{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "468af647",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import networkx as nx\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import time\n",
    "import subprocess\n",
    "import shlex\n",
    "from Bio.Blast import NCBIXML\n",
    "from collections import defaultdict\n",
    "import warnings\n",
    "from itertools import compress, count, islice\n",
    "from functools import partial\n",
    "from operator import eq\n",
    "import argparse\n",
    "import logging\n",
    "import copy\n",
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "def parseBlastXML(filePath):\n",
    "    \"\"\"\n",
    "    Convert a blast xml file to a dataframe storing all pairwise mapping information\n",
    "    \"\"\" \n",
    "    queryList, subjectList, percentIdentityList, alignmentLenList, mismatchList, gapList, qStartList, qEndList, sStartList, sEndList, evalList, bitScoreList, qSeqList, sSeqList = [],[],[],[],[],[],[],[],[],[],[],[],[],[]\n",
    "    f = open(filePath)\n",
    "    blast_records = NCBIXML.parse(f)\n",
    "    for blast_record in blast_records:\n",
    "        queryAccVer = blast_record.query[:blast_record.query.find(\" \")] if blast_record.query.find(\" \")!= -1 else blast_record.query\n",
    "        for alignment in blast_record.alignments:\n",
    "            subjectAccVer = alignment.hit_def[:alignment.hit_def.find(' ')] if alignment.hit_def.find(\" \")!= -1 else alignment.hit_def\n",
    "            for hsp in alignment.hsps:\n",
    "                percentIdentity = round(hsp.identities/hsp.align_length*100,3)\n",
    "                alignLength = hsp.align_length\n",
    "                mismatch = hsp.align_length - hsp.identities - hsp.gaps\n",
    "                gaps = hsp.gaps\n",
    "                qStart = hsp.query_start\n",
    "                qEnd = hsp.query_end\n",
    "                sStart = hsp.sbjct_start\n",
    "                sEnd = hsp.sbjct_end\n",
    "                evalue = hsp.expect\n",
    "                bitScore = int(hsp.bits)\n",
    "                qSeq = hsp.query\n",
    "                sSeq = hsp.sbjct\n",
    "                \n",
    "                queryList.append(queryAccVer)\n",
    "                subjectList.append(subjectAccVer)\n",
    "                percentIdentityList.append(percentIdentity)\n",
    "                alignmentLenList.append(alignLength)\n",
    "                mismatchList.append(mismatch)\n",
    "                gapList.append(gaps)\n",
    "                qStartList.append(qStart)\n",
    "                qEndList.append(qEnd)\n",
    "                sStartList.append(sStart)\n",
    "                sEndList.append(sEnd)\n",
    "                evalList.append(evalue)\n",
    "                bitScoreList.append(bitScore)\n",
    "                qSeqList.append(qSeq)\n",
    "                sSeqList.append(sSeq)\n",
    "    df = pd.DataFrame({'queryAccVer':queryList,'subjectAccVer':subjectList, 'identity':percentIdentityList, 'alignmentLength':alignmentLenList,\n",
    "                      'mismatches':mismatchList, 'gaps':gapList, 'qStart':qStartList,'qEnd':qEndList,'sStart':sStartList,'sEnd':sEndList,\n",
    "                      'evalue':evalList, 'bitScore':bitScoreList,'qSeq':qSeqList, 'sSeq':sSeqList})\n",
    "    return df, list(zip(qStartList, qEndList, sStartList, sEndList))\n",
    "\n",
    "def trim_fully_contain(df, check_list):\n",
    "    drop_index_list = []\n",
    "    for l in check_list:\n",
    "        qStart, qEnd, sStart, sEnd = l[0], l[1], l[2], l[3]\n",
    "        sub_df = df[(df.qStart >= qStart) & (df.qEnd <= qEnd) & (df.sStart >= sStart) &(df.sEnd <= sEnd)]\n",
    "        if sub_df.shape[0] > 1:\n",
    "            sub_df = sub_df.sort_values(by=['alignmentLength'], ascending=False)\n",
    "            overlap_index_list = list(sub_df.index.values)[1:]\n",
    "            drop_index_list += overlap_index_list\n",
    "    df = df.drop(df.index[drop_index_list]).reset_index(drop=True)\n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "id": "95c6474a",
   "metadata": {},
   "outputs": [],
   "source": [
    "def seqToBinary(seq):\n",
    "    \"\"\"\n",
    "    Bit array representation for a nuc seq; 0 means a gap, 1 means a match or mismatch.\n",
    "    \"\"\"\n",
    "    return ''.join(['0' if nuc == '-' else '1' for nuc in seq])\n",
    "\n",
    "def revComp(seq):\n",
    "    \"\"\"\n",
    "    Do the reverse complement for a given sequence; User can manually adjust a base pair reverse complement if needed.\n",
    "    \"\"\"\n",
    "    dic = {'A':'T','T':'A','C':'G','G':'C','R':'C','Y':'G','S':'G','W':'T','K':'C','M':'G','N':'A'}\n",
    "    revSeq = ''.join([dic[char] for char in seq[::-1]])\n",
    "    return revSeq\n",
    "\n",
    "def blastToDf(df, threshold, constant = 1.6446838):\n",
    "    \"\"\"\n",
    "    Input: A blast file and user preset parameter thresholds.\n",
    "    Output: A dataframe of the blast calls including two additional columns: qPair = tuple(qStart,qEnd); sPair = tuple(sStart,sEnd)\n",
    "    Convert a blast file to a dataframe after trimming according to the threshold. Given threshold is a bitscore standard that anything\n",
    "    below threshold*max_bitscore is trimmed off considered as a random match instead of a true homology.\n",
    "    \"\"\"\n",
    "    df = df.dropna()\n",
    "    #Finished reading the input blast and stored as pd\n",
    "    queryStart = list(np.array(df.qStart).astype(int))\n",
    "    queryEnd = list(np.array(df.qEnd).astype(int))\n",
    "    queryPair = list(zip(queryStart , queryEnd))\n",
    "    df = df.assign(qPair = queryPair)\n",
    "    subjectStart = list(np.array(df.sStart).astype(int))\n",
    "    subjectEnd = list(np.array(df.sEnd).astype(int))\n",
    "    subjectPair = list(zip(subjectStart,subjectEnd))\n",
    "    df = df.assign(sPair = subjectPair)\n",
    "    \n",
    "    qPair = list(df.qPair)\n",
    "    qSeq = list(df.qSeq)\n",
    "    qEdge = list(zip(qPair,qSeq))\n",
    "    df = df.assign(qEdge = qEdge)\n",
    "    sPair = list(df.sPair)\n",
    "    sSeq = list(df.sSeq)\n",
    "    sEdge = list(zip(sPair,sSeq))\n",
    "    df = df.assign(sEdge = sEdge)\n",
    "    \n",
    "    df = df[df.subjectAccVer != df.queryAccVer]\n",
    "    bitscoreThresholdList = list((np.array(df.qEnd - df.qStart)*constant+3).astype(int))\n",
    "    df = df.assign(scoreThreshold = bitscoreThresholdList)\n",
    "    df = df[df.bitScore >= threshold*df.scoreThreshold]\n",
    "    df.drop('scoreThreshold', axis=1, inplace=True)\n",
    "    df.reset_index(drop=True, inplace=True)\n",
    "    return df\n",
    "\n",
    "def union_node(df):\n",
    "    raw_bed_file = open('raw.bed','w')\n",
    "    row_num = df.shape[0]\n",
    "    \n",
    "    queryAcc = list(df.queryAccVer)\n",
    "    queryStart = list(np.array(df.qStart).astype(int))\n",
    "    queryEnd = list(np.array(df.qEnd).astype(int))\n",
    "    subjectAcc = list(df.subjectAccVer)\n",
    "    subjectStart = list(np.array(df.sStart).astype(int))\n",
    "    subjectEnd = list(np.array(df.sEnd).astype(int))\n",
    "    \n",
    "    query_entry_list = ['\\t'.join([queryAcc[i],str(np.min([queryStart[i],queryEnd[i]])),str(np.max([queryStart[i],queryEnd[i]]))]) for i in range(row_num)]\n",
    "    subject_entry_list = ['\\t'.join([subjectAcc[i],str(np.min([subjectStart[i],subjectEnd[i]])),str(np.max([subjectStart[i],subjectEnd[i]]))]) for i in range(row_num)]\n",
    "    raw_bed_file.write('\\n'.join(query_entry_list)+'\\n')\n",
    "    raw_bed_file.write('\\n'.join(subject_entry_list))\n",
    "    raw_bed_file.close()\n",
    "    \n",
    "    sort_raw_bed_command = f\"bedtools sort -i raw.bed > sort.bed\"\n",
    "    os.system(sort_raw_bed_command)\n",
    "    merge_sort_bed_commnad = f\"bedtools merge -i sort.bed > merged_sort.bed\"\n",
    "    os.system(merge_sort_bed_commnad)\n",
    "    intersect_command = 'bedtools intersect -a raw.bed -b merged_sort.bed -wo > map_node.bed'\n",
    "    os.system(intersect_command)\n",
    "    \n",
    "    map_node_df = pd.read_csv('map_node.bed', delimiter='\\t',header=None)\n",
    "    map_node_df.columns = ['id1','start1','end1','id2','start2','end2','length']\n",
    "    node_acc_list = list(map_node_df.id2)\n",
    "    node_start_list = list(np.array(map_node_df.start2).astype(int))\n",
    "    node_end_list = list(np.array(map_node_df.end2).astype(int))\n",
    "    all_node_list = list(zip(node_acc_list,zip(node_start_list,node_end_list)))\n",
    "    \n",
    "    source_node_list = all_node_list[:row_num]\n",
    "    dest_node_list = all_node_list[row_num:]\n",
    "    df['sourceNode'] = source_node_list\n",
    "    df['destNode'] = dest_node_list\n",
    "    \n",
    "    os.remove('raw.bed')\n",
    "    os.remove('sort.bed')\n",
    "    os.remove('merged_sort.bed')\n",
    "    os.remove('map_node.bed')\n",
    "\n",
    "    sPath_list = list(df['qPair'])\n",
    "    dPath_list = list(df['sPair'])\n",
    "    sourceNodeAndPath_list = list(zip(source_node_list,sPath_list))\n",
    "    destNodeAndPath_list = list(zip(dest_node_list,dPath_list))\n",
    "\n",
    "    sourceNodeAndPath_list = [(pair[0],tuple(sorted(pair[1]))) for pair in sourceNodeAndPath_list]\n",
    "    destNodeAndPath_list = [(pair[0],tuple(sorted(pair[1]))) for pair in destNodeAndPath_list]\n",
    "    map_list = list(zip(sourceNodeAndPath_list,destNodeAndPath_list))\n",
    "    map_list = [tuple(sorted((pair[0],pair[1]))) for pair in map_list]\n",
    "\n",
    "    df['align'] = map_list\n",
    "    df.drop_duplicates(subset=['align'],keep = 'first', inplace = True)\n",
    "\n",
    "    df = df.sort_values(['qStart'])\n",
    "    df.reset_index(drop=True, inplace=True)\n",
    "    \n",
    "    return df\n",
    "\n",
    "# def nth_item(n, item, iterable):\n",
    "#     indicies = compress(count(), map(partial(eq, item), iterable))\n",
    "#     return next(islice(indicies, n, None), -1)\n",
    "    \n",
    "# def choppedIndex(blockArray, offset):\n",
    "#     blockArray = seqToBinary(blockArray)\n",
    "#     if offset == 0:\n",
    "#         offset = 1\n",
    "#     try:\n",
    "#         chopIndex = nth_item(offset-1, '1', list(blockArray))\n",
    "#     except:\n",
    "#         raise ValueError(\"offset is negative\")\n",
    "\n",
    "def choppedIndex(alignment, offset):\n",
    "    basepair = 0\n",
    "    for i, base in enumerate(alignment):\n",
    "        if basepair == offset:\n",
    "            return i\n",
    "        if base != '-':\n",
    "            basepair += 1\n",
    "\n",
    "def nodePartition(position,node):\n",
    "    \"\"\"\n",
    "    Input: two intervals\n",
    "    Output: The subsequence within the chopped intervals with the landmarks flagged by the two input intervals\n",
    "    \"\"\"\n",
    "    updatedBlockList = sorted(list(set(list(position)+list(node))))\n",
    "    partitionList = [(updatedBlockList[i],updatedBlockList[i+1])for i in range(len(updatedBlockList)-1)]\n",
    "    return partitionList\n",
    "\n",
    "def multiChopNodePartition(listOfCutpoint, whole):\n",
    "    \"\"\"\n",
    "    Input: A list and an interval(tuple). The list represents the current cutpoints and the interval represents the whole length\n",
    "    Output: A list containing every single line segments(intervals) on the whole range\n",
    "    Example:\n",
    "        listOfCutpoint = [(1,10),(15,20),(25,35)]\n",
    "        whole = (1,50)\n",
    "        return [(1,10),(10,15),(15,20),(20,25),(25,35),(35,50)]\n",
    "    \"\"\"\n",
    "    updatedBlockList = list(whole)\n",
    "    for position in listOfCutpoint:\n",
    "        updatedBlockList = updatedBlockList+(list(position))\n",
    "    updatedBlockList = sorted(list(set(updatedBlockList)))\n",
    "    partitionList = [(updatedBlockList[i],updatedBlockList[i+1])for i in range(len(updatedBlockList)-1)]\n",
    "    return partitionList\n",
    "\n",
    "def partitionToTwoModules(blockNode, nucDistance, module):\n",
    "    \"\"\"\n",
    "    Input:  offset: the offset where the module should be cutted with respect to the target path\n",
    "            module: the module ready to be chopped to two parts\n",
    "            sourceDirection: the direction of the target path in the module (+ or -)\n",
    "    Output: A list containing two new modules which are subsets of the original module with the correct direction and the offset with \n",
    "            respect to each block within the original module\n",
    "    \"\"\"\n",
    "    firstPart, secondPart = nx.MultiDiGraph(), nx.MultiDiGraph()\n",
    "    seg_distance_dic = defaultdict(lambda: -1)\n",
    "    blockNode = tuple(blockNode)\n",
    "    seg_distance_dic[blockNode] = nucDistance\n",
    "    if blockNode[2] == '+':\n",
    "        first_node_dic = {blockNode:(blockNode[0],(blockNode[1][0],blockNode[1][0]+nucDistance),'+')}\n",
    "        second_node_dic = {blockNode:(blockNode[0],(blockNode[1][0]+nucDistance,blockNode[1][1]),'+')}\n",
    "    else:\n",
    "        first_node_dic = {blockNode:(blockNode[0],(blockNode[1][0]+nucDistance,blockNode[1][1]),'-')}\n",
    "        second_node_dic = {blockNode:(blockNode[0],(blockNode[1][0],blockNode[1][0]+nucDistance),'-')}\n",
    "    edgeList = list(nx.edge_bfs(module, blockNode))\n",
    "    nodeSet = set(list(module.nodes()))\n",
    "    nodeSet.discard(blockNode)\n",
    "    if not nodeSet:\n",
    "        firstPart.add_node(first_node_dic[blockNode])\n",
    "        secondPart.add_node(second_node_dic[blockNode])\n",
    "        return [firstPart,secondPart]\n",
    "    curNode = blockNode\n",
    "    curDistance = nucDistance\n",
    "    while edgeList:\n",
    "        sourceSeg, destSeg, counter = edgeList[0][0], edgeList[0][1], edgeList[0][2]\n",
    "        sourceToDestArray = module[sourceSeg][destSeg][counter]['weight']\n",
    "        destToSourceArray = module[destSeg][sourceSeg][counter]['weight']\n",
    "        sourceNode, sourceStart, sourceEnd, sourceSegDirection = sourceSeg[0], sourceSeg[1][0], sourceSeg[1][1], sourceSeg[2]\n",
    "        destNode, destStart, destEnd, destSegDirection = destSeg[0], destSeg[1][0], destSeg[1][1], destSeg[2]\n",
    "        edgeList.remove((sourceSeg,destSeg,counter))\n",
    "        edgeList.remove((destSeg,sourceSeg,counter))\n",
    "        if sourceSeg not in nodeSet and destSeg not in nodeSet:\n",
    "            continue\n",
    "        if seg_distance_dic[sourceSeg] != -1:\n",
    "            cutpoint = seg_distance_dic[sourceSeg]\n",
    "            sourceFirstNode = first_node_dic[sourceSeg]\n",
    "            sourceSecNode = second_node_dic[sourceSeg]\n",
    "            if sourceSegDirection == \"+\":\n",
    "                boundary = choppedIndex(sourceToDestArray, cutpoint)\n",
    "                source_first_array = sourceToDestArray[:boundary]\n",
    "                source_second_array = sourceToDestArray[boundary:]\n",
    "                dest_first_array = destToSourceArray[:boundary]\n",
    "                dest_second_array = destToSourceArray[boundary:]\n",
    "                destOffset = (len(dest_first_array) - dest_first_array.count('-'))\n",
    "                if destSegDirection == \"+\":\n",
    "                    dest_midpoint = destStart + destOffset\n",
    "                    if dest_midpoint > destEnd:\n",
    "                        dest_midpoint = destEnd - (len(dest_second_array) - dest_second_array.count('-'))\n",
    "                    dest_first_node = (destNode,(destStart,dest_midpoint),destSegDirection)\n",
    "                    dest_second_node = (destNode, (dest_midpoint,destEnd),destSegDirection)\n",
    "                    seg_distance_dic[destSeg] = destOffset\n",
    "                else:\n",
    "                    dest_midpoint = destEnd - destOffset\n",
    "                    if dest_midpoint < destStart:\n",
    "                        dest_midpoint = destStart + (len(dest_second_array) - dest_second_array.count('-'))\n",
    "                    dest_first_node = (destNode, (dest_midpoint,destEnd),destSegDirection)\n",
    "                    dest_second_node = (destNode,(destStart,dest_midpoint),destSegDirection)\n",
    "                    seg_distance_dic[destSeg] = dest_midpoint - destStart\n",
    "            else:\n",
    "                try:\n",
    "                    boundary = choppedIndex(sourceToDestArray[::-1], cutpoint)\n",
    "                except:\n",
    "                    raise ValueError(\"source seg not in seg dic\")\n",
    "                source_first_array = sourceToDestArray[:-boundary]\n",
    "                source_second_array = sourceToDestArray[-boundary:]\n",
    "                dest_second_array = destToSourceArray[-boundary:]\n",
    "                dest_first_array = destToSourceArray[:-boundary]\n",
    "                destOffset = (len(dest_first_array) - dest_first_array.count('-'))\n",
    "                if destSegDirection == \"+\":\n",
    "                    dest_midpoint = destStart + destOffset\n",
    "                    if dest_midpoint > destEnd:\n",
    "                        dest_midpoint = destEnd - (len(dest_second_array) - dest_second_array.count('-'))\n",
    "                    dest_first_node = (destNode,(destStart,dest_midpoint),destSegDirection)\n",
    "                    dest_second_node = (destNode, (dest_midpoint,destEnd),destSegDirection)\n",
    "                    seg_distance_dic[destSeg] = destOffset\n",
    "                else:\n",
    "                    dest_midpoint = destEnd - destOffset\n",
    "                    if dest_midpoint < destStart:\n",
    "                        dest_midpoint = destStart + (len(dest_second_array) - dest_second_array.count('-'))\n",
    "                    dest_first_node = (destNode, (dest_midpoint,destEnd),destSegDirection)\n",
    "                    dest_second_node = (destNode,(destStart,dest_midpoint),destSegDirection)\n",
    "                    seg_distance_dic[destSeg] = dest_midpoint - destStart\n",
    "            first_node_dic[destSeg] = dest_first_node\n",
    "            second_node_dic[destSeg] = dest_second_node\n",
    "            firstPart.add_edge(sourceFirstNode,dest_first_node, weight = source_first_array)\n",
    "            firstPart.add_edge(dest_first_node,sourceFirstNode, weight = dest_first_array)\n",
    "            secondPart.add_edge(sourceSecNode,dest_second_node, weight = source_second_array)\n",
    "            secondPart.add_edge(dest_second_node,sourceSecNode, weight = dest_second_array)\n",
    "            nodeSet.discard(destSeg)\n",
    "        if not nodeSet:\n",
    "            return [firstPart,secondPart]\n",
    "    raise ValueError(\"Everything is missed\")\n",
    "\n",
    "def removeOldModule(oldModule, nodeToPathDic, nodePathToModuleDic):\n",
    "    \"\"\"\n",
    "    Remove module from nodeToPathDic and nodePathToModuleDic\n",
    "    \"\"\"\n",
    "    oldModuleList = list(oldModule.nodes)\n",
    "    for (nodeName, pathTuple, direction) in oldModuleList:\n",
    "        nodeToPathDic[nodeName].discard(pathTuple)\n",
    "        nodePathToModuleDic.pop((nodeName, pathTuple),None)\n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def sanity(nodeToPathDic,nodePathToModuleDic):\n",
    "    for node in nodeToPathDic:\n",
    "        interval_list = list(sorted(nodeToPathDic[node]))\n",
    "        overlap_pairs = [(interval_list[i], interval_list[i+1]) for i in range(len(interval_list)-1) if interval_list[i+1][0]<interval_list[i][1]]\n",
    "        for tup1, tup2 in overlap_pairs:\n",
    "            tup1_len, tup2_len = tup1[1]-tup1[0], tup2[1]-tup2[0]\n",
    "            if tup1_len < tup2_len:\n",
    "                removeOldModule(nodePathToModuleDic[(node,tup1)], nodeToPathDic, nodePathToModuleDic)\n",
    "            else:\n",
    "                removeOldModule(nodePathToModuleDic[(node,tup2)], nodeToPathDic, nodePathToModuleDic)\n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic, trimLength = 20):\n",
    "    \"\"\"\n",
    "    Insert new module to nodeToPathDic and nodePathToModuleDic\n",
    "    \"\"\"\n",
    "    newModuleList = list(newModule.nodes)\n",
    "    for (nodeName, pathTuple, direction) in newModuleList:\n",
    "        if pathTuple[1] - pathTuple[0] < trimLength:\n",
    "            break\n",
    "        nodeToPathDic[nodeName].add(pathTuple)\n",
    "        nodePathToModuleDic[(nodeName, pathTuple)] = newModule\n",
    "\n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def bedtoolCall(node, nodeToPathDic, path, tempBedFile = 'bedtoolTemp.txt', tempFileA = 'tempA.bed', tempFileB = 'tempB.bed'):\n",
    "    \"\"\"\n",
    "    Call bedtool to return all segs overlapped with a given range\n",
    "    \"\"\"\n",
    "    listA = list(nodeToPathDic[node])\n",
    "    f = open(tempFileA, 'w')\n",
    "    for (start,end) in listA:\n",
    "        f.write('temp\\t'+str(int(start))+'\\t'+str(int(end))+'\\n')\n",
    "    f.close()\n",
    "    f = open(tempFileB, 'w')\n",
    "    f.write('temp\\t'+str(int(path[0]))+'\\t'+str(int(path[1])))\n",
    "    f.close()\n",
    "    command = 'bedtools intersect -a '+tempFileA+' -b '+tempFileB+' -wa'\n",
    "    process = subprocess.Popen(shlex.split(command), stdout=subprocess.PIPE)\n",
    "    output, error = process.communicate()\n",
    "    entryList = str(output, 'utf-8').split('\\n')[:-1]\n",
    "    overlappedPairs = [(node,(int(line.split('\\t')[1]),int(line.split('\\t')[2]))) for line in entryList]\n",
    "\n",
    "    return overlappedPairs\n",
    "\n",
    "def updateModuleTuple(blockNode,startOffSet,endOffSet, m_graph, destNode, destToSourcePath, sourceDirection,destDirection, sourceInModuleDirection,nodePartitionDic, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: nodePartitionDic: the new module where the node is placed. The key is the node, and the value is the module.\n",
    "            updatedModuleList: How the current module will be chopped after placing the new node. It is a list of new modules where there\n",
    "            is an internal ordering\n",
    "    Suppose already found the block to be chopped and the module which contains the block, check where to put the new node\n",
    "    Four cases in total:\n",
    "        1. The node total fits one module: add the node directly to the existing module as a new block with the correct direction \n",
    "            depending on the direction of the corresponding block\n",
    "        2. The node fits the left part of the module: chop the module to two parts first, then put the node in the first part\n",
    "        3. The node fits the right part of the module: chop the module to two parts first, then put the node in the second part\n",
    "        4. The node fits the middle part of the module: chop the module to three parts first, then put the node in the second part\n",
    "    \"\"\"\n",
    "    sourceNode = blockNode[0]\n",
    "    sourceToDestPath = (blockNode[1][0] + startOffSet, blockNode[1][0] + endOffSet)\n",
    "    block = blockNode[1]\n",
    "    sourceInModuleDirectionReverse = \"-\" if sourceInModuleDirection == \"+\" else \"+\"\n",
    "    destDirectionReverse = \"-\" if destDirection == \"+\" else \"+\"\n",
    "    destOverlapped = tuple(sorted(destToSourcePath))\n",
    "    if startOffSet == 0 and block[0] + endOffSet == block[1]:\n",
    "#       node total fit\n",
    "        new_m_graph = copy.deepcopy(m_graph)\n",
    "        if sourceInModuleDirection == sourceDirection:\n",
    "            new_m_graph.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirection), weight = sourceToDestArray)\n",
    "            new_m_graph.add_edge((destNode,destOverlapped, destDirection),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray)\n",
    "        else:\n",
    "            new_m_graph.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirectionReverse), weight = sourceToDestArray[::-1])\n",
    "            new_m_graph.add_edge((destNode,destOverlapped, destDirectionReverse),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray[::-1])\n",
    "        updatedModuleList = [new_m_graph]\n",
    "        nodePartitionDic[destOverlapped] = new_m_graph\n",
    "        return nodePartitionDic, updatedModuleList\n",
    "    \n",
    "    elif startOffSet == 0:\n",
    "#         node start fit, the single node go to the first module\n",
    "        updatedModuleList = partitionToTwoModules(blockNode, endOffSet, m_graph)\n",
    "        newModule = updatedModuleList[0] if blockNode[2] == \"+\" else updatedModuleList[1]\n",
    "        if sourceInModuleDirection == sourceDirection:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirection), weight = sourceToDestArray)\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirection),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray)\n",
    "            if blockNode[2] == \"+\":\n",
    "                updatedModuleList[0] = newModule\n",
    "            else:\n",
    "                updatedModuleList[1] = newModule\n",
    "        else:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirectionReverse), weight = sourceToDestArray[::-1])\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirectionReverse),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray[::-1])            \n",
    "            if blockNode[2] == \"+\":\n",
    "                updatedModuleList[0] = newModule\n",
    "            else:\n",
    "                updatedModuleList[1] = newModule\n",
    "        nodePartitionDic[destOverlapped] = newModule\n",
    "        return nodePartitionDic, updatedModuleList\n",
    "    \n",
    "    elif block[0] + endOffSet == block[1]:\n",
    "#       node end fit, the single node go to the second module\n",
    "        updatedModuleList = partitionToTwoModules(blockNode, startOffSet, m_graph)\n",
    "        newModule = updatedModuleList[1] if blockNode[2] == \"+\" else updatedModuleList[0]\n",
    "        if sourceInModuleDirection == sourceDirection:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirection), weight = sourceToDestArray)\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirection),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray)\n",
    "            if blockNode[2] == \"+\":\n",
    "                updatedModuleList[1] = newModule\n",
    "            else:\n",
    "                updatedModuleList[0] = newModule\n",
    "        else:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirectionReverse), weight = sourceToDestArray[::-1])\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirectionReverse),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray[::-1])            \n",
    "            if blockNode[2] == \"+\":\n",
    "                updatedModuleList[1] = newModule\n",
    "            else:\n",
    "                updatedModuleList[0] = newModule\n",
    "        nodePartitionDic[destOverlapped] = newModule\n",
    "        return nodePartitionDic, updatedModuleList\n",
    "        \n",
    "    elif startOffSet != 0 and endOffSet != 0:\n",
    "#       node fit in the middle, the single node go to the second module\n",
    "        first_module_list = partitionToTwoModules(blockNode, endOffSet, m_graph)\n",
    "        newBlockNode = (blockNode[0],(blockNode[1][0],blockNode[1][0]+endOffSet),blockNode[2])\n",
    "        targetModule = first_module_list[0] if blockNode[2] == \"+\" else first_module_list[1]\n",
    "        second_module_list = partitionToTwoModules(newBlockNode, startOffSet, targetModule)\n",
    "        if blockNode[2] == \"+\":\n",
    "            updatedModuleList = [second_module_list[0],second_module_list[1],first_module_list[1]]\n",
    "        else:\n",
    "            updatedModuleList = [second_module_list[1],second_module_list[0],first_module_list[0]]\n",
    "        newModule = updatedModuleList[1]\n",
    "        if sourceInModuleDirection == sourceDirection:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirection), weight = sourceToDestArray)\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirection),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray)\n",
    "        else:\n",
    "            newModule.add_edge((sourceNode,sourceToDestPath, sourceInModuleDirection),(destNode,destOverlapped, destDirectionReverse), weight = sourceToDestArray[::-1])\n",
    "            newModule.add_edge((destNode,destOverlapped, destDirectionReverse),(sourceNode,sourceToDestPath, sourceInModuleDirection), weight = destToSourceArray[::-1])            \n",
    "        updatedModuleList[1] = newModule\n",
    "        nodePartitionDic[destOverlapped] = newModule\n",
    "        return nodePartitionDic, updatedModuleList\n",
    "    \n",
    "def checkBlockOverlap(blockNode, m_graph, sourceNode, destNode, path, destToSourcePath, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic,nodePathToModuleDic, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: G: The directed graph after updating the new modules which contains the newly added node\n",
    "            nodePartitionDic: A dictionary containing different intervals of the original node as keys, with a module as a value for each\n",
    "            interval to indicate where the interval of the original node is placed.\n",
    "    After obtaining the nodePartitionDic and updatedModuleList from the previous function \"updateModuleTuple\", this is mainly added the\n",
    "    edges between the newly chopped modules on the directed graph and pass how the node is chopped and what the modules are for each\n",
    "    corresponding chopped intervals.\n",
    "    \"\"\"\n",
    "    oldModuleList = list(m_graph.nodes)\n",
    "    small , big = path[0], path[1]\n",
    "    if big-small < 20:\n",
    "        return nodeToPathDic, nodePathToModuleDic, nodePartitionDic, [m_graph]\n",
    "    node = blockNode[0]\n",
    "    block = blockNode[1]\n",
    "    direction = blockNode[2]\n",
    "    start, end = block[0], block[1]\n",
    "    if small >= start and big <= end:\n",
    "        startOffSet = int(small - start)\n",
    "        endOffSet = int(big - start)\n",
    "        nodePartitionDic, updatedModuleList = updateModuleTuple(blockNode,startOffSet,endOffSet, m_graph, destNode, destToSourcePath,sourceDirection,destDirection,direction, nodePartitionDic, sourceToDestArray, destToSourceArray)\n",
    "        if len(updatedModuleList) == 1:\n",
    "            newModule = updatedModuleList[0]\n",
    "            nodeToPathDic[destNode].add(tuple(sorted(destToSourcePath)))\n",
    "            for (nodeName, pathTuple, direction) in list(newModule.nodes):\n",
    "                nodePathToModuleDic[(nodeName, pathTuple)] = newModule\n",
    "        else:\n",
    "            numberOfNewModules = len(updatedModuleList)\n",
    "            for (nodeName, pathTuple, direction) in oldModuleList:\n",
    "                nodeToPathDic[nodeName].discard(pathTuple)\n",
    "                nodePathToModuleDic.pop((nodeName, pathTuple),None)\n",
    "            for moduleIndex in range(numberOfNewModules):\n",
    "                newModule = updatedModuleList[moduleIndex]\n",
    "                for (nodeName, pathTuple, direction) in list(newModule.nodes):\n",
    "                    if pathTuple[1] - pathTuple[0] < 20:\n",
    "                        break\n",
    "                    nodeToPathDic[nodeName].add(pathTuple)\n",
    "                    nodePathToModuleDic[(nodeName, pathTuple)] = newModule\n",
    "        return nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList\n",
    "\n",
    "def recursiveModuleVSNodeChecking(listOfModules, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic,nodePathToModuleDic, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: G: updated directed graph for module partition\n",
    "            nodePartitionDic: A dictionary containing different intervals of the original node as keys, with a module as a value for each\n",
    "            interval to indicate where the interval of the original node is placed.\n",
    "    This is a recursive function which recursively checking whether the node has been finished adding to the current modules.\n",
    "    It is isolated because it is recursive.\n",
    "    \"\"\"\n",
    "    sourcePathStart, sourcePathEnd = sourceToDestPath[0], sourceToDestPath[1]\n",
    "    destPathStart, destPathEnd = destToSourcePath[0], destToSourcePath[1]\n",
    "    connectedModulesLength = len(listOfModules)\n",
    "    if sourcePathEnd - sourcePathStart < 20:\n",
    "        return nodeToPathDic, nodePathToModuleDic, nodePartitionDic\n",
    "    for i in range(connectedModulesLength):\n",
    "        m_graph = listOfModules[i]\n",
    "        aModule = list(m_graph.nodes)\n",
    "        moduleToDf = pd.DataFrame(aModule, columns =['Node', 'Path', 'Direction'])\n",
    "        moduleToDf = moduleToDf[moduleToDf.Node == sourceNode]\n",
    "        pairList = list(moduleToDf['Path'])\n",
    "        start = [x[0] for x in pairList]\n",
    "        end = [x[1] for x in pairList]\n",
    "        moduleToDf = moduleToDf.assign(Start = start)\n",
    "        moduleToDf = moduleToDf.assign(End = end)\n",
    "        shouldContinue1 = moduleToDf[sourcePathStart <= moduleToDf.Start][sourcePathEnd <= moduleToDf.Start]\n",
    "        shouldContinue2 = moduleToDf[sourcePathStart >= moduleToDf.End][sourcePathEnd >= moduleToDf.End]\n",
    "        qualifiedDf = moduleToDf[~moduleToDf.index.isin(shouldContinue1.index.union(shouldContinue2.index))]\n",
    "        moduleList = qualifiedDf.values.tolist()\n",
    "        for aBlock in moduleList:\n",
    "            aBlock = aBlock[:3]\n",
    "            node = aBlock[0]\n",
    "            block = aBlock[1]\n",
    "            blockStart = block[0]\n",
    "            blockEnd = block[1]\n",
    "            blockDirection = aBlock[2]\n",
    "            if blockStart <= sourcePathStart and sourcePathEnd <= blockEnd:\n",
    "                # node total fit a given range\n",
    "                nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock, m_graph, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection, destDirection, nodePartitionDic, nodeToPathDic,nodePathToModuleDic, sourceToDestArray, destToSourceArray)\n",
    "    \n",
    "            elif sourcePathStart <= blockStart and sourcePathEnd <= blockEnd:\n",
    "                # node head overlaps but tail surpasses the range\n",
    "                offset = int(blockStart - sourcePathStart)\n",
    "                boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                leftOverDestArray = destToSourceArray[:boundary]\n",
    "                correspondingDestArray = destToSourceArray[boundary:]\n",
    "                if sourceDirection == destDirection:\n",
    "                    newLeft = destToSourcePath[0] + (len(list(leftOverDestArray)) - list(leftOverDestArray).count('-'))\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, (blockStart,sourcePathEnd), (newLeft,destToSourcePath[1]), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "                    residue = (destToSourcePath[0], newLeft)\n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockStart))\n",
    "                        leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                        correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                        leftOverDestArray = destToSourceArray[boundary:]\n",
    "                        correspondingDestArray = destToSourceArray[:boundary]\n",
    "                    newRight = destToSourcePath[1] - (len(list(leftOverDestArray)) - list(leftOverDestArray).count('-'))\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, (blockStart,sourcePathEnd), (destToSourcePath[0],newRight), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "                    residue = (newRight,destToSourcePath[1])\n",
    "                modulesToBeVisited = updatedModuleList + listOfModules[i+1:]\n",
    "                nodeToPathDic, nodePathToModuleDic, nodePartitionDic = recursiveModuleVSNodeChecking(modulesToBeVisited, sourceNode, destNode, (int(sourcePathStart),int(blockStart)), residue, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, leftOverSourceArray, leftOverDestArray)\n",
    "\n",
    "            elif sourcePathStart >= blockStart and sourcePathEnd >= blockEnd:\n",
    "                # node tail overlaps but head surpasses the range\n",
    "                offset = int(blockEnd - sourcePathStart)\n",
    "                boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                correspondingDestArray = destToSourceArray[:boundary]\n",
    "                leftOverDestArray = destToSourceArray[boundary:]\n",
    "                if sourceDirection == destDirection:\n",
    "                    newRight = destToSourcePath[0] + (len(list(correspondingDestArray)) - list(correspondingDestArray).count('-'))\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, (sourcePathStart,blockEnd), (destToSourcePath[0],newRight), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "                    residue = (newRight, destToSourcePath[1])\n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockEnd))\n",
    "                        leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                        correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                        leftOverDestArray = destToSourceArray[:boundary]\n",
    "                        correspondingDestArray = destToSourceArray[boundary:]\n",
    "                    newLeft = destToSourcePath[1] - (len(list(correspondingDestArray)) - list(correspondingDestArray).count('-'))\n",
    "                    # The head is overlapping with an already existed module, truncate the head\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, (sourcePathStart,blockEnd), (newLeft,destToSourcePath[1]), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "                    residue = (destToSourcePath[0], newLeft)\n",
    "                modulesToBeVisited = updatedModuleList + listOfModules[i+1:]\n",
    "                nodeToPathDic, nodePathToModuleDic, nodePartitionDic = recursiveModuleVSNodeChecking(modulesToBeVisited, sourceNode, destNode, (int(blockEnd),int(sourcePathEnd)), residue, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, leftOverSourceArray, leftOverDestArray)\n",
    "\n",
    "            elif sourcePathStart <= blockStart and sourcePathEnd >= blockEnd:\n",
    "                # Both head and tail surpasses the given range\n",
    "                offset1 = int(blockStart - sourcePathStart)\n",
    "                offset2 = int(blockEnd - sourcePathStart)\n",
    "                boundary1 = choppedIndex(sourceToDestArray, offset1)\n",
    "                boundary2 = choppedIndex(sourceToDestArray, offset2)\n",
    "                leftSourceArray = sourceToDestArray[:boundary1]\n",
    "                midSourceArray = sourceToDestArray[boundary1:boundary2]\n",
    "                rightSourceArray = sourceToDestArray[boundary2:]\n",
    "                leftDestArray = destToSourceArray[:boundary1]\n",
    "                midDestArray = destToSourceArray[boundary1:boundary2]\n",
    "                rightDestArray = destToSourceArray[boundary2:]\n",
    "                if sourceDirection == destDirection:\n",
    "                    newLeft = destToSourcePath[0] + (len(list(leftDestArray)) - list(leftDestArray).count('-'))\n",
    "                    newRight = newLeft + (len(list(midDestArray)) - list(midDestArray).count('-'))\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, block, (newLeft,newRight), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, midSourceArray, midDestArray)\n",
    "                    residue1 = (destToSourcePath[0], newLeft)\n",
    "                    residue2 = (newRight, destToSourcePath[1])\n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary1 = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockEnd))\n",
    "                        boundary2 = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockStart))\n",
    "                        rightSourceArray = sourceToDestArray[:boundary1]\n",
    "                        midSourceArray = sourceToDestArray[boundary1:boundary2]\n",
    "                        leftSourceArray = sourceToDestArray[boundary2:]\n",
    "                        rightDestArray = destToSourceArray[:boundary1]\n",
    "                        midDestArray = destToSourceArray[boundary1:boundary2]\n",
    "                        leftDestArray = destToSourceArray[boundary2:]\n",
    "                    newRight = destToSourcePath[1] - (len(list(leftDestArray)) - list(leftDestArray).count('-'))\n",
    "                    newLeft = newRight - (len(list(midDestArray)) - list(midDestArray).count('-'))\n",
    "                    nodeToPathDic, nodePathToModuleDic, nodePartitionDic, updatedModuleList = checkBlockOverlap(aBlock,m_graph, sourceNode, destNode, block,\n",
    "                                                            (newLeft,newRight), sourceDirection, destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, midSourceArray, midDestArray)\n",
    "                    residue1 = (newRight, destToSourcePath[1])\n",
    "                    residue2 = (destToSourcePath[0], newLeft)\n",
    "\n",
    "                modulesToBeVisited = updatedModuleList + listOfModules[i+1:]\n",
    "                nodeToPathDic, nodePathToModuleDic, nodePartitionDic = recursiveModuleVSNodeChecking(modulesToBeVisited, sourceNode, destNode, (int(sourcePathStart) ,int(blockStart)),\n",
    "                                                                                                     residue1, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, leftSourceArray, leftDestArray)\n",
    "\n",
    "                overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, (int(blockEnd), int(sourcePathEnd)))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                nodeToPathDic, nodePathToModuleDic, nodePartitionDic = recursiveModuleVSNodeChecking(modulesToBeVisited, sourceNode, destNode, (int(blockEnd), int(sourcePathEnd)),residue2, sourceDirection,destDirection, nodePartitionDic, nodeToPathDic, nodePathToModuleDic, rightSourceArray, rightDestArray)\n",
    "\n",
    "            return nodeToPathDic, nodePathToModuleDic, nodePartitionDic\n",
    "    return nodeToPathDic, nodePathToModuleDic, nodePartitionDic\n",
    "    \n",
    "def nodeModulePartition(sourceNode, destNode, sourceToDestPath, destToSourcePath,sourceDirection,destDirection, nodeToPathDic,nodePathToModuleDic, tempBedFile, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: The updated directed graph\n",
    "    Find the start where modules need to be updated, and call for the chain of commands to check whether the node overlaps \n",
    "    with current modules and update new modules\n",
    "    \"\"\"\n",
    "    overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, sourceToDestPath, tempBedFile)\n",
    "    connectedModules = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "    nodeToPathDicCopy = copy.deepcopy(nodeToPathDic)\n",
    "    nodePathToModuleDicCopy = copy.deepcopy(nodePathToModuleDic)\n",
    "    try:\n",
    "        nodeToPathDic, nodePathToModuleDic, nodePartitionDic = recursiveModuleVSNodeChecking(connectedModules, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection,destDirection, {}, nodeToPathDic,nodePathToModuleDic, sourceToDestArray, destToSourceArray)\n",
    "    except:\n",
    "        nodeToPathDic, nodePathToModuleDic = nodeToPathDicCopy, nodePathToModuleDicCopy\n",
    "        nodeToPathDic, nodePathToModuleDic = sanity(nodeToPathDic, nodePathToModuleDic)\n",
    "                \n",
    "        return nodeToPathDic,nodePathToModuleDic\n",
    "\n",
    "    destNodeBlocks = multiChopNodePartition(list(nodePartitionDic.keys()),destNode[1])\n",
    "    for element in destNodeBlocks:\n",
    "        if element not in nodePartitionDic:\n",
    "            nodeToPathDic[destNode].add(element)\n",
    "            newM_graph = nx.MultiDiGraph()\n",
    "            newM_graph.add_node((destNode, element, \"+\"),)\n",
    "            nodePathToModuleDic[(destNode,element)] = newM_graph\n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def trimShortModules(nodeToPathDic,nodePathToModuleDic, trimLength = 10):\n",
    "    for node in nodeToPathDic.keys():\n",
    "        pathList = list(nodeToPathDic[node])\n",
    "        lengthList = [int(end)-int(start) for (start,end) in pathList]\n",
    "        for (path,length) in zip(pathList,lengthList):\n",
    "            if length < trimLength:\n",
    "                nodeToPathDic[node].discard(path)\n",
    "                nodePathToModuleDic.pop((node, path),None)\n",
    "    return nodeToPathDic,nodePathToModuleDic\n",
    "\n",
    "def signReverse(module):\n",
    "    \"\"\"\n",
    "    Input: A module graph\n",
    "    Output: A module graph with sign reversed on nodes and bit array.\n",
    "    \"\"\"\n",
    "    reverseSign = nx.MultiDiGraph()\n",
    "    nodeMapping = {}\n",
    "    newNodes = []\n",
    "    for node in list(module.nodes):\n",
    "        newNode = list(node)[:]\n",
    "        newNode[2] = \"+\" if newNode[2] == \"-\" else \"-\"\n",
    "        newNode = tuple(newNode)\n",
    "        nodeMapping[node] = newNode\n",
    "        newNodes.append(newNode)\n",
    "    reverseSign.add_nodes_from(newNodes)\n",
    "    for edge in list(module.edges):\n",
    "        s = edge[0]\n",
    "        d = edge[1]\n",
    "        i = edge[2]\n",
    "        reverseSign.add_edge(nodeMapping[s],nodeMapping[d],i,weight = module[s][d][i]['weight'][::-1])\n",
    "    return reverseSign\n",
    "\n",
    "def reverseModuleOnDirection(node, path, direction, module):\n",
    "    \"\"\"\n",
    "    Output: Depending on the direction of the target block in the module, return either the module, or the module which reverses signs\n",
    "    Given the target block and its direction in the blast call, find its direction within the module to depend the further step whether\n",
    "    to keep the module as it is, or reverse the sign for each block.\n",
    "    \"\"\"\n",
    "    for aBlock in list(module.nodes):\n",
    "        blockNode = aBlock[0]\n",
    "        block = aBlock[1]\n",
    "        blockDirection = aBlock[2]\n",
    "        if node == blockNode:\n",
    "            if path[0] in range(block[0]-10,block[0]+10) and (path[1] in range(block[1]-30,block[1]+30)):\n",
    "                if direction == blockDirection:\n",
    "                    return module\n",
    "                else:\n",
    "                    return signReverse(module)\n",
    "\n",
    "def joinTwoModules(moduleA, moduleB, nodeA, pathA, aDirection, nodeB, pathB, bDirection, arrayA, arrayB):\n",
    "    \"\"\"\n",
    "    Output: A 'union' module by joining two modules\n",
    "    According to one block from each module, there are four directions that matter: the mapping directions between two blocks, and the \n",
    "    directions for two block in their module respectively. \n",
    "    \"\"\"\n",
    "    moduleUnion = copy.deepcopy(moduleA)\n",
    "    aModuleList = list(moduleA.nodes)\n",
    "    bModuleList = list(moduleB.nodes)\n",
    "    aDirectionInModuleA, bDirectionInModuleB = None, None\n",
    "    targetABlock, targetBBlock =None, None\n",
    "    for block in moduleA:\n",
    "        if block[0] == nodeA and block[1] == pathA:\n",
    "            aDirectionInModuleA = block[2]\n",
    "            targetABlock = block\n",
    "    for block in moduleB:\n",
    "        if block[0] == nodeB and block[1] == pathB:\n",
    "            bDirectionInModuleB = block[2]\n",
    "            targetBBlock = block\n",
    "    aNodeBlockSet = set([(block[0],block[1]) for block in aModuleList])\n",
    "    bNodeBlockSet = set([(block[0],block[1]) for block in bModuleList])\n",
    "    leftOverList = bNodeBlockSet.difference(aNodeBlockSet)\n",
    "    leftOverBlocks = []\n",
    "    \n",
    "    for bBlock in bModuleList:\n",
    "        if (bBlock[0],bBlock[1]) in leftOverList:\n",
    "            leftOverBlocks.append(bBlock)\n",
    "    \n",
    "    if leftOverBlocks:\n",
    "        moduleB_unique = moduleB.subgraph(leftOverBlocks)\n",
    "        if aDirection == bDirection:\n",
    "            if aDirectionInModuleA == bDirectionInModuleB:\n",
    "                moduleUnion = nx.compose(moduleUnion,moduleB_unique)\n",
    "                if aDirection == aDirectionInModuleA:\n",
    "                    moduleUnion.add_edge(targetABlock, targetBBlock, weight = arrayA)\n",
    "                    moduleUnion.add_edge(targetBBlock, targetABlock, weight = arrayB)\n",
    "                else:\n",
    "                    moduleUnion.add_edge(targetABlock, targetBBlock, weight = arrayA[::-1])\n",
    "                    moduleUnion.add_edge(targetBBlock, targetABlock, weight = arrayB[::-1])\n",
    "            else:\n",
    "                moduleUnion = nx.compose(moduleUnion,signReverse(moduleB_unique))\n",
    "                if aDirection == aDirectionInModuleA:\n",
    "                    moduleUnion.add_edge(targetABlock, (targetBBlock[0],targetBBlock[1],aDirectionInModuleA), weight = arrayA)\n",
    "                    moduleUnion.add_edge((targetBBlock[0],targetBBlock[1],aDirectionInModuleA), targetABlock, weight = arrayB)\n",
    "                else:\n",
    "                    moduleUnion.add_edge(targetABlock, (targetBBlock[0],targetBBlock[1],aDirectionInModuleA), weight = arrayA[::-1])\n",
    "                    moduleUnion.add_edge((targetBBlock[0],targetBBlock[1],aDirectionInModuleA), targetABlock, weight = arrayB[::-1])\n",
    "        else:\n",
    "            if aDirectionInModuleA == bDirectionInModuleB:\n",
    "                moduleUnion = nx.compose(moduleUnion,signReverse(moduleB_unique))\n",
    "                if aDirection == aDirectionInModuleA:\n",
    "                    moduleUnion.add_edge(targetABlock, (targetBBlock[0],targetBBlock[1],bDirection), weight = arrayA)\n",
    "                    moduleUnion.add_edge((targetBBlock[0],targetBBlock[1],bDirection), targetABlock, weight = arrayB)\n",
    "                else:\n",
    "                    moduleUnion.add_edge(targetABlock, (targetBBlock[0],targetBBlock[1],aDirection), weight = arrayA[::-1])\n",
    "                    moduleUnion.add_edge((targetBBlock[0],targetBBlock[1],aDirection), targetABlock, weight = arrayB[::-1])\n",
    "            else:\n",
    "                moduleUnion = nx.compose(moduleUnion,moduleB_unique)\n",
    "                if aDirection == aDirectionInModuleA:\n",
    "                    moduleUnion.add_edge(targetABlock, targetBBlock, weight = arrayA)\n",
    "                    moduleUnion.add_edge(targetBBlock, targetABlock, weight = arrayB)\n",
    "                else:\n",
    "                    moduleUnion.add_edge(targetABlock, targetBBlock, weight = arrayA[::-1])\n",
    "                    moduleUnion.add_edge(targetBBlock, targetABlock, weight = arrayB[::-1])\n",
    "    \n",
    "    return moduleUnion\n",
    "\n",
    "def chopModulesAndUpdateGraph(listOfModules,node, path, direction,currentPartitionBlockDic,currentModuleUpdateDic,nodeToPathDic,nodePathToModuleDic):\n",
    "    \"\"\"\n",
    "    Output: currentPartitionBlockDic: How one of the module is partitioned. The key is part of the mapping path of the blast call, and\n",
    "            the value is the module that that part is currently in\n",
    "            currentModuleUpdateDic: After adding the new pairwise path, how the current modules will be partitioned.\n",
    "    Treating a module as a node and see how it chops the current modules.\n",
    "    \"\"\"\n",
    "    pathStart = path[0]\n",
    "    pathEnd = path[1]\n",
    "    connectedModulesLength = len(listOfModules)\n",
    "    for i in range(connectedModulesLength):\n",
    "        m_graph = listOfModules[i]\n",
    "        aModule = list(m_graph.nodes)\n",
    "        moduleToDf = pd.DataFrame(aModule, columns =['Node', 'Path', 'Direction'])\n",
    "        moduleToDf = moduleToDf[moduleToDf.Node == node]\n",
    "        if moduleToDf.shape[0] == 0:\n",
    "            continue\n",
    "        pairList = list(moduleToDf['Path'])\n",
    "        start = [x[0] for x in pairList]\n",
    "        end = [x[1] for x in pairList]\n",
    "        moduleToDf = moduleToDf.assign(Start = start)\n",
    "        moduleToDf = moduleToDf.assign(End = end)\n",
    "        shouldContinue1 = moduleToDf[pathStart <= moduleToDf.Start][pathEnd <= moduleToDf.Start]\n",
    "        shouldContinue2 = moduleToDf[pathStart >= moduleToDf.End][pathEnd >= moduleToDf.End]\n",
    "        qualifiedDf = moduleToDf[~moduleToDf.index.isin(shouldContinue1.index.union(shouldContinue2.index))]\n",
    "        moduleList = qualifiedDf.values.tolist()\n",
    "        for aBlock in moduleList:\n",
    "            aBlock = aBlock[:3]\n",
    "            sourceNode = aBlock[0]\n",
    "            block = aBlock[1]\n",
    "            blockStart = block[0]\n",
    "            blockEnd = block[1]\n",
    "            blockDirection = aBlock[2]\n",
    "            if blockStart <= pathStart and pathEnd <= blockEnd:\n",
    "                leftOffset = pathStart - blockStart\n",
    "                rightOffset = pathEnd - blockStart\n",
    "                if pathStart == blockStart and pathEnd == blockEnd:\n",
    "                    currentPartitionBlockDic[(node,path,direction)] = m_graph\n",
    "                    return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "                elif pathStart == blockStart:\n",
    "                    updatedModuleList = partitionToTwoModules(aBlock, rightOffset, m_graph)\n",
    "                    currentPartitionBlockDic[(node,path,direction)] = updatedModuleList[0] if blockDirection == \"+\" else updatedModuleList[1]\n",
    "                    currentModuleUpdateDic[m_graph] = updatedModuleList\n",
    "                    return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "                elif pathEnd == blockEnd:\n",
    "                    updatedModuleList = partitionToTwoModules(aBlock, leftOffset, m_graph)\n",
    "                    currentPartitionBlockDic[(node,path,direction)] = updatedModuleList[1] if blockDirection == \"+\" else updatedModuleList[0]\n",
    "                    currentModuleUpdateDic[m_graph] = updatedModuleList\n",
    "                    return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "                else:\n",
    "                    first_module_list = partitionToTwoModules(aBlock, rightOffset, m_graph)\n",
    "                    newBlockNode = (sourceNode,(blockStart,blockStart+rightOffset),blockDirection)\n",
    "                    targetModule = first_module_list[0] if blockDirection == \"+\" else first_module_list[1]\n",
    "                    second_module_list = partitionToTwoModules(newBlockNode, leftOffset, targetModule)\n",
    "                    if blockDirection == \"+\":\n",
    "                        updatedModuleList = [second_module_list[0],second_module_list[1],first_module_list[1]]\n",
    "                    else:\n",
    "                        updatedModuleList = [second_module_list[1],second_module_list[0],first_module_list[0]]\n",
    "                    currentPartitionBlockDic[(node,path,direction)] = updatedModuleList[1]\n",
    "                    currentModuleUpdateDic[m_graph] = updatedModuleList\n",
    "                    return currentPartitionBlockDic, currentModuleUpdateDic    \n",
    "            elif pathStart >= blockStart and pathEnd >= blockEnd:\n",
    "                if pathStart-blockStart > 0 and blockEnd - pathStart != 0:\n",
    "                    updatedModuleList = partitionToTwoModules(aBlock, int(pathStart-blockStart), m_graph)\n",
    "                    currentPartitionBlockDic[(node,(pathStart,blockEnd),direction)] = updatedModuleList[1] if blockDirection == \"+\" else updatedModuleList[0]\n",
    "                    if pathStart != blockEnd:\n",
    "                        currentModuleUpdateDic[m_graph] = updatedModuleList\n",
    "                else:\n",
    "                    currentPartitionBlockDic[(node,(pathStart,blockEnd),direction)] = m_graph\n",
    "                currentPartitionBlockDic, currentModuleUpdateDic = chopModulesAndUpdateGraph(listOfModules[i:],node,(blockEnd,pathEnd),direction,currentPartitionBlockDic,currentModuleUpdateDic, nodeToPathDic,nodePathToModuleDic)\n",
    "                return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "\n",
    "            elif pathStart <= blockStart and pathEnd <= blockEnd:\n",
    "                if pathEnd - blockStart > 0 and blockEnd - pathEnd != 0:\n",
    "                    updatedModuleList = partitionToTwoModules(aBlock, int(pathEnd-blockStart), m_graph)\n",
    "                    currentPartitionBlockDic[(node,(blockStart,pathEnd),direction)] = updatedModuleList[0] if blockDirection == \"+\" else updatedModuleList[1]\n",
    "                    if pathEnd != blockEnd:\n",
    "                        currentModuleUpdateDic[m_graph] = updatedModuleList\n",
    "                else: \n",
    "                    currentPartitionBlockDic[(node,(blockStart,pathEnd),direction)] = m_graph\n",
    "\n",
    "                currentPartitionBlockDic, currentModuleUpdateDic = chopModulesAndUpdateGraph(listOfModules[i:],node,(pathStart,blockStart),direction,currentPartitionBlockDic,currentModuleUpdateDic, nodeToPathDic,nodePathToModuleDic)\n",
    "                return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "\n",
    "            elif pathStart <= blockStart and pathEnd >= blockEnd:\n",
    "                currentPartitionBlockDic[(node,(blockStart,blockEnd),direction)] = m_graph   \n",
    "                currentPartitionBlockDic, currentModuleUpdateDic = chopModulesAndUpdateGraph(listOfModules[i:],node,(pathStart,blockStart),direction,currentPartitionBlockDic,currentModuleUpdateDic, nodeToPathDic,nodePathToModuleDic)\n",
    "                overlappedPairs = bedtoolCall(node, nodeToPathDic, (blockEnd,pathEnd))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                currentPartitionBlockDic, currentModuleUpdateDic = chopModulesAndUpdateGraph(modulesToBeVisited,node,(blockEnd,pathEnd),direction,currentPartitionBlockDic,currentModuleUpdateDic, nodeToPathDic,nodePathToModuleDic)\n",
    "                return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "    return currentPartitionBlockDic, currentModuleUpdateDic\n",
    "\n",
    "def updateModuleModuleTuple(sourceNode, sourceToDestPath, block, startOffSet,endOffSet, source_m_graph, destNode, destToSourcePath,sourceDirection,destDirection, blockDirection, dest_m_graph, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: module: The new module that is formed after merging two original modules\n",
    "            updatedModuleList: The newly chopped modules which should be connected with each other\n",
    "    \"\"\"\n",
    "    if startOffSet == 0 and block[0] + endOffSet == block[1]:\n",
    "        module = joinTwoModules(source_m_graph, dest_m_graph,sourceNode, sourceToDestPath, sourceDirection, destNode, destToSourcePath, destDirection, sourceToDestArray, destToSourceArray)\n",
    "        updatedModuleList = [module]\n",
    "        return module, updatedModuleList\n",
    "    elif startOffSet == 0:\n",
    "#       the single node go to the first module\n",
    "        updatedModuleList = partitionToTwoModules((sourceNode,block,blockDirection), endOffSet, source_m_graph)\n",
    "        newModule = updatedModuleList[0] if blockDirection == \"+\" else updatedModuleList[1]\n",
    "        newModule = joinTwoModules(newModule, dest_m_graph, sourceNode, sourceToDestPath, sourceDirection, destNode, destToSourcePath, destDirection, sourceToDestArray, destToSourceArray)\n",
    "        if blockDirection == \"+\":\n",
    "            updatedModuleList[0] = newModule\n",
    "        else:\n",
    "            updatedModuleList[1] = newModule\n",
    "        return newModule,updatedModuleList\n",
    "    \n",
    "    elif block[0] + endOffSet == block[1]:\n",
    "#       the single node go to the second module\n",
    "        updatedModuleList = partitionToTwoModules((sourceNode,block,blockDirection), startOffSet, source_m_graph)\n",
    "        newModule = updatedModuleList[1] if blockDirection == \"+\" else updatedModuleList[0]\n",
    "        newModule = joinTwoModules(newModule, dest_m_graph,sourceNode, sourceToDestPath, sourceDirection, destNode, destToSourcePath, destDirection, sourceToDestArray, destToSourceArray)\n",
    "        if blockDirection == \"+\":\n",
    "            updatedModuleList[1] = newModule\n",
    "        else:\n",
    "            updatedModuleList[0] = newModule\n",
    "        return newModule,updatedModuleList\n",
    "    \n",
    "    elif startOffSet != 0 and endOffSet != 0:\n",
    "#       the single node go to the second module\n",
    "        first_module_list = partitionToTwoModules((sourceNode,block,blockDirection), endOffSet, source_m_graph)\n",
    "        newBlockNode = (sourceNode,(block[0],block[0]+endOffSet),blockDirection)\n",
    "        targetModule = first_module_list[0] if blockDirection == \"+\" else first_module_list[1]\n",
    "        second_module_list = partitionToTwoModules(newBlockNode, startOffSet, targetModule)\n",
    "        if blockDirection == \"+\":\n",
    "            updatedModuleList = [second_module_list[0],second_module_list[1],first_module_list[1]]\n",
    "        else:\n",
    "            updatedModuleList = [second_module_list[1],second_module_list[0],first_module_list[0]]\n",
    "        newModule = updatedModuleList[1]\n",
    "        newModule = joinTwoModules(newModule, dest_m_graph,\n",
    "                                sourceNode, sourceToDestPath, sourceDirection, \n",
    "                                destNode, destToSourcePath, destDirection, sourceToDestArray, destToSourceArray)\n",
    "        return newModule,updatedModuleList\n",
    "    \n",
    "def checkModuleModuleOverlap(blockNode, source_m_graph, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection, destDirection,dest_m_graph, sourceToDestArray, destToSourceArray,nodeToPathDic, nodePathToModuleDic):\n",
    "    \"\"\"\n",
    "    Output: G: The directed graph after updating new modules and linking the edges between chopped original modules\n",
    "    This function helps to see how the target block is chopped and what the offsets are in the current module set\n",
    "    \"\"\"\n",
    "    small , big = sourceToDestPath[0], sourceToDestPath[1]\n",
    "    node = blockNode[0]\n",
    "    block = blockNode[1]\n",
    "    direction = blockNode[2]\n",
    "    start, end = block[0], block[1]\n",
    "    if small >= start and big <= end:\n",
    "        startOffSet = int(small - start)\n",
    "        endOffSet = int(big - start)\n",
    "        newModule, updatedModuleList = updateModuleModuleTuple(node, sourceToDestPath, block, startOffSet,endOffSet, source_m_graph, destNode, destToSourcePath,sourceDirection,destDirection, direction,dest_m_graph, sourceToDestArray, destToSourceArray)\n",
    "        oldSourceModule = source_m_graph\n",
    "        oldDestModule = dest_m_graph\n",
    "        nodeToPathDic, nodePathToModuleDic = removeOldModule(oldSourceModule, nodeToPathDic, nodePathToModuleDic)\n",
    "        nodeToPathDic, nodePathToModuleDic = removeOldModule(oldDestModule, nodeToPathDic, nodePathToModuleDic)\n",
    "        numberOfNewModules = len(updatedModuleList)\n",
    "        for moduleIndex in range(numberOfNewModules):\n",
    "            newModule = updatedModuleList[moduleIndex]\n",
    "            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "        return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def recursiveModuleVSModuleChecking(listOfModules, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection,destDirection, dest_m_graph, nodeToPathDic, nodePathToModuleDic, sourceToDestArray, destToSourceArray):\n",
    "    \"\"\"\n",
    "    Output: G: The directed graph after updating new modules and linking the edges between chopped original modules\n",
    "    This is a recursive function that keeps updating the unmapped subsequences.\n",
    "    Also, anthoer checking that happens within this function is to check if the current block exists in more than one current module.\n",
    "    If this is the case, there are two ways to handle:\n",
    "    1.  If the overlapped region with one of the module is less than 1co00bp, simply ignore the part, which makes the case to be a mapping\n",
    "        between two subsequences within two current modules.\n",
    "    2.  If the overlapped region with one of the module is less than 100bp, simply keep the original module partitons and continue to\n",
    "        the next blast mapping pairs.\n",
    "    \"\"\"\n",
    "    sourcePathStart, sourcePathEnd = sourceToDestPath[0], sourceToDestPath[1]\n",
    "    destPathStart, destPathEnd = destToSourcePath[0], destToSourcePath[1]\n",
    "    connectedModulesLength = len(listOfModules)\n",
    "    correctDirectionModule = reverseModuleOnDirection(destNode, destToSourcePath, destDirection, dest_m_graph)\n",
    "    dest_block_direction = destDirection\n",
    "    if correctDirectionModule is None:\n",
    "        return nodeToPathDic, nodePathToModuleDic\n",
    "    if sourcePathEnd - sourcePathStart < 20:\n",
    "        return nodeToPathDic, nodePathToModuleDic\n",
    "    if not dest_m_graph in set(nodePathToModuleDic.values()):\n",
    "        return nodeToPathDic, nodePathToModuleDic\n",
    "    for i in range(connectedModulesLength):\n",
    "        source_m_graph = listOfModules[i]\n",
    "        module = list(source_m_graph.nodes)\n",
    "        if tuple(dest_m_graph) == module:\n",
    "            return nodeToPathDic, nodePathToModuleDic\n",
    "        moduleDf = pd.DataFrame(module, columns =['Node', 'Path', 'Direction']) \n",
    "        pairList = list(moduleDf['Path'])\n",
    "        start = [x[0] for x in pairList]\n",
    "        end = [x[1] for x in pairList]\n",
    "        moduleDf = moduleDf.assign(Start = start)\n",
    "        moduleDf = moduleDf.assign(End = end)\n",
    "        sourceFamilyDf = moduleDf[moduleDf.Node == sourceNode][moduleDf.Start <= sourceToDestPath[0]][moduleDf.End >= sourceToDestPath[1]]\n",
    "        sourceNotFromFamily =  sourceFamilyDf.empty\n",
    "        destFamilyDf = moduleDf[moduleDf.Node == destNode][moduleDf.Start <= destToSourcePath[0]][moduleDf.End >= destToSourcePath[1]]\n",
    "        destNotFromFamily =  destFamilyDf.empty\n",
    "        if (not sourceNotFromFamily and not destNotFromFamily):\n",
    "            return nodeToPathDic, nodePathToModuleDic\n",
    "        destInModuleDf = moduleDf[moduleDf.Node == destNode]\n",
    "        destNotInModule = destInModuleDf.empty\n",
    "        sourceModuleTailWasOverlappedDf = moduleDf[moduleDf.Node == sourceNode][moduleDf.Start <= sourceToDestPath[0]][moduleDf.End > sourceToDestPath[0]]\n",
    "        sourceModuleTailWasNotOverlapped = sourceModuleTailWasOverlappedDf.empty\n",
    "        \n",
    "        sourceModuleHeadWasOverlappedDf = moduleDf[moduleDf.Node == sourceNode][moduleDf.Start < sourceToDestPath[1]][moduleDf.End >= sourceToDestPath[1]]\n",
    "        sourceModuleHeadWasNotOverlapped = sourceModuleHeadWasOverlappedDf.empty\n",
    "        \n",
    "        sourcePerfectFitDf = moduleDf[moduleDf.Node == sourceNode][moduleDf.Start == sourceToDestPath[0]][moduleDf.End == sourceToDestPath[1]]\n",
    "        sourceNotPerfectFit = sourcePerfectFitDf.empty\n",
    "        \n",
    "        if not destNotInModule:\n",
    "            # Handling dulplication, if the module of the source path is a perfect click, and the dest module contains only one block, then\n",
    "            # this is considered as a dulplication and continue merging the two modules. \n",
    "            if not sourceNotPerfectFit:\n",
    "                if len(list(dest_m_graph.nodes)) == 1:\n",
    "                    break\n",
    "            # Check the overlapped length with the source module\n",
    "            if not sourceModuleTailWasNotOverlapped:\n",
    "                try:\n",
    "                    newLeft = int(sourceModuleTailWasOverlappedDf['End'])\n",
    "                except:\n",
    "                    newLeft = int(min(list(sourceModuleTailWasOverlappedDf['End'])))\n",
    "                if newLeft - int(sourceToDestPath[0]) > 100:\n",
    "#                   head is overlapping more than 100bp\n",
    "                    return nodeToPathDic, nodePathToModuleDic\n",
    "                else:\n",
    "#                   truncate the head and change the range with a new left starting index corresponding to the overlapping module\n",
    "                    if newLeft >= sourceToDestPath[1]:\n",
    "                        return nodeToPathDic, nodePathToModuleDic\n",
    "                    sourceToDestPath = (newLeft,sourceToDestPath[1])\n",
    "                    if sourceToDestPath[1] - newLeft < 50:\n",
    "                        return nodeToPathDic, nodePathToModuleDic\n",
    "                    \n",
    "                    offset = newLeft - int(sourceToDestPath[0])\n",
    "                    boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                    leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                    correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                    leftOverDestArray = destToSourceArray[:boundary]\n",
    "                    correspondingDestArray = destToSourceArray[boundary:]\n",
    "                    nodeToPathDic, nodePathToModuleDic = removeOldModule(dest_m_graph, nodeToPathDic, nodePathToModuleDic)\n",
    "                    if sourceDirection == destDirection:\n",
    "                        newLeft = destToSourcePath[0] + (len(leftOverDestArray) - leftOverDestArray.count('-'))\n",
    "                        choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), (len(leftOverDestArray) - leftOverDestArray.count('-')), correctDirectionModule)\n",
    "                        dest_corres_module = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                        numberOfNewModules = len(choppedModuleList)\n",
    "                        for moduleIndex in range(numberOfNewModules):\n",
    "                            newModule = choppedModuleList[moduleIndex]\n",
    "                            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                        destToSourcePath = (newLeft,destToSourcePath[1])\n",
    "                    else:\n",
    "                        if sourceDirection == \"-\":\n",
    "                            boundary = choppedIndex(sourceToDestArray, int(sourceToDestPath[1]) - newLeft)\n",
    "                            leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                            correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                            leftOverDestArray = destToSourceArray[boundary:]\n",
    "                            correspondingDestArray = destToSourceArray[:boundary]\n",
    "                        newRight = destToSourcePath[1] - (len(leftOverDestArray) - leftOverDestArray.count('-'))\n",
    "                        choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newRight - destToSourcePath[0], correctDirectionModule)\n",
    "                        dest_corres_module = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                        numberOfNewModules = len(choppedModuleList)\n",
    "                        for moduleIndex in range(numberOfNewModules):\n",
    "                            newModule = choppedModuleList[moduleIndex]\n",
    "                            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                        destToSourcePath = (destToSourcePath[0],newRight)\n",
    "                    return recursiveModuleVSModuleChecking(listOfModules, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection,destDirection, dest_corres_module,nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "            if not sourceModuleHeadWasNotOverlapped:\n",
    "                try:\n",
    "                    newRight = int(sourceModuleHeadWasOverlappedDf['Start'])\n",
    "                except:\n",
    "                    newRight = int(max(list(sourceModuleHeadWasOverlappedDf['Start'])))\n",
    "                if int(sourceToDestPath[1]) - newRight > 100:\n",
    "#                   tail is overlapping more than 100bp\n",
    "                    return nodeToPathDic, nodePathToModuleDic\n",
    "                else:\n",
    "#                   truncate the tail and change the range with a new right starting index corresponding to the overlapping module\n",
    "                    if newRight <= sourceToDestPath[0]:\n",
    "                        return nodeToPathDic, nodePathToModuleDic\n",
    "                    sourceToDestPath = (sourceToDestPath[0],newRight)\n",
    "                    if newRight - sourceToDestPath[0] < 50:\n",
    "                        return nodeToPathDic, nodePathToModuleDic\n",
    "                    offset = newRight - int(sourceToDestPath[0])\n",
    "                    boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                    correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                    leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                    correspondingDestArray = destToSourceArray[:boundary]\n",
    "                    leftOverDestArray = destToSourceArray[boundary:]\n",
    "                    nodeToPathDic, nodePathToModuleDic = removeOldModule(dest_m_graph, nodeToPathDic, nodePathToModuleDic)\n",
    "                    if sourceDirection == destDirection:\n",
    "                        newRight = destToSourcePath[0] + (len(correspondingDestArray) - correspondingDestArray.count('-'))\n",
    "                        choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), (len(correspondingDestArray) - correspondingDestArray.count('-')), correctDirectionModule)\n",
    "                        dest_corres_module = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                        numberOfNewModules = len(choppedModuleList)\n",
    "                        for moduleIndex in range(numberOfNewModules):\n",
    "                            newModule = choppedModuleList[moduleIndex]\n",
    "                            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                        destToSourcePath = (destToSourcePath[0],newRight)\n",
    "                    else:\n",
    "                        if sourceDirection == \"-\":\n",
    "                            boundary = choppedIndex(sourceToDestArray, int(sourceToDestPath[1]) - newRight)\n",
    "                            leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                            correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                            leftOverDestArray = destToSourceArray[:boundary]\n",
    "                            correspondingDestArray = destToSourceArray[boundary:]\n",
    "                        newLeft = destToSourcePath[1] - (len(correspondingDestArray) - correspondingDestArray.count('-'))\n",
    "                        choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newLeft - destToSourcePath[0], correctDirectionModule)\n",
    "                        dest_corres_module = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                        numberOfNewModules = len(choppedModuleList)\n",
    "                        for moduleIndex in range(numberOfNewModules):\n",
    "                            newModule = choppedModuleList[moduleIndex]\n",
    "                            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                        destToSourcePath = (newLeft,destToSourcePath[1])\n",
    "                    return recursiveModuleVSModuleChecking(listOfModules, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection,destDirection, dest_corres_module,nodeToPathDic, nodePathToModuleDic, correspondingSourceArray, correspondingDestArray)\n",
    "    \n",
    "    sourcePathStart, sourcePathEnd = sourceToDestPath[0], sourceToDestPath[1]\n",
    "    destPathStart, destPathEnd = destToSourcePath[0], destToSourcePath[1]\n",
    "    for i in range(connectedModulesLength):\n",
    "        source_m_graph = listOfModules[i]\n",
    "        if not source_m_graph in set(nodePathToModuleDic.values()):\n",
    "            continue\n",
    "        aModule = list(source_m_graph.nodes)\n",
    "        moduleToDf = pd.DataFrame(aModule, columns =['Node', 'Path', 'Direction'])\n",
    "        moduleToDf = moduleToDf[moduleToDf.Node == sourceNode]\n",
    "        pairList = list(moduleToDf['Path'])\n",
    "        start = [x[0] for x in pairList]\n",
    "        end = [x[1] for x in pairList]\n",
    "        moduleToDf = moduleToDf.assign(Start = start)\n",
    "        moduleToDf = moduleToDf.assign(End = end)\n",
    "        shouldContinue1 = moduleToDf[sourcePathStart <= moduleToDf.Start][sourcePathEnd <= moduleToDf.Start]\n",
    "        shouldContinue2 = moduleToDf[sourcePathStart >= moduleToDf.End][sourcePathEnd >= moduleToDf.End]\n",
    "        qualifiedDf = moduleToDf[~moduleToDf.index.isin(shouldContinue1.index.union(shouldContinue2.index))]\n",
    "        moduleList = qualifiedDf.values.tolist()\n",
    "        for aBlock in moduleList:\n",
    "            aBlock = aBlock[:3]\n",
    "            node = aBlock[0]\n",
    "            block = aBlock[1]\n",
    "            blockStart = block[0]\n",
    "            blockEnd = block[1]\n",
    "            blockDirection = aBlock[2]\n",
    "            if blockStart <= sourcePathStart and sourcePathEnd <= blockEnd:\n",
    "#               the block totally fits an existing module\n",
    "                nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, sourceToDestPath, destToSourcePath, sourceDirection, destDirection,dest_m_graph, sourceToDestArray, destToSourceArray,nodeToPathDic, nodePathToModuleDic)\n",
    "            \n",
    "            elif sourcePathStart <= blockStart and sourcePathEnd <= blockEnd:\n",
    "#               the head surpasses a given module, recursivelly check the head surpassed range\n",
    "                if source_m_graph is dest_m_graph:\n",
    "                    return nodeToPathDic, nodePathToModuleDic\n",
    "                oldModule = dest_m_graph\n",
    "                offset = int(blockStart - sourcePathStart)\n",
    "                boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                leftOverDestArray = destToSourceArray[:boundary]\n",
    "                correspondingDestArray = destToSourceArray[boundary:]\n",
    "                nodeToPathDic, nodePathToModuleDic = removeOldModule(oldModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                if sourceDirection == destDirection:\n",
    "                    newLeft = destToSourcePath[0] + (len(leftOverDestArray) - leftOverDestArray.count('-'))\n",
    "                    choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), (len(leftOverDestArray) - leftOverDestArray.count('-')), correctDirectionModule)\n",
    "                    dest_corres_module = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                    residueList = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                    numberOfNewModules = len(choppedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = choppedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, (blockStart,sourcePathEnd), (newLeft,destToSourcePath[1]), sourceDirection, destDirection, dest_corres_module, correspondingSourceArray, correspondingDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "                    residue = (destToSourcePath[0], newLeft)\n",
    "                    \n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockStart))\n",
    "                        leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                        correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                        leftOverDestArray = destToSourceArray[boundary:]\n",
    "                        correspondingDestArray = destToSourceArray[:boundary]\n",
    "                    newRight = destToSourcePath[0] + (len(correspondingDestArray) - correspondingDestArray.count('-'))\n",
    "                    choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newRight - destToSourcePath[0], correctDirectionModule)\n",
    "                    dest_corres_module = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                    residueList = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                    numberOfNewModules = len(choppedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = choppedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock,source_m_graph, sourceNode, destNode, (blockStart,sourcePathEnd), (destToSourcePath[0],newRight), sourceDirection, destDirection, dest_corres_module, correspondingSourceArray, correspondingDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "                    residue = (newRight,destToSourcePath[1])\n",
    "                overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, (sourcePathStart,blockStart))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                nodeToPathDic, nodePathToModuleDic = recursiveModuleVSModuleChecking(modulesToBeVisited, sourceNode, destNode, (sourcePathStart,blockStart), residue, sourceDirection,destDirection, residueList,nodeToPathDic, nodePathToModuleDic, leftOverSourceArray, leftOverDestArray)\n",
    "\n",
    "            elif sourcePathStart >= blockStart and sourcePathEnd >= blockEnd:\n",
    "#               the tail surpasses a given module, recursivelly check the tail surpassed range\n",
    "                if source_m_graph is dest_m_graph:\n",
    "                    return nodeToPathDic, nodePathToModuleDic\n",
    "                oldModule = dest_m_graph\n",
    "                offset = int(blockEnd - sourcePathStart)\n",
    "                boundary = choppedIndex(sourceToDestArray, offset)\n",
    "                correspondingSourceArray = sourceToDestArray[:boundary]\n",
    "                leftOverSourceArray = sourceToDestArray[boundary:]\n",
    "                correspondingDestArray = destToSourceArray[:boundary]\n",
    "                leftOverDestArray = destToSourceArray[boundary:]\n",
    "                nodeToPathDic, nodePathToModuleDic = removeOldModule(oldModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                if sourceDirection == destDirection:\n",
    "                    newRight = destToSourcePath[0] + (len(correspondingDestArray) - correspondingDestArray.count('-'))\n",
    "                    choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), (len(correspondingDestArray) - correspondingDestArray.count('-')), correctDirectionModule)\n",
    "                    dest_corres_module = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                    residueList = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                    numberOfNewModules = len(choppedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = choppedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, (sourcePathStart,blockEnd), (destToSourcePath[0],newRight), sourceDirection, destDirection, dest_corres_module, correspondingSourceArray, correspondingDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "                    residue = (newRight, destToSourcePath[1])\n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockEnd))\n",
    "                        leftOverSourceArray = sourceToDestArray[:boundary]\n",
    "                        correspondingSourceArray = sourceToDestArray[boundary:]\n",
    "                        leftOverDestArray = destToSourceArray[:boundary]\n",
    "                        correspondingDestArray = destToSourceArray[boundary:]\n",
    "                    newLeft = destToSourcePath[0] + (len(leftOverDestArray) - leftOverDestArray.count('-'))\n",
    "                    choppedModuleList = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newLeft - destToSourcePath[0], correctDirectionModule)\n",
    "                    dest_corres_module = choppedModuleList[1] if dest_block_direction == \"+\" else choppedModuleList[0]\n",
    "                    residueList = choppedModuleList[0] if dest_block_direction == \"+\" else choppedModuleList[1]\n",
    "                    numberOfNewModules = len(choppedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = choppedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, (sourcePathStart,blockEnd), (newLeft,destToSourcePath[1]), sourceDirection, destDirection, dest_corres_module, correspondingSourceArray, correspondingDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "                    residue = (destToSourcePath[0], newLeft)\n",
    "                overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, (blockEnd,sourcePathEnd))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                nodeToPathDic, nodePathToModuleDic = recursiveModuleVSModuleChecking(modulesToBeVisited, sourceNode, destNode, (blockEnd,sourcePathEnd), residue, sourceDirection,destDirection, residueList,nodeToPathDic, nodePathToModuleDic, leftOverSourceArray, leftOverDestArray)\n",
    "\n",
    "            elif sourcePathStart <= blockStart and sourcePathEnd >= blockEnd:\n",
    "#               both the head and the tail surpass a given module, recursivelly check both surpassed ranges\n",
    "                if source_m_graph is dest_m_graph:\n",
    "                    return nodeToPathDic, nodePathToModuleDic\n",
    "                offset1 = int(blockStart - sourcePathStart)\n",
    "                offset2 = int(blockEnd - sourcePathStart)\n",
    "                oldModule = dest_m_graph\n",
    "                nodeToPathDic, nodePathToModuleDic = removeOldModule(oldModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                boundary1 = choppedIndex(sourceToDestArray, offset1)\n",
    "                boundary2 = choppedIndex(sourceToDestArray, offset2)\n",
    "                leftSourceArray = sourceToDestArray[:boundary1]\n",
    "                midSourceArray = sourceToDestArray[boundary1:boundary2]\n",
    "                rightSourceArray = sourceToDestArray[boundary2:]\n",
    "                leftDestArray = destToSourceArray[:boundary1]\n",
    "                midDestArray = destToSourceArray[boundary1:boundary2]\n",
    "                rightDestArray = destToSourceArray[boundary2:]\n",
    "                if sourceDirection == destDirection:\n",
    "                    newLeft = destToSourcePath[0] + (len(leftDestArray) - leftDestArray.count('-'))\n",
    "                    newRight = newLeft + (len(midDestArray) - midDestArray.count('-'))\n",
    "                    first_module_list = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newRight - destToSourcePath[0], correctDirectionModule)\n",
    "                    newBlockNode = (destNode,(destToSourcePath[0],newRight),dest_block_direction)\n",
    "                    targetModule = first_module_list[0] if dest_block_direction == \"+\" else first_module_list[1]\n",
    "                    second_module_list = partitionToTwoModules(newBlockNode, newLeft - destToSourcePath[0], targetModule)\n",
    "                    if dest_block_direction == \"+\":\n",
    "                        updatedModuleList = [second_module_list[0],second_module_list[1],first_module_list[1]]\n",
    "                    else:\n",
    "                        updatedModuleList = [second_module_list[1],second_module_list[0],first_module_list[0]]\n",
    "                    residueList1 = updatedModuleList[0]\n",
    "                    residueList2 = updatedModuleList[2]\n",
    "                    dest_corres_module = updatedModuleList[1]\n",
    "                    numberOfNewModules = len(updatedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = updatedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, block,(newLeft,newRight), sourceDirection, destDirection, dest_corres_module, midSourceArray, midDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "\n",
    "                    residue1 = (destToSourcePath[0], newLeft)\n",
    "                    residue2 = (newRight, destToSourcePath[1])\n",
    "                else:\n",
    "                    if sourceDirection == \"-\":\n",
    "                        boundary1 = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockEnd))\n",
    "                        boundary2 = choppedIndex(sourceToDestArray, int(sourcePathEnd - blockStart))\n",
    "                        rightSourceArray = sourceToDestArray[:boundary1]\n",
    "                        midSourceArray = sourceToDestArray[boundary1:boundary2]\n",
    "                        leftSourceArray = sourceToDestArray[boundary2:]\n",
    "                        rightDestArray = destToSourceArray[:boundary1]\n",
    "                        midDestArray = destToSourceArray[boundary1:boundary2]\n",
    "                        leftDestArray = destToSourceArray[boundary2:]\n",
    "                    newRight = destToSourcePath[1] - (len(leftDestArray) - leftDestArray.count('-'))\n",
    "                    newLeft = newRight - (len(midDestArray) - midDestArray.count('-'))\n",
    "                    first_module_list = partitionToTwoModules((destNode, destToSourcePath, dest_block_direction), newRight-destToSourcePath[0], correctDirectionModule)\n",
    "                    newBlockNode = (destNode,(destToSourcePath[0],newRight),dest_block_direction)\n",
    "                    targetModule = first_module_list[0] if dest_block_direction == \"+\" else first_module_list[1]\n",
    "                    second_module_list = partitionToTwoModules(newBlockNode, newLeft-destToSourcePath[0], targetModule)\n",
    "                    if dest_block_direction == \"+\":\n",
    "                        updatedModuleList = [first_module_list[1],second_module_list[1],second_module_list[0]]\n",
    "                    else:\n",
    "                        updatedModuleList = [first_module_list[0],second_module_list[0],second_module_list[1]]\n",
    "                    residueList1 = updatedModuleList[0]\n",
    "                    residueList2 = updatedModuleList[2]\n",
    "                    dest_corres_module = updatedModuleList[1]\n",
    "                    numberOfNewModules = len(updatedModuleList)\n",
    "                    for moduleIndex in range(numberOfNewModules):\n",
    "                        newModule = updatedModuleList[moduleIndex]\n",
    "                        nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "                    nodeToPathDic, nodePathToModuleDic = checkModuleModuleOverlap(aBlock, source_m_graph, sourceNode, destNode, block,(newLeft,newRight), sourceDirection, destDirection, dest_corres_module, midSourceArray, midDestArray, nodeToPathDic, nodePathToModuleDic)\n",
    "                    residue1 = (newRight, destToSourcePath[1])\n",
    "                    residue2 = (destToSourcePath[0], newLeft)\n",
    "                overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, (sourcePathStart ,blockStart))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                nodeToPathDic, nodePathToModuleDic = recursiveModuleVSModuleChecking(modulesToBeVisited, sourceNode, destNode, (sourcePathStart ,blockStart), residue1, sourceDirection,destDirection,residueList1,nodeToPathDic, nodePathToModuleDic, leftSourceArray, leftDestArray)\n",
    "                overlappedPairs = bedtoolCall(sourceNode, nodeToPathDic, (blockEnd , sourcePathEnd))\n",
    "                modulesToBeVisited = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "                nodeToPathDic, nodePathToModuleDic = recursiveModuleVSModuleChecking(modulesToBeVisited, sourceNode, destNode, (blockEnd , sourcePathEnd), residue2, sourceDirection,destDirection,residueList2,nodeToPathDic, nodePathToModuleDic, rightSourceArray, rightDestArray)\n",
    "            return nodeToPathDic, nodePathToModuleDic\n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def moduleModulePartition(nodeA,nodeB,pathAtoB,partBtoA, directionAtoB,directionBtoA,nodeToPathDic,nodePathToModuleDic, arrayAtoB, arrayBtoA):\n",
    "    \"\"\"\n",
    "    Output: The finished module vs module partition graph\n",
    "    Suppose blast calls the two paths: pathA and pathB. \n",
    "    1. Check how pathA partitions the current modules and what module each subsequence is corresponding to. \n",
    "    2. For each subsequence, treat it as a node and find the corresponding subsequence in pathB.\n",
    "    3. Merge the two modules, one from pathA and one from pathB.\n",
    "    \"\"\"\n",
    "    overlappedPairs = bedtoolCall(nodeB, nodeToPathDic, partBtoA)\n",
    "    bConnectedModules = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "    \n",
    "    nodeToPathDicCopy = copy.deepcopy(nodeToPathDic)\n",
    "    nodePathToModuleDicCopy = copy.deepcopy(nodePathToModuleDic)\n",
    "    try:\n",
    "        partitionBlockDic, moduleUpdateDic = chopModulesAndUpdateGraph(bConnectedModules,nodeB,partBtoA,directionBtoA,{},{},nodeToPathDic,nodePathToModuleDic)\n",
    "    except:\n",
    "        nodeToPathDic, nodePathToModuleDic = nodeToPathDicCopy, nodePathToModuleDicCopy\n",
    "        for m_graph in list(nodePathToModuleDic.values()):\n",
    "            for ccIndex, cc in enumerate(nx.strongly_connected_components(m_graph)):\n",
    "                if ccIndex > 0:\n",
    "                    nodeToPathDic,nodePathToModuleDic = removeOldModule(m_graph, nodeToPathDic, nodePathToModuleDic)\n",
    "                    break\n",
    "        return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "    for oldModule in moduleUpdateDic.keys():\n",
    "        updatedModuleList = moduleUpdateDic[oldModule]\n",
    "        nodeToPathDic, nodePathToModuleDic = removeOldModule(oldModule,nodeToPathDic, nodePathToModuleDic)\n",
    "        numberOfNewModules = len(updatedModuleList)\n",
    "        for moduleIndex in range(numberOfNewModules):\n",
    "            newModule = updatedModuleList[moduleIndex]\n",
    "            nodeToPathDic, nodePathToModuleDic = updateNewModule(newModule, nodeToPathDic, nodePathToModuleDic)\n",
    "    for block in partitionBlockDic.keys():\n",
    "        blockPath = block[1]\n",
    "        correspondingModule = partitionBlockDic[block]\n",
    "        correspondingPath = None\n",
    "        leftOffset = blockPath[0] - partBtoA[0]\n",
    "        rightOffset = blockPath[1] - partBtoA[0] \n",
    "        if directionAtoB == directionBtoA:\n",
    "            leftBoundary = choppedIndex(arrayBtoA, leftOffset)\n",
    "            rightBoundary = choppedIndex(arrayBtoA, rightOffset)\n",
    "            subArrayBtoA = arrayBtoA[leftBoundary:rightBoundary]\n",
    "            subArrayAtoB = arrayAtoB[leftBoundary:rightBoundary]\n",
    "            left = pathAtoB[0]+(len(arrayAtoB[:leftBoundary]) - arrayAtoB[:leftBoundary].count('-'))\n",
    "            right = pathAtoB[0]+(len(arrayAtoB[:rightBoundary]) - arrayAtoB[:rightBoundary].count('-'))\n",
    "            correspondingPath = (left,right)\n",
    "        else:\n",
    "            if directionBtoA == '+':\n",
    "                leftBoundary = choppedIndex(arrayBtoA, leftOffset)\n",
    "                rightBoundary = choppedIndex(arrayBtoA, rightOffset)\n",
    "                subArrayBtoA = arrayBtoA[leftBoundary:rightBoundary]\n",
    "                subArrayAtoB = arrayAtoB[leftBoundary:rightBoundary]\n",
    "                right = pathAtoB[1]-(len(arrayAtoB[:leftBoundary]) - arrayAtoB[:leftBoundary].count('-'))\n",
    "                left = pathAtoB[1]-(len(arrayAtoB[:rightBoundary]) - arrayAtoB[:rightBoundary].count('-'))\n",
    "                correspondingPath = (left,right)\n",
    "            else:\n",
    "                leftBoundary = choppedIndex(arrayBtoA, int(partBtoA[1]-blockPath[1]))\n",
    "                rightBoundary = choppedIndex(arrayBtoA, int(partBtoA[1]-blockPath[0]))\n",
    "                subArrayBtoA = arrayBtoA[leftBoundary:rightBoundary]\n",
    "                subArrayAtoB = arrayAtoB[leftBoundary:rightBoundary]\n",
    "                left = pathAtoB[0]+(len(arrayAtoB[:leftBoundary]) - arrayAtoB[:leftBoundary].count('-'))\n",
    "                right = pathAtoB[0]+(len(arrayAtoB[:rightBoundary]) - arrayAtoB[:rightBoundary].count('-'))\n",
    "                correspondingPath = (left,right)\n",
    "            if correspondingPath[0] >= correspondingPath[1]:\n",
    "                continue\n",
    "        if not correspondingModule in set(nodePathToModuleDic.values()):\n",
    "            continue\n",
    "        overlappedPairs = bedtoolCall(nodeA, nodeToPathDic, correspondingPath)\n",
    "        connectedModules = [nodePathToModuleDic[pair] for pair in overlappedPairs]\n",
    "        nodeToPathDicCopy = copy.deepcopy(nodeToPathDic)\n",
    "        nodePathToModuleDicCopy = copy.deepcopy(nodePathToModuleDic)\n",
    "        try:\n",
    "            nodeToPathDic, nodePathToModuleDic = recursiveModuleVSModuleChecking(connectedModules, nodeA, nodeB, correspondingPath, blockPath, directionAtoB,directionBtoA,correspondingModule,nodeToPathDic, nodePathToModuleDic, subArrayAtoB, subArrayBtoA)\n",
    "        except:\n",
    "            nodeToPathDic, nodePathToModuleDic = nodeToPathDicCopy, nodePathToModuleDicCopy\n",
    "            nodeToPathDic, nodePathToModuleDic = sanity(nodeToPathDic, nodePathToModuleDic)\n",
    "            \n",
    "    return nodeToPathDic, nodePathToModuleDic\n",
    "\n",
    "def checkPathOverlap(moduleBlockPath,genePath):\n",
    "    \"\"\"\n",
    "    Input: Two intervals (start,end)\n",
    "    Output: A tuple(boolean, interval). The boolean is True is the two input intervals are overlapped, False if the two intervals are\n",
    "            disjoint. If the boolean is True, the returned interval is the superset of the two input intervals. If the boolean is False,\n",
    "            the returned interval is the first interval in the input.\n",
    "    \"\"\"\n",
    "    blockStart = moduleBlockPath[0]\n",
    "    blockEnd = moduleBlockPath[1]\n",
    "    geneStart = genePath[0]\n",
    "    geneEnd = genePath[1]\n",
    "    if (blockStart <= geneStart and blockEnd <= geneStart) or (geneEnd <= blockStart and geneEnd <= blockEnd):\n",
    "        return False, None, -1\n",
    "    else:\n",
    "        if (geneStart >= blockStart and geneEnd <= blockEnd):\n",
    "            return True, (geneStart,geneEnd), geneEnd-geneStart\n",
    "        elif (geneStart <= blockStart and geneEnd >= blockEnd):\n",
    "            return True, (blockStart,blockEnd), blockEnd-blockStart\n",
    "        elif (blockStart >= geneStart and blockStart <= geneEnd and blockEnd >= geneEnd):\n",
    "            return True,(blockStart,geneEnd), geneEnd-blockStart\n",
    "        elif (geneStart >= blockStart and blockEnd >= geneStart and geneEnd >= blockEnd):\n",
    "            return True, (geneStart,blockEnd),blockEnd-geneStart\n",
    "\n",
    "def main(logger, G, block_in_mhg, tempBedFile = 'bedtoolTemp.txt'):\n",
    "    counter = 0\n",
    "#     f2 = open(moduleFileName,'w')\n",
    "    total_cc_number = nx.number_strongly_connected_components(G)\n",
    "    logger.info(f\"Total {total_cc_number} cc are waiting to be visited\")\n",
    "    nodeToPathDic = defaultdict(lambda: set())\n",
    "    nodePathToModuleDic = dict()\n",
    "    for cc in nx.strongly_connected_components(G):\n",
    "        S = G.subgraph(cc)\n",
    "        nodeList = list(S.nodes())\n",
    "        counter += 1        \n",
    "        edgeList = list(S.edges())\n",
    "        numberOfNodes = len(nodeList)\n",
    "        numberOfEdges = len(edgeList)\n",
    "        logger.info(f\"cc number {counter} is on the show being visited\")\n",
    "        big_cc = False\n",
    "        if numberOfEdges > 100000:\n",
    "            logger.info(f\"cc number {counter} is a big cc, truly big one, will take a dozen of hours\")\n",
    "            big_cc = True\n",
    "            big_cc_edge_count = numberOfEdges\n",
    "        elif numberOfEdges > 10000:\n",
    "            logger.info(f\"cc number {counter} has {numberOfEdges} edges\")\n",
    "            logger.info(f\"cc number {counter} is a big cc, big but not that big, but still big; could take a few hours\")\n",
    "            big_cc = True\n",
    "            big_cc_edge_count = numberOfEdges\n",
    "        numberOfEdges = 0\n",
    "        edgeCounterDicBetweenTwoNodes = defaultdict(lambda: 0)\n",
    "        while edgeList:\n",
    "            sourceNode, destNode = edgeList[0][0], edgeList[0][1]\n",
    "            edgeIndexBetweenTwoNodes = edgeCounterDicBetweenTwoNodes[(sourceNode,destNode)]\n",
    "            sourceToDestPath = S[sourceNode][destNode][edgeIndexBetweenTwoNodes]['weight'][0]\n",
    "            sourceToDestArray = S[sourceNode][destNode][edgeIndexBetweenTwoNodes]['weight'][1]\n",
    "            edgeList.remove(((sourceNode,destNode)))\n",
    "            destToSourcePath = S[destNode][sourceNode][edgeIndexBetweenTwoNodes]['weight'][0]\n",
    "            destToSourceArray = S[destNode][sourceNode][edgeIndexBetweenTwoNodes]['weight'][1]\n",
    "            edgeList.remove(((destNode,sourceNode)))\n",
    "            edgeCounterDicBetweenTwoNodes[(sourceNode,destNode)] += 1\n",
    "            numberOfEdges += 2\n",
    "            sourceNodeAndPath = (sourceNode,tuple(sorted(sourceToDestPath)))\n",
    "            destNodeAndPath = (destNode,tuple(sorted(destToSourcePath)))\n",
    "            pair = tuple(sorted((sourceNodeAndPath,destNodeAndPath)))\n",
    "            if numberOfEdges % 500 == 0:\n",
    "                nodeToPathDic,nodePathToModuleDic = trimShortModules(nodeToPathDic,nodePathToModuleDic)\n",
    "            if big_cc:\n",
    "                if numberOfEdges % int(0.01*big_cc_edge_count) == 0:\n",
    "                    logger.info(f\"No worries! I am still working! {round(100 *numberOfEdges / big_cc_edge_count)}% of all edges in cc number {counter} are finished\")\n",
    "            sourceDirection, destDirection = \"+\", \"+\"\n",
    "            if sourceToDestPath[0] > sourceToDestPath[1]:\n",
    "                sourceDirection = \"-\"\n",
    "            if destToSourcePath[0] > destToSourcePath[1]:\n",
    "                destDirection = \"-\"\n",
    "            sourceToDestPath = tuple(sorted(sourceToDestPath))\n",
    "            destToSourcePath = tuple(sorted(destToSourcePath))\n",
    "            if sourceToDestPath[1]-sourceToDestPath[0] < 20:\n",
    "                continue\n",
    "            if sourceNode in nodeList and destNode in nodeList:\n",
    "#               case of two nodes\n",
    "                sourceNodeBlocks = nodePartition(sourceToDestPath,sourceNode[1])\n",
    "                destNodeBlocks = nodePartition(destToSourcePath,destNode[1])\n",
    "                module = nx.MultiDiGraph()\n",
    "                module.add_edge((sourceNode,sourceToDestPath, sourceDirection),(destNode,destToSourcePath, destDirection), weight = sourceToDestArray)\n",
    "                module.add_edge((destNode,destToSourcePath, destDirection),(sourceNode,sourceToDestPath, sourceDirection), weight = destToSourceArray)\n",
    "                moduleSourceNode,moduleDestNode = [],[]\n",
    "                for element in sourceNodeBlocks:\n",
    "                    if element == sourceToDestPath:\n",
    "                        moduleSourceNode.append(module)\n",
    "                    else:\n",
    "                        newM_graph = nx.MultiDiGraph()\n",
    "                        newM_graph.add_node((sourceNode, element, \"+\"),)\n",
    "                        moduleSourceNode.append(newM_graph)\n",
    "                sourcePathModulePair = list(zip(sourceNodeBlocks,moduleSourceNode))\n",
    "                for (corrsPath, corrsModule) in sourcePathModulePair:\n",
    "                    nodeToPathDic[sourceNode].add(corrsPath)\n",
    "                    nodePathToModuleDic[(sourceNode,corrsPath)] = corrsModule\n",
    "                for element in destNodeBlocks:\n",
    "                    if element == destToSourcePath:\n",
    "                        moduleDestNode.append(module)\n",
    "                    else:\n",
    "                        newM_graph = nx.MultiDiGraph()\n",
    "                        newM_graph.add_node((destNode, element, \"+\"),)\n",
    "                        moduleDestNode.append(newM_graph)\n",
    "                destPathModulePair = list(zip(destNodeBlocks,moduleDestNode))\n",
    "                for (corrsPath, corrsModule) in destPathModulePair:\n",
    "                    nodeToPathDic[destNode].add(corrsPath)\n",
    "                    nodePathToModuleDic[(destNode,corrsPath)] = corrsModule\n",
    "                nodeList.remove(sourceNode)\n",
    "                nodeList.remove(destNode)\n",
    "            elif sourceNode not in nodeList and destNode in nodeList:\n",
    "#                case of a node and a module\n",
    "                nodeToPathDic,nodePathToModuleDic = nodeModulePartition(sourceNode,destNode,sourceToDestPath,destToSourcePath,sourceDirection,destDirection,nodeToPathDic,nodePathToModuleDic,tempBedFile, sourceToDestArray, destToSourceArray)\n",
    "                nodeList.remove(destNode)\n",
    "            elif sourceNode in nodeList and destNode not in nodeList:\n",
    "#                case of a node and a module\n",
    "                nodeToPathDic,nodePathToModuleDic = nodeModulePartition(destNode,sourceNode,destToSourcePath,sourceToDestPath,destDirection,sourceDirection,nodeToPathDic,nodePathToModuleDic,tempBedFile, destToSourceArray, sourceToDestArray)\n",
    "                nodeList.remove(sourceNode)\n",
    "            else:\n",
    "#                case of two modules\n",
    "                nodeToPathDic,nodePathToModuleDic = moduleModulePartition(sourceNode,destNode,sourceToDestPath,destToSourcePath,sourceDirection,destDirection,nodeToPathDic,nodePathToModuleDic, sourceToDestArray, destToSourceArray)\n",
    "\n",
    "        nodeToPathDic,nodePathToModuleDic = trimShortModules(nodeToPathDic,nodePathToModuleDic)\n",
    "\n",
    "    os.remove('tempA.bed')\n",
    "    os.remove('tempB.bed')\n",
    "    \n",
    "#     return nodePathToModuleDic\n",
    "\n",
    "    modules = list(set([tuple(sorted(list(m_graph.nodes))) for m_graph in nodePathToModuleDic.values()]))\n",
    "    modules = [m for m in modules if len(m) >= block_in_mhg]\n",
    "    return modules\n",
    "    logger.info(\"everything finished\")\n",
    "    \n",
    "\n",
    "def mhg(blastDf, block_in_mhg):\n",
    "    logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')\n",
    "    logger = logging.getLogger()\n",
    "\n",
    "    df_start_time = time.time()\n",
    "    logger.info(f\"start building dataframe containing pairwise blastn calls\")\n",
    "    \n",
    "    blastDf = blastToDf(blastDf, threshold = 0)\n",
    "    readyToGraphDf = union_node(blastDf)\n",
    "    \n",
    "    logger.info(f\"start building an alignment graph\")\n",
    "    G = nx.MultiDiGraph()\n",
    "    first = [tuple(d) for d in readyToGraphDf[['sourceNode', 'destNode','qEdge']].values]\n",
    "    second = [tuple(d) for d in readyToGraphDf[['destNode','sourceNode', 'sEdge']].values]\n",
    "    combine = []\n",
    "    for i in range(len(first)):\n",
    "        combine.append(first[i])\n",
    "        combine.append(second[i])\n",
    "    G.add_weighted_edges_from(combine)\n",
    "    graph_end_time = time.time()\n",
    "    logger.info(f\"alignment graph finished building in time: {graph_end_time - df_start_time}\")\n",
    "#     return readyToGraphDf,G\n",
    "\n",
    "    logger.info(f\"starting traversing the alignment graph and MHG partition\")\n",
    "    mhg_list = main(logger, G, block_in_mhg)\n",
    "    partition_end_time = time.time()\n",
    "    logger.info(f\"traversing finished MHG partition finished in: {partition_end_time-graph_end_time}\")\n",
    "    return mhg_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "id": "1bd2c04b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2022-07-20 10:55:39,562 - root - INFO - start building dataframe containing pairwise blastn calls\n",
      "2022-07-20 10:55:40,049 - root - INFO - start building an alignment graph\n",
      "2022-07-20 10:55:40,155 - root - INFO - alignment graph finished building in time: 0.5931017398834229\n",
      "2022-07-20 10:55:40,155 - root - INFO - starting traversing the alignment graph and MHG partition\n",
      "2022-07-20 10:55:40,196 - root - INFO - Total 92 cc are waiting to be visited\n",
      "2022-07-20 10:55:40,477 - root - INFO - cc number 1 is on the show being visited\n",
      "2022-07-20 10:55:40,478 - root - INFO - cc number 1 has 15302 edges\n",
      "2022-07-20 10:55:40,478 - root - INFO - cc number 1 is a big cc, big but not that big, but still big; could take a few hours\n",
      "2022-07-20 10:55:59,376 - root - INFO - No worries! I am still working! 2% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:56:14,314 - root - INFO - No worries! I am still working! 4% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:56:35,271 - root - INFO - No worries! I am still working! 6% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:56:50,360 - root - INFO - No worries! I am still working! 8% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:57:08,440 - root - INFO - No worries! I am still working! 10% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:57:29,659 - root - INFO - No worries! I am still working! 12% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:57:41,991 - root - INFO - No worries! I am still working! 14% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:58:00,042 - root - INFO - No worries! I am still working! 16% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:58:19,284 - root - INFO - No worries! I am still working! 18% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:58:37,862 - root - INFO - No worries! I am still working! 20% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:59:04,660 - root - INFO - No worries! I am still working! 22% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:59:29,057 - root - INFO - No worries! I am still working! 24% of all edges in cc number 1 are finished\n",
      "2022-07-20 10:59:45,990 - root - INFO - No worries! I am still working! 26% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:00:06,039 - root - INFO - No worries! I am still working! 28% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:00:23,126 - root - INFO - No worries! I am still working! 30% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:00:41,617 - root - INFO - No worries! I am still working! 32% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:00:59,031 - root - INFO - No worries! I am still working! 34% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:01:16,239 - root - INFO - No worries! I am still working! 36% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:01:36,827 - root - INFO - No worries! I am still working! 38% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:01:55,640 - root - INFO - No worries! I am still working! 40% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:02:12,145 - root - INFO - No worries! I am still working! 42% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:02:31,816 - root - INFO - No worries! I am still working! 44% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:02:54,952 - root - INFO - No worries! I am still working! 46% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:03:12,998 - root - INFO - No worries! I am still working! 48% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:03:35,319 - root - INFO - No worries! I am still working! 50% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:03:54,927 - root - INFO - No worries! I am still working! 52% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:04:13,559 - root - INFO - No worries! I am still working! 54% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:04:37,689 - root - INFO - No worries! I am still working! 56% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:07:09,562 - root - INFO - No worries! I am still working! 58% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:11:15,262 - root - INFO - No worries! I am still working! 60% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:11:51,994 - root - INFO - No worries! I am still working! 62% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:12:29,572 - root - INFO - No worries! I am still working! 64% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:13:04,284 - root - INFO - No worries! I am still working! 66% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:13:42,965 - root - INFO - No worries! I am still working! 68% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:14:21,736 - root - INFO - No worries! I am still working! 70% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:14:55,449 - root - INFO - No worries! I am still working! 72% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:15:26,826 - root - INFO - No worries! I am still working! 74% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:16:03,258 - root - INFO - No worries! I am still working! 76% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:16:49,088 - root - INFO - No worries! I am still working! 78% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:17:36,814 - root - INFO - No worries! I am still working! 80% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:18:37,755 - root - INFO - No worries! I am still working! 82% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:22:41,303 - root - INFO - No worries! I am still working! 84% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:25:24,963 - root - INFO - No worries! I am still working! 86% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:28:31,724 - root - INFO - No worries! I am still working! 88% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:29:18,836 - root - INFO - No worries! I am still working! 90% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:30:07,633 - root - INFO - No worries! I am still working! 92% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:31:16,543 - root - INFO - No worries! I am still working! 94% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:32:25,706 - root - INFO - No worries! I am still working! 96% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:33:42,552 - root - INFO - No worries! I am still working! 98% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:35:03,994 - root - INFO - No worries! I am still working! 100% of all edges in cc number 1 are finished\n",
      "2022-07-20 11:35:04,615 - root - INFO - cc number 2 is on the show being visited\n",
      "2022-07-20 11:35:05,218 - root - INFO - cc number 3 is on the show being visited\n",
      "2022-07-20 11:35:16,015 - root - INFO - cc number 4 is on the show being visited\n",
      "2022-07-20 11:35:16,018 - root - INFO - cc number 5 is on the show being visited\n",
      "2022-07-20 11:35:16,020 - root - INFO - cc number 6 is on the show being visited\n",
      "2022-07-20 11:35:16,023 - root - INFO - cc number 7 is on the show being visited\n",
      "2022-07-20 11:35:16,581 - root - INFO - cc number 8 is on the show being visited\n",
      "2022-07-20 11:35:17,173 - root - INFO - cc number 9 is on the show being visited\n",
      "2022-07-20 11:35:17,176 - root - INFO - cc number 10 is on the show being visited\n",
      "2022-07-20 11:35:17,459 - root - INFO - cc number 11 is on the show being visited\n",
      "2022-07-20 11:35:17,744 - root - INFO - cc number 12 is on the show being visited\n",
      "2022-07-20 11:35:17,747 - root - INFO - cc number 13 is on the show being visited\n",
      "2022-07-20 11:35:17,750 - root - INFO - cc number 14 is on the show being visited\n",
      "2022-07-20 11:35:19,443 - root - INFO - cc number 15 is on the show being visited\n",
      "2022-07-20 11:35:19,446 - root - INFO - cc number 16 is on the show being visited\n",
      "2022-07-20 11:35:19,449 - root - INFO - cc number 17 is on the show being visited\n",
      "2022-07-20 11:35:19,451 - root - INFO - cc number 18 is on the show being visited\n",
      "2022-07-20 11:35:19,453 - root - INFO - cc number 19 is on the show being visited\n",
      "2022-07-20 11:35:20,364 - root - INFO - cc number 20 is on the show being visited\n",
      "2022-07-20 11:35:22,109 - root - INFO - cc number 21 is on the show being visited\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2022-07-20 11:35:22,753 - root - INFO - cc number 22 is on the show being visited\n",
      "2022-07-20 11:35:22,756 - root - INFO - cc number 23 is on the show being visited\n",
      "2022-07-20 11:35:22,758 - root - INFO - cc number 24 is on the show being visited\n",
      "2022-07-20 11:35:22,761 - root - INFO - cc number 25 is on the show being visited\n",
      "2022-07-20 11:35:24,033 - root - INFO - cc number 26 is on the show being visited\n",
      "2022-07-20 11:35:24,036 - root - INFO - cc number 27 is on the show being visited\n",
      "2022-07-20 11:35:24,322 - root - INFO - cc number 28 is on the show being visited\n",
      "2022-07-20 11:35:24,325 - root - INFO - cc number 29 is on the show being visited\n",
      "2022-07-20 11:35:24,661 - root - INFO - cc number 30 is on the show being visited\n",
      "2022-07-20 11:35:25,271 - root - INFO - cc number 31 is on the show being visited\n",
      "2022-07-20 11:35:25,908 - root - INFO - cc number 32 is on the show being visited\n",
      "2022-07-20 11:35:25,911 - root - INFO - cc number 33 is on the show being visited\n",
      "2022-07-20 11:35:25,913 - root - INFO - cc number 34 is on the show being visited\n",
      "2022-07-20 11:35:25,916 - root - INFO - cc number 35 is on the show being visited\n",
      "2022-07-20 11:35:25,918 - root - INFO - cc number 36 is on the show being visited\n",
      "2022-07-20 11:35:25,921 - root - INFO - cc number 37 is on the show being visited\n",
      "2022-07-20 11:35:26,212 - root - INFO - cc number 38 is on the show being visited\n",
      "2022-07-20 11:35:26,215 - root - INFO - cc number 39 is on the show being visited\n",
      "2022-07-20 11:35:26,218 - root - INFO - cc number 40 is on the show being visited\n",
      "2022-07-20 11:35:26,220 - root - INFO - cc number 41 is on the show being visited\n",
      "2022-07-20 11:35:26,222 - root - INFO - cc number 42 is on the show being visited\n",
      "2022-07-20 11:35:26,225 - root - INFO - cc number 43 is on the show being visited\n",
      "2022-07-20 11:35:26,228 - root - INFO - cc number 44 is on the show being visited\n",
      "2022-07-20 11:35:26,230 - root - INFO - cc number 45 is on the show being visited\n",
      "2022-07-20 11:35:26,232 - root - INFO - cc number 46 is on the show being visited\n",
      "2022-07-20 11:35:27,411 - root - INFO - cc number 47 is on the show being visited\n",
      "2022-07-20 11:35:27,414 - root - INFO - cc number 48 is on the show being visited\n",
      "2022-07-20 11:35:28,618 - root - INFO - cc number 49 is on the show being visited\n",
      "2022-07-20 11:35:28,621 - root - INFO - cc number 50 is on the show being visited\n",
      "2022-07-20 11:35:28,624 - root - INFO - cc number 51 is on the show being visited\n",
      "2022-07-20 11:35:28,626 - root - INFO - cc number 52 is on the show being visited\n",
      "2022-07-20 11:35:28,931 - root - INFO - cc number 53 is on the show being visited\n",
      "2022-07-20 11:35:28,934 - root - INFO - cc number 54 is on the show being visited\n",
      "2022-07-20 11:35:28,937 - root - INFO - cc number 55 is on the show being visited\n",
      "2022-07-20 11:35:29,259 - root - INFO - cc number 56 is on the show being visited\n",
      "2022-07-20 11:35:29,559 - root - INFO - cc number 57 is on the show being visited\n",
      "2022-07-20 11:35:29,562 - root - INFO - cc number 58 is on the show being visited\n",
      "2022-07-20 11:35:29,565 - root - INFO - cc number 59 is on the show being visited\n",
      "2022-07-20 11:35:30,149 - root - INFO - cc number 60 is on the show being visited\n",
      "2022-07-20 11:35:30,152 - root - INFO - cc number 61 is on the show being visited\n",
      "2022-07-20 11:35:30,154 - root - INFO - cc number 62 is on the show being visited\n",
      "2022-07-20 11:35:30,157 - root - INFO - cc number 63 is on the show being visited\n",
      "2022-07-20 11:35:30,159 - root - INFO - cc number 64 is on the show being visited\n",
      "2022-07-20 11:35:30,878 - root - INFO - cc number 65 is on the show being visited\n",
      "2022-07-20 11:35:31,174 - root - INFO - cc number 66 is on the show being visited\n",
      "2022-07-20 11:35:31,177 - root - INFO - cc number 67 is on the show being visited\n",
      "2022-07-20 11:35:31,180 - root - INFO - cc number 68 is on the show being visited\n",
      "2022-07-20 11:35:32,123 - root - INFO - cc number 69 is on the show being visited\n",
      "2022-07-20 11:35:32,126 - root - INFO - cc number 70 is on the show being visited\n",
      "2022-07-20 11:35:32,129 - root - INFO - cc number 71 is on the show being visited\n",
      "2022-07-20 11:35:32,131 - root - INFO - cc number 72 is on the show being visited\n",
      "2022-07-20 11:35:32,464 - root - INFO - cc number 73 is on the show being visited\n",
      "2022-07-20 11:35:32,826 - root - INFO - cc number 74 is on the show being visited\n",
      "2022-07-20 11:35:32,829 - root - INFO - cc number 75 is on the show being visited\n",
      "2022-07-20 11:35:33,131 - root - INFO - cc number 76 is on the show being visited\n",
      "2022-07-20 11:35:33,134 - root - INFO - cc number 77 is on the show being visited\n",
      "2022-07-20 11:35:34,029 - root - INFO - cc number 78 is on the show being visited\n",
      "2022-07-20 11:35:34,033 - root - INFO - cc number 79 is on the show being visited\n",
      "2022-07-20 11:35:34,035 - root - INFO - cc number 80 is on the show being visited\n",
      "2022-07-20 11:35:34,038 - root - INFO - cc number 81 is on the show being visited\n",
      "2022-07-20 11:35:34,040 - root - INFO - cc number 82 is on the show being visited\n",
      "2022-07-20 11:35:34,354 - root - INFO - cc number 83 is on the show being visited\n",
      "2022-07-20 11:35:38,632 - root - INFO - cc number 84 is on the show being visited\n",
      "2022-07-20 11:35:38,636 - root - INFO - cc number 85 is on the show being visited\n",
      "2022-07-20 11:35:38,639 - root - INFO - cc number 86 is on the show being visited\n",
      "2022-07-20 11:35:38,641 - root - INFO - cc number 87 is on the show being visited\n",
      "2022-07-20 11:35:38,644 - root - INFO - cc number 88 is on the show being visited\n",
      "2022-07-20 11:35:39,475 - root - INFO - cc number 89 is on the show being visited\n",
      "2022-07-20 11:35:39,478 - root - INFO - cc number 90 is on the show being visited\n",
      "2022-07-20 11:35:39,480 - root - INFO - cc number 91 is on the show being visited\n",
      "2022-07-20 11:35:39,483 - root - INFO - cc number 92 is on the show being visited\n",
      "2022-07-20 11:35:39,486 - root - INFO - traversing finished MHG partition finished in: 2399.3312935829163\n"
     ]
    }
   ],
   "source": [
    "df, check_list = parseBlastXML('temp_blastn/G000006925,G000299455.xml')\n",
    "df = trim_fully_contain(df, check_list)\n",
    "# mhg_list = mhg(df)\n",
    "nodePathToModuleDic = mhg(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "17cbbc4a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[((('G000006925', (1321493, 1385605)), (1337273, 1337298), '-'), (('G000299455', (2818030, 2881682)), (2865878, 2865903), '+')), ((('G000006925', (4805348, 4805944)), (4805348, 4805944), '+'), (('G000299455', (1462059, 1462655)), (1462059, 1462655), '-')), ((('G000006925', (4706174, 4706226)), (4706174, 4706226), '+'), (('G000299455', (3211410, 3211462)), (3211410, 3211462), '+')), ((('G000006925', (2756889, 2766574)), (2765664, 2765902), '-'), (('G000299455', (1277999, 1282453)), (1278671, 1278909), '+')), ((('G000006925', (2849870, 2870340)), (2858926, 2862839), '-'), (('G000299455', (1167028, 1178705)), (1167028, 1170941), '+'))]\n"
     ]
    }
   ],
   "source": [
    "print(mhg_list[:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "b9341113",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dict_keys(['G000982825', 'G000648515', 'G001887595', 'G900048035', 'G000828815', 'G000093065', 'G000195995', 'G000300455', 'G900044015', 'G900039485', 'G000973545', 'G000757785', 'G000006925', 'G000262305', 'G000299455', 'G000757825', 'G001022135', 'G000828515', 'G000164865'])\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "from itertools import groupby\n",
    "\n",
    "seqDic = {}\n",
    "genomeDirectory = './temp/'\n",
    "\n",
    "for genome in os.listdir(genomeDirectory):\n",
    "    if \"G\" not in genome or \".n\" in genome:\n",
    "        continue\n",
    "    organism = genome[:genome.find('.fa')]\n",
    "    seqFile = open(genomeDirectory+genome)\n",
    "    lines = (x[1] for x in groupby(seqFile, lambda line: line[0] == \">\"))\n",
    "    for seqName in lines:\n",
    "        seqName = seqName.__next__()[1:].strip()\n",
    "        seq = \"\".join(s.strip() for s in lines.__next__())\n",
    "        seqDic[seqName] = seq\n",
    "        \n",
    "def revComp(seq):\n",
    "    dic = {'A':'T','T':'A','C':'G','G':'C','R':'C','Y':'G','S':'G','W':'T','K':'C','M':'G','N':'A'}\n",
    "    revSeq = ''.join([dic[char] for char in seq[::-1]])\n",
    "    return revSeq\n",
    "\n",
    "def alignBlocksInModule(seqDic, moduleAsAList):\n",
    "    organismSet = set()\n",
    "    f = open('mafftInput.fasta','w')\n",
    "    for module in moduleAsAList:\n",
    "        block = list(module)\n",
    "        organismID = block[0][0]\n",
    "        pathStart = int(block[1][0])\n",
    "        pathEnd = int(block[1][1])\n",
    "        direction = block[2]\n",
    "        organismSet.add(organismID)\n",
    "        header = str(block)\n",
    "        header = header.replace('[','').replace('(','').replace(')','').replace(']','').replace(\"'\",'').replace(\" \",'').replace(',','|')\n",
    "        f.write('>'+header+'\\n')\n",
    "        subseq = seqDic[organismID][pathStart:pathEnd]\n",
    "        if direction == '+':\n",
    "            f.write(str(subseq)+'\\n')\n",
    "        else:\n",
    "            reverseComplement = revComp(subseq)\n",
    "            f.write(str(reverseComplement)+'\\n')\n",
    "    f.close()\n",
    "    \n",
    "print(seqDic.keys())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "c17f1a07",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5 [(('G000006925', (734392, 734467)), (734392, 734467), '+'), (('G000006925', (1933117, 1933207)), (1933120, 1933195), '-'), (('G000299455', (2182743, 2182833)), (2182755, 2182830), '+'), (('G000299455', (2908810, 2908900)), (2908813, 2908888), '-'), (('G000299455', (3596501, 3596576)), (3596501, 3596576), '-')]\n",
      ">G000006925|734392|734467|734392|734467|+\n",
      "-tggcaggctgttctgcatattgccgttttcgtccgtgcacaggcgccggattctgagct\n",
      "ggatacgtggatggag\n",
      ">G000006925|1933117|1933207|1933120|1933195|-\n",
      "ctggcaggctgttctgcatattgccgtcttcatccgggcacaggcaccggattcagagct\n",
      "ggatatgtggatgga-\n",
      ">G000299455|2182743|2182833|2182755|2182830|+\n",
      "-tggcgggccacactgcatattgaggtgtttttaccggcacaggtacctgattcggagct\n",
      "cgatcagtggatggaa\n",
      ">G000299455|2908810|2908900|2908813|2908888|-\n",
      "ctggcgggccacactgcatattgaggtgtttttaccggcacaggtaccggattcagagct\n",
      "tgatcagtggatgga-\n",
      ">G000299455|3596501|3596576|3596501|3596576|-\n",
      "ctggcaggcggagctgcatattgaagttttcctgcctgctcaggtgccggattcagagct\n",
      "ggattcgtggatgga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1, 24671)), (8621, 8793), '+'), (('G000006925', (2561624, 2569713)), (2567041, 2567213), '+'), (('G000299455', (4431865, 4470166)), (4441740, 4441912), '-')]\n",
      ">G000006925|1|24671|8621|8793|+\n",
      "-ttctgatcaaactggcttctacctggcagggtatccgtgctgcagaacagctggaaaaa\n",
      "gaaggtatcaactgtaacctgaccctgctgttctccttcgctcaggctcgtgcttgtgcg\n",
      "gaggcgggcgtgttcctgatctcgccgtttgttggccgtattcttgactggta\n",
      ">G000006925|2561624|2569713|2567041|2567213|+\n",
      "-ttttgatcaaactggcttcgacctgggaaggaattcgcgcggccgaagagctggaaaaa\n",
      "gaaggtattaactgcaacctgacgctgctgttttcttttacccaggcgcgggcctgtgcg\n",
      "gaagcaggcgtttttctgatttcgccgtttgtagggcgtatttatgactggta\n",
      ">G000299455|4431865|4470166|4441740|4441912|-\n",
      "attctgatcaaactggcttctacctggcagggtatccgtgctgcagaacagctggaaaaa\n",
      "gaaggcatcaactgtaacctgaccctgctgttctccttcgctcaggctcgtgcttgtgcg\n",
      "gaagcaggcgtgttcctgatctcgccatttgttggccgtattcttgactggt-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1621184, 1626752)), (1625615, 1625674), '-'), (('G000006925', (1634479, 1637262)), (1635854, 1635913), '-'), (('G000299455', (2585656, 2594816)), (2587831, 2587889), '+'), (('G000299455', (2585656, 2594816)), (2590327, 2590385), '+'), (('G000299455', (3440781, 3470001)), (3443959, 3444018), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1152\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1152\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1621184|1626752|1625615|1625674|-\n",
      "aagcgatcgggatgattaatacgccggcggatggagcgaccgcgcaaacaggcgcgtac-\n",
      ">G000006925|1634479|1637262|1635854|1635913|-\n",
      "aacctgtcaggatgattcatccgtcggcggatagagcgcccgcgtaaacatgcccgaac-\n",
      ">G000299455|2585656|2594816|2587831|2587889|+\n",
      "-acctatcaggatgattcatccgtcggcggatagagcgcccgcgtaaacacgcgcgaac-\n",
      ">G000299455|2585656|2594816|2590327|2590385|+\n",
      "-agcgatcgggatgattaatacgccggcggatggagcgaccgcgcaaacaggcgcgtac-\n",
      ">G000299455|3440781|3470001|3443959|3444018|+\n",
      "-ggcggtccggattatagacacggcgacgcatggaacgtccacgcagacaggcgcgaacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3737540, 3737636), '-'), (('G000299455', (159106, 161990)), (161723, 161819), '+'), (('G000299455', (171875, 179671)), (179574, 179671), '+')]\n",
      ">G000006925|3736321|3788106|3737540|3737636|-\n",
      "cggcgggctgggtgagcctgtttactctgcatgttatctggccgtttctgtggatctggg\n",
      "cgacgctctatcaaccggagcgtggctggggtatgc-\n",
      ">G000299455|159106|161990|161723|161819|+\n",
      "-ggcgggctgggtgagcctgtttactctgcatgttatctggccgtttctgtggatctggg\n",
      "cgacgctctatcaaccggagcgtggctggggtatgca\n",
      ">G000299455|171875|179671|179574|179671|+\n",
      "ttgctggttgggtgagtctttttacgctgcatgttatctggccatttttgtggatttggg\n",
      "ccacgctttaccgcccggagcggggatggggaatgca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1886975, 1912610)), (1902362, 1902476), '-'), (('G000299455', (2199778, 2263225)), (2209792, 2209906), '+'), (('G000299455', (2416691, 2440660)), (2432556, 2432670), '+')]\n",
      ">G000006925|1886975|1912610|1902362|1902476|-\n",
      "cagacagcatcacagcgtcagtaccatccagaacggcgtttgctacgtccatgacttctg\n",
      "cacgcgtcggcatcgggttagtaatcattgactccatcatctgggtcgccgtga-\n",
      ">G000299455|2199778|2263225|2209792|2209906|+\n",
      "-agacagcatcacagcgtcagtaccatccagaacggcgtttgctacgtccatgacttctg\n",
      "cacgcgtcggcatcgggttagtaatcattgactccatcatctgggtcgccgtgat\n",
      ">G000299455|2416691|2440660|2432556|2432670|+\n",
      "-agacagcatcactgcatcagtaccgtcgaggatggcgtttgcaacgtcaccggcttctg\n",
      "cgcgagtcgggcgtgggtttttgatcatggaatccagcatctgggtcgcagtgat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3097173, 3097629), '-'), (('G000299455', (47809, 50193)), (49228, 49684), '+'), (('G000299455', (857924, 871846)), (857924, 858380), '+'), (('G000299455', (1977697, 1978144)), (1977701, 1978144), '+'), (('G000299455', (3140498, 3142451)), (3141943, 3142399), '+'), (('G000299455', (4771847, 4785261)), (4782809, 4783265), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3097173|3097629|-\n",
      "accacgcggagtgaataaccgttttcagagaccttgttaatcagttcactgagattaaca\n",
      "ccatcgacgtcaacgacaatgcgccccatgttcagtgcctgcacgttaacgctgtcggct\n",
      "tccagcgtcagggccagtttcattgtttcgcttccggatgcttcccccgggtaatgttat\n",
      "ttaccgttctgtagttatcgcgggtcatcaggccggttgcccggcgtgcccggagaatat\n",
      "caatgctgttgattaactgcgacggggctcctgcgctgaatcccggctggtcggtacgta\n",
      "ccagcgcgtatttttcaacgagaaagttcaccgcgtcacacagtgaaattcctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcagcgaacggtgtgtcattcagcgtgaggccatagt\n",
      "gctggtccagcagtcgggtgagcagtgtctgccaga-\n",
      ">G000299455|47809|50193|49228|49684|+\n",
      "-ccacgcggagtgaataaccgttttcagagaccttgttaatcagttcactgagattaaca\n",
      "ccatcgacgtcaacgacaatgcgccccatgttcagtgcctgcacgttaacgctgtcggct\n",
      "tccagcgtcagggccagtttcatcgttttgcctccggatgcttgccctgggtaatgttat\n",
      "ttaccattctgtaattgttccgggtcatcaggccggtcgccctgcgagcccggagaatat\n",
      "caatgctgttgattaactgcgatggggctcctgcgctgaatcccggctggtcggtacgta\n",
      "ccagcgcgtatttttcaacgagaaagttcaccgcgtcacacagtgaaatccctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcagcgaacggtgtgtcattcagcgtgaggccatagt\n",
      "gctggtccagcagtcgggtgagcagtgtctgccagat\n",
      ">G000299455|857924|871846|857924|858380|+\n",
      "-ccacgcggagtgaataaccgttttcagcgactttgttaatcagttcagcgagctcaata\n",
      "ccatcgacgtcaacgacaatgcgccccatgttcagcgcctgcgcattaatgctgtcggct\n",
      "tccagcgtcagggccagtttcattgtttcgcccccggatgtttacccagggtaatgttat\n",
      "ttaccgttctgtagttgtcgcgtgtcatcaggccggttgcccggcgtgcccggagaatat\n",
      "caatgctgttgattaactgcgacggggctcctgcgctgaatcccggctggtcggtacgta\n",
      "ccagcgcgtatttttcaacgagaaagttcaccgcgtcacacagtgaaatccctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcagcgaacggtgtgtcattcagcgtgaggccatagt\n",
      "gctggtccagcagtcgggtgagcagtgtctgccagat\n",
      ">G000299455|1977697|1978144|1977701|1978144|+\n",
      "-ccacgcggagtgaataaccgttttcagcggccttattaatcagttcagtgatattaaca\n",
      "ccatcaatgtcaacgacaatgtgccccatatccagcgcctgtacgttaacgctgtcggct\n",
      "tccagtg------------tcatcatttcgcctccggatacttacccagggtaatgttat\n",
      "ttaccgttctgtaattgtcgcgggtcatcaggccggtcgccctgcgagcccggaggatat\n",
      "cgatgctgtttattaactgagagcgggtacaggcgctgaatcccggctggtcggta-gca\n",
      "ccagcgcgtatttttccacgagaaagttcaccgcatcacacagtgaaatgcctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcggcaaaaggtgtgtcattcagtgtgaggccatagt\n",
      "gctggtccagcagtcgggacagcagtatctgccagat\n",
      ">G000299455|3140498|3142451|3141943|3142399|+\n",
      "-caacactaagggagtagccgttatcgcagaccatgttaatcagttcagcgagattaaca\n",
      "ccatcgacgtcaacgacaattcgccccatattcagtgcctgtacgttaacgctgtcggct\n",
      "tccagtgtcagggccagtttcatcatttcgcctccggatgcttacccagggtaatgttat\n",
      "ttaccgttctgtaattgtcgcgggtcatcaggccggtcgccctgcgagcccggaggatat\n",
      "cgatgctgtttattaactgagagcgggtacaggcgctgaatcccggctggtcggtacgca\n",
      "ccagcgcgtatttttccacgagaaagttcaccgcatcacacagtgaaatgcctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcagcgaacggtgtgtcattcagtgtgaggccatagt\n",
      "gctggtccagcagacggatgagcagtgtctgccagat\n",
      ">G000299455|4771847|4785261|4782809|4783265|-\n",
      "accacgcggagtgaataaccgttttcagagaccttgttaatcagttcactgagattaaca\n",
      "ccatcgacgtcaacgacaatgcgccccatgttcagtgcctgcacgttaacgctgtcggct\n",
      "tccagcgtcagggccagtttcattgtttcgcttccggatgcttcccccgggtaatgttat\n",
      "ttaccgttctgtagttatcgcgggtcatcaggccggttgcccggcgtgcccggagaatat\n",
      "caatgctgttgattaactgcgacggggctcctgcgctgaatcccggctggtcggtacgta\n",
      "ccagcgcgtatttttcaacgagaaagttcaccgcgtcacacagtgaaattcctgcctcaa\n",
      "tatgctgctcaatcacacgttcatcagcgaacggtgtgtcattcagcgtgaggccatagt\n",
      "gctggtccagcagtcgggtgagcagtgtctgccaga-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (64491, 216702)), (118929, 118958), '+'), (('G000006925', (496417, 503320)), (501067, 501096), '-'), (('G000299455', (3819953, 3834467)), (3825070, 3825099), '+'), (('G000299455', (4300454, 4333834)), (4329548, 4329577), '-')]\n",
      ">G000006925|64491|216702|118929|118958|+\n",
      "-ccgcagtcagtaccacgatgttcagcgca\n",
      ">G000006925|496417|503320|501067|501096|-\n",
      "gatgccactagaatgacgaagttcagcgc-\n",
      ">G000299455|3819953|3834467|3825070|3825099|+\n",
      "-atgctaccagaatgacgaagttcagcgca\n",
      ">G000299455|4300454|4333834|4329548|4329577|-\n",
      "gccgcagtcagtaccacgatgttcagcgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3736324, 3737008), '+'), (('G000299455', (162034, 163089)), (162402, 163086), '-'), (('G000299455', (180218, 180902)), (180218, 180902), '-')]\n",
      ">G000006925|3736321|3788106|3736324|3737008|+\n",
      "-atgatccaaataaagataatgcatccaactggtcattcttagcagcactttccgcatca\n",
      "ccgaaacatggctgaaatggtcggagtaaaccgccacctgggcgtagatgccgtcgggta\n",
      "aggcatcgatatcatcgttagggtccagttcaatggttcccagcacaccgtccgtgccgg\n",
      "gaacgaccgttaatgattgcaataccccctgcgcctgataagaaccgcctggcacgacag\n",
      "gtaaaatactagtcaatttgccgtggaacacctgcccaggtagcgcgttaaacaccactt\n",
      "ccgcatcatcgccaggtttcagacgtaacagcgagttttgccgaaattgggcgacaattt\n",
      "gccgtttttgctcggggatgaagaccatcaccggacgcagcggcaaggcagctgcgtatg\n",
      "tacctgggcggatcagtacctgagtgacgtagccattgcttggtgcgcgaatgacagtct\n",
      "gctcaagattatattttgcttcagtaagttgcgctcttaagctcacaatctgcgattgct\n",
      "cgccgttgaccatactgtcgagctgactctggatctgcgcctgctccgccaccgagcctt\n",
      "tcactaacgcatcctgcgcgaggaaattttgccgcgcatcgtcgatgtcacgttccgaga\n",
      "acggattcaccgccgcctggctgcc\n",
      ">G000299455|162034|163089|162402|163086|-\n",
      "aatgatccaaataaagataatgcatccagctggtcattcttagcagcactttccgcatca\n",
      "ccgaaacatggctgaaatggtcggagtaaaccgccacctgggcgtagatgccgtcgggta\n",
      "aggcatcgatatcatcgttagggtccagttcaatggttcccagcacaccgtccgtgccgg\n",
      "gcacgaccgttaatgattgcaataccccctgcgcctgataagaaccgcctggcacgacag\n",
      "gtaaaatactagtcagtttgccgtgaaacacctgcccaggtagcgcgttaaacaccactt\n",
      "ccgcatcatcgccgggtttcagacgtaacagcgagttttgccgaaattgggcgacaattt\n",
      "gccgtttttgctcggggatgaagaccatcaccggacgcagcggcaaggcagctgcgtatg\n",
      "tacctgggcggatcagtacctgagtaacgtagccattgctcggcgcgcgaatgacagtct\n",
      "gctcaaggttatattttgcttcagtaagttgcgctcttaagctcacaatctgcgattgct\n",
      "cgccgttaaccatactgtcgagctggctctggatctgcgcctgctccgccaccgagcctt\n",
      "tcaccagcgcatcctgcgcgaggaaattttgccgcgcatcgtcgatgtcacgttccgaga\n",
      "acggattcaccgccgcctggctgc-\n",
      ">G000299455|180218|180902|180218|180902|-\n",
      "aatgatcgagataaaggtaatgcacccagctggtcatgcgtaatagcactttgcgcatca\n",
      "ccgagacatggctgaaatgatcagagtaaaccgccacctgggcgtaaataccgtctggta\n",
      "aagcgctcaaatcagtgtgctcatccagttcaatggtcgcgataacgccatctgaacccg\n",
      "gcgctgtgtttaacgtctgtaaggtgccggtcgactgataagctccgccgggaacggctg\n",
      "gactaatggctgccagcttaccgctgaataccttacctggcagagcattaaacaccactt\n",
      "ccgcatcgtcgccaggagccagtcgcagcaaggagttctgacggaactgcgccacgattt\n",
      "gtcgtttctgatcgggtataaacaccatcaccggacgtagcggcagcgacgcggcatagg\n",
      "tacccggacgaatcagtacctgggtcacataaccatcgctcggcgcacgcactatcgtct\n",
      "gctcaaggttatattttgcttccgcgagctgtgctttcaggctggcgatttgagaatgtt\n",
      "cacccaacaccaggctatccagctggctctggatctgtttttgttccgccgccgatgact\n",
      "tcacggaggcttcctgcgccagataattttgccgcgccacatcgatatcgcgttctgaaa\n",
      "acgggtttactttcgcctggctgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4289725, 4290419), '-'), (('G000299455', (1030502, 1031792)), (1030502, 1031196), '+'), (('G000299455', (4836149, 4864500)), (4859974, 4860668), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 1060\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4288513|4315006|4289725|4290419|-\n",
      "attagatagcccagccacccgcatagaaggtcaccagcgccacggcgataacgaccgtac\n",
      "cgatgttcaacttgcgccattcaccggaaaccagacgaccaatcaccagagtcgcgaagc\n",
      "cgatcatgatgcctgttacgatgttacaggtcagcacgatgaataccgccgtaaccagac\n",
      "ccgccatcgcatcaacaaaatcagcaaagtcgattttcgccacgttgctcagcatcagca\n",
      "ggccaacgtacatcagcgccggagccgttgcataccccggaacgaggtaagaaagcggag\n",
      "agaggaacagaatcagcaggaacagcacgccaacggtgatagccgtcaacccggttttac\n",
      "cgcccgccgccgtacccgccgcagactcgatgtataccgctgccggagctgcacccacca\n",
      "ggccagagaaaacgctgctcatggagtcagtggtcagtgctttcccgccgtcgatgatct\n",
      "gcccatctttatccagcaggttcgcctggccggcgacagcgcggatagttccggtggcat\n",
      "caaatactgccgtcatcaccagcgccagaacgcttggcaggactacaggattcagcgcgc\n",
      "ccataatgtccaggctgccaatcagtgaattgccgttttcatcgctcaatgaaggcatgg\n",
      "cgaaaacgccggagaaatggacgttaggatcgaa-\n",
      ">G000299455|1030502|1031792|1030502|1031196|+\n",
      "-ttagattgcccaaccacccgcgtaaaatgcgaccagtgcggcagtaatgatcaccgtac\n",
      "caatattcaacttttgccattcgcgagcaaagacacggcctacgaccagtgtcacaaagc\n",
      "ccagcataataccggtaacgatattacaagtcagaacgatgaacacggcacacaccaggc\n",
      "cagccattgcgtcaataaaatcgttgaaatccagcttcgagacgttacttaacatcagca\n",
      "aacccacgtacatcagagcgggtgcagtggcgtaaccagggatcaaaaatgacagcggtg\n",
      "ataagaacagaatcagcaggaataacgcccccactacggttgcggtcaaccctgttttac\n",
      "ccccggcggcggttcctgccgctgattcgatataaaccgctgcgggcgctgcgcccacca\n",
      "gaccggagaatattgaacttactgagtcactggtcagggctttgccgccattgatgatct\n",
      "ggttgtctttatccaacaaattcgcctgaccggcgacggcacggatggtgccagtggcgt\n",
      "cgaacactgcggtcatcaccaatgccagtacactcggaagtacggttggctggagtgcgc\n",
      "ccataatatcgaggctgaaaatcagagacttaccatcttcgccagtcagacttggcatcg\n",
      "ccaccagaccgtggtatttcaccgctgggtcaaag\n",
      ">G000299455|4836149|4864500|4859974|4860668|+\n",
      "-ttagatagcccagccacccgcatagaaggtcaccagcgccacggcgataacgaccgtac\n",
      "cgatgttcaacttgcgccattcgccggaaaccagacgaccaatcaccagagtcgcgaagc\n",
      "cgatcatgatgcctgttacgatgttacaggtcagcacgatgaataccgccgtaaccagac\n",
      "ccgccatcgcatcaacaaaatcagcaaagtcgattttcgccacgttgctcagcatcagca\n",
      "ggccaacgtacatcagcgccggagccgttgcataccccggaacgaggtaagagagcggag\n",
      "agaggaacagaatcagcaggaacagcacgccaacggtgatagccgtcaaaccggttttac\n",
      "cgcccgccgccgtacccgccgcagactcgatgtataccgctgccggagctgcacccacca\n",
      "ggccagagaaaacgctgctcatggagtcagtggtcagtgctttcccaccgtcgataattt\n",
      "gcccatctttatccagcaggttcgcctggccggcgacggcacggatagttccggtggcat\n",
      "caaatactgccgtcattaccagcgccagaacgcttggcaggactacaggattcagcgcgc\n",
      "ccataatgtccaggctgccaatcagtgagttgccgttttcatccctcaatgaaggcatgg\n",
      "cgaaaacgccggagaaatggacgttaggatcgaag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4281059, 4281090), '+'), (('G000299455', (3150858, 3151017)), (3150858, 3150889), '+'), (('G000299455', (4795454, 4823864)), (4819943, 4819974), '+')]\n",
      ">G000006925|4256570|4288375|4281059|4281090|+\n",
      "tgcagtaccggttccttttgcccgggcgtat\n",
      ">G000299455|3150858|3151017|3150858|3150889|+\n",
      "tggaggaccggttccgtttttccgggaacgt\n",
      ">G000299455|4795454|4823864|4819943|4819974|+\n",
      "tgcagtaccggttccttttgcccgggcgtat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4315393, 4315636), '+'), (('G000299455', (4865050, 4904182)), (4865417, 4865660), '+'), (('G000299455', (5406554, 5406797)), (5406554, 5406797), '-')]\n",
      ">G000006925|4315026|4354132|4315393|4315636|+\n",
      "-acgggtacgcagctgaccttcgatataaacctgagaacctttacgcagatattcgctcg\n",
      "ccacttctgccagtttgccgaacagcacaacgcggtgccattcagtctgttctttcatct\n",
      "cgccggtcgctttatcacgccaggattcggaagtagccagcgtaatgttggcaactgcgc\n",
      "caccatttggcatgtagcgtacttccgggtcctgacccagattaccaacgagaataacct\n",
      "tgtt\n",
      ">G000299455|4865050|4904182|4865417|4865660|+\n",
      "-acgggtacgcagctgaccttcgatataaacctgagaacctttacgcagatattcgctcg\n",
      "ccacttctgccagtttgccgaacagcacaacgcggtgccattcagtctgttctttcatct\n",
      "cgccggtcgctttatcacgccaggattcggaagtagccagcgtaatgttggcaactgcgc\n",
      "caccatttggcatgtagcgtacttccgggtcctgacccagattaccaacgagaataacct\n",
      "tgtt\n",
      ">G000299455|5406554|5406797|5406554|5406797|-\n",
      "tacgggtacggagttgcccctcgatgtagacctgcgcgcccttgcgcagatattcaccag\n",
      "ccacttccgcgagcttgccgaacagcaccacgcggtgccattccgtctgctcccgcatct\n",
      "cccccgtctgtttgtcgcgccagctttctgacgtggccacctgcaggtttgccactgcgc\n",
      "ccccgttggggatgtaacggacttccggatcattgcccagacgcccgacgaggatgacct\n",
      "tgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3065287, 3076770)), (3076018, 3076225), '+'), (('G000299455', (63990, 64641)), (64434, 64641), '-'), (('G000299455', (874911, 875768)), (875356, 875562), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3065287|3076770|3076018|3076225|+\n",
      "-cgccttctgttgctcactcagggcatgtaaatcatgctcaaaccacgccccctgagcat\n",
      "ctgtctgcagaatcaaccggccatggcaggaaaggcagaaacactgccggatatttctgc\n",
      "tgagcgcagagtcggcactgataatgtgctcacccggcgtgatccccagccccgttttta\n",
      "taccgttcattctgctcccccctcctca\n",
      ">G000299455|63990|64641|64434|64641|-\n",
      "ccgccttctgttgcgcactcagggcatgtaaatcatgctcaaaccacgccccttgagcat\n",
      "ctgtctgcaaaatcaaccggccacgacaggaaaggcagaaacaatgccggatatttctgc\n",
      "tgagcgcagagttggcactgataatgtgctcacccggcgtgatccccagccccattttta\n",
      "tgccgttcatcctgctaccacctcctc-\n",
      ">G000299455|874911|875768|875356|875562|-\n",
      "ccgccttctgttgcgcgctcagggcatgtaaatcgtgctcaaaccacgccccctgagcat\n",
      "ctgtctgcaaaatcaaccggccacgacaggaaagacagaaacaatgccggatatttctgc\n",
      "tgagcgcagaatcggcactgataatgtgctcacccggcgtgat-cccagccccgttttta\n",
      "taccgttcattctgcaaccacctcctc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4629818, 4630996)), (4630921, 4630996), '+'), (('G000006925', (4708951, 4710002)), (4709238, 4709313), '-'), (('G000299455', (917546, 918643)), (917879, 917954), '-'), (('G000299455', (4770510, 4771607)), (4770843, 4770918), '-')]\n",
      ">G000006925|4629818|4630996|4630921|4630996|+\n",
      "-caccggttccatccttcgctgtggcagttcggcaaggagatgggcttctctccccgact\n",
      "gtgtcgccccttcagg\n",
      ">G000006925|4708951|4710002|4709238|4709313|-\n",
      "gcaccggttccatccttcgctgtggcagttcggcaaggagatgggcttctctccccgact\n",
      "gtgtcgccccttcag-\n",
      ">G000299455|917546|918643|917879|917954|-\n",
      "gcaccggttccatccttcgctgtggcagttcggcaaggagatgggcttctctccccgact\n",
      "gtgtcgccccttcag-\n",
      ">G000299455|4770510|4771607|4770843|4770918|-\n",
      "gcaccggttccatccttcgctgtggcagttcggcaaggagatgggcttctctccccgact\n",
      "gtgtcgccccttcag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4362175, 4367702)), (4365395, 4365498), '+'), (('G000299455', (3179480, 3179583)), (3179480, 3179583), '-'), (('G000299455', (4524008, 4535561)), (4527228, 4527331), '+')]\n",
      ">G000006925|4362175|4367702|4365395|4365498|+\n",
      "-atattgaaaccaaatggctttatgattttctgactctggaaaaatgccgcaatttttcc\n",
      "caggcggcagtcagtcgcaacgtctcgcaaccggcattcagccg\n",
      ">G000299455|3179480|3179583|3179480|3179583|-\n",
      "aatatcgagacaaaatggttatacgatttcctgacactggaagcctgtcgccatttttct\n",
      "caggcggcaaaagagcgtaatctttcgcaacccgcattcagcc-\n",
      ">G000299455|4524008|4535561|4527228|4527331|+\n",
      "-atattgaaactaaatggctttatgattttctgaccctggaaaaatgccgcaatttttcc\n",
      "caggcagcagtcagtcgcaacgtctcgcaaccggcattcagccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1, 24671)), (13329, 13378), '+'), (('G000006925', (2641574, 2668966)), (2642762, 2642811), '-'), (('G000299455', (4431865, 4470166)), (4437155, 4437204), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|13329|13378|+\n",
      "-tgctgctggacgttaccccgctgtctctgggtatcgaaaccatgggcgg\n",
      ">G000006925|2641574|2668966|2642762|2642811|-\n",
      "ctgctgcttgatgtgatcccactgtcgctgggcctcgaaacgatgggcg-\n",
      ">G000299455|4431865|4470166|4437155|4437204|-\n",
      "ctgctgctggacgttaccccgctgtctctgggtatcgaaaccatgggcg-\n",
      "\n",
      "\n",
      "28 [(('G000006925', (251358, 252232)), (252183, 252232), '+'), (('G000006925', (315068, 315858)), (315068, 315118), '-'), (('G000006925', (332059, 337870)), (332059, 332109), '-'), (('G000006925', (493019, 493813)), (493019, 493069), '-'), (('G000006925', (509301, 510374)), (509301, 509351), '-'), (('G000006925', (724190, 725876)), (724190, 724240), '-'), (('G000006925', (1094884, 1095679)), (1094884, 1094934), '-'), (('G000006925', (1099363, 1100153)), (1100103, 1100153), '+'), (('G000006925', (1411510, 1412350)), (1411510, 1411560), '-'), (('G000006925', (1883764, 1885448)), (1883764, 1883814), '-'), (('G000006925', (1956633, 1961076)), (1956633, 1956683), '-'), (('G000006925', (2084425, 2085215)), (2085165, 2085215), '+'), (('G000006925', (2086041, 2087310)), (2087261, 2087310), '+'), (('G000006925', (2236395, 2237421)), (2237371, 2237421), '+'), (('G000006925', (3065287, 3076770)), (3074763, 3074813), '-'), (('G000006925', (3812816, 3813607)), (3812816, 3812866), '-'), (('G000006925', (4639759, 4640549)), (4639759, 4639809), '-'), (('G000006925', (4650084, 4650878)), (4650828, 4650878), '+'), (('G000006925', (4656735, 4658205)), (4658155, 4658205), '+'), (('G000006925', (4673259, 4674053)), (4674003, 4674053), '+'), (('G000006925', (4679888, 4680760)), (4679888, 4679938), '-'), (('G000006925', (4750261, 4751054)), (4751004, 4751054), '+'), (('G000006925', (4760730, 4761525)), (4761475, 4761525), '+'), (('G000006925', (4793569, 4794365)), (4794315, 4794365), '+'), (('G000299455', (906289, 909876)), (907039, 907088), '+'), (('G000299455', (4752701, 4766669)), (4753451, 4753500), '+'), (('G000299455', (4771847, 4785261)), (4779708, 4779757), '-'), (('G000299455', (5275700, 5279438)), (5278488, 5278538), '-')]\n",
      ">G000006925|251358|252232|252183|252232|+\n",
      "--ttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|315068|315858|315068|315118|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|332059|337870|332059|332109|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|493019|493813|493019|493069|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|509301|510374|509301|509351|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|724190|725876|724190|724240|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|1094884|1095679|1094884|1094934|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|1099363|1100153|1100103|1100153|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|1411510|1412350|1411510|1411560|-\n",
      "agttcgtggcagaacgacctgaacagctgtgggtggctgattttacttac-\n",
      ">G000006925|1883764|1885448|1883764|1883814|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|1956633|1961076|1956633|1956683|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|2084425|2085215|2085165|2085215|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|2086041|2087310|2087261|2087310|+\n",
      "--ttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|2236395|2237421|2237371|2237421|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|3065287|3076770|3074763|3074813|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|3812816|3813607|3812816|3812866|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|4639759|4640549|4639759|4639809|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|4650084|4650878|4650828|4650878|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|4656735|4658205|4658155|4658205|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|4673259|4674053|4674003|4674053|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|4679888|4680760|4679888|4679938|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000006925|4750261|4751054|4751004|4751054|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|4760730|4761525|4761475|4761525|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000006925|4793569|4794365|4794315|4794365|+\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000299455|906289|909876|907039|907088|+\n",
      "--ttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000299455|4752701|4766669|4753451|4753500|+\n",
      "--ttcgtggcagaacgacctgaccagctgtgggtggctgattttacttacg\n",
      ">G000299455|4771847|4785261|4779708|4779757|-\n",
      "-gttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      ">G000299455|5275700|5279438|5278488|5278538|-\n",
      "agttcgtggcagaacgacctgaccagctgtgggtggctgattttacttac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (201797, 202413), '-'), (('G000299455', (4235244, 4293324)), (4239812, 4240428), '+'), (('G000299455', (4789263, 4790612)), (4789997, 4790612), '-')]\n",
      ">G000006925|64491|216702|201797|202413|-\n",
      "aatcagcttatggatcccttgcgccagatcctgaatacgcatattgcggtagaaatcggg\n",
      "atcttcagcatagagatcgggcagcatatttttgatccgcaggttgagatcgtaagagcg\n",
      "cttaaattccgtcaacccacgcaataatcccattgctttggttttatcgatgccaatact\n",
      "aaagagaaacagcaggttataagggccggttttctctactacgatcccacgttcgtcgag\n",
      "gaattttgctaccagcgccgccgggatcccctcctcactcatattgccctgttcgtccat\n",
      "ccccggtgtcaaaatagtgactttaaccggatcgagaaacatatgatcggcatccgcatc\n",
      "gttaaagccgtgccattgttcgccaggcgcaacgggcctgcattcggcttcatccacctg\n",
      "cggcggttgccagatatcgaaaaaccaaccgtcagactcttcccgcagccgctggacctc\n",
      "tttgcgaaaatgcagcgcccgctccaccgaacggttaatcagccgtttgcccggattacc\n",
      "acgcagcatcgccgccgccgtcttaaccgaagcaacaatgggataactgggcgaggtggt\n",
      "ggtatgcatcataaag-\n",
      ">G000299455|4235244|4293324|4239812|4240428|+\n",
      "-atcagcttatggatcccttgcgccagatcctgaatacgcatattgcggtagaaatcggg\n",
      "atcttcagcatagagatcgggtagcatatttttgatccgcaggttgagatcgtaagagcg\n",
      "tttaaattccgtcaacccacgcaataatcccattgctttggttttatcgataccaatact\n",
      "aaagagaaacagcaggttataagggccggttttctctactacgatcccacgttcgtcaag\n",
      "gaattttgccaccagcgccgccgggatcccctcctcactcatattgccctgttcgtccat\n",
      "ccccggtgtcaaaatagtgactttaaccggatcgagaaacatatgatcggcatccgcatc\n",
      "gttaaagccgtgccactgttcgcctggcgcaacgggccagcattcggcttcatccacctg\n",
      "cggcggttgccagatatcgaaaaaccaaccgtcagactcttcccgcaaccgctggacctc\n",
      "tttgcgaaaatgcagcgcccgctccaccgaacggttaatcagccgtttgcccggattacc\n",
      "acgcagcatcgccgccgccgtctcaaccgaagcaacaatgggataactgggcgaggtggt\n",
      "ggtatgcatcataaagg\n",
      ">G000299455|4789263|4790612|4789997|4790612|-\n",
      "aatcagtttgtggatattctgagccagttcctgaatacgcatgttttcatagaattcagg\n",
      "atcttcacgatacagagacggcagcatgtttttcacacgcaggttcaggtcgaacgcacg\n",
      "tttgaagtcagtcagagcacgcagcaggctcagtgctttggtcttatcgataccgatgct\n",
      "gaacaggaacagcaggttatacggaccggttttctcaacaacgatgccatgttcgtcgag\n",
      "gtatttcgccacgatgctggccggaataccaaagtcgctcatggtgccgtctttttccat\n",
      "ccccggagtcagcagggtgactttgatcgggtcaagatacatgtgctcgttatcgatgtt\n",
      "tttgaagccgtgccaggtgctgtcagaacgcagcggccagcattcagtcgtatcgatatg\n",
      "atccggctgccatacatcaaagaaccagccatcagattccgttctcagacgtttgatctc\n",
      "tttacggaatttgatcgcacgttcaatggaaccgttgatcagacgtttacctgcattgcc\n",
      "tttcatcatcgccgcagcggtttcagtggacgccacgataccgtagtgcggagaagtggt\n",
      "ggtgtgcatcatgta--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1122423, 1168132)), (1146245, 1146288), '-'), (('G000299455', (3057394, 3103258)), (3079305, 3079348), '+'), (('G000299455', (3697938, 3726006)), (3720969, 3721012), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1122423|1168132|1146245|1146288|-\n",
      "ccgtggaaaacctgacctgctgcgttggtgtattcaccaatct-\n",
      ">G000299455|3057394|3103258|3079305|3079348|+\n",
      "-cgtggaaaacctgacctgctgcgttggtgtattcaccaatctg\n",
      ">G000299455|3697938|3726006|3720969|3721012|+\n",
      "-cgtggaaaaccgtacccgccgcgttggtgaattcaccaatctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2816684, 2833035)), (2824179, 2824284), '-'), (('G000299455', (1207004, 1224877)), (1213623, 1213728), '+'), (('G000299455', (1411666, 1461912)), (1445372, 1445477), '+')]\n",
      ">G000006925|2816684|2833035|2824179|2824284|-\n",
      "ctctcggtcggcgaccatttttttcagatcgcggttagtcgcggcaattaaacgtacgtc\n",
      "cgtctgaatgattttgttgctgccgagacgttcaaactcctgttc-\n",
      ">G000299455|1207004|1224877|1213623|1213728|+\n",
      "-tcacggtcggcgaccatttttttcagatcgcggttagtcgcggcgattagacgcacgtc\n",
      "cgtctgaatgattttgttgctgccgagacgttcaaactcctgttcc\n",
      ">G000299455|1411666|1461912|1445372|1445477|+\n",
      "-tggcgatcttcaaccatttgccacaaatcacggttggtggcggcaatgactctgacatt\n",
      "caccgggatcgttctactcccgccgagacgctcaatctcccgctcc\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3098110, 3098275), '+'), (('G000299455', (47809, 50193)), (48582, 48747), '-'), (('G000299455', (856582, 857833)), (857561, 857726), '-'), (('G000299455', (1977430, 1977596)), (1977430, 1977596), '-'), (('G000299455', (3140498, 3142451)), (3141305, 3141470), '-'), (('G000299455', (4771847, 4785261)), (4783746, 4783911), '+')]\n",
      ">G000006925|3083675|3099254|3098110|3098275|+\n",
      "-agcactgctgccctgtattgaacaggccagggaagatatccgtaccctgcgtcagctgc\n",
      "agtttctgcaacagaaccggtgagtcttactcaccagacttca-ttccactgacgccagc\n",
      "ctgaatacggctggcgttttcgtttatctgcaaaaaggaatatcgat\n",
      ">G000299455|47809|50193|48582|48747|-\n",
      "cagcactgctgccctgtattgaacaggcctgggaagatatccgtaccctgcgccagttgc\n",
      "agcttccagaccagaaccggtgagtctcattcaccatacttca-ttccactgacgccagc\n",
      "ctgaacagggctggcgttttcatttatctgcgaaaaggaatatcga-\n",
      ">G000299455|856582|857833|857561|857726|-\n",
      "tagcactgttgccctgtatcgaacaggcccgggaggatgtccgtactcttcgtcatctgc\n",
      "agcttctgcaccagaaccggtaaatatcactcaccatacttca-ttccactgacgtcagc\n",
      "ctgaacagggctggcgttttcgtttataaaccaaaagggaaaccga-\n",
      ">G000299455|1977430|1977596|1977430|1977596|-\n",
      "cagcactgttgtcctgtattaagcaggcaagagaagatgtccttaccctgcgccatctgc\n",
      "agcttctgcaccagaaccggtgaatctcactcaacaggcttcatttcccctgacgccagc\n",
      "ctgaatacagctggcgttttcatttataaacagaaaggaaaaccga-\n",
      ">G000299455|3140498|3142451|3141305|3141470|-\n",
      "cagcactgttgccctgtattgaacaggccagggaggatatccgtaccctgcgccaactgc\n",
      "agcttctgcaccagaaccggtgagtctcactcaccatacttca-ttccactgacgccggc\n",
      "ctgaacagggctggcgttttcatttatctgcgaaaaggaatatcga-\n",
      ">G000299455|4771847|4785261|4783746|4783911|+\n",
      "-agcactgctgccctgtattgaacaggccagggaagatatccgtaccctgcgtcagctgc\n",
      "agtttctgcaacagaaccggtgagtcttactcaccagacttca-ttccactgacgccagc\n",
      "ctgaatacggctggcgttttcgtttatctgcaaaaaggaatatcgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1578298, 1580868)), (1579615, 1579906), '+'), (('G000299455', (2163242, 2163548)), (2163257, 2163548), '-'), (('G000299455', (2927401, 2931337)), (2928728, 2929019), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1074\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1578298|1580868|1579615|1579906|+\n",
      "-cacgccagttgcaaaggcgtgatcaagaaaacgaaaaattaaatcaacctgagaaccat\n",
      "gcttttcttcgaacgccagaggatccgcatgaagctcgttgtgatgctcccgacacagcg\n",
      "gtagcgtgaaaatatcgtgagatttggtccccattccgccctgaccatgaccaatcaggt\n",
      "gatggggatcgtcggctggcttaccacaacacgcacacggctgtgtcttcacccagcgtt\n",
      "tgtatttctcgttaacccagcggcgacgtttaggtcgtttcatgaaagattc\n",
      ">G000299455|2163242|2163548|2163257|2163548|-\n",
      "gcacgccaattgccagtgcacgatcgataaaacgaaatatcagctccagctgggagccat\n",
      "acttctcttcaaatgccacggtatccgcatgcagctcgtcgtgatgctttctgcacaaag\n",
      "gcaacacaaagaggtcatgcgcttttgtagccattccaccctgaccgtgacctatcaggt\n",
      "ggtggggatcatcagcaggttttccacaacatgcgcacggctgcgtcttaacccagcgcg\n",
      "tgtacttttcattaacccagcggcgacgtttggggcgtaacataaaagact-\n",
      ">G000299455|2927401|2931337|2928728|2929019|+\n",
      "-cacgccagttgcaaaggcgtgatcaagaaaacgaaaaattaaatcaacctgagaaccat\n",
      "gcttttcttcgaacgccagaggatccgcatgaagctcgttgtgatgctcccgacacagcg\n",
      "gtagcgtgaaaatatcgtgagattttgtccccattccgccctgaccatgaccaatcaggt\n",
      "gatggggatcgtcggctggcttaccacaacacgcacacggctgtgtcttcacccagcgtg\n",
      "tgtatttctcgttaacccagcggcgacgtttaggtcgtttcatgaaagattc\n",
      "\n",
      "\n",
      "36 [(('G000006925', (236537, 236664)), (236561, 236662), '-'), (('G000006925', (265824, 265927)), (265827, 265927), '+'), (('G000006925', (495335, 495438)), (495338, 495438), '+'), (('G000006925', (706641, 706744)), (706644, 706744), '+'), (('G000006925', (864658, 864785)), (864659, 864760), '+'), (('G000006925', (911961, 912064)), (911962, 912064), '+'), (('G000006925', (980497, 980624)), (980499, 980599), '+'), (('G000006925', (1009400, 1009503)), (1009400, 1009500), '-'), (('G000006925', (1098479, 1098582)), (1098479, 1098580), '-'), (('G000006925', (1208359, 1208486)), (1208361, 1208461), '+'), (('G000006925', (1392468, 1392595)), (1392469, 1392570), '+'), (('G000006925', (1417818, 1417921)), (1417819, 1417921), '+'), (('G000006925', (1500992, 1501119)), (1501016, 1501117), '-'), (('G000006925', (1610724, 1610827)), (1610725, 1610827), '+'), (('G000006925', (1619381, 1619482)), (1619381, 1619482), '-'), (('G000006925', (1695541, 1695644)), (1695542, 1695644), '+'), (('G000006925', (1806835, 1806938)), (1806836, 1806938), '+'), (('G000006925', (2035520, 2035647)), (2035521, 2035622), '+'), (('G000006925', (2360205, 2360306)), (2360205, 2360306), '-'), (('G000006925', (2691556, 2691659)), (2691556, 2691656), '-'), (('G000006925', (2770126, 2770229)), (2770127, 2770229), '+'), (('G000006925', (2963697, 2963800)), (2963698, 2963800), '+'), (('G000006925', (3080301, 3080404)), (3080301, 3080402), '-'), (('G000006925', (3179206, 3179309)), (3179206, 3179307), '-'), (('G000006925', (3514083, 3514186)), (3514083, 3514184), '-'), (('G000006925', (3605511, 3605614)), (3605511, 3605612), '-'), (('G000006925', (3814994, 3815097)), (3814995, 3815097), '+'), (('G000006925', (3830887, 3830990)), (3830888, 3830990), '+'), (('G000006925', (3915487, 3915590)), (3915487, 3915588), '-'), (('G000006925', (3993365, 3993468)), (3993366, 3993468), '+'), (('G000006925', (4116789, 4116916)), (4116790, 4116891), '+'), (('G000006925', (4255689, 4255816)), (4255690, 4255791), '+'), (('G000006925', (4361401, 4361504)), (4361401, 4361502), '-'), (('G000006925', (4694651, 4694754)), (4694652, 4694754), '+'), (('G000006925', (4777914, 4778017)), (4777915, 4778017), '+'), (('G000299455', (5279601, 5279746)), (5279624, 5279725), '-')]\n",
      ">G000006925|236537|236664|236561|236662|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|265824|265927|265827|265927|+\n",
      "--gatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|495335|495438|495338|495438|+\n",
      "--gatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|706641|706744|706644|706744|+\n",
      "--gatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|864658|864785|864659|864760|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|911961|912064|911962|912064|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|980497|980624|980499|980599|+\n",
      "-cgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1009400|1009503|1009400|1009500|-\n",
      "-cgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1098479|1098582|1098479|1098580|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1208359|1208486|1208361|1208461|+\n",
      "-cgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1392468|1392595|1392469|1392570|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1417818|1417921|1417819|1417921|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|1500992|1501119|1501016|1501117|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|1610724|1610827|1610725|1610827|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|1619381|1619482|1619381|1619482|-\n",
      "acggtcacagcagtccagcgcgaacgtgactcgcagtttttctccgttatcacagcggaa\n",
      "ctcgaacccgtcagagcaccatcgctgattactttctttca-\n",
      ">G000006925|1695541|1695644|1695542|1695644|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|1806835|1806938|1806836|1806938|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|2035520|2035647|2035521|2035622|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|2360205|2360306|2360205|2360306|-\n",
      "acggtcacagcagtccagcgcgaacgtgactcgcagtttttctccgttatcacagcggaa\n",
      "ctcgaacccgtcagagcaccatcgctgattactttctttca-\n",
      ">G000006925|2691556|2691659|2691556|2691656|-\n",
      "-cgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|2770126|2770229|2770127|2770229|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|2963697|2963800|2963698|2963800|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|3080301|3080404|3080301|3080402|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|3179206|3179309|3179206|3179307|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|3514083|3514186|3514083|3514184|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|3605511|3605614|3605511|3605612|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|3814994|3815097|3814995|3815097|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|3830887|3830990|3830888|3830990|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|3915487|3915590|3915487|3915588|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|3993365|3993468|3993366|3993468|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|4116789|4116916|4116790|4116891|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|4255689|4255816|4255690|4255791|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|4361401|4361504|4361401|4361502|-\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttca-\n",
      ">G000006925|4694651|4694754|4694652|4694754|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000006925|4777914|4778017|4777915|4778017|+\n",
      "acgatcacagcagtccagcgcgaacgtgacacgcagtctctctccgttatcacagcagaa\n",
      "ctcgaacccgtcagagcaccatcgctgattgctttctttcac\n",
      ">G000299455|5279601|5279746|5279624|5279725|-\n",
      "acgatcgcaacagtccagggagaacgtaactcgcagtttttcgctatcatcgcaaccaaa\n",
      "ctcaaagccatcggaacaccaacgctggttgctttccttca-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1, 24671)), (932, 968), '+'), (('G000006925', (4143132, 4163489)), (4145517, 4145553), '+'), (('G000299455', (4431865, 4470166)), (4449566, 4449602), '-'), (('G000299455', (4996216, 5016676)), (5014255, 5014291), '-')]\n",
      ">G000006925|1|24671|932|968|+\n",
      "-gtggtgcttggacgtaacggttccgactactccgcg\n",
      ">G000006925|4143132|4163489|4145517|4145553|+\n",
      "-gtgttgctggggcgtaacggttccgactattccgcg\n",
      ">G000299455|4431865|4470166|4449566|4449602|-\n",
      "ggtggtacttggacgcaacggttccgactactccgc-\n",
      ">G000299455|4996216|5016676|5014255|5014291|-\n",
      "ggtgctgctggggcgtaacggttccgactattccgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4143132, 4163489)), (4154193, 4154528), '+'), (('G000299455', (3520976, 3527694)), (3521117, 3521452), '+'), (('G000299455', (4996216, 5016676)), (5005268, 5005603), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4143132|4163489|4154193|4154528|+\n",
      "-atttgttgcgctacatctaagggcagggtgatggattcacatcctgccagtaaacagtc\n",
      "cagcgcctgacgcggcgttttaaagctggctgccagcaccatgctttctggcgcgtgcat\n",
      "ttctaacagcgtttgcagctcctgaaccgtacgaatgccgtccccgccctgggcatctac\n",
      "gcggttaacatacggagcaacgtattttgcccctgccagtgcggctaataacccttctgc\n",
      "ggcgctatatacagcggtgccaagtgtagtaataccctattttttcagtattttaattgc\n",
      "tgccagaccttcggaagtcaccgggattttcaccac\n",
      ">G000299455|3520976|3527694|3521117|3521452|+\n",
      "-atctgctgcgccacatccagcggcagcgtaattgattcacatcctgccagtaagcagtc\n",
      "cagagcctgacgcggggttttgaaactcgctgccagcactttcgcctgcggcgcatgcat\n",
      "tttcaataactggtgtaagtcggtcacagtctgaataccgctaccgccctgagcatcaat\n",
      "acgattaacgtaaggcgcaacatattccgcacctgccagcgccgacagcaacccttgtgc\n",
      "cgcaccatataccgcggttcccagcgtcggaatcccttccgcttttaacatcttaatagt\n",
      "tgccagcccctcggcggtcaccggaactttcaccac\n",
      ">G000299455|4996216|5016676|5005268|5005603|-\n",
      "catttgttgcgctacatctaagggcagggtgatggattcacatcctgccagtaaacagtc\n",
      "cagcgcctgacgcggcgttttaaagctggccgccagcaccatgctttctggtgcgtgcat\n",
      "ttctaacagcgcttgcagctcctgaaccgtacgaatgccgtccccgccctgggcatctac\n",
      "gcggttaacatacggagcaacgtattttgcccctgccagcgcggccagtaacccttgtgc\n",
      "agcactgtacactgcggtgccaagcgtagtaataccctcttttttcagcatcttaatggc\n",
      "ggccagaccttcggaagtcaccgggattttcacca-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (722910, 723312)), (723028, 723066), '-'), (('G000006925', (901434, 901838)), (901727, 901765), '-'), (('G000299455', (2172426, 2172592)), (2172470, 2172508), '-'), (('G000299455', (2551218, 2551369)), (2551262, 2551299), '-'), (('G000299455', (2919336, 2919766)), (2919681, 2919719), '+')]\n",
      ">G000006925|722910|723312|723028|723066|-\n",
      "gtaatggttgcgttcgccagcttcagttctccggcgtt-\n",
      ">G000006925|901434|901838|901727|901765|-\n",
      "gtaatggttgcgttcgccagcttcagttctcttgcgtt-\n",
      ">G000299455|2172426|2172592|2172470|2172508|-\n",
      "gtaatggttgcgttcgccagcttcagttcactggcttt-\n",
      ">G000299455|2551218|2551369|2551262|2551299|-\n",
      "-caatggtggcgttcgcctggctgagttttgtcttcgc-\n",
      ">G000299455|2919336|2919766|2919681|2919719|+\n",
      "-taattgccgcgttcgccagcttcagttctctgacattt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2236142, 2236293)), (2236244, 2236270), '-'), (('G000299455', (2148729, 2148865)), (2148752, 2148777), '+'), (('G000299455', (3316701, 3316869)), (3316843, 3316869), '-')]\n",
      ">G000006925|2236142|2236293|2236244|2236270|-\n",
      "catccatgctccgtaaacattgaata\n",
      ">G000299455|2148729|2148865|2148752|2148777|+\n",
      "-atccatgctccgtatacattgaata\n",
      ">G000299455|3316701|3316869|3316843|3316869|-\n",
      "catccatgcaccataaacattgtaga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4351217, 4351974), '-'), (('G000299455', (3326617, 3362069)), (3335978, 3336732), '+'), (('G000299455', (4865050, 4904182)), (4901268, 4902025), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4315026|4354132|4351217|4351974|-\n",
      "ggtccgtcgcagtcggatttcggtggcgtaggattattacaaactcccaccgcgcgtatg\n",
      "gcacgggaaggggagttgagtctgaactatcgcgataacgatcagtaccgttattactca\n",
      "gcttcagtgcaactcttcccgtggctggaaacaacgctgcgctacaccgacgtgcgcacc\n",
      "cggcagtacagcagcgtcgaagcgttctctggcgatcaaacgtataaagataaagccttc\n",
      "gatctcaaactgcgtttgtgggaagagagttactggctgccgcaagtggcggttggtgcg\n",
      "cgggatatcggcggtacggggctgtttgatgcggaatatcttgttgccagcaaagcctgg\n",
      "gggccgttcgattttacgctcggcctgggctggggatatctgggcaccagcggtaatgta\n",
      "aaaaatccgctctgttcagccagtgataaatattgctatcgcgataacagctacaaacag\n",
      "gcgggatctatcgacggtagccagatgttccacggtcctgcctcactgtttggcggcgtg\n",
      "gaataccagacgccctggcaaccgctgcgtctgaaactggagtatgaaggtaataattat\n",
      "cagcaggattttgccgggaagctggagcaaaaaagtaagtttaacgtcggggcgatttat\n",
      "cgcgttaccgattgggccgacgttaaccttagctatgaacgtggcaacacctttatgttt\n",
      "ggcgtcacgctgcgcaccaactttaacgatctgcgcc-\n",
      ">G000299455|3326617|3362069|3335978|3336732|+\n",
      "-gttcgtcgcaatcagactttggcggcacaggattgttgcagatgccaaatgcgcgcatc\n",
      "gcaccggaaggtgaattcagcgtcaactaccgggataacgatcaataccggttctactcc\n",
      "acctccgtggcgctgttcccatggctggaaggcaccattcgttatacggatgtgcgcaca\n",
      "cgcaaatatagccagtgggaagatttcagcggcgatcagtcatacaaagacaaatcattc\n",
      "gattttaaacttcgcctgtgggaagaaggttactggctaccgcaagtggcgtttggtaaa\n",
      "cgtgatattgctggtacgggtctgtttgacggtgagtatctggtggccagcaagcaagcg\n",
      "gggccatttgatttcaccctcgggatggcatggggctacgccggtaatgcgggcaatatt\n",
      "accaacccgttttgccgggtgagcgataaatattgtcatcgcgcagagtctcacg---at\n",
      "gcgggcgatatcagctttagcgatatctttcgtggcccggcttccatctttggcggcatt\n",
      "gagtatcaaacgccgtggaatcccctgcgtctgaaactcgaatacgatggcaacaattac\n",
      "cagaatgatttcgctggcaaactgcctcaggcaagccatttcaacgtcggcgcagtttat\n",
      "cgcgctgccagctgggcagatctcaacctgagttatgaacgcggtaacacgttgatgttt\n",
      "ggcttcacgttacggaccaatttcaacgatctgcgccc\n",
      ">G000299455|4865050|4904182|4901268|4902025|-\n",
      "ggtccgtcgcagtcggatttcggtggcgtaggattattacaaactcccaccgcgcgtatg\n",
      "gcacgggaaggggagttgagtctgaactatcgcgataatgatcagtaccgttattactca\n",
      "gcttcagtgcaactcttcccgtggctggaaacaacgctgcgctacaccgacgtgcgcacc\n",
      "cggcagtacagcagcgtcgaagcgttctctggcgatcaaacgtataaagataaagccttc\n",
      "gatctcaaactgcgtttgtgggaagagagttactggctgccgcaagtggcggttggtgcg\n",
      "cgggatatcggcggtacggggctgtttgatgcggaatatcttgttgccagcaaagcctgg\n",
      "gggccgttcgattttacgctcggcctgggctggggatatctgggcaccagcggtaatgta\n",
      "aaaaatccgctctgttcagccagtgataaatattgctatcgcgataacagctacaaacag\n",
      "gcgggatctatcgacggtagccagatgttccacggtcctgcctcactgtttggcggcgtg\n",
      "gaataccagacgccctggcaaccgctgcgcctgaaactggagtatgaaggcaataattat\n",
      "cagcaggattttgccgggaagctggagcaaaaaagtaagtttaacgtcggtgcgatttat\n",
      "cgcgttaccgattgggccgacgttaaccttagctatgaacgtggcaacacctttatgttt\n",
      "ggcgtcacgctgcgcaccaactttaacgatctgcgcc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2735558, 2755941)), (2739285, 2739352), '-'), (('G000299455', (1302091, 1327808)), (1318682, 1318749), '+'), (('G000299455', (3626532, 3650149)), (3645608, 3645675), '-')]\n",
      ">G000006925|2735558|2755941|2739285|2739352|-\n",
      "cccgcaatacctgggcgatctgtttagctggtcagcaattggtgctcgtacaacggaatc\n",
      "gcaaact-\n",
      ">G000299455|1302091|1327808|1318682|1318749|+\n",
      "-ccgcaatacctgggcgatctgtttagctggtcagcaattggtgctcgtacaacggaatc\n",
      "gcaaactc\n",
      ">G000299455|3626532|3650149|3645608|3645675|-\n",
      "cccacaatatctcgctgacctgatgagctggggcgcaattggcgcacgtaccaccgaatc\n",
      "gcaggtg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1634479, 1637262)), (1636200, 1636660), '-'), (('G000299455', (2585656, 2594816)), (2587084, 2587544), '+'), (('G000299455', (2585656, 2594816)), (2589568, 2590028), '+'), (('G000299455', (3440781, 3470001)), (3443194, 3443654), '+')]\n",
      ">G000006925|1634479|1637262|1636200|1636660|-\n",
      "ccgatctctcgtgccaactggataattttttctgccgggatgctggtgatttttgctgcc\n",
      "cattccggcgttttagctatgccgtcaggcccttcgcccagaatataggctttataatgc\n",
      "gcgttacgtggtgcgttggcgggcagcgttttttcatcgtaaccaacacaatatttgtcg\n",
      "agaaatggctgatcgaccatgttttcagtaatcagtacccaggcaatcgcacaggccagt\n",
      "gcgccatcggtgccagggcgaatgggcagccattcatcttcacgcccggcagctgtgtcg\n",
      "ttatagcgtggatcgatgacgatcatgcgtgcgtttgaacgttcgcgggcttgctcgacg\n",
      "tagtaagtgacaccaccgccgctcatccgcgtttctgccgggttatttccgaacataacg\n",
      "accagtttcgtattggcgatatcatccgggctgttgccat-\n",
      ">G000299455|2585656|2594816|2587084|2587544|+\n",
      "-cgatctctcgtgccaactggataattttttctgccgggatgctggtgatttttgctgcc\n",
      "cattccggcgttttagctatgccgtcaggcccttcgcccagaatataggctttataatgc\n",
      "gcgttacgtggtgcgttggcgggcagagttttttcatcgtaaccaacacaatatttgtcg\n",
      "agaaatggctgatcgaccatgttttcagtaatcagtacccaggcaatcgcacaggccagt\n",
      "gcgccatcggtgcctgggcgaatgggcagccattcatcttcacgcccggcagctgtgtcg\n",
      "ttataacgtggatcgatgacgatcatgcgtgcgtttgaacgttcgcgggcttgctcgacg\n",
      "tagtaagtgacaccaccgccgctcatccgcgtttctgccgggttatttccgaacataacg\n",
      "accagtttcgtattggcgatatcatccgggctgttgccatc\n",
      ">G000299455|2585656|2594816|2589568|2590028|+\n",
      "-caatttcacgcgccagtttgatgatacggtctaccgggataccggtaatttgcgaagcc\n",
      "cactgcggcgttttcgctgtgttatcatcaccttcaccaagaatataggctttatagtga\n",
      "ccatttttgggtgcatctgcgggtaaggttttttcgtcatagccgacgcagtatttatcg\n",
      "agaaaaggttgatcaacgagattttcgttaatcaatacccaggcaatacccgcaaccagc\n",
      "gcggcatcggtgcccgggcgaatagggagccattcatcttcacgaccagcagccgtatcg\n",
      "gtatatcgcggatcgataacaatcattttggcgttcgatttctcgcgcgctttttcaaga\n",
      "agataagtgatgccaccgccgctcatgcgggtttctgccgggttgttaccaaacatcacg\n",
      "accagcttgctgttttcaatatccgtggtgctgttgccatc\n",
      ">G000299455|3440781|3470001|3443194|3443654|+\n",
      "-cgatttcacgagccagtttgatgattttgtctgccggaacaccggtgatttgtgaggcc\n",
      "cattccggtgttttagccacgccatctggcccttcacccagaatataagctttatagtgg\n",
      "ccatttttcggcgcactggctggcagggttttctcatcgtagccaacgcaatatttatcg\n",
      "aggaatgcctgatccaccaggttttcagtgatcatgacgtacgccagaccgttaaccagt\n",
      "gcggcatctgttcccggacgaatagggatccactcatcttcgcgcccggcaccggtgtcg\n",
      "gtatagcgcggatcgatgatgatcatgcgggcattagatttctggcgtgcctgttcaaga\n",
      "tagtaagtcaccccaccgccactcattcgcgtttcgccagggttattaccaaacagcact\n",
      "accagtttactgttttcgatatccgacgggctgttaccatc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1425133, 1465118)), (1452706, 1452755), '-'), (('G000006925', (3246783, 3268347)), (3250499, 3250548), '-'), (('G000299455', (679634, 695969)), (690339, 690388), '+'), (('G000299455', (1078152, 1154001)), (1135187, 1135236), '-'), (('G000299455', (2199778, 2263225)), (2250807, 2250856), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1100\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1425133|1465118|1452706|1452755|-\n",
      "aaagagatcgacgagtatttcgcgcatgtctggcaaaccatgca-ggcat\n",
      ">G000006925|3246783|3268347|3250499|3250548|-\n",
      "gcggaaattgacgcagggtttgcccgtatctggcaagtgatgcatgacg-\n",
      ">G000299455|679634|695969|690339|690388|+\n",
      "-cggaaattgacgcagggtttgcccgtatctggcaagtgatgcatgacgg\n",
      ">G000299455|1078152|1154001|1135187|1135236|-\n",
      "gaagagctggaacagcacctggcgaacgtctgggaagtcatgcgcggcg-\n",
      ">G000299455|2199778|2263225|2250807|2250856|-\n",
      "aaagagatcgacgagtatttcgcgcatgtctggcaaaccatgca-ggcat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1, 24671)), (24359, 24383), '+'), (('G000299455', (2553527, 2553551)), (2553527, 2553551), '-'), (('G000299455', (4422869, 4428117)), (4423157, 4423181), '-')]\n",
      ">G000006925|1|24671|24359|24383|+\n",
      "-tggcagcgttggttctttgccggt\n",
      ">G000299455|2553527|2553551|2553527|2553551|-\n",
      "ctggcagcgttggttctttcccgg-\n",
      ">G000299455|4422869|4428117|4423157|4423181|-\n",
      "ctggcagcgttggttctttgccgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2696888, 2728352)), (2711449, 2711472), '-'), (('G000299455', (1329940, 1344983)), (1344960, 1344983), '+'), (('G000299455', (1345136, 1375168)), (1345136, 1345159), '+')]\n",
      ">G000006925|2696888|2728352|2711449|2711472|-\n",
      "aatgccgggcttaaacccggcat-\n",
      ">G000299455|1329940|1344983|1344960|1344983|+\n",
      "-atgtcgggattaaacccggcatt\n",
      ">G000299455|1345136|1375168|1345136|1345159|+\n",
      "-ataccaggcttaaacccggcatt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1509064, 1578296)), (1577996, 1578094), '+'), (('G000299455', (2167070, 2167372)), (2167269, 2167367), '-'), (('G000299455', (2923865, 2925880)), (2925584, 2925682), '+'), (('G000299455', (3293159, 3293449)), (3293159, 3293257), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1166\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1166\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1577996|1578094|+\n",
      "-atgaatatggcgcgctgtactggattcgaaccagtgaccgattgcttagaaggcaattg\n",
      "ctctgtccggctgagctaacaacgctgaataccgataat\n",
      ">G000299455|2167070|2167372|2167269|2167367|-\n",
      "aataagtctggcgcgttgtactggattcgaaccagtgaccgattgcttagaaggcaattg\n",
      "ctctgtccggctgagctaacaacgcagggtacagataa-\n",
      ">G000299455|2923865|2925880|2925584|2925682|+\n",
      "-atgaatatggcgcgctgtactggattcgaaccagtgaccgattgcttagaaggcaattg\n",
      "ctctgtccggctgagctaacaacgcatgatgcagataat\n",
      ">G000299455|3293159|3293449|3293159|3293257|+\n",
      "-ataaatatggcgcgttgtactggattcgaaccagtgaccgattgctcagaaggcaattg\n",
      "ctctgtccggctgagctaacaacgcataatgcagataat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1965007, 1976679)), (1968636, 1968718), '+'), (('G000006925', (4557883, 4573403)), (4558850, 4558932), '-'), (('G000299455', (2122044, 2133716)), (2128304, 2128386), '-'), (('G000299455', (2122044, 2133716)), (2130005, 2130087), '-'), (('G000299455', (2737179, 2754578)), (2748601, 2748683), '+'), (('G000299455', (4487440, 4499283)), (4498234, 4498316), '+')]\n",
      ">G000006925|1965007|1976679|1968636|1968718|+\n",
      "-ccagttccgacgcctgtcgtgcgttatcggcgttttgccctaccgtggcggttaattgc\n",
      "tccatactggcggccgtttgtgc\n",
      ">G000006925|4557883|4573403|4558850|4558932|-\n",
      "gccagatggctggcctggcgcgcattctcggcgttctgtttcaccgttgcggtcagttgc\n",
      "tccatgctggctgccgtctctt-\n",
      ">G000299455|2122044|2133716|2128304|2128386|-\n",
      "gccagttgcgaggcctggcgggcgttatcggcgttttgcttcactgtcgcggtgagctgc\n",
      "tccatgctggcggcagtttctt-\n",
      ">G000299455|2122044|2133716|2130005|2130087|-\n",
      "gccagttccgacgcctgtcgtgcgttatcggcgttttgccctaccgtggcggttaattgc\n",
      "tccatactggcggccgtttgtg-\n",
      ">G000299455|2737179|2754578|2748601|2748683|+\n",
      "-ccagtttgctggcatgatgcgcgttatccgcattctgtttcaccgtcgcagtgagttgc\n",
      "tccatgctggcggcagtttgttc\n",
      ">G000299455|4487440|4499283|4498234|4498316|+\n",
      "-ccagatggctggcctgacgcgcattctcggcgttctgtttcaccgttgcggtcagttgc\n",
      "tccatgctggctgccgtctcttc\n",
      "\n",
      "\n",
      "29 [(('G000006925', (251358, 252232)), (251713, 251772), '+'), (('G000006925', (315068, 315858)), (315527, 315586), '-'), (('G000006925', (332059, 337870)), (332518, 332577), '-'), (('G000006925', (493019, 493813)), (493478, 493537), '-'), (('G000006925', (509301, 510374)), (509761, 509820), '-'), (('G000006925', (724190, 725876)), (724650, 724709), '-'), (('G000006925', (1094884, 1095679)), (1095343, 1095402), '-'), (('G000006925', (1099363, 1100153)), (1099634, 1099693), '+'), (('G000006925', (1411510, 1412350)), (1412019, 1412078), '-'), (('G000006925', (1883764, 1885448)), (1884224, 1884283), '-'), (('G000006925', (1956633, 1961076)), (1957093, 1957152), '-'), (('G000006925', (2084425, 2085215)), (2084696, 2084755), '+'), (('G000006925', (2236395, 2237421)), (2236902, 2236961), '+'), (('G000006925', (3065287, 3076770)), (3075223, 3075282), '-'), (('G000006925', (3812816, 3813607)), (3813275, 3813334), '-'), (('G000006925', (4639759, 4640549)), (4640218, 4640277), '-'), (('G000006925', (4650084, 4650878)), (4650359, 4650418), '+'), (('G000006925', (4656735, 4658205)), (4657686, 4657745), '+'), (('G000006925', (4673259, 4674053)), (4673534, 4673593), '+'), (('G000006925', (4679888, 4680760)), (4680348, 4680407), '-'), (('G000006925', (4750261, 4751054)), (4750535, 4750594), '+'), (('G000006925', (4753880, 4754990)), (4754660, 4754719), '-'), (('G000006925', (4760730, 4761525)), (4761006, 4761065), '+'), (('G000006925', (4793569, 4794365)), (4793846, 4793905), '+'), (('G000299455', (906289, 909876)), (906566, 906625), '+'), (('G000299455', (912160, 912572)), (912435, 912494), '+'), (('G000299455', (4752701, 4766669)), (4752978, 4753037), '+'), (('G000299455', (4771847, 4785261)), (4780172, 4780231), '-'), (('G000299455', (5275700, 5279438)), (5278952, 5279011), '-')]\n",
      ">G000006925|251358|252232|251713|251772|+\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|315068|315858|315527|315586|-\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|332059|337870|332518|332577|-\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|493019|493813|493478|493537|-\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|509301|510374|509761|509820|-\n",
      "tgaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccag-\n",
      ">G000006925|724190|725876|724650|724709|-\n",
      "tgaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctg-\n",
      ">G000006925|1094884|1095679|1095343|1095402|-\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|1099363|1100153|1099634|1099693|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|1411510|1412350|1412019|1412078|-\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|1883764|1885448|1884224|1884283|-\n",
      "tgaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctg-\n",
      ">G000006925|1956633|1961076|1957093|1957152|-\n",
      "tgaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctg-\n",
      ">G000006925|2084425|2085215|2084696|2084755|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|2236395|2237421|2236902|2236961|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|3065287|3076770|3075223|3075282|-\n",
      "tgaaagagctggaacgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccag-\n",
      ">G000006925|3812816|3813607|3813275|3813334|-\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|4639759|4640549|4640218|4640277|-\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|4650084|4650878|4650359|4650418|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|4656735|4658205|4657686|4657745|+\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|4673259|4674053|4673534|4673593|+\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|4679888|4680760|4680348|4680407|-\n",
      "tgaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccag-\n",
      ">G000006925|4750261|4751054|4750535|4750594|+\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|4753880|4754990|4754660|4754719|-\n",
      "-gaaagagccggaacgtgaaaatcgtgaactgcgccgcagtaacaatatccttcgcctgg\n",
      ">G000006925|4760730|4761525|4761006|4761065|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000006925|4793569|4794365|4793846|4793905|+\n",
      "-gaaagagccggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000299455|906289|909876|906566|906625|+\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtagcgatatccttcgccagg\n",
      ">G000299455|912160|912572|912435|912494|+\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccagg\n",
      ">G000299455|4752701|4766669|4752978|4753037|+\n",
      "-gaaagagctggagcgtgaaaatcgtgaactgcgccgcagtagcgatatccttcgccagg\n",
      ">G000299455|4771847|4785261|4780172|4780231|-\n",
      "tgaaagagctggaacgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccag-\n",
      ">G000299455|5275700|5279438|5278952|5279011|-\n",
      "tgaaagagctggagcgtgaaaatcgtgaactgcgccgcagtaacgatatccttcgccag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1696422, 1718400)), (1711648, 1712007), '+'), (('G000006925', (2006629, 2008384)), (2006631, 2006990), '+'), (('G000299455', (2269177, 2270929)), (2270570, 2270929), '-'), (('G000299455', (2737179, 2754578)), (2737179, 2737538), '+'), (('G000299455', (4667815, 4670271)), (4668229, 4668588), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\n",
      "Reallocating..done. *alloclen = 1120\n",
      "\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  f\b\b\r",
      "STEP    28 / 28  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  f\b\b\r",
      "STEP    28 / 28  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1696422|1718400|1711648|1712007|+\n",
      "-tgacatcctccacgccctgaaagacgtggttccctgctacgttcaagctgtcgcctgaa\n",
      "tcatttcggtgggttcctgcttcaacgagcggcctgactgcaccatctctccacaggcaa\n",
      "gaacggcgtgccccgccgctaaaatgttacgagcgccatttacatcggcgttcgctgtat\n",
      "atccacatacctggcatctgaatttactttgtgacaggcggttttctttcgctgtatgac\n",
      "cacagcacgcgcaacgctggcttgtgtacgctggcggaacagcaagcacctgaccgccac\n",
      "tccagagctgcttgtactcaagctggcggcgcatttcataccagccctgatccagtatcg\n",
      ">G000006925|2006629|2008384|2006631|2006990|+\n",
      "-tgacatcctccacgccctgaatgacgaggacccctgctacgttcaggctgtcgcctgaa\n",
      "tcatttcggtgggttcctgcttcaacgagcggcctgactgcaccatctctccacaggcaa\n",
      "gaacggcgtgccccgccgctaaaatgttacgagctccatttacatcggcgttcgctgtat\n",
      "atccacatacctggcatctgaatttactttgtgacaggcgattttctttcgctgtatgac\n",
      "cacagcacgcgcaacgctggcttgtgtacgctggcggaacagcaagcacctgaccgccac\n",
      "gccagagctgcttgtactcaagctggcggcgcatttcataccagccctgatccagtatcg\n",
      ">G000299455|2269177|2270929|2270570|2270929|-\n",
      "ttgacatcctccacgccctgaatgacgaggatccctgctacgttcaggctgtcgcctgaa\n",
      "tcatttcggtgggttcctgcttcaacgagcggcctgactgcaccatctctccacaggcaa\n",
      "gaacggcgtgccccgccgctaaaatgttacgagcgccgtttacatcggcgttcgctgtat\n",
      "atccacatacctggcatctgaattgactttgtgacaggcgattttctttcgctgtatgac\n",
      "cacagcacgcgcaacgctggcttgtgtacgctggcggaacagcaagcacctgaccgccac\n",
      "tccagagctgcttgtactcaagctggcggcgcatttcataccagccctgatccagtatc-\n",
      ">G000299455|2737179|2754578|2737179|2737538|+\n",
      "-tgacatcctccacgccctgaaggacgaggatccctgctacgttcaggctgtcgcctgaa\n",
      "tcatttcggtgggttcctgcttcaacgggcggcctgactgcaccatctctccacaggcaa\n",
      "gaacggcgtgccccgccgctaaaatgttacgagcgccatttacatcggcgttcgctgtat\n",
      "atccacatacctggcatctgaatttactttgtgacaggcggttttctttcgctgtatgac\n",
      "cacagcacgcgcaacgctggcttgtgtacgctggcggaacagcaagcacctgaccgccac\n",
      "gccagagctgcttgtactcaagctggcggcgcatttcataccagccctgatccagtatcg\n",
      ">G000299455|4667815|4670271|4668229|4668588|+\n",
      "-tgacatcctccacgccctgaaggacgtggatccctgcttcgttcaggctgtcgcctgaa\n",
      "tcatttcggtgggttcctgcttcaacgagcggcctgactgcaccatctctccacaggcaa\n",
      "gaacggcgtgccccgccgctaaaatgttacgagcgccgtttacatcggcgttcgctgtat\n",
      "atccacatacctggcatctgaatttactttgtgacaggcgattttctttcgctgtatgac\n",
      "cacagcacgcgcaacgctggcttgtgtacgctggcggaacagcaagcacctgaccgccac\n",
      "gccagagctgcttgtactcaagctggcggcgcatttcataccagccctgatccagtatcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3738363, 3738613), '-'), (('G000299455', (159106, 161990)), (160746, 160996), '+'), (('G000299455', (976832, 977082)), (976832, 977082), '-')]\n",
      ">G000006925|3736321|3788106|3738363|3738613|-\n",
      "atcatagaaccgaggaacatcggcatgtctgcgccgacgataacgcccatggtggtaatg\n",
      "gcaccgactacgccaccacgtttgccgcctaccagcttaccaccggtataaccgatcagc\n",
      "agcggtaggagataagtgatcatcggcccgaccagcttcgccagcgtctcgttcggtaac\n",
      "caccctgttggaataaataacgcggtgatgataccccacgcgataaacgcgccgatattt\n",
      "ggcatcacca-\n",
      ">G000299455|159106|161990|160746|160996|+\n",
      "-tcatagaaccgaggaacatcggcatgtctgcgccgacgataacgcccatggtagtgatg\n",
      "gcaccgactacgccgccacgttcgccgcctaccagcttaccaccggtataaccgatcagc\n",
      "agcggcaggagataagtgatcatcggcccgaccagcttcgccagcgtctcgttcggtaac\n",
      "caccctgttggaataaataacgcggtgatgataccccacgcgataaacgcgccgatattt\n",
      "ggcatcaccat\n",
      ">G000299455|976832|977082|976832|977082|-\n",
      "atcattgagccaaggaacatcgggatctctgcgccaacgatcacaccaatagtacctatt\n",
      "ccgcccatgaccgccccgcgtttaccgccgaccagatgaccaccagtagaaccaatcatc\n",
      "acgggcaataaataggtaatcatcgggccgacaattttggcgaaatgttcattcggcagc\n",
      "caaccggtgggaataaataacgcagttatgaagccccaggcaataaaagcaccaatattg\n",
      "gggatgacca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2849870, 2870340)), (2863737, 2863802), '+'), (('G000299455', (1158891, 1160694)), (1160500, 1160565), '-'), (('G000299455', (4584877, 4585231)), (4585166, 4585231), '+')]\n",
      ">G000006925|2849870|2870340|2863737|2863802|+\n",
      "-gcagtgcaatgactcgcggtttatcatcggtcagcgttgccagtagttcatccagtgct\n",
      "tcaata\n",
      ">G000299455|1158891|1160694|1160500|1160565|-\n",
      "tgcagtgcaatgactcgcggtttatcatcggtcagcgttgccagtagttcatccagtgct\n",
      "tcaat-\n",
      ">G000299455|4584877|4585231|4585166|4585231|+\n",
      "-gcagcggaatcacccgcggtttgtcgtagctcagcgtggcgaatagttcgtccagcgcg\n",
      "tcaata\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1396364, 1396395)), (1396364, 1396395), '+'), (('G000299455', (2151454, 2151485)), (2151454, 2151485), '-'), (('G000299455', (2582670, 2582701)), (2582670, 2582701), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1396364|1396395|1396364|1396395|+\n",
      "-ggcatgactactattaccaaagagcgactgc\n",
      ">G000299455|2151454|2151485|2151454|2151485|-\n",
      "cggcatgatcactattaccaaagggcgactg-\n",
      ">G000299455|2582670|2582701|2582670|2582701|-\n",
      "cggcatgatcactattaccaaagggcgactg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189492, 1208094)), (1197569, 1197608), '-'), (('G000299455', (3040484, 3040523)), (3040484, 3040523), '+'), (('G000299455', (3041304, 3050668)), (3041344, 3041383), '+')]\n",
      ">G000006925|1189492|1208094|1197569|1197608|-\n",
      "aaaacaacgggcgtgttatacgcccgtttaaatatttaa-\n",
      ">G000299455|3040484|3040523|3040484|3040523|+\n",
      "-aaacaacgggtgtgccatacgcccgtttcaatacttaac\n",
      ">G000299455|3041304|3050668|3041344|3041383|+\n",
      "-aaacaacgggcgtgttatacgcccgttgcaatattaaac\n",
      "\n",
      "\n",
      "14 [(('G000006925', (385251, 385465)), (385327, 385356), '+'), (('G000006925', (900687, 900901)), (900796, 900825), '-'), (('G000006925', (3065287, 3076770)), (3066791, 3066820), '-'), (('G000006925', (3422918, 3423132)), (3423027, 3423056), '-'), (('G000006925', (3851029, 3851243)), (3851105, 3851134), '+'), (('G000006925', (4358341, 4358555)), (4358417, 4358446), '+'), (('G000006925', (4622488, 4624460)), (4624287, 4624316), '-'), (('G000006925', (4699715, 4699878)), (4699791, 4699820), '+'), (('G000006925', (4775368, 4775582)), (4775444, 4775473), '+'), (('G000006925', (4782659, 4782873)), (4782768, 4782797), '-'), (('G000299455', (3215867, 3216640)), (3216088, 3216117), '-'), (('G000299455', (4749820, 4751622)), (4751320, 4751349), '-'), (('G000299455', (5287974, 5288758)), (5288497, 5288526), '+'), (('G000299455', (5313369, 5315826)), (5315524, 5315553), '-')]\n",
      ">G000006925|385251|385465|385327|385356|+\n",
      "-ctgagtcatttctgtcggacggcgcacat\n",
      ">G000006925|900687|900901|900796|900825|-\n",
      "cctgagtcatttctgtcggacggcgcaca-\n",
      ">G000006925|3065287|3076770|3066791|3066820|-\n",
      "ccgtggtgatgtcggttggcactcttgca-\n",
      ">G000006925|3422918|3423132|3423027|3423056|-\n",
      "cctgagtcatttctgtcggacggcgcaca-\n",
      ">G000006925|3851029|3851243|3851105|3851134|+\n",
      "-ctgagtcatttctgtcggacggcgcacat\n",
      ">G000006925|4358341|4358555|4358417|4358446|+\n",
      "-ctgagtcatttctgtcggacggcgcacat\n",
      ">G000006925|4622488|4624460|4624287|4624316|-\n",
      "ccgtggtgatgtcggttggcactcttgca-\n",
      ">G000006925|4699715|4699878|4699791|4699820|+\n",
      "-ctgagtcatttctgtcggacggcgcacat\n",
      ">G000006925|4775368|4775582|4775444|4775473|+\n",
      "-ctgagtcatttctgtcggacggcgcacat\n",
      ">G000006925|4782659|4782873|4782768|4782797|-\n",
      "cctgagtcatttctgtcggacggcgcaca-\n",
      ">G000299455|3215867|3216640|3216088|3216117|-\n",
      "cctccgtcagggctgacggggtgcgaacg-\n",
      ">G000299455|4749820|4751622|4751320|4751349|-\n",
      "ccgtggtgatgtcggttggcactcttgca-\n",
      ">G000299455|5287974|5288758|5288497|5288526|+\n",
      "-ctccgtcagggctgacggggtgcgaacgt\n",
      ">G000299455|5313369|5315826|5315524|5315553|-\n",
      "ccgtggtgatgtaggtgggcgctcttgca-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1696422, 1718400)), (1712251, 1712276), '+'), (('G000006925', (1815406, 1827803)), (1815650, 1815675), '+'), (('G000006925', (2006629, 2008384)), (2007234, 2007259), '+'), (('G000006925', (4764001, 4764173)), (4764148, 4764173), '+'), (('G000299455', (2269177, 2270929)), (2270301, 2270326), '-'), (('G000299455', (2737179, 2754578)), (2737782, 2737807), '+'), (('G000299455', (4667815, 4670271)), (4668832, 4668857), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 14\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 14\r",
      "   10 / 14\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 13  f\b\b\r",
      "STEP     2 / 13  f\b\b\r",
      "STEP     3 / 13  f\b\b\r",
      "STEP     4 / 13  f\b\b\r",
      "STEP     5 / 13  f\b\b\r",
      "STEP     6 / 13  f\b\b\r",
      "STEP     7 / 13  f\b\b\r",
      "STEP     8 / 13  f\b\b\r",
      "STEP     9 / 13  f\b\b\r",
      "STEP    10 / 13  f\b\b\r",
      "STEP    11 / 13  f\b\b\r",
      "STEP    12 / 13  f\b\b\n",
      "Reallocating..done. *alloclen = 1060\n",
      "\r",
      "STEP    13 / 13  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 14\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 14\r",
      "   10 / 14\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 13  f\b\b\r",
      "STEP     2 / 13  f\b\b\r",
      "STEP     3 / 13  f\b\b\r",
      "STEP     4 / 13  f\b\b\r",
      "STEP     5 / 13  f\b\b\r",
      "STEP     6 / 13  f\b\b\r",
      "STEP     7 / 13  f\b\b\r",
      "STEP     8 / 13  f\b\b\r",
      "STEP     9 / 13  f\b\b\r",
      "STEP    10 / 13  f\b\b\r",
      "STEP    11 / 13  f\b\b\n",
      "Reallocating..done. *alloclen = 1060\n",
      "\r",
      "STEP    12 / 13  f\b\b\r",
      "STEP    13 / 13  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1696422|1718400|1712251|1712276|+\n",
      "-tcgctgaagtctcgccagcgtcttc\n",
      ">G000006925|1815406|1827803|1815650|1815675|+\n",
      "-tcgctgaagtctcgccagcgtcttc\n",
      ">G000006925|2006629|2008384|2007234|2007259|+\n",
      "-tcgctgaagtctcgccagcgtcttc\n",
      ">G000006925|4764001|4764173|4764148|4764173|+\n",
      "-atgttggagcgctgccagcttcttc\n",
      ">G000299455|2269177|2270929|2270301|2270326|-\n",
      "gtcgctgaagtctcgccagcgtctt-\n",
      ">G000299455|2737179|2754578|2737782|2737807|+\n",
      "-tcgctgaagtctcgccagcgtcttc\n",
      ">G000299455|4667815|4670271|4668832|4668857|+\n",
      "-tcgctgaagtctcgccagcttcttc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2728788, 2734132)), (2732128, 2732225), '-'), (('G000299455', (1302091, 1327808)), (1325803, 1325900), '-'), (('G000299455', (3440781, 3470001)), (3461372, 3461469), '+')]\n",
      ">G000006925|2728788|2734132|2732128|2732225|-\n",
      "cttccggcacttcgccgttgataatacgctgcgccagaccttcaacgatggcagttttac\n",
      "cgacgccgggttcaccaatcagtaccgggttattttt-\n",
      ">G000299455|1302091|1327808|1325803|1325900|-\n",
      "cttccggcacttcgccgttgataatacgctgcgccagaccttcaacgatggcagttttac\n",
      "cgacgccgggttcaccaatcagtaccgggttattttt-\n",
      ">G000299455|3440781|3470001|3461372|3461469|+\n",
      "-ttccggcacatcgccctgaacaattcgccaggcaagaccttccgcaatcgcggttttac\n",
      "cgacaccagattcccccaccagcagcgggttgttttta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1009687, 1025529)), (1021363, 1021415), '-'), (('G000299455', (800023, 810909)), (807762, 807814), '-'), (('G000299455', (3326617, 3362069)), (3350341, 3350393), '+')]\n",
      ">G000006925|1009687|1025529|1021363|1021415|-\n",
      "agtacagacgccgcagatacgttcaacgaacgcccacgcatcgcgcgggtcg--\n",
      ">G000299455|800023|810909|807762|807814|-\n",
      "agtacatacgccacagatacgttgcacaatcatccatgcatcgcgcggatcg--\n",
      ">G000299455|3326617|3362069|3350341|3350393|+\n",
      "--tacagacgccgcagatacgttcaacgaacgcccacgcatcgcgcgggtcgcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3737636, 3737781), '-'), (('G000299455', (159106, 161990)), (161578, 161723), '+'), (('G000299455', (171875, 179671)), (179430, 179574), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1106\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3736321|3788106|3737636|3737781|-\n",
      "gaggcagatatgtttctaaactatttcgcgttgggagtgctgatcttcgtctttctggtg\n",
      "attttttatggaatcatcgcgatacatgacatcccttatctgattgccaaaaagcgcaat\n",
      "cacccccatgccgacgctattcata-\n",
      ">G000299455|159106|161990|161578|161723|+\n",
      "-aggcagatatgtttctaaactatttcgcgttgggagtgctgatcttcgtctttctggtg\n",
      "attttttatggaatcatcgcgatacatgacatcccttatctgattgccaaaaagcgcaat\n",
      "cacccccatgccgacgctattcatac\n",
      ">G000299455|171875|179671|179430|179574|+\n",
      "-agggaaatatgtttctggattattttgcactgggagtgcttatttttgtatttctggtg\n",
      "atattctatgggattattattttacatgatattccctacctgattgctaaaaaacgtaat\n",
      "catcctcatgccgacgccattcatg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (729937, 730069), '+'), (('G000299455', (2174641, 2181396)), (2178333, 2178465), '+'), (('G000299455', (2910238, 2916987)), (2913163, 2913295), '-')]\n",
      ">G000006925|726235|732999|729937|730069|+\n",
      "-cgggcgtgagatgggcgcagcaagcctttcggtaccgcaacagcaggtacagcttgacg\n",
      "ctcccgggatgctggctgaaacggacgagtacatggccggaggcaaacgacctgcccgtg\n",
      "tttaccgggtggt\n",
      ">G000299455|2174641|2181396|2178333|2178465|+\n",
      "-cgggcgcgagatgggggcagcaagtctttcggtaccacaacagcaggtacagtttgatg\n",
      "ctcccggaatgctggctgaaacggacgagtacatggccggaggtaaacgaccggcccgtg\n",
      "tttaccgggtggt\n",
      ">G000299455|2910238|2916987|2913163|2913295|-\n",
      "tcgggcgcgagatgggggcagcaagtctttcggtaccacaacagcaggtacagcttgatg\n",
      "ctcccggaatgctggctgaaacggacgagtacatggccggaggtaaacgaccggcccgtg\n",
      "tttaccgggtgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2268783, 2268820), '-'), (('G000299455', (1752647, 1813107)), (1809391, 1809428), '+'), (('G000299455', (3212276, 3212366)), (3212320, 3212357), '+')]\n",
      ">G000006925|2242494|2346249|2268783|2268820|-\n",
      "tacggctccgatgcgctcggcggtgtagtgaatatca-\n",
      ">G000299455|1752647|1813107|1809391|1809428|+\n",
      "-acggctccgatgcgctcggcggtgtagtgaatatcat\n",
      ">G000299455|3212276|3212366|3212320|3212357|+\n",
      "-atggctctgatgcgatgggcggtgtggtgaatatcat\n",
      "\n",
      "\n",
      "36 [(('G000006925', (236124, 236173)), (236138, 236173), '+'), (('G000006925', (266315, 266364)), (266315, 266350), '-'), (('G000006925', (495826, 495875)), (495826, 495861), '-'), (('G000006925', (707132, 707181)), (707132, 707167), '-'), (('G000006925', (865149, 865198)), (865149, 865184), '-'), (('G000006925', (912452, 912501)), (912452, 912487), '-'), (('G000006925', (980988, 981037)), (980988, 981023), '-'), (('G000006925', (1008963, 1009012)), (1008977, 1009012), '+'), (('G000006925', (1098042, 1098091)), (1098056, 1098091), '+'), (('G000006925', (1208850, 1208899)), (1208850, 1208885), '-'), (('G000006925', (1392959, 1393008)), (1392959, 1392994), '-'), (('G000006925', (1418309, 1418358)), (1418309, 1418344), '-'), (('G000006925', (1500579, 1500628)), (1500593, 1500628), '+'), (('G000006925', (1611215, 1611264)), (1611215, 1611250), '-'), (('G000006925', (1618934, 1618983)), (1618948, 1618983), '+'), (('G000006925', (1696032, 1696081)), (1696032, 1696067), '-'), (('G000006925', (1807326, 1807375)), (1807326, 1807361), '-'), (('G000006925', (2036011, 2036060)), (2036011, 2036046), '-'), (('G000006925', (2359768, 2359817)), (2359782, 2359817), '+'), (('G000006925', (2691119, 2691168)), (2691133, 2691168), '+'), (('G000006925', (2770617, 2770666)), (2770617, 2770652), '-'), (('G000006925', (2964188, 2964237)), (2964188, 2964223), '-'), (('G000006925', (3079863, 3079912)), (3079877, 3079912), '+'), (('G000006925', (3178769, 3178818)), (3178783, 3178818), '+'), (('G000006925', (3282350, 3282399)), (3282350, 3282385), '-'), (('G000006925', (3513646, 3513695)), (3513660, 3513695), '+'), (('G000006925', (3605074, 3605123)), (3605088, 3605123), '+'), (('G000006925', (3831378, 3831427)), (3831378, 3831413), '-'), (('G000006925', (3915050, 3915099)), (3915064, 3915099), '+'), (('G000006925', (3993856, 3993905)), (3993856, 3993891), '-'), (('G000006925', (4117280, 4117329)), (4117280, 4117315), '-'), (('G000006925', (4256180, 4256229)), (4256180, 4256215), '-'), (('G000006925', (4360964, 4361013)), (4360978, 4361013), '+'), (('G000006925', (4695142, 4695191)), (4695142, 4695177), '-'), (('G000006925', (4778404, 4778453)), (4778404, 4778439), '-'), (('G000299455', (4579022, 4579199)), (4579022, 4579058), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  d\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  d\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|236124|236173|236138|236173|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|266315|266364|266315|266350|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|495826|495875|495826|495861|-\n",
      "-caagaaaacgatgaaaaatgaactcctcaaagaag-\n",
      ">G000006925|707132|707181|707132|707167|-\n",
      "-caagaaaacgatgaaaaatgaactcctcaaagaag-\n",
      ">G000006925|865149|865198|865149|865184|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|912452|912501|912452|912487|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|980988|981037|980988|981023|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1008963|1009012|1008977|1009012|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|1098042|1098091|1098056|1098091|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|1208850|1208899|1208850|1208885|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1392959|1393008|1392959|1392994|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1418309|1418358|1418309|1418344|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1500579|1500628|1500593|1500628|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|1611215|1611264|1611215|1611250|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1618934|1618983|1618948|1618983|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|1696032|1696081|1696032|1696067|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|1807326|1807375|1807326|1807361|-\n",
      "-caagaaaacgatgaaaaatgaactcctcaaagaag-\n",
      ">G000006925|2036011|2036060|2036011|2036046|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|2359768|2359817|2359782|2359817|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|2691119|2691168|2691133|2691168|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|2770617|2770666|2770617|2770652|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|2964188|2964237|2964188|2964223|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|3079863|3079912|3079877|3079912|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|3178769|3178818|3178783|3178818|+\n",
      "--aagaaaacgatgaaaaatgaactcctcaaagaagc\n",
      ">G000006925|3282350|3282399|3282350|3282385|-\n",
      "-caagaaaacgatgaaaaatgaactcctcaaagaag-\n",
      ">G000006925|3513646|3513695|3513660|3513695|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|3605074|3605123|3605088|3605123|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|3831378|3831427|3831378|3831413|-\n",
      "-caagaaaacgatgaaaaatgaactcctcaaagaag-\n",
      ">G000006925|3915050|3915099|3915064|3915099|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|3993856|3993905|3993856|3993891|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|4117280|4117329|4117280|4117315|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|4256180|4256229|4256180|4256215|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|4360964|4361013|4360978|4361013|+\n",
      "--aagaaaacgatggaaaatgaactcctcaaagaagc\n",
      ">G000006925|4695142|4695191|4695142|4695177|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000006925|4778404|4778453|4778404|4778439|-\n",
      "-caagaaaacgatggaaaatgaactcctcaaagaag-\n",
      ">G000299455|4579022|4579199|4579022|4579058|-\n",
      "gtaagaaaacaatggaagttgagatcctcaaagaag-\n",
      "\n",
      "\n",
      "10 [(('G000006925', (384813, 384925)), (384814, 384871), '+'), (('G000006925', (901227, 901340)), (901281, 901338), '-'), (('G000006925', (3423458, 3423571)), (3423512, 3423568), '-'), (('G000006925', (3850591, 3850703)), (3850592, 3850648), '+'), (('G000006925', (4357823, 4358015)), (4357904, 4357961), '+'), (('G000006925', (4699205, 4699389)), (4699278, 4699334), '+'), (('G000006925', (4774929, 4775042)), (4774930, 4774987), '+'), (('G000006925', (4783199, 4783311)), (4783253, 4783309), '-'), (('G000299455', (4579580, 4579682)), (4579623, 4579680), '-'), (('G000299455', (5343617, 5343740)), (5343671, 5343728), '-')]\n",
      ">G000006925|384813|384925|384814|384871|+\n",
      "-aagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccaggg\n",
      ">G000006925|901227|901340|901281|901338|-\n",
      "taagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000006925|3423458|3423571|3423512|3423568|-\n",
      "-aagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000006925|3850591|3850703|3850592|3850648|+\n",
      "-aagcgccccgaaaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000006925|4357823|4358015|4357904|4357961|+\n",
      "-aagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccaggg\n",
      ">G000006925|4699205|4699389|4699278|4699334|+\n",
      "-aagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000006925|4774929|4775042|4774930|4774987|+\n",
      "taagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000006925|4783199|4783311|4783253|4783309|-\n",
      "-aagcgccccgagaagtacgtagcgtaaggattattttacagacgagaagttccagg-\n",
      ">G000299455|4579580|4579682|4579623|4579680|-\n",
      "taagcgctccgagaagtacgtagcgtaaggattattttacagctgagaagtttcagt-\n",
      ">G000299455|5343617|5343740|5343671|5343728|-\n",
      "taagcgccccgagaagtacgtggcgtaaggattattttacagccgagaggttccagg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4217758, 4228714)), (4226832, 4226871), '-'), (('G000299455', (4221161, 4233966)), (4226516, 4226555), '+'), (('G000299455', (4928594, 4951342)), (4934032, 4934071), '+')]\n",
      ">G000006925|4217758|4228714|4226832|4226871|-\n",
      "caaccgaaccggctgtttgtgtgaagtgattcacatccg-\n",
      ">G000299455|4221161|4233966|4226516|4226555|+\n",
      "-aaccgaaccggctgtttgtgtgaagtgattcacatccgc\n",
      ">G000299455|4928594|4951342|4934032|4934071|+\n",
      "-aaccgaaccggctgtttgtgtgaagtgattcacatccgc\n",
      "\n",
      "\n",
      "7 [(('G000006925', (3065287, 3076770)), (3066489, 3066538), '+'), (('G000006925', (4622488, 4624460)), (4623985, 4624034), '+'), (('G000006925', (4639077, 4639758)), (4639138, 4639187), '-'), (('G000299455', (1992068, 1992414)), (1992115, 1992164), '+'), (('G000299455', (4544552, 4544854)), (4544552, 4544601), '+'), (('G000299455', (4749820, 4751622)), (4751018, 4751067), '+'), (('G000299455', (5313369, 5315826)), (5315222, 5315271), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3065287|3076770|3066489|3066538|+\n",
      "-ggtaaagttcatgcggatgatatccccgttccggtccaggagccgggca\n",
      ">G000006925|4622488|4624460|4623985|4624034|+\n",
      "-ggtaaagtccatgccgatgatatccccgttccggtccaggagccgggca\n",
      ">G000006925|4639077|4639758|4639138|4639187|-\n",
      "cggtaaagtccatgccgatgatatccccgttccggtccaggagccgggc-\n",
      ">G000299455|1992068|1992414|1992115|1992164|+\n",
      "-ggcaaactccatgctgacgatatgcctgtcccggtgctgttgccgggca\n",
      ">G000299455|4544552|4544854|4544552|4544601|+\n",
      "-ggcaaactccatgccgatgataccccggtccaggtactgctgccgggta\n",
      ">G000299455|4749820|4751622|4751018|4751067|+\n",
      "-ggtaaagttcatgcggatgatatccccgttccggtccaggagccgggca\n",
      ">G000299455|5313369|5315826|5315222|5315271|+\n",
      "-ggtaaagtccatgctgatgatatccccgtcccggtccaggagccgggca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1804256, 1805157)), (1805133, 1805157), '+'), (('G000006925', (1805163, 1806570)), (1805163, 1805187), '+'), (('G000299455', (2719777, 2727684)), (2720655, 2720679), '+')]\n",
      ">G000006925|1804256|1805157|1805133|1805157|+\n",
      "taattaaaactgaaattcagcttt\n",
      ">G000006925|1805163|1806570|1805163|1805187|+\n",
      "taattaaagctgaaaataagcttt\n",
      ">G000299455|2719777|2727684|2720655|2720679|+\n",
      "taattaaagctgaaaataagcttt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (905848, 907049)), (906142, 906195), '+'), (('G000006925', (1397676, 1398098)), (1397743, 1397796), '-'), (('G000006925', (2062783, 2064325)), (2064193, 2064246), '+'), (('G000299455', (2541782, 2542216)), (2541861, 2541914), '-'), (('G000299455', (2927401, 2931337)), (2930420, 2930473), '+')]\n",
      ">G000006925|905848|907049|906142|906195|+\n",
      "-gaaattggcctcagattaatgttaagcgtcttgccggacgcgtaatgttaact\n",
      ">G000006925|1397676|1398098|1397743|1397796|-\n",
      "tagattgggcctcagattaatgttaagcgtcttgcaggacgcgtaatgttaac-\n",
      ">G000006925|2062783|2064325|2064193|2064246|+\n",
      "-ggattgagcctcagattaatgttaagcgtcttgcaggacgcgtaatgttaact\n",
      ">G000299455|2541782|2542216|2541861|2541914|-\n",
      "tgaaattggcctcagattaatgttaggcgtcttgcaggacgcgtaatgttaac-\n",
      ">G000299455|2927401|2931337|2930420|2930473|+\n",
      "-agattgggcctcagattaatgttaagcgtcttgcaggacgcgaaatgttaact\n",
      "\n",
      "\n",
      "20 [(('G000006925', (237940, 238704)), (238600, 238636), '+'), (('G000006925', (539107, 539868)), (539766, 539802), '+'), (('G000006925', (709671, 710707)), (710012, 710048), '-'), (('G000006925', (716695, 717456)), (717354, 717390), '+'), (('G000006925', (902326, 903088)), (902985, 903021), '+'), (('G000006925', (1095979, 1096741)), (1096046, 1096082), '-'), (('G000006925', (1632621, 1633383)), (1633280, 1633316), '+'), (('G000006925', (1834903, 1835664)), (1834969, 1835005), '-'), (('G000006925', (1878572, 1883763)), (1879231, 1879267), '+'), (('G000006925', (1944438, 1945202)), (1945097, 1945133), '+'), (('G000006925', (1962084, 1962845)), (1962743, 1962779), '+'), (('G000006925', (2069615, 2084300)), (2070274, 2070310), '+'), (('G000006925', (2346737, 2347498)), (2347396, 2347432), '+'), (('G000006925', (2477790, 2478551)), (2478449, 2478485), '+'), (('G000006925', (2953405, 2954171)), (2953472, 2953508), '-'), (('G000006925', (4392644, 4394283)), (4393588, 4393624), '-'), (('G000006925', (4798615, 4798792)), (4798615, 4798651), '+'), (('G000006925', (4823851, 4824717)), (4824022, 4824058), '-'), (('G000299455', (928393, 928731)), (928625, 928661), '+'), (('G000299455', (4559929, 4560763)), (4560587, 4560622), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1108\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|237940|238704|238600|238636|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|539107|539868|539766|539802|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|709671|710707|710012|710048|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000006925|716695|717456|717354|717390|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|902326|903088|902985|903021|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|1095979|1096741|1096046|1096082|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000006925|1632621|1633383|1633280|1633316|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|1834903|1835664|1834969|1835005|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000006925|1878572|1883763|1879231|1879267|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|1944438|1945202|1945097|1945133|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|1962084|1962845|1962743|1962779|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|2069615|2084300|2070274|2070310|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|2346737|2347498|2347396|2347432|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|2477790|2478551|2478449|2478485|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000006925|2953405|2954171|2953472|2953508|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000006925|4392644|4394283|4393588|4393624|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000006925|4798615|4798792|4798615|4798651|+\n",
      "-aaactctgcgctgaaattcctttttttcattggagt\n",
      ">G000006925|4823851|4824717|4824022|4824058|-\n",
      "taaactctgcgctaaaatttctttttttcattggag-\n",
      ">G000299455|928393|928731|928625|928661|+\n",
      "-aaactctgcgctaaaatttctttttttcattggagc\n",
      ">G000299455|4559929|4560763|4560587|4560622|+\n",
      "-aaactctgcgctgaaatttc-ttttttcattatggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (489413, 489443), '-'), (('G000299455', (316456, 316486)), (316456, 316486), '-'), (('G000299455', (3819953, 3834467)), (3832275, 3832305), '+')]\n",
      ">G000006925|487251|491968|489413|489443|-\n",
      "ggtgacaggaaaatgacaaaattgtcattt-\n",
      ">G000299455|316456|316486|316456|316486|-\n",
      "ggtgacaggataatgacaatgttgtcattt-\n",
      ">G000299455|3819953|3834467|3832275|3832305|+\n",
      "-gtgacagaaaaatgacaaaattgtcatttt\n",
      "\n",
      "\n",
      "10 [(('G000006925', (1209065, 1211067)), (1210883, 1210936), '+'), (('G000006925', (2756889, 2766574)), (2757964, 2758017), '+'), (('G000006925', (2923039, 2950048)), (2949079, 2949132), '+'), (('G000006925', (3101814, 3103074)), (3102890, 3102943), '+'), (('G000006925', (3810378, 3811029)), (3810509, 3810562), '-'), (('G000006925', (4624461, 4624660)), (4624472, 4624526), '+'), (('G000006925', (4628976, 4629562)), (4629377, 4629430), '+'), (('G000006925', (4825967, 4826151)), (4825967, 4826019), '+'), (('G000299455', (3226712, 3229721)), (3229533, 3229586), '+'), (('G000299455', (5275700, 5279438)), (5279320, 5279373), '-')]\n",
      ">G000006925|1209065|1211067|1210883|1210936|+\n",
      "-agtggaatgtatccatggagaacact-ttatcagccgggaaataatgcgggcaa\n",
      ">G000006925|2756889|2766574|2757964|2758017|+\n",
      "-agtggaatgtatccatggagaacact-ttatcagccgggaaataatgcgggcaa\n",
      ">G000006925|2923039|2950048|2949079|2949132|+\n",
      "-agtggaatgtatccatggagaacact-ttatcagccgggaaataatgcgggcaa\n",
      ">G000006925|3101814|3103074|3102890|3102943|+\n",
      "-agtggaatgtatccatggagaacact-ttatcagccgggaaataatgcgggcaa\n",
      ">G000006925|3810378|3811029|3810509|3810562|-\n",
      "aggtggaatgtatccacggggaacgct-ttagcagccgggaaataatgcgggca-\n",
      ">G000006925|4624461|4624660|4624472|4624526|+\n",
      "aaatggaatgtctccacggagaacactattatccatcgggaaatagtgcgggca-\n",
      ">G000006925|4628976|4629562|4629377|4629430|+\n",
      "aagtggaatgtctccacggagaacact-ttatcagccgggaaataatgagggca-\n",
      ">G000006925|4825967|4826151|4825967|4826019|+\n",
      "-agtggaatgtctccacggagaacact-ttatctatcgggaaatagtgcgggca-\n",
      ">G000299455|3226712|3229721|3229533|3229586|+\n",
      "-agtggaatgtatccatggagaacact-ttatcagccgggaaataatgcgggcaa\n",
      ">G000299455|5275700|5279438|5279320|5279373|-\n",
      "aagtggaatgtctccacgaagaacatt-ttatcagccgggaaataatgcgggca-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3107661, 3117031)), (3107760, 3108112), '+'), (('G000006925', (3736321, 3788106)), (3751232, 3751584), '-'), (('G000299455', (137973, 154383)), (148226, 148578), '+'), (('G000299455', (830050, 839519)), (839059, 839411), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1706\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1706\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3107661|3117031|3107760|3108112|+\n",
      "-tcagactgtgcttcacggtgtagcggaacagttcagattctcggcccaccatgcccgtc\n",
      "gcggcgcaggccacggcgatagattgcggggagatcatcttgccagttacgccgccgctg\n",
      "gtatttgctgccaccagcagagtgtcagagacgttgatttgctgcgccgtggtcgattgc\n",
      "agtgaaccaaacagggcgtttgaagaagtgtccgacccggtaaggaatacgcccagccag\n",
      "ccgagaaacggtgagaaaaacgggaacatcacgcctgtacctgccagtaccagcgccagc\n",
      "gtggtggacatgccggaatagttggtgacgaaggcgaatgccagcaccatgcc\n",
      ">G000006925|3736321|3788106|3751232|3751584|-\n",
      "atcaggctgtgtttgacagtaaagcggaacaaatcagactctttgcccaccaggcctacc\n",
      "gccgcacaggcgatagcgatagattgcggggagatcatcttaccggtgacgccaccggtg\n",
      "gtattggcggcaaccagcaacagatcagagacgccaatttgttgtgctgcggtggcttgc\n",
      "agcgcggcgaacagggcgttagatgaggtatccgacccggtcaggaatacccccagccag\n",
      "ccgaggaacggcgagaagaaggtgaatgcatgaccggtgtgcgccagtgccagtgccagt\n",
      "gttgatgacagtccggaatagttcgagataaaggcgaacgccagcaccatgc-\n",
      ">G000299455|137973|154383|148226|148578|+\n",
      "-tcaggctgtgtttgacagtaaagcggaacaaatcagactctttgcccaccaggcctacc\n",
      "gccgcacaggcgatagcgatagattgcggggagatcatcttaccggtgacgccaccggtg\n",
      "gtattggcggcaaccagcaacagatcagagacgccaatttgttgtgctgcggtggcttgc\n",
      "agcgcggcgaacagggcgttagatgaggtatccgacccggtcaggaatacccccagccag\n",
      "ccgaggaacggcgagaagaaggtgaatgcatgaccggtgtgcgccagtgccagtgccagt\n",
      "gttgatgacagtccggaatagttcgagataaaggcgaacgccagcaccatgcc\n",
      ">G000299455|830050|839519|839059|839411|-\n",
      "atcagactgtgcttcacggtgtagcggaatagttcagattctcggcccaccatgcccgtc\n",
      "gcggcgcaggccacggcgatagattgcggggagatcatcttgccagttacgccgccgctg\n",
      "gtatttgctgccaccagcagagtgtcagagacgttgatttgctgcgccgtggtcgattgc\n",
      "agtgaaccaaacagggcgtttgaagaagtgtccgagccggtaaggaatacccccagccag\n",
      "ccgaggaacggtgagaaaaacgggaacatcacgcctgtacctgccagtaccagcgccaac\n",
      "gtggtggacatgccggaatagttggtgacgaaggcgaacgccagcaccatgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2849870, 2870340)), (2849877, 2849897), '-'), (('G000299455', (1161196, 1161223)), (1161196, 1161216), '+'), (('G000299455', (1188881, 1188909)), (1188883, 1188903), '+'), (('G000299455', (1188940, 1202325)), (1188944, 1188964), '+')]\n",
      ">G000006925|2849870|2870340|2849877|2849897|-\n",
      "gttccccgctccagcggaga-\n",
      ">G000299455|1161196|1161223|1161196|1161216|+\n",
      "-ttccccgcgccagcggggat\n",
      ">G000299455|1188881|1188909|1188883|1188903|+\n",
      "-ttccccgcgccagcggggat\n",
      ">G000299455|1188940|1202325|1188944|1188964|+\n",
      "-ttccccgcgccagcggggat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3423912, 3503727)), (3457507, 3457594), '-'), (('G000299455', (434155, 488092)), (480265, 480352), '+'), (('G000299455', (4584877, 4585231)), (4584877, 4584964), '+')]\n",
      ">G000006925|3423912|3503727|3457507|3457594|-\n",
      "atcgcttcctgataggcttctaaagcatcatgggcgtttttacgcaataacccgctctgc\n",
      "cactgcggccacagccagacggtacca-\n",
      ">G000299455|434155|488092|480265|480352|+\n",
      "-tcgcttcctgataggcttctaaagcatcatgggcgtttttacgcaataacccgctctgc\n",
      "cactgcggccacagccagacggtaccac\n",
      ">G000299455|4584877|4585231|4584877|4584964|+\n",
      "-tcactttctgataggcttctgaagcgtcatgggcattcttacgcaataacccgttatac\n",
      "cactgtgaccacaaccagacggtaccgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3717879, 3736288)), (3727780, 3727818), '+'), (('G000006925', (3736321, 3788106)), (3778737, 3778775), '-'), (('G000299455', (187793, 189382)), (187795, 187833), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3717879|3736288|3727780|3727818|+\n",
      "-tttttattcccgatctcatcatcctccctttgaaatgg\n",
      ">G000006925|3736321|3788106|3778737|3778775|-\n",
      "gtttttattcccgatctcatcatcctccctttgaaatg-\n",
      ">G000299455|187793|189382|187795|187833|-\n",
      "gtttttattcccgatctcatcatcctccctttgaaatg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2034878, 2034928)), (2034878, 2034928), '+'), (('G000006925', (3606199, 3606249)), (3606199, 3606249), '-'), (('G000299455', (2065961, 2066011)), (2065961, 2066011), '-')]\n",
      ">G000006925|2034878|2034928|2034878|2034928|+\n",
      "-aaaatatgacgctaacaatgtttatgctgctgtgatgtacggtgaatctc\n",
      ">G000006925|3606199|3606249|3606199|3606249|-\n",
      "caaaatatgacgctaacaatgtttatgctgctgtgatgtacggtgaatct-\n",
      ">G000299455|2065961|2066011|2065961|2066011|-\n",
      "caaagtatgacgcgaaaaatgtctacgcagctgtaatgtacggcgaaact-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3087143, 3087229), '-'), (('G000299455', (857924, 871846)), (868281, 868367), '+'), (('G000299455', (3152907, 3154652)), (3153599, 3153687), '+')]\n",
      ">G000006925|3083675|3099254|3087143|3087229|-\n",
      "gtctattgcacggacaagagagacagtaacg--aaaacagcgcttttgtgagcagactca\n",
      "gaggcagggaggacatcatccacaacag-\n",
      ">G000299455|857924|871846|868281|868367|+\n",
      "-gctattgcacggacaagagagacagtaacg--aaaacagcgcttttgtgagcagactca\n",
      "gaggcagggaggacatcatccacaacagg\n",
      ">G000299455|3152907|3154652|3153599|3153687|+\n",
      "-gctattgcacggacaagagagacagtaacgaaaaaacagtgcttttgtgagcagactca\n",
      "gagtcagggagggcatcagtcacgacagg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2366334, 2374443)), (2373621, 2373643), '-'), (('G000006925', (2374481, 2472008)), (2431665, 2431689), '+'), (('G000299455', (1623109, 1682614)), (1648796, 1648820), '-'), (('G000299455', (1685057, 1714323)), (1707013, 1707035), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2366334|2374443|2373621|2373643|-\n",
      "--ctgatatttatgatgacgcgac-\n",
      ">G000006925|2374481|2472008|2431665|2431689|+\n",
      "-cctcaaatttaagatgacgcgaca\n",
      ">G000299455|1623109|1682614|1648796|1648820|-\n",
      "ccctcaaatttaagatgacgcgac-\n",
      ">G000299455|1685057|1714323|1707013|1707035|+\n",
      "---tgatatttatgatgacgcgaca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1509064, 1578296)), (1576360, 1576396), '+'), (('G000299455', (2167849, 2167987)), (2167951, 2167987), '-'), (('G000299455', (2923865, 2925880)), (2923947, 2923983), '+')]\n",
      ">G000006925|1509064|1578296|1576360|1576396|+\n",
      "-gcctgcatcagggtataaggggttatggccaccgga\n",
      ">G000299455|2167849|2167987|2167951|2167987|-\n",
      "cgcctgcatcagtgtgtacggcgtaatggcaaccgg-\n",
      ">G000299455|2923865|2925880|2923947|2923983|+\n",
      "-gcctgcatcagggtataaggggttatggccaccgga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2435700, 2435760), '-'), (('G000299455', (1623109, 1682614)), (1644724, 1644784), '+'), (('G000299455', (3480865, 3520974)), (3484903, 3484963), '+')]\n",
      ">G000006925|2374481|2472008|2435700|2435760|-\n",
      "ttgttggctttgatatcgatctcggtaacgagatgtgcaaacggatgcaggtgaaatgta\n",
      "-\n",
      ">G000299455|1623109|1682614|1644724|1644784|+\n",
      "-tgttggctttgatatcgatctcggtaacgagatgtgcaaacggatgcaggtgaaatgta\n",
      "c\n",
      ">G000299455|3480865|3520974|3484903|3484963|+\n",
      "-tgtcggctttgatatcgatctggcaaaagccttgtgcaaacaaatgcaggcagaatgta\n",
      "c\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1776250, 1792578)), (1776719, 1776749), '+'), (('G000299455', (2674304, 2675510)), (2674773, 2674803), '+'), (('G000299455', (4086247, 4086277)), (4086247, 4086277), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1776250|1792578|1776719|1776749|+\n",
      "-cgtggcggaaggggtcgaaaccaaagagca\n",
      ">G000299455|2674304|2675510|2674773|2674803|+\n",
      "-cgtggcggaaggggtcgaaaccaaagagca\n",
      ">G000299455|4086247|4086277|4086247|4086277|-\n",
      "tcgtggcggaaggggtagaaacccaggagc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2791567, 2791623), '-'), (('G000299455', (1237061, 1253885)), (1253805, 1253861), '+'), (('G000299455', (1253944, 1254160)), (1254078, 1254134), '+')]\n",
      ">G000006925|2770832|2816659|2791567|2791623|-\n",
      "atccgtagctcagctggatagagtactcggctacgaaccgagcggtcggaggttcg-\n",
      ">G000299455|1237061|1253885|1253805|1253861|+\n",
      "-tccgtagctcagctggatagagtactcggctacgaaccgagcggtcggaggttcga\n",
      ">G000299455|1253944|1254160|1254078|1254134|+\n",
      "-tccgtagctcagctggatagagtactcggctacgaaccgagcggtcggaggttcga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4450485, 4460192)), (4457353, 4457400), '+'), (('G000299455', (3973381, 4009751)), (3996722, 3996769), '+'), (('G000299455', (4658720, 4667733)), (4665590, 4665637), '+')]\n",
      ">G000006925|4450485|4460192|4457353|4457400|+\n",
      "atcggaatggagttaatcgcgcgtggcagtgatttctccggatcttt\n",
      ">G000299455|3973381|4009751|3996722|3996769|+\n",
      "atcggcacggagttaatcgcacgcggtatcgatttctcaggatcttt\n",
      ">G000299455|4658720|4667733|4665590|4665637|+\n",
      "atcggaatagagttaattgcacgcggcagtgatttctccggatcttt\n",
      "\n",
      "\n",
      "10 [(('G000006925', (1815406, 1827803)), (1826289, 1826332), '+'), (('G000006925', (1965007, 1976679)), (1968461, 1968505), '+'), (('G000006925', (1965007, 1976679)), (1970163, 1970206), '+'), (('G000006925', (3205438, 3224454)), (3208388, 3208431), '+'), (('G000006925', (4557883, 4573403)), (4559063, 4559106), '-'), (('G000299455', (715333, 739463)), (736451, 736494), '-'), (('G000299455', (2122044, 2133716)), (2128535, 2128578), '-'), (('G000299455', (2122044, 2133716)), (2130218, 2130261), '-'), (('G000299455', (2737179, 2754578)), (2748427, 2748470), '+'), (('G000299455', (4487440, 4499283)), (4498060, 4498103), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\n",
      "Reallocating..done. *alloclen = 1105\n",
      "\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1815406|1827803|1826289|1826332|+\n",
      "-------------------gcgcggcttcaacggcagcattcagtgccagaatattcgtc\n",
      "tg\n",
      ">G000006925|1965007|1976679|1968461|1968505|+\n",
      "------------------cgagccgcttccactgccgcattcagggccagaatattggtc\n",
      "tg\n",
      ">G000006925|1965007|1976679|1970163|1970206|+\n",
      "-------------------gcgcggcttcaaccgcggcattcagcgcgaggatattagtc\n",
      "tg\n",
      ">G000006925|3205438|3224454|3208388|3208431|+\n",
      "-------------------gcgccgcttccaccgccgcattcagggccagaatattggtc\n",
      "tg\n",
      ">G000006925|4557883|4573403|4559063|4559106|-\n",
      "------------------cgcgctgcttcaaccgccgcgttcaaagccagaatattggtc\n",
      "t-\n",
      ">G000299455|715333|739463|736451|736494|-\n",
      "------------------cgcgccgcttccaccgccgcattcagggccagaatattggtc\n",
      "t-\n",
      ">G000299455|2122044|2133716|2128535|2128578|-\n",
      "cggccctgttcacccgcacgcgcggcttcaaccgcggcattca-----------------\n",
      "--\n",
      ">G000299455|2122044|2133716|2130218|2130261|-\n",
      "------------------cgagccgcttccactgccgcattcagggccagaatattggtc\n",
      "t-\n",
      ">G000299455|2737179|2754578|2748427|2748470|+\n",
      "-------------------gcgcggcttcaacggcagcattcagtgccagaatattcgtc\n",
      "tg\n",
      ">G000299455|4487440|4499283|4498060|4498103|+\n",
      "-------------------gcgctgcttcaaccgccgcgttcaaagccagaatattggtc\n",
      "tg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3096664, 3097038), '-'), (('G000299455', (47809, 50193)), (49819, 50193), '+'), (('G000299455', (857924, 871846)), (858515, 858889), '+'), (('G000299455', (1978312, 1978956)), (1978312, 1978687), '+'), (('G000299455', (3142569, 3143200)), (3142569, 3142929), '+'), (('G000299455', (4771847, 4785261)), (4782300, 4782674), '-')]\n",
      ">G000006925|3083675|3099254|3096664|3097038|-\n",
      "gtcggataaacagccagataaacgtaaccacaggagccgagggtgtcggcttcgcaggtc\n",
      "agccctttcgcatacagtgtgacggtatgctgatggcggggattcagttcaccgccagtg\n",
      "agcatgagttccagttgtttcatcagcagcggaaaggcctggtccaggtggtacgcatct\n",
      "acgtcgctgaacctgcctctgataccggcgcggtctgcaaggtaatgcaggcggttaccc\n",
      "tcctgtaccagtcgggccccgaaacagggcgtcacggtgcagggcagcccccaccagggg\n",
      "cggtcgtgattgtcgtcgggaggcgttgtcccggggagtgtgtctgacacgataaagtcc\n",
      "tcacagaaaatcgg--\n",
      ">G000299455|47809|50193|49819|50193|+\n",
      "-tcggataaacagccagataagcgtaaccacaactgccaagggtatcggcttcgcaggtc\n",
      "agccccttcgcatacagtgtgatggtatgctgatggcggggattcagttcaccgctggtg\n",
      "agcatgagttccagttgtttcatcagcagcggaaaggcctggtccaggtggtatgcatcc\n",
      "gcgtcgctgaaccggcctctgataccggcacggtcggcaaggtaatgcaaccggttaccc\n",
      "tcctgcaccagacgggcgccgaaacagggcgtcactgtgcagggtagtccccaccaggga\n",
      "tggtcgtcattgtcatcgggatgcgttgtcccggaaagtgtgtctgacacgataaaatcc\n",
      "tcacagaaaatcggt-\n",
      ">G000299455|857924|871846|858515|858889|+\n",
      "-tcggataaacagccagataaacgtaaccacaggagccgagggtgtcggcttcgcaggtc\n",
      "agccctttcgcatacagtgtcatggtatgctgatggcggggattcagttcaccactggtg\n",
      "agcatgagctccagttgtttcatcagcagcggaaaggcctggtccaggtggtatgcatct\n",
      "gcatcgctgaaccggcctctgataccggcgcggtcggcaaggtaatgcaaccggttaccc\n",
      "ttctgcaccagacgtgccccgaaacagggcgtcacggtgcagggcagtccccaccagagg\n",
      "cggtcgtgattgtcgtcgggaggcgttgtcccggggagtgtgtctgacacgataaagtcc\n",
      "tcacagaaaatcggt-\n",
      ">G000299455|1978312|1978956|1978312|1978687|+\n",
      "-tcggataaacagccagataaacgtaaccacaactgctgagggtatcggctttgcaggtc\n",
      "agcccttttgcatacagcgtgacggtatgctgatggcggggattcagttcaccgctggtg\n",
      "agcatgagttccagttgtttcatcagcagcggaaaggcctggtccaggtggtacgcatct\n",
      "gcatcgctgaacaggcctctgataccggcgcggtcggcaaggtaatgcaaccggttaccc\n",
      "tcctgcaccagacgtgccccgaaacagggcgtcacggtgcagggcagcccccaccagggg\n",
      "cggtcgtgattgtcgtcgggaagtgttgtcccggggagtgtgtctgacacgataaaatcc\n",
      "ttacagaaaatcggct\n",
      ">G000299455|3142569|3143200|3142569|3142929|+\n",
      "-tcggataaacagccagataaacgtaaccacaggagccgagggtgtcggcatcgcaggtc\n",
      "agccctttagcatatagcgtgacggtatgctgatggcggggattcagttcaccgctggtg\n",
      "agcatgagttccagttgtttcatcagcagcggaaaggcctggtccaggtggtacgcatct\n",
      "gcatcgctgaacaggcctctgataccggcgcggtcggcaaggtaatgcaaccggttaccc\n",
      "tcctgcaccagacgggcgccgaaacagggcgtcacggtgcagggcagcccccaccagggg\n",
      "cggtcgtgattgtcgtcggga---------------agtgtgtctgacacgataaagtcc\n",
      "tcacagaaaatcggtt\n",
      ">G000299455|4771847|4785261|4782300|4782674|-\n",
      "gtcggataaacagccagataaacgtaaccacaggagccgagggtgtcggcttcgcaggtc\n",
      "agccctttcgcatacagtgtgacggtatgctgatggcggggattcagttcaccgccagtg\n",
      "agcatgagttccagttgtttcatcagcagcggaaaggcctggtccaggtggtacgcatct\n",
      "acgtcgctgaacctgcctctgataccggcgcggtctgcaaggtaatgcaggcggttaccc\n",
      "tcctgtaccagtcgggccccgaaacagggcgtcacggtgcagggcagcccccaccagggg\n",
      "cggtcgtgattgtcgtcgggaggcgttgtcccggggagtgtgtctgacacgataaagtcc\n",
      "tcacagaaaatcgg--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2412247, 2412290), '-'), (('G000299455', (1225462, 1225853)), (1225744, 1225787), '+'), (('G000299455', (1623109, 1682614)), (1668207, 1668250), '+')]\n",
      ">G000006925|2374481|2472008|2412247|2412290|-\n",
      "gataaattcatcccggttgatgtgtatatcccgggctgcccgc-\n",
      ">G000299455|1225462|1225853|1225744|1225787|+\n",
      "-ataaaatcgtcccggtagatgtctacatccctggttgcccgcc\n",
      ">G000299455|1623109|1682614|1668207|1668250|+\n",
      "-ataaattcatcccggttgatgtgtatatcccgggctgcccgcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1425133, 1465118)), (1446833, 1447137), '+'), (('G000006925', (4250578, 4250882)), (4250578, 4250882), '+'), (('G000299455', (2199778, 2263225)), (2244936, 2245240), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1616\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1616\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1425133|1465118|1446833|1447137|+\n",
      "agcagccttatccgcagcgtgctggaaagcaacggtaatagtacgaacgatgatggtcag\n",
      "tacctggcccgcagcggccagagggattgccagtgcgatacctgcaccaatgctctgatg\n",
      "acctgcgataaccagaatggtagaaatgatagaagccagagcggcgtcaggcgcaactgc\n",
      "agcaccgatgttcatccagcccagcgcgatcatttccagcgtaccaccgataataatacc\n",
      "ggttttcatatccccaagaacgatacccaccagggtacacgcgattagcggacggtgaaa\n",
      "ctga--------\n",
      ">G000006925|4250578|4250882|4250578|4250882|+\n",
      "--------tatccgccgcgtgctggaacactacggtgatggtacgggcaaacaccgtcag\n",
      "cacctggcctgccgcagccacaggcaacgcgatggcgattccggtggcgatgctttgctg\n",
      "accaacgataaccagaatggcggagattatgctggcgagtgcagaatccggagattgcgc\n",
      "cgcgccgacgttcatccagcagagagctatcaattccagcgtaccgccgagtattattcc\n",
      "ggtttttaaatcaccgagaattaaaccaatcaccgtacaggcaattaacgggcgatgagt\n",
      "ctgaaattcatc\n",
      ">G000299455|2199778|2263225|2244936|2245240|+\n",
      "--------tatccgcagcgtgctggaaagcaacggtaatagtacgaacgatgatggtcag\n",
      "tacctggcccgcagcggccagagggattgccagtgcgatacctgcaccaatgctctgatg\n",
      "acctgcgataaccagaatggtagaaatgatagaagccagagcggcgtcaggcgcaactgc\n",
      "agcaccgatgttcatccagcccagcgcgatcatttccagcgtaccaccgataataatacc\n",
      "ggttttcatatccccaagaacgatacccaccagggtacacgcgattagcggacggtgaaa\n",
      "ctgaaattcatc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3969227, 3993100)), (3973199, 3973267), '+'), (('G000299455', (1345136, 1375168)), (1345527, 1345595), '+'), (('G000299455', (5156132, 5190970)), (5186930, 5186998), '-')]\n",
      ">G000006925|3969227|3993100|3973199|3973267|+\n",
      "-tcgtagttaaagacgtgcgtcactgccggaatatgcaaaccacgcgcggcaacgtcggt\n",
      "ggcaaccag\n",
      ">G000299455|1345136|1375168|1345527|1345595|+\n",
      "-tcgaagttaaagacgtggctgacgtcaggaatgtcgataccgcgcgcggcaacatcggt\n",
      "tgcaaccag\n",
      ">G000299455|5156132|5190970|5186930|5186998|-\n",
      "atcgtagttaaagacgtgcgtcactgccggaatatgcaaaccacgcgcggcaacgtcggt\n",
      "ggcaacca-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1696422, 1718400)), (1712276, 1712615), '+'), (('G000006925', (1815406, 1827803)), (1815675, 1816014), '+'), (('G000006925', (2006629, 2008384)), (2007259, 2007598), '+'), (('G000299455', (2269177, 2270929)), (2269962, 2270301), '-'), (('G000299455', (2737179, 2754578)), (2737807, 2738146), '+'), (('G000299455', (4667815, 4670271)), (4668857, 4669196), '+')]\n",
      ">G000006925|1696422|1718400|1712276|1712615|+\n",
      "-tggtttttctgaaaactgtttacaggctcaaagactgtgccatctgacagcgtggcgag\n",
      "tttagccacgccagcatccagcccgaccattgatgctgaagggtgagccggagtggatac\n",
      "ttcactttctgtctgaatactgatgtaccacgtaccgcaggactggctgacagtgacatt\n",
      "tttcacaacacccgtgacctgacggctattccggtagcgcatccagcccagtttcggcag\n",
      "aaaaatacggctgttttcctgatcgagcttaacgccctgcgggtagcggaatgcatcatt\n",
      "ctgtcaccactttttgaatcggggaaaagcagcccgcttc\n",
      ">G000006925|1815406|1827803|1815675|1816014|+\n",
      "-tggtttttctgaaaactgtttacaggctcaaagactgtgccatctgacagcgtggcgag\n",
      "tttagccacgccagcatccagcccgaccattgatgctgaagggtgagccggagtggatac\n",
      "ttcactttctgtctgaatactgatgtaccacgtaccgcaggactggctgacagtgacatt\n",
      "tttcacaacacccgtgacctgacggctattccggtagcgcatccagcccagtttcggcag\n",
      "aaaaatacggctgttttcctgatcgagcttaacgccctgcgggtagcggaatgcatcatt\n",
      "ctgtcaccactttttgaatcggggaaaagcagcccgcttc\n",
      ">G000006925|2006629|2008384|2007259|2007598|+\n",
      "-tggtttttctgaaaactgtttacaggctcaaagactgtgccatctgacagcgaggcaag\n",
      "tttagccacgccagcatccagcccgaccattgatgctgaagggtgaaccggagttgatac\n",
      "ttcactttctgcctgaatactgatgtaccacttaccgcaggactggctgacagtgacatt\n",
      "tttcacaacacccgtgacctgacggctgttccggtagcgcatccagccaagttttggcag\n",
      "aaaaatacggctgttttcctgagcgagcttaacgccctgcggatagcggaatgcatcatt\n",
      "ctgtcaccgctttttgaatcaggggaacgccgagcgcttc\n",
      ">G000299455|2269177|2270929|2269962|2270301|-\n",
      "ctggtttttctgaaaactgtttacaggcccaaagactgtgccatctgacagcgtggcgag\n",
      "tttagccacgccagcatccagcccgatcattgatgctgaagggtgaaccggagttgatac\n",
      "ttcactttctgtctgaatactgatgtaccacttaccgcaggactggctgaccgtgacatt\n",
      "tttcacaacacccgtgacctgacggctattccggtagcgcatccagcccagtttcggcag\n",
      "aaaaatacggctgttttcctgatcgagctttacgccctgcgggtagcggaatgcatcatt\n",
      "ctgtccccgctttttgaatcggggaaaagcagcccgctt-\n",
      ">G000299455|2737179|2754578|2737807|2738146|+\n",
      "-tggtttttctgaaaactgtttacaggctcaaagactgtgccatctgacagcgtggcgag\n",
      "tttagccacgccagcatccagcccgaccattgatgctgaagggtgagccggagtggatac\n",
      "ttcactttctgtctgaatactgatgtaccacttaccgcaggactggctgacagtgacatt\n",
      "tttcacaacacccgtgacctgacggctattccggtagcgcatccagcccagtttcggcag\n",
      "aaaaatacggctgttttcctgatcgagcttaacgccctgcgggtagcggaatgcatcatt\n",
      "ctgtccccgctttttgaatcggggaaaagcagcccgcttc\n",
      ">G000299455|4667815|4670271|4668857|4669196|+\n",
      "-tggtttttctgaaaactgtttacaggctcaaagactgtaccatctgacagcgtggcgag\n",
      "tttagccacgccagcatccagcccgaccattgatgctgaagggtgaaccggagtggatac\n",
      "ttcactttcagtctgaacactgatgtaccacttaccgcaggactggctgacagtgacatt\n",
      "tttcacaacacccgcgacctgacggctattccggtagcgcatccagccaagtttcggcag\n",
      "aaaaatacggctgttttcctgatcgagcttaacgccctgcgggtagcggaatgcatcatt\n",
      "ctgtccccgctttttgaatcggggaaaagcagcccgcttc\n",
      "\n",
      "\n",
      "23 [(('G000006925', (222642, 222675)), (222642, 222675), '+'), (('G000006925', (411072, 411105)), (411072, 411105), '-'), (('G000006925', (763653, 783898)), (783865, 783898), '-'), (('G000006925', (942508, 942541)), (942508, 942541), '-'), (('G000006925', (964651, 964684)), (964651, 964684), '-'), (('G000006925', (1290847, 1306886)), (1290847, 1290880), '+'), (('G000006925', (1507639, 1507672)), (1507639, 1507672), '-'), (('G000006925', (2225776, 2225809)), (2225776, 2225809), '-'), (('G000006925', (2479115, 2479148)), (2479115, 2479148), '+'), (('G000006925', (2735523, 2735556)), (2735523, 2735556), '+'), (('G000006925', (2886690, 2886723)), (2886690, 2886723), '-'), (('G000006925', (3203998, 3204031)), (3203998, 3204031), '-'), (('G000006925', (3536714, 3536747)), (3536714, 3536747), '+'), (('G000006925', (3640025, 3640058)), (3640025, 3640058), '-'), (('G000006925', (3919037, 3919070)), (3919037, 3919070), '+'), (('G000006925', (3919072, 3938484)), (3938451, 3938484), '-'), (('G000006925', (4012439, 4012472)), (4012439, 4012472), '-'), (('G000006925', (4173097, 4173130)), (4173097, 4173130), '-'), (('G000006925', (4433872, 4433905)), (4433872, 4433905), '+'), (('G000006925', (4622451, 4622484)), (4622451, 4622484), '+'), (('G000006925', (4697193, 4697226)), (4697193, 4697226), '+'), (('G000006925', (4792237, 4792270)), (4792237, 4792270), '-'), (('G000299455', (2940021, 2958111)), (2958078, 2958111), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 23\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 23\r",
      "   10 / 23\r",
      "   20 / 23\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 22  f\b\b\r",
      "STEP     2 / 22  f\b\b\r",
      "STEP     3 / 22  f\b\b\r",
      "STEP     4 / 22  f\b\b\r",
      "STEP     5 / 22  f\b\b\r",
      "STEP     6 / 22  f\b\b\r",
      "STEP     7 / 22  f\b\b\r",
      "STEP     8 / 22  f\b\b\r",
      "STEP     9 / 22  f\b\b\r",
      "STEP    10 / 22  f\b\b\r",
      "STEP    11 / 22  f\b\b\r",
      "STEP    12 / 22  f\b\b\r",
      "STEP    13 / 22  f\b\b\r",
      "STEP    14 / 22  f\b\b\r",
      "STEP    15 / 22  f\b\b\r",
      "STEP    16 / 22  f\b\b\r",
      "STEP    17 / 22  f\b\b\r",
      "STEP    18 / 22  f\b\b\r",
      "STEP    19 / 22  f\b\b\r",
      "STEP    20 / 22  f\b\b\r",
      "STEP    21 / 22  f\b\b\r",
      "STEP    22 / 22  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 23\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 23\r",
      "   10 / 23\r",
      "   20 / 23\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 22  f\b\b\r",
      "STEP     2 / 22  f\b\b\r",
      "STEP     3 / 22  f\b\b\r",
      "STEP     4 / 22  f\b\b\r",
      "STEP     5 / 22  f\b\b\r",
      "STEP     6 / 22  f\b\b\r",
      "STEP     7 / 22  f\b\b\r",
      "STEP     8 / 22  f\b\b\r",
      "STEP     9 / 22  f\b\b\r",
      "STEP    10 / 22  f\b\b\r",
      "STEP    11 / 22  f\b\b\r",
      "STEP    12 / 22  f\b\b\r",
      "STEP    13 / 22  f\b\b\r",
      "STEP    14 / 22  f\b\b\r",
      "STEP    15 / 22  f\b\b\r",
      "STEP    16 / 22  f\b\b\r",
      "STEP    17 / 22  f\b\b\r",
      "STEP    18 / 22  f\b\b\r",
      "STEP    19 / 22  f\b\b\r",
      "STEP    20 / 22  f\b\b\r",
      "STEP    21 / 22  f\b\b\r",
      "STEP    22 / 22  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|222642|222675|222642|222675|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|411072|411105|411072|411105|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|763653|783898|783865|783898|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|942508|942541|942508|942541|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|964651|964684|964651|964684|-\n",
      "ggatcggtcaactgatccttaactgatcggtat-\n",
      ">G000006925|1290847|1306886|1290847|1290880|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|1507639|1507672|1507639|1507672|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|2225776|2225809|2225776|2225809|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|2479115|2479148|2479115|2479148|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|2735523|2735556|2735523|2735556|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|2886690|2886723|2886690|2886723|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|3203998|3204031|3203998|3204031|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|3536714|3536747|3536714|3536747|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|3640025|3640058|3640025|3640058|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|3919037|3919070|3919037|3919070|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|3919072|3938484|3938451|3938484|-\n",
      "ggatcggtcaactgatccttaactgatcggtat-\n",
      ">G000006925|4012439|4012472|4012439|4012472|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|4173097|4173130|4173097|4173130|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000006925|4433872|4433905|4433872|4433905|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|4622451|4622484|4622451|4622484|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|4697193|4697226|4697193|4697226|+\n",
      "-gatcggtcaactgatccttaactgatcggcatt\n",
      ">G000006925|4792237|4792270|4792237|4792270|-\n",
      "ggatcggtcaactgatccttaactgatcggcat-\n",
      ">G000299455|2940021|2958111|2958078|2958111|-\n",
      "ggatcggtcaactaatccttaactgatcggcat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2187913, 2198767)), (2194763, 2194936), '-'), (('G000299455', (1855036, 1876538)), (1867211, 1867384), '+'), (('G000299455', (4430274, 4430447)), (4430274, 4430447), '+')]\n",
      ">G000006925|2187913|2198767|2194763|2194936|-\n",
      "tgtgtccttaagtacgttatggcgagattactgggggcgtagcggcagcagtaaggatta\n",
      "tcagttgagttattccaataactggcggcggataagctataccctcgcggcaagccaggc\n",
      "gtatggcgagaatcatcatgaagagaaacgttttaatatttttatatcaattc-\n",
      ">G000299455|1855036|1876538|1867211|1867384|+\n",
      "-gtgtccttaagtacgttatggcgagattactgggggcgtagcggcagcagtaaggatta\n",
      "tcagttgagttattccaataactggcggcggataagctatatcctcgcggcaagccaggc\n",
      "gtatgacgagaatcatcatgaagagaaacgttttaatatttttatatcgattcc\n",
      ">G000299455|4430274|4430447|4430274|4430447|+\n",
      "-gtatcattaagtgcgctttggcggaattactggaggcgaagtggaaatgctaaagatta\n",
      "ccaactcagttactccaataactggcaacacattagttatactttctctgccagccaatc\n",
      "ttatgatgaaaataataaagaagaggagcgttttaatctgtttatctccattcc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (487251, 491968)), (488701, 488828), '-'), (('G000299455', (315576, 316388)), (315742, 315869), '-'), (('G000299455', (332631, 332758)), (332631, 332758), '+'), (('G000299455', (3819953, 3834467)), (3832890, 3833017), '+')]\n",
      ">G000006925|487251|491968|488701|488828|-\n",
      "cttatcgcctcgcaggtatgggacatgaatttcgacagcgacactaacgccattgatgtg\n",
      "gcggtgaagctgctacgcggcaaaatcgacaacgactttgagccgaagctgattcaaacc\n",
      "gtgcgcg-\n",
      ">G000299455|315576|316388|315742|315869|-\n",
      "ctgattgcctctcaggtctgggacatgaattttgacagcgacactaatgcgatcgatgtc\n",
      "gcagtaaagcgactccgcgctaaaatagacaacgattacgagacaaagctgatccagaca\n",
      "gtccggg-\n",
      ">G000299455|332631|332758|332631|332758|+\n",
      "-ttatctcgtccctggtctggaacatgaattttgacagtgatacgaatgtgattgatgtc\n",
      "gccgtgagacgtctgagaagtaaaattgatgatgactttgagccaaaactgatccatacc\n",
      "gttcgcgg\n",
      ">G000299455|3819953|3834467|3832890|3833017|+\n",
      "-ttatcgcctcgcaggtgtgggacatgaatttcgacagcgacactaacgccattgatgta\n",
      "gcggtgaagcggctgcgcggcaaaatcgacaacgactttgagccaaagctaattcagacc\n",
      "gtgcgcgg\n",
      "\n",
      "\n",
      "28 [(('G000006925', (251358, 252232)), (251778, 251899), '+'), (('G000006925', (315068, 315858)), (315402, 315522), '-'), (('G000006925', (332059, 337870)), (332393, 332513), '-'), (('G000006925', (493019, 493813)), (493353, 493473), '-'), (('G000006925', (509301, 510374)), (509635, 509755), '-'), (('G000006925', (724190, 725876)), (724524, 724644), '-'), (('G000006925', (1094884, 1095679)), (1095218, 1095338), '-'), (('G000006925', (1099363, 1100153)), (1099699, 1099819), '+'), (('G000006925', (1411510, 1412350)), (1411847, 1411968), '-'), (('G000006925', (1883764, 1885448)), (1884098, 1884218), '-'), (('G000006925', (1956633, 1961076)), (1956967, 1957087), '-'), (('G000006925', (2084425, 2085215)), (2084761, 2084881), '+'), (('G000006925', (2236395, 2237421)), (2236967, 2237087), '+'), (('G000006925', (3065287, 3076770)), (3075097, 3075217), '-'), (('G000006925', (3812816, 3813607)), (3813150, 3813270), '-'), (('G000006925', (4639759, 4640549)), (4640093, 4640213), '-'), (('G000006925', (4650084, 4650878)), (4650424, 4650544), '+'), (('G000006925', (4656735, 4658205)), (4657751, 4657871), '+'), (('G000006925', (4673259, 4674053)), (4673599, 4673719), '+'), (('G000006925', (4679888, 4680760)), (4680222, 4680342), '-'), (('G000006925', (4750261, 4751054)), (4750600, 4750720), '+'), (('G000006925', (4753880, 4754990)), (4754535, 4754655), '-'), (('G000006925', (4760730, 4761525)), (4761071, 4761191), '+'), (('G000006925', (4793569, 4794365)), (4793911, 4794031), '+'), (('G000299455', (906289, 909876)), (906631, 906751), '+'), (('G000299455', (4752701, 4766669)), (4753043, 4753163), '+'), (('G000299455', (4771847, 4785261)), (4780045, 4780166), '-'), (('G000299455', (5275700, 5279438)), (5278826, 5278946), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|251358|252232|251778|251899|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "cg---------------------------------------------\n",
      ">G000006925|315068|315858|315402|315522|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|332059|337870|332393|332513|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|493019|493813|493353|493473|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|509301|510374|509635|509755|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|724190|725876|724524|724644|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|1094884|1095679|1095218|1095338|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|1099363|1100153|1099699|1099819|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|1411510|1412350|1411847|1411968|-\n",
      "----------------------------------------------gccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "cgtattaccactgtcagcaacagcgacatcatccggataaacacagt\n",
      ">G000006925|1883764|1885448|1884098|1884218|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|1956633|1961076|1956967|1957087|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|2084425|2085215|2084761|2084881|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|2236395|2237421|2236967|2237087|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|3065287|3076770|3075097|3075217|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|3812816|3813607|3813150|3813270|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|4639759|4640549|4640093|4640213|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|4650084|4650878|4650424|4650544|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacgggggcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|4656735|4658205|4657751|4657871|+\n",
      "-cttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|4673259|4674053|4673599|4673719|+\n",
      "-cttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|4679888|4680760|4680222|4680342|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|4750261|4751054|4750600|4750720|+\n",
      "-cttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|4753880|4754990|4754535|4754655|-\n",
      "gcttattttgcaaaggcggagttcgaccgcctctggaaaaaataatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatattgccccatcaa\n",
      "-----------------------------------------------\n",
      ">G000006925|4760730|4761525|4761071|4761191|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000006925|4793569|4794365|4793911|4794031|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcggaccgctatgcagcgaactgcatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000299455|906289|909876|906631|906751|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaataatgcc-ctgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatatcgccccgtcaa\n",
      "cg---------------------------------------------\n",
      ">G000299455|4752701|4766669|4753043|4753163|+\n",
      "-cttattttgcgaaggcggagttcgaccgcctctggaaaaaataatgcc-ctgctggata\n",
      "agctgcgtgagcagtacggggtcggaccggtatgcagtgaactgcatatcgccccgtcaa\n",
      "cg---------------------------------------------\n",
      ">G000299455|4771847|4785261|4780045|4780166|-\n",
      "gcttattttgcgaaggcggagttcgaccgcctctggaaaaagtgatgccactgctggata\n",
      "agctgcgtaagctgtacggggtcggaccggtatgcagtgaactacatattgccccgtcaa\n",
      "c----------------------------------------------\n",
      ">G000299455|5275700|5279438|5278826|5278946|-\n",
      "gcttattttgcgaaggaggagttcgaccgcctctggaaaaaatgatgccactgctggata\n",
      "agctgcgtgagcagtacggggtcagaccggtatgcagcgaactgcatattgccccgtcaa\n",
      "-----------------------------------------------\n",
      "\n",
      "\n",
      "5 [(('G000006925', (747558, 747703)), (747668, 747703), '+'), (('G000006925', (1404983, 1405128)), (1405091, 1405128), '+'), (('G000006925', (1919884, 1920029)), (1919884, 1919920), '-'), (('G000006925', (2687909, 2688054)), (2687909, 2687944), '-'), (('G000299455', (2892582, 2892727)), (2892582, 2892618), '-')]\n",
      ">G000006925|747558|747703|747668|747703|+\n",
      "--gagatgaaagaggagctgagcagtctggatgattt\n",
      ">G000006925|1404983|1405128|1405091|1405128|+\n",
      "gggagctgaaggaagagctgaatagtctgaaggattt\n",
      ">G000006925|1919884|1920029|1919884|1919920|-\n",
      "gggagatgaaagaggagctgagcagtctggatgatt-\n",
      ">G000006925|2687909|2688054|2687909|2687944|-\n",
      "-ggagatgaaagaggagctgagcagtctggatgatt-\n",
      ">G000299455|2892582|2892727|2892582|2892618|-\n",
      "gagagcagaaggaagaactgaataacctgaatgatt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2449773, 2449808), '-'), (('G000299455', (1623109, 1682614)), (1630668, 1630703), '+'), (('G000299455', (3362286, 3384700)), (3367291, 3367326), '+')]\n",
      ">G000006925|2374481|2472008|2449773|2449808|-\n",
      "ctgatcggacttgttcggcgtacaagtgtacgcta-\n",
      ">G000299455|1623109|1682614|1630668|1630703|+\n",
      "-tgatcggacttgttcggcgtacaagtgtacgctat\n",
      ">G000299455|3362286|3384700|3367291|3367326|+\n",
      "-tgatcggacttgttcagcgtacacgtgttagctat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (728960, 729116), '+'), (('G000299455', (2174641, 2181396)), (2177357, 2177513), '+'), (('G000299455', (2910238, 2916987)), (2914115, 2914271), '-'), (('G000299455', (3601597, 3601753)), (3601597, 3601753), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|728960|729116|+\n",
      "-ttttgagccggtggaggacgggcagacccgtggggctaatcagttttacagcgtaatgg\n",
      "agcggatgaagatgctggactccctgcaggccacgcagcttcagtcggccattgtgaagg\n",
      "cgatgtatgcagccacaatcgaaagtgaacttgatac\n",
      ">G000299455|2174641|2181396|2177357|2177513|+\n",
      "-tttcgagccggtggaggacgggcagacccgtggagccaatcagttttacagcgttatgg\n",
      "aacggctgaagatgctggattccctgcaggcaacacagcttcagtcggccatagtgaagg\n",
      "cgatgtatgcagcgacgattgaaagtgaccttgatac\n",
      ">G000299455|2910238|2916987|2914115|2914271|-\n",
      "ttttcgagccggtggaggacgggcagacccgtggagccaatcagttttacagcgttatgg\n",
      "aacggctgaagatgctcgattccctgcaggcaacacagcttcagtcggccatagtgaagg\n",
      "cgatgtatgcagcgacgattgaaagtgaccttgata-\n",
      ">G000299455|3601597|3601753|3601597|3601753|-\n",
      "tttttgaacccgtggaggacgggcagacccgcggtgcaaatgtgttttacagcgtgatgg\n",
      "agcagatgaagatgctcgacacgctgcagaacacgcagctgcagagcgccattgtgaagg\n",
      "cgatgtatgccgccaccattgagagtgagctggata-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1792740, 1793081)), (1792786, 1792808), '-'), (('G000006925', (1842508, 1869155)), (1862037, 1862059), '-'), (('G000006925', (2008567, 2033069)), (2009613, 2009635), '-'), (('G000006925', (2242494, 2346249)), (2333386, 2333408), '+'), (('G000299455', (1720965, 1752624)), (1745702, 1745724), '-'), (('G000299455', (2786550, 2804122)), (2796994, 2797016), '-'), (('G000299455', (3386242, 3399566)), (3397736, 3397758), '-'), (('G000299455', (3608677, 3610659)), (3609723, 3609745), '-'), (('G000299455', (4156918, 4166023)), (4160470, 4160492), '-')]\n",
      ">G000006925|1792740|1793081|1792786|1792808|-\n",
      "tagatgttgttggcatcatact-\n",
      ">G000006925|1842508|1869155|1862037|1862059|-\n",
      "taaatattgttagcatcgtatt-\n",
      ">G000006925|2008567|2033069|2009613|2009635|-\n",
      "tagatgttgttcgcatcatatt-\n",
      ">G000006925|2242494|2346249|2333386|2333408|+\n",
      "-agatgttgttcgcgtcgtattt\n",
      ">G000299455|1720965|1752624|1745702|1745724|-\n",
      "tagatgttgttagcatcgtatt-\n",
      ">G000299455|2786550|2804122|2796994|2797016|-\n",
      "taaatattgttagcatcgtatt-\n",
      ">G000299455|3386242|3399566|3397736|3397758|-\n",
      "tagatgttgttcgcgtcgtact-\n",
      ">G000299455|3608677|3610659|3609723|3609745|-\n",
      "tagatgttgttcgcatcatatt-\n",
      ">G000299455|4156918|4166023|4160470|4160492|-\n",
      "taaatattattggcatcgtatt-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1209065, 1211067)), (1210585, 1210821), '+'), (('G000006925', (2756889, 2766574)), (2757666, 2757902), '+'), (('G000006925', (2923039, 2950048)), (2948781, 2949017), '+'), (('G000006925', (3101814, 3103074)), (3102592, 3102828), '+'), (('G000006925', (3810378, 3811029)), (3810624, 3810860), '-'), (('G000299455', (3226712, 3229721)), (3229235, 3229471), '+')]\n",
      ">G000006925|1209065|1211067|1210585|1210821|+\n",
      "-tacgtacagatgaaggctggctgtatctggcagtggtcattgacctgtggtcacgtgcc\n",
      "gttattggctggtcaatgtcgccacgcatgacggcgcaactggcctgcgatgccctgcag\n",
      "atggcgctgtggcggcgtaagaggccccggaacgttatcgttcacacggaccgtggaggc\n",
      "cagtactgttcagcagattatcaggcgcaactgaagcggcataatctgcgtggaagt\n",
      ">G000006925|2756889|2766574|2757666|2757902|+\n",
      "-tacgtacagatgaaggctggctgtatctggcagtggtcattgacctgtggtcacgtgcc\n",
      "gttattggctggtcaatgtcgccacgcatgacggcgcaactgccctgcgatgccctgcag\n",
      "atggcgctgtggcggcgtaagaggccccggaacgttatcgttcacacggaccgtggaggc\n",
      "cagtactgttcagcagattatcaggcgcaactgaaacggcataatctgcgtggaagt\n",
      ">G000006925|2923039|2950048|2948781|2949017|+\n",
      "-tacgtacagatgaaggctggctgtatctggcagtggtcattgacctgtggtcacgtgcc\n",
      "gttattggctggtcaatgtcgccacgcatgacggcgcaactgccctgcgatgccctgcag\n",
      "atggcgctgtggcggcgtaagaggccccggaacgttatcgttcacacggaccgtggaggc\n",
      "cagtactgttcagcagattatcaggcgcaactgaagcggcataatctgcgtggaagt\n",
      ">G000006925|3101814|3103074|3102592|3102828|+\n",
      "-tacgtacagatgaaggctggctgtatctggcagtggtcattgacctgtggtcacgtgcc\n",
      "gttattggctggtcaatgtcgccacgcatgacggcgcaactgccctgcgatgccctgcag\n",
      "atggcgctgtggcggcgtaagaggccccggaacgttatcgttcacacggaccgtggaggc\n",
      "cagtactgttcagcagattatcaggcgcaactgaaacggcataatctgcgtggaagt\n",
      ">G000006925|3810378|3811029|3810624|3810860|-\n",
      "ttgcgtaccgatgagggctggttgtatctcgcagtagtcatcgacctgtggtcacgcgcc\n",
      "gtcattggctggtcgatgtcaccgcgaatgacagcacaactggcctgtgatgcactgcaa\n",
      "atggcgttgtggcggagaagacgcccggaaagcgtcattgttcatacggaccgtggtggt\n",
      "caatactgttcaggggattatcaggcaatgctgaagcgacacaacctgcgtggcag-\n",
      ">G000299455|3226712|3229721|3229235|3229471|+\n",
      "-tacgtacagatgaaggctggctgtatctggcagtggtcattgacctgtggtcacgtgcc\n",
      "gttattggctggtcaatgtcgccacgcatgacggcgcaactggcctgcgatgccctgcag\n",
      "atggcgctgtggcggcgtaagaggccccggaacgttatcgttcacacggaccgtggaggc\n",
      "cagtactgttcagcagattatcaggcgcaactgaagcggcataatctgcgtggaagt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1253314, 1275161)), (1262541, 1262649), '-'), (('G000006925', (2964578, 2976271)), (2969387, 2969496), '+'), (('G000299455', (988799, 1028140)), (1023410, 1023519), '-'), (('G000299455', (2983402, 3007859)), (2987190, 2987298), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1253314|1275161|1262541|1262649|-\n",
      "-tcccggtcggcaagtgagtaatacgaattgccgaatcggtggtgttaacgtgctgacca\n",
      "cccgcccctgacgagcggaaagtatcaatgcgtaaatctgctgggttga-\n",
      ">G000006925|2964578|2976271|2969387|2969496|+\n",
      "-tcccggtcgggatgtgggtaatacgcaccgcagattcggtacggttaacgtgctgaccg\n",
      "cccgcgccggacgcgcgataaacgtcaatgcgcagatccgccgggttgat\n",
      ">G000299455|988799|1028140|1023410|1023519|-\n",
      "atcccggttgggatgtgggtaatacgcaccgcagattcggtacggttaacgtgctgaccg\n",
      "cccgcgccggacgcgcgataaacgtcaatgcgcagatccgccgggttga-\n",
      ">G000299455|2983402|3007859|2987190|2987298|+\n",
      "--cccggtcggcaagtgagtaatacgaattgccgaatcggtggtgttaacgtgctgacca\n",
      "cccgcccctgacgagcggaaagtatcaatgcgtaaatctgctgggttgat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (64491, 216702)), (118847, 118872), '+'), (('G000006925', (4450485, 4460192)), (4457113, 4457138), '+'), (('G000299455', (4300454, 4333834)), (4329634, 4329659), '-'), (('G000299455', (4658720, 4667733)), (4665350, 4665375), '+')]\n",
      ">G000006925|64491|216702|118847|118872|+\n",
      "-gcttttggcgcgttcccctgttgtg\n",
      ">G000006925|4450485|4460192|4457113|4457138|+\n",
      "-gctttcggcgcaacaccttcctgcg\n",
      ">G000299455|4300454|4333834|4329634|4329659|-\n",
      "cgcttttggcgcattaccctgttgt-\n",
      ">G000299455|4658720|4667733|4665350|4665375|+\n",
      "-gctttcggtgctacaccttcctgcg\n",
      "\n",
      "\n",
      "7 [(('G000006925', (2045115, 2046271)), (2045340, 2045387), '+'), (('G000006925', (2227521, 2228997)), (2228066, 2228113), '+'), (('G000299455', (2367337, 2367480)), (2367433, 2367480), '+'), (('G000299455', (2892839, 2893945)), (2893029, 2893076), '+'), (('G000299455', (2893982, 2894125)), (2894078, 2894125), '+'), (('G000299455', (3581924, 3582067)), (3582020, 3582067), '+'), (('G000299455', (4122132, 4122180)), (4122132, 4122179), '+')]\n",
      ">G000006925|2045115|2046271|2045340|2045387|+\n",
      "cgatctccagatctacagcatcctgaagtggcgcaatatgctcactg\n",
      ">G000006925|2227521|2228997|2228066|2228113|+\n",
      "cgatctccagatctacagcatcctgaagtggcgcaatatgctcactg\n",
      ">G000299455|2367337|2367480|2367433|2367480|+\n",
      "caatatccaaatctacggcatcctgaagtggcgcaatatgctcactg\n",
      ">G000299455|2892839|2893945|2893029|2893076|+\n",
      "cgatttccagatctacagcatcctgcagtggcgcaatatactcactg\n",
      ">G000299455|2893982|2894125|2894078|2894125|+\n",
      "caatttccagatctgcagcatcctgaagcggcgcaatatgctcactg\n",
      ">G000299455|3581924|3582067|3582020|3582067|+\n",
      "caatttccagatctgcagcatcctgaagcggcgcaatatgctcactg\n",
      ">G000299455|4122132|4122180|4122132|4122179|+\n",
      "cgttcttcggatctacatcatcctgaagtagcgcgatatactcactg\n",
      "\n",
      "\n",
      "28 [(('G000006925', (251358, 252232)), (251902, 252084), '+'), (('G000006925', (315068, 315858)), (315217, 315399), '-'), (('G000006925', (332059, 337870)), (332208, 332390), '-'), (('G000006925', (493019, 493813)), (493168, 493350), '-'), (('G000006925', (509301, 510374)), (509450, 509632), '-'), (('G000006925', (724190, 725876)), (724339, 724521), '-'), (('G000006925', (1094884, 1095679)), (1095033, 1095215), '-'), (('G000006925', (1099363, 1100153)), (1099822, 1100004), '+'), (('G000006925', (1411510, 1412350)), (1411659, 1411841), '-'), (('G000006925', (1883764, 1885448)), (1883913, 1884095), '-'), (('G000006925', (1956633, 1961076)), (1956782, 1956964), '-'), (('G000006925', (2084425, 2085215)), (2084884, 2085066), '+'), (('G000006925', (2236395, 2237421)), (2237090, 2237272), '+'), (('G000006925', (3065287, 3076770)), (3074912, 3075094), '-'), (('G000006925', (3812816, 3813607)), (3812965, 3813147), '-'), (('G000006925', (4639759, 4640549)), (4639908, 4640090), '-'), (('G000006925', (4650084, 4650878)), (4650547, 4650729), '+'), (('G000006925', (4656735, 4658205)), (4657874, 4658056), '+'), (('G000006925', (4673259, 4674053)), (4673722, 4673904), '+'), (('G000006925', (4679888, 4680760)), (4680037, 4680219), '-'), (('G000006925', (4750261, 4751054)), (4750723, 4750905), '+'), (('G000006925', (4753880, 4754990)), (4754346, 4754532), '-'), (('G000006925', (4760730, 4761525)), (4761194, 4761376), '+'), (('G000006925', (4793569, 4794365)), (4794034, 4794216), '+'), (('G000299455', (906289, 909876)), (906754, 906940), '+'), (('G000299455', (4752701, 4766669)), (4753166, 4753352), '+'), (('G000299455', (4771847, 4785261)), (4779856, 4780042), '-'), (('G000299455', (5275700, 5279438)), (5278637, 5278823), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\n",
      "Reallocating..done. *alloclen = 1374\n",
      "\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 28\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 28\r",
      "   10 / 28\r",
      "   20 / 28\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 27  f\b\b\r",
      "STEP     2 / 27  f\b\b\r",
      "STEP     3 / 27  f\b\b\r",
      "STEP     4 / 27  f\b\b\r",
      "STEP     5 / 27  f\b\b\r",
      "STEP     6 / 27  f\b\b\r",
      "STEP     7 / 27  f\b\b\r",
      "STEP     8 / 27  f\b\b\r",
      "STEP     9 / 27  f\b\b\r",
      "STEP    10 / 27  f\b\b\r",
      "STEP    11 / 27  f\b\b\r",
      "STEP    12 / 27  f\b\b\r",
      "STEP    13 / 27  f\b\b\r",
      "STEP    14 / 27  f\b\b\r",
      "STEP    15 / 27  f\b\b\r",
      "STEP    16 / 27  f\b\b\r",
      "STEP    17 / 27  f\b\b\r",
      "STEP    18 / 27  f\b\b\r",
      "STEP    19 / 27  f\b\b\r",
      "STEP    20 / 27  f\b\b\r",
      "STEP    21 / 27  f\b\b\r",
      "STEP    22 / 27  f\b\b\r",
      "STEP    23 / 27  f\b\b\r",
      "STEP    24 / 27  f\b\b\r",
      "STEP    25 / 27  f\b\b\r",
      "STEP    26 / 27  f\b\b\n",
      "Reallocating..done. *alloclen = 1374\n",
      "\r",
      "STEP    27 / 27  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 54\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 54\r",
      "   10 / 54\r",
      "   20 / 54\r",
      "   30 / 54\r",
      "   40 / 54\r",
      "   50 / 54\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 53  f\b\b\r",
      "STEP     2 / 53  f\b\b\r",
      "STEP     3 / 53  f\b\b\r",
      "STEP     4 / 53  f\b\b\r",
      "STEP     5 / 53  f\b\b\r",
      "STEP     6 / 53  f\b\b\r",
      "STEP     7 / 53  f\b\b\r",
      "STEP     8 / 53  f\b\b\r",
      "STEP     9 / 53  f\b\b\r",
      "STEP    10 / 53  f\b\b\r",
      "STEP    11 / 53  f\b\b\r",
      "STEP    12 / 53  f\b\b\r",
      "STEP    13 / 53  f\b\b\r",
      "STEP    14 / 53  f\b\b\r",
      "STEP    15 / 53  f\b\b\r",
      "STEP    16 / 53  f\b\b\r",
      "STEP    17 / 53  f\b\b\r",
      "STEP    18 / 53  f\b\b\r",
      "STEP    19 / 53  f\b\b\r",
      "STEP    20 / 53  f\b\b\r",
      "STEP    21 / 53  f\b\b\r",
      "STEP    22 / 53  f\b\b\r",
      "STEP    23 / 53  f\b\b\r",
      "STEP    24 / 53  f\b\b\r",
      "STEP    25 / 53  f\b\b\r",
      "STEP    26 / 53  f\b\b\r",
      "STEP    27 / 53  f\b\b\r",
      "STEP    28 / 53  f\b\b\r",
      "STEP    29 / 53  f\b\b\r",
      "STEP    30 / 53  f\b\b\r",
      "STEP    31 / 53  f\b\b\r",
      "STEP    32 / 53  f\b\b\r",
      "STEP    33 / 53  f\b\b\r",
      "STEP    34 / 53  f\b\b\r",
      "STEP    35 / 53  f\b\b\r",
      "STEP    36 / 53  f\b\b\r",
      "STEP    37 / 53  f\b\b\r",
      "STEP    38 / 53  f\b\b\r",
      "STEP    39 / 53  f\b\b\r",
      "STEP    40 / 53  f\b\b\r",
      "STEP    41 / 53  f\b\b\r",
      "STEP    42 / 53  f\b\b\r",
      "STEP    43 / 53  f\b\b\r",
      "STEP    44 / 53  f\b\b\r",
      "STEP    45 / 53  f\b\b\r",
      "STEP    46 / 53  f\b\b\r",
      "STEP    47 / 53  f\b\b\r",
      "STEP    48 / 53  f\b\b\r",
      "STEP    49 / 53  f\b\b\r",
      "STEP    50 / 53  f\b\b\r",
      "STEP    51 / 53  f\b\b\r",
      "STEP    52 / 53  f\b\b\r",
      "STEP    53 / 53  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 54\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 54\r",
      "   10 / 54\r",
      "   20 / 54\r",
      "   30 / 54\r",
      "   40 / 54\r",
      "   50 / 54\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 53  f\b\b\r",
      "STEP     2 / 53  f\b\b\r",
      "STEP     3 / 53  f\b\b\r",
      "STEP     4 / 53  f\b\b\r",
      "STEP     5 / 53  f\b\b\r",
      "STEP     6 / 53  f\b\b\r",
      "STEP     7 / 53  f\b\b\r",
      "STEP     8 / 53  f\b\b\r",
      "STEP     9 / 53  f\b\b\r",
      "STEP    10 / 53  f\b\b\r",
      "STEP    11 / 53  f\b\b\r",
      "STEP    12 / 53  f\b\b\r",
      "STEP    13 / 53  f\b\b\r",
      "STEP    14 / 53  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|251358|252232|251902|252084|+\n",
      "--taccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaatgtatcagagtggccagatgcactgtgg\n",
      "cacgtctc\n",
      ">G000006925|315068|315858|315217|315399|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaatgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|332059|337870|332208|332390|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|493019|493813|493168|493350|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|509301|510374|509450|509632|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|724190|725876|724339|724521|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|1094884|1095679|1095033|1095215|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|1099363|1100153|1099822|1100004|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggcagaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|1411510|1412350|1411659|1411841|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggcagaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|1883764|1885448|1883913|1884095|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|1956633|1961076|1956782|1956964|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|2084425|2085215|2084884|2085066|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|2236395|2237421|2237090|2237272|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaatgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|3065287|3076770|3074912|3075094|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagttcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|3812816|3813607|3812965|3813147|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggcagaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|4639759|4640549|4639908|4640090|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|4650084|4650878|4650547|4650729|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|4656735|4658205|4657874|4658056|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|4673259|4674053|4673722|4673904|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|4679888|4680760|4680037|4680219|-\n",
      "attaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|4750261|4751054|4750723|4750905|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|4753880|4754990|4754346|4754532|-\n",
      "attaccactgtcagcaacagcgacatcatcatgataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatactgcgcgtatacgatgagaatcatcaggtatacgctg\n",
      "tgcgtaaagtctggcatcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      ">G000006925|4760730|4761525|4761194|4761376|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000006925|4793569|4794365|4794034|4794216|+\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggt-\n",
      "---gtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000299455|906289|909876|906754|906940|+\n",
      "--taccactgtcagcaacagcgacatcatcctgataaacgcagttcccgtgcgcagcacg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcaccaggtgtacggtg\n",
      "tgcgtaaagtctggcgccagttgttacgggaaggaatcagggtggccagatgtacagtgg\n",
      "cgcgcctc\n",
      ">G000299455|4752701|4766669|4753166|4753352|+\n",
      "--taccactgtcagcaacagcgacatcatcctgataaacgcagttcccgtgcgcagcacg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcaccaggtgtacggtg\n",
      "tgcgtaaagtctggcgccagttgttacgggaaggaatcagggtggccagatgtacagtgg\n",
      "cgcgcctc\n",
      ">G000299455|4771847|4785261|4779856|4780042|-\n",
      "-ttaccactgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcgcg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcacaaggtatacggtg\n",
      "tgcgtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtct-\n",
      ">G000299455|5275700|5279438|5278637|5278823|-\n",
      "attaccattgtcagcaacagcgacatcatccggataaacgcagtgcccgtgcgcagcacg\n",
      "atgactggctgaagaaagagatacagcgcgtatacgatgaaaatcaccaggtatacggtg\n",
      "tgcgtaaagtctggcgtcagttgttacgggaaggtatcagagtggccagatgcactgtgg\n",
      "cacgtc--\n",
      "\n",
      "\n",
      "54 [(('G000006925', (251358, 252232)), (251362, 251422), '+'), (('G000006925', (263407, 264672)), (264609, 264669), '-'), (('G000006925', (311637, 313002)), (312939, 312999), '-'), (('G000006925', (315983, 317248)), (315986, 316046), '+'), (('G000006925', (325529, 326794)), (326731, 326791), '-'), (('G000006925', (509301, 510374)), (510310, 510370), '-'), (('G000006925', (699183, 700447)), (699186, 699246), '+'), (('G000006925', (712907, 714170)), (714107, 714167), '-'), (('G000006925', (723314, 724189)), (723317, 723377), '+'), (('G000006925', (825843, 827104)), (825845, 825905), '+'), (('G000006925', (903718, 904988)), (903726, 903786), '+'), (('G000006925', (914596, 915860)), (914599, 914659), '+'), (('G000006925', (916783, 918049)), (917986, 918046), '-'), (('G000006925', (921552, 942506)), (932350, 932410), '-'), (('G000006925', (1078088, 1090857)), (1080719, 1080779), '+'), (('G000006925', (1100850, 1102113)), (1100853, 1100913), '+'), (('G000006925', (1290847, 1306886)), (1305626, 1305686), '+'), (('G000006925', (1388754, 1391565)), (1391502, 1391562), '-'), (('G000006925', (1401103, 1402373)), (1401112, 1401172), '+'), (('G000006925', (1413769, 1415032)), (1413772, 1413832), '+'), (('G000006925', (1425133, 1465118)), (1425748, 1425808), '+'), (('G000006925', (1466327, 1467592)), (1466330, 1466390), '+'), (('G000006925', (1468138, 1500238)), (1474744, 1474804), '+'), (('G000006925', (1582509, 1591522)), (1583709, 1583769), '-'), (('G000006925', (1591523, 1592784)), (1591526, 1591586), '+'), (('G000006925', (1828287, 1829620)), (1829487, 1829547), '-'), (('G000006925', (1834428, 1834902)), (1834431, 1834491), '+'), (('G000006925', (1842508, 1869155)), (1842512, 1842572), '+'), (('G000006925', (1878572, 1883763)), (1882891, 1882951), '+'), (('G000006925', (1938560, 1939822)), (1938563, 1938623), '+'), (('G000006925', (1963295, 1964561)), (1963300, 1963360), '+'), (('G000006925', (2057197, 2058461)), (2058398, 2058458), '-'), (('G000006925', (2069615, 2084300)), (2070870, 2070930), '+'), (('G000006925', (2069615, 2084300)), (2084236, 2084296), '-'), (('G000006925', (2199538, 2210687)), (2203731, 2203791), '+'), (('G000006925', (2584954, 2636558)), (2615992, 2616052), '+'), (('G000006925', (2694303, 2695567)), (2694306, 2694366), '+'), (('G000006925', (2816684, 2833035)), (2831680, 2831740), '+'), (('G000006925', (3322761, 3361739)), (3353458, 3353518), '+'), (('G000006925', (3857305, 3858568)), (3857308, 3857368), '+'), (('G000006925', (3898713, 3912668)), (3911408, 3911468), '+'), (('G000006925', (4357823, 4358015)), (4357826, 4357887), '+'), (('G000006925', (4532444, 4533709)), (4533646, 4533706), '-'), (('G000006925', (4613266, 4614529)), (4614466, 4614526), '-'), (('G000006925', (4651394, 4651981)), (4651918, 4651978), '-'), (('G000006925', (4695532, 4696775)), (4696712, 4696772), '-'), (('G000006925', (4704225, 4705397)), (4704228, 4704288), '+'), (('G000006925', (4743489, 4744586)), (4744523, 4744583), '-'), (('G000006925', (4765915, 4766031)), (4765915, 4765975), '-'), (('G000006925', (4802939, 4805150)), (4805087, 4805147), '-'), (('G000006925', (4813486, 4814755)), (4814687, 4814747), '-'), (('G000006925', (4823511, 4823837)), (4823513, 4823573), '+'), (('G000299455', (2065333, 2065756)), (2065596, 2065656), '-'), (('G000299455', (4564841, 4566095)), (4564851, 4564911), '+')]\n",
      ">G000006925|251358|252232|251362|251422|+\n",
      "--tagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "aa\n",
      ">G000006925|263407|264672|264609|264669|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|311637|313002|312939|312999|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|315983|317248|315986|316046|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|325529|326794|326731|326791|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|509301|510374|510310|510370|-\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|699183|700447|699186|699246|+\n",
      "-atagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|712907|714170|714107|714167|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|723314|724189|723317|723377|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|825843|827104|825845|825905|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|903718|904988|903726|903786|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|914596|915860|914599|914659|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|916783|918049|917986|918046|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|921552|942506|932350|932410|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|1078088|1090857|1080719|1080779|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1100850|1102113|1100853|1100913|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1290847|1306886|1305626|1305686|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1388754|1391565|1391502|1391562|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|1401103|1402373|1401112|1401172|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1413769|1415032|1413772|1413832|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1425133|1465118|1425748|1425808|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1466327|1467592|1466330|1466390|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1468138|1500238|1474744|1474804|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1582509|1591522|1583709|1583769|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|1591523|1592784|1591526|1591586|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1828287|1829620|1829487|1829547|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|1834428|1834902|1834431|1834491|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1842508|1869155|1842512|1842572|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1878572|1883763|1882891|1882951|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1938560|1939822|1938563|1938623|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|1963295|1964561|1963300|1963360|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2057197|2058461|2058398|2058458|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|2069615|2084300|2070870|2070930|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2069615|2084300|2084236|2084296|-\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2199538|2210687|2203731|2203791|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2584954|2636558|2615992|2616052|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2694303|2695567|2694306|2694366|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|2816684|2833035|2831680|2831740|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|3322761|3361739|3353458|3353518|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|3857305|3858568|3857308|3857368|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|3898713|3912668|3911408|3911468|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|4357823|4358015|4357826|4357887|+\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "a-\n",
      ">G000006925|4532444|4533709|4533646|4533706|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|4613266|4614529|4614466|4614526|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|4651394|4651981|4651918|4651978|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagccaat\n",
      "--\n",
      ">G000006925|4695532|4696775|4696712|4696772|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|4704225|4705397|4704228|4704288|+\n",
      "-gtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagccaat\n",
      "a-\n",
      ">G000006925|4743489|4744586|4744523|4744583|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagccaat\n",
      "--\n",
      ">G000006925|4765915|4766031|4765915|4765975|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagccaat\n",
      "--\n",
      ">G000006925|4802939|4805150|4805087|4805147|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|4813486|4814755|4814687|4814747|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000006925|4823511|4823837|4823513|4823573|+\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagccaat\n",
      "--\n",
      ">G000299455|2065333|2065756|2065596|2065656|-\n",
      "ggtagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "--\n",
      ">G000299455|4564841|4566095|4564851|4564911|+\n",
      "--tagcctgagtttaacggacactccttcctgaaatagaatggcatcagaaggagctaat\n",
      "aa\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3065287, 3076770)), (3066442, 3066489), '+'), (('G000006925', (4622488, 4624460)), (4623938, 4623985), '+'), (('G000006925', (4639077, 4639758)), (4639187, 4639234), '-'), (('G000299455', (1992068, 1992414)), (1992068, 1992115), '+'), (('G000299455', (4749820, 4751622)), (4750971, 4751018), '+'), (('G000299455', (5313369, 5315826)), (5315175, 5315222), '+')]\n",
      ">G000006925|3065287|3076770|3066442|3066489|+\n",
      "-tgctggagccgctgtatgacgtcctgcgccagtatgtactgatgccc\n",
      ">G000006925|4622488|4624460|4623938|4623985|+\n",
      "-tgctggaaccgctgtatgacatcctgcgccagtatgtgctgatgccc\n",
      ">G000006925|4639077|4639758|4639187|4639234|-\n",
      "ctgctggaaccgctgtatgacatcctgcgccagtatgtgctgatgcc-\n",
      ">G000299455|1992068|1992414|1992068|1992115|+\n",
      "-tgctggcaccgctggatgaagcccttcagcactatgtcctgaccgac\n",
      ">G000299455|4749820|4751622|4750971|4751018|+\n",
      "-tgctggagccgctgtatgacgtcctgcgccagtatgtactgatgccc\n",
      ">G000299455|5313369|5315826|5315175|5315222|+\n",
      "-tgctggagccgctgtatgacgtcctgcgccagtatgtgctgatgccc\n",
      "\n",
      "\n",
      "38 [(('G000006925', (236929, 237114)), (236930, 236972), '+'), (('G000006925', (265374, 265559)), (265515, 265557), '-'), (('G000006925', (494885, 495070)), (495026, 495068), '-'), (('G000006925', (706191, 706376)), (706332, 706374), '-'), (('G000006925', (864207, 864393)), (864349, 864391), '-'), (('G000006925', (901434, 901838)), (901435, 901477), '+'), (('G000006925', (911511, 911696)), (911652, 911694), '-'), (('G000006925', (966076, 980232)), (980189, 980231), '-'), (('G000006925', (1098847, 1099032)), (1098848, 1098890), '+'), (('G000006925', (1189492, 1208094)), (1208051, 1208093), '-'), (('G000006925', (1392018, 1392203)), (1392159, 1392201), '-'), (('G000006925', (1417365, 1417553)), (1417509, 1417551), '-'), (('G000006925', (1501384, 1507637)), (1501385, 1501427), '+'), (('G000006925', (1607379, 1610459)), (1610415, 1610457), '-'), (('G000006925', (1619749, 1619934)), (1619750, 1619792), '+'), (('G000006925', (1695091, 1695276)), (1695232, 1695274), '-'), (('G000006925', (1805163, 1806570)), (1806526, 1806568), '-'), (('G000006925', (2035068, 2035255)), (2035211, 2035253), '-'), (('G000006925', (2360573, 2360758)), (2360574, 2360616), '+'), (('G000006925', (2691924, 2692109)), (2691925, 2691967), '+'), (('G000006925', (2769676, 2769861)), (2769817, 2769859), '-'), (('G000006925', (2963247, 2963432)), (2963388, 2963430), '-'), (('G000006925', (3080669, 3080854)), (3080670, 3080712), '+'), (('G000006925', (3179574, 3180459)), (3179575, 3179617), '+'), (('G000006925', (3268417, 3281594)), (3281551, 3281593), '-'), (('G000006925', (3514451, 3514636)), (3514452, 3514494), '+'), (('G000006925', (3605879, 3606167)), (3605880, 3605922), '+'), (('G000006925', (3814544, 3814729)), (3814685, 3814727), '-'), (('G000006925', (3830436, 3830622)), (3830578, 3830620), '-'), (('G000006925', (3915855, 3917645)), (3915856, 3915898), '+'), (('G000006925', (3969227, 3993100)), (3993057, 3993099), '-'), (('G000006925', (4116339, 4116524)), (4116480, 4116522), '-'), (('G000006925', (4255239, 4255424)), (4255380, 4255422), '-'), (('G000006925', (4361769, 4361954)), (4361770, 4361812), '+'), (('G000006925', (4607382, 4607555)), (4607511, 4607553), '-'), (('G000006925', (4619148, 4619330)), (4619286, 4619330), '-'), (('G000006925', (4694201, 4694386)), (4694343, 4694385), '-'), (('G000299455', (5286183, 5286375)), (5286332, 5286374), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP    15 / 53  f\b\b\r",
      "STEP    16 / 53  f\b\b\r",
      "STEP    17 / 53  f\b\b\r",
      "STEP    18 / 53  f\b\b\r",
      "STEP    19 / 53  f\b\b\r",
      "STEP    20 / 53  f\b\b\r",
      "STEP    21 / 53  f\b\b\r",
      "STEP    22 / 53  f\b\b\r",
      "STEP    23 / 53  f\b\b\r",
      "STEP    24 / 53  f\b\b\r",
      "STEP    25 / 53  f\b\b\r",
      "STEP    26 / 53  f\b\b\r",
      "STEP    27 / 53  f\b\b\r",
      "STEP    28 / 53  f\b\b\r",
      "STEP    29 / 53  f\b\b\r",
      "STEP    30 / 53  f\b\b\r",
      "STEP    31 / 53  f\b\b\r",
      "STEP    32 / 53  f\b\b\r",
      "STEP    33 / 53  f\b\b\r",
      "STEP    34 / 53  f\b\b\r",
      "STEP    35 / 53  f\b\b\r",
      "STEP    36 / 53  f\b\b\r",
      "STEP    37 / 53  f\b\b\r",
      "STEP    38 / 53  f\b\b\r",
      "STEP    39 / 53  f\b\b\r",
      "STEP    40 / 53  f\b\b\r",
      "STEP    41 / 53  f\b\b\r",
      "STEP    42 / 53  f\b\b\r",
      "STEP    43 / 53  f\b\b\r",
      "STEP    44 / 53  f\b\b\r",
      "STEP    45 / 53  f\b\b\r",
      "STEP    46 / 53  f\b\b\r",
      "STEP    47 / 53  f\b\b\r",
      "STEP    48 / 53  f\b\b\r",
      "STEP    49 / 53  f\b\b\r",
      "STEP    50 / 53  f\b\b\r",
      "STEP    51 / 53  f\b\b\r",
      "STEP    52 / 53  f\b\b\r",
      "STEP    53 / 53  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  f\b\b\r",
      "STEP    35 / 37  f\b\b\r",
      "STEP    36 / 37  f\b\b\r",
      "STEP    37 / 37  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  f\b\b\r",
      "STEP    35 / 37  f\b\b\r",
      "STEP    36 / 37  f\b\b\r",
      "STEP    37 / 37  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|236929|237114|236930|236972|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|265374|265559|265515|265557|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|494885|495070|495026|495068|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|706191|706376|706332|706374|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|864207|864393|864349|864391|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|901434|901838|901435|901477|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|911511|911696|911652|911694|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|966076|980232|980189|980231|-\n",
      "atcagtatcatgcccaaaccagacgggttaacggcagcaaag--\n",
      ">G000006925|1098847|1099032|1098848|1098890|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1189492|1208094|1208051|1208093|-\n",
      "atcagtatcatgcccaaaccagacgggttaacggcagcaaag--\n",
      ">G000006925|1392018|1392203|1392159|1392201|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1417365|1417553|1417509|1417551|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1501384|1507637|1501385|1501427|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1607379|1610459|1610415|1610457|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1619749|1619934|1619750|1619792|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1695091|1695276|1695232|1695274|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|1805163|1806570|1806526|1806568|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|2035068|2035255|2035211|2035253|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|2360573|2360758|2360574|2360616|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|2691924|2692109|2691925|2691967|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|2769676|2769861|2769817|2769859|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|2963247|2963432|2963388|2963430|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3080669|3080854|3080670|3080712|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3179574|3180459|3179575|3179617|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3268417|3281594|3281551|3281593|-\n",
      "atcagtatcatgcccaaaccagacgggttaacggcagcaaag--\n",
      ">G000006925|3514451|3514636|3514452|3514494|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3605879|3606167|3605880|3605922|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3814544|3814729|3814685|3814727|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3830436|3830622|3830578|3830620|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3915855|3917645|3915856|3915898|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|3969227|3993100|3993057|3993099|-\n",
      "atcagtatcatgcccaaaccagacgggttaacggcagcaaag--\n",
      ">G000006925|4116339|4116524|4116480|4116522|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|4255239|4255424|4255380|4255422|-\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|4361769|4361954|4361770|4361812|+\n",
      "-tcagtatcatgcccaaaccagacgggttaacggcagcaaaga-\n",
      ">G000006925|4607382|4607555|4607511|4607553|-\n",
      "-tcagtatcatgcccaaacctgacgggttaacggcagcaaaga-\n",
      ">G000006925|4619148|4619330|4619286|4619330|-\n",
      "atcagtgttatgttcgaacctgacgtgttaacagcagtaaagta\n",
      ">G000006925|4694201|4694386|4694343|4694385|-\n",
      "atcagtatcatgcccaaaccagacgggttaacggcagcaaag--\n",
      ">G000299455|5286183|5286375|5286332|5286374|-\n",
      "atcagtgtcatgcccaaaccagacgggttaacggcagcaaag--\n",
      "\n",
      "\n",
      "19 [(('G000006925', (237940, 238704)), (237940, 238329), '+'), (('G000006925', (539107, 539868)), (539107, 539495), '+'), (('G000006925', (709671, 710707)), (710319, 710707), '-'), (('G000006925', (716695, 717456)), (716695, 717083), '+'), (('G000006925', (902326, 903088)), (902326, 902714), '+'), (('G000006925', (1095979, 1096741)), (1096353, 1096741), '-'), (('G000006925', (1632621, 1633383)), (1632621, 1633009), '+'), (('G000006925', (1834903, 1835664)), (1835276, 1835664), '-'), (('G000006925', (1878572, 1883763)), (1878572, 1878960), '+'), (('G000006925', (1944438, 1945202)), (1944438, 1944826), '+'), (('G000006925', (1962084, 1962845)), (1962084, 1962472), '+'), (('G000006925', (2069615, 2084300)), (2069615, 2070003), '+'), (('G000006925', (2346737, 2347498)), (2346737, 2347125), '+'), (('G000006925', (2477790, 2478551)), (2477790, 2478178), '+'), (('G000006925', (2953405, 2954171)), (2953779, 2954171), '-'), (('G000006925', (4392644, 4394283)), (4393895, 4394283), '-'), (('G000006925', (4823851, 4824717)), (4824329, 4824717), '-'), (('G000299455', (926743, 927171)), (926743, 927131), '+'), (('G000299455', (4559929, 4560763)), (4559929, 4560317), '+')]\n",
      ">G000006925|237940|238704|237940|238329|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtcaccggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggcgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|539107|539868|539107|539495|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacgatttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|709671|710707|710319|710707|-\n",
      "tataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000006925|716695|717456|716695|717083|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|902326|903088|902326|902714|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|1095979|1096741|1096353|1096741|-\n",
      "tataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000006925|1632621|1633383|1632621|1633009|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|1834903|1835664|1835276|1835664|-\n",
      "tataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000006925|1878572|1883763|1878572|1878960|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacgatttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|1944438|1945202|1944438|1944826|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|1962084|1962845|1962084|1962472|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|2069615|2084300|2069615|2070003|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|2346737|2347498|2346737|2347125|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacgatttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|2477790|2478551|2477790|2478178|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000006925|2953405|2954171|2953779|2954171|-\n",
      "tataggtcgcatcaccagcaccacacgtgatttggctcaggtcacggcgagctgccttcc\n",
      "aaggtacgttagggatagcaacatgttcatggccaccacgtttattccggtgagtagcct\n",
      "tctgtcatgctgaccagccccagctctttcatgagcttgccaccgagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000006925|4392644|4394283|4393895|4394283|-\n",
      "tataggtcacatcacc-gcaccacacctgatttggttcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000006925|4823851|4824717|4824329|4824717|-\n",
      "tataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccaccacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgat-\n",
      ">G000299455|926743|927171|926743|927131|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggctcggtca-cggcgaactgcctttc\n",
      "aaggta-gttagggatagcaacatgttcatgaccactacgtttataccggtgagtcggct\n",
      "gctgaca-gctgaccagccccagctctttcatgagcctgccagcaagccagcgtcccatc\n",
      "tggtagcctctccgggttgccattgtggcgatgcttcttgctccggccgaaccgtggctg\n",
      "atgccatgtagctcaagtacctgactgcgtaatacagcccgtctgcagtctggtttttca\n",
      "ggacggtttttccagtatctgtagctgctgcgatgaaccccgaacacatggcagagtgtg\n",
      "accacaggataatgcgctctgagtttcccgatt\n",
      ">G000299455|4559929|4560763|4559929|4560317|+\n",
      "-ataggtcacatcacc-gcaccacacctgatttggttccgtta-cggcgaactgtcgctc\n",
      "aagatg-attcgggatagcaacgtgctcatgaccgccacgcttataccggtgagtcggct\n",
      "gctggca-actgaccagccccagctctttcatgagtctgccagcaagccagcgccccatc\n",
      "tggtagcctctctgggttgccattgtggcgatgcttcttgctccggcagagccgtggctg\n",
      "atgccatgcagttcaagtacctggctgcgtaatacagcccgtctgccgtctggtttttca\n",
      "ggacggtttttccagtatttgtagctgctgcgatggaccccgaacacatggcagagagtg\n",
      "gccacaggataacgcgccctgagtttcccgatt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (720060, 720643)), (720148, 720283), '+'), (('G000299455', (2546059, 2546618)), (2546147, 2546276), '+'), (('G000299455', (3293954, 3294425)), (3294296, 3294425), '-')]\n",
      ">G000006925|720060|720643|720148|720283|+\n",
      "-accgaagtttcacgaatgctgaactgtaatcgcgccacggtaagaaaatatgctgaaga\n",
      "taaagagggcaaagggcacgccattgttgatggtgttctgattgttcatcgtggatggga\n",
      "caggggtaaagacagt\n",
      ">G000299455|2546059|2546618|2546147|2546276|+\n",
      "-acagaagtcgccagaatactgaactgcaatcgtgctacagtcagaaaatacattgatga\n",
      "taaagatgcgaaaaagcacgccgtcgtcaatggcgtccttatggttcatcgcggatgggg\n",
      "taaagatact------\n",
      ">G000299455|3293954|3294425|3294296|3294425|-\n",
      "gaccgaagtggcccgaatactgaactgcaatcgtgccacagtcagaaaatacattggcga\n",
      "taaagaagggaaaagacacgctatcgtcaacggtgttcttatggttcaccgcggatgggg\n",
      "taaagatac-------\n",
      "\n",
      "\n",
      "4 [(('G000006925', (789803, 808424)), (799510, 799547), '-'), (('G000006925', (2069615, 2084300)), (2080378, 2080415), '-'), (('G000299455', (2006434, 2017864)), (2009056, 2009093), '+'), (('G000299455', (3528972, 3554217)), (3531127, 3531164), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1076\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|789803|808424|799510|799547|-\n",
      "cattccgcagcagctgatcctgccggataccgttcat-\n",
      ">G000006925|2069615|2084300|2080378|2080415|-\n",
      "cattccgcagcaactgattttgcccgccactgttcgt-\n",
      ">G000299455|2006434|2017864|2009056|2009093|+\n",
      "-attccgcagcaactgattttgcccgccactgttcgta\n",
      ">G000299455|3528972|3554217|3531127|3531164|+\n",
      "-attccgcagcagctgatcctgccggataccgttcatg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3065287, 3076770)), (3076671, 3076770), '+'), (('G000299455', (1996963, 1997507)), (1996963, 1997062), '-'), (('G000299455', (4541696, 4542240)), (4541696, 4541795), '-')]\n",
      ">G000006925|3065287|3076770|3076671|3076770|+\n",
      "-cagttaaaatctggactggagtttgggtaaagcaggctggttgctgttacagcactgtg\n",
      "gttgtcacacaatggcccactgacgtaaaaaacagtcagt\n",
      ">G000299455|1996963|1997507|1996963|1997062|-\n",
      "tcagttaaaatctggactggagtttgggtaaagcaggctggttgctgttacagcactgtg\n",
      "gttgtcacacaatggaacactgacgtaaaaaacagtcag-\n",
      ">G000299455|4541696|4542240|4541696|4541795|-\n",
      "tcagttaaaatctggactggagtttgggtaaagcaggctggttgctgttacagcactgtg\n",
      "gttgtcacacaatggaacactgacgtaaaaaacagtcag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1792740, 1793081)), (1792832, 1792865), '+'), (('G000006925', (2008567, 2033069)), (2009659, 2009692), '+'), (('G000299455', (3608677, 3610659)), (3609769, 3609802), '+')]\n",
      ">G000006925|1792740|1793081|1792832|1792865|+\n",
      "attaaacatgaccacctatggggatggttatat\n",
      ">G000006925|2008567|2033069|2009659|2009692|+\n",
      "ccagaacatgactgtttttggtaataaccatat\n",
      ">G000299455|3608677|3610659|3609769|3609802|+\n",
      "ccagaacatgactgtttttggtaataaccatat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (905848, 907049)), (905916, 905942), '+'), (('G000006925', (1397676, 1398098)), (1397996, 1398021), '-'), (('G000006925', (1581377, 1581641)), (1581459, 1581484), '+'), (('G000006925', (2062783, 2064325)), (2063969, 2063993), '+'), (('G000299455', (2541782, 2542216)), (2542114, 2542139), '-'), (('G000299455', (2927401, 2931337)), (2930196, 2930220), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|905848|907049|905916|905942|+\n",
      "--cacccggtcactcttacttcctggat\n",
      ">G000006925|1397676|1398098|1397996|1398021|-\n",
      "cccgcctggt--ctcttactcctcagg-\n",
      ">G000006925|1581377|1581641|1581459|1581484|+\n",
      "-tcgccgggt--ctcttactcctcaggt\n",
      ">G000006925|2062783|2064325|2063969|2063993|+\n",
      "--cgccgggt--ctcttactcctcaggt\n",
      ">G000299455|2541782|2542216|2542114|2542139|-\n",
      "ctcgccgggt--ctcttactcctcagg-\n",
      ">G000299455|2927401|2931337|2930196|2930220|+\n",
      "--cgccaggt--ctcttactcctcaggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3369261, 3369365), '-'), (('G000299455', (540757, 615358)), (571782, 571886), '+'), (('G000299455', (4235244, 4293324)), (4268868, 4268972), '+')]\n",
      ">G000006925|3361966|3403725|3369261|3369365|-\n",
      "ccatattactggggatggcaaatccaatcccgacgctcccgccgccaggcgcaaggattg\n",
      "cagtgttgatgccaattaactcaccgttaaggtttaatagtgca-\n",
      ">G000299455|540757|615358|571782|571886|+\n",
      "-catattactggggatggcaaatccaatcccgacgctcccgccgccaggcgcaaggattg\n",
      "cagtgttgatgccaattaactcaccgttaaggtttaatagtgcac\n",
      ">G000299455|4235244|4293324|4268868|4268972|+\n",
      "-catgttgctcgggatagcaaaaccgataccgatgttgccgccgtccggtgcgaggatcg\n",
      "cggtgttgataccgatcagttcgccgttcaggttaaccagcgcac\n",
      "\n",
      "\n",
      "16 [(('G000006925', (2086041, 2087310)), (2086059, 2086082), '+'), (('G000006925', (4658206, 4659163)), (4659118, 4659141), '-'), (('G000006925', (4678784, 4679315)), (4678807, 4678830), '+'), (('G000299455', (59638, 59682)), (59659, 59682), '+'), (('G000299455', (914822, 915038)), (914844, 914867), '+'), (('G000299455', (916939, 917533)), (916960, 916983), '+'), (('G000299455', (1976120, 1977257)), (1977211, 1977234), '-'), (('G000299455', (2004416, 2004799)), (2004434, 2004457), '+'), (('G000299455', (4572747, 4573338)), (4573293, 4573316), '-'), (('G000299455', (4748325, 4748919)), (4748346, 4748369), '+'), (('G000299455', (4767259, 4767478)), (4767281, 4767304), '+'), (('G000299455', (4769783, 4769911)), (4769804, 4769827), '+'), (('G000299455', (5304330, 5304511)), (5304351, 5304374), '+'), (('G000299455', (5315857, 5316391)), (5316083, 5316106), '+'), (('G000299455', (5333125, 5333340)), (5333146, 5333169), '+'), (('G000299455', (5337135, 5337200)), (5337156, 5337179), '-')]\n",
      ">G000006925|2086041|2087310|2086059|2086082|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000006925|4658206|4659163|4659118|4659141|-\n",
      "tattgacgcttat-ttatcggtga-\n",
      ">G000006925|4678784|4679315|4678807|4678830|+\n",
      "-attgacgattatcctgatggtag-\n",
      ">G000299455|59638|59682|59659|59682|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000299455|914822|915038|914844|914867|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000299455|916939|917533|916960|916983|+\n",
      "-attgacgcttat-ttattagtgag\n",
      ">G000299455|1976120|1977257|1977211|1977234|-\n",
      "tattgacgcttat-ttattggtga-\n",
      ">G000299455|2004416|2004799|2004434|2004457|+\n",
      "-attgatgcttat-ttattggtgag\n",
      ">G000299455|4572747|4573338|4573293|4573316|-\n",
      "tattgacgcttat-ttattggtga-\n",
      ">G000299455|4748325|4748919|4748346|4748369|+\n",
      "-attgacgcttat-ttattagtgag\n",
      ">G000299455|4767259|4767478|4767281|4767304|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000299455|4769783|4769911|4769804|4769827|+\n",
      "-attgacgcttat-ttattagtgag\n",
      ">G000299455|5304330|5304511|5304351|5304374|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000299455|5315857|5316391|5316083|5316106|+\n",
      "-attgacgcttat-ttattagtgag\n",
      ">G000299455|5333125|5333340|5333146|5333169|+\n",
      "-attgacgcttat-ttattggtgag\n",
      ">G000299455|5337135|5337200|5337156|5337179|-\n",
      "tattgacgcttat-ttattagtga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2638009, 2638701)), (2638509, 2638540), '-'), (('G000006925', (2638833, 2641524)), (2638833, 2638865), '-'), (('G000299455', (1402725, 1405914)), (1405384, 1405415), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 16\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 16\r",
      "   10 / 16\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 15  f\b\b\r",
      "STEP     2 / 15  f\b\b\r",
      "STEP     3 / 15  f\b\b\r",
      "STEP     4 / 15  f\b\b\r",
      "STEP     5 / 15  f\b\b\r",
      "STEP     6 / 15  f\b\b\r",
      "STEP     7 / 15  f\b\b\r",
      "STEP     8 / 15  f\b\b\r",
      "STEP     9 / 15  f\b\b\r",
      "STEP    10 / 15  f\b\b\r",
      "STEP    11 / 15  f\b\b\n",
      "Reallocating..done. *alloclen = 1048\n",
      "\r",
      "STEP    12 / 15  f\b\b\r",
      "STEP    13 / 15  f\b\b\r",
      "STEP    14 / 15  f\b\b\r",
      "STEP    15 / 15  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 16\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 16\r",
      "   10 / 16\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 15  f\b\b\r",
      "STEP     2 / 15  f\b\b\r",
      "STEP     3 / 15  f\b\b\r",
      "STEP     4 / 15  f\b\b\r",
      "STEP     5 / 15  f\b\b\r",
      "STEP     6 / 15  f\b\b\r",
      "STEP     7 / 15  f\b\b\r",
      "STEP     8 / 15  f\b\b\r",
      "STEP     9 / 15  f\b\b\r",
      "STEP    10 / 15  f\b\b\r",
      "STEP    11 / 15  f\b\b\r",
      "STEP    12 / 15  f\b\b\r",
      "STEP    13 / 15  f\b\b\r",
      "STEP    14 / 15  f\b\b\r",
      "STEP    15 / 15  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2638009|2638701|2638509|2638540|-\n",
      "gcaataaaaaaacgcc-aaattcggcggtttt--\n",
      ">G000006925|2638833|2641524|2638833|2638865|-\n",
      "-caacaaataaccgcctgattttggcggttttt-\n",
      ">G000299455|1402725|1405914|1405384|1405415|+\n",
      "--aataaaaaaccgcc-aaattcggcggtttttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2045115, 2046271)), (2046220, 2046250), '+'), (('G000006925', (2227521, 2228997)), (2228946, 2228976), '+'), (('G000299455', (2892839, 2893945)), (2893909, 2893939), '+')]\n",
      ">G000006925|2045115|2046271|2046220|2046250|+\n",
      "caaccgcagcacgttcttgcttaagacgtg\n",
      ">G000006925|2227521|2228997|2228946|2228976|+\n",
      "caaccgcagcacgttcttgcttaagacgtg\n",
      ">G000299455|2892839|2893945|2893909|2893939|+\n",
      "caaccgcagcacgttcttgcttaagacgtg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1209065, 1211067)), (1210044, 1210158), '+'), (('G000006925', (2756889, 2766574)), (2757125, 2757239), '+'), (('G000006925', (2923039, 2950048)), (2948240, 2948354), '+'), (('G000006925', (3101814, 3103074)), (3102051, 3102165), '+'), (('G000006925', (3811174, 3811622)), (3811276, 3811390), '-'), (('G000299455', (3226712, 3229721)), (3228694, 3228808), '+')]\n",
      ">G000006925|1209065|1211067|1210044|1210158|+\n",
      "-gacgtcttctgaacgtgaactggagatgtctaccgagattgcacgtctcaaacgccagc\n",
      "tggcagaacgggatgaagagctggctatcctccaaaaggccgcgacatacttcgc\n",
      ">G000006925|2756889|2766574|2757125|2757239|+\n",
      "-gacgtcttctgaacgtgaactggagatgtctaccgagattgcacgtctcaaacgccagc\n",
      "tggcagaacgggatgaagagctggctatcctccaaaaggccgcgacatacttcgc\n",
      ">G000006925|2923039|2950048|2948240|2948354|+\n",
      "-gacgtcttctgaacgtgaactggagatgtctaccgagattgcacgtctcaaacgccagc\n",
      "tggcagaacgggatgaagagctggctatcctccaaaaggccgcgacatacttcgc\n",
      ">G000006925|3101814|3103074|3102051|3102165|+\n",
      "-gacgtcttctgaacgtgaactggagatgtctaccgagattgcacgtctcaaacgccagc\n",
      "tggcagaacgggatgaagagctggctatcctccaaaaggccgcgacatacttcgc\n",
      ">G000006925|3811174|3811622|3811276|3811390|-\n",
      "tgagttcgtcagagcgcgaaagcgaactggccgctgaaaatgtccgccttaaacgacaac\n",
      "tggcggagcaggctgaggaactgtccatcctccaaaaggccgcgacatacttcg-\n",
      ">G000299455|3226712|3229721|3228694|3228808|+\n",
      "-gacgtcttctgaacgtgaactggagatgtctaccgagattgcacgtctcaaacgccagc\n",
      "tggcagaacgggatgaagagctggctatcctccaaaaggccgcgacatacttcgc\n",
      "\n",
      "\n",
      "38 [(('G000006925', (236929, 237114)), (237042, 237065), '+'), (('G000006925', (265374, 265559)), (265423, 265446), '-'), (('G000006925', (494885, 495070)), (494934, 494957), '-'), (('G000006925', (706191, 706376)), (706240, 706263), '-'), (('G000006925', (864207, 864393)), (864257, 864280), '-'), (('G000006925', (901434, 901838)), (901546, 901569), '+'), (('G000006925', (911511, 911696)), (911560, 911583), '-'), (('G000006925', (966076, 980232)), (980096, 980119), '-'), (('G000006925', (1098847, 1099032)), (1098959, 1098982), '+'), (('G000006925', (1189492, 1208094)), (1207958, 1207981), '-'), (('G000006925', (1392018, 1392203)), (1392067, 1392090), '-'), (('G000006925', (1417365, 1417553)), (1417417, 1417440), '-'), (('G000006925', (1501384, 1507637)), (1501497, 1501520), '+'), (('G000006925', (1607379, 1610459)), (1610323, 1610346), '-'), (('G000006925', (1619749, 1619934)), (1619861, 1619884), '+'), (('G000006925', (1695091, 1695276)), (1695140, 1695163), '-'), (('G000006925', (1805163, 1806570)), (1806434, 1806457), '-'), (('G000006925', (2035068, 2035255)), (2035119, 2035142), '-'), (('G000006925', (2360573, 2360758)), (2360685, 2360708), '+'), (('G000006925', (2691924, 2692109)), (2692036, 2692059), '+'), (('G000006925', (2769676, 2769861)), (2769725, 2769748), '-'), (('G000006925', (2963247, 2963432)), (2963296, 2963319), '-'), (('G000006925', (3080669, 3080854)), (3080781, 3080804), '+'), (('G000006925', (3268417, 3281594)), (3281458, 3281481), '-'), (('G000006925', (3514451, 3514636)), (3514563, 3514586), '+'), (('G000006925', (3605879, 3606167)), (3605992, 3606015), '+'), (('G000006925', (3814544, 3814729)), (3814593, 3814616), '-'), (('G000006925', (3830436, 3830622)), (3830486, 3830509), '-'), (('G000006925', (3915855, 3917645)), (3915968, 3915991), '+'), (('G000006925', (3969227, 3993100)), (3992964, 3992987), '-'), (('G000006925', (4116339, 4116524)), (4116388, 4116411), '-'), (('G000006925', (4255239, 4255424)), (4255288, 4255311), '-'), (('G000006925', (4361769, 4361954)), (4361881, 4361904), '+'), (('G000006925', (4607382, 4607555)), (4607419, 4607442), '-'), (('G000006925', (4619148, 4619330)), (4619194, 4619217), '-'), (('G000006925', (4694201, 4694386)), (4694250, 4694273), '-'), (('G000299455', (5286183, 5286375)), (5286239, 5286262), '-'), (('G000299455', (5337431, 5337481)), (5337440, 5337463), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  d\b\b\r",
      "STEP    27 / 37  d\b\b h\r",
      "STEP    28 / 37  d\b\b h\r",
      "STEP    29 / 37  d\b\b h\r",
      "STEP    30 / 37  d\b\b h\r",
      "STEP    31 / 37  d\b\b h\r",
      "STEP    32 / 37  d\b\b h\r",
      "STEP    33 / 37  d\b\b h\r",
      "STEP    34 / 37  f\b\b\n",
      "Reallocating..done. *alloclen = 1048\n",
      "\r",
      "STEP    35 / 37  d\b\b h\r",
      "STEP    36 / 37  d\b\b h\r",
      "STEP    37 / 37  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  d\b\b\r",
      "STEP    27 / 37  d\b\b h\r",
      "STEP    28 / 37  d\b\b h\r",
      "STEP    29 / 37  d\b\b h\r",
      "STEP    30 / 37  d\b\b h\r",
      "STEP    31 / 37  d\b\b h\r",
      "STEP    32 / 37  d\b\b h\r",
      "STEP    33 / 37  d\b\b h\r",
      "STEP    34 / 37  d\b\b h\r",
      "STEP    35 / 37  f\b\b\r",
      "STEP    36 / 37  d\b\b h\r",
      "STEP    37 / 37  d\b\b h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|236929|237114|237042|237065|+\n",
      "-ggaatatctgcggcagcgggctt\n",
      ">G000006925|265374|265559|265423|265446|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|494885|495070|494934|494957|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|706191|706376|706240|706263|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|864207|864393|864257|864280|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|901434|901838|901546|901569|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|911511|911696|911560|911583|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|966076|980232|980096|980119|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1098847|1099032|1098959|1098982|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1189492|1208094|1207958|1207981|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1392018|1392203|1392067|1392090|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1417365|1417553|1417417|1417440|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1501384|1507637|1501497|1501520|+\n",
      "-ggaatatctgcggcagcgggctt\n",
      ">G000006925|1607379|1610459|1610323|1610346|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1619749|1619934|1619861|1619884|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1695091|1695276|1695140|1695163|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|1805163|1806570|1806434|1806457|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|2035068|2035255|2035119|2035142|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|2360573|2360758|2360685|2360708|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|2691924|2692109|2692036|2692059|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|2769676|2769861|2769725|2769748|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|2963247|2963432|2963296|2963319|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3080669|3080854|3080781|3080804|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3268417|3281594|3281458|3281481|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3514451|3514636|3514563|3514586|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3605879|3606167|3605992|3606015|+\n",
      "-ggaatatctgcggcagcgggctt\n",
      ">G000006925|3814544|3814729|3814593|3814616|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3830436|3830622|3830486|3830509|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|3915855|3917645|3915968|3915991|+\n",
      "-ggaatatctgcggcagcgggctt\n",
      ">G000006925|3969227|3993100|3992964|3992987|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|4116339|4116524|4116388|4116411|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|4255239|4255424|4255288|4255311|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|4361769|4361954|4361881|4361904|+\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000006925|4607382|4607555|4607419|4607442|-\n",
      "gggaatatctgcggcagcgggcc-\n",
      ">G000006925|4619148|4619330|4619194|4619217|-\n",
      "ggaaatatctgaagcgacgtacc-\n",
      ">G000006925|4694201|4694386|4694250|4694273|-\n",
      "gggaatatctgcggcagcgggct-\n",
      ">G000299455|5286183|5286375|5286239|5286262|-\n",
      "gggaatatctacggcagcaggcc-\n",
      ">G000299455|5337431|5337481|5337440|5337463|+\n",
      "-ggaatatctgcggcagcgggcca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3646446, 3658113)), (3654063, 3654097), '+'), (('G000299455', (250849, 262513)), (254865, 254899), '-'), (('G000299455', (3026184, 3035686)), (3034417, 3034451), '+')]\n",
      ">G000006925|3646446|3658113|3654063|3654097|+\n",
      "-gccctgcgtagcccatgccgaaatcatccagcgc\n",
      ">G000299455|250849|262513|254865|254899|-\n",
      "agccctgcgtagcccatgccgaaatcatccagcg-\n",
      ">G000299455|3026184|3035686|3034417|3034451|+\n",
      "-gccctgagtagccagtaccgaaatcatccagcgc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1509064, 1578296)), (1513891, 1513935), '+'), (('G000006925', (3423912, 3503727)), (3458925, 3458969), '-'), (('G000299455', (434155, 488092)), (478890, 478934), '+'), (('G000299455', (2312634, 2337653)), (2318803, 2318847), '+')]\n",
      ">G000006925|1509064|1578296|1513891|1513935|+\n",
      "-ttcatggtcgcacgctgtttactgtcagtgcgggtgggcagcca\n",
      ">G000006925|3423912|3503727|3458925|3458969|-\n",
      "tttcatggtcgctcgctgtttaccgtttcggtgggtgggcagcc-\n",
      ">G000299455|434155|488092|478890|478934|+\n",
      "-ttcatggtcgctcgctgtttaccgtctcggtgggcgggcagcca\n",
      ">G000299455|2312634|2337653|2318803|2318847|+\n",
      "-ttcatggtcgcacgctgtttactgtcagtgcgggtgggcagcca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1122423, 1168132)), (1146456, 1146479), '-'), (('G000299455', (3057394, 3103258)), (3079114, 3079137), '+'), (('G000299455', (3697938, 3726006)), (3720775, 3720798), '+'), (('G000299455', (4586557, 4586617)), (4586588, 4586611), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1122423|1168132|1146456|1146479|-\n",
      "gctcggtgataccggtcaggaac-\n",
      ">G000299455|3057394|3103258|3079114|3079137|+\n",
      "-ctcggtgataccggtcaggaacg\n",
      ">G000299455|3697938|3726006|3720775|3720798|+\n",
      "-ctcagtcacaccggtcaggaacg\n",
      ">G000299455|4586557|4586617|4586588|4586611|+\n",
      "-ctcagtgattcccgtcaggaaag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3180744, 3203996)), (3203362, 3203399), '+'), (('G000299455', (739467, 747132)), (740064, 740101), '-'), (('G000299455', (1188940, 1202325)), (1199542, 1199579), '+')]\n",
      ">G000006925|3180744|3203996|3203362|3203399|+\n",
      "-tctccacctacgcaacctggtggatccgtcaggcgat\n",
      ">G000299455|739467|747132|740064|740101|-\n",
      "ttctccacctacgcaacctggtggatccgtcaggcga-\n",
      ">G000299455|1188940|1202325|1199542|1199579|+\n",
      "-tctcaacatacgcaacctggtggattcgccagacgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3700798, 3700850), '+'), (('G000299455', (269622, 287922)), (276542, 276594), '+'), (('G000299455', (536624, 536888)), (536624, 536676), '+')]\n",
      ">G000006925|3693106|3702974|3700798|3700850|+\n",
      "cacggcttgagactgataaacgccaggccgttgttttgcccctggccgctga\n",
      ">G000299455|269622|287922|276542|276594|+\n",
      "cacggcttgagactgataaacgccaggccgttgttttgcccctggccgctga\n",
      ">G000299455|536624|536888|536624|536676|+\n",
      "cacggtttcagactgacgaaggccataccggcgttttgtgcctggccgctga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3264200, 3264245), '-'), (('G000299455', (671363, 678362)), (675412, 675457), '+'), (('G000299455', (3879848, 3879893)), (3879848, 3879893), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3264200|3264245|-\n",
      "------tatgaaagttggttttattggcctggggattatgggtaaaccaat\n",
      ">G000299455|671363|678362|675412|675457|+\n",
      "atacctttaaaaaataaccaccaccgcaacaagagagatgattga------\n",
      ">G000299455|3879848|3879893|3879848|3879893|-\n",
      "------tatgaaactgggatttattggcttaggcattatgggtacaccgat\n",
      "\n",
      "\n",
      "9 [(('G000006925', (385514, 385654)), (385585, 385611), '+'), (('G000006925', (900498, 900638)), (900541, 900567), '-'), (('G000006925', (3422729, 3422869)), (3422772, 3422798), '-'), (('G000006925', (3851292, 3851432)), (3851363, 3851389), '+'), (('G000006925', (4358604, 4358744)), (4358675, 4358701), '+'), (('G000006925', (4699978, 4700118)), (4700049, 4700075), '+'), (('G000006925', (4775631, 4775771)), (4775702, 4775728), '+'), (('G000006925', (4782470, 4782610)), (4782513, 4782539), '-'), (('G000299455', (4572495, 4572632)), (4572526, 4572552), '-')]\n",
      ">G000006925|385514|385654|385585|385611|+\n",
      "-acgacccgttttcgtctttccgttcc\n",
      ">G000006925|900498|900638|900541|900567|-\n",
      "gacgacccgttttcgtctttccgttc-\n",
      ">G000006925|3422729|3422869|3422772|3422798|-\n",
      "gacgacccgttttcgtctttccgttc-\n",
      ">G000006925|3851292|3851432|3851363|3851389|+\n",
      "-acgacccgttttcgtctttccgttcc\n",
      ">G000006925|4358604|4358744|4358675|4358701|+\n",
      "-acgacccgttttcgtctttccgttcc\n",
      ">G000006925|4699978|4700118|4700049|4700075|+\n",
      "-acgacccgttttcgtctttccgttcc\n",
      ">G000006925|4775631|4775771|4775702|4775728|+\n",
      "-acgacccgttttcgtctttccgttcc\n",
      ">G000006925|4782470|4782610|4782513|4782539|-\n",
      "gacgacccgttttcgtctttccgttc-\n",
      ">G000299455|4572495|4572632|4572526|4572552|-\n",
      "tacgccctgttttcgcctttttctga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (913442, 913471)), (913444, 913471), '+'), (('G000006925', (1419301, 1419391)), (1419363, 1419391), '-'), (('G000299455', (2157991, 2158083)), (2157993, 2158020), '+')]\n",
      ">G000006925|913442|913471|913444|913471|+\n",
      "-attcagattatggtggggcgtgcaggg\n",
      ">G000006925|1419301|1419391|1419363|1419391|-\n",
      "cattcaaattatggtgggatgtgcaggg\n",
      ">G000299455|2157991|2158083|2157993|2158020|+\n",
      "-atccaaattatggtggggcgtgcaggg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (733185, 733309)), (733185, 733301), '-'), (('G000006925', (1934285, 1934732)), (1934285, 1934402), '+'), (('G000299455', (3597648, 3597997)), (3597655, 3597772), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|733185|733309|733185|733301|-\n",
      "cagcatcaaacagattatcgaaaccgtccacgacatctcctgttactgcctttttgccag\n",
      "accacctgccgccatgctggcagccacctcagcagaaacacgaaacgacgttcccg-\n",
      ">G000006925|1934285|1934732|1934285|1934402|+\n",
      "caacatcaaacagattatcgaaatcagccacgctgcctcccgttattgcattctggccag\n",
      "gccacgttctgtcattccggctgccacaccggcaaagacacggaacgccattcccgg\n",
      ">G000299455|3597648|3597997|3597655|3597772|+\n",
      "cagcatcgaacaggttatcggaatcagccacagcgcctcccgttattgcattctggccag\n",
      "gccatgttctgtcatttcggctgccacaccggcagagacacgaaacgccgttcccgg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1945366, 1956632)), (1949826, 1949896), '-'), (('G000006925', (3681874, 3691741)), (3686866, 3686936), '-'), (('G000299455', (211788, 221655)), (216593, 216663), '+'), (('G000299455', (2133804, 2148224)), (2143693, 2143763), '+')]\n",
      ">G000006925|1945366|1956632|1949826|1949896|-\n",
      "gattctggaaaacagccaggttgtggtgctgtggggaatgaacccgctgaataccttaaa\n",
      "aattgcctgg-\n",
      ">G000006925|3681874|3691741|3686866|3686936|-\n",
      "ggtgctggaacatagcgatgtcgtggtgttgtggagtgctaacccactcaatacgctgaa\n",
      "aattgcgtgg-\n",
      ">G000299455|211788|221655|216593|216663|+\n",
      "-gtgctggaacatagcgatgtcgtggtgttgtggagtgctaacccactcaatacgctgaa\n",
      "aattgcgtgga\n",
      ">G000299455|2133804|2148224|2143693|2143763|+\n",
      "-attctggaaaacagccaggttgtggtgctgtggggaatgaacccgctgaataccttaaa\n",
      "aattgcctgga\n",
      "\n",
      "\n",
      "18 [(('G000006925', (2057197, 2058461)), (2058085, 2058126), '-'), (('G000006925', (2069615, 2084300)), (2071193, 2071243), '+'), (('G000006925', (2199538, 2210687)), (2204054, 2204104), '+'), (('G000006925', (2584954, 2636558)), (2616315, 2616365), '+'), (('G000006925', (2694303, 2695567)), (2694629, 2694679), '+'), (('G000006925', (2816684, 2833035)), (2832003, 2832053), '+'), (('G000006925', (3322761, 3361739)), (3353781, 3353831), '+'), (('G000006925', (3857305, 3858568)), (3857631, 3857681), '+'), (('G000006925', (3898713, 3912668)), (3911731, 3911781), '+'), (('G000006925', (4532444, 4533709)), (4533333, 4533374), '-'), (('G000006925', (4613266, 4614529)), (4614153, 4614194), '-'), (('G000006925', (4651394, 4651981)), (4651605, 4651646), '-'), (('G000006925', (4695532, 4696775)), (4696399, 4696440), '-'), (('G000006925', (4704225, 4705397)), (4704551, 4704601), '+'), (('G000006925', (4743489, 4744586)), (4744210, 4744251), '-'), (('G000006925', (4802939, 4805150)), (4804774, 4804815), '-'), (('G000006925', (4813486, 4814755)), (4814374, 4814415), '-'), (('G000299455', (4564841, 4566095)), (4565165, 4565206), '+')]\n",
      ">G000006925|2057197|2058461|2058085|2058126|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000006925|2069615|2084300|2071193|2071243|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|2199538|2210687|2204054|2204104|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|2584954|2636558|2616315|2616365|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|2694303|2695567|2694629|2694679|+\n",
      "agcgtattttgcacaggagtcactgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|2816684|2833035|2832003|2832053|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|3322761|3361739|3353781|3353831|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|3857305|3858568|3857631|3857681|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|3898713|3912668|3911731|3911781|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      ">G000006925|4532444|4533709|4533333|4533374|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000006925|4613266|4614529|4614153|4614194|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000006925|4651394|4651981|4651605|4651646|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttagtcgaacaat-\n",
      ">G000006925|4695532|4696775|4696399|4696440|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000006925|4704225|4705397|4704551|4704601|+\n",
      "agcgtattttgcacaggagtcgctgaaaaatacgcgttagtcgaacaatg\n",
      ">G000006925|4743489|4744586|4744210|4744251|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttagtcgaacaat-\n",
      ">G000006925|4802939|4805150|4804774|4804815|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000006925|4813486|4814755|4814374|4814415|-\n",
      "--------ttgcacaggagtcgctgaaaaatacgcgttaatcgaacaat-\n",
      ">G000299455|4564841|4566095|4565165|4565206|+\n",
      "---------tgcacaggagtcgctgaaaaatacgcgttaatcgaacaatg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3268417, 3281594)), (3280564, 3280687), '+'), (('G000299455', (224130, 224312)), (224189, 224312), '+'), (('G000299455', (642419, 661467)), (656096, 656219), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1142\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1142\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3268417|3281594|3280564|3280687|+\n",
      "-cggtatcgcaaacagtagtgcaatcgtgactatcaggcaaaatggttatgtgatctatc\n",
      "aaagcaacgtgccagcgggtgcctttgaaattaacgatctctacccctcttccaacagcg\n",
      "gcga\n",
      ">G000299455|224130|224312|224189|224312|+\n",
      "-cggcatcgccaaaagcaacgctgaagtgagtatcgaacagaacggctatgtcatttatc\n",
      "gcacttttgttcagccgggtgcatttgagattaacgatctctacgcaacctccggcagcg\n",
      "gcga\n",
      ">G000299455|642419|661467|656096|656219|-\n",
      "gcggtatcgcaaacagtagtgcaatcgtgactatcaggcaaaatggttatgtgatctatc\n",
      "aaagcaacgtgccagcgggtgcctttgaaattaacgatctctacccctcttccaacagcg\n",
      "gcg-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (487251, 491968)), (489043, 489071), '+'), (('G000006925', (3146034, 3178603)), (3163314, 3163342), '-'), (('G000299455', (315576, 316388)), (316084, 316112), '+'), (('G000299455', (781497, 800022)), (782938, 782966), '+'), (('G000299455', (3819953, 3834467)), (3832647, 3832675), '-')]\n",
      ">G000006925|487251|491968|489043|489071|+\n",
      "-gtttcaccaggtagtcatctgcccccaa\n",
      ">G000006925|3146034|3178603|3163314|3163342|-\n",
      "ggtttacacagataatcgtcagctccca-\n",
      ">G000299455|315576|316388|316084|316112|+\n",
      "-gtttaaccagataatcgtccgcacccag\n",
      ">G000299455|781497|800022|782938|782966|+\n",
      "-gtttacacagataatcgtcagctcccag\n",
      ">G000299455|3819953|3834467|3832647|3832675|-\n",
      "ggcttcaccagatagtcatccgccccca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (487317, 487396), '-'), (('G000299455', (314355, 314434)), (314355, 314434), '-'), (('G000299455', (3819953, 3834467)), (3834322, 3834401), '+')]\n",
      ">G000006925|487251|491968|487317|487396|-\n",
      "ttgtttgaccgtttctatcgcgttgacccctcccgccagcgaaaaggtgaaggtagcggt\n",
      "attgggctggcgatagtga-\n",
      ">G000299455|314355|314434|314355|314434|-\n",
      "ttgtttgaccgtttttatcgggtggatccgtccagacaacgaaaaggagaaggcagcggc\n",
      "atcggccttgcgattgtga-\n",
      ">G000299455|3819953|3834467|3834322|3834401|+\n",
      "-tgtttgaccgtttctatcgcgttgacccttcccgccagcgaaaaggtgaaggtagcggt\n",
      "attgggctggcgatagtgaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (496417, 503320)), (496942, 500070), '-'), (('G000299455', (319761, 322889)), (319761, 322889), '-'), (('G000299455', (3819953, 3834467)), (3826097, 3829225), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 4 \n",
      "Reallocating..done. *alloclen = 1058\n",
      "STEP     4 / 4 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     4 / 4 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|496417|503320|496942|500070|-\n",
      "atgtcggtgcagccacatcagcttatacgccgccgggataataaacagcgacagcaaagg\n",
      "tgcggtgatcatgccgccaatcatcggtgcggcaatccggctcatcacctctgaaccagc\n",
      "ccccgttccccacagaatcggcagcagaccggcgataatcaccgccaccgtcatcgcttt\n",
      "cgggcgcacgcgcaggaccgcgccgtgatacaacgcctcatccagcttctgctcgctgaa\n",
      "tgtttgtggattattcaacgacggctcggcctctatggcgtgacgtaaatacatcagcat\n",
      "caccacgccaaattcggcggcgaccccggcaagggcaataaagccggtacccgtcgccac\n",
      "agaaagatgaaagcccatccaccacagtaaccagatgccgcccaccagcgcaaacggtac\n",
      "gctgctgataatcagcaacgcttcgcccacccgacggaatgccagatacaacagcacgaa\n",
      "gatgatcatcagtgtcatcggcaccatgagtttcagcttatggttggcgcgctccagtag\n",
      "ctcgaactgcccggagaatgccacgctggtgcccggttttaactgcactttttcagctat\n",
      "cgctttttgcaaatcgtgaaccaccgacaccatgtcacgatcgcgggcatcgatataaat\n",
      "ccagctcgtcgggcgcgcattctcggttttcagcatcgacggtccggtagagactttaac\n",
      "gtcggccacgtctgccagggtgatttgctgcttcatcggcgtcaggatcggcaactggcg\n",
      "cagtgcctgcggactatcgcgccagctttgcggataacgcagattaattggataacgggc\n",
      "aatcccttccaccgtttcgccaaccatcgccccgcccaccgcagaagtcacaaacaactg\n",
      "cacatccgccaccgtcataccgtaacgtgcggctttttcacggttaatctcaacgttgat\n",
      "atagcgcccaccttccagccgctcagcaagcgcagaagctacgcctggcaccgttcgcgc\n",
      "cacttcttcaatttgctcagccatcgcgtcgatatccgccagcacagtgccggaaacttt\n",
      "aatgccgatggggcttttaatgccggttgagagcatatcgatacggttacgaattggcgg\n",
      "cacccacagattcgccagccccggcagacgcacggtgttatccagttcctcaatgatttt\n",
      "gtccattgtcatgcctggccgccactgatcctgtggcttaagctggatggtcgtttctac\n",
      "catctccagcggagcagaatcggtggcggtttccgctttcccggttttgccaaatacccg\n",
      "cgccacttcaggtacgctcataattagcttgtcggttttttgcagcatactcgccgcctc\n",
      "tgctgcggaaatccccggcagcgtcgatggcatatacaacaagtcgccttcattgatctg\n",
      "cggtaaaaattccccgccaactttattgagcggccagagaaccgtcagcaccgaaagcgc\n",
      "cgccaccagcagcgtggttttcggccagtgcagtactttcagcaacagcggatgataaac\n",
      "acgaatcaaaaagcgattgagcggattactgctttccggcggaattttgccacggatcca\n",
      "gtaacccatcaggatagggatcactacgatcgccagcagcgccgcacccgccatcgcata\n",
      "cgttttggtgaacgccaacgggccaaacagacgcccttcctgcccttccagggtgaagat\n",
      "cgggataaacgacaacgtgataatcagcagactgataaacagcgccggccccacttcaac\n",
      "agacgcatcggtgatcacctgccagcgcgttttattatccagcgtggcgtcaggatgctg\n",
      "gtgctgccactcttccagccgtttatgcgcattctcgatcatgacgatagcggcatcgac\n",
      "catcgccccgacggcaatcgcaatgccacccagcgacataatattggcattcagcccctg\n",
      "gaagggcatgacaataaaagcaatacacaaccccagcggcaacgaaataatcgccaccag\n",
      "cgccgagcgcacatgccagagaaacggcgcgcagactaccgccaccacaataaactcttc\n",
      "cagcaacttgccgctgaggttgtcgatggcccggtcaatgagctggctgcggtcgtatgt\n",
      "tgtaactatctccacgccttccggcagactacttttcagcgtttccagtttgtccttcac\n",
      "ggcggcgatcacttctcgggcgtttttgccggatcgcaggatcaccaccccgcccgccac\n",
      "ttcgccttcgccgtttagttcggcaatgccccggcgcatcttcgggccaatctggacctt\n",
      "cgcaacatcgcgcagataaacgggcacgccattttcactggcttttaaaacgatgtgatt\n",
      "aaagtcgtcgagcgtttgtagatagccgctggcgcgcaccatatattctgcttccgccag\n",
      "ttcgattgacgaaccgcccgcttcctggtttgaagcatccagcgcgctttttacttcggc\n",
      "gagactgatgccatactgcgccaggcgctgaggatcgataaccacctgatactctttcac\n",
      "cacaccgcctaccgacgccacttccgcaacgtcaggaatggttttcagctcatatttgag\n",
      "aaaccagtcctgcaatgagcgtaaatcggcaagatcgtgcttaccgctgcgatccaccag\n",
      "tgcatattcatagatccagccaacacccgtggcatctggtcccagctcggcgctgactcc\n",
      "cgcaggcagcttaccctgtacctggttgaggtactccagcacccgcgaacgcgcccagta\n",
      "cggatcggtgccatcttcgaaaatgacatacacataagagtcgccaaactgcgagaaacc\n",
      "acgcacagtcttcgcgccaggcaccgacaacatggtggtggttagcggataagtcacctg\n",
      "attttcaacgatttgcggtgcctggccgggatagctggttttaataatcacctgcacatc\n",
      "ggagagatccggcaacgcatccactggcgtattaatgatggtccaggtgccccagatgct\n",
      "cagaaacaacgcgcccatcagcaccaaaaaacggttcgccaccgagcgacgaataatcca\n",
      "ttcaatca-\n",
      ">G000299455|319761|322889|319761|322889|-\n",
      "atgtctgcgcagccagattaatttgtaggcggcaggaataatgaacagggacagcagcgg\n",
      "agccgtgatcatcccaccaatcatgggtgccgcgatacggctcatgacttctgaacctgc\n",
      "accggttccccagagtattggcagcagacccgcaatgatcaccgccacggtcatggcttt\n",
      "cggccggacacgcagtacggcaccatgatagagggcttcatcaaggccttccggtgtgaa\n",
      "cgtctctttacgggacaattccgggtgcgcttcaatggcatgacgcagatacatcagcat\n",
      "gaccacgccaaactctgctgccaccccggccagagcgataaaccccgttccggttgcgac\n",
      "agacatatggaagccctgccagtacaggaaccatatcccgccaaccagggcgaacggcag\n",
      "gctcatcaggatcagcagggcttcatcaacccggcggaatgccagatacaacaggatgaa\n",
      "gatgatcatcaccgtcatcggcaccatcagcttcagtttcttgttggcatgctcaagcag\n",
      "ttcaaactgtccggagaatgacacgctggtacccggtctcagtttcactttctgactgat\n",
      "cgccgttttaatgtcattaaccaccgacaccatatccctgccgcgcgcgtcaatgtaaat\n",
      "ccagctggctggacgggcattttccgttttcagcatagtcggcccggaaacgaccttaat\n",
      "atccgcaacatcgcccagcgtgatctgctgcttcatcggggtcaggatcggcatctgttt\n",
      "cagggcctgcgggctgttccggtaatcctgcggatagcggatattaatcgggtaccgggc\n",
      "cacgccttcaaccgtttcccctaccgtcgcgccgccgatggctgatgagatgaacagctg\n",
      "cacatcgcccaccgtcattccgtagcgggaggctttttcccggttgatatccacatcaat\n",
      "gtagcgcccgccttccagtcgctctgcgagagcagacactacgccgggtacggttttggc\n",
      "gaccgcttcgatactctgcgccgttgcatcgatatcggacagaaccgttccggacacctt\n",
      "gatacctatcgggcttttgatcccggttgagagcatatcaatacggttacggattggcgg\n",
      "cacccagaggtttgccagccccggtaaacggacggtcctgtcgagttcttcaataatctt\n",
      "gtcaatcgtcatgcctggacgccactgatcctcaggtttgagctggatcgtggtttcaac\n",
      "catttcgagcggtgcggaatccgtggcggtctctgctttaccggtcttgccaaatacaga\n",
      "ggccacttcaggaacgcttttgattaacttgtccgtcgtctgcaggagcgctgcagcttc\n",
      "agccggagagacacccggcagggtcgacggcatatacaacaggtcgccctcgttaatctt\n",
      "cggcagaaactcaccgcccacctggctcagcggccagataacggtgaaaatggacaaggc\n",
      "cgcaaccagcagggttgtttttggccagtggagaacccgcaacagcaatggatgatacgc\n",
      "tttgatcagtacccggttcagggggttactggtctctgcaggaattttcccccggatcca\n",
      "gaatcccatcagaatcggaatgacgatgatggccagcgcggccgcgcccgccatggagta\n",
      "cgttttcgtgaatgccagcgggccaaacagacgtccttcctgaccttccagggtaaagat\n",
      "aggaataaaggacagggtgatgatcagcaggctgataaacagtgcgggtcccacttcaac\n",
      "ggaggcatcggtaatcaccttccagcgggtggcgttgtcaatctgctcacccggatgctg\n",
      "atgatcccactcctcaagccgcttatgcgcattctcaatcatcacaatggcggcatccac\n",
      "catcgcaccgacggcaatcgctatccctcccagcgacatgatattggcgttcagtccctg\n",
      "gaagtgcatgacgataaaggcgatacacagaccaagcggcagagagataatcgccaccag\n",
      "ggcagaacgtacgtgccacaggaacagggcacagacgatggccaccacgataaactcttc\n",
      "cagaagtttggaactgaggttatcaatcgcccggtcgatgagctggctgcgatcgtaggt\n",
      "ggtcacgatttcaacgccttccggcaggctggccttcagcgtctcaagtttatccctcac\n",
      "tgccgtgataacgtcgcgtgcatttttacccgaccgcaggatcaccacgccgccagcgac\n",
      "ttcgccctggccgttcagctcggcaataccacgccgcatttcaggcccggtctgcacgcg\n",
      "ggcaacatcccgcagataaaccggcacgccgttctcgcctgttttcaggacgatgttatt\n",
      "aaaatcatcaatgctctgaagataaccgctggcacggaccatatactcagcttcggccat\n",
      "ttcaacggatgagccaccggcctcctggttagacgattcaagggcctgtttcacttcggg\n",
      "caggctgataccatactgagacaattttaccggattgacctgaatctggtactgtttcac\n",
      "cacgccgccaaccgaagcgacctcagccacgttcgggatggttttcagctcaaatttcag\n",
      "gaaccagtcctgcagagaacgcagttctgaaaggtcgtgttttccgctgcgatcgacaag\n",
      "ggcatattcaaatatccagcccacccccgtggcgtccggaccgatttcagaactcacacc\n",
      "ggcgggcagttttccctgtacctggttcaggtactccagcacgcgggaacgggcccagta\n",
      "cagatcggtgccgtcttcaaaaatgacatacacatacgaatcaccgaactgtgaaaaacc\n",
      "acgcacggtttttgcgccaggcacggacagcatggtggtggtaagtggataggtgacctg\n",
      "gttttctacaatctgcggggcctggccgggatagctggttttgataatgacctgcacatc\n",
      "tgacaggtcaggcaaggcatcgacaggcgtgttaataatcgtccatgtgccccagatgct\n",
      "gagaaacagggcccccatcatgaccaggaaacggttggcgacagagcgccggataatcca\n",
      "ttcaatca-\n",
      ">G000299455|3819953|3834467|3826097|3829225|+\n",
      "-tgtcggtgcagccacatcagcttatacgccgccgggataataaacagcgacagcaaagg\n",
      "tgcggtgatcatgccgccaatcatcggtgcggcaatccggctcatcacctctgaaccagc\n",
      "ccccgttccccacagaatcggcagcagaccggcgataatcaccgccaccgtcatcgcttt\n",
      "cgggcgcacgcgcaggaccgcgccgtgatataacgcctcatccagcttctgctcgctgaa\n",
      "tgtttgtggattattcaacgacggctcggcctctatggcgtgacgtaaatacatcagcat\n",
      "caccacgccaaattcggcggcgaccccggcgagggcgataaagccagtgcccgtcgccac\n",
      "ggaaagatgaaagcccatccaccacagcaaccagatgccgcccaccagcgcaaacggtac\n",
      "gctgctgataatcagcaacgcttcgcccacccgacggaacgccagatacaacagcacgaa\n",
      "gataatcatcaacgtcatcggcaccatgagtttaagcttatggttggcgcgctccagtag\n",
      "ctcgaactgcccggagaatgccacgctggtgcccggttttaactgcactttttcagctat\n",
      "cgctttttgcaaatcgtgaaccaccgacaccatgtcacgatcgcgggcatcgatataaat\n",
      "ccagctcgtcgggcgcgcattctcggttttcagcattgacggtccggtagagactttaat\n",
      "gtcggccacgtctgccagggtgatttgctgcttcatcggcgtcagaatcggcagctggcg\n",
      "cagtgcctgcggactatcgcgccagctttgcggataacgcagattaattggataacgggc\n",
      "aattccttccaccgtttcgccaaccatcgccccgcccaccgtagaagtcacaaacaactg\n",
      "cacatccgccaccgtcataccgtaacgcgcggctttttcacggttaatctcaacgttgat\n",
      "atagcgcccaccttccagacgctcggcaagcgcagaagccacgcctggtaccgttcgtgc\n",
      "cacctcttcaatttgctccgccatcgcgtcgatatccgccagcacagtgccggaaacttt\n",
      "aattccgatggggcttttaatgccagtcgagagcatatcgatacggttacgaattggcgg\n",
      "cacccacagattcgccagccccggcagacgcacggtgttatccagttcctcaatgatttt\n",
      "gtccatcgtcatgcctggccgccactgctcctgcggcttaagctggatggtcgtttctac\n",
      "catctccagcggagcagaatcggtggcggtttccgctttcccggttttcccaaatacccg\n",
      "cgccacttcaggtacgctcataattagcttgtcggttttttgcagcatactcgccgcctc\n",
      "tgctgcggaaatccccggcagcgtcgatggcatatacaacaagtcgccttcattgatctg\n",
      "cggtaaaaattccccgccaactttattgagcggccagagaaccgtcagcaccgaaagcgc\n",
      "cgccaccagcagcgtggttttcggccagtgcagtactttcagcaacagcggatgataaac\n",
      "acgaatcaaaaagcgattgagcgggttactgctttccggcggaattttgccacggatcca\n",
      "gtagcccatcaggatcgggatcactacgatcgccagcagcgccgcacccgccatcgcata\n",
      "cgttttggtgaacgccaacgggccaaacagacgcccttcctgcccttccagggtgaagat\n",
      "cgggataaacgacaacgtgataatcagcagactgataaatagcgccggacccacttcaac\n",
      "agatgcatcggtgatcacctgccagcgcgttttattatccagcgtggcgtcaggatgctg\n",
      "gtgctgccactcttccagccgtttatgtgcattttcgatcatgacgatagcggcatcgac\n",
      "catcgccccgacggcaatcgcaatgccgcccaacgacataatattggcattcagtccctg\n",
      "gaagtgcatgacaataaaagcaatacacaaccccagcggcaacgaaataatcgccaccag\n",
      "cgccgagcgcacatgccagagaaacagcgcacagactaccgccaccacaataaactcttc\n",
      "cagcaacttgccgctgaggttgtcgatggcgcggtcaatgagctggctgcgatcgtatgt\n",
      "tgtaactatctccacgccttccggcagactacttttcagcgtttccagtttgtccttcac\n",
      "ggcggcgatcacttctcgggcgtttttgccggatcgcaggatcaccaccccgcccgccac\n",
      "ttcaccttcgccattcagttcggcaatgccccggcgcatctccgggccaatctggacctt\n",
      "cgcaacatcacgcagataaacgggcacgccattttcactggcttttaaaacgatgtgatt\n",
      "aaagtcgtcgagcgtttgcagatagccgctggcgcgcaccatatattccgcttccgccag\n",
      "ttcgatcgacgaaccgcccgcttcctggtttgaagcatccagcgcgctttttacttcggc\n",
      "tagactgatgccatattgcgccaggcgctgggaatcgataaccacctgatactctttcac\n",
      "cacgccgcccaccgacgccacttccgcaacgtcagggatggtttttagctcatatttgag\n",
      "aaaccagtcctgcaatgagcgtaaatcggccagatcgtgcttaccgctgcgatccaccag\n",
      "tgcatattcatagatccagccaacacccgtggcatctggtcccagctcggcgctgactcc\n",
      "cgcaggcagcttaccctgtacctggttgaggtactccagcacccgcgaacgcgcccagta\n",
      "cggatcggtaccatcttcgaaaatgacatacacataagagtcgccaaactgtgagaaacc\n",
      "gcgcacagtcttcgcgccaggcaccgacaacatggtggtggttagcggataagtcacctg\n",
      "attttcaacgatttgcggtgcctgaccgggatagctggttttaataatcacctgcacatc\n",
      "ggagagatccggcagcgcatccactggcgtgttaatgatggtccaggtgccccagatgct\n",
      "cagaaacaacgcgcccatcagcaccaaaaaacggttcgccaccgagcgacgaataatcca\n",
      "ttcaatcat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (119166, 119300), '+'), (('G000299455', (3819953, 3834467)), (3825307, 3825441), '+'), (('G000299455', (4300454, 4333834)), (4329206, 4329340), '-')]\n",
      ">G000006925|64491|216702|119166|119300|+\n",
      "-gaccaccgaacgagaacatgataatcgccatcatcatcaccagtccggtgaagccgtgc\n",
      "ggcaggaaaccaccttgatcccacaggttgctaacgctcgcctgcgggccgccgttgcca\n",
      "ctgaacagcagccag\n",
      ">G000299455|3819953|3834467|3825307|3825441|+\n",
      "-gaccgccgaaggagaacataattaccgccagcgacaaaatcagcccattccagccggtg\n",
      "gcaaagaaaccaccgtagcgccagaggttgtcgatactggctttctcgccgccgtgacca\n",
      "gaaaacagcagccac\n",
      ">G000299455|4300454|4333834|4329206|4329340|-\n",
      "agaccaccgaacgagaacatgataatcgccatcatcatcaccagcccggtgaagccgtgc\n",
      "ggcaggaaaccaccctgatcccacaggttgctaacggtcgcctgcgggccgccgttgcca\n",
      "ctgaatagcagcca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4478753, 4490457)), (4485497, 4485546), '+'), (('G000299455', (3672841, 3680073)), (3674108, 3674157), '+'), (('G000299455', (4713031, 4748324)), (4717175, 4717224), '-')]\n",
      ">G000006925|4478753|4490457|4485497|4485546|+\n",
      "-agaccaccagttccgccagggagttggagcccagacggtttgcaccgtg\n",
      ">G000299455|3672841|3680073|3674108|3674157|+\n",
      "-agaccaccaggtccagcagcgagttgccgcccagacggttagcgccgtg\n",
      ">G000299455|4713031|4748324|4717175|4717224|-\n",
      "aagaccaccagttccgccagggagttggagcccagacggtttgcaccgt-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (905848, 907049)), (905869, 905895), '-'), (('G000006925', (1397676, 1398098)), (1398036, 1398062), '+'), (('G000006925', (1581377, 1581641)), (1581418, 1581444), '-'), (('G000006925', (2062783, 2064325)), (2063927, 2063953), '-'), (('G000299455', (2541782, 2542216)), (2542154, 2542180), '+'), (('G000299455', (2927401, 2931337)), (2930154, 2930180), '-'), (('G000299455', (3256459, 3256588)), (3256562, 3256588), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\n",
      "Reallocating..done. *alloclen = 1044\n",
      "\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|905848|907049|905869|905895|-\n",
      "ccacctctgatgtgtcaggcatcctc-\n",
      ">G000006925|1397676|1398098|1398036|1398062|+\n",
      "-cacctctcatgtgtcaggcatcctca\n",
      ">G000006925|1581377|1581641|1581418|1581444|-\n",
      "ccacctctgatgtggcaggcatcctc-\n",
      ">G000006925|2062783|2064325|2063927|2063953|-\n",
      "ccaccgctgatgtgtcaggcatcctc-\n",
      ">G000299455|2541782|2542216|2542154|2542180|+\n",
      "-cacctctgatgtgtcaggcatcctca\n",
      ">G000299455|2927401|2931337|2930154|2930180|-\n",
      "ccacctctgatgtgtcaggcatcctc-\n",
      ">G000299455|3256459|3256588|3256562|3256588|+\n",
      "-cacctctgatgtgtcaggtatcctca\n",
      "\n",
      "\n",
      "7 [(('G000006925', (64491, 216702)), (119300, 119321), '+'), (('G000006925', (3969227, 3993100)), (3989936, 3989957), '-'), (('G000006925', (4450485, 4460192)), (4457575, 4457596), '+'), (('G000299455', (3819953, 3834467)), (3825441, 3825462), '+'), (('G000299455', (4300454, 4333834)), (4329185, 4329206), '-'), (('G000299455', (4658720, 4667733)), (4665812, 4665833), '+'), (('G000299455', (5156132, 5190970)), (5170240, 5170261), '+')]\n",
      ">G000006925|64491|216702|119300|119321|+\n",
      "-ccgccgaagatgatcatcgct\n",
      ">G000006925|3969227|3993100|3989936|3989957|-\n",
      "caggccaatgacaatcatcac-\n",
      ">G000006925|4450485|4460192|4457575|4457596|+\n",
      "-aggccgacgacaatcagcgac\n",
      ">G000299455|3819953|3834467|3825441|3825462|+\n",
      "-aggccaaagccgatcataccg\n",
      ">G000299455|4300454|4333834|4329185|4329206|-\n",
      "gccgccgaagatgatcatcgc-\n",
      ">G000299455|4658720|4667733|4665812|4665833|+\n",
      "-aggccgacgacaatcagcgac\n",
      ">G000299455|5156132|5190970|5170240|5170261|+\n",
      "-aggccaatgacaatcatcacg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (741508, 744027), '+'), (('G000006925', (1921807, 1928114)), (1923560, 1926079), '-'), (('G000299455', (2189293, 2192099)), (2189580, 2192099), '+'), (('G000299455', (2897570, 2901707)), (2899188, 2901707), '-'), (('G000299455', (3586763, 3590110)), (3587304, 3589823), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 6040\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|741508|744027|+\n",
      "-ggcggttttgtcgtttatggagcgtgaggaatgggtaaaggcagcagtaaggggcatac\n",
      "cccgcgcgaagcgaaggacaacctgaaatccacgcagttgctgagtgtgattgatgccat\n",
      "cagcgaagggccggttgacggtccggtggatggattaaaaagcgtgctgctgaacggtac\n",
      "gccggtcctggatagcgaggggaagacaaacttttccggtgttacggtggtgttccgcgc\n",
      "cggcgagcaggagcagacaccgccggaggggtttgaatcttccggctcagagactgtgct\n",
      "gggtacggaagtgaaatacgacacgccgatcacccggaccatcacgtcggcaaacattga\n",
      "tcgtctgcgcctgaccttcggtgtgcaggcactggtggaaaccacctcaaagggggaccg\n",
      "gaatccgtcggaagtccgcctgctggttcagatacagcgtaacggtggctgggtgacgga\n",
      "aaaagacatcaccattaagggcaaaaccacttcgcagtatctggcctcggtggtggtggg\n",
      "taacctgccgccgcgcccgtttaatatccggatgcgcaggatgacaccggacagcaccac\n",
      "agaccagctgcagaacaaaacgctctggtcgtcgtacaccgaaatcatcgatgtgaaaca\n",
      "gtgctacccgaacacggcgctggtcggcgtgcaggtggattcagagcagttcggtaacca\n",
      "gcaggtgagtcgcaattatcatcttcgcgggcgcattctgcaggtgccgtcgaactataa\n",
      "cccgcagacgcggcaatacagcggtatctgggacggaacgcttaagccagcatacagcaa\n",
      "caacatggcctggtgtctgtgggacatgctcactcatccgcgctacggcatggggaaacg\n",
      "tcttggtgcggcagatgtggacaaatgggcgctgtatgtcatcggccagtactgcgacca\n",
      "gtcagtgccggacggatttggcggcacggagccgcgcatcacctgtaacgcttacctgac\n",
      "cacacagcgtaaggcgtgggatgttctcagtgatttctgctcggcgatgcgctgtatgcc\n",
      "ggtatggaacgggcagacgctgacgttcgtgcaggaccgaccgtcggataaagtgtggac\n",
      "ctataaccgcagtaatgtggtgatgccggatgatggcgcgccgttccgctacagcttcag\n",
      "cgccctgaaggaccgccataatgtcgttgaggtgaactggattgatccggacaacggtca\n",
      "tgagacggcgacagagcttgtggaggacacgcaggccattgtccgttacggtcgtaacgt\n",
      "cacgaagatggatgcctttggctgtaccagccggggacaggcgcaccgcgccgggctgtg\n",
      "gctgattaaaacggaactgctggaaacgcagaccgtggacttcagcgtcggcgcagaagg\n",
      "gcttcgccatgtaccgggtgatgtcattgaaatctgcgatgatgattatgccggtatcag\n",
      "caccggtggtcgcgtactggcggtgaacagccagacccggacgctgacgctcgaccgtga\n",
      "aatcacgctgccgtcctccggtaccacgctgataagcctggttgacggaaatggcaatcc\n",
      "ggtcagcgtggaggttcagtccgtcaccgacggcgtgaaggtgaaagtgagccgggttcc\n",
      "tgacggcgttgcagaatacagcgtgtgggggctgaagctgccgacgctgcgccagcgcct\n",
      "gttccgctgcgtgagtatccgtgagaacgacgacggcacgtatgccatcaccgccgtgca\n",
      "gcatgtgccggaaaaagaggccatcgtggataacggggcacactttgacggtgaccggcg\n",
      "cggcacggtgaatggtgtcacgccgccagcggtgcagcatctgaccgccgaagtcaccgc\n",
      "agacagcggggaatatcaggtgctggcgcgatgggacacgccgaaggtggtgaagggtgt\n",
      "gagcttcctgcttcgcctgaccgtggcagcggacgacggcagtgagcggctggtcagcac\n",
      "agccaggacgacggaaaccacataccgcttcaggcagctggcgctggggaattacagtct\n",
      "gacagtccgggcggtaaatgcccgggggcagcagggcgatccggcgtcggtatcgttccg\n",
      "gattgcggcaccggcagcgcctgtcactattgaactgataccggggtattttcagataac\n",
      "ggtggtcccgaaactggctgtatatgacccgacggtgcagtttgagttctggttttcgga\n",
      "aaagcggattgcggatatcaggcaggttgaaaccagcgcccgctatcttggcacggcgct\n",
      "gtactggatagccgccagtatcaatatcaggccgggccatgattattatttttacgttcg\n",
      "cagtgtgaacaccgttggcaaatcggcatttgtggaggccgtcggtcgggcgagcgatga\n",
      "tgcggaaggttatctgagtttttataaagggttgatcaataaaacgcatctcggcaagga\n",
      ">G000006925|1921807|1928114|1923560|1926079|-\n",
      "gggcggttttgtcgtttatggagcgtgaggaatgggtaaaggcagcagtaaggggcatac\n",
      "cccgcgcgaagcgaaggacaacctgaaatccacgcagttgctgagtgtgattgatgccat\n",
      "cagcgaagggccggttgacggtccggtggatggattaaaaagcgtgctgctgaacggtac\n",
      "gccggtcctggatagcgaggggaagacaaacttttccggtgttacggtggtgttccgcgc\n",
      "cggcgagcaggagcagacaccgccggaggggtttgaatcttccggctcagagactgtgct\n",
      "gggtacggaagtgaaatacgacacgccgatcacccggaccatcacgtcggcaaacattga\n",
      "tcgtctgcgcctgaccttcggtgtgcaggcactggtggaaaccacctcaaagggggaccg\n",
      "gaatccgtcggaagtccgcctgctggttcagatacagcgtaacggtggctgggtgacgga\n",
      "aaaagacatcaccattaagggcaaaaccacttcgcagtatctggcctcggtggtggtggg\n",
      "taacctgccgccgcgcccgtttaatatccggatgcgcaggatgacaccggacagcaccac\n",
      "agaccagctgcagaacaaaacgctctggtcgtcgtacaccgaaatcatcgatgtgaaaca\n",
      "gtgctacccgaacacggcgctggtcggcgtgcaggtggattcagagcagttcggtaacca\n",
      "gcaggtgagtcgcaattatcatcttcgcgggcgcattctgcaggtgccgtcgaactataa\n",
      "cccgcagacgcggcaatacagcggtatctgggacggaacgcttaagccagcatacagcaa\n",
      "caacatggcctggtgtctgtgggacatgctcactcatccgcgctacggcatggggaaacg\n",
      "tcttggtgcggcagatgtggacaaatgggcgctgtatgtcatcggccagtactgcgacca\n",
      "gtcagtgccggacggatttggcggcacggagccgcgcatcacctgtaacgcttacctgac\n",
      "cacacagcgtaaggcgtgggatgttctcagtgatttctgctcggcgatgcgctgtatgcc\n",
      "ggtatggaacgggcagacgctgacgttcgtgcaggaccgaccgtcggataaagtgtggac\n",
      "ctataaccgcagtaatgtggtgatgccggatgatggcgcgccgttccgctacagcttcag\n",
      "cgccctgaaggaccgccataatgtcgttgaggtgaactggattgatccggacaacggtca\n",
      "tgagacggcgacagagcttgtggaggacacgcaggccattgtccgttacggtcgtaacgt\n",
      "cacgaagatggatgcctttggctgtaccagccggggacaggcgcaccgcgccgggctgtg\n",
      "gctgattaaaacggaactgctggaaacgcagaccgtggacttcagcgtcggcgcagaagg\n",
      "gcttcgccatgtaccgggtgatgtcattgaaatctgcgatgatgattatgccggtatcag\n",
      "caccggtggtcgcgtactggcggtgaacagccagacccggacgctgacgctcgaccgtga\n",
      "aatcacgctgccgtcctccggtaccacgctgataagcctggttgacggaaatggcaatcc\n",
      "ggtcagcgtggaggttcagtccgtcaccgacggcgtgaaggtgaaagtgagccgggttcc\n",
      "tgacggcgttgcagaatacagcgtgtgggggctgaagctgccgacgctgcgccagcgcct\n",
      "gttccgctgcgtgagtatccgtgagaacgacgacggcacgtatgccatcaccgccgtgca\n",
      "gcatgtgccggaaaaagaggccatcgtggataacggggcacactttgacggtgaccggcg\n",
      "cggcacggtgaatggtgtcacgccgccagcggtgcagcatctgaccgccgaagtcaccgc\n",
      "agacagcggggaatatcaggtgctggcgcgatgggacacgccgaaggtggtgaagggtgt\n",
      "gagcttcctgcttcgcctgaccgtggcagcggacgacggcagtgagcggctggtcagcac\n",
      "agccaggacgacggaaaccacataccgcttcaggcagctggcgctggggaattacagtct\n",
      "gacagtccgggcggtaaatgcccgggggcagcagggcgatccggcgtcggtatcgttccg\n",
      "gattgcggcaccggcagcgcctgtcactattgaactgataccggggtattttcagataac\n",
      "ggtggtcccgaaactggctgtatatgacccgacggtgcagtttgagttctggttttcgga\n",
      "aaagcggattgcggatatcaggcaggttgaaaccagcgcccgctatcttggcacggcgct\n",
      "gtactggatagccgccagtatcaatatcaggccgggccatgattattatttttacgttcg\n",
      "cagtgtgaacaccgttggcaaatcggcatttgtggaggccgtcggtcgggcgagcgatga\n",
      "tgcggaaggttatctgagtttttataaagggttgatcaataaaacgcatctcggcaagg-\n",
      ">G000299455|2189293|2192099|2189580|2192099|+\n",
      "-ggcggttttgtcatttatggagcgtgaggaatgggtaaaggaagcagtaaggggcatac\n",
      "cccgcgcgaagcgaaggacaacctgaagtccacgcagttgctgagtgtgatcgatgccat\n",
      "cagcgaagggccggttgaaggtccggtggatggattaaaaagcgtgctgctgaacagtac\n",
      "gccggtgctggacactgaggggaataccaacatctccggtgtcacggtggtgttccgtgc\n",
      "cggtgagcaggagcagacaccgccggagggttttgaatcctccggctccgagacggtgct\n",
      "gggtacggaagtgaaatatgacacgccgatcacccgcaccattacgtctgcaaacatcga\n",
      "ccgtctgcgctttaccttcggcgtgcaggcactggtggaaaccacctcaaagggggaccg\n",
      "gaatccgtcggaagtccgcctgctggttcagatccagcgtaatggtggctgggtgacgga\n",
      "aaaagacatcaccattaagggcaaaaccacctcgcagtatctggcctcggtggtggtggg\n",
      "taacctgccgccgcgcccgttcaatatacggatgcgcaggatgacgccggacagcaccac\n",
      "agaccagctgcagaacaaaacgctctggtcgtcatacaccgaaatcatcgatgtgaaaca\n",
      "gtgctacccgaacacggcactggtcggcgtgcaggtggattcggagcagttcggcagcca\n",
      "gcaggtgagccgtaattatcatctgcgcgggcgcattctgcaggtgccgtcgaattataa\n",
      "cccgcagacgcggcaatacagcggtatctgggacggaacgtttaaaccggcatacagcaa\n",
      "caacatggcctggtgtctgtgggatatgctgacccatccgcgctacggcatggggaaacg\n",
      "tcttggtgcggcggatgtggacaaatgggcgctgtatgtcatcggccagtactgcgacca\n",
      "gtcggtgccggacggctttggcggcacggagccgcgcatcacctgtaatgcctggctgac\n",
      "cacgcagcgtaaggcgtgggatgtgctcagtgatttctgctcggtgatgcgctgtatgcc\n",
      "ggtatggaacgggcagacgctgacgttcgtgcaggaccggccgtcggataaggtgtggac\n",
      "ctataaccgcagtaatgtggtgatgccggatgatggcgcgccgttccgctacagtttcag\n",
      "cgccctgaaggaccgccataatgccgttgaggtgaactggactgacccggataacggctg\n",
      "ggaaacggcaacagagcttgtggaggacacgcaggccattgcccgttatggtcgtaatgt\n",
      "cacgaagatggatgcctttggctgtaccagccgggggcaggcacaccgcgccgggctgtg\n",
      "gctgattaaaacggagctgctggaaacgcagaccgtggacttcagcgtgggtgccgaagg\n",
      "gcttcgccatgtaccgggcgatgttattgaaatctgcgatgatgactatgccggtatcag\n",
      "caccggtggtcgtgtgctggcggtgagcagccaggcccggacgctgacgctcgaccgtga\n",
      "aatcacgctgccatcctccggtaccacgctgataagcctggttgacggaagtggcaatcc\n",
      "ggtcagcgtggaggttcagtccgtcaccgacggcgtgaaggtgaaagtgagccgggttcc\n",
      "tgacggcgttgcagaatacagcgtgtgggggctgaagttgccgacgttgcgccagcgcct\n",
      "gttccgctgtgtgagtatccgtgagaacgacgacggcacgtatgccatcaccgccgtgca\n",
      "gcatgtaccggaaaaagaagccatcgtggataacggggcgcactttgacggcgaccagag\n",
      "cggcacggtgaatggtgtcacgccgccagcggtgcagcacctggccgcagaagtcactgc\n",
      "agacagcggggaatatcaggtgctggcgcgatgggacacaccgaaggtggtgaagggcgt\n",
      "gagcttcctgctccgtctgaccgtaacagcggacgacggcagtgagcggctggtcagcac\n",
      "ggcccggacgacggaaaccacataccgcttcaggcaactggcgctggggcgttacacgct\n",
      "gacggtccgggcggtaaatgcgtgggggcagcagggcgatccggcatcggtatcgttccg\n",
      "gattgcggcaccggcagcgcctgtcactattgaactgataccagggtattttcagataac\n",
      "agcggtcccgaaactggctgtatatgacccgacggtgcagtttgagttctggttctcgga\n",
      "aaagcggattatcgatatcaggcaggttgaaaccagcgcgcgttatcttggtacggcgct\n",
      "gtactggatagccgccagtagcaatattaagccgggttatgattattacttttatatccg\n",
      "cagcgtgaacaccgttggtaaatcggcatttgtggaggcagttggtcgggcgagcgatga\n",
      "tgcggaaggttatctgaatttttataaagggttgatcaataaaacgcatctcggcaagga\n",
      ">G000299455|2897570|2901707|2899188|2901707|-\n",
      "gggcggctttgttgtttatggagtgtgaggaatgggtaaaggaagcagtaaggggcatac\n",
      "cccgcgcgaagcgagggacaacctgaagtccacgcagctgctgagtgtgatcgatgccat\n",
      "cagcgaagggccggttgaaggtccggtggacggattaaaaagcgtgctgctgaacagtac\n",
      "gccggtgctggacagtgaggggaataccaatatatccggtgtcacggtggtgttccgggc\n",
      "aggtgagcaggagcagacaccgccggagggatttgaatcctccggctccgagacggtgct\n",
      "gggtacggaagtgaaatatgacacgccgatcacccggaccatcacgtcggcaaacatcga\n",
      "ccgtctgcgctttaccttcggtgtgcaggcactggtggaaaccacctcaaagggggacag\n",
      "gaatccgtcggaagtccgcctgctggttcagatccagcgtaacggtggctgggtgacgga\n",
      "aaaagacatcaccattaagggcaaaaccacctcgcagtatctggcctcggtggtggtgga\n",
      "taacctgccgccgcgcccgtttaatatccggatgcgcagaatgacgccggacagcaccac\n",
      "agaccagctgcagaacaaaacgctctggtcgtcatacaccgaaatcatcgatgtgaaaca\n",
      "gtgctacccgaacacggcactggtcggcgtgcaggtggactcggaacagttcggcagcca\n",
      "gcaggtgagccgtaattatcatcttcgcgggcgcattctgcaggtgccgtcgaactataa\n",
      "cccgcagacgcggcaatacagcggtatctgggacggaacgtttaagccagcatacagcaa\n",
      "caacatggcctggtgtctgtgggatatgctgacccatccgcgctacggcatggggaagcg\n",
      "tctcggtgcggcggatgtggacaaatgggcgctgtatgtcatcggccagaattgcgacca\n",
      "gtcggtgccggatggctttggtggcacggagccgcgcatcacctgtaatgcctacctgac\n",
      "cacacagcgcaaggcgtgggatgttctcagtgatttctgctcggcgatgcgctgtatgcc\n",
      "ggtatggaacgggcagacgctgacgttcgtgcaggaccgaccgtcggataaggtgtggac\n",
      "ctataaccgcagtaatgtggtgatgccggatgatggcgcgccgttccgctacagcttcag\n",
      "tgccctgaaggaccgtcataatgccgttgaggtgaactggattgacccggataacggctg\n",
      "ggagacggcgacagagcttgttgaagatacgcaggccattgcccgttacggtcgtaacgt\n",
      "cacgaagatggatgcctttggctgtaccagccgggggcaggcgcaccgcgccgggctgtg\n",
      "gctgattaaaacagaactgctggaaacgcagaccgtggacttcagcgtgggtgccgaagg\n",
      "gcttcgccatgtaccgggggatgtcattgaaatctgcgatgatgactatgcgggtatcag\n",
      "caccggcgggcgcgtgctggcggtgaacagccagacccggacgctgacgctcgaccgtga\n",
      "aatcacgctgccatcctccggtaccacgctgataagcctggttgacggtcagggtaatcc\n",
      "ggtcagcgtggaggtccagtccgtcaccgacggcgtgaaggtgaaagtgagccgtgttcc\n",
      "tgacggcgttgcagaatacagcgtgtgggggctgaagctgccgacgctgcgccagcgcct\n",
      "gttccgctgcgtgagtatccgtgagaacgatgacggcacgtatgccatcaccgccgtgca\n",
      "gcatgtaccggagaaagaagccatcgtggataacggggcgcactttgacggtgaccagag\n",
      "cggcacggtgaatggtgtcacgccgccagcagtgcagcacctgaccgccgaagtcactgc\n",
      "agacagcggggaatatcaggtgctggcgcgatgggacacaccgaaggtggtgaagggcgt\n",
      "gagcttcctgctccgtctgaccgtaacagcggacgacggcagtgagcgactggtcagcac\n",
      "ggcccggacgacggaaaccacataccgcttcacgcaactggcgctggggaactacaggct\n",
      "gacagtccgggcggtaaatgcccgggggcagcagggcgatccggcgtcggtatcgttccg\n",
      "gattgccgcaccggcagcaccgtcgaggattgagctgacgccgggctattttcagataac\n",
      "cgccacgccgcatcttgcggtttatgatccgacggtacagtttgagttctggttctcgga\n",
      "aacgcggattaccgatatcaggcaggttgaaaccacagcccgctaccttggcacggggct\n",
      "gtactggatagccgccagtatcaatatcaaaccgggccatgattattacttttatatccg\n",
      "cagtgtgaacaccgttggcaaatcggcattcgtggaggccgtcggtcgggcgagcgatga\n",
      "tgcggaaggttacctggattttttcaaaggagaaatcgggaaaacacatctggcccagg-\n",
      ">G000299455|3586763|3590110|3587304|3589823|-\n",
      "gggcggttttatcgtttatggagcatgacgaatgggtaaaggcagcagtaaggggcatac\n",
      "cccgcgcgaagcgaaagataacctgaagtccacgcagctgctgagtgtgatcgatgccat\n",
      "cagcgaagggccggttgaaggtccggtggatggattaaaaagcgtgctgctgaacagtac\n",
      "gccggtgctggacagtgaggggaataccaatatatccggcgtcacggtggtgttccgggc\n",
      "cggtgagcaggagcagacaccgccggagggatttgaatcctccggctccgagacggtgct\n",
      "cggtacagaagtgaaatatgacacgccgatcacccggaccatcacgtcggcaaacattga\n",
      "ccgtctgcgttttactttcggcgtgcaggcactggtggaaaccacctcaaagggggacag\n",
      "gaatccatcggaagtccgcctgctggttcagatacaacgtaacggtggctgggtgacgga\n",
      "aaaagacatcaccattaagggtaaaaccacttcacagtatctggcctcggtggtggtgga\n",
      "taacctgccgccgcgcccgtttaatatccggatgcgcaggatgacgccggacagcaccac\n",
      "agaccagctgcagaacaaaacgctctggtcgtcatacaccgaaattatcgatgtgaaaca\n",
      "gtgctacccgaacacggcactggtcggcgtgcaggtggactcggagcagttcggcagcca\n",
      "gcaggtgagccgtaattatcatctgcgcgggcgtattctgcaggtgccgtcgaattataa\n",
      "cccgcagacgcggcaatacagcggtatctgggacggaacgtttaaaccggcatacagcaa\n",
      "caacccggcctggtgtctgtgggatatgctgacccatccgcgctacggcatggggaaacg\n",
      "tcttggtgcggtggatgtggataaatgggcgctgtatcttatcggccagtactgcgacca\n",
      "gtcagtgccggacggctttggcggcacggagccgcgcatcacctgtaatgcctggctgac\n",
      "cacacagcgtaaggcgtgggatgtgctcagtgatttctgctctgcgatgcgctgtatgcc\n",
      "ggtatggaacgggcagacgctgacgttcgtgcaggaccgaccgtcggataaggtgtggac\n",
      "ctataaccgcagtaatgtggtgatgccggatgatggtgcgccgttccgctacagcttcag\n",
      "cgccctgaaggaccgccataatgccgttgaggtgaactggattgacccgaataacggctg\n",
      "ggagacggcgacagagcttgtggaggacacgcaggccattgcccgttacggtcgtaacgt\n",
      "cacgaagatggatgcttttggctgtaccagccgggggcaggcacaccgcgccgggctgtg\n",
      "gctgattaaaacagaactgctggaaacgcagaccgtggacttcagcgtgggtgccgaagg\n",
      "gcttcgccatgtaccgggcgatgtcattgaaatctgcgatgatgactatgccggtatccg\n",
      "caccggcgggcgcgtgctggcggtgaacagccagacccggacgctgacgctcgaccgtga\n",
      "aatcacgctgccatcctccggtaccacgctgataagcctggttgacggaagtggtaatcc\n",
      "ggtcagcgtggaggttcagtccgtcaccgacggcgtgaaggtgaaagtgagccgtgttcc\n",
      "tgacggcgttgctgaatacagcgtatgggggctgaagctgccgacgttgcgccagcgcct\n",
      "gttccgctgcgtgagtatccgtgagaacgacgacggcacgtatgccatcaccgccgtgca\n",
      "gcatgtaccggaaaaagaggccatcgtggataacggggcgcactttgacggcgaccagag\n",
      "cggcacggtaaatggtgtcacgccgccagcggtgcagcacctgaccgcagaagtcaccgc\n",
      "agacagcggggaataccaggtgctggcgcgctgggacacgccgaaggtggtgaagggcgt\n",
      "gagcttcctgcttcgcctgaccgtggcagcggatgacggccgtgagcggctggtcagcac\n",
      "ggcccggacgacggaaaccacttaccgcttcacacaactggctctggggaactacaggct\n",
      "gacagtccgggcagtaaatgcgtgggggcagcagggcgatccggcgtcggtatcgttccg\n",
      "gattgccgcaccggcagcgccgtcgcggattgagctgacgccgggctattttcagataac\n",
      "cgccacgccgcatcttgcggtttatgacccgacggtacagtttgagttctggttctcgga\n",
      "aaaacggattgcggatatcaggcaggttgaaaccagcgcgcgttatcttggtacggcact\n",
      "gtactggatagccgccagtatcaatatcaaaccgggccatgattattatttttacgttcg\n",
      "cagtgtgaacaccgttggcaaatcggcattcgtggaggctgtcggtcagccgagtgatga\n",
      "tgcatcaggctatctggattttttcaaaggcgagatagggaaaacccatctggctcagg-\n",
      "\n",
      "\n",
      "10 [(('G000006925', (386390, 386480)), (386450, 386480), '+'), (('G000006925', (898987, 899077)), (898987, 899017), '-'), (('G000006925', (2059905, 2059995)), (2059965, 2059995), '+'), (('G000006925', (3421218, 3421308)), (3421218, 3421248), '-'), (('G000006925', (3852168, 3852258)), (3852228, 3852258), '+'), (('G000006925', (4360146, 4360236)), (4360206, 4360236), '+'), (('G000006925', (4701524, 4701609)), (4701578, 4701609), '+'), (('G000006925', (4777177, 4777262)), (4777232, 4777262), '+'), (('G000006925', (4780959, 4781049)), (4780959, 4780989), '-'), (('G000299455', (5343755, 5343840)), (5343755, 5343785), '-')]\n",
      ">G000006925|386390|386480|386450|386480|+\n",
      "-cagccccaggctgtaacgcctgcttagcca\n",
      ">G000006925|898987|899077|898987|899017|-\n",
      "acagccccaggctgtaacgcctgcttagcc-\n",
      ">G000006925|2059905|2059995|2059965|2059995|+\n",
      "-cagccccaggctgtaacgcctgcttagcca\n",
      ">G000006925|3421218|3421308|3421218|3421248|-\n",
      "acagccccaggctgtaatgcctgcttagcc-\n",
      ">G000006925|3852168|3852258|3852228|3852258|+\n",
      "-cagccccaggctgtaatgcctgcttagcca\n",
      ">G000006925|4360146|4360236|4360206|4360236|+\n",
      "-cagccccaggctgtaacgcctgcttagcca\n",
      ">G000006925|4701524|4701609|4701578|4701609|+\n",
      "acagccccaggctgtaatgcctgcttagcca\n",
      ">G000006925|4777177|4777262|4777232|4777262|+\n",
      "-cagccccaggctgtaatgcctgcttagcca\n",
      ">G000006925|4780959|4781049|4780959|4780989|-\n",
      "acagccccaggctgtaacgcctgcttagcc-\n",
      ">G000299455|5343755|5343840|5343755|5343785|-\n",
      "gaatcccttggttgtaaagccagtttaacc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2008567, 2033069)), (2012135, 2012401), '-'), (('G000299455', (2082804, 2090582)), (2088325, 2088591), '+'), (('G000299455', (3614303, 3614569)), (3614303, 3614569), '-')]\n",
      ">G000006925|2008567|2033069|2012135|2012401|-\n",
      "tacaccaggatgaaaacggtgagttgtgttcaaactgtagtacaattctctccagtatga\n",
      "acaggaaagaatatgctatgaacccttatatttatcttggtggcgcaatacttgcagagg\n",
      "tcattggtacaaccttaatgaagttttcagaggggtttacacggttatggccctctgttg\n",
      "gcacaattatttgctattgtgcatcattctggttattagctcagacgctggcttatatac\n",
      "ctacagggattgcttatgctatctgg-\n",
      ">G000299455|2082804|2090582|2088325|2088591|+\n",
      "-acaccaggatgaaaacggtgagttgtgttcaaactgtagtacaattctctccagtatga\n",
      "acaggaaagaatatgctatgaacccttatatttatcttggtggcgcaatacttgcagagg\n",
      "tcattggtacaaccttaatgaagttttcagaggggtttacacggttatggccctctgttg\n",
      "gcacaattatttgctattgtgcatcattctggttattagctcagacgctggcttatatac\n",
      "ctacagggattgcttatgctatctggt\n",
      ">G000299455|3614303|3614569|3614303|3614569|-\n",
      "tacacaatgatgaaacaggtgagttgagttcaaactgtagtacaattctctccagtttga\n",
      "acaggaaagaatatgctatgaatccttatatttatcttggtggtgcaatacttgcagagg\n",
      "tcattggtacaaccttaatgaagttttcagaaggttttacacggttatggccatctgttg\n",
      "gtacaattatttgttattgtgcatcattctggttattagctcagacgctggcttatattc\n",
      "ctacagggattgcttatgctatctgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4270437, 4270460), '-'), (('G000299455', (2786550, 2804122)), (2800827, 2800850), '+'), (('G000299455', (4795454, 4823864)), (4809321, 4809344), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4256570|4288375|4270437|4270460|-\n",
      "gataaataacaggcaaaaaaaag-\n",
      ">G000299455|2786550|2804122|2800827|2800850|+\n",
      "-ataaataacaggcaaaaaaaagc\n",
      ">G000299455|4795454|4823864|4809321|4809344|-\n",
      "gataaataacaggcaaaaaaaag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (907508, 907840)), (907532, 907582), '-'), (('G000006925', (1395000, 1395332)), (1395258, 1395308), '+'), (('G000299455', (2540736, 2540915)), (2540843, 2540892), '+')]\n",
      ">G000006925|907508|907840|907532|907582|-\n",
      "ttttcaggaggcgatactgtgaatgaaattagctatcaggcttcaattac-\n",
      ">G000006925|1395000|1395332|1395258|1395308|+\n",
      "-tttcaggaggcgatactgtgaatgaaattagctatcaggcttcaattacc\n",
      ">G000299455|2540736|2540915|2540843|2540892|+\n",
      "-tttcaggaggcggtactgtgagt-aaattagctatcaggcttcaattgtc\n",
      "\n",
      "\n",
      "10 [(('G000006925', (236537, 236664)), (236537, 236561), '+'), (('G000006925', (864658, 864785)), (864760, 864785), '-'), (('G000006925', (980497, 980624)), (980599, 980624), '-'), (('G000006925', (1208359, 1208486)), (1208461, 1208486), '-'), (('G000006925', (1392468, 1392595)), (1392570, 1392595), '-'), (('G000006925', (1500992, 1501119)), (1500992, 1501016), '+'), (('G000006925', (2035520, 2035647)), (2035622, 2035647), '-'), (('G000006925', (4116789, 4116916)), (4116891, 4116916), '-'), (('G000006925', (4255689, 4255816)), (4255791, 4255816), '-'), (('G000299455', (5279601, 5279746)), (5279601, 5279624), '+')]\n",
      ">G000006925|236537|236664|236537|236561|+\n",
      "-gggcacatacaggcagagtagccg\n",
      ">G000006925|864658|864785|864760|864785|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|980497|980624|980599|980624|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|1208359|1208486|1208461|1208486|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|1392468|1392595|1392570|1392595|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|1500992|1501119|1500992|1501016|+\n",
      "-gggcacatacaggcagagtagccg\n",
      ">G000006925|2035520|2035647|2035622|2035647|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|4116789|4116916|4116891|4116916|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000006925|4255689|4255816|4255791|4255816|-\n",
      "cgggcacatacaggcagagtagccg\n",
      ">G000299455|5279601|5279746|5279601|5279624|+\n",
      "-gggaacataaaggccga-tatcgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3266087, 3266132), '-'), (('G000299455', (671363, 678362)), (673525, 673570), '+'), (('G000299455', (1078152, 1154001)), (1143592, 1143637), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3266087|3266132|-\n",
      "tctttttctggtcgct------gttcaccttcctgcaaggctttgttgata-\n",
      ">G000299455|671363|678362|673525|673570|+\n",
      "---gcggctggctgcttgataagtttggctcgaaaaaagtttatacct----\n",
      ">G000299455|1078152|1154001|1143592|1143637|+\n",
      "-ctttatctggtcgat------gtttaccttgctgcaaggcttcgtcgatat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4620693, 4621057)), (4620896, 4620927), '+'), (('G000006925', (4791871, 4792235)), (4792074, 4792105), '+'), (('G000006925', (4794974, 4795158)), (4795103, 4795134), '-'), (('G000299455', (5313369, 5315826)), (5313572, 5313603), '+')]\n",
      ">G000006925|4620693|4621057|4620896|4620927|+\n",
      "cgcttccgcaacgctggcctttcatggcctc\n",
      ">G000006925|4791871|4792235|4792074|4792105|+\n",
      "cgcttccgcaacgctggcctttcatggcctc\n",
      ">G000006925|4794974|4795158|4795103|4795134|-\n",
      "cgcttccggaaatcagggctttcatggcctt\n",
      ">G000299455|5313369|5315826|5313572|5313603|+\n",
      "cgctttcggagagctggcctttcgtggcctt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1419563, 1419894)), (1419791, 1419894), '+'), (('G000299455', (2155437, 2155540)), (2155437, 2155540), '+'), (('G000299455', (2935269, 2935589)), (2935269, 2935373), '-')]\n",
      ">G000006925|1419563|1419894|1419791|1419894|+\n",
      "-gttgtggcgtggttgtcacttaagcgtatggtcaacctgacaacccggtgtcctcaac-\n",
      "ggggaaggaataactccgccatacttaccgccgcgtcatttcgcg\n",
      ">G000299455|2155437|2155540|2155437|2155540|+\n",
      "-gttgtggtggggttgtcactcaggcgtatggtcaacctgacaatccggtgtcctcaac-\n",
      "ggggaaagagtaaccccgccatacttaccgccgcgccatttcgcg\n",
      ">G000299455|2935269|2935589|2935269|2935373|-\n",
      "cgttgcagcggggttgtcacttaagcgtatggtcaacctgacaacccggtgtcctcaacg\n",
      "ggggaaggaataaccccgccatacttaccgccgcgccatttcgc-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (216745, 221250)), (216745, 216892), '+'), (('G000006925', (3403726, 3409928)), (3409781, 3409928), '-'), (('G000006925', (3952917, 3966316)), (3952917, 3953064), '+'), (('G000006925', (4050108, 4066321)), (4050108, 4050255), '+'), (('G000006925', (4188297, 4209378)), (4188297, 4188444), '+'), (('G000006925', (4228735, 4238289)), (4228735, 4228882), '+'), (('G000299455', (1329940, 1344983)), (1329940, 1330087), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|216745|221250|216745|216892|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      ">G000006925|3403726|3409928|3409781|3409928|-\n",
      "gtccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagata-\n",
      ">G000006925|3952917|3966316|3952917|3953064|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      ">G000006925|4050108|4066321|4050108|4050255|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      ">G000006925|4188297|4209378|4188297|4188444|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      ">G000006925|4228735|4238289|4228735|4228882|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttgatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      ">G000299455|1329940|1344983|1329940|1330087|+\n",
      "-tccccttcgtctagaggcccaggacaccgccctttcacggcggtaacaggggttcgaat\n",
      "cccctaggggacgccacttgctggtttgtgagtgaaagtcacctgccttaatatctcaaa\n",
      "actcatcttcgggtgatgtttgagatat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1696422, 1718400)), (1712009, 1712104), '+'), (('G000006925', (1815406, 1827803)), (1815408, 1815503), '+'), (('G000006925', (2006629, 2008384)), (2006992, 2007087), '+'), (('G000299455', (2269177, 2270929)), (2270473, 2270568), '-'), (('G000299455', (2737179, 2754578)), (2737540, 2737635), '+'), (('G000299455', (4667815, 4670271)), (4668590, 4668685), '+')]\n",
      ">G000006925|1696422|1718400|1712009|1712104|+\n",
      "-cggtttaaacctgattttgcccggacattgcgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaat\n",
      ">G000006925|1815406|1827803|1815408|1815503|+\n",
      "-cggtttaaacctgattttgcccggacattgcgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaat\n",
      ">G000006925|2006629|2008384|2006992|2007087|+\n",
      "-cggtttaaacctgattttgcccggacattgcgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaat\n",
      ">G000299455|2269177|2270929|2270473|2270568|-\n",
      "acggtttaaacctgattttgcccggacattgcgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaa-\n",
      ">G000299455|2737179|2754578|2737540|2737635|+\n",
      "-cggtttaaacctgattttgcccggacattgcgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaat\n",
      ">G000299455|4667815|4670271|4668590|4668685|+\n",
      "-cggtttaaacctgattttgcccggacattacgccccggctggctgaccgtacccgctgc\n",
      "tgactttgacatgttgctgaccttcaaatcctcaat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4297478, 4297515), '-'), (('G000299455', (4062045, 4062082)), (4062045, 4062082), '-'), (('G000299455', (4836149, 4864500)), (4852878, 4852915), '+')]\n",
      ">G000006925|4288513|4315006|4297478|4297515|-\n",
      "tgctggcctgcgcccgcattggcgcgatgcattcggt-\n",
      ">G000299455|4062045|4062082|4062045|4062082|-\n",
      "tgctggcctgcgcgcgcattggtgctattcactcggt-\n",
      ">G000299455|4836149|4864500|4852878|4852915|+\n",
      "-gctggcctgcgcccgcattggcgcggtgcattcggtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3515370, 3534729)), (3530490, 3530517), '-'), (('G000299455', (404418, 423796)), (408630, 408657), '+'), (('G000299455', (1054286, 1054377)), (1054286, 1054313), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 110\r",
      "  101 / 110\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 110\r",
      "   10 / 110\r",
      "   20 / 110\r",
      "   30 / 110\r",
      "   40 / 110\r",
      "   50 / 110\r",
      "   60 / 110\r",
      "   70 / 110\r",
      "   80 / 110\r",
      "   90 / 110\r",
      "  100 / 110\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 109  f\b\b\r",
      "STEP     2 / 109  f\b\b\r",
      "STEP     3 / 109  f\b\b\r",
      "STEP     4 / 109  f\b\b\r",
      "STEP     5 / 109  f\b\b\r",
      "STEP     6 / 109  f\b\b\r",
      "STEP     7 / 109  f\b\b\r",
      "STEP     8 / 109  f\b\b\r",
      "STEP     9 / 109  f\b\b\r",
      "STEP    10 / 109  f\b\b\r",
      "STEP    11 / 109  f\b\b\r",
      "STEP    12 / 109  f\b\b\r",
      "STEP    13 / 109  f\b\b\r",
      "STEP    14 / 109  f\b\b\r",
      "STEP    15 / 109  f\b\b\r",
      "STEP    16 / 109  f\b\b\r",
      "STEP    17 / 109  f\b\b\r",
      "STEP    18 / 109  f\b\b\r",
      "STEP    19 / 109  f\b\b\r",
      "STEP    20 / 109  f\b\b\r",
      "STEP    21 / 109  f\b\b\r",
      "STEP    22 / 109  f\b\b\r",
      "STEP    23 / 109  f\b\b\r",
      "STEP    24 / 109  f\b\b\r",
      "STEP    25 / 109  f\b\b\r",
      "STEP    26 / 109  f\b\b\r",
      "STEP    27 / 109  f\b\b\r",
      "STEP    28 / 109  f\b\b\r",
      "STEP    29 / 109  f\b\b\r",
      "STEP    30 / 109  f\b\b\r",
      "STEP    31 / 109  f\b\b\r",
      "STEP    32 / 109  f\b\b\r",
      "STEP    33 / 109  f\b\b\r",
      "STEP    34 / 109  f\b\b\r",
      "STEP    35 / 109  f\b\b\r",
      "STEP    36 / 109  f\b\b\r",
      "STEP    37 / 109  f\b\b\r",
      "STEP    38 / 109  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3515370|3534729|3530490|3530517|-\n",
      "agatcttcgcggaatttgccttcggca-\n",
      ">G000299455|404418|423796|408630|408657|+\n",
      "-gatcttcgcgaaatttgccttcggcaa\n",
      ">G000299455|1054286|1054377|1054286|1054313|-\n",
      "agatcttcgcggaatttaccttcggca-\n",
      "\n",
      "\n",
      "110 [(('G000006925', (1, 24671)), (18887, 19255), '+'), (('G000006925', (64491, 216702)), (87122, 87490), '-'), (('G000006925', (64491, 216702)), (148312, 148680), '-'), (('G000006925', (266682, 267450)), (266863, 267231), '-'), (('G000006925', (273344, 278315)), (273563, 273931), '+'), (('G000006925', (278320, 279091)), (278503, 278871), '-'), (('G000006925', (300111, 308544)), (307956, 308324), '-'), (('G000006925', (378637, 384812)), (378820, 379188), '-'), (('G000006925', (459861, 468373)), (465936, 466304), '+'), (('G000006925', (468374, 487224)), (478932, 479300), '+'), (('G000006925', (520755, 528026)), (520936, 521304), '-'), (('G000006925', (545486, 558918)), (558369, 558737), '+'), (('G000006925', (622915, 623662)), (623134, 623502), '+'), (('G000006925', (865887, 867389)), (866840, 867208), '+'), (('G000006925', (984595, 985335)), (984777, 985146), '-'), (('G000006925', (985336, 1008622)), (985649, 986017), '+'), (('G000006925', (1025530, 1063042)), (1049791, 1050159), '+'), (('G000006925', (1189492, 1208094)), (1197867, 1198222), '+'), (('G000006925', (1239481, 1240251)), (1239700, 1240069), '+'), (('G000006925', (1290847, 1306886)), (1295292, 1295660), '+'), (('G000006925', (1321493, 1385605)), (1365648, 1366003), '-'), (('G000006925', (1398496, 1399271)), (1398717, 1399085), '+'), (('G000006925', (1409877, 1410649)), (1410062, 1410430), '-'), (('G000006925', (1424264, 1425034)), (1424447, 1424815), '-'), (('G000006925', (1425133, 1465118)), (1444499, 1444867), '+'), (('G000006925', (1509064, 1578296)), (1526898, 1527266), '+'), (('G000006925', (1509064, 1578296)), (1533708, 1534076), '+'), (('G000006925', (1509064, 1578296)), (1542985, 1543353), '+'), (('G000006925', (1509064, 1578296)), (1568372, 1568740), '-'), (('G000006925', (1509064, 1578296)), (1574330, 1574698), '-'), (('G000006925', (1670306, 1692658)), (1673591, 1673959), '+'), (('G000006925', (1729332, 1741706)), (1736155, 1736523), '-'), (('G000006925', (1758378, 1759147)), (1758597, 1758965), '+'), (('G000006925', (1759148, 1776249)), (1766747, 1767115), '+'), (('G000006925', (1759148, 1776249)), (1769213, 1769581), '-'), (('G000006925', (1776250, 1792578)), (1781132, 1781500), '+'), (('G000006925', (1776250, 1792578)), (1791295, 1791663), '+'), (('G000006925', (1793868, 1802424)), (1795310, 1795678), '-'), (('G000006925', (1842508, 1869155)), (1850230, 1850598), '-'), (('G000006925', (1842508, 1869155)), (1856168, 1856536), '-'), (('G000006925', (1940277, 1941051)), (1940496, 1940864), '+'), (('G000006925', (1976680, 1977448)), (1976861, 1977229), '-'), (('G000006925', (2036705, 2044875)), (2037336, 2037704), '-'), (('G000006925', (2050169, 2051841)), (2051254, 2051622), '-'), (('G000006925', (2088344, 2103602)), (2095259, 2095627), '+'), (('G000006925', (2120115, 2138771)), (2134829, 2135197), '-'), (('G000006925', (2172359, 2184266)), (2178011, 2178379), '+'), (('G000006925', (2187913, 2198767)), (2197068, 2197436), '-'), (('G000006925', (2198768, 2199537)), (2198988, 2199356), '+'), (('G000006925', (2199538, 2210687)), (2205732, 2206100), '-'), (('G000006925', (2211287, 2225774)), (2220852, 2221220), '+'), (('G000006925', (2235234, 2236005)), (2235415, 2235783), '-'), (('G000006925', (2242494, 2346249)), (2287664, 2288032), '+'), (('G000006925', (2374481, 2472008)), (2395816, 2396184), '-'), (('G000006925', (2756085, 2756853)), (2756266, 2756634), '-'), (('G000006925', (2816684, 2833035)), (2825420, 2825788), '+'), (('G000006925', (2849870, 2870340)), (2863021, 2863389), '-'), (('G000006925', (2957651, 2963246)), (2962874, 2963242), '-'), (('G000006925', (2976303, 3015537)), (3014852, 3015220), '+'), (('G000006925', (3016790, 3036697)), (3021683, 3022051), '+'), (('G000006925', (3038529, 3039299)), (3038748, 3039116), '+'), (('G000006925', (3104758, 3106299)), (3104978, 3105347), '+'), (('G000006925', (3179574, 3180459)), (3179871, 3180239), '-'), (('G000006925', (3180744, 3203996)), (3181762, 3182130), '-'), (('G000006925', (3237769, 3244530)), (3237950, 3238318), '-'), (('G000006925', (3268417, 3281594)), (3273952, 3274320), '-'), (('G000006925', (3322761, 3361739)), (3361149, 3361517), '-'), (('G000006925', (3361966, 3403725)), (3362149, 3362517), '-'), (('G000006925', (3608503, 3611864)), (3608689, 3609057), '-'), (('G000006925', (3612083, 3637192)), (3624452, 3624820), '+'), (('G000006925', (3637330, 3640023)), (3637549, 3637917), '+'), (('G000006925', (3658114, 3676993)), (3665379, 3665747), '+'), (('G000006925', (3693106, 3702974)), (3693289, 3693657), '-'), (('G000006925', (3703179, 3717854)), (3703361, 3703729), '-'), (('G000006925', (3717879, 3736288)), (3725645, 3726013), '-'), (('G000006925', (3736321, 3788106)), (3743241, 3743609), '+'), (('G000006925', (3815875, 3829465)), (3818224, 3818592), '+'), (('G000006925', (3831593, 3850590)), (3834705, 3835073), '+'), (('G000006925', (3858569, 3864367)), (3863780, 3864148), '-'), (('G000006925', (3939882, 3952896)), (3943015, 3943383), '-'), (('G000006925', (4002899, 4012437)), (4003080, 4003448), '-'), (('G000006925', (4069200, 4076620)), (4076035, 4076403), '-'), (('G000006925', (4091607, 4092380)), (4091827, 4092195), '+'), (('G000006925', (4094093, 4116338)), (4102533, 4102901), '+'), (('G000006925', (4174522, 4188276)), (4176744, 4177112), '-'), (('G000006925', (4256570, 4288375)), (4285198, 4285566), '+'), (('G000006925', (4288513, 4315006)), (4314418, 4314786), '-'), (('G000006925', (4362175, 4367702)), (4367153, 4367521), '+'), (('G000006925', (4378758, 4379529)), (4378977, 4379345), '+'), (('G000006925', (4389053, 4390969)), (4389273, 4389641), '+'), (('G000006925', (4392644, 4394283)), (4392830, 4393198), '-'), (('G000006925', (4397536, 4410956)), (4410369, 4410737), '-'), (('G000006925', (4462252, 4478752)), (4462472, 4462840), '+'), (('G000006925', (4478753, 4490457)), (4488610, 4488978), '+'), (('G000006925', (4497401, 4528664)), (4515051, 4515419), '-'), (('G000006925', (4535976, 4543278)), (4540691, 4541059), '-'), (('G000006925', (4557883, 4573403)), (4569904, 4570272), '-'), (('G000006925', (4645852, 4646621)), (4646033, 4646401), '-'), (('G000006925', (4656735, 4658205)), (4656917, 4657285), '-'), (('G000006925', (4785482, 4786253)), (4785703, 4786071), '+'), (('G000299455', (91523, 102755)), (91709, 92077), '-'), (('G000299455', (696017, 696792)), (696238, 696606), '+'), (('G000299455', (941725, 942502)), (941947, 942315), '+'), (('G000299455', (1581872, 1582655)), (1582058, 1582426), '-'), (('G000299455', (2052398, 2064114)), (2062138, 2062506), '-'), (('G000299455', (3182222, 3183821)), (3183268, 3183636), '+'), (('G000299455', (3672841, 3680073)), (3679480, 3679848), '-'), (('G000299455', (4588613, 4589520)), (4588964, 4589332), '+'), (('G000299455', (4592900, 4593681)), (4593088, 4593456), '-'), (('G000299455', (5323356, 5324120)), (5323541, 5323897), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP    39 / 109  f\b\b\r",
      "STEP    40 / 109  f\b\b\r",
      "STEP    41 / 109  f\b\b\r",
      "STEP    42 / 109  f\b\b\r",
      "STEP    43 / 109  f\b\b\r",
      "STEP    44 / 109  f\b\b\r",
      "STEP    45 / 109  f\b\b\r",
      "STEP    46 / 109  f\b\b\r",
      "STEP    47 / 109  f\b\b\r",
      "STEP    48 / 109  f\b\b\r",
      "STEP    49 / 109  f\b\b\r",
      "STEP    50 / 109  f\b\b\r",
      "STEP    51 / 109  f\b\b\r",
      "STEP    52 / 109  f\b\b\r",
      "STEP    53 / 109  f\b\b\r",
      "STEP    54 / 109  f\b\b\r",
      "STEP    55 / 109  f\b\b\r",
      "STEP    56 / 109  f\b\b\r",
      "STEP    57 / 109  f\b\b\r",
      "STEP    58 / 109  f\b\b\r",
      "STEP    59 / 109  f\b\b\r",
      "STEP    60 / 109  f\b\b\r",
      "STEP    61 / 109  f\b\b\r",
      "STEP    62 / 109  f\b\b\r",
      "STEP    63 / 109  f\b\b\r",
      "STEP    64 / 109  f\b\b\r",
      "STEP    65 / 109  f\b\b\r",
      "STEP    66 / 109  f\b\b\r",
      "STEP    67 / 109  f\b\b\r",
      "STEP    68 / 109  f\b\b\r",
      "STEP    69 / 109  f\b\b\r",
      "STEP    70 / 109  f\b\b\r",
      "STEP    71 / 109  f\b\b\r",
      "STEP    72 / 109  f\b\b\r",
      "STEP    73 / 109  f\b\b\r",
      "STEP    74 / 109  f\b\b\r",
      "STEP    75 / 109  f\b\b\r",
      "STEP    76 / 109  f\b\b\r",
      "STEP    77 / 109  f\b\b\r",
      "STEP    78 / 109  f\b\b\r",
      "STEP    79 / 109  f\b\b\r",
      "STEP    80 / 109  f\b\b\r",
      "STEP    81 / 109  f\b\b\r",
      "STEP    82 / 109  f\b\b\r",
      "STEP    83 / 109  f\b\b\r",
      "STEP    84 / 109  f\b\b\r",
      "STEP    85 / 109  f\b\b\r",
      "STEP    86 / 109  f\b\b\r",
      "STEP    87 / 109  f\b\b\r",
      "STEP    88 / 109  f\b\b\r",
      "STEP    89 / 109  f\b\b\r",
      "STEP    90 / 109  f\b\b\r",
      "STEP    91 / 109  f\b\b\r",
      "STEP    92 / 109  f\b\b\r",
      "STEP    93 / 109  f\b\b\r",
      "STEP    94 / 109  f\b\b\r",
      "STEP    95 / 109  f\b\b\r",
      "STEP    96 / 109  f\b\b\n",
      "Reallocating..done. *alloclen = 1740\n",
      "\r",
      "STEP    97 / 109  f\b\b\r",
      "STEP    98 / 109  f\b\b\r",
      "STEP    99 / 109  f\b\b\r",
      "STEP   100 / 109  f\b\b\r",
      "STEP   101 / 109  f\b\b\r",
      "STEP   102 / 109  f\b\b\r",
      "STEP   103 / 109  f\b\b\r",
      "STEP   104 / 109  f\b\b\r",
      "STEP   105 / 109  f\b\b\r",
      "STEP   106 / 109  f\b\b\r",
      "STEP   107 / 109  f\b\b\r",
      "STEP   108 / 109  f\b\b\r",
      "STEP   109 / 109  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 110\r",
      "  100 / 110\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 110\r",
      "   10 / 110\r",
      "   20 / 110\r",
      "   30 / 110\r",
      "   40 / 110\r",
      "   50 / 110\r",
      "   60 / 110\r",
      "   70 / 110\r",
      "   80 / 110\r",
      "   90 / 110\r",
      "  100 / 110\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 109  f\b\b\r",
      "STEP     2 / 109  f\b\b\r",
      "STEP     3 / 109  f\b\b\r",
      "STEP     4 / 109  f\b\b\r",
      "STEP     5 / 109  f\b\b\r",
      "STEP     6 / 109  f\b\b\r",
      "STEP     7 / 109  f\b\b\r",
      "STEP     8 / 109  f\b\b\r",
      "STEP     9 / 109  f\b\b\r",
      "STEP    10 / 109  f\b\b\r",
      "STEP    11 / 109  f\b\b\r",
      "STEP    12 / 109  f\b\b\r",
      "STEP    13 / 109  f\b\b\r",
      "STEP    14 / 109  f\b\b\r",
      "STEP    15 / 109  f\b\b\r",
      "STEP    16 / 109  f\b\b\r",
      "STEP    17 / 109  f\b\b\r",
      "STEP    18 / 109  f\b\b\r",
      "STEP    19 / 109  f\b\b\r",
      "STEP    20 / 109  f\b\b\r",
      "STEP    21 / 109  f\b\b\r",
      "STEP    22 / 109  f\b\b\r",
      "STEP    23 / 109  f\b\b\r",
      "STEP    24 / 109  f\b\b\r",
      "STEP    25 / 109  f\b\b\r",
      "STEP    26 / 109  f\b\b\r",
      "STEP    27 / 109  f\b\b\r",
      "STEP    28 / 109  f\b\b\r",
      "STEP    29 / 109  f\b\b\r",
      "STEP    30 / 109  f\b\b\r",
      "STEP    31 / 109  f\b\b\r",
      "STEP    32 / 109  f\b\b\r",
      "STEP    33 / 109  f\b\b\r",
      "STEP    34 / 109  f\b\b\r",
      "STEP    35 / 109  f\b\b\r",
      "STEP    36 / 109  f\b\b\r",
      "STEP    37 / 109  f\b\b\r",
      "STEP    38 / 109  f\b\b\r",
      "STEP    39 / 109  f\b\b\r",
      "STEP    40 / 109  f\b\b\r",
      "STEP    41 / 109  f\b\b\r",
      "STEP    42 / 109  f\b\b\r",
      "STEP    43 / 109  f\b\b\r",
      "STEP    44 / 109  f\b\b\r",
      "STEP    45 / 109  f\b\b\r",
      "STEP    46 / 109  f\b\b\r",
      "STEP    47 / 109  f\b\b\r",
      "STEP    48 / 109  f\b\b\r",
      "STEP    49 / 109  f\b\b\r",
      "STEP    50 / 109  f\b\b\r",
      "STEP    51 / 109  f\b\b\r",
      "STEP    52 / 109  f\b\b\r",
      "STEP    53 / 109  f\b\b\r",
      "STEP    54 / 109  f\b\b\r",
      "STEP    55 / 109  f\b\b\r",
      "STEP    56 / 109  f\b\b\r",
      "STEP    57 / 109  f\b\b\r",
      "STEP    58 / 109  f\b\b\r",
      "STEP    59 / 109  f\b\b\r",
      "STEP    60 / 109  f\b\b\r",
      "STEP    61 / 109  f\b\b\r",
      "STEP    62 / 109  f\b\b\r",
      "STEP    63 / 109  f\b\b\r",
      "STEP    64 / 109  f\b\b\r",
      "STEP    65 / 109  f\b\b\r",
      "STEP    66 / 109  f\b\b\r",
      "STEP    67 / 109  f\b\b\r",
      "STEP    68 / 109  f\b\b\r",
      "STEP    69 / 109  f\b\b\r",
      "STEP    70 / 109  f\b\b\r",
      "STEP    71 / 109  f\b\b\r",
      "STEP    72 / 109  f\b\b\r",
      "STEP    73 / 109  f\b\b\r",
      "STEP    74 / 109  f\b\b\r",
      "STEP    75 / 109  f\b\b\r",
      "STEP    76 / 109  f\b\b\r",
      "STEP    77 / 109  f\b\b\r",
      "STEP    78 / 109  f\b\b\r",
      "STEP    79 / 109  f\b\b\r",
      "STEP    80 / 109  f\b\b\r",
      "STEP    81 / 109  f\b\b\r",
      "STEP    82 / 109  f\b\b\r",
      "STEP    83 / 109  f\b\b\r",
      "STEP    84 / 109  f\b\b\r",
      "STEP    85 / 109  f\b\b\r",
      "STEP    86 / 109  f\b\b\r",
      "STEP    87 / 109  f\b\b\r",
      "STEP    88 / 109  f\b\b\r",
      "STEP    89 / 109  f\b\b\r",
      "STEP    90 / 109  f\b\b\r",
      "STEP    91 / 109  f\b\b\r",
      "STEP    92 / 109  f\b\b\r",
      "STEP    93 / 109  f\b\b\r",
      "STEP    94 / 109  f\b\b\r",
      "STEP    95 / 109  f\b\b\r",
      "STEP    96 / 109  f\b\b\r",
      "STEP    97 / 109  f\b\b\r",
      "STEP    98 / 109  f\b\b\r",
      "STEP    99 / 109  f\b\b\r",
      "STEP   100 / 109  f\b\b\r",
      "STEP   101 / 109  f\b\b\r",
      "STEP   102 / 109  f\b\b\r",
      "STEP   103 / 109  f\b\b\r",
      "STEP   104 / 109  f\b\b\r",
      "STEP   105 / 109  f\b\b\r",
      "STEP   106 / 109  f\b\b\r",
      "STEP   107 / 109  f\b\b\r",
      "STEP   108 / 109  f\b\b\r",
      "STEP   109 / 109  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|18887|19255|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaaaacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|64491|216702|87122|87490|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|64491|216702|148312|148680|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|266682|267450|266863|267231|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|273344|278315|273563|273931|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|278320|279091|278503|278871|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "agacggtgtaagtga-\n",
      ">G000006925|300111|308544|307956|308324|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "agacggtgtaagtga-\n",
      ">G000006925|378637|384812|378820|379188|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|459861|468373|465936|466304|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgtccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcgatgtaagtgaa\n",
      ">G000006925|468374|487224|478932|479300|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|520755|528026|520936|521304|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|545486|558918|558369|558737|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|622915|623662|623134|623502|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|865887|867389|866840|867208|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacagccgtcctccgtatcctgtcac\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcgatgtaagtgaa\n",
      ">G000006925|984595|985335|984777|985146|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcggcacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagcacccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|985336|1008622|985649|986017|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1025530|1063042|1049791|1050159|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtatcgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1189492|1208094|1197867|1198222|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaa-------------ctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1239481|1240251|1239700|1240069|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagcacccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1290847|1306886|1295292|1295660|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1321493|1385605|1365648|1366003|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaa-------------ctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1398496|1399271|1398717|1399085|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1409877|1410649|1410062|1410430|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1424264|1425034|1424447|1424815|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cacgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1425133|1465118|1444499|1444867|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccaaagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1509064|1578296|1526898|1527266|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1509064|1578296|1533708|1534076|+\n",
      "acttccgtcccagccttgccagatgttgtctcagattcagattatgtcgctcaatgcgct\n",
      "gagtgtaacgcttgctgataacgtgcagctttcccttcaggcgtgattcatacagcggcc\n",
      "agccatccggcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagtttt---------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "----------------\n",
      ">G000006925|1509064|1578296|1542985|1543353|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttggca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1509064|1578296|1568372|1568740|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1509064|1578296|1574330|1574698|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1670306|1692658|1673591|1673959|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1729332|1741706|1736155|1736523|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1758378|1759147|1758597|1758965|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1759148|1776249|1766747|1767115|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacaaccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1759148|1776249|1769213|1769581|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1776250|1792578|1781132|1781500|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|1776250|1792578|1791295|1791663|+\n",
      "acttccgtaccagccttgccagatgttgtctcagattcagattatgtcgctcaatgcgct\n",
      "gagtgtaacgcttgctgataacgtgcagctttcccttcaggcgtgattcatacagcggcc\n",
      "agccatccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagtttt---------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "----------------\n",
      ">G000006925|1793868|1802424|1795310|1795678|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1842508|1869155|1850230|1850598|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1842508|1869155|1856168|1856536|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|1940277|1941051|1940496|1940864|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggcggttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtgaa\n",
      ">G000006925|1976680|1977448|1976861|1977229|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2036705|2044875|2037336|2037704|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2050169|2051841|2051254|2051622|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2088344|2103602|2095259|2095627|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2120115|2138771|2134829|2135197|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2172359|2184266|2178011|2178379|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2187913|2198767|2197068|2197436|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggcggttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000006925|2198768|2199537|2198988|2199356|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2199538|2210687|2205732|2206100|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2211287|2225774|2220852|2221220|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccgcgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaaaacgtggtccacatccgtcctccgtatcttgtcat\n",
      "aggtgtcaaccagccagcgctgacgttttttagca-caaaaatagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcagtgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "gtgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2235234|2236005|2235415|2235783|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2242494|2346249|2287664|2288032|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatgaccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2374481|2472008|2395816|2396184|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2756085|2756853|2756266|2756634|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcccccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2816684|2833035|2825420|2825788|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|2849870|2870340|2863021|2863389|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2957651|2963246|2962874|2963242|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcccccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|2976303|3015537|3014852|3015220|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3016790|3036697|3021683|3022051|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3038529|3039299|3038748|3039116|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3104758|3106299|3104978|3105347|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcggcacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagcacccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3179574|3180459|3179871|3180239|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3180744|3203996|3181762|3182130|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcggcacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacatgatttagcacccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgag-tttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3237769|3244530|3237950|3238318|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3268417|3281594|3273952|3274320|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3322761|3361739|3361149|3361517|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggg\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3361966|3403725|3362149|3362517|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3608503|3611864|3608689|3609057|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------cggtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacagccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3612083|3637192|3624452|3624820|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3637330|3640023|3637549|3637917|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3658114|3676993|3665379|3665747|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3693106|3702974|3693289|3693657|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3703179|3717854|3703361|3703729|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cacgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3717879|3736288|3725645|3726013|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3736321|3788106|3743241|3743609|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3815875|3829465|3818224|3818592|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcaaaaaaaccctca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|3831593|3850590|3834705|3835073|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "agacggtgtaagtgaa\n",
      ">G000006925|3858569|3864367|3863780|3864148|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|3939882|3952896|3943015|3943383|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cacgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4002899|4012437|4003080|4003448|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4069200|4076620|4076035|4076403|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4091607|4092380|4091827|4092195|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4094093|4116338|4102533|4102901|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgtgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtatcgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4174522|4188276|4176744|4177112|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4256570|4288375|4285198|4285566|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4288513|4315006|4314418|4314786|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4362175|4367702|4367153|4367521|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4378758|4379529|4378977|4379345|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4389053|4390969|4389273|4389641|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4392644|4394283|4392830|4393198|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------cggtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttccccgaagacgtgcgccacagccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4397536|4410956|4410369|4410737|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4462252|4478752|4462472|4462840|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4478753|4490457|4488610|4488978|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000006925|4497401|4528664|4515051|4515419|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcgatgtaagtga-\n",
      ">G000006925|4535976|4543278|4540691|4541059|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4557883|4573403|4569904|4570272|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4645852|4646621|4646033|4646401|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaatgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4656735|4658205|4656917|4657285|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtga-\n",
      ">G000006925|4785482|4786253|4785703|4786071|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcgctgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtatcgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000299455|91523|102755|91709|92077|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacggcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggcggttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000299455|696017|696792|696238|696606|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggctgttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtgaa\n",
      ">G000299455|941725|942502|941947|942315|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacatagccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcctggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaatcgtgttgaggccaacgcccataatgcgggcagtag\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtgaa\n",
      ">G000299455|1581872|1582655|1582058|1582426|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggcggttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000299455|2052398|2064114|2062138|2062506|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatcgtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacatagccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaatcgtgttgaggccaacgcccataatgcgggcagttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000299455|3182222|3183821|3183268|3183636|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatctataccacgacctcaaaggccgacagcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccactgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcttaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000299455|3672841|3680073|3679480|3679848|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggctgttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000299455|4588613|4589520|4588964|4589332|+\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "-------cgtcatccataccacgacctcaaaggccgacatcaggctcagaagacgctcca\n",
      "gtgtggccagagtgcgttcaccgaagacgtgcgccacaaccgtcctccgtatcctgtcat\n",
      "acgcgtaaaacagccagcgctgacgtgatttagca-ccgacgtagccccaatgttcgtcc\n",
      "atttcagcgcagacaatcacatcactgcccggttgtatgtgcgaggttaccgactgcggc\n",
      "ctgagttttttaagtgacgtaaaaccgtgttgaggccaacgcccataatgcgtgcactgg\n",
      "cgcgacatccgacgccattcatggccatatcaatgattttctggtgcgtaccgggctgag\n",
      "aggcggtgtaagtgaa\n",
      ">G000299455|4592900|4593681|4593088|4593456|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttagcc-ccgacgtatccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgagttttttaaatggcggaaaatcgtgttgaggccaacgcccataatgcgggctgttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      ">G000299455|5323356|5324120|5323541|5323897|-\n",
      "------------------------------------------------------------\n",
      "------------------------------------------------------------\n",
      "------ccgtcatccatatcaccacgtcaaagggtgacagcaggctcataagacgcccca\n",
      "gcgtcgccatagtgcgttcaccgaatacgtgcgcaacaaccgtcttccggagcctgtcat\n",
      "acgcgtaaaacagccagcgctggcgcgatttag------------ccccactgttcgtcc\n",
      "atttccgcgcagacgatgacgtcactgcccggctgtatgcgcgaggttaccgactgcggc\n",
      "ctgag-tttttaagtgacgtaaaatcgtgttgaggccaacgcccataatgcgggcagttg\n",
      "cccggcatccaacgccattcatggccatatcaatgattttctggtgcgtaccgggttgag\n",
      "aagcggtgtaagtga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4378186, 4378731)), (4378447, 4378557), '-'), (('G000299455', (2647373, 2647918)), (2647547, 2647657), '+'), (('G000299455', (3362286, 3384700)), (3383756, 3383866), '-')]\n",
      ">G000006925|4378186|4378731|4378447|4378557|-\n",
      "ggtgatggttatacccagggcgatattttcgatggtattaactttcgcggcgcacaattg\n",
      "gcctcagatgacaatatgttacccgatagccaaagaggatttgccccggt-\n",
      ">G000299455|2647373|2647918|2647547|2647657|+\n",
      "-gtgatagttataccgatggcgatatttttgatagtgtgaactttcgtggcctcaaaata\n",
      "aattcaacagaagcgatgttgcccgatagccaacatggtttcgctccggtg\n",
      ">G000299455|3362286|3384700|3383756|3383866|-\n",
      "ggtgatgattatactgcaggcgattttttcgatagtgtcagctttcgtggtgtgcagctg\n",
      "gcgtcagatgacaacatgctgccagacagcctgaaagggtttgcgcctgt-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3094350, 3094517), '-'), (('G000299455', (50607, 52894)), (52396, 52563), '+'), (('G000299455', (857924, 871846)), (860995, 861162), '+'), (('G000299455', (1980069, 1981046)), (1980549, 1980716), '+'), (('G000299455', (3143255, 3145541)), (3145044, 3145211), '+'), (('G000299455', (5326851, 5327018)), (5326851, 5327018), '+')]\n",
      ">G000006925|3083675|3099254|3094350|3094517|-\n",
      "catctggtagctggaggtaccgtcatgggagttgagcagaataatttcagggacatgttc\n",
      "tccgtttatctctccggcccgccgcagacgcagcatgtgttttgtgtatccccggcggcc\n",
      "cgggtcgcgcacacgggtctggcaggcgaaaaagggctggaagcctt-\n",
      ">G000299455|50607|52894|52396|52563|+\n",
      "-atctggtagctggaggtaccgtcatgggagttgagcagaatgatttcagggacatgttc\n",
      "tccgtttatctctccggcccgccgcagacgcagcatatgttttgtgtatccccggcggcc\n",
      "cgggtcgcgcacacgggtctggcaggcgaagaatggctgaaagccttc\n",
      ">G000299455|857924|871846|860995|861162|+\n",
      "-atctggtagctggaggtaccgtcatgagagttgagcagaataatttcagggaaatgttc\n",
      "ttcgtttatctctccggcccgccgcagacgcagcatgtgttttgtgtatccccggcggcc\n",
      "cgggtcgcgcacacgggtctggcaggcgaagaatggctgaaagccttc\n",
      ">G000299455|1980069|1981046|1980549|1980716|+\n",
      "-atctggtagctggaggtaccgtcatgagagttgagcagaataatttcagggacatgttc\n",
      "tccgtttatctctccggcccgccgcagacgcagcatgtgttttgtgtatccccggcggcc\n",
      "cgggtcgcgcacacgggtctggcaggcgaagaacggctgaaagccttc\n",
      ">G000299455|3143255|3145541|3145044|3145211|+\n",
      "-atctggtagctggaggtaccgtcatgagagttgagcagaataatttcagggacatgttc\n",
      "tccgtttatctctccggcccgccgcagacgcagcatgtgttttgtgtatccccggcggcc\n",
      "cgggtcgcgcacacgggtctggcaggcgaaaaagggctgaaaaccttc\n",
      ">G000299455|5326851|5327018|5326851|5327018|+\n",
      "-atctgataactggatgaaccatcatgtgaattaagcaggataatttcaggaacttcttt\n",
      "cccgttaatatacccttccctgcgaagacgtaacatatgtttactgtattcgcggcgtcc\n",
      "caaatcatgaacccgactctgacaggcaaagaacggctggaaaccttc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2139144, 2139186), '-'), (('G000299455', (1911336, 1921536)), (1921194, 1921236), '+'), (('G000299455', (3326617, 3362069)), (3341463, 3341505), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2138845|2144864|2139144|2139186|-\n",
      "gcggtcaggtgccgccaaatccttctgaactgttaatgagcg-\n",
      ">G000299455|1911336|1921536|1921194|1921236|+\n",
      "gcggtcaggtgccgccaaatccttctgagctgttgatgagcg-\n",
      ">G000299455|3326617|3362069|3341463|3341505|+\n",
      "-cggtcaggtgccacctaacccgtctgaactgctgatgcgcga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4659810, 4664323)), (4664040, 4664323), '+'), (('G000299455', (5328484, 5333124)), (5328484, 5328767), '-'), (('G000299455', (5344024, 5346038)), (5345755, 5346038), '+')]\n",
      ">G000006925|4659810|4664323|4664040|4664323|+\n",
      "-tatagaaacatcaacctaattttgacaaagattaattacataggatgtagatggagcag\n",
      "tccgagattctactaatatcaattattaagcatctgttttttagttggtgtaatataatt\n",
      "tttattaaggtaagcggaaaaacaaccaaaaaaaacgcaagatgttatttttttatcatt\n",
      "ttgagccagtttggctggggggatactcttctcttaagcttcttgctcaactttaaaaat\n",
      "agttatgccagttcggtatcgataaggggcagtttggatatcga\n",
      ">G000299455|5328484|5333124|5328484|5328767|-\n",
      "atatagaaacatcaacctaattttgacaaagattaattacataggatgtggatggagcag\n",
      "tccgagattctactaatatcaattactaagcatctgttttttagttggtgtaatataatt\n",
      "tttattaaggtaagcggaaaaacaaccaaaaaaaacgcaagatgttatttttttatcatt\n",
      "ttgagctagtttggctggggggatgctcttctcttaagcttcttgctcaactttaaaaat\n",
      "agttatgccagttcggtatcgataaggggcagtttggatatcg-\n",
      ">G000299455|5344024|5346038|5345755|5346038|+\n",
      "-tatagaaacatcaacctaattttgacaaagattaattacattggatgtggatggagcag\n",
      "tccgagattctactaatatcaattactaagcatctgttttttagttggtgtaatataatt\n",
      "tttattaaggtaagcggaaaaacaaccaaaaaaaacgcaagatgttatttttttatcatt\n",
      "ttgagctagtttggttgggggtatgctcctcttttaagtttcttgctcaactttaaaaat\n",
      "agttatgctagttcagtaccgataggagtcagtttagatatcga\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3095053, 3095400), '-'), (('G000299455', (50607, 52894)), (51517, 51860), '+'), (('G000299455', (857924, 871846)), (860153, 860497), '+'), (('G000299455', (3143255, 3145541)), (3144165, 3144508), '+'), (('G000299455', (4771847, 4785261)), (4780689, 4781036), '-')]\n",
      ">G000006925|3083675|3099254|3095053|3095400|-\n",
      "agttgttgcaggtcagtctccggtgcacaggccgaagaggtggtgttctgagacaatgtt\n",
      "ttcatgtatgaaatccttgtaatgtgaataaatgagagggagtaaaggtcgggagctgac\n",
      "aacatcacgttcatcactgaccgtgatgacccggtaatatcagggatgaatgatattcgg\n",
      "ctgaaattccggacagccgtacgcgtaaccgttccggtcacagacggatggccagatgcc\n",
      "attgcctgtatactcagtctgcacagactgatttctccgggacggtaaagcagacgggta\n",
      "caccaggcgcacaggtcatcatcagaaatggaacgacaccccggcag-\n",
      ">G000299455|50607|52894|51517|51860|+\n",
      "--ttgttgcaggccagtctccggtgcacaggccgaagcggtggtgttctgagacaatgtt\n",
      "ttcatgtatgaaatccttgtaatgtgaataaatgagagggagtaaaggtcgggagctggc\n",
      "aacatcacgttcatcactgaccgtga-gacccggtaatatcagggatgaatgatattcgg\n",
      "ctgaaattccggacagctgtacgcgtaaccgttccggtcacagacggatggtcagatgcc\n",
      "attgcctg--tactcagtctgcacaggctgagctctccggaacggtaaagcaggcgggta\n",
      "caccaggtgcacaggtcatcatcagaaatggaacgacaccccggcagc\n",
      ">G000299455|857924|871846|860153|860497|+\n",
      "-gttgttgcaggtcagtctccggtgctgagaccgaagaggtggtgttctgagacaatgtt\n",
      "ttcatatgtgaaatccttgtagtgtgaataaatgcgaggaagtaaaggtcgggggctgat\n",
      "aacatcacgttcatcactgaccgtga-gacccggtaatatcagggatgaatgatattcgg\n",
      "ctgaaattccggacagccgtacgcgtaaccgttccggtcacagacggatggcctgatgcc\n",
      "attgcctg--tactcagtctgcacagactgatttctccgggacggtaaagcagacgggta\n",
      "taccaggcgcacaggtcatcatcagaaatggaacgacatcccggcagc\n",
      ">G000299455|3143255|3145541|3144165|3144508|+\n",
      "--ttgttgcaggccagtctccggtgcacaggccgaagcggtggtgttctgagacaatgtt\n",
      "ttcatgtgtgaaatccttgtaatgtgaataaatgagagggagtaaaggtcgggagctggc\n",
      "aacatcacgttcatcactgaccgtga-gacccggtaatatcagggatgaatgatattcgg\n",
      "ctgaaattccggacagccgtacgcgtaaccgttccggtcacagacggatggccagatgcc\n",
      "attgcctg--tactcagtctgcacagactgatttctccgggacggtaaagcagacgggta\n",
      "caccaggcgcacaggtcatcatcagaaatggaacgacaccccggcagc\n",
      ">G000299455|4771847|4785261|4780689|4781036|-\n",
      "agttgttgcaggtcagtctccggtgcacaggccgaagaggtggtgttctgagacaatgtt\n",
      "ttcatgtatgaaatccttgtaatgtgaataaatgagagggagtaaaggtcgggagctggc\n",
      "aacatcacgttcatcactgaccgtgatgacccggtaatatcagggatgaatgatattcgg\n",
      "ctgaaattccggacagccgtacgcgtaaccgttccggtcacagacggatggccagatgcc\n",
      "attgcctgtatactcagtctgcacagactgatttctccgggacggtaaagcagacgggta\n",
      "caccaggcgcacaggtcatcatcagaaatggaacgacaccccggcag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (520755, 528026)), (523182, 523231), '+'), (('G000299455', (3786605, 3808525)), (3806345, 3806394), '-'), (('G000299455', (4235244, 4293324)), (4280174, 4280223), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|520755|528026|523182|523231|+\n",
      "-cctgggccagcaccatcgcgatccacgctcgctggcgttgtccgccaga\n",
      ">G000299455|3786605|3808525|3806345|3806394|-\n",
      "tcctgggccagcaccatcgcgatccacgctcgctggcgttgtccgccag-\n",
      ">G000299455|4235244|4293324|4280174|4280223|+\n",
      "-cctgcgccaccagcatggcgatccacgcccgctgacgttcgccgccaga\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3096609, 3096664), '-'), (('G000299455', (857924, 871846)), (858889, 858944), '+'), (('G000299455', (1978312, 1978956)), (1978687, 1978736), '+'), (('G000299455', (3142569, 3143200)), (3142929, 3142980), '+'), (('G000299455', (4771847, 4785261)), (4782245, 4782300), '-')]\n",
      ">G000006925|3083675|3099254|3096609|3096664|-\n",
      "ttaagaatggtccggtagtggcgataattctgctcatcagaattcccttccagac--\n",
      ">G000299455|857924|871846|858889|858944|+\n",
      "-taagaatggtccggtagtggcgataattctgctcatcagaattcccttccagact-\n",
      ">G000299455|1978312|1978956|1978687|1978736|+\n",
      "--aagaatggtccggtagtggcgataattctgctcatcagaattccc------actc\n",
      ">G000299455|3142569|3143200|3142929|3142980|+\n",
      "--aagaatggtccggtggtggcgataattctgctcatcagaatgccct----gactc\n",
      ">G000299455|4771847|4785261|4782245|4782300|-\n",
      "ttaagaatggtccggtagtggcgataattctgctcatcagaattcccttccagac--\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3736321, 3788106)), (3750250, 3750270), '-'), (('G000006925', (4535976, 4543278)), (4542086, 4542106), '+'), (('G000299455', (137973, 154383)), (149540, 149560), '+'), (('G000299455', (830050, 839519)), (838053, 838073), '-')]\n",
      ">G000006925|3736321|3788106|3750250|3750270|-\n",
      "ttatagacgaacactgctgc-\n",
      ">G000006925|4535976|4543278|4542086|4542106|+\n",
      "-tatagacgaacactgctgca\n",
      ">G000299455|137973|154383|149540|149560|+\n",
      "-tatagacgaacactgctgca\n",
      ">G000299455|830050|839519|838053|838073|-\n",
      "ttatagaggaacaccgccgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2560406, 2561162)), (2560992, 2561159), '+'), (('G000006925', (4629818, 4630996)), (4629818, 4629985), '-'), (('G000299455', (5017408, 5018168)), (5017412, 5017579), '-'), (('G000299455', (5280803, 5281043)), (5280806, 5280973), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1336\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1336\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2560406|2561162|2560992|2561159|+\n",
      "--aacacacggtggcagtcacggcagcgaagtctgtcacggcctttagggttctgaccat\n",
      "ggcggtaaacctgagctgactgacaacggggacaatgaatattaacgctggtcatgagat\n",
      "aacctcaaaagcccgtattat-acatcagattcaactaattagaggcatc\n",
      ">G000006925|4629818|4630996|4629818|4629985|-\n",
      "gaaacacccggtgacagtcacggcaacggaatcgatcatgtccttttgggttctgcccgt\n",
      "gacggtaaacctgaacggactggcaacgggggcaatgaaagttaactgtgaccatgaaag\n",
      "aacc-taaaagcgggtattataacccctaattcaactaattggaggca--\n",
      ">G000299455|5017408|5018168|5017412|5017579|-\n",
      "gaaatacccggtgacagtcacggccgcggaatcgatcatgtctttttgggttctgcccgt\n",
      "gacggtaaacctgaacggactggcaacgggggcaatgaaggttaactgtgaccatgaaac\n",
      "aaccttaaaagcgagtattat-atccttaattcaactaattggaggca--\n",
      ">G000299455|5280803|5281043|5280806|5280973|-\n",
      "-aaacacacggtggcagtcacggtagcgaaatctgtcacggcctttagggttctgaccat\n",
      "ggcggtaaacctgagctgactgacaacggggacaatgaatattaacgctggacatgagat\n",
      "aacctcaaaagcccgtattat-acatcagattcaactaattagaggcat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4377056, 4377333)), (4377080, 4377132), '+'), (('G000299455', (2648771, 2649024)), (2648972, 2649024), '-'), (('G000299455', (3854619, 3854696)), (3854643, 3854695), '+')]\n",
      ">G000006925|4377056|4377333|4377080|4377132|+\n",
      "-tttcccgatattcagtggcataaggcagcacggcataaccacgccagtcggt\n",
      ">G000299455|2648771|2649024|2648972|2649024|-\n",
      "ttttctctatattctgtcgcaaatggtaatatggcatagccacgccagtcgg-\n",
      ">G000299455|3854619|3854696|3854643|3854695|+\n",
      "-tataccgataaaccgtggcatacggcatcaccgcatagccgcgccagtcggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2728788, 2734132)), (2732351, 2732394), '+'), (('G000299455', (1302091, 1327808)), (1326026, 1326069), '+'), (('G000299455', (4200530, 4200573)), (4200530, 4200573), '+')]\n",
      ">G000006925|2728788|2734132|2732351|2732394|+\n",
      "tcatcctgtttatcgacgaattacataccatggtcggcgcggg\n",
      ">G000299455|1302091|1327808|1326026|1326069|+\n",
      "tcatcctgtttatcgacgaattacataccatggtcggcgcggg\n",
      ">G000299455|4200530|4200573|4200530|4200573|+\n",
      "tcattctgtttatcgacgaagcacataccctgattggcgcggg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1956633, 1961076)), (1959199, 1959430), '-'), (('G000299455', (2133804, 2148224)), (2135450, 2135681), '+'), (('G000299455', (4166042, 4170922)), (4169225, 4169456), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1956633|1961076|1959199|1959430|-\n",
      "tcaactttatggtcattaccaaaggtgccgggcgtatagcagaagtgggtgcgcgctttg\n",
      "ttctcgatggtatgccgggtaagcagatggcgattgacgccgaccttaacgctggattga\n",
      "ttggtgaagatgaggcgaaaaaacgccgctccgaagtgactcaggaagccgatttttacg\n",
      "gctcaatggacggggcaagtaagtttgttcgcggcgatgccattgccggga-\n",
      ">G000299455|2133804|2148224|2135450|2135681|+\n",
      "tcaactttatggtcattaccaaaggtgccgggcgtatcgcagaagtgggtgcgcgctttg\n",
      "ttctcgatggtatgccgggtaagcagatggcgattgacgccgaccttaacgccggattga\n",
      "ttggtgaagatgaggcgaaaaaacgccgctccgaagtgactcaggaagccgatttttacg\n",
      "gctcaatggacggggcaagtaagtttgttcgcggcgatgccatcgccggga-\n",
      ">G000299455|4166042|4170922|4169225|4169456|+\n",
      "-caactttatcgtcgtcaccaaaggggccgagcgtatttccgaggtttctgcccgcttta\n",
      "ccctggacgcgatgcccggcaagcagatggcgattgacgccgatcttaacgccgggttga\n",
      "tcaaccaggcgcaggcgcaaacccggcgtaaagatgttgccagcgaggccgatttctacg\n",
      "gcgcaatggacggggcgtcgaagtttgtgcgcggggacgccatcgccgggat\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3082575, 3083578)), (3082873, 3082915), '+'), (('G000299455', (61438, 61772)), (61466, 61508), '-'), (('G000299455', (871858, 872859)), (872521, 872563), '-'), (('G000299455', (1994581, 1994887)), (1994581, 1994623), '-'), (('G000299455', (4539188, 4539522)), (4539216, 4539258), '-')]\n",
      ">G000006925|3082575|3083578|3082873|3082915|+\n",
      "-agcctggctgcaggcgcgtattgcacagtcccgtccgtaatt\n",
      ">G000299455|61438|61772|61466|61508|-\n",
      "aagcctggctacaggcgcgtattgcacagtcccgtccgtaat-\n",
      ">G000299455|871858|872859|872521|872563|-\n",
      "aagcctggttgcaggcgcgtattgcacaatcccgtccgtaat-\n",
      ">G000299455|1994581|1994887|1994581|1994623|-\n",
      "aagcctggctgcaggcgcgtattgcacagtcccgtccgtaat-\n",
      ">G000299455|4539188|4539522|4539216|4539258|-\n",
      "aagcctggctgcaggcgcgtattgcacagccccgcccgtaat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1, 24671)), (12576, 12631), '+'), (('G000006925', (2641574, 2668966)), (2643458, 2643513), '-'), (('G000299455', (4431865, 4470166)), (4437902, 4437957), '-')]\n",
      ">G000006925|1|24671|12576|12631|+\n",
      "-ttatcaccgtaccggcatactttaacgatgctcagcgtcaggcaaccaaagacgc\n",
      ">G000006925|2641574|2668966|2643458|2643513|-\n",
      "gttatcaccgttccggcgtactttgacgatgcccagcgtcagggcaccaaagacg-\n",
      ">G000299455|4431865|4470166|4437902|4437957|-\n",
      "gttatcaccgtaccggcatactttaacgatgctcagcgtcaggcaaccaaagacg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3060436, 3060499), '+'), (('G000006925', (3829730, 3830042)), (3829730, 3829793), '+'), (('G000299455', (4713031, 4748324)), (4743677, 4743740), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3060436|3060499|+\n",
      "atgaataaaatttattcactgaaatatagtcatattacaggtggattagttgctgtttct\n",
      "gaa\n",
      ">G000006925|3829730|3830042|3829730|3829793|+\n",
      "atgaataaaatatactcccttaaatatagtgctgccactggcggactcattgctgtttct\n",
      "gaa\n",
      ">G000299455|4713031|4748324|4743677|4743740|+\n",
      "atgaataaaatttattcactgaaatatagtcatattacaggtggattagttgctgtttct\n",
      "gaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (721457, 721684)), (721457, 721501), '-'), (('G000006925', (2060397, 2060639)), (2060595, 2060639), '+'), (('G000299455', (3608183, 3608425)), (3608381, 3608425), '+')]\n",
      ">G000006925|721457|721684|721457|721501|-\n",
      "caatgcccgttgaaattttgtccatggatttcatagcctcacct-\n",
      ">G000006925|2060397|2060639|2060595|2060639|+\n",
      "-aatgcccgttgaaattttgtccatggatttcatagcctcacctc\n",
      ">G000299455|3608183|3608425|3608381|3608425|+\n",
      "-gacacccgttgttaacttatccatggatttcataaccccacctc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2172359, 2184266)), (2181289, 2181350), '-'), (('G000299455', (665003, 670357)), (668582, 668643), '-'), (('G000299455', (1878030, 1889155)), (1880943, 1881004), '+')]\n",
      ">G000006925|2172359|2184266|2181289|2181350|-\n",
      "attgccatcgtggtgcagccgggtgtggaatttgatcacagcaatattatccattatcag\n",
      "c-\n",
      ">G000299455|665003|670357|668582|668643|-\n",
      "attgccctggtggttcaacccggcgtcgaattcgatcacaccaacgttattgattatcag\n",
      "c-\n",
      ">G000299455|1878030|1889155|1880943|1881004|+\n",
      "-ttgccatcgtggtgcagccgggtgtggaatttgatcacagcaatattatccattatcag\n",
      "cc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (827107, 836680)), (829276, 829347), '-'), (('G000299455', (3480865, 3520974)), (3501403, 3501474), '+'), (('G000299455', (4517139, 4517210)), (4517139, 4517210), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|827107|836680|829276|829347|-\n",
      "caccggacggcgaccaatacgatccgacagcggccccaggagccattgtaaaaacatccc\n",
      "gcccgccagat\n",
      ">G000299455|3480865|3520974|3501403|3501474|+\n",
      "caccggacggcgaccaatacgatccgacagcggccccaggagccattgtaaaaacatccc\n",
      "gcccgccagat\n",
      ">G000299455|4517139|4517210|4517139|4517210|-\n",
      "caccggcctgcggccaatcctgtcggaaagcggccccagcagccactgtaacgccatacc\n",
      "gcccgcgagat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (2560406, 2561162)), (2560931, 2560992), '+'), (('G000006925', (4629818, 4630996)), (4629985, 4630046), '-'), (('G000006925', (4668315, 4668899)), (4668315, 4668376), '-'), (('G000006925', (4779995, 4780327)), (4780266, 4780327), '+'), (('G000299455', (5017408, 5018168)), (5017579, 5017640), '-'), (('G000299455', (5280803, 5281043)), (5280973, 5281034), '-')]\n",
      ">G000006925|2560406|2561162|2560931|2560992|+\n",
      "--catttcagttatcagctctttcatacccggcttacgtgcttgataagtataagtgaac\n",
      "tga\n",
      ">G000006925|4629818|4630996|4629985|4630046|-\n",
      "gccatttcagtgatttggtttttaactcccggtttacgggcctcgtaacagtaagtgagc\n",
      "t--\n",
      ">G000006925|4668315|4668899|4668315|4668376|-\n",
      "gccatttcaacgatctgctccttaacgccgagttttctggcttcataagagtaagtgagc\n",
      "t--\n",
      ">G000006925|4779995|4780327|4780266|4780327|+\n",
      "-ccatttcaacgatctgctccttaacgccgggttttctggcttcataagagtaagtgagc\n",
      "tg-\n",
      ">G000299455|5017408|5018168|5017579|5017640|-\n",
      "gtcatttcagcgatttggtctttaactcccggtttacggggctcgtaacagtaactgagc\n",
      "t--\n",
      ">G000299455|5280803|5281043|5280973|5281034|-\n",
      "-ccatttcagttatcagctctttcatacccggcttacgtgcttgataagtataagtgagc\n",
      "tg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1637263, 1657511)), (1638395, 1638530), '-'), (('G000299455', (2505935, 2525742)), (2524909, 2525044), '+'), (('G000299455', (3440781, 3470001)), (3440948, 3441083), '+')]\n",
      ">G000006925|1637263|1657511|1638395|1638530|-\n",
      "ccgccaacggcaaaaaatatggaaccggcagcaatctcgttactcagaccagaagctccg\n",
      "attcgattcagcgagttaaacgctcgcagtggcgatcccaggtgcatgacagaggcaata\n",
      "aacccgatgcccatc-\n",
      ">G000299455|2505935|2525742|2524909|2525044|+\n",
      "-cgccaacggcaaaaaatatggaaccggcagcaatctcgttactcagaccagaagctccg\n",
      "attcgattcagcgagttaaacgctcgcagtggcgatcccaggtgcatgacagaggcaata\n",
      "aacccgatgcccatca\n",
      ">G000299455|3440781|3470001|3440948|3441083|+\n",
      "-cgcctacggcaaaaaagatcgaaccgctggcgatttcgttactgagtgctgaagcccct\n",
      "acccggttgagcgagttaaaagcgcgcattggtgaaccaagatggagcatagaggcgata\n",
      "aagccaatgcccatca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (905848, 907049)), (906275, 906310), '+'), (('G000006925', (4827189, 4827308)), (4827189, 4827223), '-'), (('G000299455', (2927401, 2931337)), (2930563, 2930598), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1124\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1124\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|905848|907049|906275|906310|+\n",
      "gccaatatgaaatcaatcagaaaggtgccccataa\n",
      ">G000006925|4827189|4827308|4827189|4827223|-\n",
      "gccaatatgaaatcaatcagaaaagcgttccata-\n",
      ">G000299455|2927401|2931337|2930563|2930598|+\n",
      "gccaatatgaaatcaatcagaaaggtgatccataa\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3095400, 3096309), '-'), (('G000299455', (50607, 52894)), (50607, 51517), '+'), (('G000299455', (857924, 871846)), (859244, 860153), '+'), (('G000299455', (1979011, 1979920)), (1979011, 1979920), '+'), (('G000299455', (3143255, 3145541)), (3143255, 3144165), '+'), (('G000299455', (4771847, 4785261)), (4781036, 4781945), '-')]\n",
      ">G000006925|3083675|3099254|3095400|3096309|-\n",
      "gtgacgggttaaagcagaccgtgttccgcaaaggagaaaacctggctgccaccgactatc\n",
      "agatggtccggtacccggatatcaaccagggccagtgcctgtaccagacgctcagtgata\n",
      "aggcggtctgccttgctgggggggacttcaccggacgggtgattgtgcgccagcaccacg\n",
      "gcagcggcattgtggtacagggcgcgtttaatcacttcccggggatggacttccgtacgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagctgattctggttgttcagatac\n",
      "agcacccggaactcttcacgctccagtcccgccatgttcagaatcagccattcccgtgcc\n",
      "gcatgcgtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgcagaatgaggctgcgctctctggttgtcatttctccgggcagaaaggaaagctgc\n",
      "tgcattgtccctgccaccattcagtcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcgtaatcccgcagccggtaataatggaccgtcatggcataacactccgtacga\n",
      "caggcatgatgactgtacgtcatcagacaggcagcaatgccggcagcttccggactcatt\n",
      "tcagcgcggttaccgttcatggcattgaacagtacccatgtttcgtcatcatcgtcatcc\n",
      "ggttcgggtgccataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagtcttcgcacagacggtccatccagccgaagacacggggctccagggtcacccac\n",
      "tgtggaatgaggccaaagtgctgcggccagaagctgatgcgctgttcatcagggacgagg\n",
      "gtggcaacc--\n",
      ">G000299455|50607|52894|50607|51517|+\n",
      "-tgacgggttaaagcagaccatgttcggcaaaggagaaaacctggttgccaccgactatc\n",
      "agatggtccggcacccggatatccaccaggcccagagcctgtaccagacgttcggtgata\n",
      "agccggtctgccttactgggtgtgacttcaccggacgggtgattgtgtgccagcaccaca\n",
      "gcagcggcattgtgatacagggcgcgtttaatcacttcccggggatggacttccgtgcgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagctgattctggttgttcagatac\n",
      "agcacccggaactcttcacgctccagtcccgccatgttcagaatcagccattcccgtgcc\n",
      "gcacgggtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgttgaatgaggctgcgctcgccgggcgtcatctctccgggcagaaaggaaagctgt\n",
      "tgcattgtgcttctctccattcagtcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcgtaatcccgcagccggtaataatggaccgtcatggcataacactccgtacga\n",
      "caggcatgatgactgtacgtcatcagacaggcagcaatgccggcggcttccgggctcatt\n",
      "tcagcgcggttaccgttcatggcattgaacagtacccatgtttcgtcatcatcgtcatcc\n",
      "ggttcgggtgccataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagttttcgcaaagacggtccatccagccgaagacacggggttccagggtgacccac\n",
      "tgtggaatgaggccaaaatgctgcggccagaagctgatgcgctgttcatcagggacaatg\n",
      "gtggcaaccag\n",
      ">G000299455|857924|871846|859244|860153|+\n",
      "-tgacgggttaaagcagaccgtgttccgcaaaggagaaaacctggctgccaccgactatc\n",
      "agatggtccggtacccggatatcaaccagggccagtgcctgtaccagacgctcagtgata\n",
      "aggcggtctgccttgctgggggggacttcaccggacgggtgattgtgcgccagcaccacg\n",
      "gcagcggcattgtggtacagggcgcgtttaatcacttcccggggatggacttccgtacgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagctgattctggttgttcagatac\n",
      "agcacccggaactcttcacgctccagtcccgccatgttcagaatcagccattcccgtgcc\n",
      "gcatgcgtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgctgaatgaggctgcgctcgccgggtgtcatctctccgggcagaaaggaaagttgt\n",
      "tgcattgtgcttctctccattcaatcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcgtaatcccgcagccggtaataatggaccgtcatggcataacactccgtacga\n",
      "caggcatgatgactgtacgtcatcagacaggcggcaatgccggcggcttccgggctcatt\n",
      "tcagcgcggttaccgttcatggcattgaacagtacccagttttcgccatcatcgtcatcc\n",
      "ggttcgggtgccataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagttttcgcaaagacggtccatccagccgaagatacggggctccagggtcacccac\n",
      "tgtggaatgaggccaaagtgctgcggccagaaactgatacgttgttcatcagggacgagg\n",
      "ctggcaacca-\n",
      ">G000299455|1979011|1979920|1979011|1979920|+\n",
      "-tgaagggttaaagcagaccatgttcggcaaaggagaaaacctggttgccaccgactatc\n",
      "agatggtccggcacccggatatccaccaggcccagtgcctgtaccagacgttcggtgata\n",
      "agccggtctgccttactgggtgtgacttcaccggacgggtgattgtgtgccagtaccacg\n",
      "gcagcggcattgtggtacagggcgcgtttaatcacttcccggggatggacttccgtgcgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagctgattctggttattcagatac\n",
      "agtacccggaattcttcacgctccagtcccgccatgttcagaatcagccattcccgtgct\n",
      "gcacgggtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgttgaatgaggctgcgctcgccgggcgtcatctctccgggcagaaaggaaagttgt\n",
      "tgcattgtgcttctctccattcagtcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcgtaatcccgcaaccggtaataatggaccgtcatggcataattctccgtacga\n",
      "caggcatgatgactgtacgtcatcagacaggcggcaataccggcggcttccgggctcatt\n",
      "tctgcgcggttaccgttcatggcattgaacagtacccatgtttcgtcatcatcgtcatcc\n",
      "ggttcgggtgccataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagtcttcgcacagacggtccatccagccgaagacacggggctccagggtcacccac\n",
      "tgtggaatgaggccaaaatgctgcggccagaagctgatgcgctgttcatcagggacgagg\n",
      "ctggcaacca-\n",
      ">G000299455|3143255|3145541|3143255|3144165|+\n",
      "-tgacgggttaaagcagaccatgttcggcaaaagagaaaacctggctgccaccgactatc\n",
      "agatggtccggcacccggatatccaccaggcccagtgcctgtaccagacgttccgtaata\n",
      "agccggtctgccttactgggtgtgacttcaccggacgggtgattgtgtgccagcaccacg\n",
      "gcagcggcattgtggtacagggcgcgtttaatcacttcccggggatggacttccgtgcgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagcttattctggttgttcagatac\n",
      "agcacccggaattcttcccgctccagtcccgccatgttcagaatcagccattcccgtgct\n",
      "gcacgggtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgctgaatgagactgcgctcgccgggcgtcatctctccgggcagaaaggaaagttgt\n",
      "tgcattgtccctgcctccattcagtcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcataatcccgcagctggtaataatgaaccgtcatggcataattctctgtacga\n",
      "caggcgtgatggctgtacgccatcagacaggcggcaatgccggcggcttccgggctcatt\n",
      "tcagcgcggttaccgttcatggcattgaacagtacccatgtttcgtcatcatcatcatcc\n",
      "ggttcgggggccataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagttttcgcaaagacggtccatccagccgaagacacggggttccagggtgacccac\n",
      "tgtggaatgaggccaaaatgctgcggccagaagctgatgcgctgttcatcagggactatg\n",
      "gtggcaaccag\n",
      ">G000299455|4771847|4785261|4781036|4781945|-\n",
      "gtgacgggttaaagcagaccgtgttccgcaaaggagaaaacctggctgccaccgactatc\n",
      "agatggtccggtacccggatatcaaccagggccagtgcctgtaccagacgctcagtgata\n",
      "aggcggtctgccttgctgggggggacttcaccggacgggtgattgtgcgccagcaccacg\n",
      "gcagcggcattgtggtacagggcgcgtttaatcacttcccggggatggacttccgtacgg\n",
      "ttgatggtgccggtgaagagggtttcaccggcaatcagctgattctggttgttcagatac\n",
      "agcacccggaactcttcacgctccagtcccgccatgttcagaatcagccattcccgtgcc\n",
      "gcatgcgtggaggtgaaggccacgccgggttcatgaagatggcggtccagggttttcagg\n",
      "gcccgcagaatgaggctgcgctctctggttgtcatttctccgggcagaaaggaaagctgc\n",
      "tgcattgtccctgccaccattcagtcgatgatgcgcataatggcgctgcattccggatgc\n",
      "tgcagggcgtaatcccgcagccggtaataatggaccgtcatggcataacactccgtacga\n",
      "caggcatgatgactgtacgtcatcagacaggcagcaatgccggcagcttccggactcatt\n",
      "tcagcgcggttaccgttcatggcattgaacagtacccatgtttcgtcatcatcgtcatcc\n",
      "ggttcgggtgctataaatgccccgccgttgttcagggtgtacagattccagataccaccg\n",
      "cagtagtcttcgcacagacggtccatccagccgaagacacggggctccagggtcacccac\n",
      "tgtggaatgaggccaaagtgctgcggccagaagctgatgcgctgttcatcagggacgagg\n",
      "gtggcaacc--\n",
      "\n",
      "\n",
      "10 [(('G000006925', (900174, 900410)), (900313, 900359), '+'), (('G000006925', (2058572, 2058808)), (2058623, 2058669), '-'), (('G000006925', (3422405, 3422641)), (3422544, 3422590), '+'), (('G000006925', (4358832, 4359068)), (4358882, 4358929), '-'), (('G000006925', (4612466, 4613007)), (4612507, 4612553), '-'), (('G000006925', (4622488, 4624460)), (4623804, 4623850), '+'), (('G000006925', (4700206, 4700442)), (4700256, 4700303), '-'), (('G000006925', (4775859, 4776095)), (4775909, 4775955), '-'), (('G000006925', (4782146, 4782382)), (4782285, 4782331), '+'), (('G000299455', (5313369, 5315826)), (5315041, 5315087), '+')]\n",
      ">G000006925|900174|900410|900313|900359|+\n",
      "-gttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000006925|2058572|2058808|2058623|2058669|-\n",
      "ggttacttgcacggatcctggtcagcaaatatatggaacatatccc-\n",
      ">G000006925|3422405|3422641|3422544|3422590|+\n",
      "-gttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000006925|4358832|4359068|4358882|4358929|-\n",
      "ggttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000006925|4612466|4613007|4612507|4612553|-\n",
      "-gcttctggcccatgttgttaccggaaagtatgcagaccatctgcca\n",
      ">G000006925|4622488|4624460|4623804|4623850|+\n",
      "-gcttctggcccatgttgttgctggaaaatatgcagactatctgccg\n",
      ">G000006925|4700206|4700442|4700256|4700303|-\n",
      "ggttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000006925|4775859|4776095|4775909|4775955|-\n",
      "-gttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000006925|4782146|4782382|4782285|4782331|+\n",
      "-gttacttgcacggatcctggtcagcaaatatatggaacatatccct\n",
      ">G000299455|5313369|5315826|5315041|5315087|+\n",
      "-gcttctggcccatgttgtcaccgggaaatatgcagaccatctgccg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (720060, 720643)), (720579, 720625), '+'), (('G000006925', (2008567, 2033069)), (2010193, 2010239), '-'), (('G000299455', (2082804, 2090582)), (2090485, 2090531), '+'), (('G000299455', (2546059, 2546618)), (2546572, 2546618), '+'), (('G000299455', (3293954, 3294425)), (3293954, 3294000), '-'), (('G000299455', (3608677, 3610659)), (3610317, 3610363), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1094\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1094\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|720060|720643|720579|720625|+\n",
      "-tacaaaaagccgagggggtggtggaaggaatgttgatgatgttggg\n",
      ">G000006925|2008567|2033069|2010193|2010239|-\n",
      "ttacaagaggttgaggggataattaaagggatgcttatgacattag-\n",
      ">G000299455|2082804|2090582|2090485|2090531|+\n",
      "-tacaagaggttgaggggataattaaagggatgcttatgacattaga\n",
      ">G000299455|2546059|2546618|2546572|2546618|+\n",
      "-ttcacaaagcggaggggattgttgaaggcatgctgatgatgctggg\n",
      ">G000299455|3293954|3294425|3293954|3294000|-\n",
      "cttcacaaagcggaggggattgttgaaggcatgctgatgatgctgg-\n",
      ">G000299455|3608677|3610659|3610317|3610363|-\n",
      "ttacagaaggctgagggcataattgaagggatgttaatggcattag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4770231, 4774138)), (4773446, 4773530), '+'), (('G000299455', (5296182, 5296266)), (5296182, 5296266), '+'), (('G000299455', (5400585, 5404288)), (5403822, 5403906), '+')]\n",
      ">G000006925|4770231|4774138|4773446|4773530|+\n",
      "ggctgaccgaagacgaaatcagggcagttcttgatgcggtgcgcgatgcagtatgcagtg\n",
      "tcagttgccggggggcagaagatg\n",
      ">G000299455|5296182|5296266|5296182|5296266|+\n",
      "ggctgaccgaagacgaaatcagggccgttcttgatgcggtgcgcgatgcagtatgcagtg\n",
      "tcagttgccgggtggcagaagatg\n",
      ">G000299455|5400585|5404288|5403822|5403906|+\n",
      "ggctgaccgaagacgaaatcagggcagttcttgatgcagtgcgcgatgcggtacgcagtg\n",
      "tcagttgccgggtggcagaagatg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2366334, 2374443)), (2373838, 2373885), '-'), (('G000299455', (1685057, 1714323)), (1706771, 1706818), '+'), (('G000299455', (4516187, 4516323)), (4516276, 4516323), '+')]\n",
      ">G000006925|2366334|2374443|2373838|2373885|-\n",
      "tatagccgtcaccgtcgctggtttccacggaccacaaaacatccgag-\n",
      ">G000299455|1685057|1714323|1706771|1706818|+\n",
      "-atagccgtcaccgtcgctggtttccacggaccacaaaacatccgagt\n",
      ">G000299455|4516187|4516323|4516276|4516323|+\n",
      "-ataaccatcgccttcgcgggtctttactgaccacagaatatcggagt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1425133, 1465118)), (1448743, 1448844), '+'), (('G000006925', (2138845, 2144864)), (2143288, 2143390), '+'), (('G000299455', (1911336, 1921536)), (1916990, 1917092), '-'), (('G000299455', (2199778, 2263225)), (2246838, 2246939), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1425133|1465118|1448743|1448844|+\n",
      "--gcggggctactcacgcttgttgttctcgaaattgtgctgggtatcgataacctggtct\n",
      "tcatcgccattcttgctgacaaactgccgccaaaacaacgcga\n",
      ">G000006925|2138845|2144864|2143288|2143390|+\n",
      "gggccgggttaatcacgctgattgtgatcgaactggtcctcggcattgataacctggtct\n",
      "ttattgccatcctcgccgaaaaactaccgccgaagcagcgcg-\n",
      ">G000299455|1911336|1921536|1916990|1917092|-\n",
      "gggccgggttaatcacgctgattgtgatcgaactggtcctcggcattgataacctggtct\n",
      "ttattgccatcctcgccgaaaaactaccgccgaagcagcgcg-\n",
      ">G000299455|2199778|2263225|2246838|2246939|+\n",
      "--gcggggctactcacgcttgttgttctcgaaattgtgctgggtatcgataacctggtct\n",
      "tcatcgccattcttgctgacaaactgccgccaaaacaacgcga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2569729, 2584886)), (2578711, 2578734), '-'), (('G000299455', (535426, 536523)), (535434, 535457), '+'), (('G000299455', (1462739, 1472085)), (1468891, 1468914), '+'), (('G000299455', (3918464, 3964500)), (3934949, 3934972), '-')]\n",
      ">G000006925|2569729|2584886|2578711|2578734|-\n",
      "cggcgcaccagcacaaagaacag-\n",
      ">G000299455|535426|536523|535434|535457|+\n",
      "-ggcggatcaccacaaagaacacc\n",
      ">G000299455|1462739|1472085|1468891|1468914|+\n",
      "-ggcgcaccagcacaaagaacagc\n",
      ">G000299455|3918464|3964500|3934949|3934972|-\n",
      "cggcgaaccaccacaaagaatac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (644023, 652288)), (650304, 650344), '+'), (('G000299455', (2505935, 2525742)), (2520303, 2520343), '+'), (('G000299455', (3697938, 3726006)), (3724022, 3724062), '+')]\n",
      ">G000006925|644023|652288|650304|650344|+\n",
      "cggtttatcgcctgattgaccagtacgggccaatctcgcg\n",
      ">G000299455|2505935|2525742|2520303|2520343|+\n",
      "cggtttatcgcctgattgatcagcttggtccagtctcgcg\n",
      ">G000299455|3697938|3726006|3724022|3724062|+\n",
      "cggtttatcgcctgattgaccagtacgggccaatctcgcg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (732696, 732803), '+'), (('G000299455', (2174641, 2181396)), (2181092, 2181199), '+'), (('G000299455', (2910238, 2916987)), (2910429, 2910536), '-'), (('G000299455', (3598153, 3598260)), (3598153, 3598260), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1048\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|732696|732803|+\n",
      "-cttcccgttatccgaaacactggctgaccgtgggtgatccggcgcgtgaattcaccatg\n",
      "acgcagtccgcgccgctgatggtgctgccggatccggatgagtttgtg\n",
      ">G000299455|2174641|2181396|2181092|2181199|+\n",
      "-cttcccgttatccgaaacactggctgactgtgggcgatccggcccgtgaattcaccatg\n",
      "acgcagtccgcaccgctgatggtgctgccggatccggatgagtttgtg\n",
      ">G000299455|2910238|2916987|2910429|2910536|-\n",
      "tcttcccgttacccgaaacactggctgaccgtgggcgatccgtcctgtgaattcaccatg\n",
      "acgcagtccgcgccgctgatggtgctgccggatccggatgagtttgt-\n",
      ">G000299455|3598153|3598260|3598153|3598260|-\n",
      "tctgcccgctacccgaaaaactgggtgaccaccggcgatccggcgcgtgagttcaccatg\n",
      "attcagtcagcaccgctgatgctgctggctgaccctgatgcgttcgt-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3086372, 3086439), '-'), (('G000299455', (56338, 57307)), (57043, 57110), '+'), (('G000299455', (857924, 871846)), (869071, 869138), '+'), (('G000299455', (1291086, 1291153)), (1291086, 1291153), '+'), (('G000299455', (1986939, 1987907)), (1987643, 1987710), '+'), (('G000299455', (3152907, 3154652)), (3154391, 3154458), '+')]\n",
      ">G000006925|3083675|3099254|3086372|3086439|-\n",
      "aaacagtgcattacagagtgaagatttaccggccccgcttttacccataatgccaatcac\n",
      "gggttcg--\n",
      ">G000299455|56338|57307|57043|57110|+\n",
      "--acagtgcattacagagtgaggatttaccggccccgcttttacccataatgccaatcac\n",
      "gggttcgtg\n",
      ">G000299455|857924|871846|869071|869138|+\n",
      "-aacagtgcattacagagtgaagatttaccggccccgcttttacccataatgccaatcac\n",
      "gggttcgt-\n",
      ">G000299455|1291086|1291153|1291086|1291153|+\n",
      "-aaaagggcattgcaaagcgaggattttccggcacctgttttgcccatgatgccaatcac\n",
      "gggttcgt-\n",
      ">G000299455|1986939|1987907|1987643|1987710|+\n",
      "-aacagtgcattacagagtgaggatttaccggccccgcttttacccataatgccaatcac\n",
      "gggttcgt-\n",
      ">G000299455|3152907|3154652|3154391|3154458|+\n",
      "-aacagtgcattacagagtgaggatttaccggccccgcttttccccataatgccaatcac\n",
      "gggttcgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3831593, 3850590)), (3847046, 3847167), '+'), (('G000299455', (2611787, 2611908)), (2611787, 2611908), '+'), (('G000299455', (5202806, 5268469)), (5255248, 5255369), '+')]\n",
      ">G000006925|3831593|3850590|3847046|3847167|+\n",
      "tgcagttctccgatatgtacgttaccaccaaaggtttcctgccctttgcgccagaggctg\n",
      "atttctgggtgggtaaacacggtgcgccgaaaattgaaatccagatgctggactggaaaa\n",
      "c\n",
      ">G000299455|2611787|2611908|2611787|2611908|+\n",
      "tgcagttcagcgatatttacctgacaacgcgaggttttttacccttcgcaccagaggcag\n",
      "acttctgggtcggcaaacataaactcccgcaatatgagatccagatgctggactggaaaa\n",
      "c\n",
      ">G000299455|5202806|5268469|5255248|5255369|+\n",
      "tgcagttctccgatatgtacgttaccaccaaaggtttcctgccctttgcgccagaggctg\n",
      "atttctgggtgggtaaacatggtgcgccgaaaattgaaatccagatgcttgactggaaaa\n",
      "c\n",
      "\n",
      "\n",
      "11 [(('G000006925', (385514, 385654)), (385611, 385642), '+'), (('G000006925', (900498, 900638)), (900510, 900541), '-'), (('G000006925', (3422729, 3422869)), (3422741, 3422772), '-'), (('G000006925', (3851292, 3851432)), (3851389, 3851420), '+'), (('G000006925', (4358604, 4358744)), (4358701, 4358732), '+'), (('G000006925', (4699978, 4700118)), (4700075, 4700106), '+'), (('G000006925', (4775631, 4775771)), (4775728, 4775759), '+'), (('G000006925', (4782470, 4782610)), (4782482, 4782513), '-'), (('G000299455', (4572495, 4572632)), (4572495, 4572526), '-'), (('G000299455', (4749136, 4749179)), (4749136, 4749167), '+'), (('G000299455', (5335055, 5335098)), (5335067, 5335098), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1136\n",
      "\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1136\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\n",
      "Reallocating..done. *alloclen = 1064\n",
      "\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|385514|385654|385611|385642|+\n",
      "-ccggggccagtactttcaccggagtgtcatc\n",
      ">G000006925|900498|900638|900510|900541|-\n",
      "cccggggccagtactttcaccggagtgtcat-\n",
      ">G000006925|3422729|3422869|3422741|3422772|-\n",
      "cccggggccagtactttcaccggagtgtcat-\n",
      ">G000006925|3851292|3851432|3851389|3851420|+\n",
      "-ccggggccagtactttcaccggagtgtcatc\n",
      ">G000006925|4358604|4358744|4358701|4358732|+\n",
      "-ccggggccagtactttcaccggagtgtcatc\n",
      ">G000006925|4699978|4700118|4700075|4700106|+\n",
      "-ccggggccagtactttcaccggagtgtcatc\n",
      ">G000006925|4775631|4775771|4775728|4775759|+\n",
      "-ccggggccagtactttcaccggagtgtcatc\n",
      ">G000006925|4782470|4782610|4782482|4782513|-\n",
      "cccggggccagtactttcaccggagtgtcat-\n",
      ">G000299455|4572495|4572632|4572495|4572526|-\n",
      "cccggtgccagtacctttaccggtgtgtcat-\n",
      ">G000299455|4749136|4749179|4749136|4749167|+\n",
      "-ccggtgccagtacttttactggtgtgtcatc\n",
      ">G000299455|5335055|5335098|5335067|5335098|-\n",
      "cccggtgccagtacttttactggtgtgtcat-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3040656, 3065083)), (3053931, 3053988), '+'), (('G000006925', (4462252, 4478752)), (4465823, 4465880), '-'), (('G000299455', (942831, 956197)), (942864, 942921), '-'), (('G000299455', (4713031, 4748324)), (4737171, 4737228), '+'), (('G000299455', (4771847, 4785261)), (4785177, 4785234), '+')]\n",
      ">G000006925|3040656|3065083|3053931|3053988|+\n",
      "-agatgcgaaccgggagaagggtatcagcccgtttgaatatgcaaagattaacaatcc\n",
      ">G000006925|4462252|4478752|4465823|4465880|-\n",
      "tgaatgagaacaaagagaaatgtgtcagtccgcttgctcatgcaaagactaacaatc-\n",
      ">G000299455|942831|956197|942864|942921|-\n",
      "tggatgcaaacagagaaaaatttatcagtccatttgaatatgcaaagattaataacc-\n",
      ">G000299455|4713031|4748324|4737171|4737228|+\n",
      "-agatgcgaaccgggagaagggtatcagcccgtttgaatatgcaaagattaacaatcc\n",
      ">G000299455|4771847|4785261|4785177|4785234|+\n",
      "-gaatgagaacaaagagaaatgtatcagtccgcttgctcatgcaaagactaacaatcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (201064, 201775), '-'), (('G000299455', (4235244, 4293324)), (4240450, 4241161), '+'), (('G000299455', (4789263, 4790612)), (4789263, 4789975), '-')]\n",
      ">G000006925|64491|216702|201064|201775|-\n",
      "catac-tcgcctttaatgtggatcagtgaagcctgcgataacgccgccagcattttgtgg\n",
      "gtcgattgcgtttcgaagaaaacttttcccgcaacacgctcgccgctcataccactttta\n",
      "ccctggtagattggatgaaaatgagtatacggcacccaggcggaatcgaagtggatcgac\n",
      "gggacatccagcgtctgtttgatccagtcggtgttgtagagcaaaccatcataggtggag\n",
      "ttggtgatcaccgcatgaaccggccattgtgcttgcgtagtggcagcgactttctcttcg\n",
      "atgctgtcgcgagtaaattcacggcgcgggatcccccccagaatccccaacgcattacgc\n",
      "gtcggtttcagccagactggcactacatcgttcatcatcaacagatgcgccagtgattta\n",
      "tggcagttgcggtcgatcaacagcgtactgccggatggcgcggcgtacatacccacaatt\n",
      "ttgttcgacgtcgatattccgttggtaacgatataactctgttccgcgccaaaagtccgc\n",
      "gcgatgtactcttccgcttccaggtgtggcccggtgtggtcaagcaacgaaccaagctcg\n",
      "gtgaccgaaatagagacatcagccttaagggtattcccgccgaaaaaatcataaaacaga\n",
      "cagccaaccgggcttttttgatatgcggtgccgcccatatgccccggcgtac-\n",
      ">G000299455|4235244|4293324|4240450|4241161|+\n",
      "-atac-tcgcctttaatgtggatcagcgaagcctgcgataacgccgccagcattttgtgg\n",
      "gtcgattgcgtttcgaagatcacttttcccgcaacacgctcgccgctcataccacttttc\n",
      "ccctggtagatcggatgaaaatgggtgtacggcacccaggcggaatcgaagtggatcgac\n",
      "gggacatccagcgtctgtttgatccagtcggtgttgtagagcaagccatcataggtggag\n",
      "ttggtaatcaccgcatgaaccggccattgtgcctgcgtggtggcagcgactttctcttcg\n",
      "atgctgtcgcgagtaaattcacggcgcgggatcccaccaagaatccccaacgcattacgc\n",
      "gtcggtttcagccagactggcactacatcgttcatcatcaacagatgcgccagcgattta\n",
      "tgacaattgcggtcgatcaacagcgtactgccggatggcgcggcgtacatacccacaatt\n",
      "ttgttcgacgtcgatgttccgttggtaacgatataactctgttccgcgccaaaagtccgc\n",
      "gcgatgtactcttccgcttccaggtgtggcccggtgtggtcgagcaacgaaccaagctcg\n",
      "gtgaccgaaatagagacatcagccttgagggtattcccgccgaaaaaatcataaaacagg\n",
      "cagccaaccgggcttttttgatatgcggtgccgcccatatgccccggcgtaca\n",
      ">G000299455|4789263|4790612|4789263|4789975|-\n",
      "tttacgtcacctttaacgtggatcatggaagcctgagagaacgccgccagcagtttgtga\n",
      "gtggactgggtttcgtaaatcactttcccttctacacggccaccgctcataccgcattta\n",
      "ccttcgtaaatcggtgagaagttggtgtaaggcacccacgcggagtcaaagtggatggat\n",
      "ttcacatccagtgttttcttgatgaagtcggtgttgtacagcagaccatcataggtagag\n",
      "ttagtaattacagcatgtaccggccaggttgcgtttggtgtttctttcacgcgcttagca\n",
      "atggtagcgtgctggaattcactctgtgggataccaccaagaataccgtaagcgttacgg\n",
      "gtcgggcggaaatagattggcgtaacatcgctcatcatcatcaggtgggtcagcgatttg\n",
      "tggcagttacggtcaatcagaatggtgctgcctgccggagcagagtacatgccaacaatt\n",
      "ttgttcgcagtggaagtaccgttggtcaccatgtagctgcggtctgcgttaaagacgcga\n",
      "gcgatatactgttctgcttctttgtgtggaccactgtgatccagcagagaacccagttca\n",
      "gatactgaaatggaaatatcagatttcatggtattcggaccaaagaaatcatagaacagg\n",
      "ctacctaccgggcttttctggaatgcagtaccgcccatgtgaccaggagtac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3392215, 3392592), '-'), (('G000299455', (540757, 615358)), (548534, 548911), '+'), (('G000299455', (2052398, 2064114)), (2056708, 2057084), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1116\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3361966|3403725|3392215|3392592|-\n",
      "aacttcttataacgtaaggtgaaaagctgaaaagccagccctgcgaagatgaggggctgc\n",
      "aatgagataatcttcacagaccacaaataatgtagggatgccaggatcgcgacaagatag\n",
      "acgacattacgcagcggttgctggggattacccaattttcactgcatcgtctggatagat\n",
      "gtgaatgctaaagcacgcaaaatcaaccaactgacaattctcagcattaaataggttcga\n",
      "gtggttaatttgtagtcaaataaaatcagattattcaccccatggtaccagaaatcagta\n",
      "acaggttaagcgtagtgtcgcccaggcaaagcaccataaacctaacaggcgacggaggcg\n",
      "tatcaataatggctgtt-\n",
      ">G000299455|540757|615358|548534|548911|+\n",
      "-acttcttataacgtaaggtgaaaagctgtaaagccagccctgcgaagatgaggggctgc\n",
      "aatgagataatcttcacagaccacaaataatgtagggatgccaggatcgcgacaagatag\n",
      "acgacattacgcagcggttgctggggattacccaattttcactgcatcgtctggatagat\n",
      "gtgaatgctaaagcacgcaaaatcaaccgactgacaattctcagcattaaataggttcga\n",
      "gtggttaatttgtagtcaaataaaatcagattattcaccccatgggaccagaaatcagta\n",
      "ataggttaagcgtagtgtcgcccaggcaaagcaccataaacctaacaggcgacggaggca\n",
      "tatcaataatggctgttt\n",
      ">G000299455|2052398|2064114|2056708|2057084|+\n",
      "-acttcttataccgtaaggctaaaagcagtaccgccagcccagcgtagatgagtggctgc\n",
      "ggtgaaataatcttcactgaccacaaataatgtattggggccaggatcgcgacaagatag\n",
      "acgaagttatgcaactgttgccagtgtttaccgagttttcgctgcatcgcctgggttgaa\n",
      "gtgaacgctaacgcaagcaaaattacccagctgataatacctaacgttaaataaggccgg\n",
      "gtaattaactccttacctaataacgccagattattcacgcc-cagctccagcaatgcgta\n",
      "actggttaaatgcagtgtcgcccaggcaaagcaccataagcctaacaggcggcgagtgcg\n",
      "tatcaataacggctgttt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3107661, 3117031)), (3108871, 3108922), '+'), (('G000006925', (3736321, 3788106)), (3750359, 3750410), '-'), (('G000006925', (4535976, 4543278)), (4541946, 4541997), '+'), (('G000299455', (137973, 154383)), (149400, 149451), '+'), (('G000299455', (830050, 839519)), (838162, 838213), '-')]\n",
      ">G000006925|3107661|3117031|3108871|3108922|+\n",
      "-cgtcgcaccaaagccagcagctccttccagcaacgcaccaaaggagaaacc\n",
      ">G000006925|3736321|3788106|3750359|3750410|-\n",
      "ccggtgcgccaaagcctgcggctccttcaaggaaagcgccgaaacagaaac-\n",
      ">G000006925|4535976|4543278|4541946|4541997|+\n",
      "-cggtgcgccaaagcctgcggctccttcaaggaaagcgccgaaacagaaacc\n",
      ">G000299455|137973|154383|149400|149451|+\n",
      "-cggtgcgccaaagccggcggctccttcaaggaaagcgccgaaacagaaacc\n",
      ">G000299455|830050|839519|838162|838213|-\n",
      "ccggcgcaccaaagccagccgctccttccagcaacgcaccaaaggagaaac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1578298, 1580868)), (1580308, 1580868), '+'), (('G000006925', (2062783, 2064325)), (2063185, 2063745), '+'), (('G000299455', (2927401, 2931337)), (2929418, 2929978), '+')]\n",
      ">G000006925|1578298|1580868|1580308|1580868|+\n",
      "taccattgcgccggtaccataacggtgaatgacggtttcgctgtgatgataatcgccgtg\n",
      "tggccactggcaggatttaatatggcgcaacagccagtcagacaatgcgccagcaccacc\n",
      "agcagcacgaatcacccgtgcgttactgaaaaacggcagcaatgttttgtcttccactag\n",
      "cggctggcgaacggcaggaacgaccccggacggcagattacgcatgcttttcggttccgg\n",
      "ctccaccagtacccgggtattgtggaataccggcatggattcacgaccgggcttaaggac\n",
      "caccagcccgagttccggaaccagaacaggtcgaagtaatacccgcacgttacctccaga\n",
      "tccgttgctggaatgtgcgggacggacgcggtggctgttcggagtaagggagcctgacgg\n",
      "agattatctagtgacggtagtcgaggctaagggcttttttaacctcgcatccgcgcctgc\n",
      "ggtaacactgaataagccattcggcctgttcttcagtgcatgagggatgctggtaccagt\n",
      "ctgacttaaatgcgtgagaa\n",
      ">G000006925|2062783|2064325|2063185|2063745|+\n",
      "taccattgcgccggtaccataacggtgaatgacggtttcgctgtgatgataatcgccgtg\n",
      "tggccactggcaggatttaatatggcgcaacagccagtcagacaatgcgccagcaccacc\n",
      "agcagcacgaatcacccgtgcgttactgaaaaacggcagcaatgttttgtcttccactag\n",
      "cggctggcgaacggcaggaacgaccccggacggcagattacgcatgcttttcggttccgg\n",
      "ctccaccagtacccgggtattgtggaataccggcatggattcacgaccgggcttaaggac\n",
      "caccagcccgagttccggaaccagaacaggtcgaagtaatacccgcacattacctccaga\n",
      "tccgttgctggaatgtgcaggacggacgcggtggccgttcggaataagggagcctgacga\n",
      "agattatccagtgacgataatcgagactgagggctttcttaatctcgtatccgtgtctgc\n",
      "ggtagcgctgaatcagccattcagcctgttcttcggtgcaggggtcgtgctgatacccgt\n",
      "cagatttgaatgcatgagaa\n",
      ">G000299455|2927401|2931337|2929418|2929978|+\n",
      "caccattgcgccggtaccataacggtgaatgacggtttcactgtggtgataatcgccgtg\n",
      "tggccactggcaggatttaacatggcgcagtaaccagtcagacaatgcgccagcgccacc\n",
      "agcagcacgaatcactcgttcgtcgctgaaaaatggcagtaatgatttatcctccgccag\n",
      "cggctggcgaacggcaggaacgaccccggacggcagattacgcatgcttttcggttccgg\n",
      "ctccaccagtacccgggtattgtggaataccggcatggattcacggcccggcttaacgat\n",
      "caccagcccgagttccggtaccagaacaggtcgaagtaatacccgcacgttacctccaga\n",
      "tgcgttgctggaatgtgcgggacggacgcggtgggcgttcagagtaaggaagcctgacgg\n",
      "agattatccagtgacgataatcgaggctgagggctttcttaatctcgtatccgcgtctgc\n",
      "ggtagttatgaattagccattcggcctgttcttcagtacatggtgggtgttggtaccagt\n",
      "cggttttaaatgcgtgtgaa\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3076830, 3076912)), (3076833, 3076904), '+'), (('G000299455', (63265, 63344)), (63265, 63341), '-'), (('G000299455', (874185, 874262)), (874185, 874261), '-'), (('G000299455', (1996241, 1996320)), (1996249, 1996320), '-'), (('G000299455', (4540974, 4541053)), (4540982, 4541053), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1104\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1104\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3076830|3076912|3076833|3076904|+\n",
      "-tcatggttaaatctccgtatg-----gttcatacggaatgaccacgccgtaaaaaatgc\n",
      "gcagtgccgctgaagtg\n",
      ">G000299455|63265|63344|63265|63341|-\n",
      "gtcatggttaaatctccgtatggtttcgttcatacggaataaccacgctgtaaaaaatgt\n",
      "gcagagtccctgaagt-\n",
      ">G000299455|874185|874262|874185|874261|-\n",
      "gtcatggttaaatctccgtatggtttcgttcatacggaataaccacgccgtaaaaaatgc\n",
      "gcagagtccctgaagt-\n",
      ">G000299455|1996241|1996320|1996249|1996320|-\n",
      "gtcatggttaaatctccgtatg-----gttcatacggaatagccacgtcgtaaaaaatgc\n",
      "gcagagtccctgaagt-\n",
      ">G000299455|4540974|4541053|4540982|4541053|-\n",
      "gtcatggttaaatctccgtatg-----gttcatacggaatagccacgtcgtaaaaaatac\n",
      "gcagagtccctgaagt-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (726248, 726937), '+'), (('G000299455', (2174641, 2181396)), (2174641, 2175330), '+'), (('G000299455', (2910238, 2916987)), (2916298, 2916987), '-'), (('G000299455', (3603798, 3604500)), (3603798, 3604487), '-')]\n",
      ">G000006925|726235|732999|726248|726937|+\n",
      "-gggccgatcaaaattattacctgcctaaagaatcgtcatacggtgagggggaatggaaa\n",
      "acgctgcagtttcaggtcgccatcatgaactgcatgggcaatgacgagattcgtacagtt\n",
      "aacctgattaaatctgcccgtgttggttatacaaaaatggtgcttggtgtgatcgggtat\n",
      "tttattgagcacaaatcccgcaacactcttctttttcagccgacggactcagctgctgaa\n",
      "gattttatgaagtctcacgtggaagcaacaattcgtgatgttccttgcctgaaaaaactt\n",
      "tctccctggctgggccggaaacaccgggataatacgctcacactgaagcgtttttcctct\n",
      "ggtgtggggttctggtgtttgggtggtgctgctgccaaaaactaccgtgaaaaatctgtc\n",
      "gatgtggtctgctatgacgaactctcctcgtttgaaccggatgtggaaaaagaaggttcg\n",
      "ccaacactgcttggcgataaacagtattgaaggttcggtatggccaaaatccatacgcgg\n",
      "ttcaacgccaaaaattaaaggctcctgtcagattgaaaaagctgcgaatgaatctgcgca\n",
      "tttcatgcggttttatgtcccttgccctca-tgcggggaggagcagtatctgaagtttgg\n",
      "cgatgatgcgacgtcgtttggcctgaaatgg\n",
      ">G000299455|2174641|2181396|2174641|2175330|+\n",
      "-gggccaataaacattattaccttcctaaagagtcgtcttatacccccgggcggtgggaa\n",
      "acactgccgtttcaggttggcatcatgaactgtatgggcaacgatctgattcgcacggtt\n",
      "aacctgattaaatctgcccgtgttggttatacaaagatgttgctgggagtggaggcttat\n",
      "tttattgagcataaatcacgcaacagccttctttttcagcccacggactcagctgctgaa\n",
      "gattttatgaaatctcatgttgagccaacgataagggatgttcctgcattgctggagctg\n",
      "gctccatggttcggaagaaaacaccgcgataatacgctcaccctgaagcgtttttcctcc\n",
      "ggtgtggggttctggtgtctgggtggtgcggcagcaaaaaactaccgtgaaaaatccgtg\n",
      "gatgtggtctgttatgacgagctttcctcgttcgaaccggatgttgaaaaagagggttcg\n",
      "ccaaccctgctgggggataaac-gtattgagggctctgtatggccaaaatccattcgcgg\n",
      "ctcgacgcctaaaatcaaaggctcctgccagatcgaaaaagccgctaacgagtcggcaca\n",
      "cttcatgcgtttttatgtgccctgtccgcactgtggggaggagcagtatctgaaatttgg\n",
      "cgatgatgcctcgcctttcggtcttaagtgg\n",
      ">G000299455|2910238|2916987|2916298|2916987|-\n",
      "tgggccaataaacattattaccttcctaaagagtcgtcttataccccggggcggtgggaa\n",
      "acactgccgtttcaggttgccatcatgaacagcatggggaatgaccggatccgcactgtt\n",
      "aatctgattaaatctgcccgtgttggttatacaaagatgttgctgggagtggaggcttat\n",
      "tttattgagcataaatcacgcaacagccttcttttccagcccacggattctgctgctgaa\n",
      "gattttatgaaatctcatgtggaacccacgatcagggatgtgccggttttactcgatctt\n",
      "gcaccgtggtttgggcgtaaacatcgtgataataccctcacgctgaaacgtttttcatcg\n",
      "ggcgtgggcttctggtgcctgggcggggctgccgctaaaaactaccgtgaaaaatccgtg\n",
      "gacgtggtctgctatgacgaactttcctcgttcgaaccggatgtcgaaaaagagggttcg\n",
      "ccaaccctgcttggggataaac-gtattgagggctctgtatggcccaaatccattcgcgg\n",
      "ctcgacgcctaaaatcaaaggcacctgccagatcgaaaaagcggccaacgagtcggcgca\n",
      "tttcatgcgtttttatgtgccctgcccacactgtggggaggagcagtatctgaaatttgg\n",
      "cgatgaatccacgccttttggccttaaatg-\n",
      ">G000299455|3603798|3604500|3603798|3604487|-\n",
      "tgggcggatgctaattactatctcccgaaagaatccgcataccaggaagggcgctgggaa\n",
      "acactgccctttcagcgggccatcatgaatgcgatgggcagcgactacatccgtgaggtg\n",
      "aatgtggtgaagtctgcccgtatcggttattccaaaatgctgctgggtgtttatgcctac\n",
      "tttatagagcataagcagcgcaacacccttatctggttgccgacggatggtgatgccgag\n",
      "aactttatgaaaacccacgttgagccgaccatccgcgatattccgtcgctgctggcgctg\n",
      "gctccgtggtatggcaaaaagcaccgggataacacgctcactatgaagcgtttttccaat\n",
      "ggtcgtggcttctggtgcctgggcggtaaagcggcaaaaaactaccgtgaaaagtcagtg\n",
      "gatgtggcgggttatgatgaacttgctgcctttgatgaggatattgaacaggaaggctct\n",
      "ccgacgttcctgggtgacaagc-gtattgaaggctcggtctggccaaagtccatccgtgg\n",
      "ctccacgccaaaagtgagaggcacctgtcagattgagcgtgcagccagtgaatccccgca\n",
      "ttttatgcgttttcatgttgcctgcccgcattgcggggaggagcagtatcttaaatttgg\n",
      "cgacaaagagacgccgtttggcctcaaatg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3677147, 3681777)), (3679672, 3679697), '+'), (('G000299455', (227444, 242483)), (229524, 229549), '-'), (('G000299455', (5156132, 5190970)), (5168815, 5168840), '+')]\n",
      ">G000006925|3677147|3681777|3679672|3679697|+\n",
      "-cgaatttggtcggtgatagaggatt\n",
      ">G000299455|227444|242483|229524|229549|-\n",
      "tcggatttggtcggtgatagaggat-\n",
      ">G000299455|5156132|5190970|5168815|5168840|+\n",
      "-caaaattggtcggcgagagaggatt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (720060, 720643)), (720283, 720579), '-'), (('G000299455', (2546059, 2546618)), (2546276, 2546572), '-'), (('G000299455', (3293954, 3294425)), (3294000, 3294296), '+'), (('G000299455', (3608677, 3610659)), (3610363, 3610659), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 2380\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 2380\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 1594\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 1076\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 3 \n",
      "Reallocating..done. *alloclen = 1076\n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|720060|720643|720283|720579|-\n",
      "gcttctttcctatataaccatctgagcatccatgcttccgcgccagcgtcatgaatgtca\n",
      "tcccaaacacgtagtagtcaaccagcaagtcatgtagatcgttgttgttccggttaaggc\n",
      "gggccatacacccacatataaccatcgcgtcatcgtcacagcattgcgggcgggatttta\n",
      "cttttgaagggattaatcccttaaaaccggcagcaatggacgaccaggtcacatcctcat\n",
      "gattattagccacccacgcgccccagcgctcaagaacctgctgaatatcacgcatc-\n",
      ">G000299455|2546059|2546618|2546276|2546572|-\n",
      "ggcgtttacctatacaggtgtcagagcacccatgtttccgtgccagcgccatgaacgtct\n",
      "cccccaacacgtagtaatcaaccagcaagtcatgcagatcgcgattgttccggtaaaggc\n",
      "gggctatgcacccgcatatcaccatcgcatcatcgtcacagcactgtggacgtgatttta\n",
      "ctttttcggggatcagtcccttaaatccggcagcaatgggcgaccatgtaacatcctcat\n",
      "agttatttaccgcccatgccccccagcgctcaagaacctgccggatatcacgcatc-\n",
      ">G000299455|3293954|3294425|3294000|3294296|+\n",
      "-gcgtttacctatacaggtgtcagagcacccatgtttccgtgccagcgccatgaacgtct\n",
      "cccccaacacgtaataatcaaccagcaagtcatgcagatcgctgttgttcctgttaaggc\n",
      "gagccatacacccgcatataatcatcgcgtcatcgtcacaacactgtggacgtgatttta\n",
      "ctttttcggggatcagcctcttaaatccggcagcaatgggcgaccatgtaacatcctcat\n",
      "ggttatttgccgcccatgccccccagcgctcaagaacctgccggatatcacgcatca\n",
      ">G000299455|3608677|3610659|3610363|3610659|+\n",
      "-ccttttcccgatataaccatcagagcagcaatgcttacctgccagtgacatgaatgtca\n",
      "taccgactacataataatctactaataaatcgtgcaaatcgctgttgttctttttcagac\n",
      "gggccatgcacccgcaaatgatcatcgcgtcatcgtcacaacattgcgggcgagatttta\n",
      "cttttgaagtaattaatcccttaaaaccggcggcaatggacgaccaggtcacatcttcat\n",
      "gattattagccgcccacgctccccaacgctcaagaaccatctgaatatcacgcatca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4143132, 4163489)), (4156041, 4156116), '-'), (('G000299455', (1535847, 1550961)), (1546929, 1547004), '+'), (('G000299455', (4996216, 5016676)), (5003680, 5003755), '+')]\n",
      ">G000006925|4143132|4163489|4156041|4156116|-\n",
      "tggaagcgcaggccgcattcggcaatggtgcggaaggcgttggtttgttccgcactgaaa\n",
      "tgctctatatggatc-\n",
      ">G000299455|1535847|1550961|1546929|1547004|+\n",
      "-ggaagcaccaggcgcgtttgccaacggcgcggaaggcgtcgggctgttccgtaccgaaa\n",
      "tgctgtatatggatcg\n",
      ">G000299455|4996216|5016676|5003680|5003755|+\n",
      "-ggaagcgcaggccgcattcggcaatggtgcggaaggcgttggtttgttccgcactgaaa\n",
      "tgctctatatggatcg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2242494, 2346249)), (2316122, 2316159), '-'), (('G000006925', (4288513, 4315006)), (4302792, 4302829), '+'), (('G000299455', (1752647, 1813107)), (1763000, 1763037), '+'), (('G000299455', (4836149, 4864500)), (4847565, 4847602), '-')]\n",
      ">G000006925|2242494|2346249|2316122|2316159|-\n",
      "cgtttatgccgtggctggtggggactgcactgatgca-\n",
      ">G000006925|4288513|4315006|4302792|4302829|+\n",
      "-tttattaccctggctttctgccactgcgctgctgcac\n",
      ">G000299455|1752647|1813107|1763000|1763037|+\n",
      "-gtttatgccgtggctggtggggactgcgctgatgcac\n",
      ">G000299455|4836149|4864500|4847565|4847602|-\n",
      "ctttattaccctggctttctgccactgcgctgctgca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2187913, 2198767)), (2195549, 2195588), '-'), (('G000299455', (766098, 766137)), (766098, 766137), '-'), (('G000299455', (1855036, 1876538)), (1866559, 1866598), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2187913|2198767|2195549|2195588|-\n",
      "tttatcagaaagaggttcctcctggcccgtttgcgatta-\n",
      ">G000299455|766098|766137|766098|766137|-\n",
      "tttatcaagaaaatgttcctcctggcccatttgctatta-\n",
      ">G000299455|1855036|1876538|1866559|1866598|+\n",
      "-ttatcagaaagaggttcctcctggcccgttcgcgattac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (905848, 907049)), (906710, 906744), '+'), (('G000299455', (2809699, 2818013)), (2815870, 2815904), '+'), (('G000299455', (2927401, 2931337)), (2930998, 2931032), '+')]\n",
      ">G000006925|905848|907049|906710|906744|+\n",
      "gttcaaaccagttgtctgtgtcgtaatcagcatc\n",
      ">G000299455|2809699|2818013|2815870|2815904|+\n",
      "gttcaaaccagttttcggtatcataatcagcatc\n",
      ">G000299455|2927401|2931337|2930998|2931032|+\n",
      "gttcaaaccagttgtcggtgtcgtaatcagcatc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4256570, 4288375)), (4280682, 4280719), '+'), (('G000299455', (1985988, 1986025)), (1985988, 1986025), '+'), (('G000299455', (3150481, 3150518)), (3150481, 3150518), '+'), (('G000299455', (4795454, 4823864)), (4819566, 4819603), '+')]\n",
      ">G000006925|4256570|4288375|4280682|4280719|+\n",
      "gcggttgcccggcttcatttggcccgggggtatccag\n",
      ">G000299455|1985988|1986025|1985988|1986025|+\n",
      "gcggttgtccggcctcattagggccgggggtatccag\n",
      ">G000299455|3150481|3150518|3150481|3150518|+\n",
      "gcggctggccggcttcattagggccgggggtatccag\n",
      ">G000299455|4795454|4823864|4819566|4819603|+\n",
      "gcggttgcccggcttcatttggcccgggggtatccag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4238902, 4247776)), (4242551, 4242587), '-'), (('G000299455', (540757, 615358)), (542981, 543017), '+'), (('G000299455', (4912548, 4921423)), (4917737, 4917773), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\n",
      "Reallocating..done. *alloclen = 1196\n",
      "\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  f\b\b\r",
      "STEP    28 / 28  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  f\b\b\r",
      "STEP    28 / 28  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4238902|4247776|4242551|4242587|-\n",
      "ccatgttgatatcgataatctgcgcgccgttttcca-\n",
      ">G000299455|540757|615358|542981|543017|+\n",
      "-catattgatatcaataatctgggcaccgctttccac\n",
      ">G000299455|4912548|4921423|4917737|4917773|+\n",
      "-catgttgatatcgataatctgcgcaccgttttccac\n",
      "\n",
      "\n",
      "29 [(('G000006925', (251358, 252232)), (252085, 252182), '-'), (('G000006925', (315068, 315858)), (315119, 315216), '+'), (('G000006925', (332059, 337870)), (332110, 332207), '+'), (('G000006925', (493019, 493813)), (493070, 493167), '+'), (('G000006925', (509301, 510374)), (509352, 509449), '+'), (('G000006925', (724190, 725876)), (724241, 724338), '+'), (('G000006925', (1094884, 1095679)), (1094935, 1095032), '+'), (('G000006925', (1099363, 1100153)), (1100005, 1100102), '-'), (('G000006925', (1411510, 1412350)), (1411561, 1411658), '+'), (('G000006925', (1883764, 1885448)), (1883815, 1883912), '+'), (('G000006925', (1956633, 1961076)), (1956684, 1956781), '+'), (('G000006925', (2084425, 2085215)), (2085067, 2085164), '-'), (('G000006925', (2086041, 2087310)), (2087163, 2087260), '-'), (('G000006925', (2236395, 2237421)), (2237273, 2237370), '-'), (('G000006925', (3065287, 3076770)), (3074814, 3074911), '+'), (('G000006925', (3812816, 3813607)), (3812867, 3812964), '+'), (('G000006925', (4639759, 4640549)), (4639810, 4639907), '+'), (('G000006925', (4650084, 4650878)), (4650730, 4650827), '-'), (('G000006925', (4656735, 4658205)), (4658057, 4658154), '-'), (('G000006925', (4673259, 4674053)), (4673905, 4674002), '-'), (('G000006925', (4679888, 4680760)), (4679939, 4680036), '+'), (('G000006925', (4750261, 4751054)), (4750906, 4751003), '-'), (('G000006925', (4753880, 4754990)), (4754248, 4754345), '+'), (('G000006925', (4760730, 4761525)), (4761377, 4761474), '-'), (('G000006925', (4793569, 4794365)), (4794217, 4794314), '-'), (('G000299455', (906289, 909876)), (906941, 907038), '-'), (('G000299455', (4752701, 4766669)), (4753353, 4753450), '-'), (('G000299455', (4771847, 4785261)), (4779758, 4779855), '+'), (('G000299455', (5275700, 5279438)), (5278539, 5278636), '+')]\n",
      ">G000006925|251358|252232|252085|252182|-\n",
      "tgacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgcca--\n",
      ">G000006925|315068|315858|315119|315216|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggccgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|332059|337870|332110|332207|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|493019|493813|493070|493167|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|509301|510374|509352|509449|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcatacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|724190|725876|724241|724338|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|1094884|1095679|1094935|1095032|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggactttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|1099363|1100153|1100005|1100102|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggccgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|1411510|1412350|1411561|1411658|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|1883764|1885448|1883815|1883912|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|1956633|1961076|1956684|1956781|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|2084425|2085215|2085067|2085164|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|2086041|2087310|2087163|2087260|-\n",
      "tgacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtccgataaccacca--\n",
      ">G000006925|2236395|2237421|2237273|2237370|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggccgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|3065287|3076770|3074814|3074911|+\n",
      "--acggtttacgcggtcgcctgcggcaaaggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|3812816|3813607|3812867|3812964|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|4639759|4640549|4639810|4639907|+\n",
      "--atggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|4650084|4650878|4650730|4650827|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|4656735|4658205|4658057|4658154|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccaccat-\n",
      ">G000006925|4673259|4674053|4673905|4674002|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|4679888|4680760|4679939|4680036|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      ">G000006925|4750261|4751054|4750906|4751003|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|4753880|4754990|4754248|4754345|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgacggtagtgtggaccttt\n",
      "ttaccccggagaacaccggcaagccccataaccgccatg\n",
      ">G000006925|4760730|4761525|4761377|4761474|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000006925|4793569|4794365|4794217|4794314|-\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcatacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000299455|906289|909876|906941|907038|-\n",
      "tgacggtttacgcggtcgcctgcggcaaaggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgcca--\n",
      ">G000299455|4752701|4766669|4753353|4753450|-\n",
      "tgacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgcca--\n",
      ">G000299455|4771847|4785261|4779758|4779855|+\n",
      "-gacggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccat-\n",
      ">G000299455|5275700|5279438|5278539|5278636|+\n",
      "--acggtttacgcggtcgcctgcggcaacggctttccggctgatggtcgtacggaccttt\n",
      "ttaccccggagaacaccggcaagtcccataaccgccatg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4289701, 4289725), '-'), (('G000299455', (1030502, 1031792)), (1031196, 1031220), '+'), (('G000299455', (4836149, 4864500)), (4860668, 4860692), '+')]\n",
      ">G000006925|4288513|4315006|4289701|4289725|-\n",
      "gatcaaaccgacaattgagatacc-\n",
      ">G000299455|1030502|1031792|1031196|1031220|+\n",
      "-attaagccgataatcgaaattgcg\n",
      ">G000299455|4836149|4864500|4860668|4860692|+\n",
      "-atcaaaccgacaattgagataccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2286341, 2286711), '-'), (('G000299455', (45873, 46893)), (46523, 46893), '+'), (('G000299455', (1752647, 1813107)), (1791499, 1791869), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2242494|2346249|2286341|2286711|-\n",
      "cgcctcacgtccggttttgtcggcatgttcacgggcaagggcaaacatttgcgggttagc\n",
      "ggtcgagccaaagctgctaagaaagacgccaacgaatagcaaaacaaagtagttgcgatt\n",
      "ccaagcaaaaagggtgcaggccagcacgcctaacaggcagcaaaagacaatcagcgattt\n",
      "gcgatcgccgcgcttatcagagcgcccagcgagaaactgacttaccagaatcccaatgac\n",
      "agcgctgccggtaaagaagaatcccaccatcgccggacgggcatgtacttcatcggtaag\n",
      "gaaaatactgagtgtcggggtttgcagagcacccgcaataccggtgagaaaggcaacgat\n",
      "taaaaacgcc-\n",
      ">G000299455|45873|46893|46523|46893|+\n",
      "-gtctcacggcctgttctgtcggcgtgttcacgggcgagggcgaacatttgcgggtttgc\n",
      "ggtggaagcaaaactactcagaagtacgcccgttgagaggagaatgaagtagttgcgatt\n",
      "ccacgcaaaaagcgtgcaggccagcactccaaataagcaacatagcagaatcagtaattt\n",
      "acggtcgccttgtttatcggagtgccttgccagaaattgactgaccagaattcccataat\n",
      "agcgctaccggtgaagaaaaaacctaccattataggacgggctttcagttcatctgcgag\n",
      "gaatatacttagggtaggagtctgaagagcgcccgcaatacccgtcagaaaggcgacaag\n",
      "taaaaatgccg\n",
      ">G000299455|1752647|1813107|1791499|1791869|+\n",
      "-gcctcacgtccggttttgtcggcatgttcacgggcaagggcaaacatttgcgggttagc\n",
      "ggtcgagccaaagctgctaagaaagacgccaacgaatagcaaaacaaagtagttgcgatt\n",
      "ccaggcaaaaagggtgcaggccagcacgcctaacaggcagcaaaagacaatcagcgattt\n",
      "gcgatcgccgcgcttatcagagcgcccggcgagaaactgacttaccagaatcccaatgac\n",
      "agcgctgccggtaaagaagaatcccaccatcgccggacgagcatgtacttcatcagtaag\n",
      "aaaaatactgagtgtcggggtttgcagagcgcccgcaataccggtgagaaaggcaacgat\n",
      "taaaaacgccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4490645, 4497023)), (4491020, 4491061), '-'), (('G000299455', (3918464, 3964500)), (3926764, 3926805), '-'), (('G000299455', (4688506, 4712288)), (4711872, 4711913), '+')]\n",
      ">G000006925|4490645|4497023|4491020|4491061|-\n",
      "ccgaacaaggcgtttattaccgagcagtttatcaactggtc-\n",
      ">G000299455|3918464|3964500|3926764|3926805|-\n",
      "ccgaacaaagcgtttgttaccgagcgtctgatcaactggtc-\n",
      ">G000299455|4688506|4712288|4711872|4711913|+\n",
      "-cgaacaaggcgtttattaccgagcagtttatcaactggtcg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2560406, 2561162)), (2560660, 2560742), '+'), (('G000006925', (4633099, 4633840)), (4633504, 4633586), '-'), (('G000006925', (4668315, 4668899)), (4668564, 4668646), '-'), (('G000006925', (4779995, 4780327)), (4779995, 4780076), '+'), (('G000299455', (5017408, 5018168)), (5017828, 5017910), '-')]\n",
      ">G000006925|2560406|2561162|2560660|2560742|+\n",
      "-tcccggcacgtttgatcggttcggggaccaaaagtgtaggccagtacaccgcctgtttt\n",
      "ggtgttgtacgcgtacccgagcc\n",
      ">G000006925|4633099|4633840|4633504|4633586|-\n",
      "ttcccgacaggtctcatcattccgtggcccgaaagtgtaagccaacacaccgccagtttt\n",
      "agtgttgtacgcataccagagc-\n",
      ">G000006925|4668315|4668899|4668564|4668646|-\n",
      "ttcccgacaggtctcatcattccgtggcccgcaagtgtaagccaacacaccgccagtttt\n",
      "agtgttgtacgcataccagggc-\n",
      ">G000006925|4779995|4780327|4779995|4780076|+\n",
      "-tcccgacaggtctcatcattccgtggcccgaaagtgtaagccaacacaccgccagtttt\n",
      "agtgttgtacgaataccagagc-\n",
      ">G000299455|5017408|5018168|5017828|5017910|-\n",
      "ttcacgacaggtttcgtcagtccgggggccaaaagtgtaggccagaacaccaccagtttt\n",
      "cgtgttgtacacgtaccagagc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3593248, 3593278), '-'), (('G000299455', (335504, 377237)), (342778, 342808), '+'), (('G000299455', (3554411, 3565644)), (3558551, 3558581), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3559544|3600494|3593248|3593278|-\n",
      "ggggagatttttggattccttggttcgaac-\n",
      ">G000299455|335504|377237|342778|342808|+\n",
      "-gggagatttttggttttcttggttcgaacg\n",
      ">G000299455|3554411|3565644|3558551|3558581|+\n",
      "-gggagatttttggtttgctggggccaaacg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3898713, 3912668)), (3899807, 3899844), '+'), (('G000299455', (31186, 43870)), (32280, 32317), '+'), (('G000299455', (5194758, 5202769)), (5199924, 5199961), '-')]\n",
      ">G000006925|3898713|3912668|3899807|3899844|+\n",
      "-ccaccgatgtgggtcagcatcagatgtggaccgcgca\n",
      ">G000299455|31186|43870|32280|32317|+\n",
      "-ccaccgatgtgggtcagcatcagatgtggaccgcgca\n",
      ">G000299455|5194758|5202769|5199924|5199961|-\n",
      "accacagatgtggggcagcaccagatgtgggccgcgc-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3097121, 3097173), '-'), (('G000299455', (47809, 50193)), (49684, 49736), '+'), (('G000299455', (857924, 871846)), (858380, 858432), '+'), (('G000299455', (3140498, 3142451)), (3142399, 3142451), '+'), (('G000299455', (4771847, 4785261)), (4782757, 4782809), '-')]\n",
      ">G000006925|3083675|3099254|3097121|3097173|-\n",
      "tggtgatgggagacgggcagcccgatgcctcccgtacgtgagtgtcgggtaa-\n",
      ">G000299455|47809|50193|49684|49736|+\n",
      "-ggtgacgggagacgggcagcccgatgcctcccgtacgtgagtgtcgggtaat\n",
      ">G000299455|857924|871846|858380|858432|+\n",
      "-ggtgatgggagacgggcagcccgatgcctcccgtacgtgagtgtcgggtaat\n",
      ">G000299455|3140498|3142451|3142399|3142451|+\n",
      "-ggtgacgggagacgggcagcatgatgcctcccgtacatgtgtgtcgggtaat\n",
      ">G000299455|4771847|4785261|4782757|4782809|-\n",
      "tggtgatgggagacgggcagcccgatgcctcccgtacgtgagtgtcgggtaa-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1607351, 1607378)), (1607351, 1607376), '+'), (('G000006925', (1871239, 1871266)), (1871241, 1871266), '-'), (('G000006925', (4640843, 4640872)), (4640847, 4640872), '-'), (('G000006925', (4652860, 4653326)), (4652955, 4652979), '-'), (('G000006925', (4664324, 4664351)), (4664326, 4664351), '-'), (('G000006925', (4684418, 4684875)), (4684765, 4684789), '+'), (('G000006925', (4744587, 4744895)), (4744871, 4744895), '-'), (('G000006925', (4820115, 4820143)), (4820115, 4820138), '+'), (('G000299455', (5295815, 5296178)), (5295821, 5295845), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1607351|1607378|1607351|1607376|+\n",
      "-tttgcagccgtcaggctgccgtggt\n",
      ">G000006925|1871239|1871266|1871241|1871266|-\n",
      "atttgcagccgtcaggctgccgtgg-\n",
      ">G000006925|4640843|4640872|4640847|4640872|-\n",
      "atttgcagccgtcaggctgccgtgg-\n",
      ">G000006925|4652860|4653326|4652955|4652979|-\n",
      "atttgcagccgccaggctgccgtg--\n",
      ">G000006925|4664324|4664351|4664326|4664351|-\n",
      "atttgcagccgtcaggctgccgtgg-\n",
      ">G000006925|4684418|4684875|4684765|4684789|+\n",
      "-tttgcagccgccaggctgccgtgg-\n",
      ">G000006925|4744587|4744895|4744871|4744895|-\n",
      "atttgcagccgtcaggctgccgtg--\n",
      ">G000006925|4820115|4820143|4820115|4820138|+\n",
      "-tttgcagccgtcaggctgccgtg--\n",
      ">G000299455|5295815|5296178|5295821|5295845|-\n",
      "atttgcagcctccaggctgccgtg--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (728729, 728960), '+'), (('G000299455', (2174641, 2181396)), (2177126, 2177357), '+'), (('G000299455', (2910238, 2916987)), (2914271, 2914502), '-')]\n",
      ">G000006925|726235|732999|728729|728960|+\n",
      "-ggaaaccacgcagttattccgtacgcgttttaaagccgtgagtccgaaacgggtggaca\n",
      "cgccaggacacggtatggggaaccactttctgcgggccggggttgaggtcgatcggtatg\n",
      "ggcgggcgattgcgtaccatatctgtgaggatgattttccgttctctggtgccggacgct\n",
      "gggagcggatcccgcgtgagcttcccaccgggcgtccggccatgctgcatat\n",
      ">G000299455|2174641|2181396|2177126|2177357|+\n",
      "-ggagagcacgcaactgtttcgtacgcgttttaaagccgtgagtccgaaacgggtggaca\n",
      "cgccaggacacggtatcgggaaccgttttctgcgggccggtgtggaggtcgatcgatatg\n",
      "gccgtgccgttgcgtaccatatctgtgaggatgattttcctcgctccgggagtggacgat\n",
      "gggaacggatcccgcgtgaacttcccaccgggcgtccggccatgctgcatat\n",
      ">G000299455|2910238|2916987|2914271|2914502|-\n",
      "cggagagcacgcaactgtttcgtacgcgttttaaagccgtgagtccgaaacgggtggaca\n",
      "cgccaggacacggtatcgggaaccgttttctgcgggccggtgtggaggttgatcgatatg\n",
      "gccgtgccgttgcgtaccatatctgtgaggatgattttcctcgctccgggagtggacgat\n",
      "gggaacggatcccgcgtgaacttcccaccgggcgtccggccatgctgcata-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3018127, 3018770), '+'), (('G000299455', (971316, 975991)), (974011, 974654), '-'), (('G000299455', (1477869, 1484878)), (1479281, 1479924), '+')]\n",
      ">G000006925|3016790|3036697|3018127|3018770|+\n",
      "-gaccgtcgatgtcgcgaataacgtgccagccgtaagcttcgaaacgcattgctgtgtcg\n",
      "tcggtgaaccagccttcaacgtgaccatcgatagaaataccgttgtcatcgtagaacgca\n",
      "atcagtttacccagcttcagcgtacccgccagagagcaaacttcgtgggagatgccttcc\n",
      "atcatgcagccgtcgcccatgaaggcgtaggtgtagtggtcgacaatgtcgtggcccgga\n",
      "cggttaaactgcgccgccagcgttttttctgcaatcgccataccgactgcgttggcaata\n",
      "ccctgacccagcggaccggtggtggtttccacaccagcggtataacctacttccggatgg\n",
      "cccggagttttagagtgcagctgacggaagtttttcagttcttccatcggcagatcgtaa\n",
      "ccggtgaggtgcagcaggctgtagatcagcatggagccgtggccgttggacagcacgaag\n",
      "cggtcacggtcagcccaggacggattctgcgggttgtgtttcaggaaatcacgccacagg\n",
      "acttcggcaatgtcagccatacccataggggcacccgggtgaccggatttggctttctgt\n",
      "actgcgtccatgctcagcgcacgaatagcattggcaagctcttt\n",
      ">G000299455|971316|975991|974011|974654|-\n",
      "tgaccgtcgatgtcgcgaataacgtgccagccgtaagcttcgaaacgcattgcggtgtcg\n",
      "tcggtgaaccagccttcaacgtgaccatcaatagagataccgttgtcatcgtagaacgca\n",
      "atcagtttacccagcttcagcgtacccgccagagagcaaacttcgtgggagatgccttcc\n",
      "atcatgcagccgtcgcccatgaaggcataggtgtagtggtcgacaatgtcgtggcccgga\n",
      "cggttaaactgcgccgccagcgttttttctgcaatcgccataccgactgcgttggcaata\n",
      "ccctgacccagcggaccggtggtggtttccacaccagcggtgtaacccacttccgggtga\n",
      "cccggagttttagagtgcagctgacggaagtttttcagttcttccatcggcagatcgtaa\n",
      "ccggtgaggtgcagcaggctgtagatcagcatggagccgtggccgttggacagcacgaag\n",
      "cggtcacggtcagcccaggacggattctgcgggttgtgtttcaggaaatcacgccacagg\n",
      "acttcggcaatgtcagccatacccataggggcacccgggtgaccggatttggctttctgt\n",
      "actgcgtccatgctcagcgcacgaatagcattggcaagctctt-\n",
      ">G000299455|1477869|1484878|1479281|1479924|+\n",
      "-gaccgtcgatttcatggatcacatgccagtgataggcttcaaaacgttttgccgtatcg\n",
      "tcggtaaaccagccttctgtttcaccgtcgatggaaataccgttgtgatcgtaaaaacca\n",
      "atcagcttgcccagtcccagcgtgcctgccagcgaacagacttcgtgggaaataccttcc\n",
      "atcaggcagccgtcgcccataaacacataggtgaagtgatcgacgatctcatggtctggc\n",
      "tggttaaactgcgccgccaatgtacgttccgctatcgccagcccgacggcgttcgccaaa\n",
      "ccttgtccaagcgggccggtggtggtttcaacgcctggcgtatagccaatctccgggtgg\n",
      "cctggggttttcgaatgcaactgacggaagttcttcagttcttccagcggcaggtcgtaa\n",
      "ccggtcagatgtagcaaactgtagagcagcatcgacgcgtgaccgttggaaagaataaag\n",
      "cggtcgcgatcataccaggttgggtcggtagggttatgtttaagaaaatcgttccacagc\n",
      "acttcggcaatatcagccatgcccatcggcgcgccgggatgaccagaattggctttttgt\n",
      "accgcatccatactgagtgcgcgaatcgcattggcaaggtcttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4633099, 4633840)), (4633103, 4633475), '+'), (('G000006925', (4681207, 4681605)), (4681211, 4681583), '+'), (('G000299455', (3175907, 3176990)), (3176597, 3176968), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4633099|4633840|4633103|4633475|+\n",
      "ggccgttataaaacgacgcttcagcgtggagtaatggccccccgcctgctcatcattgat\n",
      "gaaataggctatctgccgttcagtcaggaagaagcaaaactgttcttccaggtcatcgct\n",
      "aaacgttacgaaaagagcgcaatgatcctgacatccaatctaccgttcgggcagtgggat\n",
      "caaacgttcgccggtgatgcagcactgacctcagcgatgctgggccgtatcttacaccac\n",
      "tcacatgtcgttcaaatcaaaggagaaagctatcgactcagacagaaacgaaaggccggg\n",
      "gttatagcagaagctaatcctgagtaaaacggtggatcaatattgggccgttggtggaga\n",
      "tataagtggatc\n",
      ">G000006925|4681207|4681605|4681211|4681583|+\n",
      "ggccgttataaaacgacgcttcagcgtggagtaatggccccccgcctgctcatcattgat\n",
      "gaaataggctatctgccgttcagtcaggaagaagcaaaactgttcttccaggtcatcgct\n",
      "aaacgttacgaaaagagcgcaatgatcctgacatccaatctaccgttcgggcagtgggat\n",
      "caaacgttcgccggtgatgcagcactgacctcagcgatgctgggccgtatcttacaccac\n",
      "tcacatgtcgttcaaatcaaaggagaaagctatcgactcagacagaaacgaaaggccggg\n",
      "gttatagcagaagctaatcctgagtaaaacggtggatcaatattgggccgttggtggaga\n",
      "tataagtggatc\n",
      ">G000299455|3175907|3176990|3176597|3176968|+\n",
      "ggccgttataaagcgacgcttcagcgtggagtaatgg-cccccgcctgctcatcattgat\n",
      "gaaataggctatctgccgttcagtcaggaagaagcaaagctgttcttccaggtcatcgct\n",
      "aaacgttacgaaaagagcgcaatgatcctgacatccaatctgccgttcgggcagtgggat\n",
      "caaacgttcgccggtgatgcagcactgacctcagcgatgctggaccgtatcttacaccac\n",
      "tcacatgtcgttcaaatcaaaggagaaagctatcgactcagacagaaacgaaaggccggg\n",
      "gttatagcagaagctaatcctgagtaaaacggtggatcaatattgggccgttggtggaga\n",
      "tataagtggatc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (727659, 727884), '+'), (('G000299455', (2174641, 2181396)), (2176052, 2176277), '+'), (('G000299455', (2910238, 2916987)), (2915351, 2915576), '-')]\n",
      ">G000006925|726235|732999|727659|727884|+\n",
      "-gaaaacccgcaatcagcggggtgtgtatctgtgtgaggtggggacggataccgcaaaag\n",
      "aaattctctatgcccgaatgaaggccgatcccacccctgtgaatgaggccacatcgtatg\n",
      "ccatccgctttcctgataatccggagattttttcgcagacagaggcacaacaactggtgg\n",
      "cggaggaactggtggaaaagtgggaaaaaggaaagatgcgtctgtt\n",
      ">G000299455|2174641|2181396|2176052|2176277|+\n",
      "-aaaaacccgcaatcagcggggcgtgtatctgtgtgaagtggggacggacaccgcaaaag\n",
      "aaattctctatgcccgtatgaaagccgatcccacgcctgtggatgaagccacgtcgtatg\n",
      "ccatccgttttcctgatgatccggagattttttcgcagacagaggcgcagcaactggtcg\n",
      "cggaagagcttgtggagaagtgggaaaaaggaaagatgcgtctgct\n",
      ">G000299455|2910238|2916987|2915351|2915576|-\n",
      "cgaaaacccgcaatcagcggggcgtgtatctgtgtgaagtggggacggacaccgcaaaag\n",
      "aaattctctatgcccgtatgaaagccgatcccacgcctgcggatgaagccacgtcgtatg\n",
      "ccatccgttttcctgatgatccggagattttttcgcagacagaggcgcagcaactggtgg\n",
      "cggaagagcttgtggagaagtgggaaaaaggaaagatgcgtctgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2333515, 2333576), '-'), (('G000299455', (1720965, 1752624)), (1745552, 1745613), '+'), (('G000299455', (3386242, 3399566)), (3397589, 3397650), '+')]\n",
      ">G000006925|2242494|2346249|2333515|2333576|-\n",
      "aacggtgacggcgttggcggttctattacttatgattacgaaggcttcggtatcggtgct\n",
      "g-\n",
      ">G000299455|1720965|1752624|1745552|1745613|+\n",
      "-acggcgacggcgttggcggttctatcacttatgattacgaaggcttcggtatcggtgct\n",
      "gc\n",
      ">G000299455|3386242|3399566|3397589|3397650|+\n",
      "-acggcgacggtgttggcggttctatcagctacgaatacgaaggctttggtatcgttggt\n",
      "gc\n",
      "\n",
      "\n",
      "8 [(('G000006925', (2560406, 2561162)), (2560428, 2560454), '+'), (('G000006925', (3083675, 3099254)), (3083931, 3083957), '-'), (('G000006925', (4633099, 4633840)), (4633792, 4633818), '-'), (('G000006925', (4668315, 4668899)), (4668852, 4668878), '-'), (('G000006925', (4765915, 4766031)), (4765978, 4766005), '-'), (('G000006925', (4822840, 4823055)), (4822862, 4822887), '+'), (('G000299455', (857924, 871846)), (871566, 871592), '+'), (('G000299455', (5017408, 5018168)), (5018115, 5018141), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2560406|2561162|2560428|2560454|+\n",
      "--ttcgataaacgctccgatcacttttt\n",
      ">G000006925|3083675|3099254|3083931|3083957|-\n",
      "-tttcgatgaatgacccaataattttt-\n",
      ">G000006925|4633099|4633840|4633792|4633818|-\n",
      "-tttcaatgaaggagccgatgactttt-\n",
      ">G000006925|4668315|4668899|4668852|4668878|-\n",
      "ttttcaatgaaggaaccgatgacttt--\n",
      ">G000006925|4765915|4766031|4765978|4766005|-\n",
      "ttttcgataaacgcaccgataactttt-\n",
      ">G000006925|4822840|4823055|4822862|4822887|+\n",
      "--ttcaatgaaggagccgatgactttt-\n",
      ">G000299455|857924|871846|871566|871592|+\n",
      "--ttcgatgaatgacccaataacttttt\n",
      ">G000299455|5017408|5018168|5018115|5018141|-\n",
      "-tttcgataaaagcaccgatgactttt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (313003, 313138)), (313005, 313138), '+'), (('G000006925', (327498, 328010)), (327498, 327630), '-'), (('G000299455', (2525864, 2526376)), (2526243, 2526376), '+')]\n",
      ">G000006925|313003|313138|313005|313138|+\n",
      "ccgggtttatccatgccacccggcgctgcatatttatttgttgccattcaaggttgatga\n",
      "tgttcgagcgtcttaagcctgttgccagtgcaaattcaacaacagactttaatggctccg\n",
      "gacattcatcaat\n",
      ">G000006925|327498|328010|327498|327630|-\n",
      "ccgggtttatccatgccacccggcgctgcatatctatttgttgccattcaaggttgatga\n",
      "tgttcgagcgtcttaagcctgttgccagtgcaaattcaacaacagactttaatggctccg\n",
      "gacattcatcaa-\n",
      ">G000299455|2525864|2526376|2526243|2526376|+\n",
      "cagggtttacccaggcaacacgtcgctgcatatcgatttgttgccactccagattaataa\n",
      "tgtttgatctcctcaggccagttgccagtgcaaatttaactacagatttcaggggatcag\n",
      "aacatgcatcaat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4584856, 4587476)), (4586118, 4586139), '+'), (('G000299455', (3274999, 3275020)), (3274999, 3275020), '-'), (('G000299455', (4470233, 4472853)), (4471570, 4471591), '-')]\n",
      ">G000006925|4584856|4587476|4586118|4586139|+\n",
      "-gaaagcggtgaaagcggcaat\n",
      ">G000299455|3274999|3275020|3274999|3275020|-\n",
      "cgaaagcggtgaaagcggcaa-\n",
      ">G000299455|4470233|4472853|4471570|4471591|-\n",
      "cgaaagcggtgaaagcggcaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2252210, 2252230), '-'), (('G000006925', (3229824, 3229896)), (3229824, 3229844), '+'), (('G000299455', (1819843, 1851664)), (1826278, 1826298), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2242494|2346249|2252210|2252230|-\n",
      "tagctctttatcaataggcc-\n",
      ">G000006925|3229824|3229896|3229824|3229844|+\n",
      "-agctctttacaaaaagaccg\n",
      ">G000299455|1819843|1851664|1826278|1826298|+\n",
      "-agctctttatcaataggccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3146034, 3178603)), (3149386, 3149496), '-'), (('G000299455', (698804, 702950)), (701244, 701354), '+'), (('G000299455', (781497, 800022)), (796783, 796893), '+')]\n",
      ">G000006925|3146034|3178603|3149386|3149496|-\n",
      "acccacagcaatccacttaaccagttgaaaaactggaagcggcggtttggcagaccggaa\n",
      "attcccgccatggttggcagcagcgtacggacaaatgcaagaaaacgtcc-\n",
      ">G000299455|698804|702950|701244|701354|+\n",
      "-cccacagcagaccgctcatccagttgaaaaactgaaagcgcgcgttattcagccctgat\n",
      "aacccggcaatcgtcggcagcagtgttctgacaaacgcaataaagcgacca\n",
      ">G000299455|781497|800022|796783|796893|+\n",
      "-cccacagcaatccgcttaaccagttgaaaaactggaaacggcggtttggcagaccggaa\n",
      "attcccgccatggttggcagcagcgtacggacaaatgcaagaaaacgtcca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4666868, 4668030)), (4666904, 4667083), '-'), (('G000006925', (4747716, 4747948)), (4747737, 4747915), '+'), (('G000299455', (886351, 887494)), (887283, 887461), '+'), (('G000299455', (5346044, 5347202)), (5346989, 5347166), '+')]\n",
      ">G000006925|4666868|4668030|4666904|4667083|-\n",
      "catcagcatggcagtcagtctgcgggcataatttttatcgtgtgttttatggatggcttt\n",
      "ctgcatcaggcgtcgttcgtcacgggaaattggtgctatgatcggcattgctcagtccgg\n",
      "ttggtgatttgttttgatttggcgattgatcagatcgcacaatccgggctaagttccct-\n",
      ">G000006925|4747716|4747948|4747737|4747915|+\n",
      "-atcagtatgacggtgaatcgccggtcgtagtttttgttctttgttttatagggagcttt\n",
      "ctgcatcaggcatcgttcatcacggggtattggtgcaatgctcggcatcactcagtccgg\n",
      "ctggtgatttgtgatg-tttggcaatttatctgattacataatccggactgagtcccccc\n",
      ">G000299455|886351|887494|887283|887461|+\n",
      "-atcagcatggcggtgagccgccgggcgtggtttttgtcctttgttttatggatagtttt\n",
      "ctgcatcaggcgtcgttctgttctggggattggtgttatgatcggcatcgctcagtccgg\n",
      "ttggtgattttttctt-tttggcgattgatcagatcgcacaatccgggctgagttccctc\n",
      ">G000299455|5346044|5347202|5346989|5347166|+\n",
      "-atcagcatggcggtgagccgccgggcgtggtttttgtcctttgttttat-gatagtttt\n",
      "ctgcatcaggcatcgttcggttctggggattggtgctatgatcggcatcgctcagtccgc\n",
      "ttggtaattttttctt-ttgggtgattgatcagatcgcacaatccgggctgagttccctc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2791767, 2791828), '-'), (('G000006925', (2770832, 2816659)), (2791908, 2791969), '-'), (('G000299455', (1253944, 1254160)), (1254015, 1254075), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2770832|2816659|2791767|2791828|-\n",
      "tcttacttgatatggctttagtagcggtatcaaaaaatctgcagtaaagtaagtttcccg\n",
      "a--\n",
      ">G000006925|2770832|2816659|2791908|2791969|-\n",
      "tcttacttgatacggctttagtagcggtatcaaaaaatctgcagtaaagtaagtttcccg\n",
      "a--\n",
      ">G000299455|1253944|1254160|1254015|1254075|+\n",
      "-cttacttgatatggctttagtagcgatatc--aatatcagcagtaaaataaattttccc\n",
      "gat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (2045115, 2046271)), (2045244, 2045340), '+'), (('G000006925', (2227521, 2228997)), (2227970, 2228066), '+'), (('G000299455', (2367337, 2367480)), (2367337, 2367433), '+'), (('G000299455', (2892839, 2893945)), (2892933, 2893029), '+'), (('G000299455', (2893982, 2894125)), (2893982, 2894078), '+'), (('G000299455', (3581924, 3582067)), (3581924, 3582020), '+')]\n",
      ">G000006925|2045115|2046271|2045244|2045340|+\n",
      "gccactcaatatcaggtgcagttgatgtatcaacacggttcagcaacacccgatacttct\n",
      "tccaggcttccagcaacgagttttcttcctccgttg\n",
      ">G000006925|2227521|2228997|2227970|2228066|+\n",
      "gccactcaatatcaggtgcagttgatgtatcaacacggttcagcaacacccgatacttct\n",
      "tccaggcttccagcaacgagttttcttcctccgttg\n",
      ">G000299455|2367337|2367480|2367337|2367433|+\n",
      "gccactcaacatccgatgctactgttgtatcaacacgattcaacaatacccgatatgtct\n",
      "tccatgcagtcagtaacgatgcctcttcctccgtcg\n",
      ">G000299455|2892839|2893945|2892933|2893029|+\n",
      "gccactcaatatcaggtacagttgatgtatcaacacgattcaacaatacccgatatttat\n",
      "tccatgcctccagcaacgatctttcttcctccgttg\n",
      ">G000299455|2893982|2894125|2893982|2894078|+\n",
      "gccactcaatatcaggtgcagttgatgtatcaacacggttcagcaacacccgatactttt\n",
      "tccaggcttccagcaatgaggtttcttcctccgttg\n",
      ">G000299455|3581924|3582067|3581924|3582020|+\n",
      "gccattcaatatccggtgcagttgttgtattaacacggttcagcaacacccgatacttct\n",
      "tccaggcttccagcaacgaggtttcttcctccgttg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (733538, 733576)), (733538, 733576), '+'), (('G000299455', (2181912, 2181950)), (2181912, 2181950), '+'), (('G000299455', (2909693, 2909731)), (2909693, 2909731), '-')]\n",
      ">G000006925|733538|733576|733538|733576|+\n",
      "-tggatcgtattactccggatgacgggggatgctgttat\n",
      ">G000299455|2181912|2181950|2181912|2181950|+\n",
      "-tggatcgtgtttctccggatgacgggggcagctgttat\n",
      ">G000299455|2909693|2909731|2909693|2909731|-\n",
      "gtggatcgtgtttctccggatgacgggggtagctgtta-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (34451, 34567), '-'), (('G000299455', (2768331, 2768447)), (2768331, 2768447), '-'), (('G000299455', (4406425, 4422836)), (4413486, 4413602), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 31\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 31\r",
      "   10 / 31\r",
      "   20 / 31\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 30  f\b\b\r",
      "STEP     2 / 30  f\b\b\r",
      "STEP     3 / 30  f\b\b\r",
      "STEP     4 / 30  f\b\b\r",
      "STEP     5 / 30  f\b\b\r",
      "STEP     6 / 30  f\b\b\r",
      "STEP     7 / 30  f\b\b\r",
      "STEP     8 / 30  f\b\b\r",
      "STEP     9 / 30  f\b\b\r",
      "STEP    10 / 30  f\b\b\r",
      "STEP    11 / 30  f\b\b\r",
      "STEP    12 / 30  f\b\b\r",
      "STEP    13 / 30  f\b\b\r",
      "STEP    14 / 30  f\b\b\r",
      "STEP    15 / 30  f\b\b\r",
      "STEP    16 / 30  f\b\b\r",
      "STEP    17 / 30  f\b\b\r",
      "STEP    18 / 30  f\b\b\r",
      "STEP    19 / 30  f\b\b\r",
      "STEP    20 / 30  f\b\b\r",
      "STEP    21 / 30  f\b\b\r",
      "STEP    22 / 30  f\b\b\r",
      "STEP    23 / 30  f\b\b\r",
      "STEP    24 / 30  f\b\b\r",
      "STEP    25 / 30  f\b\b\r",
      "STEP    26 / 30  f\b\b\r",
      "STEP    27 / 30  f\b\b\r",
      "STEP    28 / 30  f\b\b\r",
      "STEP    29 / 30  f\b\b\r",
      "STEP    30 / 30  d\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 31\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 31\r",
      "   10 / 31\r",
      "   20 / 31\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 30  f\b\b\r",
      "STEP     2 / 30  f\b\b\r",
      "STEP     3 / 30  f\b\b\r",
      "STEP     4 / 30  f\b\b\r",
      "STEP     5 / 30  f\b\b\r",
      "STEP     6 / 30  f\b\b\r",
      "STEP     7 / 30  f\b\b\r",
      "STEP     8 / 30  f\b\b\r",
      "STEP     9 / 30  f\b\b\r",
      "STEP    10 / 30  f\b\b\r",
      "STEP    11 / 30  f\b\b\r",
      "STEP    12 / 30  f\b\b\r",
      "STEP    13 / 30  f\b\b\r",
      "STEP    14 / 30  f\b\b\r",
      "STEP    15 / 30  f\b\b\r",
      "STEP    16 / 30  f\b\b\r",
      "STEP    17 / 30  f\b\b\r",
      "STEP    18 / 30  f\b\b\r",
      "STEP    19 / 30  f\b\b\r",
      "STEP    20 / 30  f\b\b\r",
      "STEP    21 / 30  f\b\b\r",
      "STEP    22 / 30  f\b\b\r",
      "STEP    23 / 30  f\b\b\r",
      "STEP    24 / 30  f\b\b\r",
      "STEP    25 / 30  f\b\b\r",
      "STEP    26 / 30  f\b\b\r",
      "STEP    27 / 30  f\b\b\r",
      "STEP    28 / 30  f\b\b\r",
      "STEP    29 / 30  f\b\b\r",
      "STEP    30 / 30  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|25217|64488|34451|34567|-\n",
      "catatcgggcatggagcgatcctgcatggttgtgtgattggtcgcgatgcattggtcggg\n",
      "atgaacagcgtaattatggatggcgcggtcattggcgaagagagcattgttgccgc-\n",
      ">G000299455|2768331|2768447|2768331|2768447|-\n",
      "catattggtcatagcgctatccttcatggttgcattatccgccgcaatgcattagtggga\n",
      "atgaacgcggtagtgatggacggcgcggtgattggcgagaacagcattgttggcgc-\n",
      ">G000299455|4406425|4422836|4413486|4413602|+\n",
      "-atatcgggcacggagcgatcctgcatggttgtgtgattggtcgcgatgcattggtcggg\n",
      "atgaacagcgtgattatggatggcgcggtcattggcgaagagagcattgttgccgcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (3007307, 3007340), '+'), (('G000299455', (978228, 988794)), (985521, 985554), '-'), (('G000299455', (2285471, 2312598)), (2287143, 2287176), '+')]\n",
      ">G000006925|2976303|3015537|3007307|3007340|+\n",
      "-caattcgtggtacaggaagcgttagaaacgatg\n",
      ">G000299455|978228|988794|985521|985554|-\n",
      "gcaattcgtggtacaggaagcgttagaaacgat-\n",
      ">G000299455|2285471|2312598|2287143|2287176|+\n",
      "-cagttggtggtgcaggaagcgttggaaacgatg\n",
      "\n",
      "\n",
      "31 [(('G000006925', (251358, 252232)), (251636, 251665), '+'), (('G000006925', (315068, 315858)), (315634, 315663), '-'), (('G000006925', (332059, 337870)), (332625, 332654), '-'), (('G000006925', (493019, 493813)), (493585, 493614), '-'), (('G000006925', (509301, 510374)), (509868, 509897), '-'), (('G000006925', (724190, 725876)), (724757, 724786), '-'), (('G000006925', (1094884, 1095679)), (1095450, 1095479), '-'), (('G000006925', (1099363, 1100153)), (1099557, 1099586), '+'), (('G000006925', (1411510, 1412350)), (1412126, 1412155), '-'), (('G000006925', (1883764, 1885448)), (1884331, 1884360), '-'), (('G000006925', (1956633, 1961076)), (1957200, 1957229), '-'), (('G000006925', (2084425, 2085215)), (2084619, 2084648), '+'), (('G000006925', (2236395, 2237421)), (2236825, 2236854), '+'), (('G000006925', (3065287, 3076770)), (3075330, 3075359), '-'), (('G000006925', (3812816, 3813607)), (3813382, 3813411), '-'), (('G000006925', (4639759, 4640549)), (4640325, 4640354), '-'), (('G000006925', (4650084, 4650878)), (4650282, 4650311), '+'), (('G000006925', (4656735, 4658205)), (4657609, 4657638), '+'), (('G000006925', (4673259, 4674053)), (4673457, 4673486), '+'), (('G000006925', (4679888, 4680760)), (4680455, 4680484), '-'), (('G000006925', (4684418, 4684875)), (4684833, 4684862), '+'), (('G000006925', (4750261, 4751054)), (4750458, 4750487), '+'), (('G000006925', (4753880, 4754990)), (4754766, 4754795), '-'), (('G000006925', (4760730, 4761525)), (4760929, 4760958), '+'), (('G000006925', (4793569, 4794365)), (4793766, 4793798), '+'), (('G000299455', (906289, 909876)), (906489, 906518), '+'), (('G000299455', (912160, 912572)), (912358, 912387), '+'), (('G000299455', (4752701, 4766669)), (4752901, 4752930), '+'), (('G000299455', (4771847, 4785261)), (4780279, 4780308), '-'), (('G000299455', (5275700, 5279438)), (5279059, 5279088), '-'), (('G000299455', (5343200, 5343533)), (5343503, 5343532), '+')]\n",
      ">G000006925|251358|252232|251636|251665|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|315068|315858|315634|315663|-\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|332059|337870|332625|332654|-\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|493019|493813|493585|493614|-\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|509301|510374|509868|509897|-\n",
      "gcgtgtctggg---ttcgccagcatgagcggg-\n",
      ">G000006925|724190|725876|724757|724786|-\n",
      "gcgtgtctggg---gtcgccagcatgagcggg-\n",
      ">G000006925|1094884|1095679|1095450|1095479|-\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|1099363|1100153|1099557|1099586|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|1411510|1412350|1412126|1412155|-\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|1883764|1885448|1884331|1884360|-\n",
      "gcgtgtctggg---gtcgccagcatgagcggg-\n",
      ">G000006925|1956633|1961076|1957200|1957229|-\n",
      "gcgtgtctggg---ttcgccagcatgagcggg-\n",
      ">G000006925|2084425|2085215|2084619|2084648|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|2236395|2237421|2236825|2236854|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|3065287|3076770|3075330|3075359|-\n",
      "gcgtgtctggg---ttcgccagcatgagcggg-\n",
      ">G000006925|3812816|3813607|3813382|3813411|-\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4639759|4640549|4640325|4640354|-\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4650084|4650878|4650282|4650311|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4656735|4658205|4657609|4657638|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4673259|4674053|4673457|4673486|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|4679888|4680760|4680455|4680484|-\n",
      "gcgtgtctggg---gtcgccagcatgagcggg-\n",
      ">G000006925|4684418|4684875|4684833|4684862|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4750261|4751054|4750458|4750487|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000006925|4753880|4754990|4754766|4754795|-\n",
      "-cgtgtctggg---tacgccagcatgagcggga\n",
      ">G000006925|4760730|4761525|4760929|4760958|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000006925|4793569|4794365|4793766|4793798|+\n",
      "-cgtgtctggggtcttcgccagcatgagcggga\n",
      ">G000299455|906289|909876|906489|906518|+\n",
      "-cgtgtcaggg---ttcgccagcatgagcggga\n",
      ">G000299455|912160|912572|912358|912387|+\n",
      "-cgtgtctggg---gtcgccagcatgagcggga\n",
      ">G000299455|4752701|4766669|4752901|4752930|+\n",
      "-cgtgtctggg---ttcgccagcatgagcggga\n",
      ">G000299455|4771847|4785261|4780279|4780308|-\n",
      "gcgtgtctggg---ttcgccagcatgagcggg-\n",
      ">G000299455|5275700|5279438|5279059|5279088|-\n",
      "gcgtgtctggg---ttcgccagcatgagcggg-\n",
      ">G000299455|5343200|5343533|5343503|5343532|+\n",
      "-catgtctggg---tacgccagcatgagcggga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (730963, 730996), '+'), (('G000299455', (2174641, 2181396)), (2179359, 2179392), '+'), (('G000299455', (2910238, 2916987)), (2912236, 2912269), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|730963|730996|+\n",
      "-atgtgaagaggcgaaaggacgtgaacaactggc\n",
      ">G000299455|2174641|2181396|2179359|2179392|+\n",
      "-gtgtgaagaagcgaacggtcgtgaacaactggc\n",
      ">G000299455|2910238|2916987|2912236|2912269|-\n",
      "cgtgtgaagaagcgaacggtcgtgaacaactgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2569729, 2584886)), (2578017, 2578079), '+'), (('G000299455', (1462739, 1472085)), (1469546, 1469608), '-'), (('G000299455', (3918464, 3964500)), (3934252, 3934314), '+')]\n",
      ">G000006925|2569729|2584886|2578017|2578079|+\n",
      "-ccgaaataaagatggcggcatggtgcccttctctgctttcgcgacctcacgctgggaaa\n",
      "cag\n",
      ">G000299455|1462739|1472085|1469546|1469608|-\n",
      "tccgaaataaagatggcggcatggtgcccttctctgctttcgcgacctcacgctgggaga\n",
      "ca-\n",
      ">G000299455|3918464|3964500|3934252|3934314|+\n",
      "-tcgtgctgctgatggtcagatggtgccgttctcggcgttctcctcttctcgttgggagt\n",
      "acg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4642485, 4642537)), (4642485, 4642535), '+'), (('G000006925', (4654617, 4654668)), (4654617, 4654668), '+'), (('G000299455', (4545397, 4545449)), (4545398, 4545449), '-')]\n",
      ">G000006925|4642485|4642537|4642485|4642535|+\n",
      "-taaggcgcggacaggggtatcgcttagtttttcaatttcctatacgtgaa-\n",
      ">G000006925|4654617|4654668|4654617|4654668|+\n",
      "-taaggcgcggacaggggtatcgcttagtttttcaatttcctatacgtgaac\n",
      ">G000299455|4545397|4545449|4545398|4545449|-\n",
      "ataaggtacggacagcagtatcgcttagtttttcaaattcatgtatgtgaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3403726, 3409928)), (3406452, 3406483), '+'), (('G000299455', (515552, 530156)), (529946, 529977), '-'), (('G000299455', (4952297, 4970010)), (4966090, 4966121), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1042\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3403726|3409928|3406452|3406483|+\n",
      "-agtggtgctgatacccagagtcgaactgggg\n",
      ">G000299455|515552|530156|529946|529977|-\n",
      "aagtggtgctgatacccagagtcgaactggg-\n",
      ">G000299455|4952297|4970010|4966090|4966121|+\n",
      "-agtggtgctgataggcagattcgaactgccg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1241906, 1253284)), (1253227, 1253247), '+'), (('G000299455', (77918, 77975)), (77955, 77975), '-'), (('G000299455', (82782, 82839)), (82782, 82802), '+'), (('G000299455', (83942, 83999)), (83979, 83999), '-'), (('G000299455', (2983402, 3007859)), (2996517, 2996537), '-')]\n",
      ">G000006925|1241906|1253284|1253227|1253247|+\n",
      "-tatcattttcatcttgccag\n",
      ">G000299455|77918|77975|77955|77975|-\n",
      "ttatccgtttcagtttgcca-\n",
      ">G000299455|82782|82839|82782|82802|+\n",
      "-tatccgtttcagtttgccat\n",
      ">G000299455|83942|83999|83979|83999|-\n",
      "ttatccgtttcagtttgcca-\n",
      ">G000299455|2983402|3007859|2996517|2996537|-\n",
      "ttatcattttcatcttgcca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (202558, 202706), '-'), (('G000299455', (4235244, 4293324)), (4239519, 4239667), '+'), (('G000299455', (4790757, 4790905)), (4790757, 4790905), '-')]\n",
      ">G000006925|64491|216702|202558|202706|-\n",
      "ccgtgaatatccgtttcaaaaccggggtaatgttgccctacggaacaaagcatcagtaga\n",
      "aaatcgagtaccgtgcggctctctttggtcagcatttctcctggcatcaacagcggtacg\n",
      "cccggtggataaggcaggatcatatttg-\n",
      ">G000299455|4235244|4293324|4239519|4239667|+\n",
      "-cgtgaatatccgtttcaaaaccggggtaatgttgccctacggaacaaagcatcagtaga\n",
      "aaatcgagtaccgtgcggctctctttggtcagcatttctcctggcatcaacagcggtacg\n",
      "cccggtggataaggcaggatcatatttgc\n",
      ">G000299455|4790757|4790905|4790757|4790905|-\n",
      "ccgtgaatatcggtttcaaagcccggatagtgagcgccgatttcacacagcatctgcagg\n",
      "aactccagaaccggacggctttcttcggtgatcatttcacccggcattaccagaggaact\n",
      "cccggcggatacggaaggatcatattgg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4622488, 4624460)), (4622561, 4622585), '+'), (('G000006925', (4689001, 4689359)), (4689063, 4689087), '+'), (('G000299455', (3214776, 3214831)), (3214776, 3214800), '+'), (('G000299455', (5313369, 5315826)), (5313798, 5313822), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4622488|4624460|4622561|4622585|+\n",
      "cgcagatcgcccgtgaaaacggaa\n",
      ">G000006925|4689001|4689359|4689063|4689087|+\n",
      "cgtagatcgcccgtgaaaatggaa\n",
      ">G000299455|3214776|3214831|3214776|3214800|+\n",
      "cgcaactcgctcgtgagcatggga\n",
      ">G000299455|5313369|5315826|5313798|5313822|+\n",
      "cgcagatcgcccgggaaaacggaa\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1209065, 1211067)), (1210354, 1210417), '+'), (('G000006925', (2756889, 2766574)), (2757435, 2757498), '+'), (('G000006925', (2923039, 2950048)), (2948550, 2948613), '+'), (('G000006925', (3101814, 3103074)), (3102361, 3102424), '+'), (('G000006925', (4746555, 4747128)), (4746615, 4746678), '-'), (('G000006925', (4753880, 4754990)), (4754048, 4754110), '-'), (('G000006925', (4795159, 4795487)), (4795241, 4795304), '+'), (('G000299455', (1288645, 1288721)), (1288647, 1288710), '+'), (('G000299455', (3226712, 3229721)), (3229004, 3229067), '+')]\n",
      ">G000006925|1209065|1211067|1210354|1210417|+\n",
      "-agcgttacggtgccccacgcctgacggatgaactgcgtgctcagggttacccctttaac\n",
      "gtaa\n",
      ">G000006925|2756889|2766574|2757435|2757498|+\n",
      "-agcgttacggtgccccacgcctgacggatgaactgcgtgctcagggttacccctttaac\n",
      "gtaa\n",
      ">G000006925|2923039|2950048|2948550|2948613|+\n",
      "-agcgttacggtgccccacgcctgacggatgaactgcgtgctcagggttacccctttaac\n",
      "gtaa\n",
      ">G000006925|3101814|3103074|3102361|3102424|+\n",
      "-agcgttacggtgccccacgcctgacggatgaactgcgtgctcagggttacccctttaac\n",
      "gtaa\n",
      ">G000006925|4746555|4747128|4746615|4746678|-\n",
      "cagcgttacggtgcaccacgcctgacggatgaactgcgtgctcagggttaccactttaac\n",
      "gta-\n",
      ">G000006925|4753880|4754990|4754048|4754110|-\n",
      "cagcgttacggtgccccacgcctgacggatgaactgtgtgctc-tggttaccactttaac\n",
      "gta-\n",
      ">G000006925|4795159|4795487|4795241|4795304|+\n",
      "-agcgttgcggtgcaccacgcctgacggatgaactgcgtgctcagggttaccactttaac\n",
      "gtaa\n",
      ">G000299455|1288645|1288721|1288647|1288710|+\n",
      "-agcgttacggtgcaccacgcctgactgaagaactgcgtgctcagggttaccacttcaac\n",
      "gtaa\n",
      ">G000299455|3226712|3229721|3229004|3229067|+\n",
      "-agcgttacggtgccccacgcctgacggatgaactgcgtgctcagggttacccctttaac\n",
      "gtaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3423912, 3503727)), (3459034, 3459281), '-'), (('G000299455', (434155, 488092)), (478578, 478825), '+'), (('G000299455', (2312634, 2337653)), (2318491, 2318738), '+')]\n",
      ">G000006925|3423912|3503727|3459034|3459281|-\n",
      "tgggatcagcaaggcaaggagtatgtcgatttcgcgggtggcattgcagttacggcgttg\n",
      "ggccattgccatcctgcgctggtgaacgcgttaaaaacccagggcgaaactctgtggcat\n",
      "atcagtaacgttttcaccaatgaaccggcgctgcgtcttgggcgcaaactgattgaggcg\n",
      "acgtttgccgaacgcgtggtgtttatgaactccggcacggaagctaacgaaaccgccttt\n",
      "aaactgg-\n",
      ">G000299455|434155|488092|478578|478825|+\n",
      "-gggatcagcaaggcaaggagtatgtcgatttcgcgggtggcattgcagttacggcgttg\n",
      "ggccattgccatcctgcgctggtgaatgcgttaaaaacccagggcgaaactctgtggcat\n",
      "atcagtaacgttttcaccaatgaaccggcgctgcgtcttgggcgcaaactgattgaggcg\n",
      "acgtttgccgaacgcgtggtgtttatgaactccggcacggaagctaacgaaaccgccttt\n",
      "aaactggc\n",
      ">G000299455|2312634|2337653|2318491|2318738|+\n",
      "-gggatcagcaggggaaagagtatatcgacttcgcgggtggcattgcggtgaacgcgctg\n",
      "ggccatgcgcatccggaactgcgtgaagcgctgaacgaacaggcgagtaagttctggcat\n",
      "accggcaacggttacaccaacgagccggtactgcgactggcgaaaaaattgatcgacgcc\n",
      "acgtttgccgatcgcgtcttcttttgtaactccggtgcggaagccaacgaagcggcgcta\n",
      "aaactggc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3086178, 3086372), '-'), (('G000299455', (56338, 57307)), (57110, 57304), '+'), (('G000299455', (857924, 871846)), (869138, 869332), '+'), (('G000299455', (1986939, 1987907)), (1987710, 1987904), '+'), (('G000299455', (3152907, 3154652)), (3154458, 3154652), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1390\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1062\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\n",
      "Reallocating..done. *alloclen = 1110\n",
      "\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\n",
      "Reallocating..done. *alloclen = 1110\n",
      "\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3086178|3086372|-\n",
      "tgactggtgagtttgcgcagatgttccaggatgtgacgggaaagtgagtaaggcagggag\n",
      "gagagcggtttttcaattgcctcaatggcatcagacggattcatacatgtattcccggtg\n",
      "aaaaaagacaaaaccccgcgcaccggaaaggtggcgggggattcacggagataatatctg\n",
      "tctgagattattta--\n",
      ">G000299455|56338|57307|57110|57304|+\n",
      "--actggtgagtttgcgcagatgttccaggatgtgacgggaaagtgagtaaggcagagag\n",
      "gagaggggtttttcaattgcctcaatggcatcagacggattcatacatgtattcccggtg\n",
      "aaaaaagacaaaaccccgcacaccggaaaggtggcgggggactcacggagataatatctg\n",
      "tctgaaattattttga\n",
      ">G000299455|857924|871846|869138|869332|+\n",
      "-gactggtgagtttgcgcagatgttccaggatgtgacgggaaagtgagtaaggcagggag\n",
      "gagagcggtttttcaattgcctcaatggcatcagacggattcatacatgtattcccggtg\n",
      "aaaaaagacaaaaccccgcgcaccggaaaggtggcgggggattcacggagataatatctg\n",
      "tctgagattatttag-\n",
      ">G000299455|1986939|1987907|1987710|1987904|+\n",
      "-gactggtgagtttgcgcagatgttccaggatgtgacgggaaagtgagtaaggcagggag\n",
      "gagagcggtttttcaattgcctcaatggcatcagacggattcatacatgtattcccggtg\n",
      "aaaaaagacaaaaccccgcgcaccggaaaggtggcgggggattcacggaaataatatctg\n",
      "tctgaaattatctgg-\n",
      ">G000299455|3152907|3154652|3154458|3154652|+\n",
      "-gacgggtgagtttgcgcagatgttccaggatgtgacgggaaagcgagtaaggcagggag\n",
      "gagagcggtttttcaattgcctcaatggcatcagacggattcatacatgtattcccggtg\n",
      "aaaaaagacaaaaccccgcacaccggaaaggtggcgggggactcacaacgataatatctg\n",
      "tctgaaattatttag-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2153680, 2159578)), (2155288, 2155318), '+'), (('G000006925', (3361966, 3403725)), (3388144, 3388174), '-'), (('G000299455', (540757, 615358)), (552952, 552982), '+'), (('G000299455', (1902485, 1908383)), (1906745, 1906775), '-')]\n",
      ">G000006925|2153680|2159578|2155288|2155318|+\n",
      "-ggtggtggatatcggcggtggtacgactga\n",
      ">G000006925|3361966|3403725|3388144|3388174|-\n",
      "tggtggttgatatcggtggtggtaccactg-\n",
      ">G000299455|540757|615358|552952|552982|+\n",
      "-ggtggttgatatcggtggtggtaccactga\n",
      ">G000299455|1902485|1908383|1906745|1906775|-\n",
      "tggtggtggatatcggcggtggtacgactg-\n",
      "\n",
      "\n",
      "19 [(('G000006925', (385776, 386112)), (385921, 385975), '-'), (('G000006925', (899355, 899691)), (899492, 899546), '+'), (('G000006925', (2059291, 2059627)), (2059436, 2059490), '-'), (('G000006925', (3421586, 3421922)), (3421723, 3421777), '+'), (('G000006925', (3851554, 3851890)), (3851699, 3851753), '-'), (('G000006925', (4359551, 4359859)), (4359676, 4359730), '-'), (('G000006925', (4622488, 4624460)), (4623016, 4623070), '+'), (('G000006925', (4700925, 4701241)), (4701050, 4701104), '-'), (('G000006925', (4776578, 4776894)), (4776703, 4776757), '-'), (('G000006925', (4781327, 4781663)), (4781464, 4781518), '+'), (('G000299455', (1991020, 1991316)), (1991149, 1991203), '+'), (('G000299455', (3215105, 3215383)), (3215231, 3215285), '+'), (('G000299455', (4543449, 4543753)), (4543586, 4543640), '+'), (('G000299455', (4571312, 4571569)), (4571467, 4571520), '+'), (('G000299455', (4752483, 4752699)), (4752638, 4752691), '+'), (('G000299455', (5289627, 5289931)), (5289740, 5289794), '-'), (('G000299455', (5313369, 5315826)), (5314253, 5314307), '+'), (('G000299455', (5316432, 5316648)), (5316440, 5316493), '-'), (('G000299455', (5333884, 5334100)), (5334039, 5334092), '+')]\n",
      ">G000006925|385776|386112|385921|385975|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|899355|899691|899492|899546|+\n",
      "-aaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcaccaaac\n",
      ">G000006925|2059291|2059627|2059436|2059490|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|3421586|3421922|3421723|3421777|+\n",
      "-aaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaac\n",
      ">G000006925|3851554|3851890|3851699|3851753|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|4359551|4359859|4359676|4359730|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|4622488|4624460|4623016|4623070|+\n",
      "-accagataaaagtgttgtgggctgacagtgacggactgtgcctctttaccagac\n",
      ">G000006925|4700925|4701241|4701050|4701104|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|4776578|4776894|4776703|4776757|-\n",
      "aaaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaa-\n",
      ">G000006925|4781327|4781663|4781464|4781518|+\n",
      "-aaatggtgaaaatactgtgggccgatcgtgacgggttatgcctgttcgccaaac\n",
      ">G000299455|1991020|1991316|1991149|1991203|+\n",
      "-gtcaggtaaaactgctctggtccaccggcgatggtctgtgcctgctgacaaagc\n",
      ">G000299455|3215105|3215383|3215231|3215285|+\n",
      "-gtcaggtaaagctcctctggtctaccggcgatggactgtgtctgctgaccaaac\n",
      ">G000299455|4543449|4543753|4543586|4543640|+\n",
      "-gtcaggttaaactgctgtggtccaccggtgacggactgtgcctcctgaccaaac\n",
      ">G000299455|4571312|4571569|4571467|4571520|+\n",
      "--caccgtcaaaattctttgggctgatgctgatggtctgtgcctgttcaccaaac\n",
      ">G000299455|4752483|4752699|4752638|4752691|+\n",
      "--caccgttaaaattctttgggctgatgctgatggtctgtgcctgttcaccaaac\n",
      ">G000299455|5289627|5289931|5289740|5289794|-\n",
      "agtcaggttaaactgctgtggtccaccggtgacggactgtgcctcctgaccaaa-\n",
      ">G000299455|5313369|5315826|5314253|5314307|+\n",
      "-accagataaaagtgttgtgggctgacagtgacggactgtgcctcttcaccaaac\n",
      ">G000299455|5316432|5316648|5316440|5316493|-\n",
      "-acacgattaaaatcctgtgggctgatgctgatggtctgtgcctgttcaccaaa-\n",
      ">G000299455|5333884|5334100|5334039|5334092|+\n",
      "--cacgattaaaatcctgtgggctgatgctgatggtctgtgcctgttcaccaaac\n",
      "\n",
      "\n",
      "7 [(('G000006925', (720687, 720818)), (720756, 720776), '-'), (('G000006925', (1578298, 1580868)), (1578298, 1578318), '+'), (('G000006925', (1937276, 1937311)), (1937286, 1937306), '+'), (('G000299455', (2165477, 2165575)), (2165512, 2165532), '-'), (('G000299455', (2546693, 2546781)), (2546761, 2546781), '-'), (('G000299455', (2927401, 2931337)), (2927411, 2927431), '+'), (('G000299455', (3293791, 3293879)), (3293791, 3293811), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\n",
      "Reallocating..done. *alloclen = 1042\n",
      "\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|720687|720818|720756|720776|-\n",
      "aaaaaacccgcccggaggcg-\n",
      ">G000006925|1578298|1580868|1578298|1578318|+\n",
      "-aaaaacccgctcagcggcgg\n",
      ">G000006925|1937276|1937311|1937286|1937306|+\n",
      "aaaaaaaccgctcggcggcg-\n",
      ">G000299455|2165477|2165575|2165512|2165532|-\n",
      "aaaaaaaccgctcagcggcg-\n",
      ">G000299455|2546693|2546781|2546761|2546781|-\n",
      "aaaaaacccgcgcatcggcg-\n",
      ">G000299455|2927401|2931337|2927411|2927431|+\n",
      "-aaaaacccgccaggcggcgg\n",
      ">G000299455|3293791|3293879|3293791|3293811|+\n",
      "-aaaaacccgcgcatcggcgg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4620693, 4621057)), (4620873, 4620896), '+'), (('G000006925', (4684879, 4685082)), (4684879, 4684902), '-'), (('G000006925', (4791871, 4792235)), (4792051, 4792074), '+'), (('G000006925', (4794974, 4795158)), (4795134, 4795158), '-'), (('G000299455', (5313369, 5315826)), (5313549, 5313572), '+')]\n",
      ">G000006925|4620693|4621057|4620873|4620896|+\n",
      "-caactgtctgtggtatgttcgtg\n",
      ">G000006925|4684879|4685082|4684879|4684902|-\n",
      "tcaactgtctgtggtatgttcgt-\n",
      ">G000006925|4791871|4792235|4792051|4792074|+\n",
      "-caactgtctgtggtatgttcgtg\n",
      ">G000006925|4794974|4795158|4795134|4795158|-\n",
      "tcaacagtttgtggactgttcgct\n",
      ">G000299455|5313369|5315826|5313549|5313572|+\n",
      "-caacagtttgtagtatgttcgtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4619501, 4619648)), (4619541, 4619574), '+'), (('G000299455', (59768, 59801)), (59768, 59801), '+'), (('G000299455', (1990152, 1990299)), (1990192, 1990225), '+')]\n",
      ">G000006925|4619501|4619648|4619541|4619574|+\n",
      "cgttttccaccatttcatcatcactgattggcg\n",
      ">G000299455|59768|59801|59768|59801|+\n",
      "cggtttccaccgattcatcatcacttattggcg\n",
      ">G000299455|1990152|1990299|1990192|1990225|+\n",
      "cgcttttcaccatttcatcctcactgacaggcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3096309, 3096388), '-'), (('G000299455', (857924, 871846)), (859165, 859244), '+'), (('G000299455', (4771847, 4785261)), (4781945, 4782024), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3096309|3096388|-\n",
      "tgagaatagtcctcctgaatgtgaaagtaagaaacagctgagaaagagaaactgaagtga\n",
      "tatgaaggtgattgtgacg-\n",
      ">G000299455|857924|871846|859165|859244|+\n",
      "-gagaatagtcctcctgaatgtgaaagtaagaaacagctgagaaagagaaactgaagtga\n",
      "tatgaaggtgattgtgacgg\n",
      ">G000299455|4771847|4785261|4781945|4782024|-\n",
      "tgagaatagtcctcctgaatgtgaaagtaagaaacagctgagaaagagaaactgaagtga\n",
      "tatgaaggtgattgtgacg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3052938, 3053411), '-'), (('G000299455', (942831, 956197)), (943441, 943914), '+'), (('G000299455', (4713031, 4748324)), (4736178, 4736651), '-')]\n",
      ">G000006925|3040656|3065083|3052938|3053411|-\n",
      "gggtgacaggctggcctttataaccgtcaatacgcgacagaagttcaggcaggcgtgaaa\n",
      "gctccagagcagggcgatgctggcgttttaccgtggtcaatgcaccagccatatccagag\n",
      "ccgggttatatctgatgatgccagactgtacggcgtaacgcattacagcagatattcttt\n",
      "gctgcagacgactggcgatttcataaatttgttttgcttctgcagcacgaacagggataa\n",
      "gcaggtccggggtattgagtgttgtgatatcccggttgccgataaaggggaaaacgtgtg\n",
      "tctcgaggcttttgagtactttatcgctgtgtgattgtgcccattgtttattactggcat\n",
      "gccagcgtcgtgcaatatgttcaaaagagagctgtaaatcagggacaacttttttaactc\n",
      "ttttctgttcgcaagggtcaatcccctctgaaatcagttttcgggcctcatcc-\n",
      ">G000299455|942831|956197|943441|943914|+\n",
      "-ggtgacaggctggcctttataaccgtcaatacgcgacagaagttcaggcaggcgtgaaa\n",
      "gattcagagcggggcgatgctggcgttttaccgtggtcaatgcaccagccatatccagag\n",
      "ccgggttatatctgatgatgccagactgtacggcgtaacgcattacagcagatattcttt\n",
      "gctgcagacgactagcgatttcataaatttgttttgcttctgcagcacgaacagggataa\n",
      "gcaggtccggggtattgagtgttgtgatatcccggttgccgataaaggggaaaacgtgtg\n",
      "tctcgaggcttttgagtactttatcgctgtgtgattgtgcccattgtttattactggcat\n",
      "gccagcgtcgtgcaatatgttcaaaagagagctgtaaatcagggactacttttttaactc\n",
      "ttttctgttcgcaagggtcaatcccctctgaaatcagttttcgggcctcatccc\n",
      ">G000299455|4713031|4748324|4736178|4736651|-\n",
      "gggtgacaggctggcctttataaccgtcaatacgcgacagaagttcaggcaggcgtgaaa\n",
      "gctccagagcagggcgatgctggcgttttaccgtggtcaatgcaccagccatatccagag\n",
      "ccgggttatatctgatgatgccagactgtacggcgtaacgcattacagcagatattcttt\n",
      "gctgcagacgactggcgatttcataaatttgttttgcttctgcagcacgaacagggataa\n",
      "gcaggtccggggtattgagtgttgtgatatcccggttgccgataaaggggaaaacgtgtg\n",
      "tctcgaggcttttgagtactttatcgctgtgtgattgtgcccattgtttattactggcat\n",
      "gccagcgtcgtgcaatatgttcaaaagagagctgtaaatcagggacaacttttttaactc\n",
      "ttttctgttcgcaagggtcaatcccctctgaaatcagttttcgggcctcatcc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4350367, 4350507), '-'), (('G000299455', (3326617, 3362069)), (3337442, 3337582), '+'), (('G000299455', (4865050, 4904182)), (4900418, 4900558), '-')]\n",
      ">G000006925|4315026|4354132|4350367|4350507|-\n",
      "ggcaacggcttctacggtcaggtctatggaggttatctcgaaaccatgtttggcggtgcg\n",
      "ggggccgaagtgttgtatcgtccgctggacagtaactgggcgtttggtctggatgccaac\n",
      "tacgttaaacagcgcgactg-\n",
      ">G000299455|3326617|3362069|3337442|3337582|+\n",
      "-gcaatggtttctatggccaggtgtatggcggttatctggaaacgatgtatgccggtgtc\n",
      "ggttccgagctgctttatcgcccgctagatgccagctgggcgctgggtgtggacgttaac\n",
      "tacgtgaagcagcgtgactgg\n",
      ">G000299455|4865050|4904182|4900418|4900558|-\n",
      "ggcaacggcttctacggtcaggtctatggaggttatctcgaaaccatgtttggcggtgcg\n",
      "ggggccgaagtgttgtatcgtccgctggacagtaactgggcgtttggtctggatgccaac\n",
      "tacgttaaacagcgcgactg-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (2569729, 2584886)), (2578079, 2578220), '-'), (('G000006925', (3693106, 3702974)), (3700125, 3700266), '+'), (('G000299455', (269622, 287922)), (275869, 276010), '+'), (('G000299455', (535426, 536523)), (535951, 536092), '+'), (('G000299455', (1462739, 1472085)), (1469405, 1469546), '+'), (('G000299455', (3918464, 3964500)), (3934314, 3934455), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1284\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2569729|2584886|2578079|2578220|-\n",
      "actccagaccaaagccgtttggcagctgcttcactaacgattccataatatccatcgccg\n",
      "taccggtactgacccccggtgcggcttccccaacaatctcaaccgcagaataaccgttat\n",
      "agcgttccagacgcggcgagc-\n",
      ">G000006925|3693106|3702974|3700125|3700266|+\n",
      "-tgagtagccgacgcctgccggaagtttagcgaccaggtctgccataaatttcatagcgt\n",
      "caccggtacttttcccggccgccgcttcacctaaaatctccattgacgggatgccgttgt\n",
      "agcgttccaatcgcggtgaacc\n",
      ">G000299455|269622|287922|275869|276010|+\n",
      "-tgagtagccgacgcctgccggaagtttagcgaccaggtctgccataaatttcatagcgt\n",
      "caccggtacttttcccggccgccgcttcacctaaaatctccattgacgggatgccgttgt\n",
      "agcgttccaatcgcggtgaacc\n",
      ">G000299455|535426|536523|535951|536092|+\n",
      "-gtcataaccaatgcccgcaggtaattttgacgcaaggttttccatcaacgccatggcat\n",
      "cgccggaactggttcctggcgcggcttccccctgaatctccattgacggcagaccgttgt\n",
      "agcgttccagtcgcggagagcc\n",
      ">G000299455|1462739|1472085|1469405|1469546|+\n",
      "-ctccagaccaaagccgtttggcaactgcttcactaacgattccataatatccatcgccg\n",
      "taccggtactgacccccggtgcggcttccccaacaatctcaaccgcagaataaccgttat\n",
      "agcgttccagacgcggcgagcc\n",
      ">G000299455|3918464|3964500|3934314|3934455|-\n",
      "agtcatagccaacaccggtaggcagtttgctcgccagttgttccatcagctccattgctt\n",
      "caccggtacttttacccggtgccgcctggcctaagatttccatagatggcaggccgttgt\n",
      "aacgttccagacgcggcgaac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3059489, 3060076), '+'), (('G000299455', (4713031, 4748324)), (4742730, 4743317), '+'), (('G000299455', (5363107, 5363694)), (5363107, 5363694), '-')]\n",
      ">G000006925|3040656|3065083|3059489|3060076|+\n",
      "-aactggaaacggccgggcagaaatggtctctgggcttcccccccaaattccagcgcgtc\n",
      "gacctgttcaactacaacgtgctggtcagaaattatgacagcagcgccttcaaaggcgac\n",
      "cgctaccacaacacgaaaaatggcatcaacgccgacatcggtgccagtacggacctggat\n",
      "gacaactggacgctgggactggtcgcacagaacctgatctcccgtagtattgagacaaaa\n",
      "gaagtgaacggtatcacggaaaccttcaggatccgcccgcaggtgacggccggtgtctcc\n",
      "tggcacaacgcgatgttcaccactgcatttgatgtggatctgaccccggccagcggtttc\n",
      "acctccgacagcaaccgtcagtttgccgccattggcacagaatttaatgcctggaaatgg\n",
      "gcacagctgcgcgccggttaccgtcagaatctggccggtaacgacggcagtgcattcacg\n",
      "gccggggtggggatctcaccgtttgatgtggttcaccttgatgttgcaggcctgatcggc\n",
      "acggacaacacttacggtgcggtcgcacagttccagttcaccttctga\n",
      ">G000299455|4713031|4748324|4742730|4743317|+\n",
      "-aactggaaacggccgggcagaaatggtctctgggcttcccccccaaattccagcgcgtc\n",
      "gacctgttcaactacaacgtgctggtcagaaattatgacagcagcgccttcaaaggcgac\n",
      "cgctaccacaacacgaaaaatggcatcaacgccgacatcggtgccagtacggacctggat\n",
      "gacaactggacgctgggactggtcgcacagaacctgatctcccgtagtattgagacaaaa\n",
      "gaagtgaacggtatcacggaaaccttcaggatccgcccgcaggtgacggccggtgtctcc\n",
      "tggcacaacgcgatgttcaccactgcatttgatgtggatctgaccccggccagcggtttc\n",
      "acctccgacagcaaccgtcagtttgccgccattggcgcagaatttaatgcctggaaatgg\n",
      "gcacagctgcgcgccggttaccgtcagaatctggccggtaacgacggcagtgcattcacg\n",
      "gccggggtggggatctcaccgtttgatgtggttcaccttgatgttgcaggcctgatcggc\n",
      "acggacaacacttacggtgcggtcgcacagttccagttcaccttctga\n",
      ">G000299455|5363107|5363694|5363107|5363694|-\n",
      "gaactggaaacggccggacaaaaatggtctcttggcgtcacaccaaaatatcagcgtgtg\n",
      "gacctgtttaactacaacgtaacggttcgtgattacgacaaagacgattttgacggtgac\n",
      "aagtaccacaacacgaaaaacggattcaacgccgatattggtgcttatacagacctgaat\n",
      "gacaactggactgtcggtcttgttgcccagaacatcattccacgcagtattgatactaaa\n",
      "gtcgtcaatggctttaaagaaaccttcaaagttcgtccacaggcaaccgccggtgtatcc\n",
      "tggcataatgacctgtttaccactgctcttgacgttgatctgactcctgccagtggcttc\n",
      "acttctgacagcaaacgtcagtttgcaagtgttggtgctgaatttaacgcatggaaatgg\n",
      "gcgcaattgcgtgccggttatcgtcagaacatggcatcaaactccggaagcgccttcacc\n",
      "gctggcgttggtatctctccatttgacgtcgtacatatcgatgtttccggcctggtagga\n",
      "actgaccacgattacggtgcaatggcacaacttcagttcactttctg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2268820, 2268864), '-'), (('G000299455', (1752647, 1813107)), (1809347, 1809391), '+'), (('G000299455', (3212276, 3212366)), (3212276, 3212320), '+')]\n",
      ">G000006925|2242494|2346249|2268820|2268864|-\n",
      "ccatcgaacgtattgaagtggttcgtggcccgatgtcgtcgcta-\n",
      ">G000299455|1752647|1813107|1809347|1809391|+\n",
      "-catcgaacgtattgaagtggttcgtggcccgatgtcgtcgctat\n",
      ">G000299455|3212276|3212366|3212276|3212320|+\n",
      "-cattgagcgtattgaggttatcagggggccgatgtccacactgt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3952917, 3966316)), (3956314, 3956355), '-'), (('G000299455', (4212683, 4221048)), (4220983, 4221024), '+'), (('G000299455', (5202806, 5268469)), (5204762, 5204803), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3952917|3966316|3956314|3956355|-\n",
      "caccgaattcttttacaaccaccggttttacgaccggctta-\n",
      ">G000299455|4212683|4221048|4220983|4221024|+\n",
      "-accgaattcttttacaaccaccggttttatgaccggcttac\n",
      ">G000299455|5202806|5268469|5204762|5204803|+\n",
      "-accgaattcttttacaaccaccggttttatgaccggcttac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4620693, 4621057)), (4620753, 4620873), '+'), (('G000006925', (4684879, 4685082)), (4684902, 4685022), '-'), (('G000006925', (4791871, 4792235)), (4791931, 4792051), '+'), (('G000299455', (5313369, 5315826)), (5313430, 5313549), '+')]\n",
      ">G000006925|4620693|4621057|4620753|4620873|+\n",
      "-cattttgaagtggacactatcgtatggaataccggacctggatcactgaagctttacgc\n",
      "cttcatttcgaagaacatttaccacgggttgtagccggacgtcgcctgggcgtaccaaaa\n",
      "t\n",
      ">G000006925|4684879|4685082|4684902|4685022|-\n",
      "tcattttgaagtggacactatcgtatggaataccggacctggatcactgaagctttacgc\n",
      "cttcatttcgaagaacatttaccacgggttgtggccggacgtcgcctgggcgtaccaaaa\n",
      "-\n",
      ">G000006925|4791871|4792235|4791931|4792051|+\n",
      "-cattttgaagtggacactatcgtatggaataccggacctgaatcactgaagctttacgc\n",
      "cttcatttcgaagaacatttaccacgggttgtggccggatgtcgcctgggcgtaccaaaa\n",
      "t\n",
      ">G000299455|5313369|5315826|5313430|5313549|+\n",
      "-catttttaa-tggacactatcgtatgaaacaccggacctggatcactgaagctttacgt\n",
      "cttcactttgaagaacatttaccccgggttgtggccgggcgtcgcctgggtgtaccaaaa\n",
      "t\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2374913, 2374956), '-'), (('G000006925', (4551704, 4551747)), (4551704, 4551747), '-'), (('G000299455', (1685057, 1714323)), (1705748, 1705791), '+')]\n",
      ">G000006925|2374481|2472008|2374913|2374956|-\n",
      "gacgtcgaagggattgatggcgtgtttattggaccagcggatc-\n",
      ">G000006925|4551704|4551747|4551704|4551747|-\n",
      "gacgtggaaggggtcgacggcgtgtttatcggcccggcggatc-\n",
      ">G000299455|1685057|1714323|1705748|1705791|+\n",
      "-acgtcgaagggattgatggcgtgtttattggacctgcggatct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1670306, 1692658)), (1692624, 1692658), '-'), (('G000006925', (1692671, 1695090)), (1692671, 1692705), '-'), (('G000299455', (2459722, 2494618)), (2473235, 2473269), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1670306|1692658|1692624|1692658|-\n",
      "agtggaaggggggagtgagaaatcactccccctg-\n",
      ">G000006925|1692671|1695090|1692671|1692705|-\n",
      "gtggaaaggggggagtgagacatcactccccctg-\n",
      ">G000299455|2459722|2494618|2473235|2473269|+\n",
      "-gtggaaagggggagtgagaaatcactccccctgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (727935, 728422), '+'), (('G000299455', (2174641, 2181396)), (2176328, 2176819), '+'), (('G000299455', (2910238, 2916987)), (2914809, 2915300), '-')]\n",
      ">G000006925|726235|732999|727935|728422|+\n",
      "-ctatgcggcattacgtgtgtccgtgcaacgctggcaacttgatctggctgtgc-agttg\n",
      "catcccgcaaggatgcaactgcacaaccgacccttgaagaactggcagcgaagctgtccg\n",
      "gaggagtgaatggttacagtcgctgaactgcaggcgctgcgtcaggcgcgccttgattta\n",
      "ttaaccggtaaacgggttgtgtcggtacagaaagacggtcgcaggatcgaatatacggca\n",
      "gcgtccctgaatgagcttaaccgtgcgatcaacgatgcggagtcggttctgggtacgacc\n",
      "cga---cgccgccgaccgctgggagtgaggttatgaaacgaacgcctgtcctgattgatg\n",
      "tgaacggcgttccgcttcgtgagagtctcagctacaacggaggcggcgcaggatttggcg\n",
      "ggcaaatggcagagtggttaccaccggcgcagagtgccgatgcggccctgttgcctgcat\n",
      "tgcgtctgggga\n",
      ">G000299455|2174641|2181396|2176328|2176819|+\n",
      "-ctacgcggcattacgtgtgtccgtgcaacgctggcagcttgatctggctgtactggcaa\n",
      "aatcccgggaagaagagacgacccggccaacccttaaagaactggcagcgaagctgtccg\n",
      "gaggagtgaatggttacagtcgctgaactgcaggcgctgcgtcaggcgcgccttgattta\n",
      "ttaaccggtaaacgggtggtgtctgtccagaaagatggtcgcagaattgaatatacggcg\n",
      "gcttctctggatgagcttaaccgggcgatcaatgatgcggagtcggtactggggacaacc\n",
      "cggcgtcgccgtcgtccgctgggagtgaggttatgaaacgaacgcctgtcctgattgatg\n",
      "tgaacggcgttccgcttcgtgatagtctcagctacaacgggggcggtgcaggatttggcg\n",
      "ggcaaatggctgagtggttgccaccggcgcagagtgccgatgcggccctgctgcccgcgt\n",
      "tgcgtctgggga\n",
      ">G000299455|2910238|2916987|2914809|2915300|-\n",
      "cctatgcggcattacgtgtgtccgtgcaacgctggcagcttgatctggctgtactggcaa\n",
      "aatcccgggaagaagagacgacccggccaacccttaaagaactggcagcgaagctgtccg\n",
      "gaggagtgaatggttacagtcgctgaactacaggcgctgcgtcaggcgcgccttgattta\n",
      "ttaaccggtaaacgggtggtgtctgtccagaaagatggtcgcagaattgaatatacggcg\n",
      "gcttctctggatgagcttaaccgggcgatcaatgatgcggagtcggtactggggacaacc\n",
      "cgacgtcgccgtcgtccgctgggagtgaggttatgaaacgaacgcctgtcctgattgatg\n",
      "tgaacggcgttccgcttcgtgagagtctcagctacaacgggggcggcgcaggatttggcg\n",
      "ggcaaatggcggagtggttgccaccggcgcagagtgccgatgcagccctgctgcctgcgt\n",
      "tgcgtctgggg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4387885, 4387998)), (4387885, 4387998), '+'), (('G000299455', (2547946, 2548059)), (2547946, 2548059), '+'), (('G000299455', (3290769, 3290882)), (3290769, 3290882), '-')]\n",
      ">G000006925|4387885|4387998|4387885|4387998|+\n",
      "-agtacggcaccgttggccaggcactgcatattgcacggaaattactgcccttcattcct\n",
      "gataatgcagggattcttatcgttccgtgttgccgtggcggatcggcttttacc\n",
      ">G000299455|2547946|2548059|2547946|2548059|+\n",
      "-agtacggaaccgtggggcaggggctgcatatcgccaaaaaactgctgccgtttataccg\n",
      "gcgaatgcgggcattctgctggttccgtgctgtcgtggtggttcagcgttcacc\n",
      ">G000299455|3290769|3290882|3290769|3290882|-\n",
      "cagtacggaaccgtggggcaagggctgcatatcgccaaaaaattgctgccgtttataccg\n",
      "gcgaatgcgggcattctgctggttccgtgctgtcgtggtggttcagcgttcac-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3095015, 3095049), '+'), (('G000299455', (50607, 52894)), (51864, 51898), '-'), (('G000299455', (3143255, 3145541)), (3144512, 3144546), '-'), (('G000299455', (4771847, 4785261)), (4780651, 4780685), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3095015|3095049|+\n",
      "-tgattcaggcaccagaatcacccgacgccccgga\n",
      ">G000299455|50607|52894|51864|51898|-\n",
      "gtgattcaggtaccagaatcacccgacgccccgg-\n",
      ">G000299455|3143255|3145541|3144512|3144546|-\n",
      "gtgattcaggcaccagaatcacccgacgccccgg-\n",
      ">G000299455|4771847|4785261|4780651|4780685|+\n",
      "-tgattcaggcaccagaatcacccgacgccccgga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3736321, 3788106)), (3750415, 3750497), '+'), (('G000006925', (4535976, 4543278)), (4541859, 4541941), '-'), (('G000299455', (137973, 154383)), (149313, 149395), '-'), (('G000299455', (830050, 839519)), (838218, 838300), '+')]\n",
      ">G000006925|3736321|3788106|3750415|3750497|+\n",
      "-attaccgccgcattgctggtcggcctgggttttaaaccgctgtacgccgccggactgtg\n",
      "cctgattgttaacaccgcgccag\n",
      ">G000006925|4535976|4543278|4541859|4541941|-\n",
      "aattaccgccgcattgctggtcggcctgggttttaaaccgctgtacgccgccggactgtg\n",
      "cctgattgttaacaccgcgcca-\n",
      ">G000299455|137973|154383|149313|149395|-\n",
      "aattaccgccgcattgctggtcggcctgggttttaaaccgctgtacgccgccggactgtg\n",
      "cctgattgttaacaccgcgcca-\n",
      ">G000299455|830050|839519|838218|838300|+\n",
      "-attaccggtgcgctgctggtgggcctgggcttcaaaccgttatacgcggcggggctgtg\n",
      "tctgattgccaacactgcgccag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1916373, 1917424)), (1917265, 1917328), '+'), (('G000006925', (2227521, 2228997)), (2227775, 2227838), '+'), (('G000299455', (2196403, 2197468)), (2196513, 2196576), '-'), (('G000299455', (4121494, 4121667)), (4121494, 4121557), '+')]\n",
      ">G000006925|1916373|1917424|1917265|1917328|+\n",
      "-gttcaggaaactctgaaagatacctgggagtctgctgacgagtggtttgttcactaatt\n",
      "agca\n",
      ">G000006925|2227521|2228997|2227775|2227838|+\n",
      "-gttcaggagactctgaaagatacctgggagtctgctgacgagtggtttgttcactaata\n",
      "aaca\n",
      ">G000299455|2196403|2197468|2196513|2196576|-\n",
      "tgttcaggaaactctgaaagatacctgggagtctgctgacgagtggtttgttcactaatt\n",
      "agc-\n",
      ">G000299455|4121494|4121667|4121494|4121557|+\n",
      "-gttcaggagacacggaaaggtatcaggcagcctgctaacgagtggtttaatcacgaatt\n",
      "agta\n",
      "\n",
      "\n",
      "15 [(('G000006925', (385251, 385465)), (385442, 385462), '-'), (('G000006925', (900687, 900901)), (900690, 900711), '+'), (('G000006925', (3065287, 3076770)), (3066685, 3066706), '+'), (('G000006925', (3422918, 3423132)), (3422921, 3422942), '+'), (('G000006925', (3851029, 3851243)), (3851219, 3851240), '-'), (('G000006925', (4358341, 4358555)), (4358531, 4358552), '-'), (('G000006925', (4622488, 4624460)), (4624181, 4624202), '+'), (('G000006925', (4775368, 4775582)), (4775558, 4775579), '-'), (('G000006925', (4782659, 4782873)), (4782662, 4782683), '+'), (('G000299455', (1992068, 1992414)), (1992311, 1992331), '+'), (('G000299455', (3215867, 3216640)), (3215982, 3216003), '+'), (('G000299455', (4544552, 4544854)), (4544748, 4544769), '+'), (('G000299455', (4749820, 4751622)), (4751214, 4751235), '+'), (('G000299455', (5287974, 5288758)), (5288612, 5288632), '-'), (('G000299455', (5313369, 5315826)), (5315418, 5315439), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 15\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 15\r",
      "   10 / 15\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 14  f\b\b\r",
      "STEP     2 / 14  f\b\b\r",
      "STEP     3 / 14  f\b\b\r",
      "STEP     4 / 14  f\b\b\r",
      "STEP     5 / 14  f\b\b\r",
      "STEP     6 / 14  f\b\b\r",
      "STEP     7 / 14  f\b\b\r",
      "STEP     8 / 14  f\b\b\r",
      "STEP     9 / 14  f\b\b\r",
      "STEP    10 / 14  f\b\b\r",
      "STEP    11 / 14  f\b\b\n",
      "Reallocating..done. *alloclen = 1044\n",
      "\r",
      "STEP    12 / 14  f\b\b\r",
      "STEP    13 / 14  f\b\b\r",
      "STEP    14 / 14  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 15\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 15\r",
      "   10 / 15\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 14  f\b\b\r",
      "STEP     2 / 14  f\b\b\r",
      "STEP     3 / 14  f\b\b\r",
      "STEP     4 / 14  f\b\b\r",
      "STEP     5 / 14  f\b\b\r",
      "STEP     6 / 14  f\b\b\r",
      "STEP     7 / 14  f\b\b\r",
      "STEP     8 / 14  f\b\b\r",
      "STEP     9 / 14  f\b\b\r",
      "STEP    10 / 14  f\b\b\r",
      "STEP    11 / 14  f\b\b\r",
      "STEP    12 / 14  f\b\b\r",
      "STEP    13 / 14  f\b\b\r",
      "STEP    14 / 14  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|385251|385465|385442|385462|-\n",
      "gtactgcaggctgatgccta--\n",
      ">G000006925|900687|900901|900690|900711|+\n",
      "-tactgcaggctgatgcctatg\n",
      ">G000006925|3065287|3076770|3066685|3066706|+\n",
      "-tgcttcaggcagatgcttacg\n",
      ">G000006925|3422918|3423132|3422921|3422942|+\n",
      "-tactgcaggctgatgcctatg\n",
      ">G000006925|3851029|3851243|3851219|3851240|-\n",
      "gtactgcaggctgatgcctat-\n",
      ">G000006925|4358341|4358555|4358531|4358552|-\n",
      "gtactgcaggctgatgcctat-\n",
      ">G000006925|4622488|4624460|4624181|4624202|+\n",
      "-tgcttcaggccgatgcttacg\n",
      ">G000006925|4775368|4775582|4775558|4775579|-\n",
      "gtactgcaggctgatgcctat-\n",
      ">G000006925|4782659|4782873|4782662|4782683|+\n",
      "-tactgcaggctgatgcctatg\n",
      ">G000299455|1992068|1992414|1992311|1992331|+\n",
      "-tactgcaggctgatgcctac-\n",
      ">G000299455|3215867|3216640|3215982|3216003|+\n",
      "-tactgcaggcggatgcatacg\n",
      ">G000299455|4544552|4544854|4544748|4544769|+\n",
      "-tgctgcaagcggatgcgtacg\n",
      ">G000299455|4749820|4751622|4751214|4751235|+\n",
      "-tgcttcaggcagatgcttacg\n",
      ">G000299455|5287974|5288758|5288612|5288632|-\n",
      "gtactgcaggcggatgcata--\n",
      ">G000299455|5313369|5315826|5315418|5315439|+\n",
      "-tgcttcaggccgatgcttacg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1842508, 1869155)), (1864700, 1864740), '+'), (('G000299455', (2786550, 2804122)), (2799661, 2799701), '+'), (('G000299455', (3554411, 3565644)), (3554667, 3554707), '+')]\n",
      ">G000006925|1842508|1869155|1864700|1864740|+\n",
      "tatccagaccacgcgcagcaacatcagtcgcaaccagtac\n",
      ">G000299455|2786550|2804122|2799661|2799701|+\n",
      "tatccagaccacgcgcagcaacatcagtcgcaaccagtac\n",
      ">G000299455|3554411|3565644|3554667|3554707|+\n",
      "tatccagaccgcgcgcagcgatatcggtcgccaccagtac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4632298, 4632639)), (4632593, 4632639), '-'), (('G000299455', (917546, 918643)), (917546, 917592), '+'), (('G000299455', (4770510, 4771607)), (4770510, 4770556), '+')]\n",
      ">G000006925|4632298|4632639|4632593|4632639|-\n",
      "tctgcagaatgagtcgtagatggagagtggatgatgctgggggggg-\n",
      ">G000299455|917546|918643|917546|917592|+\n",
      "-ctgcagaatgagtcgtagatggagagtggatgatgcagggggtgtt\n",
      ">G000299455|4770510|4771607|4770510|4770556|+\n",
      "-ctgcagaatgagtcgtagatggagagtggatgatgcagggggtgtt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4622488, 4624460)), (4622499, 4622561), '+'), (('G000006925', (4689001, 4689359)), (4689001, 4689063), '+'), (('G000299455', (5313369, 5315826)), (5313736, 5313798), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4622488|4624460|4622499|4622561|+\n",
      "ttttccttatgaatttaaaatcgctttagtggagcagtcactgcagcccggagcctgtgt\n",
      "gg\n",
      ">G000006925|4689001|4689359|4689001|4689063|+\n",
      "cttcccttatgagttcaaaatcgccttaatggagcagtcactgcaacccagagcctgtgt\n",
      "gg\n",
      ">G000299455|5313369|5315826|5313736|5313798|+\n",
      "cttcccttatgagtttaaaatcgccttagtggagcagtcactgcagcccggagcctgtgt\n",
      "gg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1945366, 1956632)), (1950233, 1950275), '+'), (('G000299455', (2133804, 2148224)), (2143314, 2143356), '-'), (('G000299455', (3316999, 3326584)), (3325041, 3325083), '+')]\n",
      ">G000006925|1945366|1956632|1950233|1950275|+\n",
      "-ccatcggatgctgaatacgcgccgtggtgtgtacctgatccg\n",
      ">G000299455|2133804|2148224|2143314|2143356|-\n",
      "accatcggatgctgaatacgcgccgtggtgtgtacctgatcc-\n",
      ">G000299455|3316999|3326584|3325041|3325083|+\n",
      "-ccatcggataacgaatacgcgccgcgttgtgtacgtgatccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (732803, 732873), '+'), (('G000299455', (2174641, 2181396)), (2181199, 2181269), '+'), (('G000299455', (2910238, 2916987)), (2910359, 2910429), '-')]\n",
      ">G000006925|726235|732999|732803|732873|+\n",
      "-gtggtgcaggtgaaataatccgtgagcgggagctaaatgctcccgtttttcattcaatc\n",
      "aggagcgggta\n",
      ">G000299455|2174641|2181396|2181199|2181269|+\n",
      "-gtggtgcaggtgaaataatccgtgagcgggggcgaaatgcccccgtgtcttttttcaca\n",
      "ggaggctgaga\n",
      ">G000299455|2910238|2916987|2910359|2910429|-\n",
      "ggtggtacaggtgaaataatccgtgagcgggggcgaaatgcccccgtgt-ttttttcaca\n",
      "gggagctggat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2117678, 2117810)), (2117678, 2117810), '+'), (('G000006925', (3969227, 3993100)), (3981955, 3982087), '-'), (('G000299455', (5156132, 5190970)), (5178110, 5178242), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2117678|2117810|2117678|2117810|+\n",
      "-ttagccagataccattccaccgttttacgaatcccgctctcaaacgtttcttgcggttt\n",
      "ccagcccaattcgcggctaattttatcggcatcaatggcgtaacggcggtcatgccccgg\n",
      "gcgatcggcaaca\n",
      ">G000006925|3969227|3993100|3981955|3982087|-\n",
      "attagccagataccactgaaccgttttacgcattccactttcaaaggtttcctgcggcag\n",
      "ccagccaagttcacgggcaattttcgaagcatcaatggcataacgcagatcatgccccgg\n",
      "acggtcagcgac-\n",
      ">G000299455|5156132|5190970|5178110|5178242|+\n",
      "-ttagccagataccactgcaccgttttacgcattccactttcaaaggtttcctgtggcag\n",
      "ccaaccaagttcacgggcaattttcgaagcatcaatggcataacgcagatcatgccccgg\n",
      "acggtcagcgaca\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4628626, 4628837)), (4628801, 4628833), '-'), (('G000006925', (4637073, 4637285)), (4637249, 4637281), '-'), (('G000006925', (4764217, 4764425)), (4764393, 4764425), '-'), (('G000299455', (4751653, 4751865)), (4751657, 4751689), '+'), (('G000299455', (5315857, 5316391)), (5315861, 5315893), '+')]\n",
      ">G000006925|4628626|4628837|4628801|4628833|-\n",
      "aggttccgacagcggtgacgaacatgcagcgg-\n",
      ">G000006925|4637073|4637285|4637249|4637281|-\n",
      "gggttccgacagcgatgacgaacatgcggcgg-\n",
      ">G000006925|4764217|4764425|4764393|4764425|-\n",
      "gggttccgatagcggtggcgaacatgcggcgg-\n",
      ">G000299455|4751653|4751865|4751657|4751689|+\n",
      "-ggttccgacagcggtggcgaacatgcggtggt\n",
      ">G000299455|5315857|5316391|5315861|5315893|+\n",
      "-ggttccgacagcggtggtgaacatgcggcggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1122423, 1168132)), (1146725, 1146747), '-'), (('G000299455', (3057394, 3103258)), (3078846, 3078868), '+'), (('G000299455', (3697938, 3726006)), (3720498, 3720520), '+')]\n",
      ">G000006925|1122423|1168132|1146725|1146747|-\n",
      "tcttcacgacccggcgttttca-\n",
      ">G000299455|3057394|3103258|3078846|3078868|+\n",
      "-cttcacgacccggcgttttcag\n",
      ">G000299455|3697938|3726006|3720498|3720520|+\n",
      "-cttcacgacccggcgttttcag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4780499, 4780557)), (4780499, 4780557), '+'), (('G000299455', (3155737, 3155795)), (3155737, 3155795), '+'), (('G000299455', (4570894, 4570952)), (4570894, 4570952), '-'), (('G000299455', (4752065, 4752123)), (4752065, 4752123), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4780499|4780557|4780499|4780557|+\n",
      "-gaagtttgaattcattcgaatactttttacgaggttcttttcgccagtcctgtaattc\n",
      ">G000299455|3155737|3155795|3155737|3155795|+\n",
      "-gaagtttgaattcattcgaatactttttacgaggttcttttcgccagtcctgtaattc\n",
      ">G000299455|4570894|4570952|4570894|4570952|-\n",
      "cgaagtttgaattcattcgaatagtttttacgaggttcttttcgccagtcctgtaatt-\n",
      ">G000299455|4752065|4752123|4752065|4752123|-\n",
      "cgaagtttgaattcattcgaatagtttttacgaggttcttttcgccagtcctgtaatt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (434399, 459290)), (445702, 445726), '-'), (('G000299455', (3893653, 3918306)), (3907080, 3907104), '+'), (('G000299455', (4582332, 4582356)), (4582332, 4582356), '-')]\n",
      ">G000006925|434399|459290|445702|445726|-\n",
      "gctgcaacgcctgcataaagcggg-\n",
      ">G000299455|3893653|3918306|3907080|3907104|+\n",
      "-ctgcaacgcctgcataaagcggga\n",
      ">G000299455|4582332|4582356|4582332|4582356|-\n",
      "gctgcaacgactgcataaagcggg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (2977867, 2977934), '-'), (('G000299455', (988799, 1028140)), (1015089, 1015156), '+'), (('G000299455', (2612825, 2613349)), (2613090, 2613157), '-')]\n",
      ">G000006925|2976303|3015537|2977867|2977934|-\n",
      "ttcatcgaacagagagtcgtaaaacttcagcccttcttcgtttggctgagcttcatcacc\n",
      "ttttgga-\n",
      ">G000299455|988799|1028140|1015089|1015156|+\n",
      "-tcatcgaacataccatcgtagaacttcagcccttcttcgtttggctgagcttcatcacc\n",
      "ttttggaa\n",
      ">G000299455|2612825|2613349|2613090|2613157|-\n",
      "ttcatcgaacagatcatcataaaattgtaaaccctcttcattcggctcctgttcgtcacc\n",
      "ttgcgga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3322761, 3361739)), (3352659, 3352774), '+'), (('G000299455', (540757, 615358)), (585330, 585445), '-'), (('G000299455', (1472154, 1477841)), (1475881, 1475996), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3322761|3361739|3352659|3352774|+\n",
      "-cagaccagcaacccgaaaatctttgctggcggcgatatcgtccgtggttccgatctggt\n",
      "cgttaccgccatcgccgaaggtcgcaaagcggcagacggcattatgaactggctgg\n",
      ">G000299455|540757|615358|585330|585445|-\n",
      "ccagaccagcaacccgaaaatctttgctggcggcgatatcgtccgtggttccgatctggt\n",
      "ggtaaccgctattgccgaaggtcgtaaggcggcagacggtattatgaactggctg-\n",
      ">G000299455|1472154|1477841|1475881|1475996|+\n",
      "-cagaccaccaatccgaaaatcttcgccggtggcgacgccgtgcgtggtgcggatctggt\n",
      "ggttaccgcaatggcagaaggacgtcatgcggcacaggggattattgactggctgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3701037, 3701062), '+'), (('G000299455', (269622, 287922)), (276781, 276806), '+'), (('G000299455', (536624, 536888)), (536863, 536888), '+')]\n",
      ">G000006925|3693106|3702974|3701037|3701062|+\n",
      "caccgccatcccggcgcaaatcagc\n",
      ">G000299455|269622|287922|276781|276806|+\n",
      "caccgccatcccggcgcaaatcagc\n",
      ">G000299455|536624|536888|536863|536888|+\n",
      "caccaccattcctgcaacaatcagc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3019264, 3019469), '-'), (('G000299455', (971316, 975991)), (973312, 973517), '+'), (('G000299455', (3400844, 3428172)), (3424591, 3424796), '+')]\n",
      ">G000006925|3016790|3036697|3019264|3019469|-\n",
      "aacgccacgtgccccatctcgtgaccgatcaccgcttcgacttcgttatctgtcatcata\n",
      "tccatcagcccgctatagacgcggatacagccgttagccattgcaaaggcgttcacatcc\n",
      "ttcgccatatacactttgtaattcaccggctgaccgttgatattgttgcctagcgcattg\n",
      "gcaatagttgtcagacgtttagcgt-\n",
      ">G000299455|971316|975991|973312|973517|+\n",
      "-acgccacgtgccccatctcgtgaccgatcaccgcttcgacttcgttatccgtcatcata\n",
      "tccatcagcccgctatagacgcggatacagccgttagccattgcaaaggcgttcacatcc\n",
      "ttcgccatatacactttgtaattcaccggctgaccgttgatattgttgcctagcgcattg\n",
      "gcaatagttgtcagacgtttagcgta\n",
      ">G000299455|3400844|3428172|3424591|3424796|+\n",
      "-acgcgacgtggcccagttcatggcccagaacaccttcaatttcgttgtcattcatcatg\n",
      "tccatcaggccactgtagacacggacacagccgttcgccatcgcccatgcgttgacgtcg\n",
      "ctggtcatataaaccttatagttgaccggcgtgccattaatattgttacccaatgctttg\n",
      "gcgatttttgccagacgtttaccgta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1509064, 1578296)), (1577202, 1577250), '+'), (('G000299455', (2167395, 2167615)), (2167493, 2167541), '-'), (('G000299455', (2923865, 2925880)), (2924790, 2924838), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  d\b\b\r",
      "STEP    35 / 37  d\b\b h\r",
      "STEP    36 / 37  d\b\b h\r",
      "STEP    37 / 37  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  f\b\b\r",
      "STEP    35 / 37  d\b\b\r",
      "STEP    36 / 37  d\b\b h\r",
      "STEP    37 / 37  d\b\b h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1577202|1577250|+\n",
      "-taccagcacctgaatccgcgattatcccatatacctactcgctgattg\n",
      ">G000299455|2167395|2167615|2167493|2167541|-\n",
      "gtaccagcacctgaatccgcgattatcccatataactactcgctgatt-\n",
      ">G000299455|2923865|2925880|2924790|2924838|+\n",
      "-taccagcacctgaatccgcgattatcccatatacctactcgctgattg\n",
      "\n",
      "\n",
      "38 [(('G000006925', (236929, 237114)), (237085, 237112), '+'), (('G000006925', (265374, 265559)), (265377, 265404), '-'), (('G000006925', (494885, 495070)), (494888, 494915), '-'), (('G000006925', (706191, 706376)), (706194, 706221), '-'), (('G000006925', (864207, 864393)), (864211, 864238), '-'), (('G000006925', (901434, 901838)), (901588, 901616), '+'), (('G000006925', (911511, 911696)), (911514, 911541), '-'), (('G000006925', (966076, 980232)), (980050, 980077), '-'), (('G000006925', (1098847, 1099032)), (1099002, 1099029), '+'), (('G000006925', (1189492, 1208094)), (1207912, 1207939), '-'), (('G000006925', (1392018, 1392203)), (1392021, 1392048), '-'), (('G000006925', (1417365, 1417553)), (1417371, 1417398), '-'), (('G000006925', (1501384, 1507637)), (1501539, 1501566), '+'), (('G000006925', (1607379, 1610459)), (1610277, 1610304), '-'), (('G000006925', (1619749, 1619934)), (1619904, 1619931), '+'), (('G000006925', (1695091, 1695276)), (1695094, 1695121), '-'), (('G000006925', (1805163, 1806570)), (1806388, 1806415), '-'), (('G000006925', (2035068, 2035255)), (2035073, 2035100), '-'), (('G000006925', (2360573, 2360758)), (2360728, 2360755), '+'), (('G000006925', (2691924, 2692109)), (2692079, 2692106), '+'), (('G000006925', (2769676, 2769861)), (2769679, 2769706), '-'), (('G000006925', (2963247, 2963432)), (2963250, 2963277), '-'), (('G000006925', (3080669, 3080854)), (3080824, 3080851), '+'), (('G000006925', (3268417, 3281594)), (3281412, 3281439), '-'), (('G000006925', (3514451, 3514636)), (3514606, 3514633), '+'), (('G000006925', (3605879, 3606167)), (3606034, 3606061), '+'), (('G000006925', (3814544, 3814729)), (3814547, 3814574), '-'), (('G000006925', (3830436, 3830622)), (3830440, 3830467), '-'), (('G000006925', (3915855, 3917645)), (3916010, 3916037), '+'), (('G000006925', (3969227, 3993100)), (3992918, 3992945), '-'), (('G000006925', (4116339, 4116524)), (4116342, 4116369), '-'), (('G000006925', (4255239, 4255424)), (4255242, 4255269), '-'), (('G000006925', (4361769, 4361954)), (4361923, 4361950), '+'), (('G000006925', (4607382, 4607555)), (4607385, 4607412), '-'), (('G000006925', (4619148, 4619330)), (4619148, 4619175), '-'), (('G000006925', (4694201, 4694386)), (4694204, 4694231), '-'), (('G000299455', (3155225, 3155276)), (3155231, 3155258), '-'), (('G000299455', (5286183, 5286375)), (5286193, 5286220), '-')]\n",
      ">G000006925|236929|237114|237085|237112|+\n",
      "--agatgtctggaaatataggggcaaatc\n",
      ">G000006925|265374|265559|265377|265404|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|494885|495070|494888|494915|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|706191|706376|706194|706221|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|864207|864393|864211|864238|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|901434|901838|901588|901616|+\n",
      "acagatgtctggaaatataggggcaaat-\n",
      ">G000006925|911511|911696|911514|911541|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|966076|980232|980050|980077|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1098847|1099032|1099002|1099029|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|1189492|1208094|1207912|1207939|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1392018|1392203|1392021|1392048|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1417365|1417553|1417371|1417398|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1501384|1507637|1501539|1501566|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|1607379|1610459|1610277|1610304|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1619749|1619934|1619904|1619931|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|1695091|1695276|1695094|1695121|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|1805163|1806570|1806388|1806415|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|2035068|2035255|2035073|2035100|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|2360573|2360758|2360728|2360755|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|2691924|2692109|2692079|2692106|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|2769676|2769861|2769679|2769706|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|2963247|2963432|2963250|2963277|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|3080669|3080854|3080824|3080851|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|3268417|3281594|3281412|3281439|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|3514451|3514636|3514606|3514633|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|3605879|3606167|3606034|3606061|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|3814544|3814729|3814547|3814574|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|3830436|3830622|3830440|3830467|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|3915855|3917645|3916010|3916037|+\n",
      "-cagatgtctggaaatataggggcaaat-\n",
      ">G000006925|3969227|3993100|3992918|3992945|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|4116339|4116524|4116342|4116369|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|4255239|4255424|4255242|4255269|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|4361769|4361954|4361923|4361950|+\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000006925|4607382|4607555|4607385|4607412|-\n",
      "acaggtgtctggaaatataggggcaaa--\n",
      ">G000006925|4619148|4619330|4619148|4619175|-\n",
      "gctggtttctgaaaatatagaggaaaa--\n",
      ">G000006925|4694201|4694386|4694204|4694231|-\n",
      "acagatgtctggaaatataggggcaaa--\n",
      ">G000299455|3155225|3155276|3155231|3155258|-\n",
      "acaaatacccagaaatagaggttcaaa--\n",
      ">G000299455|5286183|5286375|5286193|5286220|-\n",
      "acaggtgtctggaaatataggggcaaa--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (145138, 145185), '+'), (('G000299455', (4300454, 4333834)), (4303399, 4303446), '-'), (('G000299455', (4516375, 4516518)), (4516375, 4516422), '-')]\n",
      ">G000006925|64491|216702|145138|145185|+\n",
      "-ggagatacatttgccagtggaattacgcgaactttgtgatctcaaca\n",
      ">G000299455|4300454|4333834|4303399|4303446|-\n",
      "tggagatacatttgccagtggaattacgagaactttgtgacctcaac-\n",
      ">G000299455|4516375|4516518|4516375|4516422|-\n",
      "tggagatccatttacctaaagatttacgtgaactgtgcgatctcgac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2118272, 2118477)), (2118272, 2118477), '+'), (('G000006925', (3969227, 3993100)), (3981300, 3981505), '-'), (('G000299455', (5156132, 5190970)), (5178692, 5178897), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2118272|2118477|2118272|2118477|+\n",
      "-tcaccataaacttcatcagtagaaatatgatgaaagcggaaactttttttcttttcatc\n",
      "attcagagcagaccagtaattgcgcgccgcttctaaaagaacataagtccccacaatatt\n",
      "ggtttcaataaatgccgcagggcccgttattgaacggtcaacgtggctctctgccgccag\n",
      "gtgcatcaccgcgtctggctggtgct\n",
      ">G000006925|3969227|3993100|3981300|3981505|-\n",
      "gtcaccatatacttcgtcggtggagatatgatgaaaacggaacgctgattttttatcttc\n",
      "cgtcagcgtattccagtaagcccgcgccgcttcaagcaatgtataagtcccgacaatgtt\n",
      "ggtttcaataaacgctgccgggccgtcaatagaacggtcaacatggctttcggctgccag\n",
      "atgcatgacacagtctggctgatgc-\n",
      ">G000299455|5156132|5190970|5178692|5178897|+\n",
      "-tcaccatatacttcgtcggtggagatatgatgaaaacggaacgctgattttttatcttc\n",
      "cgtcagcgtattccagtaagcccgcgccgcttcaagcaatgtataagtcccgacaatgtt\n",
      "ggtttcaataaacgctgccgggccgtcaatagaacggtcaacatggctttcggctgccag\n",
      "atgcatgacacagtctggctgatgct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4094093, 4116338)), (4096855, 4096899), '+'), (('G000299455', (2682449, 2700553)), (2686143, 2686187), '+'), (('G000299455', (5063005, 5071226)), (5068420, 5068464), '-')]\n",
      ">G000006925|4094093|4116338|4096855|4096899|+\n",
      "-cgcaaaataaggacgccagataatcacgccgctcaccagcagca\n",
      ">G000299455|2682449|2700553|2686143|2686187|+\n",
      "-cgcaaagtacggacgccagataatcaccccggtcaccagcagca\n",
      ">G000299455|5063005|5071226|5068420|5068464|-\n",
      "gcgcaaaataaggacgccagataatcacgccgctcaccagcagc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4289191, 4289701), '-'), (('G000299455', (1030502, 1031792)), (1031220, 1031730), '+'), (('G000299455', (4836149, 4864500)), (4860692, 4861202), '+')]\n",
      ">G000006925|4288513|4315006|4289191|4289701|-\n",
      "gataatggtcagcagaatgccaccagggactttcagtttttccaagccgatgatcaccgc\n",
      "cagacctgccagtgacataatcaccgggaaggtcgcgaaatcacccagcgcaacgggcag\n",
      "accatcaagcgggtttttaatcaccagaccgacaccattagcggcaatgagcagcaggaa\n",
      "cagaccgataccgatccccgtgccgtgcgccacaccgtgcggcaagttgcgcaaaatcca\n",
      "gctacggatacccgtggcagaaattaccgtaaacagcacacccatcaggaacacggcacc\n",
      "cagcgcgacaggtacgctaatatgttgccccagcaccaggctgaatgcggtaaacgccgt\n",
      "cagggagatggcgcaaccaatcgccaacggcagattagcccacagacccatcacgataga\n",
      "accgagtccggcaaccagacaggttgcaacgaaaactgccgcaggcgggaagcctgcttt\n",
      "acccaacatacctggaacgacgatgaccga-\n",
      ">G000299455|1030502|1031792|1031220|1031730|+\n",
      "-ataatcaccaacaagatcccgccgggtacgcggcacttctccagaccgaagatgaccgc\n",
      "cagccccagcaagctcatcatcaccgggaaggaggtaaacgcaccgagcgccactggcaa\n",
      "gccttcaatcgggtttttgataaccatacccacaccgttagcagcaatcagcagcagaaa\n",
      "caggccgataccgatacctgtaccgtgagcgatacccatcggtaaattacgtaagatcca\n",
      "ggtacgcacaccggttacggaaatagcggtgaagatgacgcccatcagaaagaccgcgcc\n",
      "cagtgcgacaggaacgctaatttgttgcccgagtaccaaactgaatgcggtaaatgccgt\n",
      "caaggaaatcgcacaaccaatcgccattggcaaattggcccacaaccccatcagcaacga\n",
      "gccgaagcccgcgaccagacaggtggcaacaaacacagctgcgggaggaaaacctgcttt\n",
      "gcccagcattcccggaacgacgataacggaa\n",
      ">G000299455|4836149|4864500|4860692|4861202|+\n",
      "-ataatggtcagcagaatgccaccagggactttcagcttttccaggccgatgatcaccgc\n",
      "cagacctaccagtgacataatcaccgggaaggtcgcgaaatcacccagcgcaacgggcag\n",
      "accatcaagcgggtttttaatcaccagaccgacaccgttagcggcaatgagcagcaggaa\n",
      "cagaccgataccaatccccgtgccgtgcgccacaccgtgcggcaagttgcgcaaaatcca\n",
      "gctacggatacccgtggcagaaattaccgtaaacagcacacccatcaggaacacggcacc\n",
      "cagcgcgacaggtacgctaatatgttgccccagcaccaggctgaatgcggtaaatgccgt\n",
      "cagggagatggcgcaaccaatcgccaacggcagattagcccacagacccatcacgataga\n",
      "accgagtccggcaaccagacaggttgcaacgaaaactgccgcaggcgggaagcctgcttt\n",
      "acccaacatacctggaacgacgatgaccgag\n",
      "\n",
      "\n",
      "6 [(('G000006925', (722910, 723312)), (723066, 723124), '-'), (('G000006925', (901434, 901838)), (901765, 901823), '-'), (('G000299455', (2172426, 2172592)), (2172508, 2172566), '-'), (('G000299455', (2551218, 2551369)), (2551299, 2551357), '-'), (('G000299455', (2919336, 2919766)), (2919623, 2919681), '+'), (('G000299455', (3607503, 3607572)), (3607515, 3607572), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1118\n",
      "\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 56\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 56\r",
      "   10 / 56\r",
      "   20 / 56\r",
      "   30 / 56\r",
      "   40 / 56\r",
      "   50 / 56\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 55  f\b\b\r",
      "STEP     2 / 55  f\b\b\r",
      "STEP     3 / 55  f\b\b\r",
      "STEP     4 / 55  f\b\b\r",
      "STEP     5 / 55  f\b\b\r",
      "STEP     6 / 55  f\b\b\r",
      "STEP     7 / 55  f\b\b\r",
      "STEP     8 / 55  f\b\b\r",
      "STEP     9 / 55  f\b\b\r",
      "STEP    10 / 55  f\b\b\r",
      "STEP    11 / 55  f\b\b\r",
      "STEP    12 / 55  f\b\b\r",
      "STEP    13 / 55  f\b\b\r",
      "STEP    14 / 55  f\b\b\r",
      "STEP    15 / 55  f\b\b\r",
      "STEP    16 / 55  f\b\b\r",
      "STEP    17 / 55  f\b\b\r",
      "STEP    18 / 55  f\b\b\r",
      "STEP    19 / 55  f\b\b\r",
      "STEP    20 / 55  f\b\b\r",
      "STEP    21 / 55  f\b\b\r",
      "STEP    22 / 55  f\b\b\r",
      "STEP    23 / 55  f\b\b\r",
      "STEP    24 / 55  f\b\b\r",
      "STEP    25 / 55  f\b\b\r",
      "STEP    26 / 55  f\b\b\r",
      "STEP    27 / 55  f\b\b\r",
      "STEP    28 / 55  f\b\b\r",
      "STEP    29 / 55  f\b\b\r",
      "STEP    30 / 55  f\b\b\r",
      "STEP    31 / 55  f\b\b\r",
      "STEP    32 / 55  f\b\b\r",
      "STEP    33 / 55  f\b\b\r",
      "STEP    34 / 55  f\b\b\r",
      "STEP    35 / 55  f\b\b\r",
      "STEP    36 / 55  f\b\b\r",
      "STEP    37 / 55  f\b\b\r",
      "STEP    38 / 55  f\b\b\r",
      "STEP    39 / 55  f\b\b\r",
      "STEP    40 / 55  f\b\b\r",
      "STEP    41 / 55  f\b\b\r",
      "STEP    42 / 55  f\b\b\r",
      "STEP    43 / 55  f\b\b\r",
      "STEP    44 / 55  f\b\b\r",
      "STEP    45 / 55  f\b\b\r",
      "STEP    46 / 55  f\b\b\r",
      "STEP    47 / 55  f\b\b\r",
      "STEP    48 / 55  f\b\b\r",
      "STEP    49 / 55  f\b\b\r",
      "STEP    50 / 55  f\b\b\r",
      "STEP    51 / 55  f\b\b\r",
      "STEP    52 / 55  f\b\b\r",
      "STEP    53 / 55  f\b\b\r",
      "STEP    54 / 55  f\b\b\r",
      "STEP    55 / 55  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 56\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 56\r",
      "   10 / 56\r",
      "   20 / 56\r",
      "   30 / 56\r",
      "   40 / 56\r",
      "   50 / 56\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 55  f\b\b\r",
      "STEP     2 / 55  f\b\b\r",
      "STEP     3 / 55  f\b\b\r",
      "STEP     4 / 55  f\b\b\r",
      "STEP     5 / 55  f\b\b\r",
      "STEP     6 / 55  f\b\b\r",
      "STEP     7 / 55  f\b\b\r",
      "STEP     8 / 55  f\b\b\r",
      "STEP     9 / 55  f\b\b\r",
      "STEP    10 / 55  f\b\b\r",
      "STEP    11 / 55  f\b\b\r",
      "STEP    12 / 55  f\b\b\r",
      "STEP    13 / 55  f\b\b\r",
      "STEP    14 / 55  f\b\b\r",
      "STEP    15 / 55  f\b\b\r",
      "STEP    16 / 55  f\b\b\r",
      "STEP    17 / 55  f\b\b\r",
      "STEP    18 / 55  f\b\b\r",
      "STEP    19 / 55  f\b\b\r",
      "STEP    20 / 55  f\b\b\r",
      "STEP    21 / 55  f\b\b\r",
      "STEP    22 / 55  f\b\b\r",
      "STEP    23 / 55  f\b\b\r",
      "STEP    24 / 55  f\b\b\r",
      "STEP    25 / 55  f\b\b\r",
      "STEP    26 / 55  f\b\b\r",
      "STEP    27 / 55  f\b\b\r",
      "STEP    28 / 55  f\b\b\r",
      "STEP    29 / 55  f\b\b\r",
      "STEP    30 / 55  f\b\b\r",
      "STEP    31 / 55  f\b\b\r",
      "STEP    32 / 55  f\b\b\r",
      "STEP    33 / 55  f\b\b\r",
      "STEP    34 / 55  f\b\b\r",
      "STEP    35 / 55  f\b\b\r",
      "STEP    36 / 55  f\b\b\r",
      "STEP    37 / 55  f\b\b\r",
      "STEP    38 / 55  f\b\b\r",
      "STEP    39 / 55  f\b\b\r",
      "STEP    40 / 55  f\b\b\r",
      "STEP    41 / 55  f\b\b\r",
      "STEP    42 / 55  f\b\b\r",
      "STEP    43 / 55  f\b\b\r",
      "STEP    44 / 55  f\b\b\r",
      "STEP    45 / 55  f\b\b\r",
      "STEP    46 / 55  f\b\b\r",
      "STEP    47 / 55  f\b\b\r",
      "STEP    48 / 55  f\b\b\r",
      "STEP    49 / 55  f\b\b\r",
      "STEP    50 / 55  f\b\b\r",
      "STEP    51 / 55  f\b\b\r",
      "STEP    52 / 55  f\b\b\r",
      "STEP    53 / 55  f\b\b\r",
      "STEP    54 / 55  f\b\b\r",
      "STEP    55 / 55  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|722910|723312|723066|723124|-\n",
      "ctccttcgtgtatttagcatcgagtgcatcagcatcacgctggcgctgctgcatgtca-\n",
      ">G000006925|901434|901838|901765|901823|-\n",
      "ctccgtcgtgtatttagcatcgagtgcatcagcagcacgctggcgctgctgcatgtca-\n",
      ">G000299455|2172426|2172592|2172508|2172566|-\n",
      "ttccctcgagtattttgcatcgagcgcagcaacatcgcgctggcgcacctgcatatca-\n",
      ">G000299455|2551218|2551369|2551299|2551357|-\n",
      "-tccctagtgtatctggcatcgagtgcagcaacatcacgctggcgctgctgtatgtcag\n",
      ">G000299455|2919336|2919766|2919623|2919681|+\n",
      "-tccttcgtgtattttgcatcgagcgcagcaacatcacgctgacgcatctgcatgtcag\n",
      ">G000299455|3607503|3607572|3607515|3607572|+\n",
      "--cctttgtatatctggcatcgagttctgctacgtcacgttgacgcttctgcatgtcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3263685, 3263738), '-'), (('G000299455', (671363, 678362)), (675919, 675972), '+'), (('G000299455', (3879336, 3879389)), (3879336, 3879389), '-')]\n",
      ">G000006925|3246783|3268347|3263685|3263738|-\n",
      "caaactggcgaatcaggtcattgtggcgctgaatattgccgcgatgtcagaag-------\n",
      "-----\n",
      ">G000299455|671363|678362|675919|675972|+\n",
      "---gatggcgggtt------ccgtggtgc---ataccggggaaatcggtgcaggtaacgt\n",
      "cacca\n",
      ">G000299455|3879336|3879389|3879336|3879389|-\n",
      "caaagtggcaaatcagattattgtggcgctcaatattgaagcggtttctgaag-------\n",
      "-----\n",
      "\n",
      "\n",
      "56 [(('G000006925', (263407, 264672)), (263745, 263808), '-'), (('G000006925', (311637, 313002)), (312075, 312138), '-'), (('G000006925', (315983, 317248)), (316847, 316910), '+'), (('G000006925', (325529, 326794)), (325867, 325930), '-'), (('G000006925', (330360, 332058)), (330697, 330760), '-'), (('G000006925', (507802, 509300)), (508138, 508201), '-'), (('G000006925', (699183, 700447)), (700047, 700110), '+'), (('G000006925', (712907, 714170)), (713243, 713306), '-'), (('G000006925', (724190, 725876)), (725473, 725536), '+'), (('G000006925', (825843, 827104)), (826706, 826769), '+'), (('G000006925', (903718, 904988)), (904587, 904650), '+'), (('G000006925', (914596, 915860)), (915460, 915523), '+'), (('G000006925', (916783, 918049)), (917122, 917185), '-'), (('G000006925', (921552, 942506)), (931486, 931549), '-'), (('G000006925', (1078088, 1090857)), (1081580, 1081643), '+'), (('G000006925', (1100850, 1102113)), (1101714, 1101777), '+'), (('G000006925', (1290847, 1306886)), (1306487, 1306550), '+'), (('G000006925', (1388754, 1391565)), (1390638, 1390701), '-'), (('G000006925', (1401103, 1402373)), (1401973, 1402036), '+'), (('G000006925', (1413769, 1415032)), (1414633, 1414696), '+'), (('G000006925', (1425133, 1465118)), (1426609, 1426672), '+'), (('G000006925', (1466327, 1467592)), (1467191, 1467254), '+'), (('G000006925', (1468138, 1500238)), (1475605, 1475668), '+'), (('G000006925', (1509064, 1578296)), (1575441, 1575504), '+'), (('G000006925', (1582509, 1591522)), (1582845, 1582908), '-'), (('G000006925', (1591523, 1592784)), (1592387, 1592450), '+'), (('G000006925', (1828287, 1829620)), (1828623, 1828686), '-'), (('G000006925', (1836153, 1836944)), (1836545, 1836608), '+'), (('G000006925', (1842508, 1869155)), (1843373, 1843436), '+'), (('G000006925', (1883764, 1885448)), (1885047, 1885110), '+'), (('G000006925', (1938560, 1939822)), (1939424, 1939487), '+'), (('G000006925', (1963295, 1964561)), (1964161, 1964224), '+'), (('G000006925', (2050169, 2051841)), (2050506, 2050569), '-'), (('G000006925', (2057197, 2058461)), (2057534, 2057597), '-'), (('G000006925', (2069615, 2084300)), (2071704, 2071767), '+'), (('G000006925', (2069615, 2084300)), (2083373, 2083436), '-'), (('G000006925', (2199538, 2210687)), (2204592, 2204655), '+'), (('G000006925', (2584954, 2636558)), (2616853, 2616916), '+'), (('G000006925', (2694303, 2695567)), (2695167, 2695230), '+'), (('G000006925', (2816684, 2833035)), (2832541, 2832604), '+'), (('G000006925', (3322761, 3361739)), (3354319, 3354382), '+'), (('G000006925', (3857305, 3858568)), (3858169, 3858232), '+'), (('G000006925', (3898713, 3912668)), (3912269, 3912332), '+'), (('G000006925', (4462252, 4478752)), (4463486, 4463549), '+'), (('G000006925', (4532444, 4533709)), (4532782, 4532845), '-'), (('G000006925', (4613266, 4614529)), (4613602, 4613665), '-'), (('G000006925', (4674054, 4674778)), (4674580, 4674643), '-'), (('G000006925', (4695532, 4696775)), (4695848, 4695911), '-'), (('G000006925', (4704225, 4705397)), (4704997, 4705060), '+'), (('G000006925', (4743489, 4744586)), (4743659, 4743722), '-'), (('G000006925', (4765710, 4765908)), (4765844, 4765908), '-'), (('G000006925', (4802939, 4805150)), (4804223, 4804286), '-'), (('G000006925', (4813486, 4814755)), (4813823, 4813886), '-'), (('G000299455', (2052398, 2064114)), (2063508, 2063571), '-'), (('G000299455', (3182222, 3183821)), (3182560, 3182623), '-'), (('G000299455', (4564841, 4566095)), (4565694, 4565757), '+')]\n",
      ">G000006925|263407|264672|263745|263808|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|311637|313002|312075|312138|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|315983|317248|316847|316910|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|325529|326794|325867|325930|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|330360|332058|330697|330760|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|507802|509300|508138|508201|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|699183|700447|700047|700110|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|712907|714170|713243|713306|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|724190|725876|725473|725536|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|825843|827104|826706|826769|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|903718|904988|904587|904650|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|914596|915860|915460|915523|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|916783|918049|917122|917185|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|921552|942506|931486|931549|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|1078088|1090857|1081580|1081643|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1100850|1102113|1101714|1101777|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1290847|1306886|1306487|1306550|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1388754|1391565|1390638|1390701|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccactac\n",
      "tct-\n",
      ">G000006925|1401103|1402373|1401973|1402036|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1413769|1415032|1414633|1414696|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1425133|1465118|1426609|1426672|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccactac\n",
      "tctg\n",
      ">G000006925|1466327|1467592|1467191|1467254|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1468138|1500238|1475605|1475668|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccactac\n",
      "tctg\n",
      ">G000006925|1509064|1578296|1575441|1575504|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1582509|1591522|1582845|1582908|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|1591523|1592784|1592387|1592450|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1828287|1829620|1828623|1828686|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|1836153|1836944|1836545|1836608|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1842508|1869155|1843373|1843436|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1883764|1885448|1885047|1885110|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1938560|1939822|1939424|1939487|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|1963295|1964561|1964161|1964224|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccactac\n",
      "tctg\n",
      ">G000006925|2050169|2051841|2050506|2050569|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagctcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|2057197|2058461|2057534|2057597|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|2069615|2084300|2071704|2071767|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|2069615|2084300|2083373|2083436|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|2199538|2210687|2204592|2204655|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|2584954|2636558|2616853|2616916|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|2694303|2695567|2695167|2695230|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|2816684|2833035|2832541|2832604|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|3322761|3361739|3354319|3354382|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|3857305|3858568|3858169|3858232|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|3898713|3912668|3912269|3912332|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|4462252|4478752|4463486|4463549|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|4532444|4533709|4532782|4532845|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagctcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|4613266|4614529|4613602|4613665|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|4674054|4674778|4674580|4674643|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "acc-\n",
      ">G000006925|4695532|4696775|4695848|4695911|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|4704225|4705397|4704997|4705060|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcctacctgccgggctaatccaccac\n",
      "accg\n",
      ">G000006925|4743489|4744586|4743659|4743722|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcacccacctgccgggctaatccaccac\n",
      "tcc-\n",
      ">G000006925|4765710|4765908|4765844|4765908|-\n",
      "ggtaaagccctgtttatggtgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      ">G000006925|4802939|4805150|4804223|4804286|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000006925|4813486|4814755|4813823|4813886|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000299455|2052398|2064114|2063508|2063571|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000299455|3182222|3183821|3182560|3182623|-\n",
      "ggtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tct-\n",
      ">G000299455|4564841|4566095|4565694|4565757|+\n",
      "-gtaaagccctgtttatggcgctcaggagccagcgcccacctgccgggctaatccaccac\n",
      "tctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (913580, 913905)), (913580, 913905), '+'), (('G000006925', (1418926, 1419252)), (1418926, 1419252), '-'), (('G000299455', (3303151, 3303476)), (3303151, 3303476), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1170\n",
      "\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|913580|913905|913580|913905|+\n",
      "-aggctgccatcatggctactatccttaccctttctcaccctgacgcaaccatcgaaaat\n",
      "ggtcgcgctgtcactacgtctgttgcggttgcggagttcttccggaaaatgcacaaaaac\n",
      "gttattcagaagatcgaaactctcgaatgctcgccggagtttaaccggcttaattttaag\n",
      "cccgtcacctataccgacgccaaaggcgaaaaacgcccaatgtaccaaatcaccaaaaac\n",
      "ggcttcgttttcctggtgatgggcttcactggcaaaaaagccgccgcattcaaggaagcc\n",
      "tacattgctga-gttcgaccgcatgga\n",
      ">G000006925|1418926|1419252|1418926|1419252|-\n",
      "gaggatgccatcatggctactatccctgccctttctcaccctgacgtaaccatcgaaaac\n",
      "ggtcgcgctgtcactacgtctgttgcggttgcagagttcttccacaaacgacacgacaat\n",
      "gtgttacgtgccatcgcaaatactgaatgctcacctgaatttaacgccctcaattttgag\n",
      "gacgtcacctacaccgacaccaaaggcgaaaaacgcccaatgtaccaaatcaccaaaaac\n",
      "ggcttcgttttcctggtgatgggcttcactgggaaaaaagccgctgcattcaaagaggcc\n",
      "tacatcgctgatgttcgaccgcatgg-\n",
      ">G000299455|3303151|3303476|3303151|3303476|+\n",
      "-aggctgccattatggctactatcccagtcctttctcaacctgaaatcaccatttccaat\n",
      "ggtcgcgctgttactacgtcctttgccgttgccgaatatttttgcaagcgacacgataac\n",
      "gtattgcgcgccattgcaaatattgaatgctcagagaaatttaacgccctcaattttgag\n",
      "gacgttacctacaccgacgccaaaggcgaacaacgcccaatgtaccaaatcaccaaaaac\n",
      "ggcttcgtttttctggtgatgggcttcaccggcaaaaaagcagctgcattcaaggaagcc\n",
      "tacattgctga-attcgatcgcatgga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3087285, 3087410), '-'), (('G000299455', (857924, 871846)), (868100, 868225), '+'), (('G000299455', (3152907, 3154652)), (3153420, 3153543), '+')]\n",
      ">G000006925|3083675|3099254|3087285|3087410|-\n",
      "caggtaaagggttacagtgccggaggtccggctgacggaagtttctgacgtgtggctctg\n",
      "ttgttccctgaatctcatgtctgtgtgtatctgcgcacgttgctaacaggagttattatt\n",
      "agcat-\n",
      ">G000299455|857924|871846|868100|868225|+\n",
      "-aggtaaagggttacagtgccggaggtccggctgacggaagtttctgacgtgtggctctg\n",
      "ttgttccctgaatctcatgtctgtgtgtatctgcgcacgttgctaacaggagttattatt\n",
      "agcatg\n",
      ">G000299455|3152907|3154652|3153420|3153543|+\n",
      "-aggtaaagggttacagtgccggaggtccggctg-tgggagtttctgaattgtggttctg\n",
      "ttgttccctgaatgtc-tttctgtacgtatctgtgcatgttactgacaaaggttattatt\n",
      "agctta\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4784400, 4785310)), (4784549, 4784633), '+'), (('G000006925', (4790313, 4790480)), (4790335, 4790419), '+'), (('G000006925', (4816831, 4818356)), (4818251, 4818335), '+'), (('G000299455', (5308545, 5311143)), (5309221, 5309305), '-'), (('G000299455', (5385343, 5385755)), (5385611, 5385695), '+')]\n",
      ">G000006925|4784400|4785310|4784549|4784633|+\n",
      "-gtgatattcggacgaccgtaccggcttgcgccgggcttcggcggttttacacctgtgct\n",
      "acatgtaaccaccgtatgccgcctt\n",
      ">G000006925|4790313|4790480|4790335|4790419|+\n",
      "ggtgaaacctgtcaatacgtaccggcttacgccgggcttcggcggttttactcctctgcc\n",
      "atatgcaacaacggagtgccgcct-\n",
      ">G000006925|4816831|4818356|4818251|4818335|+\n",
      "ggtgaaatctgtcaatcagtaccggctgacgccgggcttcggcggttttactccggtatc\n",
      "acatgtaaccacagcgtgccgcct-\n",
      ">G000299455|5308545|5311143|5309221|5309305|-\n",
      "ggtgaaatctatcaatcagtaccggcttacgccgggcttcggcggttttacacctgtgct\n",
      "acatgtaacctccgtatgccgcct-\n",
      ">G000299455|5385343|5385755|5385611|5385695|+\n",
      "-gtgaaatctatcaatccgtaccggcttacgccgggcttcggcggttttactccggtatc\n",
      "acatgtaaccacagcgtgccgcctt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2479150, 2517237)), (2485507, 2485545), '-'), (('G000006925', (2770832, 2816659)), (2786201, 2786239), '+'), (('G000299455', (1254267, 1277763)), (1259594, 1259632), '-'), (('G000299455', (1551165, 1574182)), (1567767, 1567805), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1078\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1078\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1076\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2479150|2517237|2485507|2485545|-\n",
      "atatgggaaccggcaatgcgttctccttattacctgca-\n",
      ">G000006925|2770832|2816659|2786201|2786239|+\n",
      "-tatgggcacaggtagcgcgttctcactgcttccggcgc\n",
      ">G000299455|1254267|1277763|1259594|1259632|-\n",
      "atatgggcacaggtagcgcattctcactgcttccggcg-\n",
      ">G000299455|1551165|1574182|1567767|1567805|+\n",
      "-tatgggaaccggcaatgcgttctccttattacctgcac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1189492, 1208094)), (1202815, 1202852), '+'), (('G000006925', (4759605, 4759642)), (4759605, 4759642), '+'), (('G000299455', (3026184, 3035686)), (3031239, 3031276), '-'), (('G000299455', (4087856, 4088016)), (4087877, 4087914), '-')]\n",
      ">G000006925|1189492|1208094|1202815|1202852|+\n",
      "--aatatagctggtttaaaaactccgtgaaaggggatga\n",
      ">G000006925|4759605|4759642|4759605|4759642|+\n",
      "-cagtacagttggtttaataatacagtcaaaggagatg-\n",
      ">G000299455|3026184|3035686|3031239|3031276|-\n",
      "-caatatagctggtttaaaaactcggtgaaaggggatg-\n",
      ">G000299455|4087856|4088016|4087877|4087914|-\n",
      "gcagtacagctggtttgataacacagtgaaaggggat--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4497401, 4528664)), (4528150, 4528172), '-'), (('G000299455', (183085, 183661)), (183581, 183606), '+'), (('G000299455', (4674171, 4686420)), (4674663, 4674685), '+')]\n",
      ">G000006925|4497401|4528664|4528150|4528172|-\n",
      "caccaccttgccgct---catatcg-\n",
      ">G000299455|183085|183661|183581|183606|+\n",
      "-accaccttgccgctggctatctcaa\n",
      ">G000299455|4674171|4686420|4674663|4674685|+\n",
      "-accaccttaccgct---catatcga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2045115, 2046271)), (2045211, 2045244), '+'), (('G000006925', (2227521, 2228997)), (2227937, 2227970), '+'), (('G000299455', (2892839, 2893945)), (2892900, 2892933), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2045115|2046271|2045211|2045244|+\n",
      "tctttttcgcttacattgtgtctggttcttccg\n",
      ">G000006925|2227521|2228997|2227937|2227970|+\n",
      "tctttttcgcttacattgtgtctggttcttccg\n",
      ">G000299455|2892839|2893945|2892900|2892933|+\n",
      "atcccaatgattactccctgacaggatttgcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2161028, 2172336)), (2164837, 2164928), '-'), (('G000299455', (1889936, 1901256)), (1894275, 1894366), '+'), (('G000299455', (1889936, 1901256)), (1897356, 1897447), '+')]\n",
      ">G000006925|2161028|2172336|2164837|2164928|-\n",
      "tagattgcatcgcgcggcgacatgccttgctcgcgctcagcagccagcgcgaagtcgatc\n",
      "atcatgatggcgttcttcttcacaataccga-\n",
      ">G000299455|1889936|1901256|1894275|1894366|+\n",
      "-aaatagcttcctgcggcgtcaggttaccgtgccgttgggcttcaagcgcaaaatcgacc\n",
      "atcataatagcgtttttcttcacgatgccgat\n",
      ">G000299455|1889936|1901256|1897356|1897447|+\n",
      "-agattgcatcgcgcggcgacatgccttgctcgcgctcagcagccagcgcgaagtcgatc\n",
      "atcatgatggcgttcttcttcacaataccgat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3096388, 3096609), '-'), (('G000299455', (50331, 50552)), (50331, 50552), '+'), (('G000299455', (857924, 871846)), (858944, 859165), '+'), (('G000299455', (1978312, 1978956)), (1978736, 1978956), '+'), (('G000299455', (3142569, 3143200)), (3142980, 3143200), '+'), (('G000299455', (4771847, 4785261)), (4782024, 4782245), '-')]\n",
      ">G000006925|3083675|3099254|3096388|3096609|-\n",
      "tcagttcagggtgacgctcatcagccggacatacggaccaaaactgtccttacggcgttc\n",
      "tgcaaacacagccagtacaccgggaatatcctgcacttcacggccggtatatgtatccgt\n",
      "gctaccgtgccagcggtatttaccggtacagaacggaaaaagacgggatgcaggatgctg\n",
      "acggtgaatacgcatggcttcaccacgggtgatgattttca-\n",
      ">G000299455|50331|50552|50331|50552|+\n",
      "-cagttcagggtgacgctcatcagccggacatacggaccaaaactgtccttacggcgttc\n",
      "tgcaaacacggccagcacaccgggaatatcctgcacttcacgaccggtatacgcttcagc\n",
      "actgccgtgccagcggtatttgccggtacagaacggaaaaagacgggatgcaggatgctg\n",
      "ttggtgaatacgcatggcttcaccacgggtgatgattttcat\n",
      ">G000299455|857924|871846|858944|859165|+\n",
      "-cagttcagggtgacgctcatcagccggacatacggaccaaaactgtccttacggcgttc\n",
      "tgcaaacacagccagtacaccgggaatatcctgcacttcacggccggtatatgtatccgt\n",
      "gctaccgtgccagcggtatttaccggtacagaacggaaaaagacgggatgcaggatgctg\n",
      "acggtgaatacgcatggcttcaccacgggtgatgattttcat\n",
      ">G000299455|1978312|1978956|1978736|1978956|+\n",
      "--agttcagggtgacgctcatcagccggacatacggaccaaaactgtccttacggcgttc\n",
      "agcaaacacggccagcacaccgggaatatcctgcacttcacgaccggtatacgcttcagc\n",
      "actgccgtgccagcggtatttaccggtacagaacggaaaaagacgggatgtcggatgttg\n",
      "ttggtgaatacgcatggcttcaccacgggtgataattttcat\n",
      ">G000299455|3142569|3143200|3142980|3143200|+\n",
      "--agttcagggtgacgctcatcagccggacatacgggccaaaactgtccttacggcgttc\n",
      "agcaaacacggccagcacaccgggaatatcctgtacttcacgaccggtatacgtctcagt\n",
      "gctgccgtgccagcggtacttaccggtgcagaacggaaaaagacgggatgcaggatgctg\n",
      "acggtgaatacgcatggcttcaccacgggtgatgattttcat\n",
      ">G000299455|4771847|4785261|4782024|4782245|-\n",
      "tcagttcagggtgacgctcatcagccggacatacggaccaaaactgtccttacggcgttc\n",
      "tgcaaacacagccagtacaccgggaatatcctgcacttcacggccggtatatgtatccgt\n",
      "gctaccgtgccagcggtatttaccggtacagaacggaaaaagacgggatgcaggatgctg\n",
      "acggtgaatacgcatggcttcaccacgggtgatgattttca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2008567, 2033069)), (2010142, 2010175), '-'), (('G000299455', (2082804, 2090582)), (2090549, 2090582), '+'), (('G000299455', (3608677, 3610659)), (3610266, 3610299), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2008567|2033069|2010142|2010175|-\n",
      "atatcgttgttattaaatctaattgatacgtta-\n",
      ">G000299455|2082804|2090582|2090549|2090582|+\n",
      "-tatcgttgttattaaatctagttgatacgttaa\n",
      ">G000299455|3608677|3610659|3610266|3610299|-\n",
      "atatcgttgttaataactctaattaatacgcca-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (3658114, 3676993)), (3670026, 3670354), '-'), (('G000006925', (3658114, 3676993)), (3670509, 3670837), '-'), (('G000006925', (3658114, 3676993)), (3670992, 3671320), '-'), (('G000299455', (227444, 242483)), (237576, 237903), '+'), (('G000299455', (227444, 242483)), (238057, 238385), '+'), (('G000299455', (2982866, 2983338)), (2982977, 2983304), '+'), (('G000299455', (2983402, 3007859)), (2983513, 2983839), '+')]\n",
      ">G000006925|3658114|3676993|3670026|3670354|-\n",
      "ccggtgaggcgcaatgttgcgggggctttatccctggtggcattggttgctggagagaga\n",
      "aaacccccgcacgttgcaggtatgcacctgacaacatcacgggggccattcttgaacct-\n",
      "gaacaactcaagaatagccgcgaaacgttgtcattacaacacaggcggctatatgacgct\n",
      "cgcagaactgggcatggccttctggcatgatttagcggctccggtcattgctggcattct\n",
      "tgccagtatgatcgtgaactggctgaacaagcggaagtaacgtgtcatgcgggcgtcagg\n",
      "ctgccgtaatggcaatttgcgcccggacc-\n",
      ">G000006925|3658114|3676993|3670509|3670837|-\n",
      "ccggtgaggcgcaatgttgcgggggcttaatccctggtggcattggttgctggagagaga\n",
      "aaacccccgcacgttgcaggtatgcacctgacaacaccacgggggccattcttgcatcc-\n",
      "aaacaatacaagaatagccgcgaaacgttgtcattacaatacaggcggctatatgacgtt\n",
      "cgcagaactgggcatggccttctggcatgatttagcggctccggtcattgctggtattct\n",
      "tgccagtatgatcgtgaactggctgaacaagcggaagtaacgtgccatgcgggcgtcagg\n",
      "ctgccgtaatggcaatttgcgcccggacc-\n",
      ">G000006925|3658114|3676993|3670992|3671320|-\n",
      "ccggtgaggcgcaatgttgcgggggcttaatccctggtggcattggttgctggagagaga\n",
      "aaacccccgcacgttgcaggtatgtacctgacaacataacgggggctaatcttgaacct-\n",
      "agaccactcaagaatagccgcgaaacgttgtcattacaacacaggcggctatatgacgct\n",
      "cgcagagctgggcatggccttctggcatgatttagcggctccggtcattgctggcattct\n",
      "tgccagtatgatcgtgaactggctgaacaagcggaagtagcgtgtcatgtgggcgtcagg\n",
      "ctgccgtaatggcaatttgcgcccggacc-\n",
      ">G000299455|227444|242483|237576|237903|+\n",
      "-cggtgaggcgcaatgttgcgggggctttatccctggtggcattggttgctggagagaga\n",
      "aaacccccgcacgttgcaggtatatacctgacaacataacgggggctaatcttgaacct-\n",
      "agaccactcaagaatagccgcgaaacgttgtcattacaacacaggcggctatatgacgct\n",
      "cgcagagctgggcatgg-cttctggcatgatttagcggctccggtcattgctggcattct\n",
      "tgccagtatgatcgtgaactggctgaacaagcggaagtagcgtgtcatgtgggcgtcagg\n",
      "ctgccgtaatggcaatttgcgcccggacca\n",
      ">G000299455|227444|242483|238057|238385|+\n",
      "ccggtgaggcgcaatgttgcgggggcttaatccctggtggcattggttgctggagagaga\n",
      "aaacccccgcacgttgcaggtatgcacctgacaacaccacgggggccattcttgcatcc-\n",
      "aaacaatacaagaatagccgcgaaacgttgtcattacaatacaggcggctatatgacgct\n",
      "cgcagaactgggcatggccttctggcatgatttagcggctccggtcattgctggtattct\n",
      "tgccagtatgatcgtgaactggctgaacaagcggaagtaacgtgccatgcgggcgtcagg\n",
      "ctgccgtaatggcaatttgcgcccggacc-\n",
      ">G000299455|2982866|2983338|2982977|2983304|+\n",
      "-cggtgaggcgcaatgttgcgggggctttatccctggtggcattggttgctggaaagaga\n",
      "aaacccccgcacgttgagaggtaaaacctgacaacagaacgggggctaatcttgaaccat\n",
      "caacaattcaagaatagccgcaaaatgttgtcattacaaca-aggcggctatatgacgct\n",
      "cgcgaagtttgccatgattttctggcacgacctggcagcaccgatcctggcggggattat\n",
      "taccgcagcgattgtcggctggtggcgtaaccggaagtaaggtg-aaaacgggcgtcagg\n",
      "ctgccgtgatggcaatatgcgcccgaaacc\n",
      ">G000299455|2983402|3007859|2983513|2983839|+\n",
      "-cggtgaggcgcaatgttgcgggggctttatccctggtggcattggttgctggaaagaga\n",
      "aaacccccgcacgttgagaggtaaaacctgacaacagaacgggggctaatcttgaacca-\n",
      "gaacaattcaagaatagccgcaaaatgttgtcattacaaca-aggcggctatatgacgct\n",
      "cgcgcagtttgccatgattttctggcacgacctggcagcaccgatcctggcgggaattat\n",
      "taccgcagcgattgtcggctggtggcgtaaccggaagtaaggtg-aaaacgggcgtcagg\n",
      "ctgccgtgatggcaatatgcgcccgaaacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3050036, 3050796), '-'), (('G000299455', (942831, 956197)), (946056, 946816), '+'), (('G000299455', (3697938, 3726006)), (3707396, 3708156), '+')]\n",
      ">G000006925|3040656|3065083|3050036|3050796|-\n",
      "aacgctgacgtaaaattgggcgatctgcttattcatgaaggatcggcgaaagatgcgcag\n",
      "aaattcgcagccaaagtctttcatgccgataaaacctattttgtgctgaacggcacatcg\n",
      "gcagcgaataaagtggtgacgaatgcgctgttaacgcgtggcgatctggtgctcttcgac\n",
      "cgtaacaaccataagtcgaatcatcacggcgcgctgattcaggcggggacgacgccggtc\n",
      "tatctggaagcttcacgtaacccgtttggtttcattggcggtattgatgcgcactgtttt\n",
      "aatgaagagtatctgcgccagcaaattcgcgacgttgcgccagaaaaagccgacctgccg\n",
      "cgcccgtttcgcctgacgattattcagctgggaacctatgacggcactgtctataacgcc\n",
      "cgtcaggtgatcgataccattgggcatctgtgtgattacattctgtttgattccgcgtgg\n",
      "gtcggttacgaacagtttatctcgatgatggcggatagctcgccgctgctgttagaactt\n",
      "aacgaaaacgatccggggatttttgtgacccagttggtgcacaaacagcaggctggattc\n",
      "tcacagacgtcgcagatccataaaaaagataaccatatccgcggacaggcgcgtttttgc\n",
      "ccgcataagcggttgaataacgcctttatgctccatgcttctaccagccctttctatccg\n",
      "ctgtttgctgcactggatgttaacgccaaaattcatgaag-\n",
      ">G000299455|942831|956197|946056|946816|+\n",
      "-acgctgacgtaaaattgggcgatctgcttattcatgaaggatcggcgaaagatgcgcag\n",
      "aaattcgcagccaaagtctttcatgccgataaaacctattttgtgctgaacggcacatcg\n",
      "gcagcgaataaagtggtgacgaatgcgctgttaacgcgtggcgatctggtgctcttcgac\n",
      "cgtaacaaccataagtcgaatcatcacggcgcgctgattcaggcgggggcgacgccggtc\n",
      "tatctggaagcttcacgcaacccgtttggtttcattggcggtattgatgcgcactgtttt\n",
      "aatgaagagtatctgcgccagcaaattcgcgacgttgcgccagaaaaagccgagctgccg\n",
      "cgcccgtttcgcctggcgattattcagctgggaacctatgacggaactgtctataacgcc\n",
      "cgtcaggtgatcgataccgttgggcatctgtgtgattacattctgtttgattccgcgtgg\n",
      "gtcggttacgaacaatttatcccgatgatggcggatagctcgccgctgctgttagaactt\n",
      "aacgaaaacgatccggggatttttgtgacccagtcggtgcacaaacagcaggctggattc\n",
      "tcacagacgtcgcagatccataaaaaagataaccatatccgcggacaggcgcgtttttgc\n",
      "ccgcataagcggttgaataacgcctttatgctccatgcttctaccagccctttctatccg\n",
      "ctgtttgctgcactggatgttaacgccaaaattcatgaagg\n",
      ">G000299455|3697938|3726006|3707396|3708156|+\n",
      "-atgctgacgtagcaatgggcgatctgctgattcatgaaggtgcgccatgtattgcacaa\n",
      "cagcatgcggctaaagtctttaatgccgataaaacctatttcgttttaaacggtacttca\n",
      "tcttctaataaagtggtcttaaacgcattgctgacgccgggcgatctggtgttgtttgac\n",
      "cgcaataaccataaatctaaccaccacggtgcgttgctacaggctggcgcgacaccggtt\n",
      "tatctggaaaccgcacgtaacccgtatggctttatcggtggcattgatgcgcactgtttt\n",
      "gaagaaagttacctgcgtgagctgattacagaagttgcaccgcagcgggcaaaagaggct\n",
      "cgtcctttccgcctcgccgttattcagctaggcacttacgacggtacgatttataacgcc\n",
      "cgtcaggtggtggataaaattggtcatctgtgtgactacatcctctttgactccgcatgg\n",
      "gtcggctatgaacagtttattccgatgatggctgactgttcgccgctgttgctggatctt\n",
      "aatgagaacgatccgggtattctggttacgcaatccgtacataaacagcaggctggtttt\n",
      "tctcaaacttcacaaatccataaaaaagacagccacatcaaagggcaacagcgttatgta\n",
      "ccgcacaaacgcatgaacaacgcctttatgatgcacgcctccaccagcccgttctatccg\n",
      "ctgtttgccgcactggatatcaacgccaaaatgcatgaagg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3087927, 3088582), '-'), (('G000299455', (54863, 56031)), (54863, 55515), '+'), (('G000299455', (857924, 871846)), (866931, 867586), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3087927|3088582|-\n",
      "ccagcgttccgccatcatccacgcgggtattagtggctgtgtgtccggtcagcacatcca\n",
      "ggcgtccgccattttccagtacgacattatcagctttaccctccacaacagagaatgctc\n",
      "ccaggcggtttgtgccggtgacggttgcagctgtactggtaaccagtgcaccgcccgtgt\n",
      "tctgggtgacatcagacgctttaccgccggcattcacctgcagctttcctttctggttga\n",
      "tggtggtatgcgcggcagttcctccttccttaatcacctgccagccatcacggtttatca\n",
      "gcgtctctgttgccgtgccaccgttgtgtacatactggttacccccctccagtcgggtgt\n",
      "tatgtgcctctccgtgtaccgtctggtcaccaccggcgtaaatgagggtgtcccgtgcca\n",
      "tccccgaagaccagataacctgccggccatttttattgatggtggtggattcagccgtcc\n",
      "ctccgaccatctgaccgctggacacattttcagactccggaccaccttctgcaccggtgt\n",
      "tgacgatggttccggttgccagtccgccatgttttatggtctggtaaccatcctggttaa\n",
      "taattgtaccggcagctttccctcccccgtgtacccactgctcacctctgttatc-\n",
      ">G000299455|54863|56031|54863|55515|+\n",
      "-cagggttccgccgtcatccacccgtgttttctgcgctgaatggccttccagtacatcca\n",
      "ggcgaccgccggactccagaacaacgccgtcagccttaccgttttccaccgtgaaattgc\n",
      "ccagacggttgctgccaagaaccgttgccgcagtactggtgaccagtgctccaccctgtt\n",
      "tcagggtgacattcgtggctgtaccaccggcgttcacctgcagtttgcctttctggttaa\n",
      "cggtggtgaaatccgccagaccaccttccttgacaatctgccagccgtcactgtttacaa\n",
      "ccgtgtcagaggctgtgcctccgttgtgcacatactggtaaccgccattcagtgtggtat\n",
      "ccagcgcatgcccgtgtaccgtctggtcgccgccggcataaaccacagtggtatttgctg\n",
      "ttccttcaacagccacaatctgacgaccatttttattgatggtggtacgtacggcatttc\n",
      "cgcgaacaaactgcccggtatcaccattttctgcatctggtcccccttccgcgccggtat\n",
      "tcacaaccgtgtcggttgccattgcgccggatttgacggcctgccagcccttctcattaa\n",
      "tgacggtacccgttgcaatcccgccttcatgtacccactgctcaccgccgttc---\n",
      ">G000299455|857924|871846|866931|867586|+\n",
      "-cagcgttccgccatcatccacacgggtattagtggctgtgtgtccgctcagcacatcca\n",
      "gacgtccgccattttccagtacgacattatcagccttacccgccagaacggagaatgctc\n",
      "ccaggcggtttgtgccggtgacggttgcagctgtgctggtaaccagtgcacctcccgtgt\n",
      "tctgggtgacatcagacgctttaccgccggcattcacctgcagctttcctttctggttga\n",
      "tggtggtatgcgcggtagttcctccttccttaatcacctgccagcccccctcgttaatta\n",
      "ccgtgttcagcgcaagtccgtatttgtgcacatactggtttcctccctccagccgggtgt\n",
      "tatgtgcctcgccgtgtaccgtctggtcaccgccggcgtaaatgagggtgtcccgtgcca\n",
      "tccccgaagaccagataacctgccggccatttttgttgatggtggtggattcagccgtcc\n",
      "ctccgaccatctgaccgctggacacattttcagactccggaccaccttctgcaccggtgt\n",
      "tgacgatggttccggttgcctgcccgccatgtttaatggtctggtagccagactggttaa\n",
      "tcaccgttccggatgcactgccgcctgcatgtatccattgttcaccaccgttaagg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3740233, 3740253), '-'), (('G000299455', (154492, 158991)), (158971, 158991), '+'), (('G000299455', (159106, 161990)), (159106, 159126), '+')]\n",
      ">G000006925|3736321|3788106|3740233|3740253|-\n",
      "accctctccgtgaggagagg-\n",
      ">G000299455|154492|158991|158971|158991|+\n",
      "-ccctctccgtgaggagaggg\n",
      ">G000299455|159106|161990|159106|159126|+\n",
      "-ccttctccatgtggagaggg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (63782, 63904), '-'), (('G000299455', (183085, 183661)), (183085, 183207), '-'), (('G000299455', (4383450, 4406324)), (4384034, 4384156), '+')]\n",
      ">G000006925|25217|64488|63782|63904|-\n",
      "aggtcgcttatatggggatattctgccgtcagttagcgccgcagttaccggatatgcagc\n",
      "aaacgctgctggataaacactatctgcgtaagcatggcgcgaaggcatattacgggcagt\n",
      "aa-\n",
      ">G000299455|183085|183661|183085|183207|-\n",
      "aatgcgcctatatgggcctattctcgcgccagcttgcgccgcagctccctgcgatgcaaa\n",
      "acgaactgctggataagcactacctgcgtaagcatggggccaatgcctattacgggcagt\n",
      "aa-\n",
      ">G000299455|4383450|4406324|4384034|4384156|+\n",
      "-ggtcgcttatatggggatattctgccgtcagttagcgccgcagttaccggatatgcagc\n",
      "aaacgctgctggataaacactatctgcgtaagcatggcgcgaaggcatattacgggcagt\n",
      "aat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3027264, 3027430), '+'), (('G000299455', (956352, 970820)), (965619, 965785), '-'), (('G000299455', (1078152, 1154001)), (1084315, 1084481), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3016790|3036697|3027264|3027430|+\n",
      "-aaatccgtgaaagtttgcaggttaaacagagtggctgggcgctgtttaaagagaatagc\n",
      "aacttccgccgcgcggtgttccttggcgtactgttacaggtaatgcagcaattcaccggg\n",
      "atgaacgtcatcatgtattacgcgccgaaaatcttcgaactggcggg\n",
      ">G000299455|956352|970820|965619|965785|-\n",
      "gaaatccgtgaaagtttgcaggttaaacagagtggctgggcgctgtttaaagagaacagc\n",
      "aacttccgccgcgcggtgttccttggcgtactgttacaggtaatgcagcaattcaccggg\n",
      "atgaacgtcatcatgtattacgcgccgaaaatcttcgaactggcgg-\n",
      ">G000299455|1078152|1154001|1084315|1084481|+\n",
      "-aaattcgtgaaagcctgaagttaaaacaaggcggttgggcactgtttaagatcaaccgt\n",
      "aacgtccgtcgtgctgtgtttctcggtatgttgttgcaggcgatgcagcagtttaccggt\n",
      "atgaacatcatcatgtactatgcgccgcgtatcttcaaaatggcggg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (789803, 808424)), (799088, 799147), '-'), (('G000006925', (2069615, 2084300)), (2079956, 2080016), '-'), (('G000299455', (2006434, 2017864)), (2009456, 2009515), '+'), (('G000299455', (3528972, 3554217)), (3531527, 3531586), '+')]\n",
      ">G000006925|789803|808424|799088|799147|-\n",
      "-ttcctgttcgagaaagtaccggtcggtacccgcgtacagtttattgatgagccggtaaa\n",
      "-\n",
      ">G000006925|2069615|2084300|2079956|2080016|-\n",
      "atatctgtttgataatgttcctgtaggcactcgtgtacaaattattgatcagccagtgaa\n",
      "-\n",
      ">G000299455|2006434|2017864|2009456|2009515|+\n",
      "--atctgtttgataatgttcctgtaggcactcgtgtacaaattattgatcagccagtgaa\n",
      "a\n",
      ">G000299455|3528972|3554217|3531527|3531586|+\n",
      "--tcctgttcgagaaagtaccggtcggtacccgcgtacagtttattgatgagccggtaaa\n",
      "a\n",
      "\n",
      "\n",
      "38 [(('G000006925', (236028, 236116)), (236028, 236106), '+'), (('G000006925', (266372, 266460)), (266382, 266460), '-'), (('G000006925', (495883, 495971)), (495893, 495971), '-'), (('G000006925', (707189, 707277)), (707199, 707277), '-'), (('G000006925', (865206, 865294)), (865216, 865294), '-'), (('G000006925', (912509, 912597)), (912519, 912597), '-'), (('G000006925', (981045, 981133)), (981055, 981133), '-'), (('G000006925', (1008867, 1008955)), (1008867, 1008945), '+'), (('G000006925', (1097946, 1098034)), (1097946, 1098024), '+'), (('G000006925', (1208907, 1208995)), (1208917, 1208995), '-'), (('G000006925', (1393016, 1393104)), (1393026, 1393104), '-'), (('G000006925', (1418366, 1418454)), (1418376, 1418454), '-'), (('G000006925', (1500483, 1500571)), (1500483, 1500561), '+'), (('G000006925', (1611272, 1611360)), (1611282, 1611360), '-'), (('G000006925', (1618838, 1618926)), (1618838, 1618916), '+'), (('G000006925', (1696089, 1696177)), (1696099, 1696177), '-'), (('G000006925', (1807383, 1807471)), (1807393, 1807471), '-'), (('G000006925', (2036068, 2036156)), (2036078, 2036156), '-'), (('G000006925', (2359672, 2359760)), (2359672, 2359750), '+'), (('G000006925', (2691023, 2691111)), (2691023, 2691101), '+'), (('G000006925', (2770674, 2770762)), (2770684, 2770762), '-'), (('G000006925', (2964245, 2964333)), (2964255, 2964333), '-'), (('G000006925', (3079767, 3079855)), (3079767, 3079845), '+'), (('G000006925', (3178673, 3178761)), (3178673, 3178751), '+'), (('G000006925', (3282407, 3282495)), (3282417, 3282495), '-'), (('G000006925', (3513550, 3513638)), (3513550, 3513628), '+'), (('G000006925', (3604978, 3605066)), (3604978, 3605056), '+'), (('G000006925', (3815542, 3815630)), (3815552, 3815630), '-'), (('G000006925', (3831435, 3831523)), (3831445, 3831523), '-'), (('G000006925', (3914954, 3915042)), (3914954, 3915032), '+'), (('G000006925', (3993913, 3994001)), (3993923, 3994001), '-'), (('G000006925', (4117337, 4117425)), (4117347, 4117425), '-'), (('G000006925', (4256237, 4256325)), (4256247, 4256325), '-'), (('G000006925', (4360868, 4360956)), (4360868, 4360946), '+'), (('G000006925', (4695199, 4695287)), (4695209, 4695287), '-'), (('G000006925', (4778461, 4778549)), (4778471, 4778549), '-'), (('G000006925', (4778619, 4778953)), (4778825, 4778903), '-'), (('G000299455', (5281068, 5281227)), (5281118, 5281196), '+')]\n",
      ">G000006925|236028|236116|236028|236106|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|266372|266460|266382|266460|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|495883|495971|495893|495971|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|707189|707277|707199|707277|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|865206|865294|865216|865294|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|912509|912597|912519|912597|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|981045|981133|981055|981133|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1008867|1008955|1008867|1008945|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|1097946|1098034|1097946|1098024|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|1208907|1208995|1208917|1208995|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1393016|1393104|1393026|1393104|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1418366|1418454|1418376|1418454|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1500483|1500571|1500483|1500561|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|1611272|1611360|1611282|1611360|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1618838|1618926|1618838|1618916|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|1696089|1696177|1696099|1696177|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|1807383|1807471|1807393|1807471|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|2036068|2036156|2036078|2036156|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|2359672|2359760|2359672|2359750|+\n",
      "-aataccaggagggaagtcttactgctgtggccgcaggagaacaggtcgttcctgcctct\n",
      "gaacttgctgctgccatga\n",
      ">G000006925|2691023|2691111|2691023|2691101|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|2770674|2770762|2770684|2770762|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|2964245|2964333|2964255|2964333|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|3079767|3079855|3079767|3079845|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|3178673|3178761|3178673|3178751|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|3282407|3282495|3282417|3282495|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|3513550|3513638|3513550|3513628|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|3604978|3605066|3604978|3605056|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|3815542|3815630|3815552|3815630|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|3831435|3831523|3831445|3831523|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|3914954|3915042|3914954|3915032|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|3993913|3994001|3993923|3994001|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|4117337|4117425|4117347|4117425|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|4256237|4256325|4256247|4256325|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|4360868|4360956|4360868|4360946|+\n",
      "-aataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatga\n",
      ">G000006925|4695199|4695287|4695209|4695287|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|4778461|4778549|4778471|4778549|-\n",
      "caataccaggaaggaagtcttactgctgtggccgccggagaacaggttgttcctgcctct\n",
      "gaacttgctgccgccatg-\n",
      ">G000006925|4778619|4778953|4778825|4778903|-\n",
      "taatatctggaaggaagtctcactgctgttgctgccggagaggatgtcgttcctgcctca\n",
      "gaactggctgccgctata-\n",
      ">G000299455|5281068|5281227|5281118|5281196|+\n",
      "-aatacctggaaggaactctcaatgctgttgctgctggtgaagacgttgttcctgcttct\n",
      "gaactggccgctgcgataa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3322761, 3361739)), (3336526, 3336575), '-'), (('G000299455', (540757, 615358)), (601530, 601579), '+'), (('G000299455', (4026769, 4031850)), (4030456, 4030505), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  f\b\b\r",
      "STEP    35 / 37  f\b\b\r",
      "STEP    36 / 37  f\b\b\r",
      "STEP    37 / 37  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 38\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 38\r",
      "   10 / 38\r",
      "   20 / 38\r",
      "   30 / 38\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 37  f\b\b\r",
      "STEP     2 / 37  f\b\b\r",
      "STEP     3 / 37  f\b\b\r",
      "STEP     4 / 37  f\b\b\r",
      "STEP     5 / 37  f\b\b\r",
      "STEP     6 / 37  f\b\b\r",
      "STEP     7 / 37  f\b\b\r",
      "STEP     8 / 37  f\b\b\r",
      "STEP     9 / 37  f\b\b\r",
      "STEP    10 / 37  f\b\b\r",
      "STEP    11 / 37  f\b\b\r",
      "STEP    12 / 37  f\b\b\r",
      "STEP    13 / 37  f\b\b\r",
      "STEP    14 / 37  f\b\b\r",
      "STEP    15 / 37  f\b\b\r",
      "STEP    16 / 37  f\b\b\r",
      "STEP    17 / 37  f\b\b\r",
      "STEP    18 / 37  f\b\b\r",
      "STEP    19 / 37  f\b\b\r",
      "STEP    20 / 37  f\b\b\r",
      "STEP    21 / 37  f\b\b\r",
      "STEP    22 / 37  f\b\b\r",
      "STEP    23 / 37  f\b\b\r",
      "STEP    24 / 37  f\b\b\r",
      "STEP    25 / 37  f\b\b\r",
      "STEP    26 / 37  f\b\b\r",
      "STEP    27 / 37  f\b\b\r",
      "STEP    28 / 37  f\b\b\r",
      "STEP    29 / 37  f\b\b\r",
      "STEP    30 / 37  f\b\b\r",
      "STEP    31 / 37  f\b\b\r",
      "STEP    32 / 37  f\b\b\r",
      "STEP    33 / 37  f\b\b\r",
      "STEP    34 / 37  f\b\b\r",
      "STEP    35 / 37  f\b\b\r",
      "STEP    36 / 37  f\b\b\r",
      "STEP    37 / 37  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3322761|3361739|3336526|3336575|-\n",
      "ttcgcagccagtgcgcgggcaatttctacacggcgacgttcacccccgg-\n",
      ">G000299455|540757|615358|601530|601579|+\n",
      "-tcgcagccagtgcgcgggcaatttctacacggcgacgttcacccccgga\n",
      ">G000299455|4026769|4031850|4030456|4030505|+\n",
      "-tcgccgccagcgcgcgggcaatccccacccgctgacgttgaccaccgga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3376233, 3376263), '-'), (('G000299455', (540757, 615358)), (564863, 564893), '+'), (('G000299455', (3773738, 3773768)), (3773738, 3773768), '-')]\n",
      ">G000006925|3361966|3403725|3376233|3376263|-\n",
      "acgcgtgacgacaaagggatggcgaccatc-\n",
      ">G000299455|540757|615358|564863|564893|+\n",
      "-cgcgtgacgacaaagggatggcgaccatcg\n",
      ">G000299455|3773738|3773768|3773738|3773768|-\n",
      "acgcgtgactacaaacggctggcgaccatc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4188297, 4209378)), (4195697, 4196882), '+'), (('G000299455', (489148, 514015)), (497322, 498507), '+'), (('G000299455', (4952297, 4970010)), (4964792, 4965977), '-')]\n",
      ">G000006925|4188297|4209378|4195697|4196882|+\n",
      "-gtctaaagaaaagtttgaacgtacaaaaccgcacgttaacgtcggtactatcggccacg\n",
      "ttgaccatggtaaaacaacgctgaccgctgcaatcactaccgtactggctaaaacctacg\n",
      "gcggtgctgctcgcgcattcgaccagatcgataacgcgccggaagaaaaagctcgtggta\n",
      "tcaccatcaacacttctcacgttgaatacgacaccccgacccgtcactacgcacacgtag\n",
      "actgcccggggcacgccgactatgttaaaaacatgatcaccggtgctgctcagatggacg\n",
      "gcgcgatcctggtagttgctgcgactgacggcccgatgccgcagactcgtgagcacatcc\n",
      "tgctgggtcgtcaggtaggcgttccgtacatcatcgtgttcctgaacaaatgcgacatgg\n",
      "ttgatgacgaagagctgctggaactggttgaaatggaagttcgtgaacttctgtctcagt\n",
      "acgacttcccgggcgacgacactccgatcgttcgtggttctgctctgaaagcgctggaag\n",
      "gcgacgcagagtgggaagcgaaaatcctggaactggctggcttcctggattcttatattc\n",
      "cggaaccagagcgtgcgattgacaagccgttcctgctgccgatcgaagacgtattctcca\n",
      "tctccggtcgtggtaccgttgttaccggtcgtgtagaacgcggtatcatcaaagttggtg\n",
      "aagaagttgaaatcgttggtatcaaagagactcagaagtctacctgtactggcgttgaaa\n",
      "tgttccgcaaactgctggacgaaggccgtgctggtgagaacgtaggtgttctgctgcgtg\n",
      "gtatcaaacgtgaagaaatcgaacgtggtcaggtactggctaagccgggcaccatcaagc\n",
      "cgcacaccaagttcgaatctgaagtgtacattctgtccaaagatgaaggcggccgtcata\n",
      "ctccgttcttcaaaggctaccgtccgcagttctacttccgtactactgacgtgactggta\n",
      "ccatcgaactgccggaaggcgtagagatggtaatgccgggcgacaacatcaaaatggttg\n",
      "ttaccctgatccacccgatcgcgatggacgacggtctgcgtttcgcaatccgtgaaggcg\n",
      "gccgtaccgttggcgcgggcgttgttgctaaagttctgggctaatt\n",
      ">G000299455|489148|514015|497322|498507|+\n",
      "-gtctaaagaaaaatttgaacgtacaaaaccgcacgttaacgttggtactatcggccacg\n",
      "ttgaccacggtaaaactactctgaccgctgcaatcaccaccgtactggctaaaacctacg\n",
      "gcggtgctgctcgtgcattcgaccagatcgataacgcgccggaagaaaaagctcgtggta\n",
      "tcaccatcaacacttctcacgttgaatatgacaccccgacccgtcactacgcacacgtag\n",
      "actgcccggggcacgccgactatgttaaaaacatgatcaccggtgctgctcagatggacg\n",
      "gcgcgatcctggtagttgctgcgactgacggcccgatgccgcagactcgtgagcacatcc\n",
      "tgctgggtcgtcaggtaggcgttccgtacatcatcgtgttcctgaacaaatgcgacatgg\n",
      "ttgatgacgaagagctgctggaactggttgaaatggaagttcgtgaacttctgtctcagt\n",
      "acgacttcccgggcgacgacactccgatcgttcgtggttctgctctgaaagcgctggaag\n",
      "gcgacgcagagtgggaagcgaaaatcctggaactggctggcttcctggattcttatattc\n",
      "cggaaccagagcgtgcgattgacaagccgttcctgctgccgatcgaagacgtattctcca\n",
      "tctccggtcgtggtaccgttgttaccggtcgtgtagaacgcggtatcatcaaagttggtg\n",
      "aagaagttgaaatcgttggtatcaaagagactcagaagtctacctgtactggcgttgaaa\n",
      "tgttccgcaaactgctggacgaaggccgtgctggtgagaacgtaggtgttctgctgcgtg\n",
      "gtatcaaacgtgaagaaatcgaacgtggtcaggtactggctaagccgggcaccatcaagc\n",
      "cgcacaccaagttcgaatctgaagtgtacattctgtccaaagatgaaggcggccgtcata\n",
      "ctccgttcttcaaaggctaccgtccgcagttctacttccgtactactgacgtgactggta\n",
      "ccatcgaactgccggaaggcgtagagatggtaatgccgggcgacaacatcaaaatggttg\n",
      "ttaccctgatccacccgatcgcgatggacgacggtctgcgtttcgcaatccgtgaaggcg\n",
      "gccgtaccgttggcgcgggcgttgttgctaaagttctgggctaatt\n",
      ">G000299455|4952297|4970010|4964792|4965977|-\n",
      "tgtctaaagaaaagtttgaacgtacaaaaccgcacgttaacgtcggtactatcggccacg\n",
      "ttgaccatggtaaaacaacgctgaccgctgcaatcactaccgtactggctaaaacctacg\n",
      "gcggtgctgctcgcgcattcgaccagatcgataacgcgccggaagaaaaagctcgtggta\n",
      "tcaccatcaacacttctcacgttgaatacgacaccccgacccgtcactacgcacacgtag\n",
      "actgcccggggcacgccgactatgttaaaaacatgatcaccggtgctgctcagatggacg\n",
      "gcgcgatcctggtagttgctgcgactgacggcccgatgccgcagactcgtgagcacatcc\n",
      "tgctgggtcgtcaggtaggcgttccgtacatcatcgtgttcctgaacaaatgcgacatgg\n",
      "ttgatgacgaagagctgctggaactggttgaaatggaagttcgtgaacttctgtctcagt\n",
      "acgacttcccgggcgacgacactccgatcgttcgtggttctgctctgaaagcgctggaag\n",
      "gcgacgcagagtgggaagcgaaaatcctggaactggctggcttcctggattcttacattc\n",
      "cggaaccagagcgtgcgattgacaagccgttcctgctgccgatcgaagacgtattctcca\n",
      "tctccggtcgtggtaccgttgttaccggtcgtgtagaacgcggtatcatcaaagttggtg\n",
      "aagaagttgaaatcgttggtatcaaagagactcagaagtctacctgtactggcgttgaaa\n",
      "tgttccgcaaactgctggacgaaggccgtgctggtgagaacgtaggtgttctgctgcgtg\n",
      "gtatcaaacgtgaagaaatcgaacgtggtcaggtactggctaagccgggcaccatcaagc\n",
      "cgcacaccaagttcgaatctgaagtgtacattctgtccaaagatgaaggcggtcgtcata\n",
      "ctccgttcttcaaaggctaccgtccgcagttctacttccgtactactgacgtgactggta\n",
      "ccatcgaactgccggaaggcgtagagatggtaatgccgggcgacaacatcaaaatggttg\n",
      "ttaccctgatccacccgatcgcgatggacgacggtctgcgtttcgcaatccgtgaaggcg\n",
      "gccgtaccgttggcgcgggcgttgtagcaaaagttctgagctaat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1945366, 1956632)), (1951118, 1951148), '+'), (('G000299455', (2133804, 2148224)), (2142441, 2142471), '-'), (('G000299455', (3316999, 3326584)), (3325991, 3326021), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1138\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1138\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1945366|1956632|1951118|1951148|+\n",
      "-gggcaatgcctttatgacaatcgatacagg\n",
      ">G000299455|2133804|2148224|2142441|2142471|-\n",
      "tgggcaatgcctttatgacaatcgatacag-\n",
      ">G000299455|3316999|3326584|3325991|3326021|+\n",
      "-gggcaatacctttatgacagtcgatgcagg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (25217, 64488)), (64210, 64278), '-'), (('G000006925', (4497401, 4528664)), (4528172, 4528240), '+'), (('G000299455', (183085, 183661)), (183513, 183581), '-'), (('G000299455', (4383450, 4406324)), (4383660, 4383728), '+'), (('G000299455', (4674171, 4686420)), (4674595, 4674663), '-')]\n",
      ">G000006925|25217|64488|64210|64278|-\n",
      "tgaaggtacgaaaaagccctcctccgacacgccaactcaccggctgctctatcaggcatt\n",
      "cccctcca-\n",
      ">G000006925|4497401|4528664|4528172|4528240|+\n",
      "-gaaggggagtatcgcccatcttccgacactgcgacgcatctcgaactctaccgtcgtta\n",
      "cccgtcgct\n",
      ">G000299455|183085|183661|183513|183581|-\n",
      "ggaaggtagcaaaaagccctcgtccgacacgccaacgcatctggcgctttaccgtcgcta\n",
      "cgccgaaa-\n",
      ">G000299455|4383450|4406324|4383660|4383728|+\n",
      "-gaaggtacgaaaaagccctcctccgacacgccaactcaccggctgctctatcaggcatt\n",
      "cccctccat\n",
      ">G000299455|4674171|4686420|4674595|4674663|-\n",
      "ggaaggggagtatcgcccatcttccgacaccgcgacgcatcttgaactctaccgtcgtta\n",
      "tccgtcgc-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (741221, 741471), '+'), (('G000006925', (1921807, 1928114)), (1926116, 1926366), '-'), (('G000299455', (2189293, 2192099)), (2189293, 2189543), '+'), (('G000299455', (2902162, 2904160)), (2902162, 2902412), '-'), (('G000299455', (3586763, 3590110)), (3589860, 3590110), '-')]\n",
      ">G000006925|739473|745780|741221|741471|+\n",
      "-tgttttctctcggtgccagtatgacgcttggcggtgttgcacagatgctggcaccgaaa\n",
      "gccagaactccccgtacacagacaacggataacggcaaacagaacacctgtttctcctca\n",
      "ctggataacatggttgcccagggcaatgttctgcctgttctgtacggtgaaatgcgcgtg\n",
      "gggtcgcgggtggtatctcaggagatcagcacggcagacgaaggggatggtggtcaggtt\n",
      "gtggtgattgg\n",
      ">G000006925|1921807|1928114|1926116|1926366|-\n",
      "ctgttttctctcggtgccagtatgacgcttggcggtgttgcacagatgctggcaccgaaa\n",
      "gccagaactccccgtacacagacaacggataacggcaaacagaacacctgtttctcctca\n",
      "ctggataacatggttgcccagggcaatgttctgcctgttctgtacggtgaaatgcgcgtg\n",
      "gggtcgcgggtggtatctcaggagatcagcacggcagacgaaggggatggtggtcaggtt\n",
      "gtggtgattg-\n",
      ">G000299455|2189293|2192099|2189293|2189543|+\n",
      "-tgttttctctcggtgccagtatggtgctcggtggtgtggcgcagatgctggcaccgaaa\n",
      "gccagaactccccgtacacagacaacggataacggtaagcagaacacctatttctcctca\n",
      "ctggataacatggttgcccagggcaatgttctgcctgttctgtacggggaaatgcgcgtg\n",
      "gggtctcgcgtggtttctcaggagatcagcacggcagacgaaggggacggtggtcaggtt\n",
      "gtggtgattgg\n",
      ">G000299455|2902162|2904160|2902162|2902412|-\n",
      "ctcttttcaatgggtgcggcgatgacgcttggtggtgtggcgcagatgctggcaccgaaa\n",
      "gccagaactccccgtacacagacaacggataacggcaaacagaacacctatttctcctca\n",
      "ctggataacatggttgcccagggcaatgtcctgcctgttctgtacggtgaaatgcgcgtg\n",
      "gggtcacgtgtggcatctcaggagatcagcacggcagatgaaggggatggtggtcaggtt\n",
      "gtggtaattg-\n",
      ">G000299455|3586763|3590110|3589860|3590110|-\n",
      "ctgttttctctcggtgccagtatggtgctcggtggtgtggcgcagatgctggcaccgaaa\n",
      "gccagaactccccgtacacagacaacggataacggcaaacagaacacctatttctcctca\n",
      "ctggataacatggttgcccagggcaatgttctgccggttctgtacggtgaaatgcgcgtg\n",
      "gggtcacgcgtggtttctcaggagatcagcacggcagacgaaggggacggtggtcaggtt\n",
      "gtggtgattg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2786930, 2787033), '+'), (('G000299455', (1254267, 1277763)), (1258800, 1258903), '-'), (('G000299455', (1551165, 1574182)), (1568503, 1568606), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1502\n",
      "\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2770832|2816659|2786930|2787033|+\n",
      "-tggcgttgtggtcgatgacggtgattgtcgcgccaatttgcggcccgatcctcggcggt\n",
      "tatatcagcgataattaccactggggctggatattcttcatcaa\n",
      ">G000299455|1254267|1277763|1258800|1258903|-\n",
      "ctggcgttgtggtcgatgacggtgattgtcgcgccaatttgcggcccgatcctcggcggt\n",
      "tatatcagcgataattaccactggggctggatattcttcatca-\n",
      ">G000299455|1551165|1574182|1568503|1568606|+\n",
      "-tggcattatggtcaatgaccgtgattatcgctccgatatgtgggccgatattgggcggt\n",
      "tatatttgcgataactttagctggggttggatatttttaatcaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4143132, 4163489)), (4158153, 4158176), '+'), (('G000299455', (4996216, 5016676)), (5001620, 5001643), '-'), (('G000299455', (5034449, 5060698)), (5058497, 5058520), '+')]\n",
      ">G000006925|4143132|4163489|4158153|4158176|+\n",
      "-atgctggcgttcgatatgggcgg\n",
      ">G000299455|4996216|5016676|5001620|5001643|-\n",
      "gatgctggcgttcgatatgggcg-\n",
      ">G000299455|5034449|5060698|5058497|5058520|+\n",
      "-atgctggcgttcgatatgggcgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189492, 1208094)), (1202852, 1202872), '+'), (('G000299455', (3026184, 3035686)), (3031219, 3031239), '-'), (('G000299455', (4087856, 4088016)), (4087856, 4087877), '-')]\n",
      ">G000006925|1189492|1208094|1202852|1202872|+\n",
      "--attagcctatgaatcctata\n",
      ">G000299455|3026184|3035686|3031219|3031239|-\n",
      "-aattagcctatgaatcctat-\n",
      ">G000299455|4087856|4088016|4087856|4087877|-\n",
      "gacttacaaagtgaatcctat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189492, 1208094)), (1203142, 1203179), '+'), (('G000299455', (3026184, 3035686)), (3030912, 3030949), '-'), (('G000299455', (4087549, 4087586)), (4087549, 4087586), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1189492|1208094|1203142|1203179|+\n",
      "-ttgttgaaatgaactggatccataacagtaaagattt\n",
      ">G000299455|3026184|3035686|3030912|3030949|-\n",
      "tttgttgaaatgaactggatccataacagtaaagatt-\n",
      ">G000299455|4087549|4087586|4087549|4087586|-\n",
      "tttgtagaagtgaactggctgcataacagtaaggatt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3117164, 3135297)), (3135107, 3135147), '+'), (('G000299455', (800023, 810909)), (810778, 810818), '-'), (('G000299455', (1207004, 1224877)), (1217525, 1217565), '+')]\n",
      ">G000006925|3117164|3135297|3135107|3135147|+\n",
      "-ctcatggcaaatccgacgtgtaccagcacccactggccca\n",
      ">G000299455|800023|810909|810778|810818|-\n",
      "gctcatggcaaatccgacgtgtaccagcacccactggccc-\n",
      ">G000299455|1207004|1224877|1217525|1217565|+\n",
      "-ctcatggcaaagccaacgtgtaccagtacccactggccca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3049636, 3049836), '-'), (('G000299455', (942831, 956197)), (947016, 947216), '+'), (('G000299455', (3697938, 3726006)), (3708356, 3708556), '+')]\n",
      ">G000006925|3040656|3065083|3049636|3049836|-\n",
      "tttgaaggatatgccgcggatcagtattttgttgatccgtgcaagctgttactcactaca\n",
      "ccaggtatcaatgccgaaaccggcgaatatagcgactttggcgttccggcgacgattctg\n",
      "gcgcactatctgcgtgagaacggcattgtgccggagaagtgcgatctcaactccattctg\n",
      "ttcttattaactccggcgga-\n",
      ">G000299455|942831|956197|947016|947216|+\n",
      "-ttgaaggatatgccgcggatcagtattttgttgatccgtgcaagctgttactcactaca\n",
      "ccaggtatcaatgccgaaaccggcgaatatagcgactttggcgttccggcgacgattctg\n",
      "gcgcactatctgcgtgagaacggcattgtgccggagaagtgcgatctcaactccattctg\n",
      "ttcttattaactccggcggaa\n",
      ">G000299455|3697938|3726006|3708356|3708556|+\n",
      "-ttgaaggctatgcagagaatcaatactttgtcgatccttgcaaactgttgctgacaacc\n",
      "ccaggtattgatgcacgtaacggcgaatatgaagcgttcggtgtacccgcgacgattctt\n",
      "gctaacttcctgcgcgaaaatggcgtagtgccggaaaaatgcgatcttaactccatcctc\n",
      "ttcctgctgactccggcagaa\n",
      "\n",
      "\n",
      "6 [(('G000006925', (4622488, 4624460)), (4622862, 4622890), '+'), (('G000299455', (4571312, 4571569)), (4571312, 4571340), '+'), (('G000299455', (4752483, 4752699)), (4752483, 4752511), '+'), (('G000299455', (5313369, 5315826)), (5314099, 5314127), '+'), (('G000299455', (5316432, 5316648)), (5316620, 5316648), '-'), (('G000299455', (5333884, 5334100)), (5333884, 5333912), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4622488|4624460|4622862|4622890|+\n",
      "-atgatatctttccctgcaggttcgcgta\n",
      ">G000299455|4571312|4571569|4571312|4571340|+\n",
      "-atgatctcactcccatcaggtacccgta\n",
      ">G000299455|4752483|4752699|4752483|4752511|+\n",
      "-atgatttcactcccatcaggtacccgta\n",
      ">G000299455|5313369|5315826|5314099|5314127|+\n",
      "-atgatatctctccctgcaggttcgcgta\n",
      ">G000299455|5316432|5316648|5316620|5316648|-\n",
      "gatgatttcactcccatcaggtacccgt-\n",
      ">G000299455|5333884|5334100|5333884|5333912|+\n",
      "-atgatttcactcccatcaggtacccgta\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4612466, 4613007)), (4612466, 4612498), '+'), (('G000006925', (4666868, 4668030)), (4666872, 4666904), '+'), (('G000006925', (4747716, 4747948)), (4747915, 4747947), '-'), (('G000299455', (886351, 887494)), (887461, 887493), '-'), (('G000299455', (5346044, 5347202)), (5347166, 5347198), '-')]\n",
      ">G000006925|4612466|4613007|4612466|4612498|+\n",
      "-taaatagctgtgtggaatagtagatcactttg\n",
      ">G000006925|4666868|4668030|4666872|4666904|+\n",
      "-taaatagctgcgcggaatagtagatccctttg\n",
      ">G000006925|4747716|4747948|4747915|4747947|-\n",
      "ctaaataagtgctcggaacagtaaaacactca-\n",
      ">G000299455|886351|887494|887461|887493|-\n",
      "ctaaatagctgcgcggaatagtaggtcacttt-\n",
      ">G000299455|5346044|5347202|5347166|5347198|-\n",
      "ctaaatagctgcgcggaatagtagatcacttt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2569729, 2584886)), (2575508, 2575540), '+'), (('G000006925', (2976303, 3015537)), (2979854, 2979886), '-'), (('G000299455', (988799, 1028140)), (1013137, 1013169), '+')]\n",
      ">G000006925|2569729|2584886|2575508|2575540|+\n",
      "-attgacctgtaagcgagtgtgtcagcaatctc\n",
      ">G000006925|2976303|3015537|2979854|2979886|-\n",
      "cattgaagattgagccagtttgttagcaatct-\n",
      ">G000299455|988799|1028140|1013137|1013169|+\n",
      "-attgaagattgagccagtttgttagcaatctc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3593278, 3593305), '-'), (('G000299455', (335504, 377237)), (342751, 342778), '+'), (('G000299455', (3554411, 3565644)), (3558524, 3558551), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3559544|3600494|3593278|3593305|-\n",
      "gatcacgtcaatttccgcattccacgc-\n",
      ">G000299455|335504|377237|342751|342778|+\n",
      "-atcacgtcaatttccgcattccacgcg\n",
      ">G000299455|3554411|3565644|3558524|3558551|+\n",
      "-atcacgtcaactttgccgttaaacgtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (412497, 434345)), (416655, 416698), '-'), (('G000299455', (3918464, 3964500)), (3936117, 3936160), '+'), (('G000299455', (5428667, 5428710)), (5428667, 5428710), '-')]\n",
      ">G000006925|412497|434345|416655|416698|-\n",
      "gatcaccgcctcgccgaattgaccatgaataaactgtacgaca-\n",
      ">G000299455|3918464|3964500|3936117|3936160|+\n",
      "-atcaccgcctcgccgaattgaccatgaataaactatacgacaa\n",
      ">G000299455|5428667|5428710|5428667|5428710|-\n",
      "gatcatcgccttgcggaactgacaatgaacaaattgtacgaca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1886975, 1912610)), (1902245, 1902291), '-'), (('G000299455', (2199778, 2263225)), (2209977, 2210023), '+'), (('G000299455', (2416691, 2440660)), (2432741, 2432787), '+')]\n",
      ">G000006925|1886975|1912610|1902245|1902291|-\n",
      "atttccacaccgaggtcgccacgtgcaaccattaccacgtcagagg-\n",
      ">G000299455|2199778|2263225|2209977|2210023|+\n",
      "-tttccacaccgaggtcgccacgtgcaaccattaccacgtcagaggc\n",
      ">G000299455|2416691|2440660|2432741|2432787|+\n",
      "-tttctacacccaggtcgccacgcgcaaccatgatgccgtcagaggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2728788, 2734132)), (2732684, 2732724), '-'), (('G000299455', (1302091, 1327808)), (1326359, 1326399), '-'), (('G000299455', (3440781, 3470001)), (3460873, 3460913), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2728788|2734132|2732684|2732724|-\n",
      "gcttcatcgatcaggtcgatggctttatccggcagctgac-\n",
      ">G000299455|1302091|1327808|1326359|1326399|-\n",
      "gcttcatcgatcaggtcgatggctttatccggcagctgac-\n",
      ">G000299455|3440781|3470001|3460873|3460913|+\n",
      "-cttcgtcgataacgtcaatggctttatccggcagatgacg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3592751, 3592791), '-'), (('G000299455', (335504, 377237)), (343265, 343305), '+'), (('G000299455', (3554411, 3565644)), (3559035, 3559075), '+')]\n",
      ">G000006925|3559544|3600494|3592751|3592791|-\n",
      "ccacccactttatgaacgaagcggaacgttgcgaccgcat-\n",
      ">G000299455|335504|377237|343265|343305|+\n",
      "-cacccactttatgaacgaagcggaacgttgcgaccgcatc\n",
      ">G000299455|3554411|3565644|3559035|3559075|+\n",
      "-cacccactttatggatgaagcagaatattgcgaccgcatc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1225684, 1225766)), (1225732, 1225766), '+'), (('G000006925', (4614585, 4614622)), (4614585, 4614619), '+'), (('G000006925', (4625138, 4625228)), (4625157, 4625191), '-'), (('G000006925', (4743337, 4743374)), (4743339, 4743374), '-'), (('G000299455', (3007901, 3026172)), (3012717, 3012751), '+')]\n",
      ">G000006925|1225684|1225766|1225732|1225766|+\n",
      "-gttgggatgagcatatcaaaatcaggaaatgaca\n",
      ">G000006925|4614585|4614622|4614585|4614619|+\n",
      "-gttggagtgaacatatcaaaaacaggtaattaca\n",
      ">G000006925|4625138|4625228|4625157|4625191|-\n",
      "-atcaaattaaatagttcaacatcagagaattata\n",
      ">G000006925|4743337|4743374|4743339|4743374|-\n",
      "tgttggagtaaacatatcaaaaacaggcaattaca\n",
      ">G000299455|3007901|3026172|3012717|3012751|+\n",
      "-gttgaattgaacatatcaaaatcaggcaattaca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4497079, 4497173)), (4497080, 4497100), '-'), (('G000299455', (2818030, 2881682)), (2846331, 2846351), '+'), (('G000299455', (2965979, 2982802)), (2973506, 2973526), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4497079|4497173|4497080|4497100|-\n",
      "gattcgttcagttgccgcct-\n",
      ">G000299455|2818030|2881682|2846331|2846351|+\n",
      "-attcgctcacttgccgcctt\n",
      ">G000299455|2965979|2982802|2973506|2973526|-\n",
      "gattcattcacttgccgcct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2786561, 2786731), '+'), (('G000299455', (1254267, 1277763)), (1259102, 1259272), '-'), (('G000299455', (1551165, 1574182)), (1568134, 1568304), '+')]\n",
      ">G000006925|2770832|2816659|2786561|2786731|+\n",
      "-ttgcgctgtcactggcgacattcatgcaggtgctgaactccaccattgctaacgtggcg\n",
      "atccccactatcgccgggaatctgggctcatcgctcagccaaggaacgtgggtaatcact\n",
      "tctttcggggtggcgaatgccatctcgatcccgcttaccggctggctggca\n",
      ">G000299455|1254267|1277763|1259102|1259272|-\n",
      "attgcgctgtcactggcgacattcatgcaggtgctggactccaccattgctaacgtggcg\n",
      "atccccactatcgccgggaatctgggctcatcgctcagccagggaacgtgggtaatcact\n",
      "tctttcggggtggcgaatgccatctcgatcccgcttaccggctggctggc-\n",
      ">G000299455|1551165|1574182|1568134|1568304|+\n",
      "-ttgcattgtcattagcgacgtttatgcaaatgttggattccactatttctaacgtcgca\n",
      "ataccgacaatatctggctttctgggagcatcaacagacgaaggcacctgggttatcacc\n",
      "tcgtttggcgtagcaaatgccattgcgatccctgttaccggcaggctggca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4490645, 4497023)), (4491303, 4491334), '+'), (('G000299455', (27008, 29721)), (29178, 29209), '+'), (('G000299455', (4688506, 4712288)), (4711599, 4711630), '-')]\n",
      ">G000006925|4490645|4497023|4491303|4491334|+\n",
      "-cgaccagcccgccaatcagcatcagcagata\n",
      ">G000299455|27008|29721|29178|29209|+\n",
      "-cggccagaccgccaatcagcatcagcaaata\n",
      ">G000299455|4688506|4712288|4711599|4711630|-\n",
      "ccgaccagcccgccaatcagcatcagcagat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3049165, 3049506), '-'), (('G000299455', (942831, 956197)), (947346, 947687), '+'), (('G000299455', (3697938, 3726006)), (3708686, 3709027), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3049165|3049506|-\n",
      "accctgcgccagttgtgtcaggagatgcacgatctgtatgtcagtttcgacgtcaaagac\n",
      "ctacaaaaagcgatgttccgccagcagagtttcccgtcagtggtgatgaacccccaggat\n",
      "gcgcatagcgcttatattcgcggtgaagtggagttggtgcggattcgtgatgccgaaggg\n",
      "cgaattgcggcagaaggggcgttgccttatccacctggcgtgctttgcgtggtacccggg\n",
      "gaagtctggggtggggcggttcaacgttatttccttgcgctggaagaaggcgtgaatttg\n",
      "ttgccgggattttcgccggagttgcaaggtgtctatagcga-\n",
      ">G000299455|942831|956197|947346|947687|+\n",
      "-ccctgcgccagttgtgtcaggagatgcacgatctgtatgtcagtttcgacgtcaaagac\n",
      "ctacaaaaagcgatgttccgccagcagagtttcccgtcagtggtgatgaacccccaggat\n",
      "gcgcatagcgcttatattcgcggtgaagtggagttggtgcggattcgtgatgccgaaggg\n",
      "cgaattgcggcagaaggggcgttgccttatccacctggcgtgctttgcgtggtacccggg\n",
      "gaagtctggggtggggcggttcaacgttatttccttgcactggaagaaggggtgaatttg\n",
      "ttgccgggattttcgccggagctgcaaggtgtttatagcgaa\n",
      ">G000299455|3697938|3726006|3708686|3709027|+\n",
      "-ccctgcgtcagttgtgtcaagaaatgcatgatttgtatgcccgtcacaacgtgaaacaa\n",
      "ctgcaaaaagagatgttccgtaaggaacacttcccacgcgtaagcatgaatccgcaagaa\n",
      "gccaactacgcctatttacgcggtgaagtggagctggttcgtctgccggatgcagaaggc\n",
      "cgtatcgctgccgaaggtgcgcttccttatcctccgggtgtgctgtgcgttgttccgggt\n",
      "gaaatctggggtggtgctgttctgcgttacttcagcgctctggaagaagggatcaacctg\n",
      "ttgccaggttttgcaccggagcttcagggtgtctatatcgaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2008567, 2033069)), (2009254, 2009305), '+'), (('G000299455', (2786550, 2804122)), (2796611, 2796662), '+'), (('G000299455', (3608677, 3610659)), (3609364, 3609415), '+')]\n",
      ">G000006925|2008567|2033069|2009254|2009305|+\n",
      "agcatacgatatcggtgcatggaccgacgttctgccagaattcggtggcga\n",
      ">G000299455|2786550|2804122|2796611|2796662|+\n",
      "aatgtacgacatcgaaggctggaccgatatgctgcctgaatttggcggtga\n",
      ">G000299455|3608677|3610659|3609364|3609415|+\n",
      "agcatacgacatcggtgcgtggactgacgttctgccagaattcggtggcga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (711984, 712053)), (711990, 712045), '-'), (('G000006925', (721022, 721098)), (721027, 721082), '-'), (('G000006925', (1936966, 1937042)), (1936981, 1937036), '+'), (('G000299455', (4952297, 4970010)), (4966473, 4966528), '+')]\n",
      ">G000006925|711984|712053|711990|712045|-\n",
      "ggaatcgaactggtgacctactgattacaagtcagttgctctgcctgctgagcta-\n",
      ">G000006925|721022|721098|721027|721082|-\n",
      "-gaatcgaactggtgacctactgattacaagtcagttgctctgcctgctgagctaa\n",
      ">G000006925|1936966|1937042|1936981|1937036|+\n",
      "-gaatcgaactggtgacctactgattacaagtcagttgctctgcctgctgagctaa\n",
      ">G000299455|4952297|4970010|4966473|4966528|+\n",
      "-gaatcgaactggtgacctactgattacaagtcagttgctctacctactgagctaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3262968, 3263065), '-'), (('G000299455', (671363, 678362)), (676592, 676689), '+'), (('G000299455', (3874295, 3874392)), (3874295, 3874392), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1245\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3262968|3263065|-\n",
      "---------------------------------------------------gggccgctg\n",
      "ggcgagaaagtgaatgccagttgggggatctccggcgatggcaaaaccgcgtttattgaa\n",
      "atggcggcggccagtgggctggagctgg\n",
      ">G000299455|671363|678362|676592|676689|+\n",
      "gaagcgatgattgcagccacccagggttctgaacgtcacgcctgggttacagggccgctg\n",
      "ggcgagaaagtgaatgccagttgggggatctccggcg-----------------------\n",
      "----------------------------\n",
      ">G000299455|3874295|3874392|3874295|3874392|-\n",
      "---------------------------------------------------gggccgatg\n",
      "ggcgaaaaagtgaatgctttttatggccttaccggcgacgggaaaacggcggtgattgag\n",
      "atggcggcagcaagtggcctgatgctgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2669115, 2684581)), (2684544, 2684581), '-'), (('G000006925', (2696888, 2728352)), (2696888, 2696925), '-'), (('G000299455', (1345136, 1375168)), (1359684, 1359721), '+')]\n",
      ">G000006925|2669115|2684581|2684544|2684581|-\n",
      "ttcacacatatcacaattgatgcattttttagtgatt-\n",
      ">G000006925|2696888|2728352|2696888|2696925|-\n",
      "ttcacacatatcacaattaatacagcgtttagtaatt-\n",
      ">G000299455|1345136|1375168|1359684|1359721|+\n",
      "-tcacacatatcacaattgatgcattttttagtgatta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4317502, 4317523), '-'), (('G000299455', (4865050, 4904182)), (4867526, 4867547), '-'), (('G000299455', (5194758, 5202769)), (5197982, 5198003), '+')]\n",
      ">G000006925|4315026|4354132|4317502|4317523|-\n",
      "ttaccgagatcgcgcagatgg-\n",
      ">G000299455|4865050|4904182|4867526|4867547|-\n",
      "ttaccgagatcgcgcagatgg-\n",
      ">G000299455|5194758|5202769|5197982|5198003|+\n",
      "-taccgagatcgcgcagatgga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3040656, 3065083)), (3058677, 3058714), '+'), (('G000299455', (4713031, 4748324)), (4741918, 4741955), '+'), (('G000299455', (4769783, 4769911)), (4769874, 4769911), '+'), (('G000299455', (5335900, 5336291)), (5335900, 5335937), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3058677|3058714|+\n",
      "-aagcagtcagacacggaatgagtgatcctcgaaaaag\n",
      ">G000299455|4713031|4748324|4741918|4741955|+\n",
      "-aagcagtcagacacggaatgagtgatcctcgaaaaag\n",
      ">G000299455|4769783|4769911|4769874|4769911|+\n",
      "-aagcagtcagacacggaatgagtgatcctcgtaaaag\n",
      ">G000299455|5335900|5336291|5335900|5335937|-\n",
      "gaagcagtcagacacggaatgagtgatcctcgtaaaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4302719, 4302792), '+'), (('G000299455', (1752647, 1813107)), (1762927, 1763000), '+'), (('G000299455', (4836149, 4864500)), (4847602, 4847675), '-')]\n",
      ">G000006925|4288513|4315006|4302719|4302792|+\n",
      "-tcggttcctggtgggcctactgcgaactcggctggggcggctggtggttctgggatccg\n",
      "gtggaaaacgcctc\n",
      ">G000299455|1752647|1813107|1762927|1763000|+\n",
      "-tcggttccgcatgggcctattacgaactcggctggggcggctggtggttctgggatccg\n",
      "gtagaaaacgcctc\n",
      ">G000299455|4836149|4864500|4847602|4847675|-\n",
      "ctcggttcctggtgggcctattgcgaactgggctggggcggctggtggttctgggatccg\n",
      "gtggaaaacgcct-\n",
      "\n",
      "\n",
      "20 [(('G000006925', (385776, 386112)), (385982, 386030), '-'), (('G000006925', (899355, 899691)), (899437, 899485), '+'), (('G000006925', (2059291, 2059627)), (2059497, 2059545), '-'), (('G000006925', (3421586, 3421922)), (3421668, 3421716), '+'), (('G000006925', (3851554, 3851890)), (3851760, 3851808), '-'), (('G000006925', (4359551, 4359859)), (4359737, 4359785), '-'), (('G000006925', (4622488, 4624460)), (4622961, 4623009), '+'), (('G000006925', (4700925, 4701241)), (4701111, 4701159), '-'), (('G000006925', (4776578, 4776894)), (4776764, 4776812), '-'), (('G000006925', (4781327, 4781663)), (4781409, 4781457), '+'), (('G000006925', (4822840, 4823055)), (4822987, 4823036), '+'), (('G000299455', (1991020, 1991316)), (1991094, 1991142), '+'), (('G000299455', (3215105, 3215383)), (3215176, 3215224), '+'), (('G000299455', (4543449, 4543753)), (4543531, 4543579), '+'), (('G000299455', (4571312, 4571569)), (4571411, 4571460), '+'), (('G000299455', (4752483, 4752699)), (4752582, 4752631), '+'), (('G000299455', (5289627, 5289931)), (5289801, 5289849), '-'), (('G000299455', (5313369, 5315826)), (5314198, 5314246), '+'), (('G000299455', (5316432, 5316648)), (5316500, 5316549), '-'), (('G000299455', (5333884, 5334100)), (5333983, 5334032), '+')]\n",
      ">G000006925|385776|386112|385982|386030|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|899355|899691|899437|899485|+\n",
      "-ctgaaagatgacccgttctccgggcatatcttcgtcttccggggccgc-\n",
      ">G000006925|2059291|2059627|2059497|2059545|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|3421586|3421922|3421668|3421716|+\n",
      "-ctgaaagatgacccgttctccgggcatatcttcgtcttccggggccgc-\n",
      ">G000006925|3851554|3851890|3851760|3851808|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|4359551|4359859|4359737|4359785|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|4622488|4624460|4622961|4623009|+\n",
      "-ctgaaggatgacccgttctccgggcatctgttcatcttccgcggacgc-\n",
      ">G000006925|4700925|4701241|4701111|4701159|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|4776578|4776894|4776764|4776812|-\n",
      "gctgaaagatgacccgttctccgggcatatcttcgtcttccggggccg--\n",
      ">G000006925|4781327|4781663|4781409|4781457|+\n",
      "-ctgaaagatgacccgttctccgggcatatcttcgtcttccggggccgc-\n",
      ">G000006925|4822840|4823055|4822987|4823036|+\n",
      "-ctggatgaagatcccttttctggtcacctgtttatctttcgtggccgac\n",
      ">G000299455|1991020|1991316|1991094|1991142|+\n",
      "-ctgaaagatgacccgatgtccggccacgtcttcatcttccggggacgc-\n",
      ">G000299455|3215105|3215383|3215176|3215224|+\n",
      "-ctgaaagacgatccgatgtcaggtcacgtttttatcttccgtgggcgt-\n",
      ">G000299455|4543449|4543753|4543531|4543579|+\n",
      "-ctgaaagacgatcccatgtccggccatgttttcattttccggggccgc-\n",
      ">G000299455|4571312|4571569|4571411|4571460|+\n",
      "-ctgaatgataatcccttctccggtcacctgtttatcttccgtggccgac\n",
      ">G000299455|4752483|4752699|4752582|4752631|+\n",
      "-ctgaatgataatcccttctccggccacctgtttatcttccgtggccgac\n",
      ">G000299455|5289627|5289931|5289801|5289849|-\n",
      "gctgaaagacgatcccatgtccggccatgttttcattttccggggccg--\n",
      ">G000299455|5313369|5315826|5314198|5314246|+\n",
      "-ctgaaggatgacccgttctccggacacctgttcatcttccgcggacgc-\n",
      ">G000299455|5316432|5316648|5316500|5316549|-\n",
      "gctggatgataaccccttctccggtcacctgtttatcttccgtggccga-\n",
      ">G000299455|5333884|5334100|5333983|5334032|+\n",
      "-ctgaatgataatcccttctccggtcacctgtttatcttccgtggccgac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (434399, 459290)), (457849, 457873), '-'), (('G000006925', (1670306, 1692658)), (1692498, 1692522), '+'), (('G000299455', (2459722, 2494618)), (2473371, 2473395), '-'), (('G000299455', (3893653, 3918306)), (3894932, 3894956), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|434399|459290|457849|457873|-\n",
      "ttcagcggccagacgttcgcgacc-\n",
      ">G000006925|1670306|1692658|1692498|1692522|+\n",
      "-tcagcggccagacgttcgcgccca\n",
      ">G000299455|2459722|2494618|2473371|2473395|-\n",
      "ttcagcggccagacgttcgcgccc-\n",
      ">G000299455|3893653|3918306|3894932|3894956|+\n",
      "-tcagcggccagacgttcgctacca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4094093, 4116338)), (4099301, 4100846), '+'), (('G000299455', (2682449, 2700553)), (2688570, 2690115), '+'), (('G000299455', (5063005, 5071226)), (5064473, 5066018), '-')]\n",
      ">G000006925|4094093|4116338|4099301|4100846|+\n",
      "-ccggggcgtccgcacctttccagtgccactgcaaccagcgacctgagttaacgattgaa\n",
      "ccgttctcttctgcgaagcaggtcgatggcagacggaacacttcggtctggatcttcgac\n",
      "gagtcaacttcgttcagctcaccgtggttctgccagaagttagaggtttcagtgttcagt\n",
      "gggtcgatggtgacgaggaacttcagtttcgacagacaaccgatcactttgtttttgttc\n",
      "gggaatgaggcaacagggttaaagccctggcagatatagccattgaccttgccctctttc\n",
      "atcatctcgaagtattgcaggacgtcgtagcctttatcccacttcggcaaccagtcaaag\n",
      "ccccagctattttccgccgtcgctttatcaccaaagaaggccttcatcatagagacgaag\n",
      "aatttcgggtagttgccccagtagtttacctggccttccagcagcggttttggcgtgttg\n",
      "gcggtaaggtaggtttgcagatcggtctgcttctcgcttggcagcgtcatgtaacctggc\n",
      "agactctgcgacagtagccccaggtccgtcagcccctgaatattggagtgaccgcgcagg\n",
      "gcgttaacgccgccgccggccatccccatattgccgagcagcagctggatcatcgccatc\n",
      "gtacgaatgttttgcgcaccaacggagtgttgcgtccagccgagggcatacaggaacgag\n",
      "gcagttttatcgtgagcactggtttctgcgatgtattcgcagactttcaggaacgcgtct\n",
      "tttggcgtaccacagatgttttcaaccacatctggcgtgtagcgggaaacgtgctgtttc\n",
      "agcaagttccacacgcagcgcggatgttgcagcgtggtatcgcgtttggcgaagccgttt\n",
      "tcgtccagttcataagtccaggaggatttatcgtacttgcgtttttccgcgtcgtagccg\n",
      "gtgaacaggccatcttcaaagccgtaatcctcacgcacgatcaggctggcgttggtatag\n",
      "gcttcggtgtattcgtggttgaatttttcattgttcagcaggtacagcaatacgcctgac\n",
      "aggaaagcaatgtcagtaccggaacgaataggggcatagtagtcagctaccgccgccgta\n",
      "cgcgtaaagcgaggatcgatcacaatcagcttcgcgccgttgtgaattttggcttccatc\n",
      "gcccagcggaacccgaccgggtgagcttcagcggcgttaccgcccatcaccacgacgagg\n",
      "ttggcgttcttgatgtcgacccagtggttggtcatcgcaccgcgaccaaatgttggagca\n",
      "agacttgctaccgttggtccgtgtcagacacgcgcctggttgtcgaccgcgagcataccc\n",
      "agcgcgcgggagaatttttgcgttaaatagccggtttcgttactcgacgcggaagcacac\n",
      "agcatcccggtggagagccagcggttaacagtcacgccttcggcgttttgcgcaatgtag\n",
      "ttagcatcgcggtcttctttcatcagtttggcgatgcgatcaaacg\n",
      ">G000299455|2682449|2700553|2688570|2690115|+\n",
      "-ccggcgcgtcctgacctttccagtgccactgcaaccagcgaccggagttagcgatagaa\n",
      "ccatcctcttcagcaaagcaggtcgaaggcagacggaatacttcagtctgaatagacgcc\n",
      "ggatcgacatcgttcgactcgccgtggttctgccagaaggtagaggtttcagtcaccagc\n",
      "ggatcgataaccaccatgtacttcagcttgctcaggcagctcaccactttgtttttgtcc\n",
      "gggaaggacgcaaccgggttaaagccctggcagaaataaccggtgactttgccttcatcc\n",
      "atcatgttgaaatacttgatgacgtcgtaggtctggtcccacttcggcagccagtcatag\n",
      "ccccagttgttctctttctgcgcggcatcgccatagaaagatttcatcaggctaacgaag\n",
      "aacttcggatagttgctccagtagttcacctgatcagctagcgtcgctttcggcgtgttc\n",
      "gcttccagatacgactgcaaatcaacctgtttttctgacggcagcgtcagataacctggc\n",
      "aggctggtagagagcaggcctaagtcagtcaagccctgaatgttggagtgaccacgcaat\n",
      "gcgttcacgccgccaccggccatacccatgttaccgagcagcaactggatcatcgccata\n",
      "gtacggatgttctgcgcacccacagtgtgctgcgtccagcccagcgcgtacaggaaggtg\n",
      "gttgtgcgatccggtgcgctggtggaggccagcacttcacacactttcaggaagtcggct\n",
      "tttggcgtaccacagatgttttcaacaacgtccggcgtgtagcgggaaacgtgcgctttc\n",
      "agcaggttccacacacagcgcggatgagtcagtgtttcatcgcgtttcgcatagccgttt\n",
      "tcatcgaactgatagttccaggacgatttatcgtattgccgtttttcagcgtcgtagccg\n",
      "ctgaacagaccgtcttcgaaagcaaaatcatcacgcaccagcaggctggcgttggtgtaa\n",
      "tgcttaacgtattcggcgttgattttgtcgttttcgatcaggtagcgcaaaacgccagac\n",
      "aggaacgtaatgtccgtaccggaacgaataggtgcgtaaatatccgccacagaagcggta\n",
      "cgcgtaaaacgaggatcgacaacgatcaaggttgcatcgttgttgtttttcgcttccatc\n",
      "gcccagcggaaaccgacgggatgcgcttcagcagcgttaccgcccattaccattacgacg\n",
      "ttagcgtttttgatatccacccagtggttggtcatcgcaccgcgaccaaatgttggagca\n",
      "agacttgctaccgttggtccgtgtcagacgcgcgcctggttgtctaccgccagcatcccg\n",
      "agggagcgggcaaatttctgtgtcagcatcccggtttcgttgctggcaccggaggcacac\n",
      "agcataccggtagaaagccaacggtttaccgttacgccctgctcgttcttttcaataaag\n",
      "ttagcgtcacggtcagctttcatcagctttgcaatacgggagaatg\n",
      ">G000299455|5063005|5071226|5064473|5066018|-\n",
      "cccggggcgtccgcacctttccagtgccactgcaaccagcgacctgagttaacgattgaa\n",
      "ccgttctcttctgcgaagcaggtcgatggcagacggaacacttcggtctggatcttcgac\n",
      "gagtcaacttcgttcagctcaccgtggttctgccagaagttagaggtttcagtgttcagt\n",
      "gggtcgatggtgacgaggaacttcagtttcgacagacaaccgatcactttgtttttgttc\n",
      "gggaatgaggcaacagggttaaagccctggcagatatagccattgaccttgccctctttc\n",
      "atcatctcgaagtactgcaggacgtcgtagcctttatcccacttcggcaaccagtcaaag\n",
      "ccccagctattttccgccgtcgctttatcaccaaagaaggccttcatcatagagacgaag\n",
      "aatttcgggtagttgccccagtagtttacctggccttccagcagcggttttggcgtgttg\n",
      "gcggtaaggtaggtttgcagatcggtctgcttctcgcttggcagcgtcatgtaacctggc\n",
      "agactctgcgacagtagccccaggtccgtcagcccctgaatattggagtgaccgcgcagg\n",
      "gcgttaacgccgccgccggccatccccatattgccgagcagcagctggatcatcgccatc\n",
      "gtacgaatgttttgcgcaccaacggagtgttgcgtccagccgagggcatacaggaacgag\n",
      "gcagttttatcgtgagcactggtttctgcgatgtattcgcagactttcaggaacgcgtct\n",
      "tttggcgtaccacagatgttttcaaccacatctggcgtgtagcgggaaacgtgctgtttc\n",
      "agcaagttccacacgcagcgcggatgttgcagcgtggtatcgcgtttggcgaagccgttt\n",
      "tcgtccagttcataagtccaggaggatttatcgtacttgcgtttttccgcgtcgtagccg\n",
      "gtgaacaggccatcttcaaagccgtaatcctcacgcactatcaggctggcgttggtatag\n",
      "gcttcggtgtattcgcggttgaatttttcattgttcagcaggtacagcaatacgcctgac\n",
      "aggaaagcaatgtcagtaccggaacgaataggggcatagtagtcagctaccgccgccgta\n",
      "cgcgtaaagcgaggatcgatcacaatcagcttcgcgccgttgtgaattttggcttccatc\n",
      "gcccagcggaacccgaccgggtgagcttcagcggcgttaccgcccatcaccacgacgagg\n",
      "ttggcgttcttgatgtcgacccagtggttggtcatcgcaccgcgaccaaatgttggagca\n",
      "agacttgctaccgttggtccgtgtcagacacgcgcctggttgtcgaccgcgagcataccc\n",
      "agcgcgcgggagaatttttgcgttaaatagccggtttcgttactcgacgcggaagcacac\n",
      "agcatcccggtggagagccagcggttaacagtcacgccttcggcgttttgcgcaatgtag\n",
      "ttagcatcgcggtcttctttcatcagtttggcgatgcgatcaaac-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (747558, 747703)), (747558, 747668), '+'), (('G000006925', (918492, 918601)), (918492, 918601), '+'), (('G000006925', (1404983, 1405128)), (1404983, 1405091), '+'), (('G000006925', (1421708, 1421817)), (1421708, 1421817), '+'), (('G000006925', (1919884, 1920029)), (1919920, 1920029), '-'), (('G000006925', (2049760, 2049869)), (2049760, 2049869), '-'), (('G000006925', (2232486, 2232595)), (2232486, 2232595), '-'), (('G000006925', (2687909, 2688054)), (2687944, 2688054), '-'), (('G000299455', (2892582, 2892727)), (2892618, 2892727), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|747558|747703|747558|747668|+\n",
      "-atgcggataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagcgg\n",
      ">G000006925|918492|918601|918492|918601|+\n",
      "-atgccgataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagcg-\n",
      ">G000006925|1404983|1405128|1404983|1405091|+\n",
      "-atgccgataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagc--\n",
      ">G000006925|1421708|1421817|1421708|1421817|+\n",
      "-atgccgataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagcg-\n",
      ">G000006925|1919884|1920029|1919920|1920029|-\n",
      "gatgcggataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagc--\n",
      ">G000006925|2049760|2049869|2049760|2049869|-\n",
      "gatgccgataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagc--\n",
      ">G000006925|2232486|2232595|2232486|2232595|-\n",
      "gatgccgataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagc--\n",
      ">G000006925|2687909|2688054|2687944|2688054|-\n",
      "gatgcggataatcagcaggtgaaactgtcgatgccggaactggaagaactggcggcagca\n",
      "atggtgcaggcgcaggtcgatcgcaacgatgagatttatcgccgtcagcg-\n",
      ">G000299455|2892582|2892727|2892618|2892727|-\n",
      "gatgctgaagatcaccaggtaaagctgtcagcgcagaaactggaagaactgctcgcatca\n",
      "atggttaaggatgaggttgatcgcaatgatgggatttattgacgtcagc--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2964578, 2976271)), (2967571, 2969115), '-'), (('G000299455', (988799, 1028140)), (1023791, 1025335), '+'), (('G000299455', (4792695, 4794239)), (4792695, 4794239), '+')]\n",
      ">G000006925|2964578|2976271|2967571|2969115|-\n",
      "ttgaaagcagggttatgaggaaccaacatgtctgaacaacacgcacagggcgctgacgcg\n",
      "gtagtcgatcttaacaatgaactgaaaacgcgtcgtgagaagctggcgaacctgcgtgag\n",
      "cagggaattgccttcccgaacgatttccgtcgcgatcatacctctgaccaattgcacgca\n",
      "gaattcgacggtaaagagaacgaagaactggaagcgctgaacatcgaagtcgccgttgct\n",
      "ggccgcatgatgacccgccgtattatgggtaaagcgtctttcgttaccctgcaggatgtg\n",
      "ggtggccgcattcagctgtacgttgcccgtgacgatctgccggaaggcgtttacaacgag\n",
      "cagttcaaaaaatgggacctcggcgacatcctcggcgcgaaaggtaaactgttcaaaacc\n",
      "aaaaccggcgaactgtctatccactgtaccgagctgcgtctgctgaccaaagcactgcgt\n",
      "ccgctgccggataaattccacggcttgcaggatcaggaagcgcgctatcgtcagcgttat\n",
      "ctggatctcatctctaacgatgaatcccgtaacacctttaaagtacgctcgcagatcctc\n",
      "tctggtattcgccagttcatggtgaaccgcggctttatggaagttgaaacgccgatgatg\n",
      "caggtgatcccaggcggtgccgctgcgcgtccgtttatcacccaccataacgcgctggat\n",
      "ctcgacatgtacctgcgtatcgcgccggaactgtacctcaagcgtctggtggtcggtggc\n",
      "ttcgagcgtgtattcgaaatcaaccgtaacttccgtaacgaaggtatttccgtacgtcat\n",
      "aacccagagttcaccatgatggaactctacatggcttacgcagattacaaagatctgatc\n",
      "gagctgaccgaatcgctgttccgtactctggcacaggatattctcggtaagacggaagtg\n",
      "acctacggcgacgtgacgctggacttcggtaaaccgttcgaaaaactgaccatgcgtgaa\n",
      "gcgatcaagaaatatcgcccggaaaccgacatggcggatctggataacttcgactctgcg\n",
      "aaagcgattgctgaatctatcggtatccacgttgagaagagctggggtctgggccgtatc\n",
      "gttaccgagatcttcgaagaagtggcagaagcacatctgattcagccgaccttcattact\n",
      "gaatatccggcagaagtttctccgctggcgcgtcgtaacgacattaacccggaaatcaca\n",
      "gaccgctttgagttcttcatcggtggtcgtgaaatcggtaacggctttagcgagctgaat\n",
      "gacgcggaagatcaggcgcaacgcttcctggatcaggttgccgcgaaagacgcaggtgac\n",
      "gacgaagcgatgttctacgatgaagattacgttaccgcactggaacatggcttaccgccg\n",
      "acagcaggtctggggattggcatcgaccgtatggtaatgctgttcaccaacagccatacc\n",
      "atccgcgacgttattctgttcccggcgatgcgtccggtgaaata-\n",
      ">G000299455|988799|1028140|1023791|1025335|+\n",
      "-tgaaagcagggttatgaggaaccaacatgtctgaacaacacgcacagggcgctgacgcg\n",
      "gtagtcgatcttaacaatgaactgaaaacgcgtcgtgagaagctggcgaacctgcgcgag\n",
      "caggggattgccttcccgaacgatttccgtcgcgatcatacctctgaccaattgcacgca\n",
      "gaattcgacggtaaagagaacgaagaactggaagcgctgaacatcgaagtcgccgttgct\n",
      "ggccgcatgatgacccgccgtattatgggtaaagcgtctttcgttaccctgcaggatgtg\n",
      "ggtggccgcattcagctgtacgttgcccgtgacgatctgccggaaggcgtttacaacgag\n",
      "cagttcaaaaaatgggacctcggcgacatcctcggcgcgaaaggtaaactgttcaaaacc\n",
      "aaaaccggcgaactgtctatccactgtaccgagctgcgtctgctgaccaaagcactgcgt\n",
      "ccgctgccggataaattccacggcttgcaggatcaggaagcgcgctatcgtcagcgttat\n",
      "ctggatctcatctctaacgatgaatcccgtaacacctttaaagtacgctcgcagatcctc\n",
      "tctggtattcgccagttcatggtgaaccgcggctttatggaagttgaaacgccgatgatg\n",
      "caggtgatcccaggcggtgccgctgcgcgtccgtttatcacccaccataacgcgctggat\n",
      "ctcgacatgtacctgcgtatcgcgccggaactgtacctcaagcgtctggtggtcggtggc\n",
      "ttcgagcgtgtattcgaaatcaaccgtaacttccgtaacgaaggtatttccgtacgtcat\n",
      "aacccagagttcaccatgatggaactctacatggcttacgcagattacaaagatctgatc\n",
      "gagctgaccgaatcgctgttccgtactctggcacaggatattctcggtaagacggaagtg\n",
      "acctacggcgacgtgacgctggacttcggtaaaccgttcgaaaaactgaccatgcgtgaa\n",
      "gcgatcaagaaatatcgcccggaaaccgacatggcggatctggataacttcgactctgcg\n",
      "aaagcgattgctgaatctatcggtatccacgttgagaagagctggggtctgggccgtatc\n",
      "gttaccgagatcttcgaagaagtggcagaagcacatctgattcagccgaccttcattact\n",
      "gaatatccggcagaagtttctccgctggcgcgtcgtaacgacattaacccggaaatcaca\n",
      "gaccgctttgagttcttcatcggtggtcgtgaaatcggtaacggctttagcgagctgaat\n",
      "gacgcggaagatcaggcgcaacgcttcctggatcaggttgccgcgaaagacgcaggtgac\n",
      "gacgaagcgatgttctacgatgaagattacgttaccgcactggaacatggcttaccgccg\n",
      "acagcaggtctggggattggcatcgaccgtatggtaatgctgttcaccaacagccatacc\n",
      "atccgcgacgttattctgttcccggcgatgcgtccggtaaaataa\n",
      ">G000299455|4792695|4794239|4792695|4794239|+\n",
      "-tgaaagctggatttagaggaaccaaaatgtctgaacaagaaacacggggagccaatgag\n",
      "gctattgattttaacgatgaactgagaaatcgccgcgaaaaactggcagcactacgtcag\n",
      "caaggtgtggcgtttcccaatgattttcgccgcgaccatacctctgaccagctgcacgaa\n",
      "gagtttgatgcgaaggataaccaggaactggaatccttaaacattgaagtctcggttgct\n",
      "ggccgaatgatgacccgtcgtatcatggggaaagcctcctttgtaacgttgcaggatgtc\n",
      "ggtggccgtattcaactgtacgttgcaagagatagcctgccagaaggtgtttataacgat\n",
      "cagtttaaaaaatgggatctgggtgacattatcggtgcccgcggtacgctgtttaagacg\n",
      "caaaccggtgagctttccattcactgtactgagttgcgcctgctgacaaaagcactacgt\n",
      "cctttaccagataaattccatggtctgcaggatcaggaagtccgttatcgtcaacgttat\n",
      "ctggacctcatcgctaacgataaatcccgtcaaacgtttgttgtccgttcaaaaattctg\n",
      "gccgctatccgccagttcatggtcgcgcgcggctttatggaagtagaaaccccgatgatg\n",
      "caggtaattccaggtggggcatctgctcgcccgtttattacccatcataacgctctggat\n",
      "ttagatatgtacctgcgtatcgcgccggagctgtatctgaaacgtctggttgtaggcggt\n",
      "tttgaacgggtattcgaaatcaaccgtaacttccgtaatgaaggtatttctgttcgccat\n",
      "aatcctgagttcacaatgatggaactctacatggcgtatgcggattaccacgatttgatt\n",
      "gaactgacagagtccctgttccgcaccctggcacaagaggttctgggtaccactaaagtc\n",
      "acttatggcgagcatgtgtttgatttcggcaaaccgtttgaaaaactcaccatgcgcgaa\n",
      "gcaatcaaaaaatatcgtccagaaaccgatatggccgacctggataattttgatgctgct\n",
      "aaagcattagctgaatctatcggtattacggtagagaaaagctgggggctgggacgtatt\n",
      "gtcacggagatctttgatgaagtggcagaagcacatctgattcagccaacctttattacg\n",
      "gaatatccggcagaagtgtcaccgctggcacgccgtaatgatgttaacccagaaatcacc\n",
      "gaccgttttgaattcttcatcggtggtcgtgaaatcggtaatggttttagcgaattaaac\n",
      "gacgcagaagatcaggctgaacgtttccaggaacaggttaacgctaaagctgcaggtgac\n",
      "gacgaagccatgttctatgacgaagattacgtgactgcgctggaatatggtctgccgcca\n",
      "accgctggtctgggtattggtatcgaccgaatgattatgctgtttacaaacagtcatact\n",
      "attcgcgacgttattctcttcccggcgatgcgcccacagaaataa\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3093917, 3093972), '-'), (('G000299455', (52904, 53020)), (52921, 52976), '+'), (('G000299455', (857924, 871846)), (861539, 861594), '+'), (('G000299455', (5407202, 5407257)), (5407202, 5407257), '-')]\n",
      ">G000006925|3083675|3099254|3093917|3093972|-\n",
      "cggataacccgacggaaatgcattcccgggtcatcctggataaacacattgcggt--\n",
      ">G000299455|52904|53020|52921|52976|+\n",
      "--gataaccagccgggaatatgttccctggtcatcttcagtaaacacattgcggtaa\n",
      ">G000299455|857924|871846|861539|861594|+\n",
      "-ggataacccgacggaaatgcattcccgggtcatcctggataaacacattgcggta-\n",
      ">G000299455|5407202|5407257|5407202|5407257|-\n",
      "cggacaaccaggcgaaaatgcgttccctgatcatcctcgataaagacgttccggt--\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1509064, 1578296)), (1577128, 1577172), '+'), (('G000299455', (2167395, 2167615)), (2167571, 2167615), '-'), (('G000299455', (2547171, 2547245)), (2547201, 2547245), '-'), (('G000299455', (2923865, 2925880)), (2924716, 2924760), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1112\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1577128|1577172|+\n",
      "-gcccctccggagaggggctggagagtggcgctatgtgccattcc\n",
      ">G000299455|2167395|2167615|2167571|2167615|-\n",
      "agcccctccggagaggggctggagagtggcgctatgtgccattg-\n",
      ">G000299455|2547171|2547245|2547201|2547245|-\n",
      "agcccctccggagaggggctggagagtggcgctatgtgccattg-\n",
      ">G000299455|2923865|2925880|2924716|2924760|+\n",
      "-gcccctccggagaggggctggagagtggcgctatgtgccattgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4543325, 4545493)), (4543704, 4544017), '-'), (('G000299455', (3786605, 3808525)), (3794498, 3794811), '-'), (('G000299455', (4499584, 4501752)), (4501060, 4501373), '+')]\n",
      ">G000006925|4543325|4545493|4543704|4544017|-\n",
      "ttctggtatcacttcggtattctgttcgaagccctgttcatcctgaccgcgctggatgcg\n",
      "ggtacccgttctggccgctttatgctgcaagacctgctgggtaacttcatcccgttcctg\n",
      "aaaaaaaccgactctctggttgccggtatcatcggtactgcgggctgtgtgggtctgtgg\n",
      "ggctacctgctgtatcagggcgtggtcgatccgctgggcggcgttaagagcctgtggccg\n",
      "ctgttcggtatctccaaccagatgctggcagccgtagcgctggtactgggcaccgttgtg\n",
      "ctgattaagatga-\n",
      ">G000299455|3786605|3808525|3794498|3794811|-\n",
      "ttctggtatcacttcgccattctgtttgaagctctgtttattctgacggcggtagatgcg\n",
      "ggtacgcgtgctgcgcgctttatgttgcaggatctgctgggcgtggtttctccgggcctg\n",
      "aagcggaccgattcactgcctgctaacctgctggcaacagcgctgtgcgtactggcgtgg\n",
      "ggctacttcctccatcagggcgtggtcgatccgctgggcggcattaacactctgtggccg\n",
      "ctgtttggtattgccaaccagatgctggcagggatggcgctgatgctctgtgccgtggtg\n",
      "ttgttcaagatga-\n",
      ">G000299455|4499584|4501752|4501060|4501373|+\n",
      "-tctggtatcacttcggtattctgttcgaagccctgttcatcctgaccgcgctggatgcg\n",
      "ggtacccgttctggccgctttatgctgcaagacctgctgggtaacttcatcccgttcctg\n",
      "aaaaaaaccgactctctggttgccggtatcatcggtactgcgggctgtgtgggtctgtgg\n",
      "ggctacctgctgtatcagggcgtggtcgatccgctgggcggcgttaagagcctgtggccg\n",
      "ctgttcggtatctccaaccagatgctggcagccgtagcgctggtactgggcaccgttgtg\n",
      "ctgattaagatgaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3251811, 3253751), '-'), (('G000299455', (679634, 695969)), (687136, 689076), '+'), (('G000299455', (3428237, 3435098)), (3431881, 3433821), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 5 \n",
      "Reallocating..done. *alloclen = 1096\n",
      "STEP     5 / 5 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 5 \n",
      "Reallocating..done. *alloclen = 1096\n",
      "STEP     5 / 5 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 37\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "   30 / 37\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP    34 / 36  h\n",
      "Reallocating..done. *alloclen = 1048\n",
      "STEP    36 / 36  h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 37\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "   30 / 37\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP    36 / 36  h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3251811|3253751|-\n",
      "tgtgggaaaaggtaatggaaggcatccgtatcgaaaacgcaacccacgcgccggttgatt\n",
      "tcgataccaatattgccaccacaattaccgctcatgatgcgggatatattaaccagccgc\n",
      "tggaaaaaattgttggcctgcaaacggatgcgccgttgaaacgtgcgctacacccgttcg\n",
      "gtggcattaatatgattaaaagttcattccacgcctatggccgtgaaatggacagtgaat\n",
      "ttgaatatctgtttaccgatctgcgtaaaacccataaccagggcgtatttgatgtttact\n",
      "caccggatatgctgcgctgccgtaaatctggcgtgctgaccggtttaccagatggctatg\n",
      "gtcgtgggcgcattatcggtgactatcgccgcgtagcgctgtatggcatcagttatctgg\n",
      "tgcgtgaacgcgaactgcaatttgccgatctccagtctcgtctggaaaaaggcgaggatc\n",
      "tggaagccaccatccgtctgcgtgaggagctggcagagcatcgtcatgcgctgttgcaga\n",
      "ttcaggaaatggcggcgaaatatggctttgatatctctcgcccggcgcagaatgcacagg\n",
      "aagcggtgcagtggctctacttcgcttatctggcggcagtgaaatcgcaaaatggcggcg\n",
      "cgatgtcgctggggcgcacggcatcgttcctggatatctacattgagcgcgactttaaag\n",
      "ctggcgtcctcaatgagcagcaggcacaggaactgatcgatcacttcatcatgaagatcc\n",
      "gtatggtacgcttcctgcgtacaccggaatttgattcgctgttctccggcgacccaatct\n",
      "gggcgacggaagtgatcggcgggatggggctggacggtcgtacgctggtgaccaaaaact\n",
      "ccttccgctatttgcacaccctgcacactatggggccggcaccggaacctaacctgacca\n",
      "ttctttggtcggaagaattaccgattgccttcaaaaaatatgccgcgcaggtgtcgatcg\n",
      "tcacctcttccttgcagtatgaaaatgacgatctgatgcgtactgacttcaacagcgacg\n",
      "attacgcgattgcctgctgcgtcagcccaatggtgattggtaagcaaatgcagttctttg\n",
      "gtgcacgcgctaacctggcgaaaacgctgctctacgcaattaacggcggggtggacgaga\n",
      "agctgaagattcaggtcgggccgaaaacagcaccgctgatggacgacgtgctggattacg\n",
      "acaaagtgatggacagcctcgatcacttcatggactggctggcggtgcagtacatcagcg\n",
      "cgctgaatatcattcactacatgcacgacaagtacagctacgaagcttcgctgatggcgc\n",
      "tgcacgatcgtgatgtctatcgcactatggcatgcggcatcgcgggcctgtcggtggcga\n",
      "cggactccctgtctgccatcaaatatgcccgcgtgaaaccaatccgtgacgaaaacggcc\n",
      "tggcggtggactttgaaatcgacggtgaatatccgcagtacggcaacaacgacgagcgcg\n",
      "tagacagcattgcctgcgacctggttgaacgctttatgaagaaaattaaagcgctgccaa\n",
      "cctatcgcaacgccgtccctacccagtcgattctgactatcacttctaacgtggtgtacg\n",
      "gccagaaaaccggtaatacgccggacggtcgtcgcgccggaacaccgttcgcgccgggcg\n",
      "ctaacccgatgcatggtcgtgaccgcaaaggtgccgtggcctcattgacgtcggtggcga\n",
      "aactgccgttcacctacgccaaagatgggatctcgtacaccttctcaatcgttcctgcgg\n",
      "cgctgggcaaagaagatccagtacgtaaaaccaaccttgtcggcctgctggatgggtatt\n",
      "tccaccacgaagcggatgtc-\n",
      ">G000299455|679634|695969|687136|689076|+\n",
      "-gtgggaaaaagtaatggaaggtatccgtatcgaaaacgcaacccacgcgccggttgatt\n",
      "tcgataccaatattgccaccacaattaccgctcatgatgcgggatatattaaccagccgc\n",
      "tggaaaaaattgttggcctgcaaacggatgcgccgttgaaacgtgcgctacacccgttcg\n",
      "gtggcattaatatgattaaaagttcattccacgcctatggccgagaaatggacagtgaat\n",
      "ttgaatatctgtttaccgatctgcgtaaaacccataaccagggcgtatttgatgtttact\n",
      "caccggatatgctgcgctgccgtaaatctggcgtgctgaccggtttaccagatggctatg\n",
      "gccgtgggcgcattatcggtgactatcgccgcgtagcgctgtatggcatcagttatctgg\n",
      "tacgtgaacgcgaactgcaatttgccgatctccagtctcgtctggaaaaaggcgaggatc\n",
      "tggaagccaccatccgtctgcgtgaggagctggcagagcatcgtcatgcgctgttgcaga\n",
      "ttcaggaaatggcggcgaaatatggctttgatatctctcgcccggcgcagaatgcgcagg\n",
      "aagcggtgcagtggctctacttcgcttatctggcggcagtgaaatcgcaaaatggcggcg\n",
      "cgatgtcgctgggccgcacggcatcgttcctcgatatctacattgagcgcgactttaaag\n",
      "ctggcgtactcaatgagcagcaggcacaggaactgatcgatcacttcatcatgaagatcc\n",
      "gtatggtacgcttcctgcgtacaccggaatttgattcgctgttctccggcgacccaatct\n",
      "gggcgacggaagtgatcggcgggatggggctggacggtcgtacgctggtgaccaaaaact\n",
      "ccttccgctatttgcacaccctgcacactatggggccggcaccggaacctaacctgacca\n",
      "ttctttggtcggaagaattaccgattgccttcaaaaaatatgccgcgcaggtgtcgatcg\n",
      "tcacctcttccttgcagtatgaaaatgacgatctgatgcgtactgacttcaacagcgacg\n",
      "attacgcgattgcctgctgcgtcagcccaatggtgattggtaagcaaatgcagttctttg\n",
      "gtgcacgcgctaacctggcgaaaacgctgctctacgcaattaacggcggggtggacgaga\n",
      "agctgaagattcaggtcgggccgaaaacagcaccgctgatggacgacgtgctggattacg\n",
      "acaaagtgatggacagcctcgatcacttcatggactggctggcggtgcagtacatcagcg\n",
      "cgctgaatatcattcactacatgcacgacaagtacagctacgaagcttcgctgatggcgc\n",
      "tgcacgatcgtgatgtctatcgcactatggcatgcggcatcgcgggcctgtcggtggcga\n",
      "cggactccctgtctgccatcaaatatgcccgcgtgaaaccaatccgtgacgaaaacggcc\n",
      "tggcggtggactttgaaatcgacggtgaatatccgcagtacggcaacaacgacgagcgcg\n",
      "tagacagcattgcctgcgacctggttgaacgctttatgaagaaaattaaagcgctgccaa\n",
      "cctatcgcaacgccgtccctacccagtcgattctgactatcacttctaacgtggtgtacg\n",
      "gccagaaaaccggtaatacgccggacggtcgtcgcgccggaacaccgttcgcgccgggcg\n",
      "ctaacccgatgcatggtcgtgaccgcaaaggtgccgtggcctcattgacgtcggtggcga\n",
      "aactgccgttcacctacgccaaagatgggatctcgtacaccttctcaatcgttcctgcgg\n",
      "cgctgggcaaagaagatccagtacgtaaaaccaaccttgtcggcctgctggatgggtatt\n",
      "tccaccacgaagcggatgtcg\n",
      ">G000299455|3428237|3435098|3431881|3433821|+\n",
      "-gtgggacaaagtaatggaaggcgttaaactggaaaaccgcactcacgcgccagttgact\n",
      "ttgacaccgctgttgcttccaccatcacctctcacgacgctggctacatcaacaaagcgt\n",
      "tggaaaaagttgttggtctacagactgaagctccgctgaaacgtgctcttatcccgttcg\n",
      "gtggtatcaaaatgatcgagggttcctgcaaagcgtacaaccgcgaactggacccgatga\n",
      "tcaaaaaaatcttcactgaataccgtaaaactcacaaccagggcgtgttcgacgtttaca\n",
      "ctccggacatcctgcgttgccgtaaatccggtgttctgaccggtctgccagatgcttatg\n",
      "gccgtggccgtatcatcggtgactaccgtcgcgttgcgctgtacggtatcgactacctga\n",
      "tgaaagacaaatacgctcagttcacctctctgcaggctgatctggaaaacggcgtaaacc\n",
      "tggaacagactatccgtctgcgcgaagaaatcgctgaacagcaccgcgctctgggtcaga\n",
      "tgaaagaaatggctgcgaaatacggctacgacatctctggtccggctaccaacgctcagg\n",
      "aagctatccagtggacttacttcggctacctggctgctgttaagtctcagaacggtgctg\n",
      "caatgtccttcggtcgtacctccaccttcctggatgtgtacatcgaacgtgacctgaaag\n",
      "ctggcaagatcaccgaacaagaagcgcaggaaatggttgaccacctggtcatgaaactgc\n",
      "gtatggttcgcttcctgcgtactccggagtacgatgaactgttctctggcgacccgatct\n",
      "gggcaaccgaatctatcggtggtatgggcctcgacggtcgtaccctggttaccaaaaaca\n",
      "gcttccgtttcctgaacaccctgtacaccatgggtccgtctccggaaccgaacatgacca\n",
      "ttctgtggtctgaaaaactgccgctgaacttcaagaaattcgccgctaaagtgtccatcg\n",
      "acacctcttctctgcagtatgagaacgatgacctgatgcgtccggacttcaacaacgatg\n",
      "actatgctatcgcttgctgcgtaagcccgatgatcgttggtaaacaaatgcagttcttcg\n",
      "gtgcgcgtgcaaacctggcgaaaaccatgctgtacgcaatcaacggcggcgttgacgaaa\n",
      "aactgaaaatgcaggttggtccgaagtctgaaccgatcaaaggcgatctcctgaactacg\n",
      "atgaagtgatggagcgcatggatcacttcatggactggctggctaaacagtacatcactg\n",
      "cactgaacatcatccactacatgcacgacaagtacagctacgaagcctctctgatggcgc\n",
      "tgcacgaccgtgacgttatccgcaccatggcgtgtggtatcgctggtctgtccgttgctg\n",
      "ctgactccctgtctgcaatcaaatatgcgaaagttaaaccgattcgtgacgaagacggtc\n",
      "tggctatcgacttcgaaatcgaaggcgaatacccgcagtttggtaacaatgatccgcgtg\n",
      "tagatgacctggctgttgacctggtagaacgtttcatgaagaaaattcagaaactgcaca\n",
      "cctaccgtgacgctatcccgactcagtctgttctgaccatcacttctaacgttgtgtatg\n",
      "gtaagaaaactggtaacaccccagacggtcgtcgtgctggcgcgccgttcggaccgggtg\n",
      "ctaacccgatgcacggtcgtgaccagaaaggtgctgtagcgtctctgacttccgttgcta\n",
      "aactgccgtttgcttacgctaaagatggtatctcctacaccttctctatcgttccgaacg\n",
      "cactgggtaaagacgacgaagttcgtaagaccaacctggcaggtctgatggatggttact\n",
      "tccaccacgaagcatccatcg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1965007, 1976679)), (1968527, 1968574), '+'), (('G000006925', (4557883, 4573403)), (4558994, 4559041), '-'), (('G000299455', (2122044, 2133716)), (2128448, 2128495), '-'), (('G000299455', (2122044, 2133716)), (2130149, 2130196), '-'), (('G000299455', (2737179, 2754578)), (2748492, 2748539), '+'), (('G000299455', (4487440, 4499283)), (4498125, 4498172), '+')]\n",
      ">G000006925|1965007|1976679|1968527|1968574|+\n",
      "-ctgataatgtcgccaattttttgcgagctggtggcgatctcctgcat\n",
      ">G000006925|4557883|4573403|4558994|4559041|-\n",
      "gctgataatatcggcgattttctgcgaactggtggagatatcgcgca-\n",
      ">G000299455|2122044|2133716|2128448|2128495|-\n",
      "gctgataatgtcggcaattttcttcgaactatcggcgatctcatgca-\n",
      ">G000299455|2122044|2133716|2130149|2130196|-\n",
      "gctgataatgtcgccaattttttgcgagctggtggcgatctcctgca-\n",
      ">G000299455|2737179|2754578|2748492|2748539|+\n",
      "-gcggtgatctcagaaattttcttcgaactcgtggagatagcgcccat\n",
      ">G000299455|4487440|4499283|4498125|4498172|+\n",
      "-ctgataatatcggcgattttctgcgaactggtggagatatcgcgcat\n",
      "\n",
      "\n",
      "37 [(('G000006925', (235782, 235958)), (235832, 235855), '+'), (('G000006925', (266530, 266681)), (266634, 266657), '-'), (('G000006925', (496041, 496215)), (496145, 496168), '-'), (('G000006925', (707347, 707521)), (707451, 707474), '-'), (('G000006925', (865364, 865540)), (865468, 865491), '-'), (('G000006925', (912667, 912841)), (912771, 912794), '-'), (('G000006925', (981203, 981377)), (981307, 981330), '-'), (('G000006925', (1008623, 1008797)), (1008670, 1008693), '+'), (('G000006925', (1097702, 1097876)), (1097749, 1097772), '+'), (('G000006925', (1209065, 1211067)), (1209169, 1209192), '-'), (('G000006925', (1393174, 1393348)), (1393278, 1393301), '-'), (('G000006925', (1418524, 1418698)), (1418628, 1418651), '-'), (('G000006925', (1500239, 1500413)), (1500286, 1500309), '+'), (('G000006925', (1618594, 1618768)), (1618641, 1618664), '+'), (('G000006925', (1696247, 1696421)), (1696351, 1696374), '-'), (('G000006925', (1807541, 1813528)), (1807645, 1807668), '-'), (('G000006925', (1828287, 1829620)), (1829550, 1829573), '-'), (('G000006925', (2036226, 2036400)), (2036330, 2036353), '-'), (('G000006925', (2347501, 2359607)), (2359475, 2359498), '+'), (('G000006925', (2690777, 2690953)), (2690825, 2690848), '+'), (('G000006925', (2770832, 2816659)), (2770936, 2770959), '-'), (('G000006925', (2964403, 2964577)), (2964507, 2964530), '-'), (('G000006925', (3079519, 3079697)), (3079569, 3079592), '+'), (('G000006925', (3146034, 3178603)), (3178476, 3178499), '+'), (('G000006925', (3282565, 3282739)), (3282669, 3282692), '-'), (('G000006925', (3513306, 3513480)), (3513352, 3513375), '+'), (('G000006925', (3604733, 3604908)), (3604780, 3604803), '+'), (('G000006925', (3815700, 3815874)), (3815804, 3815827), '-'), (('G000006925', (3831593, 3850590)), (3831697, 3831720), '-'), (('G000006925', (3914710, 3914884)), (3914756, 3914779), '+'), (('G000006925', (3994071, 3994245)), (3994175, 3994198), '-'), (('G000006925', (4117495, 4130655)), (4117599, 4117622), '-'), (('G000006925', (4256395, 4256569)), (4256499, 4256522), '-'), (('G000006925', (4360638, 4360798)), (4360670, 4360693), '+'), (('G000006925', (4695357, 4695531)), (4695461, 4695484), '-'), (('G000006925', (4778619, 4778953)), (4778723, 4778746), '-'), (('G000299455', (2004915, 2005095)), (2005023, 2005046), '-')]\n",
      ">G000006925|235782|235958|235832|235855|+\n",
      "--atagtcttaatactagtttttag\n",
      ">G000006925|266530|266681|266634|266657|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|496041|496215|496145|496168|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|707347|707521|707451|707474|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|865364|865540|865468|865491|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|912667|912841|912771|912794|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|981203|981377|981307|981330|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|1008623|1008797|1008670|1008693|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|1097702|1097876|1097749|1097772|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|1209065|1211067|1209169|1209192|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|1393174|1393348|1393278|1393301|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|1418524|1418698|1418628|1418651|-\n",
      "tgatagtcttaataccagttttt--\n",
      ">G000006925|1500239|1500413|1500286|1500309|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|1618594|1618768|1618641|1618664|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|1696247|1696421|1696351|1696374|-\n",
      "tgatagtcttaataccagttttt--\n",
      ">G000006925|1807541|1813528|1807645|1807668|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|1828287|1829620|1829550|1829573|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|2036226|2036400|2036330|2036353|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|2347501|2359607|2359475|2359498|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|2690777|2690953|2690825|2690848|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|2770832|2816659|2770936|2770959|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|2964403|2964577|2964507|2964530|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3079519|3079697|3079569|3079592|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3146034|3178603|3178476|3178499|+\n",
      "-gatagtcttaatactagttttta-\n",
      ">G000006925|3282565|3282739|3282669|3282692|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3513306|3513480|3513352|3513375|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3604733|3604908|3604780|3604803|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3815700|3815874|3815804|3815827|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3831593|3850590|3831697|3831720|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3914710|3914884|3914756|3914779|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|3994071|3994245|3994175|3994198|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|4117495|4130655|4117599|4117622|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|4256395|4256569|4256499|4256522|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|4360638|4360798|4360670|4360693|+\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|4695357|4695531|4695461|4695484|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000006925|4778619|4778953|4778723|4778746|-\n",
      "tgatagtcttaatactagttttt--\n",
      ">G000299455|2004915|2005095|2005023|2005046|-\n",
      "tgatagtcctgatacaagttttt--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4622488, 4624460)), (4623215, 4623351), '+'), (('G000006925', (4689001, 4689359)), (4689186, 4689322), '-'), (('G000299455', (5313369, 5315826)), (5314452, 5314588), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\n",
      "Reallocating..done. *alloclen = 1056\n",
      "\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4622488|4624460|4623215|4623351|+\n",
      "-tgtaaagtgaggatatggacacctcacttgctcatgagaatgcccgcctgcgggcactg\n",
      "ttgcagacgcaacaggacaccatccgccagatggcgaaatacaaccgcctgctctcacag\n",
      "cgggtggcggcttatgc\n",
      ">G000006925|4689001|4689359|4689186|4689322|-\n",
      "ttgtaaagtgaggaaatggacacctcacttgctcatgagaacgcccgcctgcgggcactg\n",
      "ttgcagacgcaacaggacactatccaccagatggctgaatacaaccgcctgctctcacag\n",
      "cggatggcggcttatg-\n",
      ">G000299455|5313369|5315826|5314452|5314588|+\n",
      "-tgtaaagtgaggatatggacacctcacttgctcatgagaacgcccgcctgcgggcactg\n",
      "ttgcagacgcaacaggacaccatccgccagatggctgaatacaaccgcctgctctcacag\n",
      "cgggtggcggcttatgc\n",
      "\n",
      "\n",
      "7 [(('G000006925', (2560406, 2561162)), (2560514, 2560541), '+'), (('G000006925', (3083675, 3099254)), (3083844, 3083871), '-'), (('G000006925', (4633099, 4633840)), (4633705, 4633732), '-'), (('G000006925', (4668315, 4668899)), (4668765, 4668791), '-'), (('G000006925', (4822840, 4823055)), (4822947, 4822974), '+'), (('G000299455', (857924, 871846)), (871652, 871677), '+'), (('G000299455', (5017408, 5018168)), (5018028, 5018055), '-')]\n",
      ">G000006925|2560406|2561162|2560514|2560541|+\n",
      "-tgcgtagcgtcagattattacgctcaa\n",
      ">G000006925|3083675|3099254|3083844|3083871|-\n",
      "gaacggagtgtcagatggtggtgctca-\n",
      ">G000006925|4633099|4633840|4633705|4633732|-\n",
      "gtgcggagggtccggttgttacgtgca-\n",
      ">G000006925|4668315|4668899|4668765|4668791|-\n",
      "gggcggagggtcaggttgttacgttc--\n",
      ">G000006925|4822840|4823055|4822947|4822974|+\n",
      "-tgcggagggtcaggttgttacgttcaa\n",
      ">G000299455|857924|871846|871652|871677|+\n",
      "---aacgggggcagatggtggtgctcaa\n",
      ">G000299455|5017408|5018168|5018028|5018055|-\n",
      "gttctcagggtcagattgttacgctca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (827107, 836680)), (831473, 831525), '-'), (('G000299455', (3480865, 3520974)), (3499220, 3499271), '+'), (('G000299455', (3520976, 3527694)), (3525335, 3525387), '+')]\n",
      ">G000006925|827107|836680|831473|831525|-\n",
      "agacatggatggtactttcttaagcgatcaaaaaacctataaccgtgagcgg-\n",
      ">G000299455|3480865|3520974|3499220|3499271|+\n",
      "-gacatggatggtactttcttaagcgatcaaaaaacctataaccgtgagcgg-\n",
      ">G000299455|3520976|3527694|3525335|3525387|+\n",
      "-gacatggacggtacttttcttaacgacgccaaaacgtacaaccaaccacgtt\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1965007, 1976679)), (1968187, 1968228), '+'), (('G000006925', (1965007, 1976679)), (1969888, 1969929), '+'), (('G000006925', (4557883, 4573403)), (4559340, 4559381), '-'), (('G000299455', (2122044, 2133716)), (2128794, 2128835), '-'), (('G000299455', (2122044, 2133716)), (2130495, 2130536), '-'), (('G000299455', (4487440, 4499283)), (4497785, 4497826), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1084\n",
      "\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1084\n",
      "\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1965007|1976679|1968187|1968228|+\n",
      "-ctggctgacagcctgtgcaacctgctcaatcccccgccgtt\n",
      ">G000006925|1965007|1976679|1969888|1969929|+\n",
      "-ttccgaaaccgccaatgcgacttgatcgatgccacggctct\n",
      ">G000006925|4557883|4573403|4559340|4559381|-\n",
      "tctcagcaaccgctaagccaacctgatcgataccacggctc-\n",
      ">G000299455|2122044|2133716|2128794|2128835|-\n",
      "tttccgaaaccgccaatgcgacttgatcgatgccacggctc-\n",
      ">G000299455|2122044|2133716|2130495|2130536|-\n",
      "tctggctgacagcctgtgcaacctgctcaatcccccgctgt-\n",
      ">G000299455|4487440|4499283|4497785|4497826|+\n",
      "-ctcagcaaccgctaagccaacctgatcgataccacggctct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1241906, 1253284)), (1250865, 1250898), '+'), (('G000299455', (840174, 840207)), (840174, 840207), '-'), (('G000299455', (2983402, 3007859)), (2998866, 2998899), '-')]\n",
      ">G000006925|1241906|1253284|1250865|1250898|+\n",
      "-tggtgtgggttcaggattaggagtcggttccgg\n",
      ">G000299455|840174|840207|840174|840207|-\n",
      "ctggtgttggctcaggatcaggtgtcggttccg-\n",
      ">G000299455|2983402|3007859|2998866|2998899|-\n",
      "ctggtgtgggttcaggattaggagtcggttccg-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1209065, 1211067)), (1210481, 1210503), '+'), (('G000006925', (2756889, 2766574)), (2757562, 2757584), '+'), (('G000006925', (2923039, 2950048)), (2948677, 2948699), '+'), (('G000006925', (3101814, 3103074)), (3102488, 3102510), '+'), (('G000006925', (3810378, 3811029)), (3810943, 3810965), '-'), (('G000006925', (4744587, 4744895)), (4744841, 4744863), '-'), (('G000006925', (4753880, 4754990)), (4753962, 4753984), '-'), (('G000006925', (4795159, 4795487)), (4795368, 4795390), '+'), (('G000299455', (3226712, 3229721)), (3229131, 3229153), '+')]\n",
      ">G000006925|1209065|1211067|1210481|1210503|+\n",
      "-ggtcagctaccgcgcacacggc\n",
      ">G000006925|2756889|2766574|2757562|2757584|+\n",
      "-ggtcagctaccgcgcacacggc\n",
      ">G000006925|2923039|2950048|2948677|2948699|+\n",
      "-ggtcagctaccgcgcacacggc\n",
      ">G000006925|3101814|3103074|3102488|3102510|+\n",
      "-ggtcagctaccgcgcacacggc\n",
      ">G000006925|3810378|3811029|3810943|3810965|-\n",
      "cggtcagctaccgtgcacatgg-\n",
      ">G000006925|4744587|4744895|4744841|4744863|-\n",
      "cggtcagttaccgcgcacatgg-\n",
      ">G000006925|4753880|4754990|4753962|4753984|-\n",
      "cgatcagttaccgtgcacacgg-\n",
      ">G000006925|4795159|4795487|4795368|4795390|+\n",
      "-ggtcagttaccgcgcacatggc\n",
      ">G000299455|3226712|3229721|3229131|3229153|+\n",
      "-ggtcagctaccgcgcacacggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3032357, 3032385), '-'), (('G000299455', (956352, 970820)), (960664, 960692), '+'), (('G000299455', (3103265, 3103293)), (3103265, 3103293), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3016790|3036697|3032357|3032385|-\n",
      "tgcccgatgcgcaagtttatagggccta-\n",
      ">G000299455|956352|970820|960664|960692|+\n",
      "-gcccgatgcgcaagtttatagggcctat\n",
      ">G000299455|3103265|3103293|3103265|3103293|-\n",
      "tgcccgatgcgcaagtttatcgggtcta-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3040656, 3065083)), (3058322, 3058653), '+'), (('G000299455', (915885, 916237)), (915885, 916216), '+'), (('G000299455', (4713031, 4748324)), (4741563, 4741894), '+'), (('G000299455', (4768323, 4769107)), (4768323, 4768653), '+'), (('G000299455', (5335900, 5336291)), (5335961, 5336291), '-')]\n",
      ">G000006925|3040656|3065083|3058322|3058653|+\n",
      "-actggtgccaggtgaatattccagtggaaacagtattcgtcccagaggaataacaaaag\n",
      "ttggaaacagcgaactgagacgtctgctttacgaagccgcctggtcttatcgtacacctg\n",
      "caaaagttggagcatggcttatatattaccgaccggactctgtaacacaatattccaaag\n",
      "atattgcatggaaagctcaacaacgattgtgttctcgttaccgaactctgacagcaaaag\n",
      "ggaaaaaatcacaagtagccattacggcggtggctcgtgagttgactggatttatgtggg\n",
      "atattgcacttgctgcccaatcatcattcagt\n",
      ">G000299455|915885|916237|915885|916216|+\n",
      "-actggtgccaggtgaatattccagtggaaacagtattcgtcccagaggaataacaaaag\n",
      "ttggaaacagcgaactgagacgtctgctttacgaagccgcctggtcttatcgtacacctg\n",
      "caaaagttggtgcatggcttatatattaccgaccggattctgtaacacaatattccaaag\n",
      "atattgcatggaaagctcaacaacgattgtgttctcgttaccgaagtctgacagcaaaag\n",
      "ggaaaaaatcacaagtagccattacggcggtggctcgtgagttgactggatttatgtggg\n",
      "atattgcacttgctgcccaatcatcattcagt\n",
      ">G000299455|4713031|4748324|4741563|4741894|+\n",
      "-actggtgccaggtgaatattccagtggaaacagtattcgtcccagaggaataacaaaag\n",
      "ttggaaacagcgaactgagacgtctgctttacgaagccgcctggtcttatcgtacacctg\n",
      "caaaagttggtgcatggcttatatattaccgaccggactctgtaacacaatattccaaag\n",
      "atattgcatggaaagctcaacaacgattgtgttctcgttaccgaactctgacagcaaaag\n",
      "ggaaaaaatcacaagtagccattacggcggtggctcgtgagttgactggatttatgtggg\n",
      "atattgcacttgctgcccaatcatcattcagt\n",
      ">G000299455|4768323|4769107|4768323|4768653|+\n",
      "-actggtgccaggtgaatattccagtggaaacagtattcgtcccagaggaataacaaaag\n",
      "ttggaaacagcgaactgagacgtctgctttacgaagccgcctggtcttatcgtacacctg\n",
      "caaaagttggggcatggcttatatattaccgaccggactctgtaacacaatattccaaag\n",
      "atattgcatggaaagctcaacaacgattgtgttctcgttaccgaagtctgacagcaaaag\n",
      "gg-aaaaatcacaagtagccattacggcggtggctcgtgagttgactggatttatgtggg\n",
      "atattgcacttgctgcccaatcatcattcagt\n",
      ">G000299455|5335900|5336291|5335961|5336291|-\n",
      "gactggtgccaggtgaatattccagtggaaacagtattcgtcccagaggaataacaaaag\n",
      "ttggaaacagcgaactgagacgtctgctttacgaagccgcctggtcttatcgtacacctg\n",
      "caaaagttggggcatggcttatatattaccgaccggactctgtaacacaatattccaaag\n",
      "atattgcatggaaagctcaacaacgattgtgttctcgttaccgaagtctgacagcaaaag\n",
      "gg-aaaaatcacaagtagccattacggcggtggctcgtgagttgactggatttatgtggg\n",
      "atattgcacttgctgcccaatcatcattcag-\n",
      "\n",
      "\n",
      "8 [(('G000006925', (711810, 711981)), (711934, 711981), '+'), (('G000006925', (720848, 721019)), (720972, 721019), '+'), (('G000006925', (1509064, 1578296)), (1578094, 1578142), '-'), (('G000006925', (1937045, 1937215)), (1937045, 1937092), '-'), (('G000006925', (2061077, 2061268)), (2061077, 2061124), '-'), (('G000299455', (2167070, 2167372)), (2167222, 2167269), '+'), (('G000299455', (2923865, 2925880)), (2925682, 2925729), '-'), (('G000299455', (3293159, 3293449)), (3293257, 3293304), '-')]\n",
      ">G000006925|711810|711981|711934|711981|+\n",
      "--accttctaagttgacggtgcgaggttcgagtcctcggtggtgggcca\n",
      ">G000006925|720848|721019|720972|721019|+\n",
      "--accttctaagttgacggtgcgaggttcgagtcctcggtggtgggcca\n",
      ">G000006925|1509064|1578296|1578094|1578142|-\n",
      "acgataacagggttgttggtgcggggggcgggtccccgatggcggtcc-\n",
      ">G000006925|1937045|1937215|1937045|1937092|-\n",
      "-aaccttctaagttgacggtccgaggttcgagtcctcggtggtgggcc-\n",
      ">G000006925|2061077|2061268|2061077|2061124|-\n",
      "-aaccctttaagttgagtgtgcgaggttcgagtccccggtggcggtcc-\n",
      ">G000299455|2167070|2167372|2167222|2167269|+\n",
      "--gccttcgaagctggttgcgcggggttcgagtcctcgatggcggtcca\n",
      ">G000299455|2923865|2925880|2925682|2925729|-\n",
      "-agccttcgaagctggctgcgcggggttcaagtccccgatggcggtcc-\n",
      ">G000299455|3293159|3293449|3293257|3293304|-\n",
      "-agccttcgaagctggctgcgcggggttcgagtcctcgatggcggtcc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (729116, 729269), '+'), (('G000299455', (2174641, 2181396)), (2177513, 2177665), '+'), (('G000299455', (2910238, 2916987)), (2913963, 2914115), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 109\r",
      "  101 / 109\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 109\r",
      "   10 / 109\r",
      "   20 / 109\r",
      "   30 / 109\r",
      "   40 / 109\r",
      "   50 / 109\r",
      "   60 / 109\r",
      "   70 / 109\r",
      "   80 / 109\r",
      "   90 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 108  f\b\b\r",
      "STEP     2 / 108  f\b\b\r",
      "STEP     3 / 108  f\b\b\r",
      "STEP     4 / 108  f\b\b\r",
      "STEP     5 / 108  f\b\b\r",
      "STEP     6 / 108  f\b\b\r",
      "STEP     7 / 108  f\b\b\r",
      "STEP     8 / 108  f\b\b\r",
      "STEP     9 / 108  f\b\b\r",
      "STEP    10 / 108  f\b\b\r",
      "STEP    11 / 108  f\b\b\r",
      "STEP    12 / 108  f\b\b\r",
      "STEP    13 / 108  f\b\b\r",
      "STEP    14 / 108  f\b\b\r",
      "STEP    15 / 108  f\b\b\r",
      "STEP    16 / 108  f\b\b\r",
      "STEP    17 / 108  f\b\b\r",
      "STEP    18 / 108  f\b\b\r",
      "STEP    19 / 108  f\b\b\r",
      "STEP    20 / 108  f\b\b\r",
      "STEP    21 / 108  f\b\b\r",
      "STEP    22 / 108  f\b\b\r",
      "STEP    23 / 108  f\b\b\r",
      "STEP    24 / 108  f\b\b\r",
      "STEP    25 / 108  f\b\b\r",
      "STEP    26 / 108  f\b\b\r",
      "STEP    27 / 108  f\b\b\r",
      "STEP    28 / 108  f\b\b\r",
      "STEP    29 / 108  f\b\b\r",
      "STEP    30 / 108  f\b\b\r",
      "STEP    31 / 108  f\b\b\r",
      "STEP    32 / 108  f\b\b\r",
      "STEP    33 / 108  f\b\b\r",
      "STEP    34 / 108  f\b\b\r",
      "STEP    35 / 108  f\b\b\r",
      "STEP    36 / 108  f\b\b\r",
      "STEP    37 / 108  f\b\b\r",
      "STEP    38 / 108  f\b\b\r",
      "STEP    39 / 108  f\b\b\r",
      "STEP    40 / 108  f\b\b\r",
      "STEP    41 / 108  f\b\b\r",
      "STEP    42 / 108  f\b\b\r",
      "STEP    43 / 108  f\b\b\r",
      "STEP    44 / 108  f\b\b\r",
      "STEP    45 / 108  f\b\b\r",
      "STEP    46 / 108  f\b\b\r",
      "STEP    47 / 108  f\b\b\r",
      "STEP    48 / 108  f\b\b\r",
      "STEP    49 / 108  f\b\b\r",
      "STEP    50 / 108  f\b\b\r",
      "STEP    51 / 108  f\b\b\r",
      "STEP    52 / 108  f\b\b\r",
      "STEP    53 / 108  f\b\b\r",
      "STEP    54 / 108  f\b\b\r",
      "STEP    55 / 108  f\b\b\r",
      "STEP    56 / 108  f\b\b\r",
      "STEP    57 / 108  f\b\b\r",
      "STEP    58 / 108  f\b\b\r",
      "STEP    59 / 108  f\b\b\r",
      "STEP    60 / 108  f\b\b\r",
      "STEP    61 / 108  f\b\b\r",
      "STEP    62 / 108  f\b\b\r",
      "STEP    63 / 108  f\b\b\r",
      "STEP    64 / 108  f\b\b\r",
      "STEP    65 / 108  f\b\b\r",
      "STEP    66 / 108  f\b\b\r",
      "STEP    67 / 108  f\b\b\r",
      "STEP    68 / 108  f\b\b\r",
      "STEP    69 / 108  f\b\b\r",
      "STEP    70 / 108  f\b\b\r",
      "STEP    71 / 108  f\b\b\r",
      "STEP    72 / 108  f\b\b\r",
      "STEP    73 / 108  f\b\b\r",
      "STEP    74 / 108  f\b\b\r",
      "STEP    75 / 108  f\b\b\r",
      "STEP    76 / 108  f\b\b\r",
      "STEP    77 / 108  f\b\b\r",
      "STEP    78 / 108  f\b\b\r",
      "STEP    79 / 108  f\b\b\r",
      "STEP    80 / 108  f\b\b\r",
      "STEP    81 / 108  f\b\b\r",
      "STEP    82 / 108  f\b\b\r",
      "STEP    83 / 108  f\b\b\r",
      "STEP    84 / 108  f\b\b\r",
      "STEP    85 / 108  f\b\b\r",
      "STEP    86 / 108  f\b\b\r",
      "STEP    87 / 108  f\b\b\r",
      "STEP    88 / 108  f\b\b\r",
      "STEP    89 / 108  f\b\b\r",
      "STEP    90 / 108  f\b\b\r",
      "STEP    91 / 108  f\b\b\r",
      "STEP    92 / 108  f\b\b\r",
      "STEP    93 / 108  f\b\b\r",
      "STEP    94 / 108  f\b\b\r",
      "STEP    95 / 108  f\b\b\r",
      "STEP    96 / 108  f\b\b\r",
      "STEP    97 / 108  f\b\b\r",
      "STEP    98 / 108  f\b\b\r",
      "STEP    99 / 108  f\b\b\r",
      "STEP   100 / 108  f\b\b\r",
      "STEP   101 / 108  f\b\b\r",
      "STEP   102 / 108  f\b\b\r",
      "STEP   103 / 108  f\b\b\r",
      "STEP   104 / 108  d\b\b\n",
      "Reallocating..done. *alloclen = 1118\n",
      "\r",
      "STEP   105 / 108  d\b\b h\r",
      "STEP   106 / 108  d\b\b h\r",
      "STEP   107 / 108  d\b\b h\r",
      "STEP   108 / 108  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 109\r",
      "   10 / 109\r",
      "   20 / 109\r",
      "   30 / 109\r",
      "   40 / 109\r",
      "   50 / 109\r",
      "   60 / 109\r",
      "   70 / 109\r",
      "   80 / 109\r",
      "   90 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 108  f\b\b\r",
      "STEP     2 / 108  f\b\b\r",
      "STEP     3 / 108  f\b\b\r",
      "STEP     4 / 108  f\b\b\r",
      "STEP     5 / 108  f\b\b\r",
      "STEP     6 / 108  f\b\b\r",
      "STEP     7 / 108  f\b\b\r",
      "STEP     8 / 108  f\b\b\r",
      "STEP     9 / 108  f\b\b\r",
      "STEP    10 / 108  f\b\b\r",
      "STEP    11 / 108  f\b\b\r",
      "STEP    12 / 108  f\b\b\r",
      "STEP    13 / 108  f\b\b\r",
      "STEP    14 / 108  f\b\b\r",
      "STEP    15 / 108  f\b\b\r",
      "STEP    16 / 108  f\b\b\r",
      "STEP    17 / 108  f\b\b\r",
      "STEP    18 / 108  f\b\b\r",
      "STEP    19 / 108  f\b\b\r",
      "STEP    20 / 108  f\b\b\r",
      "STEP    21 / 108  f\b\b\r",
      "STEP    22 / 108  f\b\b\r",
      "STEP    23 / 108  f\b\b\r",
      "STEP    24 / 108  f\b\b\r",
      "STEP    25 / 108  f\b\b\r",
      "STEP    26 / 108  f\b\b\r",
      "STEP    27 / 108  f\b\b\r",
      "STEP    28 / 108  f\b\b\r",
      "STEP    29 / 108  f\b\b\r",
      "STEP    30 / 108  f\b\b\r",
      "STEP    31 / 108  f\b\b\r",
      "STEP    32 / 108  f\b\b\r",
      "STEP    33 / 108  f\b\b\r",
      "STEP    34 / 108  f\b\b\r",
      "STEP    35 / 108  f\b\b\r",
      "STEP    36 / 108  f\b\b\r",
      "STEP    37 / 108  f\b\b\r",
      "STEP    38 / 108  f\b\b\r",
      "STEP    39 / 108  f\b\b\r",
      "STEP    40 / 108  f\b\b\r",
      "STEP    41 / 108  f\b\b\r",
      "STEP    42 / 108  f\b\b\r",
      "STEP    43 / 108  f\b\b\r",
      "STEP    44 / 108  f\b\b\r",
      "STEP    45 / 108  f\b\b\r",
      "STEP    46 / 108  f\b\b\r",
      "STEP    47 / 108  f\b\b\r",
      "STEP    48 / 108  f\b\b\r",
      "STEP    49 / 108  f\b\b\r",
      "STEP    50 / 108  f\b\b\r",
      "STEP    51 / 108  f\b\b\r",
      "STEP    52 / 108  f\b\b\r",
      "STEP    53 / 108  f\b\b\r",
      "STEP    54 / 108  f\b\b\r",
      "STEP    55 / 108  f\b\b\r",
      "STEP    56 / 108  f\b\b\r",
      "STEP    57 / 108  f\b\b\r",
      "STEP    58 / 108  f\b\b\r",
      "STEP    59 / 108  f\b\b\r",
      "STEP    60 / 108  f\b\b\r",
      "STEP    61 / 108  f\b\b\r",
      "STEP    62 / 108  f\b\b\r",
      "STEP    63 / 108  f\b\b\r",
      "STEP    64 / 108  f\b\b\r",
      "STEP    65 / 108  f\b\b\r",
      "STEP    66 / 108  f\b\b\r",
      "STEP    67 / 108  f\b\b\r",
      "STEP    68 / 108  f\b\b\r",
      "STEP    69 / 108  f\b\b\r",
      "STEP    70 / 108  f\b\b\r",
      "STEP    71 / 108  f\b\b\r",
      "STEP    72 / 108  f\b\b\r",
      "STEP    73 / 108  f\b\b\r",
      "STEP    74 / 108  f\b\b\r",
      "STEP    75 / 108  f\b\b\r",
      "STEP    76 / 108  f\b\b\r",
      "STEP    77 / 108  f\b\b\r",
      "STEP    78 / 108  f\b\b\r",
      "STEP    79 / 108  f\b\b\r",
      "STEP    80 / 108  f\b\b\r",
      "STEP    81 / 108  f\b\b\r",
      "STEP    82 / 108  f\b\b\r",
      "STEP    83 / 108  f\b\b\r",
      "STEP    84 / 108  f\b\b\r",
      "STEP    85 / 108  f\b\b\r",
      "STEP    86 / 108  f\b\b\r",
      "STEP    87 / 108  f\b\b\r",
      "STEP    88 / 108  f\b\b\r",
      "STEP    89 / 108  f\b\b\r",
      "STEP    90 / 108  f\b\b\r",
      "STEP    91 / 108  f\b\b\r",
      "STEP    92 / 108  f\b\b\r",
      "STEP    93 / 108  f\b\b\r",
      "STEP    94 / 108  f\b\b\r",
      "STEP    95 / 108  f\b\b\r",
      "STEP    96 / 108  f\b\b\r",
      "STEP    97 / 108  f\b\b\r",
      "STEP    98 / 108  f\b\b\r",
      "STEP    99 / 108  f\b\b\r",
      "STEP   100 / 108  f\b\b\r",
      "STEP   101 / 108  f\b\b\r",
      "STEP   102 / 108  f\b\b\r",
      "STEP   103 / 108  f\b\b\r",
      "STEP   104 / 108  f\b\b\r",
      "STEP   105 / 108  f\b\b\r",
      "STEP   106 / 108  f\b\b\r",
      "STEP   107 / 108  f\b\b\r",
      "STEP   108 / 108  d\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|729116|729269|+\n",
      "-cgaaaaggcatttgaatatatcgcgggggcaccgcaaggtcagcaggataatccgctta\n",
      "ttaatattctggagaagttcacccgctggtatgacacgaataacgtgacgctgggcggtg\n",
      "ttaaaattccgcaccttttttcctggcgatgcgc\n",
      ">G000299455|2174641|2181396|2177513|2177665|+\n",
      "-cgaaaaggcctttgaatatatcgccggtgcgccgcaggggcagaaggataatccgctta\n",
      "ttaatattctggataagttctccacctggtatgacacgaatagcgtgacgctgggcggtg\n",
      "tcaaaattccgcacc-ttttccccggtgatgatc\n",
      ">G000299455|2910238|2916987|2913963|2914115|-\n",
      "ccgaaaaggcctttgaatatatcgccggtgcgccgcaggggcagaaggataatccgctta\n",
      "ttaatattctggataagttctccacctggtatgacacgaatagcgtgacgctgggcggtg\n",
      "tcaaaattccgcacc-ttttccccggtgatgat-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2560406, 2561162)), (2560746, 2560922), '+'), (('G000006925', (4629818, 4630996)), (4630054, 4630231), '-'), (('G000006925', (4668315, 4668899)), (4668384, 4668560), '-'), (('G000006925', (4779995, 4780327)), (4780080, 4780257), '+'), (('G000299455', (5017408, 5018168)), (5017648, 5017824), '-')]\n",
      ">G000006925|2560406|2561162|2560746|2560922|+\n",
      "--ttgccgg-gctttactgccaacgtagctccattgctcatcaagctcgcagataaacgc\n",
      "cacatcagcatgggcaacaggcgaagaagttattcgctttggcgcgagttttttaaagtc\n",
      "cggatgacggtgttactaccaattttcagtgtcctggcggtatcgcgaaccccggcacc\n",
      ">G000006925|4629818|4630996|4630054|4630231|-\n",
      "tgttgttgc-gccttgctgccaacgaagctccactgcccatcaagttcacagataagggc\n",
      "aacatcagcgtgagcgactggcgcagatgttatccgccggggcgggagctttttaatgtc\n",
      "cgtatgacggtattgatgccgattttcagcgtacggccagtatcgcgaactccggcac-\n",
      ">G000006925|4668315|4668899|4668384|4668560|-\n",
      "-gttgttgc-gccttgctgccaacgaagcgccactgctcatcaagttcacagataagggc\n",
      "aacatcagcgtgagcgactggcgcagatgttatccgccggggcgagagctttttaatgtc\n",
      "cgtatgacggtattgatgccgattttcagtgtcctggcggtatcacggccccctgcgc-\n",
      ">G000006925|4779995|4780327|4780080|4780257|+\n",
      "-gttgttgcagccttgctgccaacgaagcgccactgctcatcaagttcacagataagggc\n",
      "aacatcagcgtgagcgactggcgcagatgttatccgccggggcgggagctttttaatgtc\n",
      "cgtatgacggtattgatgccgattttcagtgtcctggcggtatcacggccccctgcgc-\n",
      ">G000299455|5017408|5018168|5017648|5017824|-\n",
      "-gctgtcgc-gctttactgccgacaaagctccattgctgactaagtttgcagatgagtgc\n",
      "cacatcagcatgagcgaccggggatgacgttattcttcgtggcgagagttttttaaagtt\n",
      "cggatgactgtattaatgccgattttcagcgtacgggcagtatcgcgaaccccggcac-\n",
      "\n",
      "\n",
      "109 [(('G000006925', (1, 24671)), (18698, 18756), '+'), (('G000006925', (64491, 216702)), (87621, 87679), '-'), (('G000006925', (64491, 216702)), (148811, 148869), '-'), (('G000006925', (266682, 267450)), (267362, 267420), '-'), (('G000006925', (273344, 278315)), (273374, 273432), '+'), (('G000006925', (278320, 279091)), (279002, 279060), '-'), (('G000006925', (300111, 308544)), (308455, 308513), '-'), (('G000006925', (378637, 384812)), (379319, 379377), '-'), (('G000006925', (459861, 468373)), (465747, 465805), '+'), (('G000006925', (468374, 487224)), (478743, 478801), '+'), (('G000006925', (520755, 528026)), (521435, 521493), '-'), (('G000006925', (545486, 558918)), (558180, 558238), '+'), (('G000006925', (622915, 623662)), (622945, 623003), '+'), (('G000006925', (865887, 867389)), (866651, 866709), '+'), (('G000006925', (984595, 985335)), (985277, 985335), '-'), (('G000006925', (985336, 1008622)), (985460, 985518), '+'), (('G000006925', (1025530, 1063042)), (1049602, 1049660), '+'), (('G000006925', (1189492, 1208094)), (1197678, 1197736), '+'), (('G000006925', (1239481, 1240251)), (1239511, 1239569), '+'), (('G000006925', (1290847, 1306886)), (1295103, 1295161), '+'), (('G000006925', (1321493, 1385605)), (1366134, 1366192), '-'), (('G000006925', (1398496, 1399271)), (1398528, 1398586), '+'), (('G000006925', (1409877, 1410649)), (1410561, 1410619), '-'), (('G000006925', (1424264, 1425034)), (1424946, 1425004), '-'), (('G000006925', (1425133, 1465118)), (1444310, 1444368), '+'), (('G000006925', (1509064, 1578296)), (1526709, 1526767), '+'), (('G000006925', (1509064, 1578296)), (1533646, 1533704), '+'), (('G000006925', (1509064, 1578296)), (1542796, 1542854), '+'), (('G000006925', (1509064, 1578296)), (1568871, 1568929), '-'), (('G000006925', (1509064, 1578296)), (1574829, 1574887), '-'), (('G000006925', (1670306, 1692658)), (1673402, 1673460), '+'), (('G000006925', (1729332, 1741706)), (1736654, 1736712), '-'), (('G000006925', (1758378, 1759147)), (1758408, 1758466), '+'), (('G000006925', (1759148, 1776249)), (1766558, 1766616), '+'), (('G000006925', (1759148, 1776249)), (1769712, 1769770), '-'), (('G000006925', (1776250, 1792578)), (1780943, 1781001), '+'), (('G000006925', (1776250, 1792578)), (1791233, 1791291), '+'), (('G000006925', (1793868, 1802424)), (1795809, 1795867), '-'), (('G000006925', (1842508, 1869155)), (1850729, 1850787), '-'), (('G000006925', (1842508, 1869155)), (1856667, 1856725), '-'), (('G000006925', (1940277, 1941051)), (1940307, 1940365), '+'), (('G000006925', (1976680, 1977448)), (1977360, 1977418), '-'), (('G000006925', (2036705, 2044875)), (2037835, 2037893), '-'), (('G000006925', (2050169, 2051841)), (2051753, 2051811), '-'), (('G000006925', (2088344, 2103602)), (2095070, 2095128), '+'), (('G000006925', (2120115, 2138771)), (2135328, 2135386), '-'), (('G000006925', (2172359, 2184266)), (2177822, 2177880), '+'), (('G000006925', (2187913, 2198767)), (2197567, 2197625), '-'), (('G000006925', (2198768, 2199537)), (2198799, 2198857), '+'), (('G000006925', (2199538, 2210687)), (2206231, 2206289), '-'), (('G000006925', (2211287, 2225774)), (2220673, 2220721), '+'), (('G000006925', (2235234, 2236005)), (2235914, 2235972), '-'), (('G000006925', (2242494, 2346249)), (2287475, 2287533), '+'), (('G000006925', (2374481, 2472008)), (2396315, 2396373), '-'), (('G000006925', (2756085, 2756853)), (2756765, 2756823), '-'), (('G000006925', (2816684, 2833035)), (2825231, 2825289), '+'), (('G000006925', (2849870, 2870340)), (2863520, 2863578), '-'), (('G000006925', (2964578, 2976271)), (2964709, 2964767), '-'), (('G000006925', (2976303, 3015537)), (3014663, 3014721), '+'), (('G000006925', (3016790, 3036697)), (3021494, 3021552), '+'), (('G000006925', (3038529, 3039299)), (3038559, 3038617), '+'), (('G000006925', (3104758, 3106299)), (3104789, 3104847), '+'), (('G000006925', (3179574, 3180459)), (3180370, 3180428), '-'), (('G000006925', (3180744, 3203996)), (3182261, 3182319), '-'), (('G000006925', (3237769, 3244530)), (3238449, 3238507), '-'), (('G000006925', (3268417, 3281594)), (3274451, 3274509), '-'), (('G000006925', (3322761, 3361739)), (3361648, 3361706), '-'), (('G000006925', (3361966, 3403725)), (3362648, 3362706), '-'), (('G000006925', (3608503, 3611864)), (3609188, 3609246), '-'), (('G000006925', (3612083, 3637192)), (3624263, 3624321), '+'), (('G000006925', (3637330, 3640023)), (3637360, 3637418), '+'), (('G000006925', (3658114, 3676993)), (3665190, 3665248), '+'), (('G000006925', (3693106, 3702974)), (3693788, 3693846), '-'), (('G000006925', (3703179, 3717854)), (3703860, 3703918), '-'), (('G000006925', (3717879, 3736288)), (3726144, 3726202), '-'), (('G000006925', (3736321, 3788106)), (3743052, 3743110), '+'), (('G000006925', (3815875, 3829465)), (3818035, 3818093), '+'), (('G000006925', (3831593, 3850590)), (3834516, 3834574), '+'), (('G000006925', (3858569, 3864367)), (3864279, 3864337), '-'), (('G000006925', (3939882, 3952896)), (3943514, 3943572), '-'), (('G000006925', (4002899, 4012437)), (4003579, 4003637), '-'), (('G000006925', (4069200, 4076620)), (4076534, 4076592), '-'), (('G000006925', (4091607, 4092380)), (4091638, 4091696), '+'), (('G000006925', (4094093, 4116338)), (4102344, 4102402), '+'), (('G000006925', (4174522, 4188276)), (4177243, 4177301), '-'), (('G000006925', (4256570, 4288375)), (4285009, 4285067), '+'), (('G000006925', (4288513, 4315006)), (4314917, 4314975), '-'), (('G000006925', (4362175, 4367702)), (4366964, 4367022), '+'), (('G000006925', (4378758, 4379529)), (4378788, 4378846), '+'), (('G000006925', (4389053, 4390969)), (4389084, 4389142), '+'), (('G000006925', (4392644, 4394283)), (4393329, 4393387), '-'), (('G000006925', (4397536, 4410956)), (4410868, 4410926), '-'), (('G000006925', (4462252, 4478752)), (4462283, 4462341), '+'), (('G000006925', (4478753, 4490457)), (4488421, 4488479), '+'), (('G000006925', (4497401, 4528664)), (4515550, 4515608), '-'), (('G000006925', (4535976, 4543278)), (4541190, 4541248), '-'), (('G000006925', (4557883, 4573403)), (4570403, 4570461), '-'), (('G000006925', (4645852, 4646621)), (4646532, 4646590), '-'), (('G000006925', (4785482, 4786253)), (4785514, 4785572), '+'), (('G000299455', (91523, 102755)), (92208, 92266), '-'), (('G000299455', (696017, 696792)), (696049, 696107), '+'), (('G000299455', (941725, 942502)), (941758, 941816), '+'), (('G000299455', (1581872, 1582655)), (1582557, 1582615), '-'), (('G000299455', (2052398, 2064114)), (2062637, 2062695), '-'), (('G000299455', (3182222, 3183821)), (3183079, 3183137), '+'), (('G000299455', (3672841, 3680073)), (3679979, 3680037), '-'), (('G000299455', (4588613, 4589520)), (4588775, 4588833), '+'), (('G000299455', (4592900, 4593681)), (4593587, 4593645), '-'), (('G000299455', (5323356, 5324120)), (5324028, 5324086), '-')]\n",
      ">G000006925|1|24671|18698|18756|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|64491|216702|87621|87679|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|64491|216702|148811|148869|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|266682|267450|267362|267420|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|273344|278315|273374|273432|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|278320|279091|279002|279060|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|300111|308544|308455|308513|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|378637|384812|379319|379377|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|459861|468373|465747|465805|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|468374|487224|478743|478801|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|520755|528026|521435|521493|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|545486|558918|558180|558238|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|622915|623662|622945|623003|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|865887|867389|866651|866709|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|984595|985335|985277|985335|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|985336|1008622|985460|985518|+\n",
      "-tatgttcagataatgcccgatggccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1025530|1063042|1049602|1049660|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1189492|1208094|1197678|1197736|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1239481|1240251|1239511|1239569|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1290847|1306886|1295103|1295161|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1321493|1385605|1366134|1366192|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1398496|1399271|1398528|1398586|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1409877|1410649|1410561|1410619|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1424264|1425034|1424946|1425004|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1425133|1465118|1444310|1444368|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1509064|1578296|1526709|1526767|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1509064|1578296|1533646|1533704|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1509064|1578296|1542796|1542854|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1509064|1578296|1568871|1568929|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1509064|1578296|1574829|1574887|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1670306|1692658|1673402|1673460|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1729332|1741706|1736654|1736712|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1758378|1759147|1758408|1758466|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1759148|1776249|1766558|1766616|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1759148|1776249|1769712|1769770|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1776250|1792578|1780943|1781001|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1776250|1792578|1791233|1791291|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1793868|1802424|1795809|1795867|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1842508|1869155|1850729|1850787|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1842508|1869155|1856667|1856725|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|1940277|1941051|1940307|1940365|+\n",
      "-tatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|1976680|1977448|1977360|1977418|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2036705|2044875|2037835|2037893|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2050169|2051841|2051753|2051811|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2088344|2103602|2095070|2095128|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|2120115|2138771|2135328|2135386|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2172359|2184266|2177822|2177880|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|2187913|2198767|2197567|2197625|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2198768|2199537|2198799|2198857|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|2199538|2210687|2206231|2206289|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2211287|2225774|2220673|2220721|+\n",
      "-tatgttcagataatgcccga----------atgcagctccaccgattttgagaacgac\n",
      ">G000006925|2235234|2236005|2235914|2235972|-\n",
      "ttatgttcagataatgcccgatggccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2242494|2346249|2287475|2287533|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|2374481|2472008|2396315|2396373|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2756085|2756853|2756765|2756823|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2816684|2833035|2825231|2825289|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|2849870|2870340|2863520|2863578|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2964578|2976271|2964709|2964767|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|2976303|3015537|3014663|3014721|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3016790|3036697|3021494|3021552|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3038529|3039299|3038559|3038617|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3104758|3106299|3104789|3104847|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3179574|3180459|3180370|3180428|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3180744|3203996|3182261|3182319|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcaactccaccgattttgagaacga-\n",
      ">G000006925|3237769|3244530|3238449|3238507|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3268417|3281594|3274451|3274509|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3322761|3361739|3361648|3361706|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3361966|3403725|3362648|3362706|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3608503|3611864|3609188|3609246|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3612083|3637192|3624263|3624321|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3637330|3640023|3637360|3637418|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3658114|3676993|3665190|3665248|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3693106|3702974|3693788|3693846|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3703179|3717854|3703860|3703918|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3717879|3736288|3726144|3726202|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3736321|3788106|3743052|3743110|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3815875|3829465|3818035|3818093|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3831593|3850590|3834516|3834574|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|3858569|3864367|3864279|3864337|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|3939882|3952896|3943514|3943572|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4002899|4012437|4003579|4003637|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4069200|4076620|4076534|4076592|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4091607|4092380|4091638|4091696|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4094093|4116338|4102344|4102402|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4174522|4188276|4177243|4177301|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4256570|4288375|4285009|4285067|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4288513|4315006|4314917|4314975|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4362175|4367702|4366964|4367022|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4378758|4379529|4378788|4378846|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4389053|4390969|4389084|4389142|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4392644|4394283|4393329|4393387|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4397536|4410956|4410868|4410926|-\n",
      "ttatgttcagataatgcccgatggccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4462252|4478752|4462283|4462341|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4478753|4490457|4488421|4488479|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000006925|4497401|4528664|4515550|4515608|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4535976|4543278|4541190|4541248|-\n",
      "ttatgttcagataatgcccgatggccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4557883|4573403|4570403|4570461|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4645852|4646621|4646532|4646590|-\n",
      "ttatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000006925|4785482|4786253|4785514|4785572|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000299455|91523|102755|92208|92266|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000299455|696017|696792|696049|696107|+\n",
      "-tatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000299455|941725|942502|941758|941816|+\n",
      "-tatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000299455|1581872|1582655|1582557|1582615|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000299455|2052398|2064114|2062637|2062695|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000299455|3182222|3183821|3183079|3183137|+\n",
      "-tatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacgac\n",
      ">G000299455|3672841|3680073|3679979|3680037|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000299455|4588613|4589520|4588775|4588833|+\n",
      "-tatgttcagataatgcccgatgaccttgtcatgcagctccaccgattttgagaacggc\n",
      ">G000299455|4592900|4593681|4593587|4593645|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      ">G000299455|5323356|5324120|5324028|5324086|-\n",
      "ttatgttcagataatgcccgatgactttgtcatgcagctccaccgattttgagaacga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (489893, 490877), '-'), (('G000299455', (316970, 317954)), (316970, 317954), '-'), (('G000299455', (3819953, 3834467)), (3830841, 3831825), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|487251|491968|489893|490877|-\n",
      "ccaaccgccccctaaggcggtatacaaataaatttcgttaacctgacgggcataattcag\n",
      "gtcaagtaaagtttgtcgggttgcaaataaagaacgctcggcatccagcacttccagata\n",
      "acttaccgcgccgtgctgatataatgcccgcgcccgttgcagggtaatttgcagcgacgc\n",
      "cagataacgctgctgggcgctgatttgatcgtccaggctttgacgtaatgccagcgcatc\n",
      "tgccacctctttaaaggcgttctggattttctgttcataattcaccaccgactgctgctg\n",
      "gcgaatttcggcgatatccagattggcctggttgcgtccggcattaaaaatggggatctc\n",
      "aattttaggaataaaattccacatcccgctgctagcgttaaataatgacgacagatcgct\n",
      "gctggcggttgatattccgctggtcaggctgatagacgggaaaaatgccgcacgcgcagc\n",
      "accaatattagcattagacgccattaacgcgtgttcagcttccataatatccgggcgctg\n",
      "caataagatttgcgacgacaagcccgccggtaatttaacgctttgcaggctgccgctgtt\n",
      "tactgtctgcgcttgcggtagcttgccgtagcttcccaataacagttgcaatgcattatt\n",
      "cgcctgcgccagttccccctgacgtttagcgatgtcactgcgggtactttctatcacccc\n",
      "gcgagcctgttccagcgccagaacattgctgctaccggtcaacagttgtttttcgacaaa\n",
      "cgcatatgactgctgataattacgcagcgtttcttcggctatttgcaattgcgcatacgc\n",
      "cagctgctgattgaaatagctttgcgcgacattagaaaccagcaaaatatgcaccgcgcg\n",
      "ctgggcttcctcagtggctaaataattttggcgctcggcttcgctcatgttctttaagcg\n",
      "accgaaaaaatcgagatcaaagct-\n",
      ">G000299455|316970|317954|316970|317954|-\n",
      "ccaaccgccacccagcgcggtaaacagattaatttcgttaacctgccgggaataggtaag\n",
      "atcgagaatggtttgctgggtagcgaagagggaacgttctgcatccagcacttcgatgta\n",
      "actgacagcaccacttgcatacaatcctctggcacgctggagagttatctgaagtgaatc\n",
      "aagataacgctgctgtgactcaagttgctggctaaggctgtcgcgcagcgcaagcgtgtc\n",
      "ggaaacatccttaaaggctgactgaattttttgttcgtaattaaccaccgactgttgctg\n",
      "gcgaatttcagccagcttcagattggctttattcctgccagcattaaatataggaatttc\n",
      "aattttagggataaaattccacattccacttcctgacgtaaacaggctggacagctccgt\n",
      "actgcttgcggaaagaccactggtcagggtaattgaggggaaaaaggccgctcgcgctgc\n",
      "gccaatattggcatcagccgctttcagctgatattccgcttccataatatccggtcgctg\n",
      "cagcaaaatttgtgaagacagatttggtggcaattttactggggctatctccccgccttt\n",
      "catccctttttctgacggaagtgcgcggtacgttcccagcaccagttgcagggcattgtt\n",
      "tgcctgagccagatcgccttctcgtttggctatttcggcgcgggtactttcgatttgtcc\n",
      "tcgcgcctgttcaagtgccagaacgttcgtactcccggtcacgagctgctgctcaacgaa\n",
      "agcataggactgttgataatttttcagcgtttcccgcgcaatacgaagttgttcgtacgc\n",
      "cagttgctggctgaaatagctttgtgaaacgttggataccagcaggatgtgtacggcccg\n",
      "acgggcttcttcgctggcaaagtagttctggcgatcagcatcactcatgttcttaagttt\n",
      "gccgaaaaaatcgagctcatagct-\n",
      ">G000299455|3819953|3834467|3830841|3831825|+\n",
      "-caaccgccaccaagtgcggtatataaagaaatttcgttaacctgacgggcataattcag\n",
      "gtcgagtaaagtttgtcgggttgcaaataaagaacgctcggcatccagcacttccagata\n",
      "acttactgcgccgtgctgatataatgcccgcgcccgttgcaaagtaatttgcagcgacgc\n",
      "cagataacgctgctgggcgctgatttgatcgttcaggctttgacgtaatgccagcgcatc\n",
      "tgccacttctttaaaggcgttctggattttctgttcataattcactaccgactgctgctg\n",
      "gcgaatttcggcgatatccagattggcctggttgcgtccggcattaaaaatggggatctc\n",
      "aattttaggaataaaattccacatcccgctgctggcgttaaataatgacgacagatcgct\n",
      "gctggcggttgatattccgctggtcaggcttatagatgggaaaaatgccgcgcgtgcggc\n",
      "cccaatattggcattagccgccattaacgcgtgttcagcttccataatatccgggcgctg\n",
      "caataagatttgcgacgacaagcccgccggtaatttaacgctttgcaggctgtcgctgtt\n",
      "tactgtctgcgcttgcggcagcttgccgtagcttcccaataacagttgcaatgcattatt\n",
      "cgcctgcgccagttccccctgacgtttagcgatgtcactgcgggtactttctatcacccc\n",
      "gcgagcctgttccagcgccagaacattgctgctacctgtcaacagttgtttttcgacaaa\n",
      "cgcatatgactgctgataattacgcagcgtttcttcggctatttgcaattgcgcatacgc\n",
      "cagttgctgattgaaatagctttgcgcgacattagaaaccagcagaatatgcaccgcgcg\n",
      "ctgagcttcctcagtggctaaataattttgtcgctcggcttcgctcatgttctttaagcg\n",
      "accgaaaaaatcgaggtcaaagctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (616175, 618180)), (618146, 618180), '+'), (('G000299455', (3687649, 3689657)), (3689623, 3689657), '+'), (('G000299455', (3844392, 3844426)), (3844392, 3844426), '-')]\n",
      ">G000006925|616175|618180|618146|618180|+\n",
      "-cagggcaagattacgaaagcgcgctccccgcaag\n",
      ">G000299455|3687649|3689657|3689623|3689657|+\n",
      "-caggacaaatttacgaaagcctgctccccgcaag\n",
      ">G000299455|3844392|3844426|3844392|3844426|-\n",
      "gcagggcaagattacgaaagcccgctccccgcaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (729510, 729590), '+'), (('G000299455', (2174641, 2181396)), (2177906, 2177986), '+'), (('G000299455', (2910238, 2916987)), (2913642, 2913722), '-')]\n",
      ">G000006925|726235|732999|729510|729590|+\n",
      "-gcttcttcgggggattattcgtccgccacgggcgcgttttgatttttaccaggcgcgtt\n",
      "cagcctggtcacgggcagagt\n",
      ">G000299455|2174641|2181396|2177906|2177986|+\n",
      "-acttcttcgggggattattcgtccgccacgggcacgttttgatttttatcaggcgcgtt\n",
      "cagcctggtcacgggctgagt\n",
      ">G000299455|2910238|2916987|2913642|2913722|-\n",
      "cacttcttcgggggattattcgtccgccacgggcacgttttgatttttatcaggcgcgat\n",
      "cagcctggtcacgggctgag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3065287, 3076770)), (3076225, 3076671), '+'), (('G000299455', (63990, 64641)), (63990, 64434), '-'), (('G000299455', (874911, 875768)), (874911, 875356), '-'), (('G000299455', (1996963, 1997507)), (1997062, 1997507), '-'), (('G000299455', (4541696, 4542240)), (4541795, 4542240), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3065287|3076770|3076225|3076671|+\n",
      "-ctgaagcgcccggtctggcagtgtgtgcagtactgctccccataataatcgtggtgaca\n",
      "ttgcctgcagtgccagctggctttacgtactacgggtaaggtatccgggacgaattgctg\n",
      "cagacgtttaatcaaccgtatttctctgcgttccggcctgacataagggcactcatgacc\n",
      "gtgctccgtcagcccttcgtcagcgtgttcaaaccagggccgttcagtgtcgtattgcgg\n",
      "gtggtatctgagtgcactgccgcaaaggtggcaggtgtagctatcgtaaggtgcagtctg\n",
      "tgcggtacgggcaccggtcagacgtccgttgccatcaagtgcgagaaaagattttgcgta\n",
      "catagtatatgttcctcagccgccagacgacacgcaggcaacagccgtcccggtacgggc\n",
      "agcgtggtcagggtgtgaatggcggtt\n",
      ">G000299455|63990|64641|63990|64434|-\n",
      "actgaagcgcccggtctggcagtgtgtgcagtaccgctccccataataatcgtggtgaca\n",
      "ttgtctgcagtgccagctggctttacgcaccacgggtaaggcatccggtacgaattgctg\n",
      "cagacgctgaatcaagcgtacctccctgcgttccggtctgacataaggacactgctgacc\n",
      "gtgcgctgtcagcccgtcgtcagtgtgttcaaaccagggacgttcagtgtcgtattgcgg\n",
      "gtggtatctgagtgcactgccgcaaaggtggcaggtgtagcgatcataaggtgcagtctg\n",
      "tgcggtacgggcaccggtcagacgtccgttgccatcaagtgcgagaaaagattttgcgta\n",
      "catagtatatgttccttag-cgccagacgacacgcaggcaacagccgtcccggtacgggc\n",
      "agcctggtc-tggtgtgaatggcggt-\n",
      ">G000299455|874911|875768|874911|875356|-\n",
      "actgaagcgccctgtatgacagtgtgtgcagtactgctccccataataatcgtggtgaca\n",
      "ttgtctgcagtgccagctggctttacgcactacgggtaaggcgtccggtacgaattgccg\n",
      "taagcgtttaatcaacagtatttctctgcgctccggtctgacataagggcactcatgacc\n",
      "atgctcagtcagcccgtcgtcagtgtgttcaaaccagggacgttcagtgtcgtattgcgg\n",
      "gtggtatctgagagcactgccacagaggtggcaggtgtaacgatcgtatggcgctgtctg\n",
      "tgcggtacgggcaccggtcagacgtccgttgccatcaagtgcgagaaaggattttgcgta\n",
      "catcgt-catattcctcagccgccagacgacacgcaggcaacagccgtcccggtacgggc\n",
      "agcgtggtcagggtgtgaatggcggt-\n",
      ">G000299455|1996963|1997507|1997062|1997507|-\n",
      "actgaatcctccggtctggcagtgtgtgcagtaccgctccccataataatcgtggtgaca\n",
      "ttgtctgcagtgccagctggctttacgcaccacgggtaaggcatccggtacgaattgctg\n",
      "cagacgtttaatcaactgtatttctctgcgctccggcctgacataagggcactgttgacc\n",
      "gtgctctgtcagcctgtcgtcagtgtgttcaaaccagggaagttcagtgtcgtattgcgg\n",
      "atggtatctgagtgcactgccacacaagtggcaggtgtagttagcataaggtgcggcctg\n",
      "tgcagtacgggcgcccgtcagacgtccgttgccatcaagagcgataaaggattttgcgta\n",
      "catagt-catatacctcagccgccagacgacacgcaggcaacagccgtcccggtacgggc\n",
      "agcgtggtcagggtgtgaatggcggt-\n",
      ">G000299455|4541696|4542240|4541795|4542240|-\n",
      "actgaatcctccggtctggcagtgtgtgcagtactgctccccataataatcgtggtgaca\n",
      "ttgtctgcagtgccagctggctttacgcaccacgggtaaggcatccggtacgaattgctg\n",
      "cagacgtttaatcaactgtatttctctgcgctccggcctgacataagggcactgttgacc\n",
      "gtgctctgtcagcctgtcgtcagtgtgttcaaaccagggaagttcagtgtcgtattgcgg\n",
      "atggtatctgagtgcactgccacacaagtggcaggtgtagttagcataaggtgcggcctg\n",
      "tgcagtacgggcgcccgtcagacgtccgttgccatcaagagcgataaaggattttgcgta\n",
      "catagt-catattcctcagccgccagacgacacgcaggcaacagccgtcccggtacgggc\n",
      "agcgtggtcagggtgtgaatggcggt-\n",
      "\n",
      "\n",
      "32 [(('G000006925', (251358, 252232)), (251469, 251537), '+'), (('G000006925', (315068, 315858)), (315762, 315830), '-'), (('G000006925', (332059, 337870)), (332753, 332821), '-'), (('G000006925', (493019, 493813)), (493713, 493781), '-'), (('G000006925', (509301, 510374)), (509996, 510064), '-'), (('G000006925', (724190, 725876)), (724885, 724953), '-'), (('G000006925', (1094884, 1095679)), (1095578, 1095646), '-'), (('G000006925', (1099363, 1100153)), (1099390, 1099458), '+'), (('G000006925', (1411510, 1412350)), (1412254, 1412322), '-'), (('G000006925', (1883764, 1885448)), (1884459, 1884527), '-'), (('G000006925', (1956633, 1961076)), (1957328, 1957396), '-'), (('G000006925', (2084425, 2085215)), (2084452, 2084520), '+'), (('G000006925', (2236395, 2237421)), (2236658, 2236726), '+'), (('G000006925', (3065287, 3076770)), (3075458, 3075526), '-'), (('G000006925', (3812816, 3813607)), (3813510, 3813578), '-'), (('G000006925', (4639759, 4640549)), (4640453, 4640521), '-'), (('G000006925', (4650084, 4650878)), (4650115, 4650183), '+'), (('G000006925', (4656735, 4658205)), (4657442, 4657510), '+'), (('G000006925', (4673259, 4674053)), (4673290, 4673358), '+'), (('G000006925', (4679888, 4680760)), (4680583, 4680651), '-'), (('G000006925', (4750261, 4751054)), (4750291, 4750359), '+'), (('G000006925', (4753880, 4754990)), (4754894, 4754962), '-'), (('G000006925', (4760730, 4761525)), (4760762, 4760830), '+'), (('G000006925', (4793569, 4794365)), (4793599, 4793667), '+'), (('G000006925', (4828486, 4828646)), (4828543, 4828611), '-'), (('G000299455', (906289, 909876)), (906323, 906391), '+'), (('G000299455', (912160, 912572)), (912192, 912260), '+'), (('G000299455', (4752701, 4766669)), (4752734, 4752802), '+'), (('G000299455', (4771847, 4785261)), (4780407, 4780475), '-'), (('G000299455', (5275700, 5279438)), (5279187, 5279255), '-'), (('G000299455', (5295712, 5295801)), (5295735, 5295800), '+'), (('G000299455', (5343200, 5343533)), (5343336, 5343404), '+')]\n",
      ">G000006925|251358|252232|251469|251537|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|315068|315858|315762|315830|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|332059|337870|332753|332821|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|493019|493813|493713|493781|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|509301|510374|509996|510064|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|724190|725876|724885|724953|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|1094884|1095679|1095578|1095646|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|1099363|1100153|1099390|1099458|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|1411510|1412350|1412254|1412322|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|1883764|1885448|1884459|1884527|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|1956633|1961076|1957328|1957396|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|2084425|2085215|2084452|2084520|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|2236395|2237421|2236658|2236726|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|3065287|3076770|3075458|3075526|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|3812816|3813607|3813510|3813578|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4639759|4640549|4640453|4640521|-\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4650084|4650878|4650115|4650183|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4656735|4658205|4657442|4657510|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4673259|4674053|4673290|4673358|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4679888|4680760|4680583|4680651|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttctcccgaagtc\n",
      "cgtcaacg-\n",
      ">G000006925|4750261|4751054|4750291|4750359|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4753880|4754990|4754894|4754962|-\n",
      "-aactccctgagaaagaggtaaacaggatgactaaaaatacacgtttttcccctgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4760730|4761525|4760762|4760830|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4793569|4794365|4793599|4793667|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000006925|4828486|4828646|4828543|4828611|-\n",
      "-aacgccctgagaaagaggtaaacagaatgactaaaaatacacgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000299455|906289|909876|906323|906391|+\n",
      "-aactccctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtt\n",
      "cgtcatagg\n",
      ">G000299455|912160|912572|912192|912260|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtt\n",
      "cgtcatagg\n",
      ">G000299455|4752701|4766669|4752734|4752802|+\n",
      "-aacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacgg\n",
      ">G000299455|4771847|4785261|4780407|4780475|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000299455|5275700|5279438|5279187|5279255|-\n",
      "aaacttcctgagaaagaggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "cgtcaacg-\n",
      ">G000299455|5295712|5295801|5295735|5295800|+\n",
      "-aactccct--gaaagaggtaaataggataacgaaaaatacacg-ttttcccccgaagtc\n",
      "cgttaaggg\n",
      ">G000299455|5343200|5343533|5343336|5343404|+\n",
      "-aacttccagagaaagcggtaaacaggatgactaaaaatactcgtttttcccccgaagtc\n",
      "agtcatagg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2141536, 2141608), '-'), (('G000299455', (1911336, 1921536)), (1918772, 1918844), '+'), (('G000299455', (3326617, 3362069)), (3339018, 3339090), '+')]\n",
      ">G000006925|2138845|2144864|2141536|2141608|-\n",
      "cagaattccgcctgcaaccttatgacgtggtgtatgtcaccaccgccccggtttcccgct\n",
      "ggaaccgtctga-\n",
      ">G000299455|1911336|1921536|1918772|1918844|+\n",
      "cagaattccgcctgcaaccttatgacgtggtgtatgtcaccaccgccccggtttcccgct\n",
      "ggaaccgtctga-\n",
      ">G000299455|3326617|3362069|3339018|3339090|+\n",
      "-ggaattccagcttgaaccttacgacattgtttatgtcactaccgcgccgctggctcgtt\n",
      "ggaaccgtgtgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4143132, 4163489)), (4155600, 4155651), '-'), (('G000299455', (1535847, 1550961)), (1547394, 1547445), '+'), (('G000299455', (4996216, 5016676)), (5004145, 5004196), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 32\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 32\r",
      "   10 / 32\r",
      "   20 / 32\r",
      "   30 / 32\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 31  f\b\b\r",
      "STEP     2 / 31  f\b\b\r",
      "STEP     3 / 31  f\b\b\r",
      "STEP     4 / 31  f\b\b\r",
      "STEP     5 / 31  f\b\b\r",
      "STEP     6 / 31  f\b\b\r",
      "STEP     7 / 31  f\b\b\r",
      "STEP     8 / 31  f\b\b\r",
      "STEP     9 / 31  f\b\b\r",
      "STEP    10 / 31  f\b\b\r",
      "STEP    11 / 31  f\b\b\r",
      "STEP    12 / 31  f\b\b\r",
      "STEP    13 / 31  f\b\b\r",
      "STEP    14 / 31  f\b\b\r",
      "STEP    15 / 31  f\b\b\r",
      "STEP    16 / 31  f\b\b\r",
      "STEP    17 / 31  f\b\b\r",
      "STEP    18 / 31  f\b\b\r",
      "STEP    19 / 31  f\b\b\r",
      "STEP    20 / 31  f\b\b\r",
      "STEP    21 / 31  f\b\b\r",
      "STEP    22 / 31  f\b\b\r",
      "STEP    23 / 31  f\b\b\r",
      "STEP    24 / 31  f\b\b\r",
      "STEP    25 / 31  f\b\b\r",
      "STEP    26 / 31  f\b\b\r",
      "STEP    27 / 31  f\b\b\r",
      "STEP    28 / 31  f\b\b\r",
      "STEP    29 / 31  f\b\b\r",
      "STEP    30 / 31  f\b\b\r",
      "STEP    31 / 31  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 32\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 32\r",
      "   10 / 32\r",
      "   20 / 32\r",
      "   30 / 32\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 31  f\b\b\r",
      "STEP     2 / 31  f\b\b\r",
      "STEP     3 / 31  f\b\b\r",
      "STEP     4 / 31  f\b\b\r",
      "STEP     5 / 31  f\b\b\r",
      "STEP     6 / 31  f\b\b\r",
      "STEP     7 / 31  f\b\b\r",
      "STEP     8 / 31  f\b\b\r",
      "STEP     9 / 31  f\b\b\r",
      "STEP    10 / 31  f\b\b\r",
      "STEP    11 / 31  f\b\b\r",
      "STEP    12 / 31  f\b\b\r",
      "STEP    13 / 31  f\b\b\r",
      "STEP    14 / 31  f\b\b\r",
      "STEP    15 / 31  f\b\b\r",
      "STEP    16 / 31  f\b\b\r",
      "STEP    17 / 31  f\b\b\r",
      "STEP    18 / 31  f\b\b\r",
      "STEP    19 / 31  f\b\b\r",
      "STEP    20 / 31  f\b\b\r",
      "STEP    21 / 31  f\b\b\r",
      "STEP    22 / 31  f\b\b\r",
      "STEP    23 / 31  f\b\b\r",
      "STEP    24 / 31  f\b\b\r",
      "STEP    25 / 31  f\b\b\r",
      "STEP    26 / 31  f\b\b\r",
      "STEP    27 / 31  f\b\b\r",
      "STEP    28 / 31  f\b\b\r",
      "STEP    29 / 31  f\b\b\r",
      "STEP    30 / 31  f\b\b\r",
      "STEP    31 / 31  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4143132|4163489|4155600|4155651|-\n",
      "cggtgatgttcatcatcgatcaatgctgcgaagagattgatttctttagta-\n",
      ">G000299455|1535847|1550961|1547394|1547445|+\n",
      "-ggtgtgttacatcatcgaccacttctgcgatgaggtcgatttcttcagtat\n",
      ">G000299455|4996216|5016676|5004145|5004196|+\n",
      "-ggtgatgttcatcatcgatcaatgctgcgaagagactgatttctttagtat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (908032, 908771)), (908032, 908380), '-'), (('G000006925', (1394069, 1394808)), (1394459, 1394808), '+'), (('G000299455', (2538890, 2539629)), (2539280, 2539629), '+')]\n",
      ">G000006925|908032|908771|908032|908380|-\n",
      "ggaatcgcctgctgaaagacggtcagacagtgattgtggttaccgtggctgatgttatga\n",
      "gcgccctgcacgccagctatgacgacgggcagtcaggcgaaaaatttttgcgggagctgt\n",
      "gcgaagtggatctgctggttcttgatgaaattggcattcagcgcgagacgaaaaacgagc\n",
      "aggtggtgctgcaccagattgttgatcgccggacagcgtcgatgcgcagcgtggggatgc\n",
      "tgacaaacctgaactatgaggtgatgaaaacattgctcggcgagcgggtgatggatcgca\n",
      "tggtcatgaacggcgggcgctgggttaattttaactgggagagctggc-\n",
      ">G000006925|1394069|1394808|1394459|1394808|+\n",
      "gcaaccggctgatggcgaaggggcgtagcgtgattatcgtcaccgtgtccgatgtcatga\n",
      "gcgtgttgcatgagagctacgacaacggcaaatccggtgaaaaatttttacaggagcttt\n",
      "gtggtgttgacctgctggtcctggatgaaattggcatgcagcgggatacgaaaaacgagc\n",
      "aggtggtactgaaccagattgttgatcgccggacggcatcgttacgcggtgtggggatgc\n",
      "tgacaaatattaaccatgcagcgatgaatacacttctcggcgagcgggtgatggatcgca\n",
      "tggtcatgaacggcgggcgctgggtgaattttaactgggagagctggcg\n",
      ">G000299455|2538890|2539629|2539280|2539629|+\n",
      "gtaatcgtctgctgaaggccggtaaatccgtgatagtggtcaccgtggcggatgtgatga\n",
      "gcgcgttacatgccagctatgacgacggactgtcaggggaaaaatttttgcgtgaactgt\n",
      "gcgaagtggacctcctggttcttgacgaaattggcatccagcgtgagacgaaaaacgagc\n",
      "aggtggtactgcaccagattgttgaccgccggacggcatcactgcgcagtgtcgggatgc\n",
      "tgacaaacctgaatcatgccgcaatgagcacgcttcttggtgagaggattatggaccgga\n",
      "tgaccatgaacggtggtcgttgggtgaattttaactgggagagctggcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2139586, 2139636), '-'), (('G000299455', (1911336, 1921536)), (1920744, 1920794), '+'), (('G000299455', (3326617, 3362069)), (3340998, 3341048), '+')]\n",
      ">G000006925|2138845|2144864|2139586|2139636|-\n",
      "tggaagaacacggaatcagcgtctatgccagcatcccgctgtcggagtgg-\n",
      ">G000299455|1911336|1921536|1920744|1920794|+\n",
      "tggaagaacacggtatcagcgtctatgccagcatcccgctgtcggagtgg-\n",
      ">G000299455|3326617|3362069|3340998|3341048|+\n",
      "-ggaagagcacggcatcagcgtttatgccactatcccaatgtccgagtggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3268417, 3281594)), (3275589, 3275662), '-'), (('G000299455', (642419, 661467)), (660344, 660417), '+'), (('G000299455', (665003, 670357)), (666563, 666636), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3268417|3281594|3275589|3275662|-\n",
      "gcctgcatcctttcgtagttaaagctcgcttgcagcagcgacgaacgaaagcccagacgg\n",
      "gtgatatcttttt-\n",
      ">G000299455|642419|661467|660344|660417|+\n",
      "-cctgcatcctttcgtagttaaagctcgcttgcagcagcgacgaacgaaagcccagacgg\n",
      "gtgatatctttttt\n",
      ">G000299455|665003|670357|666563|666636|+\n",
      "-cctgcatacgttcgtagttaaaagacgcttgtaacagcatggaacgccaggcgacgcgg\n",
      "ttgatatctttttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1667789, 1670305)), (1669588, 1669624), '-'), (('G000006925', (4077347, 4077383)), (4077347, 4077383), '-'), (('G000299455', (2495066, 2505927)), (2495747, 2495783), '+')]\n",
      ">G000006925|1667789|1670305|1669588|1669624|-\n",
      "tgggcactggaagctgataccgtagaatacggtgaa-\n",
      ">G000006925|4077347|4077383|4077347|4077383|-\n",
      "tgggcactggtaccagataccgtagattacggtgaa-\n",
      ">G000299455|2495066|2505927|2495747|2495783|+\n",
      "-gggcactggaagctgataccgtagaatacggtgaat\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (745043, 745614), '+'), (('G000006925', (1921807, 1928114)), (1921973, 1922544), '-'), (('G000299455', (2192349, 2193633)), (2193031, 2193602), '+'), (('G000299455', (2897570, 2901707)), (2897601, 2898172), '-'), (('G000299455', (3585529, 3586267)), (3585696, 3586267), '-')]\n",
      ">G000006925|739473|745780|745043|745614|+\n",
      "-agaaatggtgccttttttattgcagaaaagcgagaggtaattatgcgtaaactttatgc\n",
      "tgccattttgtccgcagccatctgtctggctgtatccggtgcgcctgcatgggcgtctga\n",
      "acatcggtccacgctgagcgcggggtatcttcatgcctcgacgaacgctcccggcagtga\n",
      "tgatctgaacgggattaacgtgaaataccgttatgagtttacggacacgctggggctggt\n",
      "gacgtcattcagctatgccaacgctgaagatgagcaaaaaacgcattacagtgatacccg\n",
      "ctggcatgaggattccgtgcgtaaccgctggttcagcgtgatggcggggctgtctgtgcg\n",
      "cgtgaatgaatggttcagcgcgtatgcgatggcgggcgtggcttacagccgtgtgtcgac\n",
      "cttctccggggattatctccgcgtaactgacaacaaggggaaaacgcacgatgtgctgac\n",
      "cgggagtgatgacaaccgccacagcaacacgtctctggcgtggggggctggcgtgcagtt\n",
      "taacccgaccgaatccgtggccattgaccttg\n",
      ">G000006925|1921807|1928114|1921973|1922544|-\n",
      "cagaaatggtgccttttttattgcagaaaagcgagaggtaattatgcgtaaactttatgc\n",
      "tgccattttgtccgcagccatctgtctggctgtatccggtgcgcctgcatgggcgtctga\n",
      "acatcggtccacgctgagcgcggggtatcttcatgcctcgacgaacgctcccggcagtga\n",
      "tgatctgaacgggattaacgtgaaataccgttatgagtttacggacacgctggggctggt\n",
      "gacgtcattcagctatgccaacgctgaagatgagcaaaaaacgcattacagtgatacccg\n",
      "ctggcatgaggattccgtgcgtaaccgctggttcagcgtgatggcggggctgtctgtgcg\n",
      "cgtgaatgaatggttcagcgcgtatgcgatggcgggcgtggcttacagccgtgtgtcgac\n",
      "cttctccggggattatctccgcgtaactgacaacaaggggaaaacgcacgatgtgctgac\n",
      "cgggagtgatgacaaccgccacagcaacacgtctctggcgtggggggctggcgtgcagtt\n",
      "taacccgaccgaatccgtggccattgacctt-\n",
      ">G000299455|2192349|2193633|2193031|2193602|+\n",
      "-agaaatggcgccttttttattgcagaaaagcgagaggtaattatgcgtaaattatgtgc\n",
      "tgttattttgtccgcagtagtctggcaggtcgccgctgctacgccagcgagtgcagcaga\n",
      "acatcagtccacgctgagcgcggggtatctccatgcctcgacgaacgttcccggtagtga\n",
      "tgatctgaacgggattaacgtgaaataccgttatgagtttacggacgcgctggggctgat\n",
      "tacgtccttcagttatgccaatgctgaggatgagcaaaaaacgcgctacagcgatacccg\n",
      "ctggcatgaagattccgtgcgtaaccgctggttcagcgtgatggcggggccgtctgtacg\n",
      "cgtgaatgaatggttcagcgcgtatgcgatggcgggtgtggcttacagccgtgtgtcgac\n",
      "tttctccggggattatctccgcgtaactgacaacaaggggaaaacgcacgatgtgctgac\n",
      "cggaagtgatgacggtcgccacagcaacacgtctctggcgtggggggctggcgtgcagtt\n",
      "taacccgaccgaatccgtgaccattgaccttg\n",
      ">G000299455|2897570|2901707|2897601|2898172|-\n",
      "cagaaatggcgccttttttattgcagaaaagcgagaggtaattatgcgtaaattatgtgc\n",
      "tgttattttgtccgcagtagtctggcaggtcgccgctgctacgccagcgagtgcagcaga\n",
      "acatcagtccacgctgagcgcggggtatctccatgcctcgacgaacgttcccggtagtga\n",
      "tgatctgaacgggattaacgtgaaataccgttatgagtttacggacgcgctggggctgat\n",
      "tacgtccttcagttatgccaatgctgaggatgagcaaaaaacgcgctacagcgatacccg\n",
      "ctggcatgaagattccgtgcgtaaccgctggttcagcgtgatggcggggccgtctgtacg\n",
      "cgtgaatgaatggttcagcgcgtatgcgatggcgggtgtggcttacagccgtgtgtcgac\n",
      "tttctccggggattatctccgcgtaactgacaacaaggggaaaacgcacgatgtgctgac\n",
      "cggaagtgatgacggtcgccacagcaacacgtctctggcgtggggggctggcgtgcagtt\n",
      "taacccgaccgaatccgtgaccattgacctt-\n",
      ">G000299455|3585529|3586267|3585696|3586267|-\n",
      "caggaatgacgccttttttattgcagaaaagcgagaggtaattatgcgtaaagtttgtgc\n",
      "agcaattttgtccgcagccatttgtctggccgtatccggtgcgcctgcatgggcgtctga\n",
      "acatcagtccacgctgagcgcggggtatcttcatgcctcgacgaacgtccccggcagtga\n",
      "tgatctgaacgggattaacgtgaaataccgttatgagtttacggacacactggggctggt\n",
      "gacgtcattcagctatgcaggagacaagaatcgccagctgacccgttacagcgatacccg\n",
      "ctggcatgaagattccgtgcgtaaccgctggttcagcgtaatggcggggccgtctgtgcg\n",
      "cgtaaatgaatggttcagcgcgtatgcgatggcgggtgtagcttacagccgtgtgtcgac\n",
      "tttctccggggattatcttcgcgtaactgacaacaaggggaaaaagcacgatgtgctgac\n",
      "cggaagtgatgacggtcgccacagcaacacgtctctggcgtggggggctggcgtgcagtt\n",
      "taacccgaccgaatccgtggccattgatatt-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1, 24671)), (15465, 15550), '+'), (('G000006925', (905848, 907049)), (905996, 906081), '+'), (('G000006925', (1397676, 1398098)), (1397856, 1397941), '-'), (('G000006925', (1581377, 1581641)), (1581538, 1581623), '+'), (('G000006925', (2062783, 2064325)), (2064047, 2064132), '+'), (('G000299455', (2541782, 2542216)), (2541974, 2542059), '-'), (('G000299455', (2927401, 2931337)), (2930275, 2930360), '+'), (('G000299455', (3256459, 3256588)), (3256459, 3256543), '-'), (('G000299455', (4431865, 4470166)), (4434983, 4435068), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 2144\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 2144\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\n",
      "Reallocating..done. *alloclen = 1172\n",
      "\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|15465|15550|+\n",
      "-agaggtcttttctcgttaccagcgccgccactacggcggtgatacagatgacgatcagg\n",
      "gcgacaatcatcgccttatgctgctt\n",
      ">G000006925|905848|907049|905996|906081|+\n",
      "cagaggtctttcctcgttaccagtgccgtcactatgacggttaaacagatgacgatcagg\n",
      "gcgattaacatcgccttttgctgct-\n",
      ">G000006925|1397676|1398098|1397856|1397941|-\n",
      "cagaggtctttcctcgttaccagtgccgtcactatgacggttaaacagatgacgatcagg\n",
      "gcgattaacatcgccttttgctgct-\n",
      ">G000006925|1581377|1581641|1581538|1581623|+\n",
      "cagaggtctttcctcgttaccagtgccgtcactatgacggttaaacagatgacgatcagg\n",
      "gcgattaacatcgccttttgctgct-\n",
      ">G000006925|2062783|2064325|2064047|2064132|+\n",
      "cagaggtctttcctcgttaccagtgccgtcactatgacggttaaacagatgacgataagg\n",
      "gcgattaacatcgccttttgctgct-\n",
      ">G000299455|2541782|2542216|2541974|2542059|-\n",
      "cagaggtctttcctcgttaccagtgccgtcactacgacggtaatacagatgacgatcagg\n",
      "gcgactaacatcgccttttgctgct-\n",
      ">G000299455|2927401|2931337|2930275|2930360|+\n",
      "-agaggtctttcctcgttaccagtgccgtcactatgacggttaaacagatgacgatcagg\n",
      "gcgattaacatcgccttttgctgctt\n",
      ">G000299455|3256459|3256588|3256459|3256543|-\n",
      "cagaggtctttcttcgttatcagtgccgtcacaatgacggtaatacaga-gatgagcagg\n",
      "gtgattaacatcgccttttgctgct-\n",
      ">G000299455|4431865|4470166|4434983|4435068|-\n",
      "cagaggtcttttctcgttaccagcgccgccactacggcggtgatacagatgacgatcagg\n",
      "gcgacaatcatcgccttatgctgct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4350106, 4350261), '-'), (('G000299455', (3326617, 3362069)), (3337679, 3337834), '+'), (('G000299455', (4865050, 4904182)), (4900157, 4900312), '-')]\n",
      ">G000006925|4315026|4354132|4350106|4350261|-\n",
      "agcgtcgggcagtatctggcaggggataaaggcggcacgctggagatcgctaaacgcttt\n",
      "gacagcggcgtggtggtgggtggctatgccacgatcaccaacgtttcgaaagaggagtac\n",
      "ggcgaaggggacttcaccaaaggcgtgtatgtctc-\n",
      ">G000299455|3326617|3362069|3337679|3337834|+\n",
      "-gcgttgggcaatatctggcaaaagataaaggcgcaacgatcgacgtcgccaaacgcttt\n",
      "gacagcggcgtggcggtaggggtatgggcggcaatcagtaacgtatctaaagatgactac\n",
      "ggcgaaggcggctttagtaaaggtttttatatctcg\n",
      ">G000299455|4865050|4904182|4900157|4900312|-\n",
      "agcgtcgggcagtatctggcaggggataaaggcggcacgctggagatcgccaaacgcttt\n",
      "gatagcggcgtggtggttggcggctatgccacgatcactaatgtttcaaaagaagagtac\n",
      "ggcgaaggggacttcaccaaaggcgtgtatgtctc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (730996, 732147), '+'), (('G000299455', (2174641, 2181396)), (2179392, 2180543), '+'), (('G000299455', (2910238, 2916987)), (2911085, 2912236), '-')]\n",
      ">G000006925|726235|732999|730996|732147|+\n",
      "-ggcaacgctggcggcgatgccggagatgacggtggaaaaagcccgtccgattctggctg\n",
      "cttcaccgcaggcgaatgccggaccctcacttcgtgatcagattatggccctggatgagg\n",
      "caaaaggggcagaggcgcaggctgaaaaactggcggcctgtccggggatgaccgtggaga\n",
      "atgcccgggctgtactggcggcagcctcaggcagggcagaaccggtttctgcatccacaa\n",
      "ccgccatgtttgaacatttcatggcgaaccattcaccggcagcggtacagggtggcgtgc\n",
      "cacagacgtcagcagacggtgatgcggacgtgaaaatgctcatggccatgccatgaagtc\n",
      "agtgctgaccatcaatatgaggtttttacaatatggtaacgaaaaccatcactgaacagc\n",
      "gtgcggaagtacgtatttttgccggtaatgatccggcccataccgccacaggcagcagcg\n",
      "ggatttcctcggcaacaccggctctgacgcccctgatgctggatgaggataccgggaaac\n",
      "tggtggtctgggacggacagaaggcaggccatgtggttggcatactggtactgccgcttg\n",
      "aaggtacagagacggtgctgacgtattacaagtcggggacctttgcgacggaggccatcc\n",
      "actggcctgagagagtggatacacacaaaaaggcgaatgcctttgccggcagtgccctga\n",
      "gtcacgcagcgctgccgtaacattttatcaggtcactgcggtagcctgactgatttctga\n",
      "atgaaaggaactgatttatgggattgtttacgacccgccagttactcggttataccgaac\n",
      "aaaaagttaaatttcgtgcgctgtttctggagctgtttttccgccgtacggtgaatttcc\n",
      "ataccgaagaggtaatgctggacaaaattaccggaaaaacgccggtggcggcctatgtct\n",
      "ccccggttgttgaaggaaaggtgttgcgtcaccgtggcggtgaaactcgtgtactgcgcc\n",
      "cgggctacgttaagccgaaacacgaatttaattaccagcaggcggtggagcgtcttcccg\n",
      "gtgaagatccggcccagctgaacggcccggcttaccgtcgcctgcgtatcatcaccgata\n",
      "acctcaaacagg\n",
      ">G000299455|2174641|2181396|2179392|2180543|+\n",
      "-ggcaacgctggcggcgatgccggagatgacggtggaaaaagcccgcccgatcttggctg\n",
      "cctcaccgcaggcggatgccgggccctcacttcgtgatcagatcatggccctggatgagg\n",
      "caaaaggggcagaagcgcaggctgaaaaactggcggcctgcccgggaatgaccgtggaga\n",
      "acgcccgggctgtgctggctgcgggatcaggtaaggccgaaccggtctctgcatccacaa\n",
      "ccgccctgtttgaacatttcatggcgaatcattcaccggcagcggtgcggggtggcgtgt\n",
      "cacagacgtcagcagacggtgatgcggacgtgaaaatgctcatggccatgccatgaagtc\n",
      "agtgctgaccatcaatatgaggttttaacaaaatggtgacgaaaaccatcactgaacagc\n",
      "gtgcggaagtacgtatttttgccggtaatgatccggctcataccgccacaggcagcagcg\n",
      "ggatttcttctgcaacaccggctctgacgcccctgatgctggatgaagccaccgggaaac\n",
      "tggtggtctgggatggacagaaagccggtagtgcggttggcatactggtactgccgcttg\n",
      "aaggcaccgagacggtgctgacctattacaagtcggggacctttgcgacggaggcaatcc\n",
      "gctggcctgacagtgtggatgaacacaaaaaggcaaatgcctttgccggcacagccctga\n",
      "gtcacgcggctctgccgtaacacgttatcaggccaccatggtggcctgactgatttctta\n",
      "atgaaaggaactgatttatgggattgtttacgacccgccagttactcggttataccgaac\n",
      "aaaaagttaaattccgtgcgctgtttctggagctgtttttccgccgtacggtgaatttcc\n",
      "acaccgaagaggtgatgctggacaaaattaccggaaaaacgccggtggcggcctatgtct\n",
      "ccccgatcgttgaaggaaaagtgcttcgccatcgcggtggtgaaacccgcgtgttacgtc\n",
      "cgggctacgtcaagcccaaacacgaatttaattaccagcaggcggttgagcgccttcctg\n",
      "gtgaagatccggctcagctgaacgacccggcctaccgtcgtctgcgtatcattaccgata\n",
      "acctcaaacagg\n",
      ">G000299455|2910238|2916987|2911085|2912236|-\n",
      "cggcaacgctggcggcgatgccggagatgacggtggaaaaagcccgcccgatcctggcgg\n",
      "ctgcaccactggcggatgccgggccctcacttcgtgatcagatcatggccctggatgagg\n",
      "caaaaggggcagaagcgcaggctgaaaaactggcggcctgcccgggaatgaccgtggaga\n",
      "acgcccgggctgtgctggctgcgggatcaggtaaggccgaaccggtctctgcatccacaa\n",
      "ccgccctgtttgaacatttcatggcgaatcattcaccggcagcggtgcggggtggcgtgt\n",
      "cacagacgtcagcagacggtgatgcggacgtgaaaatgctcatggccatgccatgaagtc\n",
      "agtgctgaccatcaatatgaggttttaacaaaatggtgacgaaaaccatcactgaacagc\n",
      "gtgcggaagtacgtatttttgccggtaatgatccggctcataccgccacaggcagcagcg\n",
      "ggatttcttctgcaacaccggctctgacgcccctgatgctggatgaagccaccgggaaac\n",
      "tggtggtctgggatggacagaaagccggtagtgcggttggcatactggtactgccgcttg\n",
      "aaggcacagagacggtgctgacctattacaagtcggggacctttgcgacggaggcaatcc\n",
      "gctggcctgacagtgtggatgaacacaaaaaggcaaatgcctttgccggcacagccctga\n",
      "gtcacgcggctctgccgtaacacgttatcaggccaccatggtggcctgactgatttctga\n",
      "atgaaaggaactgatttatgggattgtttacgacccgccagttactcggttataccgaac\n",
      "aaaaagttaaattccgtgcgctatttctggagctgtttttccgccgtacggtgaatttcc\n",
      "acaccgaagaggtgatgctggacaaaattaccggaaaaacgccggtggcggcctatgtct\n",
      "ccccgatcgttgaaggaaaagtgctgcgccatcgtggtggtgaaacccgcgtgttgcgtc\n",
      "cgggctacgtcaagccgaaacacgaatttaattaccagcaggcggttgagcgccttcccg\n",
      "gtgaagatccggctcagctgaacgacccggcctaccgtcgtctgcgtatcatcactgata\n",
      "acctcaaacag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (2985320, 2985376), '+'), (('G000299455', (988799, 1028140)), (1007653, 1007709), '-'), (('G000299455', (3727115, 3736055)), (3729406, 3729462), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2976303|3015537|2985320|2985376|+\n",
      "-gattaacgccctgccccgccagtgggtgaatggtatgcgcggcgtcgcccaccagc\n",
      ">G000299455|988799|1028140|1007653|1007709|-\n",
      "agatttactccctgccccgccagcgggtgaatggtatgcgcggcgtcgcccaccag-\n",
      ">G000299455|3727115|3736055|3729406|3729462|+\n",
      "-gattcaccccctgccccgccagcggatggatggtatgcgcggcatcgcccaccagc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3915855, 3917645)), (3916284, 3916332), '-'), (('G000299455', (1604015, 1621831)), (1619056, 1619104), '+'), (('G000299455', (5202806, 5268469)), (5241222, 5241270), '+')]\n",
      ">G000006925|3915855|3917645|3916284|3916332|-\n",
      "actcgctggaaatagcagtaaaaacgcgtatcaagctgttctggcgac-\n",
      ">G000299455|1604015|1621831|1619056|1619104|+\n",
      "-cacgctgcaaatagcactacaaacgcgtattaagttgttctggcgacc\n",
      ">G000299455|5202806|5268469|5241222|5241270|+\n",
      "actcgctggaaatagcagtaaaaacgcgtatcaagctgttctggcgac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3700266, 3700328), '+'), (('G000299455', (269622, 287922)), (276010, 276072), '+'), (('G000299455', (535426, 536523)), (536092, 536154), '+')]\n",
      ">G000006925|3693106|3702974|3700266|3700328|+\n",
      "ataggtccattcggtagacgagtaggcagaaagcggtgccatcgtgccagaggcgttgcg\n",
      "ta\n",
      ">G000299455|269622|287922|276010|276072|+\n",
      "ataggtccattcggtagacgagtaggcagaaagcggtgccatcgtgccagaggcgttgcg\n",
      "ta\n",
      ">G000299455|535426|536523|536092|536154|+\n",
      "atacacccaatgtgaagtggtaaaggccgagaatggcaccatttcgccgttggcgctgcg\n",
      "ga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1241906, 1253284)), (1242611, 1242664), '+'), (('G000006925', (3637330, 3640023)), (3639874, 3639927), '-'), (('G000299455', (262744, 269475)), (267647, 267700), '+'), (('G000299455', (2983402, 3007859)), (3007100, 3007153), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1108\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1241906|1253284|1242611|1242664|+\n",
      "-acgccattggtccagccaaagccatcctgtaatggatattcgccaccgccgcc\n",
      ">G000006925|3637330|3640023|3639874|3639927|-\n",
      "cacaccgttagtccagccaaacccatcctgcaacggatactcgccaccgccgc-\n",
      ">G000299455|262744|269475|267647|267700|+\n",
      "-acaccgttagtccagccaaacccatcctgcaacggatactcgccaccgccgcc\n",
      ">G000299455|2983402|3007859|3007100|3007153|-\n",
      "cacgccattggtccagccaaagccatcctgtaatggatattcgccaccgccgc-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1063049, 1063191)), (1063087, 1063191), '+'), (('G000006925', (1090935, 1091083)), (1090979, 1091083), '+'), (('G000006925', (1804109, 1804251)), (1804109, 1804213), '-'), (('G000006925', (1815261, 1815403)), (1815261, 1815365), '-'), (('G000006925', (4818440, 4818585)), (4818481, 4818585), '+'), (('G000299455', (5328484, 5333124)), (5333020, 5333124), '+')]\n",
      ">G000006925|1063049|1063191|1063087|1063191|+\n",
      "-acaaaacggatgtttcttcaacaccacttttgctccatatgaacggaaccgacgattaa\n",
      "actggatggctctgattgattcagggtatgaatggcggttttttg\n",
      ">G000006925|1090935|1091083|1090979|1091083|+\n",
      "-acaaaacggatgtttcttcaacaccacttttgctccatatgaacggaaccgacgattaa\n",
      "actggatggctctgattgattcagggtatgaatggcggttttttg\n",
      ">G000006925|1804109|1804251|1804109|1804213|-\n",
      "cacaaaacggatgtttcttcaacaccacttttgctccatatgaacggaaccgacgattaa\n",
      "actggatggctctgattgattcagggtatgaatggcggtttttt-\n",
      ">G000006925|1815261|1815403|1815261|1815365|-\n",
      "cacaaaacggatgtttcttcaacaccacttttgctccatatgaacggaaccgacgattaa\n",
      "actggatggctctgattgattcagggtatgaatggcggtttttt-\n",
      ">G000006925|4818440|4818585|4818481|4818585|+\n",
      "-acaaaacggatgtttcttcaacaccacttttgctccatatgaacggaaccgacgattaa\n",
      "actggatggctctgattgattcagggtatgaatggcggttttttg\n",
      ">G000299455|5328484|5333124|5333020|5333124|+\n",
      "-ataaaatgaatgtttcttcaatatcccttgtgctccatatgaacggaaacgacgattaa\n",
      "actggatggcattcattgattcagggtatgaatggcggttttttg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (730183, 730452), '+'), (('G000299455', (2174641, 2181396)), (2178579, 2178848), '+'), (('G000299455', (2910238, 2916987)), (2912780, 2913049), '-')]\n",
      ">G000006925|726235|732999|730183|730452|+\n",
      "-aatggcggacactcaggtacggggcgtactgctggacattgacagttctggcgggcagg\n",
      "ccaccggcgcgtttgactgcgctgacatgatttaccgtctccggcagctgaagcctgtct\n",
      "gggcactgtgcaatgacacggcctgttcagccgccatgttactggcatcggcctgctccc\n",
      "gacggctggttacccagacatcccgtatcggctccattggggtgatgatgggccatgtca\n",
      "gctatgccggtcatctggcacaggccggtg\n",
      ">G000299455|2174641|2181396|2178579|2178848|+\n",
      "-aatggcagatagccaggtgcggggcgtactgctggacattgacagtccgggcgggcagg\n",
      "ccgccggtgcgtttgactgtgctgacatgatttaccgcctccgtcagcagaagccggtct\n",
      "gggcactgtgcaatgacacggcctgttctgcagccatgctgctggcgtcggcctgctccc\n",
      "gacggctggttacccagacatcccgtatcggctccattggtgtgatgatgagccatgtca\n",
      "gctatgccggtaatctggcgcaggccgggg\n",
      ">G000299455|2910238|2916987|2912780|2913049|-\n",
      "caatggcagatagccaggtgcggggcgtactgctggacattgacagtccgggcgggcagg\n",
      "ccgccggcgcgtttgactgcgctgacatgatttaccgcctccgtcagcagaagccggtct\n",
      "gggcactgtgcaatgacacggcctgttctgcagccatgctgctggcgtcggcctgctccc\n",
      "gacggctggttacccagacatcccgtatcggctccattggtgtgatgatgagccatgtca\n",
      "gctatgccggtaatctggcgcaggccggg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (386882, 411070)), (407937, 408022), '+'), (('G000006925', (2669115, 2684581)), (2671947, 2672032), '-'), (('G000299455', (1345136, 1375168)), (1372250, 1372335), '+'), (('G000299455', (3918464, 3964500)), (3943354, 3943439), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1172\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1172\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|386882|411070|407937|408022|+\n",
      "-ccggaaaaattggcgacggcaaaatcttcgtcgctgaattgcaacgcgtcattcgtatt\n",
      "cgtaccggcgaagccgacgaagcggc\n",
      ">G000006925|2669115|2684581|2671947|2672032|-\n",
      "accggcaaaatcggtgacggtaaaatcttcgtctttgacgtggctcgggtcattcgcatc\n",
      "cgtaccggtgaagaggacgacgcgg-\n",
      ">G000299455|1345136|1375168|1372250|1372335|+\n",
      "-ccggcaaaatcggtgacggtaaaatcttcgtctttgacgtggcacgggtcattcgcatc\n",
      "cgtaccggtgaagaggacgacgcggc\n",
      ">G000299455|3918464|3964500|3943354|3943439|-\n",
      "accggaaaaattggcgacggcaaaatcttcgtcgctgaattgcaacgcgtcattcgtatt\n",
      "cgtaccggcgaagccgacgaagcgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3065287, 3076770)), (3075812, 3075876), '+'), (('G000299455', (64704, 65026)), (64962, 65026), '+'), (('G000299455', (874911, 875768)), (875704, 875768), '-')]\n",
      ">G000006925|3065287|3076770|3075812|3075876|+\n",
      "-aagtctggctggtagctgtaaacagacttcgtacataccgctctgcaataaatccccgt\n",
      "accac\n",
      ">G000299455|64704|65026|64962|65026|+\n",
      "-aagtctggctggtagctgtaaacagacctcgtacataccgctctgcaataaatccccgt\n",
      "agcac\n",
      ">G000299455|874911|875768|875704|875768|-\n",
      "caaatctggctggtatctgtaaacatacttcgtacatgccgctctgcaataaatccccgt\n",
      "gcca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2479150, 2517237)), (2503563, 2503593), '-'), (('G000299455', (1535847, 1550961)), (1549499, 1549529), '+'), (('G000299455', (4201092, 4201122)), (4201092, 4201122), '-')]\n",
      ">G000006925|2479150|2517237|2503563|2503593|-\n",
      "gtctgctgttgttgccaggccgcttccagt-\n",
      ">G000299455|1535847|1550961|1549499|1549529|+\n",
      "-tctgctgttgttgccaggccgcttccagtt\n",
      ">G000299455|4201092|4201122|4201092|4201122|-\n",
      "gtctgctgctggtgccaggccgcttccagt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3322761, 3361739)), (3352340, 3352388), '+'), (('G000299455', (540757, 615358)), (585716, 585764), '-'), (('G000299455', (1029419, 1029576)), (1029528, 1029576), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3322761|3361739|3352340|3352388|+\n",
      "-gggttcccgccgcgaagtgaaaaacgcgcgggaagaaggcgtagagtt\n",
      ">G000299455|540757|615358|585716|585764|-\n",
      "cgggttcccgccgcgaagtgaaaaacgcgcgggaagaaggcgtagagt-\n",
      ">G000299455|1029419|1029576|1029528|1029576|+\n",
      "-gggctcgcgcaaagaggtggttaatgcgcgcgaggaaggtgtcgagtt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (35255, 35310), '-'), (('G000299455', (2775324, 2775379)), (2775324, 2775379), '-'), (('G000299455', (4406425, 4422836)), (4412743, 4412798), '+')]\n",
      ">G000006925|25217|64488|35255|35310|-\n",
      "aaaccagttatcgcagctgtgaacggctacgcctttggcggcggtttcgaactgg-\n",
      ">G000299455|2775324|2775379|2775324|2775379|-\n",
      "aaaccacttatcgcagccgtcaatggttacgcgcttggggcgggttgcgaactgg-\n",
      ">G000299455|4406425|4422836|4412743|4412798|+\n",
      "-aaccggttatcgcagctgtgaacggctatgcctttggcggcggtttcgaactggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (237115, 237939)), (237125, 237159), '+'), (('G000299455', (2918828, 2919155)), (2918828, 2918861), '+'), (('G000299455', (3606434, 3606478)), (3606444, 3606478), '+')]\n",
      ">G000006925|237115|237939|237125|237159|+\n",
      "tatcaaacccacccgtagataggctttgtaatgg\n",
      ">G000299455|2918828|2919155|2918828|2918861|+\n",
      "tatcaagccc-cccgcagatagactttgtaatgg\n",
      ">G000299455|3606434|3606478|3606444|3606478|+\n",
      "tatcaagcccacccgtagatgggctttgtaatgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189492, 1208094)), (1204325, 1204679), '+'), (('G000299455', (1288904, 1289259)), (1288904, 1289259), '+'), (('G000299455', (3026184, 3035686)), (3029412, 3029766), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1189492|1208094|1204325|1204679|+\n",
      "-atgttgtaaaggtaagataatctgatttatcaatattattgtgtgatttttatgtgagc\n",
      "agaagatattcatcagcaacgattacattagtcattttatttttccgacggcctcattgt\n",
      "ctaaagataagcgtacgacagtattatctgaaaagagtga-ttttttatccaactacact\n",
      "tcagcgcactgcgtgtaaaaaatgcctctttcttatgcgggatatcatcatttcatcatg\n",
      "atgtctttggtgagcggtgaacacaatacacctgcgctgtctcttcaggatgaatcccct\n",
      "ggtctggtgcctgcgggctgatgttgcagcagagcttaggtcacttagacgctact\n",
      ">G000299455|1288904|1289259|1288904|1289259|+\n",
      "-atgctgtaaaagtaaaacaatctgcgatctcaaagttattgtgtgatttttatgtgagc\n",
      "agaagatattcatcagaaacgattatgtaaatcattttattttgccaacggcctcattgt\n",
      "cgaaagaaaagaatacgcccgtatttcctgaatggattgacttttttatccaaccacact\n",
      "tcagcgcactgcgtttaaaaaatgcctcattcttatgcggaatatcatcatttcatcatg\n",
      "atgtctttgatgagcggtgaacacaatacacttgcgctgtctcttcaggatgaatcccct\n",
      "ggtctggtgcctgtgggctgatgttgcagcaaagctaaggtcgcttaaacgctact\n",
      ">G000299455|3026184|3035686|3029412|3029766|-\n",
      "aatgttgtaaaggtaagataatctgatttatcaatattattgtgtgatttttatgtgagc\n",
      "agaagatattcatcagcaacgattacattagtcattttattttgccgacggcctcattgt\n",
      "cgaaagataagcgtacgacagtattatctgaaaagagtga-ttttttatccaactacact\n",
      "tcagcgcactgcgtgtaaaaaatgcctctttcttatgcgggatatcatcatttcatcatg\n",
      "atgtctttggtgagcggtgaacacaatacacctgcgctgtctcttcaggatgaatcccct\n",
      "ggtctggtgcctgcgggctgatgttgcagcagagcttaggtcacttagacgctac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (739473, 745780)), (741036, 741221), '+'), (('G000006925', (1921807, 1928114)), (1926366, 1926551), '-'), (('G000299455', (2902162, 2904160)), (2902412, 2902597), '-')]\n",
      ">G000006925|739473|745780|741036|741221|+\n",
      "-tgatgcaggtgaaaccgaattgtctgcccgtcttaatgagccgctggcaaatggtgcag\n",
      "tgatccacatagtaccgcgtctggtgggagctaaaagtggcggtgtgtttcaggcggtgc\n",
      "tgggggcagctgttatggcggttgctatatggatgccgggggtaggaattatggcgagta\n",
      "atctgc\n",
      ">G000006925|1921807|1928114|1926366|1926551|-\n",
      "gtgatgcaggtgaaaccgaattgtctgcccgtcttaatgagccgctggcaaatggtgcag\n",
      "tgatccacatagtaccgcgtctggtgggagctaaaagtggcggtgtgtttcaggcggtgc\n",
      "tgggggcagctgttatggcggttgctatatggatgccgggggtaggaattatggcgagta\n",
      "atctg-\n",
      ">G000299455|2902162|2904160|2902412|2902597|-\n",
      "gggatgcaggagaaaccgaattatcatcccgtcttaatgagccgctggcaaatggtgccg\n",
      "tgatccacatcgtgccgcgtcttgcgggagcaaaaagtggcggtgtgtttcaggcggtgc\n",
      "tgggtgcggcgttgattgctacggcaatctggatgccggggatcagtatcgctttcagtg\n",
      "acatt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3807702, 3807962), '-'), (('G000299455', (91523, 102755)), (92417, 92677), '+'), (('G000299455', (4142626, 4142886)), (4142626, 4142886), '+')]\n",
      ">G000006925|3788320|3808082|3807702|3807962|-\n",
      "gagtgaatcaatgtaatcagcgtaccattgcatcatttcacggcggccatcgagatacag\n",
      "ggcatgattataggtacccctgatggaatttttatccacatgagctaactgcatttcaat\n",
      "ccaggcggaattaaagccttgttcatgcagaattgtactcatggtgtgtctgaaaccgtg\n",
      "ccctgtgagtcttcccttatatccaagtaattcaattacttggttaatgctttccttact\n",
      "aattggtttgcggggatcat-\n",
      ">G000299455|91523|102755|92417|92677|+\n",
      "-agtgaatcaatgtaatcagcgtaccattgcatcatttcacggcggccatcgagatacag\n",
      "ggcatgattataggtacccctgatggaatttttatccacatgagctaactgcatttcaat\n",
      "ccaggcggaattaaagccttgttcatgcagaattgtactcatggtgtgtctgaaaccgtg\n",
      "ccctgtgagtcttcccttatatccaagtaattcaattacttggttaatgctttccttact\n",
      "aattggtttgcggggatcatt\n",
      ">G000299455|4142626|4142886|4142626|4142886|+\n",
      "-agcgaatccatataatcggcgtaccactgaagcatttcccgccgtccatccagatattg\n",
      "cgcatggttgtatgtgccacgaattgagtttttatcgacgtgtgcgagctgcgtctctat\n",
      "ccacgcggtgttatagccctgctcatgcaaaatggtgctcatggtgtgccggaatccatg\n",
      "accagtaacctttccgttatagccgatgcgtttaaatacttggtttatgctagcctcgct\n",
      "cattgttttccttggatcatt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1409317, 1409460)), (1409335, 1409396), '+'), (('G000006925', (2236142, 2236293)), (2236166, 2236226), '-'), (('G000299455', (2148729, 2148865)), (2148795, 2148856), '+'), (('G000299455', (3316701, 3316869)), (3316765, 3316826), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1409317|1409460|1409335|1409396|+\n",
      "-gccccatctggctggctataaatgccgggttcgctcctgccgtcaacagccagcaggca\n",
      "aa\n",
      ">G000006925|2236142|2236293|2236166|2236226|-\n",
      "-ggcccatctggcttgcgataaaactaggattagctcctgcggcaagtgaccagcatgca\n",
      "t-\n",
      ">G000299455|2148729|2148865|2148795|2148856|+\n",
      "-ggcccatctggctggcaatgaagcttgggtttgctccggcagataatgaccagcacgca\n",
      "ta\n",
      ">G000299455|3316701|3316869|3316765|3316826|-\n",
      "tgccccatctgtgatgcaataaatgtcgggtttgctccggcagataaagcccagcacgca\n",
      "t-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (808433, 825842)), (813277, 813314), '-'), (('G000299455', (2678329, 2681788)), (2680848, 2680885), '-'), (('G000299455', (3480865, 3520974)), (3516177, 3516214), '+')]\n",
      ">G000006925|808433|825842|813277|813314|-\n",
      "ttcaacaatttgcccgagatacatcaccgccacacga-\n",
      ">G000299455|2678329|2681788|2680848|2680885|-\n",
      "ttctacaatctgcccgagatacatcaccgctacccga-\n",
      ">G000299455|3480865|3520974|3516177|3516214|+\n",
      "-tcaacaatttgtccgagatacatcaccgccacacgat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4679316, 4679373)), (4679316, 4679373), '+'), (('G000299455', (4572250, 4572307)), (4572250, 4572307), '-'), (('G000299455', (4749355, 4749412)), (4749355, 4749412), '+'), (('G000299455', (5334822, 5334879)), (5334822, 5334879), '-')]\n",
      ">G000006925|4679316|4679373|4679316|4679373|+\n",
      "-caagtaatccggggcccgctatgccgcgctctatcgggcgggacggggctggcgctt\n",
      ">G000299455|4572250|4572307|4572250|4572307|-\n",
      "gcaagtaatccggggcccgcgataccacgctctatcgggcgggacggcgccggtgct-\n",
      ">G000299455|4749355|4749412|4749355|4749412|+\n",
      "-caagtaaccccgggcccgcgataccacgctctatcggacgggatggtgccggtgctt\n",
      ">G000299455|5334822|5334879|5334822|5334879|-\n",
      "gcaagtaaccccgggcccgcgataccacgctctatcggacgggatggtgccggtgct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (39758, 39817), '-'), (('G000299455', (2411934, 2411993)), (2411934, 2411993), '-'), (('G000299455', (4406425, 4422836)), (4408236, 4408295), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1116\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|25217|64488|39758|39817|-\n",
      "gatggattttaatttaaatgatgagcaggaactgtttgtcgccggtatccgcgaactga-\n",
      ">G000299455|2411934|2411993|2411934|2411993|-\n",
      "gatggatttttctttaactgaagaacaagaactgctgctggccagtattcgcgaactga-\n",
      ">G000299455|4406425|4422836|4408236|4408295|+\n",
      "-atggattttaatttaaatgatgagcaggaactgtttgtcgccggtatccgcgaactgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2142244, 2142313), '-'), (('G000299455', (1911336, 1921536)), (1918067, 1918136), '+'), (('G000299455', (3326617, 3362069)), (3338319, 3338388), '+')]\n",
      ">G000006925|2138845|2144864|2142244|2142313|-\n",
      "tctgggatcacccggaactcaccacgccagctggtcagtaccgcagctccagcgacaccg\n",
      "gcaactggg-\n",
      ">G000299455|1911336|1921536|1918067|1918136|+\n",
      "tctgggatcacccggaactcaccacgccagccggtcagtaccgcagctccagcgacaccg\n",
      "gcaactggg-\n",
      ">G000299455|3326617|3362069|3338319|3338388|+\n",
      "-ctgggatcacccggaactaaccacccccgcagggcaataccgtagcgccagtgataccg\n",
      "gcaactgggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1425133, 1465118)), (1442091, 1442126), '-'), (('G000299455', (2199778, 2263225)), (2240965, 2241000), '-'), (('G000299455', (3738235, 3772027)), (3768895, 3768930), '+')]\n",
      ">G000006925|1425133|1465118|1442091|1442126|-\n",
      "agactcgttgaaccacttaacctgacctttaatct-\n",
      ">G000299455|2199778|2263225|2240965|2241000|-\n",
      "agactcgttgaaccacttaacctgacctttaatct-\n",
      ">G000299455|3738235|3772027|3768895|3768930|+\n",
      "-gactcattaaaccacttaacgttacctttaatctt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (732147, 732305), '+'), (('G000299455', (2174641, 2181396)), (2180543, 2180701), '+'), (('G000299455', (2910238, 2916987)), (2910927, 2911085), '-'), (('G000299455', (3598648, 3598806)), (3598648, 3598806), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|732147|732305|+\n",
      "-aagagcacgccattgtccaggtggaagagatgcaggcggtgaatgccgtgctgtatggc\n",
      "aaatacaccatggaaggagagcagtttgaaactgttgaggtcgattttggccggtcggtg\n",
      "ggaaataacattacgcaggcaggagggacggagtggagc\n",
      ">G000299455|2174641|2181396|2180543|2180701|+\n",
      "-aagagcacgctattgtccaggtggaagaaatgcaggcggtgaatgccgtgctgtatggc\n",
      "aaatacaccatggaaggggaacagtttgacacggtcgaggtggatttcggacgctctgaa\n",
      "ggaaataacattgagcaggccgacggtaaaaaatggtct\n",
      ">G000299455|2910238|2916987|2910927|2911085|-\n",
      "gaagagcacgctattgtgcaggtggaagaaatgcaggcggtgaatgccgtgctgtatggc\n",
      "aaatatacgatggaaggagaccagttcgagaaaattgaagtcgattttggccgatcgacg\n",
      "aagaataacatcatacaaggtagcggtaaggagtggtc-\n",
      ">G000299455|3598648|3598806|3598648|3598806|-\n",
      "gaagagctggccattgctcaggtcgaagagatgcaggcagtttctgccgtgctcaagggc\n",
      "aaatacaccatgaccggtgaagccttcgatccggttgaggtggatatgggccgcagtgcg\n",
      "gcgaacaacatcacgcagtccggcggcacggagtggag-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3040656, 3065083)), (3058654, 3058674), '+'), (('G000299455', (915885, 916237)), (916217, 916237), '+'), (('G000299455', (4713031, 4748324)), (4741895, 4741915), '+'), (('G000299455', (4768323, 4769107)), (4768654, 4768674), '+'), (('G000299455', (4769783, 4769911)), (4769851, 4769871), '+'), (('G000299455', (5335900, 5336291)), (5335940, 5335960), '-')]\n",
      ">G000006925|3040656|3065083|3058654|3058674|+\n",
      "-agcagaagcaaaattaaacc\n",
      ">G000299455|915885|916237|916217|916237|+\n",
      "-agcagaagcaaaattaaacc\n",
      ">G000299455|4713031|4748324|4741895|4741915|+\n",
      "-agcagaagcaaaattaaacc\n",
      ">G000299455|4768323|4769107|4768654|4768674|+\n",
      "-agcagaagcaaaattatacc\n",
      ">G000299455|4769783|4769911|4769851|4769871|+\n",
      "-agcagaagcaaaattaaacc\n",
      ">G000299455|5335900|5336291|5335940|5335960|-\n",
      "cagcagaagcaaaattaaac-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4620693, 4621057)), (4620973, 4621057), '+'), (('G000006925', (4791871, 4792235)), (4792151, 4792235), '+'), (('G000006925', (4794974, 4795158)), (4794974, 4795057), '-'), (('G000299455', (5313369, 5315826)), (5313649, 5313733), '+')]\n",
      ">G000006925|4620693|4621057|4620973|4621057|+\n",
      "aagtgcctccacagtacctgtcgtacttactgaaagcaccgttatgcccaaactcccggt\n",
      "ggtaaaaaaacgtccccggcggcc\n",
      ">G000006925|4791871|4792235|4792151|4792235|+\n",
      "aagtgcctccacagtacctgtcgtacttactgaaagcaccgttatgcccaaactcccggt\n",
      "ggtaaaaaaacgtccccggcggcc\n",
      ">G000006925|4794974|4795158|4794974|4795057|-\n",
      "aagtgcctccacagtacctgtcgtacttactgaaagcaccgttatgcccaaactcccggt\n",
      "ggtaaaaaaacgtccccggcggc-\n",
      ">G000299455|5313369|5315826|5313649|5313733|+\n",
      "acaattttccacggtaccagtcgtacgtcctgaaagcaccgttatatccgaagcccccgt\n",
      "ggtaaaaaaacgtccccggcggcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3868997, 3883532)), (3874843, 3874877), '+'), (('G000299455', (1, 26978)), (5849, 5883), '+'), (('G000299455', (767773, 781429)), (779352, 779386), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3868997|3883532|3874843|3874877|+\n",
      "ccagcgttataaaggtctgggcgagatgaacccg\n",
      ">G000299455|1|26978|5849|5883|+\n",
      "ccagcgttataaaggtctgggcgagatgaacccg\n",
      ">G000299455|767773|781429|779352|779386|+\n",
      "ccagcgttttaaaggtctgggggaaatgaacccg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (64491, 216702)), (119136, 119166), '+'), (('G000006925', (3969227, 3993100)), (3990097, 3990127), '-'), (('G000299455', (3819953, 3834467)), (3825277, 3825307), '+'), (('G000299455', (4300454, 4333834)), (4329340, 4329370), '-'), (('G000299455', (5156132, 5190970)), (5170070, 5170100), '+')]\n",
      ">G000006925|64491|216702|119136|119166|+\n",
      "-cttctgctgcggtgatccccaccagttcca\n",
      ">G000006925|3969227|3993100|3990097|3990127|-\n",
      "gcttcaccggcagtaatgccaatcagctcc-\n",
      ">G000299455|3819953|3834467|3825277|3825307|+\n",
      "-cttcagcggcagtaatcccaatcagctcca\n",
      ">G000299455|4300454|4333834|4329340|4329370|-\n",
      "gcttctgctgcggtgatccccaccagttcc-\n",
      ">G000299455|5156132|5190970|5170070|5170100|+\n",
      "-cttcaccggcagtaatgccaatcagctcca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4633849, 4633984)), (4633849, 4633984), '+'), (('G000299455', (4572024, 4572159)), (4572024, 4572159), '+'), (('G000299455', (4749503, 4749638)), (4749503, 4749638), '-'), (('G000299455', (5334596, 5334731)), (5334596, 5334731), '+')]\n",
      ">G000006925|4633849|4633984|4633849|4633984|+\n",
      "-cccgcctgttgcgacagtcacgtcatcgccgtccgttaccggaacatgtttcccgcgaa\n",
      "atacaccatctggagcctgaagaaagctgttgtccggagtgtggtggtgaactggattat\n",
      "tcgggggaaatcagcg\n",
      ">G000299455|4572024|4572159|4572024|4572159|+\n",
      "-cccgccagttgcgacagtcgcgccatcgtcgtccgttaccggcacaccttccccgtgaa\n",
      "atacaccgcctggagccagaagaaagctgttgcccggagtgtggcggtaagctggattat\n",
      "ctgggggaagtcagcg\n",
      ">G000299455|4749503|4749638|4749503|4749638|-\n",
      "ccccgccagttgcgacagtcgcgccatcgtcgtccgttaccggcacatctcccccgtgaa\n",
      "atataccgcctggagcctgaagaaagctgttgcccggagtgtggcggtgagctggattat\n",
      "ctgggggaagtcagc-\n",
      ">G000299455|5334596|5334731|5334596|5334731|+\n",
      "-cccgccagttgcgacagtcgcgccatcgtcgtccgttaccggcacatctcccccgtgaa\n",
      "atataccgcctggagcctgaagaaagctgttgcccggagtgtggcggtgagctggattat\n",
      "ctgggggaagtcagcg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (789803, 808424)), (799225, 799291), '-'), (('G000006925', (2069615, 2084300)), (2080093, 2080159), '-'), (('G000299455', (2006434, 2017864)), (2009312, 2009378), '+'), (('G000299455', (3528972, 3554217)), (3531383, 3531449), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1062\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1134\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1134\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|789803|808424|799225|799291|-\n",
      "gttccggcaggtccggataacccgatgggtctgtatgcactctacatcggtcgcctgtat\n",
      "gctatc-\n",
      ">G000006925|2069615|2084300|2080093|2080159|-\n",
      "gttcctgcggggcccgataaccccatggggctgtacgcgatttatattggcaggttgtat\n",
      "gccatc-\n",
      ">G000299455|2006434|2017864|2009312|2009378|+\n",
      "-ttcctgcggggcccgataaccccatggggctgtacgcgatttatattggcaggttgtat\n",
      "gccatcc\n",
      ">G000299455|3528972|3554217|3531383|3531449|+\n",
      "-ttccggcaggtccggataacccgatggggctgtatgcactctacatcggtcgcctgtac\n",
      "gctatcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3027652, 3027805), '+'), (('G000299455', (956352, 970820)), (965244, 965397), '-'), (('G000299455', (1078152, 1154001)), (1084712, 1084865), '+')]\n",
      ">G000006925|3016790|3036697|3027652|3027805|+\n",
      "-gatgtttattgtcggttttgccatgagtgccggtccgctgatttgggtactgtgctccg\n",
      "aaattcagccgctgaaaggccgcgattttggcatcacctgctccaccgccaccaactgga\n",
      "ttgccaacatgatcgttggcgcaacgttcctgac\n",
      ">G000299455|956352|970820|965244|965397|-\n",
      "tgatgtttattgtcggttttgccatgagtgccggtccgctgatttgggtactgtgctccg\n",
      "aaattcagccgctgaaaggccgcgattttggcatcacctgctccaccgccaccaactgga\n",
      "ttgccaacatgatcgttggcgcaacgttcctga-\n",
      ">G000299455|1078152|1154001|1084712|1084865|+\n",
      "-gatgtgtattgccggttatgcgatgagcgccgcgccagtggtgtggatcctgtgctctg\n",
      "aaattcagccgctgaaatgccgcgatttcggtattacctgttcgacgacgacaaactggg\n",
      "tgtcgaatatgattatcggcgcgaccttcctgac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2266523, 2266568), '-'), (('G000299455', (1752647, 1813107)), (1811644, 1811689), '+'), (('G000299455', (4037524, 4037569)), (4037524, 4037569), '-')]\n",
      ">G000006925|2242494|2346249|2266523|2266568|-\n",
      "tgaagttctcgtcattgcaggttaatccggaaagccagtacagca-\n",
      ">G000299455|1752647|1813107|1811644|1811689|+\n",
      "-gaagttctcgtcattgcaggttaatccggaaagccagtacagcac\n",
      ">G000299455|4037524|4037569|4037524|4037569|-\n",
      "tgaaattctgctcgttgcaggtcaggcccgaaagccagtacagca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1916373, 1917424)), (1917012, 1917265), '+'), (('G000006925', (2227521, 2228997)), (2227521, 2227775), '+'), (('G000299455', (2196403, 2197468)), (2196576, 2196829), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1916373|1917424|1917012|1917265|+\n",
      "-cactgtttatccatacagttaaaaataata-ctgtataaaaacacagtgtagagggatt\n",
      "tttatgcgttttgaaatctgcatagccaaagagaaaatgactaaaatgccaaccggtgct\n",
      "gtggatgcgttaaaggaagaattaacccgacgcatcagtaaacgttatgacgatgtagag\n",
      "gtgatcgtaaaagccaccagcaacgatggcctttctgttacgcgcaccgccgataaagat\n",
      "tcagctaaaactttt\n",
      ">G000006925|2227521|2228997|2227521|2227775|+\n",
      "-cactggttatccatacagataaaaataacacctgtacataactacagttacaggagggt\n",
      "gttatgcgtgttgagatctgtatcgctaaagaaaaaatcactaaaatgccaaacggtgct\n",
      "gtggatgcgttaaaggaagaattaacccgacgcatcagtaaacgttatgacgatgtagag\n",
      "gtgatcgtaaaagccaccagcaacgatggcctttctgttacacgcaccgcagataaggat\n",
      "tctgcaaaaacattt\n",
      ">G000299455|2196403|2197468|2196576|2196829|-\n",
      "acactgtttatccatacagttaaaaatgata-ctgtatacaaacacagtgtagagggact\n",
      "tttatgcgtattgaaatctgcatagccaaagaaaaaatgactaaaatgccaaccggtgct\n",
      "gtggatgcgttaaaggaagaattaacccgacgcatcagtaaacgttatgacgatgtagag\n",
      "gtgatcgtaaaagctaccagcaacgatggcctttctgttacgcgcaccgccgataaagat\n",
      "tcagctaaaacttt-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (734657, 734866)), (734658, 734854), '+'), (('G000006925', (1932721, 1932930)), (1932733, 1932929), '-'), (('G000299455', (2183018, 2183226)), (2183018, 2183214), '+'), (('G000299455', (2908419, 2908627)), (2908431, 2908627), '-'), (('G000299455', (3596117, 3596314)), (3596117, 3596313), '-')]\n",
      ">G000006925|734657|734866|734658|734854|+\n",
      "-aaggtgcggggacgacgttctggatgtataccggcaacggcgatgcgtttgcgaaccct\n",
      "ttgtcggacactgactggctgcgtcttgcgatggtgaaggatctgcaacctggcgaaatg\n",
      "accgctgatgcagaagatgacacttatctcgatgataaagatgcagactggaaaacgaca\n",
      "acccaggggcagaaatc\n",
      ">G000006925|1932721|1932930|1932733|1932929|-\n",
      "aaaggtgcggggacaacgttctggatgtataccggcaacggcgatgcgtttgcgaaccct\n",
      "ttgtcggacactgactggctgcgtcttgcgatggtgaaggatctgcaacctggcgaaatg\n",
      "accgctgatgcagaagatgacacttatctcgatgatgaagatgcagactggaaaacgaca\n",
      "acccaggggcagaaat-\n",
      ">G000299455|2183018|2183226|2183018|2183214|+\n",
      "-aaggtgccggtaccactctgtgggtttacaacggcaagggtgatgcttatgcaaacccg\n",
      "ttgtcagacgatgactggcagcgactggctaaggtgaaggatctgacgccgggcgagatg\n",
      "acggcagaatcctacgatgataactacctggatgatgaagacgcggactggaccgcgacc\n",
      "gggcaggggcagaaatc\n",
      ">G000299455|2908419|2908627|2908431|2908627|-\n",
      "aaaggtgccggtaccactctgtgggtttacaacggcaagggtgatgcttatgcaaacccg\n",
      "ttgtcagacgctgactggcagcgactggctaaggtgaaggatctgacgccgggcgagatg\n",
      "acggcagaatcctacgatgataactacctggatgatgaagacgcagactggaccgcgacc\n",
      "gggcaggggcagaaat-\n",
      ">G000299455|3596117|3596314|3596117|3596313|-\n",
      "aaaggggccgggaccacactgtgggtttataaggggaacggtgacccttatgcgaacccg\n",
      "ctttcagacgttgactggtcgcgtctggctaaagttaaagacctgacgcccggcgaactg\n",
      "accgctgagtcctatgacgacagctatctcgatgatgaagatgcggactggactgcgacc\n",
      "gggcaggggcagaaat-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (267770, 272851)), (271425, 271457), '+'), (('G000006925', (3322761, 3361739)), (3336575, 3336607), '-'), (('G000299455', (540757, 615358)), (601498, 601530), '+'), (('G000299455', (4026769, 4031850)), (4030424, 4030456), '+')]\n",
      ">G000006925|267770|272851|271425|271457|+\n",
      "-aaacggttcgtcgagtaataacagctggggat\n",
      ">G000006925|3322761|3361739|3336575|3336607|-\n",
      "caaacggttcgtcgagcagaataaatttcgga-\n",
      ">G000299455|540757|615358|601498|601530|+\n",
      "-aaacggttcgtcgagcagaataaatttcggat\n",
      ">G000299455|4026769|4031850|4030424|4030456|+\n",
      "-aaacggttcgtcgagtaataacagctggggat\n",
      "\n",
      "\n",
      "36 [(('G000006925', (235782, 235958)), (235858, 235958), '+'), (('G000006925', (266530, 266681)), (266530, 266631), '-'), (('G000006925', (496041, 496215)), (496041, 496142), '-'), (('G000006925', (707347, 707521)), (707347, 707448), '-'), (('G000006925', (865364, 865540)), (865364, 865465), '-'), (('G000006925', (912667, 912841)), (912667, 912768), '-'), (('G000006925', (981203, 981377)), (981203, 981304), '-'), (('G000006925', (1008623, 1008797)), (1008696, 1008797), '+'), (('G000006925', (1097702, 1097876)), (1097775, 1097876), '+'), (('G000006925', (1209065, 1211067)), (1209065, 1209166), '-'), (('G000006925', (1393174, 1393348)), (1393174, 1393275), '-'), (('G000006925', (1418524, 1418698)), (1418524, 1418625), '-'), (('G000006925', (1500239, 1500413)), (1500312, 1500413), '+'), (('G000006925', (1618594, 1618768)), (1618667, 1618768), '+'), (('G000006925', (1696247, 1696421)), (1696247, 1696348), '-'), (('G000006925', (1807541, 1813528)), (1807541, 1807642), '-'), (('G000006925', (2036226, 2036400)), (2036226, 2036327), '-'), (('G000006925', (2347501, 2359607)), (2359501, 2359602), '+'), (('G000006925', (2690777, 2690953)), (2690851, 2690953), '+'), (('G000006925', (2770832, 2816659)), (2770832, 2770933), '-'), (('G000006925', (2964403, 2964577)), (2964403, 2964504), '-'), (('G000006925', (3079519, 3079697)), (3079595, 3079697), '+'), (('G000006925', (3146034, 3178603)), (3178502, 3178603), '+'), (('G000006925', (3282565, 3282739)), (3282565, 3282666), '-'), (('G000006925', (3513306, 3513480)), (3513378, 3513480), '+'), (('G000006925', (3604733, 3604908)), (3604806, 3604908), '+'), (('G000006925', (3815700, 3815874)), (3815700, 3815801), '-'), (('G000006925', (3831593, 3850590)), (3831593, 3831694), '-'), (('G000006925', (3914710, 3914884)), (3914782, 3914884), '+'), (('G000006925', (3994071, 3994245)), (3994071, 3994172), '-'), (('G000006925', (4117495, 4130655)), (4117495, 4117596), '-'), (('G000006925', (4256395, 4256569)), (4256395, 4256496), '-'), (('G000006925', (4360638, 4360798)), (4360696, 4360798), '+'), (('G000006925', (4695357, 4695531)), (4695357, 4695458), '-'), (('G000006925', (4778619, 4778953)), (4778619, 4778720), '-'), (('G000299455', (2004915, 2005095)), (2004920, 2005020), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1394\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1394\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 36\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 36\r",
      "   10 / 36\r",
      "   20 / 36\r",
      "   30 / 36\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 35  f\b\b\r",
      "STEP     2 / 35  f\b\b\r",
      "STEP     3 / 35  f\b\b\r",
      "STEP     4 / 35  f\b\b\r",
      "STEP     5 / 35  f\b\b\r",
      "STEP     6 / 35  f\b\b\r",
      "STEP     7 / 35  f\b\b\r",
      "STEP     8 / 35  f\b\b\r",
      "STEP     9 / 35  f\b\b\r",
      "STEP    10 / 35  f\b\b\r",
      "STEP    11 / 35  f\b\b\r",
      "STEP    12 / 35  f\b\b\r",
      "STEP    13 / 35  f\b\b\r",
      "STEP    14 / 35  f\b\b\r",
      "STEP    15 / 35  f\b\b\r",
      "STEP    16 / 35  f\b\b\r",
      "STEP    17 / 35  f\b\b\r",
      "STEP    18 / 35  f\b\b\r",
      "STEP    19 / 35  f\b\b\r",
      "STEP    20 / 35  f\b\b\r",
      "STEP    21 / 35  f\b\b\r",
      "STEP    22 / 35  f\b\b\r",
      "STEP    23 / 35  f\b\b\r",
      "STEP    24 / 35  f\b\b\r",
      "STEP    25 / 35  f\b\b\r",
      "STEP    26 / 35  f\b\b\r",
      "STEP    27 / 35  f\b\b\r",
      "STEP    28 / 35  f\b\b\r",
      "STEP    29 / 35  f\b\b\r",
      "STEP    30 / 35  f\b\b\r",
      "STEP    31 / 35  f\b\b\r",
      "STEP    32 / 35  f\b\b\r",
      "STEP    33 / 35  f\b\b\r",
      "STEP    34 / 35  f\b\b\r",
      "STEP    35 / 35  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|235782|235958|235858|235958|+\n",
      "--tgtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|266530|266681|266530|266631|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|496041|496215|496041|496142|-\n",
      "cttgtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|707347|707521|707347|707448|-\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|865364|865540|865364|865465|-\n",
      "cttgtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|912667|912841|912667|912768|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|981203|981377|981203|981304|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|1008623|1008797|1008696|1008797|+\n",
      "-tagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|1097702|1097876|1097775|1097876|+\n",
      "-tagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|1209065|1211067|1209065|1209166|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|1393174|1393348|1393174|1393275|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|1418524|1418698|1418524|1418625|-\n",
      "cttgtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|1500239|1500413|1500312|1500413|+\n",
      "-tagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|1618594|1618768|1618667|1618768|+\n",
      "-tagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|1696247|1696421|1696247|1696348|-\n",
      "cttgtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|1807541|1813528|1807541|1807642|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|2036226|2036400|2036226|2036327|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|2347501|2359607|2359501|2359602|+\n",
      "-tagtcattggagagcagatgattgatgtcttaggaccggagaaacgcagacggcgtact\n",
      "acacaggaaaagatcgctattgttcagcagagttttgaaccg\n",
      ">G000006925|2690777|2690953|2690851|2690953|+\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|2770832|2816659|2770832|2770933|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|2964403|2964577|2964403|2964504|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|3079519|3079697|3079595|3079697|+\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|3146034|3178603|3178502|3178603|+\n",
      "-tagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|3282565|3282739|3282565|3282666|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|3513306|3513480|3513378|3513480|+\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|3604733|3604908|3604806|3604908|+\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|3815700|3815874|3815700|3815801|-\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|3831593|3850590|3831593|3831694|-\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|3914710|3914884|3914782|3914884|+\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|3994071|3994245|3994071|3994172|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|4117495|4130655|4117495|4117596|-\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|4256395|4256569|4256395|4256496|-\n",
      "ctagtcattggagtacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|4360638|4360798|4360696|4360798|+\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaaccg\n",
      ">G000006925|4695357|4695531|4695357|4695458|-\n",
      "ctagtcattggagaacagattattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000006925|4778619|4778953|4778619|4778720|-\n",
      "ctagtcattggagaacagatgattgatgtcttagggccggagaaacgcagacggcgtacc\n",
      "acacaggaaaagatcgcaattgttcagcagagctttgaacc-\n",
      ">G000299455|2004915|2005095|2004920|2005020|-\n",
      "gtagtcattagtgaacagataattgttaacttagaaccagtaaaatgcaaacgacgcacc\n",
      "acgctgggaaa-atcgctgctattcagcagagttttgtacc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2696888, 2728352)), (2710559, 2710609), '-'), (('G000299455', (1345136, 1375168)), (1345999, 1346049), '+'), (('G000299455', (3554411, 3565644)), (3555102, 3555152), '+')]\n",
      ">G000006925|2696888|2728352|2710559|2710609|-\n",
      "aagcccatatccagcatacggtctgcttcgtcgaggatcagcgtttcaac-\n",
      ">G000299455|1345136|1375168|1345999|1346049|+\n",
      "-agcccatatccagcatacggtctgcttcgtcgaggatcagcgtttcaacc\n",
      ">G000299455|3554411|3565644|3555102|3555152|+\n",
      "-agcccatgtcgagcatgcggtcagcttcatcgaggacgaggatttcaacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3612083, 3637192)), (3616887, 3617013), '-'), (('G000299455', (288342, 302750)), (297823, 297949), '+'), (('G000299455', (2459722, 2494618)), (2475919, 2476045), '+')]\n",
      ">G000006925|3612083|3637192|3616887|3617013|-\n",
      "gagccgatattgatcgacatatagaacagggtgaatgcgccatcaagccgcggatctttc\n",
      "ggcggatagcacttcgaaagcaagctggctgggttagctttaaacaggccgttaccgaca\n",
      "gcgata-\n",
      ">G000299455|288342|302750|297823|297949|+\n",
      "-agccgatattgatcgacatatagaacagggtgaatgcgccatcaagccgcggatctttc\n",
      "ggcggatagcacttcgaaagcaagctggctgggttagctttaaacaggccgttaccgaca\n",
      "gcgatag\n",
      ">G000299455|2459722|2494618|2475919|2476045|+\n",
      "-agccgatgttgacggacatgtagtacatggtgaatgcaccgtccagacgcgggtcgttt\n",
      "ttctcatagcatgtagaaagcagagaagacgggttagctttaaacaggccgttaccgacc\n",
      "gcaatag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4302314, 4302355), '+'), (('G000299455', (1752647, 1813107)), (1762519, 1762560), '+'), (('G000299455', (4836149, 4864500)), (4848039, 4848080), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4288513|4315006|4302314|4302355|+\n",
      "-ggggcggtcatgaaggttcgctgctgctttgggtgctgctg\n",
      ">G000299455|1752647|1813107|1762519|1762560|+\n",
      "-ggggcgcgcatgaaggctcgctcctgctgtgggtgctgctg\n",
      ">G000299455|4836149|4864500|4848039|4848080|-\n",
      "tggggcggccatgaaggttcgctgctgctttgggtgctgct-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (386882, 411070)), (405380, 405420), '+'), (('G000299455', (3194526, 3194571)), (3194531, 3194571), '-'), (('G000299455', (3918464, 3964500)), (3945956, 3945996), '-'), (('G000299455', (4221161, 4233966)), (4227824, 4227864), '+')]\n",
      ">G000006925|386882|411070|405380|405420|+\n",
      "-tttccggtgggcaaaaacagcgtatctccattgctcgagc\n",
      ">G000299455|3194526|3194571|3194531|3194571|-\n",
      "ctttccggcggtcaaaaacagcgtatattcattgcccgag-\n",
      ">G000299455|3918464|3964500|3945956|3945996|-\n",
      "ctttccggtgggcaaaaacagcgtatctccattgctcgag-\n",
      ">G000299455|4221161|4233966|4227824|4227864|+\n",
      "-tttccggtgggcagaaacagcgtgtggcgattgcccgtgc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (539869, 540982)), (540175, 540238), '-'), (('G000006925', (4608018, 4608127)), (4608051, 4608114), '+'), (('G000299455', (3184088, 3184197)), (3184101, 3184164), '-'), (('G000299455', (3786605, 3808525)), (3790489, 3790552), '+')]\n",
      ">G000006925|539869|540982|540175|540238|-\n",
      "atgccggaaaaaacagctgagcattaccgcaataaaatcgccatttatttgcactggtat\n",
      "cag-\n",
      ">G000006925|4608018|4608127|4608051|4608114|+\n",
      "-tgccgtaaagaacggcagaacattaccggaacaagatagcggtctacctgcactggtat\n",
      "caga\n",
      ">G000299455|3184088|3184197|3184101|3184164|-\n",
      "atgccggaaagaacggcagaacattaccgtaataaaattgctgtctacctgcgctggtat\n",
      "cag-\n",
      ">G000299455|3786605|3808525|3790489|3790552|+\n",
      "-tgccggaaaaaacagctgagcattaccgcaataaaatcgccatttatttgcactggtat\n",
      "caga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (38954, 39057), '-'), (('G000299455', (2411127, 2411230)), (2411127, 2411230), '-'), (('G000299455', (4406425, 4422836)), (4408996, 4409099), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1128\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|25217|64488|38954|39057|-\n",
      "tgcgcctttgaagatgcggcgcgctacgccaaccagcgcgtgcagtttggcgaggctatt\n",
      "ggtcgtttccagttgattcaggaaaaattcgcccacatggcga-\n",
      ">G000299455|2411127|2411230|2411127|2411230|-\n",
      "tgcgcctttgaagatgccgcccgctatgccaaccaacgtatcgcttttggtaagcccatt\n",
      "ggtcataaccagatgatccaggaaaaactggcgctgatggcga-\n",
      ">G000299455|4406425|4422836|4408996|4409099|+\n",
      "-gcgcctttgaagatgcggcgcgctacgccaatcagcgcgtgcagtttggcgaggctatt\n",
      "ggtcgtttccagttgattcaggaaaaattcgcccacatggcgat\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1209065, 1211067)), (1210822, 1210871), '+'), (('G000006925', (2756889, 2766574)), (2757903, 2757952), '+'), (('G000006925', (2923039, 2950048)), (2949018, 2949067), '+'), (('G000006925', (3101814, 3103074)), (3102829, 3102878), '+'), (('G000006925', (3810378, 3811029)), (3810574, 3810623), '-'), (('G000006925', (4628976, 4629562)), (4629316, 4629365), '+'), (('G000299455', (3226712, 3229721)), (3229472, 3229521), '+'), (('G000299455', (5275700, 5279438)), (5279385, 5279434), '-')]\n",
      ">G000006925|1209065|1211067|1210822|1210871|+\n",
      "-tgagcgcaaaaggttgctgctacgataatgcctgcgtggaaagcttctt\n",
      ">G000006925|2756889|2766574|2757903|2757952|+\n",
      "-tgagcgcaaaaggttgctgctacgataatgcctgcgtggaaagcttctt\n",
      ">G000006925|2923039|2950048|2949018|2949067|+\n",
      "-tgagcgcaaaaggttgctgctacgataatgcctgcgtggaaagcttctt\n",
      ">G000006925|3101814|3103074|3102829|3102878|+\n",
      "-tgagcgcaaaaggttgctgctacgataatgcctgcgtggaaagcttctt\n",
      ">G000006925|3810378|3811029|3810574|3810623|-\n",
      "atgagtgcgaaaggctactgttatgacaatgcctgtgtggaaagcttct-\n",
      ">G000006925|4628976|4629562|4629316|4629365|+\n",
      "atgagcgcaaagggctgctgctacgataatgcctgcgtggaaagcttct-\n",
      ">G000299455|3226712|3229721|3229472|3229521|+\n",
      "-tgagcgcaaaaggttgctgctacgataatgcctgcgtggaaagcttctt\n",
      ">G000299455|5275700|5279438|5279385|5279434|-\n",
      "atgcgcgccaagggatactgctacgataatgcctgcgtggaaaggttct-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1637263, 1657511)), (1649367, 1649496), '-'), (('G000006925', (3559544, 3600494)), (3583205, 3583334), '-'), (('G000299455', (335504, 377237)), (352722, 352851), '+'), (('G000299455', (2505935, 2525742)), (2513951, 2514080), '+')]\n",
      ">G000006925|1637263|1657511|1649367|1649496|-\n",
      "ggcacggaaagcagcatacccaccggtcctaacaaccatccccaaaaaatcaacgacaaa\n",
      "aataccaccaatgtcgaaagccccagcccacgtcccatgatgcgcggctccagaatattg\n",
      "ccgaagacc-\n",
      ">G000006925|3559544|3600494|3583205|3583334|-\n",
      "ggcaccgaaagtagcatccctaccgggccgagcagccatccccagattaacaatgaaaga\n",
      "aataccaccatggtggacatccccaggcgatggcccatcatccgtggttctaaaatattg\n",
      "ccgatgacc-\n",
      ">G000299455|335504|377237|352722|352851|+\n",
      "-gcaccgaaagtagcatccctaccgggccgagcagccatccccagattaacaatgaaaga\n",
      "aataccaccatggtggacatccccaggcgatggcccatcatccgtggctctaaaatattg\n",
      "ccgatgacca\n",
      ">G000299455|2505935|2525742|2513951|2514080|+\n",
      "-gcacggaaagcagcatacccaccggtcctaacaaccatccccaaaaaatcaacgacaaa\n",
      "aataccaccaatgtggaaagccccagcccacgtcccatgatgcgcggctccagaatattg\n",
      "ccgaagacca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3807300, 3807370), '-'), (('G000299455', (91523, 102755)), (93009, 93079), '+'), (('G000299455', (4143218, 4143288)), (4143218, 4143288), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1260\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1260\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3788320|3808082|3807300|3807370|-\n",
      "agatcgggggctgggttatagtctgcccggccagtaacaattgcatagcggaacacctca\n",
      "ccacagcgct-\n",
      ">G000299455|91523|102755|93009|93079|+\n",
      "-gatcgggggctgggttatagtctgcccggccagtaacaattgcatagcggaacacctca\n",
      "ccacagcgctg\n",
      ">G000299455|4143218|4143288|4143218|4143288|+\n",
      "-gatccggtgcagggttatactcagcacgtccagtgacaattgcatagcggaaaacttcc\n",
      "ccgcatcgctg\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1209065, 1211067)), (1209915, 1210035), '-'), (('G000006925', (2756889, 2766574)), (2756996, 2757116), '-'), (('G000006925', (2923039, 2950048)), (2948111, 2948231), '-'), (('G000006925', (3101814, 3103074)), (3101922, 3102042), '-'), (('G000006925', (3811174, 3811622)), (3811399, 3811519), '+'), (('G000299455', (3226712, 3229721)), (3228565, 3228685), '-'), (('G000299455', (4548262, 4548497)), (4548271, 4548394), '+')]\n",
      ">G000006925|1209065|1211067|1209915|1210035|-\n",
      "tgctgtttactgcgccagttgtagagct---gtgattcatacaggctgagttcacgggct\n",
      "gcggcagtaacaccgatgcgttcagcaagcttcagggcttcactgcgaaattcaggcgaa\n",
      "tgc-\n",
      ">G000006925|2756889|2766574|2756996|2757116|-\n",
      "tgttgtttactgcgccagttgtagagct---gtgattcatacaggctgagttcacgggct\n",
      "gcggcagtaacaccgatgcgttcagcaagcttcagggcttcactgcgaaattcaggcgaa\n",
      "tgc-\n",
      ">G000006925|2923039|2950048|2948111|2948231|-\n",
      "tgctgtttactgcgccagttgtagagct---gtgattcatacaggctgagttcacgggct\n",
      "gcggcagtaacaccgatgcgttcagcaagcttcagggcttcactgcgaaattcaggcgaa\n",
      "tgc-\n",
      ">G000006925|3101814|3103074|3101922|3102042|-\n",
      "tgctgtttactgcgccagttgtagagct---gtgattcatacaggctgagttcacgggct\n",
      "gcggcagtaacaccgatgcgttcagcaagcttcagggcttcactgcgaaattcaggcgaa\n",
      "tgc-\n",
      ">G000006925|3811174|3811622|3811399|3811519|+\n",
      "-gctgtttactgcgccaggcataaagct---gagattcatacaggctgagttcacgggct\n",
      "gcggcggccacaccgatgcgttcagcgagtttcagggcttcgttacgaaattcaggcgta\n",
      "tgtt\n",
      ">G000299455|3226712|3229721|3228565|3228685|-\n",
      "tgctgtttactgcgccagttgtagcgct---gtgattcatacaggctgagttcacgggct\n",
      "gcggcagtaacaccgatgcgttcagcaagcttcagggcttcactgcgaaattcaggcgaa\n",
      "tgc-\n",
      ">G000299455|4548262|4548497|4548271|4548394|+\n",
      "-gctgtttactgcgacagatgtaaagctgtggtgattcatgcagactaagttcacggact\n",
      "ccggcagtaacaccgatgcattctgcaagcttcagatcttaactacgaaattcatgcgaa\n",
      "tggt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3554331, 3558215)), (3557753, 3557784), '-'), (('G000006925', (4315026, 4354132)), (4342311, 4342342), '-'), (('G000299455', (335504, 377237)), (376928, 376959), '+'), (('G000299455', (4865050, 4904182)), (4892361, 4892392), '-')]\n",
      ">G000006925|3554331|3558215|3557753|3557784|-\n",
      "tacgttactcacgatcaggttgaggcgatga-\n",
      ">G000006925|4315026|4354132|4342311|4342342|-\n",
      "tacgtcacccacgatcaggtcgaagcgatga-\n",
      ">G000299455|335504|377237|376928|376959|+\n",
      "-acgttactcacgatcaggttgaagcgatgac\n",
      ">G000299455|4865050|4904182|4892361|4892392|-\n",
      "tacgtcacccacgatcaggtcgaagcgatga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3750206, 3750250), '-'), (('G000299455', (137973, 154383)), (149560, 149604), '+'), (('G000299455', (830050, 839519)), (838009, 838053), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3736321|3788106|3750206|3750250|-\n",
      "aataatgatccacgcgatgggccacaaaccatagaagaagccat-\n",
      ">G000299455|137973|154383|149560|149604|+\n",
      "-ataatgatccacgcgatgggccacaaaccatagaagaagccata\n",
      ">G000299455|830050|839519|838009|838053|-\n",
      "gacaataatccacgcaattggccataatccataaataaagccat-\n",
      "\n",
      "\n",
      "15 [(('G000006925', (385776, 386112)), (385828, 385872), '-'), (('G000006925', (899355, 899691)), (899595, 899639), '+'), (('G000006925', (2059291, 2059627)), (2059343, 2059387), '-'), (('G000006925', (3421586, 3421922)), (3421826, 3421870), '+'), (('G000006925', (3851554, 3851890)), (3851606, 3851650), '-'), (('G000006925', (4359551, 4359859)), (4359604, 4359648), '-'), (('G000006925', (4622488, 4624460)), (4623119, 4623163), '+'), (('G000006925', (4700925, 4701241)), (4700976, 4701020), '-'), (('G000006925', (4776578, 4776894)), (4776629, 4776673), '-'), (('G000006925', (4781327, 4781663)), (4781567, 4781611), '+'), (('G000299455', (1991020, 1991316)), (1991252, 1991296), '+'), (('G000299455', (3215105, 3215383)), (3215334, 3215378), '+'), (('G000299455', (4543449, 4543753)), (4543689, 4543733), '+'), (('G000299455', (5289627, 5289931)), (5289647, 5289691), '-'), (('G000299455', (5313369, 5315826)), (5314356, 5314400), '+')]\n",
      ">G000006925|385776|386112|385828|385872|-\n",
      "-tgcacctgacgccagctcagttatccatgctactggaggggatc-\n",
      ">G000006925|899355|899691|899595|899639|+\n",
      "--gcacctgacgccagctcagttatccatgctactggaggggatcg\n",
      ">G000006925|2059291|2059627|2059343|2059387|-\n",
      "-tgcacctgacgccagctcagttatccatgctactggaggggatc-\n",
      ">G000006925|3421586|3421922|3421826|3421870|+\n",
      "--gcacctgacgccggctcagttatccatgctactggaggggatcg\n",
      ">G000006925|3851554|3851890|3851606|3851650|-\n",
      "-tgcacctgacgccagctcagttatccatgctactggaggggatc-\n",
      ">G000006925|4359551|4359859|4359604|4359648|-\n",
      "gtacacctgacgccagctcagttatccatgctactggaggggat--\n",
      ">G000006925|4622488|4624460|4623119|4623163|+\n",
      "--gcaccttactccggcccagttatccatgcttcttgagggcatcg\n",
      ">G000006925|4700925|4701241|4700976|4701020|-\n",
      "--gcacctgacgccagctcagttatccatgctactggaggggatcg\n",
      ">G000006925|4776578|4776894|4776629|4776673|-\n",
      "--gcacctgacgccagctcagttatccatgctactggaggggatcg\n",
      ">G000006925|4781327|4781663|4781567|4781611|+\n",
      "--gcacctgacgccggctcagttatccatgctactggaggggatcg\n",
      ">G000299455|1991020|1991316|1991252|1991296|+\n",
      "--gttcctgacgccggcgcaactggcgatgctgatggaaggtatcg\n",
      ">G000299455|3215105|3215383|3215334|3215378|+\n",
      "--gttcctcacaccggcacagctggcgatgctgctggaaggtatcg\n",
      ">G000299455|4543449|4543753|4543689|4543733|+\n",
      "--gttccttacgcaggcgcagctggcgatgctgctggaaggtatcg\n",
      ">G000299455|5289627|5289931|5289647|5289691|-\n",
      "-tgttccttacacaggcgcagctggcgatgctgctggaaggtatc-\n",
      ">G000299455|5313369|5315826|5314356|5314400|+\n",
      "--gcaccttactccggctcagttatccatgcttcttgaaggtatca\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1, 24671)), (15422, 15451), '+'), (('G000006925', (905848, 907049)), (905954, 905983), '+'), (('G000006925', (1397676, 1398098)), (1397955, 1397984), '-'), (('G000006925', (1581377, 1581641)), (1581495, 1581524), '+'), (('G000006925', (2062783, 2064325)), (2064004, 2064033), '+'), (('G000299455', (2541782, 2542216)), (2542073, 2542102), '-'), (('G000299455', (2927401, 2931337)), (2930232, 2930261), '+'), (('G000299455', (4431865, 4470166)), (4435082, 4435111), '-')]\n",
      ">G000006925|1|24671|15422|15451|+\n",
      "-aaaacagcaacctccgtctggccagttcg\n",
      ">G000006925|905848|907049|905954|905983|+\n",
      "-aagacagcgacctccgtctggccggttcg\n",
      ">G000006925|1397676|1398098|1397955|1397984|-\n",
      "gaagacagcgacctccgtctggccggttc-\n",
      ">G000006925|1581377|1581641|1581495|1581524|+\n",
      "gaagacagcgacctccgtctggccggttc-\n",
      ">G000006925|2062783|2064325|2064004|2064033|+\n",
      "gaagacagcgacctccgtctggccggttc-\n",
      ">G000299455|2541782|2542216|2542073|2542102|-\n",
      "gaagacagcaacctccgtctgaccggttc-\n",
      ">G000299455|2927401|2931337|2930232|2930261|+\n",
      "-aagacagcgacctccgtctggccggttcg\n",
      ">G000299455|4431865|4470166|4435082|4435111|-\n",
      "gaaaacagcaacctccgtctggccagttc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2923039, 2950048)), (2944906, 2944937), '-'), (('G000006925', (4026192, 4050087)), (4040691, 4040722), '-'), (('G000299455', (1078152, 1154001)), (1081221, 1081252), '+'), (('G000299455', (5094571, 5134770)), (5120240, 5120271), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 15\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 15\r",
      "   10 / 15\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 14  f\b\b\r",
      "STEP     2 / 14  f\b\b\r",
      "STEP     3 / 14  f\b\b\r",
      "STEP     4 / 14  f\b\b\r",
      "STEP     5 / 14  f\b\b\r",
      "STEP     6 / 14  f\b\b\r",
      "STEP     7 / 14  f\b\b\n",
      "Reallocating..done. *alloclen = 1090\n",
      "\r",
      "STEP     8 / 14  f\b\b\r",
      "STEP     9 / 14  f\b\b\r",
      "STEP    10 / 14  f\b\b\r",
      "STEP    11 / 14  f\b\b\r",
      "STEP    12 / 14  f\b\b\r",
      "STEP    13 / 14  f\b\b\r",
      "STEP    14 / 14  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 15\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 15\r",
      "   10 / 15\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 14  f\b\b\r",
      "STEP     2 / 14  f\b\b\r",
      "STEP     3 / 14  f\b\b\r",
      "STEP     4 / 14  f\b\b\r",
      "STEP     5 / 14  f\b\b\r",
      "STEP     6 / 14  f\b\b\r",
      "STEP     7 / 14  f\b\b\r",
      "STEP     8 / 14  f\b\b\r",
      "STEP     9 / 14  f\b\b\r",
      "STEP    10 / 14  f\b\b\r",
      "STEP    11 / 14  f\b\b\r",
      "STEP    12 / 14  f\b\b\r",
      "STEP    13 / 14  f\b\b\n",
      "Reallocating..done. *alloclen = 1091\n",
      "\r",
      "STEP    14 / 14  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\n",
      "Reallocating..done. *alloclen = 1060\n",
      "\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1064\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2923039|2950048|2944906|2944937|-\n",
      "tcaatggtggcgcgatcgcactcggtcaccc-\n",
      ">G000006925|4026192|4050087|4040691|4040722|-\n",
      "tcaatggtggtgcgatcgcgctgggtcatcc-\n",
      ">G000299455|1078152|1154001|1081221|1081252|+\n",
      "-caatggtggtgcgatcgcacttggtcacccg\n",
      ">G000299455|5094571|5134770|5120240|5120271|+\n",
      "-caatggtggtgcgatcgcgctgggtcatccg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1509064, 1578296)), (1577250, 1577275), '+'), (('G000299455', (2167395, 2167615)), (2167468, 2167493), '-'), (('G000299455', (2547013, 2547107)), (2547082, 2547107), '-'), (('G000299455', (2923865, 2925880)), (2924838, 2924863), '+'), (('G000299455', (3177753, 3177778)), (3177753, 3177778), '-'), (('G000299455', (3291704, 3291790)), (3291704, 3291729), '+')]\n",
      ">G000006925|1509064|1578296|1577250|1577275|+\n",
      "-cccctccgcacagggggattcacca\n",
      ">G000299455|2167395|2167615|2167468|2167493|-\n",
      "gcccctccgcacagggggattcacc-\n",
      ">G000299455|2547013|2547107|2547082|2547107|-\n",
      "gcccctccgctcagggggattcacc-\n",
      ">G000299455|2923865|2925880|2924838|2924863|+\n",
      "-cccctccgcacagggggattcacca\n",
      ">G000299455|3177753|3177778|3177753|3177778|-\n",
      "gccccaccgcacagggggattcacc-\n",
      ">G000299455|3291704|3291790|3291704|3291729|+\n",
      "-cccctccacacagggggattcacca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2569729, 2584886)), (2577818, 2577947), '+'), (('G000299455', (1462739, 1472085)), (1469678, 1469807), '-'), (('G000299455', (3918464, 3964500)), (3934053, 3934182), '+')]\n",
      ">G000006925|2569729|2584886|2577818|2577947|+\n",
      "-cgcagttgcagattgatatcgaccagcgtaaagctcaggcgctgggcgttgatatcgac\n",
      "gatattaacgacacactgcaaaccgcctggggttcgagctatgtgaatgactttatggat\n",
      "cgcggtcgcg\n",
      ">G000299455|1462739|1472085|1469678|1469807|-\n",
      "ccgcagttgcagattgatatcgaccagcgtaaagctcaggcgctgggcgttgctatcgac\n",
      "gatattaacgacacactgcaaaccgcctggggttcgagctatgtgaatgactttatggat\n",
      "cgcggtcgc-\n",
      ">G000299455|3918464|3964500|3934053|3934182|+\n",
      "-cgcagtttaagattgatatcgaccaggaaaaagcgcaggcgctgggtgtttctatcaac\n",
      "gacattaacaccactctgggcgctgcatggggtggtagctatgtgaacgactttatcgac\n",
      "cgcggtcgtg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1842508, 1869155)), (1862401, 1862431), '+'), (('G000006925', (2242494, 2346249)), (2333031, 2333061), '-'), (('G000299455', (1720965, 1752624)), (1746049, 1746079), '+'), (('G000299455', (2786550, 2804122)), (2797362, 2797392), '+'), (('G000299455', (3608677, 3610659)), (3610055, 3610085), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1842508|1869155|1862401|1862431|+\n",
      "-gtcgctttaggtctggtttatcagttctaa\n",
      ">G000006925|2242494|2346249|2333031|2333061|-\n",
      "cgtagctctgggtctggtttaccagttcta-\n",
      ">G000299455|1720965|1752624|1746049|1746079|+\n",
      "-gtagctcttggcctggtttaccagttctaa\n",
      ">G000299455|2786550|2804122|2797362|2797392|+\n",
      "-gtcgctttaggtctggtttatcagttctaa\n",
      ">G000299455|3608677|3610659|3610055|3610085|+\n",
      "-gttgctgtaggtatggtttaccagttctaa\n",
      "\n",
      "\n",
      "29 [(('G000006925', (251358, 252232)), (251666, 251687), '+'), (('G000006925', (315068, 315858)), (315612, 315633), '-'), (('G000006925', (332059, 337870)), (332603, 332624), '-'), (('G000006925', (493019, 493813)), (493563, 493584), '-'), (('G000006925', (509301, 510374)), (509846, 509867), '-'), (('G000006925', (724190, 725876)), (724735, 724756), '-'), (('G000006925', (1094884, 1095679)), (1095428, 1095449), '-'), (('G000006925', (1099363, 1100153)), (1099587, 1099608), '+'), (('G000006925', (1411510, 1412350)), (1412104, 1412125), '-'), (('G000006925', (1883764, 1885448)), (1884309, 1884330), '-'), (('G000006925', (1956633, 1961076)), (1957178, 1957199), '-'), (('G000006925', (2084425, 2085215)), (2084649, 2084670), '+'), (('G000006925', (2236395, 2237421)), (2236855, 2236876), '+'), (('G000006925', (3065287, 3076770)), (3075308, 3075329), '-'), (('G000006925', (3812816, 3813607)), (3813360, 3813381), '-'), (('G000006925', (4639759, 4640549)), (4640303, 4640324), '-'), (('G000006925', (4650084, 4650878)), (4650312, 4650333), '+'), (('G000006925', (4656735, 4658205)), (4657639, 4657660), '+'), (('G000006925', (4673259, 4674053)), (4673487, 4673508), '+'), (('G000006925', (4679888, 4680760)), (4680433, 4680454), '-'), (('G000006925', (4750261, 4751054)), (4750488, 4750509), '+'), (('G000006925', (4753880, 4754990)), (4754745, 4754765), '-'), (('G000006925', (4760730, 4761525)), (4760959, 4760980), '+'), (('G000006925', (4793569, 4794365)), (4793799, 4793820), '+'), (('G000299455', (906289, 909876)), (906519, 906540), '+'), (('G000299455', (912160, 912572)), (912388, 912409), '+'), (('G000299455', (4752701, 4766669)), (4752931, 4752952), '+'), (('G000299455', (4771847, 4785261)), (4780257, 4780278), '-'), (('G000299455', (5275700, 5279438)), (5279037, 5279058), '-')]\n",
      ">G000006925|251358|252232|251666|251687|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|315068|315858|315612|315633|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|332059|337870|332603|332624|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|493019|493813|493563|493584|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|509301|510374|509846|509867|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|724190|725876|724735|724756|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|1094884|1095679|1095428|1095449|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|1099363|1100153|1099587|1099608|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|1411510|1412350|1412104|1412125|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|1883764|1885448|1884309|1884330|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|1956633|1961076|1957178|1957199|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|2084425|2085215|2084649|2084670|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|2236395|2237421|2236855|2236876|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|3065287|3076770|3075308|3075329|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|3812816|3813607|3813360|3813381|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4639759|4640549|4640303|4640324|-\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4650084|4650878|4650312|4650333|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4656735|4658205|4657639|4657660|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4673259|4674053|4673487|4673508|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4679888|4680760|4680433|4680454|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000006925|4750261|4751054|4750488|4750509|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4753880|4754990|4754745|4754765|-\n",
      "-accgggggcggtgaggtggg-\n",
      ">G000006925|4760730|4761525|4760959|4760980|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000006925|4793569|4794365|4793799|4793820|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000299455|906289|909876|906519|906540|+\n",
      "-accaggggcggtgatggaggg\n",
      ">G000299455|912160|912572|912388|912409|+\n",
      "-accgggggcggtgatggaggg\n",
      ">G000299455|4752701|4766669|4752931|4752952|+\n",
      "-accaggggcggtgatggaggg\n",
      ">G000299455|4771847|4785261|4780257|4780278|-\n",
      "taccgggggcggtgatggagg-\n",
      ">G000299455|5275700|5279438|5279037|5279058|-\n",
      "taccgggggcggtgatggagg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (511335, 520753)), (511581, 511660), '-'), (('G000299455', (3808566, 3816764)), (3816674, 3816753), '+'), (('G000299455', (3818101, 3818615)), (3818525, 3818604), '+')]\n",
      ">G000006925|511335|520753|511581|511660|-\n",
      "tcgcccgcagccatcactgaccgcgggcgaaagtgtaaagcaggtgccttaccatcctga\n",
      "cctgacaaccggatatgcg-\n",
      ">G000299455|3808566|3816764|3816674|3816753|+\n",
      "-cgcccgcagccatcactgaccgcgggcgaaagtgtaaagcaggtgccttaccactctga\n",
      "cctgacaaccggatatgcgg\n",
      ">G000299455|3818101|3818615|3818525|3818604|+\n",
      "-cgcccgcagccatcactgaccgcgggcgaaagtgtaaagcaggtgccttaccatcctga\n",
      "cctgacaaccggatatgcgg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3094517, 3094867), '-'), (('G000299455', (50607, 52894)), (52046, 52396), '+'), (('G000299455', (857924, 871846)), (860645, 860995), '+'), (('G000299455', (1980069, 1981046)), (1980199, 1980549), '+'), (('G000299455', (3143255, 3145541)), (3144694, 3145044), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  d\b\b\r",
      "STEP    28 / 28  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 29\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 29\r",
      "   10 / 29\r",
      "   20 / 29\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 28  f\b\b\r",
      "STEP     2 / 28  f\b\b\r",
      "STEP     3 / 28  f\b\b\r",
      "STEP     4 / 28  f\b\b\r",
      "STEP     5 / 28  f\b\b\r",
      "STEP     6 / 28  f\b\b\r",
      "STEP     7 / 28  f\b\b\r",
      "STEP     8 / 28  f\b\b\r",
      "STEP     9 / 28  f\b\b\r",
      "STEP    10 / 28  f\b\b\r",
      "STEP    11 / 28  f\b\b\r",
      "STEP    12 / 28  f\b\b\r",
      "STEP    13 / 28  f\b\b\r",
      "STEP    14 / 28  f\b\b\r",
      "STEP    15 / 28  f\b\b\r",
      "STEP    16 / 28  f\b\b\r",
      "STEP    17 / 28  f\b\b\r",
      "STEP    18 / 28  f\b\b\r",
      "STEP    19 / 28  f\b\b\r",
      "STEP    20 / 28  f\b\b\r",
      "STEP    21 / 28  f\b\b\r",
      "STEP    22 / 28  f\b\b\r",
      "STEP    23 / 28  f\b\b\r",
      "STEP    24 / 28  f\b\b\r",
      "STEP    25 / 28  f\b\b\r",
      "STEP    26 / 28  f\b\b\r",
      "STEP    27 / 28  f\b\b\r",
      "STEP    28 / 28  d\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3094517|3094867|-\n",
      "ccgcctttcagcatattctcctggatggtctgataagtactccacaggtccttaccgtaa\n",
      "tcctcccggcgtcgtggcgtcagaatgtcggcggtggtgacgggctgatgttcgtcacca\n",
      "taacggtaagtcagtgccgcctgtgccagcgcctggcgtgccggtggcggcaggaccagc\n",
      "gactgcatggcatcacgtttctcctcaatccggtcaaacacgcccaccacttcgtaagcc\n",
      "ccttcaataactttctccaccacgtttccccggtgcggaacacgcacttcccccagagac\n",
      "tgaccacagacacacccgttctggcagacgaatctgaagtaacccggcag-\n",
      ">G000299455|50607|52894|52046|52396|+\n",
      "-cgcctttcagcatattctcctggatggtctgatatgcgctccacaggtccttaccgtaa\n",
      "tcctcccggcgtcgtggcgtcagaatgtcggcggtggtgacgggctgatgttcgtcacca\n",
      "taacggtaagtcagcgccgcctgtgccagcgcctggcgtgccggtggcggcaggaccagc\n",
      "gactgcatggcatcacgcttttcctcaatccggtcaaaaacgcccaccacctcgtaagcc\n",
      "ccttcaatgactctgtccaccacatttccccgatgcggaacacgcacttcccccagagac\n",
      "tgaccacagacgcatccgttctgacagacgaacctgaagtaacccggcagc\n",
      ">G000299455|857924|871846|860645|860995|+\n",
      "-cgcctttcagcatattctcctggatggtctgataagtactccacaggtccttaccgtaa\n",
      "tcctccctgcgtcgcggtgtcagaatgtcagcggtggtgacgggctgatgttcgtcacca\n",
      "taacggtaagtcagtgccgcctgtgccagcgcctggcgtgccggtggcggcagaatcagc\n",
      "gactgcatggcatcacgcttttcctcaatccggtcaaaaacccccaccacctcgtaagcc\n",
      "ccttcaatgactctgtccaccacatttccccggtgcggaacacgcacttcccccagagac\n",
      "tgaccacagacgcatccgttctggcagacgaacctgaagtaacccggcagc\n",
      ">G000299455|1980069|1981046|1980199|1980549|+\n",
      "-cgcctttcagcatattctcctggatggtctgatatgcgctccacaggtccttaccgtaa\n",
      "tcctcccggcgtcgcggcgtcagaatgtcggcggtggtgacgggctgatgttcgtcacca\n",
      "taacggtaagtcagtgccgcctgtgccagcgcctggcgtgccggtggcggcaggaccagc\n",
      "gactgcatggcatcacgcttctcctctatccggtcaaacacgcccaccacctcgtaagcc\n",
      "ccttcgataactttctccactacatttccccggtgtggaacacgcacttcccccagagac\n",
      "tggccacagacgcacccgttctggcagacgaacctgaagtaacccggcagc\n",
      ">G000299455|3143255|3145541|3144694|3145044|+\n",
      "-cgcctttcagcatattctcctggatggtctgataagcactccacaggtccttaccgtaa\n",
      "tcctcccggcgtcgtggcgtcagaatgtcggcggtggtgacgggctgatgttcgtcacca\n",
      "taacggtaagtcagtgccgcctgtgccagcgcctggcgtgccggtggcggcaggaccagc\n",
      "gactgcatggcatcacgtttctcttcaatccggtcaaaaacccccaccacctcgtaagcc\n",
      "ccttcgataactttctccactacatttccccggtgtggaacacgcacttcccccagagac\n",
      "tggccacagacgcacccgttctggcagacgaacctgaagtaacccggcagc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4666868, 4668030)), (4667207, 4667537), '-'), (('G000006925', (4802939, 4805150)), (4803432, 4803762), '+'), (('G000299455', (886351, 887494)), (886831, 887159), '+'), (('G000299455', (5346044, 5347202)), (5346537, 5346866), '+')]\n",
      ">G000006925|4666868|4668030|4667207|4667537|-\n",
      "gaagatggatatccacttcatcttcataaaagaccggatgctctgcgctgcattcgtcca\n",
      "gtgctttatggattgctgccatcttttcatctttatgcgggtcacggatacgcagagttg\n",
      "gcgcagcccttcgccacacaatccccacagacggcaaccagcggcgaacggttccggcat\n",
      "ttaactggcaaccggttatctcattgatttttattgccagcagttctgtactccagcgtg\n",
      "aacgctggtagccaaagtcgccgggagaatgttttaccagctcacgtaacagtgtgcaga\n",
      "tatgctcaaacggccagcgacgggcacgcc-\n",
      ">G000006925|4802939|4805150|4803432|4803762|+\n",
      "-aagatggatatccacttcatcttcataaaagaccggatgctctgcgctgcattcgtcca\n",
      "gtgctttatggattgctgccatcttttcatctttatgcgggtcacggatacgcagagttg\n",
      "gcgcagcccttcgccacacaatccccacagacggcaaccagcggcgaacggttccggcat\n",
      "ttaactggcaaccggttatctcattgatttttattgccagcagttctgtactccagcgtg\n",
      "aacgctggtagccaaagtcgccgggagaatgttttaccagctcacgtaacagtgtgcaga\n",
      "tatgctcaaacggccagcgacgggcacgccc\n",
      ">G000299455|886351|887494|886831|887159|+\n",
      "-gagatgaatatccac-tcatcttcgtaaaacacggggttttccgcactgcatttcgcca\n",
      "gcgtttcatgaatagcggccattttttcttctttgtggggggcacggatacgcggtgtcg\n",
      "acgcggctctgcgctataccagtccggcagcaggtaaccatcgccggatggttggagcat\n",
      "gtagcgggcaaccggtgacatcacggatttttatcgccagaagttcggtactccagcggg\n",
      "aacgctgataacctaaatcgccaggagaatgcttaac-agctcacgcagcaatgcgcaaa\n",
      "tatgttcaaacggccagcgcctcccacgtcc\n",
      ">G000299455|5346044|5347202|5346537|5346866|+\n",
      "-gagatgaatatccac-tcatcttcgtaaaacacggggttttccgcactgcatttcgcca\n",
      "gcgcttcatgaatagcggccattttttcttctttgtatgggtaacggatacacagtgtcg\n",
      "acgcggctctgcgctatacaagtccggcagcaggtaaccatcgccggatggttgaaacat\n",
      "gtagcgggcaaccggtaacatcacggatttttatcgccagaagttcggtactccagcggg\n",
      "aacgctgataaccgaaatcgccaggagaatacttaacaagctcacgcagcaatgctcaaa\n",
      "tatgttcgaacggccagcgcctcccacgtcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (608503, 616145)), (615677, 615729), '+'), (('G000299455', (3680085, 3687629)), (3687259, 3687311), '+'), (('G000299455', (4792177, 4792229)), (4792177, 4792229), '+')]\n",
      ">G000006925|608503|616145|615677|615729|+\n",
      "tgggctttgcggaactgtttatcgacccggttgccatgtcgcaaattacgcg\n",
      ">G000299455|3680085|3687629|3687259|3687311|+\n",
      "tgggctttgcggaactgtttatcgacccggttgccatgtcgcaaattacgcg\n",
      ">G000299455|4792177|4792229|4792177|4792229|+\n",
      "tgggctttgccgaactctttattgacccggtggcgattgcgcaaatcacgcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3322761, 3361739)), (3359713, 3359737), '-'), (('G000006925', (4385267, 4385291)), (4385267, 4385291), '+'), (('G000299455', (540757, 615358)), (579634, 579658), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3322761|3361739|3359713|3359737|-\n",
      "ttgatctggtataacaggtataaa-\n",
      ">G000006925|4385267|4385291|4385267|4385291|+\n",
      "-tgatctggtataacaggtataaag\n",
      ">G000299455|540757|615358|579634|579658|+\n",
      "-tgatctggtataacaggtataaag\n",
      "\n",
      "\n",
      "7 [(('G000006925', (2560406, 2561162)), (2560454, 2560514), '+'), (('G000006925', (3083675, 3099254)), (3083871, 3083931), '-'), (('G000006925', (4633099, 4633840)), (4633732, 4633792), '-'), (('G000006925', (4668315, 4668899)), (4668791, 4668852), '-'), (('G000006925', (4822840, 4823055)), (4822887, 4822947), '+'), (('G000299455', (857924, 871846)), (871592, 871652), '+'), (('G000299455', (5017408, 5018168)), (5018055, 5018115), '-')]\n",
      ">G000006925|2560406|2561162|2560454|2560514|+\n",
      "--cgtggatctcaactgagcgtgagaagcagattgttttacgacccaaccgcttaatgcg\n",
      "gg\n",
      ">G000006925|3083675|3099254|3083871|3083931|-\n",
      "-tcgtgaatttccacagcacgtgaaaagtaaatcgtttttcggtccagtcgtttgatgcg\n",
      "g-\n",
      ">G000006925|4633099|4633840|4633732|4633792|-\n",
      "-tcgtggatttccacggaacgcgagaagcagattgttttacgtgccagccgcttgatgcg\n",
      "g-\n",
      ">G000006925|4668315|4668899|4668791|4668852|-\n",
      "ttcgtggatttccacggaacgcgagaagcagattgttttacgtgccagccgcttgatgcg\n",
      "g-\n",
      ">G000006925|4822840|4823055|4822887|4822947|+\n",
      "--cgtggatttccacggaacgcgagaagcagattgttttacgtgccagccgcttgatgcg\n",
      "gg\n",
      ">G000299455|857924|871846|871592|871652|+\n",
      "--cgtgaatttccacggagcatgaaaagcaaatcgttttccggtccagccgtttgatgtg\n",
      "gg\n",
      ">G000299455|5017408|5018168|5018055|5018115|-\n",
      "-tcatgaagttctacagaacgcgagaaacagatggttctgcgggccagacgtttgattcg\n",
      "g-\n",
      "\n",
      "\n",
      "8 [(('G000006925', (3065287, 3076770)), (3066642, 3066685), '+'), (('G000006925', (4622488, 4624460)), (4624138, 4624181), '+'), (('G000299455', (1992068, 1992414)), (1992268, 1992311), '+'), (('G000299455', (3215867, 3216640)), (3215939, 3215982), '+'), (('G000299455', (4544552, 4544854)), (4544705, 4544748), '+'), (('G000299455', (4749820, 4751622)), (4751171, 4751214), '+'), (('G000299455', (5287974, 5288758)), (5288632, 5288675), '-'), (('G000299455', (5313369, 5315826)), (5315375, 5315418), '+')]\n",
      ">G000006925|3065287|3076770|3066642|3066685|+\n",
      "-aaaggtatccatccacagaatcacctagccggttacagcggtg\n",
      ">G000006925|4622488|4624460|4624138|4624181|+\n",
      "-aaaggtatccatccacagaatcatctggccggttacagcggag\n",
      ">G000299455|1992068|1992414|1992268|1992311|+\n",
      "-aaaggtatccaccctcagacccatcttgcaggcttcagtggcg\n",
      ">G000299455|3215867|3216640|3215939|3215982|+\n",
      "-aaaggcatccatccgcagacccatcttgcggggttcagtggtg\n",
      ">G000299455|4544552|4544854|4544705|4544748|+\n",
      "-aaaggcatccatccgcagactcatcttgcttgcttcagcggtg\n",
      ">G000299455|4749820|4751622|4751171|4751214|+\n",
      "-aaaggtatccatccacagaatcacctagccggttacagcggtg\n",
      ">G000299455|5287974|5288758|5288632|5288675|-\n",
      "aaaaggcatccatccgcagacccatcttgcggggttcagtggt-\n",
      ">G000299455|5313369|5315826|5315375|5315418|+\n",
      "-aaaggtatccatccacaaaatcacctggccggttacagcggtg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (739473, 745780)), (744568, 745043), '+'), (('G000006925', (1921807, 1928114)), (1922544, 1923019), '-'), (('G000299455', (2192349, 2193633)), (2192556, 2193031), '+'), (('G000299455', (2897570, 2901707)), (2898172, 2898647), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1952\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1952\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|744568|745043|+\n",
      "-cgggttctgggaaaattgtccgcgaaccagattgaaggcgatctcgttaaaacagtggg\n",
      "caaacctttcccacgggactcccgggcaccggagaggtggccatcagggaccattaccgt\n",
      "cagggtttatgacgatcagccgtttgatcggcaaattgttattcccgcggtggcgtttcg\n",
      "cggtgctaaacatgagcggaagaataacaatatttattcgtcatgccgcctgatagtgaa\n",
      "gaaaaacggtgctgaaatttataaccgaacgaccctggataatacgctgatatatacggg\n",
      "tgttattgatatgcctgccggtcacggtcacatgacgctggagttttctgtatcggcatg\n",
      "gctggtaaatggctggtatcccacagcaagtatcagcgatttgctggttgttgtgatgaa\n",
      "gaaatccacagcaggtatcagtatcagctgaattttataacccagatacgggcacc\n",
      ">G000006925|1921807|1928114|1922544|1923019|-\n",
      "tcgggttctgggaaaattgtccgcgaaccagattgaaggcgatctcgttaaaacagtggg\n",
      "caaacctttcccacgggactcccgggcaccggagaggtggccatcagggaccattaccgt\n",
      "cagggtttatgacgatcagccgtttgatcggcaaattgttattcccgcggtggcgtttcg\n",
      "cggtgctaaacatgagcggaagaataacaatatttattcgtcatgccgcctgatagtgaa\n",
      "gaaaaacggtgctgaaatttataaccgaacgaccctggataatacgctgatatatacggg\n",
      "tgttattgatatgcctgccggtcacggtcacatgacgctggagttttctgtatcggcatg\n",
      "gctggtaaatggctggtatcccacagcaagtatcagcgatttgctggttgttgtgatgaa\n",
      "gaaatccacagcaggtatcagtatcagctgaattttataacccagatacgggcac-\n",
      ">G000299455|2192349|2193633|2192556|2193031|+\n",
      "-cgggttctgggaaaactgtccgcgaaccagattgaaggcgatctcgttaaaacagtggg\n",
      "caaagctttcccccgtgactcccgggcaccggagcggtggccatcagggaccattaccgt\n",
      "cagggtttatgacgatcagccgtttgaccggcaaattgttattccggctgtggcattcag\n",
      "tggcgctaagcatgagagagagcatactgatatttactcctcatgccgtctgatagtgcg\n",
      "gaaaaacggtgctgaaatttataaccgtaccgcgctggataatacgctgatttacagtgg\n",
      "tgttattgatatgcctgccggtcacggtcacatgacactggagttttcggtgtcagcatg\n",
      "gctggtaaataactggtatcccacagcaagtatcagcgatttgctggttgtggtgatgaa\n",
      "gaaagccactgcaggcatcacgattagctgaattttataacccagatacgggcgcc\n",
      ">G000299455|2897570|2901707|2898172|2898647|-\n",
      "tcgggttctgggaaaattgtccgccaaccagattgaaggcgatctcgttaaaacagtggg\n",
      "caaagctttcccccgggactcccgtgcaccggaacggtggccatcagggaccatcaccgt\n",
      "cagggtttatgacgatcagccgtttgaccggcagattgttattccggcggtggcattcag\n",
      "cggcgctaaacatgagcgggagaataacgatatttattcgtcatgccgcctgatagtacg\n",
      "gaaaaacggtgctgaaatttataaccgtaccgcgctggataatacgctgatttacagtgg\n",
      "tgttattgatatgcctgccggtcacggtcacatgacactggagttttcggtgtcagcatg\n",
      "gctggtaaataactggtatcccacagcaagtatcagcgatttgctggttgtggtgatgaa\n",
      "gaaagccactgcaggcatcacgattagctgaattttataacccagatacgggcgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1776250, 1792578)), (1786121, 1786423), '+'), (('G000006925', (4094093, 4116338)), (4097446, 4097748), '+'), (('G000299455', (2682449, 2700553)), (2686718, 2687020), '+'), (('G000299455', (5063005, 5071226)), (5067571, 5067873), '-')]\n",
      ">G000006925|1776250|1792578|1786121|1786423|+\n",
      "-ggcagaccgtgatacagctccggctgatcggcgtgatgcagcacgtacataacgtgcgt\n",
      "accaccgaccccttccgggttgtagacgccagcatgttcgtaaccacgcgctttcagttt\n",
      "cgccacgcgctgttccgccagctccagcatctccttcttggtgccgaagtggatagcccc\n",
      "ggtcggacaggttttcacacaagccggttcctggccgacgctgacgcgatcgacgcagag\n",
      "cgtgcatttatatacccggttatcctctttgttgaggcgcggaatattaaacggacaccc\n",
      "ggc\n",
      ">G000006925|4094093|4116338|4097446|4097748|+\n",
      "-ggcaagccatgatacagatttggcttgtcagcatggtgcagcacgtacatgacgtgtgt\n",
      "accaccgacgcctgccggatcgtacagacccgcattgtcgtaaccgcgggttttcagctc\n",
      "agcaacgcgctcgctcgccagcgttttcatcgactctttcgtgccgaagtgaatcgcgcc\n",
      "cgttgggcaggtcttcacgcaggctggttcttgcccaaccaccacgcggtcaacgcacag\n",
      "cgtacatttgtagacgcggttgtcttccgggttgaggcgcggaatgtcgaacggacagcc\n",
      "cgc\n",
      ">G000299455|2682449|2700553|2686718|2687020|+\n",
      "-ggcagaccgtgatacagctccggctgatcggcgtgatgcagcacgtacataacgtgcgt\n",
      "accaccgaccccttccgggttgtagacgccagcatgttcgtaaccacgcgctttcagttt\n",
      "cgccacgcgctgttccgccagctccagcatctccttcttggtgccgaagtggatagcccc\n",
      "ggtcggacaggttttcacacaagccggttcctggccgacgctgacgcgatcgacgcagag\n",
      "cgtgcatttatatacccggttatcctctttgttgaggcgcggaatattaaacggacaccc\n",
      "ggc\n",
      ">G000299455|5063005|5071226|5067571|5067873|-\n",
      "cggcaagccatgatacagatttggcttgtcagcatggtgcagcacgtacatgacgtgtgt\n",
      "accaccgacgcctgccggatcgtacagacccgcattgtcgtaaccgcgggttttcagctc\n",
      "agcaacgcgctcgctcgccagcgttttcatcgactctttcgtaccgaagtgaatcgcgcc\n",
      "cgttgggcaggtcttcacgcaggctggttcttgcccaaccaccacgcggtcaacgcacag\n",
      "cgtacatttgtagacgcggttgtcttccgggttgaggcgcggaatgtcgaacggacagcc\n",
      "cg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2333183, 2333226), '-'), (('G000299455', (1720965, 1752624)), (1745884, 1745927), '+'), (('G000299455', (3608677, 3610659)), (3609905, 3609933), '+')]\n",
      ">G000006925|2242494|2346249|2333183|2333226|-\n",
      "tgtcatcaatggtcgtaactacgacgacgaagatatcctgaaa-\n",
      ">G000299455|1720965|1752624|1745884|1745927|+\n",
      "-accatcggtactcgtaactacgacgacgaagatatcctgaaat\n",
      ">G000299455|3608677|3610659|3609905|3609933|+\n",
      "----------------gtttggggtgatcaggacctggttgaat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (732444, 732548), '+'), (('G000299455', (2174641, 2181396)), (2180840, 2180944), '+'), (('G000299455', (2910238, 2916987)), (2910684, 2910788), '-'), (('G000299455', (3598405, 3598509)), (3598405, 3598509), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|732444|732548|+\n",
      "-tggatacccgtcgtggctcaaattcacagctggaaacggcagtgaaagacctgggggcg\n",
      "gtggtgtccttcaaagggtattatggtgatctggcaatcgtggtg\n",
      ">G000299455|2174641|2181396|2180840|2180944|+\n",
      "-tggatacccgtcgcggctcaaattcacaactcgaaacggcagtgaaagacctgggagcg\n",
      "gtggtgtctttcaaagggtattacggtgatctggccattgtggtg\n",
      ">G000299455|2910238|2916987|2910684|2910788|-\n",
      "atggatacccgtcgcggttcaaattcacaactcgaaacggcagtgaaagatctgggcgca\n",
      "gtggtatccttcaaggggtattacggcgatctggccattgtggt-\n",
      ">G000299455|3598405|3598509|3598405|3598509|-\n",
      "ctggatacccgtcgcggctctaattccgagctggagacagcggtaaaagacctgggcgaa\n",
      "gcggtgtcctataaggggatgtatggcgatacggcgatcgtcgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (3004197, 3004230), '-'), (('G000299455', (978228, 988794)), (988631, 988664), '+'), (('G000299455', (988799, 1028140)), (988799, 988831), '+')]\n",
      ">G000006925|2976303|3015537|3004197|3004230|-\n",
      "cccgctctgcgggtctttttttcgccaaaagca-\n",
      ">G000299455|978228|988794|988631|988664|+\n",
      "-ccgctctgcgggtctttttttcgccaaaagcaa\n",
      ">G000299455|988799|1028140|988799|988831|+\n",
      "-ccgctctgcggg-gattttttcgccaaaagcaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (58872, 58897), '-'), (('G000299455', (4235244, 4293324)), (4238085, 4238110), '-'), (('G000299455', (4383450, 4406324)), (4389041, 4389066), '+')]\n",
      ">G000006925|25217|64488|58872|58897|-\n",
      "gaagtggcgctggcgcgtgaagatg-\n",
      ">G000299455|4235244|4293324|4238085|4238110|-\n",
      "gaagtggcgctggcgcgggaagatg-\n",
      ">G000299455|4383450|4406324|4389041|4389066|+\n",
      "-aagtggcgctggcgcgtgaagatgc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1582509, 1591522)), (1584951, 1584989), '-'), (('G000006925', (2242494, 2346249)), (2289529, 2289567), '-'), (('G000299455', (1752647, 1813107)), (1789419, 1789457), '+'), (('G000299455', (2596243, 2603992)), (2597421, 2597459), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1582509|1591522|1584951|1584989|-\n",
      "cctgatgcgcacggtgaaatgcaccaaagcctaaatga-\n",
      ">G000006925|2242494|2346249|2289529|2289567|-\n",
      "cctgatgagcgcggtgaaaggcgccaaagccaaaatga-\n",
      ">G000299455|1752647|1813107|1789419|1789457|+\n",
      "-ctgatgagcgcagtgaaaggcgccaaagccaaaatgaa\n",
      ">G000299455|2596243|2603992|2597421|2597459|-\n",
      "cctgatgcgcacggtgaaatgcaccaaagcctaaatga-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (739473, 745780)), (745614, 745645), '+'), (('G000006925', (1921807, 1928114)), (1921942, 1921973), '-'), (('G000299455', (2192349, 2193633)), (2193602, 2193633), '+'), (('G000299455', (2897570, 2901707)), (2897570, 2897601), '-'), (('G000299455', (3268692, 3268726)), (3268695, 3268726), '-'), (('G000299455', (3585529, 3586267)), (3585665, 3585696), '-')]\n",
      ">G000006925|739473|745780|745614|745645|+\n",
      "-cttatgaaggttccggcagtggcgactggcg\n",
      ">G000006925|1921807|1928114|1921942|1921973|-\n",
      "gcttatgaaggttccggcagtggcgactggc-\n",
      ">G000299455|2192349|2193633|2193602|2193633|+\n",
      "-cttatgaaggttccggtagtggcgactggcg\n",
      ">G000299455|2897570|2901707|2897570|2897601|-\n",
      "gcttatgaaggttccggtagtggcgactggc-\n",
      ">G000299455|3268692|3268726|3268695|3268726|-\n",
      "gcttacgaaggttccggctctggcgactggc-\n",
      ">G000299455|3585529|3586267|3585665|3585696|-\n",
      "gcttatgaaggctccggcagtggtgactggc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (985336, 1008622)), (986434, 986498), '-'), (('G000006925', (4379878, 4379942)), (4379878, 4379942), '+'), (('G000299455', (3362286, 3384700)), (3384402, 3384466), '+')]\n",
      ">G000006925|985336|1008622|986434|986498|-\n",
      "atatcgacacgataaacccctggcgttattttttgtcctttttcgaaacgagacaaatcg\n",
      "gcca-\n",
      ">G000006925|4379878|4379942|4379878|4379942|+\n",
      "-tatcgacgcgatacgtccctggcggtaattcttgcccattttcaaaacgcgataaatcg\n",
      "gccac\n",
      ">G000299455|3362286|3384700|3384402|3384466|+\n",
      "-tatcgacacgataaacccctggcgttattttttgtcctttttcgaaacgagacaaatcg\n",
      "gccac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (732305, 732444), '+'), (('G000299455', (2174641, 2181396)), (2180701, 2180840), '+'), (('G000299455', (2910238, 2916987)), (2910788, 2910927), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1064\n",
      "\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|732305|732444|+\n",
      "-gctcaggaccgcgatacctttgatccgacgcatgatctggatgcgtactgtgattttgc\n",
      "ttccggcaccatcaatattgccattatggatggtaaggtctggcgtctgctgaatggctt\n",
      "taagctgttccgcgaaaaac\n",
      ">G000299455|2174641|2181396|2180701|2180840|+\n",
      "-gagcaggaccgtgatacgtttgatccgacgcatgatattgacctttactgcgatcaggc\n",
      "cagcggtcttgtgaatattgccattatggacggtacggtctggcgtctgctgaatggctt\n",
      "taagctgttccgcgaaaaac\n",
      ">G000299455|2910238|2916987|2910788|2910927|-\n",
      "aaaacaggatcgtgacacgttcgacccgacatatgatatcgaccttttctgtgatcaggc\n",
      "cagtggtcttgtgaatattgccattatggacggtaccgtctggcgtctgctgaatggctt\n",
      "taagcttttccgcgaaaaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4630997, 4631130)), (4630997, 4631130), '+'), (('G000006925', (4696951, 4697084)), (4696951, 4697084), '-'), (('G000299455', (4588613, 4589520)), (4588613, 4588745), '+')]\n",
      ">G000006925|4630997|4631130|4630997|4631130|+\n",
      "-tgagggatccccacaaaattactgctaataaaccagcaccatatttcggtaggttctgg\n",
      "acagatggtttagaactgtgctcagtactgttcgtcttaaaataagtggcgagtgtcgaa\n",
      "gaacattcttcgct\n",
      ">G000006925|4696951|4697084|4696951|4697084|-\n",
      "ctgagggatccccacaaaattactgctaataaaccagcaccatatttcggtaggttctgg\n",
      "acagatggtttagaactgtgctcagtactgttcgtcttaaaataagtggcgagtgtcgaa\n",
      "gaacattcttcgc-\n",
      ">G000299455|4588613|4589520|4588613|4588745|+\n",
      "-tgagggatccccat-gaatcagcgctaataaaccaacaccatattatggtcggttctgg\n",
      "cgaagtgattaagaacggtgatcagcaccattcgttttaaaattagcgaggagtatcgca\n",
      "agatattcttcgct\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3099048, 3099082), '+'), (('G000299455', (856582, 857833)), (856754, 856788), '-'), (('G000299455', (3140498, 3142451)), (3140498, 3140532), '-'), (('G000299455', (4771847, 4785261)), (4784684, 4784718), '+')]\n",
      ">G000006925|3083675|3099254|3099048|3099082|+\n",
      "-aaaggctggaactgattgacagaattaacgatat\n",
      ">G000299455|856582|857833|856754|856788|-\n",
      "aaaaggctggaactgattgacagaattaacgata-\n",
      ">G000299455|3140498|3142451|3140498|3140532|-\n",
      "agaaaaccagaactggttgacagactgaatgata-\n",
      ">G000299455|4771847|4785261|4784684|4784718|+\n",
      "-aaaggctggaactgattgacagaattaacgatat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3259272, 3259293), '+'), (('G000299455', (679634, 695969)), (681595, 681616), '-'), (('G000299455', (1604015, 1621831)), (1608064, 1608085), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3259272|3259293|+\n",
      "-aaatgagcgttttttgatagt\n",
      ">G000299455|679634|695969|681595|681616|-\n",
      "taaatgagcgttttttgatag-\n",
      ">G000299455|1604015|1621831|1608064|1608085|-\n",
      "taaatgagcgttttttgatag-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (728644, 728729), '+'), (('G000299455', (2174641, 2181396)), (2177041, 2177126), '+'), (('G000299455', (2910238, 2916987)), (2914502, 2914587), '-'), (('G000299455', (3601984, 3602069)), (3601984, 3602069), '-')]\n",
      ">G000006925|726235|732999|728644|728729|+\n",
      "-aacgcacgtttacggaatttatccgtgaaggggtgggcgtgcatgcctttaacggcgaa\n",
      "atctttgtgcagccggtctgggatac\n",
      ">G000299455|2174641|2181396|2177041|2177126|+\n",
      "-aacgcacgtttacggaatttatccgtgaaggtgtgggcgttcatgcgtttaacggcgaa\n",
      "atctttgtgcagccggtctgggatac\n",
      ">G000299455|2910238|2916987|2914502|2914587|-\n",
      "aaacgcacgtttacggaatttatccgtgaaggtgtgggcgttcatgcgtttaacggcgaa\n",
      "atctttgtgcagccggtctgggata-\n",
      ">G000299455|3601984|3602069|3601984|3602069|-\n",
      "aaacgcacgtttaccatgatgattcgggaaggtgtggccatgcacgcctttaacggtgaa\n",
      "ctgttcgttcaggccacctgggata-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1, 24671)), (15551, 15590), '+'), (('G000006925', (905848, 907049)), (906082, 906121), '+'), (('G000006925', (1397676, 1398098)), (1397817, 1397855), '-'), (('G000006925', (2062783, 2064325)), (2064133, 2064171), '+'), (('G000299455', (2541782, 2542216)), (2541935, 2541973), '-'), (('G000299455', (2927401, 2931337)), (2930361, 2930399), '+'), (('G000299455', (4431865, 4470166)), (4434943, 4434982), '-')]\n",
      ">G000006925|1|24671|15551|15590|+\n",
      "-attgctctcttctccttgaccttacggtcagtaagaggc\n",
      ">G000006925|905848|907049|906082|906121|+\n",
      "catagcctgcttctccttg-cctttcggcatgtaagaggc\n",
      ">G000006925|1397676|1398098|1397817|1397855|-\n",
      "catagcctgcttctccttg-cctttcggcacgtaagagg-\n",
      ">G000006925|2062783|2064325|2064133|2064171|+\n",
      "catagcctgcttctccttg-cctttcggcacgtaagagg-\n",
      ">G000299455|2541782|2542216|2541935|2541973|-\n",
      "catagcctgcttctccttg-cctttcggcgcgtaagagg-\n",
      ">G000299455|2927401|2931337|2930361|2930399|+\n",
      "-atagcctgcttccccttg-cctttcggcacgtaagaggc\n",
      ">G000299455|4431865|4470166|4434943|4434982|-\n",
      "cattgctctcttctccttgaccttacggtcagtaagagg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4632298, 4632639)), (4632306, 4632593), '-'), (('G000006925', (4708951, 4710002)), (4708951, 4709237), '+'), (('G000299455', (917546, 918643)), (917592, 917878), '+'), (('G000299455', (4770510, 4771607)), (4770556, 4770843), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4632298|4632639|4632306|4632593|-\n",
      "ttgtcgaagctcaccagattttcaccaggatgcacgtcatactcttttttctccggaggc\n",
      "agtgccagcatggactgctgctcttcgagccagcgatcacagggacgggcctggattgtt\n",
      "tcatgctttcgttggttagcgacatcgtgcagccagcgcagaccgtggcggttggctgtt\n",
      "tcaacatcgacagtgctccccatcgggcgcaggcgagtcattagtgggatgtaaaaactg\n",
      "ttacgggtgtactgcaccatccgttccaccttacctttagtctgtgc-\n",
      ">G000006925|4708951|4710002|4708951|4709237|+\n",
      "-tgtcgaagctcaccagattttcaccaggatgcacgtcatactcttttttctccggaggc\n",
      "agtgccagcatggactgctgctcttcgagccagcgatcacagggacgggcctggattgtt\n",
      "tcatgctttcgttggttagcgacatcgtgcagccagcgcagaccgtggcggttggctgtt\n",
      "tcaacatcgacagtgctccccatcgggcgcaggcgagtcattagtgggatgtaaaaactg\n",
      "ttacgggtgtactgcaccatccgttccaccttacctttagtctgtgc-\n",
      ">G000299455|917546|918643|917592|917878|+\n",
      "-tgtcgaagttcaccagattttcatcaagatgcacgtcatactcttttttctccggaggc\n",
      "agtgccagcatggactgctgctcttcgagccagcgatcgcagggacgtgcctggattgtt\n",
      "tcatgctttcgttggttagcgacatcgtgcagccagcgcagaccgtggcggttggctgtt\n",
      "tcaacatcgacagtgatccccatcgggcgcaggcgagtcattagtgggatgtaaaaactg\n",
      "ttacgggtgtactgcaccatccgttccaccttacctttagtctgtgc-\n",
      ">G000299455|4770510|4771607|4770556|4770843|+\n",
      "-tgtcgaagttcaccagattttcatcaagatgcacgtcatactcttttttctccggaggc\n",
      "agtgccagcatggactgctgctcttcgagccagcgatcgcagggacgtgcctggattgtt\n",
      "tcatgctttcgttggttagcgacatcgtgcagccagcgcagaccgtggcggttggctgtt\n",
      "tcaacatcgacagtgatccccatcgggcgcaggcgagtcattagtgggatgtaaaaactg\n",
      "ttacgggtgtactgcaccatccgttccaccttacctttagtctgtgcc\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1209065, 1211067)), (1210213, 1210246), '+'), (('G000006925', (2756889, 2766574)), (2757294, 2757327), '+'), (('G000006925', (2923039, 2950048)), (2948409, 2948442), '+'), (('G000006925', (3101814, 3103074)), (3102220, 3102253), '+'), (('G000006925', (3811174, 3811622)), (3811188, 3811221), '-'), (('G000006925', (4753880, 4754990)), (4754214, 4754247), '-'), (('G000299455', (3226712, 3229721)), (3228863, 3228896), '+')]\n",
      ">G000006925|1209065|1211067|1210213|1210246|+\n",
      "-tcaaagcaatgtgccgcgtgctccgggtggccc\n",
      ">G000006925|2756889|2766574|2757294|2757327|+\n",
      "-tcaaagcaatgtgccgcgtgctccgggtggccc\n",
      ">G000006925|2923039|2950048|2948409|2948442|+\n",
      "-tcaaagcaatgtgccgcgtgctccgggtggccc\n",
      ">G000006925|3101814|3103074|3102220|3102253|+\n",
      "-tcaaagcaatgtgccgcgtgctccgggtggccc\n",
      ">G000006925|3811174|3811622|3811188|3811221|-\n",
      "atcaaagcgatgtgtcgtgtacttcgggttgcc-\n",
      ">G000006925|4753880|4754990|4754214|4754247|-\n",
      "atcaaggcaatgtgccgcgtactccgggtggcc-\n",
      ">G000299455|3226712|3229721|3228863|3228896|+\n",
      "-tcaaagcaatgtgccgcgtgctccgggtggccc\n",
      "\n",
      "\n",
      "109 [(('G000006925', (1, 24671)), (18792, 18842), '+'), (('G000006925', (64491, 216702)), (87535, 87585), '-'), (('G000006925', (64491, 216702)), (148725, 148775), '-'), (('G000006925', (266682, 267450)), (267276, 267326), '-'), (('G000006925', (273344, 278315)), (273468, 273518), '+'), (('G000006925', (278320, 279091)), (278916, 278966), '-'), (('G000006925', (300111, 308544)), (308369, 308419), '-'), (('G000006925', (378637, 384812)), (379233, 379283), '-'), (('G000006925', (459861, 468373)), (465841, 465891), '+'), (('G000006925', (468374, 487224)), (478837, 478887), '+'), (('G000006925', (520755, 528026)), (521349, 521399), '-'), (('G000006925', (545486, 558918)), (558274, 558324), '+'), (('G000006925', (622915, 623662)), (623039, 623089), '+'), (('G000006925', (865887, 867389)), (866745, 866795), '+'), (('G000006925', (984595, 985335)), (985191, 985241), '-'), (('G000006925', (985336, 1008622)), (985554, 985604), '+'), (('G000006925', (1025530, 1063042)), (1049696, 1049746), '+'), (('G000006925', (1189492, 1208094)), (1197772, 1197822), '+'), (('G000006925', (1239481, 1240251)), (1239605, 1239655), '+'), (('G000006925', (1290847, 1306886)), (1295197, 1295247), '+'), (('G000006925', (1321493, 1385605)), (1366048, 1366098), '-'), (('G000006925', (1398496, 1399271)), (1398622, 1398672), '+'), (('G000006925', (1409877, 1410649)), (1410475, 1410525), '-'), (('G000006925', (1424264, 1425034)), (1424860, 1424910), '-'), (('G000006925', (1425133, 1465118)), (1444404, 1444454), '+'), (('G000006925', (1509064, 1578296)), (1526803, 1526853), '+'), (('G000006925', (1509064, 1578296)), (1542890, 1542940), '+'), (('G000006925', (1509064, 1578296)), (1568785, 1568835), '-'), (('G000006925', (1509064, 1578296)), (1574743, 1574793), '-'), (('G000006925', (1670306, 1692658)), (1673496, 1673546), '+'), (('G000006925', (1729332, 1741706)), (1736568, 1736618), '-'), (('G000006925', (1758378, 1759147)), (1758502, 1758552), '+'), (('G000006925', (1759148, 1776249)), (1766652, 1766702), '+'), (('G000006925', (1759148, 1776249)), (1769626, 1769676), '-'), (('G000006925', (1776250, 1792578)), (1781037, 1781087), '+'), (('G000006925', (1793868, 1802424)), (1795723, 1795773), '-'), (('G000006925', (1842508, 1869155)), (1850643, 1850693), '-'), (('G000006925', (1842508, 1869155)), (1856581, 1856631), '-'), (('G000006925', (1940277, 1941051)), (1940401, 1940451), '+'), (('G000006925', (1976680, 1977448)), (1977274, 1977324), '-'), (('G000006925', (2036705, 2044875)), (2037749, 2037799), '-'), (('G000006925', (2050169, 2051841)), (2051667, 2051717), '-'), (('G000006925', (2088344, 2103602)), (2095164, 2095214), '+'), (('G000006925', (2120115, 2138771)), (2135242, 2135292), '-'), (('G000006925', (2172359, 2184266)), (2177916, 2177966), '+'), (('G000006925', (2187913, 2198767)), (2197481, 2197531), '-'), (('G000006925', (2198768, 2199537)), (2198893, 2198943), '+'), (('G000006925', (2199538, 2210687)), (2206145, 2206195), '-'), (('G000006925', (2211287, 2225774)), (2220757, 2220807), '+'), (('G000006925', (2235234, 2236005)), (2235828, 2235878), '-'), (('G000006925', (2242494, 2346249)), (2287569, 2287619), '+'), (('G000006925', (2374481, 2472008)), (2396229, 2396279), '-'), (('G000006925', (2756085, 2756853)), (2756679, 2756729), '-'), (('G000006925', (2816684, 2833035)), (2825325, 2825375), '+'), (('G000006925', (2849870, 2870340)), (2863434, 2863484), '-'), (('G000006925', (2964578, 2976271)), (2964623, 2964673), '-'), (('G000006925', (2976303, 3015537)), (3014757, 3014807), '+'), (('G000006925', (3016790, 3036697)), (3021588, 3021638), '+'), (('G000006925', (3038529, 3039299)), (3038653, 3038703), '+'), (('G000006925', (3104758, 3106299)), (3104883, 3104933), '+'), (('G000006925', (3179574, 3180459)), (3180284, 3180334), '-'), (('G000006925', (3180744, 3203996)), (3182175, 3182225), '-'), (('G000006925', (3237769, 3244530)), (3238363, 3238413), '-'), (('G000006925', (3268417, 3281594)), (3274365, 3274415), '-'), (('G000006925', (3322761, 3361739)), (3361562, 3361612), '-'), (('G000006925', (3361966, 3403725)), (3362562, 3362612), '-'), (('G000006925', (3608503, 3611864)), (3609102, 3609152), '-'), (('G000006925', (3612083, 3637192)), (3624357, 3624407), '+'), (('G000006925', (3637330, 3640023)), (3637454, 3637504), '+'), (('G000006925', (3658114, 3676993)), (3665284, 3665334), '+'), (('G000006925', (3693106, 3702974)), (3693702, 3693752), '-'), (('G000006925', (3703179, 3717854)), (3703774, 3703824), '-'), (('G000006925', (3717879, 3736288)), (3726058, 3726108), '-'), (('G000006925', (3736321, 3788106)), (3743146, 3743196), '+'), (('G000006925', (3815875, 3829465)), (3818129, 3818179), '+'), (('G000006925', (3831593, 3850590)), (3834610, 3834660), '+'), (('G000006925', (3858569, 3864367)), (3864193, 3864243), '-'), (('G000006925', (3939882, 3952896)), (3943428, 3943478), '-'), (('G000006925', (4002899, 4012437)), (4003493, 4003543), '-'), (('G000006925', (4069200, 4076620)), (4076448, 4076498), '-'), (('G000006925', (4091607, 4092380)), (4091732, 4091782), '+'), (('G000006925', (4094093, 4116338)), (4102438, 4102488), '+'), (('G000006925', (4174522, 4188276)), (4177157, 4177207), '-'), (('G000006925', (4256570, 4288375)), (4285103, 4285153), '+'), (('G000006925', (4288513, 4315006)), (4314831, 4314881), '-'), (('G000006925', (4362175, 4367702)), (4367058, 4367108), '+'), (('G000006925', (4378758, 4379529)), (4378882, 4378932), '+'), (('G000006925', (4389053, 4390969)), (4389178, 4389228), '+'), (('G000006925', (4392644, 4394283)), (4393243, 4393293), '-'), (('G000006925', (4397536, 4410956)), (4410782, 4410832), '-'), (('G000006925', (4462252, 4478752)), (4462377, 4462427), '+'), (('G000006925', (4478753, 4490457)), (4488515, 4488565), '+'), (('G000006925', (4497401, 4528664)), (4515464, 4515514), '-'), (('G000006925', (4535976, 4543278)), (4541104, 4541154), '-'), (('G000006925', (4557883, 4573403)), (4570317, 4570367), '-'), (('G000006925', (4618262, 4618312)), (4618262, 4618312), '-'), (('G000006925', (4645852, 4646621)), (4646446, 4646496), '-'), (('G000006925', (4656735, 4658205)), (4657330, 4657380), '-'), (('G000006925', (4785482, 4786253)), (4785608, 4785658), '+'), (('G000299455', (91523, 102755)), (92122, 92172), '-'), (('G000299455', (696017, 696792)), (696143, 696193), '+'), (('G000299455', (941725, 942502)), (941852, 941902), '+'), (('G000299455', (1581872, 1582655)), (1582471, 1582521), '-'), (('G000299455', (2052398, 2064114)), (2062551, 2062601), '-'), (('G000299455', (3182222, 3183821)), (3183173, 3183223), '+'), (('G000299455', (3672841, 3680073)), (3679893, 3679943), '-'), (('G000299455', (4588613, 4589520)), (4588869, 4588919), '+'), (('G000299455', (4592900, 4593681)), (4593501, 4593551), '-'), (('G000299455', (5323356, 5324120)), (5323942, 5323992), '-')]\n",
      ">G000006925|1|24671|18792|18842|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|64491|216702|87535|87585|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|64491|216702|148725|148775|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|266682|267450|267276|267326|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|273344|278315|273468|273518|+\n",
      "-agcttcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|278320|279091|278916|278966|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|300111|308544|308369|308419|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|378637|384812|379233|379283|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|459861|468373|465841|465891|+\n",
      "-agattcagattatgtcgctcaatgtgctgagtgtaacgcttgctgataac\n",
      ">G000006925|468374|487224|478837|478887|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|520755|528026|521349|521399|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|545486|558918|558274|558324|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|622915|623662|623039|623089|+\n",
      "-agcttcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|865887|867389|866745|866795|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|984595|985335|985191|985241|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|985336|1008622|985554|985604|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1025530|1063042|1049696|1049746|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1189492|1208094|1197772|1197822|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1239481|1240251|1239605|1239655|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1290847|1306886|1295197|1295247|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1321493|1385605|1366048|1366098|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1398496|1399271|1398622|1398672|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1409877|1410649|1410475|1410525|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1424264|1425034|1424860|1424910|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1425133|1465118|1444404|1444454|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1509064|1578296|1526803|1526853|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1509064|1578296|1542890|1542940|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1509064|1578296|1568785|1568835|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1509064|1578296|1574743|1574793|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1670306|1692658|1673496|1673546|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1729332|1741706|1736568|1736618|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1758378|1759147|1758502|1758552|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1759148|1776249|1766652|1766702|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1759148|1776249|1769626|1769676|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1776250|1792578|1781037|1781087|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|1793868|1802424|1795723|1795773|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1842508|1869155|1850643|1850693|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1842508|1869155|1856581|1856631|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|1940277|1941051|1940401|1940451|+\n",
      "-agattcaggttatgccgctcaattcgctgcgtatatcgcttgctgattac\n",
      ">G000006925|1976680|1977448|1977274|1977324|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2036705|2044875|2037749|2037799|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2050169|2051841|2051667|2051717|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2088344|2103602|2095164|2095214|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2120115|2138771|2135242|2135292|-\n",
      "cagcttcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2172359|2184266|2177916|2177966|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2187913|2198767|2197481|2197531|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000006925|2198768|2199537|2198893|2198943|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2199538|2210687|2206145|2206195|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2211287|2225774|2220757|2220807|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2235234|2236005|2235828|2235878|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2242494|2346249|2287569|2287619|+\n",
      "-agattcagattatgtcgttcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2374481|2472008|2396229|2396279|-\n",
      "cagcttcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2756085|2756853|2756679|2756729|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2816684|2833035|2825325|2825375|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|2849870|2870340|2863434|2863484|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2964578|2976271|2964623|2964673|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|2976303|3015537|3014757|3014807|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3016790|3036697|3021588|3021638|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3038529|3039299|3038653|3038703|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3104758|3106299|3104883|3104933|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3179574|3180459|3180284|3180334|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3180744|3203996|3182175|3182225|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3237769|3244530|3238363|3238413|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3268417|3281594|3274365|3274415|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3322761|3361739|3361562|3361612|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3361966|3403725|3362562|3362612|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3608503|3611864|3609102|3609152|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3612083|3637192|3624357|3624407|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3637330|3640023|3637454|3637504|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3658114|3676993|3665284|3665334|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3693106|3702974|3693702|3693752|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3703179|3717854|3703774|3703824|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3717879|3736288|3726058|3726108|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3736321|3788106|3743146|3743196|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3815875|3829465|3818129|3818179|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3831593|3850590|3834610|3834660|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|3858569|3864367|3864193|3864243|-\n",
      "cagcttcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|3939882|3952896|3943428|3943478|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4002899|4012437|4003493|4003543|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4069200|4076620|4076448|4076498|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4091607|4092380|4091732|4091782|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4094093|4116338|4102438|4102488|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4174522|4188276|4177157|4177207|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4256570|4288375|4285103|4285153|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4288513|4315006|4314831|4314881|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4362175|4367702|4367058|4367108|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4378758|4379529|4378882|4378932|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4389053|4390969|4389178|4389228|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4392644|4394283|4393243|4393293|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4397536|4410956|4410782|4410832|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4462252|4478752|4462377|4462427|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4478753|4490457|4488515|4488565|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000006925|4497401|4528664|4515464|4515514|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4535976|4543278|4541104|4541154|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4557883|4573403|4570317|4570367|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4618262|4618312|4618262|4618312|-\n",
      "caggttcagattatgtcgctctatccgctgagtgcatttcttgcttataa-\n",
      ">G000006925|4645852|4646621|4646446|4646496|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4656735|4658205|4657330|4657380|-\n",
      "cagattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataa-\n",
      ">G000006925|4785482|4786253|4785608|4785658|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000299455|91523|102755|92122|92172|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000299455|696017|696792|696143|696193|+\n",
      "-agattcaggttatgccgctcaattcgctgcgtatatcgcttgctgattac\n",
      ">G000299455|941725|942502|941852|941902|+\n",
      "-agattcaggttatgccgctcaattcgctgcgtatatcgcttgctgattac\n",
      ">G000299455|1581872|1582655|1582471|1582521|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000299455|2052398|2064114|2062551|2062601|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000299455|3182222|3183821|3183173|3183223|+\n",
      "-agattcaggttatgccgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000299455|3672841|3680073|3679893|3679943|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000299455|4588613|4589520|4588869|4588919|+\n",
      "-agattcagattatgtcgctcaatgcgctgagtgtaacgcttgctgataac\n",
      ">G000299455|4592900|4593681|4593501|4593551|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      ">G000299455|5323356|5324120|5323942|5323992|-\n",
      "cagattcaggttatgccgctcaattcgctgcgtatatcgcttgctgatta-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3898713, 3912668)), (3900628, 3900765), '+'), (('G000299455', (31186, 43870)), (33101, 33238), '+'), (('G000299455', (4578866, 4579002)), (4578868, 4579002), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 109\r",
      "  101 / 109\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 109\r",
      "   10 / 109\r",
      "   20 / 109\r",
      "   30 / 109\r",
      "   40 / 109\r",
      "   50 / 109\r",
      "   60 / 109\r",
      "   70 / 109\r",
      "   80 / 109\r",
      "   90 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 108  f\b\b\r",
      "STEP     2 / 108  f\b\b\r",
      "STEP     3 / 108  f\b\b\r",
      "STEP     4 / 108  f\b\b\r",
      "STEP     5 / 108  f\b\b\r",
      "STEP     6 / 108  f\b\b\r",
      "STEP     7 / 108  f\b\b\r",
      "STEP     8 / 108  f\b\b\r",
      "STEP     9 / 108  f\b\b\r",
      "STEP    10 / 108  f\b\b\r",
      "STEP    11 / 108  f\b\b\r",
      "STEP    12 / 108  f\b\b\r",
      "STEP    13 / 108  f\b\b\r",
      "STEP    14 / 108  f\b\b\r",
      "STEP    15 / 108  f\b\b\r",
      "STEP    16 / 108  f\b\b\r",
      "STEP    17 / 108  f\b\b\r",
      "STEP    18 / 108  f\b\b\r",
      "STEP    19 / 108  f\b\b\r",
      "STEP    20 / 108  f\b\b\r",
      "STEP    21 / 108  f\b\b\r",
      "STEP    22 / 108  f\b\b\r",
      "STEP    23 / 108  f\b\b\r",
      "STEP    24 / 108  f\b\b\r",
      "STEP    25 / 108  f\b\b\r",
      "STEP    26 / 108  f\b\b\r",
      "STEP    27 / 108  f\b\b\r",
      "STEP    28 / 108  f\b\b\r",
      "STEP    29 / 108  f\b\b\r",
      "STEP    30 / 108  f\b\b\r",
      "STEP    31 / 108  f\b\b\r",
      "STEP    32 / 108  f\b\b\r",
      "STEP    33 / 108  f\b\b\r",
      "STEP    34 / 108  f\b\b\r",
      "STEP    35 / 108  f\b\b\r",
      "STEP    36 / 108  f\b\b\r",
      "STEP    37 / 108  f\b\b\r",
      "STEP    38 / 108  f\b\b\r",
      "STEP    39 / 108  f\b\b\r",
      "STEP    40 / 108  f\b\b\r",
      "STEP    41 / 108  f\b\b\r",
      "STEP    42 / 108  f\b\b\r",
      "STEP    43 / 108  f\b\b\r",
      "STEP    44 / 108  f\b\b\r",
      "STEP    45 / 108  f\b\b\r",
      "STEP    46 / 108  f\b\b\r",
      "STEP    47 / 108  f\b\b\r",
      "STEP    48 / 108  f\b\b\r",
      "STEP    49 / 108  f\b\b\r",
      "STEP    50 / 108  f\b\b\r",
      "STEP    51 / 108  f\b\b\r",
      "STEP    52 / 108  f\b\b\r",
      "STEP    53 / 108  f\b\b\r",
      "STEP    54 / 108  f\b\b\r",
      "STEP    55 / 108  f\b\b\r",
      "STEP    56 / 108  f\b\b\r",
      "STEP    57 / 108  f\b\b\r",
      "STEP    58 / 108  f\b\b\r",
      "STEP    59 / 108  f\b\b\r",
      "STEP    60 / 108  f\b\b\r",
      "STEP    61 / 108  f\b\b\r",
      "STEP    62 / 108  f\b\b\r",
      "STEP    63 / 108  f\b\b\r",
      "STEP    64 / 108  f\b\b\r",
      "STEP    65 / 108  f\b\b\r",
      "STEP    66 / 108  f\b\b\r",
      "STEP    67 / 108  f\b\b\r",
      "STEP    68 / 108  f\b\b\r",
      "STEP    69 / 108  f\b\b\r",
      "STEP    70 / 108  f\b\b\r",
      "STEP    71 / 108  f\b\b\r",
      "STEP    72 / 108  f\b\b\r",
      "STEP    73 / 108  f\b\b\r",
      "STEP    74 / 108  f\b\b\r",
      "STEP    75 / 108  f\b\b\r",
      "STEP    76 / 108  f\b\b\r",
      "STEP    77 / 108  f\b\b\r",
      "STEP    78 / 108  f\b\b\r",
      "STEP    79 / 108  f\b\b\r",
      "STEP    80 / 108  f\b\b\r",
      "STEP    81 / 108  f\b\b\r",
      "STEP    82 / 108  f\b\b\r",
      "STEP    83 / 108  f\b\b\r",
      "STEP    84 / 108  f\b\b\r",
      "STEP    85 / 108  f\b\b\r",
      "STEP    86 / 108  f\b\b\r",
      "STEP    87 / 108  f\b\b\r",
      "STEP    88 / 108  f\b\b\r",
      "STEP    89 / 108  f\b\b\r",
      "STEP    90 / 108  f\b\b\r",
      "STEP    91 / 108  f\b\b\r",
      "STEP    92 / 108  f\b\b\r",
      "STEP    93 / 108  f\b\b\r",
      "STEP    94 / 108  f\b\b\r",
      "STEP    95 / 108  f\b\b\r",
      "STEP    96 / 108  f\b\b\r",
      "STEP    97 / 108  f\b\b\r",
      "STEP    98 / 108  f\b\b\r",
      "STEP    99 / 108  f\b\b\r",
      "STEP   100 / 108  f\b\b\r",
      "STEP   101 / 108  f\b\b\r",
      "STEP   102 / 108  f\b\b\n",
      "Reallocating..done. *alloclen = 1102\n",
      "\r",
      "STEP   103 / 108  d\b\b\r",
      "STEP   104 / 108  d\b\b h\r",
      "STEP   105 / 108  d\b\b h\r",
      "STEP   106 / 108  d\b\b h\r",
      "STEP   107 / 108  d\b\b h\r",
      "STEP   108 / 108  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 109\r",
      "   10 / 109\r",
      "   20 / 109\r",
      "   30 / 109\r",
      "   40 / 109\r",
      "   50 / 109\r",
      "   60 / 109\r",
      "   70 / 109\r",
      "   80 / 109\r",
      "   90 / 109\r",
      "  100 / 109\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 108  f\b\b\r",
      "STEP     2 / 108  f\b\b\r",
      "STEP     3 / 108  f\b\b\r",
      "STEP     4 / 108  f\b\b\r",
      "STEP     5 / 108  f\b\b\r",
      "STEP     6 / 108  f\b\b\r",
      "STEP     7 / 108  f\b\b\r",
      "STEP     8 / 108  f\b\b\r",
      "STEP     9 / 108  f\b\b\r",
      "STEP    10 / 108  f\b\b\r",
      "STEP    11 / 108  f\b\b\r",
      "STEP    12 / 108  f\b\b\r",
      "STEP    13 / 108  f\b\b\r",
      "STEP    14 / 108  f\b\b\r",
      "STEP    15 / 108  f\b\b\r",
      "STEP    16 / 108  f\b\b\r",
      "STEP    17 / 108  f\b\b\r",
      "STEP    18 / 108  f\b\b\r",
      "STEP    19 / 108  f\b\b\r",
      "STEP    20 / 108  f\b\b\r",
      "STEP    21 / 108  f\b\b\r",
      "STEP    22 / 108  f\b\b\r",
      "STEP    23 / 108  f\b\b\r",
      "STEP    24 / 108  f\b\b\r",
      "STEP    25 / 108  f\b\b\r",
      "STEP    26 / 108  f\b\b\r",
      "STEP    27 / 108  f\b\b\r",
      "STEP    28 / 108  f\b\b\r",
      "STEP    29 / 108  f\b\b\r",
      "STEP    30 / 108  f\b\b\r",
      "STEP    31 / 108  f\b\b\r",
      "STEP    32 / 108  f\b\b\r",
      "STEP    33 / 108  f\b\b\r",
      "STEP    34 / 108  f\b\b\r",
      "STEP    35 / 108  f\b\b\r",
      "STEP    36 / 108  f\b\b\r",
      "STEP    37 / 108  f\b\b\r",
      "STEP    38 / 108  f\b\b\r",
      "STEP    39 / 108  f\b\b\r",
      "STEP    40 / 108  f\b\b\r",
      "STEP    41 / 108  f\b\b\r",
      "STEP    42 / 108  f\b\b\r",
      "STEP    43 / 108  f\b\b\r",
      "STEP    44 / 108  f\b\b\r",
      "STEP    45 / 108  f\b\b\r",
      "STEP    46 / 108  f\b\b\r",
      "STEP    47 / 108  f\b\b\r",
      "STEP    48 / 108  f\b\b\r",
      "STEP    49 / 108  f\b\b\r",
      "STEP    50 / 108  f\b\b\r",
      "STEP    51 / 108  f\b\b\r",
      "STEP    52 / 108  f\b\b\r",
      "STEP    53 / 108  f\b\b\r",
      "STEP    54 / 108  f\b\b\r",
      "STEP    55 / 108  f\b\b\r",
      "STEP    56 / 108  f\b\b\r",
      "STEP    57 / 108  f\b\b\r",
      "STEP    58 / 108  f\b\b\r",
      "STEP    59 / 108  f\b\b\r",
      "STEP    60 / 108  f\b\b\r",
      "STEP    61 / 108  f\b\b\r",
      "STEP    62 / 108  f\b\b\r",
      "STEP    63 / 108  f\b\b\r",
      "STEP    64 / 108  f\b\b\r",
      "STEP    65 / 108  f\b\b\r",
      "STEP    66 / 108  f\b\b\r",
      "STEP    67 / 108  f\b\b\r",
      "STEP    68 / 108  f\b\b\r",
      "STEP    69 / 108  f\b\b\r",
      "STEP    70 / 108  f\b\b\r",
      "STEP    71 / 108  f\b\b\r",
      "STEP    72 / 108  f\b\b\r",
      "STEP    73 / 108  f\b\b\r",
      "STEP    74 / 108  f\b\b\r",
      "STEP    75 / 108  f\b\b\r",
      "STEP    76 / 108  f\b\b\r",
      "STEP    77 / 108  f\b\b\r",
      "STEP    78 / 108  f\b\b\r",
      "STEP    79 / 108  f\b\b\r",
      "STEP    80 / 108  f\b\b\r",
      "STEP    81 / 108  f\b\b\r",
      "STEP    82 / 108  f\b\b\r",
      "STEP    83 / 108  f\b\b\r",
      "STEP    84 / 108  f\b\b\r",
      "STEP    85 / 108  f\b\b\r",
      "STEP    86 / 108  f\b\b\r",
      "STEP    87 / 108  f\b\b\r",
      "STEP    88 / 108  f\b\b\r",
      "STEP    89 / 108  f\b\b\r",
      "STEP    90 / 108  f\b\b\r",
      "STEP    91 / 108  f\b\b\r",
      "STEP    92 / 108  f\b\b\r",
      "STEP    93 / 108  f\b\b\r",
      "STEP    94 / 108  f\b\b\r",
      "STEP    95 / 108  f\b\b\r",
      "STEP    96 / 108  f\b\b\r",
      "STEP    97 / 108  f\b\b\r",
      "STEP    98 / 108  f\b\b\r",
      "STEP    99 / 108  f\b\b\r",
      "STEP   100 / 108  f\b\b\r",
      "STEP   101 / 108  f\b\b\r",
      "STEP   102 / 108  f\b\b\r",
      "STEP   103 / 108  f\b\b\r",
      "STEP   104 / 108  f\b\b\r",
      "STEP   105 / 108  f\b\b\r",
      "STEP   106 / 108  d\b\b\r",
      "STEP   107 / 108  d\b\b h\r",
      "STEP   108 / 108  d\b\b h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3898713|3912668|3900628|3900765|+\n",
      "ccgctcaaggcttgaacaacatcgcgcttatcgttaaggtaagcgcgtttttttttaccc\n",
      "gccaggacaagaccatgatcaccgttgcccttatagacgatcacctcatcgtccgctccg\n",
      "gctttgcgcagctgctg\n",
      ">G000299455|31186|43870|33101|33238|+\n",
      "ccgctcaaggcttgaacaacatcgcgcttatcgttaaggtaagcgcgtttttttttaccc\n",
      "gccaggacaagaccatgatcaccgttgcccttatagacgatcacctcatcgtccgctccg\n",
      "gctttgcgcagctgctg\n",
      ">G000299455|4578866|4579002|4578868|4579002|+\n",
      "ccactcaaggcctgaacagcggcgcgcttatcgttaaggtaagcgcg--cttttttaccc\n",
      "gccaggacaagactatgat-agcgttgcccttatagacgatcacattatcgtccactcca\n",
      "aatttgcgcaactactg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1578298, 1580868)), (1579906, 1580273), '+'), (('G000006925', (2062783, 2064325)), (2062783, 2063150), '+'), (('G000299455', (2927401, 2931337)), (2929019, 2929383), '+')]\n",
      ">G000006925|1578298|1580868|1579906|1580273|+\n",
      "cggagactcaggatcaacggcaatgctgaccaccgtcttttcctgttgtgggttctgttg\n",
      "ctggtgggcgtgaggcagcggcgcaagattttttgtgcgctgcttcagtatgctggtggc\n",
      "ggtctgctctcccggcacgatgtcgctttcacggtacattgagcggattttttccgcacg\n",
      "caaccccagcgaacgacgtaataccgcttccggtagcgcgtccgccacctgattgcggac\n",
      "cgcccaccaggataattcagccagagataattcacgctcctgcgtaccgcttattgcgtg\n",
      "accgatgacgtcaatcatccatgctgacaggttttgatgagcaagttgctcgagtgattc\n",
      "ggatgtc\n",
      ">G000006925|2062783|2064325|2062783|2063150|+\n",
      "cggagattccggatcaacggcaatgctgaccaccgtctcttcctgtggtggattctgttg\n",
      "ctggtgggcgtgaggcagcggcgcaagattttttgtgcgctgcttcagtatgctggtggc\n",
      "ggtctgctctcccggcacgatgtcgctttcacggtacattgagcggattttttccgcacg\n",
      "caaccccagcgaacgacgtaataccgcttccggtagcgcgtccgccacctgattgcggac\n",
      "cgcccaccaggataattcagccagagataattcacgctcctgcgtaccgcttattgcgtg\n",
      "accgatgacgtcaatcatccatgctgacaggttttgatgagcaagttgctcgagtgattc\n",
      "ggatgtc\n",
      ">G000299455|2927401|2931337|2929019|2929383|+\n",
      "cggagactcaggatcaacggcaatgctgaccaccgtctcttcctgtggcgggtt---ttg\n",
      "ctggtgggcgtgaggcagcggcgcaagattttttgtgcgctgcttcagtatgctggtggc\n",
      "ggtctgctctcccggtacgatgtcgctttcacggtacattgagcggattttttccgcacg\n",
      "caaccccagcgaacgacgtaataccgcttccggtagcgcgtccgccacctgattgcggac\n",
      "cgcccaccaggataattcagccagcgataattcccgttcctgcgagccattcattgcatg\n",
      "gcgtatgacgtcaatcatccatgcaaacaggttttggtgagcaagttgcccgagtgattc\n",
      "ggaggtc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1634479, 1637262)), (1636660, 1636878), '-'), (('G000299455', (2585656, 2594816)), (2586866, 2587084), '+'), (('G000299455', (2585656, 2594816)), (2589350, 2589568), '+'), (('G000299455', (3440781, 3470001)), (3442976, 3443194), '+')]\n",
      ">G000006925|1634479|1637262|1636660|1636878|-\n",
      "atgtaaaaacggaaatctgcgttttaacaggattctcgagcatcgggaaccattctaccc\n",
      "ccagatcccagctaccttcgcgtacacctgagttgccgccgtttatgccgacgttaccgg\n",
      "taagaacggaaagcatggcaatagcgcgggacgtttgttcgccgttggagtgtcgttgtg\n",
      "gcccccaaccctgacaaatataagcaggttttgctgaa-\n",
      ">G000299455|2585656|2594816|2586866|2587084|+\n",
      "-tgtaaagacggaaatctgcgttttaacagggttctcaagcatcgggaaccattctaccc\n",
      "ccagatcccagctaccttcgcgtacacctgagttgccgccgtttatgccgacgttaccgg\n",
      "tgagaacggaaagcatggcaatagcgcgggacgtttgctcgccgttggaatgtcgttgtg\n",
      "gcccccaaccctgacaaatataagcaggttttgctgaac\n",
      ">G000299455|2585656|2594816|2589350|2589568|+\n",
      "-gctgaagcaggagattgacgttttgaccggattatccagcaccggcaggcgttctatgg\n",
      "taatggtataggtcgattcacgcgcgccactatttccgccgctgatgccgacattgcccg\n",
      "tcaaaataggtaacatagcaatagcgcgtgcagtcagttcgccgtttgcctggcgttgcg\n",
      "gcccccagccctggcagatataagcgggttttgctgtgc\n",
      ">G000299455|3440781|3470001|3442976|3443194|+\n",
      "-cataaacatagaaatgctggtctggatcgggttttcgagggtcggcatacggacaaacg\n",
      "gcaggctataggaaccttcacgcgcgccggtgttaccgccgttaatgccgacgttaccgg\n",
      "tcagaatcgccagcatcgagatagcgcgagtcgcgatttcgccgttggcgtggcgctgtg\n",
      "gcccccatccctggctgataaacgccggtttggtactgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3049506, 3049636), '-'), (('G000299455', (942831, 956197)), (947216, 947346), '+'), (('G000299455', (3697938, 3726006)), (3708556, 3708686), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3049506|3049636|-\n",
      "aagccacgagaagctggcacaactggtggcgatgctggcgcaatttgaacagcatattga\n",
      "ggatgactcgtcgctggctgaggtgttgccgagcgtttataacaaatatccggtgcgcta\n",
      "tcgcgactac-\n",
      ">G000299455|942831|956197|947216|947346|+\n",
      "-agccacgagaagctggcacaactggtagagatgttggcgcagtttgaacagcatattga\n",
      "ggatgactcgccgctggctgaggtgttgccgagcgtttataacaaatatccggtgcgcta\n",
      "tcgcgactaca\n",
      ">G000299455|3697938|3726006|3708556|3708686|+\n",
      "-gatatggccaagcttcagcaacttgttgccctgctggtacgcttcgaaaaactgcttga\n",
      "gtccgacgcgccattagcagaagtgctgccttccatctacaaacagcatgaagagcgcta\n",
      "cgccggttata\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1945366, 1956632)), (1949896, 1949987), '-'), (('G000006925', (3681874, 3691741)), (3686936, 3687027), '-'), (('G000299455', (211788, 221655)), (216502, 216593), '+'), (('G000299455', (2133804, 2148224)), (2143602, 2143693), '+'), (('G000299455', (3316999, 3326584)), (3324704, 3324795), '-')]\n",
      ">G000006925|1945366|1956632|1949896|1949987|-\n",
      "gcggcgattattcaaccggggcagcacaggtgatcatgccgcatgtggtcggttctgtag\n",
      "aggtgtatgaacagcaggcctcctggccgct-\n",
      ">G000006925|3681874|3691741|3686936|3687027|-\n",
      "tgggggattattcgaccggtgcggcacaggcgatcatgccgtatgtcgtaggtggcagcg\n",
      "aagtttatcaacagcaaaccagttggccgct-\n",
      ">G000299455|211788|221655|216502|216593|+\n",
      "-gggggattattcgaccggcgcggcacaggcgatcatgccgtatgtcgtgggtggtagcg\n",
      "aagtttatcaacagcaaaccagttggccgctg\n",
      ">G000299455|2133804|2148224|2143602|2143693|+\n",
      "-cggcgattattcaaccggggcagcacaggtgatcatgccgcatgtggtcggttctgtag\n",
      "aggtgtatgaacagcagacctcctggccgctg\n",
      ">G000299455|3316999|3326584|3324704|3324795|-\n",
      "gcggagattactctaccggtgctgcgcaggtgatcctgccgcgcgtggttggctcaatgg\n",
      "aagtgtatgaacagcaaacctcctggccgct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (77403, 77424), '-'), (('G000299455', (2818030, 2881682)), (2871446, 2871467), '-'), (('G000299455', (4334060, 4383441)), (4370544, 4370565), '+')]\n",
      ">G000006925|64491|216702|77403|77424|-\n",
      "gcaaccgtaactttgaaggcc-\n",
      ">G000299455|2818030|2881682|2871446|2871467|-\n",
      "gcaaccgtaactttgaaggcc-\n",
      ">G000299455|4334060|4383441|4370544|4370565|+\n",
      "-caaccgtaactttgaaggccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4652860, 4653326)), (4652860, 4652935), '+'), (('G000006925', (4815962, 4816132)), (4815962, 4816036), '-'), (('G000299455', (5311401, 5311590)), (5311496, 5311571), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1184\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1184\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4652860|4653326|4652860|4652935|+\n",
      "agagacctgaacattgtgacgatgatctgtgtcggcgcacaaatcatgtctgattacggc\n",
      "cgtttaatgtgattt\n",
      ">G000006925|4815962|4816132|4815962|4816036|-\n",
      "agcgacttgagaattgtggcgaagatctgtgttggcgcacaaatcaacggggattactgt\n",
      "cgtttaatgtgatt-\n",
      ">G000299455|5311401|5311590|5311496|5311571|+\n",
      "agtgacttgaaagttgtgacgatgatctgtgttggcgcacaaatcatgtctgattacgac\n",
      "cgtttaatgtaattt\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1, 24671)), (15652, 15675), '-'), (('G000006925', (905848, 907049)), (906202, 906227), '-'), (('G000006925', (1397676, 1398098)), (1397712, 1397736), '+'), (('G000006925', (2062783, 2064325)), (2064253, 2064278), '-'), (('G000006925', (4827189, 4827308)), (4827282, 4827308), '+'), (('G000299455', (211707, 211774)), (211744, 211767), '+'), (('G000299455', (2541782, 2542216)), (2541830, 2541854), '+'), (('G000299455', (2927401, 2931337)), (2930480, 2930505), '-'), (('G000299455', (4431865, 4470166)), (4434858, 4434881), '+')]\n",
      ">G000006925|1|24671|15652|15675|-\n",
      "--ttagctgaacggcagatagagaa-\n",
      ">G000006925|905848|907049|906202|906227|-\n",
      "tgaacaccaaaaggcagatagagaa-\n",
      ">G000006925|1397676|1398098|1397712|1397736|+\n",
      "--gccactgtaaggcagacagagaaa\n",
      ">G000006925|2062783|2064325|2064253|2064278|-\n",
      "tgaacaccaaaaggcagatagagaa-\n",
      ">G000006925|4827189|4827308|4827282|4827308|+\n",
      "tgaacaccaaaaggtaggtagagaaa\n",
      ">G000299455|211707|211774|211744|211767|+\n",
      "---tcaccgaaaggcagacagagaaa\n",
      ">G000299455|2541782|2542216|2541830|2541854|+\n",
      "--gccaccgtaaggcagacagagaaa\n",
      ">G000299455|2927401|2931337|2930480|2930505|-\n",
      "tgaacactgaaaggcagatagagaa-\n",
      ">G000299455|4431865|4470166|4434858|4434881|+\n",
      "---tagctgaacggcagatagagaaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (739473, 745780)), (745648, 745780), '-'), (('G000006925', (1921807, 1928114)), (1921807, 1921939), '+'), (('G000299455', (3585529, 3586267)), (3585529, 3585662), '+')]\n",
      ">G000006925|739473|745780|745648|745780|-\n",
      "tctttcagtacacctgaaatctttactgccatattcaccccac-aaaaagcccgcctgaa\n",
      "ccagcgggctgtcataacactgtgttacctggctaatcagaacttataaccgacacccac\n",
      "gatgaaaccgtca-\n",
      ">G000006925|1921807|1928114|1921807|1921939|+\n",
      "-ctttcagtacacctgaaatctttactgccatattcaccccac-aaaaagcccgcctgaa\n",
      "ccagcgggctgtcataacactgtgttacctggctaatcagaacttataaccgacacccac\n",
      "gatgaaaccgtcag\n",
      ">G000299455|3585529|3586267|3585529|3585662|+\n",
      "-ctttcagtacacctgaaatctttactgccatattcaccccacaaaaaagcccgcctgaa\n",
      "ccggcgggctgtcataacactgtgttacctggctaatcagaatttataaccgacacccac\n",
      "gatgaaaccgtcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2139957, 2140032), '-'), (('G000299455', (1911336, 1921536)), (1920348, 1920423), '+'), (('G000299455', (3326617, 3362069)), (3340602, 3340677), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2138845|2144864|2139957|2140032|-\n",
      "aggcggaaatctccaagctgtacaccaaagttcacccggcgtaccgcacgctgctggaga\n",
      "aacgtcaggcgctgg-\n",
      ">G000299455|1911336|1921536|1920348|1920423|+\n",
      "aggcggaaatctccaagctgtacaccaaagttcaccccgcgtaccgcacgctgctggaga\n",
      "aacgtcaggcgctgg-\n",
      ">G000299455|3326617|3362069|3340602|3340677|+\n",
      "-ggcagagatctcccagctgtataagaaagatcacccaacttatcgtgcgctgctggaaa\n",
      "aacgccagacgctgga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3099230, 3099251), '+'), (('G000299455', (856582, 857833)), (856585, 856606), '-'), (('G000299455', (4771847, 4785261)), (4784866, 4784887), '+')]\n",
      ">G000006925|3083675|3099254|3099230|3099251|+\n",
      "-tcgattccgagtccggcacca\n",
      ">G000299455|856582|857833|856585|856606|-\n",
      "ttcgattccgagtccggcacc-\n",
      ">G000299455|4771847|4785261|4784866|4784887|+\n",
      "-tcgattccgagtccggcacca\n",
      "\n",
      "\n",
      "10 [(('G000006925', (385251, 385465)), (385305, 385327), '+'), (('G000006925', (900687, 900901)), (900825, 900847), '-'), (('G000006925', (3422918, 3423132)), (3423056, 3423078), '-'), (('G000006925', (3851029, 3851243)), (3851083, 3851105), '+'), (('G000006925', (4358341, 4358555)), (4358395, 4358417), '+'), (('G000006925', (4699715, 4699878)), (4699769, 4699791), '+'), (('G000006925', (4775368, 4775582)), (4775422, 4775444), '+'), (('G000006925', (4782659, 4782873)), (4782797, 4782819), '-'), (('G000299455', (3215867, 3216640)), (3216117, 3216139), '-'), (('G000299455', (5287974, 5288758)), (5288475, 5288497), '+')]\n",
      ">G000006925|385251|385465|385305|385327|+\n",
      "-ctgctatccgtctgagcgcttc\n",
      ">G000006925|900687|900901|900825|900847|-\n",
      "tctgctatccgtctgagcgctt-\n",
      ">G000006925|3422918|3423132|3423056|3423078|-\n",
      "tctgctatccgtctgagcgctt-\n",
      ">G000006925|3851029|3851243|3851083|3851105|+\n",
      "-ctgctatccgtctgagcgcttc\n",
      ">G000006925|4358341|4358555|4358395|4358417|+\n",
      "-ctgctatccgtctgagcgcttc\n",
      ">G000006925|4699715|4699878|4699769|4699791|+\n",
      "-ctgctatccgtctgagcgcttc\n",
      ">G000006925|4775368|4775582|4775422|4775444|+\n",
      "-ctgctatccgtctgagcgcttc\n",
      ">G000006925|4782659|4782873|4782797|4782819|-\n",
      "tctgctatccgtctgagcgctt-\n",
      ">G000299455|3215867|3216640|3216117|3216139|-\n",
      "tcgccgatccgtttcagcgctt-\n",
      ">G000299455|5287974|5288758|5288475|5288497|+\n",
      "-cgccgatccgtttcagcgcttc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1634479, 1637262)), (1636878, 1636977), '-'), (('G000299455', (2585656, 2594816)), (2586767, 2586866), '+'), (('G000299455', (2585656, 2594816)), (2589251, 2589350), '+'), (('G000299455', (3440781, 3470001)), (3442877, 3442976), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1634479|1637262|1636878|1636977|-\n",
      "agcaccataaaaacttgatggggacatccagtttttcttttccacgaacaccatcacggg\n",
      "tcgcggtcattttcgtaccatggtcgatggcatctgtcc-\n",
      ">G000299455|2585656|2594816|2586767|2586866|+\n",
      "-gcaccataaaaacttgatggggacatccagtttttcttttccacgaacaccatcacggg\n",
      "tcgcggtcatttccttaccatggtcgatggcatctgtcca\n",
      ">G000299455|2585656|2594816|2589251|2589350|+\n",
      "-gttccagatgaacttaatggacacatccagtttatctttgccgcggacgccgtcgcgga\n",
      "ttgccgtcatttgcggaccatgatcgatagcatctgtcca\n",
      ">G000299455|3440781|3470001|3442877|3442976|+\n",
      "-gttccagatcattttgatcggcacatccagcttatctttcccgcgcacaccatcacgta\n",
      "gcgccgtcatttccgggccacgttcaatggcatcggtcca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (808433, 825842)), (819176, 819204), '-'), (('G000299455', (1535674, 1535721)), (1535674, 1535702), '-'), (('G000299455', (3480865, 3520974)), (3510281, 3510309), '+')]\n",
      ">G000006925|808433|825842|819176|819204|-\n",
      "ccggtgccaaagtcatcaatcgatatcc-\n",
      ">G000299455|1535674|1535721|1535674|1535702|-\n",
      "ccggtgccaaagtcatcaatcgcaatcc-\n",
      ">G000299455|3480865|3520974|3510281|3510309|+\n",
      "-cggtgccaaagtcatcaatcgatatccg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3083675, 3099254)), (3098596, 3098706), '+'), (('G000299455', (47809, 50193)), (48151, 48261), '-'), (('G000299455', (856582, 857833)), (857130, 857240), '-'), (('G000299455', (1576512, 1576622)), (1576512, 1576622), '-'), (('G000299455', (3140498, 3142451)), (3140874, 3140984), '-'), (('G000299455', (4771847, 4785261)), (4784232, 4784342), '+')]\n",
      ">G000006925|3083675|3099254|3098596|3098706|+\n",
      "-gatggatgcgcagacccgggacacatggtacaggtcactggaatatgataattttccgg\n",
      "aaatcagcgaagcgaacattctgagcacatttgaacaactgcaccagaaca\n",
      ">G000299455|47809|50193|48151|48261|-\n",
      "taatggatgcgcaggcccgtgatacatggtaccgctcactggagtacgataactttccgg\n",
      "aaatcagtgaagcgaacatttggagcacatttgagcaactgcaccagaat-\n",
      ">G000299455|856582|857833|857130|857240|-\n",
      "taatggatgcacaggcccgtgatacatggtaccggtcactggagcacgataattttccgg\n",
      "aaatcagtgaagcgaacattctgagcacatttgaacaactacaccagaat-\n",
      ">G000299455|1576512|1576622|1576512|1576622|-\n",
      "tgatgaattcgcaggaccgtgatacacgggacaggtcactgaatgacgacaatgtcctgg\n",
      "aaatcagcgaaccgcgcatctgaagtacatttgagcgactgtaccagaac-\n",
      ">G000299455|3140498|3142451|3140874|3140984|-\n",
      "taatggatacgcaggcccgtgatacatggtaccggtcactggagtacgataattttccgg\n",
      "aaatcagcgaagcgaacattctgagcacatttgaacaactgcaccagaat-\n",
      ">G000299455|4771847|4785261|4784232|4784342|+\n",
      "-gatggatgcgcagacccgggacacatggtacaggtcactggaatatgataattttccgg\n",
      "aaatcagcgaagcgaacattctgagcacatttgaacaactgcaccagaaca\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1696422, 1718400)), (1713278, 1713359), '+'), (('G000006925', (1815406, 1827803)), (1816677, 1816758), '+'), (('G000006925', (2006629, 2008384)), (2008262, 2008343), '+'), (('G000299455', (2269177, 2270929)), (2269218, 2269299), '-'), (('G000299455', (2737179, 2754578)), (2738804, 2738885), '+'), (('G000299455', (4667815, 4670271)), (4669859, 4669940), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1696422|1718400|1713278|1713359|+\n",
      "-gaaggtgcgccaatatccgtcatccgccagtacattgaacaacagcaaacacctggtca\n",
      "ggtggaaaaccgcgccttatat\n",
      ">G000006925|1815406|1827803|1816677|1816758|+\n",
      "-ggaggtgcgccaatatccgtcatccgccaatacattgaacaacagcaaacacctggtca\n",
      "ggtggaaaaccgcgccttatat\n",
      ">G000006925|2006629|2008384|2008262|2008343|+\n",
      "-ggaggtgcgccaatatccgtcatccgccaatacattgaacaacagcaaacacctggtca\n",
      "gatgaaaaaacgagccttatat\n",
      ">G000299455|2269177|2270929|2269218|2269299|-\n",
      "cggaggtgcgccaatatccgtcatccgccaatacattgaacaacagcaaacacctggtca\n",
      "ggtggaaaaccgcgccttata-\n",
      ">G000299455|2737179|2754578|2738804|2738885|+\n",
      "-ggaggtgcgccaatatccgtcatccgccaatacattgaacaacagcaaacacctggtca\n",
      "ggtggaaaaccgcgccttatat\n",
      ">G000299455|4667815|4670271|4669859|4669940|+\n",
      "-ggaggtgcgccaatatccgccatccgtcaatacattgaacaacagcaaacacctggtta\n",
      "ggtggaaaaccgcgccgtatat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1025530, 1063042)), (1056434, 1056456), '+'), (('G000006925', (1025530, 1063042)), (1056458, 1056480), '+'), (('G000006925', (1306887, 1319577)), (1312218, 1312240), '-'), (('G000299455', (2883599, 2892315)), (2890937, 2890959), '+'), (('G000299455', (3248978, 3256131)), (3255566, 3255588), '-'), (('G000299455', (3248978, 3256131)), (3255632, 3255654), '-')]\n",
      ">G000006925|1025530|1063042|1056434|1056456|+\n",
      "--atcagaagcaggtaaaaaaggt--\n",
      ">G000006925|1025530|1063042|1056458|1056480|+\n",
      "----caagagcagtcacggcaaaagc\n",
      ">G000006925|1306887|1319577|1312218|1312240|-\n",
      "gcatctgaagcgggtaaaaaag----\n",
      ">G000299455|2883599|2892315|2890937|2890959|+\n",
      "-catctgaagcgggtaaaaaagg---\n",
      ">G000299455|3248978|3256131|3255566|3255588|-\n",
      "gcatctgaagcaggtaaaaaag----\n",
      ">G000299455|3248978|3256131|3255632|3255654|-\n",
      "gcatcagaagcaggtaaaaaag----\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1168137, 1173523)), (1171514, 1171553), '+'), (('G000299455', (2728771, 2728908)), (2728796, 2728835), '+'), (('G000299455', (3050799, 3056185)), (3052769, 3052808), '-')]\n",
      ">G000006925|1168137|1173523|1171514|1171553|+\n",
      "-ccagcgctgagagcgactcatccagcaacaacagacgag\n",
      ">G000299455|2728771|2728908|2728796|2728835|+\n",
      "-ccagtgcgccgagcggttcatccaacagcaatacgcgcg\n",
      ">G000299455|3050799|3056185|3052769|3052808|-\n",
      "tccagcgctgagagcgactcatccagcaacaacagacga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3093872, 3093917), '-'), (('G000299455', (52904, 53020)), (52976, 53020), '+'), (('G000299455', (857924, 871846)), (861594, 861639), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3093872|3093917|-\n",
      "aagcggttatgacagcaaccgcctgcccgtatgagaaagatcctt-\n",
      ">G000299455|52904|53020|52976|53020|+\n",
      "--gctgttttgacggcaacagcctgttcgcgtgagaaagggccttc\n",
      ">G000299455|857924|871846|861594|861639|+\n",
      "-agctgttatgacagcaaccgcctgcccgtatgagaaagatccttc\n",
      "\n",
      "\n",
      "31 [(('G000006925', (251358, 252232)), (251538, 251589), '+'), (('G000006925', (315068, 315858)), (315710, 315761), '-'), (('G000006925', (332059, 337870)), (332701, 332752), '-'), (('G000006925', (493019, 493813)), (493661, 493712), '-'), (('G000006925', (509301, 510374)), (509944, 509995), '-'), (('G000006925', (724190, 725876)), (724833, 724884), '-'), (('G000006925', (1094884, 1095679)), (1095526, 1095577), '-'), (('G000006925', (1099363, 1100153)), (1099459, 1099510), '+'), (('G000006925', (1411510, 1412350)), (1412202, 1412253), '-'), (('G000006925', (1883764, 1885448)), (1884407, 1884458), '-'), (('G000006925', (1956633, 1961076)), (1957276, 1957327), '-'), (('G000006925', (2084425, 2085215)), (2084521, 2084572), '+'), (('G000006925', (2236395, 2237421)), (2236727, 2236778), '+'), (('G000006925', (3065287, 3076770)), (3075406, 3075457), '-'), (('G000006925', (3812816, 3813607)), (3813458, 3813509), '-'), (('G000006925', (4639759, 4640549)), (4640401, 4640452), '-'), (('G000006925', (4650084, 4650878)), (4650184, 4650235), '+'), (('G000006925', (4656735, 4658205)), (4657511, 4657562), '+'), (('G000006925', (4673259, 4674053)), (4673359, 4673410), '+'), (('G000006925', (4679888, 4680760)), (4680531, 4680582), '-'), (('G000006925', (4750261, 4751054)), (4750360, 4750411), '+'), (('G000006925', (4753880, 4754990)), (4754842, 4754893), '-'), (('G000006925', (4760730, 4761525)), (4760831, 4760882), '+'), (('G000006925', (4793569, 4794365)), (4793668, 4793719), '+'), (('G000006925', (4828486, 4828646)), (4828492, 4828542), '-'), (('G000299455', (906289, 909876)), (906392, 906442), '+'), (('G000299455', (912160, 912572)), (912261, 912311), '+'), (('G000299455', (4752701, 4766669)), (4752803, 4752854), '+'), (('G000299455', (4771847, 4785261)), (4780355, 4780406), '-'), (('G000299455', (5275700, 5279438)), (5279135, 5279186), '-'), (('G000299455', (5343200, 5343533)), (5343405, 5343456), '+')]\n",
      ">G000006925|251358|252232|251538|251589|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|315068|315858|315710|315761|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|332059|337870|332701|332752|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|493019|493813|493661|493712|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|509301|510374|509944|509995|-\n",
      "gcagtccgtatggttctggaaagtcagggagaatatgactcacaatgggcg-\n",
      ">G000006925|724190|725876|724833|724884|-\n",
      "gcagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000006925|1094884|1095679|1095526|1095577|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|1099363|1100153|1099459|1099510|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|1411510|1412350|1412202|1412253|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|1883764|1885448|1884407|1884458|-\n",
      "gcagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000006925|1956633|1961076|1957276|1957327|-\n",
      "gcagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000006925|2084425|2085215|2084521|2084572|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|2236395|2237421|2236727|2236778|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|3065287|3076770|3075406|3075457|-\n",
      "gcagtccgtatggttctggaaagtcagagcgaatatgactcacaatgggcg-\n",
      ">G000006925|3812816|3813607|3813458|3813509|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4639759|4640549|4640401|4640452|-\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4650084|4650878|4650184|4650235|+\n",
      "-cagtccgtatggttctggaaagtcagggagaatatgactcacaatgggcga\n",
      ">G000006925|4656735|4658205|4657511|4657562|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4673259|4674053|4673359|4673410|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4679888|4680760|4680531|4680582|-\n",
      "gcagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000006925|4750261|4751054|4750360|4750411|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4753880|4754990|4754842|4754893|-\n",
      "-cagttcgtatggttctggaaagtcagggcgaatatgactcacaatgggcag\n",
      ">G000006925|4760730|4761525|4760831|4760882|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4793569|4794365|4793668|4793719|+\n",
      "-cagtccgtatggttctggaaagtcagggcgaatatgactcacaatgggcga\n",
      ">G000006925|4828486|4828646|4828492|4828542|-\n",
      "-cagttcgtatggttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000299455|906289|909876|906392|906442|+\n",
      "-tgatacgtatagttctggaaagtcag-gcgaatatgactcacaatgggcgg\n",
      ">G000299455|912160|912572|912261|912311|+\n",
      "-tgattcgtatagttctggaaagtcag-gcgaatatgactcataatgggcgg\n",
      ">G000299455|4752701|4766669|4752803|4752854|+\n",
      "-cagtccgtatggttctggaaagtcagagcgaatatgactcacaatgggcga\n",
      ">G000299455|4771847|4785261|4780355|4780406|-\n",
      "gcagtccgtatggttctggaaagtcagagcgaatatgactcacaatgggcg-\n",
      ">G000299455|5275700|5279438|5279135|5279186|-\n",
      "gcagtccgtatagttctggaaagtcagggcgaatatgactcacaatgggcg-\n",
      ">G000299455|5343200|5343533|5343405|5343456|+\n",
      "-tgattcgtatggttctggaaagtcagggcgaatatgactcacaatgggcgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (496417, 503320)), (496847, 496903), '-'), (('G000299455', (319622, 319678)), (319622, 319678), '-'), (('G000299455', (3819953, 3834467)), (3829264, 3829320), '+')]\n",
      ">G000006925|496417|503320|496847|496903|-\n",
      "gcgcatccgctccagtgcgccagaaatattggcttcggaatcaatcaggaacaggc-\n",
      ">G000299455|319622|319678|319622|319678|-\n",
      "gcgcatacgttccagtgcgcccgtaatattggcttcagagtcaatgaggaacaggc-\n",
      ">G000299455|3819953|3834467|3829264|3829320|+\n",
      "-cgcatccgctccagtgcgccagaaatattggcttcggaatcaatcaggaacaggcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3370611, 3370659), '-'), (('G000299455', (540757, 615358)), (570468, 570516), '+'), (('G000299455', (4235244, 4293324)), (4268972, 4269020), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 31\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 31\r",
      "   10 / 31\r",
      "   20 / 31\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 30  f\b\b\r",
      "STEP     2 / 30  f\b\b\r",
      "STEP     3 / 30  f\b\b\r",
      "STEP     4 / 30  f\b\b\r",
      "STEP     5 / 30  f\b\b\r",
      "STEP     6 / 30  f\b\b\r",
      "STEP     7 / 30  f\b\b\r",
      "STEP     8 / 30  f\b\b\r",
      "STEP     9 / 30  f\b\b\r",
      "STEP    10 / 30  f\b\b\r",
      "STEP    11 / 30  f\b\b\r",
      "STEP    12 / 30  f\b\b\r",
      "STEP    13 / 30  f\b\b\r",
      "STEP    14 / 30  f\b\b\r",
      "STEP    15 / 30  f\b\b\r",
      "STEP    16 / 30  f\b\b\r",
      "STEP    17 / 30  f\b\b\r",
      "STEP    18 / 30  f\b\b\r",
      "STEP    19 / 30  f\b\b\r",
      "STEP    20 / 30  f\b\b\r",
      "STEP    21 / 30  f\b\b\r",
      "STEP    22 / 30  f\b\b\r",
      "STEP    23 / 30  f\b\b\r",
      "STEP    24 / 30  f\b\b\r",
      "STEP    25 / 30  f\b\b\n",
      "Reallocating..done. *alloclen = 1104\n",
      "\r",
      "STEP    26 / 30  f\b\b\r",
      "STEP    27 / 30  f\b\b\r",
      "STEP    28 / 30  f\b\b\r",
      "STEP    29 / 30  f\b\b\r",
      "STEP    30 / 30  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 31\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 31\r",
      "   10 / 31\r",
      "   20 / 31\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 30  f\b\b\r",
      "STEP     2 / 30  f\b\b\r",
      "STEP     3 / 30  f\b\b\r",
      "STEP     4 / 30  f\b\b\r",
      "STEP     5 / 30  f\b\b\r",
      "STEP     6 / 30  f\b\b\r",
      "STEP     7 / 30  f\b\b\r",
      "STEP     8 / 30  f\b\b\r",
      "STEP     9 / 30  f\b\b\r",
      "STEP    10 / 30  f\b\b\r",
      "STEP    11 / 30  f\b\b\r",
      "STEP    12 / 30  f\b\b\r",
      "STEP    13 / 30  f\b\b\r",
      "STEP    14 / 30  f\b\b\r",
      "STEP    15 / 30  f\b\b\r",
      "STEP    16 / 30  f\b\b\r",
      "STEP    17 / 30  f\b\b\r",
      "STEP    18 / 30  f\b\b\r",
      "STEP    19 / 30  f\b\b\r",
      "STEP    20 / 30  f\b\b\r",
      "STEP    21 / 30  f\b\b\r",
      "STEP    22 / 30  f\b\b\r",
      "STEP    23 / 30  f\b\b\r",
      "STEP    24 / 30  f\b\b\r",
      "STEP    25 / 30  f\b\b\r",
      "STEP    26 / 30  f\b\b\r",
      "STEP    27 / 30  f\b\b\r",
      "STEP    28 / 30  f\b\b\r",
      "STEP    29 / 30  f\b\b\r",
      "STEP    30 / 30  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3361966|3403725|3370611|3370659|-\n",
      "ccgccagagttaccgtggttaatggaagcatcggtttgtaggaagttt-\n",
      ">G000299455|540757|615358|570468|570516|+\n",
      "-cgccagagttaccgtggttaatggaagcatcggtttgtaggaagtttt\n",
      ">G000299455|4235244|4293324|4268972|4269020|+\n",
      "-caccggagttaccacggttgatcgctgcatcggtctggatgaagtttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1582509, 1591522)), (1583773, 1583810), '+'), (('G000299455', (2018078, 2018117)), (2018078, 2018115), '-'), (('G000299455', (2596243, 2603992)), (2596243, 2596280), '+')]\n",
      ">G000006925|1582509|1591522|1583773|1583810|+\n",
      "-ccatcttgctggtgacgctgcgtattattcagggatt\n",
      ">G000299455|2018078|2018117|2018078|2018115|-\n",
      "cctattttgctggtgacactgcgtgccattcagggat-\n",
      ">G000299455|2596243|2603992|2596243|2596280|+\n",
      "-ccatcttgctggtgacgctgcgtattattcaggggtt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4612466, 4613007)), (4612692, 4612910), '-'), (('G000006925', (4622488, 4624460)), (4623448, 4623665), '+'), (('G000299455', (5313369, 5315826)), (5314685, 5314902), '+')]\n",
      ">G000006925|4612466|4613007|4612692|4612910|-\n",
      "gatacaggaagcacaggagcgaatcagcgcacttcaggaggaaatggcggaaacactggg\n",
      "tgagcaatatgacccggtactgccatcttccctgcgccagtcttcagcccgtaaaccgtt\n",
      "accagcctcacttccccgtgccccccgggtcatccggccggaagaggaatgctgtccggc\n",
      "ctgtggtggtgaactcagtccactgggatgtgatgtgt\n",
      ">G000006925|4622488|4624460|4623448|4623665|+\n",
      "-atactggaagcacaggagcgaatcagcgcacttcaggaagaaatggcggaaacgctggg\n",
      "tgagcaatatgacccggtactgccatctcccctgcgccagtcttcagcccgtaaaccgtt\n",
      "acctgcctcacttccccgtgaaacccgggtcatccgtccggaagaggaatgctgcccggc\n",
      "ctgtggtggcgaactcagttctctgggatgtgatgtgt\n",
      ">G000299455|5313369|5315826|5314685|5314902|+\n",
      "-atacaggaagcacaggagcgaatcagcgcacttcaggaagaaatggcggaaacgctggg\n",
      "tgagcaatatgacccggtactgccatccgccctgcgccagtcttcagcccgtaaaccgtt\n",
      "accggcctcacttccccgtgaaacccgggttatccggccggaagaggaatgctgtcctgc\n",
      "ctgtggtggtgaactcagttctctgggatgtgatgtgt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (491387, 491961), '-'), (('G000299455', (318605, 319185)), (318605, 319185), '-'), (('G000299455', (3819953, 3834467)), (3829757, 3830331), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|487251|491968|491387|491961|-\n",
      "accacgttatctttggcgatattcattcccgcgcgcagatcaaacgcggtgatcacgcca\n",
      "tcaatgggcgctttgagcgtaaagcgagtctggattttttgcgtggcgatcaggtggcga\n",
      "atatccgcctccggcattcccgccagtcgcagccgctcaagaatgccttcagtctgggtc\n",
      "gccgtaccgccggtttcgcgcaacagtaaatactcactctgcgcttccacccagtcggga\n",
      "atggtcaggtcgagaagcggtgtgcccttttgtactttatcgcccacggtaagcggatac\n",
      "accttgtcgataaacccggcagcgcgagcctgcacaatggcatactgatactcgttgtaa\n",
      "ctgacattcgtcgggaaactctgggcaaaagtcagcggcccgcgcgtgacggtagccgtt\n",
      "ttcacccccagattctgcgtctgagtcgggtcaatgcgca------caccagacgcagaa\n",
      "ctctcttcatcggcatatttcggcaccagatccatatccataaacggcgatttacctggt\n",
      "ttatcgaaccgcgtattgggatacatcgggtcgtaccaga-\n",
      ">G000299455|318605|319185|318605|319185|-\n",
      "accaccttatctttcgaaatattcatgccggtgcgcaggtcaaatgcagtaatgacacca\n",
      "tcaataggtgctttaatggtaaaacgggtctgaatgctccgggtagaacgcagtctctga\n",
      "atatcctcttccggcatacctgccaggcgaagccgttccagaacgcctttaatttgcgtg\n",
      "gaagtaccaccggtgctggataacagaaggaattcactctgtgcttccacccagtccgga\n",
      "atggtaatatcgataagtggcgtccccttcttcacatgatcgccaatcgtcatggggtag\n",
      "actttttcgacaaacccgtcagaacgcgcctgcacaataacaaactgatactcgttatag\n",
      "ctgacattagccgggattgtctgagaataattcagcattcctcgcgtgactttttgcgtt\n",
      "tttaatcccagattctgaacctgggttgggtcgatacggatcccgccactgcttttatcg\n",
      "ccactgtcatcagcatattttggcaccaggtccatatccataaagggcgattttccgggt\n",
      "ttatcaaatttggtatccggtttcattgggtcataccaga-\n",
      ">G000299455|3819953|3834467|3829757|3830331|+\n",
      "-ccacgttatctttggcgatattcattcccgcgcgcagatcaaacgcggtgatcacgcca\n",
      "tcaatgggcgctttgagcgtaaagcgagtctggattttttgcgtggcgatcaggcggcga\n",
      "atatccgcctccggcattcccgccagtcgcagccgctcaagaatgccttcagtctgggtc\n",
      "gccgtaccgccggtttcgcgcagcagtaaatactcactctgcgcttccacccagtcagga\n",
      "atggtcaggtcgagaagcggtgtgcccttttgtactttatcgcccacggtaagcggatac\n",
      "accttgtcgataaacccggcagcgcgagcctgcacaatggcatactgatactcgttgtaa\n",
      "ctgacattcgccgggaaactctgggcaaaagtcagcggcccgcgcgtgacggtagccgtt\n",
      "ttcacccccagattctgcgtctgagtcgggtcaatgcgca------caccagacgcagaa\n",
      "ctctcttcatcggcatatttcggcaccagatccatatccataaacggcgatttacctggt\n",
      "ttatcgaaccgcgtattggggtacatcgggtcgtaccagaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1956633, 1961076)), (1959022, 1959103), '-'), (('G000299455', (2133804, 2148224)), (2135778, 2135859), '+'), (('G000299455', (4166042, 4170922)), (4169553, 4169634), '+')]\n",
      ">G000006925|1956633|1961076|1959022|1959103|-\n",
      "gaccattggcgacggtctggtggcacaaattccggcgctggtgatttctaccgccgcggg\n",
      "gggcatcgttacgcgtgtcag-\n",
      ">G000299455|2133804|2148224|2135778|2135859|+\n",
      "gaccattggcgacggtctggtggcacaaattccggcgctggtgatttctaccgccgcggg\n",
      "ggtcatcgttacgcgtgtcag-\n",
      ">G000299455|4166042|4170922|4169553|4169634|+\n",
      "-accatcggcgacggtctggtggcgcagatcccttccctgctgctctccacggcggcggc\n",
      "gattatcgtcacccgcgtcagc\n",
      "\n",
      "\n",
      "11 [(('G000006925', (385251, 385465)), (385251, 385280), '+'), (('G000006925', (900687, 900901)), (900872, 900901), '-'), (('G000006925', (3422918, 3423132)), (3423103, 3423132), '-'), (('G000006925', (3851029, 3851243)), (3851029, 3851057), '+'), (('G000006925', (4358341, 4358555)), (4358341, 4358369), '+'), (('G000006925', (4699715, 4699878)), (4699715, 4699743), '+'), (('G000006925', (4775368, 4775582)), (4775368, 4775396), '+'), (('G000006925', (4782659, 4782873)), (4782844, 4782873), '-'), (('G000299455', (918823, 918877)), (918848, 918877), '-'), (('G000299455', (4544905, 4544959)), (4544930, 4544959), '-'), (('G000299455', (4771787, 4771841)), (4771812, 4771841), '-')]\n",
      ">G000006925|385251|385465|385251|385280|+\n",
      "-caagccgttcctctgccggactgccacgt\n",
      ">G000006925|900687|900901|900872|900901|-\n",
      "gcaagccgttcctctgccggactgccacg-\n",
      ">G000006925|3422918|3423132|3423103|3423132|-\n",
      "gcaagccgttcctctgccggactgccacg-\n",
      ">G000006925|3851029|3851243|3851029|3851057|+\n",
      "-caagccgttcctctgccggactgccacg-\n",
      ">G000006925|4358341|4358555|4358341|4358369|+\n",
      "-caagccgttcctctgccggactgccacg-\n",
      ">G000006925|4699715|4699878|4699715|4699743|+\n",
      "-caagccgttcctctgccggactgccacg-\n",
      ">G000006925|4775368|4775582|4775368|4775396|+\n",
      "-caagccgttcctctgccggactgccacg-\n",
      ">G000006925|4782659|4782873|4782844|4782873|-\n",
      "gcaagccgttcctctgccggactgccacg-\n",
      ">G000299455|918823|918877|918848|918877|-\n",
      "gcaagccgctgctctgccggcattcccct-\n",
      ">G000299455|4544905|4544959|4544930|4544959|-\n",
      "gcaagccgctgctctgccggcattcccct-\n",
      ">G000299455|4771787|4771841|4771812|4771841|-\n",
      "gcaagccgctgctctgccggcattcccct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2143633, 2143933), '+'), (('G000299455', (1911336, 1921536)), (1916447, 1916747), '-'), (('G000299455', (2199778, 2263225)), (2247181, 2247481), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2138845|2144864|2143633|2143933|+\n",
      "tctggggcgtggtgacgcaaattgtggtactggacgccattttctcactcgactcagtga\n",
      "ttaccgccgtcgggatggtcgaccatttactggtcatgatggccgccgtggttatcgcta\n",
      "tcagcctgatgttgatggccagcaagccgttaacgcaatttgttaacagtcacccgacca\n",
      "tcgttattctctgcttaagcttcctgttgatgattggctttagcctggtggcagaaggtt\n",
      "tcggcttcgtcattccgaaaggctacctgtacgctgccattggtttctcggtgatgatcg\n",
      ">G000299455|1911336|1921536|1916447|1916747|-\n",
      "tctggggggtggtgacacaaattgtggtactggacgccatcttctcactcgactcagtga\n",
      "ttaccgccgtcgggatggtcgaccatttactggtcatgatggccgccgtggttatcgcta\n",
      "tcagcctgatgttgatggccagcaagccgttaacgcaatttgttaacagtcacccgacca\n",
      "tcgttattctctgcttaagcttcctgttgatgattggctttagcctggtggcagaaggtt\n",
      "tcggcttcgtcattccgaaaggctacctgtacgctgccattggtttctcggtgatgatcg\n",
      ">G000299455|2199778|2263225|2247181|2247481|+\n",
      "tctgggtggtcgtcacacagattgtcatccttgacgccgtcttctcgttggatgcggtaa\n",
      "ttactgcagtagggatggttaaccatctgccggtgatgatggcggcggtagtgattgcga\n",
      "tggcggttatgttgctggcatccaaaccgctgacgcgattcgttaaccagcaccccacgg\n",
      "tggtggtgctctgtctgagcttcctgttaatgattggtctgagtctggtggcagaaggtt\n",
      "tcggtttccacattccgaaaggttacctgtatgccgcgattggcttctcgatcatcatcg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (729590, 729684), '+'), (('G000299455', (2174641, 2181396)), (2177986, 2178080), '+'), (('G000299455', (2910238, 2916987)), (2913548, 2913642), '-'), (('G000299455', (3601027, 3601121)), (3601027, 3601121), '-')]\n",
      ">G000006925|726235|732999|729590|729684|+\n",
      "-ggattagctccggaagaatggccattgacgggctcaaggaggtccaggaatcggtgatg\n",
      "cgcattgaggccggactgagcacgtatgagaaaga\n",
      ">G000299455|2174641|2181396|2177986|2178080|+\n",
      "-ggattggtgccggaagaatggccattgacgggctcaaggaggttcaggaatcggtgatg\n",
      "cgtattgattccggactgagcacgtatgagaaaga\n",
      ">G000299455|2910238|2916987|2913548|2913642|-\n",
      "tggattggagccggaagaatggccattgacgggctcaaggaggttcaggaatcagtgatg\n",
      "cgcattgaggccggactgagcacgtatgagaaag-\n",
      ">G000299455|3601027|3601121|3601027|3601121|-\n",
      "tggataggctccggtcgtatggccatcgatggtctgaaagaagttcaggaagcggtgatg\n",
      "ctgatagaagccggactgagcacctacgagaaag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3094023, 3094350), '-'), (('G000299455', (50607, 52894)), (52563, 52889), '+'), (('G000299455', (857924, 871846)), (861162, 861488), '+'), (('G000299455', (1980069, 1981046)), (1980716, 1981046), '+'), (('G000299455', (3143255, 3145541)), (3145211, 3145541), '+')]\n",
      ">G000006925|3083675|3099254|3094023|3094350|-\n",
      "cccgctgcagactttccagtacggtgatggtggggatgtacgtatagttctggcttcggg\n",
      "acgtgtgtttctcttccccgaaaatccccggcacatggtgcatcagctcttcatgtgtca\n",
      "gcggacggtcacggcgtatctggttc---gcataaccaaaacgactgactaattgcatac\n",
      "tttgctccttatcggtggttaagatttactggtgcaacaaatgaaaaaagccgcgtctcc\n",
      "cggagaagacgcggcctgatggatgaaatcaatgaagtttattgtctgagaagccctcca\n",
      "ctggcgagccacgcattaagctgttttccg-\n",
      ">G000299455|50607|52894|52563|52889|+\n",
      "-ccgctgcaggctttccagtacggtgatggtgggaatgtacgcatatcgtttactgcggg\n",
      "aagtatgccggtcttcaccgaaaataccgggaacatagtgcatcagttcttcgtgtgtca\n",
      "gcggacggtcacggcgtatctggttc---gtataaccaaaacgactggctagtcgcataa\n",
      "tttgctccttatcggtggttaagatttactggtgtaataaatg-aaaaagccacgtctcc\n",
      "cggagaagacgcggcctgacagatgaaatgaatgacgtttattgtctgagaagcccttaa\n",
      "ctggcgagctgagtattaagctgtgttccgg\n",
      ">G000299455|857924|871846|861162|861488|+\n",
      "-ccgctgcaggctttccagtacggtgatggtggggatgtacgtatagttctggcttcggg\n",
      "acgtgtgtttatcttccccgaaaatccccggcacatagtgcatcagttcttcgtgtgtca\n",
      "gcggacggtcacggcgtatctggttc---gcataaccaaaacgactggctagtcgcataa\n",
      "tttgctccttatcggtggttaagatttactggtgtaataaatg-aaaaagccacgtctcc\n",
      "cggagaagacgcggcctgatgggtgaaatgaatgacgtttattgtctgagaagaccgtca\n",
      "ctggcgagccgcgtattaagcacctttccgg\n",
      ">G000299455|1980069|1981046|1980716|1981046|+\n",
      "-ccgctgcagattttccaggacggtaatggtgggaatgtacgcataccgttcactgcggg\n",
      "aggtgtgccggtcttccccaaaaatactgggtacatggcgtatcagttcttcatgtgtca\n",
      "gcggacggtcacggcgtatctggtttgcagcataaccaaaacgacttgctaatcgcatac\n",
      "tttgctccttatcggtggttaagatttactggtgcaataaatgaaaaaagccgcgtctcc\n",
      "cggagaagacgcggcctgacgggtgaaatgaataaggtttattgtctgagaggcctttca\n",
      "ctggcgagccacgcattaagcggctttccgg\n",
      ">G000299455|3143255|3145541|3145211|3145541|+\n",
      "-ccgctgcaggctttccagcacggtgatggtggggatgtacgtatagttctggcttcggg\n",
      "atgtgtgtttatcttctccgaaaatccccggcacatggtgcatcagctcttcatgtgtca\n",
      "gcggacggtcacggcgtatctggtttgcagcataaccaaaacgactgactaatcgcatac\n",
      "tttgctccttatcggtggttaagatttactggtgcaataaatgaaaaaagccgcgtctcc\n",
      "cggagaagacacggcctgactggtgaaatgaataaggtttattgtctgagaggcctttca\n",
      "ctggcgagccacgcattaagcggctttccgg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (865572, 865846)), (865728, 865763), '-'), (('G000006925', (2242494, 2346249)), (2252100, 2252136), '-'), (('G000006925', (3229897, 3229934)), (3229897, 3229933), '+'), (('G000299455', (1819843, 1851664)), (1826372, 1826408), '+'), (('G000299455', (3470062, 3470342)), (3470223, 3470259), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|865572|865846|865728|865763|-\n",
      "gagggcgcgtcttaca-ggacgttacctcgcgcccg-\n",
      ">G000006925|2242494|2346249|2252100|2252136|-\n",
      "gagggcgcgttttacagggatgttacctcgcgcccg-\n",
      ">G000006925|3229897|3229934|3229897|3229933|+\n",
      "-agggcgcatcttacagggacgttaccttgcgcccga\n",
      ">G000299455|1819843|1851664|1826372|1826408|+\n",
      "-agggcgcgttttacagggatgttacctcgcgcccga\n",
      ">G000299455|3470062|3470342|3470223|3470259|-\n",
      "gagggcgcgttttacagggatgttacctcgcgcccg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1413472, 1413749)), (1413472, 1413728), '+'), (('G000299455', (2528985, 2529262)), (2528985, 2529241), '+'), (('G000299455', (3619129, 3619385)), (3619129, 3619385), '-')]\n",
      ">G000006925|1413472|1413749|1413472|1413728|+\n",
      "-tttagtcctccaaatcaatatcaacctggtggtgggcaatggtttcagccatgtaccgg\n",
      "atgtgttctgccatgcgctcctgaaactcaacatcatcatcaaatgcgtggctgattgcc\n",
      "tgtttattggcaccgtggcgttgcaaatggtcgatgcagagcgattcaaacaggtgctga\n",
      "ggcagacctttttccaggtcgtccgccagttccgtttctttttcttcacgaacgatctgc\n",
      "tggtagtgacgagccca\n",
      ">G000299455|2528985|2529262|2528985|2529241|+\n",
      "-tttatacctctgaatcaatattaatttggtgacgggcgatggtttcagccatgtagcgg\n",
      "atgtgttctgccatgcgttcctgaaaatcgacatcgtcatcaaatgcacgggaaatagct\n",
      "tttttgctggccccgtgacgttgcagattatcgatgcatagcgattcaaacaggtgttgg\n",
      "ggcagacctttttccaggtcgtctgccagctcagcttcagtttcttcacgggcaatttgc\n",
      "tggtagtgtcgtgccca\n",
      ">G000299455|3619129|3619385|3619129|3619385|-\n",
      "ttttatacctctgaatcaatatcaacctggtggtgagcaatggtttcaaccatgtactgg\n",
      "atgtgttctgccatgcgctcctgaaactcaacatcgtcatcaaacgcacgggtaatggct\n",
      "tttttgctggccccgtggcgttgcaaatgatcgatgcatagcgattcaaacaggtgctgg\n",
      "ggcaggcctttttccatgtcgtctgccagttctgcctctttctcttcacgggcgatctgc\n",
      "tggtagtgacgcgccc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (730069, 730183), '+'), (('G000299455', (2174641, 2181396)), (2178465, 2178579), '+'), (('G000299455', (2910238, 2916987)), (2913049, 2913163), '-'), (('G000299455', (3600528, 3600642)), (3600528, 3600642), '-')]\n",
      ">G000006925|726235|732999|730069|730183|+\n",
      "-gaacggtattgctgtactgccggtgaccggcacgctggtacaccggctgggtggcatgc\n",
      "ggccattttccggaatgactggctatgacggcattgtcgcctgtcttcagcaggc\n",
      ">G000299455|2174641|2181396|2178465|2178579|+\n",
      "-gaacggtattgctgtactgccggtgacaggcacgctggtgcaccggctggggggtatgc\n",
      "ggccattttccggaatgacaggctatgacggcattgtcgcctgtcttcagcaggc\n",
      ">G000299455|2910238|2916987|2913049|2913163|-\n",
      "tgaacggtattgctgtactgccggtgaccggcacgctggtgcaccggctgggcggtatgc\n",
      "ggccattttccggaatgacaggctatgacggcattgtcgcctgtcttcagcagg-\n",
      ">G000299455|3600528|3600642|3600528|3600642|-\n",
      "tgaacggcatcgccgtgctgccggtgtccggcacgctggtcagccggacgcgggcgctgc\n",
      "agccgtattcggggatgaccggttacaacggcattatcgcccgtctgcaacagg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (127893, 127954), '+'), (('G000006925', (64491, 216702)), (128084, 128145), '+'), (('G000299455', (4300454, 4333834)), (4320541, 4320602), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1124\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1124\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|64491|216702|127893|127954|+\n",
      "--tggcgctgttcgggagttgcagactgaatacgctcacgcgctgcctctcttcgctgct\n",
      "ggt\n",
      ">G000006925|64491|216702|128084|128145|+\n",
      "-ctggcgctgctcaggagaggccgactgataacgctgacgagcggcgtccttttgctgct\n",
      "gg-\n",
      ">G000299455|4300454|4333834|4320541|4320602|-\n",
      "cctggcgctgttcgggagttgcagactgaatacgctcacgcgctgcctctcttcgctgct\n",
      "g--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1425133, 1465118)), (1446020, 1446111), '+'), (('G000006925', (4249830, 4249921)), (4249830, 4249921), '+'), (('G000299455', (2199778, 2263225)), (2244123, 2244214), '+')]\n",
      ">G000006925|1425133|1465118|1446020|1446111|+\n",
      "acgtacggttccccagaagatcgggtcgcctacaccagccagcggccccatcaaaccgac\n",
      "tttgataccgttgatagcaccgtcatcgatc--------\n",
      ">G000006925|4249830|4249921|4249830|4249921|+\n",
      "--------ttccccagaccagtggatcgccaactcctgccaatggtcccatcagaccgac\n",
      "tttgatgccgttgatggtaccgtcatcaatttccgcgcc\n",
      ">G000299455|2199778|2263225|2244123|2244214|+\n",
      "--------ttccccagaagatcgggtcgcctacaccagccagcggccccatcaaaccgac\n",
      "tttgataccgttgatagcaccgtcgtcgatctctgcgcc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (730452, 730498), '+'), (('G000299455', (2174641, 2181396)), (2178848, 2178894), '+'), (('G000299455', (2910238, 2916987)), (2912734, 2912780), '-'), (('G000299455', (3600213, 3600259)), (3600213, 3600259), '-')]\n",
      ">G000006925|726235|732999|730452|730498|+\n",
      "-tggatatcacgctgatttacgccggatcgcacaaggtggatggcaa\n",
      ">G000299455|2174641|2181396|2178848|2178894|+\n",
      "-tggatatcacgctgatttatgccggggcgcacaaggtggatggcaa\n",
      ">G000299455|2910238|2916987|2912734|2912780|-\n",
      "gtggatatcacgctgatttatgccggggcgcacaaggtggatggca-\n",
      ">G000299455|3600213|3600259|3600213|3600259|-\n",
      "gtggaaatcacgctgatttacagcggcagccataaggtggatggca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (728502, 728644), '+'), (('G000299455', (2174641, 2181396)), (2176899, 2177041), '+'), (('G000299455', (2910238, 2916987)), (2914587, 2914729), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1190\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1190\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|728502|728644|+\n",
      "-cacatgtttcttatcagctaccgtctgaactggcgttggctggggatgcgggagaccac\n",
      "ggcaaaaagttttgtcggggaggtggaagcagcctggtcggaatacgcagaaggaatgtt\n",
      "tggcgagatcgacgtggaaggga\n",
      ">G000299455|2174641|2181396|2176899|2177041|+\n",
      "-catatgtttctgattagctaccgtccgaactggcgctggctggggatgcgggagaccgc\n",
      "ggcaaaaagttttgtcgatgaggtggaggcggcctggtcagaatacgcagaagggatgtt\n",
      "tggtgagatcgacgtggaaggga\n",
      ">G000299455|2910238|2916987|2914587|2914729|-\n",
      "gcatatgtttctgatcagctaccgtccgaactggcgctggctggggatgcgggagactgc\n",
      "ggcaaaaagttttgtcgatgaggtggaggcggcctggtcggaatacgcagaagggatgtt\n",
      "tggcgagatcgacgtagaaggg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3268417, 3281594)), (3280517, 3280564), '+'), (('G000299455', (224130, 224312)), (224142, 224189), '+'), (('G000299455', (642419, 661467)), (656219, 656266), '-'), (('G000299455', (3856002, 3856049)), (3856002, 3856049), '-')]\n",
      ">G000006925|3268417|3281594|3280517|3280564|+\n",
      "-gataatatgttgccaaacagccagcgcggatttgccccaacggtacg\n",
      ">G000299455|224130|224312|224142|224189|+\n",
      "-gatgaaatgttgccggacagccagcgtggctttgctccggtagtgcg\n",
      ">G000299455|642419|661467|656219|656266|-\n",
      "cgataatatgttgccaaacagccagcgcggatttgccccaacggtac-\n",
      ">G000299455|3856002|3856049|3856002|3856049|-\n",
      "cgataatatgtatcctgatagccagcaagggtttgccccaacggtac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (943933, 964649)), (951845, 951869), '-'), (('G000299455', (3400844, 3428172)), (3417193, 3417217), '+'), (('G000299455', (4865050, 4904182)), (4892534, 4892558), '+')]\n",
      ">G000006925|943933|964649|951845|951869|-\n",
      "acgctgacgctgaccgccagagag-\n",
      ">G000299455|3400844|3428172|3417193|3417217|+\n",
      "-cgctgacgctgaccgccagagagc\n",
      ">G000299455|4865050|4904182|4892534|4892558|+\n",
      "-cgctgacgctgaccaccagagagc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (789803, 808424)), (808283, 808350), '-'), (('G000006925', (4143132, 4163489)), (4154126, 4154193), '+'), (('G000299455', (3520976, 3527694)), (3521050, 3521117), '+'), (('G000299455', (4996216, 5016676)), (5005603, 5005670), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1136\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1136\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|789803|808424|808283|808350|-\n",
      "tgccaaacgctccctgccagtcctgctcaaacttcgtcacagcggcttccaccgccggat\n",
      "agctaat-\n",
      ">G000006925|4143132|4163489|4154126|4154193|+\n",
      "-gccaaatgcggcattccagtcgtgttcgaacttctctatagctgactctaccgcagggg\n",
      "tgttgagc\n",
      ">G000299455|3520976|3527694|3521050|3521117|+\n",
      "-gccaaacgctccctgccagtcctgctcaaacttcgccacagcggcatcaaccgccggat\n",
      "agctaatc\n",
      ">G000299455|4996216|5016676|5005603|5005670|-\n",
      "tgccaaatgcggcattccagtcgtgttcgaacttctctatagctgactctaccgcagggg\n",
      "tgttgag-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1578298, 1580868)), (1579266, 1579369), '+'), (('G000006925', (1938092, 1938404)), (1938092, 1938195), '+'), (('G000006925', (2062140, 2062463)), (2062140, 2062243), '+'), (('G000299455', (2927401, 2931337)), (2928379, 2928482), '+')]\n",
      ">G000006925|1578298|1580868|1579266|1579369|+\n",
      "ccagacgtccaccagatactggctgaccacgaacgatattgatttcatcaaactgctcat\n",
      "cgtccattaacactcccgcatgcgtcagcgcatccagcggtgc\n",
      ">G000006925|1938092|1938404|1938092|1938195|+\n",
      "ccaaccgaccaccaggaacaggcagtccgcgcacaatattggtttcatcaaactgctcgt\n",
      "cgtcgataagcaaccccgcatgtgtcagtgcatccagtggtgc\n",
      ">G000006925|2062140|2062463|2062140|2062243|+\n",
      "ccagccgcccaccaggaacgagctgaccgcgcacaatattgatttcatcaaactgctcgt\n",
      "cgtctataagtagtccagcatgcgtcagcgcatccagcggtgc\n",
      ">G000299455|2927401|2931337|2928379|2928482|+\n",
      "ccagccgccccccaggaacgcgctgaccgcgcacaatattgatttcatcaaactgctcgt\n",
      "cgtctatgagaagtccggcatgcgtcagcgcatccagtggtgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2069615, 2084300)), (2083510, 2083537), '-'), (('G000299455', (3182222, 3183821)), (3182697, 3182724), '-'), (('G000299455', (4564841, 4566095)), (4565593, 4565620), '+')]\n",
      ">G000006925|2069615|2084300|2083510|2083537|-\n",
      "ccacccgggaaggatggctgtatctcg-\n",
      ">G000299455|3182222|3183821|3182697|3182724|-\n",
      "ccacacaggagggatggttgtacctcg-\n",
      ">G000299455|4564841|4566095|4565593|4565620|+\n",
      "-cacacaggagggatggttgtacctcgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3868997, 3883532)), (3873889, 3873992), '+'), (('G000299455', (1, 26978)), (4895, 4998), '+'), (('G000299455', (767773, 781429)), (778908, 779011), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3868997|3883532|3873889|3873992|+\n",
      "cgaactgtacctggtggaaggggactccgcgggcggctctgcgaagcaggggcgtaaccg\n",
      "caagaaccaggcgattctgccgctgaagggtaaaatccttaac\n",
      ">G000299455|1|26978|4895|4998|+\n",
      "cgaactgtacctggtggaaggggactccgcgggcggctctgcgaagcaggggcgtaaccg\n",
      "caagaaccaggcgattctgccgctgaagggtaaaatcctcaac\n",
      ">G000299455|767773|781429|778908|779011|+\n",
      "cgaactgttccttgtggaaggtgactccgcaggcggatctgccaagcaggcgcgcgatcg\n",
      "cgaatatcaggcgatcatgccactgaaaggtaagatccttaac\n",
      "\n",
      "\n",
      "5 [(('G000006925', (865572, 865846)), (865658, 865680), '-'), (('G000006925', (2242494, 2346249)), (2252030, 2252052), '-'), (('G000006925', (3229969, 3230122)), (3230015, 3230037), '+'), (('G000299455', (1819843, 1851664)), (1826456, 1826478), '+'), (('G000299455', (3470062, 3470342)), (3470153, 3470175), '-')]\n",
      ">G000006925|865572|865846|865658|865680|-\n",
      "ttgttcgccagagcccgggggt-\n",
      ">G000006925|2242494|2346249|2252030|2252052|-\n",
      "ttgttcgccagagcccgggggg-\n",
      ">G000006925|3229969|3230122|3230015|3230037|+\n",
      "-tgttcgccagagcctgggggtg\n",
      ">G000299455|1819843|1851664|1826456|1826478|+\n",
      "-tgttcgccagagcccgggggtg\n",
      ">G000299455|3470062|3470342|3470153|3470175|-\n",
      "ttgttcgccagagcccgggggt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2045115, 2046271)), (2045388, 2046220), '+'), (('G000006925', (2227521, 2228997)), (2228114, 2228946), '+'), (('G000299455', (2892839, 2893945)), (2893077, 2893909), '+')]\n",
      ">G000006925|2045115|2046271|2045388|2046220|+\n",
      "atccctggatgtagaactgtgtggtgacggtcttccagccattcggctcctgctgtatcg\n",
      "aagcataccaggctatttcaatatcgctatgctgcggcagcatttaaccccttgtaattc\n",
      "atcgccataattgatttaattcacaaataaaactataatatggtgaaattaatgaaaaaa\n",
      "aacacagatgatggggctaaaatttacacaccacttaccctaaagctttatgactggtgg\n",
      "gttttgggagtatcaaatcggcttgcatggggatgtcctacaaaggaacaccttcttcca\n",
      "cactttctggaacatttaggtaacaaccatctggatattggtgttggaactgggttttac\n",
      "cttactcacgtacctgagagtagtctgatatctttaatggatttgaacgaagctagcctg\n",
      "aacgcggcatctacaagggctggggaatcaaaaattaaacataaaattagccatgatgtt\n",
      "tttgaaccttatcctgcggcgttacatggtcaatttgattccatttccatgttttacctt\n",
      "cttcactgcctgcctggaaatatatctacaaaaagctgtgtaatacgcaatgcggcgcag\n",
      "gccttaactgatgatggaactctatacggagccacaattcttggcgatggagttgtgcac\n",
      "aatagcttcggtcaaaaactgatgcgcatttacaatcagaaaggcatcttttcaaacaca\n",
      "aaagattccgaagaaggcttaacacatatactctcagagcatttcgagaatgttaaaacc\n",
      "aaggttcaaggtactgtagtaatgttttccgcttcagggaaaaaatagcatc\n",
      ">G000006925|2227521|2228997|2228114|2228946|+\n",
      "atccctggatgtagaactgtgtggtgacggtcttccagccattcggctcctgctgtatcg\n",
      "aagcataccaggctatttcaatatcgctatgctgcggcagcatttaaccccttgtaattc\n",
      "atcgccataattgatttaattcacaaataaaactataatatggtgaaattaatgaaaaaa\n",
      "aacacagatgatggggctaaaatttacacaccacttaccctaaagctttatgactggtgg\n",
      "gttttgggagtatcaaatcggcttgcatggggatgtcctacaaaggaacaccttcttcca\n",
      "cactttctggaacatttaggtaacaaccatctggatattggtgttggaactgggttttac\n",
      "cttactcacgtacctgagagtagtctgatatctttaatggatttgaacgaagctagcctg\n",
      "aacgcggcatctacaagggctggggaatcaaaaattaaacataaaattagccatgatgtt\n",
      "tttgaaccttatcctgcggcgttacatggtcaatttgattccatttccatgttttacctt\n",
      "cttcactgcctgcctggaaatatatctacaaaaagctgtgtaatacgcaatgcggcgcag\n",
      "gccttaactgatgatggaactctatacggagccacaattcttggcgatggagttgtgcac\n",
      "aatagcttcggtcaaaaactgatgcgcatttacaatcagaaaggcatcttttcaaacaca\n",
      "aaagattccgaagaaggcttaacacatatactctcagagcatttcgagaatgttaaaacc\n",
      "aaggttcaaggtactgtagtaatgttttccgcttcagggaaaaaatagcatc\n",
      ">G000299455|2892839|2893945|2893077|2893909|+\n",
      "attcctggatgtagaactgtgtggtgacggtcttccagccattcggctcctgctgtatcg\n",
      "aagcataccaggctatttcaatatcgctatgctgcagcagcatttaaccccttgtaattc\n",
      "atcgccataattgatttaattcacaaataaaactataacatggtgaaatcaatgaaaaaa\n",
      "aacacagatgatggggctaaaatttacacaccacttaccctaaagctttatgactggtgg\n",
      "gttttgggagtatcaaatcggcttgcatggggatgtcctacaaaggaacaccttcttcca\n",
      "cactttctggaacatgtaggtaacaaccatctggatattggtgttggaactgggttttac\n",
      "cttactcacgtacctgagagtagtctgatatctttaatggatttgaacgaagctagcctg\n",
      "aacgcggcatctacaagggctggggaatcaaaaattaaacataaaattagccatgatgtt\n",
      "tttgaaccttatcccgcggcgttacatggtcaatttgattccatttccatgttttacctt\n",
      "cttcactgcctgcctggaaatatatctacaaaaagctgtgtaatacgcaatgcggcgcag\n",
      "gccttaactgacgatggaactctatacggagccacaattcttggcgatggagttgtgcac\n",
      "aatagcttcggtcaaaaactgatgcgcatttacaatcagaaaggcatcttttcaaacaca\n",
      "aaagattccgaagaaggcttaacacatatactctcagagcatttcgagaatgttaaaacc\n",
      "aaggttcaaggtactgtagtaatgttttccgcttcagggaaaaaatagcatc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (877185, 898585)), (877213, 877252), '-'), (('G000006925', (1069482, 1076758)), (1072402, 1072441), '-'), (('G000299455', (3226561, 3226635)), (3226578, 3226617), '+'), (('G000299455', (3226712, 3229721)), (3226759, 3226798), '+'), (('G000299455', (3440781, 3470001)), (3459470, 3459509), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1080\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|877185|898585|877213|877252|-\n",
      "ttaaagaagagctcgtacgaaagtgcgagcttttttttc-\n",
      ">G000006925|1069482|1076758|1072402|1072441|-\n",
      "ttaaagaagagcgcgtacgaaagtacgggcttttttttc-\n",
      ">G000299455|3226561|3226635|3226578|3226617|+\n",
      "-taaagaagagctcgtacgaaagtacgggcttttttttcg\n",
      ">G000299455|3226712|3229721|3226759|3226798|+\n",
      "-taaagaagagcccgtacgaaagtacgggcttttttttcg\n",
      ">G000299455|3440781|3470001|3459470|3459509|+\n",
      "-taaagaagagctcgtacgaaagtgcgagcttttttttcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2786239, 2786282), '+'), (('G000299455', (1254267, 1277763)), (1259551, 1259594), '-'), (('G000299455', (1551165, 1574182)), (1567805, 1567848), '+')]\n",
      ">G000006925|2770832|2816659|2786239|2786282|+\n",
      "-aaaatgcgaccggtaactggatcaaagtcgttcagcgtctacc\n",
      ">G000299455|1254267|1277763|1259551|1259594|-\n",
      "caaaatgcgaccggtaactggatcaaagtcgttcagcgtctac-\n",
      ">G000299455|1551165|1574182|1567805|1567848|+\n",
      "-aaaatgcgacagggaactggatcaaaatcgttcagcgtgtacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (488828, 489043), '-'), (('G000299455', (315576, 316388)), (315869, 316084), '-'), (('G000299455', (3819953, 3834467)), (3832675, 3832890), '+')]\n",
      ">G000006925|487251|491968|488828|489043|-\n",
      "cgttcgcttttgctgaactgctggcgcgggtgcgcactctcctgcggcgcggggcggcgg\n",
      "tgattatcgaaagtcagtttcaggttgccgatctgatggtcgatctcgtcagccgcaaag\n",
      "tcacccgcagcggcacgcgcatcactctgaccagtaaagaatttactctgctggagtttt\n",
      "ttctccgccatcagggcgaagtgctgccccgctcg-\n",
      ">G000299455|315576|316388|315869|316084|-\n",
      "cctttgcgtttgccgaactgctcgcccgggtgagaacccttctgaggcggggaaacacga\n",
      "tgatcacggaaagccagtttaaggtggctgacctctcgattgatctcgtatccagaaaag\n",
      "tcagtcgcgccggaaaccgcattgtgctcaccagtaaagagttcagcctgctggaattct\n",
      "tcattcgccatcagggagaggttcttccccgctcc-\n",
      ">G000299455|3819953|3834467|3832675|3832890|+\n",
      "-gttcgcttttgctgaactgctggcgcgggtgcgcactctcctgcggcgcggggcggcgg\n",
      "tgattatcgaaagtcagtttcaggttgccgatctgatggtcgatctcgtcagccgcaaag\n",
      "tcacccgcagcggcacgcgcatcactctgaccagtaaagagtttactctgctggagtttt\n",
      "tcctccgccatcagggcgaagtgctgccccgctcgc\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1696422, 1718400)), (1712104, 1712251), '+'), (('G000006925', (1815406, 1827803)), (1815503, 1815650), '+'), (('G000006925', (2006629, 2008384)), (2007087, 2007234), '+'), (('G000006925', (4764001, 4764173)), (4764001, 4764148), '+'), (('G000299455', (2269177, 2270929)), (2270326, 2270473), '-'), (('G000299455', (2737179, 2754578)), (2737635, 2737782), '+'), (('G000299455', (4667815, 4670271)), (4668685, 4668832), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1696422|1718400|1712104|1712251|+\n",
      "-gactatcattgcgtggtttttgctgacggtcgttgtgactttgtgaaggtagtccctgc\n",
      "ggatatttgcgatacaggaatgcagtcgctgtattttgcgtttctgcttctgccagttgt\n",
      "tgctgaatttgaccttccggcttaactg\n",
      ">G000006925|1815406|1827803|1815503|1815650|+\n",
      "-gactatcattgcgtggtttttgctgacggccgttgtgactttgtgaaggtagtccctgc\n",
      "ggatatttgcgatacaggaatgcagtcgctgtattttgcgtttctgcttctgccagttgt\n",
      "tgctgaatttgaccttgcggcttaactg\n",
      ">G000006925|2006629|2008384|2007087|2007234|+\n",
      "-gactatcattacgtggtttttgctgacggtcgttgtgactttgtgaaggtagtccctgc\n",
      "ggatatttgcgatacaggaatgcagtcgctgtattttgcgtttctgcttctgccagttgt\n",
      "tgctgaatttgaccttgcggcttaactg\n",
      ">G000006925|4764001|4764173|4764001|4764148|+\n",
      "-gacgatcatcacgtgttttttgctgacagtcgctgtgactttgtgaaagtagtctctgc\n",
      "gtatgtttgcgatatgggaatgcagttgctgtattttgtgtttctgcttctgccagttgt\n",
      "tgctgaatttaaccttgcggcttaactg\n",
      ">G000299455|2269177|2270929|2270326|2270473|-\n",
      "tgactatcattgcgtggtttttgctgacggtcgttgtgactttatgaaggtagtccctgc\n",
      "ggatatttgcgatacgggaatgcagtcgctgtattttgcgtttctgcttctgccagttat\n",
      "tgctgaatttgaccttgcggcttaact-\n",
      ">G000299455|2737179|2754578|2737635|2737782|+\n",
      "-gactatcattgcgtggtttttgctgacggccgttgtgactttgtgaaggtagtccctgc\n",
      "ggatatttgcgatacaggaatgcagtcgctgtattttgcgtttctgcttctgccagttgt\n",
      "tgctgaatttgaccttgcggcttaactg\n",
      ">G000299455|4667815|4670271|4668685|4668832|+\n",
      "-gactatcattgcgtggtttttgctgacggtcgttgtgactttgtgaaggtagtccctgc\n",
      "ggatttttgcgatacaggaatgcagtcgctgtattttgcgtttctgcttctgccagttgt\n",
      "tgctgaatttgaccttgcggcttaactg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3040656, 3065083)), (3052415, 3052516), '-'), (('G000006925', (4462252, 4478752)), (4466192, 4466293), '+'), (('G000299455', (942831, 956197)), (944336, 944437), '+'), (('G000299455', (4713031, 4748324)), (4735655, 4735756), '-')]\n",
      ">G000006925|3040656|3065083|3052415|3052516|-\n",
      "accaaggacacggggattttcaatcccctgctctaccgactgagctatccgggcaacggg\n",
      "gcgcattaaacctgattcgcctcgtctcgtcaatcaaattt-\n",
      ">G000006925|4462252|4478752|4466192|4466293|+\n",
      "-ccaaggacacggggattttcaatcccctgctctaccgactgagctatccgggcaacggg\n",
      "gcgcattaaaccgtaatctgcacatctcgtcaacctaatttc\n",
      ">G000299455|942831|956197|944336|944437|+\n",
      "-ccaaggacacggggattttcaatcccctgctctaccgactgagctatccgggcaacggg\n",
      "gcgcattaaacctgattcgcctcgtctcgtcaatcaaatttc\n",
      ">G000299455|4713031|4748324|4735655|4735756|-\n",
      "accaaggacacggggattttcaatcccctgctctaccgactgagctatccgggcaacggg\n",
      "gcgcattaaaccgtaatctgcacatctcgtcaacctaattt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2141740, 2142061), '-'), (('G000299455', (1911336, 1921536)), (1918319, 1918640), '+'), (('G000299455', (3326617, 3362069)), (3338571, 3338892), '+')]\n",
      ">G000006925|2138845|2144864|2141740|2142061|-\n",
      "aatccggtcaacaggcgatcaccaacgtaccactgaccattctcgacgccatcaacgccg\n",
      "caggtggcctgaccgacaccgctgactggcgcaacgtggtgctaacacacaatggtcgtg\n",
      "aagaacgca-tttctttacaggcgctgatgcaaaacggcgacctcaatcagaaccgactg\n",
      "ctttatcccggcgatattctctacgtgccgcgtaatgatgatctgaaagtatttgtgatg\n",
      "ggtgaagtgaagaaacagagcaccctgaaaatggactttagcggcatgaccctgactgaa\n",
      "gctctggccaatgccgaaggta-\n",
      ">G000299455|1911336|1921536|1918319|1918640|+\n",
      "aatccggtcaacaggcgattaccaacgtaccactgaccattctcgatgccatcaacgccg\n",
      "caggtggcctgaccgacaccgctgactggcgcaacgtggtgctaacacacaatggtcgtg\n",
      "aagaacgca-tttctttgcaggcgctgatgcaaaacggcgacctcaatcagaaccgactg\n",
      "ctttaccccggcgatattctctacgtgccgcgtaatgatgatctgaaagtatttgtgatg\n",
      "ggtgaagtgaagaaacagagcaccctgaaaatggactttagcggtatgaccctgactgaa\n",
      "gccctgggcaatgctgaaggta-\n",
      ">G000299455|3326617|3362069|3338571|3338892|+\n",
      "-atcgggtcagcagccgattaccaatattcctctgaccatcatggatgcgattaacgccg\n",
      "cgggcggtctgaccgcagatgcagactggcgcaacgtggtgctcacccagaatgg-cgta\n",
      "aagaccaaagtgaacctctatgcgctgatgcagcgtggcgacttgcgccagaacaaactg\n",
      "ctgcacccgggcgatattttatttattccgcgcaacgatgatctgaaagtcttcgtgatg\n",
      "ggtgaagtgggtaaacagagcacgctgaaaatggaccgcagcggtatgaccctggcggaa\n",
      "gcgctgggtaacgctgagggtat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (459861, 468373)), (463187, 463217), '+'), (('G000006925', (4238902, 4247776)), (4240691, 4240721), '-'), (('G000299455', (3880492, 3886348)), (3882992, 3883022), '-'), (('G000299455', (4912548, 4921423)), (4919603, 4919633), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1204\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1204\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1062\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|459861|468373|463187|463217|+\n",
      "-gccgctgcatgcttccggtgcgggcaaagc\n",
      ">G000006925|4238902|4247776|4240691|4240721|-\n",
      "tgccgatgcacgcttccggtgcgggtaaag-\n",
      ">G000299455|3880492|3886348|3882992|3883022|-\n",
      "tgccgctgcatgcttccggtgcgggcaaag-\n",
      ">G000299455|4912548|4921423|4919603|4919633|+\n",
      "-gccgatgcacgcttccggtgcgggtaaagc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4535175, 4535565)), (4535185, 4535376), '+'), (('G000006925', (4535976, 4543278)), (4535980, 4536171), '+'), (('G000299455', (4504808, 4509341)), (4509146, 4509337), '-'), (('G000299455', (4509788, 4510178)), (4509977, 4510168), '-')]\n",
      ">G000006925|4535175|4535565|4535185|4535376|+\n",
      "-cattcccacccttccggaagttcaaacggtttttcctcatcgctaattggcggcagtgg\n",
      "tttttgtttttttattttcccttctttcaccaattgagctttttcctgcgcaatacgttt\n",
      "gagcagttcagaggcgggttcgtcattcggatcctgcggcacaagtttgcccattacggc\n",
      "cagttgcagaat\n",
      ">G000006925|4535976|4543278|4535980|4536171|+\n",
      "-cactcccatccctccggcaattcaaacggcttctcttcctcactaatttctggcagtgg\n",
      "tttttgctttttaattttgccttgcttcaccagctctgctttttccgccgcaatacgctt\n",
      "cagcaggtccgacgccggttcatcattcggatcctgcggcaccagtttaccgcgcaccgc\n",
      "cagttccagaat\n",
      ">G000299455|4504808|4509341|4509146|4509337|-\n",
      "ccactcccatccctccggcaattcaaacggcttctcttcctcgctaatttccggcagtgg\n",
      "tttttgctttttaattttaccttgcttcaccagctctgctttttccgccgcaatacgctt\n",
      "cagcagctccgacgccggttcatcgttcgggtcttgcggcaccagtttaccgcgcactgc\n",
      "cagttccagaa-\n",
      ">G000299455|4509788|4510178|4509977|4510168|-\n",
      "ccactcccacccttccggaagttcaaacggtttttcctcatcgctaatcgggggcaacgg\n",
      "tttttgtttttttattttcccttctttcaccagttgagctttttcctgcgcaattcgttt\n",
      "gagcagttcagaggcgggttcgtcattcggatcctgcggcacaagtttacccattacggc\n",
      "cagttgcagaa-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3097629, 3098003), '+'), (('G000299455', (47809, 50193)), (48854, 49228), '-'), (('G000299455', (3140498, 3142451)), (3141577, 3141943), '-'), (('G000299455', (4771847, 4785261)), (4783265, 4783639), '+')]\n",
      ">G000006925|3083675|3099254|3097629|3098003|+\n",
      "-cgataaatccgaccaacacgcaaccagtacaccaccaccgttaacaacccttacctgca\n",
      "tacgttgcagcaccgcacatatcacggaaacggacaacgcctggctgtactcgctgtcac\n",
      "accagactaatgacgacggcgaaacagaatggattcattttacaggcagcggctatctgt\n",
      "tacgtaccgatgcatggtcgtacccggttctgcggcttaaacgcctgggcctgtccagaa\n",
      "cgttccgtcgtctggtcgtcactctcacccggcgttatggcgtcagtctcattcatctgg\n",
      "acgccggcgcagaatgtctgccggggttccccactttcgactggtaatcaggacaaaaga\n",
      "cactggatgaaatta\n",
      ">G000299455|47809|50193|48854|49228|-\n",
      "tcgataaatccgaccaacacgcaaccagtacaccaccaccgttaacaacccttacctgca\n",
      "tacgttgcagcaccgcacatatcacggaaacggacaacgcctggctgtactcgctgtcac\n",
      "accagactaatgacgacggcgaaacagaatggattcattttacaggcagcggctatctgt\n",
      "tacgtaccgatgcatggtcgtacccggttctgcggcttaaacgcctgggcctgtccagaa\n",
      "cgttccgtcgtctggtcgtcactctcacccggcgttatggcgtcagtctcattcatctgg\n",
      "acgccggcgcagaatgtctgccggggttccccactttcgactggtaatcaggacaaaaga\n",
      "cactggatgaaatt-\n",
      ">G000299455|3140498|3142451|3141577|3141943|-\n",
      "ttgatgaatctgaccggaccgcaactgacagcatgccaccctccgctgccctgagcggta\n",
      "tacgctgcagcaccgcacatatcacggaaacagacaacgcctggctgtactcactgtcgc\n",
      "atcagaccagtgacgttggtgaatcagaatggattcatttcacaggcagcggctatctgt\n",
      "tacgtaccgatgcgtggtcgtatccggttctgcgacttaaacgactgggactgtcaaaaa\n",
      "cgttccgtcgtctggttgtcacactcatccggcgttatggcgtcagcctcattcatctgg\n",
      "atgccagcgctgaatgcctgcccggtttacccactttcgactggtaaccagga-------\n",
      "-acaacatgaaatc-\n",
      ">G000299455|4771847|4785261|4783265|4783639|+\n",
      "-cgataaatccgaccaacacgcaaccagtacaccaccaccgttaacaacccttacctgca\n",
      "tacgttgcagcaccgcacatatcacggaaacggacaacgcctggctgtactcgctgtcac\n",
      "accagactaatgacgacggcgaaacagaatggattcattttacaggcagcggctatctgt\n",
      "tacgtaccgatgcatggtcgtacccggttctgcggcttaaacgcctgggcctgtccagaa\n",
      "cgttccgtcgtctggtcgtcactctcacccggcgttatggcgtcagtctcattcatctgg\n",
      "acgccggcgcagaatgtctgccggggttccccactttcgactggtaatcaggacaaaaga\n",
      "cactggatgaaatta\n",
      "\n",
      "\n",
      "19 [(('G000006925', (237115, 237939)), (237457, 237563), '+'), (('G000006925', (538623, 539106)), (538624, 538730), '+'), (('G000006925', (710708, 711191)), (711084, 711190), '-'), (('G000006925', (716211, 716694)), (716212, 716318), '+'), (('G000006925', (901842, 902325)), (901843, 901949), '+'), (('G000006925', (1096742, 1097225)), (1097118, 1097224), '-'), (('G000006925', (1632137, 1632620)), (1632138, 1632244), '+'), (('G000006925', (1835665, 1836148)), (1836041, 1836147), '-'), (('G000006925', (1878088, 1878571)), (1878089, 1878195), '+'), (('G000006925', (1943954, 1944437)), (1943955, 1944061), '+'), (('G000006925', (1961600, 1962083)), (1961601, 1961707), '+'), (('G000006925', (2069131, 2069614)), (2069132, 2069238), '+'), (('G000006925', (2346253, 2346736)), (2346254, 2346360), '+'), (('G000006925', (2473341, 2477789)), (2477307, 2477413), '+'), (('G000006925', (2954172, 2954655)), (2954548, 2954654), '-'), (('G000006925', (4394284, 4394772)), (4394660, 4394765), '-'), (('G000006925', (4824718, 4825201)), (4825094, 4825200), '-'), (('G000299455', (4557177, 4559928)), (4557177, 4557283), '+'), (('G000299455', (4579022, 4579199)), (4579093, 4579199), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1384\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1384\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\n",
      "Reallocating..done. *alloclen = 1214\n",
      "\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|237115|237939|237457|237563|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|538623|539106|538624|538730|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|710708|711191|711084|711190|-\n",
      "gtggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgta-\n",
      ">G000006925|716211|716694|716212|716318|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|901842|902325|901843|901949|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgtaa\n",
      ">G000006925|1096742|1097225|1097118|1097224|-\n",
      "gtggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgta-\n",
      ">G000006925|1632137|1632620|1632138|1632244|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgtaa\n",
      ">G000006925|1835665|1836148|1836041|1836147|-\n",
      "gtggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgta-\n",
      ">G000006925|1878088|1878571|1878089|1878195|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|1943954|1944437|1943955|1944061|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|1961600|1962083|1961601|1961707|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|2069131|2069614|2069132|2069238|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgtaa\n",
      ">G000006925|2346253|2346736|2346254|2346360|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcactgtaa\n",
      ">G000006925|2473341|2477789|2477307|2477413|+\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000006925|2954172|2954655|2954548|2954654|-\n",
      "gtggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgta-\n",
      ">G000006925|4394284|4394772|4394660|4394765|-\n",
      "-tggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgta-\n",
      ">G000006925|4824718|4825201|4825094|4825200|-\n",
      "gtggtcaacaaaaactggccaccgagttagagtttttccagtatcgattttccgattcgt\n",
      "ttgggggtaacccaccgttatattcgtgcggtcttagtgcgctgta-\n",
      ">G000299455|4557177|4559928|4557177|4557283|+\n",
      "-tggtcaactaaaactagcctccgcgttagagtttttccagtatcggttttctgattcgt\n",
      "ttggtggtaacccaccattatattcgtgcggtcttagtgcgctgtaa\n",
      ">G000299455|4579022|4579199|4579093|4579199|+\n",
      "-tggtcaacaaaaactggtcacaatttcagtttttttccagtatctattttccgattcgt\n",
      "tcggcggtaatccaccgttatattcatgaggccggagtgcactataa\n",
      "\n",
      "\n",
      "18 [(('G000006925', (237115, 237939)), (237697, 237938), '+'), (('G000006925', (538623, 539106)), (538864, 539105), '+'), (('G000006925', (710708, 711191)), (710709, 710950), '-'), (('G000006925', (716211, 716694)), (716452, 716693), '+'), (('G000006925', (901842, 902325)), (902083, 902324), '+'), (('G000006925', (1096742, 1097225)), (1096743, 1096984), '-'), (('G000006925', (1632137, 1632620)), (1632378, 1632619), '+'), (('G000006925', (1835665, 1836148)), (1835666, 1835907), '-'), (('G000006925', (1878088, 1878571)), (1878329, 1878570), '+'), (('G000006925', (1943954, 1944437)), (1944195, 1944436), '+'), (('G000006925', (1961600, 1962083)), (1961841, 1962082), '+'), (('G000006925', (2069131, 2069614)), (2069372, 2069613), '+'), (('G000006925', (2346253, 2346736)), (2346494, 2346735), '+'), (('G000006925', (2473341, 2477789)), (2477547, 2477788), '+'), (('G000006925', (2954172, 2954655)), (2954173, 2954414), '-'), (('G000006925', (4394284, 4394772)), (4394285, 4394526), '-'), (('G000006925', (4824718, 4825201)), (4824719, 4824960), '-'), (('G000299455', (4557177, 4559928)), (4557417, 4557657), '+')]\n",
      ">G000006925|237115|237939|237697|237938|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|538623|539106|538864|539105|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|710708|711191|710709|710950|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "a-\n",
      ">G000006925|716211|716694|716452|716693|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|901842|902325|902083|902324|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|1096742|1097225|1096743|1096984|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgctcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "a-\n",
      ">G000006925|1632137|1632620|1632378|1632619|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|1835665|1836148|1835666|1835907|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "a-\n",
      ">G000006925|1878088|1878571|1878329|1878570|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|1943954|1944437|1944195|1944436|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|1961600|1962083|1961841|1962082|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|2069131|2069614|2069372|2069613|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|2346253|2346736|2346494|2346735|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|2473341|2477789|2477547|2477788|+\n",
      "-gctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      ">G000006925|2954172|2954655|2954173|2954414|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagcccg\n",
      "gtggttttcaggtgttcaggtagagagcatcggctaggtacggccagcgcttacccgtcc\n",
      "a-\n",
      ">G000006925|4394284|4394772|4394285|4394526|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "a-\n",
      ">G000006925|4824718|4825201|4824719|4824960|-\n",
      "ggctcatactctgtctgatctggtatcgccacaataactgccggaactgcctgctcgtat\n",
      "aatgactgccctgatcgctgtggaacatcaccccgccgggcttaccacgggtttcccatg\n",
      "ccatttccagcgctttcatggtgagcctgctgtccggcgagaacgacatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "a-\n",
      ">G000299455|4557177|4559928|4557417|4557657|+\n",
      "-gctcatactctgtctgatctggtatcgccacagtaactgccggaactgcctgctcgtat\n",
      "aatgactgccttgatcgct-tggaacatcaccccgacgggcttaccacgggtttcccatg\n",
      "ccatttccagtgctttcatggtaagcctgctgtccggcgagaacgagatggcccagccca\n",
      "ctggttttcttgcgaacaggtcgagaacaacggcgaggtacgcccagcgcttacccgtcc\n",
      "ag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (295768, 299358)), (296723, 296750), '-'), (('G000006925', (1425133, 1465118)), (1463118, 1463145), '+'), (('G000299455', (2199778, 2263225)), (2261225, 2261252), '+'), (('G000299455', (4166042, 4170922)), (4169940, 4169967), '+')]\n",
      ">G000006925|295768|299358|296723|296750|-\n",
      "gctggtggcgctggtggacaaagcgca-\n",
      ">G000006925|1425133|1465118|1463118|1463145|+\n",
      "-ctgctggcgctggtggaaaaagcgcag\n",
      ">G000299455|2199778|2263225|2261225|2261252|+\n",
      "-ctgctggcgctggtggaaaaagcgcag\n",
      ">G000299455|4166042|4170922|4169940|4169967|+\n",
      "-ctggtggcactggtcgataaagcgcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3800049, 3800078), '-'), (('G000299455', (91523, 102755)), (100301, 100330), '+'), (('G000299455', (1033723, 1033752)), (1033723, 1033752), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3788320|3808082|3800049|3800078|-\n",
      "caggccgtttgccagcacgccgcctttca-\n",
      ">G000299455|91523|102755|100301|100330|+\n",
      "-aggccgtttgccagcacgccacctttcag\n",
      ">G000299455|1033723|1033752|1033723|1033752|+\n",
      "-agaccgtctgccagcacgccgcctttcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2161028, 2172336)), (2163233, 2163278), '-'), (('G000299455', (1889936, 1901256)), (1895910, 1895955), '+'), (('G000299455', (1889936, 1901256)), (1899006, 1899051), '+')]\n",
      ">G000006925|2161028|2172336|2163233|2163278|-\n",
      "acgaatccggtggcgatagttaaagccatcagtgtcaggttattg-\n",
      ">G000299455|1889936|1901256|1895910|1895955|+\n",
      "-cgaaaccagtagcgatggtgagcgccattaacgaaaggttattga\n",
      ">G000299455|1889936|1901256|1899006|1899051|+\n",
      "-cgaatccggtggcgatagttaacgccatcagtgtcaggttattga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3017064, 3017738), '+'), (('G000299455', (971316, 975991)), (975043, 975717), '-'), (('G000299455', (1477869, 1484878)), (1478218, 1478892), '+')]\n",
      ">G000006925|3016790|3036697|3017064|3017738|+\n",
      "-gcagtcagtttttcgtaggcagcaaccgccagttcaacttcagaaccggtggcgatgaa\n",
      "gatcagttccggctgaccagcacagtctttcagcacataaccaccgcgcgcgatgtttgc\n",
      "cagctgctcttcagtacgttcctgctgcgccaggttctgacgggagaggataagcgcagt\n",
      "cgggccgtcctgacgctcaacgccgtatttccacgcgaccgcggattcaacctggtcaca\n",
      "cggacgccatgtagacatgttcggggtcacgcgcagagaagcgacctgctctaccggctg\n",
      "gtgagtcgggccatcttcgcccagaccgatggagtcgtgggtgtaaaccatcacctggcg\n",
      "ctgtttcatcagcgcagccatacgtacggcgttacgtgcgtattccacgaacatcaggaa\n",
      "ggtggaggtgtacggcaggaagccaccgtgcagggagataccgttagcaatcgcggtcat\n",
      "accgaactcgcgaacaccgtagtggatgtagttacctgcagcatcttcgttgattgcttt\n",
      "agaaccagaccacagggtcaggttagacggtgccaggtcagcagagccgccgaggaattc\n",
      "aggcaacagcgggccgaacgcttcgatagcattctgcgacgctttacggctggcgatttt\n",
      "cgccggattagcctg\n",
      ">G000299455|971316|975991|975043|975717|-\n",
      "ggcagtcagtttttcgtaggcagcaacagccagttcaacttctgaaccggtagcgatgaa\n",
      "aatcagttccggctgaccggcgcagtctttcagcacataaccaccgcgcgcgatgtttgc\n",
      "cagttgctcttcagttcgttcctgctgcgccaggttctgacgggagaggatcagtgcggt\n",
      "cgggccgtcctgacgctcaacaccgtatttccacgcgaccgcggattcaacctggtcaca\n",
      "cggacgccatgtagacatgttcggggttacgcgcagagaagcgacctgctctaccggctg\n",
      "gtgagtcgggccatcttcgcccagaccgatggagtcgtgggtgtaaaccatcacctgacg\n",
      "ctgtttcatcagcgcagccatacgtacggcgttacgtgcgtattccacgaacatcaggaa\n",
      "ggtggaggtgtacggcaggaaaccaccgtgcagggagataccgttagcaatcgcggtcat\n",
      "accgaactcgcgaacaccgtagtggatgtagttacccgcagcatcttcgttgattgcttt\n",
      "agaaccagaccacagggtcaggttagacggtgccaggtcagcagaaccgccgaggaattc\n",
      "cggcaacagcggaccgaaagcttcgatagcattctgagacgctttacggctggcgatttt\n",
      "cgccgggttagcct-\n",
      ">G000299455|1477869|1484878|1478218|1478892|+\n",
      "-cctgctaatttctctgccgcttgcagggtaatttccatctctgaaccggtggcaatcaa\n",
      "aataatatctggcttaccgccgctgtctttcagcacataaccgccacgagcaatctcttt\n",
      "aacctgatccggcgtacgttccacctgggccagattctgccttgagaggatcagtgccgt\n",
      "cggtccgttgtggcgctcaaccgccagcttccagcccaccgccgcttccacctgatcgca\n",
      "cggtcgccaggtgctgaaatttggcgttaagcgcaggctggccagttgctcaacagcctg\n",
      "gtgcgtcggaccatcttcgcccaggccaattgagtcgtgggtataaaccataatctgccg\n",
      "cgctttcatcagtgccgccatccgcgcggcgttacgggcgtattcaacaaacatcaggaa\n",
      "cgtcgcggtatacggcacaaagccgccgtggtgcgcgatgccgttggcgatagcggtcat\n",
      "gccaaattcacgcaccccgtagtgaatgtagttgcccgctggatcttccttcagcgaaac\n",
      "agaacctttccagatggtcaggttgctgggagccagatccgccgaaccgccgagcaactc\n",
      "aggcagcatcggcccgtaagcgttaagcgtattttgcgaagccttacgggtagcgatttt\n",
      "cgccggatttgcctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1611778, 1618593)), (1614366, 1614397), '-'), (('G000299455', (2625906, 2635627)), (2633008, 2633039), '+'), (('G000299455', (4829613, 4829644)), (4829613, 4829644), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1611778|1618593|1614366|1614397|-\n",
      "agcgaggcggtaggttcatcgaggatcagaa-\n",
      ">G000299455|2625906|2635627|2633008|2633039|+\n",
      "-gcgaggctgtaggttcatcgaggatcagaat\n",
      ">G000299455|4829613|4829644|4829613|4829644|-\n",
      "agcgaggcggtaggttcgtcgagcagcagaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (827107, 836680)), (831341, 831472), '-'), (('G000299455', (3480865, 3520974)), (3499272, 3499403), '+'), (('G000299455', (3520976, 3527694)), (3525388, 3525519), '+')]\n",
      ">G000006925|827107|836680|831341|831472|-\n",
      "ttatggctcagtatcagcaaatgaaagcacaaggcattcgctttgtggtcgccagcggga\n",
      "atcaatattatcagttgatctctttcttccctgaaattgcgaatgaaattgcctttgtgg\n",
      "ctgaaaacggc-\n",
      ">G000299455|3480865|3520974|3499272|3499403|+\n",
      "ttatggctcagtatcagcaaatgaaagcacaaggcattcgctttgtggtcgccagcggga\n",
      "atcaatattatcagttgatctctttcttccctgaaattgcgaatgaaattgcctttgtgg\n",
      "ctgaaaacggc-\n",
      ">G000299455|3520976|3527694|3525388|3525519|+\n",
      "-tatggcgcaatatcaggaactgaaaaagcgcggcattgagttcgttgtcgccagcggta\n",
      "atcagtattaccagcttatttcgttctttcctgagctaaaggatgagatctcttttgtcg\n",
      "cggaaaacggcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (907508, 907840)), (907689, 907840), '+'), (('G000006925', (1395000, 1395332)), (1395000, 1395151), '-'), (('G000299455', (2931465, 2931723)), (2931572, 2931723), '+')]\n",
      ">G000006925|907508|907840|907689|907840|+\n",
      "-tgaccagataaggtccggatcaacaacaaccggtttcctcacctttactctcgagagtt\n",
      "ttttgcgggcgctttgccagtccttgcgagcctgttcagacgggaataagccgtagccag\n",
      "aattatatacatcaccacttgcgaccagttct\n",
      ">G000006925|1395000|1395332|1395000|1395151|-\n",
      "atgaccagataaggtccggatcaacaacaaccggtttcctcacctttactctcgagagtt\n",
      "ttttgcgggcgctttgccagtccttgcgagcctgttcagacgggaataagccgtagccag\n",
      "aattatatacatcaccacttgcgaccagttc-\n",
      ">G000299455|2931465|2931723|2931572|2931723|+\n",
      "-cgaccagataaggtccggatcaaccactgcaggtttcttcacctttgcccttgagagtt\n",
      "ttttgcgagcattttgccagtccttacgcgcctgttcagacgggaataacccgtaaccag\n",
      "agttgtatacatcgccactggcaaccagttct\n",
      "\n",
      "\n",
      "4 [(('G000006925', (712850, 712894)), (712850, 712878), '+'), (('G000006925', (721457, 721684)), (721501, 721528), '+'), (('G000006925', (2060397, 2060639)), (2060568, 2060595), '-'), (('G000299455', (3608183, 3608425)), (3608354, 3608381), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|712850|712894|712850|712878|+\n",
      "-cctacggcacctccgccggcagtgccgg\n",
      ">G000006925|721457|721684|721501|721528|+\n",
      "-cctacggcacctccgccggcagtgccg-\n",
      ">G000006925|2060397|2060639|2060568|2060595|-\n",
      "gcctacggcacctccgcaggcagtgct--\n",
      ">G000299455|3608183|3608425|3608354|3608381|-\n",
      "gcctatggcacctcagcaggtagtgcc--\n",
      "\n",
      "\n",
      "4 [(('G000006925', (734007, 734128)), (734102, 734128), '+'), (('G000006925', (1933459, 1933520)), (1933459, 1933484), '-'), (('G000299455', (2182436, 2182497)), (2182471, 2182497), '+'), (('G000299455', (2909146, 2909207)), (2909146, 2909172), '-')]\n",
      ">G000006925|734007|734128|734102|734128|+\n",
      "-tccggaccgctgacacaggcatttga\n",
      ">G000006925|1933459|1933520|1933459|1933484|-\n",
      "-tccggaccgctgacacaggcatttg-\n",
      ">G000299455|2182436|2182497|2182471|2182497|+\n",
      "-gcggccccactgaaacaggcatttga\n",
      ">G000299455|2909146|2909207|2909146|2909172|-\n",
      "ggcggccccactgaaacaggcatttg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (728422, 728502), '+'), (('G000299455', (2174641, 2181396)), (2176819, 2176899), '+'), (('G000299455', (2910238, 2916987)), (2914729, 2914809), '-'), (('G000299455', (3602211, 3602291)), (3602211, 3602291), '-')]\n",
      ">G000006925|726235|732999|728422|728502|+\n",
      "-atgcccgggcagatgatctggtgcgcaataacggtatcgcggctaatgcggtggccctg\n",
      "cataaggatcatatcgtcggg\n",
      ">G000299455|2174641|2181396|2176819|2176899|+\n",
      "-atgcccgggcagatgatctggtgcgcaataacggaatagcggccaatgcggtggccctg\n",
      "cataaggatcacattgtcggg\n",
      ">G000299455|2910238|2916987|2914729|2914809|-\n",
      "aatgcccgggcagatgatctggtgcgcaataacgggatagcggccaatgcggtggccctg\n",
      "cataaggatcaaattgtcgg-\n",
      ">G000299455|3602211|3602291|3602211|3602291|-\n",
      "aatgcccgcgcagacgatctggtgcgcaataacggctatgccgccaacgccatccagctg\n",
      "catcaggatcatatcgtcgg-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2161028, 2172336)), (2171351, 2171376), '-'), (('G000006925', (4598788, 4607203)), (4601833, 4601858), '-'), (('G000299455', (423834, 433368)), (431548, 431573), '-'), (('G000299455', (1889936, 1901256)), (1890896, 1890921), '+'), (('G000299455', (4431865, 4470166)), (4455878, 4455903), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1052\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2161028|2172336|2171351|2171376|-\n",
      "tagtcatcggcgccaatctccagcc-\n",
      ">G000006925|4598788|4607203|4601833|4601858|-\n",
      "-agtcgtcagcgccaatttccagccc\n",
      ">G000299455|423834|433368|431548|431573|-\n",
      "tagtcgtcagcgccaatctccaggc-\n",
      ">G000299455|1889936|1901256|1890896|1890921|+\n",
      "-aatcatctgcgccaatctccagccc\n",
      ">G000299455|4431865|4470166|4455878|4455903|+\n",
      "-agtcgtcagcgccaatttccagccc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4143132, 4163489)), (4153321, 4153354), '-'), (('G000299455', (3786605, 3808525)), (3793393, 3793426), '+'), (('G000299455', (4996216, 5016676)), (5006443, 5006476), '+')]\n",
      ">G000006925|4143132|4163489|4153321|4153354|-\n",
      "tggcggcggctcacgcggtgcataacggcctga-\n",
      ">G000299455|3786605|3808525|3793393|3793426|+\n",
      "-ggcggcggctcatgccgtgcataacggtctgac\n",
      ">G000299455|4996216|5016676|5006443|5006476|+\n",
      "-ggcggcggctcacgcggtgcataacggcctgac\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1209065, 1211067)), (1210503, 1210585), '+'), (('G000006925', (2756889, 2766574)), (2757584, 2757666), '+'), (('G000006925', (2923039, 2950048)), (2948699, 2948781), '+'), (('G000006925', (3101814, 3103074)), (3102510, 3102592), '+'), (('G000006925', (3810378, 3811029)), (3810860, 3810943), '-'), (('G000006925', (4753880, 4754990)), (4753880, 4753962), '-'), (('G000006925', (4795159, 4795487)), (4795390, 4795472), '+'), (('G000299455', (3226712, 3229721)), (3229153, 3229235), '+')]\n",
      ">G000006925|1209065|1211067|1210503|1210585|+\n",
      "-ctgcctgtgtcagaaaatctgttggagcaggatttttacgccagtggcccg-aaccaga\n",
      "agtgggcaggagacatcacgtact\n",
      ">G000006925|2756889|2766574|2757584|2757666|+\n",
      "-ctgcctgtgtcagaaaatctgttggagcaggatttttacgccagtggcccg-aaccaga\n",
      "agtggccaggagacatcacgtact\n",
      ">G000006925|2923039|2950048|2948699|2948781|+\n",
      "-ctgcctgtgtcagaaaatctgttggagcaggatttttacgccagtggcccg-aaccaga\n",
      "agtgggcaggagacatcacgtact\n",
      ">G000006925|3101814|3103074|3102510|3102592|+\n",
      "-ctgcctgtgtcagaaaatctgttggagcaggatttttacgccagtggcccg-aaccaga\n",
      "agtggccaggagacatcacgtact\n",
      ">G000006925|3810378|3811029|3810860|3810943|-\n",
      "cctgcccgtattggagaatctgctggagcaggacttcagcgccagtggcccgaaaccaga\n",
      "agtgggcgggtgacatcacgtac-\n",
      ">G000006925|4753880|4754990|4753880|4753962|-\n",
      "cctgcctgtatcagaaaatctgctgacgcaggatttttacgccagtggcccg-aaccaga\n",
      "agtgggcgggagacatcacgtac-\n",
      ">G000006925|4795159|4795487|4795390|4795472|+\n",
      "-ctgcccgtgtcagaaaatctgctggagcaggatttttacgccagtggtccg-aaccaga\n",
      "agtgggcgggagacatcacgtact\n",
      ">G000299455|3226712|3229721|3229153|3229235|+\n",
      "-ctgcctgtgtcagaaaatctgttggagcaggatttttacgccagtggcccg-aaccaga\n",
      "agtgggcaggagacatcacgtact\n",
      "\n",
      "\n",
      "8 [(('G000006925', (2560406, 2561162)), (2560406, 2560428), '+'), (('G000006925', (3083675, 3099254)), (3083957, 3083979), '-'), (('G000006925', (4633099, 4633840)), (4633818, 4633840), '-'), (('G000006925', (4668315, 4668899)), (4668878, 4668899), '-'), (('G000006925', (4765915, 4766031)), (4766005, 4766026), '-'), (('G000006925', (4822840, 4823055)), (4822840, 4822862), '+'), (('G000299455', (857924, 871846)), (871544, 871566), '+'), (('G000299455', (5017408, 5018168)), (5018141, 5018162), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2560406|2561162|2560406|2560428|+\n",
      "-ccaattagtagaacatgtgttt\n",
      ">G000006925|3083675|3099254|3083957|3083979|-\n",
      "tccagtcatcagaacatatgtt-\n",
      ">G000006925|4633099|4633840|4633818|4633840|-\n",
      "tccaatcagtagaacatgtgct-\n",
      ">G000006925|4668315|4668899|4668878|4668899|-\n",
      "tccaatcattagaatatgtgc--\n",
      ">G000006925|4765915|4766031|4766005|4766026|-\n",
      "tccaattagtagaacatatgt--\n",
      ">G000006925|4822840|4823055|4822840|4822862|+\n",
      "-ccaatcagtagaatatgtgttt\n",
      ">G000299455|857924|871846|871544|871566|+\n",
      "-ccagtcagcagaatatgttttt\n",
      ">G000299455|5017408|5018168|5018141|5018162|-\n",
      "tccaattagtagaacatatat--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3658114, 3676993)), (3670482, 3670509), '-'), (('G000299455', (227444, 242483)), (237903, 237930), '+'), (('G000299455', (2982866, 2983338)), (2983304, 2983331), '+')]\n",
      ">G000006925|3658114|3676993|3670482|3670509|-\n",
      "aggccgcaggggtgaaactctgcggcc-\n",
      ">G000299455|227444|242483|237903|237930|+\n",
      "-ggccgcaggagtgaaactctgcggcct\n",
      ">G000299455|2982866|2983338|2983304|2983331|+\n",
      "-ggccgcaggggtaacagcctgcggcca\n",
      "\n",
      "\n",
      "18 [(('G000006925', (237115, 237939)), (237563, 237627), '+'), (('G000006925', (538623, 539106)), (538730, 538794), '+'), (('G000006925', (710708, 711191)), (711020, 711084), '-'), (('G000006925', (716211, 716694)), (716318, 716382), '+'), (('G000006925', (901842, 902325)), (901949, 902013), '+'), (('G000006925', (1096742, 1097225)), (1097054, 1097118), '-'), (('G000006925', (1632137, 1632620)), (1632244, 1632308), '+'), (('G000006925', (1835665, 1836148)), (1835977, 1836041), '-'), (('G000006925', (1878088, 1878571)), (1878195, 1878259), '+'), (('G000006925', (1943954, 1944437)), (1944061, 1944125), '+'), (('G000006925', (1961600, 1962083)), (1961707, 1961771), '+'), (('G000006925', (2069131, 2069614)), (2069238, 2069302), '+'), (('G000006925', (2346253, 2346736)), (2346360, 2346424), '+'), (('G000006925', (2473341, 2477789)), (2477413, 2477477), '+'), (('G000006925', (2954172, 2954655)), (2954484, 2954548), '-'), (('G000006925', (4394284, 4394772)), (4394596, 4394660), '-'), (('G000006925', (4824718, 4825201)), (4825030, 4825094), '-'), (('G000299455', (4557177, 4559928)), (4557283, 4557347), '+')]\n",
      ">G000006925|237115|237939|237563|237627|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|538623|539106|538730|538794|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|710708|711191|711020|711084|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000006925|716211|716694|716318|716382|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|901842|902325|901949|902013|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|1096742|1097225|1097054|1097118|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000006925|1632137|1632620|1632244|1632308|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|1835665|1836148|1835977|1836041|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000006925|1878088|1878571|1878195|1878259|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|1943954|1944437|1944061|1944125|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|1961600|1962083|1961707|1961771|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|2069131|2069614|2069238|2069302|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|2346253|2346736|2346360|2346424|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|2473341|2477789|2477413|2477477|+\n",
      "-tatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "caccg\n",
      ">G000006925|2954172|2954655|2954484|2954548|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000006925|4394284|4394772|4394596|4394660|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000006925|4824718|4825201|4825030|4825094|-\n",
      "atatccaacgatatagtccgttatggcgtgagctgcctcgctgaagcttacgtaacccac\n",
      "cacc-\n",
      ">G000299455|4557177|4559928|4557283|4557347|+\n",
      "-tatccaacgatatagtccgttattgcgtgagctgcatcgctgaagcttacatagcccgt\n",
      "cgctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2728788, 2734132)), (2733428, 2733467), '+'), (('G000299455', (1302091, 1327808)), (1327103, 1327142), '+'), (('G000299455', (4201688, 4201857)), (4201688, 4201727), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 18\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 18\r",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 17  f\b\b\r",
      "STEP     2 / 17  f\b\b\r",
      "STEP     3 / 17  f\b\b\r",
      "STEP     4 / 17  f\b\b\r",
      "STEP     5 / 17  f\b\b\r",
      "STEP     6 / 17  f\b\b\r",
      "STEP     7 / 17  f\b\b\r",
      "STEP     8 / 17  f\b\b\r",
      "STEP     9 / 17  f\b\b\r",
      "STEP    10 / 17  f\b\b\r",
      "STEP    11 / 17  f\b\b\r",
      "STEP    12 / 17  f\b\b\r",
      "STEP    13 / 17  f\b\b\r",
      "STEP    14 / 17  f\b\b\r",
      "STEP    15 / 17  f\b\b\r",
      "STEP    16 / 17  f\b\b\r",
      "STEP    17 / 17  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2728788|2734132|2733428|2733467|+\n",
      "tcgatatgtccgaatttatggagaaacactcggtgtctc\n",
      ">G000299455|1302091|1327808|1327103|1327142|+\n",
      "tcgatatgtccgagtttatggagaaacactcggtgtctc\n",
      ">G000299455|4201688|4201857|4201688|4201727|+\n",
      "tcaatatgtccgagttccaggaaaaacacaccgtctcgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1122423, 1168132)), (1157439, 1157515), '-'), (('G000299455', (2006434, 2017864)), (2009379, 2009455), '+'), (('G000299455', (3057394, 3103258)), (3068095, 3068171), '+')]\n",
      ">G000006925|1122423|1168132|1157439|1157515|-\n",
      "acggtacgaacgccgatttcggcattggcatgcgggtaagttctggctgtattcgtctgc\n",
      "gggatgacgatatcaa-\n",
      ">G000299455|2006434|2017864|2009379|2009455|+\n",
      "-cggtaccaatgccaattttggtattgggctccgtgtaagtcagggctgtattcgtctgc\n",
      "gtaatgacgatatcaaa\n",
      ">G000299455|3057394|3103258|3068095|3068171|+\n",
      "-cggtacgaacgccgatttcggcattggcatgcgggtaagttctggctgtattcgtctgc\n",
      "gggatgacgatatcaaa\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1621184, 1626752)), (1625674, 1625722), '-'), (('G000006925', (1634479, 1637262)), (1635913, 1635961), '-'), (('G000299455', (2585656, 2594816)), (2587783, 2587831), '+'), (('G000299455', (2585656, 2594816)), (2590279, 2590327), '+'), (('G000299455', (3440781, 3470001)), (3443911, 3443959), '+')]\n",
      ">G000006925|1621184|1626752|1625674|1625722|-\n",
      "tcgaatttgccttcaccgcgtttacccacgcgtttcattgggtcattc-\n",
      ">G000006925|1634479|1637262|1635913|1635961|-\n",
      "tcaaatttaccttcaccgcgctcgccgacgcgcttcatgggatatttc-\n",
      ">G000299455|2585656|2594816|2587783|2587831|+\n",
      "-caaatttaccttcaccgcgcttgccgacgcgcttcatgggatatttca\n",
      ">G000299455|2585656|2594816|2590279|2590327|+\n",
      "-cgaatttgccttcaccgcgtgtacccacgcgtttcattgggtaattca\n",
      ">G000299455|3440781|3470001|3443911|3443959|+\n",
      "-caaatttgccttcaccgcgcgccccgactcgtttcatcggatatttca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3087244, 3087272), '-'), (('G000299455', (56182, 56238)), (56195, 56223), '+'), (('G000299455', (857924, 871846)), (868238, 868266), '+'), (('G000299455', (3152907, 3154652)), (3153556, 3153584), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1098\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3087244|3087272|-\n",
      "aacgatcggtaaaataggtatcgatcgt-\n",
      ">G000299455|56182|56238|56195|56223|+\n",
      "-acgatcgatacctgttttatcgatcgtt\n",
      ">G000299455|857924|871846|868238|868266|+\n",
      "-acgatcgataaaataggtatcgatcgtt\n",
      ">G000299455|3152907|3154652|3153556|3153584|+\n",
      "-acgatcgataaaatagatattgatcgtt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (25217, 64488)), (64090, 64112), '-'), (('G000006925', (4497401, 4528664)), (4528338, 4528360), '+'), (('G000299455', (183085, 183661)), (183393, 183415), '-'), (('G000299455', (4383450, 4406324)), (4383826, 4383848), '+'), (('G000299455', (4674171, 4686420)), (4674475, 4674497), '-')]\n",
      ">G000006925|25217|64488|64090|64112|-\n",
      "tacggcaccattccctgcaccc-\n",
      ">G000006925|4497401|4528664|4528338|4528360|+\n",
      "-ttggtgatatcccgtgtacgcg\n",
      ">G000299455|183085|183661|183393|183415|-\n",
      "tatggtgccatcccctgcacgc-\n",
      ">G000299455|4383450|4406324|4383826|4383848|+\n",
      "-acggcaccattccctgcacccg\n",
      ">G000299455|4674171|4686420|4674475|4674497|-\n",
      "tttggtgacatcccgtgtacgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1670306, 1692658)), (1685621, 1685676), '-'), (('G000299455', (2459722, 2494618)), (2480121, 2480176), '+'), (('G000299455', (2459722, 2494618)), (2480217, 2480272), '+'), (('G000299455', (2459722, 2494618)), (2480313, 2480368), '+')]\n",
      ">G000006925|1670306|1692658|1685621|1685676|-\n",
      "aacctgttcttctggctccgcattggcctgttgctgttccagcttgcgtgctttg-\n",
      ">G000299455|2459722|2494618|2480121|2480176|+\n",
      "-acctgttcttctggttctgcgttagtctgttgctgttccagcttgcgcgctttgg\n",
      ">G000299455|2459722|2494618|2480217|2480272|+\n",
      "-acctgttcttctggttccgcattagcctgttgctgttccagtttgcgcgctttgg\n",
      ">G000299455|2459722|2494618|2480313|2480368|+\n",
      "-acctgttcttctggttccgcattagcctgttgctgttccagcttgcgcgctttgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3048399, 3048822), '-'), (('G000299455', (942831, 956197)), (948031, 948454), '+'), (('G000299455', (1528204, 1528627)), (1528204, 1528627), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3048399|3048822|-\n",
      "aacttctttttcgacaaacaccgaaccagagatgttgaagaagtcgaatgcgcaaccgta\n",
      "aacgatcatcgacagtaccagcagtacagtaccgaacggagtcgggtcgccgtaagcaaa\n",
      "cagcgcaaaacgcaggatccacgccacaatactgatcatcattacgttcttaataccgta\n",
      "gcggcttaagaagaacgggatggtcagaatgaacagggtttcagagatctgcgaaatcga\n",
      "catgatgattgacgcatgctgcacgatgaagctgctggcaaacatcggatctttgtcgaa\n",
      "gctgtgcaggaaggtattaccgaacatgttggtaatctgcagttccgcgcccagcagcat\n",
      "tgagaagatgaagaagattgccatacgcttgtttttaaacagcgcgaatgcatcgaggcc\n",
      "cag-\n",
      ">G000299455|942831|956197|948031|948454|+\n",
      "-acttctttttcgacaaacaccgaaccagagatgttgaagaagtcgaatgcgcaaccgta\n",
      "aacgatcatcgacagtaccagcagtacagtaccgaacggagtcgggtcgccgtaagcaaa\n",
      "cagcgcaaaacgcaggatccacgccacaatactgatcatcattacgttcttaataccgta\n",
      "gcggcttaagaagaacgggatggtcagaatgaacagggtttcagagatctgcgaaatcga\n",
      "catgatgattgacgcatgctgcacaataaagctgctggcaaacatcggatctttgtcgaa\n",
      "gctgtgcaggaaggtattaccgaacatgttggtaatctgcagttccgcgcccagcagcat\n",
      "tgagaagataaagaagattgccatacgcttgtttttaaacagcgcgaatgcatcgaggcc\n",
      "cagc\n",
      ">G000299455|1528204|1528627|1528204|1528627|-\n",
      "aacttcctgttcgacaaataccgaaccagaaatattgaagaaatcgaatgcacagccata\n",
      "aacaatcatcgacagcagcagcaaaataaatccggttgttgacggatcgccataggcgaa\n",
      "gaagccaaagcgcagcgtccaggccaccatactcatcagcatgacggttttaatgccaaa\n",
      "tcgctttaaaaaaaatgggatagtcagtataaagcccacttctgccatctgtgaaactga\n",
      "cagtaaaatggagggatatttcaccacaaaactgtcagcaaactccgggttacgggcgaa\n",
      "atcatgtaggaacggattaccaaaaacgttggtaatttgcagtaccgcacccagcatcat\n",
      "ggcaaagaggaaaaagatggccatgcgtggatttttaaacagcacgaaggcatccagacc\n",
      "cag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3737154, 3737365), '+'), (('G000299455', (162034, 163089)), (162045, 162256), '-'), (('G000299455', (179861, 180072)), (179861, 180072), '-')]\n",
      ">G000006925|3736321|3788106|3737154|3737365|+\n",
      "-gcgcctggtaacgaaccgggtcgagcttaaaaagcacctcgcccttttgaataagctga\n",
      "ttattcttgtcagtgacttcagtaacaattcccgtcacctgtggcgtgatagggatcgct\n",
      "atcactgccttttgcgcggtaaaagtgtaagggtggttgtagttcatcaacaaaatcaaa\n",
      "ccactcaccagaaacacgcctcccagcgccgc\n",
      ">G000299455|162034|163089|162045|162256|-\n",
      "cgcgcctggtaacgaaccgggtcgagcttaaaaagcacctcgcccttttgaataagctga\n",
      "ttattcttgtcagtgacttcagtaacaattcccgtcacctgtggcgtgatagggatcgct\n",
      "atcactgccttttgcgcggtaaaagtgtaagggtggttgtagttcatcaacaaaattaaa\n",
      "ccactcaccagaaacacgcctcccagcgccg-\n",
      ">G000299455|179861|180072|179861|180072|-\n",
      "cgcgcctgataacgcgtcgggtccagtcgaaatagcacctcaccttttttaatcagcgta\n",
      "tttttcttatccgtcacttcgatcaccacgcctgtcacctgtgggacaacaggaatagaa\n",
      "ataaccgctttttgcgctttgaaggtatacggatggttatagttcattaacagaattaaa\n",
      "ccactgacaataaatatgccacccagggccg-\n",
      "\n",
      "\n",
      "16 [(('G000006925', (385776, 386112)), (385872, 385894), '-'), (('G000006925', (899355, 899691)), (899573, 899595), '+'), (('G000006925', (2059291, 2059627)), (2059387, 2059409), '-'), (('G000006925', (3421586, 3421922)), (3421804, 3421826), '+'), (('G000006925', (3851554, 3851890)), (3851650, 3851672), '-'), (('G000006925', (4359551, 4359859)), (4359648, 4359670), '-'), (('G000006925', (4622488, 4624460)), (4623097, 4623119), '+'), (('G000006925', (4700925, 4701241)), (4701020, 4701042), '-'), (('G000006925', (4776578, 4776894)), (4776673, 4776695), '-'), (('G000006925', (4781327, 4781663)), (4781545, 4781567), '+'), (('G000299455', (1991020, 1991316)), (1991230, 1991252), '+'), (('G000299455', (3215105, 3215383)), (3215312, 3215334), '+'), (('G000299455', (4543449, 4543753)), (4543667, 4543689), '+'), (('G000299455', (4571312, 4571569)), (4571547, 4571569), '+'), (('G000299455', (5289627, 5289931)), (5289691, 5289713), '-'), (('G000299455', (5313369, 5315826)), (5314334, 5314356), '+')]\n",
      ">G000006925|385776|386112|385872|385894|-\n",
      "-ccggtgacccgggaagggaaag-\n",
      ">G000006925|899355|899691|899573|899595|+\n",
      "--cggtgacccgggaagggaaagt\n",
      ">G000006925|2059291|2059627|2059387|2059409|-\n",
      "-ccggtgacccgggaagggaaag-\n",
      ">G000006925|3421586|3421922|3421804|3421826|+\n",
      "--cggtgacccgggaagggaaagt\n",
      ">G000006925|3851554|3851890|3851650|3851672|-\n",
      "-ccggtgacccgggaagggaaag-\n",
      ">G000006925|4359551|4359859|4359648|4359670|-\n",
      "gccggtgacccgggaagggaaa--\n",
      ">G000006925|4622488|4624460|4623097|4623119|+\n",
      "--cggtcacccgtgacggcaaggt\n",
      ">G000006925|4700925|4701241|4701020|4701042|-\n",
      "--cggtgacccgggaagggaaagt\n",
      ">G000006925|4776578|4776894|4776673|4776695|-\n",
      "--cggtgacccgggaagggaaagt\n",
      ">G000006925|4781327|4781663|4781545|4781567|+\n",
      "--cggtgacccgggaagggaaagt\n",
      ">G000299455|1991020|1991316|1991230|1991252|+\n",
      "--cctcagcccgcgatggcaaagt\n",
      ">G000299455|3215105|3215383|3215312|3215334|+\n",
      "--cgtcagcccgggatggcaaagt\n",
      ">G000299455|4543449|4543753|4543667|4543689|+\n",
      "--cgtcagcccgtgatggcaaagt\n",
      ">G000299455|4571312|4571569|4571547|4571569|+\n",
      "--ctgcggtacgtgacggcaaggt\n",
      ">G000299455|5289627|5289931|5289691|5289713|-\n",
      "-ccgtcagcccgtgatggcaaag-\n",
      ">G000299455|5313369|5315826|5314334|5314356|+\n",
      "--cagtcacccgtgacggcaaggt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (739473, 739616), '+'), (('G000006925', (1921807, 1928114)), (1927971, 1928114), '-'), (('G000299455', (2187515, 2189078)), (2187515, 2187658), '+'), (('G000299455', (2902162, 2904160)), (2904017, 2904160), '-'), (('G000299455', (3590325, 3591889)), (3591746, 3591889), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 16\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 16\r",
      "   10 / 16\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 15  f\b\b\r",
      "STEP     2 / 15  f\b\b\r",
      "STEP     3 / 15  f\b\b\r",
      "STEP     4 / 15  f\b\b\r",
      "STEP     5 / 15  f\b\b\r",
      "STEP     6 / 15  f\b\b\r",
      "STEP     7 / 15  f\b\b\r",
      "STEP     8 / 15  f\b\b\r",
      "STEP     9 / 15  f\b\b\r",
      "STEP    10 / 15  f\b\b\r",
      "STEP    11 / 15  f\b\b\r",
      "STEP    12 / 15  f\b\b\r",
      "STEP    13 / 15  f\b\b\n",
      "Reallocating..done. *alloclen = 1047\n",
      "\r",
      "STEP    14 / 15  f\b\b\r",
      "STEP    15 / 15  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 16\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 16\r",
      "   10 / 16\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 15  f\b\b\r",
      "STEP     2 / 15  f\b\b\r",
      "STEP     3 / 15  f\b\b\r",
      "STEP     4 / 15  f\b\b\r",
      "STEP     5 / 15  f\b\b\r",
      "STEP     6 / 15  f\b\b\r",
      "STEP     7 / 15  f\b\b\r",
      "STEP     8 / 15  f\b\b\r",
      "STEP     9 / 15  f\b\b\r",
      "STEP    10 / 15  f\b\b\r",
      "STEP    11 / 15  f\b\b\r",
      "STEP    12 / 15  f\b\b\r",
      "STEP    13 / 15  f\b\b\r",
      "STEP    14 / 15  f\b\b\r",
      "STEP    15 / 15  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1288\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|739473|739616|+\n",
      "-ttgaacaggtggtgaactgatgcaggatatccggcaggaaacactgaatgaatgcaccc\n",
      "gtgcggagcagtcagcccgggtggagctctgggaaatcgatctgacagaggtcggtggtg\n",
      "agcgttattttttctgtaatgagc\n",
      ">G000006925|1921807|1928114|1927971|1928114|-\n",
      "tttgaacaggtggtgaactgatgcaggatatccggcaggaaacactgaatgaatgcaccc\n",
      "gtgcggagcagtcagcccgggtggagctctgggaaatcgatctgacagaggtcggtggtg\n",
      "agcgttattttttctgtaatgag-\n",
      ">G000299455|2187515|2189078|2187515|2187658|+\n",
      "-ttgaacaggtggtgaactgatgcaggatatccggcaggaaacactgaatgaatgcaccc\n",
      "gtgcggagcagtcggccagcgtggtgctctgggaaatcgatctgacagaggtcggtggag\n",
      "aacgttattttttctgtaatgagc\n",
      ">G000299455|2902162|2904160|2904017|2904160|-\n",
      "tttgaacaggtggtgaactgatgcaggatatccggcaggaaacactgaatgaatgcaccc\n",
      "gtgcggagcagtcggccagcgtggtgctctgggaaatcgatctgacagaggtcggtggag\n",
      "aacgttattttttctgtaatgag-\n",
      ">G000299455|3590325|3591889|3591746|3591889|-\n",
      "tttaaacaggtggtgaactgatgcaggatatccggcaggaaacactgaatgaatgcaccc\n",
      "gtgcggagcagtcggccagcgtggtgctctgggaaatcgacctgacagaggtcggtggag\n",
      "aacgttattttttctgtaatgag-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (332059, 337870)), (333687, 333728), '-'), (('G000006925', (1600039, 1605691)), (1603481, 1603522), '-'), (('G000299455', (2620477, 2625905)), (2623695, 2623736), '-'), (('G000299455', (3230094, 3230143)), (3230094, 3230135), '+'), (('G000299455', (3581442, 3581499)), (3581442, 3581483), '-'), (('G000299455', (4009752, 4023455)), (4013894, 4013935), '+')]\n",
      ">G000006925|332059|337870|333687|333728|-\n",
      "atgaccccaggtatcgttgatgctcttgaaatggtcgatat-\n",
      ">G000006925|1600039|1605691|1603481|1603522|-\n",
      "atgtccatagttatcgttaatgcttttgaaatagtcgatat-\n",
      ">G000299455|2620477|2625905|2623695|2623736|-\n",
      "atgtccatagttatcgttaatgcttttgaaatagtcgatat-\n",
      ">G000299455|3230094|3230143|3230094|3230135|+\n",
      "-tgcccccaggtgtcgttgacttttttgaagtggtcgatatc\n",
      ">G000299455|3581442|3581499|3581442|3581483|-\n",
      "atgcccccatttgtcattgatgcttttgaaattgtcaatat-\n",
      ">G000299455|4009752|4023455|4013894|4013935|+\n",
      "-tgaccccaggtatcgttgatgctcttgaaatggtcgatatc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1729332, 1741706)), (1731478, 1731562), '-'), (('G000006925', (4288513, 4315006)), (4300649, 4300733), '+'), (('G000299455', (2416691, 2440660)), (2438430, 2438514), '+'), (('G000299455', (4836149, 4864500)), (4849661, 4849745), '-')]\n",
      ">G000006925|1729332|1741706|1731478|1731562|-\n",
      "tatcacttttttcgccagtcatgtcagcactgcgaagatgctccgtgcatcgacgtatgc\n",
      "ccgacgggggcatcgtggcgtgat-\n",
      ">G000006925|4288513|4315006|4300649|4300733|+\n",
      "-atcgcttcttccgtaagtcttgccagcactgcgatcatgcgccgtgcgttgacgtctgc\n",
      "ccgaccggtgcgtcttttcgtgatg\n",
      ">G000299455|2416691|2440660|2438430|2438514|+\n",
      "-atcacttttttcgccagtcatgtcagcactgcgaagatgctccgtgcatcgacgtatgc\n",
      "ccgacaggggcgtcgtggcgtgatg\n",
      ">G000299455|4836149|4864500|4849661|4849745|-\n",
      "tatcgcttcttccgtaagtcttgccagcactgcgatcatgcgccgtgcgttgacgtctgc\n",
      "ccgaccggtgcgtcttttcgtgat-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1509064, 1578296)), (1514117, 1514255), '+'), (('G000006925', (3423912, 3503727)), (3458605, 3458743), '-'), (('G000299455', (434155, 488092)), (479116, 479254), '+'), (('G000299455', (2312634, 2337653)), (2319029, 2319167), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1170\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1170\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1514117|1514255|+\n",
      "-tgaccgccacaatgccttgctgatttttgatgaggtacaaaccggcgtcgggcgcactg\n",
      "gtgagctgtatgcctatatgcactacggcgtgacgcccgatctgttaaccaccgccaaag\n",
      "cgctgggcggcggtttccc\n",
      ">G000006925|3423912|3503727|3458605|3458743|-\n",
      "gcgatcaacatcaggcattattggtgtttgatgaagtgcagtgcgggatggggcggactg\n",
      "gcgatttgtttgcttacaggtactacggtgtgacgccggatattctgacttctgcgaaag\n",
      "cgttaggcggcggcttcc-\n",
      ">G000299455|434155|488092|479116|479254|+\n",
      "-cgatcaacatcaggcattattggtgtttgatgaagtgcagtgcgggatggggcggactg\n",
      "gcgatttgtttgcttacatgcactacggtgtgacgccggatattctgacctctgcgaaag\n",
      "cgttaggcggcggcttccc\n",
      ">G000299455|2312634|2337653|2319029|2319167|+\n",
      "-tgaccgccacaatgccttgctgatttttgatgaggtacaaaccggcgtcgggcgcactg\n",
      "gtgagctgtatgcctatatgcactacggcgtgacgcccgatctgttaaccaccgccaaag\n",
      "cgctgggcggcggtttccc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1965007, 1976679)), (1968393, 1968412), '-'), (('G000006925', (3205438, 3224454)), (3208319, 3208339), '-'), (('G000299455', (715333, 739463)), (736543, 736563), '+')]\n",
      ">G000006925|1965007|1976679|1968393|1968412|-\n",
      "-aatcttgccagccgtagcg-\n",
      ">G000006925|3205438|3224454|3208319|3208339|-\n",
      "tcatttagccagccgcagcg-\n",
      ">G000299455|715333|739463|736543|736563|+\n",
      "-catttagccagccgcagcgc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3097038, 3097121), '-'), (('G000299455', (47809, 50193)), (49736, 49819), '+'), (('G000299455', (857924, 871846)), (858432, 858515), '+'), (('G000299455', (4771847, 4785261)), (4782674, 4782757), '-')]\n",
      ">G000006925|3083675|3099254|3097038|3097121|-\n",
      "tgtattcatcttaagttctctgtaacaggtggttataaaggggagatatccggtgttata\n",
      "cggtggttgcgggcgctgccggt-\n",
      ">G000299455|47809|50193|49736|49819|+\n",
      "-gtattcatcttaagttctctgtaacaggtggtgataaaggggagatatccggggttata\n",
      "cggtgattgcgggctctgccggtg\n",
      ">G000299455|857924|871846|858432|858515|+\n",
      "-gtattcatcttaagttctctgtaacaggtggttataaaggggagatatccggtgttata\n",
      "cggtggttgcgggcgctgccggtg\n",
      ">G000299455|4771847|4785261|4782674|4782757|-\n",
      "tgtattcatcttaaattctctgtaacaggtggttataaaggggagatatccggtgttata\n",
      "cggtggttgcgggcgctgccggt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1424044, 1424128)), (1424044, 1424063), '-'), (('G000299455', (2196403, 2197468)), (2196403, 2196422), '-'), (('G000299455', (3580143, 3580251)), (3580231, 3580251), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1424044|1424128|1424044|1424063|-\n",
      "tacggtttcctttacgatt-\n",
      ">G000299455|2196403|2197468|2196403|2196422|-\n",
      "cgctgttctctttacgatt-\n",
      ">G000299455|3580143|3580251|3580231|3580251|+\n",
      "tgcggttttctttatgattt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3571036, 3571069), '+'), (('G000299455', (335504, 377237)), (364982, 365015), '-'), (('G000299455', (739467, 747132)), (740136, 740169), '+')]\n",
      ">G000006925|3559544|3600494|3571036|3571069|+\n",
      "-ctttcatcaggccgatgttaccttcctgaatca\n",
      ">G000299455|335504|377237|364982|365015|-\n",
      "gctttcatcaggccgatgttaccttcctgaatc-\n",
      ">G000299455|739467|747132|740136|740169|+\n",
      "-ctttcatcagaccgatgttgccttcctgaatca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (24672, 25214)), (24672, 25214), '+'), (('G000006925', (25217, 64488)), (25240, 25782), '-'), (('G000299455', (4406425, 4422836)), (4422270, 4422812), '+')]\n",
      ">G000006925|24672|25214|24672|25214|+\n",
      "-cgcgttttcaacaatgctgatagcgcggtctaccccggcacaaaaaccacgcgggttgg\n",
      "ccaacaggatctgcatgttacgcctccagtgccggatcgatttccagcacttcaatatca\n",
      "aaatgaacggtctgcccggccagcggatggttgaaatcaacggtaatggagtcgccgtta\n",
      "atttcgcggatcacaccaggcatctcactgccatccattgcggtaaaaagcatgattgcg\n",
      "ccaatttctggctcgcctgcatccataaattcacggcgggagaagtactgaatcaggtcc\n",
      "ggtgacggcacgccaaacgccgcatctggctccagcgagaaggtggttttgtcgcccact\n",
      "ttcagccccagcaggtgttgctccagcccttcagaaagagaagcatcgcccaggcggaac\n",
      "agcgccggtttaccgttgttgcgagtagactcggcggtggtgccatcgtcgagttttagc\n",
      "gtgaagtgcaccaggacggcgctattgctctgtacagattcagacatgcaggtttgctct\n",
      "tat\n",
      ">G000006925|25217|64488|25240|25782|-\n",
      "gcgcgttttcaacaatgctgatagcgcggtctaccccggcacaaaaaccacgcgggttgg\n",
      "ccaacaggatctgcatgttacgcctccagtgccggatcgatttccagcacttcaatatca\n",
      "aaatgaacggtctgcccggccagcggatggttgaaatcaacggtaatggagtcgccgtta\n",
      "atttcgcggatcacaccaggcatctcactgccatccattgcggtaaaaagcatgattgcg\n",
      "ccaatttctggctcgcctgcatccataaattcacggcgggagaagtactgaatcaggtcc\n",
      "ggtgacggcacgccaaacgccgcatctggctccagcgagaaggtggttttgtcgcccact\n",
      "ttcagccccagcaggtgttgctccagcccttcagaaagagaagcatcgcccaggcggaac\n",
      "agcgccggtttaccgttgttgcgagtagactcggcggtggtgccatcgtcgagttttagc\n",
      "gtgaagtgcaccaggacggcgctattgctctgtacagattcagacatgcaggtttgctct\n",
      "ta-\n",
      ">G000299455|4406425|4422836|4422270|4422812|+\n",
      "-cgcgttttcaacaatgctgatagcgcggtctaccccggcacaaaaaccacgcgggttgg\n",
      "ccaacaggatctgcatgttacgcctccagtgccggatcgatttccagcacttcaatatca\n",
      "aaatgaacggtctgcccggccagcggatggttgaaatcaacggtaatggagtcgccgtta\n",
      "atttcgcggatcacgccaggcatctcactgccatccattgcggtaaaaagcatgattgcg\n",
      "ccaatttctggctcgcctgcatccataaattcacggcgggagaagtactgaatcaggtcc\n",
      "ggtgacggcacgccaaacgccgcatctggctccaacgagaaggtggttttatcgcccact\n",
      "ttcagccccaacaggtgttgctccagcccttcagaaagagaagcatcacccaggcggaac\n",
      "agcgccggtttaccgttgttgcgggtagactcggcggtggtgccatcgtcgagttttagc\n",
      "gtgaagtgcaccaggacggcgctattgctctgtacagattcagacatgcaggtttgctct\n",
      "tat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4557883, 4573403)), (4573025, 4573059), '+'), (('G000006925', (4557883, 4573403)), (4573149, 4573176), '+'), (('G000299455', (4472908, 4487234)), (4484551, 4484578), '-'), (('G000299455', (4472908, 4487234)), (4484666, 4484699), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4557883|4573403|4573025|4573059|+\n",
      "-cagtcatcttactttttttgatatcgccataatt--\n",
      ">G000006925|4557883|4573403|4573149|4573176|+\n",
      "----------cagtgcgagcaatatcaacgtgatttt\n",
      ">G000299455|4472908|4487234|4484551|4484578|-\n",
      "---------ccagtgcgagcaatatcaacgtggttt-\n",
      ">G000299455|4472908|4487234|4484666|4484699|-\n",
      "acagtcatcttactttttttgatatcgcctcgt----\n",
      "\n",
      "\n",
      "3 [(('G000006925', (722152, 722887)), (722152, 722359), '+'), (('G000299455', (2170692, 2170900)), (2170692, 2170900), '+'), (('G000299455', (2920506, 2921242)), (2921034, 2921242), '-')]\n",
      ">G000006925|722152|722887|722152|722359|+\n",
      "-tgaaacgcattggccgttgcatggggtatttgcaggtggcggcagagattgaagcactt\n",
      "gccagtggtgcagacatggatgctgcggtgcgggaggctcttcttcgtgatttcaatact\n",
      "cc-ccctttaagaaagagcctgatgactggatccagccggggctgacttatcttaaaagg\n",
      "cgtatataagttggctcgttatttgttgc\n",
      ">G000299455|2170692|2170900|2170692|2170900|+\n",
      "-tgaagcgcattggtcgttgcatgggatatctgcaggtagcagctgagattgaaatgatg\n",
      "gccagtggtgctgacaatgatgcagttgtgctggaggctcttctgcgtgattttgataat\n",
      "acgccctttaagaaagcacctgttgactggatgcagccggggatgacttatctgaaaggg\n",
      "cgtatataaggtggctcgttatctgttgc\n",
      ">G000299455|2920506|2921242|2921034|2921242|-\n",
      "atgaagcgcgttggccgttgcatgggatatctgcagatagctgctgagattgaagcgctg\n",
      "gccagtggtgcggacaaggatgcagttgtgcgggaggctcttcttcgtgattttgataat\n",
      "ccgccctttaaaaaagtgccggcttactggtttcatccaggactgacttatcttaaagga\n",
      "cgtatataagctggctcgttatctgttg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3117164, 3135297)), (3130875, 3131764), '+'), (('G000299455', (288342, 302750)), (300525, 301413), '+'), (('G000299455', (812187, 830040)), (815440, 816329), '-')]\n",
      ">G000006925|3117164|3135297|3130875|3131764|+\n",
      "-tcgaagttggtaacggcatcgcgggtacgggtaatttcatagccggacgcattcatatt\n",
      "gacgacgaagccagcaggggcaatccccaccagtaccagcattaccaggccgatcccttt\n",
      "ctgcccgtcgttcgcgccgtgcgaaaacgccacgcccgcagcggaaacaatcagcgcaat\n",
      "acgcgtccagaatggtggtttacgtttgccctttttctttttgcgatcttccggaatgcg\n",
      "gtgaatacggtcacgctttttcgtcccactccagaaacggcgcagcaggaaaatcagacc\n",
      "acccgcaatgaccaggccgacgataggggaaacaatcagcgaggagaaaattttggttac\n",
      "ttcacgcaggtttaacgcatccatcaccgatgagccggttaacagcgcgttggttaaacc\n",
      "gatgccgataatcgcaccaatcaaggtgtgcgaactggaggccggtaaaccaaagaacca\n",
      "cgttcccaggttccagataatcgccgccagcagcatggaaaagaccatcgccaaaccgtg\n",
      "agttgaccccatattcagcaacaaatcgattggtaacatatggacaatggcataggcaac\n",
      "gctaagtccgcccaataacacgccaaaaaagttaaaaaatgccgccatcaccacagcaag\n",
      "ttgtggctgcatggcacgagtataaataacggttgccaccgcattcgccgtgtcatgaaa\n",
      "accattgattgcttcgtagaacaacacaaatgccagagcaagtaataacaaaagccccgt\n",
      "atatatatcaaggccaacaaataaatttagcatatattagattacgccattttgaattta\n",
      "cgaacg-gacgcattatcagtgactttaacggcatgggcaaagtgaaatat\n",
      ">G000299455|288342|302750|300525|301413|+\n",
      "-tcgacgttgttgatggcatcacgggtacgggtgatttcgtagccagtggcattcatgtt\n",
      "caccacgaagcctgctggcgcgacgccaatcaataccaacataaccagaccaatgccttt\n",
      "ctgaccatcgttcgcgccgtgcgaaaacgccacgccgatagcggaaaggatcagcgcgat\n",
      "acgcgtccagaacggcggctttttcttgccgtctttcttttcacgctccgctggggtcag\n",
      "gtggatacgggcgcgtttcttggtgccgctccagtagcgacgcagcaagaaaatcagacc\n",
      "gccagcaaacaccaggccgacaataggggaaacgatcagagaaccgaaaatacttaatac\n",
      "tttcgggatattgagtgcatccaccactgacgtcccggtcatcaacgcattggttaaacc\n",
      "aatcccgatgatcgcgccaatcagcgtatgagagctggatgcaggtaaaccaaagtacca\n",
      "ggtacccaggttccagataatcgccgccagcaacatagagaacaccatggcaaggccatg\n",
      "agacgatcccatattaagcagcagatccgtcggcagcatatgcacaatggcataggcaac\n",
      "actcagaccacccagcaaaacacccaaaaagttgaacaccgccgccataaccacggcgag\n",
      "ctgagaacgcatcgcgcgggtatagataacggttgccacggcgttggctgtgtcatggaa\n",
      "accattgatggcttcgtagaacagcacaaaagccagtgcaagcaataataacagcccggt\n",
      "atgcaaatccaggccagcaaacaaatgtagcatagg--acgttacgccattttgaggaca\n",
      "tgaacgcggcgcattatcagtgactttcgcggcgcgggcaaagtgaaatat\n",
      ">G000299455|812187|830040|815440|816329|-\n",
      "ttcgaagttggagacggcatcgcgagtacgggtaatttcatagccggacgcgttcatatt\n",
      "gacgacgaagccagcaggggcaatccccaccagtaccagcatcaccaggccgatcccttt\n",
      "ctgcccgtcgttcgcgccgtgcgaaaacgccacgcccgcagcggaaacaatcagcgcaat\n",
      "acgcgtccagaatggcggtttacgtttgccttttttctttttgcgatcttccggaatgcg\n",
      "gtgaatacggtcacgctttttcgtcccgctccagaaacggcgcagcaggaaaatcagacc\n",
      "acccgcaatgaccaggccgacgataggggaaacaatcagcgaggagaaaattttggtcac\n",
      "ttcacgcaggtttaacgcatccatcaccgatgagccggttaacagcgcgttggttaaacc\n",
      "gatgccgataatcgcgccaatcaaggtgtgcgaactggaggccggtaaaccgaagaacca\n",
      "cgtccccaggttccatataatcgccgccagcagcatggaaaagaccatcgccagaccgtg\n",
      "ggttgaccccatattcagcaacaaatcggttggcaacatatggacgatggcataggcgac\n",
      "gctaagcccgccaagcaacacaccaaaaaagttaaaaaacgccgccattactaccgcaag\n",
      "atgcggctgcatggcacgagtataaataacggttgccaccgcgttcgccgtgtcgtgaaa\n",
      "accattgattgcttcgtagaacaacacaaatgccagagcaagtaataacaaaagccccgt\n",
      "atatatatcaaggccaacaaataaatttagcatatattagattacgccattttgaattta\n",
      "cgaacg-gacgcattatcagtgactttcacggcgtgggcaaagtgaaata-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4790313, 4790480)), (4790419, 4790439), '+'), (('G000006925', (4816831, 4818356)), (4818336, 4818356), '+'), (('G000299455', (5326102, 5326126)), (5326102, 5326122), '-'), (('G000299455', (5385343, 5385755)), (5385695, 5385715), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     2 / 2 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4790313|4790480|4790419|4790439|+\n",
      "tacatgccgctggcgcggca-\n",
      ">G000006925|4816831|4818356|4818336|4818356|+\n",
      "-ccatgccgctggcgcggcat\n",
      ">G000299455|5326102|5326126|5326102|5326122|-\n",
      "tccatgccgctgtcgcggca-\n",
      ">G000299455|5385343|5385755|5385695|5385715|+\n",
      "-ccatgccgctgacgcggcat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3087437, 3087927), '-'), (('G000299455', (54863, 56031)), (55515, 55999), '+'), (('G000299455', (857924, 871846)), (867586, 868073), '+'), (('G000299455', (3152907, 3154652)), (3152907, 3153393), '+')]\n",
      ">G000006925|3083675|3099254|3087437|3087927|-\n",
      "cagcgtggtgttcaccgccagtccgttaaggctctgccctccgccatcacgaataaccgt\n",
      "atcactggcagttcctcctgcctgcacaatctgacgaccatttgcggtgacagtggtgtt\n",
      "tctgcctgtgccacccgcttttatccattgcccgccggtgttttcgtcactgtccggccc\n",
      "cagctcaagcccggtactgacagtcacgccatcagctgttccggatacaaactggttgtc\n",
      "atggtttaccagtgttccgccattcactgtttcacccgggtgcacaacgatgtcagcagc\n",
      "cagcaccgggagtgacgtgaccgcggcaagagacagtgcaaccgccacaccgccacgttt\n",
      "accctgtgcgcgggccagttcggaggcaaccacgaaagcgcccgtaatgtgattccatac\n",
      "cagcctgtagcaggtattcagatgtcgtttcatcagcatttccttaatcagagtgagggt\n",
      "gaataaaaaa-\n",
      ">G000299455|54863|56031|55515|55999|+\n",
      "-agagtggtattcactgcctgcccctgaaggctctgtccgcctccggcactgataaccgt\n",
      "gtctgaaacgcttcctccggcattcactctctgaagaccaccaccggtgacagtagtgtt\n",
      "gttggcgataccgccattttgtatccattgcccgccggtattggcctcgttatccggccc\n",
      "atactccagaccggtactgatggtcattccgttggccgtaccgagaacaatctggttgtc\n",
      "atgatttgtcagtgttccgccgctcacggtttctcccgcctgtacaaccgtgtcagcagc\n",
      "cagtgccggaactgatgtgacagcagcaagagacagcgcaatcgccacaccggtgcgttt\n",
      "tccccgtgagcgcgccagttcggaagccaccaccagggtgcccgtaatgtgattccatac\n",
      "cagcctgtagctggtgttcagatgtcgtttcatcagcttttccttaca------gagggt\n",
      "gaataaaaaag\n",
      ">G000299455|857924|871846|867586|868073|+\n",
      "----acagtaccgtaagcctgcccgcgcagattctgtcctccaccggcactgataaccgt\n",
      "atccgtggcactgccacctgccatgacatcctgcagaccgtttgcggtgaccgtggtgtt\n",
      "tctggcagtgccaccccgggctatctgctgaccgccggtgttgtcatcagagtccggccc\n",
      "cagctcaagcccggtactgacagtcacgccatcagctgttccggatacaaactggttgtc\n",
      "atggtttaccagtgttccgccattcactgtttcacccgggtgcacaacgatgtcagcagc\n",
      "cagcaccgggagtgacgtgactgcggcaagagacagtgcaaccgccacaccgccacgttt\n",
      "accccgtgcgcgggccagttcggaggcaaccacgaaagcgcccgtaatgtgattccatac\n",
      "cagcctgtagcaggtattcagatgtcgtttcatcagcatttccttaatcagagtgagggt\n",
      "gaataaaaaag\n",
      ">G000299455|3152907|3154652|3152907|3153393|+\n",
      "-agcgtggtgttcactgcccgtccctgaaggctctgtccgcctccggcactgataaccgt\n",
      "gtctgagacacttccaccggggttcactctctgaagaccaccactggtgacagtcgtttt\n",
      "gttggctgttccgccatcctgtacccattgcccgccggtattggcctcgttatccggccc\n",
      "atactccagcccggtactgatggtcattccgttggtcgtaccgaagacaatctggttgtc\n",
      "atgatttgccagtgttccgccgttcacggtttctcccgggtgcacaacgatgtcagcagc\n",
      "cagcaccgggagtgacgtgactgcggcaagagacagtgcaaccgccacaccgccacgttt\n",
      "accccgtgcgcgggccagttcggaggcaaccacgaaagcgcccgtcatgtgattccatac\n",
      "cagcctgtagctggtgttcagatgtcgtttcatcagcttttccttagat----tgagggt\n",
      "gaataaaaaag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1397350, 1397414)), (1397356, 1397376), '+'), (('G000299455', (2541462, 2541521)), (2541463, 2541483), '+'), (('G000299455', (2582178, 2582236)), (2582216, 2582236), '-'), (('G000299455', (3311958, 3311996)), (3311964, 3311984), '+')]\n",
      ">G000006925|1397350|1397414|1397356|1397376|+\n",
      "-tggatgccgctaccggaacc\n",
      ">G000299455|2541462|2541521|2541463|2541483|+\n",
      "-tggatgccgctaccagaacc\n",
      ">G000299455|2582178|2582236|2582216|2582236|-\n",
      "ctggatgccgctaccagaac-\n",
      ">G000299455|3311958|3311996|3311964|3311984|+\n",
      "-tggatgccgctaccagaacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1168137, 1173523)), (1171489, 1171514), '+'), (('G000299455', (2728771, 2728908)), (2728771, 2728796), '+'), (('G000299455', (3050799, 3056185)), (3052808, 3052833), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1168137|1173523|1171489|1171514|+\n",
      "-tgcatttgcttacgcagtttgtaat\n",
      ">G000299455|2728771|2728908|2728771|2728796|+\n",
      "-tgcatctgctcacgcaatttgagat\n",
      ">G000299455|3050799|3056185|3052808|3052833|-\n",
      "ctgcatttgcttgcgcagtttgtaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (727171, 727454), '+'), (('G000299455', (2174641, 2181396)), (2175564, 2175847), '+'), (('G000299455', (2910238, 2916987)), (2915781, 2916064), '-')]\n",
      ">G000006925|726235|732999|727171|727454|+\n",
      "-tgtgactggctggatgcactgaaagatcccaacggcgtgaaaacttttgtgaacaccac\n",
      "gctgggcgagacctgggaagaggccgtgggcgaaaaactcgatcaccaggtactgatgga\n",
      "taaggttgtgccttacacggcaacggtgcctgtccgggtggtttatctgacggcgggcat\n",
      "tgactcgcagcgaaaccgttttgagatgtatgtctgggggtgggctccgggagaggaagc\n",
      "ctttctggtggataaaatcatcatcatgggacgtcctgatgagg\n",
      ">G000299455|2174641|2181396|2175564|2175847|+\n",
      "-tatgactggctggatgcactgaaagatcccaacggcctgaaaacctttgtgaacaccac\n",
      "gctgggcgagacctgggaagaggccgtgggcgaaaaactcgatcaccaggtactgatgga\n",
      "taaggttgtgcgttacacggcggcggtgcctgcccgggtggtttatctgacggcgggcat\n",
      "tgactcgcagcgaaaccgtttcgagatgtatgtctggggatgggctccgggagaggaagc\n",
      "ctttctggtggataaaatcatcattatggggcgtcccgatgagg\n",
      ">G000299455|2910238|2916987|2915781|2916064|-\n",
      "ctatgactggctggatgcactgaaagatcccaacggcctgaaaacctttgtgaacaccac\n",
      "gctgggcgagacctgggaagaggccgtgggcgaaaaactcgatcaccaggtactgatgga\n",
      "taaggtcgtgcattacacggcggcggtgccagcccgggtggtttatctgacggcgggcat\n",
      "tgactcgcagcgaaaccgttttgagatgtatgtctggggatgggcaccgggagaggaagc\n",
      "ttttctggtggataaaatcatcattatgggccgtcccgatgag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4117495, 4130655)), (4120303, 4120333), '-'), (('G000299455', (423834, 433368)), (431604, 431634), '+'), (('G000299455', (5034449, 5060698)), (5044772, 5044802), '+')]\n",
      ">G000006925|4117495|4130655|4120303|4120333|-\n",
      "ggtggcacgtattcgcgcaatcctgcgccg-\n",
      ">G000299455|423834|433368|431604|431634|+\n",
      "-ctggcccgtattcgtgcggtgctgcgtcgt\n",
      ">G000299455|5034449|5060698|5044772|5044802|+\n",
      "-gtggcacgtattcgcgcgatcctgcgccgt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2227199, 2227358)), (2227338, 2227358), '-'), (('G000006925', (2236395, 2237421)), (2236395, 2236414), '-'), (('G000299455', (1819843, 1851664)), (1840752, 1840772), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2227199|2227358|2227338|2227358|-\n",
      "cacgcgcgtaacgtgacagg-\n",
      ">G000006925|2236395|2237421|2236395|2236414|-\n",
      "-acgcgcgtaacgtgacagg-\n",
      ">G000299455|1819843|1851664|1840752|1840772|+\n",
      "-acgcgcgtaacgtgacaggg\n",
      "\n",
      "\n",
      "19 [(('G000006925', (237940, 238704)), (238384, 238600), '+'), (('G000006925', (539107, 539868)), (539550, 539766), '+'), (('G000006925', (709671, 710707)), (710048, 710264), '-'), (('G000006925', (716695, 717456)), (717138, 717354), '+'), (('G000006925', (902326, 903088)), (902769, 902985), '+'), (('G000006925', (1095979, 1096741)), (1096082, 1096298), '-'), (('G000006925', (1632621, 1633383)), (1633064, 1633280), '+'), (('G000006925', (1834903, 1835664)), (1835005, 1835221), '-'), (('G000006925', (1878572, 1883763)), (1879015, 1879231), '+'), (('G000006925', (1944438, 1945202)), (1944881, 1945097), '+'), (('G000006925', (1962084, 1962845)), (1962527, 1962743), '+'), (('G000006925', (2069615, 2084300)), (2070058, 2070274), '+'), (('G000006925', (2346737, 2347498)), (2347180, 2347396), '+'), (('G000006925', (2477790, 2478551)), (2478233, 2478449), '+'), (('G000006925', (2953405, 2954171)), (2953508, 2953724), '-'), (('G000006925', (4392644, 4394283)), (4393624, 4393840), '-'), (('G000006925', (4823851, 4824717)), (4824058, 4824274), '-'), (('G000299455', (928393, 928731)), (928409, 928625), '+'), (('G000299455', (4559929, 4560763)), (4560371, 4560587), '+')]\n",
      ">G000006925|237940|238704|238384|238600|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|539107|539868|539550|539766|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|709671|710707|710048|710264|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000006925|716695|717456|717138|717354|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|902326|903088|902769|902985|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|1095979|1096741|1096082|1096298|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000006925|1632621|1633383|1633064|1633280|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|1834903|1835664|1835005|1835221|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000006925|1878572|1883763|1879015|1879231|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|1944438|1945202|1944881|1945097|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|1962084|1962845|1962527|1962743|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|2069615|2084300|2070058|2070274|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|2346737|2347498|2347180|2347396|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|2477790|2478551|2478233|2478449|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000006925|2953405|2954171|2953508|2953724|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000006925|4392644|4394283|4393624|4393840|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000006925|4823851|4824717|4824058|4824274|-\n",
      "tttcattctccatttcaatgcgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgtt-\n",
      ">G000299455|928393|928731|928409|928625|+\n",
      "-ttcattctccatttcaatgcgttgtagctttttcctcagcttacgtatttcgatttgtt\n",
      "ctggtgttatcggagaggcttttggtgttttgccctgacgctcatcacgcagttgtttga\n",
      "cccatcttgtcattgtggaaaggccaacatccatagctttggcggcatctgccaccgtgt\n",
      "atttctggtcaacaaccagttgagcggattcgcgttt\n",
      ">G000299455|4559929|4560763|4560371|4560587|+\n",
      "-ttcattttccatttcaatacgttgtagctttttcctgagctcacggatttcaatttgtt\n",
      "ccggggtaatgggggaggcttttggtgtttttccctgccgctcatcacgtaattgtttca\n",
      "cccatcgcgtcattgtggaaaggccgacatccatagcgctggctgcatctgccacggtgt\n",
      "atttctggtcaacgaccagttgagcggattcgcgttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2638009, 2638701)), (2638540, 2638701), '-'), (('G000006925', (2638833, 2641524)), (2638865, 2639023), '-'), (('G000299455', (1402725, 1405914)), (1405226, 1405384), '+')]\n",
      ">G000006925|2638009|2638701|2638540|2638701|-\n",
      "acggcggaaagggccggtgtcgataatacgctaaataaacaataattactctcttttgct\n",
      "tgacaaaaagagagttactggtgagtattgt-tttgctgtgaagcacctgcgttgctcat\n",
      "gtaacacaagtcaacctgctggaaaggccgcgaaccagacca--\n",
      ">G000006925|2638833|2641524|2638865|2639023|-\n",
      "acggcggaaagggccgcagtggataatacactaacagaacaataataactctcttttgct\n",
      "tgacaaaaagagagttactggcgagtattgtctccgctgagacgcgcttgcattactacg\n",
      "gtaata-----caacctgctggaaaggccgcgaaccggaccag-\n",
      ">G000299455|1402725|1405914|1405226|1405384|+\n",
      "-cggcggaaagggccgcagtggataatacactaacagaacaataataactctcttttgct\n",
      "tgacaaaaagagagttactggcgagtattgtctccgctgagacgcgcttgcattactacg\n",
      "gtaata-----caacctgctggaaaggccgcgaaccagaccagc\n",
      "\n",
      "\n",
      "8 [(('G000006925', (2366334, 2374443)), (2374003, 2374027), '-'), (('G000006925', (2374481, 2472008)), (2431281, 2431305), '+'), (('G000006925', (3503771, 3513305)), (3512780, 3512804), '-'), (('G000299455', (423834, 433368)), (424335, 424359), '+'), (('G000299455', (1623109, 1682614)), (1649180, 1649204), '-'), (('G000299455', (1685057, 1714323)), (1706629, 1706653), '+'), (('G000299455', (4300454, 4333834)), (4303158, 4303182), '+'), (('G000299455', (4515930, 4516155)), (4516131, 4516155), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\n",
      "Reallocating..done. *alloclen = 1050\n",
      "\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1168\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2366334|2374443|2374003|2374027|-\n",
      "tgataaaacagcaatggcaccacc-\n",
      ">G000006925|2374481|2472008|2431281|2431305|+\n",
      "-gataaaacagcatcgggatcacca\n",
      ">G000006925|3503771|3513305|3512780|3512804|-\n",
      "tggtaaaacaacaatggcaccacc-\n",
      ">G000299455|423834|433368|424335|424359|+\n",
      "-ggtaaaacaacaatagcaccacca\n",
      ">G000299455|1623109|1682614|1649180|1649204|-\n",
      "tgataaaacagcatcgggatcacc-\n",
      ">G000299455|1685057|1714323|1706629|1706653|+\n",
      "-gataaaacagcaatggcaccacca\n",
      ">G000299455|4300454|4333834|4303158|4303182|+\n",
      "-gataaaacagtatcggcaccacca\n",
      ">G000299455|4515930|4516155|4516131|4516155|+\n",
      "-gataaaacagcatcgggatgacca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (489080, 489347), '-'), (('G000299455', (315576, 316388)), (316121, 316388), '-'), (('G000299455', (3819953, 3834467)), (3832371, 3832638), '+')]\n",
      ">G000006925|487251|491968|489080|489347|-\n",
      "atgaaactgttgattgtcgaagatgaaaagaaaaccggagaatacttgaccaaagggtta\n",
      "accgaagccggttttgtggtcgatttggccgacaacgggctgaatggctaccatctggcg\n",
      "atgaccggtgattatgacctgataatcctcgatattatgctgccagacgtgaacggctgg\n",
      "gatatcgtgcgcatgttgcgctccgccaataaagggatgccgattctgttgcttaccgcg\n",
      "cttggcaccattgaacatcgcgtcaag-\n",
      ">G000299455|315576|316388|316121|316388|-\n",
      "atgaaaatattgatcgtcgaagacgaaattaaaacaggtgaatatctcagcaaagggctt\n",
      "acagaggcagggttcgtagtggatcacgctgataatggtcttaccggatatcatctcgcc\n",
      "atgacagccgagtatgatttagtcattctggatatcatgctacctgatgtgaacggctgg\n",
      "gatatcatccgcatgctgcgcactgccggaaagggtatgccggtcttactgctgacagcc\n",
      "ctcggcacgatcgaacatagggtcaaa-\n",
      ">G000299455|3819953|3834467|3832371|3832638|+\n",
      "-tgaaactgttgattgtcgaagatgaaaagaaaaccggagaatacttgaccaaagggtta\n",
      "accgaagccggttttgtggtcgatttggccgacaacgggttgaatggctaccatctggcg\n",
      "atgaccggtgattatgatctgataatcctcgatattatgctgccggacgtgaacggctgg\n",
      "gatatcgtgcgcatgttgcgttccgccaataaagggatgccgattctgttgcttaccgcg\n",
      "cttggcaccattgaacatcgcgtcaagg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (865572, 865846)), (865574, 865657), '-'), (('G000006925', (2242494, 2346249)), (2251946, 2252029), '-'), (('G000006925', (3229969, 3230122)), (3230038, 3230121), '+'), (('G000299455', (1819843, 1851664)), (1826479, 1826562), '+'), (('G000299455', (3242963, 3243106)), (3243022, 3243105), '+'), (('G000299455', (3470062, 3470342)), (3470069, 3470152), '-')]\n",
      ">G000006925|865572|865846|865574|865657|-\n",
      "cagggggcggcggcgactggctgccccctgcgcgctccctccgccagtggctttatatgg\n",
      "cattgcacatgaaaggagcgcaa-\n",
      ">G000006925|2242494|2346249|2251946|2252029|-\n",
      "cagggggcggcggcgattggtcgccccctgcgcgctccttgcgccagtgattatatgaag\n",
      "caatgcacatgacaggagcgcaa-\n",
      ">G000006925|3229969|3230122|3230038|3230121|+\n",
      "-agggggcggcggcgactggctgccccctgcgcgctccctgcgccagtgactttatatgg\n",
      "cataacacgttaaaggagcgcaac\n",
      ">G000299455|1819843|1851664|1826479|1826562|+\n",
      "-agggggcggcggcgattggccgccccctgcgcgctccttgcgccagtgattatatgaag\n",
      "caatgcacatgacaggagcgcaac\n",
      ">G000299455|3242963|3243106|3243022|3243105|+\n",
      "-agggggcggcggcgattggccgccccctgcgcgctccctgcaccattcaatttatatgg\n",
      "cgaggcacattaagggagcgcaac\n",
      ">G000299455|3470062|3470342|3470069|3470152|-\n",
      "catggggcggcggcgattggccgccccctgcgcgctccttgcgccagtggcaatatgtgg\n",
      "cttagctcatgaaaggagcgcaa-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2641574, 2668966)), (2661995, 2662018), '+'), (('G000006925', (4433907, 4437456)), (4436214, 4436237), '+'), (('G000299455', (1375183, 1402556)), (1382132, 1382155), '-'), (('G000299455', (4626810, 4646234)), (4644992, 4645015), '+'), (('G000299455', (5202806, 5268469)), (5216738, 5216761), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2641574|2668966|2661995|2662018|+\n",
      "-actttttgctgattgccaccaga\n",
      ">G000006925|4433907|4437456|4436214|4436237|+\n",
      "-actttttgctgattgccaccgga\n",
      ">G000299455|1375183|1402556|1382132|1382155|-\n",
      "cactttttgctgattgccgccag-\n",
      ">G000299455|4626810|4646234|4644992|4645015|+\n",
      "-actttttgctgattgccgccaga\n",
      ">G000299455|5202806|5268469|5216738|5216761|+\n",
      "-actttttgctgattgccaccgga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3612083, 3637192)), (3617344, 3617433), '-'), (('G000299455', (288342, 302750)), (297403, 297492), '+'), (('G000299455', (2459722, 2494618)), (2475499, 2475588), '+')]\n",
      ">G000006925|3612083|3637192|3617344|3617433|-\n",
      "gcaaagaagttcagcgatgttggcatctgggcgtagagaatgtaaaacaccaccgcttcg\n",
      "agcatcaggacaaaggcgacaaacatttt-\n",
      ">G000299455|288342|302750|297403|297492|+\n",
      "-caaagaagttcagcgatgttggcatctgggcgtagagaatgtaaaacaccaccgcttcg\n",
      "agcatcaggacaaaggcgacaaacatttta\n",
      ">G000299455|2459722|2494618|2475499|2475588|+\n",
      "-caaagaagttcagtgacgttggcatctggctgtacagcacgaagaagataatggcttcg\n",
      "agcatcaggatgaaggcaacgatcatttta\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2976303, 3015537)), (2978687, 2978710), '-'), (('G000299455', (988799, 1028140)), (1014313, 1014336), '+'), (('G000299455', (1229970, 1230012)), (1229970, 1229994), '+'), (('G000299455', (2613922, 2614096)), (2613922, 2613945), '-')]\n",
      ">G000006925|2976303|3015537|2978687|2978710|-\n",
      "ccgttttcgacaataaacagcgg--\n",
      ">G000299455|988799|1028140|1014313|1014336|+\n",
      "-cgttttcgacaataaacagcggc-\n",
      ">G000299455|1229970|1230012|1229970|1229994|+\n",
      "-cgttttccaccagaaacagcggct\n",
      ">G000299455|2613922|2614096|2613922|2613945|-\n",
      "ccattttcgacaataaacagcgg--\n",
      "\n",
      "\n",
      "108 [(('G000006925', (1, 24671)), (18759, 18792), '+'), (('G000006925', (64491, 216702)), (87585, 87618), '-'), (('G000006925', (64491, 216702)), (148775, 148808), '-'), (('G000006925', (266682, 267450)), (267326, 267359), '-'), (('G000006925', (273344, 278315)), (273435, 273468), '+'), (('G000006925', (278320, 279091)), (278966, 278999), '-'), (('G000006925', (300111, 308544)), (308419, 308452), '-'), (('G000006925', (378637, 384812)), (379283, 379316), '-'), (('G000006925', (459861, 468373)), (465808, 465841), '+'), (('G000006925', (468374, 487224)), (478804, 478837), '+'), (('G000006925', (520755, 528026)), (521399, 521432), '-'), (('G000006925', (545486, 558918)), (558241, 558274), '+'), (('G000006925', (622915, 623662)), (623006, 623039), '+'), (('G000006925', (865887, 867389)), (866712, 866745), '+'), (('G000006925', (984595, 985335)), (985241, 985274), '-'), (('G000006925', (985336, 1008622)), (985521, 985554), '+'), (('G000006925', (1025530, 1063042)), (1049663, 1049696), '+'), (('G000006925', (1189492, 1208094)), (1197739, 1197772), '+'), (('G000006925', (1239481, 1240251)), (1239572, 1239605), '+'), (('G000006925', (1290847, 1306886)), (1295164, 1295197), '+'), (('G000006925', (1321493, 1385605)), (1366098, 1366131), '-'), (('G000006925', (1398496, 1399271)), (1398589, 1398622), '+'), (('G000006925', (1409877, 1410649)), (1410525, 1410558), '-'), (('G000006925', (1424264, 1425034)), (1424910, 1424943), '-'), (('G000006925', (1425133, 1465118)), (1444371, 1444404), '+'), (('G000006925', (1509064, 1578296)), (1526770, 1526803), '+'), (('G000006925', (1509064, 1578296)), (1542857, 1542890), '+'), (('G000006925', (1509064, 1578296)), (1568835, 1568868), '-'), (('G000006925', (1509064, 1578296)), (1574793, 1574826), '-'), (('G000006925', (1670306, 1692658)), (1673463, 1673496), '+'), (('G000006925', (1729332, 1741706)), (1736618, 1736651), '-'), (('G000006925', (1758378, 1759147)), (1758469, 1758502), '+'), (('G000006925', (1759148, 1776249)), (1766619, 1766652), '+'), (('G000006925', (1759148, 1776249)), (1769676, 1769709), '-'), (('G000006925', (1776250, 1792578)), (1781004, 1781037), '+'), (('G000006925', (1793868, 1802424)), (1795773, 1795806), '-'), (('G000006925', (1842508, 1869155)), (1850693, 1850726), '-'), (('G000006925', (1842508, 1869155)), (1856631, 1856664), '-'), (('G000006925', (1940277, 1941051)), (1940368, 1940401), '+'), (('G000006925', (1976680, 1977448)), (1977324, 1977357), '-'), (('G000006925', (2036705, 2044875)), (2037799, 2037832), '-'), (('G000006925', (2050169, 2051841)), (2051717, 2051750), '-'), (('G000006925', (2088344, 2103602)), (2095131, 2095164), '+'), (('G000006925', (2120115, 2138771)), (2135292, 2135325), '-'), (('G000006925', (2172359, 2184266)), (2177883, 2177916), '+'), (('G000006925', (2187913, 2198767)), (2197531, 2197564), '-'), (('G000006925', (2198768, 2199537)), (2198860, 2198893), '+'), (('G000006925', (2199538, 2210687)), (2206195, 2206228), '-'), (('G000006925', (2211287, 2225774)), (2220724, 2220757), '+'), (('G000006925', (2235234, 2236005)), (2235878, 2235911), '-'), (('G000006925', (2242494, 2346249)), (2287536, 2287569), '+'), (('G000006925', (2374481, 2472008)), (2396279, 2396312), '-'), (('G000006925', (2756085, 2756853)), (2756729, 2756762), '-'), (('G000006925', (2816684, 2833035)), (2825292, 2825325), '+'), (('G000006925', (2849870, 2870340)), (2863484, 2863517), '-'), (('G000006925', (2964578, 2976271)), (2964673, 2964706), '-'), (('G000006925', (2976303, 3015537)), (3014724, 3014757), '+'), (('G000006925', (3016790, 3036697)), (3021555, 3021588), '+'), (('G000006925', (3038529, 3039299)), (3038620, 3038653), '+'), (('G000006925', (3104758, 3106299)), (3104850, 3104883), '+'), (('G000006925', (3179574, 3180459)), (3180334, 3180367), '-'), (('G000006925', (3180744, 3203996)), (3182225, 3182258), '-'), (('G000006925', (3237769, 3244530)), (3238413, 3238446), '-'), (('G000006925', (3268417, 3281594)), (3274415, 3274448), '-'), (('G000006925', (3322761, 3361739)), (3361612, 3361645), '-'), (('G000006925', (3361966, 3403725)), (3362612, 3362645), '-'), (('G000006925', (3608503, 3611864)), (3609152, 3609185), '-'), (('G000006925', (3612083, 3637192)), (3624324, 3624357), '+'), (('G000006925', (3637330, 3640023)), (3637421, 3637454), '+'), (('G000006925', (3658114, 3676993)), (3665251, 3665284), '+'), (('G000006925', (3693106, 3702974)), (3693752, 3693785), '-'), (('G000006925', (3703179, 3717854)), (3703824, 3703857), '-'), (('G000006925', (3717879, 3736288)), (3726108, 3726141), '-'), (('G000006925', (3736321, 3788106)), (3743113, 3743146), '+'), (('G000006925', (3815875, 3829465)), (3818096, 3818129), '+'), (('G000006925', (3831593, 3850590)), (3834577, 3834610), '+'), (('G000006925', (3858569, 3864367)), (3864243, 3864276), '-'), (('G000006925', (3939882, 3952896)), (3943478, 3943511), '-'), (('G000006925', (4002899, 4012437)), (4003543, 4003576), '-'), (('G000006925', (4069200, 4076620)), (4076498, 4076531), '-'), (('G000006925', (4091607, 4092380)), (4091699, 4091732), '+'), (('G000006925', (4094093, 4116338)), (4102405, 4102438), '+'), (('G000006925', (4174522, 4188276)), (4177207, 4177240), '-'), (('G000006925', (4256570, 4288375)), (4285070, 4285103), '+'), (('G000006925', (4288513, 4315006)), (4314881, 4314914), '-'), (('G000006925', (4362175, 4367702)), (4367025, 4367058), '+'), (('G000006925', (4378758, 4379529)), (4378849, 4378882), '+'), (('G000006925', (4389053, 4390969)), (4389145, 4389178), '+'), (('G000006925', (4392644, 4394283)), (4393293, 4393326), '-'), (('G000006925', (4397536, 4410956)), (4410832, 4410865), '-'), (('G000006925', (4462252, 4478752)), (4462344, 4462377), '+'), (('G000006925', (4478753, 4490457)), (4488482, 4488515), '+'), (('G000006925', (4497401, 4528664)), (4515514, 4515547), '-'), (('G000006925', (4535976, 4543278)), (4541154, 4541187), '-'), (('G000006925', (4557883, 4573403)), (4570367, 4570400), '-'), (('G000006925', (4645852, 4646621)), (4646496, 4646529), '-'), (('G000006925', (4656735, 4658205)), (4657380, 4657413), '-'), (('G000006925', (4785482, 4786253)), (4785575, 4785608), '+'), (('G000299455', (91523, 102755)), (92172, 92205), '-'), (('G000299455', (696017, 696792)), (696110, 696143), '+'), (('G000299455', (941725, 942502)), (941819, 941852), '+'), (('G000299455', (1581872, 1582655)), (1582521, 1582554), '-'), (('G000299455', (2052398, 2064114)), (2062601, 2062634), '-'), (('G000299455', (3182222, 3183821)), (3183140, 3183173), '+'), (('G000299455', (3672841, 3680073)), (3679943, 3679976), '-'), (('G000299455', (4588613, 4589520)), (4588836, 4588869), '+'), (('G000299455', (4592900, 4593681)), (4593551, 4593584), '-'), (('G000299455', (5323356, 5324120)), (5323992, 5324025), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 108\r",
      "  101 / 108\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 108\r",
      "   10 / 108\r",
      "   20 / 108\r",
      "   30 / 108\r",
      "   40 / 108\r",
      "   50 / 108\r",
      "   60 / 108\r",
      "   70 / 108\r",
      "   80 / 108\r",
      "   90 / 108\r",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 107  f\b\b\r",
      "STEP     2 / 107  f\b\b\r",
      "STEP     3 / 107  f\b\b\r",
      "STEP     4 / 107  f\b\b\r",
      "STEP     5 / 107  f\b\b\r",
      "STEP     6 / 107  f\b\b\r",
      "STEP     7 / 107  f\b\b\r",
      "STEP     8 / 107  f\b\b\r",
      "STEP     9 / 107  f\b\b\r",
      "STEP    10 / 107  f\b\b\r",
      "STEP    11 / 107  f\b\b\r",
      "STEP    12 / 107  f\b\b\r",
      "STEP    13 / 107  f\b\b\r",
      "STEP    14 / 107  f\b\b\r",
      "STEP    15 / 107  f\b\b\r",
      "STEP    16 / 107  f\b\b\r",
      "STEP    17 / 107  f\b\b\r",
      "STEP    18 / 107  f\b\b\r",
      "STEP    19 / 107  f\b\b\r",
      "STEP    20 / 107  f\b\b\r",
      "STEP    21 / 107  f\b\b\r",
      "STEP    22 / 107  f\b\b\r",
      "STEP    23 / 107  f\b\b\r",
      "STEP    24 / 107  f\b\b\r",
      "STEP    25 / 107  f\b\b\r",
      "STEP    26 / 107  f\b\b\r",
      "STEP    27 / 107  f\b\b\r",
      "STEP    28 / 107  f\b\b\r",
      "STEP    29 / 107  f\b\b\r",
      "STEP    30 / 107  f\b\b\r",
      "STEP    31 / 107  f\b\b\r",
      "STEP    32 / 107  f\b\b\r",
      "STEP    33 / 107  f\b\b\r",
      "STEP    34 / 107  f\b\b\r",
      "STEP    35 / 107  f\b\b\r",
      "STEP    36 / 107  f\b\b\r",
      "STEP    37 / 107  f\b\b\r",
      "STEP    38 / 107  f\b\b\r",
      "STEP    39 / 107  f\b\b\r",
      "STEP    40 / 107  f\b\b\r",
      "STEP    41 / 107  f\b\b\r",
      "STEP    42 / 107  f\b\b\r",
      "STEP    43 / 107  f\b\b\r",
      "STEP    44 / 107  f\b\b\r",
      "STEP    45 / 107  f\b\b\r",
      "STEP    46 / 107  f\b\b\r",
      "STEP    47 / 107  f\b\b\r",
      "STEP    48 / 107  f\b\b\r",
      "STEP    49 / 107  f\b\b\r",
      "STEP    50 / 107  f\b\b\r",
      "STEP    51 / 107  f\b\b\r",
      "STEP    52 / 107  f\b\b\r",
      "STEP    53 / 107  f\b\b\r",
      "STEP    54 / 107  f\b\b\r",
      "STEP    55 / 107  f\b\b\r",
      "STEP    56 / 107  f\b\b\r",
      "STEP    57 / 107  f\b\b\r",
      "STEP    58 / 107  f\b\b\r",
      "STEP    59 / 107  f\b\b\r",
      "STEP    60 / 107  f\b\b\r",
      "STEP    61 / 107  f\b\b\r",
      "STEP    62 / 107  f\b\b\r",
      "STEP    63 / 107  f\b\b\r",
      "STEP    64 / 107  f\b\b\r",
      "STEP    65 / 107  f\b\b\r",
      "STEP    66 / 107  f\b\b\r",
      "STEP    67 / 107  f\b\b\r",
      "STEP    68 / 107  f\b\b\r",
      "STEP    69 / 107  f\b\b\r",
      "STEP    70 / 107  f\b\b\r",
      "STEP    71 / 107  f\b\b\r",
      "STEP    72 / 107  f\b\b\r",
      "STEP    73 / 107  f\b\b\r",
      "STEP    74 / 107  f\b\b\r",
      "STEP    75 / 107  f\b\b\r",
      "STEP    76 / 107  f\b\b\r",
      "STEP    77 / 107  f\b\b\r",
      "STEP    78 / 107  f\b\b\r",
      "STEP    79 / 107  f\b\b\r",
      "STEP    80 / 107  f\b\b\r",
      "STEP    81 / 107  f\b\b\r",
      "STEP    82 / 107  f\b\b\r",
      "STEP    83 / 107  f\b\b\r",
      "STEP    84 / 107  f\b\b\r",
      "STEP    85 / 107  f\b\b\r",
      "STEP    86 / 107  f\b\b\r",
      "STEP    87 / 107  f\b\b\r",
      "STEP    88 / 107  f\b\b\r",
      "STEP    89 / 107  f\b\b\r",
      "STEP    90 / 107  f\b\b\r",
      "STEP    91 / 107  f\b\b\r",
      "STEP    92 / 107  f\b\b\r",
      "STEP    93 / 107  f\b\b\r",
      "STEP    94 / 107  f\b\b\r",
      "STEP    95 / 107  f\b\b\r",
      "STEP    96 / 107  f\b\b\r",
      "STEP    97 / 107  f\b\b\r",
      "STEP    98 / 107  f\b\b\r",
      "STEP    99 / 107  f\b\b\n",
      "Reallocating..done. *alloclen = 1068\n",
      "\r",
      "STEP   100 / 107  d\b\b\r",
      "STEP   101 / 107  d\b\b\r",
      "STEP   102 / 107  d\b\b h\r",
      "STEP   103 / 107  f\b\b\r",
      "STEP   104 / 107  d\b\b h\r",
      "STEP   105 / 107  d\b\b h\r",
      "STEP   106 / 107  d\b\b h\r",
      "STEP   107 / 107  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 108\r",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 108\r",
      "   10 / 108\r",
      "   20 / 108\r",
      "   30 / 108\r",
      "   40 / 108\r",
      "   50 / 108\r",
      "   60 / 108\r",
      "   70 / 108\r",
      "   80 / 108\r",
      "   90 / 108\r",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 107  f\b\b\r",
      "STEP     2 / 107  f\b\b\r",
      "STEP     3 / 107  f\b\b\r",
      "STEP     4 / 107  f\b\b\r",
      "STEP     5 / 107  f\b\b\r",
      "STEP     6 / 107  f\b\b\r",
      "STEP     7 / 107  f\b\b\r",
      "STEP     8 / 107  f\b\b\r",
      "STEP     9 / 107  f\b\b\r",
      "STEP    10 / 107  f\b\b\r",
      "STEP    11 / 107  f\b\b\r",
      "STEP    12 / 107  f\b\b\r",
      "STEP    13 / 107  f\b\b\r",
      "STEP    14 / 107  f\b\b\r",
      "STEP    15 / 107  f\b\b\r",
      "STEP    16 / 107  f\b\b\r",
      "STEP    17 / 107  f\b\b\r",
      "STEP    18 / 107  f\b\b\r",
      "STEP    19 / 107  f\b\b\r",
      "STEP    20 / 107  f\b\b\r",
      "STEP    21 / 107  f\b\b\r",
      "STEP    22 / 107  f\b\b\r",
      "STEP    23 / 107  f\b\b\r",
      "STEP    24 / 107  f\b\b\r",
      "STEP    25 / 107  f\b\b\r",
      "STEP    26 / 107  f\b\b\r",
      "STEP    27 / 107  f\b\b\r",
      "STEP    28 / 107  f\b\b\r",
      "STEP    29 / 107  f\b\b\r",
      "STEP    30 / 107  f\b\b\r",
      "STEP    31 / 107  f\b\b\r",
      "STEP    32 / 107  f\b\b\r",
      "STEP    33 / 107  f\b\b\r",
      "STEP    34 / 107  f\b\b\r",
      "STEP    35 / 107  f\b\b\r",
      "STEP    36 / 107  f\b\b\r",
      "STEP    37 / 107  f\b\b\r",
      "STEP    38 / 107  f\b\b\r",
      "STEP    39 / 107  f\b\b\r",
      "STEP    40 / 107  f\b\b\r",
      "STEP    41 / 107  f\b\b\r",
      "STEP    42 / 107  f\b\b\r",
      "STEP    43 / 107  f\b\b\r",
      "STEP    44 / 107  f\b\b\r",
      "STEP    45 / 107  f\b\b\r",
      "STEP    46 / 107  f\b\b\r",
      "STEP    47 / 107  f\b\b\r",
      "STEP    48 / 107  f\b\b\r",
      "STEP    49 / 107  f\b\b\r",
      "STEP    50 / 107  f\b\b\r",
      "STEP    51 / 107  f\b\b\r",
      "STEP    52 / 107  f\b\b\r",
      "STEP    53 / 107  f\b\b\r",
      "STEP    54 / 107  f\b\b\r",
      "STEP    55 / 107  f\b\b\r",
      "STEP    56 / 107  f\b\b\r",
      "STEP    57 / 107  f\b\b\r",
      "STEP    58 / 107  f\b\b\r",
      "STEP    59 / 107  f\b\b\r",
      "STEP    60 / 107  f\b\b\r",
      "STEP    61 / 107  f\b\b\r",
      "STEP    62 / 107  f\b\b\r",
      "STEP    63 / 107  f\b\b\r",
      "STEP    64 / 107  f\b\b\r",
      "STEP    65 / 107  f\b\b\r",
      "STEP    66 / 107  f\b\b\r",
      "STEP    67 / 107  f\b\b\r",
      "STEP    68 / 107  f\b\b\r",
      "STEP    69 / 107  f\b\b\r",
      "STEP    70 / 107  f\b\b\r",
      "STEP    71 / 107  f\b\b\r",
      "STEP    72 / 107  f\b\b\r",
      "STEP    73 / 107  f\b\b\r",
      "STEP    74 / 107  f\b\b\r",
      "STEP    75 / 107  f\b\b\r",
      "STEP    76 / 107  f\b\b\r",
      "STEP    77 / 107  f\b\b\r",
      "STEP    78 / 107  f\b\b\r",
      "STEP    79 / 107  f\b\b\r",
      "STEP    80 / 107  f\b\b\r",
      "STEP    81 / 107  f\b\b\r",
      "STEP    82 / 107  f\b\b\r",
      "STEP    83 / 107  f\b\b\r",
      "STEP    84 / 107  f\b\b\r",
      "STEP    85 / 107  f\b\b\r",
      "STEP    86 / 107  f\b\b\r",
      "STEP    87 / 107  f\b\b\r",
      "STEP    88 / 107  f\b\b\r",
      "STEP    89 / 107  f\b\b\r",
      "STEP    90 / 107  f\b\b\r",
      "STEP    91 / 107  f\b\b\r",
      "STEP    92 / 107  f\b\b\r",
      "STEP    93 / 107  f\b\b\r",
      "STEP    94 / 107  f\b\b\r",
      "STEP    95 / 107  f\b\b\r",
      "STEP    96 / 107  f\b\b\r",
      "STEP    97 / 107  f\b\b\r",
      "STEP    98 / 107  f\b\b\r",
      "STEP    99 / 107  f\b\b\r",
      "STEP   100 / 107  f\b\b\r",
      "STEP   101 / 107  f\b\b\r",
      "STEP   102 / 107  d\b\b\r",
      "STEP   103 / 107  f\b\b\r",
      "STEP   104 / 107  f\b\b\r",
      "STEP   105 / 107  d\b\b\r",
      "STEP   106 / 107  d\b\b h\r",
      "STEP   107 / 107  d\b\b h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|18759|18792|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|64491|216702|87585|87618|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|64491|216702|148775|148808|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|266682|267450|267326|267359|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|273344|278315|273435|273468|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|278320|279091|278966|278999|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|300111|308544|308419|308452|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|378637|384812|379283|379316|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|459861|468373|465808|465841|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|468374|487224|478804|478837|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|520755|528026|521399|521432|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|545486|558918|558241|558274|+\n",
      "-gacttccgtaccggccttgccagatgttgtctc\n",
      ">G000006925|622915|623662|623006|623039|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|865887|867389|866712|866745|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|984595|985335|985241|985274|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|985336|1008622|985521|985554|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1025530|1063042|1049663|1049696|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1189492|1208094|1197739|1197772|+\n",
      "-gacttccgtaccggccttgccagatgttgtctc\n",
      ">G000006925|1239481|1240251|1239572|1239605|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1290847|1306886|1295164|1295197|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1321493|1385605|1366098|1366131|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1398496|1399271|1398589|1398622|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|1409877|1410649|1410525|1410558|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1424264|1425034|1424910|1424943|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|1425133|1465118|1444371|1444404|+\n",
      "-gacatccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1509064|1578296|1526770|1526803|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1509064|1578296|1542857|1542890|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1509064|1578296|1568835|1568868|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1509064|1578296|1574793|1574826|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1670306|1692658|1673463|1673496|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|1729332|1741706|1736618|1736651|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1758378|1759147|1758469|1758502|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|1759148|1776249|1766619|1766652|+\n",
      "-aacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|1759148|1776249|1769676|1769709|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|1776250|1792578|1781004|1781037|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|1793868|1802424|1795773|1795806|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|1842508|1869155|1850693|1850726|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|1842508|1869155|1856631|1856664|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|1940277|1941051|1940368|1940401|+\n",
      "-gacttccgtcccagccgtgccaggtgctgcctc\n",
      ">G000006925|1976680|1977448|1977324|1977357|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2036705|2044875|2037799|2037832|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2050169|2051841|2051717|2051750|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2088344|2103602|2095131|2095164|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|2120115|2138771|2135292|2135325|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2172359|2184266|2177883|2177916|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|2187913|2198767|2197531|2197564|-\n",
      "cgacttccgtcccagccgtgccaggtgctgcct-\n",
      ">G000006925|2198768|2199537|2198860|2198893|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|2199538|2210687|2206195|2206228|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2211287|2225774|2220724|2220757|+\n",
      "-gacttccgtaccagccttgccaaatgttgtctc\n",
      ">G000006925|2235234|2236005|2235878|2235911|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2242494|2346249|2287536|2287569|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|2374481|2472008|2396279|2396312|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2756085|2756853|2756729|2756762|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2816684|2833035|2825292|2825325|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|2849870|2870340|2863484|2863517|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2964578|2976271|2964673|2964706|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|2976303|3015537|3014724|3014757|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3016790|3036697|3021555|3021588|+\n",
      "-ggcttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3038529|3039299|3038620|3038653|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|3104758|3106299|3104850|3104883|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|3179574|3180459|3180334|3180367|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3180744|3203996|3182225|3182258|-\n",
      "taacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3237769|3244530|3238413|3238446|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3268417|3281594|3274415|3274448|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3322761|3361739|3361612|3361645|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|3361966|3403725|3362612|3362645|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3608503|3611864|3609152|3609185|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|3612083|3637192|3624324|3624357|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3637330|3640023|3637421|3637454|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3658114|3676993|3665251|3665284|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3693106|3702974|3693752|3693785|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3703179|3717854|3703824|3703857|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3717879|3736288|3726108|3726141|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|3736321|3788106|3743113|3743146|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|3815875|3829465|3818096|3818129|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3831593|3850590|3834577|3834610|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|3858569|3864367|3864243|3864276|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|3939882|3952896|3943478|3943511|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4002899|4012437|4003543|4003576|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4069200|4076620|4076498|4076531|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4091607|4092380|4091699|4091732|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4094093|4116338|4102405|4102438|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|4174522|4188276|4177207|4177240|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4256570|4288375|4285070|4285103|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4288513|4315006|4314881|4314914|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4362175|4367702|4367025|4367058|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4378758|4379529|4378849|4378882|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4389053|4390969|4389145|4389178|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4392644|4394283|4393293|4393326|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|4397536|4410956|4410832|4410865|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4462252|4478752|4462344|4462377|+\n",
      "-gacttccgtcccagccttgccagatgttgtctc\n",
      ">G000006925|4478753|4490457|4488482|4488515|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000006925|4497401|4528664|4515514|4515547|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|4535976|4543278|4541154|4541187|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4557883|4573403|4570367|4570400|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|4645852|4646621|4646496|4646529|-\n",
      "tgacttccgtcccagccttgccagatgttgtct-\n",
      ">G000006925|4656735|4658205|4657380|4657413|-\n",
      "tgacttccgtaccagccttgccagatgttgtct-\n",
      ">G000006925|4785482|4786253|4785575|4785608|+\n",
      "-gacttccgtaccagccttgccagatgttgtctc\n",
      ">G000299455|91523|102755|92172|92205|-\n",
      "cgacttccgtcccagtcgtgccaggtgctgcct-\n",
      ">G000299455|696017|696792|696110|696143|+\n",
      "-gacttccgtcccagccgtgccaggtgctgcctc\n",
      ">G000299455|941725|942502|941819|941852|+\n",
      "-gacttccgtcccagccgtgccaggtgctgcctc\n",
      ">G000299455|1581872|1582655|1582521|1582554|-\n",
      "cgacttccgtcccagtcgtgccaggtgctgcct-\n",
      ">G000299455|2052398|2064114|2062601|2062634|-\n",
      "cgacttccgtcccagccgtgccaggtgctgcct-\n",
      ">G000299455|3182222|3183821|3183140|3183173|+\n",
      "-gacttccgtcccagccgtgccaggtgctgcctc\n",
      ">G000299455|3672841|3680073|3679943|3679976|-\n",
      "cgacttccgtcccagccgtgccaggtgctgcct-\n",
      ">G000299455|4588613|4589520|4588836|4588869|+\n",
      "-aacttccgtcccagccttgccagatgttgtctc\n",
      ">G000299455|4592900|4593681|4593551|4593584|-\n",
      "cgacttccgtcccagccgtgccaggtgctgcct-\n",
      ">G000299455|5323356|5324120|5323992|5324025|-\n",
      "cgacttccgtcccagccgtgccaggtgctgcct-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3040656, 3065083)), (3053988, 3054015), '+'), (('G000299455', (942831, 956197)), (942839, 942864), '-'), (('G000299455', (4713031, 4748324)), (4737228, 4737255), '+'), (('G000299455', (4771847, 4785261)), (4785234, 4785261), '+')]\n",
      ">G000006925|3040656|3065083|3053988|3054015|+\n",
      "-attaaaatagtaagcagtccgggcaat\n",
      ">G000299455|942831|956197|942839|942864|-\n",
      "cattaaaataataa--aatccgggcaa-\n",
      ">G000299455|4713031|4748324|4737228|4737255|+\n",
      "-attaaaatagtaagcagtccgggcaat\n",
      ">G000299455|4771847|4785261|4785234|4785261|+\n",
      "-attaaaatagtaagcgctccggacaat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2333074, 2333106), '-'), (('G000299455', (1720965, 1752624)), (1746004, 1746036), '+'), (('G000299455', (3608677, 3610659)), (3610010, 3610042), '+')]\n",
      ">G000006925|2242494|2346249|2333074|2333106|-\n",
      "cgacaaccagttcactcgtgacgctggcatca-\n",
      ">G000299455|1720965|1752624|1746004|1746036|+\n",
      "-gacaaccagttcactcgtgacgctggcatcaa\n",
      ">G000299455|3608677|3610659|3610010|3610042|+\n",
      "-aagagcgatttcacgaaagcatctggcgttgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3573008, 3573034), '-'), (('G000299455', (335504, 377237)), (363017, 363043), '+'), (('G000299455', (4221161, 4233966)), (4227644, 4227670), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3559544|3600494|3573008|3573034|-\n",
      "cgccgccagattggcatgattttcca-\n",
      ">G000299455|335504|377237|363017|363043|+\n",
      "-gccgccagattggcatgattttccag\n",
      ">G000299455|4221161|4233966|4227644|4227670|+\n",
      "-gccgccagattggtatgattttccag\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1063049, 1063191)), (1063049, 1063087), '+'), (('G000006925', (1090935, 1091083)), (1090941, 1090979), '+'), (('G000006925', (1804109, 1804251)), (1804213, 1804251), '-'), (('G000006925', (1815261, 1815403)), (1815365, 1815403), '-'), (('G000006925', (4619766, 4619804)), (4619766, 4619804), '+'), (('G000006925', (4751699, 4751737)), (4751699, 4751737), '+'), (('G000006925', (4818440, 4818585)), (4818443, 4818481), '+'), (('G000299455', (5328484, 5333124)), (5332982, 5333020), '+')]\n",
      ">G000006925|1063049|1063191|1063049|1063087|+\n",
      "-gataggaattaaaaccccaaaaagattaaaaaaacacc\n",
      ">G000006925|1090935|1091083|1090941|1090979|+\n",
      "-gataggaattaaaaccccaaaaagattaaaaaaacacc\n",
      ">G000006925|1804109|1804251|1804213|1804251|-\n",
      "cgataggaattaaaaccccaaaaagattaaaaaaacac-\n",
      ">G000006925|1815261|1815403|1815365|1815403|-\n",
      "cgataggaattaaaaccccaaaaagattaaaaaaacac-\n",
      ">G000006925|4619766|4619804|4619766|4619804|+\n",
      "-gataggaatttaaatccccaaaagattaaaaaagcacc\n",
      ">G000006925|4751699|4751737|4751699|4751737|+\n",
      "-gataggaatttaaatccccaaaagattaaaaaagcacc\n",
      ">G000006925|4818440|4818585|4818443|4818481|+\n",
      "-gataggaattaaaaccccaaaaagattaaaaaaacacc\n",
      ">G000299455|5328484|5333124|5332982|5333020|+\n",
      "-gataggaattaaaaacctaaaaagattaaaaaaacacc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (608503, 616145)), (614534, 614991), '+'), (('G000299455', (3680085, 3687629)), (3686116, 3686573), '+'), (('G000299455', (4791040, 4791497)), (4791040, 4791497), '+')]\n",
      ">G000006925|608503|616145|614534|614991|+\n",
      "cacaaccgcgcgctatttactatgtcgttgcgctgcaaatctgggaatattttagctttt\n",
      "acggcatgcgtgccctgctgattctctatctcaccaatcaactaaaatacaacgatactc\n",
      "acgcctacgagttatttagcgcctactgttcgctggtgtatgtcacgccaatcctcggtg\n",
      "gctttttggcggataaagttctcggcaatcgcatggcggtgatgctgggggcgttgttga\n",
      "tggcgatcggtcatgtggtgctgggtgccagtg-agatccatccgtcattcctctatctg\n",
      "tccctggcgattatcgtctgcggctatggcctgtttaaatctaacgtaagctgtctgctc\n",
      "ggcgagctgtatgagccaaccgatccgcgtcgtgatggcggtttctcgctgatgtatgcg\n",
      "gcgggtaacgtggggtctattatcgcacctatcgcctg\n",
      ">G000299455|3680085|3687629|3686116|3686573|+\n",
      "cacaaccgcgcgctatttactatgtcgttgcgctgcaaatctgggaatattttagctttt\n",
      "acggcatgcgtgccctgctgattctctatctcaccaatcaactaaaatacaacgatactc\n",
      "acgcctacgagttatttagcgcctactgttcgctggtgtatgtcacgccaatcctcggtg\n",
      "gctttttggcggataaagttctcggcaatcgcatggcggtgatgctgggggcgttgttga\n",
      "tggcgatcggtcatgtggtgctgggtgccagtg-agatccatccgtcattcctctatctg\n",
      "tccctggcgattatcgtttgtggctatggtctgtttaaatccaatgtcagttgcctgctc\n",
      "ggcgagctgtatgagccaaccgatccgcgtcgtgatggcggtttctcgctgatgtatgca\n",
      "gcgggtaacgtggggtctattatcgcacctatcgcctg\n",
      ">G000299455|4791040|4791497|4791040|4791497|+\n",
      "cacagccgcgcgcgatatactatatcgtggcgatccaaatctgggagtacttcagttttt\n",
      "acggcatgcgtgccttactcattctctatctcacccatcagcttggttttgatgataacc\n",
      "atgccatcagcctgttcagcgcatatgcttctctggtttacgttacccctattctcggcg\n",
      "gctggcttgccgaccgcctgctcggcaaccgcactgcagtgattgccggcgcgctgttaa\n",
      "tgacccttggccatgtggtgctgggcattgatacaaattcaaccttta-gcctgtatctg\n",
      "gcgctggcaatcattatttgtggctacggtttattcaaatcaaacatcagctgtttgctt\n",
      "ggcgagctctacgacgagaacgatcatcgacgtgatggcggtttttcgctgctgtatgct\n",
      "gcgggcaatatcggttctatcgccgccccaatcgcctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4816831, 4818356)), (4817983, 4818101), '-'), (('G000299455', (5308545, 5311143)), (5309454, 5309572), '+'), (('G000299455', (5385343, 5385755)), (5385343, 5385461), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4816831|4818356|4817983|4818101|-\n",
      "tcgatacgtccagccagaagactgacagcgatgatgatg--tagttacaacattcataat\n",
      "taaaagcgactctgttcccgccctttgggcgggggcggcggcgcttttcagttatgaggg\n",
      "---\n",
      ">G000299455|5308545|5311143|5309454|5309572|+\n",
      "tcaatacttccagccagaagactgacagcgacggcaatgcttagttacaacattcataat\n",
      "taaaagcgactctgttccggcccttcgggccggggc-----cgcttttcagttatgaggg\n",
      "agg\n",
      ">G000299455|5385343|5385755|5385343|5385461|-\n",
      "ggaggacttccagccagaagactggcagcgatgataatg--tagttacaacattcataat\n",
      "taaaagcgatgttgttcccgccctttgggcgggggcggcctcgcttttcagttatgaggg\n",
      "---\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2569729, 2584886)), (2577102, 2577135), '-'), (('G000006925', (3693106, 3702974)), (3701210, 3701243), '+'), (('G000299455', (269622, 287922)), (276954, 276987), '+'), (('G000299455', (1462739, 1472085)), (1470490, 1470523), '+'), (('G000299455', (3918464, 3964500)), (3933331, 3933364), '-')]\n",
      ">G000006925|2569729|2584886|2577102|2577135|-\n",
      "gtggcacacagagccggagtgaggagcatcgct-\n",
      ">G000006925|3693106|3702974|3701210|3701243|+\n",
      "-tggcgcacagggcaggggtcaggctcattgcca\n",
      ">G000299455|269622|287922|276954|276987|+\n",
      "-tggcgcacagggcaggggtcaggctcattgcca\n",
      ">G000299455|1462739|1472085|1470490|1470523|+\n",
      "-tggcacacagagccggagtgaggatcatcgcta\n",
      ">G000299455|3918464|3964500|3933331|3933364|-\n",
      "gtggcacaaagagccggagtcaggatcaacgct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3146034, 3178603)), (3149117, 3149290), '-'), (('G000299455', (698804, 702950)), (701450, 701623), '+'), (('G000299455', (781497, 800022)), (796989, 797162), '+')]\n",
      ">G000006925|3146034|3178603|3149117|3149290|-\n",
      "ggtattccctaaccagctcccctgaatataacttagccagcagcccagacttgctgcggc\n",
      "ggtcagaatcgcaatcgtaggcagaaaatccataaccccttgggcaatcaatgcgcctgc\n",
      "cagtatcaacaagctgtcgcctggcaaaaatgaggcgggcagcaggccgtttt-\n",
      ">G000299455|698804|702950|701450|701623|+\n",
      "-gtattgcccagccatcgcccctgaatatagctgacccagcagccgaggctggcggcaac\n",
      "ggtcagcagcagaatcgtttgcgggtaacccatcgcgcctttcgcaatcaacacgccgac\n",
      "caataccagtaaactgtcgcccggtaaaaaggccgctggaagcaagccgttttc\n",
      ">G000299455|781497|800022|796989|797162|+\n",
      "-gtattccctaaccagcgcccctgaatataacttagccagcagcccagacttgctgcggc\n",
      "ggtcagaatcgcaatcgtaggcagaaaatccataaccccctgggcaatcaatgcgcctgc\n",
      "cagtatcaacaagctgtcgcctggcaaaaatgaggcgggcagcaggccgttttc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (281565, 281827)), (281565, 281827), '+'), (('G000299455', (2711456, 2711718)), (2711456, 2711718), '-'), (('G000299455', (3836324, 3836586)), (3836324, 3836586), '+'), (('G000299455', (4187335, 4187597)), (4187335, 4187597), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|281565|281827|281565|281827|+\n",
      "-gcccgcagagcgccatcgtcaccggcccggcggggcaggaaatctggacggataaatat\n",
      "ggccgggtgaaggtgcagtttggctgggaccgttacggaaagaatgacgaaaacagttcc\n",
      "tgctggatccgcgtcagctatccgtgggcgggtaaaggattcggcatgatccagatcccg\n",
      "cgtattggccaggaagtactggtggatttcaaaaacggcgatccggatttgcccatcatc\n",
      "gtggggcgcacctataaccagga\n",
      ">G000299455|2711456|2711718|2711456|2711718|-\n",
      "ggcccgcagagcgctgtcgtgacgggaccggcaggcgaggaaatcttctgtgatgaacat\n",
      "ggccgcgtgcgggtgaaatttaactgggaccgttataacccgtcaaaccaggacagttca\n",
      "tgctggagccgtgtggcacaggcgtgggcaggcaccggattcggtaacctggcgataccg\n",
      "cgtgtgggtcaggaggtgattgtggacttcctcaacggcgatccggaccagccgatcatt\n",
      "ttggggcgcacctaccaccagg-\n",
      ">G000299455|3836324|3836586|3836324|3836586|+\n",
      "-gcccgcagagcgctgtcgtgacaggaccggcaggcgaggaaatcttctgcgacgaacat\n",
      "ggtcgcgtgcgggtgaagttcaactgggaccgttataacccggcagaccaagacagttcg\n",
      "tgctggatccgtgtggcacaggcgtgggcaggcaccggttttggccacctggcgataccg\n",
      "cgtgtgggtcaggaggtgattgtagacttcctcaacggcgatccggaccagccgatcatt\n",
      "atggggcgcacctaccaccagga\n",
      ">G000299455|4187335|4187597|4187335|4187597|-\n",
      "ggcccgcagagcgccgtcgtgacgggaccggcaggcgaggaaatcttctgtgatgaacat\n",
      "ggccgcgtgcgggtgaaatttaactgggaccgttataacccgtcaaaccaggacagttca\n",
      "tgctggatccgtgtggcacaggcgtgggcaggcaccggattcggtaacctggcgataccg\n",
      "cgtgtgggtcaggaggtgattgtggacttcctcaacggcgatccggaccagccgatcatt\n",
      "atggggcgcacctaccaccagg-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (739473, 745780)), (739616, 739643), '+'), (('G000006925', (1921807, 1928114)), (1927944, 1927971), '-'), (('G000299455', (2187515, 2189078)), (2187658, 2187685), '+'), (('G000299455', (2553791, 2553818)), (2553791, 2553818), '+'), (('G000299455', (2902162, 2904160)), (2903990, 2904017), '-'), (('G000299455', (3284561, 3284588)), (3284561, 3284588), '-'), (('G000299455', (3590325, 3591889)), (3591719, 3591746), '-')]\n",
      ">G000006925|739473|745780|739616|739643|+\n",
      "-agaacgaaaaaggtgagccggtcacct\n",
      ">G000006925|1921807|1928114|1927944|1927971|-\n",
      "cagaacgaaaaaggtgagccggtcacc-\n",
      ">G000299455|2187515|2189078|2187658|2187685|+\n",
      "-agaacgaaaaaggtgagccggtcacct\n",
      ">G000299455|2553791|2553818|2553791|2553818|+\n",
      "-agaacgaaaaaggtgagctggtcacct\n",
      ">G000299455|2902162|2904160|2903990|2904017|-\n",
      "cagaacgaaaaaggtgagccggtcacc-\n",
      ">G000299455|3284561|3284588|3284561|3284588|-\n",
      "cagaacgaaaaaggtgaactggtcacc-\n",
      ">G000299455|3590325|3591889|3591719|3591746|-\n",
      "cagaacgaaaaaggtgagccggtcacc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (511335, 520753)), (518682, 518731), '-'), (('G000299455', (2042624, 2042673)), (2042624, 2042673), '+'), (('G000299455', (3808566, 3816764)), (3810117, 3810166), '+')]\n",
      ">G000006925|511335|520753|518682|518731|-\n",
      "tacccctttcggcctgccggtggagccagaggtaaagatgatataagcc-\n",
      ">G000299455|2042624|2042673|2042624|2042673|+\n",
      "-acccctttcggcgtaccggtagagccggaggtgtagataatgtaggccg\n",
      ">G000299455|3808566|3816764|3810117|3810166|+\n",
      "-acccctttcggcctgccggtggagccagaggtaaagatgatataagccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3026952, 3027047), '+'), (('G000299455', (956352, 970820)), (966002, 966097), '-'), (('G000299455', (1078152, 1154001)), (1084003, 1084098), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3016790|3036697|3026952|3027047|+\n",
      "-cttataccgcaccactgtacctgtcagaaattgcgccggaaaaaattcgcggcagtatg\n",
      "atctcgatgtatcagttgatgatcactatcgggatc\n",
      ">G000299455|956352|970820|966002|966097|-\n",
      "tcttataccgcaccgctgtacctctctgaaattgcgccggaaaaaattcgcggcagtatg\n",
      "atctcgatgtatcagttgatgatcactatcgggat-\n",
      ">G000299455|1078152|1154001|1084003|1084098|+\n",
      "-cttacaccgctcctctgtatctttctgaaatggcaagtgaaaacgttcgcggtaagatg\n",
      "atcagtatgtaccagttgatggtcacactcggcatc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1578298, 1580868)), (1579541, 1579567), '+'), (('G000006925', (1938092, 1938404)), (1938367, 1938393), '+'), (('G000006925', (2062140, 2062463)), (2062415, 2062441), '+'), (('G000299455', (2545759, 2545785)), (2545759, 2545785), '-'), (('G000299455', (2927401, 2931337)), (2928654, 2928680), '+')]\n",
      ">G000006925|1578298|1580868|1579541|1579567|+\n",
      "-cgccagtaagtgttcaccgtcggcgg\n",
      ">G000006925|1938092|1938404|1938367|1938393|+\n",
      "-cgccagtaagtattcaccgtcggcgg\n",
      ">G000006925|2062140|2062463|2062415|2062441|+\n",
      "-cgccagtaggtgttcaccgttggcgg\n",
      ">G000299455|2545759|2545785|2545759|2545785|-\n",
      "tcgccagtaggtattcaccgtcggcg-\n",
      ">G000299455|2927401|2931337|2928654|2928680|+\n",
      "-cgccagtaagtgttcaccgtcggcgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (200779, 200816), '-'), (('G000299455', (4235244, 4293324)), (4241409, 4241446), '+'), (('G000299455', (4788978, 4789015)), (4788978, 4789015), '-')]\n",
      ">G000006925|64491|216702|200779|200816|-\n",
      "tcgagactgtactcatcccagtcaaaaatcacgccgc-\n",
      ">G000299455|4235244|4293324|4241409|4241446|+\n",
      "-cgagactgtactcatcccagtcaaaaatcacgccgca\n",
      ">G000299455|4788978|4789015|4788978|4789015|-\n",
      "tcgagattatatttatcccagtcaaaaataacgccgc-\n",
      "\n",
      "\n",
      "11 [(('G000006925', (386390, 386480)), (386420, 386449), '+'), (('G000006925', (898987, 899077)), (899018, 899047), '-'), (('G000006925', (2059905, 2059995)), (2059935, 2059964), '+'), (('G000006925', (3421218, 3421308)), (3421249, 3421278), '-'), (('G000006925', (3852168, 3852258)), (3852198, 3852227), '+'), (('G000006925', (4360146, 4360236)), (4360176, 4360205), '+'), (('G000006925', (4701524, 4701609)), (4701548, 4701577), '+'), (('G000006925', (4777177, 4777262)), (4777201, 4777230), '+'), (('G000006925', (4780959, 4781049)), (4780990, 4781019), '-'), (('G000299455', (4579385, 4579439)), (4579385, 4579414), '-'), (('G000299455', (5343755, 5343840)), (5343786, 5343815), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 11\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 10  f\b\b\r",
      "STEP     2 / 10  f\b\b\r",
      "STEP     3 / 10  f\b\b\r",
      "STEP     4 / 10  f\b\b\r",
      "STEP     5 / 10  f\b\b\r",
      "STEP     6 / 10  f\b\b\r",
      "STEP     7 / 10  f\b\b\r",
      "STEP     8 / 10  f\b\b\r",
      "STEP     9 / 10  f\b\b\r",
      "STEP    10 / 10  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1074\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|386390|386480|386420|386449|+\n",
      "-tatcgtgttctctggcgatccgggcaaca\n",
      ">G000006925|898987|899077|899018|899047|-\n",
      "atatcgtgttctctggcgatccgggcaac-\n",
      ">G000006925|2059905|2059995|2059935|2059964|+\n",
      "-tatcgtgttctctggcgatccgggcaaca\n",
      ">G000006925|3421218|3421308|3421249|3421278|-\n",
      "atatcgtgttctctggcgatccgggcaac-\n",
      ">G000006925|3852168|3852258|3852198|3852227|+\n",
      "-tatcgtgttctctggcgatccgggcaaca\n",
      ">G000006925|4360146|4360236|4360176|4360205|+\n",
      "-tatcgtgttctctggcgatccgggcaaca\n",
      ">G000006925|4701524|4701609|4701548|4701577|+\n",
      "atatcgtgttctctggcgatccgggcaac-\n",
      ">G000006925|4777177|4777262|4777201|4777230|+\n",
      "atatcgtgttctctggcgatccgggcaac-\n",
      ">G000006925|4780959|4781049|4780990|4781019|-\n",
      "atatcgtgttctctggcgatccgggcaac-\n",
      ">G000299455|4579385|4579439|4579385|4579414|-\n",
      "atatcatgctcccgggcaatacgggcaac-\n",
      ">G000299455|5343755|5343840|5343786|5343815|-\n",
      "atatcgtgctcccgggcaatacgggcaac-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1306887, 1319577)), (1317437, 1317473), '-'), (('G000006925', (3423912, 3503727)), (3459785, 3459821), '-'), (('G000299455', (434155, 488092)), (478038, 478074), '+'), (('G000299455', (2883599, 2892315)), (2885703, 2885739), '+')]\n",
      ">G000006925|1306887|1319577|1317437|1317473|-\n",
      "tgcccattattggcatttgcctcggacatcaggcaa-\n",
      ">G000006925|3423912|3503727|3459785|3459821|-\n",
      "tgccgattcttggcgtctgcctcggtcatcaggcaa-\n",
      ">G000299455|434155|488092|478038|478074|+\n",
      "-gccgattcttggcgtctgcctcggtcatcaggcaat\n",
      ">G000299455|2883599|2892315|2885703|2885739|+\n",
      "-gcccattattggcatttgcctcggacatcaggcaat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3969227, 3993100)), (3990938, 3991017), '-'), (('G000299455', (3844542, 3852289)), (3851758, 3851837), '+'), (('G000299455', (5156132, 5190970)), (5169179, 5169258), '+')]\n",
      ">G000006925|3969227|3993100|3990938|3991017|-\n",
      "aaatggcgcgcccgacaggattcgaacctgagacctctgcctccggagggcagcgctcta\n",
      "tccagctgagctacgggcg-\n",
      ">G000299455|3844542|3852289|3851758|3851837|+\n",
      "-aatggcgcgccctgcaggattcgaacctgcggcccacgacttagaaggtcgttgctcta\n",
      "tccaactgagctaagggcgc\n",
      ">G000299455|5156132|5190970|5169179|5169258|+\n",
      "-aatggcgcgcccgacaggattcgaacctgagacctctgcctccggagggcagcgctcta\n",
      "tccagctgagctacgggcgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3180744, 3203996)), (3183293, 3183461), '-'), (('G000006925', (3180744, 3203996)), (3184044, 3184212), '-'), (('G000299455', (747195, 760809)), (759697, 759865), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3180744|3203996|3183293|3183461|-\n",
      "ccctttgctgactccccgattgttgatgagcgccggacctcgcgcaacgctttcccagag\n",
      "gcagttttgccttccccgagcggtcacgcaaaacgttgcaacgacctgatcatttgcagt\n",
      "ttttagcggaacgcagatataaaaaatccccggctttttgtgccaggg-\n",
      ">G000006925|3180744|3203996|3184044|3184212|-\n",
      "ccccttgctgactccccgattgttgatgagcgccggaccccgcgcaacgctttccctgaa\n",
      "gcagttttgccttccccgagcggtcacgcaaaacgctgcaacgtcctgagcatttgcacc\n",
      "ttttagcggaacgcagatataaaaaatccccggctttttgggccaggg-\n",
      ">G000299455|747195|760809|759697|759865|+\n",
      "-cctttgctgactccccgattgttgatgagcgccggacctcgcgcaacgctttcccagag\n",
      "gcagttttgccttccccgagcggtcacgcaaaacgttgcaacgacctgatcatttgcagt\n",
      "ttttagcggaacgcagatataaaaaatccccggctttttgtgccaggga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (25217, 64488)), (63968, 64090), '-'), (('G000299455', (183085, 183661)), (183271, 183393), '-'), (('G000299455', (4383450, 4406324)), (4383848, 4383970), '+')]\n",
      ">G000006925|25217|64488|63968|64090|-\n",
      "gcaaaatgaccgacgcagaaatcaacggcgaatatgagtgggaaaccggtaacgtcatcg\n",
      "tagaaacctttgaaaaacagggtatcgatgcagcgcaaatgcccggcgttctggtccatt\n",
      "cc-\n",
      ">G000299455|183085|183661|183271|183393|-\n",
      "gacagatgaccgcagaggagattaacggcgaatatgaatatcagaccggcgaagtgatca\n",
      "ttgaaaccttcgaagaacgtggcaggagtccggcacaaatcccggcggtgctggtgcatt\n",
      "ct-\n",
      ">G000299455|4383450|4406324|4383848|4383970|+\n",
      "-caaaatgaccgacgcagaaatcaacggcgaatatgagtgggaaaccggtaacgtcatcg\n",
      "tagaaacctttgaaaaacagggtatcgatgcagcgcaaatgcccggcgttctggtccatt\n",
      "ccc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3423912, 3503727)), (3461256, 3461320), '-'), (('G000299455', (434155, 488092)), (476539, 476603), '+'), (('G000299455', (3856374, 3866784)), (3862239, 3862303), '+')]\n",
      ">G000006925|3423912|3503727|3461256|3461320|-\n",
      "ggtttttataacaacactacctttcaccgcgtcattcctggctttatgattcagggcggc\n",
      "ggtt-\n",
      ">G000299455|434155|488092|476539|476603|+\n",
      "-gtttttataacaacactacctttcaccgcgtcattcctggctttatgattcagggcggc\n",
      "ggttt\n",
      ">G000299455|3856374|3866784|3862239|3862303|+\n",
      "-gtttttacaacaacaccattttccaccgtgttatcaacggctttatgattcagggcggc\n",
      "ggttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1009687, 1025529)), (1025025, 1025529), '-'), (('G000006925', (1025530, 1063042)), (1025530, 1026035), '-'), (('G000299455', (3326617, 3362069)), (3346225, 3346730), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 30\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 30\r",
      "   10 / 30\r",
      "   20 / 30\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 29  f\b\b\r",
      "STEP     2 / 29  f\b\b\r",
      "STEP     3 / 29  f\b\b\r",
      "STEP     4 / 29  f\b\b\r",
      "STEP     5 / 29  f\b\b\r",
      "STEP     6 / 29  f\b\b\r",
      "STEP     7 / 29  f\b\b\r",
      "STEP     8 / 29  f\b\b\r",
      "STEP     9 / 29  f\b\b\r",
      "STEP    10 / 29  f\b\b\r",
      "STEP    11 / 29  f\b\b\r",
      "STEP    12 / 29  f\b\b\r",
      "STEP    13 / 29  f\b\b\r",
      "STEP    14 / 29  f\b\b\r",
      "STEP    15 / 29  f\b\b\r",
      "STEP    16 / 29  f\b\b\r",
      "STEP    17 / 29  f\b\b\r",
      "STEP    18 / 29  f\b\b\r",
      "STEP    19 / 29  f\b\b\r",
      "STEP    20 / 29  f\b\b\r",
      "STEP    21 / 29  f\b\b\r",
      "STEP    22 / 29  f\b\b\r",
      "STEP    23 / 29  f\b\b\r",
      "STEP    24 / 29  f\b\b\r",
      "STEP    25 / 29  f\b\b\r",
      "STEP    26 / 29  f\b\b\r",
      "STEP    27 / 29  f\b\b\r",
      "STEP    28 / 29  d\b\b\r",
      "STEP    29 / 29  d\b\b h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 30\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 30\r",
      "   10 / 30\r",
      "   20 / 30\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 29  f\b\b\r",
      "STEP     2 / 29  f\b\b\r",
      "STEP     3 / 29  f\b\b\r",
      "STEP     4 / 29  f\b\b\r",
      "STEP     5 / 29  f\b\b\r",
      "STEP     6 / 29  f\b\b\r",
      "STEP     7 / 29  f\b\b\r",
      "STEP     8 / 29  f\b\b\r",
      "STEP     9 / 29  f\b\b\r",
      "STEP    10 / 29  f\b\b\r",
      "STEP    11 / 29  f\b\b\r",
      "STEP    12 / 29  f\b\b\r",
      "STEP    13 / 29  f\b\b\r",
      "STEP    14 / 29  f\b\b\r",
      "STEP    15 / 29  f\b\b\r",
      "STEP    16 / 29  f\b\b\r",
      "STEP    17 / 29  f\b\b\r",
      "STEP    18 / 29  f\b\b\r",
      "STEP    19 / 29  f\b\b\r",
      "STEP    20 / 29  f\b\b\r",
      "STEP    21 / 29  f\b\b\r",
      "STEP    22 / 29  f\b\b\r",
      "STEP    23 / 29  f\b\b\r",
      "STEP    24 / 29  f\b\b\r",
      "STEP    25 / 29  f\b\b\r",
      "STEP    26 / 29  f\b\b\r",
      "STEP    27 / 29  f\b\b\r",
      "STEP    28 / 29  f\b\b\r",
      "STEP    29 / 29  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1150\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1150\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1009687|1025529|1025025|1025529|-\n",
      "tttttcgtgggtgcagctcccgccagccattgacatacctcgttaacctgatg-gcacag\n",
      "tcgaccaaatcttctggcgctgccagcaccacttccggtattgggcagatttcataactt\n",
      "tccagtaacgggccttttaaggtgtccgtacagcgtaaatgccagacatggcgtaacccc\n",
      "gtggcgttgatatagctctcgccatagccaatggtacgaatttgaatatttcccggccca\n",
      "ttgagacgcgagagaaaaagccgatcagcctcgctgatgggtaattgcgtcagattaatg\n",
      "ctgtccggctgcgcgtcggggttacgtacatgtgccagtaactcatgcgccaacggtagg\n",
      "ccattcatcaggccatcaatgggcggcggtaacagcgaatctgtcggcaagagattttgc\n",
      "gttgccgcctgccacaacgccagcggcgcgcagccagcttccagtttgtcctccagcaac\n",
      "ttttcgccgcgacgtctgcgcaccc-\n",
      ">G000006925|1025530|1063042|1025530|1026035|-\n",
      "ttttacgtcggtgcagcttccgccagccactgacatacctcgctaagccgctgcgcagag\n",
      "tcgaccaaatcttctggcgctgccagcaccacttccggtattgggcagatttcataactt\n",
      "tccagtaacgggccttttaaggtgtccgtacagcgtaaatgccagacatggcgtaacccc\n",
      "gtggcgttgatatagctctcgccatagccaatggtacgaatctgaatatttcccggccca\n",
      "ttgagacgcgagagaaaaagccgatcagcctcgctgatgggtaattgcgtcagattaatg\n",
      "ctgtgcggctgcgcgtcggggttacgtacatgtgccagtaactcatgcgccaacggtagg\n",
      "ccattcatcaggccatcaatgggcggcggtaacagcgaatctgtcggcaagagattttgc\n",
      "gttgccgcctgccacaacgccagcggcgcgcagccagcctccagtttgtcctccagcaac\n",
      "ttttcgccgcgacgtctgcgcaccc-\n",
      ">G000299455|3326617|3362069|3346225|3346730|+\n",
      "-tttacgtcggtgcagcttccgccagccactgacatacctcgctaagccgctgcgcagag\n",
      "tcgaccaaatcttctggcgctgccagcaccacttccggtatcgggcagatttcataactt\n",
      "tccagtaaagggccttttaaggtgtccgtacagcgtaaatgccagacatggcgtaacccc\n",
      "gtggcgttgatatagctctcgccatagccaatggtacgaatctgaatatttcccggccca\n",
      "cagagacgtgagagaaaaagccgatcagcctcgctgatgggtaattgcgtcagattaatg\n",
      "ctgtgcggctgcgcgtcggggttacgtacatgtgccagtaactcatgcgccaacggtagg\n",
      "ccattcatcaggccatcaatgggcggcggtaacagcgaatctgtcggcaagagattttgc\n",
      "gttgccgcctgccacaacgccagcggcgcacagccagcttccagtttgtcctccagcaac\n",
      "ttttcgccgcgacgtctgcgcacccg\n",
      "\n",
      "\n",
      "30 [(('G000006925', (251358, 252232)), (251688, 251712), '+'), (('G000006925', (315068, 315858)), (315587, 315611), '-'), (('G000006925', (332059, 337870)), (332578, 332602), '-'), (('G000006925', (493019, 493813)), (493538, 493562), '-'), (('G000006925', (509301, 510374)), (509821, 509845), '-'), (('G000006925', (724190, 725876)), (724710, 724734), '-'), (('G000006925', (1094884, 1095679)), (1095403, 1095427), '-'), (('G000006925', (1099363, 1100153)), (1099609, 1099633), '+'), (('G000006925', (1411510, 1412350)), (1412079, 1412103), '-'), (('G000006925', (1883764, 1885448)), (1884284, 1884308), '-'), (('G000006925', (1956633, 1961076)), (1957153, 1957177), '-'), (('G000006925', (2084425, 2085215)), (2084671, 2084695), '+'), (('G000006925', (2236395, 2237421)), (2236877, 2236901), '+'), (('G000006925', (3065287, 3076770)), (3075283, 3075307), '-'), (('G000006925', (3812816, 3813607)), (3813335, 3813359), '-'), (('G000006925', (4639759, 4640549)), (4640278, 4640302), '-'), (('G000006925', (4650084, 4650878)), (4650334, 4650358), '+'), (('G000006925', (4656735, 4658205)), (4657661, 4657685), '+'), (('G000006925', (4673259, 4674053)), (4673509, 4673533), '+'), (('G000006925', (4679888, 4680760)), (4680408, 4680432), '-'), (('G000006925', (4750261, 4751054)), (4750510, 4750534), '+'), (('G000006925', (4753880, 4754990)), (4754720, 4754744), '-'), (('G000006925', (4760730, 4761525)), (4760981, 4761005), '+'), (('G000006925', (4793569, 4794365)), (4793821, 4793845), '+'), (('G000299455', (906289, 909876)), (906541, 906565), '+'), (('G000299455', (912160, 912572)), (912410, 912434), '+'), (('G000299455', (3840886, 3840911)), (3840886, 3840909), '-'), (('G000299455', (4752701, 4766669)), (4752953, 4752977), '+'), (('G000299455', (4771847, 4785261)), (4780232, 4780256), '-'), (('G000299455', (5275700, 5279438)), (5279012, 5279036), '-')]\n",
      ">G000006925|251358|252232|251688|251712|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|315068|315858|315587|315611|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|332059|337870|332578|332602|-\n",
      "-tcatcaccgctgaacgtcagcgtc\n",
      ">G000006925|493019|493813|493538|493562|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|509301|510374|509821|509845|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|724190|725876|724710|724734|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|1094884|1095679|1095403|1095427|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|1099363|1100153|1099609|1099633|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|1411510|1412350|1412079|1412103|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|1883764|1885448|1884284|1884308|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|1956633|1961076|1957153|1957177|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|2084425|2085215|2084671|2084695|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|2236395|2237421|2236877|2236901|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|3065287|3076770|3075283|3075307|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|3812816|3813607|3813335|3813359|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4639759|4640549|4640278|4640302|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4650084|4650878|4650334|4650358|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4656735|4658205|4657661|4657685|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4673259|4674053|4673509|4673533|+\n",
      "-tcatcaccgctgaacgtcagcgtc\n",
      ">G000006925|4679888|4680760|4680408|4680432|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000006925|4750261|4751054|4750510|4750534|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4753880|4754990|4754720|4754744|-\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4760730|4761525|4760981|4761005|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000006925|4793569|4794365|4793821|4793845|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000299455|906289|909876|906541|906565|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000299455|912160|912572|912410|912434|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000299455|3840886|3840911|3840886|3840909|-\n",
      "--catcacggctgaacgtcagcgtc\n",
      ">G000299455|4752701|4766669|4752953|4752977|+\n",
      "-tcaccaccgctgaacgtcagcgtc\n",
      ">G000299455|4771847|4785261|4780232|4780256|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      ">G000299455|5275700|5279438|5279012|5279036|-\n",
      "ctcaccaccgctgaacgtcagcgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1009687, 1025529)), (1021703, 1021777), '-'), (('G000299455', (800023, 810909)), (808102, 808176), '-'), (('G000299455', (3326617, 3362069)), (3349979, 3350053), '+')]\n",
      ">G000006925|1009687|1025529|1021703|1021777|-\n",
      "atcaggttggcttctggcggcagtttgtactgcgggtgcccccagtagccattgcggaag\n",
      "atccccaactgccc--\n",
      ">G000299455|800023|810909|808102|808176|-\n",
      "atcaggttcacttccggcggcagtttcatcgctgggtgaccccagtagccattagcgaaa\n",
      "atacccaactgacc--\n",
      ">G000299455|3326617|3362069|3349979|3350053|+\n",
      "--caggttggcttctggcggcaacttgtactgcgggtgcccccagtagccattgcggaag\n",
      "atccccaactgcccgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (488535, 488701), '-'), (('G000299455', (315576, 316388)), (315576, 315742), '-'), (('G000299455', (3819953, 3834467)), (3833017, 3833183), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1106\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|487251|491968|488535|488701|-\n",
      "gcgtgggttacatgcttgaggtgccggatggtcagtaagccatttcagcgcccgttttcg\n",
      "ctggcaacccgcctgaccttttttatcagcctggccaccatcgcggcgtttttcgccttt\n",
      "gcatggatcataatccactcagtaaaagtgcattttgccgagcagg-\n",
      ">G000299455|315576|316388|315576|315742|-\n",
      "gcgtgggctacatgctggaggtcccggatgcatagcaaaccttccagacgccctttctca\n",
      "ctcgctctgcggctgaccttttttatcagcctgtccacgatactggcttttatcgcattc\n",
      "acctggtttatgctgcattctgttgaaaatcattttgccgagcagg-\n",
      ">G000299455|3819953|3834467|3833017|3833183|+\n",
      "-cgtgggttacatgcttgaggtgccggatggtcagtaagccatttcagcgcccgttttcg\n",
      "ctggcaacccgcctgaccttttttatcagcctggccaccatcgcagcgtttttcgccttt\n",
      "gcatggatcatgatccactcagtaaaagtgcattttgccgagcagga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1425133, 1465118)), (1445795, 1445844), '+'), (('G000006925', (4249605, 4249654)), (4249605, 4249654), '+'), (('G000299455', (2199778, 2263225)), (2243898, 2243947), '+')]\n",
      ">G000006925|1425133|1465118|1445795|1445844|+\n",
      "caatgcccccatgacaaacaggccgaggatagacgccccttccgtcagt--------\n",
      ">G000006925|4249605|4249654|4249605|4249654|+\n",
      "--------ccatcacaaacagtccgagaatcgacgcgccttcggtgagtttttgcag\n",
      ">G000299455|2199778|2263225|2243898|2243947|+\n",
      "--------ccatgacaaacaggccgaggatagacgccccttccgtcagtttttgcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (907933, 907985)), (907933, 907985), '+'), (('G000006925', (1394855, 1394907)), (1394855, 1394907), '-'), (('G000299455', (2540467, 2540519)), (2540467, 2540519), '-')]\n",
      ">G000006925|907933|907985|907933|907985|+\n",
      "-attcgcgtcttcagttcttcccgctgttcgggagtaaaaggtttcgccataa\n",
      ">G000006925|1394855|1394907|1394855|1394907|-\n",
      "aattcgcgtcttcagttcttcccgctgttcgggagtaaaaggtttcgccata-\n",
      ">G000299455|2540467|2540519|2540467|2540519|-\n",
      "aattcgtgccttcagttcttcccgctgttcgtgtgtaaaaggtttcgccata-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3939882, 3952896)), (3944512, 3944540), '-'), (('G000006925', (4433907, 4437456)), (4436143, 4436171), '+'), (('G000299455', (197629, 211581)), (200354, 200382), '+'), (('G000299455', (4626810, 4646234)), (4644921, 4644949), '+'), (('G000299455', (5202806, 5268469)), (5216667, 5216695), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1087\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1087\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3939882|3952896|3944512|3944540|-\n",
      "acatctacgccacgggtaggctcatcaa-\n",
      ">G000006925|4433907|4437456|4436143|4436171|+\n",
      "-catcaatgccacgggttagctcatcaag\n",
      ">G000299455|197629|211581|200354|200382|+\n",
      "-tatcgatacccctggtaggttcatcaag\n",
      ">G000299455|4626810|4646234|4644921|4644949|+\n",
      "-catcaatgccgcgggttggctcgtcgag\n",
      ">G000299455|5202806|5268469|5216667|5216695|+\n",
      "-catctacgccacgggtaggttcatcaag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1025530, 1063042)), (1056392, 1056434), '+'), (('G000006925', (1306887, 1319577)), (1312306, 1312348), '-'), (('G000299455', (2883599, 2892315)), (2890829, 2890871), '+'), (('G000299455', (3248978, 3256131)), (3255654, 3255696), '-')]\n",
      ">G000006925|1025530|1063042|1056392|1056434|+\n",
      "--tcgaggcggttccggcaattttgcagaagaccgcgaaagagc\n",
      ">G000006925|1306887|1319577|1312306|1312348|-\n",
      "catcgtggtagttcaggaaatttcgccgaagaccgtgagaag--\n",
      ">G000299455|2883599|2892315|2890829|2890871|+\n",
      "-atcgtggtggttcaggaaatttcgccgaagaccgtgagaagg-\n",
      ">G000299455|3248978|3256131|3255654|3255696|-\n",
      "catcgaggcggttccggcaattttgcagaagaccgcgaaaga--\n",
      "\n",
      "\n",
      "12 [(('G000006925', (385776, 386112)), (385776, 385808), '-'), (('G000006925', (899355, 899691)), (899659, 899691), '+'), (('G000006925', (2059291, 2059627)), (2059291, 2059323), '-'), (('G000006925', (3421586, 3421922)), (3421890, 3421922), '+'), (('G000006925', (3851554, 3851890)), (3851554, 3851586), '-'), (('G000006925', (4359551, 4359859)), (4359551, 4359584), '-'), (('G000006925', (4622488, 4624460)), (4623183, 4623215), '+'), (('G000006925', (4689001, 4689359)), (4689322, 4689354), '-'), (('G000006925', (4700925, 4701241)), (4700925, 4700956), '-'), (('G000006925', (4776578, 4776894)), (4776578, 4776609), '-'), (('G000006925', (4781327, 4781663)), (4781631, 4781663), '+'), (('G000299455', (5313369, 5315826)), (5314420, 5314452), '+')]\n",
      ">G000006925|385776|386112|385776|385808|-\n",
      "-gacagaacggcctggcatccggatataacccg-\n",
      ">G000006925|899355|899691|899659|899691|+\n",
      "--acagaacggcctggcatccggatataacccgt\n",
      ">G000006925|2059291|2059627|2059291|2059323|-\n",
      "-gacagaacggcctggcatccggatataacccg-\n",
      ">G000006925|3421586|3421922|3421890|3421922|+\n",
      "--acagaacggcctggtatccggatataacccgt\n",
      ">G000006925|3851554|3851890|3851554|3851586|-\n",
      "-gacagaacggcctggcatccggatataacccg-\n",
      ">G000006925|4359551|4359859|4359551|4359584|-\n",
      "ggacagaacggcctggcatccggatataacccg-\n",
      ">G000006925|4622488|4624460|4623183|4623215|+\n",
      "--acggaacgcgctggcatccgcatataacccgt\n",
      ">G000006925|4689001|4689359|4689322|4689354|-\n",
      "-aacggaacgtgctggaatccgcatataacccg-\n",
      ">G000006925|4700925|4701241|4700925|4700956|-\n",
      "--acagaacggcctggcatccggatataacccg-\n",
      ">G000006925|4776578|4776894|4776578|4776609|-\n",
      "--acagaacggcctggcatccggatataacccg-\n",
      ">G000006925|4781327|4781663|4781631|4781663|+\n",
      "--acagaacggcctggtatccggatataacccgt\n",
      ">G000299455|5313369|5315826|5314420|5314452|+\n",
      "--acggaacgcgctggaatccgcatataacccgt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (729269, 729510), '+'), (('G000299455', (2174641, 2181396)), (2177665, 2177906), '+'), (('G000299455', (2910238, 2916987)), (2913722, 2913963), '-'), (('G000299455', (3601204, 3601445)), (3601204, 3601445), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|729269|729510|+\n",
      "-tgaacctgcagacagcgcaggattcagacaatggattttcggcgcttgaacaggcgctg\n",
      "ctgcggtatattgcagccggtcttggcgtttcctacgaacagttgtcccgtgattactcg\n",
      "aaggtcagttactcaagtgcccgcgcctccgccaatgagtcgtggcgctattttatgggg\n",
      "cggcgaaaatttattgcggcccggctggccacgcagatgttttcctgctggctggaagag\n",
      "gc\n",
      ">G000299455|2174641|2181396|2177665|2177906|+\n",
      "-tgaaacttcagaccgcgcaggattcagacaatggattttcggcgcttgaacaggcgctg\n",
      "ctgcggtatatcgccgccggtcttggcgtttcctacgaacagttgtcccgtgattactcg\n",
      "aaggtcagttattcaagtgcccgcgcctccgccaatgagtcgtggcgctattttatgggg\n",
      "cggcgaaaatttattgcgtcccggctggccacgcagatgttttcctgctggctggaagag\n",
      "gc\n",
      ">G000299455|2910238|2916987|2913722|2913963|-\n",
      "ctgaaacttcagaccgcgcaggattcagacaatggattttcggcgcttgaacaggcgctg\n",
      "ctgcggtatatcgccgccggtcttggcgtttcctacgaacagttgtcccgtgattactcg\n",
      "aaggtcagttattcaagtgcccgcgcatccgccaatgagtcgtggcgctattttatgggg\n",
      "cggcgaaaatttattgcgtcccggctggccacgcagatgttttcctgctggctggaagag\n",
      "g-\n",
      ">G000299455|3601204|3601445|3601204|3601445|-\n",
      "ctgaacctgcagacggctcaggacacggataacggctactccgtgtttgagcagtcactg\n",
      "ttgcggtatattgctgccgggctgggtgtctcgtatgagcagctttcccggaattacgcc\n",
      "cagatgagctactccacggcacgggccagtgcgaacgagtcgtgggcgtactttatgggg\n",
      "cggcgaaaattcgtcgcatcccgtcaggcgagccagatgtttctgtgctggctggaagag\n",
      "g-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3423912, 3503727)), (3457300, 3457483), '-'), (('G000299455', (434155, 488092)), (480376, 480559), '+'), (('G000299455', (4584877, 4585231)), (4584987, 4585166), '+')]\n",
      ">G000006925|3423912|3503727|3457300|3457483|-\n",
      "atggtatcgataagacgtggaaggatgtactgctcaccgttcagccacaacagttgcagg\n",
      "gtatacactgcggtaatagtaaaaccgaccgttgcccagccgtagtttttgcgcaatatc\n",
      "aggtagctggcgagggtggtaatcagcatcaacgtcagggtgtaactttcgggaatttta\n",
      "aag-\n",
      ">G000299455|434155|488092|480376|480559|+\n",
      "-tggtatcgataagacgtggaaggatgtactgctcaccgttcagccacaacagttgcagg\n",
      "gtatacactgcggtaatggtaaaaccgaccgtcgcccagccgtagtttttgcgcaatatc\n",
      "aggtagctggcgagggtggtaatcagcatcaaggtcagggtgtaaccttcgggaatttta\n",
      "aagt\n",
      ">G000299455|4584877|4585231|4584987|4585166|+\n",
      "-tggtgttgataagacacggaaggatgtactgctcgccattcagccataacagttgcagg\n",
      "gtatatacagcagtaatggtaaaaccaaccttca-ccagccgtagtttttgcacaatatc\n",
      "aggcagctggtgaaggtggtaatcagcatcaa---cagggtataaccttcggaaatttta\n",
      "aagt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (300111, 308544)), (306949, 306974), '+'), (('G000299455', (3608677, 3610659)), (3609698, 3609723), '-'), (('G000299455', (4156918, 4166023)), (4160445, 4160470), '-')]\n",
      ">G000006925|300111|308544|306949|306974|+\n",
      "-cagacctgttgcccatgcttctgca\n",
      ">G000299455|3608677|3610659|3609698|3609723|-\n",
      "tcagaccagctgcccatacttcagc-\n",
      ">G000299455|4156918|4166023|4160445|4160470|-\n",
      "tcagacctgtagcccatgcttctgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2069615, 2084300)), (2083558, 2083713), '-'), (('G000299455', (3182222, 3183821)), (3182745, 3182900), '-'), (('G000299455', (4564841, 4566095)), (4565417, 4565572), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2069615|2084300|2083558|2083713|-\n",
      "tcgtgaccgactggcacgtcttcgtaaggagctgaggctacgctgtaagcagaaacgcaa\n",
      "gttcagagcaactacgaacccgaaccacaatctgccagttgcgccaaatctgctgaacca\n",
      "gacgttcgctcctacagcaccaaatcaggtctggg-\n",
      ">G000299455|3182222|3183821|3182745|3182900|-\n",
      "tcgtgaccgactggcacgtcttcgtaaggagctaaggctacgctgtaagcagaaacgcaa\n",
      "gttcagagcgactacgaactcgaaccacaatctgccagttgcgccaaatctgctgaacca\n",
      "gacgttcgctcctacagcaccaaatcaggtctggg-\n",
      ">G000299455|4564841|4566095|4565417|4565572|+\n",
      "-cgtgaccgactggcacgtcttcgtaaggagctaaggctacgctgtaagcagaaacgcaa\n",
      "gttcagagcgactacgaactcgaaccacaatctgccagttgcgccaaatctgctgaacca\n",
      "gacgttcgctcctacagcaccaaatcaggtctgggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4094093, 4116338)), (4096668, 4096703), '+'), (('G000299455', (2682449, 2700553)), (2685956, 2685991), '+'), (('G000299455', (5063005, 5071226)), (5068616, 5068651), '-')]\n",
      ">G000006925|4094093|4116338|4096668|4096703|+\n",
      "-cacggtaccagcgcgggtgatgtttcttcgcccag\n",
      ">G000299455|2682449|2700553|2685956|2685991|+\n",
      "-cacgataccagcgcggatggtgtttcttcgcccag\n",
      ">G000299455|5063005|5071226|5068616|5068651|-\n",
      "tcacggtaccagcgcgggtgatgtttcttcgccca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3099082, 3099230), '+'), (('G000299455', (856582, 857833)), (856606, 856754), '-'), (('G000299455', (4771847, 4785261)), (4784718, 4784866), '+')]\n",
      ">G000006925|3083675|3099254|3099082|3099230|+\n",
      "-aatagccaggtactttccttctgtgctctcagcctgatcccgagagtgattccctttcg\n",
      "atatcaaaagggactgcgggtataattgagggtacatttttaatcaaaccaaacaaattg\n",
      "cattaatatcaatatgataaaagaaattt\n",
      ">G000299455|856582|857833|856606|856754|-\n",
      "taatagccaggtactttccttctgtgctctcagcctgatcccgagtgtgattccctttcg\n",
      "atatcaaaagggactgcgggtataattgagggtacatttttaatcaaaccaaataaatta\n",
      "tattaatatcaatatgataaaagaaatt-\n",
      ">G000299455|4771847|4785261|4784718|4784866|+\n",
      "-aatagccaggtactttccttctgtgctctcagcctgatcccgagagtgattccctttcg\n",
      "atatcaaaagggactgcgggtataattgagggtacatttttaatcaaaccaaacaaattg\n",
      "cattaatatcaatatgataaaagaaattt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3805050, 3805080), '-'), (('G000006925', (4078751, 4078781)), (4078751, 4078781), '-'), (('G000299455', (91523, 102755)), (95299, 95329), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3788320|3808082|3805050|3805080|-\n",
      "ttcgataccgtcgactacggcgagtggtgc-\n",
      ">G000006925|4078751|4078781|4078751|4078781|-\n",
      "tccgacaccgtcgagtacggcgagtggcgc-\n",
      ">G000299455|91523|102755|95299|95329|+\n",
      "-ccgataccgtcgactacggcgagtggtgca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4002899, 4012437)), (4005977, 4006147), '-'), (('G000299455', (5147115, 5155219)), (5155049, 5155219), '+'), (('G000299455', (5156132, 5190970)), (5156132, 5156302), '+')]\n",
      ">G000006925|4002899|4012437|4005977|4006147|-\n",
      "ctttatgaagtctgaaaagtgaggaaatgctgcgagtataacgcttactcgtgaggattt\n",
      "atgttctgctggcagtttcgtgatcaacacctcgcgccaggatggctttatgcactatcg\n",
      "caatgtggtttgtttgttgtcttgttcattgtttttatcgtcggcatggg-\n",
      ">G000299455|5147115|5155219|5155049|5155219|+\n",
      "-tttatgaagtctgaaaagtgaggaaatgctgcgagtataacgcttacacgtggggattt\n",
      "atgttttgctggtagtttcgtgatcggcacctcgcgccaggatggctttatgcactatcg\n",
      "caatgtggtttgtttgttgtcttgttcattgttcttatcgtcggcatgggg\n",
      ">G000299455|5156132|5190970|5156132|5156302|+\n",
      "-tttatagattctgaaaagtgacgatttgccgcaaaagttacgcttacgtgcgaggaatt\n",
      "atgttctgctggtagtttcgtgatcggcacctcgcgccaggatggctttatgcactatcg\n",
      "caatgtggtttgtttgttgtcttgttcattgtttttatcgtcggcatgggg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4367703, 4374506)), (4369181, 4369210), '+'), (('G000299455', (698804, 702950)), (702199, 702228), '+'), (('G000299455', (4524008, 4535561)), (4530236, 4530265), '+')]\n",
      ">G000006925|4367703|4374506|4369181|4369210|+\n",
      "ttttccagatgctgccacattgccagctt\n",
      ">G000299455|698804|702950|702199|702228|+\n",
      "ttttccaggtgctgccacattgccagctt\n",
      ">G000299455|4524008|4535561|4530236|4530265|+\n",
      "ttttccagatgctgccacattgccagctt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (64491, 216702)), (118673, 118741), '+'), (('G000006925', (496417, 503320)), (501284, 501352), '-'), (('G000299455', (3819953, 3834467)), (3824814, 3824882), '+'), (('G000299455', (4300454, 4333834)), (4329765, 4329833), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1138\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1138\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1164\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1164\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|64491|216702|118673|118741|+\n",
      "-tgcgccagactgatcatcgcccagttgatcaccagtgcggataccaccagtgccattaa\n",
      "caggccaaa\n",
      ">G000006925|496417|503320|501284|501352|-\n",
      "atgcgccaggcagatcataatccagttcaacagcaacgttgctaccaccagcgccatcag\n",
      "cagaccaa-\n",
      ">G000299455|3819953|3834467|3824814|3824882|+\n",
      "-tgcgccagacagatcataatccagttcaacagcagcgttgctaccaccagcgccatcag\n",
      "cagaccaaa\n",
      ">G000299455|4300454|4333834|4329765|4329833|-\n",
      "atgcgccaggctaatcatcgcccagttgattaccagtgcagataccaccagcgccattaa\n",
      "cagtccga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3806850, 3806942), '-'), (('G000299455', (91523, 102755)), (93437, 93529), '+'), (('G000299455', (4143646, 4143738)), (4143646, 4143738), '+')]\n",
      ">G000006925|3788320|3808082|3806850|3806942|-\n",
      "cttttctttaccagcaaagcggtacttcatgcgccagtatttagaaccattagtgttaat\n",
      "cagtagatacataccgccgccatcagcaagct-\n",
      ">G000299455|91523|102755|93437|93529|+\n",
      "-ttttctttaccagcaaagcggtacttcatgcgccagtatttagaaccattagtgttaat\n",
      "cagtagatacataccgccgccatcagcaagctt\n",
      ">G000299455|4143646|4143738|4143646|4143738|+\n",
      "-ttctctttaccagctacacgatacttgaagcgccaatattttcctccactaggttttac\n",
      "caagagatacagaccaccaccatcagccagctt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2172359, 2184266)), (2181794, 2181875), '-'), (('G000006925', (3268417, 3281594)), (3269603, 3269684), '+'), (('G000299455', (665003, 670357)), (669090, 669171), '-'), (('G000299455', (1878030, 1889155)), (1880423, 1880504), '+')]\n",
      ">G000006925|2172359|2184266|2181794|2181875|-\n",
      "ctgattgaagcaacgtcaaaccaggtcaatcaatttggcggttataccggaatgacaccg\n",
      "gcagactttcgcgaatttgtt-\n",
      ">G000006925|3268417|3281594|3269603|3269684|+\n",
      "-tgattgaagcaacctccaatcaggtagaccagttcggcggttataccggaatgacgccc\n",
      "gccgattttcgcggctttgttt\n",
      ">G000299455|665003|670357|669090|669171|-\n",
      "ctgattgaagcaacctccaatcaggtagaccagttcggcggttataccggaatgacgccc\n",
      "gccgattttcgcggctttgtt-\n",
      ">G000299455|1878030|1889155|1880423|1880504|+\n",
      "-tgattgaagcaacgtcaaaccaggtcaatcaatttggcggttataccggaatgacaccg\n",
      "gcagactttcgcgaatttgttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3898713, 3912668)), (3898901, 3899022), '+'), (('G000299455', (31186, 43870)), (31374, 31495), '+'), (('G000299455', (5194758, 5202769)), (5200737, 5200858), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3898713|3912668|3898901|3899022|+\n",
      "-cgggtgcgactaacctggtgaccgccattgctgatgcgcggctggactccatcccgctg\n",
      "atttgcatcactggtcaggttcccgcctcgatgatcggcaccgacgccttccaagaagtg\n",
      "ga\n",
      ">G000299455|31186|43870|31374|31495|+\n",
      "-cgggtgcgactaacctggtgaccgccattgccgatgcgcggctggactccatcccgctg\n",
      "atttgcatcaccggtcaggttcccgcctcgatgatcggcaccgacgccttccaggaagtg\n",
      "ga\n",
      ">G000299455|5194758|5202769|5200737|5200858|-\n",
      "ccgggcgcaaccaacctgataaccgggcttgcggacgcactgttagattccatccctgtc\n",
      "gttgccatcaccggtcaagtgtccgcaccttttatcggcaccgacgcatttcaggaagtg\n",
      "g-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1842508, 1869155)), (1862062, 1862095), '+'), (('G000299455', (2786550, 2804122)), (2797019, 2797052), '+'), (('G000299455', (4156918, 4166023)), (4160495, 4160528), '+')]\n",
      ">G000006925|1842508|1869155|1862062|1862095|+\n",
      "agcaaccatgtattcagaaacgcgtaatatgac\n",
      ">G000299455|2786550|2804122|2797019|2797052|+\n",
      "ggcaaccatgtattcagaaacgcgtaatatgac\n",
      ">G000299455|4156918|4166023|4160495|4160528|+\n",
      "ggcaactttttattctgaaatacgcaaaatgac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3815875, 3829465)), (3829063, 3829140), '+'), (('G000299455', (3207505, 3210112)), (3208761, 3208833), '+'), (('G000299455', (4752701, 4766669)), (4756287, 4756364), '-')]\n",
      ">G000006925|3815875|3829465|3829063|3829140|+\n",
      "-gatctgcgcccaatatcctgagccggaactttgctgcaatgaaggccaacgaaaaatgg\n",
      "gggagtatgttatagagt\n",
      ">G000299455|3207505|3210112|3208761|3208833|+\n",
      "-gatctgcgcccaatatcctgagccggaactttg-----acaaaggccaacgaaaaatgg\n",
      "gagcggatgttacagagt\n",
      ">G000299455|4752701|4766669|4756287|4756364|-\n",
      "tgatctgcgcccaatatcctgagccggaactttgctgcaatgaaggccaacgaaaaatgg\n",
      "gggagtatgttatagag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3098003, 3098110), '+'), (('G000299455', (47809, 50193)), (48747, 48854), '-'), (('G000299455', (856582, 857833)), (857726, 857833), '-'), (('G000299455', (3140498, 3142451)), (3141470, 3141577), '-'), (('G000299455', (4771847, 4785261)), (4783639, 4783746), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3098003|3098110|+\n",
      "-ttaaccacagaaacggcgctggatattctgattgcgtggctgcaggacaatattgactg\n",
      "cgaatcagggattatcttcgacaacgatgaggataaaacggattcggc\n",
      ">G000299455|47809|50193|48747|48854|-\n",
      "attaaccacagaaacggcgctggatattctgattgcgtggctgcaggacaatattgactg\n",
      "cgaatcagggattatcttcgacaacgatgaggataaaacggattcgg-\n",
      ">G000299455|856582|857833|857726|857833|-\n",
      "attaaccactgaaacagcgctggatatcctcattgcgtggctgcaggacaatatcgactg\n",
      "cgaatcgggaattatctttgacaacgatgaggataaaacggattcgg-\n",
      ">G000299455|3140498|3142451|3141470|3141577|-\n",
      "attaaccacggaaacagcgctggatattctgattgcgtggctgcaggacaatatcgactg\n",
      "cgaatcggggattatctttgacaacgatgaatataaaacggattcgg-\n",
      ">G000299455|4771847|4785261|4783639|4783746|+\n",
      "-ttaaccacagaaacggcgctggatattctgattgcgtggctgcaggacaatattgactg\n",
      "cgaatcagggattatcttcgacaacgatgaggataaaacggattcggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3738936, 3739110), '-'), (('G000299455', (159106, 161990)), (160249, 160423), '+'), (('G000299455', (977405, 977579)), (977405, 977579), '-')]\n",
      ">G000006925|3736321|3788106|3738936|3739110|-\n",
      "gcggattcatcagcacatacgggaagtagatttcgtggataccacccaggaagtggatga\n",
      "ttgccgcaccgcccgcagactgtttagcgctaccacgaccaaagaacatgtacgccagca\n",
      "gcacgcccatacctggacctgggttagcttcaatcaggaagaagattgatttac-\n",
      ">G000299455|159106|161990|160249|160423|+\n",
      "-cggattcatcagcacatacgggaagtagatttcgtggataccacccaggaagtggatga\n",
      "ttgccgcaccgcccgcagactgtttagcactaccacgaccaaagaacatgtacgccagca\n",
      "gcacgcccatacctggacctgggttagcttcaatcaggaagaagatggatttacc\n",
      ">G000299455|977405|977579|977405|977579|-\n",
      "gcggcttcatcagcacatatgggaaatacagttcgtggatcccaccgaggaagtgaataa\n",
      "tcatcgccccgggcgcagaacgtttactcatccctttaccaaacaaggtaaacgccagca\n",
      "gcagccccaggcccggacctgggttagaggccaccataaagaagatggatttac-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1225684, 1225766)), (1225696, 1225732), '+'), (('G000006925', (4625138, 4625228)), (4625191, 4625228), '-'), (('G000299455', (3007901, 3026172)), (3012681, 3012717), '+')]\n",
      ">G000006925|1225684|1225766|1225696|1225732|+\n",
      "-aatggtttgtcatccataatttaaccttcttctgat\n",
      ">G000006925|4625138|4625228|4625191|4625228|-\n",
      "caactgtttttcattcataaattcaccttcatttgat\n",
      ">G000299455|3007901|3026172|3012681|3012717|+\n",
      "-aattgtttgtcgtccataatttaaccttcatttgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4693409, 4693572)), (4693448, 4693522), '-'), (('G000299455', (883494, 883657)), (883544, 883618), '+'), (('G000299455', (3786605, 3808525)), (3791253, 3791327), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4693409|4693572|4693448|4693522|-\n",
      "attggtacagagctgggaatggaccaggatgaagtgctgaggctgaaacagattagcggg\n",
      "ctgacagagttgtt-\n",
      ">G000299455|883494|883657|883544|883618|+\n",
      "-ttggcactgagctgggtatggaccaggatgaagtactgagactgaaacagattagcggg\n",
      "ctgacggaattgttt\n",
      ">G000299455|3786605|3808525|3791253|3791327|+\n",
      "-ttggcaaagagctgggaatggacagcgacgaagtgctgcgcctgaagcaaattaacggt\n",
      "ctgcaagagctgttt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2117901, 2118205)), (2118032, 2118205), '+'), (('G000006925', (3969227, 3993100)), (3981548, 3981721), '-'), (('G000299455', (5156132, 5190970)), (5178476, 5178649), '+')]\n",
      ">G000006925|2117901|2118205|2118032|2118205|+\n",
      "-ttaccttccagagcattcagaataaccaatggaataagtttttccgggaagtgataagg\n",
      "accataattgttcgagcaattagtcacaatggttggtaaaccataagtccgcttccaggc\n",
      "acgtaccaaatgatcgctggaagctttagacgcagaatacgggctacttggcgc\n",
      ">G000006925|3969227|3993100|3981548|3981721|-\n",
      "tttacccgccagcgcgttgaggatcatcagcgggatcagtttttccggaaagtggtaagg\n",
      "accgtagttattcgagcagttggtgatcagcgtcggcagaccgtaggtccgcagccaggc\n",
      "gcgcaccagatggtcgctgctggcttttgacgcggaataagggctactcggcg-\n",
      ">G000299455|5156132|5190970|5178476|5178649|+\n",
      "-ttacccgccagcgcgttgaggatcatcagcgggatcagtttttccggaaagtggtaagg\n",
      "accgtagttattcgagcagttggtgatcagcgtcggcagaccgtaggtccgcagccaggc\n",
      "gcgcaccagatggtcgctgctggcttttgacgcggaataagggctgctcggcgc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2404231, 2404271), '-'), (('G000299455', (1225017, 1225065)), (1225025, 1225065), '+'), (('G000299455', (1623109, 1682614)), (1676226, 1676266), '+')]\n",
      ">G000006925|2374481|2472008|2404231|2404271|-\n",
      "cgctgcattttctgtggtctgtgcgaagaagcctgtccga-\n",
      ">G000299455|1225017|1225065|1225025|1225065|+\n",
      "-gctgcatcttctgtggacgctgcgaagaagtctgcccgac\n",
      ">G000299455|1623109|1682614|1676226|1676266|+\n",
      "-gctgcattttctgtggtctgtgcgaagaagcctgtccgac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4271042, 4271283), '+'), (('G000299455', (3844542, 3852289)), (3851369, 3851610), '+'), (('G000299455', (4795454, 4823864)), (4809926, 4810167), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4256570|4288375|4271042|4271283|+\n",
      "gatgtatatgttacgtaattgtgtaagtgacagcgtttatcaaattattgaaagcgatat\n",
      "tcacaaagactggaatcttagtatggtagccagttgtttatgtcttagcccaagtctgtt\n",
      "aaagaaaaagttgaaaagcgaaaacaccagttatagccaaataatcaccacctgccgcat\n",
      "gcgttatgccgtaaatgaattaatgatggacggtaaaaatatctctcaggtatcacagtc\n",
      "c\n",
      ">G000299455|3844542|3852289|3851369|3851610|+\n",
      "gatgtatatcttacgcagcagcgtccgcgacagcgtctgccgcattattcaaagcgatat\n",
      "tcagcattactggaatctgcgaattgtcgccagttcgctatgtttaagccccagcctgct\n",
      "caaaaagaaattaaaaaacgaaaataccagctatagccagattgtcacagagtgtcgtat\n",
      "gcgttacgccgtacagatgttattgatggataacaaaaatatcactcaggtggcgcaatt\n",
      "a\n",
      ">G000299455|4795454|4823864|4809926|4810167|+\n",
      "gatgtatatgttacgtaattgtgtaagtgacagcgtttatcaaattattgaaagcgatat\n",
      "tcacaaagactggaatcttagtatggtagccagttgtttatgtcttagcccaagtctgtt\n",
      "aaagaaaaagttgaaaagcgaaaacaccagttatagccaaataatcaccacctgccgcat\n",
      "gcgttatgccgtaaatgaattaatgatggacggtaaaaatatctctcaggtatcacagtc\n",
      "c\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3939882, 3952896)), (3944540, 3944576), '-'), (('G000299455', (197629, 211581)), (200318, 200354), '+'), (('G000299455', (5202806, 5268469)), (5216631, 5216667), '+')]\n",
      ">G000006925|3939882|3952896|3944540|3944576|-\n",
      "tggttaatcagttgatagatctcttttttcgcgccg-\n",
      ">G000299455|197629|211581|200318|200354|+\n",
      "-ggttaattaatttgtagatctcgtatttcgcgccaa\n",
      ">G000299455|5202806|5268469|5216631|5216667|+\n",
      "-ggttaatcagttgatagatctcttttttcgcgccga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2138845, 2144864)), (2142062, 2142243), '-'), (('G000299455', (1911336, 1921536)), (1918137, 1918318), '+'), (('G000299455', (3326617, 3362069)), (3338389, 3338570), '+')]\n",
      ">G000006925|2138845|2144864|2142062|2142243|-\n",
      "acagcctgacggcactatgttttacccgtatatcggcaaggttcacgtagtcgggaaaac\n",
      "gctcgctgaaatccgcagtgatattaccgggcgcttagcgacgtacatcgctgacccgca\n",
      "ggtggacgttaatatcgccgccttccgctcacaaaaggcctatatctcaggtcaggtgaa\n",
      "t-\n",
      ">G000299455|1911336|1921536|1918137|1918318|+\n",
      "acagcctgacggcactatgttttatccgtatatcggcaaggttcacgtagtggggaaaac\n",
      "gctcgctgaaatccgcagtgatattaccgggcgcttagcaacgtacatcgctgacccgca\n",
      "ggtggacgttaatatcgccgccttccgctcacaaaaggcctatatctcaggtcaggtgaa\n",
      "t-\n",
      ">G000299455|3326617|3362069|3338389|3338570|+\n",
      "-aacgccgatggtgcgattttctacccctatattggtcgcctgaaagtggccgggaaaac\n",
      "acttacccaggtacgcaacgagattacagcgcgtctcgattcagtcattgaaagcccgca\n",
      "agtcgatgtcagcgtcgcggcgttccgatcgcaaaaagcgtacgtcaccggcgaagtcag\n",
      "ca\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3086439, 3087143), '-'), (('G000299455', (56338, 57307)), (56338, 57043), '+'), (('G000299455', (857924, 871846)), (868367, 869071), '+'), (('G000299455', (1986939, 1987907)), (1986940, 1987643), '+'), (('G000299455', (3152907, 3154652)), (3153687, 3154391), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1222\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1222\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3086439|3087143|-\n",
      "gttcagaagaagatccagttccataccgcgcgggcaacagagaccacggtgtcacgaaca\n",
      "gcgcgcagggccgtacggacaacagaggcaacacagacgctctccgctgtgtcaaatatc\n",
      "cggtccaccgcaccggtaaactgttcacgggcctgagagcggacagactccgtgcgcagc\n",
      "tcgtcctgcagtcgggtcatcagggggctggcggcatggtcgggaagcgctgtcatgagc\n",
      "gcactgaccagcgtatccagttcccagccggtgcgggccgatacggcaacaaccggatgt\n",
      "acgggccggaacagacggaataccgcctccgttttctcacgaatgttctgttcctgtgcg\n",
      "ggagaaggctgaatacctgccatatcccattcatggcagggttccgttttgtcagcctgc\n",
      "gtcaccacaaacagcacctgctgatgtccgcactgcaggatgtgtcgccagaaatactca\n",
      "tccacagacagggcccggtcatcggctttaatcagccacagcaccaggtccagttcaggc\n",
      "agaatgtcacggtacagggcttcatactctgcatccctgtcccggctctcgcccacctcg\n",
      "ggcaggtcagtgataaccatgctgtgaccgtgaccactcagacggaagcgctgcacttcc\n",
      "cgggtgccggcgtgaacatcactgaccggtgtgacctccccctg--\n",
      ">G000299455|56338|57307|56338|57043|+\n",
      "-ttcagaagaaaatccagttccatacagcgcgggcaacagagaccaccgtgtcacggacg\n",
      "gcacgcagaaccgtacgggcaacagaggcaacacagacgctctccgccgtgtcaaatatc\n",
      "cggtccaccgcaccggtaaactgttcacgggcctgagcgcggacagattccgtgcgcagc\n",
      "tcgtcctgcaggcgggtcatcaggggactggcggcatggtcgggaagtgctgtcattagc\n",
      "gcactgaccagcgtatccagttcccagccggtgcgggccgatacggcaacaaccggatgt\n",
      "acgggccggaacagacggaataccgcctccgttttctcccgaatgttctgtgcctgtgcg\n",
      "ggagaaggctgaatgccggccatatcccattcatggcagggctccgttttgtcggcctgc\n",
      "gtcaccacaaacagcacctgctgatgtccccggtgcaggatgtgtcgccagaaatactca\n",
      "tccacagacagggcacggtcatcggctttaatcagccacagtaccaggtcaagttcaggc\n",
      "agaatgtcacggtacagggcttcatactctgcatccctgtcccggctctcgcccaccccg\n",
      "ggcaggtcagtgataaccatgctgtgaccatgtccactcagacggaagcgccgcacttcc\n",
      "cgggtgccggcgtgaacatcactgaccggggtgacctccccctgaa\n",
      ">G000299455|857924|871846|868367|869071|+\n",
      "-ttcagaagaagatccagttccataccgcgcgggcaacagagaccacggtgtcacgaaca\n",
      "gcgcgcagggccgtacggacaacagaggcaacacagacgctctccgctgtgtcaaatatc\n",
      "cggtccaccgcaccggtaaactgttcacgggcctgagagcggacagactccgtgcgcagc\n",
      "tcgtcctgcagtcgggtcatcagggggctggcggcatggtcgggaagcgctgtcatgagc\n",
      "gcactgaccagcgtatccagttcccagccggtgcgggccgatacggcaacaaccggatgt\n",
      "acgggccggaacagacggaataccgcctccgttttctcacgaatgttctgttcctgtgcg\n",
      "ggagaaggctgaatacctgccatatcccattcatggcagggttccgttttgtcagcctgc\n",
      "gtcaccacaaacagcacctgctgatgtccgcactgcaggatgtgtcgccagaaatactca\n",
      "tccacagacagggcccggtcatcggctttaatcagccacagcaccaggtccagttcaggc\n",
      "agaatgtcacggtacagggcttcatactctgcatccctgtcccggctctcgcccaccccg\n",
      "ggcaggtcagtgataaccatgctgtgaccgtgaccactcagacggaagcgctgcacttcc\n",
      "cgggtgccggcgtgaacatcactgaccggtgtgacctccccctga-\n",
      ">G000299455|1986939|1987907|1986940|1987643|+\n",
      "-ttcagaagaaaatccagttccataccgcgcgggcaacagagaccaccgagtcacggacg\n",
      "gcgcgcagaaccgtgcgtgcaacagaggcaatgcagacgctctccgccgtgtcaaatatc\n",
      "cggtccaccgcaccggtaaactgttcacgggcctgagaccggacagactccgtgcgcagc\n",
      "tcgtcctgcaggcgggtcatcaggggactggcggcatggtcgggaagtgctgtcatgagc\n",
      "gcactgaccagcgtatccagttcccagccggtgcgggccgatacggcaacaacccgatgt\n",
      "acgggccggaacagacggaataccg-ctccgttttttcgcgaatgttctgtgcctgtgcg\n",
      "ggagaaggctgaataccggccatatcccattcatggcagggctccgttttatcggcctgc\n",
      "atcaccacaaacagtacccgctgatgtccccggtgcaggatgtgtcgccagaaatactca\n",
      "tccacagacagggcacggtcatcggctttaatcagccacagcaccaggtccagttcgggc\n",
      "agaatgtcacggtacagggcttcatactctgcatccctgtcccggctctcgcccaccccg\n",
      "ggcaggtcagtgataaccatgctgtgaccgtggccactcagacggaagcgccgcacttcc\n",
      "cgggggccggcgtgaacatcactgaccggggtgacctccccctga-\n",
      ">G000299455|3152907|3154652|3153687|3154391|+\n",
      "-ttcagaagaaaatccagttccataccgcacgggcaacagagaccaccgtgtcacgaaca\n",
      "gcgcgcagggccgtacggacaacagaggcaacacagacgctctccgctgtgtcaaatatc\n",
      "cggtccaccgcaccggtaaactgttcacgggcctgagagcggacagattccgtgcgcagc\n",
      "tcgtcctgcagtcgggtcatcaggggactggcggcatggtcgggaagcgctgtcatgagc\n",
      "gcactgaccagcgtatccagttcccagccggtgcgggccgatacggccacaaccggatgt\n",
      "acgggccggaacagacggaataccgcctccgttttttcgcgaatattctgtgcctgtaca\n",
      "ggagaaggctgaataccggccatatcccattcatggcagggctccgttttgtcggcctgc\n",
      "gtcaccacaaacagcacctgctgatgtccccggtgcaggatgtgtcgccagaaatactca\n",
      "tccacagacagggcacggtcatcggctttaatcagccacagtaccaggtccagttcaggc\n",
      "agaatgtcacggtacagggcttcatactctgcatccctgtcccgactctcgcccactccg\n",
      "ggcaggtcagtgataaccatgctgtgaccgtggccactcagacggaagcgctgcacttcc\n",
      "cgggtgccggcgtgaacatcactgaccggggtgacctccccctga-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (789803, 808424)), (799547, 799657), '-'), (('G000006925', (2069615, 2084300)), (2080415, 2080525), '-'), (('G000299455', (2006434, 2017864)), (2008946, 2009056), '+'), (('G000299455', (3528972, 3554217)), (3531017, 3531127), '+')]\n",
      ">G000006925|789803|808424|799547|799657|-\n",
      "actcagccgctggagtattttgccgcggagtaccagatggggctttccaatatgatggaa\n",
      "gcgaacccgggtgtggataccttcctgccgaaaggcggtactgtactgaa-\n",
      ">G000006925|2069615|2084300|2080415|2080525|-\n",
      "actcagccgctggagacttttgccgcacaatacgggcaagggttaagtaatatgctggaa\n",
      "gcgaatccgggcgctgatgtttttttgccgaagcctggctcacaactcac-\n",
      ">G000299455|2006434|2017864|2008946|2009056|+\n",
      "-cccagccgctggagacttttgccgcacaatacgggcaagggttaagtaacatgctggaa\n",
      "gcgaatccgggcgctgatgtttttttgccgaagcctggctcacaactcacc\n",
      ">G000299455|3528972|3554217|3531017|3531127|+\n",
      "-ctcagccgctggagtattttgccgcggagtaccagatggggctttccaatatgatggaa\n",
      "gcgaacccgggtgtggataccttcctgccgaaaggcggtactgtactgaac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3065287, 3076770)), (3066405, 3066435), '+'), (('G000006925', (4622488, 4624460)), (4623901, 4623931), '+'), (('G000299455', (4749820, 4751622)), (4750934, 4750964), '+'), (('G000299455', (5313369, 5315826)), (5315138, 5315168), '+')]\n",
      ">G000006925|3065287|3076770|3066405|3066435|+\n",
      "gccacactgggtcgctggaccggtgccgtt\n",
      ">G000006925|4622488|4624460|4623901|4623931|+\n",
      "gccacactggggcgctggacaggtgctgtt\n",
      ">G000299455|4749820|4751622|4750934|4750964|+\n",
      "gccacactgggtcgctggaccggtgccgtt\n",
      ">G000299455|5313369|5315826|5315138|5315168|+\n",
      "gccacactggggcgctggacaggtgctgtt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1659262, 1667779)), (1663148, 1664639), '-'), (('G000006925', (4256570, 4288375)), (4261347, 4262838), '+'), (('G000299455', (2495066, 2505927)), (2500550, 2502041), '+'), (('G000299455', (4795454, 4823864)), (4800231, 4801722), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\n",
      "Reallocating..done. *alloclen = 1279\n",
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1659262|1667779|1663148|1664639|-\n",
      "tgaaagtcgcacccgaagcgttaactctgttggcgcgtcaggcgtttcatgatgcgtcgt\n",
      "ttatgctgcgtccggctcaccaacaacaggtggccgacattctgcgtgacccggaggcca\n",
      "gcgaaaatgataaatatgtggcgctgcaattcctgcgtaactccgacatcgcggcgaaag\n",
      "gcgttctgccaacctgtcaggataccggcaccgcgattattgttggtaaaaaagggcagc\n",
      "gtgtatggaccggtggtggtgatgaagcggcgctggcgcgcggtgtctataacacttata\n",
      "tcgaagataatctgcgctactcgcaaaacgcgccgctggatatgtataaagaagtgaata\n",
      "ccggcaccaatctgccagcgcagatcgatctttatgccgttgatggcgacgaatacaaat\n",
      "tcctctgtatcgcaaaaggtggtggttcggcaaacaagacgtatctctatcaggaaacca\n",
      "aagcgttactgacgccggggaaactgaaaaattacctggttgagaagatgcgcacgctgg\n",
      "gtaccgcggcctgtcctccgtatcatattgcgttcgttattggtggaacttctgcagaaa\n",
      "cgaaccttaaaacggtgaaactggcttccgcgaaatactatgatgaactgccaacggaag\n",
      "ggaatgagcacggtcaggcgttccgcgatgtggaactggaaaaagaattactgatcgaag\n",
      "cgcaaaatctcggtctgggtgcgcagtttggtggtaaatacttcgctcacgacatccgcg\n",
      "tgattcgcctgccacgtcacggcgcatcctgtccggttggtatgggtgtttcctgctctg\n",
      "ctgaccgtaatatcaaagcgaagatcaaccgtcaggggatctggatcgaaaaactggaac\n",
      "ataatccaggtaaatatatccctgaagagctgcgtaaagcgggagaaggcgaagcggtgc\n",
      "gcgttgaccttaaccgtccgatgaaagagatcctcgcacagttgtcgcagtatcccgttt\n",
      "ctacacgcttatcgcttaacggcacgattatcgttggtcgtgatattgctcacgccaaac\n",
      "tgaaagagcggatggataacggtgaagggctgccgcaatacattaaagatcatccgattt\n",
      "actacgcgggtccggccaaaacgccggaaggttatgcctccggttctcttggcccaacga\n",
      "ccgccggacggatggattcttatgtcgatcaactgcaagcgcagggcggaagtatgatca\n",
      "tgctggcgaaaggcaaccgcagccagcaggtgacggatgcctgtaaaaaacacggcggct\n",
      "tctaccttggcagtatcggtggtccggccgctgtattggcgcagggaagtattaagagcc\n",
      "tggaatgtgttgaatatccggaactgggaatggaagccatctggaaaattgaagtggaag\n",
      "atttcccggcgtttatccttgtggatgataaaggaaatgacttcttccagc-\n",
      ">G000006925|4256570|4288375|4261347|4262838|+\n",
      "-gaaagtggaaccagaagccctgaccctgctggcgcagcaagcctttcacgacgcttctt\n",
      "tcatgctacgtccggcacaccagaaacaggttgcggctattcttcacgatccagaagcca\n",
      "gcgaaaacgacaagtacgtggcactgcaattcttaagaaactccgaaatcgccgccaaag\n",
      "gcgtgctgccgacctgccaggataccggcaccgcgatcatcgtcggtaaaaaaggccagc\n",
      "gcgtgtggaccggtggcggtgatgaagaagcgctgtcgaaaggcgtctataacacctata\n",
      "tcgaagataacctgcgctactcgcaaaacgcggcgctggacatgtacaaagaggtcaaca\n",
      "ccggcactaacctgcctgcgcaaatcgacctgtacgcggtagatggcgatgagtacaaat\n",
      "tcctttgcgttgcgaaaggcggcggctctgccaacaaaacgtatctctaccaggaaacca\n",
      "aagccctgctgactcccggcaaactgaaaaacttcctcgtcgagaaaatgcgtaccctcg\n",
      "gtactgcagcctgcccgccgtaccatatcgcgtttgtgattggcggtacgtctgcggaaa\n",
      "acaacctgaaaaccgtcaaattagcaagcgctcactattacgatgaactgccgacggaag\n",
      "ggaacgaacatggtcaggcgttccgcgatgtccagctggaacaggaactgctggaagagg\n",
      "cccagaaactcggtcttggcgcgcagtttggcggtaaatacttcgcgcacgacattcgcg\n",
      "ttatccgtctgccacgtcacggcgcatcctgcccggtaggcatgggcgtctcctgctccg\n",
      "ctgaccgtaacattaaagcgaaaatcaaccgcgaaggtatctggatcgaaaaactggaac\n",
      "acaatctaggtcagtacattccacaagaactgcgtcaggccggtgaaggcgaagcggtga\n",
      "aggtcgaccttaaccgcccgatgaaagagatcctcgcccagctttcgcaatatccggttt\n",
      "ccactcgtttgtcgctcactggcaccattattgtgggtcgagatattgcacacgccaagc\n",
      "tgaaagagctgattgacgccggtaaagaacttccgcagtacatcaaagatcacccgatct\n",
      "actacgcgggtccggcgaaaacccctgccggttatccatcaggttcacttggcccaacca\n",
      "ccgcaggtcgtatggactcctatgtggatctgctgcaatcccacggcggcagcatgatca\n",
      "tgctggcgaaaggcaaccgcagccagcaggttaccgacgcgtgtcataaacacggcggct\n",
      "tctacctcggtagcatcggcggtccggcggcggtactggcgcagcagagcatcaagcatc\n",
      "tggaatgcgtcgcttatccggagctgggtatggaagctatctggaaaatcgaagtagaag\n",
      "atttcccggcgtttatcctggtcgatgacaaaggtaacgacttcttccagca\n",
      ">G000299455|2495066|2505927|2500550|2502041|+\n",
      "-gaaagtcgcacccgaagcgttaactctgttggcgcgtcaggcatttcatgatgcgtcat\n",
      "ttatgctgcgtccggctcaccaacaacaggtggccgacattctgcgtgacccggaagcca\n",
      "gcgaaaatgataaatatgtggcgctgcaattcctgcgcaactccgacatcgcggcgaaag\n",
      "gcgttctgccaacctgtcaggataccggcaccgcgattattgttggtaaaaaagggcagc\n",
      "gtgtatggaccggtggcggtgatgaagcggcgctggcacgcggtgtctataacacttata\n",
      "tcgaagataatctgcgctactcgcaaaacgcgccgctggatatgtataaagaggtgaata\n",
      "ccggcaccaatctaccagcgcagatcgatctttatgccgttgatggcgacgagtacaaat\n",
      "tcctctgtatcgccaaaggtggcggttcggctaacaagacgtatctctatcaggaaacca\n",
      "aagcgttactaacgccggggaaactgaaaaattacctggttgagaagatgcgcacgctgg\n",
      "gtacggcggcctgtcctccgtatcatattgcgttcgttattggtggaacttctgcagaaa\n",
      "cgaaccttaaaacggtcaaactggcttccgctaaatactatgatgaactgccaacggaag\n",
      "ggaatgagcacggtcaggcgttccgcgatgtggaactggaaaaagaattgctgatcgaag\n",
      "cgcaaaatcttggtctgggtgcgcagtttggtggtaaatacttcgctcacgacatccgcg\n",
      "tgattcgcctgccacgtcacggcgcatcctgcccggtcggtatgggcgtctcctgctctg\n",
      "ctgaccgtaatatcaaagcgaagatcaaccgtcaggggatctggatcgaaaaactggaac\n",
      "ataacccaggcaaatatatcccggaagagctgcgcaaagcgggagaaggcgaagcggtgt\n",
      "gcgttgaccttaaccgtccgatgaaagagatcctcgcacagttgtcgcagtatcccgttt\n",
      "ctacacgcttatcgcttaacggcacgattatcgtcggtcgtgatattgctcacgccaaac\n",
      "tgaaagagcggatggataacggtgaagggctgccgcagtacattaaagatcatccgattt\n",
      "actacgcgggtccggccaaaacgccggaaggttatgcctccggttctcttggcccaacga\n",
      "ccgccggacggatggattcttatgtcgatcaactgcaagcgcagggcggaagtatgatca\n",
      "tgctggcgaaaggcaaccgcagccagcaggtgacggatgcctgtaaaaaacacggcggct\n",
      "tctaccttggcagtatcggtggtccggccgctgtattggcgcagggaagtattaagagcc\n",
      "tggaatgtgttgaatatccggaactgggaatggaagccatctggaaaattgaagtggaag\n",
      "atttcccggcgtttatccttgtggatgataaaggaaatgacttcttccagca\n",
      ">G000299455|4795454|4823864|4800231|4801722|+\n",
      "-gaaagtggaaccagaagccctgaccctgctggcgcagcaagcctttcacgacgcttctt\n",
      "ttatgctccgcccggcacaccagaaacaggttgcggctattcttcacgatccagaagcca\n",
      "gcgaaaacgacaagtacgtggcgctgcaattcttaagaaactccgaaatcgccgccaaag\n",
      "gcgtgctgccgacctgccaggataccggcaccgcgatcatcgtcggtaaaaaaggccagc\n",
      "gcgtgtggaccggtggcggtgatgaagaagcgctgtcgaaaggcgtctataacacctata\n",
      "tcgaagataacctgcgctattcacagaatgcggcgctggacatgtacaaagaggtcaaca\n",
      "ccggcactaacctgcctgcgcaaatcgacctgtacgcggtagatggcgatgagtacaaat\n",
      "tcctttgcgttgcgaaaggcggcggctctgccaacaaaacgtatctctaccaggaaacca\n",
      "aagccctgctgactcccggcaaactgaaaaacttcctcgtcgagaaaatgcgtaccctcg\n",
      "gtactgcagcctgcccgccgtaccatatcgcgtttgtgattggcggtacgtctgcggaaa\n",
      "ccaacctgaaaaccgtcaagttagcaagcgctcactattacgatgaactgccgacggaag\n",
      "ggaacgaacatggtcaggcgttccgcgatgtccagctggaacaggaactgctggaagagg\n",
      "cccagaaactcggtcttggcgcgcagtttggcggtaaatacttcgcgcacgacattcgcg\n",
      "ttatccgtctgccacgtcacggcgcatcctgcccggtcggcatgggcgtctcctgctccg\n",
      "ctgaccgtaacattaaagcgaaaatcaaccgcgaaggtatctggatcgaaaaactggaac\n",
      "acaacccaggccagtacattccacaagaactgcgccaggccggtgaaggcgaagcggtga\n",
      "aagttgaccttaaccgcccgatgaaagagatcctcgcccagctttcgcaatacccggtat\n",
      "ccactcgtttgtcgctcaccggcaccattatcgtgggccgagatattgcacacgccaagc\n",
      "tgaaagagctgattgacgccggtaaagaacttccgcagtacatcaaagatcacccgatct\n",
      "actacgcgggtccggcgaaaacccctgccggttatccatcaggttcacttggcccaacca\n",
      "ccgcaggtcgtatggactcctacgtggatctgctgcaatcccacggcggcagcatgatca\n",
      "tgctagcgaaaggtaaccgcagtcagcaggttaccgacgcgtgtcataaacacggcggct\n",
      "tctacctcggtagcatcggcggtccggcggcggtactggcgcagcagagcatcaagcatc\n",
      "tggaatgcgtcgcttatccggagctgggtatggaagctatctggaaaatcgaagtagaag\n",
      "atttcccggcgtttatcctggtcgatgacaaaggtaacgacttcttccagca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2065055, 2065094)), (2065064, 2065093), '+'), (('G000299455', (2151341, 2151371)), (2151341, 2151370), '+'), (('G000299455', (2582367, 2582587)), (2582557, 2582586), '+'), (('G000299455', (3311224, 3311253)), (3311224, 3311253), '-')]\n",
      ">G000006925|2065055|2065094|2065064|2065093|+\n",
      "-ggctctgcttccagcgatgcccgtgcaat\n",
      ">G000299455|2151341|2151371|2151341|2151370|+\n",
      "-ggctctgcttccagcgatatcagtgcaat\n",
      ">G000299455|2582367|2582587|2582557|2582586|+\n",
      "-ggctctgcctcaagcgatgccagtgcaat\n",
      ">G000299455|3311224|3311253|3311224|3311253|-\n",
      "cggctctgcttccagcgatgccagcgcaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3265484, 3265598), '-'), (('G000299455', (671363, 678362)), (674059, 674173), '+'), (('G000299455', (1078152, 1154001)), (1144126, 1144240), '+')]\n",
      ">G000006925|3246783|3268347|3265484|3265598|-\n",
      "---------------------------------------------------tcttcctca\n",
      "cctggttcccgatttatctggtgcaggaaaaaggcatgtcgattctgaaagtgggtctgg\n",
      "tcgcctcgattccggcactgtgtggttttgcgggcggcgtgctgg-\n",
      ">G000299455|671363|678362|674059|674173|+\n",
      "tgatgttgggcgtatttttcggacaatattttatcaacaccatcacctggttcttcctca\n",
      "cctggttcccgatttatctggtgcaggaaaaaggcatgtcgattctgaaagtgg------\n",
      "----------------------------------------------\n",
      ">G000299455|1078152|1154001|1144126|1144240|+\n",
      "----------------------------------------------------tttcatta\n",
      "cctggtttccggtttatctggtgcaggcgcgtgggatgtcgattctgaaagcgggctttg\n",
      "tggcttccgttccggcggtttgcggttttatcggcggtgtgctggg\n",
      "\n",
      "\n",
      "53 [(('G000006925', (263407, 264672)), (264433, 264514), '-'), (('G000006925', (311637, 313002)), (312763, 312844), '-'), (('G000006925', (315983, 317248)), (316141, 316222), '+'), (('G000006925', (325529, 326794)), (326555, 326636), '-'), (('G000006925', (330360, 332058)), (331385, 331466), '-'), (('G000006925', (509301, 510374)), (510135, 510216), '-'), (('G000006925', (699183, 700447)), (699341, 699422), '+'), (('G000006925', (712907, 714170)), (713931, 714012), '-'), (('G000006925', (723314, 724189)), (723472, 723553), '+'), (('G000006925', (825843, 827104)), (826000, 826081), '+'), (('G000006925', (903718, 904988)), (903881, 903962), '+'), (('G000006925', (914596, 915860)), (914754, 914835), '+'), (('G000006925', (916783, 918049)), (917810, 917891), '-'), (('G000006925', (921552, 942506)), (932174, 932255), '-'), (('G000006925', (1078088, 1090857)), (1080874, 1080955), '+'), (('G000006925', (1100850, 1102113)), (1101008, 1101089), '+'), (('G000006925', (1290847, 1306886)), (1305781, 1305862), '+'), (('G000006925', (1388754, 1391565)), (1391326, 1391407), '-'), (('G000006925', (1401103, 1402373)), (1401267, 1401348), '+'), (('G000006925', (1413769, 1415032)), (1413927, 1414008), '+'), (('G000006925', (1425133, 1465118)), (1425903, 1425984), '+'), (('G000006925', (1466327, 1467592)), (1466485, 1466566), '+'), (('G000006925', (1468138, 1500238)), (1474899, 1474980), '+'), (('G000006925', (1582509, 1591522)), (1583533, 1583614), '-'), (('G000006925', (1591523, 1592784)), (1591681, 1591762), '+'), (('G000006925', (1828287, 1829620)), (1829311, 1829392), '-'), (('G000006925', (1834428, 1834902)), (1834586, 1834667), '+'), (('G000006925', (1842508, 1869155)), (1842667, 1842748), '+'), (('G000006925', (1878572, 1883763)), (1883046, 1883127), '+'), (('G000006925', (1938560, 1939822)), (1938718, 1938799), '+'), (('G000006925', (1963295, 1964561)), (1963455, 1963536), '+'), (('G000006925', (2057197, 2058461)), (2058222, 2058303), '-'), (('G000006925', (2069615, 2084300)), (2071025, 2071106), '+'), (('G000006925', (2069615, 2084300)), (2084061, 2084142), '-'), (('G000006925', (2199538, 2210687)), (2203886, 2203967), '+'), (('G000006925', (2584954, 2636558)), (2616147, 2616228), '+'), (('G000006925', (2694303, 2695567)), (2694461, 2694542), '+'), (('G000006925', (2816684, 2833035)), (2831835, 2831916), '+'), (('G000006925', (3322761, 3361739)), (3353613, 3353694), '+'), (('G000006925', (3857305, 3858568)), (3857463, 3857544), '+'), (('G000006925', (3898713, 3912668)), (3911563, 3911644), '+'), (('G000006925', (4532444, 4533709)), (4533470, 4533551), '-'), (('G000006925', (4613266, 4614529)), (4614290, 4614371), '-'), (('G000006925', (4651394, 4651981)), (4651742, 4651823), '-'), (('G000006925', (4679888, 4680760)), (4680679, 4680760), '+'), (('G000006925', (4695532, 4696775)), (4696536, 4696617), '-'), (('G000006925', (4704225, 4705397)), (4704383, 4704464), '+'), (('G000006925', (4743489, 4744586)), (4744347, 4744428), '-'), (('G000006925', (4802939, 4805150)), (4804911, 4804992), '-'), (('G000006925', (4813486, 4814755)), (4814511, 4814592), '-'), (('G000006925', (4823511, 4823837)), (4823668, 4823749), '+'), (('G000299455', (2065333, 2065756)), (2065420, 2065501), '-'), (('G000299455', (4564841, 4566095)), (4565005, 4565086), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 53\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 53\r",
      "   10 / 53\r",
      "   20 / 53\r",
      "   30 / 53\r",
      "   40 / 53\r",
      "   50 / 53\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 52  f\b\b\r",
      "STEP     2 / 52  f\b\b\r",
      "STEP     3 / 52  f\b\b\r",
      "STEP     4 / 52  f\b\b\r",
      "STEP     5 / 52  f\b\b\r",
      "STEP     6 / 52  f\b\b\r",
      "STEP     7 / 52  f\b\b\r",
      "STEP     8 / 52  f\b\b\r",
      "STEP     9 / 52  f\b\b\r",
      "STEP    10 / 52  f\b\b\r",
      "STEP    11 / 52  f\b\b\r",
      "STEP    12 / 52  f\b\b\r",
      "STEP    13 / 52  f\b\b\r",
      "STEP    14 / 52  f\b\b\r",
      "STEP    15 / 52  f\b\b\r",
      "STEP    16 / 52  f\b\b\r",
      "STEP    17 / 52  f\b\b\r",
      "STEP    18 / 52  f\b\b\r",
      "STEP    19 / 52  f\b\b\r",
      "STEP    20 / 52  f\b\b\r",
      "STEP    21 / 52  f\b\b\r",
      "STEP    22 / 52  f\b\b\r",
      "STEP    23 / 52  f\b\b\r",
      "STEP    24 / 52  f\b\b\r",
      "STEP    25 / 52  f\b\b\r",
      "STEP    26 / 52  f\b\b\r",
      "STEP    27 / 52  f\b\b\r",
      "STEP    28 / 52  f\b\b\r",
      "STEP    29 / 52  f\b\b\r",
      "STEP    30 / 52  f\b\b\r",
      "STEP    31 / 52  f\b\b\r",
      "STEP    32 / 52  f\b\b\r",
      "STEP    33 / 52  f\b\b\r",
      "STEP    34 / 52  f\b\b\r",
      "STEP    35 / 52  f\b\b\r",
      "STEP    36 / 52  f\b\b\r",
      "STEP    37 / 52  f\b\b\r",
      "STEP    38 / 52  f\b\b\r",
      "STEP    39 / 52  f\b\b\r",
      "STEP    40 / 52  f\b\b\r",
      "STEP    41 / 52  f\b\b\r",
      "STEP    42 / 52  f\b\b\r",
      "STEP    43 / 52  f\b\b\r",
      "STEP    44 / 52  f\b\b\r",
      "STEP    45 / 52  f\b\b\r",
      "STEP    46 / 52  f\b\b\r",
      "STEP    47 / 52  f\b\b\n",
      "Reallocating..done. *alloclen = 1164\n",
      "\r",
      "STEP    48 / 52  f\b\b\r",
      "STEP    49 / 52  f\b\b\r",
      "STEP    50 / 52  f\b\b\r",
      "STEP    51 / 52  f\b\b\r",
      "STEP    52 / 52  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 53\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 53\r",
      "   10 / 53\r",
      "   20 / 53\r",
      "   30 / 53\r",
      "   40 / 53\r",
      "   50 / 53\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 52  f\b\b\r",
      "STEP     2 / 52  f\b\b\r",
      "STEP     3 / 52  f\b\b\r",
      "STEP     4 / 52  f\b\b\r",
      "STEP     5 / 52  f\b\b\r",
      "STEP     6 / 52  f\b\b\r",
      "STEP     7 / 52  f\b\b\r",
      "STEP     8 / 52  f\b\b\r",
      "STEP     9 / 52  f\b\b\r",
      "STEP    10 / 52  f\b\b\r",
      "STEP    11 / 52  f\b\b\r",
      "STEP    12 / 52  f\b\b\r",
      "STEP    13 / 52  f\b\b\r",
      "STEP    14 / 52  f\b\b\r",
      "STEP    15 / 52  f\b\b\r",
      "STEP    16 / 52  f\b\b\r",
      "STEP    17 / 52  f\b\b\r",
      "STEP    18 / 52  f\b\b\r",
      "STEP    19 / 52  f\b\b\r",
      "STEP    20 / 52  f\b\b\r",
      "STEP    21 / 52  f\b\b\r",
      "STEP    22 / 52  f\b\b\r",
      "STEP    23 / 52  f\b\b\r",
      "STEP    24 / 52  f\b\b\r",
      "STEP    25 / 52  f\b\b\r",
      "STEP    26 / 52  f\b\b\r",
      "STEP    27 / 52  f\b\b\r",
      "STEP    28 / 52  f\b\b\r",
      "STEP    29 / 52  f\b\b\r",
      "STEP    30 / 52  f\b\b\r",
      "STEP    31 / 52  f\b\b\r",
      "STEP    32 / 52  f\b\b\r",
      "STEP    33 / 52  f\b\b\r",
      "STEP    34 / 52  f\b\b\r",
      "STEP    35 / 52  f\b\b\r",
      "STEP    36 / 52  f\b\b\r",
      "STEP    37 / 52  f\b\b\r",
      "STEP    38 / 52  f\b\b\r",
      "STEP    39 / 52  f\b\b\r",
      "STEP    40 / 52  f\b\b\r",
      "STEP    41 / 52  f\b\b\r",
      "STEP    42 / 52  f\b\b\r",
      "STEP    43 / 52  f\b\b\r",
      "STEP    44 / 52  f\b\b\r",
      "STEP    45 / 52  f\b\b\r",
      "STEP    46 / 52  f\b\b\r",
      "STEP    47 / 52  f\b\b\r",
      "STEP    48 / 52  f\b\b\r",
      "STEP    49 / 52  f\b\b\r",
      "STEP    50 / 52  f\b\b\r",
      "STEP    51 / 52  f\b\b\r",
      "STEP    52 / 52  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|263407|264672|264433|264514|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|311637|313002|312763|312844|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|315983|317248|316141|316222|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|325529|326794|326555|326636|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|330360|332058|331385|331466|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|509301|510374|510135|510216|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|699183|700447|699341|699422|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|712907|714170|713931|714012|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|723314|724189|723472|723553|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|825843|827104|826000|826081|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|903718|904988|903881|903962|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|914596|915860|914754|914835|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|916783|918049|917810|917891|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|921552|942506|932174|932255|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|1078088|1090857|1080874|1080955|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1100850|1102113|1101008|1101089|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1290847|1306886|1305781|1305862|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1388754|1391565|1391326|1391407|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|1401103|1402373|1401267|1401348|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1413769|1415032|1413927|1414008|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1425133|1465118|1425903|1425984|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1466327|1467592|1466485|1466566|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1468138|1500238|1474899|1474980|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1582509|1591522|1583533|1583614|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|1591523|1592784|1591681|1591762|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1828287|1829620|1829311|1829392|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|1834428|1834902|1834586|1834667|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgagttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1842508|1869155|1842667|1842748|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1878572|1883763|1883046|1883127|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1938560|1939822|1938718|1938799|+\n",
      "-ccgattattccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|1963295|1964561|1963455|1963536|+\n",
      "-ccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|2057197|2058461|2058222|2058303|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|2069615|2084300|2071025|2071106|+\n",
      "-ccgattattccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|2069615|2084300|2084061|2084142|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|2199538|2210687|2203886|2203967|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|2584954|2636558|2616147|2616228|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|2694303|2695567|2694461|2694542|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|2816684|2833035|2831835|2831916|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|3322761|3361739|3353613|3353694|+\n",
      "-ccgattattccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|3857305|3858568|3857463|3857544|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|3898713|3912668|3911563|3911644|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      ">G000006925|4532444|4533709|4533470|4533551|-\n",
      "cccgattatcccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|4613266|4614529|4614290|4614371|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|4651394|4651981|4651742|4651823|-\n",
      "cccgattatctgttcctgaaggcactttaggacaatgggttaccgccgcaagaaaagggc\n",
      "tcaatacaccaggctcccgca-\n",
      ">G000006925|4679888|4680760|4680679|4680760|+\n",
      "-ccgattatctgttcctgaaggcactttaggacaatgggttaccgccgcaagaaaaggac\n",
      "tcaatacaccaggctcccgcac\n",
      ">G000006925|4695532|4696775|4696536|4696617|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|4704225|4705397|4704383|4704464|+\n",
      "-ccgattatctgttcctgaaggcactttaggacaatgggttaccgccgcaagaaaagggc\n",
      "tcaatacaccaggctcccgcac\n",
      ">G000006925|4743489|4744586|4744347|4744428|-\n",
      "cccgattatctgttcctgaaggcactttaggacaatgggttaccgccgcaagaaaagggc\n",
      "tcaatacaccaggctcccgca-\n",
      ">G000006925|4802939|4805150|4804911|4804992|-\n",
      "cccgattattccttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|4813486|4814755|4814511|4814592|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000006925|4823511|4823837|4823668|4823749|+\n",
      "cccgattatctgttcctgaaggcactttaggacaatgggttaccgccgcaagaaaagggc\n",
      "tcaatacaccaggctcccgca-\n",
      ">G000299455|2065333|2065756|2065420|2065501|-\n",
      "cccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgca-\n",
      ">G000299455|4564841|4566095|4565005|4565086|+\n",
      "-ccgattatctcttcctgaaggcactttaggacaatgggttaccgccgccagaaaagggc\n",
      "tcggtactcctggttcccgcac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (338191, 374547)), (371416, 371441), '-'), (('G000006925', (1611778, 1618593)), (1616865, 1616890), '-'), (('G000299455', (2625906, 2635627)), (2630514, 2630539), '+'), (('G000299455', (3973381, 4009751)), (3976486, 3976511), '+')]\n",
      ">G000006925|338191|374547|371416|371441|-\n",
      "atcgggaaaccaccaataccttcgt-\n",
      ">G000006925|1611778|1618593|1616865|1616890|-\n",
      "-tcgggaatccaccaataccttcgta\n",
      ">G000299455|2625906|2635627|2630514|2630539|+\n",
      "-tcgggaatccaccaataccttcgta\n",
      ">G000299455|3973381|4009751|3976486|3976511|+\n",
      "-tcgggaaaccgccaataccttcgta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4117495, 4130655)), (4130254, 4130321), '-'), (('G000299455', (978228, 988794)), (979689, 979756), '+'), (('G000299455', (5034449, 5060698)), (5034783, 5034850), '+')]\n",
      ">G000006925|4117495|4130655|4130254|4130321|-\n",
      "gatattgctgttgatccgattgaaggcacgcgcatgacggcgatgggccaggctaacgcg\n",
      "ctggcgg-\n",
      ">G000299455|978228|988794|979689|979756|+\n",
      "-atatcgcggttgacccgattgaaggcacgcggatggtagcgatggggcagagcaatgcg\n",
      "ctggcggt\n",
      ">G000299455|5034449|5060698|5034783|5034850|+\n",
      "-atattgctgttgatccgattgaaggcacgcgcatgacggcgatgggccaggctaacgcg\n",
      "ctggcggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3423912, 3503727)), (3451522, 3451565), '-'), (('G000299455', (434155, 488092)), (486294, 486337), '+'), (('G000299455', (3362286, 3384700)), (3373729, 3373772), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1122\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3423912|3503727|3451522|3451565|-\n",
      "tttgattaatgtcgatttacctgcgccgttgcggcctaacaga-\n",
      ">G000299455|434155|488092|486294|486337|+\n",
      "-ttgattaatgtcgatttacccgcgccgttgcggcctaacagac\n",
      ">G000299455|3362286|3384700|3373729|3373772|+\n",
      "-ttcattaacgtcgatttgcccgcgccgttgcggcccaccagac\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739026, 739086)), (739026, 739086), '+'), (('G000006925', (1928501, 1928561)), (1928501, 1928561), '-'), (('G000299455', (2187071, 2187131)), (2187071, 2187131), '+'), (('G000299455', (2904544, 2904604)), (2904544, 2904604), '-'), (('G000299455', (3592273, 3592333)), (3592273, 3592333), '-')]\n",
      ">G000006925|739026|739086|739026|739086|+\n",
      "-atcaataacgacggcacgaacgggcagataggtccggctgccctgaaggtggtttatga\n",
      "c\n",
      ">G000006925|1928501|1928561|1928501|1928561|-\n",
      "gatcaataacgacggcacgaacgggcagataggtccggctgccctgaaggtggtttatga\n",
      "-\n",
      ">G000299455|2187071|2187131|2187071|2187131|+\n",
      "-attaacaacgacggcacgaacgggcagataggtccggctgctctgaaggcggtgtatga\n",
      "c\n",
      ">G000299455|2904544|2904604|2904544|2904604|-\n",
      "gattaacaacgacggcacgaacgggcagataggtccggctgctctgaaggcggtgtatga\n",
      "-\n",
      ">G000299455|3592273|3592333|3592273|3592333|-\n",
      "gattaacaacgacggcacgaacgggcagataggtccggctgctctgaaggcggtgtatga\n",
      "-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3082575, 3083578)), (3082607, 3082827), '+'), (('G000299455', (61438, 61772)), (61554, 61772), '-'), (('G000299455', (871858, 872859)), (872609, 872827), '-'), (('G000299455', (1994581, 1994887)), (1994669, 1994887), '-'), (('G000299455', (4539188, 4539522)), (4539304, 4539522), '-')]\n",
      ">G000006925|3082575|3083578|3082607|3082827|+\n",
      "-ttttttacgtttctccccggttgttatgccggtgaaggtggtgcgtcgttttcatcacc\n",
      "aacgccggttgagattaacaacatccggaggaacattctcatgaataccccagtttcact\n",
      "gatggatgaccagatggtcgacatgtcgtttatcactcagctgaccggcctgaccgataa\n",
      "gtggttttataaactcatcaagcatggagcctttccgcccc\n",
      ">G000299455|61438|61772|61554|61772|-\n",
      "attttttacgtttctccccggttgttatgccggtgaaggtggtgcgttgttttcatgacc\n",
      "-acaccggctg-gtttaacaacatccggaggaacattctcatgaccaccccagtttcgct\n",
      "gatggatgaccagatggtcgatatggcgtttatcactcagctgaccggcctgaccgataa\n",
      "gtggttttacaagctcatcaaggatggggcctttccggct-\n",
      ">G000299455|871858|872859|872609|872827|-\n",
      "attttttacgtttctccccagttgttatgccagtgaaggtggtgcgt--ttttcatcacc\n",
      "aacgccggttgagattaacaacatccggaggaacattttcatgaataccccagtttcgct\n",
      "gatggatgaccagatggtcgacatggcgtttatcactcaactgaccggcttaaccgataa\n",
      "gtggttttacaagctcatcaaggatggtgcctttccggcc-\n",
      ">G000299455|1994581|1994887|1994669|1994887|-\n",
      "attttttacgtttctccccggttgttatgccggtgaaggtggcgcgctgttttcatgacc\n",
      "-acaccggctg-gtttaacaacatccggaggaacattctcatggctaccccagtttcgct\n",
      "gatggatgaccagatggtcgacatggcgtttatcactcaactgaccggcctaaccgataa\n",
      "gtggtttgacaaactcatcaaagatgggggctttcctgcg-\n",
      ">G000299455|4539188|4539522|4539304|4539522|-\n",
      "attttttacgtttctccccggttgttatgccggtgaaggtggcgcgctgttttcatgacc\n",
      "-acaccggctg-gtttaacaacatccggaggaacattctcatggctaccccagtttcact\n",
      "gatggatgaccagatggtcgacatggcatttatcactcaactgaccggcctgacagataa\n",
      "gtggttttacaggctcatcagggatggggcctttccggct-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2479150, 2517237)), (2505619, 2505647), '-'), (('G000006925', (4143132, 4163489)), (4155572, 4155600), '-'), (('G000299455', (1535847, 1550961)), (1547445, 1547473), '+'), (('G000299455', (4996216, 5016676)), (5004196, 5004224), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1058\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2479150|2517237|2505619|2505647|-\n",
      "tcggctccaacgatatgacccagtatct-\n",
      ">G000006925|4143132|4163489|4155572|4155600|-\n",
      "ttggtagtaatgacctgacgcaatatct-\n",
      ">G000299455|1535847|1550961|1547445|1547473|+\n",
      "-cggttctaacgatatgacccagtatctg\n",
      ">G000299455|4996216|5016676|5004196|5004224|+\n",
      "-tggtagtaatgacctgacgcaatatctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3053586, 3053669), '-'), (('G000299455', (942831, 956197)), (943183, 943266), '+'), (('G000299455', (4713031, 4748324)), (4736826, 4736909), '-')]\n",
      ">G000006925|3040656|3065083|3053586|3053669|-\n",
      "gtgcacccgtaaagatcagaccattttcaccagcccatgtctgaagttctgccagtatgg\n",
      "ctacaacctgtttgctgagtggc-\n",
      ">G000299455|942831|956197|943183|943266|+\n",
      "-tgcacccgtaaatatcagaccattttcacctgcccatgtctgaagttctgtcagtatgg\n",
      "ctacagcctgtttactgagtggca\n",
      ">G000299455|4713031|4748324|4736826|4736909|-\n",
      "gtgcacccgtaaagatcagaccattttcaccagcccatgtctgaagttctgccagtatgg\n",
      "ctacaacctgtttgctgagtggc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (709671, 710707)), (709770, 709943), '+'), (('G000006925', (722910, 723312)), (723139, 723312), '-'), (('G000299455', (2919336, 2919766)), (2919435, 2919608), '+')]\n",
      ">G000006925|709671|710707|709770|709943|+\n",
      "-cgtcccggataccgagaacgtttcgtccagcaactggagagagttcgacggtggcatca\n",
      "ttgcccatgccggaggcgctggaggtttcggctgaggatggcacagggcattttcctttg\n",
      "acgagcacccgaccaccattatcaagcttgcgccgaagagcatcattttcagct\n",
      ">G000006925|722910|723312|723139|723312|-\n",
      "ccgtcccggataccgagaacgtttcgtccagcaactggagagagttcgacggtggcatca\n",
      "ttgcccatgccggaggcgctggaggtttcggctgaggatggcacagggcattttcctttg\n",
      "acgagcacccgaccaccattatcaagcttgcgccgaagagcatcattttcagc-\n",
      ">G000299455|2919336|2919766|2919435|2919608|+\n",
      "-cgtcccggataccgagaacgtttcgtccagcaactggagagagttcgacggtggcatca\n",
      "ttgcccatgccggaggcgctggaggtttcggctgaggatggcacagggcattttcctttg\n",
      "acgagcaccctgccaccattatcaagcttgcgacgaagagcatcattttcagct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (908032, 908771)), (908414, 908771), '-'), (('G000006925', (1394069, 1394808)), (1394069, 1394426), '+'), (('G000299455', (2538890, 2539629)), (2538890, 2539247), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|908032|908771|908414|908771|-\n",
      "ccgccaggcttcagggggtaacggaccatgaaaaatattgcgacaggcggcgttctggaa\n",
      "cgcatccgccgactgaccccgcaacatgtaaccgccccattcagaacggtagcggagtgg\n",
      "cgcgagtggcaacttgctgaaggccagaaacgtagcgaggagatcaaccgcctgaatcgc\n",
      "cagttgcgggtggaaaaaattctgaatcgctcaggcatccagccgttgcaccgtaaatgc\n",
      "tcgtttgcgaattaccaggtgcagaacgacggccagcgatacgcgttaagccaggcgaaa\n",
      "tccatcgccgatgaactgatgaccgggtgtacaaattttgcgttcagcggaaaacct-\n",
      ">G000006925|1394069|1394808|1394069|1394426|+\n",
      "-cgccaggcttcagggggtaacagaccatgaaaaatattgcgacaggaggcgttctggag\n",
      "cgtatccgcagactgaccccaccatatgtaaccgccccattcagaacggttgcggagtgg\n",
      "cgcgagtggcaacttgctgaaggccagaaacgttgcgaggagatcaaccgcctgaatcgt\n",
      "cagttgcgggtggaaaaaatcctgaaccgctccggcatccagccgttgcaccggaagtgt\n",
      "tcgtttgcgaattaccgggcgcagaacgacggtcagcgacatgcactgagtcaggcgaaa\n",
      "tccatagctgacgaactgatgacaggctgtacgaattttgtgttcagcggtaagcccg\n",
      ">G000299455|2538890|2539629|2538890|2539247|+\n",
      "-cgccgggtttcagggggtaatcagccatgaaaaacatttcgactggagggattcttgaa\n",
      "cgggtgcgccgtctggcaccaccgcacgtggcagcaccgttccggacgaccgacgaatgg\n",
      "cgggaatggcaactggctgagggccgtaagcgcagcgaggaaattaaccgcctgaatcat\n",
      "caggcgcgggttgaaaaaatcctgaaccgtgcgggcatccagccgcttcacaggaagtgc\n",
      "tcattcgggaactaccgggtgcagaacgacggtcagcgccatgctctgagccaggcgaaa\n",
      "tccattgtcgatgaattgatgaccagttatacaaacttcgtgttcagcgggaatccgg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1241906, 1253284)), (1253247, 1253284), '+'), (('G000299455', (77918, 77975)), (77918, 77955), '-'), (('G000299455', (82782, 82839)), (82802, 82839), '+'), (('G000299455', (83942, 83999)), (83942, 83979), '-')]\n",
      ">G000006925|1241906|1253284|1253247|1253284|+\n",
      "-gatcgcacgtaatcacgttattcggggactttatctg\n",
      ">G000299455|77918|77975|77918|77955|-\n",
      "tgatcgcattcaatcacgttgttccggtacttaatct-\n",
      ">G000299455|82782|82839|82802|82839|+\n",
      "-gatcgcattcaatcacgttgttccggtacttaatctg\n",
      ">G000299455|83942|83999|83942|83979|-\n",
      "tgatcgcattcaatcacgttgttccggtacttaatct-\n",
      "\n",
      "\n",
      "7 [(('G000006925', (722910, 723312)), (722984, 723028), '-'), (('G000006925', (901434, 901838)), (901683, 901727), '-'), (('G000299455', (2172426, 2172592)), (2172426, 2172470), '-'), (('G000299455', (2551218, 2551369)), (2551218, 2551262), '-'), (('G000299455', (2919336, 2919766)), (2919719, 2919763), '+'), (('G000299455', (3286869, 3286916)), (3286869, 3286914), '+'), (('G000299455', (3607589, 3607707)), (3607589, 3607634), '+')]\n",
      ">G000006925|722910|723312|722984|723028|-\n",
      "tttgtcgcgctgctctttgtaggtaatggcgttatcacggtaat--\n",
      ">G000006925|901434|901838|901683|901727|-\n",
      "tttgtcgcgctgctctttgtaggtaatggcgttatcacggtaat--\n",
      ">G000299455|2172426|2172592|2172426|2172470|-\n",
      "gttatcgcgctgcgctttgtaggtaatcgcgttatcacggtaat--\n",
      ">G000299455|2551218|2551369|2551218|2551262|-\n",
      "atcatctcgctgctctttgtaggtgatggcgttatcacggtaat--\n",
      ">G000299455|2919336|2919766|2919719|2919763|+\n",
      "-ttgtcgcgctgggctttgtaggtaatggcgttatcacggtaatg-\n",
      ">G000299455|3286869|3286916|3286869|3286914|+\n",
      "-ttatcgcgctgctctttgtaagcgatggcgttatcacggtaatga\n",
      ">G000299455|3607589|3607707|3607589|3607634|+\n",
      "-ttgtcgcgctgttctttgtaggtcatggcgttatcacggtaatga\n",
      "\n",
      "\n",
      "12 [(('G000006925', (1209065, 1211067)), (1210944, 1210970), '-'), (('G000006925', (2756889, 2766574)), (2758025, 2758051), '-'), (('G000006925', (2923039, 2950048)), (2949140, 2949166), '-'), (('G000006925', (3101814, 3103074)), (3102951, 3102977), '-'), (('G000006925', (3810378, 3811029)), (3810475, 3810501), '+'), (('G000006925', (4624461, 4624660)), (4624534, 4624560), '-'), (('G000006925', (4627110, 4627241)), (4627117, 4627143), '-'), (('G000006925', (4628976, 4629562)), (4629438, 4629464), '-'), (('G000006925', (4825967, 4826151)), (4826027, 4826053), '-'), (('G000299455', (3226712, 3229721)), (3229594, 3229620), '-'), (('G000299455', (4548160, 4548254)), (4548227, 4548253), '+'), (('G000299455', (5275700, 5279438)), (5279286, 5279312), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1209065|1211067|1210944|1210970|-\n",
      "cgattgtaatcacattcgatataatt-\n",
      ">G000006925|2756889|2766574|2758025|2758051|-\n",
      "cgattgtaatcacattcgatataatt-\n",
      ">G000006925|2923039|2950048|2949140|2949166|-\n",
      "cgattgtaatcacattcgatataatt-\n",
      ">G000006925|3101814|3103074|3102951|3102977|-\n",
      "cgattgtaatcacattcgatataatt-\n",
      ">G000006925|3810378|3811029|3810475|3810501|+\n",
      "-gattgtaatcacactcgatataatta\n",
      ">G000006925|4624461|4624660|4624534|4624560|-\n",
      "-gattgtaatcgcactcgatataacta\n",
      ">G000006925|4627110|4627241|4627117|4627143|-\n",
      "-gattgtaatcgcacttgatataatta\n",
      ">G000006925|4628976|4629562|4629438|4629464|-\n",
      "-gattgtaatcacattcgatataatta\n",
      ">G000006925|4825967|4826151|4826027|4826053|-\n",
      "-gattgtaattgcactcgatataatta\n",
      ">G000299455|3226712|3229721|3229594|3229620|-\n",
      "cgattgtaatcacattcgatataatt-\n",
      ">G000299455|4548160|4548254|4548227|4548253|+\n",
      "-gattgtaatcgcactcgatataatta\n",
      ">G000299455|5275700|5279438|5279286|5279312|+\n",
      "-gattgtaatcacattcgatataatta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3054193, 3054878), '+'), (('G000006925', (3809691, 3810377)), (3809691, 3810377), '+'), (('G000299455', (4713031, 4748324)), (4737433, 4738118), '+')]\n",
      ">G000006925|3040656|3065083|3054193|3054878|+\n",
      "ttttcggctgctctggtatccccgtccaacacaggtagttattataaacggtaaaaaa-t\n",
      "gtattcactgttatgaaatcaatagtcctgacgtactgtttaatgagcttgtgttctgaa\n",
      "ccttgtacaaccagttcatttgatatcgaattataaacaaaacattgcatccaacttttt\n",
      "caatatggacaatagctgtccatacggtgattttatattcgataaatcaatggatttcat\n",
      "tgagtcacaggactggtaatggatgaaaatgctttagggtttgcctcatactggcgcaac\n",
      "tcgcttgcggatgctgagtcaggaaagggcagttttaaacggaaagacgcccaaaatttc\n",
      "actcactggcatgggatagcggcgggacgtcttgacgaagcgattgtcagtaaatttttt\n",
      "gagggagaaaaagacgatgtcgaaacggtcgatgtcatcttgcgcccaaaagtttatttc\n",
      "cggttactgcagcatggtaaggaccgttccgcaggcgcgcctgatattgttaccccgata\n",
      "gtgacgccagccttgctaagccgtgagggttttttatatccgacgccagcgacctccatt\n",
      "cccagagacctgcttgaacctttgccaaaaggagcattttcgattggtgagattgggcag\n",
      "tatgacaaatacaagacgacccatac\n",
      ">G000006925|3809691|3810377|3809691|3810377|+\n",
      "ttttcggttgaactggtatccccgtccaacacaggcagccattataaacagtaaaaaatt\n",
      "gtattcgctgttatgaaatcaatagtcctgacgtactgttgaatgtgcttgtgttctgaa\n",
      "cctcgtacaaccagtacatttgatatcgaattataaacaaaacattgcatccaacttttt\n",
      "taatatggacaatagttgtccatacggtggttttatattcgataaatcaatggatttcat\n",
      "tgagtcacaggactggtaatggatgaaaatgctttaaggtttgcctcatactggcgcaac\n",
      "tcgcttgcagatgctgagtcaggaaagggctgttttgaacggaaagacgcccaaaatttc\n",
      "actcactggcatgggatagcggcgggacgtcttgacgaagcgattgtcaataaatttttt\n",
      "aagggagaaaaagacgatgtcgaaactgtcaatgtcatcttacgcccaaaagtttatttc\n",
      "cggttactgcagcatggtaaggaccgttccgcaggcacgcctgatattgttacaccgata\n",
      "gtgacgccagccttgctaagccgtgagggttttttatatccgacgccagcgacctccatt\n",
      "cccagagacctgcttgaacctttgccaaaaggagcattttcgattggtgagattgggcag\n",
      "tatgacaaatacaagacgacccatac\n",
      ">G000299455|4713031|4748324|4737433|4738118|+\n",
      "ttttcggctgctctggtatccccgtccaacacaggtagttattataaacggtaaaaaa-t\n",
      "gtattcactgttatgaaatcaatagtcctgacgtactgtttaatgagcttgtgttctgaa\n",
      "ccttgtacaaccagttcatttgatatcgaattataaacaaaacattgcatccaacttttt\n",
      "caatatggacaatagctgtccatacggtgattttatattcgataaatcaatggatttcat\n",
      "tgagtcacaggactggtaatggatgaaaatgctttagggtttgcctcatactggcgcaac\n",
      "tcgcttgcggatgctgagtcaggaaagggcagttttaaacggaaagacgcccaaaatttc\n",
      "actcactggcatgggatagcggcgggacgtcttgacgaagcgattgtcagtaaatttttt\n",
      "gagggagaaaaagacgatgtcgaaacggtcgatgtcatcttgcgcccaaaagtttatttc\n",
      "cggttactgcagcatggtaaggaccgttccgcaggcgcgtctgatattgttaccccgata\n",
      "gtgacgccagccttgctaagccgtgagggttttttatatccgacgccagcgacctccatt\n",
      "cccagagacctgcttgaacctttgccaaaaggagcattttcgattggtgagattgggcag\n",
      "tatgacaaatacaagacgacccatac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4410958, 4411070)), (4411019, 4411070), '+'), (('G000006925', (4411073, 4432480)), (4411073, 4411124), '+'), (('G000299455', (4619870, 4625532)), (4619931, 4619982), '+')]\n",
      ">G000006925|4410958|4411070|4411019|4411070|+\n",
      "ctgcgagttatttaactcctgcgaattatttaacttctgtgaattatttaa\n",
      ">G000006925|4411073|4432480|4411073|4411124|+\n",
      "ctgcgaatcatttaactcctgtatattatttaacttctgatgattatttaa\n",
      ">G000299455|4619870|4625532|4619931|4619982|+\n",
      "ctgcgagttattcaagtcctgagaattatttaactcctgcggaatatttaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3559544, 3600494)), (3593708, 3593750), '-'), (('G000299455', (335504, 377237)), (342306, 342348), '+'), (('G000299455', (3554411, 3565644)), (3557893, 3557935), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3559544|3600494|3593708|3593750|-\n",
      "ggcaggtaaactctccggcgggatgaagcaaaaacttgggct-\n",
      ">G000299455|335504|377237|342306|342348|+\n",
      "-gcaggtaaactctccggcgggatgaagcaaaaacttgggctg\n",
      ">G000299455|3554411|3565644|3557893|3557935|+\n",
      "-gcgggcaagctctccggtgggatgaaacaaaaactcggtctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3898713, 3912668)), (3900387, 3900628), '+'), (('G000299455', (31186, 43870)), (32860, 33101), '+'), (('G000299455', (4574079, 4575225)), (4574249, 4574489), '+')]\n",
      ">G000006925|3898713|3912668|3900387|3900628|+\n",
      "ccatccgggcgtaatgacccacgtttgtggcctttttgcccgccgcgcttttaacgttga\n",
      "aggcattctttgtctgccgattcaggacagcgacaaaagccatatctggctactggtcaa\n",
      "tgacgaccagcgtctggagcagatgataagccaaatcgctaagctggaagatgtcgtgaa\n",
      "agtgcagcgtaatcagtccgatccgacgatgtttaacaagatcgcggtgttttttcagta\n",
      "a\n",
      ">G000299455|31186|43870|32860|33101|+\n",
      "ccatccgggcgtaatgacccacgtttgtggcctttttgcccgccgcgcttttaacgttga\n",
      "aggcattctttgtctgccgattcaggacagcgacaaaagccatatctggctactggtcaa\n",
      "tgacgaccagcgtctggagcagatgataagccaaatcgataagctggaagatgtggtgaa\n",
      "agtgcagcgtaatcagtccgatccgactatgtttaacaagattgcggtgttttttcagta\n",
      "a\n",
      ">G000299455|4574079|4575225|4574249|4574489|+\n",
      "ccatccgggtgtaatgactcacctttgtgggc-ttttgcccgtcgcgctcttaacgttga\n",
      "agacattctctgcctgccgattcaggactgcgacaaaagccatatctggctactggtcaa\n",
      "ggatggccagcgtctggagtaggtgataagccaaatagataagctggaatatgtggtgaa\n",
      "agtgcaacgtaatcagtccaacccgacgatgtttaacaagattgtggtgttttttcagta\n",
      "a\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (727130, 727171), '+'), (('G000299455', (2174641, 2181396)), (2175523, 2175564), '+'), (('G000299455', (2910238, 2916987)), (2916064, 2916105), '-'), (('G000299455', (3603564, 3603605)), (3603564, 3603605), '-')]\n",
      ">G000006925|726235|732999|727130|727171|+\n",
      "-ggacggcgtacagtccgttcaccaccagggtacagatagtc\n",
      ">G000299455|2174641|2181396|2175523|2175564|+\n",
      "-ggacggcgtacagtccgttcaccacctgggtacagattgtc\n",
      ">G000299455|2910238|2916987|2916064|2916105|-\n",
      "tggacggcgtacagtccgttcaccacctgggtacagatagt-\n",
      ">G000299455|3603564|3603605|3603564|3603605|-\n",
      "tggacagcgtacagcccgttcaccacctgggtgcagattgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4026192, 4050087)), (4041017, 4041060), '+'), (('G000299455', (1604015, 1621831)), (1611314, 1611357), '-'), (('G000299455', (5094571, 5134770)), (5119902, 5119945), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1126\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1126\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4026192|4050087|4041017|4041060|+\n",
      "-cgccgttaccgtaccgtttactggatcaaacgccggacgcagc\n",
      ">G000299455|1604015|1621831|1611314|1611357|-\n",
      "ccgccgttaccgttccgtgtttacgatcaaacgccggacgcag-\n",
      ">G000299455|5094571|5134770|5119902|5119945|-\n",
      "ccgccgttaccgtaccgtttactggatcaaacgccggacgcag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1965007, 1976679)), (1968574, 1968636), '+'), (('G000006925', (4557883, 4573403)), (4558932, 4558994), '-'), (('G000299455', (2122044, 2133716)), (2128386, 2128448), '-'), (('G000299455', (2122044, 2133716)), (2130087, 2130149), '-'), (('G000299455', (4487440, 4499283)), (4498172, 4498234), '+')]\n",
      ">G000006925|1965007|1976679|1968574|1968636|+\n",
      "-ggtgtgagtcatggtactgacctgaacaccgcccgcctgcgccgttgtcgcggcatttt\n",
      "ttg\n",
      ">G000006925|4557883|4573403|4558932|4558994|-\n",
      "tagtctgcaccacgttatctaccactttgccgccgcgttgcgccgtttcagaagcactta\n",
      "at-\n",
      ">G000299455|2122044|2133716|2128386|2128448|-\n",
      "tcgttttcactacgccatccaccactttgccgccgtgctgggcggtgtcggaggcacttc\n",
      "gc-\n",
      ">G000299455|2122044|2133716|2130087|2130149|-\n",
      "tggtgtgagtcatggtactgacctgaacaccgcccgcctgcgccgttgtcgcggcatttt\n",
      "tt-\n",
      ">G000299455|4487440|4499283|4498172|4498234|+\n",
      "-ggtctgtactacgttatccaccactttgccgccgcgttgcgccgtttcagaagcactta\n",
      "acg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3065287, 3076770)), (3066874, 3066901), '-'), (('G000006925', (4622488, 4624460)), (4624370, 4624397), '-'), (('G000299455', (3215867, 3216640)), (3216171, 3216198), '-'), (('G000299455', (4749820, 4751622)), (4751403, 4751430), '-'), (('G000299455', (5287974, 5288758)), (5288416, 5288443), '+'), (('G000299455', (5313369, 5315826)), (5315607, 5315634), '-')]\n",
      ">G000006925|3065287|3076770|3066874|3066901|-\n",
      "ttgccgccagacgctgttctgctgaac-\n",
      ">G000006925|4622488|4624460|4624370|4624397|-\n",
      "ttgccgccagacgctgttctgctgaac-\n",
      ">G000299455|3215867|3216640|3216171|3216198|-\n",
      "gttcggcaaggcgctgttccgccgtca-\n",
      ">G000299455|4749820|4751622|4751403|4751430|-\n",
      "ttgccgccagacgctgttctgctgaac-\n",
      ">G000299455|5287974|5288758|5288416|5288443|+\n",
      "-ttcggcaaggcgctgttccgccgtcat\n",
      ">G000299455|5313369|5315826|5315607|5315634|-\n",
      "ttgccgccagacgctgttctgctgaac-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (727454, 727659), '+'), (('G000299455', (2174641, 2181396)), (2175847, 2176052), '+'), (('G000299455', (2910238, 2916987)), (2915576, 2915781), '-'), (('G000299455', (3603076, 3603281)), (3603076, 3603281), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 52\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 52\r",
      "   10 / 52\r",
      "   20 / 52\r",
      "   30 / 52\r",
      "   40 / 52\r",
      "   50 / 52\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 51  f\b\b\r",
      "STEP     2 / 51  f\b\b\r",
      "STEP     3 / 51  f\b\b\r",
      "STEP     4 / 51  f\b\b\r",
      "STEP     5 / 51  f\b\b\r",
      "STEP     6 / 51  f\b\b\r",
      "STEP     7 / 51  f\b\b\r",
      "STEP     8 / 51  f\b\b\r",
      "STEP     9 / 51  f\b\b\r",
      "STEP    10 / 51  f\b\b\r",
      "STEP    11 / 51  f\b\b\r",
      "STEP    12 / 51  f\b\b\r",
      "STEP    13 / 51  f\b\b\r",
      "STEP    14 / 51  f\b\b\r",
      "STEP    15 / 51  f\b\b\r",
      "STEP    16 / 51  f\b\b\r",
      "STEP    17 / 51  f\b\b\r",
      "STEP    18 / 51  f\b\b\r",
      "STEP    19 / 51  f\b\b\r",
      "STEP    20 / 51  f\b\b\r",
      "STEP    21 / 51  f\b\b\r",
      "STEP    22 / 51  f\b\b\r",
      "STEP    23 / 51  f\b\b\r",
      "STEP    24 / 51  f\b\b\r",
      "STEP    25 / 51  f\b\b\r",
      "STEP    26 / 51  f\b\b\r",
      "STEP    27 / 51  f\b\b\r",
      "STEP    28 / 51  f\b\b\r",
      "STEP    29 / 51  f\b\b\r",
      "STEP    30 / 51  f\b\b\r",
      "STEP    31 / 51  f\b\b\r",
      "STEP    32 / 51  f\b\b\r",
      "STEP    33 / 51  f\b\b\r",
      "STEP    34 / 51  f\b\b\r",
      "STEP    35 / 51  f\b\b\r",
      "STEP    36 / 51  f\b\b\r",
      "STEP    37 / 51  f\b\b\r",
      "STEP    38 / 51  f\b\b\r",
      "STEP    39 / 51  f\b\b\r",
      "STEP    40 / 51  f\b\b\r",
      "STEP    41 / 51  f\b\b\r",
      "STEP    42 / 51  f\b\b\r",
      "STEP    43 / 51  f\b\b\r",
      "STEP    44 / 51  f\b\b\r",
      "STEP    45 / 51  f\b\b\r",
      "STEP    46 / 51  f\b\b\r",
      "STEP    47 / 51  f\b\b\n",
      "Reallocating..done. *alloclen = 1150\n",
      "\r",
      "STEP    48 / 51  f\b\b\r",
      "STEP    49 / 51  f\b\b\r",
      "STEP    50 / 51  f\b\b\r",
      "STEP    51 / 51  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 52\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 52\r",
      "   10 / 52\r",
      "   20 / 52\r",
      "   30 / 52\r",
      "   40 / 52\r",
      "   50 / 52\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 51  f\b\b\r",
      "STEP     2 / 51  f\b\b\r",
      "STEP     3 / 51  f\b\b\r",
      "STEP     4 / 51  f\b\b\r",
      "STEP     5 / 51  f\b\b\r",
      "STEP     6 / 51  f\b\b\r",
      "STEP     7 / 51  f\b\b\r",
      "STEP     8 / 51  f\b\b\r",
      "STEP     9 / 51  f\b\b\r",
      "STEP    10 / 51  f\b\b\r",
      "STEP    11 / 51  f\b\b\r",
      "STEP    12 / 51  f\b\b\r",
      "STEP    13 / 51  f\b\b\r",
      "STEP    14 / 51  f\b\b\r",
      "STEP    15 / 51  f\b\b\r",
      "STEP    16 / 51  f\b\b\r",
      "STEP    17 / 51  f\b\b\r",
      "STEP    18 / 51  f\b\b\r",
      "STEP    19 / 51  f\b\b\r",
      "STEP    20 / 51  f\b\b\r",
      "STEP    21 / 51  f\b\b\r",
      "STEP    22 / 51  f\b\b\r",
      "STEP    23 / 51  f\b\b\r",
      "STEP    24 / 51  f\b\b\r",
      "STEP    25 / 51  f\b\b\r",
      "STEP    26 / 51  f\b\b\r",
      "STEP    27 / 51  f\b\b\r",
      "STEP    28 / 51  f\b\b\r",
      "STEP    29 / 51  f\b\b\r",
      "STEP    30 / 51  f\b\b\r",
      "STEP    31 / 51  f\b\b\r",
      "STEP    32 / 51  f\b\b\r",
      "STEP    33 / 51  f\b\b\r",
      "STEP    34 / 51  f\b\b\r",
      "STEP    35 / 51  f\b\b\r",
      "STEP    36 / 51  f\b\b\r",
      "STEP    37 / 51  f\b\b\r",
      "STEP    38 / 51  f\b\b\r",
      "STEP    39 / 51  f\b\b\r",
      "STEP    40 / 51  f\b\b\r",
      "STEP    41 / 51  f\b\b\r",
      "STEP    42 / 51  f\b\b\r",
      "STEP    43 / 51  f\b\b\r",
      "STEP    44 / 51  f\b\b\r",
      "STEP    45 / 51  f\b\b\r",
      "STEP    46 / 51  f\b\b\r",
      "STEP    47 / 51  f\b\b\r",
      "STEP    48 / 51  f\b\b\r",
      "STEP    49 / 51  f\b\b\r",
      "STEP    50 / 51  f\b\b\r",
      "STEP    51 / 51  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|727454|727659|+\n",
      "-aagagacgctgttacgtgtggatgcggcgatcaataaaaaataccgccatgcagacggt\n",
      "accgaaatgaccatttcccgtgtctgctgggatatcggggggatcgatggcgaaatcgtg\n",
      "tatcagaggtcaaaaaaacacggtattttccgggtactgccggtaaaaggtgcctctgtc\n",
      "tatggcaagccggtgatcaccatgcc\n",
      ">G000299455|2174641|2181396|2175847|2176052|+\n",
      "-aagagacgctgttacgtttggatgcggcgatcaacaaaaaataccgccatgcagacgga\n",
      "accgaaatgactatttcccgtgtctgctgggacatcggggggatcgatggcgaaatcgtt\n",
      "tatcagaggtcaaaaaaacacggtgttttccgggtgctgccggtaaaaggcgcatctgtc\n",
      "tatggcaagccggtgatcaccatgcc\n",
      ">G000299455|2910238|2916987|2915576|2915781|-\n",
      "gaagagacgctgttacgtgtggatgcggcgatcaacaaaaaatactgccatgcagacgga\n",
      "accgaaatgaccatttcccgtgtctgctgggacaccggggggatcgatggtgaaattgtc\n",
      "tatcagaggtcaaaaaaacacggtgttttccgggtgctgccggtaaaaggcgcatctgtc\n",
      "tatggcaagccggtgatcaccatgc-\n",
      ">G000299455|3603076|3603281|3603076|3603281|-\n",
      "gaacagacgctgctgcgtgtggatgaggccatcaataaaacctatacccgccggaatggt\n",
      "gcagaaatgtcgatatcccgtatctgctgggatactggcgggattgacccgaccattgtg\n",
      "tatgaacgctcgaaaaaacatgggctgttccgggtgatccccattaaaggggcatccgtc\n",
      "tacggaaagccggtggccagcatgc-\n",
      "\n",
      "\n",
      "52 [(('G000006925', (263407, 264672)), (264514, 264588), '-'), (('G000006925', (311637, 313002)), (312844, 312918), '-'), (('G000006925', (315983, 317248)), (316067, 316141), '+'), (('G000006925', (325529, 326794)), (326636, 326710), '-'), (('G000006925', (330360, 332058)), (331466, 331540), '-'), (('G000006925', (509301, 510374)), (510216, 510290), '-'), (('G000006925', (699183, 700447)), (699267, 699341), '+'), (('G000006925', (712907, 714170)), (714012, 714086), '-'), (('G000006925', (723314, 724189)), (723398, 723472), '+'), (('G000006925', (825843, 827104)), (825926, 826000), '+'), (('G000006925', (903718, 904988)), (903807, 903881), '+'), (('G000006925', (914596, 915860)), (914680, 914754), '+'), (('G000006925', (916783, 918049)), (917891, 917965), '-'), (('G000006925', (921552, 942506)), (932255, 932329), '-'), (('G000006925', (1078088, 1090857)), (1080800, 1080874), '+'), (('G000006925', (1100850, 1102113)), (1100934, 1101008), '+'), (('G000006925', (1290847, 1306886)), (1305707, 1305781), '+'), (('G000006925', (1388754, 1391565)), (1391407, 1391481), '-'), (('G000006925', (1401103, 1402373)), (1401193, 1401267), '+'), (('G000006925', (1413769, 1415032)), (1413853, 1413927), '+'), (('G000006925', (1425133, 1465118)), (1425829, 1425903), '+'), (('G000006925', (1466327, 1467592)), (1466411, 1466485), '+'), (('G000006925', (1468138, 1500238)), (1474825, 1474899), '+'), (('G000006925', (1582509, 1591522)), (1583614, 1583688), '-'), (('G000006925', (1591523, 1592784)), (1591607, 1591681), '+'), (('G000006925', (1828287, 1829620)), (1829392, 1829466), '-'), (('G000006925', (1834428, 1834902)), (1834512, 1834586), '+'), (('G000006925', (1842508, 1869155)), (1842593, 1842667), '+'), (('G000006925', (1878572, 1883763)), (1882972, 1883046), '+'), (('G000006925', (1938560, 1939822)), (1938644, 1938718), '+'), (('G000006925', (1963295, 1964561)), (1963381, 1963455), '+'), (('G000006925', (2057197, 2058461)), (2058303, 2058377), '-'), (('G000006925', (2069615, 2084300)), (2070951, 2071025), '+'), (('G000006925', (2069615, 2084300)), (2084142, 2084216), '-'), (('G000006925', (2199538, 2210687)), (2203812, 2203886), '+'), (('G000006925', (2584954, 2636558)), (2616073, 2616147), '+'), (('G000006925', (2694303, 2695567)), (2694387, 2694461), '+'), (('G000006925', (2816684, 2833035)), (2831761, 2831835), '+'), (('G000006925', (3322761, 3361739)), (3353539, 3353613), '+'), (('G000006925', (3857305, 3858568)), (3857389, 3857463), '+'), (('G000006925', (3898713, 3912668)), (3911489, 3911563), '+'), (('G000006925', (4532444, 4533709)), (4533551, 4533625), '-'), (('G000006925', (4613266, 4614529)), (4614371, 4614445), '-'), (('G000006925', (4651394, 4651981)), (4651823, 4651897), '-'), (('G000006925', (4695532, 4696775)), (4696617, 4696691), '-'), (('G000006925', (4704225, 4705397)), (4704309, 4704383), '+'), (('G000006925', (4743489, 4744586)), (4744428, 4744502), '-'), (('G000006925', (4802939, 4805150)), (4804992, 4805066), '-'), (('G000006925', (4813486, 4814755)), (4814592, 4814666), '-'), (('G000006925', (4823511, 4823837)), (4823594, 4823668), '+'), (('G000299455', (2065333, 2065756)), (2065501, 2065575), '-'), (('G000299455', (4564841, 4566095)), (4564931, 4565005), '+')]\n",
      ">G000006925|263407|264672|264514|264588|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|311637|313002|312844|312918|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|315983|317248|316067|316141|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|325529|326794|326636|326710|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|330360|332058|331466|331540|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|509301|510374|510216|510290|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|699183|700447|699267|699341|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|712907|714170|714012|714086|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|723314|724189|723398|723472|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|825843|827104|825926|826000|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|903718|904988|903807|903881|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|914596|915860|914680|914754|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|916783|918049|917891|917965|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|921552|942506|932255|932329|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|1078088|1090857|1080800|1080874|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1100850|1102113|1100934|1101008|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1290847|1306886|1305707|1305781|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1388754|1391565|1391407|1391481|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcaat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|1401103|1402373|1401193|1401267|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1413769|1415032|1413853|1413927|+\n",
      "-tactctaaatagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1425133|1465118|1425829|1425903|+\n",
      "-tactctaaatagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1466327|1467592|1466411|1466485|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1468138|1500238|1474825|1474899|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1582509|1591522|1583614|1583688|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|1591523|1592784|1591607|1591681|+\n",
      "-tactctaaatagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1828287|1829620|1829392|1829466|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|1834428|1834902|1834512|1834586|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1842508|1869155|1842593|1842667|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1878572|1883763|1882972|1883046|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|1938560|1939822|1938644|1938718|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaagacgcttc\n",
      ">G000006925|1963295|1964561|1963381|1963455|+\n",
      "-tactctaaatagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|2057197|2058461|2058303|2058377|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|2069615|2084300|2070951|2071025|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaagacgcttc\n",
      ">G000006925|2069615|2084300|2084142|2084216|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|2199538|2210687|2203812|2203886|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcctc\n",
      ">G000006925|2584954|2636558|2616073|2616147|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|2694303|2695567|2694387|2694461|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|2816684|2833035|2831761|2831835|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|3322761|3361739|3353539|3353613|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaagacgcttc\n",
      ">G000006925|3857305|3858568|3857389|3857463|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|3898713|3912668|3911489|3911563|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|4532444|4533709|4533551|4533625|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcaat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|4613266|4614529|4614371|4614445|-\n",
      "ttactctaaatagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|4651394|4651981|4651823|4651897|-\n",
      "ttactctacggagtttaaagccgaagccgtcaaaacggttcctgaaaatcagctttcgat\n",
      "cagtgaaggagctt-\n",
      ">G000006925|4695532|4696775|4696617|4696691|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|4704225|4705397|4704309|4704383|+\n",
      "-tactctacggagtttaaagccgaagccgtcaaaacggttcctgaaaatcagctttcgat\n",
      "cagtgaaggcgcttc\n",
      ">G000006925|4743489|4744586|4744428|4744502|-\n",
      "ttactctacggagtttaaagccgaagccgtcaaaacggttcctgaaaatcagctttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|4802939|4805150|4804992|4805066|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaagacgctt-\n",
      ">G000006925|4813486|4814755|4814592|4814666|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000006925|4823511|4823837|4823594|4823668|+\n",
      "ttactctacggagtttaaagccgaagccgtcaaaacggttcctgaaaatcagctttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000299455|2065333|2065756|2065501|2065575|-\n",
      "ttactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgctt-\n",
      ">G000299455|4564841|4566095|4564931|4565005|+\n",
      "-tactctaaagagttcaaagccgaagctgtcagaacggttcttgaaaatcaactttcgat\n",
      "cagtgaaggcgcttc\n",
      "\n",
      "\n",
      "10 [(('G000006925', (900174, 900410)), (900174, 900276), '+'), (('G000006925', (2058572, 2058808)), (2058706, 2058808), '-'), (('G000006925', (3422405, 3422641)), (3422405, 3422507), '+'), (('G000006925', (4358832, 4359068)), (4358966, 4359068), '-'), (('G000006925', (4612466, 4613007)), (4612590, 4612692), '-'), (('G000006925', (4622488, 4624460)), (4623665, 4623767), '+'), (('G000006925', (4700206, 4700442)), (4700340, 4700442), '-'), (('G000006925', (4775859, 4776095)), (4775992, 4776095), '-'), (('G000006925', (4782146, 4782382)), (4782146, 4782248), '+'), (('G000299455', (5313369, 5315826)), (5314902, 5315004), '+')]\n",
      ">G000006925|900174|900410|900174|900276|+\n",
      "-cagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccacttc\n",
      ">G000006925|2058572|2058808|2058706|2058808|-\n",
      "tcagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccactt-\n",
      ">G000006925|3422405|3422641|3422405|3422507|+\n",
      "-cagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccacttc\n",
      ">G000006925|4358832|4359068|4358966|4359068|-\n",
      "tcagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccactt-\n",
      ">G000006925|4612466|4613007|4612590|4612692|-\n",
      "-cagagcagctggagcttatcagcagcgcctttaaggttatcgaaaaacaacgccctaaa\n",
      "ctggcctgtcgccggtgcgaccatatcgtgcaggcaccggtgc\n",
      ">G000006925|4622488|4624460|4623665|4623767|+\n",
      "-cagagcagctggagcttatcagcagcgcctttaaggttatcgaaacacaacgtccgaaa\n",
      "ctggcctgttgccggtgtgaccatatcgtgcaggcaccggtgc\n",
      ">G000006925|4700206|4700442|4700340|4700442|-\n",
      "tcagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccactt-\n",
      ">G000006925|4775859|4776095|4775992|4776095|-\n",
      "tcagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccacttc\n",
      ">G000006925|4782146|4782382|4782146|4782248|+\n",
      "-cagagcaactggatatcattaataccgcctttaaagttatcgaaaccatacgtcccaaa\n",
      "ctggcctgtagccggtgtgatgtcatcgttcaggcaccacttc\n",
      ">G000299455|5313369|5315826|5314902|5315004|+\n",
      "-cagagcaactggagcttatcagcagcgcctttaaggttatcgaaacacaacgtccgaaa\n",
      "caggcctgttgccggtgcgaccatatcgtgcaggcaccagtac\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3098706, 3099048), '+'), (('G000299455', (47809, 50193)), (47809, 48151), '-'), (('G000299455', (856582, 857833)), (856788, 857130), '-'), (('G000299455', (3140498, 3142451)), (3140532, 3140874), '-'), (('G000299455', (4771847, 4785261)), (4784342, 4784684), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3098706|3099048|+\n",
      "-aagatgaagtgtttgaacggggagtgatcaacgtcttcaaagggctgagctgggattac\n",
      "aaaaccaactcaccctgtaaatttggcagtaaaattatcgtcaacaacctggtgaggtgg\n",
      "gaccggtgggggtttcatcttatcaccgggcaacagactgaccgactcgtcgacctggaa\n",
      "agaatgctacatctgttcagcggcaaaccgatccccgacaaccgggaaaacatcactatt\n",
      "cgtctggatggtcacatccagtccgttcagggtaaagagcgctatgaagacgagatgttc\n",
      "atcattaaatactttaagaagggatctgcacacatcactttca\n",
      ">G000299455|47809|50193|47809|48151|-\n",
      "aaggatgaagtgtttgagcgaggagtgatcaacgtcttcagagggctgagctggaattac\n",
      "aaaaccaactccccctgtaaatttggcagtaaaattatcgtcaacaatctggtgaggggg\n",
      "gaccggtggggattgcatcttatcaccgggcaacaggcagatcgactcgccgacctggaa\n",
      "agaatgctgtatctgttcagtggcaaaccaatccctgacaaccgggaaaatatcatcatt\n",
      "cggctggatgatcacatccgctccgttcagggcaaagcgtgctatgaagatgagatgttc\n",
      "aacatcagatactttaagaagggctccgcacacatcacgttc-\n",
      ">G000299455|856582|857833|856788|857130|-\n",
      "aaggatgaggtgtttgagcgaggggtgatcaacgtcttcaggggactgaactggaattac\n",
      "aaaaccaattgcccctgcaaatttggcagtaaaattatcgtcaacaacctagtgaggtgg\n",
      "gaccggtgggggtttcatcttatcaccgggcaacagactgaccgactcgtcgacctggaa\n",
      "agaatgctacatctgttcagcggcaaaccgatccccgacaaccgggaaaacatcactatt\n",
      "cgtctggatggtcacatccagtccgttcagggtaaagagcgctatgaagacgagatgttc\n",
      "atcattaaatactttaagaagggatctgcacacatcactttc-\n",
      ">G000299455|3140498|3142451|3140532|3140874|-\n",
      "aaggatgaggtgtttgagcgaggggtgatcaacgtcttcagagggctgaactggaattac\n",
      "aaaaccaattgcccctgtaaatttggcagtaaaattatcgtcaacaatctggtgagatgg\n",
      "gaccggtggggatttcatcttatcaccggacaacaggctgaccgactcgccgatctggaa\n",
      "agaatgttgcacctgttcagcggtaaaccaatccccgacaaccggggaaacatcaccctt\n",
      "cgtctggatgatcacattcagtctgttcagggtaaagagagctatgaagatgagatgttc\n",
      "agcatcagatactttaagaagggctccgcgcatatcacgttc-\n",
      ">G000299455|4771847|4785261|4784342|4784684|+\n",
      "-aagatgaagtgtttgaacggggagtgatcaacgtcttcaaagggctgagctgggattac\n",
      "aaaaccaactcaccctgtaaatttggcagtaaaattatcgtcaacaacctggtgaggtgg\n",
      "gaccggtgggggtttcatcttatcaccgggcaacagactgaccgactcgtcgacctggaa\n",
      "agaatgctacatctgttcagcggcaaaccgatccccgacaaccgggaaaacatcactatt\n",
      "cgtctggatggtcacatccagtccgttcagggtaaagagcgctatgaagacgagatgttc\n",
      "atcattaaatactttaagaagggatctgcacacatcactttca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3646446, 3658113)), (3651362, 3651393), '-'), (('G000299455', (250849, 262513)), (257566, 257597), '+'), (('G000299455', (3480865, 3520974)), (3485128, 3485159), '+')]\n",
      ">G000006925|3646446|3658113|3651362|3651393|-\n",
      "acaaaacgtttggcgaactgaaaggcaaacg-\n",
      ">G000299455|250849|262513|257566|257597|+\n",
      "-caaaacgtttggcgaactgaaaggcaaacgt\n",
      ">G000299455|3480865|3520974|3485128|3485159|+\n",
      "-caaaacgtttgccgatctgaaaggcaaacgt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (726937, 727130), '+'), (('G000299455', (2174641, 2181396)), (2175330, 2175523), '+'), (('G000299455', (2910238, 2916987)), (2916105, 2916298), '-')]\n",
      ">G000006925|726235|732999|726937|727130|+\n",
      "-gagaagggtaaaccggaaacggtgtattacctgtgtgaacataatggctgcgtgatccg\n",
      "tcagtctgaacttgatcagaccggggggcgctggatttgtgacaataccgggatgtggac\n",
      "gtgtgacggtctgacgtttttcagtgcttcgggtaatgaaattccgccgccgcgctccat\n",
      "cactttccatatat\n",
      ">G000299455|2174641|2181396|2175330|2175523|+\n",
      "-gagaagaataaaccagaaagtgttttctacctttgtgagcatcatggctgtgtgatcca\n",
      "tcagtctgagcttgaccagagtaacgggcggtggatctgtgaaaacacgggcatgtggac\n",
      "ccgtgacggcctgatgtttttcagcgcccggggtgatgaaattccgccgccgcgttccat\n",
      "cactttccatatct\n",
      ">G000299455|2910238|2916987|2916105|2916298|-\n",
      "ggagaaggacagccccgaaagcgttttctacctctgtgaacatcatggctgcgtgatcca\n",
      "tcagtctgagcttgaccagagcaacgggcggtggatctgtgaaaacacggggatgtggac\n",
      "ccgtgacggtctgacgtttttcagcgccgcggataatgaaattccgccgccgcgctccat\n",
      "cacgttccatatc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (808433, 825842)), (825068, 825592), '+'), (('G000299455', (3480865, 3520974)), (3503891, 3504415), '-'), (('G000299455', (3738235, 3772027)), (3762470, 3762994), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|808433|825842|825068|825592|+\n",
      "-atcctgcgagcctgactaaaatcatgaccagctatgtggttgggcaggcgcttaaggcc\n",
      "gataagattaaactcaccgatatggtgacggtcggtaaagatgcctgggcgacgggaaat\n",
      "ccggcactgcgtggttcatcggtaatgttcctcaaaccgggcgatcaggtttcggtggca\n",
      "gacttgaacaaaggtgtgattatccagtccggtaatgacgcctgtattgcgctggcagat\n",
      "tacgttgccgggagccaggagtcctttattggtctgatgaatggttatgccaaaaaactg\n",
      "gggctgaccaacactaccttccagacggtgcacggcctggatgcgccggggcagttcagt\n",
      "accgcgcgcgatatggcattgctgggtaaagcattgatccacgatgtgccggaagagtac\n",
      "gccattcataaagagaaagagttcaccttcaacaaaattcgtcagcctaaccgtaaccgt\n",
      "ctgctgtggagcagcaatctgaatgttgatggcatgaagacagga\n",
      ">G000299455|3480865|3520974|3503891|3504415|-\n",
      "gatcccgcgagcctgactaaaatcatgaccagctatgtggttgggcaggcgcttaaggcc\n",
      "gataagattaaactcaccgatatggtgacggtcggtaaagatgcctgggcgacgggaaat\n",
      "ccggcactgcgtggttcatcggtaatgttcctcaaaccgggcgatcaggtttcggtggca\n",
      "gacttgaacaaaggtgtgattatccagtccggtaatgacgcctgtattgcgctggccgat\n",
      "tacgttgccgggagccaggagtcattcattggtttgatgaatggttatgccaaaaaactg\n",
      "ggtctgaccaacactaccttccagacggtgcacggtctggatgcgccggggcagttcagt\n",
      "accgcgcgcgatatggcattgctgggtaaagcattgatccacgatgtgccggaagagtac\n",
      "gccattcataaagagaaagagttcaccttcaacaaaattcgtcagcctaaccgtaaccgt\n",
      "ctgctgtggagcagcaatctgaatgttgatggcatgaagacagg-\n",
      ">G000299455|3738235|3772027|3762470|3762994|+\n",
      "-atcctgccagcctgaccaaaatgatgaccagttacgttatcggccaggcaatgaaagcc\n",
      "ggtaaatttaaagaaactgatttagtcactatcggcaacgacgcatgggccaccggtaac\n",
      "ccggtgtttaaaggttcttcgctgatgttcctcaaaccgggcatgcaggttccggtttct\n",
      "cagctgatccgcggtattaacctgcaatcgggtaacgatgcttgtgtcgccatggccgat\n",
      "tttgccgctggtagccaggacgcttttgttggcttgatgaacagctacgttaacgcactg\n",
      "ggcctgaaaaatacccacttccagacggtacatggtctggatgctgatggtcagtacagc\n",
      "tccgcgcgagatatggcgctgatcggccaggcattgatccgtgacgtaccgaatgaatac\n",
      "tcgatctataaagaaaaagaatttacgtttaacggtattcgccagctgaaccgtaacggc\n",
      "ctgttatgggataacagcctgaatgtcgacggcatcaaaaccgga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1578298, 1580868)), (1579423, 1579541), '+'), (('G000006925', (1938092, 1938404)), (1938249, 1938367), '+'), (('G000006925', (2062140, 2062463)), (2062297, 2062415), '+'), (('G000299455', (2927401, 2931337)), (2928536, 2928654), '+')]\n",
      ">G000006925|1578298|1580868|1579423|1579541|+\n",
      "ctttatcgccagccttccggacaggcttaatttcagccgctgctggcgaacaataagcgc\n",
      "cacagcccggcgataacgctttccctcctccgagataaaatatgtgctgccacggcgt\n",
      ">G000006925|1938092|1938404|1938249|1938367|+\n",
      "ctttatcgccagtcttccggacaggcttaatttcagtcgctgctgacggacaataagcgc\n",
      "cacatcccggcgataacgctcaccggcttttgatacaaaatatgtgctgccacgacgt\n",
      ">G000006925|2062140|2062463|2062297|2062415|+\n",
      "ctttatcgccagccttccggacaggtttaatttcagtcgctgctggcgaacaattagcgc\n",
      "cacatcacggcgataacgctcaccggcttttgatacaaaatatgtgctgccacgacgt\n",
      ">G000299455|2927401|2931337|2928536|2928654|+\n",
      "ctttatcgccagccttccggacaggcttaatttcagccgctgctggcgaacaataagcgc\n",
      "cacagcccggcgataacgctttccctcctccgagataaaatatgtgctgccacggcgt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4674917, 4675058)), (4674917, 4675056), '-'), (('G000006925', (4784400, 4785310)), (4784401, 4784540), '+'), (('G000006925', (4816831, 4818356)), (4818101, 4818241), '+'), (('G000299455', (5308545, 5311143)), (5309315, 5309454), '-'), (('G000299455', (5385343, 5385755)), (5385462, 5385601), '+')]\n",
      ">G000006925|4674917|4675058|4674917|4675056|-\n",
      "-tacacgctcgtaagcggccctcacggcccgctaacgcggagatacgccccgacttcggg\n",
      "taaaaccctcgtcgggaccactccgaccgcgcacaaaagcattttcatcgccgatagcgg\n",
      "gtatggcttggcagggccgg-\n",
      ">G000006925|4784400|4785310|4784401|4784540|+\n",
      "-tacacgctcgtaagcggccctcacggcccgctaacgcggagatacgccccgactgcggg\n",
      "t-aaacccttgtcgggaccactccgaccgcgcacagaggcattttcatcgctgacaggcc\n",
      "gcatggcctcgcagagccgga\n",
      ">G000006925|4816831|4818356|4818101|4818241|+\n",
      "atacacgctcgtaagcggccctcacggcccgctaacgcggagatacgccccgacttcggg\n",
      "taaaaccctcgtcgggaccactccgaccgcgcacagaagctctatcatggctgaaaacgg\n",
      "gtatggcttggcagggctgg-\n",
      ">G000299455|5308545|5311143|5309315|5309454|-\n",
      "atacacgctcgtaagcggccctgacggcccgctaacgcggagatacgccccgacttcggg\n",
      "t-aaaccctcgtcgggaccactccgaccgcgcacagaagctctctcatggctgaaaacag\n",
      "gtatggtctggcagggccgg-\n",
      ">G000299455|5385343|5385755|5385462|5385601|+\n",
      "-tacacgctcgtaagcggccctcacggcccgctaacgcggagatacgccccgactgcggg\n",
      "t-aaacccttgtcgggaccactccgaccgcgcacagaagctctgtcatgactgaaaacgg\n",
      "gtatggtctagcagggctggg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (907508, 907840)), (907509, 907530), '+'), (('G000006925', (1395000, 1395332)), (1395310, 1395331), '-'), (('G000299455', (2540736, 2540915)), (2540894, 2540915), '-'), (('G000299455', (2583214, 2583238)), (2583215, 2583236), '+'), (('G000299455', (3310892, 3310914)), (3310892, 3310913), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|907508|907840|907509|907530|+\n",
      "-ctcctttgatgcgaatgccag\n",
      ">G000006925|1395000|1395332|1395310|1395331|-\n",
      "tctcctttgatgcgaatgcca-\n",
      ">G000299455|2540736|2540915|2540894|2540915|-\n",
      "tctcctttgatgcgaatgcca-\n",
      ">G000299455|2583214|2583238|2583215|2583236|+\n",
      "-ctcctttgatgcgaatgccag\n",
      ">G000299455|3310892|3310914|3310892|3310913|-\n",
      "tctcctttgatgcgaatgcca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3750765, 3751074), '-'), (('G000006925', (4535976, 4543278)), (4541282, 4541591), '+'), (('G000299455', (137973, 154383)), (148736, 149045), '+')]\n",
      ">G000006925|3736321|3788106|3750765|3751074|-\n",
      "taggctgtagcctcgctgaccactggtggcatacgggcaaccagtttattgaggtacggc\n",
      "accggaatgttgatcacccactcatacagcgcaccacccgatgcgaacagggctttaaac\n",
      "ggcgggatgctccacagtgttacggtagcggtcaggaacaggaacggtgtccaggcacgt\n",
      "aacacctgacccgcggtgtaaccggtgtgggccagcgtcatatcaacctgtgacgccccc\n",
      "aagtcaccgaagcggaatacgcgaactggttgccagcgtttgaggaacagcgtcaggcag\n",
      "agcagtgat-\n",
      ">G000006925|4535976|4543278|4541282|4541591|+\n",
      "-aggctgtagcctcgctgaccactggtggcatacgggcaaccagtttattgaggtacggc\n",
      "accggaatgttgatcacccactcatacagcgcaccacccgatgcgaacagggctttaaac\n",
      "ggcgggatgctccacagtgttacggtagcggtcaggaacaggaacggtgtccaggcacgt\n",
      "aacacctgacccgcggtgtaaccggtgtgggccagcgtcatatcaacctgtgacgccccc\n",
      "aagtcaccgaagcggaatacgcgaactggttgccagcgtttgaggaacagcgtcaggcag\n",
      "agcagtgata\n",
      ">G000299455|137973|154383|148736|149045|+\n",
      "-aggctgtagcctcgctgaccactggtggcatacgggcaaccagtttatcgaggtacggc\n",
      "accggaatgttgatcacccactcatacagcgcaccacccgatgcgaacagggctttaaac\n",
      "ggcgggatgctccacagtgttacggtagcggtcaggaacaggaacggtgtccaggcacgt\n",
      "aacacctgacccgcggtgtaaccggtgtgggccagcgtcatatcaacctgtgacgccccc\n",
      "aagtcaccgaagcggaatacgcgaactggttgccagcgtttgaggaacagcgtcaggcag\n",
      "agcagtgata\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2728788, 2734132)), (2732282, 2732315), '-'), (('G000299455', (1302091, 1327808)), (1325957, 1325990), '-'), (('G000299455', (3440781, 3470001)), (3461282, 3461315), '+')]\n",
      ">G000006925|2728788|2734132|2732282|2732315|-\n",
      "cttttaaacgttcttcaaactcaccgcgatatt-\n",
      ">G000299455|1302091|1327808|1325957|1325990|-\n",
      "cttttaaacgttcttcaaactcaccgcgatatt-\n",
      ">G000299455|3440781|3470001|3461282|3461315|+\n",
      "-tttaaaacgtttttcaaagtcgccgcgatattt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4362175, 4367702)), (4366043, 4366105), '+'), (('G000299455', (3178864, 3178926)), (3178864, 3178926), '-'), (('G000299455', (4524008, 4535561)), (4527876, 4527938), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1066\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4362175|4367702|4366043|4366105|+\n",
      "-ttgtctcttcgatgagcgagcttttaaagcaggttgccctcgacggctgtgggattgcc\n",
      "tgg\n",
      ">G000299455|3178864|3178926|3178864|3178926|-\n",
      "tttatctcttcgatgagcgagttattaaagaatatggcgctcaacggatacggcattgcc\n",
      "tg-\n",
      ">G000299455|4524008|4535561|4527876|4527938|+\n",
      "-ttgtctcttcgatgagcgagcttttaaagcaggttgccctcgacggctgtgggattgcc\n",
      "tgg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1397350, 1397414)), (1397380, 1397412), '+'), (('G000006925', (1943279, 1943950)), (1943283, 1943315), '+'), (('G000299455', (2150074, 2150741)), (2150709, 2150741), '-'), (('G000299455', (2541462, 2541521)), (2541487, 2541519), '+'), (('G000299455', (2582178, 2582236)), (2582180, 2582212), '-')]\n",
      ">G000006925|1397350|1397414|1397380|1397412|+\n",
      "-caggaggtgaatcaatgacctggcctgaggca\n",
      ">G000006925|1943279|1943950|1943283|1943315|+\n",
      "-caggaggtgaatcaatgagctggcctgatgca\n",
      ">G000299455|2150074|2150741|2150709|2150741|-\n",
      "gcaggaggtgaatcaatgagctggcctgatgc-\n",
      ">G000299455|2541462|2541521|2541487|2541519|+\n",
      "-caggaggtgcgccaatgaactggcctgaagca\n",
      ">G000299455|2582178|2582236|2582180|2582212|-\n",
      "gaaggaggtacgccaatgacctagcctgaagc-\n",
      "\n",
      "\n",
      "20 [(('G000006925', (237940, 238704)), (238329, 238367), '+'), (('G000006925', (539107, 539868)), (539495, 539533), '+'), (('G000006925', (709671, 710707)), (710280, 710318), '-'), (('G000006925', (716695, 717456)), (717083, 717121), '+'), (('G000006925', (902326, 903088)), (902714, 902752), '+'), (('G000006925', (1095979, 1096741)), (1096314, 1096352), '-'), (('G000006925', (1632621, 1633383)), (1633009, 1633047), '+'), (('G000006925', (1834903, 1835664)), (1835237, 1835275), '-'), (('G000006925', (1878572, 1883763)), (1878960, 1878998), '+'), (('G000006925', (1944438, 1945202)), (1944826, 1944864), '+'), (('G000006925', (1962084, 1962845)), (1962472, 1962510), '+'), (('G000006925', (2069615, 2084300)), (2070003, 2070041), '+'), (('G000006925', (2346737, 2347498)), (2347125, 2347163), '+'), (('G000006925', (2477790, 2478551)), (2478178, 2478216), '+'), (('G000006925', (2953405, 2954171)), (2953740, 2953778), '-'), (('G000006925', (4392644, 4394283)), (4393857, 4393895), '-'), (('G000006925', (4823851, 4824717)), (4824290, 4824328), '-'), (('G000299455', (926743, 927171)), (927131, 927169), '+'), (('G000299455', (1290596, 1290651)), (1290596, 1290634), '+'), (('G000299455', (4559929, 4560763)), (4560317, 4560355), '+')]\n",
      ">G000006925|237940|238704|238329|238367|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|539107|539868|539495|539533|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|709671|710707|710280|710318|-\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|716695|717456|717083|717121|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|902326|903088|902714|902752|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1095979|1096741|1096314|1096352|-\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1632621|1633383|1633009|1633047|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1834903|1835664|1835237|1835275|-\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1878572|1883763|1878960|1878998|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1944438|1945202|1944826|1944864|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|1962084|1962845|1962472|1962510|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|2069615|2084300|2070003|2070041|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|2346737|2347498|2347125|2347163|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|2477790|2478551|2478178|2478216|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|2953405|2954171|2953740|2953778|-\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000006925|4392644|4394283|4393857|4393895|-\n",
      "tatcgagaactgttcagggagtctgacatcaagagcgc-\n",
      ">G000006925|4823851|4824717|4824290|4824328|-\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000299455|926743|927171|927131|927169|+\n",
      "-atcgagaactgttcagggagtctgacatcaagagcgcg\n",
      ">G000299455|1290596|1290651|1290596|1290634|+\n",
      "-atcgtgaaccgttcagcgagtcggacattaagagcgcg\n",
      ">G000299455|4559929|4560763|4560317|4560355|+\n",
      "-atcgaaaactgttcagggagtctgacatcaagagcgcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4478753, 4490457)), (4485925, 4485972), '+'), (('G000299455', (3672841, 3680073)), (3674533, 3674580), '+'), (('G000299455', (4713031, 4748324)), (4716749, 4716796), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 20\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 20\r",
      "   10 / 20\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 19  f\b\b\r",
      "STEP     2 / 19  f\b\b\r",
      "STEP     3 / 19  f\b\b\r",
      "STEP     4 / 19  f\b\b\r",
      "STEP     5 / 19  f\b\b\r",
      "STEP     6 / 19  f\b\b\r",
      "STEP     7 / 19  f\b\b\r",
      "STEP     8 / 19  f\b\b\r",
      "STEP     9 / 19  f\b\b\r",
      "STEP    10 / 19  f\b\b\r",
      "STEP    11 / 19  f\b\b\r",
      "STEP    12 / 19  f\b\b\r",
      "STEP    13 / 19  f\b\b\r",
      "STEP    14 / 19  f\b\b\r",
      "STEP    15 / 19  f\b\b\r",
      "STEP    16 / 19  f\b\b\r",
      "STEP    17 / 19  f\b\b\r",
      "STEP    18 / 19  f\b\b\r",
      "STEP    19 / 19  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4478753|4490457|4485925|4485972|+\n",
      "-ccatttttgttgaccagaataccgccttcaccacggcagccttcggt\n",
      ">G000299455|3672841|3680073|3674533|3674580|+\n",
      "-ccatgtttgttcagcagataaccgccttcaccacggcaaccttcggt\n",
      ">G000299455|4713031|4748324|4716749|4716796|-\n",
      "gccatttttgttgaccagaataccgccttcaccgcggcagccttcgg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4620693, 4621057)), (4620927, 4620951), '+'), (('G000006925', (4791871, 4792235)), (4792105, 4792129), '+'), (('G000006925', (4794974, 4795158)), (4795079, 4795103), '-'), (('G000299455', (5313369, 5315826)), (5313603, 5313627), '+')]\n",
      ">G000006925|4620693|4621057|4620927|4620951|+\n",
      "tgcccgcaggtatgtcggagcagg\n",
      ">G000006925|4791871|4792235|4792105|4792129|+\n",
      "tgcccgcaggtatgtcggagcagg\n",
      ">G000006925|4794974|4795158|4795079|4795103|-\n",
      "tgcctgcaggtatgtcggagcagg\n",
      ">G000299455|5313369|5315826|5313603|5313627|+\n",
      "tgcccgcaggcatgtcggagcagg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (3082575, 3083578)), (3082827, 3082873), '+'), (('G000299455', (61438, 61772)), (61508, 61554), '-'), (('G000299455', (871858, 872859)), (872563, 872609), '-'), (('G000299455', (1994581, 1994887)), (1994623, 1994669), '-'), (('G000299455', (3206911, 3206957)), (3206911, 3206957), '-'), (('G000299455', (4539188, 4539522)), (4539258, 4539304), '-')]\n",
      ">G000006925|3082575|3083578|3082827|3082873|+\n",
      "-ccatcaaactgggccgtagctcccgctggctgaaaagcgaagtgga\n",
      ">G000299455|61438|61772|61508|61554|-\n",
      "cccatcaaactcggccgcagctcccgctggctgaaaagtgaagtgg-\n",
      ">G000299455|871858|872859|872563|872609|-\n",
      "cccatcaaactcggccgcagctcccgatggctgaaaagtgaagtgg-\n",
      ">G000299455|1994581|1994887|1994623|1994669|-\n",
      "cccatcaaaatggggcgcagctcccgctggctgaaaagtgaagtgg-\n",
      ">G000299455|3206911|3206957|3206911|3206957|-\n",
      "cccatcaaactggggcgcagcagccgctggtacaaaagtgaagtgg-\n",
      ">G000299455|4539188|4539522|4539258|4539304|-\n",
      "cccatcaagctgggccgcagctcccgctggctgaaaagtgaagtgg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (727884, 727935), '+'), (('G000299455', (2174641, 2181396)), (2176277, 2176328), '+'), (('G000299455', (2910238, 2916987)), (2915300, 2915351), '-'), (('G000299455', (3602804, 3602855)), (3602804, 3602855), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|726235|732999|727884|727935|+\n",
      "-gtgggataacaaaaagcggcgtaacgaagcgctggactgcctggtgtatgc\n",
      ">G000299455|2174641|2181396|2176277|2176328|+\n",
      "-gtgggataacaaaaagcggcgtaacgaagcgctggactgcctggtgtatgc\n",
      ">G000299455|2910238|2916987|2915300|2915351|-\n",
      "tgtgggataacaaaaagcggcgtaacgaagcgctggactgcctggtgtatg-\n",
      ">G000299455|3602804|3602855|3602804|3602855|-\n",
      "tgtgggacagcaaaaagcgacgcaatgaggcgctcgactgcttcgtttatg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4387645, 4387715)), (4387664, 4387706), '+'), (('G000299455', (2375107, 2375149)), (2375107, 2375149), '-'), (('G000299455', (2547706, 2547776)), (2547725, 2547767), '+'), (('G000299455', (3291052, 3291122)), (3291061, 3291103), '-')]\n",
      ">G000006925|4387645|4387715|4387664|4387706|+\n",
      "-tgctggtcagtctaatgccatggcgtatggcgaaggactgcc\n",
      ">G000299455|2375107|2375149|2375107|2375149|-\n",
      "ttgctggtcaatccaatgccatgtcatatggtgatggaatgc-\n",
      ">G000299455|2547706|2547776|2547725|2547767|+\n",
      "-tgctggtcagtccaatggtatgtcatacggtgagggactgcc\n",
      ">G000299455|3291052|3291122|3291061|3291103|-\n",
      "ttgctggtcagtccaatggtatgtcatacggtgagggactgc-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4620693, 4621057)), (4620951, 4620973), '+'), (('G000006925', (4704162, 4704221)), (4704199, 4704221), '+'), (('G000006925', (4791871, 4792235)), (4792129, 4792151), '+'), (('G000006925', (4794974, 4795158)), (4795057, 4795079), '-'), (('G000299455', (5313369, 5315826)), (5313627, 5313649), '+')]\n",
      ">G000006925|4620693|4621057|4620951|4620973|+\n",
      "aacttgatgcccttctttacgg\n",
      ">G000006925|4704162|4704221|4704199|4704221|+\n",
      "aacttgactcctgtctttacgg\n",
      ">G000006925|4791871|4792235|4792129|4792151|+\n",
      "aacttgatgcccttctttacgg\n",
      ">G000006925|4794974|4795158|4795057|4795079|-\n",
      "agcttgatgcccttctttacgg\n",
      ">G000299455|5313369|5315826|5313627|5313649|+\n",
      "aacttgatgcctgcctttacgg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3736321, 3788106)), (3750497, 3750765), '+'), (('G000006925', (4535976, 4543278)), (4541591, 4541859), '-'), (('G000299455', (137973, 154383)), (149045, 149313), '-'), (('G000299455', (830050, 839519)), (838300, 838568), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3736321|3788106|3750497|3750765|+\n",
      "-tggcatttggcgcgatgggcattccaatcctggttgccggacaagtaacaggcatcgac\n",
      "agctttgagattggccagatggtggggcgtcagctaccgtttatgaccattatcgtgctg\n",
      "ttctggatcatggcgattatggacggctggcgcggtatcaaagagacgtggcctgcggtc\n",
      "gtggttgcaggcggctcgtttgccatcgctcagtaccttagctctaacttcattgggccg\n",
      "gagctgccggacattatctcttcgctggt\n",
      ">G000006925|4535976|4543278|4541591|4541859|-\n",
      "gtggcatttggcgcgatgggcattccaatcctggttgccggacaagtaacaggcatcgac\n",
      "agctttgagattggccagatggtggggcgtcagctaccgtttatgaccattatcgtgctg\n",
      "ttctggatcatggcgattatggacggctggcgcggtatcaaagagacgtggcctgcggtc\n",
      "gtggttgcaggcggctcgtttgccatcgctcagtaccttagctctaacttcattgggccg\n",
      "gagctgccggacattatctcttcgctgg-\n",
      ">G000299455|137973|154383|149045|149313|-\n",
      "gtggcatttggcgcgatgggcattccaatcctggttgccggacaagtaacaggcatcgac\n",
      "agttttgagattggccagatggtggggcgtcagctaccgtttatgaccattatcgtgctg\n",
      "ttctggatcatggcgattatggacggctggcgcggtatcaaagagacgtggcctgcggtc\n",
      "gtggttgcgggcggctcgtttgccatcgctcagtaccttagctccaacttcattgggccg\n",
      "gagctgccggacattatctcttcgctgg-\n",
      ">G000299455|830050|839519|838300|838568|+\n",
      "-tggcgtttggtgcgttgggcgtgccgattctggtcgccggtcaggtaacgggaatcgat\n",
      "ccgttccacattggcgcaatggcgggacgtcagttaccgttcctgtcggttcttgtgccg\n",
      "ttctggctggtagcaatgatggacggctggaaaggggtgaaagagacgtggccagcggcg\n",
      "ctggttgctgggggaagctttgctgtcactcagttctttacctctaactatattggtccg\n",
      "gaactgccggatattacttcggcgctggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (3013767, 3013852), '-'), (('G000006925', (4248405, 4248490)), (4248405, 4248490), '-'), (('G000299455', (978228, 988794)), (979009, 979094), '+')]\n",
      ">G000006925|2976303|3015537|3013767|3013852|-\n",
      "ctggcggaagatggctgtctgaacttctttgccgggctgacggataaaaacttcaaagtg\n",
      "ccgtttaatttctacaacgtccatt-\n",
      ">G000006925|4248405|4248490|4248405|4248490|-\n",
      "ctggcgacagatggttgcctgaatttcttcgccggaccgcaggataaacatttcagcgcg\n",
      "ccaattaatttctacgatgtgcatt-\n",
      ">G000299455|978228|988794|979009|979094|+\n",
      "-tggcggaagatggctgtctgaacttctttgccgggccgacggataaaaacttcaaagtg\n",
      "ccgtttaatttctacaacgtccatta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1122423, 1168132)), (1145995, 1146048), '-'), (('G000299455', (3057394, 3103258)), (3079545, 3079598), '+'), (('G000299455', (3697938, 3726006)), (3721206, 3721259), '+')]\n",
      ">G000006925|1122423|1168132|1145995|1146048|-\n",
      "atcggcacaaagcgtttaccggcaaagaagccaagatactcaggcagcttaat-\n",
      ">G000299455|3057394|3103258|3079545|3079598|+\n",
      "-tcggcacaaagcgtttaccggcaaagaagccaagatactcaggcagcttaata\n",
      ">G000299455|3697938|3726006|3721206|3721259|+\n",
      "-tcggcacaaagcgtttgccgccgaagaagctcaggaagtccggcagtttaata\n",
      "\n",
      "\n",
      "3 [(('G000006925', (914470, 914504)), (914470, 914504), '+'), (('G000006925', (1416851, 1416885)), (1416851, 1416885), '-'), (('G000299455', (2936028, 2936062)), (2936028, 2936062), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|914470|914504|914470|914504|+\n",
      "-atgccagcaatggcagggattcgctcaaccttaa\n",
      ">G000006925|1416851|1416885|1416851|1416885|-\n",
      "tatgccagcaatggcagggattcgctcaacctta-\n",
      ">G000299455|2936028|2936062|2936028|2936062|+\n",
      "-atgccagcaatggcagggatttgttcacccttaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4271312, 4271355), '+'), (('G000299455', (3844542, 3852289)), (3851639, 3851682), '+'), (('G000299455', (4795454, 4823864)), (4810196, 4810239), '+')]\n",
      ">G000006925|4256570|4288375|4271312|4271355|+\n",
      "ttctgtctttaaagacttctacggtatgacgccgctgcattat\n",
      ">G000299455|3844542|3852289|3851639|3851682|+\n",
      "ctctgtttttaaggcgttttacggcctgacaccgttgaattat\n",
      ">G000299455|4795454|4823864|4810196|4810239|+\n",
      "ttctgtctttaaagacttctacggtatgacgccgctgcattat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4026192, 4050087)), (4027193, 4027218), '-'), (('G000299455', (3288931, 3288956)), (3288931, 3288956), '+'), (('G000299455', (5094571, 5134770)), (5133744, 5133769), '+')]\n",
      ">G000006925|4026192|4050087|4027193|4027218|-\n",
      "cacttctgcatcaagacgcgtttcc-\n",
      ">G000299455|3288931|3288956|3288931|3288956|+\n",
      "-acttctgcatcaagacgcgcttcca\n",
      ">G000299455|5094571|5134770|5133744|5133769|+\n",
      "-acttctgcatcaagacgcgtttcca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189025, 1189082)), (1189047, 1189078), '+'), (('G000299455', (2920445, 2920480)), (2920445, 2920476), '+'), (('G000299455', (4140164, 4140217)), (4140186, 4140217), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1189025|1189082|1189047|1189078|+\n",
      "ggccaacgtcattttttagcgaaaaattctg\n",
      ">G000299455|2920445|2920480|2920445|2920476|+\n",
      "ggccaacgtcatttattagcaaaatattctg\n",
      ">G000299455|4140164|4140217|4140186|4140217|+\n",
      "ggccaacggcattttgcagcaaaatattctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1637263, 1657511)), (1637715, 1638293), '-'), (('G000299455', (2505935, 2525742)), (2525146, 2525724), '+'), (('G000299455', (3440781, 3470001)), (3441188, 3441766), '+')]\n",
      ">G000006925|1637263|1657511|1637715|1638293|-\n",
      "cattgacccaatacggtaaagataaccagtggccattcatgccatccatttcccatttta\n",
      "cacttcctccggattagccagataaccagtggtatcgccggtcgggcggctgttggcgtt\n",
      "aggtttgataacgatattgggttttgtgaaatgcgcgcgcggcagtggcgcgacggcagc\n",
      "cagagtgccgtgcttctggcgcagctcttcaattggaccgaactcaagtgcacgcagcgg\n",
      "gcaggattcgacgcatatgggttgtttcccctcggcgacgcgcgaataacaaccatcgca\n",
      "cttcgtcatgtgccctttttcagcattgtactgtggtgcgccgtacggacaagccatgtg\n",
      "gcagtagcggcagccgatacaaacatcttcatcaaccaccacaaagccatcttcacgctt\n",
      "atgcattgcgccgctggggcagacttttgtacatgcggggtcatcgcaatggttacagga\n",
      "gatggagagataataagcgaacacgttctggtgccagatgccattatcttcctgccagtc\n",
      "gcagccagcatattcataaatacggcggaaactgactt-\n",
      ">G000299455|2505935|2525742|2525146|2525724|+\n",
      "-attgacccaatacggtaaagataaccagtggccattcatgccatccatttcccatttta\n",
      "cacttcctccggattagccagataaccagtggtatcgccggtcgggcggctgttggcgtt\n",
      "aggtttgataacgatattgggttttgtgaaatgcgcgcgcggcagtggcgcgacggcagc\n",
      "cagagtgccgtgcttctggcgcagctcttcaattggaccgaactcaagtgcacgcagcgg\n",
      "gcaggattcgacgcatatgggttgtttcccctcggcgacgcgcgaataacaaccatcgca\n",
      "cttcgtcatgtgccctttttcagcattgtactgtggtgcgccgtacggacaagccatgtg\n",
      "gcagtagcggcagccgatacaaacatcttcatcaaccaccacaaagccatcttcacgctt\n",
      "atgcattgcgccgctggggcagacttttgtacatgcggggtcatcgcaatggttacagga\n",
      "gatggagagataataagcgaacacgttctggtgccagacgccattatcttcctgccagtc\n",
      "gccgccagcgtattcataaatacggcggaaactgacttc\n",
      ">G000299455|3440781|3470001|3441188|3441766|+\n",
      "-attgcccgaagaccgtgaagatcatcagcggccattcatgccatccacttcccatctca\n",
      "cacctccttcgggtttgccagatagccagtggtatccccggtcgggcggctattggcatt\n",
      "gggtttgatcacaatattcggtttggtaaagtgagctcgcggcaacggcgcaacggccgc\n",
      "cagatcgccatgttttttacgcagctcgtcgatagggccgaaatccagcgcccgcagcgg\n",
      "acaagattcaacacagatcggctttttaccctccgcaacacggtcataacaaccatcgca\n",
      "tttggtcatatggcctttcgtttcgttatattgcggtgcgccatacgggcaagccatatg\n",
      "gcagtagcggcagccaatgcacacatcttcatcgaccacaacaaaaccatcttcacgttt\n",
      "atgcatcgcaccgctcgggcagactttagtacaagccggatcttcgcagtggttacatga\n",
      "aatcgacagatagtaggcaaacacgttctggtgccagacaccgttatcttcctgccagtc\n",
      "gccgccagcatattcataaatgcggcggaagctgacttc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2008567, 2033069)), (2010338, 2010396), '-'), (('G000299455', (2082804, 2090582)), (2090329, 2090387), '+'), (('G000299455', (3612234, 3612293)), (3612234, 3612293), '-')]\n",
      ">G000006925|2008567|2033069|2010338|2010396|-\n",
      "attggagcctgaacaa-tccggtgacttctgtgccaatcgggaacgtttataagcacat-\n",
      ">G000299455|2082804|2090582|2090329|2090387|+\n",
      "-ttggagtctgaacaa-tccggtgacttctgcgctaatcggggacgtttctaagcacata\n",
      ">G000299455|3612234|3612293|3612234|3612293|-\n",
      "attggagcctgaacaactccggtgacttctgcgctaaacggggacgtttatgcgcacat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2211287, 2225774)), (2213470, 2213958), '-'), (('G000299455', (1819843, 1851664)), (1851171, 1851659), '+'), (('G000299455', (1851781, 1854451)), (1851781, 1852267), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2211287|2225774|2213470|2213958|-\n",
      "aaaaatatctggaaagcagatcacagtaaataaaaaaaactgctggtgaggcaggttctt\n",
      "ttatttactattttcaggcctaacaccactgcctgtgcaacatcgtgttccaacgaacga\n",
      "attataaccacctctggacagtgttgcaattaagataatgagatttataatcctgactgt\n",
      "tgggaagttgcgccggatcgaacctatttcaagtcctggctaccggacgggtggtgccgc\n",
      "aggtagtgctctgatccggagtgcaacatggcgactcaacaaacgtaaaccagtatcgac\n",
      "aggcagtgtaagggtaaggtgctggtgttgtgctccttaataaggagagcggatggacac\n",
      "gctgacacaaaagttaaccgtgctcattgctgtactggagctattagtggctctgttacg\n",
      "gttgattgatttgttgaagtaacgggcaaattgaaacgcatctaagggggaataccgttc\n",
      "cccccttt-\n",
      ">G000299455|1819843|1851664|1851171|1851659|+\n",
      "-aaaatatctggaaagcagatcgcagtaaataaaaaaacctgctggtgaggcaggttctt\n",
      "ttatttactattttcaggcctaacaccacagcctgtgcaacatcgtgttccaacgaacga\n",
      "attataaccacggctggaccccgttgcaattaagataatgagatttataatccagaccgt\n",
      "tgggacgttgcgccggatcgaacctatttcaagtcctggctaccggacggggggtgctgc\n",
      "aggtggtgccctgatccggagtgcaacatcgtggtccaacatacgtaaaccagtatcgac\n",
      "aggcagtgtaagggtaaggtgctggtgttgcgctccttaataaggagagcggatggactc\n",
      "gctgacacaaaagttaaccgtgctcattgctgtactggagctattagtggctctgttacg\n",
      "gttgattgatttgttgaagtaacgggcaaattgaaacgcatctaagggggaaaaccgctt\n",
      "cccccttta\n",
      ">G000299455|1851781|1854451|1851781|1852267|+\n",
      "-aaaatatctggaaagtagatcgcagaaaataaaaaaacctgctggtgaggcaggttctt\n",
      "ttatttactacagtgacgct--aacttgctgattgtgcaacatttcgtaccaacgaataa\n",
      "attataaccatatccacactccattgcaattcagtgagtgagatttataatcctgcccgt\n",
      "tgggaggttgtgccggatcggaccaattttaagtcctggctgccggacgggtggtgccgc\n",
      "aggcggtgccctgatccggagagcaacgtttcgtaccaacatacgtaaaccagtatcaac\n",
      "aggcagtgtaagggtaaggtgctggtgttgcgctccttaataaggagagcggatggacac\n",
      "gctgacacaaaagttaaccgtgctcattgccgtactggagttattagtggctctgttacg\n",
      "gttgattgatttgttgaagtaacgggcaaattgaaacgcatctaagggggaataccgttt\n",
      "cccccttta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3268417, 3281594)), (3270777, 3270832), '-'), (('G000299455', (661679, 661734)), (661679, 661734), '+'), (('G000299455', (665003, 670357)), (667942, 667997), '+')]\n",
      ">G000006925|3268417|3281594|3270777|3270832|-\n",
      "acgccgacctgaccgtgaatcaagcgttcatcaatccggcttaaaacaatatttg-\n",
      ">G000299455|661679|661734|661679|661734|+\n",
      "-cgccaacctgaccgtgaaccagacggttatcaatacgggttaagagaatatttgg\n",
      ">G000299455|665003|670357|667942|667997|+\n",
      "-cgccgacctgaccgtgaatcaagcgttcatcaatccggcttaaaacaatatttgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3815875, 3829465)), (3829013, 3829060), '-'), (('G000299455', (3214599, 3214646)), (3214599, 3214646), '-'), (('G000299455', (4752701, 4766669)), (4756367, 4756414), '+')]\n",
      ">G000006925|3815875|3829465|3829013|3829060|-\n",
      "ctctcaggaatttcaggatctgccagacggtgctgagacgacgctta-\n",
      ">G000299455|3214599|3214646|3214599|3214646|-\n",
      "ctctcaggaatttcaggatctgccagacggtgctgagacgacgctta-\n",
      ">G000299455|4752701|4766669|4756367|4756414|+\n",
      "-tctcaggaatttcaggatctgccagacggtgctgagacgacgcttac\n",
      "\n",
      "\n",
      "5 [(('G000006925', (734007, 734128)), (734076, 734102), '+'), (('G000006925', (1933459, 1933520)), (1933484, 1933510), '-'), (('G000299455', (2182436, 2182497)), (2182445, 2182471), '+'), (('G000299455', (2909146, 2909207)), (2909172, 2909198), '-'), (('G000299455', (3596860, 3596955)), (3596860, 3596886), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1054\n",
      "\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\n",
      "Reallocating..done. *alloclen = 1050\n",
      "\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|734007|734128|734076|734102|+\n",
      "-ccattgatgtggtgaaaatcccgatg\n",
      ">G000006925|1933459|1933520|1933484|1933510|-\n",
      "-ctgtggaggtagtgaaaatcccgctg\n",
      ">G000299455|2182436|2182497|2182445|2182471|+\n",
      "-ccattcaggtggtgaaaatcccgatg\n",
      ">G000299455|2909146|2909207|2909172|2909198|-\n",
      "cccattcaggtggtgaaaatcccgat-\n",
      ">G000299455|3596860|3596955|3596860|3596886|-\n",
      "cccattgatgtggtgaaaatcccgat-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (64491, 216702)), (145076, 145100), '-'), (('G000006925', (2366334, 2374443)), (2373677, 2373701), '-'), (('G000006925', (2374481, 2472008)), (2431607, 2431631), '+'), (('G000006925', (3503771, 3513305)), (3512454, 3512478), '-'), (('G000299455', (423834, 433368)), (424661, 424685), '+'), (('G000299455', (1623109, 1682614)), (1648854, 1648878), '-'), (('G000299455', (1685057, 1714323)), (1706955, 1706979), '+'), (('G000299455', (4300454, 4333834)), (4303484, 4303508), '+'), (('G000299455', (4516375, 4516518)), (4516460, 4516484), '+')]\n",
      ">G000006925|64491|216702|145076|145100|-\n",
      "tttaaataccgcgtcatgcggtgt-\n",
      ">G000006925|2366334|2374443|2373677|2373701|-\n",
      "tttaaataccgcatcatgcgggga-\n",
      ">G000006925|2374481|2472008|2431607|2431631|+\n",
      "-ttaaataccgcatcatgcggcgtg\n",
      ">G000006925|3503771|3513305|3512454|3512478|-\n",
      "tttgaacagcgcatcatgtggggt-\n",
      ">G000299455|423834|433368|424661|424685|+\n",
      "-ttgaacagcgcatcgtgtggggtg\n",
      ">G000299455|1623109|1682614|1648854|1648878|-\n",
      "tttaaataccgcatcatgcggcgt-\n",
      ">G000299455|1685057|1714323|1706955|1706979|+\n",
      "-ttaaacaccgcatcatgcggggag\n",
      ">G000299455|4300454|4333834|4303484|4303508|+\n",
      "-ttaaacactgcatcatgcggtgtg\n",
      ">G000299455|4516375|4516518|4516460|4516484|+\n",
      "-ttaaataatgcgtcatgcggcgtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2641574, 2668966)), (2645914, 2645964), '+'), (('G000299455', (1078152, 1154001)), (1121382, 1121432), '+'), (('G000299455', (1375183, 1402556)), (1398166, 1398216), '-')]\n",
      ">G000006925|2641574|2668966|2645914|2645964|+\n",
      "-atacagcgcaccgatacctttcgggccatagattttgtgaccggagaaag\n",
      ">G000299455|1078152|1154001|1121382|1121432|+\n",
      "-atacagcacgccgatacctgtcgggccatacagtttgtgacctgaaaaag\n",
      ">G000299455|1375183|1402556|1398166|1398216|-\n",
      "catacagcgcaccgatacctttcgggccatagattttgtgaccggagaaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (2977934, 2978126), '-'), (('G000299455', (988799, 1028140)), (1014897, 1015089), '+'), (('G000299455', (2612825, 2613349)), (2613157, 2613349), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2976303|3015537|2977934|2978126|-\n",
      "ctggttgtttatttcgttgaaggtcatccagtatttgactttatgcttgtagcgttcgaa\n",
      "taccacttccgcgaaacggacaaagaaatcaacgaatttacggttggtccagctaccgta\n",
      "ttgctgcaccagatgcagcggcatttcaaagtgggagagggttatcaccggttcgatgtt\n",
      "gtatttcagcag-\n",
      ">G000299455|988799|1028140|1014897|1015089|+\n",
      "-tggttgttaatttcgttgaaggtcatccagtatttgactttatgcttatagcgttcaaa\n",
      "caccacttccgcgaaacggacaaagaaatcaaccactttacggttggtccagctaccgta\n",
      "ttgctgcaccagatgcagcggcatttcaaagtgggagagggtgatcaccggttcgatgtt\n",
      "gtatttcagcagt\n",
      ">G000299455|2612825|2613349|2613157|2613349|-\n",
      "ctgattattgatttcgttaaacgtcatccagtactttactttttctttatagcgcgtgaa\n",
      "gaccgttgatgcgaagcggataaaaaagtcgatcagtttacggtttcgccagccaccata\n",
      "ttttgtcaccagatgataaggcatctcaaaatgtgaaagcgtcaccacaggttccattcc\n",
      "ctgcttaaggca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (313442, 313947)), (313509, 313822), '+'), (('G000006925', (326814, 327127)), (326814, 327127), '-'), (('G000299455', (2526680, 2527185)), (2526747, 2527060), '+')]\n",
      ">G000006925|313442|313947|313509|313822|+\n",
      "-ctttctcttcaagccacctgacacacgcttcctcgaacgttatatcaggtatttcacca\n",
      "agtttgctgacccgccatgcttcagcctttagcttgtcatggagttctgtcgcctgcctt\n",
      "ttgtcctttgttccaagagactgtttaaatcttttaccgttcggcaatgtgaaactggcg\n",
      "taccatatttcacctctgcggaagagtgacattttctttcctctgttatgccatcacccg\n",
      "cgctcacctggacagtatgcagcggagactgaagagccgcaatgcaggcttgtcgtgttg\n",
      "tgaggtaaggagat\n",
      ">G000006925|326814|327127|326814|327127|-\n",
      "gctttctcttcaagccacctgacacacgcttcctcgaacgttatatcaggtatttcacca\n",
      "agtttgctgacccgccatgcttcagcctttagcttgtcatggagttctgtcgcctgcctt\n",
      "ttgtcctttgttccaagagactgtttaaatcttttaccgttcggcaatgtgaaactggcg\n",
      "taccatatttcacctctgcggaagagtgacattttctttcctctgttatgccatcacccg\n",
      "cgctcacctggacagtatgcagcggagactgaagagccgcaatgcaggcttgtcgtgttg\n",
      "tgaggtaaggaga-\n",
      ">G000299455|2526680|2527185|2526747|2527060|+\n",
      "-ctttttcctcaagccatctcaggcaggcttcttcgaaagtgacatctggcatatccccc\n",
      "agtctgtctactcgccagagttctgcttttcgcttgtcgtgcaactcctgagcttgccgc\n",
      "ttgtcctttgtgccaagagactccttaattcgcttcccgcccgggagcgagtacgaggcg\n",
      "taccatatttcacctctgcggaagagtgacattttctttcctctgttatgccatcacccg\n",
      "cgctcacctggacagtatgcagcggtgaatgaagtgctgcaatgcaggcttgccgggttg\n",
      "tgagataaggtgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (732548, 732696), '+'), (('G000299455', (2174641, 2181396)), (2180944, 2181092), '+'), (('G000299455', (2910238, 2916987)), (2910536, 2910684), '-')]\n",
      ">G000006925|726235|732999|732548|732696|+\n",
      "-gcaaaaacgtcctatgtggcagaggacggtaccgaaaaacgttatctgccggacggcac\n",
      "actggtactgggaaatacggccgcagagggtattcgttgctacggtgccattcaggatgc\n",
      "gcaggcattgtctgaaggcgtggtggcgt\n",
      ">G000299455|2174641|2181396|2180944|2181092|+\n",
      "-gcgaaaacgtcttatgtggcagacgacggtaccgaaaaacgttatctgccggtgggcac\n",
      "actggtcctggggaatacggcagcagagggcattcgttgctatggtgccattcaggatgc\n",
      "gcaggcgttgtccgaaggtgtggtggcct\n",
      ">G000299455|2910238|2916987|2910536|2910684|-\n",
      "agcgaaaacgtcttatgtggcagaggacggtaccgaaaaacgttatctgccggagggcac\n",
      "gctggtgctggggaatacggcagcagaggggattcgttgctatggagccattaaggatgc\n",
      "acaggcgttgtctgaaggggtggtggct-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4256570, 4288375)), (4281090, 4281218), '+'), (('G000299455', (1986396, 1986524)), (1986396, 1986524), '+'), (('G000299455', (3150858, 3151017)), (3150889, 3151017), '+'), (('G000299455', (4795454, 4823864)), (4819974, 4820102), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4256570|4288375|4281090|4281218|+\n",
      "ggcgaataagcgtcggcagcgcagtcattgggcgattacgattaggcagaacctccgtac\n",
      "caacaatggcattaatggtggttgatttccctgctttcatggtaccgacaattgccagca\n",
      "ccatttcc\n",
      ">G000299455|1986396|1986524|1986396|1986524|+\n",
      "ggcggatgagcgtcggtacagaggtcatggggcggttacggttaggcagaatttcctgcc\n",
      "ccacaatggcgttgatggtggttgatttgcctgctttcatcgtgccgacaacagccagca\n",
      "ccatctcc\n",
      ">G000299455|3150858|3151017|3150889|3151017|+\n",
      "ggcggatgagtgtcggtacagaggtcatggggcggttacggttaggcagaatttcctgcc\n",
      "ccacaatggcgttgatggtggttgatttgcctgctttcatcgtgccgacaacagccagca\n",
      "ccatctcc\n",
      ">G000299455|4795454|4823864|4819974|4820102|+\n",
      "ggcgaataagcgtcggcagcgcagtcattgggcgattacgattaggcagaacctccgtac\n",
      "caacaatggcattaatggtggttgatttccctgctttcatggtaccgacaattgccagca\n",
      "ccatttcc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2728788, 2734132)), (2733467, 2733557), '-'), (('G000299455', (1302091, 1327808)), (1327142, 1327232), '-'), (('G000299455', (3440781, 3470001)), (3460403, 3460493), '+'), (('G000299455', (4201688, 4201857)), (4201727, 4201816), '-')]\n",
      ">G000006925|2728788|2734132|2733467|2733557|-\n",
      "ggatgacggaatacggacgacgacgcacagcttcggtcaggtagccaccttcttcataac\n",
      "cgacatatcccggaggcgcaccaaccagac-\n",
      ">G000299455|1302091|1327808|1327142|1327232|-\n",
      "ggatgacggaatacggacgacgacgcacagcttcggtcaggtagccaccttcttcataac\n",
      "cgacatatcccggaggcgcaccaaccagac-\n",
      ">G000299455|3440781|3470001|3460403|3460493|+\n",
      "-cagcaccgcatgtggatgcttgatgaccgcatcagtcagcaaacctccctgatcaaaac\n",
      "caacgtatcccggaggcgcaccaatcagacg\n",
      ">G000299455|4201688|4201857|4201727|4201816|-\n",
      "-caccaccgagtacggtttctggcgaatcgcttcggtcagtacgccgccttcgccatagc\n",
      "cgacatagcccggaggggagccaatcagcc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3230192, 3237768)), (3233146, 3233180), '+'), (('G000006925', (3246783, 3268347)), (3267338, 3267372), '-'), (('G000299455', (671363, 678362)), (672286, 672320), '+'), (('G000299455', (707538, 715112)), (712124, 712158), '-')]\n",
      ">G000006925|3230192|3237768|3233146|3233180|+\n",
      "------tttcacagcccagaccaatcaccagcactgcgcc\n",
      ">G000006925|3246783|3268347|3267338|3267372|-\n",
      "gcgaaatattgtgaatggtacgaataggtacaac------\n",
      ">G000299455|671363|678362|672286|672320|+\n",
      "------tttcacaacccagaccaatcaccattacttcgcc\n",
      ">G000299455|707538|715112|712124|712158|-\n",
      "-----ttttcacagcccagaccaatcaccagcactgcgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1189492, 1208094)), (1202712, 1202815), '+'), (('G000299455', (3026184, 3035686)), (3031276, 3031379), '-'), (('G000299455', (4087856, 4088016)), (4087914, 4088016), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 55\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 55\r",
      "   10 / 55\r",
      "   20 / 55\r",
      "   30 / 55\r",
      "   40 / 55\r",
      "   50 / 55\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 54  f\b\b\r",
      "STEP     2 / 54  f\b\b\r",
      "STEP     3 / 54  f\b\b\r",
      "STEP     4 / 54  f\b\b\r",
      "STEP     5 / 54  f\b\b\r",
      "STEP     6 / 54  f\b\b\r",
      "STEP     7 / 54  f\b\b\r",
      "STEP     8 / 54  f\b\b\r",
      "STEP     9 / 54  f\b\b\r",
      "STEP    10 / 54  f\b\b\r",
      "STEP    11 / 54  f\b\b\r",
      "STEP    12 / 54  f\b\b\r",
      "STEP    13 / 54  f\b\b\r",
      "STEP    14 / 54  f\b\b\r",
      "STEP    15 / 54  f\b\b\r",
      "STEP    16 / 54  f\b\b\r",
      "STEP    17 / 54  f\b\b\r",
      "STEP    18 / 54  f\b\b\r",
      "STEP    19 / 54  f\b\b\r",
      "STEP    20 / 54  f\b\b\r",
      "STEP    21 / 54  f\b\b\r",
      "STEP    22 / 54  f\b\b\r",
      "STEP    23 / 54  f\b\b\r",
      "STEP    24 / 54  f\b\b\r",
      "STEP    25 / 54  f\b\b\r",
      "STEP    26 / 54  f\b\b\r",
      "STEP    27 / 54  f\b\b\r",
      "STEP    28 / 54  f\b\b\r",
      "STEP    29 / 54  f\b\b\r",
      "STEP    30 / 54  f\b\b\r",
      "STEP    31 / 54  f\b\b\r",
      "STEP    32 / 54  f\b\b\r",
      "STEP    33 / 54  f\b\b\r",
      "STEP    34 / 54  f\b\b\r",
      "STEP    35 / 54  f\b\b\r",
      "STEP    36 / 54  f\b\b\r",
      "STEP    37 / 54  f\b\b\r",
      "STEP    38 / 54  f\b\b\r",
      "STEP    39 / 54  f\b\b\r",
      "STEP    40 / 54  f\b\b\r",
      "STEP    41 / 54  f\b\b\r",
      "STEP    42 / 54  f\b\b\r",
      "STEP    43 / 54  f\b\b\r",
      "STEP    44 / 54  f\b\b\r",
      "STEP    45 / 54  f\b\b\r",
      "STEP    46 / 54  f\b\b\r",
      "STEP    47 / 54  f\b\b\r",
      "STEP    48 / 54  f\b\b\r",
      "STEP    49 / 54  f\b\b\r",
      "STEP    50 / 54  f\b\b\n",
      "Reallocating..done. *alloclen = 1146\n",
      "\r",
      "STEP    51 / 54  f\b\b\r",
      "STEP    52 / 54  f\b\b\r",
      "STEP    53 / 54  f\b\b\r",
      "STEP    54 / 54  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 55\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 55\r",
      "   10 / 55\r",
      "   20 / 55\r",
      "   30 / 55\r",
      "   40 / 55\r",
      "   50 / 55\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 54  f\b\b\r",
      "STEP     2 / 54  f\b\b\r",
      "STEP     3 / 54  f\b\b\r",
      "STEP     4 / 54  f\b\b\r",
      "STEP     5 / 54  f\b\b\r",
      "STEP     6 / 54  f\b\b\r",
      "STEP     7 / 54  f\b\b\r",
      "STEP     8 / 54  f\b\b\r",
      "STEP     9 / 54  f\b\b\r",
      "STEP    10 / 54  f\b\b\r",
      "STEP    11 / 54  f\b\b\r",
      "STEP    12 / 54  f\b\b\r",
      "STEP    13 / 54  f\b\b\r",
      "STEP    14 / 54  f\b\b\r",
      "STEP    15 / 54  f\b\b\r",
      "STEP    16 / 54  f\b\b\r",
      "STEP    17 / 54  f\b\b\r",
      "STEP    18 / 54  f\b\b\r",
      "STEP    19 / 54  f\b\b\r",
      "STEP    20 / 54  f\b\b\r",
      "STEP    21 / 54  f\b\b\r",
      "STEP    22 / 54  f\b\b\r",
      "STEP    23 / 54  f\b\b\r",
      "STEP    24 / 54  f\b\b\r",
      "STEP    25 / 54  f\b\b\r",
      "STEP    26 / 54  f\b\b\r",
      "STEP    27 / 54  f\b\b\r",
      "STEP    28 / 54  f\b\b\r",
      "STEP    29 / 54  f\b\b\r",
      "STEP    30 / 54  f\b\b\r",
      "STEP    31 / 54  f\b\b\r",
      "STEP    32 / 54  f\b\b\r",
      "STEP    33 / 54  f\b\b\r",
      "STEP    34 / 54  f\b\b\r",
      "STEP    35 / 54  f\b\b\r",
      "STEP    36 / 54  f\b\b\r",
      "STEP    37 / 54  f\b\b\r",
      "STEP    38 / 54  f\b\b\r",
      "STEP    39 / 54  f\b\b\r",
      "STEP    40 / 54  f\b\b\r",
      "STEP    41 / 54  f\b\b\r",
      "STEP    42 / 54  f\b\b\r",
      "STEP    43 / 54  f\b\b\r",
      "STEP    44 / 54  f\b\b\r",
      "STEP    45 / 54  f\b\b\r",
      "STEP    46 / 54  f\b\b\r",
      "STEP    47 / 54  f\b\b\r",
      "STEP    48 / 54  f\b\b\r",
      "STEP    49 / 54  f\b\b\r",
      "STEP    50 / 54  f\b\b\r",
      "STEP    51 / 54  f\b\b\r",
      "STEP    52 / 54  f\b\b\r",
      "STEP    53 / 54  f\b\b\r",
      "STEP    54 / 54  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1189492|1208094|1202712|1202815|+\n",
      "-tatcgttcgaaagggagtgtcagaggctatagcgcagggctgtatgccacttggtttgc\n",
      "cgatgacatcagtaaaaaaggcgcatacattgactcctgggcgc\n",
      ">G000299455|3026184|3035686|3031276|3031379|-\n",
      "ttatcgttcgaaagggagtgtcagaggctattgcgcagggctgtatgccatttggtttgc\n",
      "cgatgacatcagtaaaaaaggcgcatacattgacgcctgggcg-\n",
      ">G000299455|4087856|4088016|4087914|4088016|-\n",
      "ttatcgtgcaaaagcgagtgtgaacggatatagcaccggcctctatgccacctggtatgc\n",
      "cgatgacgagtcgcgtaatggcgcgtatctcgacagttgggc--\n",
      "\n",
      "\n",
      "55 [(('G000006925', (263407, 264672)), (263661, 263733), '-'), (('G000006925', (311637, 313002)), (311991, 312063), '-'), (('G000006925', (315983, 317248)), (316922, 316994), '+'), (('G000006925', (325529, 326794)), (325783, 325855), '-'), (('G000006925', (330360, 332058)), (330613, 330685), '-'), (('G000006925', (507802, 509300)), (508054, 508126), '-'), (('G000006925', (699183, 700447)), (700122, 700194), '+'), (('G000006925', (712907, 714170)), (713159, 713231), '-'), (('G000006925', (724190, 725876)), (725548, 725620), '+'), (('G000006925', (825843, 827104)), (826781, 826853), '+'), (('G000006925', (903718, 904988)), (904662, 904734), '+'), (('G000006925', (914596, 915860)), (915535, 915607), '+'), (('G000006925', (916783, 918049)), (917038, 917110), '-'), (('G000006925', (921552, 942506)), (931402, 931474), '-'), (('G000006925', (1078088, 1090857)), (1081655, 1081727), '+'), (('G000006925', (1100850, 1102113)), (1101789, 1101861), '+'), (('G000006925', (1290847, 1306886)), (1306562, 1306634), '+'), (('G000006925', (1388754, 1391565)), (1390554, 1390626), '-'), (('G000006925', (1401103, 1402373)), (1402048, 1402120), '+'), (('G000006925', (1413769, 1415032)), (1414708, 1414780), '+'), (('G000006925', (1425133, 1465118)), (1426684, 1426756), '+'), (('G000006925', (1466327, 1467592)), (1467266, 1467338), '+'), (('G000006925', (1468138, 1500238)), (1475680, 1475752), '+'), (('G000006925', (1509064, 1578296)), (1575516, 1575588), '+'), (('G000006925', (1582509, 1591522)), (1582761, 1582833), '-'), (('G000006925', (1591523, 1592784)), (1592462, 1592534), '+'), (('G000006925', (1828287, 1829620)), (1828539, 1828611), '-'), (('G000006925', (1836153, 1836944)), (1836620, 1836692), '+'), (('G000006925', (1842508, 1869155)), (1843448, 1843520), '+'), (('G000006925', (1883764, 1885448)), (1885122, 1885194), '+'), (('G000006925', (1938560, 1939822)), (1939499, 1939571), '+'), (('G000006925', (1963295, 1964561)), (1964236, 1964308), '+'), (('G000006925', (2050169, 2051841)), (2050422, 2050494), '-'), (('G000006925', (2057197, 2058461)), (2057450, 2057522), '-'), (('G000006925', (2069615, 2084300)), (2071779, 2071851), '+'), (('G000006925', (2199538, 2210687)), (2204667, 2204739), '+'), (('G000006925', (2584954, 2636558)), (2616928, 2617000), '+'), (('G000006925', (2694303, 2695567)), (2695242, 2695314), '+'), (('G000006925', (2816684, 2833035)), (2832616, 2832688), '+'), (('G000006925', (3322761, 3361739)), (3354394, 3354466), '+'), (('G000006925', (3857305, 3858568)), (3858244, 3858316), '+'), (('G000006925', (3898713, 3912668)), (3912344, 3912416), '+'), (('G000006925', (4462252, 4478752)), (4463561, 4463633), '+'), (('G000006925', (4532444, 4533709)), (4532698, 4532770), '-'), (('G000006925', (4613266, 4614529)), (4613518, 4613590), '-'), (('G000006925', (4695532, 4696775)), (4695764, 4695836), '-'), (('G000006925', (4704225, 4705397)), (4705072, 4705144), '+'), (('G000006925', (4743489, 4744586)), (4743575, 4743647), '-'), (('G000006925', (4765710, 4765908)), (4765761, 4765833), '-'), (('G000006925', (4802939, 4805150)), (4804139, 4804211), '-'), (('G000006925', (4813486, 4814755)), (4813739, 4813811), '-'), (('G000299455', (2052398, 2064114)), (2063424, 2063496), '-'), (('G000299455', (3182222, 3183821)), (3182476, 3182548), '-'), (('G000299455', (4564841, 4566095)), (4565769, 4565841), '+'), (('G000299455', (5287395, 5287744)), (5287660, 5287732), '-')]\n",
      ">G000006925|263407|264672|263661|263733|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|311637|313002|311991|312063|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|315983|317248|316922|316994|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|325529|326794|325783|325855|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|330360|332058|330613|330685|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|507802|509300|508054|508126|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|699183|700447|700122|700194|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|712907|714170|713159|713231|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|724190|725876|725548|725620|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|825843|827104|826781|826853|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|903718|904988|904662|904734|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|914596|915860|915535|915607|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|916783|918049|917038|917110|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgcaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|921552|942506|931402|931474|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|1078088|1090857|1081655|1081727|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1100850|1102113|1101789|1101861|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1290847|1306886|1306562|1306634|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1388754|1391565|1390554|1390626|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|1401103|1402373|1402048|1402120|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1413769|1415032|1414708|1414780|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1425133|1465118|1426684|1426756|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1466327|1467592|1467266|1467338|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1468138|1500238|1475680|1475752|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1509064|1578296|1575516|1575588|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1582509|1591522|1582761|1582833|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|1591523|1592784|1592462|1592534|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1828287|1829620|1828539|1828611|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|1836153|1836944|1836620|1836692|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1842508|1869155|1843448|1843520|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1883764|1885448|1885122|1885194|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1938560|1939822|1939499|1939571|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|1963295|1964561|1964236|1964308|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|2050169|2051841|2050422|2050494|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|2057197|2058461|2057450|2057522|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|2069615|2084300|2071779|2071851|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|2199538|2210687|2204667|2204739|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|2584954|2636558|2616928|2617000|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|2694303|2695567|2695242|2695314|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|2816684|2833035|2832616|2832688|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|3322761|3361739|3354394|3354466|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|3857305|3858568|3858244|3858316|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgcaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|3898713|3912668|3912344|3912416|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|4462252|4478752|4463561|4463633|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|4532444|4533709|4532698|4532770|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagactggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4613266|4614529|4613518|4613590|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4695532|4696775|4695764|4695836|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4704225|4705397|4705072|4705144|+\n",
      "-agtactgcgcatacgattacagggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000006925|4743489|4744586|4743575|4743647|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4765710|4765908|4765761|4765833|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4802939|4805150|4804139|4804211|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000006925|4813486|4814755|4813739|4813811|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtctggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000299455|2052398|2064114|2063424|2063496|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000299455|3182222|3183821|3182476|3182548|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      ">G000299455|4564841|4566095|4565769|4565841|+\n",
      "-agtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggta\n",
      ">G000299455|5287395|5287744|5287660|5287732|-\n",
      "cagtactgcgcatacgattaccgggtcatacaggagcagtttggtctgaaaacatcaatg\n",
      "tcgcgtaaaggt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3746595, 3746621), '-'), (('G000299455', (137973, 154383)), (153190, 153216), '+'), (('G000299455', (137973, 154383)), (153466, 153492), '+')]\n",
      ">G000006925|3736321|3788106|3746595|3746621|-\n",
      "gtggcaatattggaagtgttattggc-\n",
      ">G000299455|137973|154383|153190|153216|+\n",
      "-tggcaatattggaagtgttattggcg\n",
      ">G000299455|137973|154383|153466|153492|+\n",
      "-tggcaatgttgctggtattagtagcg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (722152, 722887)), (722418, 722881), '+'), (('G000299455', (2171310, 2171838)), (2171369, 2171832), '+'), (('G000299455', (2550313, 2550782)), (2550313, 2550776), '+'), (('G000299455', (2920506, 2921242)), (2920512, 2920975), '-'), (('G000299455', (3288014, 3288486)), (3288014, 3288477), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|722152|722887|722418|722881|+\n",
      "-ttctggcgctgattgccgcaggcgcgtctgctccccagatacttgaccagtttctgaac\n",
      "gaaaaagagggtaatcacacaacggcataccgtgatggttctggcatatggaccatctgt\n",
      "cgtggtgccacaatggtggatggtaagcctgtcataccgggaatgaagctgtcgaaggaa\n",
      "aaatgtgaccaggttaacgctattgaacgtgataaggcgctggcatgggtggagcgcaat\n",
      "attaaagtacctctgaccgaaccccagaaagcgggtatagcgtcattttgcccctataac\n",
      "attggccccggtaagtgtttcccgtcgacgttttataagcggctgaatgccggtgatcgt\n",
      "aagggcgcatgcgaggcgattcgctggtggataaaagatgttgggcgcgattgccgcata\n",
      "cgttcaaataactgctatggacaggttattcgtcgtgaccagga\n",
      ">G000299455|2171310|2171838|2171369|2171832|+\n",
      "-ttctggcgttgattgccatcggtgcgcctgcgcctgatattctcgaccagtttctggat\n",
      "gaaaaagaaggtaaccacacaacggcataccgtgatggtgcgggtatctggaccatctgc\n",
      "cgcggtgccacaatggtggatggtaaacctgttattccgggcatgaaactgtcgaaggaa\n",
      "aaatgcgaccaggttaatgccattgaacgggataaggcgctggcatgggtggagcgcaat\n",
      "attaaagttccactgaccgaaccacagaaagcgggtatagcgtcattctgtccctataac\n",
      "attggccccggtaagtgtttcccgtcgacgttttataagcggctgaatgctggtgaccgc\n",
      "aggggagcgtgtgaggcgattcgctggtggattaaggacggtggcagagactgccgtatc\n",
      "cgctcaaataactgttatggtcaggtatcccgtcgtgaccagga\n",
      ">G000299455|2550313|2550782|2550313|2550776|+\n",
      "-ttctggcgctgattgctgcgggcgcttctgcgcctgaaatcctcgaccagtttcttgac\n",
      "gaaaaagaaggtaaccacaccacggcataccgtgatggtgcgggtatctggactatctgc\n",
      "cgcggtgccatcctggtggatggcaaacctgtcgttccgggcatgaagttgtcgaaggaa\n",
      "aaatgcgaccgggttaacgccattgagcgtgataaggcgctggcatgggtggagaaaaac\n",
      "atcagagtgccactgaccgaaccgcagaaagcgggcatcgcgtcattctgtccgtacaac\n",
      "atcgggccttcgaaatgcttcacctcaacgttttaccggaaactgaacgcaggagaccgg\n",
      "aaaggtgcatgtgctgaaatccgccgatggatatatgacggcggcagggactgtcggaac\n",
      "cgctcaaataactgttacgggcaggtatcgcggcgtgaccagga\n",
      ">G000299455|2920506|2921242|2920512|2920975|-\n",
      "gttctggcgctgattgccgctggtgcgcctgcgcctgacattctcgaccagtttctggat\n",
      "gaaaaggaaggtaaccacaccacggcataccgtgatggcgcgggtatctggaccatctgt\n",
      "cgcggtgccattctggtggatggtaaacctgtcgtcccgggcatgaagttgtcgaaggag\n",
      "aaatgcgaccaggttaacgccatcgaacgtgataaggcgctggcatgggtggagaaaaac\n",
      "atcagagtgccattgaccgaaccccagaaagcggggatcgcgtcattctgtccgtacaac\n",
      "attggtcccggtaagtgtttcccgtcgacgttttacagacgaattaatgctggtgatcgc\n",
      "aggggagcatgtgaggcgattcgctggtggattaaggacggtggcagagactgccgtatt\n",
      "cgctcaaataactgttacggtcaggtatcccgtcgtgaccagg-\n",
      ">G000299455|3288014|3288486|3288014|3288477|-\n",
      "gttctggcgctgattgccgcaggtgcttctgcgcctgaaatcctcgaccagtttctggat\n",
      "gaaaaggaaggcaaccacaccacggcataccgtgatggtgtgggtatctggaccatctgc\n",
      "cgtggtgccacccaggtggatggtaagcttgtcgtccccggcatgaagttgtcgaaggaa\n",
      "aaatgcgaccaggttaacgccattgaacgtgataaggcgctggcatgggtggcgaaaaac\n",
      "atcagagtgccactgaccgaacctcagaaagcggggatcgcgtcattctgtccttacaac\n",
      "attggccccggtaagtgtttcccgtcgacgttttataaacgaattaatgcaggcgatcgc\n",
      "aggggagcgtgtgaagcgattcgctggtggattaaggacggtggcagagactgccgtatc\n",
      "cgctcaaataattgctacggtcaggtatcccgtcgcgaccagg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1, 24671)), (9074, 9136), '+'), (('G000006925', (2561624, 2569713)), (2567494, 2567556), '+'), (('G000299455', (4431865, 4470166)), (4441397, 4441459), '-')]\n",
      ">G000006925|1|24671|9074|9136|+\n",
      "-agttcctgtggcagcacaaccaggatccaatggcagtagataaactggcggaaggtatc\n",
      "cgt\n",
      ">G000006925|2561624|2569713|2567494|2567556|+\n",
      "-agttccgttgggagcacaatcaggatgcgatggcggtagaaaaactgtctgaaggcatt\n",
      "cgt\n",
      ">G000299455|4431865|4470166|4441397|4441459|-\n",
      "gagttcctgtggcagcacaaccaggatccaatggcagtagataaactggcggaaggtatc\n",
      "cg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1103067, 1122318)), (1104038, 1104082), '+'), (('G000006925', (2479150, 2517237)), (2498954, 2498998), '-'), (('G000299455', (3103316, 3122569)), (3121554, 3121598), '-')]\n",
      ">G000006925|1103067|1122318|1104038|1104082|+\n",
      "-tccggccagaaccacgccataccggtttccatcaggcccatgcc\n",
      ">G000006925|2479150|2517237|2498954|2498998|-\n",
      "gtcgggccagaaccatgccatgccggtttctaccagcgccatgc-\n",
      ">G000299455|3103316|3122569|3121554|3121598|-\n",
      "gtccggccagaaccacgccatgccggtttccatcaggcccatgc-\n",
      "\n",
      "\n",
      "13 [(('G000006925', (2086041, 2087310)), (2086106, 2086177), '+'), (('G000006925', (4658206, 4659163)), (4659023, 4659094), '-'), (('G000006925', (4678784, 4679315)), (4678854, 4678925), '+'), (('G000299455', (914822, 915038)), (914891, 914962), '+'), (('G000299455', (916939, 917533)), (917007, 917078), '+'), (('G000299455', (1976120, 1977257)), (1977116, 1977187), '-'), (('G000299455', (2004416, 2004799)), (2004481, 2004552), '+'), (('G000299455', (4572747, 4573338)), (4573198, 4573269), '-'), (('G000299455', (4748325, 4748919)), (4748393, 4748464), '+'), (('G000299455', (4767259, 4767478)), (4767328, 4767399), '+'), (('G000299455', (5304330, 5304511)), (5304398, 5304469), '+'), (('G000299455', (5315857, 5316391)), (5316130, 5316201), '+'), (('G000299455', (5333125, 5333340)), (5333193, 5333264), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 13\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 13\r",
      "   10 / 13\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 12  f\b\b\r",
      "STEP     2 / 12  f\b\b\r",
      "STEP     3 / 12  f\b\b\r",
      "STEP     4 / 12  f\b\b\r",
      "STEP     5 / 12  f\b\b\r",
      "STEP     6 / 12  f\b\b\r",
      "STEP     7 / 12  f\b\b\r",
      "STEP     8 / 12  f\b\b\r",
      "STEP     9 / 12  f\b\b\r",
      "STEP    10 / 12  f\b\b\r",
      "STEP    11 / 12  f\b\b\n",
      "Reallocating..done. *alloclen = 1144\n",
      "\r",
      "STEP    12 / 12  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 13\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 13\r",
      "   10 / 13\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 12  f\b\b\r",
      "STEP     2 / 12  f\b\b\r",
      "STEP     3 / 12  f\b\b\r",
      "STEP     4 / 12  f\b\b\r",
      "STEP     5 / 12  f\b\b\r",
      "STEP     6 / 12  f\b\b\r",
      "STEP     7 / 12  f\b\b\r",
      "STEP     8 / 12  f\b\b\r",
      "STEP     9 / 12  f\b\b\r",
      "STEP    10 / 12  f\b\b\r",
      "STEP    11 / 12  f\b\b\r",
      "STEP    12 / 12  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1056\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2086041|2087310|2086106|2086177|+\n",
      "-gtcaacacggttggagggccattcaggcagtatgctcagaatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000006925|4658206|4659163|4659023|4659094|-\n",
      "cgtcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ctggatcgata-\n",
      ">G000006925|4678784|4679315|4678854|4678925|+\n",
      "-gtcaactcggttggaaggccattccggcagtacgctcagaatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|914822|915038|914891|914962|+\n",
      "-atcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|916939|917533|917007|917078|+\n",
      "-gtcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|1976120|1977257|1977116|1977187|-\n",
      "cgtcaacacggttggagggccattccggcagtatgctcagaatatggcgcagatacgctt\n",
      "ccggatcgata-\n",
      ">G000299455|2004416|2004799|2004481|2004552|+\n",
      "-atcaatacggttggaaggttattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcaatgt\n",
      ">G000299455|4572747|4573338|4573198|4573269|-\n",
      "catcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgata-\n",
      ">G000299455|4748325|4748919|4748393|4748464|+\n",
      "-gtcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|4767259|4767478|4767328|4767399|+\n",
      "-gccaactcggttggaaggccattccggcagtacgctcagaatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|5304330|5304511|5304398|5304469|+\n",
      "-gccaactcggttggaaggccattccggcagtacgctcagaatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|5315857|5316391|5316130|5316201|+\n",
      "-gtcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      ">G000299455|5333125|5333340|5333193|5333264|+\n",
      "-atcaactcggttggaaggccattccggcagtacgctcaggatatggcgcagatacgctt\n",
      "ccggatcgatac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (720687, 720818)), (720694, 720721), '+'), (('G000299455', (2546693, 2546781)), (2546700, 2546727), '+'), (('G000299455', (3293791, 3293879)), (3293845, 3293872), '-'), (('G000299455', (3608552, 3608633)), (3608606, 3608633), '-')]\n",
      ">G000006925|720687|720818|720694|720721|+\n",
      "-ttacgtacgtaaaaaacgggatattct\n",
      ">G000299455|2546693|2546781|2546700|2546727|+\n",
      "-ttacgatcgtaaaaatctgcatatcat\n",
      ">G000299455|3293791|3293879|3293845|3293872|-\n",
      "tttacgatcgtaaaaatctgcatatca-\n",
      ">G000299455|3608552|3608633|3608606|3608633|-\n",
      "tttacgtacgtaaaaaattggttatgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4783423, 4783456)), (4783423, 4783453), '+'), (('G000299455', (78155, 78188)), (78155, 78185), '+'), (('G000299455', (82572, 82602)), (82572, 82602), '-'), (('G000299455', (84179, 84210)), (84179, 84209), '+')]\n",
      ">G000006925|4783423|4783456|4783423|4783453|+\n",
      "-ctacctcatcgataactttgcaacagtgcc\n",
      ">G000299455|78155|78188|78155|78185|+\n",
      "-ctgcctcatcgctaactttgcaacagtgcc\n",
      ">G000299455|82572|82602|82572|82602|-\n",
      "gctgcctcatcgctaactttgcaacagtgc-\n",
      ">G000299455|84179|84210|84179|84209|+\n",
      "-ctgcctcatcgctaactttgcaacagtgcc\n",
      "\n",
      "\n",
      "52 [(('G000006925', (263407, 264672)), (264395, 264433), '-'), (('G000006925', (311637, 313002)), (312725, 312763), '-'), (('G000006925', (315983, 317248)), (316222, 316260), '+'), (('G000006925', (325529, 326794)), (326517, 326555), '-'), (('G000006925', (330360, 332058)), (331347, 331385), '-'), (('G000006925', (509301, 510374)), (510097, 510135), '-'), (('G000006925', (699183, 700447)), (699422, 699460), '+'), (('G000006925', (712907, 714170)), (713893, 713931), '-'), (('G000006925', (723314, 724189)), (723553, 723591), '+'), (('G000006925', (825843, 827104)), (826081, 826119), '+'), (('G000006925', (903718, 904988)), (903962, 904000), '+'), (('G000006925', (914596, 915860)), (914835, 914873), '+'), (('G000006925', (916783, 918049)), (917772, 917810), '-'), (('G000006925', (921552, 942506)), (932136, 932174), '-'), (('G000006925', (1078088, 1090857)), (1080955, 1080993), '+'), (('G000006925', (1100850, 1102113)), (1101089, 1101127), '+'), (('G000006925', (1290847, 1306886)), (1305862, 1305900), '+'), (('G000006925', (1388754, 1391565)), (1391288, 1391326), '-'), (('G000006925', (1401103, 1402373)), (1401348, 1401386), '+'), (('G000006925', (1413769, 1415032)), (1414008, 1414046), '+'), (('G000006925', (1425133, 1465118)), (1425984, 1426022), '+'), (('G000006925', (1466327, 1467592)), (1466566, 1466604), '+'), (('G000006925', (1468138, 1500238)), (1474980, 1475018), '+'), (('G000006925', (1582509, 1591522)), (1583495, 1583533), '-'), (('G000006925', (1591523, 1592784)), (1591762, 1591800), '+'), (('G000006925', (1828287, 1829620)), (1829273, 1829311), '-'), (('G000006925', (1834428, 1834902)), (1834667, 1834705), '+'), (('G000006925', (1842508, 1869155)), (1842748, 1842786), '+'), (('G000006925', (1878572, 1883763)), (1883127, 1883165), '+'), (('G000006925', (1938560, 1939822)), (1938799, 1938837), '+'), (('G000006925', (1963295, 1964561)), (1963536, 1963574), '+'), (('G000006925', (2057197, 2058461)), (2058184, 2058222), '-'), (('G000006925', (2069615, 2084300)), (2071106, 2071144), '+'), (('G000006925', (2069615, 2084300)), (2084023, 2084061), '-'), (('G000006925', (2199538, 2210687)), (2203967, 2204005), '+'), (('G000006925', (2584954, 2636558)), (2616228, 2616266), '+'), (('G000006925', (2694303, 2695567)), (2694542, 2694580), '+'), (('G000006925', (2816684, 2833035)), (2831916, 2831954), '+'), (('G000006925', (3322761, 3361739)), (3353694, 3353732), '+'), (('G000006925', (3857305, 3858568)), (3857544, 3857582), '+'), (('G000006925', (3898713, 3912668)), (3911644, 3911682), '+'), (('G000006925', (4532444, 4533709)), (4533432, 4533470), '-'), (('G000006925', (4613266, 4614529)), (4614252, 4614290), '-'), (('G000006925', (4651394, 4651981)), (4651704, 4651742), '-'), (('G000006925', (4695532, 4696775)), (4696498, 4696536), '-'), (('G000006925', (4704225, 4705397)), (4704464, 4704502), '+'), (('G000006925', (4743489, 4744586)), (4744309, 4744347), '-'), (('G000006925', (4802939, 4805150)), (4804873, 4804911), '-'), (('G000006925', (4813486, 4814755)), (4814473, 4814511), '-'), (('G000006925', (4823511, 4823837)), (4823749, 4823787), '+'), (('G000299455', (2065333, 2065756)), (2065382, 2065420), '-'), (('G000299455', (4564841, 4566095)), (4565086, 4565124), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 52\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 52\r",
      "   10 / 52\r",
      "   20 / 52\r",
      "   30 / 52\r",
      "   40 / 52\r",
      "   50 / 52\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 51  f\b\b\r",
      "STEP     2 / 51  f\b\b\r",
      "STEP     3 / 51  f\b\b\r",
      "STEP     4 / 51  f\b\b\r",
      "STEP     5 / 51  f\b\b\r",
      "STEP     6 / 51  f\b\b\r",
      "STEP     7 / 51  f\b\b\r",
      "STEP     8 / 51  f\b\b\r",
      "STEP     9 / 51  f\b\b\r",
      "STEP    10 / 51  f\b\b\r",
      "STEP    11 / 51  f\b\b\r",
      "STEP    12 / 51  f\b\b\r",
      "STEP    13 / 51  f\b\b\r",
      "STEP    14 / 51  f\b\b\r",
      "STEP    15 / 51  f\b\b\r",
      "STEP    16 / 51  f\b\b\r",
      "STEP    17 / 51  f\b\b\r",
      "STEP    18 / 51  f\b\b\r",
      "STEP    19 / 51  f\b\b\r",
      "STEP    20 / 51  f\b\b\r",
      "STEP    21 / 51  f\b\b\r",
      "STEP    22 / 51  f\b\b\r",
      "STEP    23 / 51  f\b\b\r",
      "STEP    24 / 51  f\b\b\r",
      "STEP    25 / 51  f\b\b\r",
      "STEP    26 / 51  f\b\b\r",
      "STEP    27 / 51  f\b\b\r",
      "STEP    28 / 51  f\b\b\r",
      "STEP    29 / 51  f\b\b\r",
      "STEP    30 / 51  f\b\b\r",
      "STEP    31 / 51  f\b\b\r",
      "STEP    32 / 51  f\b\b\r",
      "STEP    33 / 51  f\b\b\r",
      "STEP    34 / 51  f\b\b\r",
      "STEP    35 / 51  f\b\b\r",
      "STEP    36 / 51  f\b\b\r",
      "STEP    37 / 51  f\b\b\r",
      "STEP    38 / 51  f\b\b\r",
      "STEP    39 / 51  f\b\b\r",
      "STEP    40 / 51  f\b\b\r",
      "STEP    41 / 51  f\b\b\r",
      "STEP    42 / 51  f\b\b\r",
      "STEP    43 / 51  f\b\b\r",
      "STEP    44 / 51  f\b\b\r",
      "STEP    45 / 51  f\b\b\r",
      "STEP    46 / 51  f\b\b\r",
      "STEP    47 / 51  f\b\b\r",
      "STEP    48 / 51  f\b\b\r",
      "STEP    49 / 51  f\b\b\n",
      "Reallocating..done. *alloclen = 1078\n",
      "\r",
      "STEP    50 / 51  f\b\b\r",
      "STEP    51 / 51  d\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 52\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 52\r",
      "   10 / 52\r",
      "   20 / 52\r",
      "   30 / 52\r",
      "   40 / 52\r",
      "   50 / 52\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 51  f\b\b\r",
      "STEP     2 / 51  f\b\b\r",
      "STEP     3 / 51  f\b\b\r",
      "STEP     4 / 51  f\b\b\r",
      "STEP     5 / 51  f\b\b\r",
      "STEP     6 / 51  f\b\b\r",
      "STEP     7 / 51  f\b\b\r",
      "STEP     8 / 51  f\b\b\r",
      "STEP     9 / 51  f\b\b\r",
      "STEP    10 / 51  f\b\b\r",
      "STEP    11 / 51  f\b\b\r",
      "STEP    12 / 51  f\b\b\r",
      "STEP    13 / 51  f\b\b\r",
      "STEP    14 / 51  f\b\b\r",
      "STEP    15 / 51  f\b\b\r",
      "STEP    16 / 51  f\b\b\r",
      "STEP    17 / 51  f\b\b\r",
      "STEP    18 / 51  f\b\b\r",
      "STEP    19 / 51  f\b\b\r",
      "STEP    20 / 51  f\b\b\r",
      "STEP    21 / 51  f\b\b\r",
      "STEP    22 / 51  f\b\b\r",
      "STEP    23 / 51  f\b\b\r",
      "STEP    24 / 51  f\b\b\r",
      "STEP    25 / 51  f\b\b\r",
      "STEP    26 / 51  f\b\b\r",
      "STEP    27 / 51  f\b\b\r",
      "STEP    28 / 51  f\b\b\r",
      "STEP    29 / 51  f\b\b\r",
      "STEP    30 / 51  f\b\b\r",
      "STEP    31 / 51  f\b\b\r",
      "STEP    32 / 51  f\b\b\r",
      "STEP    33 / 51  f\b\b\r",
      "STEP    34 / 51  f\b\b\r",
      "STEP    35 / 51  f\b\b\r",
      "STEP    36 / 51  f\b\b\r",
      "STEP    37 / 51  f\b\b\r",
      "STEP    38 / 51  f\b\b\r",
      "STEP    39 / 51  f\b\b\r",
      "STEP    40 / 51  f\b\b\r",
      "STEP    41 / 51  f\b\b\r",
      "STEP    42 / 51  f\b\b\r",
      "STEP    43 / 51  f\b\b\r",
      "STEP    44 / 51  f\b\b\r",
      "STEP    45 / 51  f\b\b\r",
      "STEP    46 / 51  f\b\b\r",
      "STEP    47 / 51  f\b\b\r",
      "STEP    48 / 51  f\b\b\r",
      "STEP    49 / 51  f\b\b\r",
      "STEP    50 / 51  f\b\b\r",
      "STEP    51 / 51  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|263407|264672|264395|264433|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|311637|313002|312725|312763|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|315983|317248|316222|316260|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|325529|326794|326517|326555|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|330360|332058|331347|331385|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|509301|510374|510097|510135|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|699183|700447|699422|699460|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|712907|714170|713893|713931|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|723314|724189|723553|723591|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|825843|827104|826081|826119|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|903718|904988|903962|904000|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|914596|915860|914835|914873|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|916783|918049|917772|917810|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|921552|942506|932136|932174|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|1078088|1090857|1080955|1080993|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1100850|1102113|1101089|1101127|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1290847|1306886|1305862|1305900|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1388754|1391565|1391288|1391326|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|1401103|1402373|1401348|1401386|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1413769|1415032|1414008|1414046|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1425133|1465118|1425984|1426022|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1466327|1467592|1466566|1466604|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1468138|1500238|1474980|1475018|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1582509|1591522|1583495|1583533|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|1591523|1592784|1591762|1591800|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1828287|1829620|1829273|1829311|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|1834428|1834902|1834667|1834705|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1842508|1869155|1842748|1842786|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1878572|1883763|1883127|1883165|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1938560|1939822|1938799|1938837|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|1963295|1964561|1963536|1963574|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|2057197|2058461|2058184|2058222|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|2069615|2084300|2071106|2071144|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|2069615|2084300|2084023|2084061|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|2199538|2210687|2203967|2204005|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|2584954|2636558|2616228|2616266|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|2694303|2695567|2694542|2694580|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|2816684|2833035|2831916|2831954|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|3322761|3361739|3353694|3353732|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|3857305|3858568|3857544|3857582|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|3898713|3912668|3911644|3911682|+\n",
      "-gctggctgaactggaatctgaaattctgcaactgcgta\n",
      ">G000006925|4532444|4533709|4533432|4533470|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|4613266|4614529|4614252|4614290|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|4651394|4651981|4651704|4651742|-\n",
      "cagtggctgagctggaatctgaagttatgcaactgcgt-\n",
      ">G000006925|4695532|4696775|4696498|4696536|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|4704225|4705397|4704464|4704502|+\n",
      "-agtggctgagctggaatctgaagttatgcaactgcgta\n",
      ">G000006925|4743489|4744586|4744309|4744347|-\n",
      "cagtggctgagctggaatctgaagttatgcaactgcgt-\n",
      ">G000006925|4802939|4805150|4804873|4804911|-\n",
      "cgctggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|4813486|4814755|4814473|4814511|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000006925|4823511|4823837|4823749|4823787|+\n",
      "cagtggctgagctggaatctgaagttatgcaactgcgt-\n",
      ">G000299455|2065333|2065756|2065382|2065420|-\n",
      "cggtggctgaactggaatctgaaattctgcaactgcgt-\n",
      ">G000299455|4564841|4566095|4565086|4565124|+\n",
      "-ggtggctgaactggaatctgaaattctgcaactgcgta\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4815021, 4815104)), (4815027, 4815070), '+'), (('G000299455', (62391, 62434)), (62391, 62434), '+'), (('G000299455', (873437, 873523)), (873437, 873480), '+'), (('G000299455', (5428759, 5428842)), (5428793, 5428836), '-')]\n",
      ">G000006925|4815021|4815104|4815027|4815070|+\n",
      "-aaacaaaacaggaatggctgtatcagttacggcgatgttcatc\n",
      ">G000299455|62391|62434|62391|62434|+\n",
      "-aaaccaaagaggactggctgtatcagttccgtcgttgttcatc\n",
      ">G000299455|873437|873523|873437|873480|+\n",
      "-aaaccaaagaagactggctgtatcagttccgtcgttgttcatc\n",
      ">G000299455|5428759|5428842|5428793|5428836|-\n",
      "aaaacaaaacaggaatggttatatcagttacgtcgctgttcca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3703179, 3717854)), (3714681, 3714702), '-'), (('G000299455', (197629, 211581)), (200827, 200848), '+'), (('G000299455', (5299263, 5299284)), (5299263, 5299284), '+')]\n",
      ">G000006925|3703179|3717854|3714681|3714702|-\n",
      "gacatcattaactcgtttaat-\n",
      ">G000299455|197629|211581|200827|200848|+\n",
      "-acatcattaactcgtttaata\n",
      ">G000299455|5299263|5299284|5299263|5299284|+\n",
      "-acatcattaactcgtttaata\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (744361, 744568), '+'), (('G000006925', (1921807, 1928114)), (1923019, 1923226), '-'), (('G000299455', (2192349, 2193633)), (2192349, 2192556), '+'), (('G000299455', (2897570, 2901707)), (2898647, 2898854), '-'), (('G000299455', (3586763, 3590110)), (3586763, 3586970), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|744361|744568|+\n",
      "-ccgatgtttgttggtcagggcgatcagatattcatgaacgacgtgttcctgaaacgcct\n",
      "gacggctccgaccattaccagcggcggtaatcctccggcattttccctgacacctggcgg\n",
      "acggctgacggcgaaaaatgccgatatcagcggtaacgtgaacgcgaactccgggacgct\n",
      "caacaacgtcacgattaataagaactgt\n",
      ">G000006925|1921807|1928114|1923019|1923226|-\n",
      "gccgatgtttgttggtcagggcgatcagatattcatgaacgacgtgttcctgaaacgcct\n",
      "gacggctccgaccattaccagcggcggtaatcctccggcattttccctgacacctggcgg\n",
      "acggctgacggcgaaaaatgccgatatcagcggtaacgtgaacgcgaactccgggacgct\n",
      "caacaacgtcacgattaataagaactg-\n",
      ">G000299455|2192349|2193633|2192349|2192556|+\n",
      "-ccgatgtttgtggcgcagggcaaccagatattcatgaacgacgtgttcctgaagcgcct\n",
      "gacggcccccaccattaccagcggcggtaatcctccggcattttccctgacaccggacgg\n",
      "gcggctgacggcgaaaaatgccgatatcagcggtaacgtgaatgcgaactccgggacgct\n",
      "caacaacgtcacgattaacgagaactgc\n",
      ">G000299455|2897570|2901707|2898647|2898854|-\n",
      "gccgatgtttgttggtcagggcgatcagatattcatgaacgacgtgttcctgaaacgcct\n",
      "gacggctccgaccattaccagcggcggtaatcctccggcattttccctgacaccggacgg\n",
      "gcggctgacggcgaaaaatgccgatatcagcggtaacgtgaatgcgaactccgggacgct\n",
      "caacaacgtcacgattaacgagaactg-\n",
      ">G000299455|3586763|3590110|3586763|3586970|-\n",
      "gccgatgtttgttggtcagggcgatcagatattcatgaatgaagtgttcctgaaacgcct\n",
      "gacggcccccaccattaccagcggcggtaatcctccggcattttccctgacatcagacgg\n",
      "gagactgacggcgaaaaatgcggatatcagtggcagtgtgaatgcgaactcaggaacgct\n",
      "caacaatgtcacgattaaccagaactg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4288354, 4288375), '-'), (('G000006925', (4288513, 4315006)), (4288513, 4288534), '-'), (('G000299455', (4836149, 4864500)), (4861858, 4861879), '+')]\n",
      ">G000006925|4256570|4288375|4288354|4288375|-\n",
      "ccttgtggcgctttagttttg-\n",
      ">G000006925|4288513|4315006|4288513|4288534|-\n",
      "ccttgtggcgctttagttttg-\n",
      ">G000299455|4836149|4864500|4861858|4861879|+\n",
      "-cttgtggcgctttagttttgt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3815875, 3829465)), (3816585, 3816654), '+'), (('G000299455', (3207505, 3210112)), (3208692, 3208761), '+'), (('G000299455', (4542964, 4543033)), (4542964, 4543033), '+'), (('G000299455', (4564765, 4564834)), (4564765, 4564834), '+'), (('G000299455', (5290347, 5290416)), (5290347, 5290416), '-')]\n",
      ">G000006925|3815875|3829465|3816585|3816654|+\n",
      "-taagcgtccagcgaaatccatctggtcataatctgaagcatccgacaaagtggtgtcca\n",
      "ccaaataagt\n",
      ">G000299455|3207505|3210112|3208692|3208761|+\n",
      "-taagcgtccagcgaaatccatctggtcataatctgaagcatccgacaaagtggtgtcca\n",
      "ccaaataagt\n",
      ">G000299455|4542964|4543033|4542964|4543033|+\n",
      "-taagcgtacagcctgaaccgtctggtcagaatctgacgaattagacaaagtggtgtcca\n",
      "ccaaataagt\n",
      ">G000299455|4564765|4564834|4564765|4564834|+\n",
      "-taagcgtacagcctgaaccgtctggtcagaatctgacgaattagacaaagtggtgtcca\n",
      "ccaaataagt\n",
      ">G000299455|5290347|5290416|5290347|5290416|-\n",
      "gtaagcgtacagcctgaaccgtctggtcagaatctgacgaattagacaaagtggtgtcca\n",
      "ccaaataag-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1659262, 1667779)), (1664639, 1664758), '+'), (('G000006925', (4256570, 4288375)), (4261229, 4261347), '-'), (('G000299455', (2495066, 2505927)), (2500432, 2500550), '-'), (('G000299455', (4795454, 4823864)), (4800113, 4800231), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1062\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1659262|1667779|1664639|1664758|+\n",
      "-aaatctcctgcccttcaaattcagatacgctaacgtgttcactggttagcaggtaatac\n",
      "tcagtatcattttttttgagtggaaaaggagactgataatgaaagggtttgtttgacatt\n",
      ">G000006925|4256570|4288375|4261229|4261347|-\n",
      "aggatggtttcgccgtcgaagtcggcaacgctaacgtaatcggaagtgagtagatagtat\n",
      "tcggtattgtctttccccatcgggaaaggtgcctggtagataaagggtttgtttgaca--\n",
      ">G000299455|2495066|2505927|2500432|2500550|-\n",
      "aaaatctcctgcccttcaaattcagatacgctaacgtgttcgctggttagcaggtaatac\n",
      "tcagtatcatcttttttgagtggaaaaggagcctgataatgaaagggtttgtttgaca--\n",
      ">G000299455|4795454|4823864|4800113|4800231|-\n",
      "aggatggtttcgccgtcgaagtcggcaacgctaacgtaatcggaagtgagtagatagtat\n",
      "tcggtattgtctttccccatcgggaaaggtgcctggtagataaagggtttgtttgaca--\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1, 24671)), (9136, 9166), '-'), (('G000006925', (2561624, 2569713)), (2567556, 2567586), '-'), (('G000299455', (1477869, 1484878)), (1479977, 1480007), '+'), (('G000299455', (4431865, 4470166)), (4441367, 4441397), '+')]\n",
      ">G000006925|1|24671|9136|9166|-\n",
      "ttccagtttttcctggtcaatagcaaactt-\n",
      ">G000006925|2561624|2569713|2567556|2567586|-\n",
      "ttccagtttgcgttgatcaacagcgaacag-\n",
      ">G000299455|1477869|1484878|1479977|1480007|+\n",
      "-tccagtttgcgttgatcaacggcgaacaga\n",
      ">G000299455|4431865|4470166|4441367|4441397|+\n",
      "-tccagtttttcctggtcaatagcaaactta\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2187913, 2198767)), (2195087, 2195126), '-'), (('G000299455', (1855036, 1876538)), (1867021, 1867060), '+'), (('G000299455', (4430108, 4430147)), (4430108, 4430147), '+')]\n",
      ">G000006925|2187913|2198767|2195087|2195126|-\n",
      "tggcggcctggcgttattcgtcgcgtgattaccggacat-\n",
      ">G000299455|1855036|1876538|1867021|1867060|+\n",
      "-ggcggcctggcgttattcatcgcgtgattaccggacatt\n",
      ">G000299455|4430108|4430147|4430108|4430147|+\n",
      "-ggccgcctggcgttatgcttcgcaggattacaggacatt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3082575, 3083578)), (3082915, 3082943), '+'), (('G000299455', (61438, 61772)), (61438, 61466), '-'), (('G000299455', (871858, 872859)), (872493, 872521), '-'), (('G000299455', (4539188, 4539522)), (4539188, 4539216), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3082575|3083578|3082915|3082943|+\n",
      "-cctgaccgttatccgttccaccgcagca\n",
      ">G000299455|61438|61772|61438|61466|-\n",
      "ttctgccccttatccgttcacccgcagc-\n",
      ">G000299455|871858|872859|872493|872521|-\n",
      "tcctgaccattacccgttcaaccgcagc-\n",
      ">G000299455|4539188|4539522|4539188|4539216|-\n",
      "ttctgaccgttatccgttcacccgcagc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3087410, 3087437), '-'), (('G000299455', (54863, 56031)), (55999, 56026), '+'), (('G000299455', (857924, 871846)), (868073, 868100), '+'), (('G000299455', (3152907, 3154652)), (3153393, 3153420), '+')]\n",
      ">G000006925|3083675|3099254|3087410|3087437|-\n",
      "gccggtatccacagatgtggataccgg-\n",
      ">G000299455|54863|56031|55999|56026|+\n",
      "-ccggtacccacaaaagtggataccggc\n",
      ">G000299455|857924|871846|868073|868100|+\n",
      "-ccggtatccacagatgtggataccggc\n",
      ">G000299455|3152907|3154652|3153393|3153420|+\n",
      "-ccggtacccacaaacgtggataccggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4367703, 4374506)), (4371142, 4371173), '-'), (('G000299455', (2596243, 2603992)), (2597699, 2597730), '+'), (('G000299455', (4524008, 4535561)), (4532197, 4532228), '-')]\n",
      ">G000006925|4367703|4374506|4371142|4371173|-\n",
      "gcgattgtctctctaacggtcacggaaaaag-\n",
      ">G000299455|2596243|2603992|2597699|2597730|+\n",
      "-cgattgtctctctgacaatcaccgaaaaagg\n",
      ">G000299455|4524008|4535561|4532197|4532228|-\n",
      "gcgattgtctctctgacggtcacggaaaaag-\n",
      "\n",
      "\n",
      "10 [(('G000006925', (385056, 385246)), (385157, 385191), '+'), (('G000006925', (900906, 901096)), (900961, 900995), '-'), (('G000006925', (3423137, 3423327)), (3423192, 3423226), '-'), (('G000006925', (3850834, 3851024)), (3850935, 3850969), '+'), (('G000006925', (4358146, 4358336)), (4358247, 4358281), '+'), (('G000006925', (4699520, 4699710)), (4699621, 4699655), '+'), (('G000006925', (4775173, 4775363)), (4775274, 4775308), '+'), (('G000006925', (4782878, 4783068)), (4782933, 4782967), '-'), (('G000299455', (4749820, 4751622)), (4751485, 4751519), '-'), (('G000299455', (5313369, 5315826)), (5315689, 5315723), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 10\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 9  f\b\b\r",
      "STEP     2 / 9  f\b\b\r",
      "STEP     3 / 9  f\b\b\r",
      "STEP     4 / 9  f\b\b\r",
      "STEP     5 / 9  f\b\b\r",
      "STEP     6 / 9  f\b\b\r",
      "STEP     7 / 9  f\b\b\r",
      "STEP     8 / 9  f\b\b\r",
      "STEP     9 / 9  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|385056|385246|385157|385191|+\n",
      "-gccatctccgcatgtttcgacagcgttttcctct\n",
      ">G000006925|900906|901096|900961|900995|-\n",
      "cgccatctccgcatgtttcgacagcgttttcctc-\n",
      ">G000006925|3423137|3423327|3423192|3423226|-\n",
      "cgccatctccgcatgtttcgacagcgttttcctc-\n",
      ">G000006925|3850834|3851024|3850935|3850969|+\n",
      "-gccatctccgcatgtttcgacagcgttttcctct\n",
      ">G000006925|4358146|4358336|4358247|4358281|+\n",
      "-gccatctccgcatgtttcgacagcgttttcctct\n",
      ">G000006925|4699520|4699710|4699621|4699655|+\n",
      "-gccatctccgcatgtttcgacagcgttttcctct\n",
      ">G000006925|4775173|4775363|4775274|4775308|+\n",
      "-gccatctccgcatgtttcgacagcgttttcctct\n",
      ">G000006925|4782878|4783068|4782933|4782967|-\n",
      "cgccatctccgcatgtttcgacagcgttttcctc-\n",
      ">G000299455|4749820|4751622|4751485|4751519|-\n",
      "tgccgtatccgagtgacgcgacagtgttttcatc-\n",
      ">G000299455|5313369|5315826|5315689|5315723|-\n",
      "tgccgtatctgagtgacgcgacagtgttttcatc-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (1209065, 1211067)), (1210429, 1210473), '+'), (('G000006925', (2756889, 2766574)), (2757510, 2757554), '+'), (('G000006925', (2923039, 2950048)), (2948625, 2948669), '+'), (('G000006925', (3101814, 3103074)), (3102436, 3102480), '+'), (('G000006925', (3810378, 3811029)), (3810973, 3811017), '-'), (('G000006925', (4746555, 4747128)), (4746559, 4746603), '-'), (('G000006925', (4753880, 4754990)), (4753992, 4754036), '-'), (('G000006925', (4795159, 4795487)), (4795316, 4795360), '+'), (('G000299455', (3226712, 3229721)), (3229079, 3229123), '+')]\n",
      ">G000006925|1209065|1211067|1210429|1210473|+\n",
      "-caagcctgcgccgtcagggactgagggcaaaggcctcccggaag\n",
      ">G000006925|2756889|2766574|2757510|2757554|+\n",
      "-caagcctgcgccgtcagggactgagggcaaaggcctcccggaag\n",
      ">G000006925|2923039|2950048|2948625|2948669|+\n",
      "-caagcctgcgccgtcagggactgagggcaaaggcctcccggaag\n",
      ">G000006925|3101814|3103074|3102436|3102480|+\n",
      "-caagcctgcgccgtcagggactgagggcaaaggcctcccggaag\n",
      ">G000006925|3810378|3811029|3810973|3811017|-\n",
      "gccagcctgcgtcgtcaggggctgcgggcgaaagccagccggaa-\n",
      ">G000006925|4746555|4747128|4746559|4746603|-\n",
      "gcaagcctgcgccgtcaggggctgagggcaaaggcctcccggaa-\n",
      ">G000006925|4753880|4754990|4753992|4754036|-\n",
      "gccagtctgcgccgtcaggaactgagcgcaaaagcctcccagaa-\n",
      ">G000006925|4795159|4795487|4795316|4795360|+\n",
      "-caagcctgcgccgtcaggggctgagggcaaaggcctcccggaag\n",
      ">G000299455|3226712|3229721|3229079|3229123|+\n",
      "-caagcctgcgccgtcagggactgagggcaaaggcctcccggaag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3637330, 3640023)), (3638935, 3639011), '+'), (('G000299455', (262744, 269475)), (268563, 268639), '-'), (('G000299455', (2983402, 3007859)), (3006155, 3006231), '+')]\n",
      ">G000006925|3637330|3640023|3638935|3639011|+\n",
      "-gtcacatccccaacggcaaccgcacctattatttgagccgctcgcaaccaccggttttt\n",
      "gcgctgatggtggagtt\n",
      ">G000299455|262744|269475|268563|268639|-\n",
      "ggtcacatccccaacggcaaccgcacctattatttgagccgatcgcaaccaccggttttt\n",
      "gcgctgatggtggagt-\n",
      ">G000299455|2983402|3007859|3006155|3006231|+\n",
      "-gtcatattcccaacggcaaccgcagttactatttaagccgctcgcaaccgcccttcttt\n",
      "gccctgatggtagagtt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1409140, 1409182)), (1409140, 1409182), '-'), (('G000006925', (1842508, 1869155)), (1863189, 1863231), '+'), (('G000299455', (2786550, 2804122)), (2798150, 2798192), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1409140|1409182|1409140|1409182|-\n",
      "aaaatcaacaaattacaaagaaagaacaatacaacctgaaca-\n",
      ">G000006925|1842508|1869155|1863189|1863231|+\n",
      "-aaatcaagaaattagtaagaaagaacaatacaacctgaacaa\n",
      ">G000299455|2786550|2804122|2798150|2798192|+\n",
      "-aaatcaagaaattagtaagaaagaacaatacaacctgaacaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3695478, 3695987), '+'), (('G000299455', (269622, 287922)), (271224, 271733), '+'), (('G000299455', (2665739, 2672184)), (2666153, 2666662), '+')]\n",
      ">G000006925|3693106|3702974|3695478|3695987|+\n",
      "ataaatcctacttttttattgccttcaaataaatttaaggagttcgaaatggaccagaag\n",
      "ctgttaacggatttccgctcagaactactcgattcacgttttggcgcaaaggccatttct\n",
      "actatcgcggagtcaaaacgatttccgctgcacgaaatgcgcgatgatgtcgcattccag\n",
      "attatcaatgatgaattatatcttgatggcaacgctcgtcagaacctggccactttctgc\n",
      "cagacctgggacgacgaaaacgtccataaattgatggatttgtcgatcaataaaaactgg\n",
      "atcgacaaagaagaatatccgcaatccgcagccatcgacctgcgttgcgtaaatatggtt\n",
      "gccgatctgtggcatgcgcctgcgccgaaaaatggtcaggccgttggcaccaacaccatt\n",
      "ggttcttccgaggcctgtatgctcggcgggatggcgatgaaatggcgttggcgcaagcgt\n",
      "atggaagctgcaggcaaaccaacggataa\n",
      ">G000299455|269622|287922|271224|271733|+\n",
      "ataaatcctacttttttattgccttcaaataaatttaaggagttcgaaatggaccagaag\n",
      "ctgttaacggatttccgctcagaactactcgattcacgttttggcgcaaaggccatttct\n",
      "actatcgcggagtcaaaacgatttccgctgcacgaaatgcgcgatgatgtcgcattccag\n",
      "attatcaatgatgaattatatcttgatggcaacgctcgtcagaacctggccactttctgc\n",
      "cagacctgggacgacgaaaacgtccataaattgatggatttgtcgatcaataaaaactgg\n",
      "atcgacaaagaagaatatccgcaatccgcagccatcgacctgcgttgcgtaaatatggtt\n",
      "gccgatctgtggcatgcgcctgcgccgaaaaatggtcaggccgttggcaccaacaccatt\n",
      "ggttcttccgaggcctgtatgctcggcgggatggcgatgaaatggcgttggcgcaagcgt\n",
      "atggaagctgcaggcaaaccaacggataa\n",
      ">G000299455|2665739|2672184|2666153|2666662|+\n",
      "ataaatcctacttttttaatgcgatccaatcattttaaggagtttaaaatggataagaag\n",
      "caagtaacggatttaaggtcggaactactcgattcacgttttggtgcgaagtctatttcc\n",
      "actatcgcagaatcaaaacgttttccgctgcacgaaatgcgcgacgatgtcgcattccag\n",
      "attatcaatgacgaattatatcttgatggcaacgctcgtcagaacctggccactttctgc\n",
      "cagacctgggacgacgaaaatgtccacaaattgatggatttatccattaacaaaaactgg\n",
      "atcgacaaagaagaatatccgcaatccgcagccatcgacctgcgttgcgtaaatatggtt\n",
      "gccgatctgtggcatgcgcctgcgccgaaaaatggtcaggccgttggcaccaacaccatt\n",
      "ggttcttccgaggcctgtatgctcggcgggatggcgatgaaatggcgttggcgcaagcgt\n",
      "atggaagctgcaggcaaaccaacggataa\n",
      "\n",
      "\n",
      "12 [(('G000006925', (386390, 386480)), (386394, 386420), '-'), (('G000006925', (898987, 899077)), (899047, 899072), '+'), (('G000006925', (2059905, 2059995)), (2059909, 2059935), '-'), (('G000006925', (3421218, 3421308)), (3421278, 3421303), '+'), (('G000006925', (3852168, 3852258)), (3852172, 3852198), '-'), (('G000006925', (4360146, 4360236)), (4360150, 4360176), '-'), (('G000006925', (4701524, 4701609)), (4701524, 4701548), '-'), (('G000006925', (4777177, 4777262)), (4777177, 4777201), '-'), (('G000006925', (4780959, 4781049)), (4781019, 4781044), '+'), (('G000299455', (1990734, 1990764)), (1990734, 1990759), '+'), (('G000299455', (4579385, 4579439)), (4579414, 4579439), '+'), (('G000299455', (5343755, 5343840)), (5343815, 5343840), '+')]\n",
      ">G000006925|386390|386480|386394|386420|-\n",
      "tcaataataacctgctgtttaaatgg\n",
      ">G000006925|898987|899077|899047|899072|+\n",
      "-caataataacctgctgtttaaatgg\n",
      ">G000006925|2059905|2059995|2059909|2059935|-\n",
      "tcaataataacctgctgtttaaatgg\n",
      ">G000006925|3421218|3421308|3421278|3421303|+\n",
      "-caataataacctgctgtttaaatgg\n",
      ">G000006925|3852168|3852258|3852172|3852198|-\n",
      "tcaatgataacctgctgtttaaatgg\n",
      ">G000006925|4360146|4360236|4360150|4360176|-\n",
      "tcaataataacctgctgtttaaatgg\n",
      ">G000006925|4701524|4701609|4701524|4701548|-\n",
      "-caataataacctgctggttaaatg-\n",
      ">G000006925|4777177|4777262|4777177|4777201|-\n",
      "-caataataacctgctggttaaatg-\n",
      ">G000006925|4780959|4781049|4781019|4781044|+\n",
      "-caatgataacctgctgtttaaatgg\n",
      ">G000299455|1990734|1990764|1990734|1990759|+\n",
      "-caatgataacctgctctttaaatgg\n",
      ">G000299455|4579385|4579439|4579414|4579439|+\n",
      "-caatgataatcttctgttcaaatgg\n",
      ">G000299455|5343755|5343840|5343815|5343840|+\n",
      "-caatgataatcttctgttcaaatgg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1, 24671)), (5387, 5423), '+'), (('G000006925', (2479150, 2517237)), (2513987, 2514023), '+'), (('G000299455', (1535847, 1550961)), (1539067, 1539103), '-'), (('G000299455', (4431865, 4470166)), (4445111, 4445147), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 12\r",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 11  f\b\b\r",
      "STEP     2 / 11  f\b\b\r",
      "STEP     3 / 11  f\b\b\r",
      "STEP     4 / 11  f\b\b\r",
      "STEP     5 / 11  f\b\b\r",
      "STEP     6 / 11  f\b\b\r",
      "STEP     7 / 11  f\b\b\r",
      "STEP     8 / 11  f\b\b\r",
      "STEP     9 / 11  f\b\b\r",
      "STEP    10 / 11  f\b\b\r",
      "STEP    11 / 11  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1074\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1240\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|5387|5423|+\n",
      "-cgaccacggctggtggaaacaacattatgaatggcg\n",
      ">G000006925|2479150|2517237|2513987|2514023|+\n",
      "-agatcgtgattactggcatcgcaattatgaatggcg\n",
      ">G000299455|1535847|1550961|1539067|1539103|-\n",
      "gcgaccgtgactactggcatcgcaattatgaatggc-\n",
      ">G000299455|4431865|4470166|4445111|4445147|-\n",
      "gcgaccacggctggtggaaacaacattatgaatggc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (504650, 507319)), (507173, 507292), '-'), (('G000006925', (511335, 520753)), (511695, 511814), '-'), (('G000299455', (3818101, 3818615)), (3818371, 3818490), '+'), (('G000299455', (3819953, 3834467)), (3820224, 3820343), '+')]\n",
      ">G000006925|504650|507319|507173|507292|-\n",
      "tcatctttccagacctcagagtaaacctgaaatcggaaagcttcaggcacccgcagttat\n",
      "cttacccttcctcaacaaaatagctgtttatttatacagcaaatcctgtttcggacttc-\n",
      ">G000006925|511335|520753|511695|511814|-\n",
      "tcatctttccaaacctcagagttaacctgaaaccggaaagcttcaggcacccgccgttat\n",
      "cttacccttccttagcaaaatagctgtttatttatacagtgagtcctgtttcggacttc-\n",
      ">G000299455|3818101|3818615|3818371|3818490|+\n",
      "-catctttccagacctcagagttaacctgaaaccggaaagcttcaggcacccgccgttat\n",
      "cttacccttcctcaacaaaatagctgtttatttatacagtgaatcctgtttcggacttct\n",
      ">G000299455|3819953|3834467|3820224|3820343|+\n",
      "-catctttccagacctcagagtaaacctgaaaccggaaagcttcaggcacccgcagttat\n",
      "cttacccttgcttagcaaaatagctgtttatttatacagtaagtcctgtttcggacttct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (2978710, 2978861), '-'), (('G000299455', (988799, 1028140)), (1014162, 1014313), '+'), (('G000299455', (2613922, 2614096)), (2613945, 2614096), '-')]\n",
      ">G000006925|2976303|3015537|2978710|2978861|-\n",
      "gtcgatgcagccccacggtgtgtagcccatcagatccacaccatcgtaagtcactgcttt\n",
      "tttcatctcttcaatatgggcgcgcaggtagtcaatgcggtagtcgtcgttgatgctgcc\n",
      "atcttcttccactttgtcgtaagcgccaaaa-\n",
      ">G000299455|988799|1028140|1014162|1014313|+\n",
      "-tcgatgcagccccacggtgtgtagcccatcagatccacaccatcgtaagtcactgcttt\n",
      "tttcatctcttcaatatgggcgcgcaggtagtcaatgcggtagtcgtcgttgatgctgcc\n",
      "atcttcttccactttgtcgtaagcgccaaaac\n",
      ">G000299455|2613922|2614096|2613945|2614096|-\n",
      "gtcaatgcagccccacggggtgtagccaattaagtcaacaccatcttcaactacggcttt\n",
      "tttcatttcccgaatatgggaagcaaagtaatcaatgcgatagtgatcgttcaccgtgcc\n",
      "gtcagcttgtctctggtcaaccgcaccaaat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4354245, 4357769)), (4357530, 4357551), '+'), (('G000299455', (4904188, 4907712)), (4907473, 4907494), '+'), (('G000299455', (5359167, 5359188)), (5359167, 5359188), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4354245|4357769|4357530|4357551|+\n",
      "tgatggagctttccgcactgt\n",
      ">G000299455|4904188|4907712|4907473|4907494|+\n",
      "tgatggagctttccgcactgt\n",
      ">G000299455|5359167|5359188|5359167|5359188|+\n",
      "tgatggagctttccgcactgt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3969227, 3993100)), (3990176, 3990220), '-'), (('G000006925', (4450485, 4460192)), (4457309, 4457353), '+'), (('G000299455', (3973381, 4009751)), (3996678, 3996722), '+'), (('G000299455', (4658720, 4667733)), (4665546, 4665590), '+'), (('G000299455', (5156132, 5190970)), (5169977, 5170021), '+')]\n",
      ">G000006925|3969227|3993100|3990176|3990220|-\n",
      "gtgacgataacgaaaatcgcgcctacgtagaaaatcaggatccg-\n",
      ">G000006925|4450485|4460192|4457309|4457353|+\n",
      "-acataatcacaatcagcgcgaagacgtagaacataatgatacgg\n",
      ">G000299455|3973381|4009751|3996678|3996722|+\n",
      "-acataatgacgaacagcgtaccgacgtagaacaccagaatacgc\n",
      ">G000299455|4658720|4667733|4665546|4665590|+\n",
      "-acataattacaatcagcgcgaagacgtagaacataatgatacgg\n",
      ">G000299455|5156132|5190970|5169977|5170021|+\n",
      "-tgacgataacgaaaatcgcgcctacgtagaaaatcaggatccgc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2728788, 2734132)), (2733557, 2733592), '-'), (('G000299455', (905566, 905601)), (905566, 905601), '-'), (('G000299455', (1302091, 1327808)), (1327232, 1327267), '-'), (('G000299455', (3440781, 3470001)), (3460368, 3460403), '+'), (('G000299455', (4201688, 4201857)), (4201816, 4201851), '-')]\n",
      ">G000006925|2728788|2734132|2733557|2733592|-\n",
      "gacatccggatgcgctttttccacttcatccagca-\n",
      ">G000299455|905566|905601|905566|905601|-\n",
      "gacatccggatgtgctttttcaatttcatcaagca-\n",
      ">G000299455|1302091|1327808|1327232|1327267|-\n",
      "gacatccggatgcgctttttccacttcatccagca-\n",
      ">G000299455|3440781|3470001|3460368|3460403|+\n",
      "-acgtccggatgcgctttctcgatttcgtccagcaa\n",
      ">G000299455|4201688|4201857|4201816|4201851|-\n",
      "-acatccgggtgcgctttttccacttcatcaagcag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4288513, 4315006)), (4289129, 4289150), '-'), (('G000299455', (1030502, 1031792)), (1031771, 1031792), '+'), (('G000299455', (4836149, 4864500)), (4861243, 4861264), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4288513|4315006|4289129|4289150|-\n",
      "cctgacggacagtgcttccac-\n",
      ">G000299455|1030502|1031792|1031771|1031792|+\n",
      "-ctgacgaacggtactgccacg\n",
      ">G000299455|4836149|4864500|4861243|4861264|+\n",
      "-ctgacggacagtgcttccacg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4622488, 4624460)), (4622587, 4622616), '+'), (('G000299455', (3214776, 3214831)), (3214802, 3214831), '+'), (('G000299455', (5313369, 5315826)), (5313824, 5313853), '+')]\n",
      ">G000006925|4622488|4624460|4622587|4622616|+\n",
      "aacgataacctgctcttcaactggcgcca\n",
      ">G000299455|3214776|3214831|3214802|3214831|+\n",
      "aacgataatttgctgttcaagtggcgcca\n",
      ">G000299455|5313369|5315826|5313824|5313853|+\n",
      "aacgataacctgctcttcaactggcgcca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1836945, 1842499)), (1839664, 1839688), '+'), (('G000299455', (2754964, 2760515)), (2757683, 2757707), '+'), (('G000299455', (3626532, 3650149)), (3634433, 3634457), '+')]\n",
      ">G000006925|1836945|1842499|1839664|1839688|+\n",
      "tcgcccagccccagcgcggtcatc\n",
      ">G000299455|2754964|2760515|2757683|2757707|+\n",
      "tcgcccagccccagcgcggtcatc\n",
      ">G000299455|3626532|3650149|3634433|3634457|+\n",
      "tcgcccagcgccagcgcggtcatc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1637263, 1657511)), (1637263, 1637411), '-'), (('G000299455', (2585656, 2594816)), (2585948, 2586096), '+'), (('G000299455', (2585656, 2594816)), (2588432, 2588580), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1637263|1657511|1637263|1637411|-\n",
      "gcatttctccattattgttaaatacccgcacggtatcgccatgacggataccgcgtgcct\n",
      "gagcatcaatggggttgatccacacctcttgtgggcaggcctgctgtaacacatcaatat\n",
      "tgccgtagctggagtgggtacgcgcttt-\n",
      ">G000299455|2585656|2594816|2585948|2586096|+\n",
      "-catttctccattattgttaaatacccgcacggtatcgccatgacggataccgcgagcct\n",
      "gagcatcaatggggttgatccacacctcttgtgggcaggcctgctgtaacacatcaatat\n",
      "tgccgtagctggagtgggtacgcgccttg\n",
      ">G000299455|2585656|2594816|2588432|2588580|+\n",
      "-aacttcgccacgatggttaaacacccgcaccatatcgccgttggcaatcccacgtttct\n",
      "gcgcatctatagggttgatccacacctcctgacggcaggcagccttcaggagatcaatat\n",
      "tgccgtaggtcgagtgagtacgggatttg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (608503, 616145)), (615752, 615813), '+'), (('G000299455', (3680085, 3687629)), (3687334, 3687395), '+'), (('G000299455', (4792243, 4792304)), (4792243, 4792304), '+')]\n",
      ">G000006925|608503|616145|615752|615813|+\n",
      "gcgtattaaccggcatctacatgctgctttctggcgcgatcgcgaactatctggcgggcg\n",
      "t\n",
      ">G000299455|3680085|3687629|3687334|3687395|+\n",
      "gcgtattaaccggcatctacatgctgctttctggcgcgatcgcgaactatctggcgggcg\n",
      "t\n",
      ">G000299455|4792243|4792304|4792243|4792304|+\n",
      "gcgtattaaccggtatttatatgctggcaacaggcgcggtcgccaactggctggcaggcg\n",
      "t\n",
      "\n",
      "\n",
      "3 [(('G000006925', (721893, 722007)), (721893, 722007), '+'), (('G000299455', (2170421, 2170535)), (2170421, 2170535), '+'), (('G000299455', (2921399, 2921513)), (2921399, 2921513), '-')]\n",
      ">G000006925|721893|722007|721893|722007|+\n",
      "-tcagaaaaagaggcggcttacctgcggcctgagtacgtcattacggttaacagtgccgg\n",
      "agaggaagagcataaaagtgattttattgcgtctgcaccggataaagaccaggaa\n",
      ">G000299455|2170421|2170535|2170421|2170535|+\n",
      "-tcagaaaaagaggcgtcttgtctgcagtctgagtactccattacggttaaaagtgccgg\n",
      "agaggaaggaaataaacgttattttattgcgtctgcacctgataaagatcaggaa\n",
      ">G000299455|2921399|2921513|2921399|2921513|-\n",
      "gtcagaaaaagaggcgtcttgtctgcagtctgagtactccattacggttaaaagtgccgg\n",
      "agaggaaggaaataaacgttattttattgcgtctgcacctgataaagatcagga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3701718, 3701950), '+'), (('G000299455', (269622, 287922)), (277462, 277694), '+'), (('G000299455', (3918464, 3964500)), (3932624, 3932856), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3693106|3702974|3701718|3701950|+\n",
      "-ataaaccgtcttcagacttgcggggaaataggctgataagcggttcagctcctctttga\n",
      "ctgcccgcgaggtatccagggcgtttgctccggcagccagtttgatggcgatcccggcag\n",
      "caggtttgccgttatagcgcgccacggtggaataatcttccgccccaagttcgacgcggg\n",
      "cgacatcacgcagcagcacttgcgaaccatcttgctgaactttcaacaggatt\n",
      ">G000299455|269622|287922|277462|277694|+\n",
      "-ataaaccgtcttcagacttgcggggaaataggctgataagcggttcagctcctctttga\n",
      "ctgcccgcgaggtatccagggcgtttgctccggtagccagtttgatggcgatcccggcag\n",
      "caggtttgccgttatagcgcgccacggtggaataatcttccgctccaagttcgacgcggg\n",
      "cgacatcacgcagcagcacttgcgaaccatcttgctgaactttcaacaggatt\n",
      ">G000299455|3918464|3964500|3932624|3932856|-\n",
      "ggtaaacaattttcagacctgacgggaagaacggttccatcttcgccagttcagcacgga\n",
      "ttgccgcagcggtatccagcgcgtttgcaccggtcgccagcttgatccccagaccggaag\n",
      "ccggttggccgttaaactctgcgatgatgtcgtagttctcaccacccagctcaatcttcg\n",
      "ccacatcacgcagcagtacgcgggaaccatcctgattcactttcagcaggat-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1886975, 1912610)), (1903664, 1903802), '-'), (('G000006925', (4798846, 4802842)), (4802122, 4802260), '+'), (('G000299455', (2199778, 2263225)), (2208466, 2208604), '+'), (('G000299455', (4560768, 4564764)), (4564044, 4564182), '+')]\n",
      ">G000006925|1886975|1912610|1903664|1903802|-\n",
      "atgtttgccaccgcgccgcaagcgatggcaatgatggctgagttggcaatacgcgggccg\n",
      "gagaaaattcagccgcgcgttggctggcaagggctggagatcatcgaagagatgcggcgt\n",
      "aataacgagaaagttatc-\n",
      ">G000006925|4798846|4802842|4802122|4802260|+\n",
      "-tgtttgccacagcattgcaatccatagtgatgatggcagaactggcaattcgtggtccg\n",
      "gaaaagttccagaaacgagtgttctggaaaggcctcgaaattcttgaggaaattcggcac\n",
      "aataacagaaatgtgattt\n",
      ">G000299455|2199778|2263225|2208466|2208604|+\n",
      "-tgtttgccaccgcgccgcaagcgatggcaatgatggctgagttggcaatacgcggaccg\n",
      "gagaaaattcagccgcgcgttgactggcaagggctggagatcatcgaagagatgaggcgt\n",
      "aataacgagaaagttatct\n",
      ">G000299455|4560768|4564764|4564044|4564182|+\n",
      "-tgtttgccacagcattgcaatccatagtgttgatggcagaactggcaattcgtggtccg\n",
      "gaaaagttccagaaacgagtgttctggaaaggcctcgaaattcttgaggaaattcgacac\n",
      "aataatagaaatgtgattt\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1696422, 1718400)), (1712617, 1713278), '+'), (('G000006925', (1815406, 1827803)), (1816016, 1816677), '+'), (('G000006925', (2006629, 2008384)), (2007600, 2008262), '+'), (('G000299455', (2090586, 2097706)), (2090586, 2091247), '-'), (('G000299455', (2269177, 2270929)), (2269299, 2269960), '-'), (('G000299455', (2737179, 2754578)), (2738148, 2738804), '+'), (('G000299455', (4667815, 4670271)), (4669198, 4669859), '+')]\n",
      ">G000006925|1696422|1718400|1712617|1713278|+\n",
      "-gaagaagtttttgtaagcccgctcaaggtctttcagtgactgttgcaatggctgtgagg\n",
      "gagaatctttaagccattgcgtttcagtggcatttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgacgtgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacactcctgttgac\n",
      "cacctggtcttaactggaatttaaatgcctgtaatcgtttcacaggtcaaatgatacttt\n",
      "ggtctgtgaaaaaagaaaccgatatccggcgtggaaggcattgtgttttcctgatgcatg\n",
      "ttcacctagtctttgtcaccagataccgacgccagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgcttattttgaagctgaactggttgaaatgg\n",
      "atggcgaaccagatcacgtccatttgttaattaactaacctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaggcgtatcgggtaggttactgcgacgagatcgacca\n",
      "gata-tgcagtcaggtattactacaaaaggcgttttgtggagtcctggctattttgccag\n",
      "tagctgc\n",
      ">G000006925|1815406|1827803|1816016|1816677|+\n",
      "-gaagaagtttttgtaagcccgctcaaggtctttcagtgactgttgcaatggctgtgagg\n",
      "gagaatctttaagccattgcgtttcagtggcatttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgacgtgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacactcctgttgac\n",
      "cacctggtcttaactggaatttaaatgcctgtaatcgtttcacaggtcaaatgatacttt\n",
      "ggtctgtgaaaaaagaaaccgatatccggcgtggaaggcattgtgttttcctgatgcatg\n",
      "ttcacctagtctttgtcaccagataccgacgccagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgcttattttgaagctgaactggttgaaatgg\n",
      "atggcgaaccagatcacgtccatttgttaattaactaacctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaggcgtatcgggtaggttactgcgacgagatcgacca\n",
      "gatattgcagtcaggtattactac-aaaggcgttttgtggagtcctggctattttgccag\n",
      "tagctgc\n",
      ">G000006925|2006629|2008384|2007600|2008262|+\n",
      "-gaaaaagtttttgcaggcctgctcagggtctttcagtgactgttgcaatggctgtgacg\n",
      "gagcatctttaatccattgcgtttcagtggcatttttccactcaaccaatcaggaagcca\n",
      "ttttcgcgtaagggatgtctttgttaccggccttatgattctcattctgacgtgccagag\n",
      "tacgactgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacgcgcctgttgac\n",
      "caccgggtcttaaccggaatttgaatacctgtagtcgtttcacagggcaaattatacttt\n",
      "ggtctgtgaaaa---aaactgatatccgacgcggaaggcactgtgttttcctgatgcatg\n",
      "ctcaccgggtctttgttaccaagtaccgacgccagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgcttattttgaagctgaactggttgaaatgg\n",
      "atggcgaaccagatcacgtccatctgttaattaactatccccccaaactggcgatatcca\n",
      "gtctggtaaacaaacagcctcaaaggcgtatcgggtaggttactgcgacgggatcgacca\n",
      "gatattgctgccaggtattaccgc-aaagatgttttgtggcgtcccggctattttgccag\n",
      "tagctgc\n",
      ">G000299455|2090586|2097706|2090586|2091247|-\n",
      "ggaagaagtttttgtaagcccgctcagggtctttcagtgactgttgcaatggctgtgagg\n",
      "gcgaatctttaagccattgcgtttcagtggcatttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgaagcgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacgctcctgttgac\n",
      "cacctggtcttaaccggaatttgaatgcctgtagtcgtttcatgggccaaatcatacttt\n",
      "ggcctatgaaaaaagaaaccgatatccgacgcggaaggcattgtgttttcctgaagcatg\n",
      "ttcacctagtcttcgttactaaataccgacgtcagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgctgattttgaagctgaacgggttgaaatgg\n",
      "atggcggaccagatcacgtccatttgttaattaactatcctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaggcgtatcgagtaggttactgcgacgagatcgacca\n",
      "gatattgcagtcaggtattacgac-aaaggcgttttgtggagtcctggctattttgtcag\n",
      "tagctg-\n",
      ">G000299455|2269177|2270929|2269299|2269960|-\n",
      "ggaagaagtttttgtaagcccgctcaaggtctttcagtgactgttgcaatggctgtgagg\n",
      "gcgaatctttaagccattgcgtttcagtggcgtttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgacgtgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacactcctgttgac\n",
      "cacctggtcttaactggaatttaaatgcctgtaatcgtttcacaggtcaaatgatacttt\n",
      "ggtctgtgaaaaaagaaaccgatatccggcgtggaaggcattgtgttttcctgatgcatg\n",
      "ttcacctggtctttgtcaccagataccgactccagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgctgattttgaagctgaacttgttgaaatgg\n",
      "atggcgaaccagatcacgtccatttgttaattaactatcctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaggcgtatcgggtaggttactgcgacgagatcgacca\n",
      "gatattgcagtcaggtattactac-aaaggcgttttgtggagtcctggctattttgccag\n",
      "tagctg-\n",
      ">G000299455|2737179|2754578|2738148|2738804|+\n",
      "-gaagaagtttttgtaagcccgctcaaggtctttcagtgactgttgcaatggctgtgagg\n",
      "gagaatctttaagccattgcgtttcagtggcatttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgacgtgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacactcctgttgac\n",
      "cacctggtcttaactggaatttaaatgcctgtaatcgtttcacaggtcaaatgatacttt\n",
      "ggtctgtgaataaagaaaccgatatccggcgtggaaggcattgtgttttcctgatgcatg\n",
      "ttcacctggtctttgtcaccagataccgacgccagattttt-----tgacgcgacagaaa\n",
      "aactacgcacttacttttcaaatgtatgtgcttattttgaagctgaactggttgaaatgg\n",
      "atggcgaaccagatcacgtccatttgttaattaactaacctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaagcgtatcgggtaggttactgcgacgagatcgacca\n",
      "gatattgcagtcaggtattactac-aaaggcgttttgtggagtcctggctattttgccag\n",
      "tagctgc\n",
      ">G000299455|4667815|4670271|4669198|4669859|+\n",
      "-gaagaagtttttgtaagcccgctcaaggtctttcagtgactgttgcaatggctgtgacg\n",
      "gggcatctttaagccattgcgtttcagtggcatttttccactcaaccagccaggaagcca\n",
      "ttttcccgtaagggatgtatttattaccggcctcatgattctcattctgaagcgccagag\n",
      "cacgattgaaaacgaaacgacatgcgcctgcgaagcgcctcatttcacgctcctgttgac\n",
      "cacctggtcttaactggaatttaaatgcctgtagtcgtttcacaggccaaatgatacttt\n",
      "ggtctgtgaaaaaagaaaccgatatccggcgtggaaggcattgtgttttcctgatgcatg\n",
      "ttcacctggtctttgtcaccaaataccgacgccagatttttgaccatgacgcgacagaaa\n",
      "aactacgcacttacttttcaaaggtatgtgctgattttgaagctgaactggttgaaatgg\n",
      "atggcgaaccagatcacgtccatttgttaattaactatcctcccaaactggcgatatcca\n",
      "gtctggt----aaacagcctcaaaggcgtatcgggtaggttactgcgacgagatcgacca\n",
      "gatattgcagtcaggtattactac-aaaggcgttttgtggagtcctggctattttgccag\n",
      "tagctgc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3107661, 3117031)), (3108922, 3109011), '+'), (('G000006925', (3736321, 3788106)), (3750270, 3750359), '-'), (('G000006925', (4535976, 4543278)), (4541997, 4542086), '+'), (('G000299455', (137973, 154383)), (149451, 149540), '+'), (('G000299455', (830050, 839519)), (838073, 838162), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1180\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1180\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3107661|3117031|3108922|3109011|+\n",
      "-aatcagtaacacctgcaaacgctgatcgtcggtgatggagataaccgagctgcggataa\n",
      "tatcgaactgcccgctggcaacggttaatt\n",
      ">G000006925|3736321|3788106|3750270|3750359|-\n",
      "cgacgatcagcatttgcagacgctggtcaggggttatcgaaagaatagacgagcgaatga\n",
      "tgtcaaactgcccggttttcaccgagatc-\n",
      ">G000006925|4535976|4543278|4541997|4542086|+\n",
      "-gacgatcagcatttgcagacgctggtcaggggttatcgaaagaatagacgagcgaatga\n",
      "tgtcaaactgcccggttttcaccgagatct\n",
      ">G000299455|137973|154383|149451|149540|+\n",
      "-gacgatcagcatttgcagacgctggtcaggggttatcgaaagaatagacgagcgaatga\n",
      "tgtcaaactgcccggttttcaccgagatct\n",
      ">G000299455|830050|839519|838073|838162|-\n",
      "caatcagtaacacctgcaaacgctgatcgtcggtgatggagataaccgagctgcggataa\n",
      "tatcgaactgaccgctggcaacggttaat-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2888115, 2906456)), (2904347, 2904420), '+'), (('G000299455', (1078152, 1154001)), (1121432, 1121505), '-'), (('G000299455', (2416691, 2440660)), (2429255, 2429328), '+')]\n",
      ">G000006925|2888115|2906456|2904347|2904420|+\n",
      "-tgatggttgatggtgctcaggggacggtgcatttccccgcggatgttcagcaactggat\n",
      "atcgatttctatgc\n",
      ">G000299455|1078152|1154001|1121432|1121505|-\n",
      "gtgatggttgatggtgctcagggggcagtgcatttccccgcggatgttcagcaactggat\n",
      "attgatttctatg-\n",
      ">G000299455|2416691|2440660|2429255|2429328|+\n",
      "-tgctggtggatggtgctcaggcggtgatgcatcatccggtggatgttcaggcgctggat\n",
      "tgcgacttttacgt\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1965007, 1976679)), (1968745, 1968766), '+'), (('G000006925', (1965007, 1976679)), (1970446, 1970467), '+'), (('G000006925', (4557883, 4573403)), (4558802, 4558823), '-'), (('G000299455', (2122044, 2133716)), (2128256, 2128277), '-'), (('G000299455', (2122044, 2133716)), (2129957, 2129978), '-'), (('G000299455', (2737179, 2754578)), (2748710, 2748731), '+'), (('G000299455', (4487440, 4499283)), (4498343, 4498364), '+')]\n",
      ">G000006925|1965007|1976679|1968745|1968766|+\n",
      "-acttgagagatcgttattgcc\n",
      ">G000006925|1965007|1976679|1970446|1970467|+\n",
      "-ggaggaaagatcggtgttgcc\n",
      ">G000006925|4557883|4573403|4558802|4558823|-\n",
      "gcgaagagagatcattattgc-\n",
      ">G000299455|2122044|2133716|2128256|2128277|-\n",
      "gggaggaaagatcggtgttgc-\n",
      ">G000299455|2122044|2133716|2129957|2129978|-\n",
      "gacttgagagatcgttattgc-\n",
      ">G000299455|2737179|2754578|2748710|2748731|+\n",
      "-agatgacaggtccgcattgcc\n",
      ">G000299455|4487440|4499283|4498343|4498364|+\n",
      "-cgaagagagatcgttattgcc\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1509064, 1578296)), (1575941, 1575973), '+'), (('G000299455', (2169561, 2170070)), (2169882, 2169914), '-'), (('G000299455', (2550010, 2550272)), (2550085, 2550117), '-'), (('G000299455', (2921864, 2922373)), (2922020, 2922052), '+'), (('G000299455', (3288518, 3288775)), (3288673, 3288705), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\n",
      "Reallocating..done. *alloclen = 1044\n",
      "\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1575941|1575973|+\n",
      "-cgttcccaccgctgacgtggtgtatgacacac\n",
      ">G000299455|2169561|2170070|2169882|2169914|-\n",
      "ccgttcccaccgctgacgtggtgtatgacaca-\n",
      ">G000299455|2550010|2550272|2550085|2550117|-\n",
      "ccgttcccaccgctgacgtggtgtatgacaca-\n",
      ">G000299455|2921864|2922373|2922020|2922052|+\n",
      "-cgttcccaccgctgacgtggtgtatgacacac\n",
      ">G000299455|3288518|3288775|3288673|3288705|+\n",
      "-cgttcccaccgctgacgtggtgtatgacacac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3205438, 3224454)), (3214397, 3214426), '-'), (('G000299455', (715333, 739463)), (730453, 730482), '+'), (('G000299455', (4050418, 4050447)), (4050418, 4050447), '-')]\n",
      ">G000006925|3205438|3224454|3214397|3214426|-\n",
      "gccggattcattgcccagcgaccagatga-\n",
      ">G000299455|715333|739463|730453|730482|+\n",
      "-ccggattcattgcccagcgaccagatgat\n",
      ">G000299455|4050418|4050447|4050418|4050447|-\n",
      "gcctgattcattccccagcgaccagatga-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1103067, 1122318)), (1103443, 1103491), '+'), (('G000006925', (2479150, 2517237)), (2499542, 2499590), '-'), (('G000299455', (1551165, 1574182)), (1553714, 1553762), '+'), (('G000299455', (3103316, 3122569)), (3122145, 3122193), '-')]\n",
      ">G000006925|1103067|1122318|1103443|1103491|+\n",
      "-cgggcgtgttttaaagcgacggtgtaaccacatatactgctctggtgc\n",
      ">G000006925|2479150|2517237|2499542|2499590|-\n",
      "ccgggcgcgttttaaagcgacggtggatccagaggtactgctccggtg-\n",
      ">G000299455|1551165|1574182|1553714|1553762|+\n",
      "-cgggcgcgttttaaagcgacggtggatccagaggtactgctccggtgc\n",
      ">G000299455|3103316|3122569|3122145|3122193|-\n",
      "ccgggcgtgttttaaagcgacggtgtaaccacatatactgctctggtg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3637330, 3640023)), (3639154, 3639185), '+'), (('G000299455', (262744, 269475)), (268389, 268420), '-'), (('G000299455', (2983402, 3007859)), (3006377, 3006408), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1098\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3637330|3640023|3639154|3639185|+\n",
      "-tcaaccgttactgggacgatcgcgacacgcc\n",
      ">G000299455|262744|269475|268389|268420|-\n",
      "ctcaaccgttattgggacgatcgcgacacgc-\n",
      ">G000299455|2983402|3007859|3006377|3006408|+\n",
      "-tcaaccgctactgggacgatcgcgatacgcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (722910, 723312)), (722920, 722981), '+'), (('G000006925', (901434, 901838)), (901619, 901680), '+'), (('G000299455', (3607589, 3607707)), (3607636, 3607697), '-')]\n",
      ">G000006925|722910|723312|722920|722981|+\n",
      "-gtcaccgcgattatctccgctctggttatctgcatcatcgtctgcctgtcatgggctgt\n",
      "ta\n",
      ">G000006925|901434|901838|901619|901680|+\n",
      "-gtcaccgcgattatctccgctctggttatctgcatcatcgtctgcctgtcatgggctgt\n",
      "ta\n",
      ">G000299455|3607589|3607707|3607636|3607697|-\n",
      "agtaactgcgattatcttcgctctggttatctgcatcatcgtctgcctgtcatgggctgt\n",
      "t-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (311360, 311542)), (311364, 311542), '+'), (('G000006925', (317354, 317532)), (317354, 317532), '-'), (('G000006925', (327498, 328010)), (327827, 328005), '-'), (('G000299455', (2525864, 2526376)), (2525868, 2526046), '+')]\n",
      ">G000006925|311360|311542|311364|311542|+\n",
      "-tttgccgtgcgtgttcggtaaggtgattaggtgcgaggtgagcatatcgacgaaccatt\n",
      "tcgatagactcccagcctcccatttcctgtaacactgataacgggactccggcttgaacc\n",
      "agccaacttgcccaggtgtgtctcaagtcgtgaaatctgaaatcatcaataccagcccg\n",
      ">G000006925|317354|317532|317354|317532|-\n",
      "attttgctcgcgtgctcggtaagatgcccggctgaaaggtgggcgtatctttgaaccatt\n",
      "tcgagagtttcccatcctcccatctctttaagcgcaagaagagagacaccggactgaacc\n",
      "agccagcttgcccaggtatgcctcaggtcatggaagcggaagttgctaatgcctgccc-\n",
      ">G000006925|327498|328010|327827|328005|-\n",
      "-tttgccgtgcgtgttcggtaaggtgattaggtgcaaggtgagcatatcgacgaaccatt\n",
      "tcgatagactcccagcctcccatttcctgtaacactgacaacgggactccggcttgaacc\n",
      "agccaacttgcccaggtgtgtctcaagtcgtgaaatctgaaatcatcaatacctgctcg\n",
      ">G000299455|2525864|2526376|2525868|2526046|+\n",
      "-ttttcctcgcgtgctctgtcaaatgattaggtgcaaggtgtgcgtacctacgcaccatt\n",
      "tcgatagactcccagccgcccatttcctgaagtactgacagtggtacacctgactgaatt\n",
      "aaccaactggcccaggtatggcggaggtcatgaaaacggaaattttcaattcctgcccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3027894, 3027941), '+'), (('G000299455', (956352, 970820)), (965108, 965155), '-'), (('G000299455', (1078152, 1154001)), (1084954, 1085001), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3016790|3036697|3027894|3027941|+\n",
      "-cggaaaccaaacacgtttcgctggaacatattgaacgtaatctgatg\n",
      ">G000299455|956352|970820|965108|965155|-\n",
      "ccggaaaccaaacacgtttcgctggaacatattgaacgtaatctgat-\n",
      ">G000299455|1078152|1154001|1084954|1085001|+\n",
      "-cggaaaccaaaaatgtcacgctggaacatatcgaacgcaaactgatg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (712060, 712136)), (712060, 712136), '-'), (('G000006925', (721099, 721175)), (721099, 721175), '-'), (('G000006925', (1936889, 1936965)), (1936889, 1936965), '+'), (('G000006925', (2060921, 2060997)), (2060921, 2060997), '+'), (('G000006925', (4188297, 4209378)), (4195423, 4195499), '-'), (('G000299455', (4952297, 4970010)), (4966175, 4966251), '+')]\n",
      ">G000006925|712060|712136|712060|712136|-\n",
      "tggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgca-\n",
      ">G000006925|721099|721175|721099|721175|-\n",
      "tggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgca-\n",
      ">G000006925|1936889|1936965|1936889|1936965|+\n",
      "-ggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgcat\n",
      ">G000006925|2060921|2060997|2060921|2060997|+\n",
      "-ggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgcat\n",
      ">G000006925|4188297|4209378|4195423|4195499|-\n",
      "tggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgca-\n",
      ">G000299455|4952297|4970010|4966175|4966251|+\n",
      "-ggagcgggcagcgggaatcgaacccgcatcatcagcttggaaggctgaggtaatagcca\n",
      "ttatacgatgcccgcat\n",
      "\n",
      "\n",
      "19 [(('G000006925', (385776, 386112)), (386030, 386083), '-'), (('G000006925', (899355, 899691)), (899384, 899437), '+'), (('G000006925', (2059291, 2059627)), (2059545, 2059598), '-'), (('G000006925', (3421586, 3421922)), (3421615, 3421668), '+'), (('G000006925', (3851554, 3851890)), (3851808, 3851861), '-'), (('G000006925', (4359551, 4359859)), (4359785, 4359838), '-'), (('G000006925', (4622488, 4624460)), (4622908, 4622961), '+'), (('G000006925', (4700925, 4701241)), (4701159, 4701212), '-'), (('G000006925', (4776578, 4776894)), (4776812, 4776865), '-'), (('G000006925', (4781327, 4781663)), (4781356, 4781409), '+'), (('G000299455', (1991020, 1991316)), (1991041, 1991094), '+'), (('G000299455', (3215105, 3215383)), (3215123, 3215176), '+'), (('G000299455', (4543449, 4543753)), (4543478, 4543531), '+'), (('G000299455', (4571312, 4571569)), (4571358, 4571411), '+'), (('G000299455', (4752483, 4752699)), (4752529, 4752582), '+'), (('G000299455', (5289627, 5289931)), (5289849, 5289902), '-'), (('G000299455', (5313369, 5315826)), (5314145, 5314198), '+'), (('G000299455', (5316432, 5316648)), (5316549, 5316602), '-'), (('G000299455', (5333884, 5334100)), (5333930, 5333983), '+')]\n",
      ">G000006925|385776|386112|386030|386083|-\n",
      "atcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacac-\n",
      ">G000006925|899355|899691|899384|899437|+\n",
      "-tcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacacg\n",
      ">G000006925|2059291|2059627|2059545|2059598|-\n",
      "atcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacac-\n",
      ">G000006925|3421586|3421922|3421615|3421668|+\n",
      "-tcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacacg\n",
      ">G000006925|3851554|3851890|3851808|3851861|-\n",
      "atcacagatatgcgttgtggcttcaatggcctagcttcgaaggtgcagaacac-\n",
      ">G000006925|4359551|4359859|4359785|4359838|-\n",
      "atcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacac-\n",
      ">G000006925|4622488|4624460|4622908|4622961|+\n",
      "-tcaccgatatgcgaaatggctttaacggcctggtctcaaaagttcagaacgtc\n",
      ">G000006925|4700925|4701241|4701159|4701212|-\n",
      "atcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacac-\n",
      ">G000006925|4776578|4776894|4776812|4776865|-\n",
      "atcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacac-\n",
      ">G000006925|4781327|4781663|4781356|4781409|+\n",
      "-tcacagatatgcgttgtggcttcaatggcctggcttcgaaggtgcagaacacg\n",
      ">G000299455|1991020|1991316|1991041|1991094|+\n",
      "-tcaccgatatgcgcaacggcttcaatggcctcgccgcaaaggtgcagaccgcg\n",
      ">G000299455|3215105|3215383|3215123|3215176|+\n",
      "-tcaccgatatgagaaacggcttcaacggcctggcggcaaaggtgcagacgacg\n",
      ">G000299455|4543449|4543753|4543478|4543531|+\n",
      "-tcaccgatatgagaaatggcttcaacggcctggctgcgaaagtacagacggcg\n",
      ">G000299455|4571312|4571569|4571358|4571411|+\n",
      "-ttaccgatatgcgtaaatccttcaacggactgggagaacaggtacaacatgtg\n",
      ">G000299455|4752483|4752699|4752529|4752582|+\n",
      "-ttaccgatatgcgtaaatccttcaacggactgggagaacaggtacaacatgtg\n",
      ">G000299455|5289627|5289931|5289849|5289902|-\n",
      "atcaccgatatgagaaatggcttcaacggcctggctgcgaaagtacagacggc-\n",
      ">G000299455|5313369|5315826|5314145|5314198|+\n",
      "-tcaccgatatgcgaaatggctttaacggcctggcatcaaaagttcagaacgtc\n",
      ">G000299455|5316432|5316648|5316549|5316602|-\n",
      "gttactgatatgcgtaaatccttcaacggtctgggggagcagatacagcatgt-\n",
      ">G000299455|5333884|5334100|5333930|5333983|+\n",
      "-ttaccgatatgcgtaaatccttcaatggactgggagaacaggtacaatatgtg\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1582509, 1591522)), (1585799, 1585885), '+'), (('G000006925', (4367703, 4374506)), (4370517, 4370603), '-'), (('G000299455', (2596243, 2603992)), (2598269, 2598355), '+'), (('G000299455', (4524008, 4535561)), (4531572, 4531658), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\n",
      "Reallocating..done. *alloclen = 1108\n",
      "\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\n",
      "Reallocating..done. *alloclen = 1108\n",
      "\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1582509|1591522|1585799|1585885|+\n",
      "-atcagcacattaatgactgtatggaagatgaacattatcgtcatgcggcgtatggcatg\n",
      "atgttgcaggaacaagcgccgacgctg\n",
      ">G000006925|4367703|4374506|4370517|4370603|-\n",
      "tatgaaaccattgccgacaccatgactaacccggattatcgcaaagcggcctttgccctg\n",
      "atgatgcaggaacaagcgccaacgct-\n",
      ">G000299455|2596243|2603992|2598269|2598355|+\n",
      "-atcagcacattaatgactgcatggaagatgaacattatcgttatgcggcgtatggcttg\n",
      "atgttgcaggaacaagcgccgacgctg\n",
      ">G000299455|4524008|4535561|4531572|4531658|-\n",
      "tatgaaactattgccgacaccatgactaacccggattatcgcaaagcggcctttgccctg\n",
      "atgatgcaggaacaagcgccaacgct-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (64491, 216702)), (109540, 109573), '-'), (('G000006925', (1977451, 1998377)), (1987259, 1987292), '+'), (('G000299455', (2104790, 2119714)), (2109862, 2109895), '-'), (('G000299455', (2959070, 2965951)), (2964757, 2964790), '-'), (('G000299455', (4334060, 4383441)), (4339171, 4339204), '+')]\n",
      ">G000006925|64491|216702|109540|109573|-\n",
      "attttttaccagaaccgcacgggcaaggatcgt-\n",
      ">G000006925|1977451|1998377|1987259|1987292|+\n",
      "-tttcttaccactgccgcacggacaaggatcgtt\n",
      ">G000299455|2104790|2119714|2109862|2109895|-\n",
      "atttcttaccactaccgcacgggcaaggatcgt-\n",
      ">G000299455|2959070|2965951|2964757|2964790|-\n",
      "attttttacctgaaccacaagggcagggatcgt-\n",
      ">G000299455|4334060|4383441|4339171|4339204|+\n",
      "-ttttttaccagaaccgcacgggcaaggatcgtt\n",
      "\n",
      "\n",
      "7 [(('G000006925', (900498, 900638)), (900567, 900604), '+'), (('G000006925', (3422729, 3422869)), (3422798, 3422835), '+'), (('G000006925', (4358604, 4358744)), (4358637, 4358675), '-'), (('G000006925', (4775631, 4775771)), (4775664, 4775702), '-'), (('G000006925', (4782470, 4782610)), (4782539, 4782576), '+'), (('G000299455', (4572495, 4572632)), (4572552, 4572589), '+'), (('G000299455', (4749820, 4751622)), (4751091, 4751128), '+')]\n",
      ">G000006925|900498|900638|900567|900604|+\n",
      "-tgtgggtatacgtcagggatgatcgtaatgcgggttc\n",
      ">G000006925|3422729|3422869|3422798|3422835|+\n",
      "-tgtgggtatacgtcagggatgatcgtaatgcgggttc\n",
      ">G000006925|4358604|4358744|4358637|4358675|-\n",
      "ctgtgggtatacgtcagggatgatcgtaatgcgggttc\n",
      ">G000006925|4775631|4775771|4775664|4775702|-\n",
      "ctgtgggtatacgtcagggatgatcgtaatgcgggttc\n",
      ">G000006925|4782470|4782610|4782539|4782576|+\n",
      "-tgtgggtatacgtcagggatgatcgtaatgcgggttc\n",
      ">G000299455|4572495|4572632|4572552|4572589|+\n",
      "-tctggacgtatgtccgggatgatcgcaatgtgggttc\n",
      ">G000299455|4749820|4751622|4751091|4751128|+\n",
      "-tgtgggtctatgtccgtgatgaccgtaacgccggttc\n",
      "\n",
      "\n",
      "30 [(('G000006925', (251358, 252232)), (251595, 251634), '+'), (('G000006925', (315068, 315858)), (315665, 315704), '-'), (('G000006925', (332059, 337870)), (332656, 332695), '-'), (('G000006925', (493019, 493813)), (493616, 493655), '-'), (('G000006925', (509301, 510374)), (509899, 509938), '-'), (('G000006925', (724190, 725876)), (724788, 724827), '-'), (('G000006925', (1094884, 1095679)), (1095481, 1095520), '-'), (('G000006925', (1099363, 1100153)), (1099516, 1099555), '+'), (('G000006925', (1411510, 1412350)), (1412157, 1412196), '-'), (('G000006925', (1883764, 1885448)), (1884362, 1884401), '-'), (('G000006925', (1956633, 1961076)), (1957231, 1957270), '-'), (('G000006925', (2084425, 2085215)), (2084578, 2084617), '+'), (('G000006925', (2236395, 2237421)), (2236784, 2236823), '+'), (('G000006925', (3065287, 3076770)), (3075361, 3075400), '-'), (('G000006925', (3812816, 3813607)), (3813413, 3813452), '-'), (('G000006925', (4639759, 4640549)), (4640356, 4640395), '-'), (('G000006925', (4650084, 4650878)), (4650241, 4650280), '+'), (('G000006925', (4656735, 4658205)), (4657568, 4657607), '+'), (('G000006925', (4673259, 4674053)), (4673416, 4673455), '+'), (('G000006925', (4679888, 4680760)), (4680486, 4680525), '-'), (('G000006925', (4750261, 4751054)), (4750417, 4750456), '+'), (('G000006925', (4753880, 4754990)), (4754797, 4754836), '-'), (('G000006925', (4760730, 4761525)), (4760888, 4760927), '+'), (('G000006925', (4793569, 4794365)), (4793725, 4793764), '+'), (('G000299455', (906289, 909876)), (906448, 906487), '+'), (('G000299455', (912160, 912572)), (912317, 912356), '+'), (('G000299455', (4752701, 4766669)), (4752860, 4752899), '+'), (('G000299455', (4771847, 4785261)), (4780310, 4780349), '-'), (('G000299455', (5275700, 5279438)), (5279090, 5279129), '-'), (('G000299455', (5343200, 5343533)), (5343462, 5343501), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 30\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 30\r",
      "   10 / 30\r",
      "   20 / 30\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 29  f\b\b\r",
      "STEP     2 / 29  f\b\b\r",
      "STEP     3 / 29  f\b\b\r",
      "STEP     4 / 29  f\b\b\r",
      "STEP     5 / 29  f\b\b\r",
      "STEP     6 / 29  f\b\b\r",
      "STEP     7 / 29  f\b\b\r",
      "STEP     8 / 29  f\b\b\r",
      "STEP     9 / 29  f\b\b\r",
      "STEP    10 / 29  f\b\b\r",
      "STEP    11 / 29  f\b\b\r",
      "STEP    12 / 29  f\b\b\r",
      "STEP    13 / 29  f\b\b\r",
      "STEP    14 / 29  f\b\b\r",
      "STEP    15 / 29  f\b\b\r",
      "STEP    16 / 29  f\b\b\r",
      "STEP    17 / 29  f\b\b\r",
      "STEP    18 / 29  f\b\b\r",
      "STEP    19 / 29  f\b\b\r",
      "STEP    20 / 29  f\b\b\r",
      "STEP    21 / 29  f\b\b\r",
      "STEP    22 / 29  f\b\b\r",
      "STEP    23 / 29  f\b\b\r",
      "STEP    24 / 29  f\b\b\r",
      "STEP    25 / 29  f\b\b\r",
      "STEP    26 / 29  f\b\b\r",
      "STEP    27 / 29  f\b\b\n",
      "Reallocating..done. *alloclen = 1080\n",
      "\r",
      "STEP    28 / 29  f\b\b\r",
      "STEP    29 / 29  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 30\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 30\r",
      "   10 / 30\r",
      "   20 / 30\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 29  f\b\b\r",
      "STEP     2 / 29  f\b\b\r",
      "STEP     3 / 29  f\b\b\r",
      "STEP     4 / 29  f\b\b\r",
      "STEP     5 / 29  f\b\b\r",
      "STEP     6 / 29  f\b\b\r",
      "STEP     7 / 29  f\b\b\r",
      "STEP     8 / 29  f\b\b\r",
      "STEP     9 / 29  f\b\b\r",
      "STEP    10 / 29  f\b\b\r",
      "STEP    11 / 29  f\b\b\r",
      "STEP    12 / 29  f\b\b\r",
      "STEP    13 / 29  f\b\b\r",
      "STEP    14 / 29  f\b\b\r",
      "STEP    15 / 29  f\b\b\r",
      "STEP    16 / 29  f\b\b\r",
      "STEP    17 / 29  f\b\b\r",
      "STEP    18 / 29  f\b\b\r",
      "STEP    19 / 29  f\b\b\r",
      "STEP    20 / 29  f\b\b\r",
      "STEP    21 / 29  f\b\b\r",
      "STEP    22 / 29  f\b\b\r",
      "STEP    23 / 29  f\b\b\r",
      "STEP    24 / 29  f\b\b\r",
      "STEP    25 / 29  f\b\b\r",
      "STEP    26 / 29  f\b\b\r",
      "STEP    27 / 29  f\b\b\r",
      "STEP    28 / 29  f\b\b\r",
      "STEP    29 / 29  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|251358|252232|251595|251634|+\n",
      "-gttccattgccccaaagattggctgtacgccggagactc\n",
      ">G000006925|315068|315858|315665|315704|-\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|332059|337870|332656|332695|-\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|493019|493813|493616|493655|-\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|509301|510374|509899|509938|-\n",
      "tgttccattgccccaaagattggctgtacgccggagact-\n",
      ">G000006925|724190|725876|724788|724827|-\n",
      "tgttccattgctccaaagattggctgtacgccggagact-\n",
      ">G000006925|1094884|1095679|1095481|1095520|-\n",
      "-gttccattgccccaaagattggctgtacgccggagactc\n",
      ">G000006925|1099363|1100153|1099516|1099555|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|1411510|1412350|1412157|1412196|-\n",
      "-gttccattgccccaaagattggctgtacgccggagactc\n",
      ">G000006925|1883764|1885448|1884362|1884401|-\n",
      "tgttccattgctccaaagattggctgtacgccggagact-\n",
      ">G000006925|1956633|1961076|1957231|1957270|-\n",
      "tgttccattgccccaaagattggctgtacgccggagact-\n",
      ">G000006925|2084425|2085215|2084578|2084617|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|2236395|2237421|2236784|2236823|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|3065287|3076770|3075361|3075400|-\n",
      "tgttctattgctccaaagattggctgtacgccggagact-\n",
      ">G000006925|3812816|3813607|3813413|3813452|-\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4639759|4640549|4640356|4640395|-\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4650084|4650878|4650241|4650280|+\n",
      "-gttccattgccccaaagattggctgtacgccggagactc\n",
      ">G000006925|4656735|4658205|4657568|4657607|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4673259|4674053|4673416|4673455|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4679888|4680760|4680486|4680525|-\n",
      "tgttccattgctccaaagattggctgtacgccggagact-\n",
      ">G000006925|4750261|4751054|4750417|4750456|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4753880|4754990|4754797|4754836|-\n",
      "-gttccattgccccaaagattggctgtatactggaaactc\n",
      ">G000006925|4760730|4761525|4760888|4760927|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000006925|4793569|4794365|4793725|4793764|+\n",
      "-gttccattgctccaaagattggctgtacgccggagactc\n",
      ">G000299455|906289|909876|906448|906487|+\n",
      "-gttccattgccccaaagattggctgtacgccggagactc\n",
      ">G000299455|912160|912572|912317|912356|+\n",
      "-gttccattgccccaaagattggctgtacaccggagactc\n",
      ">G000299455|4752701|4766669|4752860|4752899|+\n",
      "-gttccattgctccaaacattggctgtacgccggagactc\n",
      ">G000299455|4771847|4785261|4780310|4780349|-\n",
      "tgttccattgctccaaagattggctgtacgccggagact-\n",
      ">G000299455|5275700|5279438|5279090|5279129|-\n",
      "tgttccattgctccaaagattggctgtacgccggagact-\n",
      ">G000299455|5343200|5343533|5343462|5343501|+\n",
      "-gttccattgccccaaagattggctgtacaccggagactc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (907508, 907840)), (907582, 907689), '+'), (('G000006925', (1395000, 1395332)), (1395151, 1395258), '-'), (('G000299455', (2540736, 2540915)), (2540736, 2540843), '-'), (('G000299455', (2931465, 2931723)), (2931465, 2931572), '+')]\n",
      ">G000006925|907508|907840|907582|907689|+\n",
      "-ttaccctgatagaaagccagtacacgctgcatagcttcgctcttccggcactcgcgaca\n",
      "gattatattcaggcgcctgtcgtagcggcgtatttctccgtcaggtaa\n",
      ">G000006925|1395000|1395332|1395151|1395258|-\n",
      "attaccctgatagaaagccagtacacgctgcatagcttcgctcttccggcactcgcgaca\n",
      "gattatattcaggcgcctgtcgtagcggcgtatttctctgtcaggta-\n",
      ">G000299455|2540736|2540915|2540736|2540843|-\n",
      "attcccctgatagaaagccagcacgcgctgcataacttcactattccggcactcgcgaca\n",
      "gattatgttcaggcgtctatcgtagcggcgaatttctccgtctggta-\n",
      ">G000299455|2931465|2931723|2931465|2931572|+\n",
      "-tttccttgataaaatgccagtacacgctgcatagcttcgctcttccggcactcgcgaca\n",
      "gattatattcaggcgcctgtcgtagcggcgtatttcgccgtctggtaa\n",
      "\n",
      "\n",
      "4 [(('G000006925', (739473, 745780)), (744027, 744361), '+'), (('G000006925', (1921807, 1928114)), (1923226, 1923560), '-'), (('G000299455', (2897570, 2901707)), (2898854, 2899188), '-'), (('G000299455', (3586763, 3590110)), (3586970, 3587304), '-')]\n",
      ">G000006925|739473|745780|744027|744361|+\n",
      "-gttgtggacgcagattgataacggtcagcttgcgccggacctgactgaaatcaggacgt\n",
      "ccattacgaatgtcagcaatgaaatcacgcaaaccgtcaataaaaaactggaaaatcaga\n",
      "gtgctgcaatccagcagatacagaaagttcaggttgatacaaataataacctgaacagca\n",
      "tgtgggctgtgaagctgcaacagatgaaggacggacgcctttatattgcgggtatcggtg\n",
      "ccggtattgagaatacgccagcaggaatgcagagtcaggtgctgctggcggcagacagga\n",
      "ttgcgatgattaatcctgcgaatggcaacacaaag\n",
      ">G000006925|1921807|1928114|1923226|1923560|-\n",
      "agttgtggacgcagattgataacggtcagcttgcgccggacctgactgaaatcaggacgt\n",
      "ccattacgaatgtcagcaatgaaatcacgcaaaccgtcaataaaaaactggaaaatcaga\n",
      "gtgctgcaatccagcagatacagaaagttcaggttgatacaaataataacctgaacagca\n",
      "tgtgggctgtgaagctgcaacagatgaaggacggacgcctttatattgcgggtatcggtg\n",
      "ccggtattgagaatacgccagcaggaatgcagagtcaggtgctgctggcggcagacagga\n",
      "ttgcgatgattaatcctgcgaatggcaacacaaa-\n",
      ">G000299455|2897570|2901707|2898854|2899188|-\n",
      "agttgtggacgcagattgataacggtcagcttgcgccggacctggctgaaatcaggacgt\n",
      "ccattacgaatgtcagcaatgaaatcacgcagaccgtcaataaaaaactggaagaccaga\n",
      "gtgcggcaatccagcagatacagaaagttcaggttgatacaaataataacctgaacagca\n",
      "tgtgggccgtgaaactgcagcagatgaaggacggacgcctttatattgcgggtatcggag\n",
      "ccggtattgagaatacgccagcaggtatgcagagtcaggtgcttctggctgctgaccgga\n",
      "ttgcgatgattaatcctgcgaatggcaacacaaa-\n",
      ">G000299455|3586763|3590110|3586970|3587304|-\n",
      "agctgtggacgcagattgataacggtcagcttgcgcctgacctgactgaaatcaggacgt\n",
      "ccataacggatgtcagcaatgaaataacacagaccgtcaataagaaactggaagaccaga\n",
      "gtgcagcgatccagcagatacagaaggttcaggttgatacaaataataacctgaacagca\n",
      "tgtgggcagtgaagctgcagcagatgcaggacggacgcctttatattgcgggtatcggtg\n",
      "ccggtattgagaacacccctgacggcatgcagagtcaggtgctgctggcagcagacagga\n",
      "ttgcgatgattaatcctgcgaatggcaacacaaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3052652, 3052787), '-'), (('G000299455', (942831, 956197)), (944065, 944200), '+'), (('G000299455', (4713031, 4748324)), (4735892, 4736027), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3040656|3065083|3052652|3052787|-\n",
      "aagccttgtcagtgggctttgcagcccggatttttgcgtcagtcagtgccatagttcagc\n",
      "cttctgaatgcgttaaagagggtacaaattctgtcgaactggggtatacccgcaattata\n",
      "cccacacaatggggt-\n",
      ">G000299455|942831|956197|944065|944200|+\n",
      "-agccttgtcagtaggctttgctgcccggatttttgcgtcagtcagtgccatagttcagc\n",
      "cttctgaatgcgttaaagagggtacaaattctgtcgaactggggtatgcccacaattata\n",
      "cccacacaatgaggtt\n",
      ">G000299455|4713031|4748324|4735892|4736027|-\n",
      "aagccttgtcagtgggctttgcagcccggatttttgcgtcagtcagtgccatagttcagc\n",
      "cttctgaatgcgttaaagagggtacaaattctgtcgaactggggtatacccgcaattata\n",
      "cccacacaatggggt-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1776250, 1792578)), (1786451, 1787727), '+'), (('G000006925', (4094093, 4116338)), (4097776, 4099052), '+'), (('G000299455', (2682449, 2700553)), (2687048, 2688324), '+'), (('G000299455', (5063005, 5071226)), (5066267, 5067543), '-')]\n",
      ">G000006925|1776250|1792578|1786451|1787727|+\n",
      "-tccgactggaaatcgacaatcccgttagcgtactgaatgattgcaccagcagacgggca\n",
      "cgccttcaggcagcccggatcttcacagtgcatacagccgtctttacggatcagccactc\n",
      "cagcttgccattctgttcggtttcgctaaagcgcatcaccgtccaggacttggcgctcag\n",
      "atcggcggggttatcgtaaaccccgacgcagtgccccacttcatcacggatgtcgttcca\n",
      "ctccgaacacgccacctgacaggctttacagccgatacaggtggaaacgtcgataagttt\n",
      "cgcgacttctgctttgtaatcacgcacctgagaaggcggcgtgatggagttagttgcgga\n",
      "ccttttgataatatcctgcgtttccatagccatctgttcgcccccttacgccttctcgat\n",
      "gttgactaagaacgctttatattccggcgtttgcgagtttgcatcaccgacattcggcgt\n",
      "cagagtgttagcgatataacctttacgcgcgacaccctcaaagccccagtggattggaat\n",
      "acccaccgtttcaacctgctgaccgtttacattcagcggtttcagacggcgcgttaccac\n",
      "tgccaccgcgcggataaagcctcgcttgctggagacagtgacacgatcgccattattaat\n",
      "gcctttcgccgccgcctgcgtttcgctgatttccacaaactgttccggctgagcaattgc\n",
      "gttgagcaacgcgtgcttggtccaggtgtggaagtgctcggtcagacgataggtcgtacc\n",
      "cacatacgggaactgctcttttttacccatccgcagcgcgtcttgttcatacagacgaac\n",
      "aaccgggttagacaccacgttcggatgcagcgggttagtgcccagcggcgtttcaatcgg\n",
      "ctcgtagtgttccgggaacggaccttccgccattttgttgatggcaaacaggcgtcccat\n",
      "cccttccggctgcataataaacggcccggttggcgtacccggtgcggcattgccgaagtc\n",
      "cggaatatcgttacccgtccacttgctgccgttccactggatcagcatccgtttcggatc\n",
      "ccacggtttaccgttgatatccgccgaagcacggttgtagagtacgcgacggttgagcgg\n",
      "ccacgcccaggcccatcccagcgtattccccagaccggacgggtcggagttatcgcggtt\n",
      "tgccatctggttaccctgctctgtccagctaccggtgtagatccagcaagaagatgcggt\n",
      "tgtaccgtcatcacgca\n",
      ">G000006925|4094093|4116338|4097776|4099052|+\n",
      "-tcggactggaagtcgacgataccgttggcatactgaataattgcaccttccgccgggca\n",
      "cgctttcaggcagcctggatcggaacagtgcatacagccgtctttgcggatcagccattc\n",
      "cagtttgtcgttctgctccacttccgagaagcgcattaccgtccacgatttggcgcttaa\n",
      "atcattggggttgtcgtacaccccaatgttattgccgacggtatcgcgaatgtcgttcca\n",
      "ctctgaacacgccacctgacaggctttacagccgatacaggtggtaacgtcgatgagttt\n",
      "cgccacttcttcctggaagtcccgcgcctgaggcgcgggggtcagaccgttagtcgcgga\n",
      "acgacgaatgatatcttgagattgataagccatatgtcgtctccgttacaccttttccac\n",
      "attcacaaggaaggacttaaactccggcgtctgcgtgttcgcatcaccaacgaatggcgt\n",
      "taaagtattggcaataaagccttttttagctacgccttcatagccccagtgaatcggaat\n",
      "accgatggtatcgatatctttgccgttcgctttcagcgtgcgaatacgtttggtcaccac\n",
      "cgctttggctttgatatagccacggttggaggagactttcacggtatcgccctgggcaat\n",
      "gccaagtttattcgccagcgactccccgatttccacaaactgctctggctgcaaaatcgc\n",
      "gttcaacagcgcgtgtttggtccagtagtggaagtgctcggtcagacgataggtggttcc\n",
      "gacatacgggaacttatcggctttacccaatgcttcggcgtcgtctttaaagatacgtgc\n",
      "agctgggttcgagataacgtttggatgcagcgggttagttcccagcggcgtttcaaacgg\n",
      "ctcgtagtgttccgggaacggaccttccgccatcttatcgagggcaaacagacgccccat\n",
      "gccttcctgctgcatgataaacggcccgacgccgctacccggaggcgctgcgctgtagtc\n",
      "cggaatatcccagccggtccacttagtgccgtcccatttcagcaactgacgcttcggatc\n",
      "ccacgggttaccctgcggatctgcggaggcgcggttatacagaatgcggcggttaagcgg\n",
      "ccatgcccatgcccagcccagcgtgttaccgaggccagacggatcggcgttatcacgacg\n",
      "cgccatctggttgccttccggcgtccagctaccggcgaaaatccagcagccacaggaggt\n",
      "tgtaccgtcatcgcgca\n",
      ">G000299455|2682449|2700553|2687048|2688324|+\n",
      "-tccgactggaaatcgacaatcccgttagcgtactgaatgattgcaccagcagacgggca\n",
      "cgccttcaggcagccgggatcttcacagtgcatacagccgtctttacggatcagccactc\n",
      "cagcttgccgttctgttcggtttcgctaaagcgcatcaccgtccaggacttggcgctcag\n",
      "atcggcggggttatcgtaaaccccgacgcagtgccccacttcatcacggatgtcgttcca\n",
      "ctccgaacacgccacctgacaggctttacagccgatacaggtggaaacgtcgataagttt\n",
      "tgcgacttctgctttgtaatcacgcacctgagaaggcggcgtgatggagttagttgcgga\n",
      "ccttttgataatgtcctgcgtttccatagccatctgttcgcccccttacgccttctcgat\n",
      "gttgactaagaacgctttatattccggcgtttgcgagtttgcatcaccgacattcggcgt\n",
      "cagagtgttagcgatataacctttacgcgcgacaccctcaaagccccagtggattggaat\n",
      "acccaccgtttcaacctgctgaccatttacattcagtggtttcagacgacgcgttaccac\n",
      "agccaccgcgcggataaagccacgcttgcttgagacagtgacacgatcgccattattaat\n",
      "gcctttcgccgccgccagcgtttcgctgatttccacaaactgttccggctgagcaattgc\n",
      "gttgagcaatgcgtgcttggtccaggtgtggaagtgctcggtcagacgataggtcgtacc\n",
      "cacatacgggaactgctcttttttacccatccgcagcgcgtcttgttcatacagacgaac\n",
      "aaccgggttagacaccacgttcggatgcagcgggttagtgcccagcggcgtttcaattgg\n",
      "ctcgtagtgttccgggaacggaccttccgccattttgttgatggcaaacaggcgtcccat\n",
      "cccttccggctgcatgataaacggcccggttggcgtacccggtgcggcattgccgaagtc\n",
      "cggaatatcgttacccgtccacttgctgccgttccactggatcagcatccgtttcggatc\n",
      "ccacggtttaccgttgatatccgccgaagcacggttgtacagcacgcgacggttgagcgg\n",
      "ccacgcccaggcccatcccagcgtattccccagaccggacgggtcggagttatcgcggtt\n",
      "agccatctggttgccctgctctgtccagctaccggtgtagatccagcaagaagatgcggt\n",
      "tgtaccgtcatcacgca\n",
      ">G000299455|5063005|5071226|5066267|5067543|-\n",
      "ctcggactggaagtcgacgataccgttggcatactgaataattgcaccttccgccgggca\n",
      "cgctttcaggcagcctggatcggaacagtgcatacagccgtctttgcggatcagccattc\n",
      "cagtttgtcgttctgctccacttccgagaagcgcattaccgtccacgatttggcgcttaa\n",
      "atcattggggttgtcgtacaccccaatgttattgccgacggtatcgcgaatgtcgttcca\n",
      "ctctgaacacgccacctgacaggctttacagccgatacaggtggtgacgtcgatgagttt\n",
      "cgccacttcttcctggaagtcccgcgcctgaggcgcgggggtcagaccgttagtcgcgga\n",
      "acgacgaatgatatcttgagattgataagccatatgtcgtctccgttacaccttttccac\n",
      "attcacaaggaaggacttaaactccggcgtctgcgtgttcgcatcaccaacgaatggcgt\n",
      "taacgtattggcaataaagccttttttagctacgccttcataaccccagtgaatcggaat\n",
      "accgatggtatcgatatctttgccgttcgctttcagcgtacgaatacgtttggtcaccac\n",
      "cgctttggctttgatatagccacggttggaggagactttcacggtatcgccctgggcaat\n",
      "gccaagtttattcgccagcgactccccgatttccacaaactgctctggttgcaaaatcgc\n",
      "gttcaacagcgcgtgtttggtccagtagtggaagtgctcggtcagacgataggtggttcc\n",
      "gacatacgggaacttatcggctttacccaatgcttcggcgtcgtctttaaagatacgtgc\n",
      "agccgggttcgagataacgtttggatgcagcgggttagttcccagcggcgtttcaaacgg\n",
      "ctcgtagtgttccgggaacggaccttccgccatcttatcgagggcaaacagacgccccat\n",
      "gccttcctgctgcatgataaacggcccgacgccgctacccggaggcgctgcgctgtagtc\n",
      "cggaatatcccagccggtccacttagtgccgtcccatttcagcaactgacgcttcggatc\n",
      "ccacgggttaccctgcggatctgcggaggcgcggttatacagaatgcggcggttaagcgg\n",
      "ccatgcccatgcccagcccagcgtgttaccgaggccagacggatcggcgttatcacgacg\n",
      "cgccatctggttgccttccggcgtccagctaccggcgaaaatccagcagccacaggaggt\n",
      "tgtaccgtcatcgcgc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1, 24671)), (5329, 5387), '+'), (('G000006925', (2479150, 2517237)), (2513929, 2513987), '+'), (('G000299455', (1535847, 1550961)), (1539103, 1539161), '-'), (('G000299455', (4431865, 4470166)), (4445147, 4445205), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1118\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1118\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1166\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1166\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|5329|5387|+\n",
      "-aattacagataggcgatcgtgataatcgtggctattactgggatggaggtcactggcg\n",
      ">G000006925|2479150|2517237|2513929|2513987|+\n",
      "-aattacaaattggcgatcgcgatcattacggtaattactgggacggtggtcactggcg\n",
      ">G000299455|1535847|1550961|1539103|1539161|-\n",
      "aaattacaaattggcgatcgcgatcattacggtaattactgggacggtggtcactggc-\n",
      ">G000299455|4431865|4470166|4445147|4445205|-\n",
      "aaattacagataggcgatcgtgataatcgtggctattactgggatggaggtcactggc-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2523229, 2535003)), (2532359, 2532441), '-'), (('G000006925', (4143132, 4163489)), (4155876, 4155958), '+'), (('G000299455', (1513899, 1525673)), (1516461, 1516543), '+'), (('G000299455', (1535847, 1550961)), (1547087, 1547169), '-'), (('G000299455', (4996216, 5016676)), (5003838, 5003920), '-')]\n",
      ">G000006925|2523229|2535003|2532359|2532441|-\n",
      "gcgcgccagccgaggaacgggttctcttctttcgggaagttcatgtatggcagctctttg\n",
      "tcgccgccgatgtccatggtac-\n",
      ">G000006925|4143132|4163489|4155876|4155958|+\n",
      "-cgcgataaccgaggaacgggtttgcctctgcgggaatgtttagataatcaacgggtttg\n",
      "tcgccgccaatgtccatagtgcg\n",
      ">G000299455|1513899|1525673|1516461|1516543|+\n",
      "-cgcgccagccgaggaacgggttctcttctttcgggaagttcatgtatggcagctctttg\n",
      "tcgccgccgatgtccatggtacg\n",
      ">G000299455|1535847|1550961|1547087|1547169|-\n",
      "gcgcgatagccgaggaacgggttctcttcctggggaatattcagataaggaatgctttta\n",
      "tcgcctccgatgtccattgtgc-\n",
      ">G000299455|4996216|5016676|5003838|5003920|-\n",
      "gcgcgataaccgaggaacgggttagcttctgccggaatgttcagataatcaacgggtttg\n",
      "tcgccgccaatgtccatagtgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3369099, 3369213), '-'), (('G000299455', (540757, 615358)), (571934, 572048), '+'), (('G000299455', (4235244, 4293324)), (4269020, 4269134), '+')]\n",
      ">G000006925|3361966|3403725|3369099|3369213|-\n",
      "tccagaccttcaagattcaacccgctgcggcctaatgcggaaacaatgccagaggtggcg\n",
      "gtttgcccaaggccaaatgggttaccgaccgctacggcaaaatcaccgacgcgc-\n",
      ">G000299455|540757|615358|571934|572048|+\n",
      "-ccagaccttcaagattcaacccgctgcggcctaatgcggaaacaatgccagaggtggcg\n",
      "gtttgcccaaggccaaatgggttaccgaccgctacggcaaaatcaccgacgcgca\n",
      ">G000299455|4235244|4293324|4269020|4269134|+\n",
      "-cgtagttttcggcattcaggccgctacgccccagcgcagagacaatcccggaagttacc\n",
      "gtctctcccagaccaaacgggttaccaatcgctacggtgtaatcacccacgcgca\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1209065, 1211067)), (1210158, 1210213), '+'), (('G000006925', (2756889, 2766574)), (2757239, 2757294), '+'), (('G000006925', (2923039, 2950048)), (2948354, 2948409), '+'), (('G000006925', (3101814, 3103074)), (3102165, 3102220), '+'), (('G000006925', (3811174, 3811622)), (3811221, 3811276), '-'), (('G000299455', (3226712, 3229721)), (3228808, 3228863), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1209065|1211067|1210158|1210213|+\n",
      "-gaagcgcctgaaatgaagtatgtctttattgaaaaacatcaggctgagttcagca\n",
      ">G000006925|2756889|2766574|2757239|2757294|+\n",
      "-gaagcgcctgaaatgaagtatgtctttattgaaaaacatcaggctgagttcagca\n",
      ">G000006925|2923039|2950048|2948354|2948409|+\n",
      "-gaagcgcctgaaatgaagtatgtctttattgaaaaacatcaggctgagttcagca\n",
      ">G000006925|3101814|3103074|3102165|3102220|+\n",
      "-gaagcgcctgaaatgaagtatgtctttattgaaaaacatcaggctgagttcagca\n",
      ">G000006925|3811174|3811622|3811221|3811276|-\n",
      "cgaagcgcctgaaatgaagtatgtctttatcgaaaatcatcgggcagagttcagc-\n",
      ">G000299455|3226712|3229721|3228808|3228863|+\n",
      "-gaagcgcctgaaatgaagtatgtctttattgaaaaacatcaggctgagttcagca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2584954, 2636558)), (2602665, 2602717), '+'), (('G000299455', (1411666, 1461912)), (1444184, 1444236), '-'), (('G000299455', (3428237, 3435098)), (3431410, 3431462), '+')]\n",
      ">G000006925|2584954|2636558|2602665|2602717|+\n",
      "-gtttgtcgccagtggctttgagcactgtatcgccaatttgtttgtgattccg\n",
      ">G000299455|1411666|1461912|1444184|1444236|-\n",
      "tgtttgtcgccagtggctttgagcactgtatcgccaatttgtttgtgattcc-\n",
      ">G000299455|3428237|3435098|3431410|3431462|+\n",
      "-gtttgttgccagcggttttgagcacagtatcgcaaacatgtttatgatcccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2976303, 3015537)), (2977602, 2977796), '-'), (('G000299455', (988799, 1028140)), (1015227, 1015421), '+'), (('G000299455', (2612825, 2613349)), (2612825, 2613019), '-')]\n",
      ">G000006925|2976303|3015537|2977602|2977796|-\n",
      "cttgtagtgaccataaaaatcaacggcttcatggtttggatagtattttcctggcaagac\n",
      "ttctttggtgatttcgcgcggcacgccgtgtgcgccacaggtcagaacgtcacaaatgct\n",
      "cggtcctttgccgcctttgttccagccgccttcgacctgatgagcggcaacggcgccacc\n",
      "ccataagaaatctt-\n",
      ">G000299455|988799|1028140|1015227|1015421|+\n",
      "-ttgtagtgaccataaaaatcaacggcttcatggtttggatagtatttgcctggcaagac\n",
      "ttctttggtgatttcgcgcggcacgccgtgtgcgccaccggtcagaacgtcacaaatgct\n",
      "cggccctttgccgcctttgttccagccgccttcgacctgatgagcggcaactgcgccgcc\n",
      "ccataagaaatcttt\n",
      ">G000299455|2612825|2613349|2612825|2613019|-\n",
      "tttatagcgatgataaaaatcaattgcttcatgattgggataattaagcccgtcgataac\n",
      "gccttctgtcacttcacgcggcaccccgtgagcgccagcagtcatcacatcagcgatgct\n",
      "gatgccttttcctccttcattccagccaccttccaactgatgcgcggctaccgcaccgcc\n",
      "ccataaaaaacctt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3266261, 3266313), '-'), (('G000299455', (671363, 678362)), (673344, 673396), '+'), (('G000299455', (1078152, 1154001)), (1143411, 1143463), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3266261|3266313|-\n",
      "aactacgccgatcgtgcaacgctgtctattgctggtaccgaagtggcaaaag-\n",
      ">G000299455|671363|678362|673344|673396|+\n",
      "gtgcatacccgctatttaatattactgattatttttattgttaccgccgtta-\n",
      ">G000299455|1078152|1154001|1143411|1143463|+\n",
      "-actacggcgaccgcgctacgctttctattgccggttcggaaatggccaaaga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (118896, 118916), '+'), (('G000299455', (3819953, 3834467)), (3825037, 3825057), '+'), (('G000299455', (4300454, 4333834)), (4329590, 4329610), '-')]\n",
      ">G000006925|64491|216702|118896|118916|+\n",
      "-tgcaatatacgcagctgttg\n",
      ">G000299455|3819953|3834467|3825037|3825057|+\n",
      "-tagagtaaaccccgctgtta\n",
      ">G000299455|4300454|4333834|4329590|4329610|-\n",
      "ttgcaatatacgcagctgtt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4497401, 4528664)), (4517423, 4517463), '-'), (('G000299455', (767773, 781429)), (771715, 771755), '+'), (('G000299455', (4674171, 4686420)), (4684595, 4684635), '+')]\n",
      ">G000006925|4497401|4528664|4517423|4517463|-\n",
      "acgtttccgccttcgcgcgagtagatcggtttacgcacat-\n",
      ">G000299455|767773|781429|771715|771755|+\n",
      "-cgtttgcgccttcacgggagaagatcggtttaaccacata\n",
      ">G000299455|4674171|4686420|4684595|4684635|+\n",
      "-cgtttccgccttcgcgcgagtagatcggtttacgcacata\n",
      "\n",
      "\n",
      "5 [(('G000006925', (739473, 745780)), (739643, 741036), '+'), (('G000006925', (1921807, 1928114)), (1926551, 1927944), '-'), (('G000299455', (2187515, 2189078)), (2187685, 2189078), '+'), (('G000299455', (2902162, 2904160)), (2902597, 2903990), '-'), (('G000299455', (3590325, 3591889)), (3590325, 3591719), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1520\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|739643|741036|+\n",
      "-ggcaggggcggcagtatcaggcataccccattcaggggtcggggtttgaactgaacggc\n",
      "aggggctgtgctgcccgtccgacactgacggtctctaacctgcacggcatggtcaccggg\n",
      "atggcggaagatctgcagagtctggtcggcggaacggtggtccggcgtaaggtttacgcc\n",
      "cgttttctggatgcggtgaacttcgtcaacggaaacagcgacgccgatccggaacaggag\n",
      "gtgatcagccgctggcgcatcgagcagtgcagcgaactgagcgcggtcagtgcctctttt\n",
      "gtgctggccacaccaacggagacggatggcgcggttttcccggggcgtatcatgctggcg\n",
      "aatacctgtatgtggacttaccgttctgatgagtgtggttacacgggcagggcagtggct\n",
      "gacgagttcgacaaaccaacaacggatatccggaaggacaaatgcagcaagtgtatgcgc\n",
      "gggtgtgagttgcgcaacaataccggtaatttcggcggtttcctttccatcaataaactt\n",
      "tctcagt-aaatccatgacacagacagaatcagcgattctggcgcacgcccggcgatgtg\n",
      "cgccagcggagtcgtgtggcttcgtggtgagaacgccggagggagacaggtatcttccca\n",
      "gcgagaatatctccggtgagccggaggaacggttccggatggctccggaggactggctgc\n",
      "gggcacaaatgcagggtgagattgtggcactggtccacagtcatcccggtggtctgccct\n",
      "ggctgagtgaggctgaccggcggctgcaggtgcagagtgatttgccgtggtggctggtct\n",
      "gccgggggacgattcacaagttccgctgtgtgccgcatctcaccgggcggcgctttgagc\n",
      "acggggtgacggactgttacacgctgttccgggatgcttatcatctggcggggattgaga\n",
      "tgccggattttcatcgcggggatgactggtggcgtcacggtcagaatctctatctggata\n",
      "atctggaggccacagggctgtatcaggtgccgttgtcatcagcacaaccgggcgatgtgc\n",
      "tgctgtgctgttttggttcatcggtgccgaatcatgccgccatttactgcggcgatggtg\n",
      "agctgctgcaccatattcctgaacaactgagtaaacgagagaggtataccgacaaatggc\n",
      "agcgacgcacacactccctctggcgtcaccgggaatggcacgcatctgcctttacgggga\n",
      "tttgcaacgatttggccgccgcatcgaccttcgtgtgaaaacgggggctgaagccatccg\n",
      "cgcactggccacacagctcccggcgtttcgtcagaaactgagtgacggctggtatcaggt\n",
      "gcgcattgccgggcg\n",
      ">G000006925|1921807|1928114|1926551|1927944|-\n",
      "tggcaggggcggcagtatcaggcataccccattcaggggtcggggtttgaactgaacggc\n",
      "aggggctgtgctgcccgtccgacactgacggtctctaacctgcacggcatggtcaccggg\n",
      "atggcggaagatctgcagagtctggtcggcggaacggtggtccggcgtaaggtttacgcc\n",
      "cgttttctggatgcggtgaacttcgtcaacggaaacagcgacgccgatccggaacaggag\n",
      "gtgatcagccgctggcgcatcgagcagtgcagcgaactgagcgcggtcagtgcctctttt\n",
      "gtgctggccacaccaacggagacggatggcgcggttttcccggggcgtatcatgctggcg\n",
      "aatacctgtatgtggacttaccgttctgatgagtgtggttacacgggcagggcagtggct\n",
      "gacgagttcgacaaaccaacaacggatatccggaaggacaaatgcagcaagtgtatgcgc\n",
      "gggtgtgagttgcgcaacaataccggtaatttcggcggtttcctttccatcaataaactt\n",
      "tctcagt-aaatccatgacacagacagaatcagcgattctggcgcacgcccggcgatgtg\n",
      "cgccagcggagtcgtgtggcttcgtggtgagaacgccggagggagacaggtatcttccca\n",
      "gcgagaatatctccggtgagccggaggaacggttccggatggctccggaggactggctgc\n",
      "gggcacaaatgcagggtgagattgtggcactggtccacagtcatcccggtggtctgccct\n",
      "ggctgagtgaggctgaccggcggctgcaggtgcagagtgatttgccgtggtggctggtct\n",
      "gccgggggacgattcacaagttccgctgtgtgccgcatctcaccgggcggcgctttgagc\n",
      "acggggtgacggactgttacacgctgttccgggatgcttatcatctggcggggattgaga\n",
      "tgccggattttcatcgcggggatgactggtggcgtcacggtcagaatctctatctggata\n",
      "atctggaggccacagggctgtatcaggtgccgttgtcatcagcacaaccgggcgatgtgc\n",
      "tgctgtgctgttttggttcatcggtgccgaatcatgccgccatttactgcggcgatggtg\n",
      "agctgctgcaccatattcctgaacaactgagtaaacgagagaggtataccgacaaatggc\n",
      "agcgacgcacacactccctctggcgtcaccgggaatggcacgcatctgcctttacgggga\n",
      "tttgcaacgatttggccgccgcatcgaccttcgtgtgaaaacgggggctgaagccatccg\n",
      "cgcactggccacacagctcccggcgtttcgtcagaaactgagtgacggctggtatcaggt\n",
      "gcgcattgccgggc-\n",
      ">G000299455|2187515|2189078|2187685|2189078|+\n",
      "-ggcaggggaggcagtatcaggcgtatcccattcaggggagcggttttgaactgaatggc\n",
      "aaaggcaccagtacgcggcccacgctggcagtctctaacctgtacggcatggtcaccggt\n",
      "atggtggaagatatgcagagtctggtcggcggaacggtggtccggcgtaaggtttacgcc\n",
      "cgttttctggatgcggtgaacttcgtcaacggaaacagagacgccgatccggagcaggag\n",
      "gtgatcagccgctggcgcatcgagcagtgcagcgaactgagcgcggtgagtgcctccttt\n",
      "gtactgtccacgccgacggaaacggatggcgctgtttttccggggcgtatcatgctggcc\n",
      "aacacctgcacctggacctatcgcggtgatgagtgcggttatcacggtccggcagtcgcg\n",
      "gatgaatatgaccagccgacgtccgatatcacgaaggataaatgcagcaaatgcctgagc\n",
      "ggctgtaagtttcgcaataacgtcggcaactttggcggcttcctttccattaacaaactt\n",
      "tcgcagt-aatcccatgacagagacagaatcagcgattctggcgcacgcccggcgatgtg\n",
      "cgccagcggagtcgtgcggcttcgtggtaagcacgccggagggggaaagatatttcccct\n",
      "gcgtgaatatctccggtgagccggaggcgtatttccggatggctccggaggactggctgc\n",
      "aggcagagatgcagggtgagattgtggcgctggtccacagccaccccggtggtctgccct\n",
      "ggctgagtgaggctgaccggcggctgcaggtgcagagcgatttgccgtggtggctggtct\n",
      "gccggggggcgattcacaagttccgctgtgtgccgcatctcaccgggcggcgctttgagt\n",
      "acggggtgacggactgttacacactgttccgggatgcttaccatctggcggggattgaga\n",
      "tgccggattttcatcgcggggatgactggtggcgtcacggtcagaatctctatcttgaca\n",
      "atatggaggctactggtttttaccgtgtcgcactgacagaggcgcagccgggcgatgtgc\n",
      "tgctgtgctgttttggttcatcggtgccgaatcacgccgccatttactgcggcgacggtg\n",
      "agctgctgcaccatattcctgaacaactgagtaaacgagagaggtataccgacaaatggc\n",
      "agcgacgcacacactccctctggcgtcaccgggaatggcacgcatctgcctttacgggga\n",
      "tttgcaacgatttggccgccgcatcgaccttcgtgtgaaaacgggggctgaagccatccg\n",
      "ggcactggccacacagctcccggcgtttcgtcagaaactgagcgacggctggtatcaggt\n",
      "acggattgccgggcg\n",
      ">G000299455|2902162|2904160|2902597|2903990|-\n",
      "tggcaggggaggcagtatcaggcgtatcccattcaggggagcggttttgaactgaatggc\n",
      "aaaggcaccagtacgcggcccacgctggcagtctctaacctgtacggcatggtcaccggt\n",
      "atggtggaagatatgcagagtctggtcggcggaacggtggtccggcgtaaggtttacgcc\n",
      "cgttttctggatgcggtgaacttcgtcaacggaaacagagacgccgatccggagcaggag\n",
      "gtgatcagccgctggcgcatcgagcagtgcagcgaactgagcgcggtgagtgcctccttt\n",
      "gtactgtccacgccgacggaaacggatggcgctgtttttccggggcgtatcatgctggcc\n",
      "aacacctgcacctggacctatcgcggtgatgagtgcggttatcacggtccggcagtcgcg\n",
      "gatgaatatgaccagccgacgtccgatatcacgaaggataaatgcagcaaatgcctgagc\n",
      "ggctgtaagtttcgcaataacgtcggcaactttggcggcttcctttccattaacaaactt\n",
      "tcgcagt-aatcccatgacagagacagaatcagcgattctggcgcacgcccggcgatgtg\n",
      "cgccagcggagtcgtgcggcttcgtggtaagcacgccggagggggaaagatatttcccct\n",
      "gcgtgaatatctccggtgagccggaggcgtatttccggatggctccggaggactggctgc\n",
      "aggcagagatgcagggtgagattgtggcgctggtccacagccaccccggtggtctgccct\n",
      "ggctgagtgaggctgaccggcggctgcaggtgcagagcgatttgccgtggtggctggtct\n",
      "gccggggggcgattcacaagttccgctgtgtgccgcatctcaccgggcggcgctttgagt\n",
      "acggggtgacggactgttacacactgttccgggatgcttaccatctggcggggattgaga\n",
      "tgccggattttcatcgcggggatgactggtggcgtcacggtcagaatctctatcttgaca\n",
      "atatggaggctactggtttttaccgtgtcgcactgacagaggcgcagccgggcgatgtgc\n",
      "tgctgtgctgttttggttcatcggtgccgaatcacgccgccatttactgcggcgacggtg\n",
      "agctgctgcaccatattcctgaacaactgagtaaacgagagaggtataccgacaaatggc\n",
      "agcgacgcacacactccctctggcgtcaccgggaatggcacgcatctgcctttacgggga\n",
      "tttgcaacgatttggccgccgcatcgaccttcgagtgaaaacgggggctgaagccatccg\n",
      "ggcactggccacacagctcccggcgtttcgtcagaaactgagcgatggctggtatcaggt\n",
      "acggattgccgggc-\n",
      ">G000299455|3590325|3591889|3590325|3591719|-\n",
      "tggcaggggcgacagtatcagccgtatcccattcaggggagcggttttgaactgaatggc\n",
      "aaaggcaccagtacgcgccccacgctgacggtttctaacctgtacggcatgatcaccggg\n",
      "atggcggaagatctgcagagtctggtcggcggaacggtggtccggcgtaaggtttacgcc\n",
      "cgctttctggatgcggtgaacttcgtcaacggaaacagcgaagccgatccggagcaggag\n",
      "gtgatcagccgctggcgcatcgagcagtgcagcgaactgagcgcggtcagtgcctctttt\n",
      "gtactgtccacgccgactgaaacggatggcgctgtttttccgggacgtatcatgctggcc\n",
      "aacacctgcacctggacctatcgcggcgatgagtgcggttatagcggtccggcggtcgcg\n",
      "gatgaatatgaccagccgacgtccgatatcacgaaggataaatgcagcaaatgcctgagt\n",
      "ggctgtaagtttcgcaataatgtcggcaactttggcagcttcctttccattaacaaactt\n",
      "tcgcagtaaatcccatgacagagacagaatcagcgattctggcgcacgcccggcgatgtg\n",
      "cgccagcggagtcgtgcggcttcgtggtgagaacgccggaggaggaaagatattttccct\n",
      "gcgtgaatatctccggtgagccggaggagtatttccggatgtcgccggaggactggctga\n",
      "gtgcagaaatgcagggagagattgtggcgctggtccacagccaccccggtggtctgccct\n",
      "ggctgagtgaggccgaccggcggctgcaggtacagagtgatttgccgtggtggctggtct\n",
      "gccgggtggcgattcataaattccgctgtgtgccgcatctcaccgggcggcgctttgagc\n",
      "acggggtaacggactgttacacgctgttccgggacgcttaccatctggcgggaattgaga\n",
      "tgccggattttcatcgcggggataactggtggcgtcacggtcagaatctctatctggata\n",
      "atctggaggccacagggctgtatcaggtgccgttgtcagcggcgcagccgggcgatgtgc\n",
      "tgctgtgctgttttggttcatcggtgccgaatcatgccgccatttactgtggtgatggcg\n",
      "agctgctgcaccatattcctgaacaactgagcaaacgagagaggtataccgacaaatggc\n",
      "agcgacgcacacactccctctggcgtcaccgggcatggcgcgcatctgcctttacgggga\n",
      "tttgcaacgatttggccgccgcatcgaccttcgtgtgaaaacgggggctgaagccatccg\n",
      "cgcactggccacacagctcccggtgtttcgtcagaaactgagcgacggctggtatcaggt\n",
      "acggattgccgggc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (64491, 216702)), (119385, 119644), '+'), (('G000006925', (496417, 503320)), (500381, 500640), '-'), (('G000299455', (3819953, 3834467)), (3825526, 3825785), '+'), (('G000299455', (4300454, 4333834)), (4328862, 4329121), '-')]\n",
      ">G000006925|64491|216702|119385|119644|+\n",
      "-tcaggttgatggcgttaatcaccacaaagaatacggcggcagaaacccaggttgggatt\n",
      "tccggataccagaactgaatgtatttacccacggcagtcagctctgccatggcaactaaa\n",
      "acgtacagtacccagtagttccagccagaggcgaaaccggcaaaactgccccagtattta\n",
      "taagcaaagtggctaaaggagcctgcgacaggttcttcgaccaccatttcacccagctga\n",
      "cgcatgatcagaaaggcgat\n",
      ">G000006925|496417|503320|500381|500640|-\n",
      "accaggttaacggcgttgatgataataaagaaggcggcagcccaaatccacgttggaaca\n",
      "tccgggaaccagtactgcatatagatgcccgcagcggtcagctctgccattcccaccagc\n",
      "acgaacattacccagtagttccagccagagaggaagcccgcaaacggtccccagtattta\n",
      "taggcaaagtgggcaaatgaaccggataccggctcttcgaccaccatctcgccaagctgg\n",
      "cgcataatcaggaaagcga-\n",
      ">G000299455|3819953|3834467|3825526|3825785|+\n",
      "-ccaggttaacggcgttgatgataataaagaaggcggcagcccaaatccacgttggaaca\n",
      "tccgggaaccagtactgcatatagatgcccgcagcggtcagctctgccattcccaccagc\n",
      "acgaacattacccagtagttccagccagagaggaagcccgcaaacggtccccagtattta\n",
      "taggcaaagtgggcaaatgaaccggataccggctcttcgaccaccatctcgccaagctgg\n",
      "cgcataatcaggaaagcgat\n",
      ">G000299455|4300454|4333834|4328862|4329121|-\n",
      "gtcaggttgatggcgttaatcaccacaaagaatacggcggcagaaacccaggtggggatt\n",
      "tccggataccagaactgaatgtatttacccacggcagtcagctcagccatggcaactaaa\n",
      "acgtacagtacccagtagttccagccagaggcgaaaccggcaaaactgccccagtattta\n",
      "taagcaaagtggctaaaggagcctgcgacaggttcttcgaccaccatttcacccagctga\n",
      "cgcatgatcagaaaggcga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4652860, 4653326)), (4652979, 4653326), '+'), (('G000006925', (4684418, 4684875)), (4684418, 4684765), '-'), (('G000299455', (5295815, 5296178)), (5295845, 5296178), '+')]\n",
      ">G000006925|4652860|4653326|4652979|4653326|+\n",
      "-cggggtgtggcagaatatgagctttcagagtaaccatccagaatgagttcccatgtcac\n",
      "acaacctggagcaccagaaagtccacaccagaatggtgaaagaggttctgaaagccgtcg\n",
      "ccagagccaataatcacccttataagtccgtatttgcagactttatcaccggtcatccct\n",
      "cctgcacggtctgtttctgggaaaccttccacaaaatgtacccggattcaccgtacgaat\n",
      "acgtcacgttctgccatacctgccgtcgttttg-atttatacgaaacagaagcggagatg\n",
      "aaggcggatgacccgaagtggtggtgattttcagccccgaccgctgaaa\n",
      ">G000006925|4684418|4684875|4684418|4684765|-\n",
      "tcggggtgtggcagaatatgagctttcagagtaaccatccagaatgagttcccatgtcac\n",
      "acaacctggagcaccagaaagtccacaccagaatggtgaaagaggttctgaaagccgtcg\n",
      "ccagagccaataatcacccttataagtccgtatttgcagactttatcaccggtcatccct\n",
      "cctgcacggtctgtttctgggaaaccttccacaaaatgtacccggattcaccgtacgaat\n",
      "acgtcacgttctgccatacctgccgtcgttttg-atttatacgaaacagaagcggagatg\n",
      "aaggcggatgacccgaagtggtggtgattttcagccccgaccgctgaa-\n",
      ">G000299455|5295815|5296178|5295845|5296178|+\n",
      "-cggggtgtggcagaatatgagctttcagagtaaccatccagaatgagttcccatgtcac\n",
      "ac-acctgaag--------------caccagaatggtgaaagaggttctgaaagccgtcg\n",
      "ccagagccaataatcccccttatcagtccgtatttgcagactttatcgccggtcatccct\n",
      "cctgcacggtctgtttctgggaaaccttccacaaaatgtacccggattcaccgtacgaat\n",
      "acgtcacgttctgccatacctgccgtcgttttgtatttatacgaaacagaagcggagatg\n",
      "aaggcggatgacccgaagtggtggtgattttcagcctcgtccgctgaaa\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1275166, 1289454)), (1286559, 1286824), '-'), (('G000006925', (2118864, 2119959)), (2119641, 2119906), '+'), (('G000299455', (1942270, 1943364)), (1942323, 1942588), '-'), (('G000299455', (2959070, 2965951)), (2961702, 2961967), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1520\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1532\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1532\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1275166|1289454|1286559|1286824|-\n",
      "ccctgacgaacttgcataatagtcacgtgcggtggacaaatagactgcacttcatcaagc\n",
      "agttgacgttttacacgtttttccagcattgcttccagttcaaaactggtatcaaagtgg\n",
      "ttttcaatagagtttttagatgagtgtgtaaccagcacaatttcagtaatgccagccgca\n",
      "atacattcattcacgacgtattgaattaatggcttatcgacaagtggcagcatctctttc\n",
      "gggatggctttcgtcgccggcaaca-\n",
      ">G000006925|2118864|2119959|2119641|2119906|+\n",
      "-cctgacgcacgttcataatgatcacgcccggcggacagatggactgtacttccgccagc\n",
      "agttgacgcttcacgcgctgctcaaggagtgattctaactcataagaggtgtcgaagtgg\n",
      "ttttcgaccgcgttcttggacgcgtgagttaccaggaggatttctttgatccctgcagcc\n",
      "acaatctcgtcaacaatgtactgaatcattggcttgtcgacgattggtagcatctctttg\n",
      "ggtatcgccttagtggcaggcaacat\n",
      ">G000299455|1942270|1943364|1942323|1942588|-\n",
      "ccctgacgcacgttcataatggttacgcccggcgggcagatcgactgcacttccgccagc\n",
      "agctgacgcttcacacgctgttcaaggagagattctaattcataagaggtgtcaaagtgg\n",
      "ttttcgaccgcgttcttggacgcatgagttaccaggaggatttctttgatccctgcagcc\n",
      "acaatctcgtcaacaatgtactgaatcattggcttgtcgacgatcggtagcatctctttg\n",
      "ggtatcgccttagtggcaggcaaca-\n",
      ">G000299455|2959070|2965951|2961702|2961967|+\n",
      "-cctgacgaacttgcataatagtcacgtgcggtggacaaatagactgcacttcatcaagc\n",
      "agttgacgttttacacgtttttccagcattgcttccagttcaaaactggtatcaaagtgg\n",
      "ttttcaatagagtttttagatgagtgtgtaaccagcacaatttcagtaatgccagccgca\n",
      "atacattcattcacgacgtattgaattaatggcttatcgacaagtggcagcatctctttc\n",
      "gggatggctttcgtcgccggcaacat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3040656, 3065083)), (3052891, 3052938), '-'), (('G000299455', (942831, 956197)), (943914, 943961), '+'), (('G000299455', (4143590, 4143639)), (4143590, 4143637), '+'), (('G000299455', (4713031, 4748324)), (4736131, 4736178), '-')]\n",
      ">G000006925|3040656|3065083|3052891|3052938|-\n",
      "cgttttgtacgagcttcggagagagaaacttctggatacacaccaag-\n",
      ">G000299455|942831|956197|943914|943961|+\n",
      "-gttttgtacgagcttcggagagagaaacttctggatacacaccaagt\n",
      ">G000299455|4143590|4143639|4143590|4143637|+\n",
      "-gttttgcacgagcatcagccaaagtaacttcaggatacacacctagt\n",
      ">G000299455|4713031|4748324|4736131|4736178|-\n",
      "cgttttgtacgagcttcggagagagaaacttctggatacacaccaag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1406932, 1406960)), (1406932, 1406960), '-'), (('G000006925', (2374481, 2472008)), (2391228, 2391256), '-'), (('G000299455', (1685057, 1714323)), (1689435, 1689463), '+')]\n",
      ">G000006925|1406932|1406960|1406932|1406960|-\n",
      "ggctggtcgctgattggcgatgcgctgt-\n",
      ">G000006925|2374481|2472008|2391228|2391256|-\n",
      "ggctggccgctgattggcgatgtgctgt-\n",
      ">G000299455|1685057|1714323|1689435|1689463|+\n",
      "-gctggccgctgattggcgatgtgctgtc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4392644, 4394283)), (4393419, 4393521), '+'), (('G000006925', (4823851, 4824717)), (4823853, 4823955), '+'), (('G000299455', (4911254, 4911358)), (4911254, 4911356), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4392644|4394283|4393419|4393521|+\n",
      "-aatccgcgcccgatccagcagtgttttaagttcacgctgctcgcgttcagtcatctctt\n",
      "tttgggtaatacgggggagtgccatgatggtgccctcagtgag\n",
      ">G000006925|4823851|4824717|4823853|4823955|+\n",
      "-aatccgcgcccgatccagcagtgttttaagttcacgctgctcgcgttcagtcatctctt\n",
      "tttgggtaatacgggggagtgccatgatggtgccctcagtgag\n",
      ">G000299455|4911254|4911358|4911254|4911356|-\n",
      "caatccgcgcccgatccagcagtgttttaagttcacgctgctcgcgttcagtcatctctt\n",
      "tttgggtaatacgggggagtgccatgatggtgccctcagtga-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3016790, 3036697)), (3016861, 3017042), '+'), (('G000299455', (971316, 975991)), (975739, 975920), '-'), (('G000299455', (1477869, 1484878)), (1478015, 1478196), '+')]\n",
      ">G000006925|3016790|3036697|3016861|3017042|+\n",
      "-ccggagcagattcaccgaaggtggtcatgccgacgatagcgccgttcaggccaacatac\n",
      "ttgtaccagtagtcagcaatacccgcttctaccgcaacgcgtgcagtaaccgctttcggc\n",
      "agtacggattcacggtaagccgcatcctgcttgtcaaatgcgtcggtagacggcatggac\n",
      "ac\n",
      ">G000299455|971316|975991|975739|975920|-\n",
      "gccggagcagattcaccgaaagtggtcataccgacgatagcaccgttcaggccaacatac\n",
      "ttgtaccagtagtcagcaatacccgcttctacagcaacgcgtgcagtaaccgctttcggc\n",
      "agtacggattcacggtaagcagcatcctgcttgtcaaatgcgtcggtagacggcatggac\n",
      "a-\n",
      ">G000299455|1477869|1484878|1478015|1478196|+\n",
      "-ccggagcagattccccgtaacccgtcatcccgacaattgcccctttcagaccaacatac\n",
      "ttgtaccagtaatcggcaatacctgcttccaccgccacgcgagccgcaacgttagaaggc\n",
      "aacaccgactcccgatattcctcatcctgggcgtcgaaaatatcggtcgagggcagggaa\n",
      "ac\n",
      "\n",
      "\n",
      "120 [(('G000006925', (1, 24671)), (19255, 19377), '+'), (('G000006925', (64491, 216702)), (87000, 87122), '-'), (('G000006925', (64491, 216702)), (148190, 148312), '-'), (('G000006925', (266682, 267450)), (266741, 266863), '-'), (('G000006925', (273344, 278315)), (273931, 274053), '+'), (('G000006925', (278320, 279091)), (278381, 278503), '-'), (('G000006925', (300111, 308544)), (307834, 307956), '-'), (('G000006925', (378637, 384812)), (378698, 378820), '-'), (('G000006925', (459861, 468373)), (466304, 466426), '+'), (('G000006925', (468374, 487224)), (479300, 479422), '+'), (('G000006925', (520755, 528026)), (520814, 520936), '-'), (('G000006925', (545486, 558918)), (558737, 558859), '+'), (('G000006925', (622915, 623662)), (623502, 623624), '+'), (('G000006925', (865887, 867389)), (867208, 867330), '+'), (('G000006925', (921552, 942506)), (932413, 932535), '+'), (('G000006925', (984595, 985335)), (984655, 984777), '-'), (('G000006925', (985336, 1008622)), (986017, 986139), '+'), (('G000006925', (1025530, 1063042)), (1050159, 1050281), '+'), (('G000006925', (1189492, 1208094)), (1198222, 1198344), '+'), (('G000006925', (1239481, 1240251)), (1240069, 1240191), '+'), (('G000006925', (1290847, 1306886)), (1295660, 1295782), '+'), (('G000006925', (1321493, 1385605)), (1365526, 1365648), '-'), (('G000006925', (1398496, 1399271)), (1399085, 1399207), '+'), (('G000006925', (1409877, 1410649)), (1409940, 1410062), '-'), (('G000006925', (1424264, 1425034)), (1424325, 1424447), '-'), (('G000006925', (1425133, 1465118)), (1444867, 1444989), '+'), (('G000006925', (1509064, 1578296)), (1527266, 1527388), '+'), (('G000006925', (1670306, 1692658)), (1673959, 1674081), '+'), (('G000006925', (1729332, 1741706)), (1736033, 1736155), '-'), (('G000006925', (1758378, 1759147)), (1758965, 1759087), '+'), (('G000006925', (1759148, 1776249)), (1767115, 1767237), '+'), (('G000006925', (1776250, 1792578)), (1781500, 1781622), '+'), (('G000006925', (1793868, 1802424)), (1795188, 1795310), '-'), (('G000006925', (1842508, 1869155)), (1850108, 1850230), '-'), (('G000006925', (1940277, 1941051)), (1940864, 1940986), '+'), (('G000006925', (1976680, 1977448)), (1976739, 1976861), '-'), (('G000006925', (2036705, 2044875)), (2037214, 2037336), '-'), (('G000006925', (2050169, 2051841)), (2051132, 2051254), '-'), (('G000006925', (2088344, 2103602)), (2095627, 2095749), '+'), (('G000006925', (2120115, 2138771)), (2134707, 2134829), '-'), (('G000006925', (2172359, 2184266)), (2178379, 2178501), '+'), (('G000006925', (2187913, 2198767)), (2196946, 2197068), '-'), (('G000006925', (2198768, 2199537)), (2199356, 2199478), '+'), (('G000006925', (2199538, 2210687)), (2205610, 2205732), '-'), (('G000006925', (2211287, 2225774)), (2221220, 2221342), '+'), (('G000006925', (2211287, 2225774)), (2222690, 2222812), '+'), (('G000006925', (2235234, 2236005)), (2235293, 2235415), '-'), (('G000006925', (2242494, 2346249)), (2288032, 2288154), '+'), (('G000006925', (2374481, 2472008)), (2395694, 2395816), '-'), (('G000006925', (2374481, 2472008)), (2464668, 2464790), '+'), (('G000006925', (2756085, 2756853)), (2756144, 2756266), '-'), (('G000006925', (2816684, 2833035)), (2825788, 2825910), '+'), (('G000006925', (2849870, 2870340)), (2862899, 2863021), '-'), (('G000006925', (2849870, 2870340)), (2864700, 2864822), '-'), (('G000006925', (2957651, 2963246)), (2962752, 2962874), '-'), (('G000006925', (2976303, 3015537)), (3015220, 3015342), '+'), (('G000006925', (3016790, 3036697)), (3022051, 3022173), '+'), (('G000006925', (3038529, 3039299)), (3039116, 3039238), '+'), (('G000006925', (3104758, 3106299)), (3105347, 3105469), '+'), (('G000006925', (3104758, 3106299)), (3106116, 3106238), '+'), (('G000006925', (3179574, 3180459)), (3179749, 3179871), '-'), (('G000006925', (3180744, 3203996)), (3181640, 3181762), '-'), (('G000006925', (3237769, 3244530)), (3237828, 3237950), '-'), (('G000006925', (3268417, 3281594)), (3273830, 3273952), '-'), (('G000006925', (3268417, 3281594)), (3278198, 3278320), '+'), (('G000006925', (3322761, 3361739)), (3361027, 3361149), '-'), (('G000006925', (3361966, 3403725)), (3362027, 3362149), '-'), (('G000006925', (3361966, 3403725)), (3400957, 3401079), '+'), (('G000006925', (3361966, 3403725)), (3403544, 3403666), '+'), (('G000006925', (3608503, 3611864)), (3608567, 3608689), '-'), (('G000006925', (3608503, 3611864)), (3611154, 3611276), '-'), (('G000006925', (3612083, 3637192)), (3624820, 3624942), '+'), (('G000006925', (3612083, 3637192)), (3627287, 3627409), '-'), (('G000006925', (3637330, 3640023)), (3637917, 3638039), '+'), (('G000006925', (3658114, 3676993)), (3665747, 3665869), '+'), (('G000006925', (3693106, 3702974)), (3693167, 3693289), '-'), (('G000006925', (3703179, 3717854)), (3703239, 3703361), '-'), (('G000006925', (3717879, 3736288)), (3725523, 3725645), '-'), (('G000006925', (3717879, 3736288)), (3727880, 3728002), '-'), (('G000006925', (3736321, 3788106)), (3743609, 3743731), '+'), (('G000006925', (3736321, 3788106)), (3762435, 3762557), '+'), (('G000006925', (3736321, 3788106)), (3778553, 3778675), '+'), (('G000006925', (3815875, 3829465)), (3818592, 3818714), '+'), (('G000006925', (3831593, 3850590)), (3835073, 3835195), '+'), (('G000006925', (3831593, 3850590)), (3845259, 3845381), '-'), (('G000006925', (3858569, 3864367)), (3863658, 3863780), '-'), (('G000006925', (3939882, 3952896)), (3942893, 3943015), '-'), (('G000006925', (4002899, 4012437)), (4002958, 4003080), '-'), (('G000006925', (4069200, 4076620)), (4075913, 4076035), '-'), (('G000006925', (4091607, 4092380)), (4092195, 4092317), '+'), (('G000006925', (4094093, 4116338)), (4102901, 4103023), '+'), (('G000006925', (4174522, 4188276)), (4176622, 4176744), '-'), (('G000006925', (4256570, 4288375)), (4285566, 4285688), '+'), (('G000006925', (4288513, 4315006)), (4314296, 4314418), '-'), (('G000006925', (4362175, 4367702)), (4367521, 4367643), '+'), (('G000006925', (4378758, 4379529)), (4379345, 4379467), '+'), (('G000006925', (4389053, 4390969)), (4389641, 4389763), '+'), (('G000006925', (4392644, 4394283)), (4392708, 4392830), '-'), (('G000006925', (4397536, 4410956)), (4410247, 4410369), '-'), (('G000006925', (4462252, 4478752)), (4462840, 4462962), '+'), (('G000006925', (4462252, 4478752)), (4465114, 4465236), '-'), (('G000006925', (4462252, 4478752)), (4478043, 4478165), '-'), (('G000006925', (4478753, 4490457)), (4488978, 4489100), '+'), (('G000006925', (4497401, 4528664)), (4514929, 4515051), '-'), (('G000006925', (4497401, 4528664)), (4519646, 4519768), '+'), (('G000006925', (4535976, 4543278)), (4540569, 4540691), '-'), (('G000006925', (4557883, 4573403)), (4569782, 4569904), '-'), (('G000006925', (4645852, 4646621)), (4645911, 4646033), '-'), (('G000006925', (4656735, 4658205)), (4656795, 4656917), '-'), (('G000006925', (4785482, 4786253)), (4786071, 4786193), '+'), (('G000299455', (91523, 102755)), (91587, 91709), '-'), (('G000299455', (696017, 696792)), (696606, 696728), '+'), (('G000299455', (941725, 942502)), (942315, 942437), '+'), (('G000299455', (1581872, 1582655)), (1581936, 1582058), '-'), (('G000299455', (2052398, 2064114)), (2062016, 2062138), '-'), (('G000299455', (3182222, 3183821)), (3183636, 3183758), '+'), (('G000299455', (3672841, 3680073)), (3679358, 3679480), '-'), (('G000299455', (4588613, 4589520)), (4589332, 4589454), '+'), (('G000299455', (4592900, 4593681)), (4592966, 4593088), '-'), (('G000299455', (5323356, 5324120)), (5323419, 5323541), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 120\r",
      "  101 / 120\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 120\r",
      "   10 / 120\r",
      "   20 / 120\r",
      "   30 / 120\r",
      "   40 / 120\r",
      "   50 / 120\r",
      "   60 / 120\r",
      "   70 / 120\r",
      "   80 / 120\r",
      "   90 / 120\r",
      "  100 / 120\r",
      "  110 / 120\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 119  f\b\b\r",
      "STEP     2 / 119  f\b\b\r",
      "STEP     3 / 119  f\b\b\r",
      "STEP     4 / 119  f\b\b\r",
      "STEP     5 / 119  f\b\b\r",
      "STEP     6 / 119  f\b\b\r",
      "STEP     7 / 119  f\b\b\r",
      "STEP     8 / 119  f\b\b\r",
      "STEP     9 / 119  f\b\b\r",
      "STEP    10 / 119  f\b\b\r",
      "STEP    11 / 119  f\b\b\r",
      "STEP    12 / 119  f\b\b\r",
      "STEP    13 / 119  f\b\b\r",
      "STEP    14 / 119  f\b\b\r",
      "STEP    15 / 119  f\b\b\r",
      "STEP    16 / 119  f\b\b\r",
      "STEP    17 / 119  f\b\b\r",
      "STEP    18 / 119  f\b\b\r",
      "STEP    19 / 119  f\b\b\r",
      "STEP    20 / 119  f\b\b\r",
      "STEP    21 / 119  f\b\b\r",
      "STEP    22 / 119  f\b\b\r",
      "STEP    23 / 119  f\b\b\r",
      "STEP    24 / 119  f\b\b\r",
      "STEP    25 / 119  f\b\b\r",
      "STEP    26 / 119  f\b\b\r",
      "STEP    27 / 119  f\b\b\r",
      "STEP    28 / 119  f\b\b\r",
      "STEP    29 / 119  f\b\b\r",
      "STEP    30 / 119  f\b\b\r",
      "STEP    31 / 119  f\b\b\r",
      "STEP    32 / 119  f\b\b\r",
      "STEP    33 / 119  f\b\b\r",
      "STEP    34 / 119  f\b\b\r",
      "STEP    35 / 119  f\b\b\r",
      "STEP    36 / 119  f\b\b\r",
      "STEP    37 / 119  f\b\b\r",
      "STEP    38 / 119  f\b\b\r",
      "STEP    39 / 119  f\b\b\r",
      "STEP    40 / 119  f\b\b\r",
      "STEP    41 / 119  f\b\b\r",
      "STEP    42 / 119  f\b\b\r",
      "STEP    43 / 119  f\b\b\r",
      "STEP    44 / 119  f\b\b\r",
      "STEP    45 / 119  f\b\b\r",
      "STEP    46 / 119  f\b\b\r",
      "STEP    47 / 119  f\b\b\r",
      "STEP    48 / 119  f\b\b\r",
      "STEP    49 / 119  f\b\b\r",
      "STEP    50 / 119  f\b\b\r",
      "STEP    51 / 119  f\b\b\r",
      "STEP    52 / 119  f\b\b\r",
      "STEP    53 / 119  f\b\b\r",
      "STEP    54 / 119  f\b\b\r",
      "STEP    55 / 119  f\b\b\r",
      "STEP    56 / 119  f\b\b\r",
      "STEP    57 / 119  f\b\b\r",
      "STEP    58 / 119  f\b\b\r",
      "STEP    59 / 119  f\b\b\r",
      "STEP    60 / 119  f\b\b\r",
      "STEP    61 / 119  f\b\b\r",
      "STEP    62 / 119  f\b\b\r",
      "STEP    63 / 119  f\b\b\r",
      "STEP    64 / 119  f\b\b\r",
      "STEP    65 / 119  f\b\b\r",
      "STEP    66 / 119  f\b\b\r",
      "STEP    67 / 119  f\b\b\r",
      "STEP    68 / 119  f\b\b\r",
      "STEP    69 / 119  f\b\b\r",
      "STEP    70 / 119  f\b\b\r",
      "STEP    71 / 119  f\b\b\r",
      "STEP    72 / 119  f\b\b\r",
      "STEP    73 / 119  f\b\b\r",
      "STEP    74 / 119  f\b\b\r",
      "STEP    75 / 119  f\b\b\r",
      "STEP    76 / 119  f\b\b\r",
      "STEP    77 / 119  f\b\b\r",
      "STEP    78 / 119  f\b\b\r",
      "STEP    79 / 119  f\b\b\r",
      "STEP    80 / 119  f\b\b\r",
      "STEP    81 / 119  f\b\b\r",
      "STEP    82 / 119  f\b\b\r",
      "STEP    83 / 119  f\b\b\r",
      "STEP    84 / 119  f\b\b\r",
      "STEP    85 / 119  f\b\b\r",
      "STEP    86 / 119  f\b\b\r",
      "STEP    87 / 119  f\b\b\r",
      "STEP    88 / 119  f\b\b\r",
      "STEP    89 / 119  f\b\b\r",
      "STEP    90 / 119  f\b\b\r",
      "STEP    91 / 119  f\b\b\r",
      "STEP    92 / 119  f\b\b\r",
      "STEP    93 / 119  f\b\b\r",
      "STEP    94 / 119  f\b\b\r",
      "STEP    95 / 119  f\b\b\r",
      "STEP    96 / 119  f\b\b\r",
      "STEP    97 / 119  f\b\b\r",
      "STEP    98 / 119  f\b\b\r",
      "STEP    99 / 119  f\b\b\r",
      "STEP   100 / 119  f\b\b\r",
      "STEP   101 / 119  f\b\b\r",
      "STEP   102 / 119  f\b\b\r",
      "STEP   103 / 119  f\b\b\r",
      "STEP   104 / 119  f\b\b\r",
      "STEP   105 / 119  f\b\b\r",
      "STEP   106 / 119  f\b\b\r",
      "STEP   107 / 119  f\b\b\r",
      "STEP   108 / 119  f\b\b\r",
      "STEP   109 / 119  f\b\b\r",
      "STEP   110 / 119  f\b\b\r",
      "STEP   111 / 119  f\b\b\r",
      "STEP   112 / 119  f\b\b\r",
      "STEP   113 / 119  f\b\b\r",
      "STEP   114 / 119  f\b\b\n",
      "Reallocating..done. *alloclen = 1246\n",
      "\r",
      "STEP   115 / 119  f\b\b\r",
      "STEP   116 / 119  f\b\b\r",
      "STEP   117 / 119  f\b\b\r",
      "STEP   118 / 119  f\b\b\r",
      "STEP   119 / 119  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 120\r",
      "  100 / 120\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 120\r",
      "   10 / 120\r",
      "   20 / 120\r",
      "   30 / 120\r",
      "   40 / 120\r",
      "   50 / 120\r",
      "   60 / 120\r",
      "   70 / 120\r",
      "   80 / 120\r",
      "   90 / 120\r",
      "  100 / 120\r",
      "  110 / 120\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 119  f\b\b\r",
      "STEP     2 / 119  f\b\b\r",
      "STEP     3 / 119  f\b\b\r",
      "STEP     4 / 119  f\b\b\r",
      "STEP     5 / 119  f\b\b\r",
      "STEP     6 / 119  f\b\b\r",
      "STEP     7 / 119  f\b\b\r",
      "STEP     8 / 119  f\b\b\r",
      "STEP     9 / 119  f\b\b\r",
      "STEP    10 / 119  f\b\b\r",
      "STEP    11 / 119  f\b\b\r",
      "STEP    12 / 119  f\b\b\r",
      "STEP    13 / 119  f\b\b\r",
      "STEP    14 / 119  f\b\b\r",
      "STEP    15 / 119  f\b\b\r",
      "STEP    16 / 119  f\b\b\r",
      "STEP    17 / 119  f\b\b\r",
      "STEP    18 / 119  f\b\b\r",
      "STEP    19 / 119  f\b\b\r",
      "STEP    20 / 119  f\b\b\r",
      "STEP    21 / 119  f\b\b\r",
      "STEP    22 / 119  f\b\b\r",
      "STEP    23 / 119  f\b\b\r",
      "STEP    24 / 119  f\b\b\r",
      "STEP    25 / 119  f\b\b\r",
      "STEP    26 / 119  f\b\b\r",
      "STEP    27 / 119  f\b\b\r",
      "STEP    28 / 119  f\b\b\r",
      "STEP    29 / 119  f\b\b\r",
      "STEP    30 / 119  f\b\b\r",
      "STEP    31 / 119  f\b\b\r",
      "STEP    32 / 119  f\b\b\r",
      "STEP    33 / 119  f\b\b\r",
      "STEP    34 / 119  f\b\b\r",
      "STEP    35 / 119  f\b\b\r",
      "STEP    36 / 119  f\b\b\r",
      "STEP    37 / 119  f\b\b\r",
      "STEP    38 / 119  f\b\b\r",
      "STEP    39 / 119  f\b\b\r",
      "STEP    40 / 119  f\b\b\r",
      "STEP    41 / 119  f\b\b\r",
      "STEP    42 / 119  f\b\b\r",
      "STEP    43 / 119  f\b\b\r",
      "STEP    44 / 119  f\b\b\r",
      "STEP    45 / 119  f\b\b\r",
      "STEP    46 / 119  f\b\b\r",
      "STEP    47 / 119  f\b\b\r",
      "STEP    48 / 119  f\b\b\r",
      "STEP    49 / 119  f\b\b\r",
      "STEP    50 / 119  f\b\b\r",
      "STEP    51 / 119  f\b\b\r",
      "STEP    52 / 119  f\b\b\r",
      "STEP    53 / 119  f\b\b\r",
      "STEP    54 / 119  f\b\b\r",
      "STEP    55 / 119  f\b\b\r",
      "STEP    56 / 119  f\b\b\r",
      "STEP    57 / 119  f\b\b\r",
      "STEP    58 / 119  f\b\b\r",
      "STEP    59 / 119  f\b\b\r",
      "STEP    60 / 119  f\b\b\r",
      "STEP    61 / 119  f\b\b\r",
      "STEP    62 / 119  f\b\b\r",
      "STEP    63 / 119  f\b\b\r",
      "STEP    64 / 119  f\b\b\r",
      "STEP    65 / 119  f\b\b\r",
      "STEP    66 / 119  f\b\b\r",
      "STEP    67 / 119  f\b\b\r",
      "STEP    68 / 119  f\b\b\r",
      "STEP    69 / 119  f\b\b\r",
      "STEP    70 / 119  f\b\b\r",
      "STEP    71 / 119  f\b\b\r",
      "STEP    72 / 119  f\b\b\r",
      "STEP    73 / 119  f\b\b\r",
      "STEP    74 / 119  f\b\b\r",
      "STEP    75 / 119  f\b\b\r",
      "STEP    76 / 119  f\b\b\r",
      "STEP    77 / 119  f\b\b\r",
      "STEP    78 / 119  f\b\b\r",
      "STEP    79 / 119  f\b\b\r",
      "STEP    80 / 119  f\b\b\r",
      "STEP    81 / 119  f\b\b\r",
      "STEP    82 / 119  f\b\b\r",
      "STEP    83 / 119  f\b\b\r",
      "STEP    84 / 119  f\b\b\r",
      "STEP    85 / 119  f\b\b\r",
      "STEP    86 / 119  f\b\b\r",
      "STEP    87 / 119  f\b\b\r",
      "STEP    88 / 119  f\b\b\r",
      "STEP    89 / 119  f\b\b\r",
      "STEP    90 / 119  f\b\b\r",
      "STEP    91 / 119  f\b\b\r",
      "STEP    92 / 119  f\b\b\r",
      "STEP    93 / 119  f\b\b\r",
      "STEP    94 / 119  f\b\b\r",
      "STEP    95 / 119  f\b\b\r",
      "STEP    96 / 119  f\b\b\r",
      "STEP    97 / 119  f\b\b\r",
      "STEP    98 / 119  f\b\b\r",
      "STEP    99 / 119  f\b\b\r",
      "STEP   100 / 119  f\b\b\r",
      "STEP   101 / 119  f\b\b\r",
      "STEP   102 / 119  f\b\b\r",
      "STEP   103 / 119  f\b\b\r",
      "STEP   104 / 119  f\b\b\r",
      "STEP   105 / 119  f\b\b\r",
      "STEP   106 / 119  f\b\b\r",
      "STEP   107 / 119  f\b\b\r",
      "STEP   108 / 119  f\b\b\r",
      "STEP   109 / 119  f\b\b\r",
      "STEP   110 / 119  f\b\b\r",
      "STEP   111 / 119  f\b\b\r",
      "STEP   112 / 119  f\b\b\r",
      "STEP   113 / 119  f\b\b\r",
      "STEP   114 / 119  f\b\b\r",
      "STEP   115 / 119  f\b\b\r",
      "STEP   116 / 119  f\b\b\r",
      "STEP   117 / 119  f\b\b\r",
      "STEP   118 / 119  f\b\b\n",
      "Reallocating..done. *alloclen = 1246\n",
      "\r",
      "STEP   119 / 119  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|19255|19377|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|64491|216702|87000|87122|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|64491|216702|148190|148312|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|266682|267450|266741|266863|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|273344|278315|273931|274053|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgcagatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|278320|279091|278381|278503|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|300111|308544|307834|307956|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|378637|384812|378698|378820|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|459861|468373|466304|466426|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|468374|487224|479300|479422|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|520755|528026|520814|520936|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|545486|558918|558737|558859|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|622915|623662|623502|623624|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|865887|867389|867208|867330|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|921552|942506|932413|932535|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|984595|985335|984655|984777|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|985336|1008622|986017|986139|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1025530|1063042|1050159|1050281|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1189492|1208094|1198222|1198344|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1239481|1240251|1240069|1240191|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1290847|1306886|1295660|1295782|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1321493|1385605|1365526|1365648|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1398496|1399271|1399085|1399207|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagtgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1409877|1410649|1409940|1410062|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1424264|1425034|1424325|1424447|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ctttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1425133|1465118|1444867|1444989|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1509064|1578296|1527266|1527388|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1670306|1692658|1673959|1674081|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1729332|1741706|1736033|1736155|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1758378|1759147|1758965|1759087|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1759148|1776249|1767115|1767237|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1776250|1792578|1781500|1781622|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|1793868|1802424|1795188|1795310|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1842508|1869155|1850108|1850230|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|1940277|1941051|1940864|1940986|+\n",
      "-ctgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aag\n",
      ">G000006925|1976680|1977448|1976739|1976861|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2036705|2044875|2037214|2037336|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2050169|2051841|2051132|2051254|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2088344|2103602|2095627|2095749|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2120115|2138771|2134707|2134829|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2172359|2184266|2178379|2178501|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgctgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2187913|2198767|2196946|2197068|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000006925|2198768|2199537|2199356|2199478|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2199538|2210687|2205610|2205732|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgcagatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2211287|2225774|2221220|2221342|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2211287|2225774|2222690|2222812|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2235234|2236005|2235293|2235415|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2242494|2346249|2288032|2288154|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2374481|2472008|2395694|2395816|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2374481|2472008|2464668|2464790|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2756085|2756853|2756144|2756266|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2816684|2833035|2825788|2825910|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|2849870|2870340|2862899|2863021|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2849870|2870340|2864700|2864822|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2957651|2963246|2962752|2962874|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|2976303|3015537|3015220|3015342|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3016790|3036697|3022051|3022173|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3038529|3039299|3039116|3039238|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3104758|3106299|3105347|3105469|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3104758|3106299|3106116|3106238|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3179574|3180459|3179749|3179871|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3180744|3203996|3181640|3181762|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3237769|3244530|3237828|3237950|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgcagatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3268417|3281594|3273830|3273952|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3268417|3281594|3278198|3278320|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3322761|3361739|3361027|3361149|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatga\n",
      "aa-\n",
      ">G000006925|3361966|3403725|3362027|3362149|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3361966|3403725|3400957|3401079|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3361966|3403725|3403544|3403666|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3608503|3611864|3608567|3608689|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3608503|3611864|3611154|3611276|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3612083|3637192|3624820|3624942|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3612083|3637192|3627287|3627409|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3637330|3640023|3637917|3638039|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3658114|3676993|3665747|3665869|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3693106|3702974|3693167|3693289|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3703179|3717854|3703239|3703361|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3717879|3736288|3725523|3725645|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgctgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3717879|3736288|3727880|3728002|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3736321|3788106|3743609|3743731|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3736321|3788106|3762435|3762557|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3736321|3788106|3778553|3778675|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3815875|3829465|3818592|3818714|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgcagatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3831593|3850590|3835073|3835195|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|3831593|3850590|3845259|3845381|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3858569|3864367|3863658|3863780|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|3939882|3952896|3942893|3943015|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4002899|4012437|4002958|4003080|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4069200|4076620|4075913|4076035|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4091607|4092380|4092195|4092317|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgcagatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4094093|4116338|4102901|4103023|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4174522|4188276|4176622|4176744|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4256570|4288375|4285566|4285688|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4288513|4315006|4314296|4314418|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4362175|4367702|4367521|4367643|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4378758|4379529|4379345|4379467|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4389053|4390969|4389641|4389763|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4392644|4394283|4392708|4392830|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4397536|4410956|4410247|4410369|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4462252|4478752|4462840|4462962|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4462252|4478752|4465114|4465236|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4462252|4478752|4478043|4478165|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4478753|4490457|4488978|4489100|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4497401|4528664|4514929|4515051|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4497401|4528664|4519646|4519768|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000006925|4535976|4543278|4540569|4540691|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4557883|4573403|4569782|4569904|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4645852|4646621|4645911|4646033|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagattgcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4656735|4658205|4656795|4656917|-\n",
      "actgtagttgccatgttttacggcaatgagagcagagattgcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aa-\n",
      ">G000006925|4785482|4786253|4786071|4786193|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccacgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000299455|91523|102755|91587|91709|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000299455|696017|696792|696606|696728|+\n",
      "-ctgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aag\n",
      ">G000299455|941725|942502|942315|942437|+\n",
      "-ctgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aag\n",
      ">G000299455|1581872|1582655|1581936|1582058|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000299455|2052398|2064114|2062016|2062138|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000299455|3182222|3183821|3183636|3183758|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccaccccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000299455|3672841|3680073|3679358|3679480|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000299455|4588613|4589520|4589332|4589454|+\n",
      "-ctgtagttgccatgttttacggcaatgagagcagagatagcgctgatgtccggcagtgc\n",
      "ttttgccgttacgcaccatgccttcagtagcggagcaggaaggacatctgatggaaatgg\n",
      "aag\n",
      ">G000299455|4592900|4593681|4592966|4593088|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggacagctgatagaaacag\n",
      "aa-\n",
      ">G000299455|5323356|5324120|5323419|5323541|-\n",
      "actgcagttgccatgttttacggcagtgagagcagagatagcgctgatgtccggcggtgc\n",
      "ttttgccgttacgcaccaccccgtcagtagctgaacaggagggatagctgatagaaacag\n",
      "aa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4659810, 4664323)), (4662313, 4662468), '+'), (('G000299455', (5328484, 5333124)), (5330339, 5330494), '-'), (('G000299455', (5344024, 5346038)), (5344024, 5344179), '+')]\n",
      ">G000006925|4659810|4664323|4662313|4662468|+\n",
      "-tttcaggatgaaatattgcttggtagaactgtcataatataggtctcccggcgctcctc\n",
      "tccccccaaaaatactgacagtattaaccggtacattgatgtcactggcttttaatgtct\n",
      "gaaggtcgagtgttaccgttgattttttctcactgt\n",
      ">G000299455|5328484|5333124|5330339|5330494|-\n",
      "ctttcaggatgaaatattgcttggtagaactgtcataatacaggtctcccggcgctcctc\n",
      "tccccccaaaaatactgacagtattaaccggtacattgatgtcactggcttttaatgtct\n",
      "gaaggtcgagtgttaccgttgattttttctcactg-\n",
      ">G000299455|5344024|5346038|5344024|5344179|+\n",
      "-tttcaggatgtaatattgcccagtagaagtgtcataatacaggtctcccggcgttcctt\n",
      "tccccccaaagatactgacagtattaaccggtacattgatgtcactggcttttaatgtct\n",
      "gaaagtcgagtgttaccgttgattttttctcactgt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (434399, 459290)), (446228, 446282), '-'), (('G000299455', (324915, 324969)), (324915, 324969), '+'), (('G000299455', (3893653, 3918306)), (3906524, 3906578), '+')]\n",
      ">G000006925|434399|459290|446228|446282|-\n",
      "cggtgctgattatcgcctgcccgtgtgcgctgggactggcgacgccgatgtcga-\n",
      ">G000299455|324915|324969|324915|324969|+\n",
      "-ggtcctgattattgcctgtccctgcgcgctggggctggccacgccgatgtcgat\n",
      ">G000299455|3893653|3918306|3906524|3906578|+\n",
      "-ggtactgattatcgcctgtccgtgtgcgctggggctggcgacgccgatgtcgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (459384, 459436)), (459416, 459436), '-'), (('G000006925', (3612083, 3637192)), (3623447, 3623467), '-'), (('G000299455', (288342, 302750)), (291369, 291389), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|459384|459436|459416|459436|-\n",
      "gataaagttctcatcgttct-\n",
      ">G000006925|3612083|3637192|3623447|3623467|-\n",
      "gataaagcaatcatcgttct-\n",
      ">G000299455|288342|302750|291369|291389|+\n",
      "-ataaagcaatcatcgttctt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3554331, 3558215)), (3554332, 3554626), '+'), (('G000299455', (377310, 380460)), (380166, 380460), '-'), (('G000299455', (1999926, 2000220)), (1999926, 2000219), '-')]\n",
      ">G000006925|3554331|3558215|3554332|3554626|+\n",
      "-taataccccgccgttaaatcatccaccgagcgcgggtcggatgcgccgtacaactcacc\n",
      "gtccggcccaaccataatgctttgtgtgctgcccatcgcctctttcagcgccactttctg\n",
      "accttttgcttccagcagcttgagcgtatccgggctaaaccctttttcgacacgcagctc\n",
      "gtccggcaaccactgatggtggaaacgcggcgcattggtcgcttcggcgacgttcatgcc\n",
      "ataatcgatgctattcaccaccatttgcagcaccgtggtgataatccggctaccg\n",
      ">G000299455|377310|380460|380166|380460|-\n",
      "ttagtaccccgccgttaaatcatccaccgagcgcgggtcggatgcgccgtacaactcacc\n",
      "gtccggcccaaccataatgctttgtgtactgcccatcgcctctttcagcgccactttctg\n",
      "accttttgcttccagcagcttgagcgtatccgggctaaaccctttttcgacacgcagctc\n",
      "gtccggcaaccactgatggtggaaacgcggcgcattggtcgcttcggcgacgttcatgcc\n",
      "ataatcgatgctattcaccaccatttgcagcactgtagtgatgatccggctacc-\n",
      ">G000299455|1999926|2000220|1999926|2000219|-\n",
      "ttaataccccgccgttaaatcatccaccaagcgcgagtcggatgcgccgtacaactcacc\n",
      "atccgacccaaccataatgctttgcgtgctgcccatcgcctcattcagcgccactttctg\n",
      "accttttgcatccagcagcttgagcgtatccggactaaaccc-ttttcaacacgcagctc\n",
      "gtccggcaaccactgatagtggaaacgaggcgcattggtcgcttcggcaacattcattcc\n",
      "acaattgatgctgttcaccaccatttgcagcacggtagtgatgatccagctacc-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (732873, 732972), '+'), (('G000006925', (1934285, 1934732)), (1934630, 1934729), '-'), (('G000299455', (2174641, 2181396)), (2181269, 2181368), '+'), (('G000299455', (2910238, 2916987)), (2910260, 2910359), '-')]\n",
      ">G000006925|726235|732999|732873|732972|+\n",
      "-tggctacgaaagaagaaaacctgaaacgtctttgtgagctggctgagaggctggggcgt\n",
      "gaaccggatgtctccggtagtgcggcagatattgcgcagc\n",
      ">G000006925|1934285|1934732|1934630|1934729|-\n",
      "atggcaaagaaagacgataacctgaaacgtctgagagaactggcagcatcgctgggacgt\n",
      "gagccggatatctcgggaagcgcagcagatattgcgcag-\n",
      ">G000299455|2174641|2181396|2181269|2181368|+\n",
      "-tggcaacaaaagaagaaaatctgaatcgtcttcgtcaacttgctggcctgctggggcgc\n",
      "gaggcggatatgtcggggagtgctgcggatattgcgcaac\n",
      ">G000299455|2910238|2916987|2910260|2910359|-\n",
      "atggcaacaaaagaagaaaatcagaaacgtctttgtgaactggctggtctgctggggcgc\n",
      "gaggcggatatgtcggggagtgctgcggatatcgcacag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4117495, 4130655)), (4124155, 4124217), '+'), (('G000299455', (1508916, 1513891)), (1509391, 1509453), '+'), (('G000299455', (5034449, 5060698)), (5040887, 5040949), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1200\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4117495|4130655|4124155|4124217|+\n",
      "-ccaccattgttttcctggtgcgtaagggcaatccgaagcagatccatgactggaacgat\n",
      "ctg\n",
      ">G000299455|1508916|1513891|1509391|1509453|+\n",
      "-ccaccatgggcttcctggtgcgtaagggcaacccgaagaatatccacgactggaacgac\n",
      "ctg\n",
      ">G000299455|5034449|5060698|5040887|5040949|-\n",
      "tccaccattgttttcctggtgcgtaagggcaatccgaagcagatccatgactggaacgat\n",
      "ct-\n",
      "\n",
      "\n",
      "13 [(('G000006925', (385251, 385465)), (385282, 385305), '+'), (('G000006925', (900687, 900901)), (900847, 900871), '-'), (('G000006925', (3422918, 3423132)), (3423078, 3423102), '-'), (('G000006925', (3851029, 3851243)), (3851058, 3851083), '+'), (('G000006925', (4358341, 4358555)), (4358371, 4358395), '+'), (('G000006925', (4699715, 4699878)), (4699745, 4699769), '+'), (('G000006925', (4775368, 4775582)), (4775398, 4775422), '+'), (('G000006925', (4782659, 4782873)), (4782819, 4782843), '-'), (('G000299455', (918823, 918877)), (918823, 918847), '-'), (('G000299455', (3215867, 3216640)), (3216139, 3216162), '-'), (('G000299455', (4544905, 4544959)), (4544905, 4544928), '-'), (('G000299455', (4771787, 4771841)), (4771787, 4771810), '-'), (('G000299455', (5287974, 5288758)), (5288452, 5288475), '+')]\n",
      ">G000006925|385251|385465|385282|385305|+\n",
      "--ctccgcttctatgtcgtataact\n",
      ">G000006925|900687|900901|900847|900871|-\n",
      "atctccgcttctatgtcgtataac-\n",
      ">G000006925|3422918|3423132|3423078|3423102|-\n",
      "atctccgcttctatgtcgtataac-\n",
      ">G000006925|3851029|3851243|3851058|3851083|+\n",
      "atctccgcttctatgtcgtataact\n",
      ">G000006925|4358341|4358555|4358371|4358395|+\n",
      "-tctccgcttctatgtcgtataact\n",
      ">G000006925|4699715|4699878|4699745|4699769|+\n",
      "-tctccgcttctatgtcgtataact\n",
      ">G000006925|4775368|4775582|4775398|4775422|+\n",
      "-tctccgcttctatgtcgtataact\n",
      ">G000006925|4782659|4782873|4782819|4782843|-\n",
      "atctccgcttctatgtcgtataac-\n",
      ">G000299455|918823|918877|918823|918847|-\n",
      "atatccgcctctatggcgtacaac-\n",
      ">G000299455|3215867|3216640|3216139|3216162|-\n",
      "-tctctgcctcgatggcgtacagt-\n",
      ">G000299455|4544905|4544959|4544905|4544928|-\n",
      "-tatccgcctctatggcgtacaac-\n",
      ">G000299455|4771787|4771841|4771787|4771810|-\n",
      "-tatccgcctctatggcgtacaac-\n",
      ">G000299455|5287974|5288758|5288452|5288475|+\n",
      "--ctctgcctcgatggcgtacagtt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1729332, 1741706)), (1739970, 1740002), '-'), (('G000299455', (2416691, 2440660)), (2430765, 2430797), '+'), (('G000299455', (3057394, 3103258)), (3067812, 3067844), '+')]\n",
      ">G000006925|1729332|1741706|1739970|1740002|-\n",
      "aaccttgcagagctgcgcctttattattatcc-\n",
      ">G000299455|2416691|2440660|2430765|2430797|+\n",
      "-accttgcagagctgcgcctttattattatccg\n",
      ">G000299455|3057394|3103258|3067812|3067844|+\n",
      "-acattgcggagctgcgtctctattactacccg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (865572, 865846)), (865786, 865815), '-'), (('G000006925', (2242494, 2346249)), (2252159, 2252188), '-'), (('G000006925', (3229824, 3229896)), (3229865, 3229894), '+'), (('G000299455', (1819843, 1851664)), (1826320, 1826349), '+'), (('G000299455', (3242963, 3243106)), (3242988, 3243017), '+'), (('G000299455', (3470062, 3470342)), (3470282, 3470311), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 13\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 13\r",
      "   10 / 13\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 12  f\b\b\r",
      "STEP     2 / 12  f\b\b\r",
      "STEP     3 / 12  f\b\b\r",
      "STEP     4 / 12  f\b\b\r",
      "STEP     5 / 12  f\b\b\r",
      "STEP     6 / 12  f\b\b\r",
      "STEP     7 / 12  f\b\b\r",
      "STEP     8 / 12  f\b\b\r",
      "STEP     9 / 12  f\b\b\r",
      "STEP    10 / 12  f\b\b\r",
      "STEP    11 / 12  f\b\b\r",
      "STEP    12 / 12  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 13\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 13\r",
      "   10 / 13\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 12  f\b\b\r",
      "STEP     2 / 12  f\b\b\r",
      "STEP     3 / 12  f\b\b\r",
      "STEP     4 / 12  f\b\b\r",
      "STEP     5 / 12  f\b\b\r",
      "STEP     6 / 12  f\b\b\r",
      "STEP     7 / 12  f\b\b\r",
      "STEP     8 / 12  f\b\b\r",
      "STEP     9 / 12  f\b\b\r",
      "STEP    10 / 12  f\b\b\r",
      "STEP    11 / 12  f\b\b\r",
      "STEP    12 / 12  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1060\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|865572|865846|865786|865815|-\n",
      "aaatcgctgaagcgttccccgaaggccgg-\n",
      ">G000006925|2242494|2346249|2252159|2252188|-\n",
      "aaatcgccgaggcgttccctgaaggccgg-\n",
      ">G000006925|3229824|3229896|3229865|3229894|+\n",
      "-aatcgccgaagcgttccctggaggccggg\n",
      ">G000299455|1819843|1851664|1826320|1826349|+\n",
      "-aattgccgaggcgttccctgaaggccggg\n",
      ">G000299455|3242963|3243106|3242988|3243017|+\n",
      "-aatcgctgaagcgttccccgaaggccggg\n",
      ">G000299455|3470062|3470342|3470282|3470311|-\n",
      "aaatcgccgaggcgttccctgaaggccgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (865572, 865846)), (865680, 865725), '-'), (('G000299455', (1819843, 1851664)), (1826411, 1826456), '+'), (('G000299455', (3470062, 3470342)), (3470175, 3470220), '-')]\n",
      ">G000006925|865572|865846|865680|865725|-\n",
      "cggttgccgaaggggataagctgaaggtaccgcgcagcggcgatt-\n",
      ">G000299455|1819843|1851664|1826411|1826456|+\n",
      "-ggtagccgtaagggataagccgagggcaccgcgcagcggcgattt\n",
      ">G000299455|3470062|3470342|3470175|3470220|-\n",
      "cggtagccgtaagggataagtcgagggcaccgcgcagcggcgatt-\n",
      "\n",
      "\n",
      "23 [(('G000006925', (237940, 238704)), (238653, 238698), '+'), (('G000006925', (539107, 539868)), (539819, 539864), '+'), (('G000006925', (709671, 710707)), (709950, 709995), '-'), (('G000006925', (716695, 717456)), (717407, 717452), '+'), (('G000006925', (902326, 903088)), (903038, 903083), '+'), (('G000006925', (1095979, 1096741)), (1095984, 1096029), '-'), (('G000006925', (1420617, 1420678)), (1420622, 1420667), '-'), (('G000006925', (1632621, 1633383)), (1633333, 1633378), '+'), (('G000006925', (1834903, 1835664)), (1834907, 1834952), '-'), (('G000006925', (1878572, 1883763)), (1879284, 1879329), '+'), (('G000006925', (1944438, 1945202)), (1945150, 1945195), '+'), (('G000006925', (1962084, 1962845)), (1962796, 1962841), '+'), (('G000006925', (2069615, 2084300)), (2070327, 2070372), '+'), (('G000006925', (2346737, 2347498)), (2347449, 2347494), '+'), (('G000006925', (2477790, 2478551)), (2478502, 2478547), '+'), (('G000006925', (2556960, 2557020)), (2556971, 2557016), '+'), (('G000006925', (2953405, 2954171)), (2953410, 2953455), '-'), (('G000006925', (4392644, 4394283)), (4393526, 4393571), '-'), (('G000006925', (4798615, 4798792)), (4798668, 4798713), '+'), (('G000006925', (4823851, 4824717)), (4823960, 4824005), '-'), (('G000299455', (928393, 928731)), (928678, 928723), '+'), (('G000299455', (1290912, 1290981)), (1290929, 1290974), '+'), (('G000299455', (4559929, 4560763)), (4560639, 4560684), '+')]\n",
      ">G000006925|237940|238704|238653|238698|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|539107|539868|539819|539864|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|709671|710707|709950|709995|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|716695|717456|717407|717452|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|902326|903088|903038|903083|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|1095979|1096741|1095984|1096029|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|1420617|1420678|1420622|1420667|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|1632621|1633383|1633333|1633378|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|1834903|1835664|1834907|1834952|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|1878572|1883763|1879284|1879329|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|1944438|1945202|1945150|1945195|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|1962084|1962845|1962796|1962841|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|2069615|2084300|2070327|2070372|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|2346737|2347498|2347449|2347494|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|2477790|2478551|2478502|2478547|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000006925|2556960|2557020|2556971|2557016|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaatttagtgtgccac\n",
      ">G000006925|2953405|2954171|2953410|2953455|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|4392644|4394283|4393526|4393571|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000006925|4798615|4798792|4798668|4798713|+\n",
      "-gtgaacatatcacctctgttcaggtagccaaattcagtgtgccac\n",
      ">G000006925|4823851|4824717|4823960|4824005|-\n",
      "ggtgagcatatcacctctgttcaggtggccaaattcagtgtgcca-\n",
      ">G000299455|928393|928731|928678|928723|+\n",
      "-gtgagcatatcacctctgttcaggtggccaaattcagtgtgccac\n",
      ">G000299455|1290912|1290981|1290929|1290974|+\n",
      "-gtgagaatatcacctttcatcaggtggccaaatttagtgtgccac\n",
      ">G000299455|4559929|4560763|4560639|4560684|+\n",
      "-gtgagcatatcacctctgttcaggtagctaaatttagtgtgccac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2770832, 2816659)), (2791543, 2791567), '-'), (('G000299455', (1237061, 1253885)), (1253861, 1253885), '+'), (('G000299455', (1253944, 1254160)), (1254134, 1254158), '+'), (('G000299455', (1254267, 1277763)), (1254267, 1254291), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 23\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 23\r",
      "   10 / 23\r",
      "   20 / 23\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 22  f\b\b\r",
      "STEP     2 / 22  f\b\b\r",
      "STEP     3 / 22  f\b\b\r",
      "STEP     4 / 22  f\b\b\r",
      "STEP     5 / 22  f\b\b\r",
      "STEP     6 / 22  f\b\b\r",
      "STEP     7 / 22  f\b\b\r",
      "STEP     8 / 22  f\b\b\r",
      "STEP     9 / 22  f\b\b\r",
      "STEP    10 / 22  f\b\b\r",
      "STEP    11 / 22  f\b\b\r",
      "STEP    12 / 22  f\b\b\r",
      "STEP    13 / 22  f\b\b\r",
      "STEP    14 / 22  f\b\b\r",
      "STEP    15 / 22  f\b\b\r",
      "STEP    16 / 22  f\b\b\r",
      "STEP    17 / 22  f\b\b\r",
      "STEP    18 / 22  f\b\b\r",
      "STEP    19 / 22  f\b\b\r",
      "STEP    20 / 22  f\b\b\r",
      "STEP    21 / 22  f\b\b\r",
      "STEP    22 / 22  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 23\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 23\r",
      "   10 / 23\r",
      "   20 / 23\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 22  f\b\b\r",
      "STEP     2 / 22  f\b\b\r",
      "STEP     3 / 22  f\b\b\r",
      "STEP     4 / 22  f\b\b\r",
      "STEP     5 / 22  f\b\b\r",
      "STEP     6 / 22  f\b\b\r",
      "STEP     7 / 22  f\b\b\r",
      "STEP     8 / 22  f\b\b\r",
      "STEP     9 / 22  f\b\b\r",
      "STEP    10 / 22  f\b\b\r",
      "STEP    11 / 22  f\b\b\r",
      "STEP    12 / 22  f\b\b\r",
      "STEP    13 / 22  f\b\b\r",
      "STEP    14 / 22  f\b\b\r",
      "STEP    15 / 22  f\b\b\r",
      "STEP    16 / 22  f\b\b\r",
      "STEP    17 / 22  f\b\b\r",
      "STEP    18 / 22  f\b\b\r",
      "STEP    19 / 22  f\b\b\r",
      "STEP    20 / 22  f\b\b\r",
      "STEP    21 / 22  f\b\b\r",
      "STEP    22 / 22  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2770832|2816659|2791543|2791567|-\n",
      "aatcctcccggatgcaccatattc-\n",
      ">G000299455|1237061|1253885|1253861|1253885|+\n",
      "-atcctcccggatgcaccatattct\n",
      ">G000299455|1253944|1254160|1254134|1254158|+\n",
      "-atcctcccggatgcaccatattct\n",
      ">G000299455|1254267|1277763|1254267|1254291|+\n",
      "-atcctcccggatgcaccatcttct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2116355, 2116567)), (2116355, 2116567), '+'), (('G000006925', (3969227, 3993100)), (3982241, 3982453), '-'), (('G000299455', (5156132, 5190970)), (5177744, 5177956), '+')]\n",
      ">G000006925|2116355|2116567|2116355|2116567|+\n",
      "-tcaccaatgataaatgcttgcgccagaccatccggactcggttgtactttatattgcag\n",
      "attaagcccccactggcttccatcaccaagcaattgttgaaaacgtggtgtatcctgcgg\n",
      "cgtactgataatcagaatatctcgaatacccgccaacataagcgttgaaagcggataata\n",
      "aatcatcggtttatcataaatcggtagcagttg\n",
      ">G000006925|3969227|3993100|3982241|3982453|-\n",
      "ttcaccgatgataaaggcctgcgccagaccgtccgggctgggctgttcggcatattccag\n",
      "ctggataccgaactcactaccatcgcccagcaggcgctggaaataacctttatcttccgg\n",
      "cgtagtgatgatgagaatttcgcggataccggccagcatcagcaccgacagcggatagta\n",
      "aatcattggcttatcgtaaatcggcaacagtt-\n",
      ">G000299455|5156132|5190970|5177744|5177956|+\n",
      "-tcaccgatgataaaggcctgcgccagaccgtccgggctgggctgttcggcgtattccag\n",
      "ctggataccgaactcactaccatcgcccagcaggcgctggaaataacctttatcttccgg\n",
      "cgtagtgatgatgagaatttcgcggataccggccaacatcagcaccgacagcggatagta\n",
      "aatcattggcttatcgtaaatcggcaacagttg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1425133, 1465118)), (1452755, 1452785), '-'), (('G000006925', (3246783, 3268347)), (3250548, 3250578), '-'), (('G000299455', (679634, 695969)), (690309, 690339), '+'), (('G000299455', (1078152, 1154001)), (1135236, 1135266), '-'), (('G000299455', (2199778, 2263225)), (2250856, 2250886), '-')]\n",
      ">G000006925|1425133|1465118|1452755|1452785|-\n",
      "atgcagaacgaactggcgctgcacagcaag-\n",
      ">G000006925|3246783|3268347|3250548|3250578|-\n",
      "atgcacaacgagctggcgctgcgcagcaaa-\n",
      ">G000299455|679634|695969|690309|690339|+\n",
      "-tgcacaacgagctagcgctgcgcagtaaag\n",
      ">G000299455|1078152|1154001|1135236|1135266|-\n",
      "atgaaaaacgagctggcgctgcacagcaaa-\n",
      ">G000299455|2199778|2263225|2250856|2250886|-\n",
      "atgcagaacgaactggcgctgcacagcaag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (722152, 722887)), (722359, 722409), '+'), (('G000299455', (2171310, 2171838)), (2171310, 2171360), '+'), (('G000299455', (2920506, 2921242)), (2920984, 2921034), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|722152|722887|722359|722409|+\n",
      "-cgataaatcctgataaatatccatgagcgcaaaaatcaaatacggcctgt\n",
      ">G000299455|2171310|2171838|2171310|2171360|+\n",
      "-cgataaatcctgataaatatccatgaacaccaaaatcaaatatggcctgt\n",
      ">G000299455|2920506|2921242|2920984|2921034|-\n",
      "ccgataaatcctgataaatatccatgaacaccaaaatcaaatacggcctg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4612466, 4613007)), (4612910, 4613007), '-'), (('G000006925', (4622488, 4624460)), (4623351, 4623448), '+'), (('G000006925', (4689001, 4689359)), (4689089, 4689186), '-'), (('G000299455', (5313369, 5315826)), (5314588, 5314685), '+')]\n",
      ">G000006925|4612466|4613007|4612910|4613007|-\n",
      "cttccgaaatcaaccgactgaaggcgctggtcgcgaaactgcagcgtatgcagttcggta\n",
      "aaagctcagaaaaacttcgcgcaaaaaccgaacggtg-\n",
      ">G000006925|4622488|4624460|4623351|4623448|+\n",
      "-ttccgaaatcaaccggctgaaggcgctggttgcgaaactgcagcgcatgcagttcggta\n",
      "aaagctcagaaaaacttcgcgcaaaaaccgaacggcag\n",
      ">G000006925|4689001|4689359|4689089|4689186|-\n",
      "cttccgaaatcaaccgactgaaggcgctggttgcgaaactgcagcgtatgcagttcggta\n",
      "aaagctcagaaaaacttcgcgcaaaaaccgaacggca-\n",
      ">G000299455|5313369|5315826|5314588|5314685|+\n",
      "-ttccgaaatcaaccggctgaaggcgctggttgcgaaactgcaacgtatgcagttcggta\n",
      "aaagctcagaaaaacttcgtgcaaaaaccgaacggcag\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1815406, 1827803)), (1826176, 1826202), '+'), (('G000006925', (1965007, 1976679)), (1970050, 1970076), '+'), (('G000006925', (4557883, 4573403)), (4559193, 4559219), '-'), (('G000299455', (2122044, 2133716)), (2128647, 2128673), '-'), (('G000299455', (2122044, 2133716)), (2130348, 2130374), '-'), (('G000299455', (2737179, 2754578)), (2748314, 2748340), '+'), (('G000299455', (4487440, 4499283)), (4497947, 4497973), '+')]\n",
      ">G000006925|1815406|1827803|1826176|1826202|+\n",
      "-gactgattcactgatcaagccttcaa\n",
      ">G000006925|1965007|1976679|1970050|1970076|+\n",
      "-gacggagtcttcaatgagggctttga\n",
      ">G000006925|4557883|4573403|4559193|4559219|-\n",
      "ccaccgagtcttcaatcaggctttta-\n",
      ">G000299455|2122044|2133716|2128647|2128673|-\n",
      "cgacggagtcttcaatgagggctttg-\n",
      ">G000299455|2122044|2133716|2130348|2130374|-\n",
      "tgactgactcttcgatcagccctttg-\n",
      ">G000299455|2737179|2754578|2748314|2748340|+\n",
      "-gactgattcactgatcaagccttcaa\n",
      ">G000299455|4487440|4499283|4497947|4497973|+\n",
      "-caccgagtcttcaatcaggcttttaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3083675, 3099254)), (3089328, 3089517), '-'), (('G000299455', (857924, 871846)), (865996, 866185), '+'), (('G000299455', (3151327, 3152597)), (3152409, 3152597), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\n",
      "Reallocating..done. *alloclen = 1054\n",
      "\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3083675|3099254|3089328|3089517|-\n",
      "cttattcccctggataaaggccccttcctccgtggtggcaccgttaatggcttccaccac\n",
      "ctgaatacccttaccgctggtcgccagccccgacgcactgttgccggcgttcaccaggtt\n",
      "caggatggtttttccggttgccctgccaccgtcaatgaccagtctgtcagcattgttctg\n",
      "tgccatatc-\n",
      ">G000299455|857924|871846|865996|866185|+\n",
      "-atattcccctggacaaaagccccttcctccgtggtggcaccgttaatggcctcaaccac\n",
      "ctgaatacccttaccggtggtcgccagccccgtaccactgttgccggcgttcaccagatt\n",
      "caggatggtttttccggttgccctgccaccgtcaatgaccagtctgtcagcattgttctg\n",
      "cgccatatcc\n",
      ">G000299455|3151327|3152597|3152409|3152597|+\n",
      "--tgttcccctggacaaaggccccttcctccgtggtggcaccgttaatggcttcaaccac\n",
      "ctgaatacccttaccgctggtcgccagccccgacgcactgttgccggcgttcaccaggtt\n",
      "caggatggtttttccggttgccctgccgccgtcaatgaccagtctgtcagcattgttctg\n",
      "tgccatatcc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2849870, 2870340)), (2864256, 2864311), '+'), (('G000299455', (1158891, 1160694)), (1159991, 1160046), '-'), (('G000299455', (4585236, 4585291)), (4585236, 4585291), '+')]\n",
      ">G000006925|2849870|2870340|2864256|2864311|+\n",
      "-cctcaccttgcagggtctggaacatctcgttaatcgggtactgcatagcattctc\n",
      ">G000299455|1158891|1160694|1159991|1160046|-\n",
      "ccctcaccttgcagggtctggaacatctcgttaatcgggtactgcatagcattct-\n",
      ">G000299455|4585236|4585291|4585236|4585291|+\n",
      "-cctcaccttgcagggtctggaaaatctagctaatcgggtactgcataacactctc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3681874, 3691741)), (3691121, 3691157), '-'), (('G000299455', (211788, 221655)), (212373, 212409), '+'), (('G000299455', (3326617, 3362069)), (3333105, 3333141), '+')]\n",
      ">G000006925|3681874|3691741|3691121|3691157|-\n",
      "gtcagcgttgaaccattttacgataccagtcatttt-\n",
      ">G000299455|211788|221655|212373|212409|+\n",
      "-tcagcgttgaaccattttacgataccagtcatttta\n",
      ">G000299455|3326617|3362069|3333105|3333141|+\n",
      "-tctgcgttaaaccattttactaaaccagtcatttta\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1509064, 1578296)), (1577275, 1577336), '+'), (('G000299455', (2167395, 2167615)), (2167407, 2167468), '-'), (('G000299455', (2547013, 2547107)), (2547022, 2547082), '-'), (('G000299455', (2923865, 2925880)), (2924863, 2924924), '+'), (('G000299455', (3291704, 3291790)), (3291729, 3291790), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1577275|1577336|+\n",
      "-tgccagtttcttttaacaaactccccgcaaaccagacaacagtcaaccgcctgaattgt\n",
      "ga\n",
      ">G000299455|2167395|2167615|2167407|2167468|-\n",
      "atgccagtttcttttaacaaactccccgcaaaacagacaactgtcaaccgtctgaattgt\n",
      "g-\n",
      ">G000299455|2547013|2547107|2547022|2547082|-\n",
      "atgc-agttttttctaacaaattctcatccgggcagacaacattcaactgacttaattgt\n",
      "g-\n",
      ">G000299455|2923865|2925880|2924863|2924924|+\n",
      "-tgccagtttcttttaacaaactccccgcaaaccagacaacagtcaaccgcctgaattgt\n",
      "ga\n",
      ">G000299455|3291704|3291790|3291729|3291790|+\n",
      "-tgcgaaatttttttaacaaatgctcagtctgacaggcaactgtcaactgactgaattgt\n",
      "ga\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3608058, 3608452)), (3608058, 3608452), '+'), (('G000006925', (4392199, 4392593)), (4392199, 4392593), '+'), (('G000299455', (4670382, 4673512)), (4670382, 4670775), '-')]\n",
      ">G000006925|3608058|3608452|3608058|3608452|+\n",
      "-actccatcgtaagcattacacttcgttgctaacttattgaatgcttgccagtcagagag\n",
      "atctagtttgtggatctcaacagtacgtctgccatcgcaattatgatctcttccacgatt\n",
      "tgacagtaatgtagcgcgtgcgtattgatgaaattttaattttgctggtttgtcggtgga\n",
      "cgaaaaacggggtaaatatgcggggaatgaataaatagttgttggtaatgtcattttatt\n",
      "gttgcctgtttaatgtttttacaaatatcgcagttgaatgttcaaagaaaaagaattgat\n",
      "tttaataaaaataagtttgttttctgaagtggtgaaaatagcggatggtgaaagatatta\n",
      "atatcaaaaatcgctcatcacggatataacgatgc\n",
      ">G000006925|4392199|4392593|4392199|4392593|+\n",
      "-actccatcgtaagcattacacttcgttgctaacttattgaatgcttgccagtcagagag\n",
      "atctagtttgtggatctcaacagtacgtctgccatcgcaattatgatctcttccacgatt\n",
      "tgacagtaatgtagcgcgtgcgtattgatgaaattttaattttgctggtttgtcggtgga\n",
      "cgaaaaacggggtaaatatgcggggaatgaataaatagttgttggtaatgtcattttatt\n",
      "gttgcctgtttaatgtttttacaaatatcgcagttgaatgttcaaagaaaaagaattgat\n",
      "tttaataaaaataagtttgttttctgaagtggtgaaaatagcggatggtgaaagatatta\n",
      "atatcaaaaatcgctcatcacggatataacgatgc\n",
      ">G000299455|4670382|4673512|4670382|4670775|-\n",
      "tactccatcgtaagcattacacttcgttgctaacttattgaatgcttgccagtcagagag\n",
      "atctagtttgtggatctcaacagtacgtctgccatcgcaattatgatctcttccacgatt\n",
      "tggcagtaatgtagcacgtgcatattgatgaaattttaattttactggtttgtcggtgga\n",
      "cgaaaaatggggtgaatatgcaggaaatgaataaatagtcgttggtaatgtcattgtaat\n",
      "gttacctgtttaatgtttttataaatatcgcagctg-atgttcacagaaaaagaatcgat\n",
      "attaataaaaataagtttgttttctgaagtagtgaaaatagcggatggtgaaagatatta\n",
      "atatcaaaaatcgctcattatggatataacgatg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3939882, 3952896)), (3944475, 3944512), '-'), (('G000006925', (4433907, 4437456)), (4436171, 4436209), '+'), (('G000299455', (4626810, 4646234)), (4644949, 4644987), '+'), (('G000299455', (5202806, 5268469)), (5216695, 5216733), '+')]\n",
      ">G000006925|3939882|3952896|3944475|3944512|-\n",
      "ggatcaatactttgggg-gtgtcatcagaccgcgggca-\n",
      ">G000006925|4433907|4437456|4436171|4436209|+\n",
      "-gatcagaaattgcggtcgggtcagtagccaacgtgaaa\n",
      ">G000299455|4626810|4646234|4644949|4644987|+\n",
      "-gatcagaaattgcggtcgggtcagtagccaacgtgaaa\n",
      ">G000299455|5202806|5268469|5216695|5216733|+\n",
      "-gatcaacactttggggcgtgtcatcagaccgcgggcaa\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1397159, 1397202)), (1397168, 1397201), '+'), (('G000299455', (1577719, 1577762)), (1577720, 1577753), '-'), (('G000299455', (2150760, 2150803)), (2150761, 2150794), '-'), (('G000299455', (2541283, 2541318)), (2541284, 2541317), '+'), (('G000299455', (2582367, 2582587)), (2582367, 2582400), '-'), (('G000299455', (3311764, 3311807)), (3311773, 3311806), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1397159|1397202|1397168|1397201|+\n",
      "-cggtaactctggctggctggataagctgtagtg\n",
      ">G000299455|1577719|1577762|1577720|1577753|-\n",
      "ccggtaactccggatggctggataagctgtagt-\n",
      ">G000299455|2150760|2150803|2150761|2150794|-\n",
      "ccggtaactccggatggttggataacctgtagt-\n",
      ">G000299455|2541283|2541318|2541284|2541317|+\n",
      "-cggtaactctggatggttggataagctgtagtg\n",
      ">G000299455|2582367|2582587|2582367|2582400|-\n",
      "ccggtaactccggatggttggataagctgtagt-\n",
      ">G000299455|3311764|3311807|3311773|3311806|+\n",
      "-cggtaactccggatagttggataagctgtagtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1637263, 1657511)), (1644126, 1644206), '-'), (('G000299455', (1529161, 1529241)), (1529161, 1529241), '+'), (('G000299455', (2505935, 2525742)), (2519242, 2519322), '+')]\n",
      ">G000006925|1637263|1657511|1644126|1644206|-\n",
      "cgttactttgttgctgttgcggaagagctgcattttgggcgcgcggctgcccgcctgaat\n",
      "atttcgcaaccgccgctaag-\n",
      ">G000299455|1529161|1529241|1529161|1529241|+\n",
      "-gttattttcttgccgtagcggaagagttgcattttggccgcgcagcagcgcgtttaaat\n",
      "atgtctcagcctccgctcagt\n",
      ">G000299455|2505935|2525742|2519242|2519322|+\n",
      "-gttactttgttgctgttgcggaagagctgcatttcgggcgcgccgctgcccgcctgaat\n",
      "atttcgcaaccgccgctaagt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (2242494, 2346249)), (2290220, 2290267), '+'), (('G000006925', (4367703, 4374506)), (4370713, 4370760), '-'), (('G000299455', (1752647, 1813107)), (1788719, 1788766), '-'), (('G000299455', (2596243, 2603992)), (2598112, 2598159), '+'), (('G000299455', (4524008, 4535561)), (4531768, 4531815), '-')]\n",
      ">G000006925|2242494|2346249|2290220|2290267|+\n",
      "-cagtgggtggttgaagataacttcatcgctgggcgtcctgcctggga\n",
      ">G000006925|4367703|4374506|4370713|4370760|-\n",
      "tcagtgggtgattgaagataacttcgttaatggtcgcccggactggg-\n",
      ">G000299455|1752647|1813107|1788719|1788766|-\n",
      "ccagtgggtggtggaagataacttcgtcgctgggcgtcctgcctggg-\n",
      ">G000299455|2596243|2603992|2598112|2598159|+\n",
      "-cagtgggtaatagaagataactttgttgccggacgtccagaatggga\n",
      ">G000299455|4524008|4535561|4531768|4531815|-\n",
      "tcagtgggtgattgaagataacttcgttaatggtcgtccggactggg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3788320, 3808082)), (3806474, 3806497), '-'), (('G000299455', (47579, 47602)), (47579, 47602), '+'), (('G000299455', (91523, 102755)), (93883, 93906), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3788320|3808082|3806474|3806497|-\n",
      "tggcggaagatcacaggagtcga-\n",
      ">G000299455|47579|47602|47579|47602|+\n",
      "-ggcggaagatcacaggagtcgaa\n",
      ">G000299455|91523|102755|93883|93906|+\n",
      "-ggcggaagatcacaggagtcgaa\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2870402, 2886688)), (2871560, 2871637), '-'), (('G000006925', (3788320, 3808082)), (3793666, 3793743), '+'), (('G000299455', (102882, 112192)), (106768, 106845), '-'), (('G000299455', (1078152, 1154001)), (1152766, 1152843), '+')]\n",
      ">G000006925|2870402|2886688|2871560|2871637|-\n",
      "gttgagatccaaatccgcaccaaacagatgcatgaagatgcggagttgggtgttgctgcg\n",
      "cactggaaatataaaga-\n",
      ">G000006925|3788320|3808082|3793666|3793743|+\n",
      "-ttgaggtccagatccgtaccgaagatatggatcagatggcggagatgggtgttgccgcg\n",
      "cactgggcttataaagag\n",
      ">G000299455|102882|112192|106768|106845|-\n",
      "gttgaggtccagatccgtaccgaagatatggatcagatggcggagatgggtgttgccgcg\n",
      "cactgggcttataaaga-\n",
      ">G000299455|1078152|1154001|1152766|1152843|+\n",
      "-ttgagatccaaatccgcaccaaacagatgcatgaagatgcagagttgggtgttgctgcg\n",
      "cactggaaatataaagag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (64491, 216702)), (201775, 201797), '-'), (('G000299455', (4235244, 4293324)), (4240428, 4240450), '+'), (('G000299455', (4789263, 4790612)), (4789975, 4789997), '-')]\n",
      ">G000006925|64491|216702|201775|201797|-\n",
      "-gcttcgttaaacgcctcttcgt-\n",
      ">G000299455|4235244|4293324|4240428|4240450|+\n",
      "--cttcgttaaacgcctcttcgtc\n",
      ">G000299455|4789263|4790612|4789975|4789997|-\n",
      "ggcttcgttaaaggtttcttcg--\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4094093, 4116338)), (4100846, 4100867), '+'), (('G000299455', (2682449, 2700553)), (2690115, 2690136), '+'), (('G000299455', (5063005, 5071226)), (5064452, 5064473), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1156\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1156\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4094093|4116338|4100846|4100867|+\n",
      "-cctcttcccaactgatttgct\n",
      ">G000299455|2682449|2700553|2690115|2690136|+\n",
      "-cttcttcccagctaatgcgct\n",
      ">G000299455|5063005|5071226|5064452|5064473|-\n",
      "gcctcttcccaactgatttgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3250578, 3250603), '-'), (('G000299455', (679634, 695969)), (690284, 690309), '+'), (('G000299455', (1078152, 1154001)), (1135266, 1135291), '-')]\n",
      ">G000006925|3246783|3268347|3250578|3250603|-\n",
      "cggcctgtctatatctggcctgatg-\n",
      ">G000299455|679634|695969|690284|690309|+\n",
      "-ggcctgtctatatctggcctgatga\n",
      ">G000299455|1078152|1154001|1135266|1135291|-\n",
      "cgggctgtcactctctggcctgatg-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (712805, 712845)), (712811, 712845), '+'), (('G000006925', (1509064, 1578296)), (1575973, 1576007), '-'), (('G000299455', (2169561, 2170070)), (2169848, 2169882), '+'), (('G000299455', (2550010, 2550272)), (2550051, 2550085), '+'), (('G000299455', (2921864, 2922373)), (2922052, 2922086), '-'), (('G000299455', (3288518, 3288775)), (3288705, 3288739), '-')]\n",
      ">G000006925|712805|712845|712811|712845|+\n",
      "-gggatatgtatcagatggaaaaaatatcaacagg\n",
      ">G000006925|1509064|1578296|1575973|1576007|-\n",
      "cggggtatggaccagatggaaaaaatcacaacag-\n",
      ">G000299455|2169561|2170070|2169848|2169882|+\n",
      "-ggggtatgtaccagatggaaaaaatcacaacagg\n",
      ">G000299455|2550010|2550272|2550051|2550085|+\n",
      "-ggggtatgtaccagatggaaaaaatcacaacagg\n",
      ">G000299455|2921864|2922373|2922052|2922086|-\n",
      "cggggtatgtaccagatggaaaaaatcacaacag-\n",
      ">G000299455|3288518|3288775|3288705|3288739|-\n",
      "cggggtatgaaccagatggaaaaaatcacaacag-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (913347, 913425)), (913350, 913373), '+'), (('G000006925', (1419463, 1419488)), (1419463, 1419485), '-'), (('G000299455', (2157901, 2157977)), (2157902, 2157925), '+'), (('G000299455', (2389476, 2389513)), (2389488, 2389510), '-'), (('G000299455', (3302922, 3302956)), (3302922, 3302945), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|913347|913425|913350|913373|+\n",
      "-acctcataaaacgggtgccggga\n",
      ">G000006925|1419463|1419488|1419463|1419485|-\n",
      "cacctcataaaacgggggccgg--\n",
      ">G000299455|2157901|2157977|2157902|2157925|+\n",
      "-accttataaagcgggtgccggga\n",
      ">G000299455|2389476|2389513|2389488|2389510|-\n",
      "cacctcataaaacgggtgccgg--\n",
      ">G000299455|3302922|3302956|3302922|3302945|+\n",
      "-acctcataaaacgggtgtcggga\n",
      "\n",
      "\n",
      "4 [(('G000006925', (510419, 510447)), (510419, 510447), '+'), (('G000006925', (3692630, 3692658)), (3692630, 3692658), '-'), (('G000006925', (4642783, 4642811)), (4642783, 4642811), '-'), (('G000299455', (5286933, 5286961)), (5286933, 5286961), '+')]\n",
      ">G000006925|510419|510447|510419|510447|+\n",
      "-atctaaaaaagctgaaagccttagctca\n",
      ">G000006925|3692630|3692658|3692630|3692658|-\n",
      "tatctaaaaaagctgaaagccttagctc-\n",
      ">G000006925|4642783|4642811|4642783|4642811|-\n",
      "tatctaaaaaagctgaaagccttagctc-\n",
      ">G000299455|5286933|5286961|5286933|5286961|+\n",
      "-atctaaaaaagctgaaagccttagctca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2772046, 2772114), '-'), (('G000299455', (1254267, 1277763)), (1273722, 1273790), '+'), (('G000299455', (2959070, 2965951)), (2960840, 2960908), '+')]\n",
      ">G000006925|2770832|2816659|2772046|2772114|-\n",
      "gcgtccggcgaaatataaattcactgatgttaacggtgaaactaaaacctggaccggtca\n",
      "gggccgta-\n",
      ">G000299455|1254267|1277763|1273722|1273790|+\n",
      "-cgtccggcgaaatataaattcaccgatgttaacggtgaaactaaaacctggaccggtca\n",
      "gggccgtac\n",
      ">G000299455|2959070|2965951|2960840|2960908|+\n",
      "-cgtccggcaaaatatagctacgttgacgaaaacggcgaaactaaaacctggactggcca\n",
      "aggccgtac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (496417, 503320)), (496418, 496742), '-'), (('G000299455', (319193, 319517)), (319193, 319517), '-'), (('G000299455', (3819953, 3834467)), (3829425, 3829749), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|496417|503320|496418|496742|-\n",
      "catcaacggtaatcacccgctgttcgttgccggtatcaatcagcgcctgtgacggaatga\n",
      "gcagcatcggttcgctggcggtgttgagttgcaaccaggcgttcatgcccggttttagcg\n",
      "cctcgtcggcgttgtcgacttccagacgcagctgcagcgtgcgggtcgcggcatccacgc\n",
      "caggtagcagcgtccatttgcggatggtgagtgttttatccggtcgcgccggaacggtga\n",
      "gggtaaactgcgaggcatctttcaccagccaggcgatggactccgggatcgcagcagtga\n",
      "cccacaccgggtccataccctgaa-\n",
      ">G000299455|319193|319517|319193|319517|-\n",
      "catcaacagtaatcacgcgctgttctttgccggtatcgataacggcctggcttggtatca\n",
      "gcagcatctcctggcttctggtattcagtttcagataggcattcatgcccggcttgagaa\n",
      "actcatccttattagaaacctggagacggacctgaagcgtacgggttgtctgatccacgc\n",
      "tgggaagaatattccatttttcgacatggaatgttttatccggataagccggtaccgaaa\n",
      "tttcaaactgcgacgtatctttcagcagataggcgatagattctggcactgcagcgctga\n",
      "tccagaccgggtccattccctgaa-\n",
      ">G000299455|3819953|3834467|3829425|3829749|+\n",
      "-atcaacggtaatcacccgctgttcgttgccggtatcaatcagcgcctgtgacggaatga\n",
      "gcagcatcggttcgctggcggtattgagttgcaaccaggcgttcattcccggttttagcg\n",
      "cctcgtcggcgttgtcgacttccagacgcagctgcagcgtgcgggtcgcggcatccacgc\n",
      "caggtagcagcgtccatttgcggatggtgagtgttttatccggtcgcgccggaacggtga\n",
      "gggtaaactgcgaggcatctttcaccagccaggcgatggactccgggatcgcagcagtga\n",
      "cccacaccgggtccataccctgaat\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1886975, 1912610)), (1903354, 1903664), '-'), (('G000006925', (4798846, 4802842)), (4802260, 4802570), '+'), (('G000299455', (2199778, 2263225)), (2208604, 2208914), '+'), (('G000299455', (4560768, 4564764)), (4564182, 4564492), '+')]\n",
      ">G000006925|1886975|1912610|1903354|1903664|-\n",
      "tttctggtgccgcacggttgggccgtcgatattcctgccatgctgatggcctcgcaaggg\n",
      "cagaaaatggcagcgatgttccataatcagggcaacccggtttttgattatgtctggaac\n",
      "acggtgcgtcgtcgctttggcggtcgtctgcatgcgagaaatgacggtattaaaccattc\n",
      "atccagtcggtacgtcaggggtactggggatattatttacccgatcaggatcatggccca\n",
      "gagcacagcgaatttgtggatttctttgccacctataaagcgacgttgcccgcgattggt\n",
      "cgtttgatga-\n",
      ">G000006925|4798846|4802842|4802260|4802570|+\n",
      "-ttctggttccccatggctggagcgtggatattcctgcaatgttgctggcagctcagggg\n",
      "gaaaaaatggccgccatgtttcatcagcaacgaaatccagtgattgattatgtctggaat\n",
      "tctgtacggcgtaaattcggggggcgcttacattcccgggaggatgggataaaaccattt\n",
      "attcagtcagtacgccagggatactgggggtattaccttccagatcaggatcatggtcct\n",
      "gaatacagtgaatttgctgatttttttgcgacctataaagcgacattaccaattattgga\n",
      "cgtctgatgaa\n",
      ">G000299455|2199778|2263225|2208604|2208914|+\n",
      "-ttctggtgccgcacggttgggccgtcgatattcctgccatgctgatggcctcgcaaggg\n",
      "cagaaaatggcagcgatgttccataatcagggcaacccggtttttgattatgtctggaac\n",
      "acggtgcgtcgtcgctttggcggtcgtctgcatgcgagaaatgacggtattaaaccattc\n",
      "atccagtcggtacgtcaggggtactggggatattatttacccgatcaggatcatggccca\n",
      "gagcacagcgaatttgtggatttctttgccacctataaagcgacgttgcccgcgattggt\n",
      "cgtttgatgaa\n",
      ">G000299455|4560768|4564764|4564182|4564492|+\n",
      "-ttctggttccccatggctggagcgtggatattcctgcaatgttgctggcagctcaggga\n",
      "aaaaaaatggccgccatgtttcatcagcaacgaaatccagtgattgattatgtctggaat\n",
      "tcagtacggcgtaaattcggggggcgcttacatgcccgggaggatgggataaaaccattt\n",
      "attcagtcagtacgccagggatactggggatattacctcccagatcaggatcatggtcct\n",
      "gaatacagtgaatttgctgatttttttgcgacctataaagcgacattaccaatcattgga\n",
      "cgtctgatgaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3180744, 3203996)), (3183611, 3183648), '-'), (('G000299455', (747195, 760809)), (759134, 759171), '+'), (('G000299455', (747195, 760809)), (759510, 759547), '+')]\n",
      ">G000006925|3180744|3203996|3183611|3183648|-\n",
      "ttgacgctgattttttttcaacctaaagtaaaggaac-\n",
      ">G000299455|747195|760809|759134|759171|+\n",
      "-tgacgctgttgttttttcaacctaaagtaaaggaaca\n",
      ">G000299455|747195|760809|759510|759547|+\n",
      "-tgacgctgattttttttcaacctaaagtaaaggaaca\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4497401, 4528664)), (4527183, 4527485), '-'), (('G000299455', (184280, 184582)), (184280, 184582), '+'), (('G000299455', (4674171, 4686420)), (4675350, 4675652), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4497401|4528664|4527183|4527485|-\n",
      "acgtcatagcccgccagctggatcacgcgaatgccaacgtcctgggcgaactggatggct\n",
      "ttacgcataatctccagcccctgcgcccgcaccgcgtcatcttcgctgcccaacgggaaa\n",
      "cgacgatgggcagaaaggcacatggacggcacacgcacgccggtttcaacaatagcattc\n",
      "accagcgccagacgctgttcgcggctccagtcgaggcgcgacaggcgctcgttagtttca\n",
      "tctaccgacatctcgacaaaatcgaagcccaacgtttttgccagtcgcaggcgttccagc\n",
      "ca-\n",
      ">G000299455|184280|184582|184280|184582|+\n",
      "-cgtcgtagcccgccagctggatggtgcgtatgccgagatcgcgtgccagacgaatcgct\n",
      "ttgctcatgatctcccgtgctcgctggcgtaccgcgtcgtcacggctaccaaaaggaaag\n",
      "cgacgatgtgcggacaggcacatcgacgggatcccaacacccgtttcgatcatcgcggca\n",
      "accagcgaagtcctttgtgcggtgctccagtcaagacgtgaaagccgttcgtcggtttca\n",
      "tccaccgacatttcgacaaaatcaaaaccgcagctttttgccagcaccagccgctccggc\n",
      "cag\n",
      ">G000299455|4674171|4686420|4675350|4675652|+\n",
      "-cgtcatagcccgccagctggatcacgcgaatgccaacgtcctgggcgaactggatggct\n",
      "ttacgcataatctccagcccctgcgcccgcaccgcgtcatcttcgctgcccaacgggaaa\n",
      "cgacgatgggcagaaaggcacatggacggcacacgcacgccggtttcaacaatcgcattc\n",
      "accagcgccagacgctgctcgcggctccagtcgaggcgcgacaggcgctcgtcagtttca\n",
      "tctaccgacatctcgacaaaatcgaagcccaacgtttttgccagtcgcaggcgttccagc\n",
      "cag\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3717879, 3736288)), (3722736, 3722779), '-'), (('G000299455', (189977, 197562)), (192662, 192705), '+'), (('G000299455', (1030353, 1030396)), (1030353, 1030396), '+')]\n",
      ">G000006925|3717879|3736288|3722736|3722779|-\n",
      "ggggccggcctgtgttgaagcctgcccgacaaaggcgttgcag-\n",
      ">G000299455|189977|197562|192662|192705|+\n",
      "-ggaccggcctgtgttgaagcctgcccgacaaaggcgttgcagt\n",
      ">G000299455|1030353|1030396|1030353|1030396|+\n",
      "-gggccagcctgtgttgaatcttgcccgacacaggcgctgcagt\n",
      "\n",
      "\n",
      "7 [(('G000006925', (262348, 262446)), (262348, 262446), '+'), (('G000006925', (542113, 545355)), (542113, 542211), '-'), (('G000006925', (1619955, 1620053)), (1619955, 1620053), '+'), (('G000006925', (2033070, 2033168)), (2033070, 2033168), '+'), (('G000006925', (3607959, 3608057)), (3607959, 3608057), '-'), (('G000006925', (4392100, 4392198)), (4392100, 4392198), '-'), (('G000299455', (5287745, 5287970)), (5287872, 5287970), '-')]\n",
      ">G000006925|262348|262446|262348|262446|+\n",
      "-gaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgctt\n",
      ">G000006925|542113|545355|542113|542211|-\n",
      "tgaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgct-\n",
      ">G000006925|1619955|1620053|1619955|1620053|+\n",
      "-gaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgctt\n",
      ">G000006925|2033070|2033168|2033070|2033168|+\n",
      "-gaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgctt\n",
      ">G000006925|3607959|3608057|3607959|3608057|-\n",
      "tgaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgct-\n",
      ">G000006925|4392100|4392198|4392100|4392198|-\n",
      "tgaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgct-\n",
      ">G000299455|5287745|5287970|5287872|5287970|-\n",
      "tgaatcgccacggataatctagacacttccgagccgttgataatactggttttcatattc\n",
      "tgtcggtgacatctgttcgctagaaccatgccgacgct-\n",
      "\n",
      "\n",
      "19 [(('G000006925', (237115, 237939)), (237627, 237697), '+'), (('G000006925', (538623, 539106)), (538794, 538864), '+'), (('G000006925', (710708, 711191)), (710950, 711020), '-'), (('G000006925', (716211, 716694)), (716382, 716452), '+'), (('G000006925', (901842, 902325)), (902013, 902083), '+'), (('G000006925', (1096742, 1097225)), (1096984, 1097054), '-'), (('G000006925', (1632137, 1632620)), (1632308, 1632378), '+'), (('G000006925', (1835665, 1836148)), (1835907, 1835977), '-'), (('G000006925', (1878088, 1878571)), (1878259, 1878329), '+'), (('G000006925', (1943954, 1944437)), (1944125, 1944195), '+'), (('G000006925', (1961600, 1962083)), (1961771, 1961841), '+'), (('G000006925', (2069131, 2069614)), (2069302, 2069372), '+'), (('G000006925', (2346253, 2346736)), (2346424, 2346494), '+'), (('G000006925', (2473341, 2477789)), (2477477, 2477547), '+'), (('G000006925', (2954172, 2954655)), (2954414, 2954484), '-'), (('G000006925', (4394284, 4394772)), (4394526, 4394596), '-'), (('G000006925', (4824718, 4825201)), (4824960, 4825030), '-'), (('G000299455', (1289895, 1289965)), (1289895, 1289965), '+'), (('G000299455', (4557177, 4559928)), (4557347, 4557417), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 19\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 19\r",
      "   10 / 19\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 18  f\b\b\r",
      "STEP     2 / 18  f\b\b\r",
      "STEP     3 / 18  f\b\b\r",
      "STEP     4 / 18  f\b\b\r",
      "STEP     5 / 18  f\b\b\r",
      "STEP     6 / 18  f\b\b\r",
      "STEP     7 / 18  f\b\b\r",
      "STEP     8 / 18  f\b\b\r",
      "STEP     9 / 18  f\b\b\r",
      "STEP    10 / 18  f\b\b\r",
      "STEP    11 / 18  f\b\b\r",
      "STEP    12 / 18  f\b\b\r",
      "STEP    13 / 18  f\b\b\r",
      "STEP    14 / 18  f\b\b\r",
      "STEP    15 / 18  f\b\b\r",
      "STEP    16 / 18  f\b\b\r",
      "STEP    17 / 18  f\b\b\r",
      "STEP    18 / 18  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|237115|237939|237627|237697|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|538623|539106|538794|538864|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|710708|711191|710950|711020|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000006925|716211|716694|716382|716452|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|901842|902325|902013|902083|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|1096742|1097225|1096984|1097054|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000006925|1632137|1632620|1632308|1632378|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|1835665|1836148|1835907|1835977|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000006925|1878088|1878571|1878259|1878329|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|1943954|1944437|1944125|1944195|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|1961600|1962083|1961771|1961841|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|2069131|2069614|2069302|2069372|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|2346253|2346736|2346424|2346494|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|2473341|2477789|2477477|2477547|+\n",
      "-gcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgccg\n",
      ">G000006925|2954172|2954655|2954414|2954484|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000006925|4394284|4394772|4394526|4394596|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000006925|4824718|4825201|4824960|4825030|-\n",
      "ggcatccattcgttcttcagactcctgaagaagcgttccattgggctgttatcccagcag\n",
      "tttccgcgcc-\n",
      ">G000299455|1289895|1289965|1289895|1289965|+\n",
      "-gcacccattctgttttcagactgcggaagaagcgttccatggggctgttatcccagcag\n",
      "tttccccgccg\n",
      ">G000299455|4557177|4559928|4557347|4557417|+\n",
      "-gcacccattagttcttcagactcctgaagaagcgctccattgggctgttatcccagcag\n",
      "tttccacgccg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3736321, 3788106)), (3768543, 3768564), '-'), (('G000006925', (4765176, 4765197)), (4765176, 4765197), '+'), (('G000299455', (112406, 137079)), (131132, 131153), '+')]\n",
      ">G000006925|3736321|3788106|3768543|3768564|-\n",
      "cttccaccacccagtcaaact-\n",
      ">G000006925|4765176|4765197|4765176|4765197|+\n",
      "-ttccaccacccagtcaaactt\n",
      ">G000299455|112406|137079|131132|131153|+\n",
      "-ttccaccacccagtcaaactt\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1189492, 1208094)), (1203283, 1203320), '+'), (('G000299455', (2642264, 2642301)), (2642264, 2642301), '+'), (('G000299455', (3026184, 3035686)), (3030771, 3030808), '-'), (('G000299455', (4087398, 4087448)), (4087408, 4087445), '-')]\n",
      ">G000006925|1189492|1208094|1203283|1203320|+\n",
      "-gggggaatgtgggcgtgcagctgggtgataatggcta\n",
      ">G000299455|2642264|2642301|2642264|2642301|+\n",
      "-gggggaatgtcggtgtgcaactaggtgataaaggcta\n",
      ">G000299455|3026184|3035686|3030771|3030808|-\n",
      "tgggggaatgtgggcgtgcagctgggtgataatggct-\n",
      ">G000299455|4087398|4087448|4087408|4087445|-\n",
      "tgggggaatgtgggcgttcaggttgccgataggggat-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1578298, 1580868)), (1580273, 1580308), '+'), (('G000006925', (2062783, 2064325)), (2063150, 2063185), '+'), (('G000299455', (2162912, 2162947)), (2162912, 2162947), '-'), (('G000299455', (2927401, 2931337)), (2929383, 2929418), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1578298|1580868|1580273|1580308|+\n",
      "-tggtcacgcagctggttgtcgcagtgccagcacaa\n",
      ">G000006925|2062783|2064325|2063150|2063185|+\n",
      "-tggtcacgcagctggttgtcgcagtgccagcacaa\n",
      ">G000299455|2162912|2162947|2162912|2162947|-\n",
      "ctgatcgcgcagctggttatcgcagtgccagcaca-\n",
      ">G000299455|2927401|2931337|2929383|2929418|+\n",
      "-tggtcgcgcagctggttgtcgcagtgccagcacaa\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1659262, 1667779)), (1667760, 1667779), '+'), (('G000006925', (1667789, 1670305)), (1667940, 1667960), '+'), (('G000299455', (2495066, 2505927)), (2497410, 2497430), '-')]\n",
      ">G000006925|1659262|1667779|1667760|1667779|+\n",
      "--gaattttcactccggcgtt\n",
      ">G000006925|1667789|1670305|1667940|1667960|+\n",
      "-aggtttttctctccggcgtt\n",
      ">G000299455|2495066|2505927|2497410|2497430|-\n",
      "gagatttttctctccggcgt-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3040656, 3065083)), (3064409, 3065083), '+'), (('G000299455', (4713031, 4748324)), (4747650, 4748324), '+'), (('G000299455', (4769108, 4769782)), (4769108, 4769782), '+')]\n",
      ">G000006925|3040656|3065083|3064409|3065083|+\n",
      "atctgcctgttatgccttaaacatgagcgaataaaaaaagaaactatcaggcaaaaaaat\n",
      "gcacatattctggagcatggctggaatgcaactgagttctcatggttccgatacggacag\n",
      "tataacgaaacgggtatttacatctcgatcgagaagaccattatcattacaatcaccgtt\n",
      "tctggtgggtgctttaaaaaagaatacagcattgtctcccacatgctggtcactgacacc\n",
      "ataacagaggccaccctgtacgaaaacgggctgtatacgcgccacatcagactcagtcgc\n",
      "ccagtatctgacagcaaataccccttgccacccggatctcagctcattaaaaatatgacc\n",
      "ttacgactgcgcctgcaggatcagcaggaagaaacatccgtgacattatttcagggaaaa\n",
      "atgagtaccgacggaaataactattacatcatcaaaggtaaagtaagctcagtacttctg\n",
      "ttattaaaaatgctgcagattaatcacgcctgatctctgtcccggtattttagcccgagg\n",
      "tatattctgaaatatcattcatcatgaccggcgtttgtataaacgccggtcattcgctaa\n",
      "cagcttcaccgttccttcccccatccaggttccgcatgagcgtgttgtaatgttcataag\n",
      "cgtattatcgtaag\n",
      ">G000299455|4713031|4748324|4747650|4748324|+\n",
      "atctgcctgttatgccttaaacatgagcgaataaaaaaagaaactatcaggcaaaaaaat\n",
      "gcacatattctggagcatggctggaatgcaactgagttctcatggttccgatacggacag\n",
      "tataacgaaacgggtatttacatctcgatcgagaagaccattatcattacaatcaccgtt\n",
      "tctggtgggtgctttaaaaaagaatacagcattgtctcccacatgctggtcactgacacc\n",
      "ataacagaggccaccctgtacgaaaacgggctgtatacgcgccacatcagactcagtcgc\n",
      "ccagtatctgacagcaaataccccttgccacccggatctcagctcattaaaaatatgacc\n",
      "ttacgactgcgcctgcaggatcagcaggaagaaacatccgtgacattatttcagggaaaa\n",
      "atgagtaccgacggaaataactattacatcatcaaaggtaaagtaagctcagtacttctg\n",
      "ttattaaaaatgctgcagattaatcacgcctgatctctgtcccggtattttagcccgagg\n",
      "tatattctgaaatatcattcatcatgaccggcgtttgtataaacgccggtcattcgctaa\n",
      "cagcttcaccgttccttcccccatccaggttccgcatgagcgtgttgtaatgttcataag\n",
      "cgtattatcgtaag\n",
      ">G000299455|4769108|4769782|4769108|4769782|+\n",
      "atctgcctgttatgccttaaacatgagcgaataaaaaaagaaactatcaggcaaaaaaat\n",
      "gcacatattctggagcatggctggaatgcaactgagttctcatggttccgatacggacag\n",
      "tataacgaaacgggtatttacatctcgatcgagaagaccattatcattacaatcaccgtt\n",
      "tctggtgggtgctttaaaaaagaatacagcattgtctcccacatgctggtcactgacacc\n",
      "ataacagaggccaccctgtacgaaaacgggctgtatacgcgccacatcagactcagtcgc\n",
      "ccagtatctgacagcaaataccccttgccacccggatctcagctcattaaaaatatgacc\n",
      "ttacgactgcgcctgcaggatcagcaggaagaaacatccgtgacattatttcagggaaaa\n",
      "atgagtaccgacggaaataactattacatcatcaaaggtaaagtaagctcagtacttctg\n",
      "ttattaaaaatgctgcagattaatcacgcctgatctctgtcccggtattttagcccgagg\n",
      "tatattctgaaatatcattcatcatgaccggcgtttgtataaacgccggtcattcgctaa\n",
      "cagcttcaccgttccttcccccatccaggttccgcatgagcgtgttgtaatgttcataag\n",
      "cgtattatcgtaag\n",
      "\n",
      "\n",
      "4 [(('G000006925', (4666868, 4668030)), (4667104, 4667153), '-'), (('G000006925', (4802939, 4805150)), (4803816, 4803865), '+'), (('G000299455', (886351, 887494)), (887213, 887262), '+'), (('G000299455', (5346044, 5347202)), (5346920, 5346968), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4666868|4668030|4667104|4667153|-\n",
      "agcgtccaacagaggagcgggcgcagcagagcgttctggcaacgtcgct-\n",
      ">G000006925|4802939|4805150|4803816|4803865|+\n",
      "-gcgtccaacagaggagcgggcgcagcagagcgttctggcaacgtcgctg\n",
      ">G000299455|886351|887494|887213|887262|+\n",
      "-gcgaccaatcgatgaacgggcacagcagagcgttctggcaacataactg\n",
      ">G000299455|5346044|5347202|5346920|5346968|+\n",
      "-gcga-caatcgatgaacggtcacagcagagcgttctggtaacataactg\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1670306, 1692658)), (1685676, 1685717), '-'), (('G000299455', (2459722, 2494618)), (2480080, 2480121), '+'), (('G000299455', (2459722, 2494618)), (2480176, 2480217), '+'), (('G000299455', (2459722, 2494618)), (2480272, 2480313), '+'), (('G000299455', (2459722, 2494618)), (2480368, 2480409), '+')]\n",
      ">G000006925|1670306|1692658|1685676|1685717|-\n",
      "gcgcgggcaatagcggcttcgacggcagctttgcgcggatc-\n",
      ">G000299455|2459722|2494618|2480080|2480121|+\n",
      "-cgcgggcaatagccgcggcaactgccgctttgcgcggatca\n",
      ">G000299455|2459722|2494618|2480176|2480217|+\n",
      "-cacgggcaatagcggcttcgacggcggctttgcgcggatca\n",
      ">G000299455|2459722|2494618|2480272|2480313|+\n",
      "-cacgggcaatagcggcttcgacggcggctttgcgcggatcg\n",
      ">G000299455|2459722|2494618|2480368|2480409|+\n",
      "-cgcgggcgatagctgcttcaacggcagttttacgtggatca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1253314, 1275161)), (1262649, 1262694), '-'), (('G000006925', (2964578, 2976271)), (2969343, 2969387), '+'), (('G000299455', (988799, 1028140)), (1023519, 1023563), '-'), (('G000299455', (2983402, 3007859)), (2987145, 2987190), '+')]\n",
      ">G000006925|1253314|1275161|1262649|1262694|-\n",
      "ctttgtttttatgttgtgaacgttcgtcctgacattcaacaacaa-\n",
      ">G000006925|2964578|2976271|2969343|2969387|+\n",
      "-tttgttcttgtgctgggaacggtcgttctggcactgggtcacga-\n",
      ">G000299455|988799|1028140|1023519|1023563|-\n",
      "ctttgttcttgtgctgggaacggtcgttctggcattgggtcacg--\n",
      ">G000299455|2983402|3007859|2987145|2987190|+\n",
      "-tttgtttttatgttgtgaacgttcatcctgacattcaacaacaat\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4620693, 4621057)), (4620693, 4620753), '+'), (('G000006925', (4684879, 4685082)), (4685022, 4685082), '-'), (('G000006925', (4699205, 4699389)), (4699206, 4699266), '+'), (('G000006925', (4791871, 4792235)), (4791871, 4791931), '+'), (('G000299455', (5313369, 5315826)), (5313370, 5313430), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4620693|4621057|4620693|4620753|+\n",
      "-taaccggctcatttaaaccgtctggtctgtttcctccggctccacaaaaataatgtcca\n",
      "t\n",
      ">G000006925|4684879|4685082|4685022|4685082|-\n",
      "gtaaccggctcatttaaaccgtctggtctgttacctccggttccacaaaaataatgtcca\n",
      "-\n",
      ">G000006925|4699205|4699389|4699206|4699266|+\n",
      "-taaccggctcatttaaaccgtctggtctgttacctccggttccacaaaaataatgtcca\n",
      "t\n",
      ">G000006925|4791871|4792235|4791871|4791931|+\n",
      "-taaccggctcatttaaaccgtctggtctgttacctccggttccacaaaaataatgtcca\n",
      "t\n",
      ">G000299455|5313369|5315826|5313370|5313430|+\n",
      "-taaccggctcatttaaaccgtctggtctgtttcctccggctctacaaaaataatgtcca\n",
      "t\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1509064, 1578296)), (1513963, 1514028), '+'), (('G000006925', (3423912, 3503727)), (3458832, 3458897), '-'), (('G000299455', (434155, 488092)), (478962, 479027), '+'), (('G000299455', (2312634, 2337653)), (2318875, 2318940), '+')]\n",
      ">G000006925|1509064|1578296|1513963|1514028|+\n",
      "-cggcggatattcgtcatgctgcatataacgatattaactctgccagcgcgctgattgac\n",
      "gacgcc\n",
      ">G000006925|3423912|3503727|3458832|3458897|-\n",
      "ccggcagacatcatccacgttccctttaacgatctccatgcagtgaaagcggtgatggat\n",
      "gatca-\n",
      ">G000299455|434155|488092|478962|479027|+\n",
      "-cggcagacatcatccacgttccctttaacgatctccacgcagtgaaagcggtgatggat\n",
      "gatcac\n",
      ">G000299455|2312634|2337653|2318875|2318940|+\n",
      "-caccggatattcgtcatgctgcatataacgatattaactctgccagcgcgctgattgac\n",
      "gacgcc\n",
      "\n",
      "\n",
      "39 [(('G000006925', (236929, 237114)), (236972, 237033), '+'), (('G000006925', (265374, 265559)), (265455, 265515), '-'), (('G000006925', (494885, 495070)), (494966, 495026), '-'), (('G000006925', (706191, 706376)), (706272, 706332), '-'), (('G000006925', (864207, 864393)), (864289, 864349), '-'), (('G000006925', (901434, 901838)), (901477, 901537), '+'), (('G000006925', (911511, 911696)), (911592, 911652), '-'), (('G000006925', (966076, 980232)), (980128, 980189), '-'), (('G000006925', (1098847, 1099032)), (1098890, 1098950), '+'), (('G000006925', (1189492, 1208094)), (1207990, 1208051), '-'), (('G000006925', (1392018, 1392203)), (1392099, 1392159), '-'), (('G000006925', (1417365, 1417553)), (1417449, 1417509), '-'), (('G000006925', (1501384, 1507637)), (1501427, 1501488), '+'), (('G000006925', (1607379, 1610459)), (1610355, 1610415), '-'), (('G000006925', (1619749, 1619934)), (1619792, 1619852), '+'), (('G000006925', (1695091, 1695276)), (1695172, 1695232), '-'), (('G000006925', (1805163, 1806570)), (1806466, 1806526), '-'), (('G000006925', (2035068, 2035255)), (2035151, 2035211), '-'), (('G000006925', (2360573, 2360758)), (2360616, 2360676), '+'), (('G000006925', (2691924, 2692109)), (2691967, 2692027), '+'), (('G000006925', (2769676, 2769861)), (2769757, 2769817), '-'), (('G000006925', (2963247, 2963432)), (2963328, 2963388), '-'), (('G000006925', (3080669, 3080854)), (3080712, 3080772), '+'), (('G000006925', (3179574, 3180459)), (3179617, 3179678), '+'), (('G000006925', (3268417, 3281594)), (3281490, 3281551), '-'), (('G000006925', (3514451, 3514636)), (3514494, 3514554), '+'), (('G000006925', (3605879, 3606167)), (3605922, 3605983), '+'), (('G000006925', (3814544, 3814729)), (3814625, 3814685), '-'), (('G000006925', (3830436, 3830622)), (3830518, 3830578), '-'), (('G000006925', (3915855, 3917645)), (3915898, 3915959), '+'), (('G000006925', (3969227, 3993100)), (3992996, 3993057), '-'), (('G000006925', (4116339, 4116524)), (4116420, 4116480), '-'), (('G000006925', (4255239, 4255424)), (4255320, 4255380), '-'), (('G000006925', (4361769, 4361954)), (4361812, 4361872), '+'), (('G000006925', (4607382, 4607555)), (4607451, 4607511), '-'), (('G000006925', (4619148, 4619330)), (4619226, 4619286), '-'), (('G000006925', (4645165, 4645252)), (4645165, 4645225), '+'), (('G000006925', (4694201, 4694386)), (4694282, 4694343), '-'), (('G000299455', (5286183, 5286375)), (5286271, 5286332), '-')]\n",
      ">G000006925|236929|237114|236972|237033|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "cg\n",
      ">G000006925|265374|265559|265455|265515|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|494885|495070|494966|495026|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|706191|706376|706272|706332|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|864207|864393|864289|864349|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|901434|901838|901477|901537|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttac\n",
      "c-\n",
      ">G000006925|911511|911696|911592|911652|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|966076|980232|980128|980189|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1098847|1099032|1098890|1098950|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1189492|1208094|1207990|1208051|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1392018|1392203|1392099|1392159|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1417365|1417553|1417449|1417509|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1501384|1507637|1501427|1501488|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "cg\n",
      ">G000006925|1607379|1610459|1610355|1610415|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1619749|1619934|1619792|1619852|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1695091|1695276|1695172|1695232|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|1805163|1806570|1806466|1806526|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|2035068|2035255|2035151|2035211|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|2360573|2360758|2360616|2360676|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|2691924|2692109|2691967|2692027|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|2769676|2769861|2769757|2769817|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|2963247|2963432|2963328|2963388|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3080669|3080854|3080712|3080772|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3179574|3180459|3179617|3179678|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "cg\n",
      ">G000006925|3268417|3281594|3281490|3281551|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3514451|3514636|3514494|3514554|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3605879|3606167|3605922|3605983|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "cg\n",
      ">G000006925|3814544|3814729|3814625|3814685|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3830436|3830622|3830518|3830578|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|3915855|3917645|3915898|3915959|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "cg\n",
      ">G000006925|3969227|3993100|3992996|3993057|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|4116339|4116524|4116420|4116480|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|4255239|4255424|4255320|4255380|-\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|4361769|4361954|4361812|4361872|+\n",
      "-accttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000006925|4607382|4607555|4607451|4607511|-\n",
      "-accttgcagaggcattcgagcattataacgaatggcatccgcatagtgcgctggattat\n",
      "c-\n",
      ">G000006925|4619148|4619330|4619226|4619286|-\n",
      "-gtctcgcgaaggtgttcaaacattataatgaatggcatcctcatagtgcgcagggttat\n",
      "c-\n",
      ">G000006925|4645165|4645252|4645165|4645225|+\n",
      "-accttgcagtggcgatcgagcattacaatgaaaaccatccgcatagtgcgttgggttat\n",
      "c-\n",
      ">G000006925|4694201|4694386|4694282|4694343|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgctgggttat\n",
      "c-\n",
      ">G000299455|5286183|5286375|5286271|5286332|-\n",
      "aaccttgcagaggcgttcgagcattataacgaatggcatccgcatagtgcgttgggttat\n",
      "c-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3246783, 3268347)), (3265598, 3265650), '-'), (('G000299455', (671363, 678362)), (674007, 674059), '+'), (('G000299455', (1078152, 1154001)), (1144074, 1144126), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 39\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 39\r",
      "   10 / 39\r",
      "   20 / 39\r",
      "   30 / 39\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 38  f\b\b\r",
      "STEP     2 / 38  f\b\b\r",
      "STEP     3 / 38  f\b\b\r",
      "STEP     4 / 38  f\b\b\r",
      "STEP     5 / 38  f\b\b\r",
      "STEP     6 / 38  f\b\b\r",
      "STEP     7 / 38  f\b\b\r",
      "STEP     8 / 38  f\b\b\r",
      "STEP     9 / 38  f\b\b\r",
      "STEP    10 / 38  f\b\b\r",
      "STEP    11 / 38  f\b\b\r",
      "STEP    12 / 38  f\b\b\r",
      "STEP    13 / 38  f\b\b\r",
      "STEP    14 / 38  f\b\b\r",
      "STEP    15 / 38  f\b\b\r",
      "STEP    16 / 38  f\b\b\r",
      "STEP    17 / 38  f\b\b\r",
      "STEP    18 / 38  f\b\b\r",
      "STEP    19 / 38  f\b\b\r",
      "STEP    20 / 38  f\b\b\r",
      "STEP    21 / 38  f\b\b\r",
      "STEP    22 / 38  f\b\b\r",
      "STEP    23 / 38  f\b\b\r",
      "STEP    24 / 38  f\b\b\r",
      "STEP    25 / 38  f\b\b\r",
      "STEP    26 / 38  f\b\b\r",
      "STEP    27 / 38  f\b\b\r",
      "STEP    28 / 38  f\b\b\r",
      "STEP    29 / 38  f\b\b\r",
      "STEP    30 / 38  f\b\b\r",
      "STEP    31 / 38  f\b\b\r",
      "STEP    32 / 38  f\b\b\r",
      "STEP    33 / 38  f\b\b\r",
      "STEP    34 / 38  f\b\b\r",
      "STEP    35 / 38  f\b\b\r",
      "STEP    36 / 38  f\b\b\r",
      "STEP    37 / 38  f\b\b\r",
      "STEP    38 / 38  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 39\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 39\r",
      "   10 / 39\r",
      "   20 / 39\r",
      "   30 / 39\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 38  f\b\b\r",
      "STEP     2 / 38  f\b\b\r",
      "STEP     3 / 38  f\b\b\r",
      "STEP     4 / 38  f\b\b\r",
      "STEP     5 / 38  f\b\b\r",
      "STEP     6 / 38  f\b\b\r",
      "STEP     7 / 38  f\b\b\r",
      "STEP     8 / 38  f\b\b\r",
      "STEP     9 / 38  f\b\b\r",
      "STEP    10 / 38  f\b\b\r",
      "STEP    11 / 38  f\b\b\r",
      "STEP    12 / 38  f\b\b\r",
      "STEP    13 / 38  f\b\b\r",
      "STEP    14 / 38  f\b\b\r",
      "STEP    15 / 38  f\b\b\r",
      "STEP    16 / 38  f\b\b\r",
      "STEP    17 / 38  f\b\b\r",
      "STEP    18 / 38  f\b\b\r",
      "STEP    19 / 38  f\b\b\r",
      "STEP    20 / 38  f\b\b\r",
      "STEP    21 / 38  f\b\b\r",
      "STEP    22 / 38  f\b\b\r",
      "STEP    23 / 38  f\b\b\r",
      "STEP    24 / 38  f\b\b\r",
      "STEP    25 / 38  f\b\b\r",
      "STEP    26 / 38  f\b\b\r",
      "STEP    27 / 38  f\b\b\r",
      "STEP    28 / 38  f\b\b\r",
      "STEP    29 / 38  f\b\b\r",
      "STEP    30 / 38  f\b\b\r",
      "STEP    31 / 38  f\b\b\r",
      "STEP    32 / 38  f\b\b\r",
      "STEP    33 / 38  f\b\b\r",
      "STEP    34 / 38  f\b\b\r",
      "STEP    35 / 38  f\b\b\r",
      "STEP    36 / 38  f\b\b\r",
      "STEP    37 / 38  f\b\b\r",
      "STEP    38 / 38  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3246783|3268347|3265598|3265650|-\n",
      "atgatgttgggcgtatttttcggacaatattttatcaacaccatcacctggt-------\n",
      ">G000299455|671363|678362|674007|674059|+\n",
      "---gcagcaagcg---gtcccaaactgcattacatcaa-gcaattgctctctaaccgca\n",
      ">G000299455|1078152|1154001|1144074|1144126|+\n",
      "-tgatgatcggcgtttatatcggtcagtactgtatcaacgccctgacttactt------\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3079099, 3079131)), (3079101, 3079123), '-'), (('G000299455', (62449, 62476)), (62452, 62474), '+'), (('G000299455', (873437, 873523)), (873499, 873521), '+'), (('G000299455', (1995529, 1995556)), (1995534, 1995556), '+'), (('G000299455', (4540164, 4540191)), (4540167, 4540189), '+')]\n",
      ">G000006925|3079099|3079131|3079101|3079123|-\n",
      "agtgattgcccacacgcgttat-\n",
      ">G000299455|62449|62476|62452|62474|+\n",
      "-gtgatttcccacacgcgttata\n",
      ">G000299455|873437|873523|873499|873521|+\n",
      "-gtgatttcccacacacgttata\n",
      ">G000299455|1995529|1995556|1995534|1995556|+\n",
      "-atgatttcccacacgcgttata\n",
      ">G000299455|4540164|4540191|4540167|4540189|+\n",
      "-gtgaattcccacacgcgttata\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2792107, 2792145), '-'), (('G000299455', (1237061, 1253885)), (1253706, 1253744), '+'), (('G000299455', (3136481, 3140315)), (3140223, 3140261), '+')]\n",
      ">G000006925|2770832|2816659|2792107|2792145|-\n",
      "tggtgaggtggccgagaggctgaaggcgctcccctgct-\n",
      ">G000299455|1237061|1253885|1253706|1253744|+\n",
      "-ggtgaggtggccgagaggctgaaggcgctcccctgcta\n",
      ">G000299455|3136481|3140315|3140223|3140261|+\n",
      "-ggtgaggtgtccgagtggctgaaggagcacgcctggaa\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1509064, 1578296)), (1576396, 1576498), '+'), (('G000299455', (2167849, 2167987)), (2167849, 2167951), '-'), (('G000299455', (2547477, 2547579)), (2547477, 2547579), '-'), (('G000299455', (2923865, 2925880)), (2923983, 2924085), '+'), (('G000299455', (3291249, 3291351)), (3291249, 3291351), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1206\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\n",
      "Reallocating..done. *alloclen = 1206\n",
      "\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1509064|1578296|1576396|1576498|+\n",
      "-ctaccaaacggctgccagccctctttcagtttatgtgtcagcttttccgcaagatctga\n",
      "cggcggcgccgccctgacaacatcatagtgtttaaatgccatg\n",
      ">G000299455|2167849|2167987|2167849|2167951|-\n",
      "tccgccatatggctgccagccctctttcagtttgtgtgtcagcttttccgcaaggtctga\n",
      "cggcgacgccgccctgacaacatcatagtgtttaaatgccat-\n",
      ">G000299455|2547477|2547579|2547477|2547579|-\n",
      "actaccataaggctgccacccttccttcagtttttgagttattcgtttcgcaaggtctga\n",
      "cggcgacgccgccctgacaacatcatagtgtttaaatgccat-\n",
      ">G000299455|2923865|2925880|2923983|2924085|+\n",
      "-ctaccaaacggctgccagccctctttcagtttatgtgtcagcttttccgcaagatctga\n",
      "cggcgacgccgccctgacaacatcatagtgtttaaatgccatg\n",
      ">G000299455|3291249|3291351|3291249|3291351|+\n",
      "-ctaccataaggctgccacccttccttcagtttttgagttattcgttccgcaaggtctga\n",
      "cggcgacgccgccctgaccacgtcatagtgtttaaatgccatg\n",
      "\n",
      "\n",
      "6 [(('G000006925', (717939, 717969)), (717939, 717964), '+'), (('G000006925', (908032, 908771)), (908385, 908410), '-'), (('G000006925', (1394069, 1394808)), (1394429, 1394454), '+'), (('G000006925', (4557883, 4573403)), (4566919, 4566944), '-'), (('G000299455', (2538890, 2539629)), (2539250, 2539275), '+'), (('G000299455', (4487440, 4499283)), (4490219, 4490244), '+')]\n",
      ">G000006925|717939|717969|717939|717964|+\n",
      "-ccggaaaaaatcatctggctgtggc\n",
      ">G000006925|908032|908771|908385|908410|-\n",
      "-ccgggtagaatcacttagcggcagc\n",
      ">G000006925|1394069|1394808|1394429|1394454|+\n",
      "-ccggaaaaaatcatcttgcagccgc\n",
      ">G000006925|4557883|4573403|4566919|4566944|-\n",
      "accggcaaaaaccatctggcggcgg-\n",
      ">G000299455|2538890|2539629|2539250|2539275|+\n",
      "-ccggaaaaaatcatctggcggcggc\n",
      ">G000299455|4487440|4499283|4490219|4490244|+\n",
      "-ccggcaaaaaccatctggcggcggc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2242494, 2346249)), (2299602, 2299632), '-'), (('G000299455', (1752647, 1813107)), (1779354, 1779384), '+'), (('G000299455', (3480865, 3520974)), (3516733, 3516763), '+')]\n",
      ">G000006925|2242494|2346249|2299602|2299632|-\n",
      "gactcacccaccagcgccagcgtttcgcca-\n",
      ">G000299455|1752647|1813107|1779354|1779384|+\n",
      "-actcacccaccagcgccagcgtttcgccag\n",
      ">G000299455|3480865|3520974|3516733|3516763|+\n",
      "-actcgcccaccagcgatagcgtttcgccag\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1578298, 1580868)), (1579371, 1579420), '+'), (('G000006925', (1938092, 1938404)), (1938197, 1938246), '+'), (('G000006925', (2062140, 2062463)), (2062245, 2062294), '+'), (('G000299455', (2545903, 2545957)), (2545906, 2545955), '-'), (('G000299455', (2927401, 2931337)), (2928484, 2928533), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1578298|1580868|1579371|1579420|+\n",
      "-tcaggatattgtccaggtcgcgacgacgcttatccggtggctctgcaat\n",
      ">G000006925|1938092|1938404|1938197|1938246|+\n",
      "-tcaggatattgtccaggtcacgacgccgcttatccggtggctctgcaat\n",
      ">G000006925|2062140|2062463|2062245|2062294|+\n",
      "-tcaggatattgtccaggtcacgacggcgcttatccggtggctctgcaat\n",
      ">G000299455|2545903|2545957|2545906|2545955|-\n",
      "ttcaggatgttatcgaggtcgcgacggcgggaatctggtggttcagcaa-\n",
      ">G000299455|2927401|2931337|2928484|2928533|+\n",
      "-tcagaatattgtccaggtcgcgacgacgcttatccggtggctctgcaat\n",
      "\n",
      "\n",
      "6 [(('G000006925', (2118762, 2118801)), (2118764, 2118792), '-'), (('G000006925', (2138845, 2144864)), (2142780, 2142808), '-'), (('G000006925', (3736321, 3788106)), (3777674, 3777702), '-'), (('G000299455', (112406, 137079)), (121993, 122022), '+'), (('G000299455', (1911336, 1921536)), (1917572, 1917600), '+'), (('G000299455', (1943430, 1943472)), (1943440, 1943468), '+')]\n",
      ">G000006925|2118762|2118801|2118764|2118792|-\n",
      "ggtagctgttaagccaggggcggtagcg--\n",
      ">G000006925|2138845|2144864|2142780|2142808|-\n",
      "--tagctaaaaagccaggggcggtagcgtg\n",
      ">G000006925|3736321|3788106|3777674|3777702|-\n",
      "-gtagctgttgagcctggggcggtagcgt-\n",
      ">G000299455|112406|137079|121993|122022|+\n",
      "-gtagctgttgagcctggggcggtagcgtg\n",
      ">G000299455|1911336|1921536|1917572|1917600|+\n",
      "--tagctaaaaagccaggggcggtagcgtg\n",
      ">G000299455|1943430|1943472|1943440|1943468|+\n",
      "--tagctgttaagcctggggcggtagcgtg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (729684, 729878), '+'), (('G000299455', (2174641, 2181396)), (2178080, 2178274), '+'), (('G000299455', (2910238, 2916987)), (2913354, 2913548), '-')]\n",
      ">G000006925|726235|732999|729684|729878|+\n",
      "-actggcactgatgggcgaagattatcaggacattttccgccagcaggtcagggaatccg\n",
      "cagagcgcgaaaaagccggactctcacgtccggtgtggatagcccaggcgtatcagcagc\n",
      "agatagcggagagtcgcaggccggaagaggagacaacatcacgtgagacgtaatctttca\n",
      "catattatcgctgcg\n",
      ">G000299455|2174641|2181396|2178080|2178274|+\n",
      "-gctggcgctgatgggcgaggattatcaggacattttccgccagcaggtcagggaatctg\n",
      "cagagcgggaaaaagccggactctcacgtccggtgtggatagcgcaggcgtatcagcagc\n",
      "agatagcggagagtcgcaggccggaagaggagacaacaccacgtgagacgtaatctttca\n",
      "cacattattgcagca\n",
      ">G000299455|2910238|2916987|2913354|2913548|-\n",
      "agctggcgctgatgggcgaggattatcaggacattttccgccagcaggtcagggaatctg\n",
      "cagagcgggaaaaagccggactctcacgtccggtgtggatagcgcaggcgtatcagcagc\n",
      "agatagcggagagtcgcaggccggaagaggagacaacaccacgtgagacgtaatctttca\n",
      "cacattattgccgc-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (709671, 710707)), (709710, 709735), '+'), (('G000299455', (2551572, 2551597)), (2551572, 2551597), '-'), (('G000299455', (2919336, 2919766)), (2919375, 2919400), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|709671|710707|709710|709735|+\n",
      "-ggcattggatcctgatgtactcctg\n",
      ">G000299455|2551572|2551597|2551572|2551597|-\n",
      "aggcattgcatcctgatgtactcct-\n",
      ">G000299455|2919336|2919766|2919375|2919400|+\n",
      "-ggcattgcgtcctgatgtattcctg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (487251, 491968)), (487689, 487967), '-'), (('G000299455', (314727, 315005)), (314727, 315005), '-'), (('G000299455', (3819953, 3834467)), (3833751, 3834029), '+')]\n",
      ">G000006925|487251|491968|487689|487967|-\n",
      "cgtgccgattgagctggaacagctggtactgtcgttcaaccatatgatcgagcgtattga\n",
      "ggatgtctttactcgccagtccaatttctcagcggatatcgcccacgaaattcgcacacc\n",
      "aattactaatctcattacgcaaacggaaatcgccctcagccagtcgcgcagccagaagga\n",
      "gctggaagatgtgctctactctaatctcgaagagctgacgcgaatggcgaaaatggtcag\n",
      "cgatatgctgtttctcgctcaggccgataacaaccagc-\n",
      ">G000299455|314727|315005|314727|315005|-\n",
      "cgttcccattgagctggagcaactggttatctcgttcaatcatatgattggaaagattga\n",
      "ggatgtctttacccgccaggccaatttctctgccgatatcgcgcatgagatcagaacgcc\n",
      "catcaccaatctggtgacgcagactgaaatcgcactgagtcaggatcgaacacagaggga\n",
      "acttgaggatgtcctctattccagtcttgaagagtataaccggatgaccaaaatggtcag\n",
      "cgatatgctgttcctggcacaggcagataataatcagc-\n",
      ">G000299455|3819953|3834467|3833751|3834029|+\n",
      "-gtgcccattgagctggaacagctggtactgtcgttcaaccatatgatcgagcgtattga\n",
      "ggatgtctttacccgccagtccaatttctcagcggatatcgctcacgaaattcgcacacc\n",
      "aattactaatctcataacgcaaacggaaatcgccctcagccagtcgcgcagccagaagga\n",
      "actggaagatgtgctctactctaatctcgaagagctgacgcgaatggcgaaaatggtcag\n",
      "cgatatgctgtttctcgctcaggccgataacaaccagct\n",
      "\n",
      "\n",
      "8 [(('G000006925', (262642, 262772)), (262644, 262772), '-'), (('G000006925', (541787, 541917)), (541787, 541914), '+'), (('G000006925', (1620249, 1620379)), (1620251, 1620379), '-'), (('G000006925', (2033364, 2033494)), (2033366, 2033494), '-'), (('G000006925', (3607633, 3607763)), (3607633, 3607760), '+'), (('G000006925', (4391774, 4391904)), (4391774, 4391901), '+'), (('G000006925', (4532273, 4532403)), (4532273, 4532400), '+'), (('G000299455', (5287745, 5287970)), (5287745, 5287872), '+')]\n",
      ">G000006925|262642|262772|262644|262772|-\n",
      "ttatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|541787|541917|541787|541914|+\n",
      "-tatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|1620249|1620379|1620251|1620379|-\n",
      "ttatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggtggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|2033364|2033494|2033366|2033494|-\n",
      "ttatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggtggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|3607633|3607763|3607633|3607760|+\n",
      "-tatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|4391774|4391904|4391774|4391901|+\n",
      "-tatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000006925|4532273|4532403|4532273|4532400|+\n",
      "-tatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacaggtgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      ">G000299455|5287745|5287970|5287745|5287872|+\n",
      "-tatcggctggtccatgcaatcccggatgacaaaggacattgtcctgaacgcactgctga\n",
      "tggctgtatggcggcgtaatcccgaaaaacagatgctggttcattcggatcagggcagtc\n",
      "agtacaca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (739473, 745780)), (741471, 741508), '+'), (('G000006925', (1921807, 1928114)), (1926079, 1926116), '-'), (('G000299455', (2189293, 2192099)), (2189543, 2189580), '+'), (('G000299455', (3586763, 3590110)), (3589823, 3589860), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\n",
      "Reallocating..done. *alloclen = 1076\n",
      "\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|739473|745780|741471|741508|+\n",
      "-tcgctgatgcaaaatgtttcatgtgaaaccgcctgcg\n",
      ">G000006925|1921807|1928114|1926079|1926116|-\n",
      "gtcgctgatgcaaaatgtttcatgtgaaaccgcctgc-\n",
      ">G000299455|2189293|2192099|2189543|2189580|+\n",
      "-tcgctgatgcaaaatgttttatgtgaaaccgcctgcg\n",
      ">G000299455|3586763|3590110|3589823|3589860|-\n",
      "gtcgctgatgcaaaatgttttatgtgaaaccgcctcc-\n",
      "\n",
      "\n",
      "6 [(('G000006925', (1509064, 1578296)), (1577172, 1577200), '+'), (('G000299455', (2167395, 2167615)), (2167543, 2167571), '-'), (('G000299455', (2547171, 2547245)), (2547173, 2547201), '-'), (('G000299455', (2923865, 2925880)), (2924760, 2924788), '+'), (('G000299455', (3177840, 3177871)), (3177843, 3177871), '-'), (('G000299455', (3291615, 3291643)), (3291615, 3291643), '+')]\n",
      ">G000006925|1509064|1578296|1577172|1577200|+\n",
      "-atggtgccgggtgcctcccggtgaattc\n",
      ">G000299455|2167395|2167615|2167543|2167571|-\n",
      "catggtgccgggtgcctcccggtgaatt-\n",
      ">G000299455|2547171|2547245|2547173|2547201|-\n",
      "catggtgccgggtgcctcccggtgagtt-\n",
      ">G000299455|2923865|2925880|2924760|2924788|+\n",
      "-atggtgccgggtgcctcccggtgaattc\n",
      ">G000299455|3177840|3177871|3177843|3177871|-\n",
      "catggtgccgggtgcctcccggtgaatt-\n",
      ">G000299455|3291615|3291643|3291615|3291643|+\n",
      "-atggtgccgggtgcctcccggtgagttc\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1405168, 1405262)), (1405237, 1405259), '+'), (('G000006925', (1916373, 1917424)), (1917331, 1917353), '-'), (('G000006925', (2045115, 2046271)), (2045115, 2045137), '-'), (('G000006925', (2227521, 2228997)), (2227841, 2227863), '-'), (('G000299455', (2196403, 2197468)), (2196488, 2196510), '+'), (('G000299455', (3580143, 3580251)), (3580144, 3580166), '-'), (('G000299455', (4121494, 4121667)), (4121559, 4121581), '-')]\n",
      ">G000006925|1405168|1405262|1405237|1405259|+\n",
      "--ttctaggcgttatcgattttac\n",
      ">G000006925|1916373|1917424|1917331|1917353|-\n",
      "atttccagccgttactgatttt--\n",
      ">G000006925|2045115|2046271|2045115|2045137|-\n",
      "atttccagccgttaccgatttt--\n",
      ">G000006925|2227521|2228997|2227841|2227863|-\n",
      "atttccagccgttaccgatttt--\n",
      ">G000299455|2196403|2197468|2196488|2196510|+\n",
      "-tttccagccgttacagatttta-\n",
      ">G000299455|3580143|3580251|3580144|3580166|-\n",
      "atttccagctgttgccgatttt--\n",
      ">G000299455|4121494|4121667|4121559|4121581|-\n",
      "-tttccagccgttaccgatttta-\n",
      "\n",
      "\n",
      "9 [(('G000006925', (711810, 711981)), (711910, 711934), '+'), (('G000006925', (720848, 721019)), (720948, 720972), '+'), (('G000006925', (1509064, 1578296)), (1578142, 1578169), '-'), (('G000006925', (1937045, 1937215)), (1937092, 1937116), '-'), (('G000006925', (2061077, 2061268)), (2061124, 2061148), '-'), (('G000299455', (2167070, 2167372)), (2167198, 2167222), '+'), (('G000299455', (2923865, 2925880)), (2925729, 2925753), '-'), (('G000299455', (3293159, 3293449)), (3293304, 3293328), '-'), (('G000299455', (4069402, 4069442)), (4069402, 4069429), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\n",
      "Reallocating..done. *alloclen = 1058\n",
      "\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 6\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 5  f\b\b\r",
      "STEP     2 / 5  f\b\b\r",
      "STEP     3 / 5  f\b\b\r",
      "STEP     4 / 5  f\b\b\r",
      "STEP     5 / 5  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 9\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 8  f\b\b\r",
      "STEP     2 / 8  f\b\b\r",
      "STEP     3 / 8  f\b\b\r",
      "STEP     4 / 8  f\b\b\r",
      "STEP     5 / 8  f\b\b\r",
      "STEP     6 / 8  f\b\b\r",
      "STEP     7 / 8  f\b\b\r",
      "STEP     8 / 8  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|711810|711981|711910|711934|+\n",
      "-ctagctcatccggatagagcatca--\n",
      ">G000006925|720848|721019|720948|720972|+\n",
      "-ctagctcatccggatagagcatca--\n",
      ">G000006925|1509064|1578296|1578142|1578169|-\n",
      "attagcttatcaggaagagcagacgac\n",
      ">G000006925|1937045|1937215|1937092|1937116|-\n",
      "actagctcatccggatagagcatc---\n",
      ">G000006925|2061077|2061268|2061124|2061148|-\n",
      "actagctcatcaggaaagagcgtc---\n",
      ">G000299455|2167070|2167372|2167198|2167222|+\n",
      "-ttagctcatcgggatagagcgcca--\n",
      ">G000299455|2923865|2925880|2925729|2925753|-\n",
      "attagctcatcgggatagagcgcc---\n",
      ">G000299455|3293159|3293449|3293304|3293328|-\n",
      "attagctcatcgggacagagcgcc---\n",
      ">G000299455|4069402|4069442|4069402|4069429|-\n",
      "attagctcatcaggaagagcagacgac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2374481, 2472008)), (2402489, 2402520), '-'), (('G000299455', (1623109, 1682614)), (1677977, 1678008), '+'), (('G000299455', (4554421, 4554452)), (4554421, 4554452), '+')]\n",
      ">G000006925|2374481|2472008|2402489|2402520|-\n",
      "catgctgatgtgggcgacgctgatgctgctg-\n",
      ">G000299455|1623109|1682614|1677977|1678008|+\n",
      "-atgctgatgtgggcgacgctgatgctgctgg\n",
      ">G000299455|4554421|4554452|4554421|4554452|+\n",
      "-atgctgctgggggccacgctgatgctgctgg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2117901, 2118205)), (2117901, 2118002), '+'), (('G000006925', (3969227, 3993100)), (3981751, 3981852), '-'), (('G000299455', (5156132, 5190970)), (5178345, 5178446), '+')]\n",
      ">G000006925|2117901|2118205|2117901|2118002|+\n",
      "-cgttatgtccaccaatgttataagtttcacccgctttaccttcggttacgacggtatat\n",
      "aacgcacgagcatgatcctctacatacaaccagtcgcgaatt\n",
      ">G000006925|3969227|3993100|3981751|3981852|-\n",
      "tcgttgtgaccaccaatattataggtttcaccgactttcccggtggtcgccacgcaatac\n",
      "agcgcgcgggcgtgatcttccacatacagccagtcacggat-\n",
      ">G000299455|5156132|5190970|5178345|5178446|+\n",
      "-cgttgtgaccaccaatattataggtttcaccgactttcccggtggtcgccacgcaatac\n",
      "agcgcgcgggcgtgatcttccacatacagccagtcacggatt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4013864, 4015519)), (4015376, 4015519), '-'), (('G000299455', (5145404, 5145547)), (5145404, 5145547), '+'), (('G000299455', (5147115, 5155219)), (5147115, 5147258), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4013864|4015519|4015376|4015519|-\n",
      "ttacaaccagttcttccgcttaaagtacagatacggtgccaggcccgcgaggatcataaa\n",
      "gataatcgcgccagggtagccgaaactccacttcagttctggcataaactcaaagttcat\n",
      "cccgtagctggaagcaaccagcg-\n",
      ">G000299455|5145404|5145547|5145404|5145547|+\n",
      "-tacagccagttcttacgcttaaagtacagatacggtgtcaggcccgcaaggatcataaa\n",
      "gataatcgcgccagggtagccaaagctccacttcagttctggcataaactcaaagttcat\n",
      "gccatagctggaggcaatccgcgt\n",
      ">G000299455|5147115|5155219|5147115|5147258|+\n",
      "-tacagccagttcttgcgcttaaagtacagatacggtgccaggcccgcgaggatcataaa\n",
      "gataatcgcgcctgggtagccgaaactccacttcagttctggcataaactcaaagttcat\n",
      "cccatagctggaagcaacgagcgt\n",
      "\n",
      "\n",
      "5 [(('G000006925', (720687, 720818)), (720721, 720756), '+'), (('G000299455', (2165477, 2165575)), (2165477, 2165512), '+'), (('G000299455', (2546693, 2546781)), (2546727, 2546761), '+'), (('G000299455', (3293791, 3293879)), (3293811, 3293845), '-'), (('G000299455', (3608552, 3608633)), (3608571, 3608606), '-')]\n",
      ">G000006925|720687|720818|720721|720756|+\n",
      "-gttaagagtggttgcttcgccacacagcttaaacc\n",
      ">G000299455|2165477|2165575|2165477|2165512|+\n",
      "-gttaagagtggttacttcgccacacaacttaaacc\n",
      ">G000299455|2546693|2546781|2546727|2546761|+\n",
      "-gataagagtggttacattgccacgcagccgaacc-\n",
      ">G000299455|3293791|3293879|3293811|3293845|-\n",
      "tgataagagtggttacattgccacgctgcttaac--\n",
      ">G000299455|3608552|3608633|3608571|3608606|-\n",
      "tgttaagagtggttacttcgtcacacagcttaaac-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (308545, 311277)), (311231, 311277), '-'), (('G000006925', (317551, 317665)), (317619, 317665), '+'), (('G000006925', (328093, 328139)), (328093, 328139), '+'), (('G000299455', (4156918, 4166023)), (4156918, 4156964), '+')]\n",
      ">G000006925|308545|311277|311231|311277|-\n",
      "atggtgccgataataggagtcgaacctacgaccttcgcattacgaa-\n",
      ">G000006925|317551|317665|317619|317665|+\n",
      "-tggtgccgataataggagtcgaacctacgaccttcgcattacgaat\n",
      ">G000006925|328093|328139|328093|328139|+\n",
      "-tggtgccgataataggagtcgaacctacgaccttcgcattacgaat\n",
      ">G000299455|4156918|4166023|4156918|4156964|+\n",
      "-tggtgccgataataggagtcgaacctacgaccttcgcattacgaat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3868997, 3883532)), (3872859, 3873017), '+'), (('G000299455', (1, 26978)), (3865, 4023), '+'), (('G000299455', (767773, 781429)), (777884, 778042), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3868997|3883532|3872859|3873017|+\n",
      "ggcgcggcattccgaccggtattcacccggaagagggcgtatcggcggcggaagtgatca\n",
      "tgaccgttctgcacgcaggcggtaaatttgacgataactcctataaagtgtccggcggtc\n",
      "tgcacggcgttggtgtttcggtagtaaacgccctgtcg\n",
      ">G000299455|1|26978|3865|4023|+\n",
      "ggcgcggtattccgaccggtattcacccggaagagggcgtatcggcggcggaagtgatca\n",
      "tgaccgttctgcacgcaggcggtaaatttgacgataactcctataaagtgtccggcggtc\n",
      "tgcacggcgttggtgtttcggtagtaaacgccctgtcg\n",
      ">G000299455|767773|781429|777884|778042|+\n",
      "ggcgcgggatgccggtggatattcacccggaagagggtgtaccggcggttgaactgattc\n",
      "tttgccgtctgcatgcaggcggtaaattctctaacaaaaattaccagttctctggcggcc\n",
      "tgcatggcgtggggatttcggtggttaacgccctgtcg\n",
      "\n",
      "\n",
      "3 [(('G000006925', (726235, 732999)), (730498, 730963), '+'), (('G000299455', (2174641, 2181396)), (2178894, 2179359), '+'), (('G000299455', (2910238, 2916987)), (2912269, 2912734), '-')]\n",
      ">G000006925|726235|732999|730498|730963|+\n",
      "-tcagtttgaatccctgccagcaaaggttcgccaggacatgcagcagcgcattgatgcgg\n",
      "cgcgccggatgtttgctgaaaaagtggcgatgtataccggtttatctgtggatgcggtta\n",
      "tgggaacagaggccgccgtttttgaaggtcagtccggtattgaggccgaactggcggatg\n",
      "aattaatcaatgcgtcggacgccatcagcgtgatggctgcgacgctgaacacacacgata\n",
      "caggaggcactatgccgcaattaactgcaacggaggctgccgcgcaggagaaccagcgag\n",
      "tgatggggatcctgacgtgtcaggaagcgaaaggacgtgaacaactggccacgatgctgg\n",
      "cagggcaacagggtatgagcgttgagcaggcccgggcgattctggccgtggcggcaccgc\n",
      "aacaatctggtgtatctgcgcagagtgaagccgatcgcattatggc\n",
      ">G000299455|2174641|2181396|2178894|2179359|+\n",
      "-tcagtttgaagtcttaccggcagaggtgcgtcaggacatgcagcagcgggttgatgcgg\n",
      "cgcaccggatgtttgccgaaaaagtggcgatgtataccgggttgtctgtggatgcggtca\n",
      "cgggaacagaggccgccgtttttgaaggtcagtccggcattgaggccgggctggcggatg\n",
      "aattaatcaatgcgtcggatgccatcagtgtgatggccacggcgctgaacagtaatgtca\n",
      "gaggaggcactatgccgcaattaactgcaacggaagccgccgtgcaggagaaccagcgag\n",
      "tgatggggatcctgacatgccaggaagcgaaaggacgtgaacagcttgccacgatgctgg\n",
      "cagggcaacagggcatgagcgttgaacaggcccgggcgattctggccgcggcggcaccgc\n",
      "agcagctggtggcatccgcgcagagtgaagccgatcgcattatggc\n",
      ">G000299455|2910238|2916987|2912269|2912734|-\n",
      "atcagtttgaagtcttaccggcagaggtgcgtcaggacatgcagcagcgggttgatgcgg\n",
      "cgcaccggatgtttgccgaaaaagtggcgatgtataccgggttgtctgtggatgcggtca\n",
      "cgggaacagaggccgccgtttttgaaggtcagtccggcattgaggccgggctggcggatg\n",
      "aattaatcaatgcgtcggatgccatcagtgtgatggccacggcgctgaacagtaatgtca\n",
      "gaggaggcactatgccgcaattaactgcaacggaagccgccgtgcaggagaaccagcgag\n",
      "tgatggggatcctgacatgccaggaagcgaaaggacgtgaacagcttgccacgatgctgg\n",
      "cagggcaacagggcatgagcgttgaacaggcccgggcgattctggccgcggcggcaccgc\n",
      "agcagctggtggcatccgcgcagagtgaagccgatcgcattatgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2669115, 2684581)), (2673026, 2673118), '-'), (('G000299455', (1053988, 1054080)), (1053988, 1054080), '+'), (('G000299455', (1345136, 1375168)), (1371164, 1371256), '+')]\n",
      ">G000006925|2669115|2684581|2673026|2673118|-\n",
      "aaaccatttattgctattaactgtggcgcattgcccgagcaattgctggagtcggagctg\n",
      "tttggtcatgcgcgtggcgcgtttactggcgc-\n",
      ">G000299455|1053988|1054080|1053988|1054080|+\n",
      "-aaccctttattgctatcaactgtgccgcgattccggagcagcttctggaaagcgaactg\n",
      "ttcggttatgttaaaggcgcatttactggcgct\n",
      ">G000299455|1345136|1375168|1371164|1371256|+\n",
      "-aaccatttattgctattaactgtggcgcattgcccgagcaattgctggagtcggagctg\n",
      "tttggtcatgcgcgtggcgcgtttactggcgct\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3361966, 3403725)), (3368888, 3368943), '-'), (('G000299455', (540757, 615358)), (572204, 572259), '+'), (('G000299455', (4235244, 4293324)), (4269290, 4269345), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3361966|3403725|3368888|3368943|-\n",
      "acatggttgttggtcagcacatagcctttactggcgttgatgatgacaccggaac-\n",
      ">G000299455|540757|615358|572204|572259|+\n",
      "-catggttgttggtcagcacatagcctttactggcgttgatgatgacaccggaacc\n",
      ">G000299455|4235244|4293324|4269290|4269345|+\n",
      "-catggttgttggtgacgacatagcctttatcggcatcaatgatgacgccggaacc\n",
      "\n",
      "\n",
      "4 [(('G000006925', (913030, 913272)), (913107, 913261), '-'), (('G000006925', (1419563, 1419894)), (1419573, 1419738), '+'), (('G000299455', (2532173, 2532413)), (2532183, 2532338), '+'), (('G000299455', (2935269, 2935589)), (2935436, 2935589), '-')]\n",
      ">G000006925|913030|913272|913107|913261|-\n",
      "-gatgcggtgcctggtgcctccaggtgacgttaaccagttaacaattacagttggctttc\n",
      "ccacccaaaccaataaggactaacatgac-----------ttttaactgtgccgcgtgcg\n",
      "catagccgcattcaccgcatcacaaaattcactttaaaaagggcgg\n",
      ">G000006925|1419563|1419894|1419573|1419738|+\n",
      "-gatgcggtgcctggtgcctccaggtgacgttaaccagttaacaattaacgccggataca\n",
      "gggaaacccataaccacccgatacgtgggaactaatcgctttttaactgttccgcgtgcg\n",
      "ctcagccgcattcaccgcatcacaaaattcactttaaaaagggcgg\n",
      ">G000299455|2532173|2532413|2532183|2532338|+\n",
      "-gatgcggtgcctggtgcctccgggtgacgttaaccagttaacaactaacgccgggtcag\n",
      "gggacgatg--------actttccgtgacatcctgtcg--gtttaactgttccgcgtgcg\n",
      "catagccgcattcaccgcatcacaaaattcactttaaaaagggcgg\n",
      ">G000299455|2935269|2935589|2935436|2935589|-\n",
      "tgatgcggtgcctggtgcctccaggtgacgttaaccagttaacaattaacgccggataca\n",
      "gagaatccacccataacactgtttttgg------------ttttaactgttccgcgtgcg\n",
      "ctcagccgcattcaccacatcacaaaattcactttaaaaagggcg-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (3083675, 3099254)), (3089517, 3090598), '-'), (('G000299455', (53117, 54201)), (53120, 54201), '+'), (('G000299455', (857924, 871846)), (864915, 865996), '+'), (('G000299455', (3151327, 3152597)), (3151327, 3152409), '+')]\n",
      ">G000006925|3083675|3099254|3089517|3090598|-\n",
      "atcagaaggttacattcagtgtggcctgactgttataaccttcagcgctgctgccgttga\n",
      "tgctgtgggcataactggcctgaacgcccagggtgatattttcccggacacgggcttcca\n",
      "gtccggcctgcaggtccagtgacgtaccattctgtgacggtgagaacgtcatgttactgc\n",
      "cggctgcggctgtacccatgctcatgtccccccgggagctgaaggtgcggataacagaag\n",
      "gctgtacccaccagttcaccggcagttcacgcacactgtgtttggcactgccgcgcaggg\n",
      "tgtcacgggatgaggtgcctttaccaaagttcatatcgtggtggctgcccagacggaagc\n",
      "cggcacgcacatgttgtgcactgccatgcccgaacttcacataactggcgttatcctggc\n",
      "cgtcatccagggagagtccctgccaggtgtactgcagttgcggctccagcatcagattgt\n",
      "cagtgatactgaagggcagaccggtttccagcgaacccagccagccccagccccggacgc\n",
      "ggaagtcgttattgtccgatgaggctttcatgctgtggcgggttccctgggccacaatgt\n",
      "cagcccacaggccggaggcgttgtgtatcagattcaggtatccgcccaggctgccggcat\n",
      "catcccggaccgtgccggcacgggagccgtcatcatccttaacatcaacggaagaatggc\n",
      "cagcagcaccatatacccccgcggtcacagacataccggcaacctctgttctgagcaggt\n",
      "caccctccagacggacgaagccatagctgccgctgctttccggcgtggctccacgggcaa\n",
      "taccgccgttgttatcgtgaccgagatgaccgccctgaatgctgagacggacgctgttat\n",
      "tttcaccgcttacaccggtctgatggctgcgggagcctgccagaatccggtcatagtcca\n",
      "ttgcctgtgtcagcatggaggcatacagggggacttctgcacgataagcattttcactgc\n",
      "gcagataccagctctcatcactgtcccggttgagggagtagttaaaggcaccggcctgca\n",
      "g--\n",
      ">G000299455|53117|54201|53120|54201|+\n",
      "-tcagaaagtcacattcagcgtggcctgaccgttataaccttcagcgctgctgccgctga\n",
      "cgctgtgggcataaccggcctgaacgcccagggtgatattttcccggacacgggcttcca\n",
      "gtccggcctgcaggtccagtgacgtgccattccgggacggtgagaacgtcatgttactgc\n",
      "cggcggcggctgtacccatgctcatgtcaccccgggagctgaaagtgcggataacagaag\n",
      "gctgtacccaccagttcaccggcagttcactcacacggtgttttgtactgtcgcgcagag\n",
      "tgtcacgggatgaggtgccttcgccaaaggtcatatcgttgtggctgcccagacggaagc\n",
      "cggcacgcacatgttgtgcactgccatgcccgaacttcacataaccggcgttatcctggc\n",
      "cgtcatccagggaaagtccctgccaggtatactgcagttgtggctccagcatcaggttgt\n",
      "cagtgatactgaagggcagaccggtttccagtgagcccagccagccccagccgcgggcgc\n",
      "ggaagtcgttattgtccgatgacgctttcatgctgtggcgggttccctgtgccacaatgt\n",
      "cagcccacaggccggaggacgtgtgtgtcagattcaggtatccgcccaggctgccggcat\n",
      "catcccggaccgtgccggcgcgggaaccgtcatcatccttaacatcaacggaagaatggc\n",
      "ctgcggcaccatgcacccctgtcgtcagagacataccggcaacctctgttctgagcaggt\n",
      "caccctccagacggacgaagccatagctgccgctgctttccggcgtggctccacgggcaa\n",
      "taccgccgttgttatcgtgaccgagatgaccgccctgaatgctgagacggacgctgttat\n",
      "tttcaccgtttacaccggtctgatggctgcgggagcctgccagaatccggtcatagtcca\n",
      "ttgcctgtgtcagcatggatgtatacagggggacttcagcacgataagcattttcactgc\n",
      "gcaggtaccagtcttcatcgctgtcacggttcagggtgtagttaaaggcgccggcctgaa\n",
      "gc-\n",
      ">G000299455|857924|871846|864915|865996|+\n",
      "-tcagaaggtcacattcagcgtggcctgactgttataaccttcagcgctgctgccgctga\n",
      "cgctgtgggcataaccggcctgaacgcccagggtgatattttcccggacacgggcttcca\n",
      "gtccggcctgcagatccagtgaagtgccattctgtgagggagagaacgtcatcccgctgc\n",
      "ctgcggtggaggtccccacacgcatatcaccccgggagctgaaggtgcggataacggaag\n",
      "gctgtacccaccagttcaccggcagttcacgcacactgtgttttgcactgtcacgcaggg\n",
      "gggcacgggatgaggtgccttcaccaaaggtcatatcgttgtggctgcccagacggaaac\n",
      "cggcacgcacatgttgtgcactgccatgcccgaacttcacataaccggcgttgtccttac\n",
      "cgtcatccagcgaaagcccctgccaggtgtactgcagccgtggctccagcatcaggttgt\n",
      "cagtaatactgaagggcagaccggtttccagtgagcccagccagccccagccccgtgcgc\n",
      "ggaagtcgttattgcccgatgacgctttcatactgtggcgggttccctgtgccacaatgt\n",
      "cagcccacaggccggaggaggtgtgtgtcaggttcatatatccgcccaggctgccggcat\n",
      "catcccggaccgtgccggcgcgggagccgtcatcatccttaacatcaacggaagaatggc\n",
      "ctgcagcaccatacactcctgtcgtcagtgacataccggcgacctctgttctgagcaggt\n",
      "caccctccagacggacgaagccatagctgccgctgctttccggcgtggccccacgggcaa\n",
      "taccgccgttgttatcgtgaccgagatggccgccctgaatgctgagacggacgctgttat\n",
      "tttcaccgtttacaccggtctgatggctgcgtgagcctgccagaatccggtcatagtcca\n",
      "ttgcctgtgtcagcatggaggcgtacagggggacttcagcacgataacgttcttcactgc\n",
      "gcagataccagctctcatcactgtcccggttgagggtgtagttaaaggccccggcctgca\n",
      "gc-\n",
      ">G000299455|3151327|3152597|3151327|3152409|+\n",
      "-tcagaaagtcacattcagcgtggcctgaccgttatagccttcagcgctgctgccgctga\n",
      "cgctgtgggcataaccggcctgaacgcccagggtgatattttcccggacacgggcttcca\n",
      "gtccggcctgcaggtccagtgatgtgccattctgggacggtgagaacgtcatgttactgc\n",
      "cggctgcggctgtacccatgctcatgtctccccgggaactgaaggtacggataacagaag\n",
      "gctgtacccaccagttcaccggcagttcacgcatactgtgttttgcactgtcacgcaggg\n",
      "gggcacgggatgaggtgccttcgccaaaggtcatatcgttgtggctgcccagacggaaac\n",
      "cggcacgcacatgttgtgcactgccatgcccgaacttcacataaccggcgttgtccttac\n",
      "cgtcatccagggaaagtccctgccaggtatactgcagttgtggctccagcatcaggttgt\n",
      "cagtgatactgaagggcagaccggtttccagtgaacccagccagccccaaccccgggcgc\n",
      "ggaagtcgttattgtccgttgacgctttcatgctgtggcgggttcccagggccacaatgt\n",
      "cagcccacagtccggaggcgttgtgtatcagattcaggtatccgcccaggctgccggcat\n",
      "catcccggaccgtgccggcacgggagccgtcatcatccttaacatcaacggaagaatggc\n",
      "cagcagcaccatatacccccgcggtcacagacataccggcaacctctgttctcatcaggt\n",
      "caccctccagacggacgaatccatagctgccgctgctttccggcgtggccccacgggcaa\n",
      "taccgccattgttatcgtgaccgagatgaccgccctgaatgctgagacggacgctgttgt\n",
      "tttcaccatttacaccggtctgatggctgcgggagcctgccacaatccggtcatagtcca\n",
      "ttgcctgtgtcagcatggaggcatacagggggacttctgcacgataagcattttcactgc\n",
      "gcagataccagctctcatcactgtcccggttgagggagtagttaaaggcaccggcctgca\n",
      "gcc\n",
      "\n",
      "\n",
      "108 [(('G000006925', (1, 24671)), (18842, 18883), '+'), (('G000006925', (64491, 216702)), (87494, 87535), '-'), (('G000006925', (64491, 216702)), (148684, 148725), '-'), (('G000006925', (266682, 267450)), (267235, 267276), '-'), (('G000006925', (273344, 278315)), (273518, 273559), '+'), (('G000006925', (278320, 279091)), (278875, 278916), '-'), (('G000006925', (300111, 308544)), (308328, 308369), '-'), (('G000006925', (378637, 384812)), (379192, 379233), '-'), (('G000006925', (459861, 468373)), (465891, 465932), '+'), (('G000006925', (468374, 487224)), (478887, 478928), '+'), (('G000006925', (520755, 528026)), (521308, 521349), '-'), (('G000006925', (545486, 558918)), (558324, 558365), '+'), (('G000006925', (622915, 623662)), (623089, 623130), '+'), (('G000006925', (865887, 867389)), (866795, 866836), '+'), (('G000006925', (984595, 985335)), (985150, 985191), '-'), (('G000006925', (985336, 1008622)), (985604, 985645), '+'), (('G000006925', (1025530, 1063042)), (1049746, 1049787), '+'), (('G000006925', (1189492, 1208094)), (1197822, 1197863), '+'), (('G000006925', (1239481, 1240251)), (1239655, 1239696), '+'), (('G000006925', (1290847, 1306886)), (1295247, 1295288), '+'), (('G000006925', (1321493, 1385605)), (1366007, 1366048), '-'), (('G000006925', (1398496, 1399271)), (1398672, 1398713), '+'), (('G000006925', (1409877, 1410649)), (1410434, 1410475), '-'), (('G000006925', (1424264, 1425034)), (1424819, 1424860), '-'), (('G000006925', (1425133, 1465118)), (1444454, 1444495), '+'), (('G000006925', (1509064, 1578296)), (1526853, 1526894), '+'), (('G000006925', (1509064, 1578296)), (1542940, 1542981), '+'), (('G000006925', (1509064, 1578296)), (1568744, 1568785), '-'), (('G000006925', (1509064, 1578296)), (1574702, 1574743), '-'), (('G000006925', (1670306, 1692658)), (1673546, 1673587), '+'), (('G000006925', (1729332, 1741706)), (1736527, 1736568), '-'), (('G000006925', (1758378, 1759147)), (1758552, 1758593), '+'), (('G000006925', (1759148, 1776249)), (1766702, 1766743), '+'), (('G000006925', (1759148, 1776249)), (1769585, 1769626), '-'), (('G000006925', (1776250, 1792578)), (1781087, 1781128), '+'), (('G000006925', (1793868, 1802424)), (1795682, 1795723), '-'), (('G000006925', (1842508, 1869155)), (1850602, 1850643), '-'), (('G000006925', (1842508, 1869155)), (1856540, 1856581), '-'), (('G000006925', (1940277, 1941051)), (1940451, 1940492), '+'), (('G000006925', (1976680, 1977448)), (1977233, 1977274), '-'), (('G000006925', (2036705, 2044875)), (2037708, 2037749), '-'), (('G000006925', (2050169, 2051841)), (2051626, 2051667), '-'), (('G000006925', (2088344, 2103602)), (2095214, 2095255), '+'), (('G000006925', (2120115, 2138771)), (2135201, 2135242), '-'), (('G000006925', (2172359, 2184266)), (2177966, 2178007), '+'), (('G000006925', (2187913, 2198767)), (2197440, 2197481), '-'), (('G000006925', (2198768, 2199537)), (2198943, 2198984), '+'), (('G000006925', (2199538, 2210687)), (2206104, 2206145), '-'), (('G000006925', (2211287, 2225774)), (2220807, 2220848), '+'), (('G000006925', (2235234, 2236005)), (2235787, 2235828), '-'), (('G000006925', (2242494, 2346249)), (2287619, 2287660), '+'), (('G000006925', (2374481, 2472008)), (2396188, 2396229), '-'), (('G000006925', (2756085, 2756853)), (2756638, 2756679), '-'), (('G000006925', (2816684, 2833035)), (2825375, 2825416), '+'), (('G000006925', (2849870, 2870340)), (2863393, 2863434), '-'), (('G000006925', (2964578, 2976271)), (2964582, 2964623), '-'), (('G000006925', (2976303, 3015537)), (3014807, 3014848), '+'), (('G000006925', (3016790, 3036697)), (3021638, 3021679), '+'), (('G000006925', (3038529, 3039299)), (3038703, 3038744), '+'), (('G000006925', (3104758, 3106299)), (3104933, 3104974), '+'), (('G000006925', (3179574, 3180459)), (3180243, 3180284), '-'), (('G000006925', (3180744, 3203996)), (3182134, 3182175), '-'), (('G000006925', (3237769, 3244530)), (3238322, 3238363), '-'), (('G000006925', (3268417, 3281594)), (3274324, 3274365), '-'), (('G000006925', (3322761, 3361739)), (3361521, 3361562), '-'), (('G000006925', (3361966, 3403725)), (3362521, 3362562), '-'), (('G000006925', (3608503, 3611864)), (3609061, 3609102), '-'), (('G000006925', (3612083, 3637192)), (3624407, 3624448), '+'), (('G000006925', (3637330, 3640023)), (3637504, 3637545), '+'), (('G000006925', (3658114, 3676993)), (3665334, 3665375), '+'), (('G000006925', (3693106, 3702974)), (3693661, 3693702), '-'), (('G000006925', (3703179, 3717854)), (3703733, 3703774), '-'), (('G000006925', (3717879, 3736288)), (3726017, 3726058), '-'), (('G000006925', (3736321, 3788106)), (3743196, 3743237), '+'), (('G000006925', (3815875, 3829465)), (3818179, 3818220), '+'), (('G000006925', (3831593, 3850590)), (3834660, 3834701), '+'), (('G000006925', (3858569, 3864367)), (3864152, 3864193), '-'), (('G000006925', (3939882, 3952896)), (3943387, 3943428), '-'), (('G000006925', (4002899, 4012437)), (4003452, 4003493), '-'), (('G000006925', (4069200, 4076620)), (4076407, 4076448), '-'), (('G000006925', (4091607, 4092380)), (4091782, 4091823), '+'), (('G000006925', (4094093, 4116338)), (4102488, 4102529), '+'), (('G000006925', (4174522, 4188276)), (4177116, 4177157), '-'), (('G000006925', (4256570, 4288375)), (4285153, 4285194), '+'), (('G000006925', (4288513, 4315006)), (4314790, 4314831), '-'), (('G000006925', (4362175, 4367702)), (4367108, 4367149), '+'), (('G000006925', (4378758, 4379529)), (4378932, 4378973), '+'), (('G000006925', (4389053, 4390969)), (4389228, 4389269), '+'), (('G000006925', (4392644, 4394283)), (4393202, 4393243), '-'), (('G000006925', (4397536, 4410956)), (4410741, 4410782), '-'), (('G000006925', (4462252, 4478752)), (4462427, 4462468), '+'), (('G000006925', (4478753, 4490457)), (4488565, 4488606), '+'), (('G000006925', (4497401, 4528664)), (4515423, 4515464), '-'), (('G000006925', (4535976, 4543278)), (4541063, 4541104), '-'), (('G000006925', (4557883, 4573403)), (4570276, 4570317), '-'), (('G000006925', (4645852, 4646621)), (4646405, 4646446), '-'), (('G000006925', (4656735, 4658205)), (4657289, 4657330), '-'), (('G000006925', (4785482, 4786253)), (4785658, 4785699), '+'), (('G000299455', (91523, 102755)), (92081, 92122), '-'), (('G000299455', (696017, 696792)), (696193, 696234), '+'), (('G000299455', (941725, 942502)), (941902, 941943), '+'), (('G000299455', (1581872, 1582655)), (1582430, 1582471), '-'), (('G000299455', (2052398, 2064114)), (2062510, 2062551), '-'), (('G000299455', (3182222, 3183821)), (3183223, 3183264), '+'), (('G000299455', (3672841, 3680073)), (3679852, 3679893), '-'), (('G000299455', (4588613, 4589520)), (4588919, 4588960), '+'), (('G000299455', (4592900, 4593681)), (4593460, 4593501), '-'), (('G000299455', (5323356, 5324120)), (5323901, 5323942), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "  101 / 108\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP   101 / 107  h\n",
      "Reallocating..done. *alloclen = 1084\n",
      "STEP   107 / 107  h\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "  100 / 108\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP   107 / 107  h\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     8 / 11 \n",
      "Reallocating..done. *alloclen = 1058\n",
      "STEP    11 / 11 \n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 12\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "   10 / 12\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP    11 / 11 \n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|18842|18883|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|64491|216702|87494|87535|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|64491|216702|148684|148725|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|266682|267450|267235|267276|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|273344|278315|273518|273559|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|278320|279091|278875|278916|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|300111|308544|308328|308369|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|378637|384812|379192|379233|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|459861|468373|465891|465932|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|468374|487224|478887|478928|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|520755|528026|521308|521349|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|545486|558918|558324|558365|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|622915|623662|623089|623130|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|865887|867389|866795|866836|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|984595|985335|985150|985191|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|985336|1008622|985604|985645|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1025530|1063042|1049746|1049787|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1189492|1208094|1197822|1197863|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1239481|1240251|1239655|1239696|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1290847|1306886|1295247|1295288|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1321493|1385605|1366007|1366048|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1398496|1399271|1398672|1398713|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|1409877|1410649|1410434|1410475|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|1424264|1425034|1424819|1424860|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1425133|1465118|1444454|1444495|+\n",
      "-gtgcagcttttccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1509064|1578296|1526853|1526894|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1509064|1578296|1542940|1542981|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1509064|1578296|1568744|1568785|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|1509064|1578296|1574702|1574743|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1670306|1692658|1673546|1673587|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|1729332|1741706|1736527|1736568|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|1758378|1759147|1758552|1758593|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|1759148|1776249|1766702|1766743|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1759148|1776249|1769585|1769626|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1776250|1792578|1781087|1781128|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|1793868|1802424|1795682|1795723|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1842508|1869155|1850602|1850643|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1842508|1869155|1856540|1856581|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|1940277|1941051|1940451|1940492|+\n",
      "-gtgcagctttcccttcaggcgggattcatacagcggccagc\n",
      ">G000006925|1976680|1977448|1977233|1977274|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2036705|2044875|2037708|2037749|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2050169|2051841|2051626|2051667|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2088344|2103602|2095214|2095255|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|2120115|2138771|2135201|2135242|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2172359|2184266|2177966|2178007|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|2187913|2198767|2197440|2197481|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000006925|2198768|2199537|2198943|2198984|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|2199538|2210687|2206104|2206145|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2211287|2225774|2220807|2220848|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccaac\n",
      ">G000006925|2235234|2236005|2235787|2235828|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2242494|2346249|2287619|2287660|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|2374481|2472008|2396188|2396229|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2756085|2756853|2756638|2756679|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2816684|2833035|2825375|2825416|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|2849870|2870340|2863393|2863434|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2964578|2976271|2964582|2964623|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|2976303|3015537|3014807|3014848|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|3016790|3036697|3021638|3021679|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3038529|3039299|3038703|3038744|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3104758|3106299|3104933|3104974|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3179574|3180459|3180243|3180284|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3180744|3203996|3182134|3182175|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3237769|3244530|3238322|3238363|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3268417|3281594|3274324|3274365|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3322761|3361739|3361521|3361562|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3361966|3403725|3362521|3362562|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3608503|3611864|3609061|3609102|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3612083|3637192|3624407|3624448|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3637330|3640023|3637504|3637545|+\n",
      "-gtgcagctttcccttcaggcgtggttcatacagcggccagc\n",
      ">G000006925|3658114|3676993|3665334|3665375|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3693106|3702974|3693661|3693702|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|3703179|3717854|3703733|3703774|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3717879|3736288|3726017|3726058|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3736321|3788106|3743196|3743237|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3815875|3829465|3818179|3818220|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3831593|3850590|3834660|3834701|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|3858569|3864367|3864152|3864193|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|3939882|3952896|3943387|3943428|-\n",
      "cgtgcagctttcccttcaggcgtgactcatacagcggccag-\n",
      ">G000006925|4002899|4012437|4003452|4003493|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4069200|4076620|4076407|4076448|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4091607|4092380|4091782|4091823|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4094093|4116338|4102488|4102529|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4174522|4188276|4177116|4177157|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4256570|4288375|4285153|4285194|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4288513|4315006|4314790|4314831|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4362175|4367702|4367108|4367149|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4378758|4379529|4378932|4378973|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggacagc\n",
      ">G000006925|4389053|4390969|4389228|4389269|+\n",
      "-gtgcagctttcccttcaggcatgattcatacagcggccagc\n",
      ">G000006925|4392644|4394283|4393202|4393243|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4397536|4410956|4410741|4410782|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4462252|4478752|4462427|4462468|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4478753|4490457|4488565|4488606|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000006925|4497401|4528664|4515423|4515464|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4535976|4543278|4541063|4541104|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4557883|4573403|4570276|4570317|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggccag-\n",
      ">G000006925|4645852|4646621|4646405|4646446|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|4656735|4658205|4657289|4657330|-\n",
      "cgtgcagctttcccttcaggcgtgattcatacagcggacag-\n",
      ">G000006925|4785482|4786253|4785658|4785699|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000299455|91523|102755|92081|92122|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000299455|696017|696792|696193|696234|+\n",
      "-gtgcagctttcccttcaggcgggattcatacagcggccagc\n",
      ">G000299455|941725|942502|941902|941943|+\n",
      "-gtgcagctttcccttcaggcgggattcatacagcggccagt\n",
      ">G000299455|1581872|1582655|1582430|1582471|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000299455|2052398|2064114|2062510|2062551|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000299455|3182222|3183821|3183223|3183264|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000299455|3672841|3680073|3679852|3679893|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000299455|4588613|4589520|4588919|4588960|+\n",
      "-gtgcagctttcccttcaggcgtgattcatacagcggccagc\n",
      ">G000299455|4592900|4593681|4593460|4593501|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      ">G000299455|5323356|5324120|5323901|5323942|-\n",
      "cgtgcagctttcccttcaggcgggattcatacagcggccag-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (726235, 732999)), (729878, 729937), '+'), (('G000299455', (2174641, 2181396)), (2178274, 2178333), '+'), (('G000299455', (2910238, 2916987)), (2913295, 2913354), '-'), (('G000299455', (3600777, 3600836)), (3600777, 3600836), '-')]\n",
      ">G000006925|726235|732999|729878|729937|+\n",
      "-gcattcaatgagccgctgcttctggagcccgcctatgcgcgggttttcttttgcgcgct\n",
      ">G000299455|2174641|2181396|2178274|2178333|+\n",
      "-gcattcaatgaaccgctgcttctggagcccgcctatgcgcgggttttcttttgcgcgct\n",
      ">G000299455|2910238|2916987|2913295|2913354|-\n",
      "agcattcaatgaaccgctgcttctggagcccgcctatgcgcgggttttcttttgcgcgc-\n",
      ">G000299455|3600777|3600836|3600777|3600836|-\n",
      "ggcctttaatgagccgctgatgcttgaacccgcctatgcgcgggttttcttttgtgcgc-\n",
      "\n",
      "\n",
      "12 [(('G000006925', (1209065, 1211067)), (1211038, 1211066), '-'), (('G000006925', (2756889, 2766574)), (2758119, 2758147), '-'), (('G000006925', (2923039, 2950048)), (2949234, 2949262), '-'), (('G000006925', (3101814, 3103074)), (3103045, 3103073), '-'), (('G000006925', (3810378, 3811029)), (3810379, 3810407), '+'), (('G000006925', (4624461, 4624660)), (4624628, 4624656), '-'), (('G000006925', (4627110, 4627241)), (4627211, 4627239), '-'), (('G000006925', (4628976, 4629562)), (4629532, 4629560), '-'), (('G000006925', (4825967, 4826151)), (4826121, 4826149), '-'), (('G000006925', (4826153, 4826197)), (4826167, 4826195), '-'), (('G000299455', (3226712, 3229721)), (3229688, 3229716), '-'), (('G000299455', (5275617, 5275694)), (5275620, 5275648), '+')]\n",
      ">G000006925|1209065|1211067|1211038|1211066|-\n",
      "gatcctacccacgtaatatggacacagg-\n",
      ">G000006925|2756889|2766574|2758119|2758147|-\n",
      "gatcctacccacgtaatatggacacagg-\n",
      ">G000006925|2923039|2950048|2949234|2949262|-\n",
      "gatcctacccacgtaatatggacacagg-\n",
      ">G000006925|3101814|3103074|3103045|3103073|-\n",
      "gatcctacccacgtaatatggacacagg-\n",
      ">G000006925|3810378|3811029|3810379|3810407|+\n",
      "-atcctacccacgtaatgtggacacggcc\n",
      ">G000006925|4624461|4624660|4624628|4624656|-\n",
      "-atcctacccacataatatggacacagtc\n",
      ">G000006925|4627110|4627241|4627211|4627239|-\n",
      "-atcctacccacgtaatatgaacacaggc\n",
      ">G000006925|4628976|4629562|4629532|4629560|-\n",
      "-atcctacccacgtaatatggacacaggc\n",
      ">G000006925|4825967|4826151|4826121|4826149|-\n",
      "-agcctacccacgtaatatggacacaggc\n",
      ">G000006925|4826153|4826197|4826167|4826195|-\n",
      "gatcctacccacataatatggacgcagt-\n",
      ">G000299455|3226712|3229721|3229688|3229716|-\n",
      "gatcctacccacgtaatatggacacagg-\n",
      ">G000299455|5275617|5275694|5275620|5275648|+\n",
      "-atcccaccctcgtaatatggacacaggt\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4342604, 4342669), '+'), (('G000299455', (3480865, 3520974)), (3489986, 3490051), '+'), (('G000299455', (4865050, 4904182)), (4892654, 4892719), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|4315026|4354132|4342604|4342669|+\n",
      "ttcaggccaaatgacatgttttctgctactgacaggtggggatagagcgcgtaagactga\n",
      "aacac\n",
      ">G000299455|3480865|3520974|3489986|3490051|+\n",
      "ttcaggccaaaagcgatgttctgttccacggtcatatgcggaaacagcgcgtaagactga\n",
      "aacat\n",
      ">G000299455|4865050|4904182|4892654|4892719|+\n",
      "ttcaggccaaatgacatgttttctgctactgacaggtggggatagagcgcgtaagactga\n",
      "aacac\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4256570, 4288375)), (4271283, 4271312), '+'), (('G000299455', (3844542, 3852289)), (3851610, 3851639), '+'), (('G000299455', (4795454, 4823864)), (4810167, 4810196), '+')]\n",
      ">G000006925|4256570|4288375|4271283|4271312|+\n",
      "tgcggctacaacagtacgtcgtactttat\n",
      ">G000299455|3844542|3852289|3851610|3851639|+\n",
      "tgtggctatagcagcacgtcatactttat\n",
      ">G000299455|4795454|4823864|4810167|4810196|+\n",
      "tgcggctacaacagtacgtcgtactttat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1306887, 1319577)), (1310862, 1310982), '-'), (('G000299455', (2883599, 2892315)), (2892195, 2892315), '+'), (('G000299455', (2940021, 2958111)), (2940021, 2940141), '+')]\n",
      ">G000006925|1306887|1319577|1310862|1310982|-\n",
      "aattaaaaacgatatcctgctgagaacataaacacccacggatcgaggcgtacgctatcg\n",
      "tgttgctgtgcaccgcctagcttataattggcggtggtatcgatatccatgtaccacact\n",
      "-\n",
      ">G000299455|2883599|2892315|2892195|2892315|+\n",
      "-attaaaaacgatatcctgctgagaacataaacacccacggatcgaggcgtacgctatcg\n",
      "tgttgctgtgcaccgcctagcttataattggcggtggtatcgatatccatgtaccagact\n",
      "g\n",
      ">G000299455|2940021|2958111|2940021|2940141|+\n",
      "-attaaaaacgatatcctgcggagaacataaacacccacggatccagtcgtaccgagtct\n",
      "tttacggtagtaacaccagatttgtacttagccgtggtatcgatatccatgtaccacact\n",
      "g\n",
      "\n",
      "\n",
      "7 [(('G000006925', (1, 24671)), (15590, 15610), '+'), (('G000006925', (905848, 907049)), (906121, 906142), '+'), (('G000006925', (1397676, 1398098)), (1397796, 1397817), '-'), (('G000006925', (2062783, 2064325)), (2064171, 2064193), '+'), (('G000299455', (2541782, 2542216)), (2541914, 2541935), '-'), (('G000299455', (2927401, 2931337)), (2930399, 2930420), '+'), (('G000299455', (4431865, 4470166)), (4434923, 4434943), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 7\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 6  f\b\b\r",
      "STEP     2 / 6  f\b\b\r",
      "STEP     3 / 6  f\b\b\r",
      "STEP     4 / 6  f\b\b\r",
      "STEP     5 / 6  f\b\b\r",
      "STEP     6 / 6  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1|24671|15590|15610|+\n",
      "--actctacatgtgttcagcat\n",
      ">G000006925|905848|907049|906121|906142|+\n",
      "-taacctacatgtgcaaagcat\n",
      ">G000006925|1397676|1398098|1397796|1397817|-\n",
      "ctaacctacatttgtgagaca-\n",
      ">G000006925|2062783|2064325|2064171|2064193|+\n",
      "ctaacctacatgtgttcagcat\n",
      ">G000299455|2541782|2542216|2541914|2541935|-\n",
      "ctaacctacatgtgtctagca-\n",
      ">G000299455|2927401|2931337|2930399|2930420|+\n",
      "-taacctacatttgtgagacat\n",
      ">G000299455|4431865|4470166|4434923|4434943|-\n",
      "-cactctacatgtgttcagca-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2770832, 2816659)), (2771988, 2772016), '-'), (('G000299455', (1254267, 1277763)), (1273820, 1273848), '+'), (('G000299455', (2959070, 2965951)), (2960941, 2960969), '+')]\n",
      ">G000006925|2770832|2816659|2771988|2772016|-\n",
      "aaggtaaatctctcgacgatttcctgat-\n",
      ">G000299455|1254267|1277763|1273820|1273848|+\n",
      "-aggtaaatctctcgacgatttcctgatc\n",
      ">G000299455|2959070|2965951|2960941|2960969|+\n",
      "-aggtaaatccctcgacgatttcctgatc\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1842508, 1869155)), (1861301, 1861347), '-'), (('G000299455', (2066598, 2066708)), (2066662, 2066708), '+'), (('G000299455', (2786550, 2804122)), (2796258, 2796304), '-')]\n",
      ">G000006925|1842508|1869155|1861301|1861347|-\n",
      "cggcgagcagggcaggaattaaaagtgccagtacttttcttttcat-\n",
      ">G000299455|2066598|2066708|2066662|2066708|+\n",
      "-aactaataacgccgggaccagcattgccagaacttttcttttcatt\n",
      ">G000299455|2786550|2804122|2796258|2796304|-\n",
      "cggcgagcagggcaggaattaaaagtgccagtactttgcttttcat-\n",
      "\n",
      "\n",
      "8 [(('G000006925', (3180744, 3203996)), (3183461, 3183507), '-'), (('G000006925', (3180744, 3203996)), (3183837, 3183883), '-'), (('G000006925', (3180744, 3203996)), (3184212, 3184258), '-'), (('G000299455', (747195, 760809)), (759275, 759321), '+'), (('G000299455', (747195, 760809)), (759651, 759697), '+'), (('G000299455', (1901701, 1901773)), (1901701, 1901747), '-'), (('G000299455', (1902029, 1902101)), (1902029, 1902075), '-'), (('G000299455', (1902357, 1902474)), (1902357, 1902404), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3180744|3203996|3183461|3183507|-\n",
      "ttccttctgtaaggccc-cttcctcgggaggggctttcccgtttcag-\n",
      ">G000006925|3180744|3203996|3183837|3183883|-\n",
      "ttccttctataaggccc-cttcctcgggaggggctttcccgtttcag-\n",
      ">G000006925|3180744|3203996|3184212|3184258|-\n",
      "ttccttctataaggccc-cttcctcgggaggggctttcccgtttcag-\n",
      ">G000299455|747195|760809|759275|759321|+\n",
      "-tccttctataaggccc-cttcctcgggaggggctttcccgtttcagc\n",
      ">G000299455|747195|760809|759651|759697|+\n",
      "-tccttctgtaaggccc-cttcctcgggaggggctttcccgtttcagc\n",
      ">G000299455|1901701|1901773|1901701|1901747|-\n",
      "ttccttatttaaggccc-cttcctcgggaggggctttcccgtttcag-\n",
      ">G000299455|1902029|1902101|1902029|1902075|-\n",
      "ttccttatttaaggccc-cttcctcgggaggggctttcccgtttcag-\n",
      ">G000299455|1902357|1902474|1902357|1902404|-\n",
      "ttccttattaaaagccctcttctccgggagaggctttcccgtttcag-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (2907928, 2923032)), (2923012, 2923032), '-'), (('G000006925', (2923039, 2950048)), (2923039, 2923059), '-'), (('G000299455', (1078152, 1154001)), (1103092, 1103112), '+')]\n",
      ">G000006925|2907928|2923032|2923012|2923032|-\n",
      "gatggtgctgatggtgatga-\n",
      ">G000006925|2923039|2950048|2923039|2923059|-\n",
      "gatggtgctgatggtgctga-\n",
      ">G000299455|1078152|1154001|1103092|1103112|+\n",
      "-atggtgctgatggtgatgat\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3969227, 3993100)), (3991581, 3991619), '+'), (('G000299455', (2650678, 2664983)), (2657490, 2657528), '+'), (('G000299455', (5156132, 5190970)), (5168577, 5168615), '-')]\n",
      ">G000006925|3969227|3993100|3991581|3991619|+\n",
      "-gtaatctgaactgtgactactgtttttatctcgaaaaa\n",
      ">G000299455|2650678|2664983|2657490|2657528|+\n",
      "-gtaatctcaaatgtgattactgtttttaccttgaaaaa\n",
      ">G000299455|5156132|5190970|5168577|5168615|-\n",
      "tgtaatctgaactgtgactactgtttttatctcgaaaa-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (3693106, 3702974)), (3696051, 3696931), '+'), (('G000299455', (269622, 287922)), (271797, 272675), '+'), (('G000299455', (2665739, 2672184)), (2666726, 2667606), '+')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|3693106|3702974|3696051|3696931|+\n",
      "gagctgcgtgagatccctatgcgccccggtcagttgtttatggacccgaaacgcatgatt\n",
      "gaagcctgtgacgaaaacaccatcggcgtggtgccgactttcggcgtgacctacaccggt\n",
      "aactatgagttcccacaaccgctgcacgatgcgctggataaattccaggccgacaccggt\n",
      "atcgacatcgacatgcacatcgacgctgccagcggtggcttcctggcaccgttcgtcgcc\n",
      "ccggatatcgtctgggacttccgcctgccgcgtgtgaaatcgatcagtgcttcaggccat\n",
      "aaattcggtctggctccgctgggctgcggctgggttatctggcgtgacgaagaagcgctg\n",
      "ccgcaggaactggtgttcaacgttgactacctgggtggtcaaattggtacttttgccatc\n",
      "aacttctcccgcccggcgggtcaggtaattgcacagtactatgaattcctgcgcctcggt\n",
      "cgtgaaggctataccaaagtacagaacgcctcttaccaggttgccgcttatctggcggat\n",
      "gaaatcgccaaacaggggccgtatgagttcatctgtacgggtcgcccggacgaaggcatc\n",
      "ccggcggtttgcttcaaactgaaagatggtgaagatccgggatacaccctgtacgacctc\n",
      "tctgaacgtctgcgtctgcgcggctggcaggttccggccttcactctcggcggtgaagcc\n",
      "accgacatcgtggtgatgcgcattatgtgtcgtcgcggcttcgaaatggactttgctgaa\n",
      "ctgttgctggaagactacaaagcctccctgaaatatctcagcgatcacccgaaactgcag\n",
      "ggtattgcccagcagaacagctttaaacacacctgataac\n",
      ">G000299455|269622|287922|271797|272675|+\n",
      "gagctgcgtgagatccctatgcgccccggtcagttgtttatggacccgaaacgcatgatt\n",
      "gaagcctgtgacgaaaacaccatcggcgtggtgccgactttcggcgtgacctacaccggt\n",
      "aactatgagttcccacaaccgctgcacgatgcgctggataaattccaggccg--acaggt\n",
      "atcgacatcgacatgcacatcgacgctgccagcggtggcttcctggcaccgttcgtcgcc\n",
      "ccggatatcgtctgggacttccgcctgccgcgtgtgaaatcgatcagtgcttcaggccat\n",
      "aaattcggtctggctccgctgggctgcggctgggttatctggcgtgacgaagaagcgctg\n",
      "ccgcaggaactggtgttcaacgttgactacctgggtggtcaaattggtacttttgccatc\n",
      "aacttctcccgcccggcgggtcaggtaattgcacagtactatgaattcctgcgcctcggt\n",
      "cgtgaaggctacaccaaagtacagaacgcctcttaccaggttgccgcttatctggcggat\n",
      "gaaatcgccaaactggggccgtatgagttcatctgtacgggtcgcccggacgaaggcatc\n",
      "ccggcggtttgcttcaaactgaaagatggtgaagatccgggatacaccctgtacgacctc\n",
      "tctgaacgtctgcgtctgcgcggctggcaggttccggccttcactctcggcggtgaagcc\n",
      "accgacatcgtggtgatgcgcattatgtgtcgtcgcggcttcgaaatggactttgctgaa\n",
      "ctgttgctggaagactacaaagcctccctgaaatatctcagcgatcacccgaaactgcag\n",
      "ggtattgcccagcagaacagctttaaacacacctgataac\n",
      ">G000299455|2665739|2672184|2666726|2667606|+\n",
      "gagctgcgtgagatccctatgcgccccggtcagttgtttatggacccgaaacgcatgatt\n",
      "gaagcctgtgacgaaaacaccatcggcgtggtgccgactttcggcgtgacctacactggt\n",
      "aactatgagttcccacaaccgctgcacgatgcgctggataaattccaggccgatacaggt\n",
      "atcgacatcgacatgcacatcgacgctgccagcggtggcttcctggcaccgttcgtcgcc\n",
      "ccggatatcgtctgggacttccgcctgccgcgtgtgaaatcgatcagtgcttcaggccat\n",
      "aaattcggtctggctccgctgggctgcggctgggttatctggcgtgacgaagaagcgctg\n",
      "ccgcaggaactggtgttcaacgttgactacctgggtggtcaaattggtacttttgccatc\n",
      "aacttctcccgcccggcgggtcaggtaattgcacagtactatgaattcctgcgcctcggt\n",
      "cgtgaaggctacaccaaagtacagaacgcctcttaccaggttgccgcttatctggcggat\n",
      "gaaatcgccaaactggggccgtatgagttcatctgtacgggtcgcccggacgaaggcatc\n",
      "ccggcggtttgcttcaaactgaaagatggtgaagatccgggatacaccctgtacgacctc\n",
      "tctgaacgtctgcgtctgcgcggctggcaggttccggccttcactctcggcggtgaagcc\n",
      "accgacatcgtggtgatgcgcattatgtgtcgtcgcggcttcgaaatggactttgctgaa\n",
      "ctgttgctggaagactacaaagcctccctgaaatatctcagcgatcacccgaaactgcag\n",
      "ggtattgcccagcagaacagctttaaacatacctgataac\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1177057, 1177245)), (1177168, 1177228), '+'), (('G000006925', (3814062, 3814543)), (3814252, 3814312), '-'), (('G000299455', (3207505, 3210112)), (3207695, 3207755), '-'), (('G000299455', (4139671, 4139852)), (4139675, 4139735), '-')]\n",
      ">G000006925|1177057|1177245|1177168|1177228|+\n",
      "-tcttcgttggtcattgtgtgcgcctgtaaagcaatgcgccgtagtacctcacaccacgg\n",
      "c\n",
      ">G000006925|3814062|3814543|3814252|3814312|-\n",
      "ttcttcgttagtcattgtctgcaccgggaaagcaatttaccggagtacctcacactacgg\n",
      "-\n",
      ">G000299455|3207505|3210112|3207695|3207755|-\n",
      "ttcttcgttagtcattgtctgcacctggaaagcaatttaccggagtacctcacactacgg\n",
      "-\n",
      ">G000299455|4139671|4139852|4139675|4139735|-\n",
      "ttctttgttggtcatggcgctgccctgtaaaacgatgcaccgtagttcctcacaccacgg\n",
      "-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4094093, 4116338)), (4100867, 4101207), '+'), (('G000299455', (2682449, 2700553)), (2690136, 2690476), '+'), (('G000299455', (5063005, 5071226)), (5064112, 5064452), '-')]\n",
      ">G000006925|4094093|4116338|4100867|4101207|+\n",
      "-gccatttatcagaacctggcgcacggtattccggaaacttcagacggctttcggagtgg\n",
      "atgaaatccaccaggccagcgcctttcggacaaagtgcaccgcggttgaccgggtgatcc\n",
      "ggatcgccttcgatatggaagatagatgctttggcgttttttgctccgtcaccgaggctg\n",
      "tacatcaacagcccacagcctacggaacaataggtgcaggtattacgggtttcgcgggtg\n",
      "cgcagcagtttatactgccgggtttccgcgagtgctacgctgggtgcaaaacccagtgcc\n",
      "gctgccgtggtgcctgccataccgccagcgcagatcttaaa\n",
      ">G000299455|2682449|2700553|2690136|2690476|+\n",
      "-gccatttgtcagaacctggcgcacgatattccgggtagcgcagacggttttcgctgttg\n",
      "acgtaatccagcaaaccggccccttttgggcacagcgcaccacggcttaccggatgatcc\n",
      "gggtcaccttcaatgtgataaatcgcttctctggcgtttttcgcgccatcacccaggcta\n",
      "tacatcaatagcccgcaacctacggaacagtatgtgcaggtgttacggatctctttagcg\n",
      "cgtaataatttgtagtttcgcgcctgagccagtgcttgcttcggggcaaagcccaatgcc\n",
      "gctaccgttgttccagccataccgcccgcgcagattttaaa\n",
      ">G000299455|5063005|5071226|5064112|5064452|-\n",
      "tgccatttatcagaacctggcgcacggtattccggaaacttcagacggctttcggagtgg\n",
      "atgaaatccaccaggccagcgcctttcggacaaagtgcaccgcggttgaccgggtgatcc\n",
      "ggatcgccttcgatatggaagatagatgctttggcgttttttgctccgtcaccgaggctg\n",
      "tacatcaacagcccacagcctacggaacaataggtgcaggtattacgggtttcgcgggtg\n",
      "cgcagcagtttatactgccgggtttccgcgagcgctacgctgggtgcaaaacccagtgcc\n",
      "gctgccgtggtgcctgccataccgccagcgcagatcttaa-\n",
      "\n",
      "\n",
      "4 [(('G000006925', (2560406, 2561162)), (2560558, 2560660), '+'), (('G000006925', (4633099, 4633840)), (4633586, 4633688), '-'), (('G000006925', (4668315, 4668899)), (4668646, 4668748), '-'), (('G000299455', (5017408, 5018168)), (5017910, 5018011), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|2560406|2561162|2560558|2560660|+\n",
      "-ttgccggtcagatgcttattcttcggcacctcccggccatagctgccccagtcatcgct\n",
      "ggtcagcatgccgatgttgaagggtgtaagcagtgccagtagc\n",
      ">G000006925|4633099|4633840|4633586|4633688|-\n",
      "tttgccggtcaggtgtttctccttcggcacttctctggcgtagctgccccagtcatcgct\n",
      "ggtcaccatgccgatacagaacggagtcaacagggctaacag-\n",
      ">G000006925|4668315|4668899|4668646|4668748|-\n",
      "tttgccggtcaggtgtttctcctccggcacttctctggcgtagctgccccagtcatcgct\n",
      "ggtcaccatgccgatacagaacggagtcaacagggctaacag-\n",
      ">G000299455|5017408|5018168|5017910|5018011|-\n",
      "cttaccagtcagatgcttctccttcggtaactctctagcgtaactaccccattcgtcagt\n",
      "ggtcaacatgccgatggtaaaagg-gtaagcagcgcccgcaa-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (4781327, 4781663)), (4781525, 4781545), '+'), (('G000299455', (1991020, 1991316)), (1991210, 1991230), '+'), (('G000299455', (3215105, 3215383)), (3215292, 3215312), '+'), (('G000299455', (4543449, 4543753)), (4543647, 4543667), '+'), (('G000299455', (5289627, 5289931)), (5289713, 5289733), '-')]\n",
      ">G000006925|4781327|4781663|4781525|4781545|+\n",
      "-acggggccgcttcgtctggc\n",
      ">G000299455|1991020|1991316|1991210|1991230|+\n",
      "-acgtggtcgcttcgcctggc\n",
      ">G000299455|3215105|3215383|3215292|3215312|+\n",
      "-gcgcggccgcttcgcctggc\n",
      ">G000299455|4543449|4543753|4543647|4543667|+\n",
      "-gcgtgggcgcttcgcctggc\n",
      ">G000299455|5289627|5289931|5289713|5289733|-\n",
      "agcgtgggcgcttcgcctgg-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (1509064, 1578296)), (1576755, 1576889), '+'), (('G000299455', (2923865, 2925880)), (2924343, 2924477), '+'), (('G000299455', (3040967, 3041101)), (3040967, 3041101), '+')]\n",
      ">G000006925|1509064|1578296|1576755|1576889|+\n",
      "aatgaagtggattgtgattgatacagttatccagccatcatgcggaatatctttttcagt\n",
      "catatggagtaaaataaaattaataatctggtatcaatcggatgctttcttacctcctga\n",
      "aagtatatttacac\n",
      ">G000299455|2923865|2925880|2924343|2924477|+\n",
      "aatgaagtggattgtgattgatacagttatccagccatcatgcggaatatctttttcagt\n",
      "catatggagtaaaataaaattaataatctggtatcaatcggatgctttcttacctcctga\n",
      "aagtatatttacac\n",
      ">G000299455|3040967|3041101|3040967|3041101|+\n",
      "aatgaagtggatagtaattgacacgataattcaaccttcatgcggcatatctttttcagc\n",
      "catatggggtaatatgaaaatgatcatctggtatcaatctactatatttctccctcctgg\n",
      "cagtatatttacac\n",
      "\n",
      "\n",
      "5 [(('G000006925', (1425133, 1465118)), (1452019, 1452578), '-'), (('G000006925', (3246783, 3268347)), (3249812, 3250371), '-'), (('G000299455', (679634, 695969)), (690516, 691075), '+'), (('G000299455', (1078152, 1154001)), (1134500, 1135059), '-'), (('G000299455', (2199778, 2263225)), (2250120, 2250679), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 5\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 4  f\b\b\r",
      "STEP     2 / 4  f\b\b\r",
      "STEP     3 / 4  f\b\b\r",
      "STEP     4 / 4  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\n",
      "Reallocating..done. *alloclen = 1046\n",
      "\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1425133|1465118|1452019|1452578|-\n",
      "gactgggtaaacatgtttgcgctggcagttaacgaagaaaacgccgccggtggtcgtgtg\n",
      "gtaactgcgccaaccaacggtgcctgcggtatcgttccggcagtgctggcttactatgac\n",
      "cactttattgaatcggtcagcccagacatctatacccgttactttatggcagcgggcgcg\n",
      "attggtgcattgtataaaatgaacgcctctatttccggtgcggaagttggttgccagggc\n",
      "gaagtgggtgttgcctgttcaatggctgctgcgggtcttgcagaactgctgggcggtagc\n",
      "ccggaacaggtttgcgtggcggcggaaattggcatggaacacaaccttggtttaacctgc\n",
      "gacccggttgcagggcaggttcaggtgccgtgcattgagcgtaatgccattgcctctgtg\n",
      "aaggcgattaacgccgcacggatggctctgcgccgcaccagtgcaccgcgcgtctcgctg\n",
      "gataaggtcatcgaaacgatgtacgaaaccggtaaggacatgaacgccaaataccgcgaa\n",
      "acctcacgcggtggtctgg-\n",
      ">G000006925|3246783|3268347|3249812|3250371|-\n",
      "gactggatcaacatgtacgcgctggcggttagtgaagaaaacgcagctggcgggcgcgtg\n",
      "gtaacggcaccgactaacggtgcgtgcggcattattccggcagtgctggcttattacgat\n",
      "aagttccgtcgtccggtaaacgagcggtcaattgcccgctattttctggcagcgggggct\n",
      "attggcgcgctgtataaaatgaacgcctccatctctggcgcggaggtcggctgtcagggg\n",
      "gagattggcgtggcctgttcaatggcagcggcagggttaactgaactactgggcggcagt\n",
      "ccggcgcaggtatgcaatgcggcggaaatcgcgatggagcataaccttgggctgacctgc\n",
      "gatccggttgccggacaggtacaaatcccgtgcattgaacgtaatgccattaatgccgtg\n",
      "aaagcagtaaacgccgcgcggatggcgatgcgccgcacctcggcaccgcgtgtttcactc\n",
      "gataaagtgatcgagacgatgtatgaaaccggcaaagatatgaacgataaataccgcgaa\n",
      "acatcacgcggaggactgg-\n",
      ">G000299455|679634|695969|690516|691075|+\n",
      "-actggatcaacatgtacgcgctggcggttagtgaagaaaacgcagctggcgggcgcgtg\n",
      "gtaacggcaccgactaacggtgcgtgcggcattattccggcagtgctggcttattacgat\n",
      "aagttccgtcgtccggtaaacgagcggtcaattgcccgctattttctggcagcgggggct\n",
      "attggcgcgctgtataaaatgaacgcctccatctctggcgcggaggtcggctgtcagggg\n",
      "gagattggcgtggcctgttcaatggcggcggcagggttaactgaactactgggcggcagt\n",
      "ccggcgcaggtatgcaatgcggcggaaatcgcgatggagcataaccttgggctgacctgc\n",
      "gatccggttgccggacaggtacaaatcccgtgcattgaacgtaatgccattaatgccgtg\n",
      "aaagcagtaaacgccgcgcggatggcgatgcgccgcacctcggcaccgcgtgtttcactc\n",
      "gataaagtgatcgagacgatgtatgaaaccggcaaagatatgaacgataaataccgcgaa\n",
      "acatcacgcggaggactggc\n",
      ">G000299455|1078152|1154001|1134500|1135059|-\n",
      "gactggatcaacatgtttgcactggcagtgaacgaagagaacgccgctggcggtcgcgtg\n",
      "gtgactgcgccgactaacggtgcgtgcgggattatcccggctgtgctggcgtactacgat\n",
      "aagtttatccgcgaagtgaacgctaactcactggctcgttacctgctggtagccagtgcc\n",
      "attggttctctttataagatgaacgcgtcaatttctggtgctgaagtcggctgccagggt\n",
      "gaagttggtgtggcgtgctcaatggcggcggctggtctggcagaactgttaggtgcaagc\n",
      "ccggcgcaggtgtgcatcgcggcggaaatcgccatggagcacaacctcggtctgacgtgt\n",
      "gacccggtcgccggacaggtccaggtgccatgcatcgagcgtaacgccattgcggcagta\n",
      "aaagcggtgaacgccgcacgtatggcgctacgccgtaccagcgagccgcgcgtctgcctc\n",
      "gataaagttatcgaaaccatgtacgaaacaggtaaagatatgaacgccaagtaccgcgaa\n",
      "acctctcgcggcggcctgg-\n",
      ">G000299455|2199778|2263225|2250120|2250679|-\n",
      "gactgggtaaacatgtttgcgctggcagttaacgaagaaaacgccgccggtggtcgtgtg\n",
      "gtaactgcgccaaccaacggtgcctgcggtatcgttccggcagtgctggcttactatgac\n",
      "cactttattgaatcggtcagcccggacatctatacccgttactttatggcagcgggcgcg\n",
      "attggtgcattgtataaaatgaacgcctctatttccggtgcggaagttggttgccagggc\n",
      "gaagtgggtgttgcctgttcaatggctgctgcgggtcttgcagaactgctgggcggtagc\n",
      "ccggaacaggtttgcgtggcggcggaaattggcatggaacacaaccttggtttaacctgc\n",
      "gacccggttgcagggcaggttcaggtgccgtgcattgagcgtaatgccattgcctctgtg\n",
      "aaggcgattaacgccgcacggatggctctgcgccgcaccagtgcaccgcgcgtctcgctg\n",
      "gataaggtcatcgaaacgatgtacgaaaccggtaaggacatgaacgccaaataccgcgaa\n",
      "acctcacgcggtggtctgg-\n",
      "\n",
      "\n",
      "5 [(('G000006925', (3083675, 3099254)), (3098275, 3098596), '+'), (('G000299455', (47809, 50193)), (48261, 48582), '-'), (('G000299455', (856582, 857833)), (857240, 857561), '-'), (('G000299455', (3140498, 3142451)), (3140984, 3141305), '-'), (('G000299455', (4771847, 4785261)), (4783911, 4784232), '+')]\n",
      ">G000006925|3083675|3099254|3098275|3098596|+\n",
      "-tatgtctgaaatcacagcctctcgtccggaagtggtcaacgggcatacggacgttatct\n",
      "gctcaaccagtatccgccacattctggctgtacgaaagagtacattgctgcaaatcgaca\n",
      "cacttatccgacaactggctgaagtatcagcaatgacagaaagtattagaggtaaaacgg\n",
      "ctctggactgggccatgaaacaggattttcgctgcggttgctggctgatggagaaaccgg\n",
      "aaaccgcaatgaaagccatcacacgcaatcttgatcgcgaactctggcgtgatctgatgc\n",
      "aacgttccgggatgctgagtct\n",
      ">G000299455|47809|50193|48261|48582|-\n",
      "ttatgtctgcaatcacagtctcccgtccggaagtggtcaacgggcatacggacgttatct\n",
      "gctccacctcagtcagccacattctggctgtgcgaaagactacactgctgcaaatcgaca\n",
      "cacttatccggcaactggctgaaatctcatcaatgacagaaagtattggaggtaaaactt\n",
      "ctctggactgggccatgaaacaggattttcgctgtggctgctggctcatggagaaaccgg\n",
      "aaaccgcaatgaaagccatcacccgcaatcttgaccgcgaaatctggcgtgacctgatgc\n",
      "aacgttccgggatgctgtcat-\n",
      ">G000299455|856582|857833|857240|857561|-\n",
      "taatgtcagacatcacaatctcccgtccggaagtggtcaccgggcatacggacgttatct\n",
      "gctcaaccagtatccgccacattctggctgtacgaaagagtacactgctgcaaatcgaca\n",
      "cacttatccggcaactggctgaaatatcagcaatgacagaaagtattggcggtaaagccg\n",
      "ccccggactgggccatgaaacaggattttcgctgcggctgctggctgatggagaaaccgg\n",
      "aaaccgcaatgaaagccatcacacgcaatctcgatcgcgaaatctggcgtgacctgatgc\n",
      "aacgttcagggatgctttcct-\n",
      ">G000299455|3140498|3142451|3140984|3141305|-\n",
      "ttatgtctgcaatcacagtctcccgtccggaagtggtcaacgggcatacggacgttattt\n",
      "gctcaacctcagtcagccacattctggctgtacgaaagagtacactgttgcaaatcgaca\n",
      "cacttatccgacaactggctgaaatctcggtgctgacggaaagcatcggaggtaaaactg\n",
      "ccccggactgggccatgaaacaggattttcgctgcggctgctggctgatggagaaacccg\n",
      "aaacagcaatgaaagccattacccggaatcttgatcgcgaaatctggcgtgacctgatgc\n",
      "aacgttccgggatgctgtcat-\n",
      ">G000299455|4771847|4785261|4783911|4784232|+\n",
      "-tatgtctgaaatcacagcctctcgtccggaagtggtcaacgggcatacggacgttatct\n",
      "gctcaaccagtatccgccacattctggctgtacgaaagagtacattgctgcaaatcgaca\n",
      "cacttatccgacaactggctgaagtatcagcaatgacagaaagtattagaggtaaaacgg\n",
      "ctctggactgggccatgaaacaggattttcgctgcggttgctggctgatggagaaaccgg\n",
      "aaaccgcaatgaaagccatcacacgcaatcttgatcgcgaactctggcgtgatctgatgc\n",
      "aacgttccgggatgctgagtct\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1696422, 1718400)), (1713369, 1713391), '+'), (('G000006925', (1815406, 1827803)), (1816768, 1816790), '+'), (('G000006925', (2006629, 2008384)), (2008353, 2008375), '+'), (('G000006925', (2560295, 2560325)), (2560303, 2560325), '-'), (('G000006925', (2684711, 2684743)), (2684721, 2684743), '+'), (('G000299455', (2269177, 2270929)), (2269186, 2269208), '-'), (('G000299455', (2737179, 2754578)), (2738895, 2738917), '+'), (('G000299455', (4667815, 4670271)), (4669950, 4669972), '+')]\n",
      ">G000006925|1696422|1718400|1713369|1713391|+\n",
      "-aaggacgaggttttacggcgca\n",
      ">G000006925|1815406|1827803|1816768|1816790|+\n",
      "-aaggatggggttttacggcgca\n",
      ">G000006925|2006629|2008384|2008353|2008375|+\n",
      "-aaggacgggcttttacggtgca\n",
      ">G000006925|2560295|2560325|2560303|2560325|-\n",
      "gaaggacgggattttacggtgt-\n",
      ">G000006925|2684711|2684743|2684721|2684743|+\n",
      "-aaggacggggctttacggcgca\n",
      ">G000299455|2269177|2270929|2269186|2269208|-\n",
      "gaaggacggggttttacggcgc-\n",
      ">G000299455|2737179|2754578|2738895|2738917|+\n",
      "-aaggatggggttttacggcaca\n",
      ">G000299455|4667815|4670271|4669950|4669972|+\n",
      "-aaggacggggttttacggcgca\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1582509, 1591522)), (1585940, 1585964), '+'), (('G000006925', (4367703, 4374506)), (4370435, 4370459), '-'), (('G000299455', (2596243, 2603992)), (2598410, 2598434), '+'), (('G000299455', (4524008, 4535561)), (4531490, 4531514), '-')]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 8\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 7  f\b\b\r",
      "STEP     2 / 7  f\b\b\r",
      "STEP     3 / 7  f\b\b\r",
      "STEP     4 / 7  f\b\b\r",
      "STEP     5 / 7  f\b\b\r",
      "STEP     6 / 7  f\b\b\r",
      "STEP     7 / 7  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 3  f\b\b\r",
      "STEP     2 / 3  f\b\b\r",
      "STEP     3 / 3  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">G000006925|1582509|1591522|1585940|1585964|+\n",
      "-gcaacccggcgttacgtcatcgaa\n",
      ">G000006925|4367703|4374506|4370435|4370459|-\n",
      "agcaacccgtctctgcgtcaccgt-\n",
      ">G000299455|2596243|2603992|2598410|2598434|+\n",
      "-gcaacccggcgttacgtcatcgaa\n",
      ">G000299455|4524008|4535561|4531490|4531514|-\n",
      "agcaacccgtctctgcgtcaccgt-\n",
      "\n",
      "\n",
      "8 [(('G000006925', (1209065, 1211067)), (1210261, 1210352), '+'), (('G000006925', (2756889, 2766574)), (2757342, 2757433), '+'), (('G000006925', (2923039, 2950048)), (2948457, 2948548), '+'), (('G000006925', (3101814, 3103074)), (3102268, 3102359), '+'), (('G000006925', (4746555, 4747128)), (4746680, 4746774), '-'), (('G000006925', (4753880, 4754990)), (4754111, 4754199), '-'), (('G000006925', (4795159, 4795487)), (4795159, 4795239), '+'), (('G000299455', (3226712, 3229721)), (3228911, 3229002), '+')]\n",
      ">G000006925|1209065|1211067|1210261|1210352|+\n",
      "-cgtggtgtcagcggcggacaaggataagcacgcgtcagcagttccgccaacactgcgac\n",
      "agcgttgtcctc---gcggcttttacccggtcaaa\n",
      ">G000006925|2756889|2766574|2757342|2757433|+\n",
      "-cgtggtgtcagcggcggacaaggataagcacgcgtcagcagttccgccaacactgcgac\n",
      "agcgttgtcctc---gcggcttttacccggtcaaa\n",
      ">G000006925|2923039|2950048|2948457|2948548|+\n",
      "-cgtggtgtcagcggcggacaaggataagcacgcgtcagcagttccgccaacactgcgac\n",
      "agcgttgtcctc---gcggcttttacccggtcaaa\n",
      ">G000006925|3101814|3103074|3102268|3102359|+\n",
      "-cgtggtgtcagcggcggacaaggataagcacgcgtcagcagttccgccaacactgcgac\n",
      "agcgttgtcctc---gcggcttttacccggtcaaa\n",
      ">G000006925|4746555|4747128|4746680|4746774|-\n",
      "acgtggtgtcagcgtcggacaggaataagccctcgtcagcagttccgtcagcactgcgac\n",
      "agcgttgtcctcgcggcggcctttacccggtcaa-\n",
      ">G000006925|4753880|4754990|4754111|4754199|-\n",
      "acgtggtgtcagcggcggacaaggataaacacacgtccgcagttccgtcagcactgcgac\n",
      "agcgttgtccgc---aaggcttttacccagt----\n",
      ">G000006925|4795159|4795487|4795159|4795239|+\n",
      "-cgtggtgtcagcggcggacagggataagccctcgtcagcagttccgtcagcactgcgac\n",
      "agcgtt--------------ctttacccggtcaaa\n",
      ">G000299455|3226712|3229721|3228911|3229002|+\n",
      "-cgtggtgtcagcggcggacaaggataagcacgcgtcagcagttccgccaacactgcgac\n",
      "agcgttgtcctc---gcggcttttacccggtcaaa\n",
      "\n",
      "\n",
      "4 [(('G000006925', (1582509, 1591522)), (1586015, 1586058), '+'), (('G000006925', (2242494, 2346249)), (2290593, 2290636), '+'), (('G000299455', (1752647, 1813107)), (1788350, 1788393), '-'), (('G000299455', (2596243, 2603992)), (2598485, 2598528), '+')]\n",
      ">G000006925|1582509|1591522|1586015|1586058|+\n",
      "-attctgttcgctggcatctggcgcatgacagcaagttcgatct-\n",
      ">G000006925|2242494|2346249|2290593|2290636|+\n",
      "-caggtattcgcatacatctggggcgtga-aacggactggtcgtt\n",
      ">G000299455|1752647|1813107|1788350|1788393|-\n",
      "gcaggtattcgcatacatctggggtgcga-aacggactggtcgt-\n",
      ">G000299455|2596243|2603992|2598485|2598528|+\n",
      "-attctgttcgctggcatctggcgcatgacagcaagttcgatct-\n",
      "\n",
      "\n",
      "3 [(('G000006925', (4315026, 4354132)), (4330365, 4330395), '+'), (('G000299455', (2082753, 2082783)), (2082753, 2082783), '+'), (('G000299455', (4865050, 4904182)), (4880414, 4880444), '+')]\n",
      ">G000006925|4315026|4354132|4330365|4330395|+\n",
      "tccacgacctctttttctgcctgatctttc\n",
      ">G000299455|2082753|2082783|2082753|2082783|+\n",
      "tcctctacctctttttctgccagatctttc\n",
      ">G000299455|4865050|4904182|4880414|4880444|+\n",
      "tccacgacctctttttctgcctgatctttc\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "len(mhg_list)\n",
    "for m in mhg_list:\n",
    "    if len(m) > 2:\n",
    "        print(len(m),list(m))\n",
    "        alignBlocksInModule(seqDic, list(m))\n",
    "        command = \"mafft mafftInput.fasta > mafftOutput.fasta\"\n",
    "        os.system(command)\n",
    "        with open('mafftOutput.fasta') as f:\n",
    "            alignment = f.read()\n",
    "        f.close()\n",
    "        print(alignment)\n",
    "        print(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "0237aba4",
   "metadata": {},
   "outputs": [],
   "source": [
    "f = open(\"G000006925,G000299455.txt\",'w')\n",
    "for module in mhg_list:\n",
    "    f.write(str(module)+'\\n')\n",
    "f.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "id": "b7c1db92",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['>G000006925|734392|734467|734392|734467|+', 'tccacgacctctttttctgcctgatctttc', '>G000299455|2082753|2082783|2082753|2082783|+', 'tcctctacctctttttctgccagatctttc', '>G000299455|4865050|4904182|4880414|4880444|+', 'tccacgacctctttttctgcctgatctttc']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "nthread = 0\n",
      "nthreadpair = 0\n",
      "nthreadtb = 0\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "\r",
      "    1 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "\r",
      "    0 / 3\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "\r",
      "STEP     1 / 2  f\b\b\r",
      "STEP     2 / 2  f\b\b\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "0 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-2 (Fast but rough)\n",
      " Progressive method (guide trees were built 2 times.)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "command = \"mafft mafftInput.fasta\"\n",
    "process = subprocess.Popen(shlex.split(command), stdout=subprocess.PIPE)\n",
    "output, error = process.communicate()\n",
    "entryList = str(output, 'utf-8').split('\\n')[:-1]\n",
    "print(entryList)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
